# Genomic variations in the MIKK panel {#MIKK-genomes-chap}



## The Medaka Inbred Kiyosu-Karlruhe (MIKK) panel^[This chapter sets out my contributions to @fitzgeraldMedakaInbredKiyosuKarlsruhe2022 and @legerGenomicVariationsEpigenomic2022, which were published as companion papers in the journal *Genome Biology* to introduce the MIKK panel to the scientific community, and describe the genetic characteristics of the MIKK panel that would make it a useful resource for other researchers who wish to explore the genetics of quantitative traits in vertebrates. I am joint-first author on both papers.] 

I introduced the MIKK panel in section \@ref(MIKK-background) of the previous chapter, and here I describe several of its genomic and phenotypic characteristics which I determined are relevant to its utility for the genetic mapping of complex traits. These include a visualisation of the inbreeding trajectory of the panel (section \@ref(inbreeding-sec)), an exploration of the evolutionary history of the MIKK panel's founding population (section \@ref(introgression-sec)), measurement of the levels of homozygosity across the panel (section \@ref(nuc-div)), an assessment of the panel's allele-frequency distribution and rate of linkage disequilibrium (LD) decay (section \@ref(ld-decay-sec)), and a characterisation of the structural variants present in a smaller sample of lines using Oxford Nanopore long-read sequencing data (section \@ref(mikksv-sec)).

## Genomic characterisation of the MIKK panel

### MIKK panel DNA sequence dataset {#non-sib-calls}

In 2018, our collaborators sequenced 79 of the 80 extant MIKK panel lines -- together with several wild Kiyosu samples and individuals from the established *iCab* medaka strain -- from male brain samples using Illumina short-read sequencing technology. Tomas Fitzgerald from the Birney Group at EMBL-EBI then aligned these sequences to the *HdrR* medaka reference and called variants to produce the **MIKK Illumina call set**, in the form of a .vcf file containing single nucleotide polymorphism (SNP) and small insertion-deletion (INDEL) calls for each line. To avoid allele frequency biases introduced by the 16 pairs/triplets of "sibling lines" (see \@ref(inbreeding-sec)), I removed each pairâ€™s arbitrarily-labelled second sibling line from the variant call set, leaving 63 MIKK panel lines (**MIKK non-sibling call set**), and used only those calls for the analyses in sections \@ref(nuc-div) and \@ref(ld-decay-sec) below.

I also performed a pilot study on 9 MIKK panel lines using Oxford Nanopore Technologies (**ONT**) long-read sequence data. Adrien Leger, then of the Birney Group at EMBL-EBI, aligned these sequences to the *HdrR* medaka reference [@yatesEnsembl20202020], and called variants to produce the **MIKK ONT call set** in the form of a .vcf file containing structural variants calls for each line with tags for insertions (INS), deletions (DEL), duplications (SUP), inversions (INV) and translocations (TRA). In the work described below, I used these variant call sets as the primary datasets.

### Assessing the inbreeding trajectory of the MIKK panel {#inbreeding-sec}

The MIKK panel was bred from a wild population of medaka found in the Kiyosu area near Toyohashi, Aichi Prefecture, in southern Japan [@spivakovGenomicPhenotypicCharacterization2014]. From this wild population, the Loosli Group at KIT set up random crosses of single mating pairs to create 115 'founder families'. For each founder family, they then set up between two and five single full-sibling-pair inbreeding crosses, which resulted in 253 F1 lines. Lines derived from the same founder family are referred to as 'sibling lines'. Over the course of the next eight generations of inbreeding, they used only one mating pair per line. I generated **Figure \@ref(fig:InbreedingFigure)A** and **B** from the inbreeding data provided by the Loosli Group. **Figure \@ref(fig:InbreedingFigure)A** shows the number of lines that survived over the course of the first 14 generations of the inbreeding program, and the various causes for the termination of other lines. **Figure \@ref(fig:InbreedingFigure)B** shows the average fecundity levels of the surviving lines at generation F~16~. 

(ref:InbreedingFigure) Inbreeding, fecundity and eye size in the MIKK panel lines. **A**: Status of all MIKK panel lines during the first 14 generations of inbreeding, showing cause of death for non-extant lines. **B**: Average fecundity of MIKK panel lines in generation F~16~, as measured during peak egg production in July 2020. **C**: Distribution of mean relative eye size for female and male medaka across all MIKK panel lines.

\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_genome/20211213_final_figure} \caption{(ref:InbreedingFigure)}(\#fig:InbreedingFigure)
\end{figure}



In addition, the Birney Group at EMBL-EBI generated morphometric data for the MIKK panel lines to demonstrate the distribution of physical phenotypes across the MIKK panel. I used this data on relative eye diameters to generate **Fig. \@ref(fig:InbreedingFigure)C**, where the trait appears to have a roughly normal distribution, with minimal differences in the mean between sexes. I also computed this trait's broad-sense heritability $\eta^2_{(partial)} = \hat{H^2} = SS_{line} / (SS_{line} + SS_{residuals}) =$ 0.56. This value is on the higher range of heritability estimates for morphological traits across different species inluding mouse and human [@visscherHeritabilityGenomicsEra2008; @claesGenomewideMappingGlobaltolocal2018; @weinbergCraniofacialGeneticsWhere2018], and indicates that the majority of variance observed across the panel is driven by differences between lines in their genetics. This suggests the MIKK panel will be a promising resource for discovering the genetic variants that are responsible for such differences. 

### Levels of homozygosity

To assess whether the inbreeding process resulted in high levels of homozygosity in the MIKK panel, for each line I extracted from the MIKK Illumina call set all non-missing, biallelic calls (both SNPs and indels) with *bcftools* [@danecekTwelveYearsSAMtools2021], counted the number of SNPs within non-overlapping, 5-kb bins, and calculated the proportion of SNPs within each bin that were homozgyous. To provide an example of the levels of homozygosity in the MIKK panel, **Figure \@ref(fig:circos-55-2)** is a circos plot generated with circlize [@guCirclizeImplementsEnhances2014] for the MIKK panel line 55-2, featuring the proportion of homozygous SNPs per 5-kb bin (blue), and the total number of SNPs in each bin (yellow). This plot is representative of the plots generated for all MIKK panel lines. Almost all 5-kb bins across the genome contain a very high proportion of homozygous variants, and all other MIKK panel lines show similar levels of homozygosity. Although there are more sophisticated means of measuring homozygosity -- for example, by using a Hidden Markov Model to separate out regions of residual homozygosity^[As described by @fitzgeraldMedakaInbredKiyosuKarlsruhe2022.] -- this confirms that the breeding program was successful in establishing a panel of inbred lines with levels of homozygosity comparable to established medaka strains that have been inbred for dozens of generations, such as the *iCab* strain at generation F~24~ which was sequenced with the MIKK panel  (**Figure \@ref(fig:circos-iCab-F24)**).

(ref:circos-55-2) Proportion of homozygous SNPs within 5 kb bins in the MIKK panel line 55-2 (*blue*), and number of SNPs in each bin (*yellow*).

\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_genome/circos/55-2} \caption{(ref:circos-55-2)}(\#fig:circos-55-2)
\end{figure}

(ref:circos-iCab-F24) Proportion of homozygous SNPs within 5 kb bins in the previously established *iCab* strain at generation F~24~ (dark grey), and number of SNPs in each bin (yellow).

\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_genome/circos/iCab_F24} \caption{(ref:circos-iCab-F24)}(\#fig:circos-iCab-F24)
\end{figure}

Medaka has a male heterogametic XX-XY sex determination system, but because the sex chromosomes are karyotypically similar, they have traditionally been assigned the label of "chromosome 1" [@fitzgeraldMedakaInbredKiyosuKarlsruhe2022]. The authors there identified the sex determination region as residing within the locus of 1:15.9-16.5 Mb. As the individuals who were sequenced were male and therefore heterozygous at that locus, one would expect to observe a sharp decrease in homozygosity around that region. However, in this figure -- which was generated from the same data as @fitzgeraldMedakaInbredKiyosuKarlsruhe2022 -- such a narrow region is barely visible due to both the thickness of the plot lines, and the non-uniformity of low-homozygosity across that region.

### Nucleotide diversity {#nuc-div}

I also sought to assess the levels of genetic diversity in the MIKK panel, and to compare them with the levels of genetic diversity present in the wild Kiyosu population. If the levels were similar, it would confirm that the MIKK panel provided a reasonable representation of the variation present in its originating population, and that any selection for loci associated with domestication was not widespread across the genome. As a means of assessing genetic diversity in the MIKK panel, I calculated the widely-used measure of 'nucleotide diversity' ($\hat{\pi}$) [@neiMathematicalModelStudying1979] within 500-kb non-overlapping windows across the genome of the 63 lines in the MIKK non-sibling call set (see \@ref(non-sib-calls)), and compared this to the nucleotide diversity in 7 wild medaka from the same Kiyosu population from which the MIKK panel was derived. Mean and median nucleotide diversity in both the MIKK panel and wild Kiyosu medaka were similar, and slightly higher in the MIKK panel (mean: MIKK = 0.0038, wild = 0.0037; median: MIKK = 0.0033, wild = 0.0031). This may have been caused by the smaller sample size of wild Kiyosu medaka, as mean $\hat{\pi}$ for a random sample of 7 MIKK lines dropped to 0.0037, although it is unexpected that the highly-inbred MIKK lines have similar levels of $\hat{\pi}$ to wild Kiyosu individuals. The patterns of varying nucleotide diversity across the genome are shared between the MIKK panel and wild Kiyosu medaka, where regions with high levels of repeat content tend to have higher nucleotide diversity (r = 0.386, $p$ \< 0.001) (**Fig. \@ref(fig:NucleotideDiversity)**). I also calculated $\hat{\pi}$ for each line individually, and as expected, levels of $\hat{\pi}$ around the (XX/XY) sex determination region of 1:~16-17 Mb are elevated in all lines relative to the consistently low levels found in most other chromosomes.

(ref:NucleotideDiversity) Circos plot with nucleotide diversity ($\hat{\pi}$) calculated within 500-kb non-overlapping windows for 63 non-sibling lines from the MIKK panel (*green*) and 7 wild Kiyosu medaka samples from the same originating population (*purple*); proportion of sequence classified as repeats by RepeatMasker (*blue*); mean mapping quality (*pink*); and sex determination locus (*red arrow*).

\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_genome/supp_01_pi_circos} \caption{(ref:NucleotideDiversity)}(\#fig:NucleotideDiversity)
\end{figure}

As expected, there is a peak of $\hat{\pi}$ around the sex determination locus (as indicated by the red arrow). However, there are far higher peaks on several chromosomes -- most notably for chromosomes 2, 11, and 18 -- which correspond closely to the regions I identified as containing large (>250 kb) inversions that appear to be shared across at least some of the MIKK panel (**Fig. \@ref(fig:SVInvs)**). These regions are also enriched for large deletions and duplications.[@legerGenomicVariationsEpigenomic2022] Inversions cause permanent heterozygosity [@hoffmannChromosomalInversionPolymorphisms2004], and duplications and deletions may have increased the density of called SNPs in these regions [@fredmanComplexSNPrelatedSequence2004], so the observed depressions in homozygosity at these loci may be the result of such large structural variants that are present in the MIKK panel's genomes. Overall, this analysis confirms that the MIKK panel shows similar levels of homozygosity compared to classical laboratory inbred medaka strains, with the additional complication of structural variation and inversion.

(ref:SVInvs) Inversions identified in 9 MIKK panel lines using a combination of ONT long-read and Illumina short-read sequences (see Chapter \@ref(mikksv-sec) below).

\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_genome/20210224_sv_invs_lines} \caption{(ref:SVInvs)}(\#fig:SVInvs)
\end{figure}

### Introgression with northern Japanese and eastern Korean medaka populations {#introgression-sec}

There is an open question as to whether the Northern and Southern Japanese medaka populations have been able to interbreed since being separated on either side of the Hida and Akaishi Mountains (commonly known in the West as the 'Japanese Alps') [@naruseMedakaModelOrganogenesis2011a; @spivakovGenomicPhenotypicCharacterization2014]. I therefore sought to determine whether there was evidence of 'introgression' -- that is, the transfer of genes through interbreeding -- between the southern Japanese, northern Japanese and eastern Korean medaka populations. To this end, I used the 50-fish multiple alignment from Ensembl release 102 to obtain the aligned genome sequences for the established medaka inbred lines *HdrR* (southern Japan), *HNI* (northern Japan), and *HSOK* (eastern Korea), as well as the most recent common ancestor of all three strains [@howeEnsembl20212021] (**Figure \@ref(fig:ABBABABA)A**). Using the phylogenetic tree provided with the dataset and the *ape* R package [@paradisApeEnvironmentModern2019], I identified the ancestral base of those three strains. For each locus with a non-missing base for *HdrR*, I assigned the allele in that ancestral base as the 'ancestral' allele, and the alternative allele as the 'derived' allele, and then combined that dataset with the MIKK Illumina call set and variant calls for the southern Japanese *iCab* strain (see \@ref(non-sib-calls)). 

I chose to use the ABBA-BABA test to explore the levels of introgression between the southern Japanese Kiyosu population from which the MIKK panel was derived, and its ancestrally-related northern Japanese and eastern Korean medaka populations. The ABBA-BABA test is used to measure the proportion of shared genome between populations, and was initially developed to determine whether humans had interbred with neanderthals [@durandTestingAncientAdmixture2011]. It involves comparing three ingroup populations, denoted as P~1~, P~2~ and P~3~, with an outgroup population, denoted as O, which is intended to define the ancestral state. The populations are presumed to have the evolutionary relationship (((P~1~, P~2~), P~3~), O). The letters in *ABBA-BABA* represent the alleles possessed respectively by those populations. Therefore *ABBA* represents a locus where P~1~ and O share the *A* allele, and P~2~ and P~3~ share the *B* allele. Likewise, *BABA* represents a locus where P~1~ and P~3~ share the *B* allele, and P~2~ and O share the *A* allele. In circumstances where the populations have not significantly interbred after deviation,^[Assuming there are no large deviations created by recurrent mutations, ancestral population structure, or variable substitution rates [@durandTestingAncientAdmixture2011].] the ratio of *ABBA* to *BABA* sites is expected to be equal. On the other hand, interbreeding between  P~2~ and P~3~ would create a statistically significant deviation from this ratio. 

Using the scripts provided by the first author of @martinEvaluatingUseABBA2015 on their GitHub page [@martinSimonhmartinGenomicsGeneral2022], I applied this ABBA-BABA test to measure of the proportion of shared genome in 500-kb sliding windows between the MIKK panel and either *iCab*, *HNI*, or *HSOK* (**Figure \@ref(fig:ABBABABA)**), with a modified 'admixture proportion' statistic $\hat{f}_d$ [@martinEvaluatingUseABBA2015]. Like the inbred lines of the MIKK panel, the *iCab* strain is derived from a southern medaka population, and represents a population for which I expect a substantial mixing between the two. The focus of the exploration here is rather on the degrees of mixing between the MIKK panel and either the *HNI* strain from northern Japan, and the *HSOK* strain from eastern Korea. 

(ref:ABBABABA) **Figure 2**: ABBA-BABA analysis. **A**. Phylogenetic tree generated from the Ensembl release 102 50-fish multiple alignment, showing only the medaka lines used in the ABBA-BABA analysis. **B**. Schema of the comparisons carried out in the ABBA-BABA analysis. **C**. Circos plot comparing introgression ($\hat{f}_d$) between the MIKK panel and either *iCab* (yellow), *HNI* (orange), or *HSOK* (purple), calculated within 500-kb sliding windows using a minimum of 250 SNPs per window.

\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_genome/07_introgression} \caption{(ref:ABBABABA)}(\#fig:ABBABABA)
\end{figure}

Based on the genome-wide mean $\hat{f}_d$, the MIKK panel shares approximately 25% of its genome with *iCab*, 9% with *HNI*, and 12% with *HSOK*. These results provide evidence that the MIKK panel's originating population has more recently interbred with medaka from eastern Korea than with medaka from northern Japan, which is surprising given Japan and Korea are separated by the Korean Strait, whereas the founding populations of the MIKK panel and the *HNI* strain are separated by a "mere" mountain range. This supports the findings in @spivakovGenomicPhenotypicCharacterization2014, where the authors found little evidence of significant interbreeding between southern and northern Japanese medaka since the populations diverged. Although the proportional difference between *HNI* and *HSOK* is small, this further supports the general finding that northern and southern Japanese medaka strains show low levels of interbreeding that may be a result of geographical isolation or genome divergence through a process of speciation [@katsumuraMedakaPopulationGenome2019].

### LD decay {#ld-decay-sec}

Linkage disequilibrium (**LD**) describes the "non-independent association of two alleles in a population" [@uffelmannGenomewideAssociationStudies2021]. It usually occurs between loci that are physically proximate to one another on the chromosome, as a recombination event is less likely to occur between them, causing a tendency for them to be inherited together. Genetic mapping requires variants to segregate in order to determine how they differ in their effects, and so regions with high LD make it difficult to identify the causal variants that reside in those regions, because they tend to be inherited together. If the MIKK panel's LD structure was relatively low compared to humans, it would indicate that one would be able to map variants at a higher resolution, in turn making it easier to identify variants that cause differences in traits of interest. For this purpose I analysed the MIKK panel's allele frequency distribution and LD structure to assess their likely effects on genetic mapping. To remove allele-frequency biases introduced by the presence of sibling lines in the MIKK panel, I used only the MIKK non-sibling call set (see section \@ref(non-sib-calls) above). 

To assess how accurately one may be able to map genetic variants using the MIKK panel relative to a human dataset, I compared the MIKK panelâ€™s minor allele frequency (MAF) distribution and LD structure against that of the 2,504 humans in the 1KG Phase 3 release.[@10002015global] To prepare the **"1KG call set"**, I first downloaded the .vcf files for each autosome from the projectâ€™s FTP site, then merged them into a single VCF using GATK [@mckennaGenomeAnalysisToolkit2010]. I then used PLINK [@changSecondgenerationPLINKRising2015; @purcellPLINK] to calculate the minor allele frequencies for all non-missing, biallelic SNPs in both the MIKK non-sibling and 1KG call sets (*N* SNPs = 16,395,558 and 81,042,381 respectively) (**Fig. \@ref(fig:LDdecay)A**). As expected, the 1KG and MIKK panel calls are similarly enriched for low-frequency variants, albeit to a lesser extent in the MIKK panel, which is expected due to its smaller sample size.

To determine the rate of LD decay in the MIKK panel and compare it to that in the 1KG sample, for both the MIKK non-sibling and 1KG call sets, I used PLINK to compute $r^2$ on each autosome for all pairs of non-missing, biallelic SNPs with MAF $>$ 0.10 within 10 kb of one another (for 1KG and the MIKK panel respectively ~ 5.5M and ~ 3M SNPs, with a total number of pairwise $r^2$ observations of 204,152,922 and 146,785,673). I then grouped the $r^2$ observations for each pair of SNPs based on their distance from one another into non-overlapping bins of 100 bp in length, and calculated the mean $r^2$ in each of those bins to generate **Fig. \@ref(fig:LDdecay)B** using the mean $r^2$ and left boundary of each bin.

(ref:LDdecay) Minor allele frequency distributions and LD decay for biallelic, non-missing SNPs in the 1000 Genomes Phase 3 variant calls (N = 2,504) (1KG), and the MIKK panel Illumina-based calls excluding one of each pair of sibling lines (N = 63), across all autosomes (1KG: chrs 1-22; MIKK: chrs 1-24). **A**: Histogram of allele frequencies in the 1KG and MIKK panel calls. **B**: LD decay for each autosome, calculated by taking the mean $r^2$ of pairs of SNPs with MAF $>$ 0.1 within non-overlapping 100 bp windows of distance from one another, up to a maximum of 10 kb. LD decays faster on chromosome 2 for the MIKK panel due to its higher recombination rate.

\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_genome/08_ld_decay} \caption{(ref:LDdecay)}(\#fig:LDdecay)
\end{figure}

Based on the 1KG calls under these parameters, LD decays in humans to a mean $r^2$ of around 0.2-0.35 at a distance of 10 kb, whereas the MIKK panel reaches this level within 1 kb, with a mean $r^2$ of 0.3-0.4 at a distance of ~100 bp. This implies that when a causal variant is present in at least two lines in the MIKK panel, one may be able to map causal variants at a higher resolution than in humans. I note that LD decays faster in chromosome 2 of the MIKK panel relative to the other chromosomes. This suggests that it has a much higher recombination rate, which is consistent with the linkage map described in @naruseDetailedLinkageMap2000, showing a higher genetic distance per Mb for this chromosome. This higher recombination rate in chromosome 2 may in turn be caused by its relatively high proportion of repeat content (**Fig. \@ref(fig:repeats)A**).

(ref:repeats) Repeat content in the *HdrR* genome based on RepeatMasker results obtained by Jack Monahan. **A**. Proportion of repeat content per-chromosome. **B**. log10 of repeat lengths and counts per repeat class. "Misc" includes all repeats assigned to their own specific class, for example "(GAG)n" or "(GATCCA)n". **C**.  Circos plot showing repeat length (radial axes) by locus (angular axis) and repeat class (track). 

\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_genome/20210401_hdrr_repeats_final} \caption{(ref:repeats)}(\#fig:repeats)
\end{figure}

## Structural variation in the MIKK panel {#mikksv-sec}

As an alternative to the variation pangenome approach described in @legerGenomicVariationsEpigenomic2022, I explored the structural variants (SVs) present in 9 of the MIKK panel lines in a reference-anchored manner, similar to many human studies. Differences in SVs between panel lines is another important class of genetic variation that could cause or contribute to significant phenotypic differences. Here I used ONT long-read data obtained for 9 of the 12 selected lines, allowing me to characterise larger SVs in the MIKK panel and to create a more extensive picture of genomic rearrangements compared to available medaka reference genomes. Adrien Leger from the Birney Group at EMBL-EBI first called structural variants using only the ONT long reads, producing a set of structural variants classified into five types: deletions (**DEL**), insertions (**INS**), translocations (**TRA**), duplications (**DUP**) and inversions (**INV**). I then "polished" the called DEL and INS variants with Illumina short reads to improve their accuracy, using the software *SViper* v2.0.0 [@mehringerSmehringerSViper2017]. The polishing process filtered out 7.4% of DEL and 12.8% of INS variants, and adjusted the breakpoints (i.e. start and end positions) for 75-77% of DEL and INS variants in each sample by a mean of 23 bp for the start position, and 33 bp for the end position. This process produced a total of 143,326 filtered SVs.

The 9 "polished" samples contained a mean per-sample count of approximately 37K DEL variants (12% singletons), 29.5K INS variants (14%), 3.5K TRA variants (9%), 2.5K DUP (7%) and 600 INV (7%) (**Fig. \@ref(fig:SV-main)D**). DEL variants were up to 494 kb in length, with 90% of unique DEL variants shorter than 3.8 kb. INS variants were only up to 13.8 kb in length -- as limited by the ONT read lengths -- with 90% of unique INS variants shorter than 2 kb. DUP and INV variants tended to be longer, with a mean length of 19 and 70.5 kb respectively (**Fig. \@ref(fig:SV-main)A**). **Fig. \@ref(fig:SV-main)E** shows the per-sample distribution of DEL variants across the genome. Most large DEL variants over 250 kb in length were common among the MIKK panel lines. A number of large DEL variants appear to have accumulated within the 0-10 Mb region of chromosome 2, which is enriched for repeats in the HdrR reference genome (**Fig. \@ref(fig:repeats)**).

SVs were generally enriched in regions covered by repeats. While only 16% of bases in the *HdrR* reference were classified as repeats (irrespective of strand), those bases overlapped with 72% of DEL, 63% of DUP, 81% of INV and 35% of TRA variant regions. However, repeat bases only overlapped with 21% of INS variants. I also assessed each SV's probability of Loss-of-function Intolerance (**pLI**) [@lekAnalysisProteincodingGenetic2016] by calculating the logarithm of odds (LOD) for the pLI scores of all human orthologous genes overlapping the variant (based on the Exome Aggregation Consortium (ExAC) database [@fullerMeasuringIntoleranceMutation2019]) (**Fig. \@ref(fig:SV-main)B,C**). 30,357 out of 134,088 DEL, INS, DUP and INV variants overlapped at least one gene, and 9% of those had a score greater than 10, indicating a high probability that the SV would cause a loss of function. Two INS variants on chr2 had an outlying LOD score of 57 as a result of overlapping medaka gene ENSORLG00000003411, which has a pLI score of 1 -- the highest intolerance to variants causing a loss of function. This gene is homologous with human genes *SCN1A*, *SCN2A* and *SCN3A*, which encode sodium channels and have been associated with neuronal and sleep disorders. I did not find evidence that longer SVs tended to have a higher probability of causing a loss of function (**Fig. \@ref(fig:SV-main)B**).

(ref:SV-cap) Polished SVs in 9 MIKK panel lines sequenced with ONT. DEL: deletion; INS: insertion; TRA: translocation; DUP: duplication; INV: inversion. **A**. Aggregate log10 counts and lengths of distinct SVs by type, excluding TRA. **B**. pLI LOD scores in distinct SVs by SV type. **C**. Histogram of LOD scores by SV type. **D**. Total and singleton counts of SV types per sample. **E**. Circos plot showing per-sample distribution and lengths of DEL variants across the genome.


\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_genome/20210408_sv_main} \caption{(ref:SV-cap)}(\#fig:SV-main)
\end{figure}

A completely different approach to analysing these ONT long-read data was to generate independent assemblies of each of the samples, and then construct a consolidated graph genome. This was performed by Adrien Leger, as described in @legerGenomicVariationsEpigenomic2021. I compared these reference-anchored, polished INS and DEL calls with the high-quality graph-based alternative paths and large-scale deletions, respectively. I found that 2 of the 19 regions covered by graph-based alternative paths, and 4 of the 16 regions covered by graph-based deletions, had no SVs that overlapped those regions at all, which suggests they would have been missed entirely when using a reference-anchored approach alone.

With the exception of one alternative path on chromosome 20, the alternative paths were not captured by INS variants, which only covered up to 63% of the bases in each region, and in many cases substantially less. On the other hand, for 8 of the 16 graph-based deletions, the DEL variants covered at least 85% of the bases in those regions. The other 8 graph-based deletions were either not at all covered by DEL variants, or only slightly. This indicates, as expected, that the reference-based approach is better at detecting large-scale deletions than alternative paths ("insertions"), but still misses around half of such variants relative to the graph-based approach.
