# Variation in the frequency of trait-associated alleles across global human populations

The genetic component of human traits varies between groups of people across the globe due to differences in their allele frequencies. In this study we use the well-established fixation index measure ($F_{ST}$) with the 1000 Genomes Project phase 3 dataset to systematically quantify the variation between human populations of the allele frequencies of SNPs associated with traits from the GWAS Catalog. We generate a simple null model of these FST measures by comparing them to randomly-selected SNPs matched by allele frequencies in the European (EUR) population. We find many common traits such as height and body mass index show little difference in their FST distributions relative to randomly-selected SNPs, whereas traits such as skin pigmentation, hair shape, and eye colour show striking differences. Traits related to educational attainment (EA) show low differences in FST levels, similar to those observed for body height and weight. We conclude that variation in the genetics of traits related to EA is caused by variation in the ancestral human population, and is not likely to substantially contribute to differences in EA between groups of people, including commonly used groupings by self-identified ethnicity. 

## Background

Traits can vary between individuals and populations for a variety of reasons. In many cases, variation in the environment may be the most important factor, but genetic drift and differential selection (meaning selection acting to a greater degree in one population than another since their divergence) can also contribute to a difference in the mean value of a trait between populations.1 Both these processes can lead to differences in allele frequencies between populations (barring demographic phenomena such as migration), but manifest in different ways, allowing us in principle to distinguish between their effects. For example, it can be shown that at loci associated with a trait under differential selection, one can expect greater variance in the fixation index (FST) – a statistic which quantifies the relative variance in allele frequency between groups – than at equivalent neutral loci.2 In 1973, Lewontin & Krakauer proposed a test for selection based on this principle, which they applied to a limited dataset of human genetic polymorphisms.3 Subsequent studies using larger datasets have extended this approach, for example by enabling the comparison to be made to empirical genome-wide FST distributions rather than theoretical expectations, and other refinements to account for the demographic history of the populations compared.4–8
Early studies focused on relatively simple polymorphisms, but more recently studies have investigated selection on traits with complex polygenic architecture, taking advantage of large-scale genome sequencing and the growing catalogue of GWAS results for thousands of traits.9,10 GWAS results provide, for a given trait and in the cohort studied, an estimate of the additive effect on that trait of many loci genome-wide. In combination with models of ancestral demography, they can thus be used to infer correlated changes in allele frequency expected under selection, and an application of a generalised version of the FST comparison was developed and applied in this context by Berg & Coop in 2014.8 Studies such as this and others have found evidence for differential selection on traits such as height and skin pigmentation in the populations analysed.11,12

Here we illustrate the distribution of FST scores for loci associated with 587 traits, a subset of the GWAS Catalog which passed our criteria for suitable polygenic traits (Methods). Our analysis is empirical, in that we do not explicitly formulate a statistical test for drift or selection, differential or otherwise. Using high-coverage sequence data for 2,504 individuals from the 1000 Genomes Project phase 3 release, for each trait in the GWAS Catalog we calculated the distribution of FST across all approximately-unlinked SNPs associated with it (trait SNPs), and compared these FST distributions with the FST distributions of random-selected SNPs that were matched to the trait SNPs by their allele frequencies in European populations (control SNPs). Our results show that traits related to the physical correlates of “race” (such as skin-pigmentation, eye colour, and hair shape) tend to have relatively high FST values – signifying relatively high variance in allele frequencies between populations – whereas traits related to intelligence (such as self-reported EA, mathematical ability, and cognitive function measurement) tend to have lower FST values that are similar to those of most polygenic traits such as height and body mass index.

## Methods

Methods
The code and pipeline used for this analysis is set out in full in the notebooks and linked repository available here: https://brettellebi.github.io/human_traits_fst/. The analyses were performed with R version 4.1.0,26 Snakemake,27 and several R packages in addition to those cited below.28–34
1000 Genomes dataset 
As the reference for human genomic variation across diverse populations, we used the New York Genome Center high-coverage, phased .vcf files35 for the 2,504 individuals described in the 1000 Genomes phase 3 release.36 We then annotated those .vcf files with human SNP IDs from dbSNP release 9606.37
GWAS Catalog
We used the R package gwasrapidd38 to query all traits in the GWAS Catalog39 (NTRAITS = 3,459) as of 9 August 2021. For 541 of these traits, no matching variant IDs could be pulled out from the 1000 Genomes VCFs, leaving NTRAITS = 3,008. 
Linkage disequilibrium
To obtain our “trait SNP” dataset, for each trait, we sought to isolate the SNP closest to each of its true causal variants, and exclude the SNPs in linkage disequilibrium (LD) with them. To this end, we used PLINK 1.940,41 to “clump” the SNPs associated with each of the remaining 3,008 traits, using an index variant p-value threshold of 10-8,42 r2 threshold of 0.1,43 and base window size of 1 Mb. This process left us with 2,045 traits with at least one index SNP that met the p-value threshold. The index SNPs for each trait formed our set of trait SNPs. In order to target relatively polygenic traits, we further filtered out traits with fewer than 10 trait SNPs, leaving NTRAITS = 587. Supplementary Fig. 4 shows the counts of unique SNP IDs associated with each trait before and after clumping, and an interactive version is available in the notebook cited above. 
Control SNPs
To obtain our “control SNP” dataset, we assigned each trait SNP to one of 20 bins based on its minor allele frequency in European populations (as provided in the original 1000 Genomes .vcf files under the column header ‘INFO/AC_EUR’). For example, if a trait SNP had a minor allele frequency of 0.08 in European populations, it was assigned to the (0.05, 0.1] bin. We did the same for all (unassociated) SNPs in the .vcf files, then paired each trait SNP with a random SNP from the .vcf file in the equivalent bin. These allele-frequency-paired random SNPs formed our set of “control SNPs”, which we used to infer the FST distribution of a random set of SNPs with the same allele frequencies as the trait SNPs, and against which we could compare the FST  distribution of the trait SNPs.
FST and ranking traits by signed Kolmogorov-Smirnov D statistic
We then calculated FST  for each of the trait SNPs and their matched control SNPs using the Weir and Cockerham method,44 as implemented in the R package pegas.45 To rank all traits based on the directional difference in FST distributions between trait and control SNPs, we ran three Kolmogorov-Smirnov (KS) tests for each trait t with xt = FST, trait SNPs and yt = FST, control SNPs : 1) two-sided (Dt) ; 2) one-sided “greater” (Dt+) ; and 3) one-sided “less” (Dt–). We note that Dt  = max(Dt+, Dt–), where  Dt+ is the greatest vertical distance attained by the eCDF of xt over the eCDF of yt, and Dt– is the greatest vertical distance attained by the eCDF of yt over the eCDF of xt.46,47 Accordingly, we used a comparison of Dt+ and Dt– to created a signed D statistic (DtS), based on the logic that trait SNPs with a lower overall FST than control SNPs tend to have a higher D under the “greater” test than the “less” test, and vice versa. 
Therefore, DtS:
Dt+ >Dt–:       –Dt
Dt+ =Dt–:        0
Dt+ <Dt–:          Dt

This allowed us to rank the traits based on the direction and size of the distance between the FST distributions for trait and control SNPs, from large negative DtS (FST for traits SNPs is lower overall than control SNPs) to large positive DtS (FST for trait SNPs is higher overall than control SNPs). We then plotted the distributions and eCDFs of hits and controls for each trait using the R packages tidyverse,48 ggridges,49 cowplot,50 and plotly.51


## Results

In Fig. \@ref(fig:FstMain) we present the FST distributions of trait SNPs for an illustrative subset of 28 human traits, ranked by the signed Kolmogorov-Smirnov test statistic D when compared with their matched control SNPs (Methods). Fig. 1A shows the densities of SNPs as a function of FST , and Figs. 1B and C show their empirical Cumulative Distribution Functions (eCDFs). Fig. 1B includes the eCDFs of control SNPs in grey. eCDF figures for all 587 traits that passed our filters (Methods) are provided in Supplementary Fig. 1.

```{r}

```


