<!DOCTYPE html>
<html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<title>Chapter 3 Cross-references | Genetic analysis of quantitative traits in medaka fish and humans</title>
<meta name="author" content="Ian Brettell">
<meta name="description" content="Cross-references make it easier for your readers to find and link to elements in your book.  3.1 Chapters and sub-chapters There are two steps to cross-reference any heading: Label the heading: #...">
<meta name="generator" content="bookdown 0.26 with bs4_book()">
<meta property="og:title" content="Chapter 3 Cross-references | Genetic analysis of quantitative traits in medaka fish and humans">
<meta property="og:type" content="book">
<meta property="og:description" content="Cross-references make it easier for your readers to find and link to elements in your book.  3.1 Chapters and sub-chapters There are two steps to cross-reference any heading: Label the heading: #...">
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="Chapter 3 Cross-references | Genetic analysis of quantitative traits in medaka fish and humans">
<meta name="twitter:description" content="Cross-references make it easier for your readers to find and link to elements in your book.  3.1 Chapters and sub-chapters There are two steps to cross-reference any heading: Label the heading: #...">
<!-- JS --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://kit.fontawesome.com/6ecbd6c532.js" crossorigin="anonymous"></script><script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="libs/bootstrap-4.6.0/bootstrap.min.css" rel="stylesheet">
<script src="libs/bootstrap-4.6.0/bootstrap.bundle.min.js"></script><script src="libs/bs3compat-0.3.1/transition.js"></script><script src="libs/bs3compat-0.3.1/tabs.js"></script><script src="libs/bs3compat-0.3.1/bs3compat.js"></script><link href="libs/bs4_book-1.0.0/bs4_book.css" rel="stylesheet">
<script src="libs/bs4_book-1.0.0/bs4_book.js"></script><style>
    #content {
        font-size: 0.9rem;
    }
    </style>
<script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- CSS --><style type="text/css">
    
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  </style>
<style type="text/css">
    /* Used with Pandoc 2.11+ new --citeproc when CSL is used */
    div.csl-bib-body { }
    div.csl-entry {
      clear: both;
        }
    .hanging div.csl-entry {
      margin-left:2em;
      text-indent:-2em;
    }
    div.csl-left-margin {
      min-width:2em;
      float:left;
    }
    div.csl-right-inline {
      margin-left:2em;
      padding-left:1em;
    }
    div.csl-indent {
      margin-left: 2em;
    }
  </style>
<link rel="stylesheet" href="style.css">
</head>
<body data-spy="scroll" data-target="#toc">

<div class="container-fluid">
<div class="row">
  <header class="col-sm-12 col-lg-3 sidebar sidebar-book"><a class="sr-only sr-only-focusable" href="#content">Skip to main content</a>

    <div class="d-flex align-items-start justify-content-between">
      <h1>
        <a href="index.html" title="">Genetic analysis of quantitative traits in medaka fish and humans</a>
      </h1>
      <button class="btn btn-outline-primary d-lg-none ml-2 mt-1" type="button" data-toggle="collapse" data-target="#main-nav" aria-expanded="true" aria-controls="main-nav"><i class="fas fa-bars"></i><span class="sr-only">Show table of contents</span></button>
    </div>

    <div id="main-nav" class="collapse-lg">
      <form role="search">
        <input id="search" class="form-control" type="search" placeholder="Search" aria-label="Search">
</form>

      <nav aria-label="Table of contents"><h2>Table of contents</h2>
        <ul class="book-toc list-unstyled">
<li><a class="" href="index.html"><span class="header-section-number">1</span> About</a></li>
<li><a class="" href="genomic-variations-in-the-mikk-panel.html"><span class="header-section-number">2</span> Genomic variations in the MIKK panel</a></li>
<li><a class="" href="references.html">References</a></li>
<li><a class="active" href="cross.html"><span class="header-section-number">3</span> Cross-references</a></li>
</ul>

        <div class="book-extra">
          <p><a id="book-repo" href="https://github.com/brettellebi/PhD-thesis">View book source <i class="fab fa-github"></i></a></p>
        </div>
      </nav>
</div>
  </header><main class="col-sm-12 col-md-9 col-lg-7" id="content"><div id="cross" class="section level1" number="3">
<h1>
<span class="header-section-number">3</span> Cross-references<a class="anchor" aria-label="anchor" href="#cross"><i class="fas fa-link"></i></a>
</h1>
<p>Cross-references make it easier for your readers to find and link to elements in your book.</p>
<div id="chapters-and-sub-chapters" class="section level2" number="3.1">
<h2>
<span class="header-section-number">3.1</span> Chapters and sub-chapters<a class="anchor" aria-label="anchor" href="#chapters-and-sub-chapters"><i class="fas fa-link"></i></a>
</h2>
<p>There are two steps to cross-reference any heading:</p>
<ol style="list-style-type: decimal">
<li>Label the heading: <code># Hello world {#nice-label}</code>.
<ul>
<li>Leave the label off if you like the automated heading generated based on your heading title: for example, <code># Hello world</code> = <code># Hello world {#hello-world}</code>.</li>
<li>To label an un-numbered heading, use: <code># Hello world {-#nice-label}</code> or <code>{# Hello world .unnumbered}</code>.</li>
</ul>
</li>
<li>Next, reference the labeled heading anywhere in the text using <code>\@ref(nice-label)</code>; for example, please see Chapter <a href="cross.html#cross">3</a>.
<ul>
<li>If you prefer text as the link instead of a numbered reference use: <a href="cross.html#cross">any text you want can go here</a>.</li>
</ul>
</li>
</ol>
</div>
<div id="captioned-figures-and-tables" class="section level2" number="3.2">
<h2>
<span class="header-section-number">3.2</span> Captioned figures and tables<a class="anchor" aria-label="anchor" href="#captioned-figures-and-tables"><i class="fas fa-link"></i></a>
</h2>
<p>Figures and tables <em>with captions</em> can also be cross-referenced from elsewhere in your book using <code>\@ref(fig:chunk-label)</code> and <code>\@ref(tab:chunk-label)</code>, respectively.</p>
<p>See Figure <a href="cross.html#fig:nice-fig">3.1</a>.</p>
<div class="sourceCode" id="cb4"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="https://rdrr.io/r/graphics/par.html">par</a></span><span class="op">(</span>mar <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html">c</a></span><span class="op">(</span><span class="fl">4</span>, <span class="fl">4</span>, <span class="fl">.1</span>, <span class="fl">.1</span><span class="op">)</span><span class="op">)</span>
<span class="fu"><a href="https://rdrr.io/r/graphics/plot.default.html">plot</a></span><span class="op">(</span><span class="va">pressure</span>, type <span class="op">=</span> <span class="st">'b'</span>, pch <span class="op">=</span> <span class="fl">19</span><span class="op">)</span></code></pre></div>
<div class="figure" style="text-align: center">
<span style="display:block;" id="fig:nice-fig"></span>
<img src="02-cross-refs_files/figure-html/nice-fig-1.png" alt="Plot with connected points showing that vapor pressure of mercury increases exponentially as temperature increases." width="80%"><p class="caption">
Figure 3.1: Here is a nice figure!
</p>
</div>
<p>Don’t miss Table <a href="cross.html#tab:nice-tab">3.1</a>.</p>
<div class="sourceCode" id="cb5"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu">knitr</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/knitr/man/kable.html">kable</a></span><span class="op">(</span>
  <span class="fu"><a href="https://rdrr.io/r/utils/head.html">head</a></span><span class="op">(</span><span class="va">pressure</span>, <span class="fl">10</span><span class="op">)</span>, caption <span class="op">=</span> <span class="st">'Here is a nice table!'</span>,
  booktabs <span class="op">=</span> <span class="cn">TRUE</span>
<span class="op">)</span></code></pre></div>
<div class="inline-table"><table class="table table-sm">
<caption>
<span id="tab:nice-tab">Table 3.1: </span>Here is a nice table!</caption>
<thead><tr class="header">
<th align="right">temperature</th>
<th align="right">pressure</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="right">0</td>
<td align="right">0.0002</td>
</tr>
<tr class="even">
<td align="right">20</td>
<td align="right">0.0012</td>
</tr>
<tr class="odd">
<td align="right">40</td>
<td align="right">0.0060</td>
</tr>
<tr class="even">
<td align="right">60</td>
<td align="right">0.0300</td>
</tr>
<tr class="odd">
<td align="right">80</td>
<td align="right">0.0900</td>
</tr>
<tr class="even">
<td align="right">100</td>
<td align="right">0.2700</td>
</tr>
<tr class="odd">
<td align="right">120</td>
<td align="right">0.7500</td>
</tr>
<tr class="even">
<td align="right">140</td>
<td align="right">1.8500</td>
</tr>
<tr class="odd">
<td align="right">160</td>
<td align="right">4.2000</td>
</tr>
<tr class="even">
<td align="right">180</td>
<td align="right">8.8000</td>
</tr>
</tbody>
</table></div>
<div id="refs" class="references csl-bib-body hanging-indent">
<div id="ref-GlobalReferenceHuman2015" class="csl-entry">
<span>“A Global Reference for Human Genetic Variation.”</span> <em>Nature</em> 526, no. 7571 (2015): 68–74. <a href="https://doi.org/10.1038/nature15393">https://doi.org/10.1038/nature15393</a>.
</div>
<div id="ref-barrettHaploviewAnalysisVisualization2005" class="csl-entry">
Barrett, J. C., B. Fry, J. Maller, and M. J. Daly. <span>“Haploview: Analysis and Visualization of <span>LD</span> and Haplotype Maps.”</span> <em>Bioinformatics</em> 21, no. 2 (January 15, 2005): 263–65. <a href="https://doi.org/10.1093/bioinformatics/bth457">https://doi.org/10.1093/bioinformatics/bth457</a>.
</div>
<div id="ref-changSecondgenerationPLINKRising2015" class="csl-entry">
Chang, Christopher C, Carson C Chow, Laurent CAM Tellier, Shashaank Vattikuti, Shaun M Purcell, and James J Lee. <span>“Second-Generation <span>PLINK</span>: Rising to the Challenge of Larger and Richer Datasets.”</span> <em>GigaScience</em> 4, no. 1 (December 2015): 7. <a href="https://doi.org/10.1186/s13742-015-0047-8">https://doi.org/10.1186/s13742-015-0047-8</a>.
</div>
<div id="ref-fitzgerald2022" class="csl-entry">
Fitzgerald, Tomas, Ian Brettell, Adrien Leger, Nadeshda Wolf, Natalja Kusminski, Jack Monahan, Carl Barton, et al. <span>“The Medaka Inbred Kiyosu-Karlsruhe (MIKK) Panel.”</span> <em>Genome Biology</em> 23, no. 1 (February 21, 2022): 59. <a href="https://doi.org/10.1186/s13059-022-02623-z">https://doi.org/10.1186/s13059-022-02623-z</a>.
</div>
<div id="ref-fredmanComplexSNPrelatedSequence2004" class="csl-entry">
Fredman, David, Stefan J. White, Susanna Potter, Evan E. Eichler, Johan T. Den Dunnen, and Anthony J. Brookes. <span>“Complex <span class="nocase">SNP-related</span> Sequence Variation in Segmental Genome Duplications.”</span> <em>Nature Genetics</em> 36, no. 8, 8 (August 2004): 861–66. <a href="https://doi.org/10.1038/ng1401">https://doi.org/10.1038/ng1401</a>.
</div>
<div id="ref-hoffmannChromosomalInversionPolymorphisms2004" class="csl-entry">
Hoffmann, Ary A., Carla M. Sgr‘o, and Andrew R. Weeks. <span>“Chromosomal Inversion Polymorphisms and Adaptation.”</span> <em>Trends in Ecology &amp; Evolution</em> 19, no. 9 (September 1, 2004): 482–88. <a href="https://doi.org/10.1016/j.tree.2004.06.013">https://doi.org/10.1016/j.tree.2004.06.013</a>.
</div>
<div id="ref-IndexPubRelease102" class="csl-entry">
<span>“Index of /Pub/Release-102/Emf/Ensembl-Compara/Multiple_alignments/50_fish.epo/.”</span> Accessed January 25, 2022. <a href="https://ftp.ensembl.org/pub/release-102/emf/ensembl-compara/multiple_alignments/50_fish.epo/">https://ftp.ensembl.org/pub/release-102/emf/ensembl-compara/multiple_alignments/50_fish.epo/</a>.
</div>
<div id="ref-leger2022" class="csl-entry">
Leger, Adrien, Ian Brettell, Jack Monahan, Carl Barton, Nadeshda Wolf, Natalja Kusminski, Cathrin Herder, et al. <span>“Genomic Variations and Epigenomic Landscape of the Medaka Inbred Kiyosu-Karlsruhe (MIKK) Panel.”</span> <em>Genome Biology</em> 23, no. 1 (February 21, 2022): 58. <a href="https://doi.org/10.1186/s13059-022-02602-4">https://doi.org/10.1186/s13059-022-02602-4</a>.
</div>
<div id="ref-legerGenomicVariationsEpigenomic2022" class="csl-entry">
———, et al. <span>“Genomic Variations and Epigenomic Landscape of the <span>Medaka Inbred Kiyosu-Karlsruhe</span> (<span>MIKK</span>) Panel.”</span> <em>Genome Biology</em> 23, no. 1 (February 21, 2022): 58. <a href="https://doi.org/10.1186/s13059-022-02602-4">https://doi.org/10.1186/s13059-022-02602-4</a>.
</div>
<div id="ref-martinEvaluatingUseABBA2015" class="csl-entry">
Martin, Simon H., John W. Davey, and Chris D. Jiggins. <span>“Evaluating the <span>Use</span> of <span>ABBA</span>–<span>BABA Statistics</span> to <span>Locate Introgressed Loci</span>.”</span> <em>Molecular Biology and Evolution</em> 32, no. 1 (January 1, 2015): 244–57. <a href="https://doi.org/10.1093/molbev/msu269">https://doi.org/10.1093/molbev/msu269</a>.
</div>
<div id="ref-mckennaGenomeAnalysisToolkit2010" class="csl-entry">
McKenna, Aaron, Matthew Hanna, Eric Banks, Andrey Sivachenko, Kristian Cibulskis, Andrew Kernytsky, Kiran Garimella, et al. <span>“The <span>Genome Analysis Toolkit</span>: <span>A MapReduce</span> Framework for Analyzing Next-Generation <span>DNA</span> Sequencing Data.”</span> <em>Genome Research</em> 20, no. 9 (January 9, 2010): 1297–1303. <a href="https://doi.org/10.1101/gr.107524.110">https://doi.org/10.1101/gr.107524.110</a>.
</div>
<div id="ref-MIKKPanelGenome2021" class="csl-entry">
<em><span>MIKK</span> Panel Genome Analysis</em>. 2021. Reprint, <span>birneylab</span>, 2021. <a href="https://github.com/birneylab/MIKK_genome_main_paper">https://github.com/birneylab/MIKK_genome_main_paper</a>.
</div>
<div id="ref-purcellPLINK" class="csl-entry">
Purcell, Shaun M, and Christopher C Chang. <em><span>PLINK</span> 1.9</em>, n.d. <a href="https://www.cog-genomics.org/plink/1.9/">www.cog-genomics.org/plink/1.9/</a>.
</div>
<div id="ref-spivakov2014" class="csl-entry">
Spivakov, Mikhail, Thomas O Auer, Ravindra Peravali, Ian Dunham, Dirk Dolle, Asao Fujiyama, Atsushi Toyoda, et al. <span>“Genomic and Phenotypic Characterization of a Wild Medaka Population: Towards the Establishment of an Isogenic Population Genetic Resource in Fish.”</span> <em>G3 Genes|Genomes|Genetics</em> 4, no. 3 (March 1, 2014): 433–45. <a href="https://doi.org/10.1534/g3.113.008722">https://doi.org/10.1534/g3.113.008722</a>.
</div>
<div id="ref-rcoreteamLanguageEnvironmentStatistical2021" class="csl-entry">
Team, R Core. <em>R: <span>A</span> Language and Environment for Statistical Computing</em> (version 4.1.0). <span>Vienna, Austria</span>: <span>R Foundation for Statistical Computing</span>, 2021. <a href="https://www.R-project.org/">https://www.R-project.org/</a>.
</div>
<div id="ref-wickhamWelcomeTidyverse2019" class="csl-entry">
Wickham, Hadley, Mara Averick, Jennifer Bryan, Winston Chang, Lucy D’Agostino McGowan, Romain François, Garrett Grolemund, et al. <span>“Welcome to the <span>Tidyverse</span>.”</span> <em>Journal of Open Source Software</em> 4, no. 43 (November 21, 2019): 1686. <a href="https://doi.org/10.21105/joss.01686">https://doi.org/10.21105/joss.01686</a>.
</div>
</div>
</div>
</div>
























  <div class="chapter-nav">
<div class="prev"><a href="references.html">References</a></div>
<div class="empty"></div>
</div></main><div class="col-md-3 col-lg-2 d-none d-md-block sidebar sidebar-chapter">
    <nav id="toc" data-toggle="toc" aria-label="On this page"><h2>On this page</h2>
      <ul class="nav navbar-nav">
<li><a class="nav-link" href="#cross"><span class="header-section-number">3</span> Cross-references</a></li>
<li><a class="nav-link" href="#chapters-and-sub-chapters"><span class="header-section-number">3.1</span> Chapters and sub-chapters</a></li>
<li><a class="nav-link" href="#captioned-figures-and-tables"><span class="header-section-number">3.2</span> Captioned figures and tables</a></li>
</ul>

      <div class="book-extra">
        <ul class="list-unstyled">
<li><a id="book-source" href="https://github.com/brettellebi/PhD-thesis/blob/master/book/02-cross-refs.Rmd">View source <i class="fab fa-github"></i></a></li>
          <li><a id="book-edit" href="https://github.com/brettellebi/PhD-thesis/edit/master/book/02-cross-refs.Rmd">Edit this page <i class="fab fa-github"></i></a></li>
        </ul>
</div>
    </nav>
</div>

</div>
</div> <!-- .container -->

<footer class="bg-primary text-light mt-5"><div class="container"><div class="row">

  <div class="col-12 col-md-6 mt-3">
    <p>"<strong>Genetic analysis of quantitative traits in medaka fish and humans</strong>" was written by Ian Brettell. It was last built on 2022-05-26.</p>
  </div>

  <div class="col-12 col-md-6 mt-3">
    <p>This book was built by the <a class="text-light" href="https://bookdown.org">bookdown</a> R package.</p>
  </div>

</div></div>
</footer><!-- dynamically load mathjax for compatibility with self-contained --><script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script><script type="text/x-mathjax-config">const popovers = document.querySelectorAll('a.footnote-ref[data-toggle="popover"]');
for (let popover of popovers) {
  const div = document.createElement('div');
  div.setAttribute('style', 'position: absolute; top: 0, left:0; width:0, height:0, overflow: hidden; visibility: hidden;');
  div.innerHTML = popover.getAttribute('data-content');

  var has_math = div.querySelector("span.math");
  if (has_math) {
    document.body.appendChild(div);
    MathJax.Hub.Queue(["Typeset", MathJax.Hub, div]);
    MathJax.Hub.Queue(function() {
      popover.setAttribute('data-content', div.innerHTML);
      document.body.removeChild(div);
    })
  }
}
</script>
</body>
</html>
