<!DOCTYPE html>
<html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<title>Chapter 6 Variation in the frequency of trait-associated alleles across global human populations | Genetic analysis of quantitative traits in medaka fish and humans</title>
<meta name="author" content="Ian Brettell">
<meta name="description" content="Here we illustrate the distribution of FST scores for loci associated with 587 traits, a subset of the GWAS Catalog which passed our criteria for suitable polygenic traits (Methods). Our analysis...">
<meta name="generator" content="bookdown 0.26 with bs4_book()">
<meta property="og:title" content="Chapter 6 Variation in the frequency of trait-associated alleles across global human populations | Genetic analysis of quantitative traits in medaka fish and humans">
<meta property="og:type" content="book">
<meta property="og:description" content="Here we illustrate the distribution of FST scores for loci associated with 587 traits, a subset of the GWAS Catalog which passed our criteria for suitable polygenic traits (Methods). Our analysis...">
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="Chapter 6 Variation in the frequency of trait-associated alleles across global human populations | Genetic analysis of quantitative traits in medaka fish and humans">
<meta name="twitter:description" content="Here we illustrate the distribution of FST scores for loci associated with 587 traits, a subset of the GWAS Catalog which passed our criteria for suitable polygenic traits (Methods). Our analysis...">
<!-- JS --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://kit.fontawesome.com/6ecbd6c532.js" crossorigin="anonymous"></script><script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="libs/bootstrap-4.6.0/bootstrap.min.css" rel="stylesheet">
<script src="libs/bootstrap-4.6.0/bootstrap.bundle.min.js"></script><script src="libs/bs3compat-0.3.1/transition.js"></script><script src="libs/bs3compat-0.3.1/tabs.js"></script><script src="libs/bs3compat-0.3.1/bs3compat.js"></script><link href="libs/bs4_book-1.0.0/bs4_book.css" rel="stylesheet">
<script src="libs/bs4_book-1.0.0/bs4_book.js"></script><style>
    #content {
        font-size: 0.9rem;
    }
    </style>
<script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- CSS --><style type="text/css">
    
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  </style>
<style type="text/css">
    /* Used with Pandoc 2.11+ new --citeproc when CSL is used */
    div.csl-bib-body { }
    div.csl-entry {
      clear: both;
        }
    .hanging div.csl-entry {
      margin-left:2em;
      text-indent:-2em;
    }
    div.csl-left-margin {
      min-width:2em;
      float:left;
    }
    div.csl-right-inline {
      margin-left:2em;
      padding-left:1em;
    }
    div.csl-indent {
      margin-left: 2em;
    }
  </style>
<link rel="stylesheet" href="style.css">
</head>
<body data-spy="scroll" data-target="#toc">

<div class="container-fluid">
<div class="row">
  <header class="col-sm-12 col-lg-3 sidebar sidebar-book"><a class="sr-only sr-only-focusable" href="#content">Skip to main content</a>

    <div class="d-flex align-items-start justify-content-between">
      <h1>
        <a href="index.html" title="">Genetic analysis of quantitative traits in medaka fish and humans</a>
      </h1>
      <button class="btn btn-outline-primary d-lg-none ml-2 mt-1" type="button" data-toggle="collapse" data-target="#main-nav" aria-expanded="true" aria-controls="main-nav"><i class="fas fa-bars"></i><span class="sr-only">Show table of contents</span></button>
    </div>

    <div id="main-nav" class="collapse-lg">
      <form role="search">
        <input id="search" class="form-control" type="search" placeholder="Search" aria-label="Search">
</form>

      <nav aria-label="Table of contents"><h2>Table of contents</h2>
        <ul class="book-toc list-unstyled"><li><a class="" href="index.html"><span class="header-section-number">Chapter 6</span> Variation in the frequency of trait-associated alleles across global human populations</a></li></ul>

        <div class="book-extra">
          <p><a id="book-repo" href="https://github.com/brettellebi/PhD-thesis">View book source <i class="fab fa-github"></i></a></p>
        </div>
      </nav>
</div>
  </header><main class="col-sm-12 col-md-9 col-lg-7" id="content"><div id="Fst-chap" class="section level1" number="6">
<h1>
<span class="header-section-number">6</span> Variation in the frequency of trait-associated alleles across global human populations<a class="anchor" aria-label="anchor" href="#Fst-chap"><i class="fas fa-link"></i></a>
</h1>
<p>Here we illustrate the distribution of FST scores for loci associated with 587 traits, a subset of the GWAS Catalog which passed our criteria for suitable polygenic traits (Methods). Our analysis is empirical, in that we do not explicitly formulate a statistical test for drift or selection, differential or otherwise. Using high-coverage sequence data for 2,504 individuals from the 1000 Genomes Project phase 3 release, for each trait in the GWAS Catalog we calculated the distribution of FST across all approximately-unlinked SNPs associated with it (trait SNPs), and compared these FST distributions with the FST distributions of random-selected SNPs that were matched to the trait SNPs by their allele frequencies in European populations (control SNPs). Our results show that traits related to the physical correlates of “race” (such as skin-pigmentation, eye colour, and hair shape) tend to have relatively high FST values – signifying relatively high variance in allele frequencies between populations – whereas traits related to intelligence (such as self-reported EA, mathematical ability, and cognitive function measurement) tend to have lower FST values that are similar to those of most polygenic traits such as height and body mass index.</p>
<div id="datasets" class="section level2" number="6.1">
<h2>
<span class="header-section-number">6.1</span> Datasets<a class="anchor" aria-label="anchor" href="#datasets"><i class="fas fa-link"></i></a>
</h2>
<div id="genomes" class="section level3" number="6.1.1">
<h3>
<span class="header-section-number">6.1.1</span> 1000 Genomes<a class="anchor" aria-label="anchor" href="#genomes"><i class="fas fa-link"></i></a>
</h3>
<p>As the reference for human genomic variation across diverse populations, we used the New York Genome Center high-coverage, phased .vcf files<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;&lt;span&gt;“Index of /Vol1/Ftp/Data_collections/&lt;span&gt;1000g&lt;/span&gt;_2504_high_coverage/Working/20201028_3202_phased/,”&lt;/span&gt; accessed March 24, 2022, &lt;a href="http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/data_collections/1000G_2504_high_coverage/working/20201028_3202_phased/" role="doc-biblioref"&gt;http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/data_collections/1000G_2504_high_coverage/working/20201028_3202_phased/&lt;/a&gt;.&lt;/p&gt;'><sup>1</sup></a></span> for the 2,504 individuals described in the 1000 Genomes phase 3 release.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;1000 Genomes Project Consortium et al., &lt;span&gt;“A Global Reference for Human Genetic Variation,”&lt;/span&gt; &lt;em&gt;Nature&lt;/em&gt; 526, no. 7571 (2015): 68.&lt;/p&gt;"><sup>2</sup></a></span> We then annotated those .vcf files with human SNP IDs from dbSNP release 9606.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Elizabeth M. Smigielski et al., &lt;span&gt;“&lt;span class="nocase"&gt;dbSNP&lt;/span&gt;: A Database of Single Nucleotide Polymorphisms,”&lt;/span&gt; &lt;em&gt;Nucleic Acids Research&lt;/em&gt; 28, no. 1 (January 1, 2000): 352–55, &lt;a href="https://doi.org/10.1093/nar/28.1.352" role="doc-biblioref"&gt;https://doi.org/10.1093/nar/28.1.352&lt;/a&gt;.&lt;/p&gt;'><sup>3</sup></a></span></p>
</div>
<div id="gwas-catalog" class="section level3" number="6.1.2">
<h3>
<span class="header-section-number">6.1.2</span> GWAS Catalog<a class="anchor" aria-label="anchor" href="#gwas-catalog"><i class="fas fa-link"></i></a>
</h3>
<p>We used the R package <em>gwasrapidd</em><span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Ramiro Magno and Ana-Teresa Maia, &lt;span&gt;“Gwasrapidd: An &lt;span&gt;R&lt;/span&gt; Package to Query, Download and Wrangle &lt;span&gt;GWAS&lt;/span&gt; Catalog Data,”&lt;/span&gt; &lt;em&gt;Bioinformatics&lt;/em&gt; 36, no. 2 (January 15, 2020): 649–50, &lt;a href="https://doi.org/10.1093/bioinformatics/btz605" role="doc-biblioref"&gt;https://doi.org/10.1093/bioinformatics/btz605&lt;/a&gt;.&lt;/p&gt;'><sup>4</sup></a></span> to query all traits in the GWAS Catalog<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Jacqueline MacArthur et al., &lt;span&gt;“The New &lt;span&gt;NHGRI-EBI Catalog&lt;/span&gt; of Published Genome-Wide Association Studies (&lt;span&gt;GWAS Catalog&lt;/span&gt;),”&lt;/span&gt; &lt;em&gt;Nucleic Acids Research&lt;/em&gt; 45, no. D1 (January 4, 2017): D896–901, &lt;a href="https://doi.org/10.1093/nar/gkw1133" role="doc-biblioref"&gt;https://doi.org/10.1093/nar/gkw1133&lt;/a&gt;.&lt;/p&gt;'><sup>5</sup></a></span> as of 9 August 2021 (<span class="math inline">\(N_{TRAITS}\)</span> = 3,459). For 541 of these traits, no matching variant IDs could be pulled out from the 1000 Genomes VCFs, leaving <span class="math inline">\(N_{TRAITS}\)</span> = 3,008.</p>
</div>
<div id="linkage-disequilibrium" class="section level3" number="6.1.3">
<h3>
<span class="header-section-number">6.1.3</span> Linkage disequilibrium<a class="anchor" aria-label="anchor" href="#linkage-disequilibrium"><i class="fas fa-link"></i></a>
</h3>
<p>To obtain our “trait SNP” dataset, for each trait, we sought to isolate the SNP closest to each of its true causal variants, and exclude the SNPs in linkage disequilibrium (LD) with them. To this end, we used PLINK 1.9<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Christopher C Chang et al., &lt;span&gt;“Second-Generation &lt;span&gt;PLINK&lt;/span&gt;: Rising to the Challenge of Larger and Richer Datasets,”&lt;/span&gt; &lt;em&gt;GigaScience&lt;/em&gt; 4, no. 1 (December 2015): 7, &lt;a href="https://doi.org/10.1186/s13742-015-0047-8" role="doc-biblioref"&gt;https://doi.org/10.1186/s13742-015-0047-8&lt;/a&gt;; Shaun M Purcell and Christopher C Chang, &lt;em&gt;&lt;span&gt;PLINK&lt;/span&gt; 1.9&lt;/em&gt;, n.d., &lt;a href="https://www.cog-genomics.org/plink/1.9/" role="doc-biblioref"&gt;www.cog-genomics.org/plink/1.9/&lt;/a&gt;.&lt;/p&gt;'><sup>6</sup></a></span> to “clump” the SNPs associated with each of the remaining 3,008 traits, using an index variant p-value threshold of <span class="math inline">\(10^{-8}\)</span>,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Orestis A. Panagiotou, John P. A. Ioannidis, and Genome-Wide Significance Project, &lt;span&gt;“What Should the Genome-Wide Significance Threshold Be? &lt;span&gt;Empirical&lt;/span&gt; Replication of Borderline Genetic Associations,”&lt;/span&gt; &lt;em&gt;International Journal of Epidemiology&lt;/em&gt; 41, no. 1 (February 2012): 273–86, &lt;a href="https://doi.org/10.1093/ije/dyr178" role="doc-biblioref"&gt;https://doi.org/10.1093/ije/dyr178&lt;/a&gt;.&lt;/p&gt;'><sup>7</sup></a></span> <span class="math inline">\(r^2\)</span> threshold of 0.1,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;W. G. Hill and Alan Robertson, &lt;span&gt;“Linkage Disequilibrium in Finite Populations,”&lt;/span&gt; &lt;em&gt;Theoretical and Applied Genetics&lt;/em&gt; 38, no. 6 (June 1, 1968): 226–31, &lt;a href="https://doi.org/10.1007/BF01245622" role="doc-biblioref"&gt;https://doi.org/10.1007/BF01245622&lt;/a&gt;.&lt;/p&gt;'><sup>8</sup></a></span> and base window size of 1 Mb. This process left us with 2,045 traits with at least one index SNP that met the p-value threshold. The index SNPs for each trait formed our set of trait SNPs. In order to target relatively polygenic traits, we further filtered out traits with fewer than 10 trait SNPs, leaving <span class="math inline">\(N_{TRAITS}\)</span> = 587. Supplementary Fig. 4 shows the counts of unique SNP IDs associated with each trait before and after clumping, and an interactive version is available in the notebook cited above.</p>
</div>
<div id="control-snps" class="section level3" number="6.1.4">
<h3>
<span class="header-section-number">6.1.4</span> Control SNPs<a class="anchor" aria-label="anchor" href="#control-snps"><i class="fas fa-link"></i></a>
</h3>
<p>To obtain our “control SNP” dataset, we assigned each trait SNP to one of 20 bins based on its minor allele frequency in European populations (as provided in the original 1000 Genomes .vcf files under the column header ‘INFO/AC_EUR’). For example, if a trait SNP had a minor allele frequency of 0.08 in European populations, it was assigned to the <span class="math inline">\((0.05, 0.1]\)</span> bin. We did the same for all (unassociated) SNPs in the .vcf files, then paired each trait SNP with a random SNP from the .vcf file in the equivalent bin. These allele-frequency-paired random SNPs formed our set of “control SNPs”, which we used to infer the <span class="math inline">\(F_{ST}\)</span> distribution of a random set of SNPs with the same allele frequencies as the trait SNPs, and against which we could compare the <span class="math inline">\(F_{ST}\)</span> distribution of the trait SNPs.</p>
</div>
<div id="f_st-and-ranking-traits-by-signed-kolmogorov-smirnov-d-statistic" class="section level3" number="6.1.5">
<h3>
<span class="header-section-number">6.1.5</span> <span class="math inline">\(F_{ST}\)</span> and ranking traits by signed Kolmogorov-Smirnov <span class="math inline">\(D\)</span> statistic<a class="anchor" aria-label="anchor" href="#f_st-and-ranking-traits-by-signed-kolmogorov-smirnov-d-statistic"><i class="fas fa-link"></i></a>
</h3>
<p>We then calculated <span class="math inline">\(F_{ST}\)</span> for each of the trait SNPs and their matched control SNPs using the Weir and Cockerham method,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;B. S. Weir and C. Clark Cockerham, &lt;span&gt;“Estimating &lt;span&gt;F-Statistics&lt;/span&gt; for the &lt;span&gt;Analysis&lt;/span&gt; of &lt;span&gt;Population Structure&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Evolution&lt;/em&gt; 38, no. 6 (1984): 1358–70, &lt;a href="https://doi.org/10.2307/2408641" role="doc-biblioref"&gt;https://doi.org/10.2307/2408641&lt;/a&gt;.&lt;/p&gt;'><sup>9</sup></a></span> as implemented in the R package <em>pegas</em>.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Emmanuel Paradis, &lt;span&gt;“Pegas: An &lt;span&gt;R&lt;/span&gt; Package for Population Genetics with an Integrated–Modular Approach,”&lt;/span&gt; &lt;em&gt;Bioinformatics&lt;/em&gt; 26, no. 3 (February 1, 2010): 419–20, &lt;a href="https://doi.org/10.1093/bioinformatics/btp696" role="doc-biblioref"&gt;https://doi.org/10.1093/bioinformatics/btp696&lt;/a&gt;.&lt;/p&gt;'><sup>10</sup></a></span> To rank all traits based on the directional difference in <span class="math inline">\(F_{ST}\)</span> distributions between trait and control SNPs, we ran three Kolmogorov-Smirnov (KS) tests for each trait <span class="math inline">\(t\)</span> with <span class="math inline">\(x_t\)</span> = <span class="math inline">\(F_{ST, trait SNPs}\)</span> and <span class="math inline">\(y_t\)</span> = <span class="math inline">\(F_{ST, control SNPs}\)</span>:</p>
<ol style="list-style-type: decimal">
<li><p>two-sided (<span class="math inline">\(D_t\)</span>) ;</p></li>
<li><p>one-sided “greater” (<span class="math inline">\(D_t^+\)</span>) ; and</p></li>
<li><p>one-sided “less” (<span class="math inline">\(D_t^-\)</span>).</p></li>
</ol>
<p>Therefore, <span class="math inline">\(D_t^S\)</span>:</p>
<p><span class="math display">\[
\begin{aligned}
D_t^– &gt; D_t^+ &amp;: &amp;-D_t \\
D_t^- = D_t^+ &amp;: &amp;0 \\
D_t^- &lt; D_t^+ &amp;: &amp;D_t \\
\end{aligned}
\]</span></p>
<p>In <strong>Figure <a href="Fst-chap.html#fig:FstMain">6.1</a></strong> we present the <span class="math inline">\(F_{ST}\)</span> distributions of trait SNPs for an illustrative subset of 28 human traits, ranked by the signed Kolmogorov-Smirnov test statistic D when compared with their matched control SNPs. <strong>Figure <a href="Fst-chap.html#fig:FstMain">6.1</a>A</strong> shows the densities of SNPs as a function of <span class="math inline">\(F_{ST}\)</span>, and <strong>Figure <a href="Fst-chap.html#fig:FstMain">6.1</a>B and C</strong> show their empirical Cumulative Distribution Functions (eCDFs). <strong>Figure <a href="Fst-chap.html#fig:FstMain">6.1</a>B</strong> includes the eCDFs of control SNPs in grey. eCDF figures for all 587 traits that passed our filters (Methods) are provided in Supplementary Fig. 1.</p>

<div class="figure">
<span style="display:block;" id="fig:FstMain"></span>
<img src="figs/fst/20220314_final_0.1_1000.png" alt="Distributions of FST across 28 illustrative human traits, ranked by signed-D (Kolmogorov-Smirnov test) comparing trait and control SNPs. A. FST density ridge plots with SNP markers. B. Empirical Cumulative Distribution Functions (eCDFs) of FST for trait-associated (colour) and random control (grey) SNPs, faceted by trait. C. Consolidated eCDFs of trait-associated SNPs from (B). eCDFs for all traits are included in Supplementary Figure 1." width="100%"><p class="caption">
Figure 6.1: Distributions of FST across 28 illustrative human traits, ranked by signed-D (Kolmogorov-Smirnov test) comparing trait and control SNPs. <strong>A</strong>. FST density ridge plots with SNP markers. <strong>B</strong>. Empirical Cumulative Distribution Functions (eCDFs) of FST for trait-associated (colour) and random control (grey) SNPs, faceted by trait. <strong>C</strong>. Consolidated eCDFs of trait-associated SNPs from (B). eCDFs for all traits are included in Supplementary Figure 1.
</p>
</div>

</div>
</div>
</div>

  <div class="chapter-nav">
<div class="empty"></div>
<div class="empty"></div>
</div></main><div class="col-md-3 col-lg-2 d-none d-md-block sidebar sidebar-chapter">
    <nav id="toc" data-toggle="toc" aria-label="On this page"><h2>On this page</h2>
      <div id="book-on-this-page"></div>

      <div class="book-extra">
        <ul class="list-unstyled">
          
        </ul>
</div>
    </nav>
</div>

</div>
</div> <!-- .container -->

<footer class="bg-primary text-light mt-5"><div class="container"><div class="row">

  <div class="col-12 col-md-6 mt-3">
    <p>"<strong>Genetic analysis of quantitative traits in medaka fish and humans</strong>" was written by Ian Brettell. It was last built on 2022-06-09.</p>
  </div>

  <div class="col-12 col-md-6 mt-3">
    <p>This book was built by the <a class="text-light" href="https://bookdown.org">bookdown</a> R package.</p>
  </div>

</div></div>
</footer><!-- dynamically load mathjax for compatibility with self-contained --><script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script><script type="text/x-mathjax-config">const popovers = document.querySelectorAll('a.footnote-ref[data-toggle="popover"]');
for (let popover of popovers) {
  const div = document.createElement('div');
  div.setAttribute('style', 'position: absolute; top: 0, left:0; width:0, height:0, overflow: hidden; visibility: hidden;');
  div.innerHTML = popover.getAttribute('data-content');

  var has_math = div.querySelector("span.math");
  if (has_math) {
    document.body.appendChild(div);
    MathJax.Hub.Queue(["Typeset", MathJax.Hub, div]);
    MathJax.Hub.Queue(function() {
      popover.setAttribute('data-content', div.innerHTML);
      document.body.removeChild(div);
    })
  }
}
</script>
</body>
</html>
