<!DOCTYPE html>
<html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<title>Chapter 6 Variation in the frequency of trait-associated alleles across global human populations | Japanese courage: a genetic analysis of complex traits in medaka fish and humans</title>
<meta name="author" content="Ian Brettell">
<meta name="description" content="6.1 Background Humans have long sought to use genetic information to predict an individual’s likely value for a given trait, in our own species and in other organisms (Chapter 1). As seen in...">
<meta name="generator" content="bookdown 0.28 with bs4_book()">
<meta property="og:title" content="Chapter 6 Variation in the frequency of trait-associated alleles across global human populations | Japanese courage: a genetic analysis of complex traits in medaka fish and humans">
<meta property="og:type" content="book">
<meta property="og:description" content="6.1 Background Humans have long sought to use genetic information to predict an individual’s likely value for a given trait, in our own species and in other organisms (Chapter 1). As seen in...">
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="Chapter 6 Variation in the frequency of trait-associated alleles across global human populations | Japanese courage: a genetic analysis of complex traits in medaka fish and humans">
<meta name="twitter:description" content="6.1 Background Humans have long sought to use genetic information to predict an individual’s likely value for a given trait, in our own species and in other organisms (Chapter 1). As seen in...">
<!-- JS --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://kit.fontawesome.com/6ecbd6c532.js" crossorigin="anonymous"></script><script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="libs/bootstrap-4.6.0/bootstrap.min.css" rel="stylesheet">
<script src="libs/bootstrap-4.6.0/bootstrap.bundle.min.js"></script><script src="libs/bs3compat-0.4.0/transition.js"></script><script src="libs/bs3compat-0.4.0/tabs.js"></script><script src="libs/bs3compat-0.4.0/bs3compat.js"></script><link href="libs/bs4_book-1.0.0/bs4_book.css" rel="stylesheet">
<script src="libs/bs4_book-1.0.0/bs4_book.js"></script><style>
    #content {
        font-size: 0.9rem;
    }
    </style>
<script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- CSS --><style type="text/css">
    
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  </style>
<style type="text/css">
    /* Used with Pandoc 2.11+ new --citeproc when CSL is used */
    div.csl-bib-body { }
    div.csl-entry {
      clear: both;
        }
    .hanging div.csl-entry {
      margin-left:2em;
      text-indent:-2em;
    }
    div.csl-left-margin {
      min-width:2em;
      float:left;
    }
    div.csl-right-inline {
      margin-left:2em;
      padding-left:1em;
    }
    div.csl-indent {
      margin-left: 2em;
    }
  </style>
<link rel="stylesheet" href="style.css">
</head>
<body data-spy="scroll" data-target="#toc">

<div class="container-fluid">
<div class="row">
  <header class="col-sm-12 col-lg-3 sidebar sidebar-book"><a class="sr-only sr-only-focusable" href="#content">Skip to main content</a>

    <div class="d-flex align-items-start justify-content-between">
      <h1>
        <a href="index.html" title="">Japanese courage: a genetic analysis of complex traits in medaka fish and humans</a>
      </h1>
      <button class="btn btn-outline-primary d-lg-none ml-2 mt-1" type="button" data-toggle="collapse" data-target="#main-nav" aria-expanded="true" aria-controls="main-nav"><i class="fas fa-bars"></i><span class="sr-only">Show table of contents</span></button>
    </div>

    <div id="main-nav" class="collapse-lg">
      <form role="search">
        <input id="search" class="form-control" type="search" placeholder="Search" aria-label="Search">
</form>

      <nav aria-label="Table of contents"><h2>Table of contents</h2>
        <ul class="book-toc list-unstyled">
<li><a class="" href="index.html">About</a></li>
<li><a class="" href="Introduction.html"><span class="header-section-number">1</span> Introduction</a></li>
<li><a class="" href="MIKK-genomes-chap.html"><span class="header-section-number">2</span> Genomic variations in the MIKK panel</a></li>
<li><a class="" href="Pilot-chap.html"><span class="header-section-number">3</span> Classification of bold/shy behaviours in 5 inbred medaka lines</a></li>
<li><a class="" href="Somite-chap.html"><span class="header-section-number">4</span> Genetic loci associated with somite development periodicity</a></li>
<li><a class="" href="MIKK-F2-chap.html"><span class="header-section-number">5</span> Genetic linkage study of bold/shy behaviours in the MIKK panel</a></li>
<li><a class="active" href="Fst-chap.html"><span class="header-section-number">6</span> Variation in the frequency of trait-associated alleles across global human populations</a></li>
<li><a class="" href="references.html">References</a></li>
<li class="book-part">Appendix</li>
<li><a class="" href="hmm-states-with-0.05-second-interval.html"><span class="header-section-number">A</span> 15 HMM states with 0.05 second interval</a></li>
<li><a class="" href="four-panel-app.html"><span class="header-section-number">B</span> Four-panel plots for MIKK lines selected for the F2 cross</a></li>
<li><a class="" href="hmm-state-time-dependence-for-all-mikk-panel-lines.html"><span class="header-section-number">C</span> HMM state time dependence for all MIKK panel lines</a></li>
<li><a class="" href="f2-recombination-karyoplot-with-missing-calls.html"><span class="header-section-number">D</span> F2 recombination karyoplot with missing calls</a></li>
<li><a class="" href="loco-grm-for-chromosome-1.html"><span class="header-section-number">E</span> LOCO GRM for chromosome 1</a></li>
<li><a class="" href="ecdf-of-all-polygenic-traits-in-the-gwas-catalog-ranked-by-d_ts.html"><span class="header-section-number">F</span> eCDF of all polygenic traits in the GWAS Catalog ranked by \({D_t^S}\)</a></li>
</ul>

        <div class="book-extra">
          <p><a id="book-repo" href="https://github.com/brettellebi/PhD-thesis">View book source <i class="fab fa-github"></i></a></p>
        </div>
      </nav>
</div>
  </header><main class="col-sm-12 col-md-9 col-lg-7" id="content"><div id="Fst-chap" class="section level1" number="6">
<h1>
<span class="header-section-number">6</span> Variation in the frequency of trait-associated alleles across global human populations<a class="anchor" aria-label="anchor" href="#Fst-chap"><i class="fas fa-link"></i></a>
</h1>
<div id="Fst-background" class="section level2" number="6.1">
<h2>
<span class="header-section-number">6.1</span> Background<a class="anchor" aria-label="anchor" href="#Fst-background"><i class="fas fa-link"></i></a>
</h2>
<p>Humans have long sought to use genetic information to predict an individual’s likely value for a given trait, in our own species and in other organisms (Chapter <a href="Introduction.html#Introduction">1</a>). As seen in previous chapters, an individual’s phenotypic value at a given point in time is the product of complex interactions between their genome and their environment, beginning from embryonic development and continuing throughout their lifetimes. It is now clear that “complex” traits such as height, intelligence, and behaviour are highly polygenic, meaning that they are genetically influenced by hundreds or thousands of genetic variants, each exerting a small effect in one or the other direction along the trait’s spectrum.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;G. Sella and N. Barton, &lt;span&gt;“Thinking &lt;span&gt;About&lt;/span&gt; the &lt;span&gt;Evolution&lt;/span&gt; of &lt;span&gt;Complex Traits&lt;/span&gt; in the &lt;span&gt;Era&lt;/span&gt; of &lt;span&gt;Genome-Wide Association Studies&lt;/span&gt;.”&lt;/span&gt; &lt;em&gt;Annual Review of Genomics and Human Genetics&lt;/em&gt;, 2019, &lt;a href="https://doi.org/10.1146/annurev-genom-083115-022316" role="doc-biblioref"&gt;https://doi.org/10.1146/annurev-genom-083115-022316&lt;/a&gt;.&lt;/p&gt;'><sup>287</sup></a></span></p>
<p>A richer understanding of the cumulative effect of genetic variants on any trait allows for the prediction of the value that an individual is most likely to have for that trait. Of all human traits, diseases are particularly salient; in 2018, the global healthcare industry was valued at US$8 trillion, and predicted to increase to US$12 trillion by 2022.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;&lt;span&gt;“The $11.9 &lt;span&gt;Trillion Global Healthcare Market&lt;/span&gt;: &lt;span&gt;Key Opportunities&lt;/span&gt; &amp;amp; &lt;span&gt;Strategies&lt;/span&gt; (2014-2022) - &lt;span&gt;ResearchAndMarkets&lt;/span&gt;.com,”&lt;/span&gt; June 25, 2019, &lt;a href="https://www.businesswire.com/news/home/20190625005862/en/The-11.9-Trillion-Global-Healthcare-Market-Key-Opportunities-Strategies-2014-2022---ResearchAndMarkets.com" role="doc-biblioref"&gt;https://www.businesswire.com/news/home/20190625005862/en/The-11.9-Trillion-Global-Healthcare-Market-Key-Opportunities-Strategies-2014-2022---ResearchAndMarkets.com&lt;/a&gt;.&lt;/p&gt;'><sup>288</sup></a></span> This strong financial imperative complements the moral imperative to reduce suffering, together driving the question of how to use genetic information to improve human health.</p>
<p>Recent technological developments have made it possible to sequence human genomes at scale, and it is thought that by combining detailed genetic information with with other environmental and phenotypic information (such as lifestyle or clinical factors), clinicians could move towards the practice of “precision medicine”, where interventions could be tailored to their patients’ unique risk profiles.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Naomi R. Wray, Michael E. Goddard, and Peter M. Visscher, &lt;span&gt;“Prediction of Individual Genetic Risk to Disease from Genome-Wide Association Studies,”&lt;/span&gt; &lt;em&gt;Genome Research&lt;/em&gt; 17, no. 10 (January 10, 2007): 1520–28, &lt;a href="https://doi.org/10.1101/gr.6665407" role="doc-biblioref"&gt;https://doi.org/10.1101/gr.6665407&lt;/a&gt;.&lt;/p&gt;'><sup>289</sup></a></span> The use of genetic information to predict individuals’ values for a trait of interest entails the construction of metrics known as “polygenic scores” (<strong>PGS</strong>). When the trait is a disease, PGS is commonly known as polygenic risk scores (<strong>PRS</strong>), or genetic risk profiling, but I use the term PGS to encompass both disease and non-disease traits.</p>
<div id="polygenic-scores-pgs-and-genome-wide-association-studies-gwas" class="section level3" number="6.1.1">
<h3>
<span class="header-section-number">6.1.1</span> Polygenic Scores (PGS) and Genome-Wide Association Studies (GWAS)<a class="anchor" aria-label="anchor" href="#polygenic-scores-pgs-and-genome-wide-association-studies-gwas"><i class="fas fa-link"></i></a>
</h3>
<div id="pgs" class="section level4" number="6.1.1.1">
<h4>
<span class="header-section-number">6.1.1.1</span> PGS<a class="anchor" aria-label="anchor" href="#pgs"><i class="fas fa-link"></i></a>
</h4>
<p>PGS using genetic information alone show modest yet reliable accuracy for the prediction of complex traits:<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Alicia R. Martin et al., &lt;span&gt;“Clinical Use of Current Polygenic Risk Scores May Exacerbate Health Disparities,”&lt;/span&gt; &lt;em&gt;Nature Genetics&lt;/em&gt; 51, no. 4, 4 (April 2019): 584–91, &lt;a href="https://doi.org/10.1038/s41588-019-0379-x" role="doc-biblioref"&gt;https://doi.org/10.1038/s41588-019-0379-x&lt;/a&gt;.&lt;/p&gt;'><sup>290</sup></a></span> the correlations between PGS and the trait value as measured by <span class="math inline">\(R^2\)</span> have reached 0.24 for height,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Loic Yengo et al., &lt;span&gt;“Meta-Analysis of Genome-Wide Association Studies for Height and Body Mass Index in  700000 Individuals of &lt;span&gt;European&lt;/span&gt; Ancestry,”&lt;/span&gt; &lt;em&gt;Human Molecular Genetics&lt;/em&gt; 27, no. 20 (October 15, 2018): 3641–49, &lt;a href="https://doi.org/10.1093/hmg/ddy271" role="doc-biblioref"&gt;https://doi.org/10.1093/hmg/ddy271&lt;/a&gt;.&lt;/p&gt;'><sup>291</sup></a></span> and 0.12-0.16 for educational attainment.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Aysu Okbay et al., &lt;span&gt;“Polygenic Prediction of Educational Attainment Within and Between Families from Genome-Wide Association Analyses in 3 Million Individuals,”&lt;/span&gt; &lt;em&gt;Nature Genetics&lt;/em&gt;, March 31, 2022, 1–13, &lt;a href="https://doi.org/10.1038/s41588-022-01016-z" role="doc-biblioref"&gt;https://doi.org/10.1038/s41588-022-01016-z&lt;/a&gt;.&lt;/p&gt;'><sup>292</sup></a></span> PGS also improve predictions beyond non-genetic clinical models for a variety of health-related traits, including breast cancer,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Paige Maas et al., &lt;span&gt;“Breast &lt;span&gt;Cancer Risk From Modifiable&lt;/span&gt; and &lt;span&gt;Nonmodifiable Risk Factors Among White Women&lt;/span&gt; in the &lt;span&gt;United States&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;JAMA Oncology&lt;/em&gt; 2, no. 10 (October 1, 2016): 1295–1302, &lt;a href="https://doi.org/10.1001/jamaoncol.2016.1025" role="doc-biblioref"&gt;https://doi.org/10.1001/jamaoncol.2016.1025&lt;/a&gt;.&lt;/p&gt;'><sup>293</sup></a></span> prostate cancer,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Fredrick R. Schumacher et al., &lt;span&gt;“Association Analyses of More Than 140,000 Men Identify 63 New Prostate Cancer Susceptibility Loci,”&lt;/span&gt; &lt;em&gt;Nature Genetics&lt;/em&gt; 50, no. 7, 7 (July 2018): 928–36, &lt;a href="https://doi.org/10.1038/s41588-018-0142-8" role="doc-biblioref"&gt;https://doi.org/10.1038/s41588-018-0142-8&lt;/a&gt;.&lt;/p&gt;'><sup>294</sup></a></span> and type I diabetes.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Seth A. Sharp et al., &lt;span&gt;“Development and &lt;span&gt;Standardization&lt;/span&gt; of an &lt;span&gt;Improved Type&lt;/span&gt; 1 &lt;span&gt;Diabetes Genetic Risk Score&lt;/span&gt; for &lt;span&gt;Use&lt;/span&gt; in &lt;span&gt;Newborn Screening&lt;/span&gt; and &lt;span&gt;Incident Diagnosis&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Diabetes Care&lt;/em&gt; 42, no. 2 (January 11, 2019): 200–207, &lt;a href="https://doi.org/10.2337/dc18-1785" role="doc-biblioref"&gt;https://doi.org/10.2337/dc18-1785&lt;/a&gt;.&lt;/p&gt;'><sup>295</sup></a></span> The predictive accuracy of PGS scores can be further improved by combining genetic information with lifestyle and clinical factors, as seen with cardiovascular disease.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Amit V. Khera et al., &lt;span&gt;“Genome-Wide Polygenic Scores for Common Diseases Identify Individuals with Risk Equivalent to Monogenic Mutations,”&lt;/span&gt; &lt;em&gt;Nature Genetics&lt;/em&gt; 50, no. 9, 9 (September 2018): 1219–24, &lt;a href="https://doi.org/10.1038/s41588-018-0183-z" role="doc-biblioref"&gt;https://doi.org/10.1038/s41588-018-0183-z&lt;/a&gt;; Iftikhar J. Kullo et al., &lt;span&gt;“Incorporating a &lt;span&gt;Genetic Risk Score Into Coronary Heart Disease Risk Estimates&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Circulation&lt;/em&gt; 133, no. 12 (March 22, 2016): 1181–88, &lt;a href="https://doi.org/10.1161/CIRCULATIONAHA.115.020109" role="doc-biblioref"&gt;https://doi.org/10.1161/CIRCULATIONAHA.115.020109&lt;/a&gt;; Pradeep Natarajan et al., &lt;span&gt;“Polygenic &lt;span&gt;Risk Score Identifies Subgroup With Higher Burden&lt;/span&gt; of &lt;span&gt;Atherosclerosis&lt;/span&gt; and &lt;span&gt;Greater Relative Benefit From Statin Therapy&lt;/span&gt; in the &lt;span&gt;Primary Prevention Setting&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Circulation&lt;/em&gt; 135, no. 22 (May 30, 2017): 2091–101, &lt;a href="https://doi.org/10.1161/CIRCULATIONAHA.116.024436" role="doc-biblioref"&gt;https://doi.org/10.1161/CIRCULATIONAHA.116.024436&lt;/a&gt;; Martine Paquette et al., &lt;span&gt;“Polygenic Risk Score Predicts Prevalence of Cardiovascular Disease in Patients with Familial Hypercholesterolemia,”&lt;/span&gt; &lt;em&gt;Journal of Clinical Lipidology&lt;/em&gt; 11, no. 3 (May 1, 2017): 725–732.e5, &lt;a href="https://doi.org/10.1016/j.jacl.2017.03.019" role="doc-biblioref"&gt;https://doi.org/10.1016/j.jacl.2017.03.019&lt;/a&gt;; Emmi Tikkanen et al., &lt;span&gt;“Genetic &lt;span&gt;Risk Prediction&lt;/span&gt; and a 2-&lt;span&gt;Stage Risk Screening Strategy&lt;/span&gt; for &lt;span&gt;Coronary Heart Disease&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Arteriosclerosis, Thrombosis, and Vascular Biology&lt;/em&gt; 33, no. 9 (September 2013): 2261–66, &lt;a href="https://doi.org/10.1161/ATVBAHA.112.301120" role="doc-biblioref"&gt;https://doi.org/10.1161/ATVBAHA.112.301120&lt;/a&gt;; Luanluan Sun et al., &lt;span&gt;“Polygenic Risk Scores in Cardiovascular Risk Prediction: &lt;span&gt;A&lt;/span&gt; Cohort Study and Modelling Analyses,”&lt;/span&gt; &lt;em&gt;PLOS Medicine&lt;/em&gt; 18, no. 1 (January 14, 2021): e1003498, &lt;a href="https://doi.org/10.1371/journal.pmed.1003498" role="doc-biblioref"&gt;https://doi.org/10.1371/journal.pmed.1003498&lt;/a&gt;.&lt;/p&gt;'><sup>296</sup></a></span></p>
</div>
<div id="gwas" class="section level4" number="6.1.1.2">
<h4>
<span class="header-section-number">6.1.1.2</span> GWAS<a class="anchor" aria-label="anchor" href="#gwas"><i class="fas fa-link"></i></a>
</h4>
<p>PGS are calculated for an individual by summing trait-associated alleles identified by genome-wide association studies (<strong>GWAS</strong>), as weighted by the alleles’ effect sizes.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;L. Duncan et al., &lt;span&gt;“Analysis of Polygenic Risk Score Usage and Performance in Diverse Human Populations,”&lt;/span&gt; &lt;em&gt;Nature Communications&lt;/em&gt; 10, no. 1, 1 (July 25, 2019): 3328, &lt;a href="https://doi.org/10.1038/s41467-019-11112-0" role="doc-biblioref"&gt;https://doi.org/10.1038/s41467-019-11112-0&lt;/a&gt;.&lt;/p&gt;'><sup>297</sup></a></span> GWAS aim to identify genetic variants associated with traits by comparing the allele frequencies of individuals who share similar ancestries, but differ in values for the trait in question.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Uffelmann et al., &lt;span&gt;“Genome-Wide Association Studies.”&lt;/span&gt;&lt;/p&gt;"><sup>298</sup></a></span> As of 2021, over 5,700 GWAS have been performed for more than 3,330 traits.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Uffelmann et al.&lt;/p&gt;"><sup>299</sup></a></span></p>
<p>However, most GWAS have been performed with individuals of European ancestry, despite only constituting around 16% of the present global population. Although the proportion of participants in GWAS from a non-European background increased from 4% in 2009 to 16% in 2016),<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Alice B. Popejoy and Stephanie M. Fullerton, &lt;span&gt;“Genomics Is Failing on Diversity,”&lt;/span&gt; &lt;em&gt;Nature&lt;/em&gt; 538, no. 7624, 7624 (October 2016): 161–64, &lt;a href="https://doi.org/10.1038/538161a" role="doc-biblioref"&gt;https://doi.org/10.1038/538161a&lt;/a&gt;.&lt;/p&gt;'><sup>300</sup></a></span> as of 2019, 79% of all GWAS participants recorded in the GWAS Catalog were of European ancestry, and the proportion of non-European individuals has remained the same or reduced since late 2014.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Martin et al., &lt;span&gt;“Clinical Use of Current Polygenic Risk Scores May Exacerbate Health Disparities.”&lt;/span&gt;&lt;/p&gt;"><sup>301</sup></a></span> This bias extends to PGS studies, where as of 2019, only 67% of them included only participants of European ancestry, with another 19% including only East Asian ancestry participants, and only 3.8% with cohorts of African, Hispanic, or Indigenous ancestry.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Duncan et al., &lt;span&gt;“Analysis of Polygenic Risk Score Usage and Performance in Diverse Human Populations.”&lt;/span&gt;&lt;/p&gt;"><sup>302</sup></a></span></p>
<p>It is therefore unsurprising that PGS scores are far better at predicting disease risk in individuals of European ancestry than in those of non-European ancestry.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Alicia R Martin et al., &lt;span&gt;“Human Demographic History Impacts Genetic Risk Prediction Across Diverse Populations,”&lt;/span&gt; &lt;em&gt;The American Journal of Human Genetics&lt;/em&gt; 100, no. 4 (2017): 635–49, &lt;a href="https://doi.org/10.1016/j.ajhg.2017.03.004" role="doc-biblioref"&gt;https://doi.org/10.1016/j.ajhg.2017.03.004&lt;/a&gt;; Martin et al., &lt;span&gt;“Clinical Use of Current Polygenic Risk Scores May Exacerbate Health Disparities.”&lt;/span&gt;&lt;/p&gt;'><sup>303</sup></a></span> Indeed, the predictive accuracy of PGS scores decays with genetic divergence of the GWAS “independent” or “test” sample, from the “discovery” or “training” sample, as established in both humans,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Martin et al., &lt;span&gt;“Human Demographic History Impacts Genetic Risk Prediction Across Diverse Populations”&lt;/span&gt;; Martin et al., &lt;span&gt;“Clinical Use of Current Polygenic Risk Scores May Exacerbate Health Disparities.”&lt;/span&gt;&lt;/p&gt;"><sup>304</sup></a></span> and livestock.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Samuel A. Clark et al., &lt;span&gt;“The Importance of Information on Relatives for the Prediction of Genomic Breeding Values and the Implications for the Makeup of Reference Data Sets in Livestock Breeding Schemes,”&lt;/span&gt; &lt;em&gt;Genetics Selection Evolution&lt;/em&gt; 44, no. 1 (February 9, 2012): 4, &lt;a href="https://doi.org/10.1186/1297-9686-44-4" role="doc-biblioref"&gt;https://doi.org/10.1186/1297-9686-44-4&lt;/a&gt;; David Habier et al., &lt;span&gt;“The Impact of Genetic Relationship Information on Genomic Breeding Values in &lt;span&gt;German Holstein&lt;/span&gt; Cattle,”&lt;/span&gt; &lt;em&gt;Genetics Selection Evolution&lt;/em&gt; 42, no. 1 (February 19, 2010): 5, &lt;a href="https://doi.org/10.1186/1297-9686-42-5" role="doc-biblioref"&gt;https://doi.org/10.1186/1297-9686-42-5&lt;/a&gt;; M. Pszczola et al., &lt;span&gt;“Reliability of Direct Genomic Values for Animals with Different Relationships Within and to the Reference Population,”&lt;/span&gt; &lt;em&gt;Journal of Dairy Science&lt;/em&gt; 95, no. 1 (January 1, 2012): 389–400, &lt;a href="https://doi.org/10.3168/jds.2011-4338" role="doc-biblioref"&gt;https://doi.org/10.3168/jds.2011-4338&lt;/a&gt;.&lt;/p&gt;'><sup>305</sup></a></span></p>
<p>Compared to PGS scores for those of European ancestry, PGS scores across multiple traits are ~64-78% less accurate for individuals of African ancestry,,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Duncan et al., &lt;span&gt;“Analysis of Polygenic Risk Score Usage and Performance in Diverse Human Populations”&lt;/span&gt;; Martin et al., &lt;span&gt;“Clinical Use of Current Polygenic Risk Scores May Exacerbate Health Disparities.”&lt;/span&gt;&lt;/p&gt;"><sup>306</sup></a></span> ~50% less accurate for individuals of East-Asian ancestry, and ~37% less accurate for individuals of South-Asian ancestry.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Martin et al., &lt;span&gt;“Clinical Use of Current Polygenic Risk Scores May Exacerbate Health Disparities.”&lt;/span&gt;&lt;/p&gt;"><sup>307</sup></a></span></p>
</div>
</div>
<div id="contributors-to-pgs-non-transferability" class="section level3" number="6.1.2">
<h3>
<span class="header-section-number">6.1.2</span> Contributors to PGS non-transferability<a class="anchor" aria-label="anchor" href="#contributors-to-pgs-non-transferability"><i class="fas fa-link"></i></a>
</h3>
<p>What explains this disparity in predictive value? A number of factors may be responsible, including:</p>
<ol style="list-style-type: decimal">
<li><p>The failure of GWAS to identify causal variants that either do not exist or are not identifiable within the “discovery” sample, for both technological and methodological reasons;<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Martin et al.&lt;/p&gt;"><sup>308</sup></a></span></p></li>
<li><p>The sample populations may differ in linkage disequilibrium (<strong>LD</strong>) – the correlation structure of the genome – which would change the estimated effect sizes of the causal variants, even when the causal variants themselves are the same;<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Martin et al.&lt;/p&gt;"><sup>309</sup></a></span></p></li>
<li><p>Allele frequencies of the causal variants, and the distribution of the effect sizes of the causal variants, may differ between populations;<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Martin et al., &lt;span&gt;“Human Demographic History Impacts Genetic Risk Prediction Across Diverse Populations”&lt;/span&gt;; Marco Scutari, Ian Mackay, and David Balding, &lt;span&gt;“Using &lt;span&gt;Genetic Distance&lt;/span&gt; to &lt;span&gt;Infer&lt;/span&gt; the &lt;span&gt;Accuracy&lt;/span&gt; of &lt;span&gt;Genomic Prediction&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;PLOS Genetics&lt;/em&gt; 12, no. 9 (September 2, 2016): e1006288, &lt;a href="https://doi.org/10.1371/journal.pgen.1006288" role="doc-biblioref"&gt;https://doi.org/10.1371/journal.pgen.1006288&lt;/a&gt;.&lt;/p&gt;'><sup>310</sup></a></span> and</p></li>
<li><p>The environments and demographies of populations tend to differ. Such differences are often correlated with genetic divergence due to geography, making it difficult to determine whether the associations are driven by the differences between population in their genetics, or their environments.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Martin et al., &lt;span&gt;“Clinical Use of Current Polygenic Risk Scores May Exacerbate Health Disparities”&lt;/span&gt;; Sini Kerminen et al., &lt;span&gt;“Geographic &lt;span&gt;Variation&lt;/span&gt; and &lt;span&gt;Bias&lt;/span&gt; in the &lt;span&gt;Polygenic Scores&lt;/span&gt; of &lt;span&gt;Complex Diseases&lt;/span&gt; and &lt;span&gt;Traits&lt;/span&gt; in &lt;span&gt;Finland&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;The American Journal of Human Genetics&lt;/em&gt; 104, no. 6 (June 6, 2019): 1169–81, &lt;a href="https://doi.org/10.1016/j.ajhg.2019.05.001" role="doc-biblioref"&gt;https://doi.org/10.1016/j.ajhg.2019.05.001&lt;/a&gt;.&lt;/p&gt;'><sup>311</sup></a></span></p></li>
</ol>
<p>The first three factors can degrade predictive performance even in the absence of biological and environmental differences. On the other hand, environmental and demographic differences can drive forces of natural selection can in turn drive differences in causal genetic architecture.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Martin et al., &lt;span&gt;“Clinical Use of Current Polygenic Risk Scores May Exacerbate Health Disparities.”&lt;/span&gt;&lt;/p&gt;"><sup>312</sup></a></span></p>
<p>I will discuss each of these factors in turn before addressing point (3) in this analysis.</p>
<div id="fst-discovery-sec" class="section level4" number="6.1.2.1">
<h4>
<span class="header-section-number">6.1.2.1</span> Technological and methodological limitations of GWAS<a class="anchor" aria-label="anchor" href="#fst-discovery-sec"><i class="fas fa-link"></i></a>
</h4>
<p>The power to discover a causal variant through GWAS depends on the variant’s effect size and frequency in the study population.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Martin et al.; P. C. Sham et al., &lt;span&gt;“Power of &lt;span&gt;Linkage&lt;/span&gt; Versus &lt;span&gt;Association Analysis&lt;/span&gt; of &lt;span&gt;Quantitative Traits&lt;/span&gt;, by &lt;span&gt;Use&lt;/span&gt; of &lt;span&gt;Variance-Components Models&lt;/span&gt;, for &lt;span&gt;Sibship Data&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;The American Journal of Human Genetics&lt;/em&gt; 66, no. 5 (May 1, 2000): 1616–30, &lt;a href="https://doi.org/10.1086/302891" role="doc-biblioref"&gt;https://doi.org/10.1086/302891&lt;/a&gt;.&lt;/p&gt;'><sup>313</sup></a></span> That is to say, the stronger the variant’s effect, or the more common it is, the more likely it is to be discovered. Rare variants tend to have stronger effect sizes,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Kyoko Watanabe et al., &lt;span&gt;“A Global Overview of Pleiotropy and Genetic Architecture in Complex Traits,”&lt;/span&gt; &lt;em&gt;Nature Genetics&lt;/em&gt; 51, no. 9, 9 (September 2019): 1339–48, &lt;a href="https://doi.org/10.1038/s41588-019-0481-0" role="doc-biblioref"&gt;https://doi.org/10.1038/s41588-019-0481-0&lt;/a&gt;.&lt;/p&gt;'><sup>314</sup></a></span> likely due to purifying selection,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Park et al., &lt;span&gt;“Distribution of Allele Frequencies and Effect Sizes and Their Interrelationships for Common Genetic Susceptibility Variants.”&lt;/span&gt;&lt;/p&gt;"><sup>315</sup></a></span> and tend not to be shared across populations.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Simon Gravel et al., &lt;span&gt;“Demographic History and Rare Allele Sharing Among Human Populations,”&lt;/span&gt; &lt;em&gt;Proceedings of the National Academy of Sciences&lt;/em&gt; 108, no. 29 (2011): 11983–88, &lt;a href="https://doi.org/10.1073/pnas.1019276108" role="doc-biblioref"&gt;https://doi.org/10.1073/pnas.1019276108&lt;/a&gt;; 1000 Genomes Project Consortium et al., &lt;span&gt;“A Global Reference for Human Genetic Variation,”&lt;/span&gt; &lt;em&gt;Nature&lt;/em&gt; 526, no. 7571 (2015): 68.&lt;/p&gt;'><sup>316</sup></a></span> This is particularly relevant for African populations, as they have a much greater level of genetic variance than other populations due to the human species having originated on that continent.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Consortium et al., &lt;span&gt;“A Global Reference for Human Genetic Variation.”&lt;/span&gt;&lt;/p&gt;"><sup>317</sup></a></span> Therefore, if GWAS aren’t performed on diverse populations, PGS can’t take into account the rare variants present in non-European populations that are likely to exert stronger effects on the trait of interest. There are also several other issues that can affect the discoverability of causal variants through GWAS, including the technology used for genotyping, the selection of the cohort, and the necessary exclusion of genotypic outliers.
With respect to genotyping technologies, GWAS often use data from SNP microarrays. These do not sequence the whole genome, but rather a selection (from several hundred thousand to millions) of genetic markers intended to present <em>common</em> genetic variation,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Eleonora Porcu et al., &lt;span&gt;“Genotype &lt;span&gt;Imputation&lt;/span&gt; in &lt;span&gt;Genome-Wide Association Studies&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Current Protocols in Human Genetics&lt;/em&gt; 78, no. 1 (2013): 1.25.1–14, &lt;a href="https://doi.org/10.1002/0471142905.hg0125s78" role="doc-biblioref"&gt;https://doi.org/10.1002/0471142905.hg0125s78&lt;/a&gt;.&lt;/p&gt;'><sup>318</sup></a></span> which accordingly tend to neglect rare genetic variants.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Uffelmann et al., &lt;span&gt;“Genome-Wide Association Studies.”&lt;/span&gt;&lt;/p&gt;"><sup>319</sup></a></span> To increase the density of genotypes, which would increase the likelihood of refining the association signal and identifying causal variants, researchers often “impute” variants that aren’t sequenced directly.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Porcu et al., &lt;span&gt;“Genotype &lt;span&gt;Imputation&lt;/span&gt; in &lt;span&gt;Genome-Wide Association Studies&lt;/span&gt;.”&lt;/span&gt;&lt;/p&gt;"><sup>320</sup></a></span> The imputation process involves “phasing” the study genotypes onto the genotypes of a “reference panel.”<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Shane McCarthy et al., &lt;span&gt;“A Reference Panel of 64,976 Haplotypes for Genotype Imputation,”&lt;/span&gt; &lt;em&gt;Nature Genetics&lt;/em&gt; 48, no. 10, 10 (October 2016): 1279–83, &lt;a href="https://doi.org/10.1038/ng.3643" role="doc-biblioref"&gt;https://doi.org/10.1038/ng.3643&lt;/a&gt;.&lt;/p&gt;'><sup>321</sup></a></span> However, if the reference panel does not sufficiently represent the population in the study sample, they are likely to miss or incorrectly impute those genotypes.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Martin et al., &lt;span&gt;“Clinical Use of Current Polygenic Risk Scores May Exacerbate Health Disparities.”&lt;/span&gt;&lt;/p&gt;"><sup>322</sup></a></span> Again, this is particularly problematic for African populations.</p>
<p>The lack of representation of rare variants in SNP microarrays can be overcome by using next-generation sequencing technologies such as whole-genome sequencing (<strong>WGS</strong>) and whole-exome sequencing (<strong>WES</strong>). (The former seeks to sequence the full genome, and the latter of only targets the coding regions of the genome.) These methods are more expensive than SNP microarrays, which hinders their widespread use at scale, and although their costs are continuing to decrease rapidly, there is a question as to whether they return a proportionate benefit in all use cases.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Katharina Schwarze et al., &lt;span&gt;“Are Whole-Exome and Whole-Genome Sequencing Approaches Cost-Effective? &lt;span&gt;A&lt;/span&gt; Systematic Review of the Literature,”&lt;/span&gt; &lt;em&gt;Genetics in Medicine&lt;/em&gt; 20, no. 10 (October 1, 2018): 1122–30, &lt;a href="https://doi.org/10.1038/gim.2017.247" role="doc-biblioref"&gt;https://doi.org/10.1038/gim.2017.247&lt;/a&gt;.&lt;/p&gt;'><sup>323</sup></a></span></p>
<p>A second limitation is the selection of GWAS cohorts, which can introduce selection and collider biases.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Uffelmann et al., &lt;span&gt;“Genome-Wide Association Studies.”&lt;/span&gt;&lt;/p&gt;"><sup>324</sup></a></span> For instance, the UK Biobank, which contains genetic and phenotypic data on 500,000 participants who volunteered for inclusion between 2006 and 2010, tend to be older, female, healthier, and wealthier than non-participants.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Anna Fry et al., &lt;span&gt;“Comparison of &lt;span&gt;Sociodemographic&lt;/span&gt; and &lt;span&gt;Health-Related Characteristics&lt;/span&gt; of &lt;span&gt;UK Biobank Participants With Those&lt;/span&gt; of the &lt;span&gt;General Population&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;American Journal of Epidemiology&lt;/em&gt; 186, no. 9 (November 1, 2017): 1026–34, &lt;a href="https://doi.org/10.1093/aje/kwx246" role="doc-biblioref"&gt;https://doi.org/10.1093/aje/kwx246&lt;/a&gt;.&lt;/p&gt;'><sup>325</sup></a></span> This creaties the possibility of confounding genetic associations with environmental factors, which I discuss further in <a href="Fst-chap.html#fst-env-sec">6.1.2.4</a> below.</p>
<p>A third limitation is the “quality control” step that is required during the GWAS process.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Uffelmann et al., &lt;span&gt;“Genome-Wide Association Studies.”&lt;/span&gt;&lt;/p&gt;"><sup>326</sup></a></span> To avoid confounding from population stratification, which can lead to overestimated heritability and biased PGS, GWAS cohorts are filtered to include only those with similar ancestries – or relative genetic homogeneity – by clustering individuals through principal component analysis (<strong>PCA</strong>) of their genotypes, and excluding outliers. I elaborate on the issue of population stratification in section <a href="Fst-chap.html#fst-env-sec">6.1.2.4</a> below, but at present, a statistical model for GWAS that can include cohorts with diverse ancestries without the risk of serious confounding is yet to be developed.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Jeremy J Berg et al., &lt;span&gt;“Reduced Signal for Polygenic Adaptation of Height in &lt;span&gt;UK Biobank&lt;/span&gt;,”&lt;/span&gt; ed. Magnus Nordborg et al., &lt;em&gt;eLife&lt;/em&gt; 8 (March 21, 2019): e39725, &lt;a href="https://doi.org/10.7554/eLife.39725" role="doc-biblioref"&gt;https://doi.org/10.7554/eLife.39725&lt;/a&gt;.&lt;/p&gt;'><sup>327</sup></a></span></p>
</div>
<div id="differences-in-ld" class="section level4" number="6.1.2.2">
<h4>
<span class="header-section-number">6.1.2.2</span> Differences in LD<a class="anchor" aria-label="anchor" href="#differences-in-ld"><i class="fas fa-link"></i></a>
</h4>
<p>Because GWAS SNP markers are often not the causal variants themselves, but merely in physical proximity to them, the estimated effect size of a SNP marker depends on the extent to which it is in LD with the causal variant.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Hakhamanesh Mostafavi et al., &lt;span&gt;“Variable Prediction Accuracy of Polygenic Scores Within an Ancestry Group,”&lt;/span&gt; ed. Ruth Loos, Michael B Eisen, and Paul O’Reilly, &lt;em&gt;eLife&lt;/em&gt; 9 (January 30, 2020): e48376, &lt;a href="https://doi.org/10.7554/eLife.48376" role="doc-biblioref"&gt;https://doi.org/10.7554/eLife.48376&lt;/a&gt;; Jonathan K. Pritchard and Molly Przeworski, &lt;span&gt;“Linkage &lt;span&gt;Disequilibrium&lt;/span&gt; in &lt;span&gt;Humans&lt;/span&gt;: &lt;span&gt;Models&lt;/span&gt; and &lt;span&gt;Data&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;The American Journal of Human Genetics&lt;/em&gt; 69, no. 1 (July 1, 2001): 1–14, &lt;a href="https://doi.org/10.1086/321275" role="doc-biblioref"&gt;https://doi.org/10.1086/321275&lt;/a&gt;.&lt;/p&gt;'><sup>328</sup></a></span> To illustrate the problem, if a SNP has an LD <span class="math inline">\(r^2\)</span> with a causal variant of 0.8 in the discovery population and 0.6 in the target population, it would explain 25% = (1 - 0.6/0.8) less trait variation in the target population, and would therefore be less predictive.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Ying Wang et al., &lt;span&gt;“Theoretical and Empirical Quantification of the Accuracy of Polygenic Scores in Ancestry Divergent Populations,”&lt;/span&gt; &lt;em&gt;Nature Communications&lt;/em&gt; 11, no. 1, 1 (July 31, 2020): 3865, &lt;a href="https://doi.org/10.1038/s41467-020-17719-y" role="doc-biblioref"&gt;https://doi.org/10.1038/s41467-020-17719-y&lt;/a&gt;.&lt;/p&gt;'><sup>329</sup></a></span></p>
<p>These differences in effect-size estimates may typically be small for most regions of the genome, but as PGS sum across all such effects, they aggregate these population differences.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Martin et al., &lt;span&gt;“Clinical Use of Current Polygenic Risk Scores May Exacerbate Health Disparities”&lt;/span&gt;; Berg et al., &lt;span&gt;“Reduced Signal for Polygenic Adaptation of Height in &lt;span&gt;UK Biobank&lt;/span&gt;.”&lt;/span&gt;&lt;/p&gt;"><sup>330</sup></a></span> Previous empirical and simulation studies have shown that accuracy of PGS scores decay with increased genetic differentiation (<span class="math inline">\(F_{ST}\)</span> – described below in <a href="Fst-chap.html#Fst-descr">6.1.3</a>) and LD differences between populations.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Habier et al., &lt;span&gt;“The Impact of Genetic Relationship Information on Genomic Breeding Values in &lt;span&gt;German Holstein&lt;/span&gt; Cattle”&lt;/span&gt;; Pszczola et al., &lt;span&gt;“Reliability of Direct Genomic Values for Animals with Different Relationships Within and to the Reference Population”&lt;/span&gt;; Scutari, Mackay, and Balding, &lt;span&gt;“Using &lt;span&gt;Genetic Distance&lt;/span&gt; to &lt;span&gt;Infer&lt;/span&gt; the &lt;span&gt;Accuracy&lt;/span&gt; of &lt;span&gt;Genomic Prediction&lt;/span&gt;”&lt;/span&gt;; Wang et al., &lt;span&gt;“Theoretical and Empirical Quantification of the Accuracy of Polygenic Scores in Ancestry Divergent Populations.”&lt;/span&gt;&lt;/p&gt;"><sup>331</sup></a></span> The issue may be addressed to a degree by using LD information from an external reference panel as a prior to infer the posterior mean effect size of a genetic variant – <span class="citation">Bjarni J Vilhjálmsson et al.<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;&lt;span&gt;“Modeling Linkage Disequilibrium Increases Accuracy of Polygenic Risk Scores,”&lt;/span&gt; &lt;em&gt;The American Journal of Human Genetics&lt;/em&gt; 97, no. 4 (2015): 576–92.&lt;/p&gt;"><sup>332</sup></a></span> demonstrated through simulations that this could improve PGS predictive accuracy. Yet the most appropriate means of deal with differences between populations in LD remains an active area of research.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Duncan et al., &lt;span&gt;“Analysis of Polygenic Risk Score Usage and Performance in Diverse Human Populations.”&lt;/span&gt;&lt;/p&gt;"><sup>333</sup></a></span></p>
</div>
<div id="differences-in-allele-frequencies" class="section level4" number="6.1.2.3">
<h4>
<span class="header-section-number">6.1.2.3</span> Differences in allele frequencies<a class="anchor" aria-label="anchor" href="#differences-in-allele-frequencies"><i class="fas fa-link"></i></a>
</h4>
<p>Causal variants can differ in both frequency and effect size between different ancestry groups, e.g. for lactase persistence,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Laure S’egurel and C’eline Bon, &lt;span&gt;“On the &lt;span&gt;Evolution&lt;/span&gt; of &lt;span&gt;Lactase Persistence&lt;/span&gt; in &lt;span&gt;Humans&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Annual Review of Genomics and Human Genetics&lt;/em&gt; 18 (August 1, 2017): 297–319, &lt;a href="https://doi.org/10.1146/annurev-genom-091416-035340" role="doc-biblioref"&gt;https://doi.org/10.1146/annurev-genom-091416-035340&lt;/a&gt;.&lt;/p&gt;'><sup>334</sup></a></span> or skin pigmentation.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Kaustubh Adhikari et al., &lt;span&gt;“A &lt;span&gt;GWAS&lt;/span&gt; in &lt;span&gt;Latin Americans&lt;/span&gt; Highlights the Convergent Evolution of Lighter Skin Pigmentation in &lt;span&gt;Eurasia&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Nature Communications&lt;/em&gt; 10, no. 1, 1 (January 21, 2019): 358, &lt;a href="https://doi.org/10.1038/s41467-018-08147-0" role="doc-biblioref"&gt;https://doi.org/10.1038/s41467-018-08147-0&lt;/a&gt;.&lt;/p&gt;'><sup>335</sup></a></span> If a causal allele is rare in the GWAS discovery population, even if it is discovered (see <a href="Fst-chap.html#fst-discovery-sec">6.1.2.1</a>), it is likely to have noisy effect size estimates, and therefore likely to inaccurately estimate its effect size in a different population where it exists at a higher frequency.</p>
<p>Differences in allele frequencies between populations can arise through random genetic drift, or be driven by selective pressures towards the trait optima for a given environment.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Arbel Harpak and Molly Przeworski, &lt;span&gt;“The Evolution of Group Differences in Changing Environments,”&lt;/span&gt; &lt;em&gt;PLOS Biology&lt;/em&gt; 19, no. 1 (January 25, 2021): e3001072, &lt;a href="https://doi.org/10.1371/journal.pbio.3001072" role="doc-biblioref"&gt;https://doi.org/10.1371/journal.pbio.3001072&lt;/a&gt;.&lt;/p&gt;'><sup>336</sup></a></span> However, evolutionary biologists have found that differences between populations in the mean values for traits tend to occur through small, coordinated shifts in their allele frequencies.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Berg et al., &lt;span&gt;“Reduced Signal for Polygenic Adaptation of Height in &lt;span&gt;UK Biobank&lt;/span&gt;”&lt;/span&gt;; Michael D Edge and Graham Coop, &lt;span&gt;“Reconstructing the &lt;span&gt;History&lt;/span&gt; of &lt;span&gt;Polygenic Scores Using Coalescent Trees&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Genetics&lt;/em&gt; 211, no. 1 (January 1, 2019): 235–62, &lt;a href="https://doi.org/10.1534/genetics.118.301687" role="doc-biblioref"&gt;https://doi.org/10.1534/genetics.118.301687&lt;/a&gt;.&lt;/p&gt;'><sup>337</sup></a></span> In Chapter <a href="Fst-chap.html#Fst-chap">6</a>, I explore the differences in allele frequencies across populations for all polygenic traits in the GWAS Catalog, and confirm that with few exceptions – including skin pigmentation, and HIV viral load – the differences in allele frequencies between populations tends to be small.</p>
</div>
<div id="fst-env-sec" class="section level4" number="6.1.2.4">
<h4>
<span class="header-section-number">6.1.2.4</span> Differences in environment<a class="anchor" aria-label="anchor" href="#fst-env-sec"><i class="fas fa-link"></i></a>
</h4>
<p>Genes continuously interact with each other (GxG, or “epistasis”),<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Pierre-Alexis Gros, Herv’e Le Nagard, and Olivier Tenaillon, &lt;span&gt;“The &lt;span&gt;Evolution&lt;/span&gt; of &lt;span&gt;Epistasis&lt;/span&gt; and &lt;span&gt;Its Links With Genetic Robustness&lt;/span&gt;, &lt;span&gt;Complexity&lt;/span&gt; and &lt;span&gt;Drift&lt;/span&gt; in a &lt;span&gt;Phenotypic Model&lt;/span&gt; of &lt;span&gt;Adaptation&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Genetics&lt;/em&gt; 182, no. 1 (May 1, 2009): 277–93, &lt;a href="https://doi.org/10.1534/genetics.108.099127" role="doc-biblioref"&gt;https://doi.org/10.1534/genetics.108.099127&lt;/a&gt;.&lt;/p&gt;'><sup>338</sup></a></span> the genes of one’s parents (“genetic nurture”, <span class="citation">Kong et al.<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;&lt;span&gt;“The Nature of Nurture.”&lt;/span&gt;&lt;/p&gt;"><sup>339</sup></a></span>) or social companions (“social genetic effects”),<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Benjamin W. Domingue et al., &lt;span&gt;“The Social Genome of Friends and Schoolmates in the &lt;span&gt;National Longitudinal Study&lt;/span&gt; of &lt;span&gt;Adolescent&lt;/span&gt; to &lt;span&gt;Adult Health&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Proceedings of the National Academy of Sciences&lt;/em&gt; 115, no. 4 (January 23, 2018): 702–7, &lt;a href="https://doi.org/10.1073/pnas.1711803115" role="doc-biblioref"&gt;https://doi.org/10.1073/pnas.1711803115&lt;/a&gt;; Baud et al., &lt;span&gt;“Genetic &lt;span&gt;Variation&lt;/span&gt; in the &lt;span&gt;Social Environment Contributes&lt;/span&gt; to &lt;span&gt;Health&lt;/span&gt; and &lt;span&gt;Disease&lt;/span&gt;.”&lt;/span&gt;&lt;/p&gt;'><sup>340</sup></a></span><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;As also explored in Chapters &lt;a href="Pilot-chap.html#Pilot-chap"&gt;3&lt;/a&gt; and &lt;a href="MIKK-F2-chap.html#MIKK-F2-chap"&gt;5&lt;/a&gt;&lt;/p&gt;'><sup>341</sup></a> and the wider non-genetic environment (GxE).</p>
<p>The respective contributions of genetics and environment to traits with social value – such as intelligence – is highly contentious, especially when there are apparent differences between populations in the mean values for those traits. PGS measure the proportion of variance within a population that is explained by genetics. Because PRS summarises a <em>proportion</em> of the total variance, when studying a population that is subject to greater environmental variation, the variance attributable to genetic factors will proportionately reduce. The corollary being that when studying a population where the environment is held constant, the proportion of variance for that trait that is explained by genetic factors will approach 1. Therefore, increases in the amount of environmental variance that a population is exposed to will reduce the accuracy of PGS predictions when applied to that population.</p>
<p>Different environments are also often correlated with population structure.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Berg et al., &lt;span&gt;“Reduced Signal for Polygenic Adaptation of Height in &lt;span&gt;UK Biobank&lt;/span&gt;.”&lt;/span&gt;&lt;/p&gt;"><sup>342</sup></a></span> For example, in East Asia, there is a greater proportion of individuals of East-Asian ancestry than there is of European ancestry, and <em>vice versa</em> in Europe. Those East-Asian individuals will therefore tend to share more of their genetic background with each other than with Europeans, and that population structure will be correlated with the different environments that exist in East Asia compared to Europe. This makes it difficult to determine whether it is the differences in their environments or the differences in their genetics that is driving the discrepancies between the mean values for traits between those populations. These complexities are unlikely to be resolved in the near future, which makes it attractive to turn to model organisms to address more basic biological questions regarding GxE in relation to complex traits,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Leif Andersson and Michel Georges, &lt;span&gt;“Domestic-Animal Genomics: Deciphering the Genetics of Complex Traits,”&lt;/span&gt; &lt;em&gt;Nature Reviews Genetics&lt;/em&gt; 5, no. 3, 3 (March 2004): 202–12, &lt;a href="https://doi.org/10.1038/nrg1294" role="doc-biblioref"&gt;https://doi.org/10.1038/nrg1294&lt;/a&gt;.&lt;/p&gt;'><sup>343</sup></a></span> as we have done with respect to behaviour in Chapters <a href="Pilot-chap.html#Pilot-chap">3</a> and <a href="MIKK-F2-chap.html#MIKK-F2-chap">5</a>.</p>
</div>
</div>
<div id="Fst-descr" class="section level3" number="6.1.3">
<h3>
<span class="header-section-number">6.1.3</span> <span class="math inline">\(F_{ST}\)</span><a class="anchor" aria-label="anchor" href="#Fst-descr"><i class="fas fa-link"></i></a>
</h3>
<p>The widely-used fixation index (<span class="math inline">\(F_{ST}\)</span>) was introduced independently by Sewall Wright<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Sewall Wright, &lt;span&gt;“The &lt;span&gt;Genetical Structure&lt;/span&gt; of &lt;span&gt;Populations&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Annals of Eugenics&lt;/em&gt; 15, no. 1 (1949): 323–54, &lt;a href="https://doi.org/10.1111/j.1469-1809.1949.tb02451.x" role="doc-biblioref"&gt;https://doi.org/10.1111/j.1469-1809.1949.tb02451.x&lt;/a&gt;.&lt;/p&gt;'><sup>344</sup></a></span> and Gustave Malécot<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Gustave Mal’ecot, &lt;span&gt;“Mathématiques de l’hérédité,”&lt;/span&gt; 1948.&lt;/p&gt;"><sup>345</sup></a></span> as a metric for measuring the genetic diversity between populations.<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;In Wright’s notation, &lt;span class="math inline"&gt;\(F\)&lt;/span&gt; refers to “fixation” of an allele, and &lt;span class="math inline"&gt;\(_{ST}\)&lt;/span&gt; refers to “subpopulations within the total population”.&lt;/p&gt;'><sup>346</sup></a> It quantifies the relative variance in allele frequency between groups compared to within groups, reflecting the combined effects of genetic drift, migration, mutation, and selection.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Kent E. Holsinger and Bruce S. Weir, &lt;span&gt;“Genetics in Geographically Structured Populations: Defining, Estimating and Interpreting &lt;span&gt;FST&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Nature Reviews Genetics&lt;/em&gt; 10, no. 9, 9 (September 2009): 639–50, &lt;a href="https://doi.org/10.1038/nrg2611" role="doc-biblioref"&gt;https://doi.org/10.1038/nrg2611&lt;/a&gt;.&lt;/p&gt;'><sup>347</sup></a></span> The metric ranges from 0 to 1, where loci with high <span class="math inline">\(F_{ST}\)</span> values – that is, loci with a large relative between-group variance in allele frequencies – may have been subject to selection or different demographic processes.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Holsinger and Weir.&lt;/p&gt;"><sup>348</sup></a></span> The metric has customarily been used to identify regions of the genome have been subject to diversifying selection.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Joshua M. Akey et al., &lt;span&gt;“Interrogating a &lt;span&gt;High-Density SNP Map&lt;/span&gt; for &lt;span&gt;Signatures&lt;/span&gt; of &lt;span&gt;Natural Selection&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Genome Research&lt;/em&gt; 12, no. 12 (January 12, 2002): 1805–14, &lt;a href="https://doi.org/10.1101/gr.631202" role="doc-biblioref"&gt;https://doi.org/10.1101/gr.631202&lt;/a&gt;; Feng Guo, Dipak K. Dey, and Kent E. Holsinger, &lt;span&gt;“A &lt;span&gt;Bayesian Hierarchical Model&lt;/span&gt; for &lt;span&gt;Analysis&lt;/span&gt; of &lt;span&gt;Single-Nucleotide Polymorphisms Diversity&lt;/span&gt; in &lt;span&gt;Multilocus&lt;/span&gt;, &lt;span&gt;Multipopulation Samples&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Journal of the American Statistical Association&lt;/em&gt; 104, no. 485 (March 1, 2009): 142–54, &lt;a href="https://doi.org/10.1198/jasa.2009.0010" role="doc-biblioref"&gt;https://doi.org/10.1198/jasa.2009.0010&lt;/a&gt;; Bruce S. Weir et al., &lt;span&gt;“Measures of Human Population Structure Show Heterogeneity Among Genomic Regions,”&lt;/span&gt; &lt;em&gt;Genome Research&lt;/em&gt; 15, no. 11 (January 11, 2005): 1468–76, &lt;a href="https://doi.org/10.1101/gr.4398405" role="doc-biblioref"&gt;https://doi.org/10.1101/gr.4398405&lt;/a&gt;.&lt;/p&gt;'><sup>349</sup></a></span></p>
<p>I first sought to explore the distribution of <span class="math inline">\(F_{ST}\)</span> for SNPs across the human genome. As the reference for human genomic variation across diverse populations, I used the New York Genome Center high-coverage, phased .vcf files<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;&lt;span&gt;“Index of /Vol1/Ftp/Data_collections/&lt;span&gt;1000g&lt;/span&gt;_2504_high_coverage/Working/20201028_3202_phased/,”&lt;/span&gt; accessed March 24, 2022, &lt;a href="http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/data_collections/1000G_2504_high_coverage/working/20201028_3202_phased/" role="doc-biblioref"&gt;http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/data_collections/1000G_2504_high_coverage/working/20201028_3202_phased/&lt;/a&gt;.&lt;/p&gt;'><sup>350</sup></a></span> for the 2,504 individuals from 26 populations from across the globe, as described in the 1000 Genomes phase 3 release.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Consortium et al., &lt;span&gt;“A Global Reference for Human Genetic Variation.”&lt;/span&gt;&lt;/p&gt;"><sup>351</sup></a></span> I then annotated those .vcf files with human SNP IDs from dbSNP release 9606,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Elizabeth M. Smigielski et al., &lt;span&gt;“&lt;span class="nocase"&gt;dbSNP&lt;/span&gt;: A Database of Single Nucleotide Polymorphisms,”&lt;/span&gt; &lt;em&gt;Nucleic Acids Research&lt;/em&gt; 28, no. 1 (January 1, 2000): 352–55, &lt;a href="https://doi.org/10.1093/nar/28.1.352" role="doc-biblioref"&gt;https://doi.org/10.1093/nar/28.1.352&lt;/a&gt;.&lt;/p&gt;'><sup>352</sup></a></span> and calculated <span class="math inline">\(F_{ST}\)</span> for each of the ~69M SNPs in that dataset with PLINK 1.9.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Chang et al., &lt;span&gt;“Second-Generation &lt;span&gt;PLINK&lt;/span&gt;”&lt;/span&gt;; Purcell and Chang, &lt;em&gt;&lt;span&gt;PLINK&lt;/span&gt; 1.9&lt;/em&gt;.&lt;/p&gt;"><sup>353</sup></a></span> Figure <a href="Fst-chap.html#fig:fst-circos">6.1</a> shows the location and <span class="math inline">\(F_{ST}\)</span> value for all SNPs in the 1000 Genomes dataset, and Figure <a href="Fst-chap.html#fig:fst-histo">6.2</a> shows their distribution across the range of <span class="math inline">\(F_{ST}\)</span> values.</p>
<div class="figure">
<span style="display:block;" id="fig:fst-circos"></span>
<img src="figs/fst/circos_all_fst.png" alt="(ref:fst-circos)" width="100%"><p class="caption">
Figure 6.1: (ref:fst-circos)
</p>
</div>
<div class="figure">
<span style="display:block;" id="fig:fst-histo"></span>
<img src="figs/fst/histo_fst_all.png" alt="(ref:fst-histo)" width="100%"><p class="caption">
Figure 6.2: (ref:fst-histo)
</p>
</div>
<p>Figure <a href="Fst-chap.html#fig:fst-circos">6.1</a> shows that with very few exceptions, <span class="math inline">\(F_{ST}\)</span> remains below 0.6, although specific regions do appear to be differentially selected, as indicated be the peaks on chromosomes 1, 2, and 17. Figure <a href="Fst-chap.html#fig:fst-histo">6.2</a> makes it clear that the vast majority of SNPs have very low <span class="math inline">\(F_{ST}\)</span>, with a genome-wide mean of 0.019. The question then is whether the SNPs that have a detectable effect on traits tend to have higher <span class="math inline">\(F_{ST}\)</span> values – and therefore greater variation in allele frequencies between populations – than random SNPs with similar allele frequencies in European populations. If so, this would suggest that variation in allele frequencies might contribute significantly to the non-transferability of PRS scores derived from European-focused GWAS.</p>
</div>
</div>
<div id="Fst-analysis-chap" class="section level2" number="6.2">
<h2>
<span class="header-section-number">6.2</span> Analysis<a class="anchor" aria-label="anchor" href="#Fst-analysis-chap"><i class="fas fa-link"></i></a>
</h2>
<p>In this analysis I explore the distribution of <span class="math inline">\(F_{ST}\)</span> scores for loci associated with 587 traits, a subset of the GWAS Catalog that passed our criteria for suitable polygenic traits (see section <a href="Fst-chap.html#Fst-analysis-chap">6.2</a>). Using high-coverage sequence data for 2,504 individuals from the 1000 Genomes Project phase 3 release, for each trait in the GWAS Catalog I calculated the distribution of <span class="math inline">\(F_{ST}\)</span> across all approximately-unlinked SNPs associated with it (<strong>trait SNPs</strong>), and compared these <span class="math inline">\(F_{ST}\)</span> distributions with the <span class="math inline">\(F_{ST}\)</span> distributions of random-selected SNPs that were matched to the trait SNPs by their allele frequencies in European populations (<strong>control SNPs</strong>).</p>
<div id="gwas-catalog" class="section level4" number="6.2.0.1">
<h4>
<span class="header-section-number">6.2.0.1</span> GWAS Catalog<a class="anchor" aria-label="anchor" href="#gwas-catalog"><i class="fas fa-link"></i></a>
</h4>
<p>I used the R package <em>gwasrapidd</em><span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Ramiro Magno and Ana-Teresa Maia, &lt;span&gt;“Gwasrapidd: An &lt;span&gt;R&lt;/span&gt; Package to Query, Download and Wrangle &lt;span&gt;GWAS&lt;/span&gt; Catalog Data,”&lt;/span&gt; &lt;em&gt;Bioinformatics&lt;/em&gt; 36, no. 2 (January 15, 2020): 649–50, &lt;a href="https://doi.org/10.1093/bioinformatics/btz605" role="doc-biblioref"&gt;https://doi.org/10.1093/bioinformatics/btz605&lt;/a&gt;.&lt;/p&gt;'><sup>354</sup></a></span> to query all traits in the GWAS Catalog<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Jacqueline MacArthur et al., &lt;span&gt;“The New &lt;span&gt;NHGRI-EBI Catalog&lt;/span&gt; of Published Genome-Wide Association Studies (&lt;span&gt;GWAS Catalog&lt;/span&gt;),”&lt;/span&gt; &lt;em&gt;Nucleic Acids Research&lt;/em&gt; 45, no. D1 (January 4, 2017): D896–901, &lt;a href="https://doi.org/10.1093/nar/gkw1133" role="doc-biblioref"&gt;https://doi.org/10.1093/nar/gkw1133&lt;/a&gt;.&lt;/p&gt;'><sup>355</sup></a></span> as of 9 August 2021 (<span class="math inline">\(N_{TRAITS}\)</span> = 3,459). For 541 of these traits, no matching variant IDs could be pulled out from the 1000 Genomes VCFs, leaving <span class="math inline">\(N_{TRAITS}\)</span> = 3,008.</p>
</div>
<div id="linkage-disequilibrium" class="section level3" number="6.2.1">
<h3>
<span class="header-section-number">6.2.1</span> Linkage disequilibrium<a class="anchor" aria-label="anchor" href="#linkage-disequilibrium"><i class="fas fa-link"></i></a>
</h3>
<p>To obtain the “trait SNP” dataset, for each trait, I sought to isolate the SNP closest to each of its true causal variants, and exclude the SNPs in LD with them. To this end, I used PLINK 1.9<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Chang et al., &lt;span&gt;“Second-Generation &lt;span&gt;PLINK&lt;/span&gt;”&lt;/span&gt;; Purcell and Chang, &lt;em&gt;&lt;span&gt;PLINK&lt;/span&gt; 1.9&lt;/em&gt;.&lt;/p&gt;"><sup>356</sup></a></span> to “clump” the SNPs associated with each of the remaining 3,008 traits, using an “index” SNP p-value threshold of <span class="math inline">\(10^{-8}\)</span>,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Orestis A. Panagiotou, John P. A. Ioannidis, and Genome-Wide Significance Project, &lt;span&gt;“What Should the Genome-Wide Significance Threshold Be? &lt;span&gt;Empirical&lt;/span&gt; Replication of Borderline Genetic Associations,”&lt;/span&gt; &lt;em&gt;International Journal of Epidemiology&lt;/em&gt; 41, no. 1 (February 2012): 273–86, &lt;a href="https://doi.org/10.1093/ije/dyr178" role="doc-biblioref"&gt;https://doi.org/10.1093/ije/dyr178&lt;/a&gt;.&lt;/p&gt;'><sup>357</sup></a></span> <span class="math inline">\(r^2\)</span> threshold of 0.1,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;W. G. Hill and Alan Robertson, &lt;span&gt;“Linkage Disequilibrium in Finite Populations,”&lt;/span&gt; &lt;em&gt;Theoretical and Applied Genetics&lt;/em&gt; 38, no. 6 (June 1, 1968): 226–31, &lt;a href="https://doi.org/10.1007/BF01245622" role="doc-biblioref"&gt;https://doi.org/10.1007/BF01245622&lt;/a&gt;.&lt;/p&gt;'><sup>358</sup></a></span> and base window size of 1 Mb. This process revealed 2,045 traits with at least one index SNP that met the p-value threshold. The index SNPs for each trait formed the set of trait SNPs, and <strong>Figure <a href="Fst-chap.html#fig:FstSnpCount">6.3</a></strong> shows the counts of unique SNP IDs associated with each trait before and after clumping. In order to target relatively polygenic traits, I further filtered out traits with fewer than 10 trait SNPs, leaving <span class="math inline">\(N_{TRAITS}\)</span> = 587.</p>

<div class="figure">
<span style="display:block;" id="fig:FstSnpCount"></span>
<img src="figs/fst/0.1_1000_20220314_snp_counts.png" alt="log10 counts of associated SNPs for each trait before and after the clumping process, which involved: a) excluding all SNPs with a p-value greater than \(10^{-8}\); and b) starting with the SNPs with the lowest p-values (“index” SNPs), excluding all other SNPs within a 1 Mb region of the index SNP with an LD \(r^2\) of more than 0.1. [CHANGE COLOUR]" width="100%"><p class="caption">
Figure 6.3: log<sub>10</sub> counts of associated SNPs for each trait before and after the clumping process, which involved: a) excluding all SNPs with a <em>p</em>-value greater than <span class="math inline">\(10^{-8}\)</span>; and b) starting with the SNPs with the lowest <em>p</em>-values (“index” SNPs), excluding all other SNPs within a 1 Mb region of the index SNP with an LD <span class="math inline">\(r^2\)</span> of more than 0.1. [CHANGE COLOUR]
</p>
</div>
</div>
<div id="control-snps" class="section level3" number="6.2.2">
<h3>
<span class="header-section-number">6.2.2</span> Control SNPs<a class="anchor" aria-label="anchor" href="#control-snps"><i class="fas fa-link"></i></a>
</h3>
<p>To obtain the “control SNP” dataset, I assigned each trait SNP to one of 20 bins based on its minor allele frequency in European populations (as provided in the original 1000 Genomes .vcf files under the column header ‘INFO/AC_EUR’). For example, if a trait SNP had a minor allele frequency of 0.08 in European populations, it was assigned to the <span class="math inline">\((0.05, 0.1]\)</span> bin. I did the same for all (un-associated) SNPs in the .vcf files, then paired each trait SNP with a random SNP from the .vcf file in the equivalent bin. These allele-frequency-paired random SNPs formed the set of “control SNPs”, which I used to infer the <span class="math inline">\(F_{ST}\)</span> distribution of a random set of SNPs with the same allele frequencies as the trait SNPs, and against which I could compare the <span class="math inline">\(F_{ST}\)</span> distribution of the trait SNPs.</p>
</div>
<div id="f_st-and-ranking-traits-by-signed-kolmogorov-smirnov-d-statistic" class="section level3" number="6.2.3">
<h3>
<span class="header-section-number">6.2.3</span> <span class="math inline">\(F_{ST}\)</span> and ranking traits by signed Kolmogorov-Smirnov <span class="math inline">\(D\)</span> statistic<a class="anchor" aria-label="anchor" href="#f_st-and-ranking-traits-by-signed-kolmogorov-smirnov-d-statistic"><i class="fas fa-link"></i></a>
</h3>
<p>I then calculated <span class="math inline">\(F_{ST}\)</span> for each of the trait SNPs and their matched control SNPs using the Weir and Cockerham method,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;B. S. Weir and C. Clark Cockerham, &lt;span&gt;“Estimating &lt;span&gt;F-Statistics&lt;/span&gt; for the &lt;span&gt;Analysis&lt;/span&gt; of &lt;span&gt;Population Structure&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Evolution&lt;/em&gt; 38, no. 6 (1984): 1358–70, &lt;a href="https://doi.org/10.2307/2408641" role="doc-biblioref"&gt;https://doi.org/10.2307/2408641&lt;/a&gt;.&lt;/p&gt;'><sup>359</sup></a></span> as implemented in the R package <em>pegas</em>.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Emmanuel Paradis, &lt;span&gt;“Pegas: An &lt;span&gt;R&lt;/span&gt; Package for Population Genetics with an Integrated–Modular Approach,”&lt;/span&gt; &lt;em&gt;Bioinformatics&lt;/em&gt; 26, no. 3 (February 1, 2010): 419–20, &lt;a href="https://doi.org/10.1093/bioinformatics/btp696" role="doc-biblioref"&gt;https://doi.org/10.1093/bioinformatics/btp696&lt;/a&gt;.&lt;/p&gt;'><sup>360</sup></a></span> I then sought to rank all traits based on the directional difference in <span class="math inline">\(F_{ST}\)</span> distributions between trait and control SNPs. The Kolmogorov-Smirnov (KS) test is an appropriate test to measure the differences between distributions,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;W. J. Conover, &lt;em&gt;Practical &lt;span&gt;Nonparametric Statistics&lt;/span&gt;&lt;/em&gt; (&lt;span&gt;John Wiley &amp;amp; Sons&lt;/span&gt;, 1999), &lt;a href="https://books.google.com?id=n_39DwAAQBAJ" role="doc-biblioref"&gt;https://books.google.com?id=n_39DwAAQBAJ&lt;/a&gt;.&lt;/p&gt;'><sup>361</sup></a></span> but the <em>p</em>-values are strongly affected by the number of samples in each distribution (here, the number of loci in each trait), and it does not measure whether one distribution is overall higher or lower than the other distribution. These limitations made it impossible to use the KS test <em>p</em>-values to compare the directional distances between trait and control SNPs across traits that have very different numbers of SNPs. I accordingly formulated the following method for ranking the traits based on the directional distance of the <span class="math inline">\(F_{ST}\)</span> distributions of the trait SNPs compared to the control SNPs:</p>
<p>First, I ran three KS tests for each trait <span class="math inline">\(t\)</span> with <span class="math inline">\(x_t\)</span> = <span class="math inline">\(F_{ST, trait SNPs}\)</span> and <span class="math inline">\(y_t\)</span> = <span class="math inline">\(F_{ST, control SNPs}\)</span>:</p>
<ol style="list-style-type: decimal">
<li><p>two-sided (<span class="math inline">\(D_t\)</span>) ;</p></li>
<li><p>one-sided “greater” (<span class="math inline">\(D_t^+\)</span>) ; and</p></li>
<li><p>one-sided “less” (<span class="math inline">\(D_t^-\)</span>).</p></li>
</ol>
<p>I note that <span class="math inline">\(D_{t} = max(D_t^+, D_t^-)\)</span>, where <span class="math inline">\(D_t^+\)</span> is the greatest vertical distance attained by the eCDF of <span class="math inline">\(x_t\)</span> over the eCDF of <span class="math inline">\(y_t\)</span>, and <span class="math inline">\(D_t^-\)</span> is the greatest vertical distance attained by the eCDF of <span class="math inline">\(y_t\)</span> over the eCDF of <span class="math inline">\(x_t\)</span>.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Conover; J. Durbin, &lt;em&gt;Distribution &lt;span&gt;Theory&lt;/span&gt; for &lt;span&gt;Tests Based&lt;/span&gt; on &lt;span&gt;Sample Distribution Function&lt;/span&gt;&lt;/em&gt; (&lt;span&gt;SIAM&lt;/span&gt;, 1973), &lt;a href="https://books.google.com?id=zAryCrT1IUYC" role="doc-biblioref"&gt;https://books.google.com?id=zAryCrT1IUYC&lt;/a&gt;.&lt;/p&gt;'><sup>362</sup></a></span> I then used a comparison of <span class="math inline">\(D_t^+\)</span> and <span class="math inline">\(D_t^-\)</span> to formulate a “signed D statistic” (<span class="math inline">\(D_t^S\)</span>), based on the logic that trait SNPs with a lower overall <span class="math inline">\(F_{ST}\)</span> than control SNPs tend to have a higher <span class="math inline">\(D\)</span> under the “greater” test than the “less” test, and vice versa.</p>
<p>Therefore, <span class="math inline">\({D_t^S}\)</span>:</p>
<p><span class="math display">\[
\begin{aligned}
D_t^- &gt; D_t^+ &amp;: &amp;-D_t \\
D_t^- = D_t^+ &amp;: &amp;0 \\
D_t^- &lt; D_t^+ &amp;: &amp;D_t \\
\end{aligned}
\]</span></p>
<p>In <strong>Figure <a href="Fst-chap.html#fig:FstMain">6.4</a></strong> I present the <span class="math inline">\(F_{ST}\)</span> distributions of trait SNPs for an illustrative subset of 28 human traits, ranked by <span class="math inline">\({D_t^S}\)</span> when compared with their matched control SNPs. <strong>Figure <a href="Fst-chap.html#fig:FstMain">6.4</a>A</strong> shows the densities of SNPs as a function of <span class="math inline">\(F_{ST}\)</span>, and <strong>Figure <a href="Fst-chap.html#fig:FstMain">6.4</a>B and C</strong> show their empirical Cumulative Distribution Functions (eCDFs). <strong>Figure <a href="Fst-chap.html#fig:FstMain">6.4</a>B</strong> includes the eCDFs of control SNPs in grey. eCDF figures for all 587 traits that passed our filters (Methods) are provided in the Appendix <a href="ecdf-of-all-polygenic-traits-in-the-gwas-catalog-ranked-by-d_ts.html#fig:eCDFall">F.1</a>.</p>

<div class="figure">
<span style="display:block;" id="fig:FstMain"></span>
<img src="figs/fst/0.1_1000_20220314_final.png" alt="Distributions of \(F_{ST}\) across 28 illustrative human traits, ranked by signed-D (Kolmogorov-Smirnov test) comparing trait and control SNPs. A. \(F_{ST}\) density ridge plots with SNP markers. B. Empirical Cumulative Distribution Functions (eCDFs) of \(F_{ST}\) for trait-associated (colour) and random control (grey) SNPs, faceted by trait. C. Consolidated eCDFs of trait-associated SNPs from (B). eCDFs for all traits are included in the Appendix." width="100%"><p class="caption">
Figure 6.4: Distributions of <span class="math inline">\(F_{ST}\)</span> across 28 illustrative human traits, ranked by signed-D (Kolmogorov-Smirnov test) comparing trait and control SNPs. <strong>A</strong>. <span class="math inline">\(F_{ST}\)</span> density ridge plots with SNP markers. <strong>B</strong>. Empirical Cumulative Distribution Functions (eCDFs) of <span class="math inline">\(F_{ST}\)</span> for trait-associated (colour) and random control (grey) SNPs, faceted by trait. <strong>C</strong>. Consolidated eCDFs of trait-associated SNPs from (B). eCDFs for all traits are included in the Appendix.
</p>
</div>
<p>These results show that for most traits, especially those that are highly polygenic, the <span class="math inline">\(F_{ST}\)</span> of their associated alleles tend to be low, and to differ little from their matched control SNPs. This suggests that the causal variants of polygenic traits are generally shared across global populations at similar frequencies. <span class="citation">Wang et al.<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;&lt;span&gt;“Theoretical and Empirical Quantification of the Accuracy of Polygenic Scores in Ancestry Divergent Populations.”&lt;/span&gt;&lt;/p&gt;"><sup>363</sup></a></span> determined through simulations that differences in LD and allele frequencies between populations can explain 70-80% of the loss of PGS relative accuracy for traits like body mass index and type 2 diabetes. This builds on the work of others [CITE PAPERS CITED BY BERG AND COOP AND ELABORATE]. As most of these variants have been discovered in European populations, the poor transferability of PGS between populations does not appear to be primarily driven by differences in the allele frequencies of these discovered variants, but perhaps rather by as-yet undiscovered variants of larger effect sizes in non-European populations, as well as differences between these populations in LD structure and environment.</p>
<p>There is a question as to whether the higher distributions of <span class="math inline">\(F_{ST}\)</span> observed for hair shape, eye colour, skin pigmentation, and HIV infection, are due to those variants being discovered in diverse populations, such as those in Africa. For instance, if the alleles were predominantly discovered through GWAS on African populations, which contain an outsized proportion of genetic variation, those alleles would be more likely to show lower frequencies in other populations, and consequently higher <span class="math inline">\(F_{ST}\)</span>. To examine the number of individuals from different ancestries that were used in these studies, I generated Figure <a href="Fst-chap.html#fig:fst-n-indivs">6.5</a>. This figure shows that GWAS performed for the more comprehensively-studied phenotypes such as body height, BMI, and educational attainment, included individuals from many diverse populations, whereas the GWAS on pigmentation-related traits tended to focus on European or Latin American populations, as well as those of African ancestry in some cases. As expected, the GWAS on HIV-related traits included a higher proportion of individuals of African ancestry, which may explain the higher levels of <span class="math inline">\(F_{ST}\)</span> observed there. I note that the GWAS Catalog data did not allow me to determine whether different studies on the same trait used the same cohorts (e.g. from UK Biobank), which would have the effect of inflating the counts shown in Figure <a href="Fst-chap.html#fig:fst-n-indivs">6.5</a>.</p>
<div class="figure">
<span style="display:block;" id="fig:fst-n-indivs"></span>
<img src="figs/fst/20220324_n_individuals.png" alt="(ref:fst-n-indivs)" width="100%"><p class="caption">
Figure 6.5: (ref:fst-n-indivs)
</p>
</div>
</div>
</div>
<div id="implications" class="section level2" number="6.3">
<h2>
<span class="header-section-number">6.3</span> Implications<a class="anchor" aria-label="anchor" href="#implications"><i class="fas fa-link"></i></a>
</h2>
<div id="limitation-of-f_st" class="section level3" number="6.3.1">
<h3>
<span class="header-section-number">6.3.1</span> Limitation of <span class="math inline">\(F_{ST}\)</span><a class="anchor" aria-label="anchor" href="#limitation-of-f_st"><i class="fas fa-link"></i></a>
</h3>
<p>The first relevant limitation of <span class="math inline">\(F_{ST}\)</span> is that it does not correspond to similarities or differences between populations in trait values. This is because <span class="math inline">\(F_{ST}\)</span> does not take into account the effect size or the direction of effect of the trait-associated allele, so for highly-polygenic traits like the ones shown here, <span class="math inline">\(F_{ST}\)</span> is almost entirely decoupled from the mean additive genetic value between populations.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Jeremy J. Berg and Graham Coop, &lt;span&gt;“A &lt;span&gt;Population Genetic Signal&lt;/span&gt; of &lt;span&gt;Polygenic Adaptation&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;PLOS Genetics&lt;/em&gt; 10, no. 8 (August 7, 2014): e1004412, &lt;a href="https://doi.org/10.1371/journal.pgen.1004412" role="doc-biblioref"&gt;https://doi.org/10.1371/journal.pgen.1004412&lt;/a&gt;.&lt;/p&gt;'><sup>364</sup></a></span> This means that even when a trait’s associated alleles have low <span class="math inline">\(F_{ST}\)</span>, as we see for most traits here, it does not follow that the population mean values for that trait would be the same due to genetic similarities between those populations. As <span class="citation">A. Kremer and V. Le Corre<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;&lt;span&gt;“Decoupling of Differentiation Between Traits and Their Underlying Genes in Response to Divergent Selection,”&lt;/span&gt; &lt;em&gt;Heredity&lt;/em&gt; 108, no. 4, 4 (April 2012): 375–85, &lt;a href="https://doi.org/10.1038/hdy.2011.81" role="doc-biblioref"&gt;https://doi.org/10.1038/hdy.2011.81&lt;/a&gt;.&lt;/p&gt;'><sup>365</sup></a></span> determined by through simulations, for highly polygenic traits, mean trait differences are driven by small, coordinated shifts in allele frequencies, which would not be detected by <span class="math inline">\(F_{ST}\)</span>.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Val’erie Le Corre and Antoine Kremer, &lt;span&gt;“The Genetic Differentiation at Quantitative Trait Loci Under Local Adaptation,”&lt;/span&gt; &lt;em&gt;Molecular Ecology&lt;/em&gt; 21, no. 7 (2012): 1548–66, &lt;a href="https://doi.org/10.1111/j.1365-294X.2012.05479.x" role="doc-biblioref"&gt;https://doi.org/10.1111/j.1365-294X.2012.05479.x&lt;/a&gt;.&lt;/p&gt;'><sup>366</sup></a></span></p>
<p>The second relevant limitation is that <span class="math inline">\(F_{ST}\)</span> is not sufficiently powered to detect selection for polygenic traits,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Jeremy J. Berg and Graham Coop, &lt;span&gt;“A &lt;span&gt;Population Genetic Signal&lt;/span&gt; of &lt;span&gt;Polygenic Adaptation&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;PLOS Genetics&lt;/em&gt; 10, no. 8 (August 7, 2014): e1004412, &lt;a href="https://doi.org/10.1371/journal.pgen.1004412" role="doc-biblioref"&gt;https://doi.org/10.1371/journal.pgen.1004412&lt;/a&gt;.&lt;/p&gt;'><sup>367</sup></a></span> which means that these results cannot be used to determine the extent to which alleles are under selection in a given population. This is apparent in our analysis, where in contrast with <span class="citation">Fernando Racimo, Jeremy J Berg, and Joseph K Pickrell<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;&lt;span&gt;“Detecting &lt;span&gt;Polygenic Adaptation&lt;/span&gt; in &lt;span&gt;Admixture Graphs&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Genetics&lt;/em&gt; 208, no. 4 (April 1, 2018): 1565–84, &lt;a href="https://doi.org/10.1534/genetics.117.300489" role="doc-biblioref"&gt;https://doi.org/10.1534/genetics.117.300489&lt;/a&gt;.&lt;/p&gt;'><sup>368</sup></a></span>, height and BMI, which have been found to be under strong differential selection, appear similar to other polygenic traits.</p>
</div>
<div id="the-importance-of-environment-and-a-push-for-diversity-in-genetic-studies" class="section level3" number="6.3.2">
<h3>
<span class="header-section-number">6.3.2</span> The importance of environment, and a push for diversity in genetic studies<a class="anchor" aria-label="anchor" href="#the-importance-of-environment-and-a-push-for-diversity-in-genetic-studies"><i class="fas fa-link"></i></a>
</h3>
<p>The obvious solution to this issue of non-transferability of PGS scores to diverse populations is to increase the representation of those populations in GWAS and PGS studies, as has been often proposed elsewhere.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Martin et al., &lt;span&gt;“Human Demographic History Impacts Genetic Risk Prediction Across Diverse Populations”&lt;/span&gt;; Alicia R Martin et al., &lt;span&gt;“The Critical Needs and Challenges for Genetic Architecture Studies in &lt;span&gt;Africa&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Current Opinion in Genetics &amp;amp; Development&lt;/em&gt;, Genetics of &lt;span&gt;Human Origins&lt;/span&gt;, 53 (December 1, 2018): 113–20, &lt;a href="https://doi.org/10.1016/j.gde.2018.08.005" role="doc-biblioref"&gt;https://doi.org/10.1016/j.gde.2018.08.005&lt;/a&gt;; Stephanie A. Bien et al., &lt;span&gt;“The &lt;span&gt;Future&lt;/span&gt; of &lt;span&gt;Genomic Studies Must Be Globally Representative&lt;/span&gt;: &lt;span&gt;Perspectives&lt;/span&gt; from &lt;span&gt;PAGE&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Annual Review of Genomics and Human Genetics&lt;/em&gt; 20, no. 1 (August 31, 2019): 181–200, &lt;a href="https://doi.org/10.1146/annurev-genom-091416-035517" role="doc-biblioref"&gt;https://doi.org/10.1146/annurev-genom-091416-035517&lt;/a&gt;; Martin et al., &lt;span&gt;“Clinical Use of Current Polygenic Risk Scores May Exacerbate Health Disparities”&lt;/span&gt;; Giorgio Sirugo, Scott M. Williams, and Sarah A. Tishkoff, &lt;span&gt;“The &lt;span&gt;Missing Diversity&lt;/span&gt; in &lt;span&gt;Human Genetic Studies&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Cell&lt;/em&gt; 177, no. 1 (March 21, 2019): 26–31, &lt;a href="https://doi.org/10.1016/j.cell.2019.02.048" role="doc-biblioref"&gt;https://doi.org/10.1016/j.cell.2019.02.048&lt;/a&gt;; Genevieve L. Wojcik et al., &lt;span&gt;“Genetic Analyses of Diverse Populations Improves Discovery for Complex Traits,”&lt;/span&gt; &lt;em&gt;Nature&lt;/em&gt; 570, no. 7762, 7762 (June 2019): 514–18, &lt;a href="https://doi.org/10.1038/s41586-019-1310-4" role="doc-biblioref"&gt;https://doi.org/10.1038/s41586-019-1310-4&lt;/a&gt;.&lt;/p&gt;'><sup>369</sup></a></span> [ELABORATE]</p>

</div>
</div>
</div>

  <div class="chapter-nav">
<div class="prev"><a href="MIKK-F2-chap.html"><span class="header-section-number">5</span> Genetic linkage study of bold/shy behaviours in the MIKK panel</a></div>
<div class="next"><a href="references.html">References</a></div>
</div></main><div class="col-md-3 col-lg-2 d-none d-md-block sidebar sidebar-chapter">
    <nav id="toc" data-toggle="toc" aria-label="On this page"><h2>On this page</h2>
      <ul class="nav navbar-nav">
<li><a class="nav-link" href="#Fst-chap"><span class="header-section-number">6</span> Variation in the frequency of trait-associated alleles across global human populations</a></li>
<li>
<a class="nav-link" href="#Fst-background"><span class="header-section-number">6.1</span> Background</a><ul class="nav navbar-nav">
<li><a class="nav-link" href="#polygenic-scores-pgs-and-genome-wide-association-studies-gwas"><span class="header-section-number">6.1.1</span> Polygenic Scores (PGS) and Genome-Wide Association Studies (GWAS)</a></li>
<li><a class="nav-link" href="#contributors-to-pgs-non-transferability"><span class="header-section-number">6.1.2</span> Contributors to PGS non-transferability</a></li>
<li><a class="nav-link" href="#Fst-descr"><span class="header-section-number">6.1.3</span> \(F_{ST}\)</a></li>
</ul>
</li>
<li>
<a class="nav-link" href="#Fst-analysis-chap"><span class="header-section-number">6.2</span> Analysis</a><ul class="nav navbar-nav">
<li><a class="nav-link" href="#linkage-disequilibrium"><span class="header-section-number">6.2.1</span> Linkage disequilibrium</a></li>
<li><a class="nav-link" href="#control-snps"><span class="header-section-number">6.2.2</span> Control SNPs</a></li>
<li><a class="nav-link" href="#f_st-and-ranking-traits-by-signed-kolmogorov-smirnov-d-statistic"><span class="header-section-number">6.2.3</span> \(F_{ST}\) and ranking traits by signed Kolmogorov-Smirnov \(D\) statistic</a></li>
</ul>
</li>
<li>
<a class="nav-link" href="#implications"><span class="header-section-number">6.3</span> Implications</a><ul class="nav navbar-nav">
<li><a class="nav-link" href="#limitation-of-f_st"><span class="header-section-number">6.3.1</span> Limitation of \(F_{ST}\)</a></li>
<li><a class="nav-link" href="#the-importance-of-environment-and-a-push-for-diversity-in-genetic-studies"><span class="header-section-number">6.3.2</span> The importance of environment, and a push for diversity in genetic studies</a></li>
</ul>
</li>
</ul>

      <div class="book-extra">
        <ul class="list-unstyled">
<li><a id="book-source" href="https://github.com/brettellebi/PhD-thesis/blob/master/book/06-Fst.Rmd">View source <i class="fab fa-github"></i></a></li>
          <li><a id="book-edit" href="https://github.com/brettellebi/PhD-thesis/edit/master/book/06-Fst.Rmd">Edit this page <i class="fab fa-github"></i></a></li>
        </ul>
</div>
    </nav>
</div>

</div>
</div> <!-- .container -->

<footer class="bg-primary text-light mt-5"><div class="container"><div class="row">

  <div class="col-12 col-md-6 mt-3">
    <p>"<strong>Japanese courage: a genetic analysis of complex traits in medaka fish and humans</strong>" was written by Ian Brettell. It was last built on 2022-08-31.</p>
  </div>

  <div class="col-12 col-md-6 mt-3">
    <p>This book was built by the <a class="text-light" href="https://bookdown.org">bookdown</a> R package.</p>
  </div>

</div></div>
</footer><!-- dynamically load mathjax for compatibility with self-contained --><script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.9/latest.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script><script type="text/x-mathjax-config">const popovers = document.querySelectorAll('a.footnote-ref[data-toggle="popover"]');
for (let popover of popovers) {
  const div = document.createElement('div');
  div.setAttribute('style', 'position: absolute; top: 0, left:0; width:0, height:0, overflow: hidden; visibility: hidden;');
  div.innerHTML = popover.getAttribute('data-content');

  var has_math = div.querySelector("span.math");
  if (has_math) {
    document.body.appendChild(div);
    MathJax.Hub.Queue(["Typeset", MathJax.Hub, div]);
    MathJax.Hub.Queue(function() {
      popover.setAttribute('data-content', div.innerHTML);
      document.body.removeChild(div);
    })
  }
}
</script>
</body>
</html>
