<!DOCTYPE html>
<html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<title>Chapter 6 Variation in the frequency of trait-associated alleles across global human populations | Genetic analysis of quantitative traits in medaka fish and humans</title>
<meta name="author" content="Ian Brettell">
<meta name="description" content="6.1 Background As seen in previous chapters, an individual’s phenotypic value at a given point in time is the product of complex interactions between their genome and their environment,...">
<meta name="generator" content="bookdown 0.26 with bs4_book()">
<meta property="og:title" content="Chapter 6 Variation in the frequency of trait-associated alleles across global human populations | Genetic analysis of quantitative traits in medaka fish and humans">
<meta property="og:type" content="book">
<meta property="og:description" content="6.1 Background As seen in previous chapters, an individual’s phenotypic value at a given point in time is the product of complex interactions between their genome and their environment,...">
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="Chapter 6 Variation in the frequency of trait-associated alleles across global human populations | Genetic analysis of quantitative traits in medaka fish and humans">
<meta name="twitter:description" content="6.1 Background As seen in previous chapters, an individual’s phenotypic value at a given point in time is the product of complex interactions between their genome and their environment,...">
<!-- JS --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://kit.fontawesome.com/6ecbd6c532.js" crossorigin="anonymous"></script><script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="libs/bootstrap-4.6.0/bootstrap.min.css" rel="stylesheet">
<script src="libs/bootstrap-4.6.0/bootstrap.bundle.min.js"></script><script src="libs/bs3compat-0.3.1/transition.js"></script><script src="libs/bs3compat-0.3.1/tabs.js"></script><script src="libs/bs3compat-0.3.1/bs3compat.js"></script><link href="libs/bs4_book-1.0.0/bs4_book.css" rel="stylesheet">
<script src="libs/bs4_book-1.0.0/bs4_book.js"></script><style>
    #content {
        font-size: 0.9rem;
    }
    </style>
<script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- CSS --><style type="text/css">
    
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  </style>
<style type="text/css">
    /* Used with Pandoc 2.11+ new --citeproc when CSL is used */
    div.csl-bib-body { }
    div.csl-entry {
      clear: both;
        }
    .hanging div.csl-entry {
      margin-left:2em;
      text-indent:-2em;
    }
    div.csl-left-margin {
      min-width:2em;
      float:left;
    }
    div.csl-right-inline {
      margin-left:2em;
      padding-left:1em;
    }
    div.csl-indent {
      margin-left: 2em;
    }
  </style>
<link rel="stylesheet" href="style.css">
</head>
<body data-spy="scroll" data-target="#toc">

<div class="container-fluid">
<div class="row">
  <header class="col-sm-12 col-lg-3 sidebar sidebar-book"><a class="sr-only sr-only-focusable" href="#content">Skip to main content</a>

    <div class="d-flex align-items-start justify-content-between">
      <h1>
        <a href="index.html" title="">Genetic analysis of quantitative traits in medaka fish and humans</a>
      </h1>
      <button class="btn btn-outline-primary d-lg-none ml-2 mt-1" type="button" data-toggle="collapse" data-target="#main-nav" aria-expanded="true" aria-controls="main-nav"><i class="fas fa-bars"></i><span class="sr-only">Show table of contents</span></button>
    </div>

    <div id="main-nav" class="collapse-lg">
      <form role="search">
        <input id="search" class="form-control" type="search" placeholder="Search" aria-label="Search">
</form>

      <nav aria-label="Table of contents"><h2>Table of contents</h2>
        <ul class="book-toc list-unstyled">
<li><a class="" href="index.html"><span class="header-section-number">1</span> About</a></li>
<li><a class="" href="Introduction.html"><span class="header-section-number">2</span> Introduction</a></li>
<li><a class="" href="MIKK-genomes-chap.html"><span class="header-section-number">3</span> Genomic variations in the MIKK panel</a></li>
<li><a class="" href="Pilot-chap.html"><span class="header-section-number">4</span> Classification of bold/shy behaviours in 5 inbred medaka lines</a></li>
<li><a class="" href="MIKK-F2-chap.html"><span class="header-section-number">5</span> Bold/shy behaviours in the MIKK panel</a></li>
<li><a class="active" href="Fst-chap.html"><span class="header-section-number">6</span> Variation in the frequency of trait-associated alleles across global human populations</a></li>
<li><a class="" href="references.html">References</a></li>
</ul>

        <div class="book-extra">
          <p><a id="book-repo" href="https://github.com/brettellebi/PhD-thesis">View book source <i class="fab fa-github"></i></a></p>
        </div>
      </nav>
</div>
  </header><main class="col-sm-12 col-md-9 col-lg-7" id="content"><div id="Fst-chap" class="section level1" number="6">
<h1>
<span class="header-section-number">6</span> Variation in the frequency of trait-associated alleles across global human populations<a class="anchor" aria-label="anchor" href="#Fst-chap"><i class="fas fa-link"></i></a>
</h1>
<div id="Fst-background" class="section level2" number="6.1">
<h2>
<span class="header-section-number">6.1</span> Background<a class="anchor" aria-label="anchor" href="#Fst-background"><i class="fas fa-link"></i></a>
</h2>
<p>As seen in previous chapters, an individual’s phenotypic value at a given point in time is the product of complex interactions between their genome and their environment, accumulating during their initial development and continuing throughout their lifetimes.</p>
<p>A richer understanding of the cumulative effect of genetic variants on any trait allows for the prediction of the value that an individual is most likely to have for that trait. Of all human traits, diseases are particularly salient – in 2018, the global healthcare industry was valued at US$8 trillion, and predicted to increase to US$12 trillion by 2022.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;&lt;span&gt;“The $11.9 &lt;span&gt;Trillion Global Healthcare Market&lt;/span&gt;: &lt;span&gt;Key Opportunities&lt;/span&gt; &amp;amp; &lt;span&gt;Strategies&lt;/span&gt; (2014-2022) - &lt;span&gt;ResearchAndMarkets&lt;/span&gt;.com,”&lt;/span&gt; June 25, 2019, &lt;a href="https://www.businesswire.com/news/home/20190625005862/en/The-11.9-Trillion-Global-Healthcare-Market-Key-Opportunities-Strategies-2014-2022---ResearchAndMarkets.com" role="doc-biblioref"&gt;https://www.businesswire.com/news/home/20190625005862/en/The-11.9-Trillion-Global-Healthcare-Market-Key-Opportunities-Strategies-2014-2022---ResearchAndMarkets.com&lt;/a&gt;.&lt;/p&gt;'><sup>41</sup></a></span> This strong financial imperative complements the moral imperative to reduce suffering, together driving many to determine how to use genetic information to improve human health.</p>
<p>Humans have long sought to use genetic information to predict an individual’s likely value for a given trait, in humans and other organisms (Chapter <a href="Introduction.html#Introduction">2</a>). More recently, since it became possible to sequence human genomes at scale, it is thought that by combining detailed genetic information with with other environmental and phenotypic information (such as lifestyle or clinical factors), clinicians could move towards the practice of “precision medicine”, where interventions could be tailored to their patients’ unique risk profiles.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Naomi R. Wray, Michael E. Goddard, and Peter M. Visscher, &lt;span&gt;“Prediction of Individual Genetic Risk to Disease from Genome-Wide Association Studies,”&lt;/span&gt; &lt;em&gt;Genome Research&lt;/em&gt; 17, no. 10 (January 10, 2007): 1520–28, &lt;a href="https://doi.org/10.1101/gr.6665407" role="doc-biblioref"&gt;https://doi.org/10.1101/gr.6665407&lt;/a&gt;.&lt;/p&gt;'><sup>42</sup></a></span></p>
<div id="polygenic-risk-scores-prs-and-genome-wide-association-studies-gwas" class="section level3" number="6.1.1">
<h3>
<span class="header-section-number">6.1.1</span> Polygenic Risk Scores (PRS) and Genome-Wide Association Studies (GWAS)<a class="anchor" aria-label="anchor" href="#polygenic-risk-scores-prs-and-genome-wide-association-studies-gwas"><i class="fas fa-link"></i></a>
</h3>
<p>The genetic component of precision medicine requires the construction of “polygenic risk scores” (<strong>PRS</strong>), also known as genetic risk profiling. PRS seek to predict the likelihood of a given trait based on an individual’s genomic variants. To date, PRS studies have shown reliable yet modest improvement above clinical models to predictions for a variety of health-related traits including blood pressure, breast cancer,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Paige Maas et al., &lt;span&gt;“Breast &lt;span&gt;Cancer Risk From Modifiable&lt;/span&gt; and &lt;span&gt;Nonmodifiable Risk Factors Among White Women&lt;/span&gt; in the &lt;span&gt;United States&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;JAMA Oncology&lt;/em&gt; 2, no. 10 (October 1, 2016): 1295–1302, &lt;a href="https://doi.org/10.1001/jamaoncol.2016.1025" role="doc-biblioref"&gt;https://doi.org/10.1001/jamaoncol.2016.1025&lt;/a&gt;.&lt;/p&gt;'><sup>43</sup></a></span> prostate cancer,,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Fredrick R. Schumacher et al., &lt;span&gt;“Association Analyses of More Than 140,000 Men Identify 63 New Prostate Cancer Susceptibility Loci,”&lt;/span&gt; &lt;em&gt;Nature Genetics&lt;/em&gt; 50, no. 7, 7 (July 2018): 928–36, &lt;a href="https://doi.org/10.1038/s41588-018-0142-8" role="doc-biblioref"&gt;https://doi.org/10.1038/s41588-018-0142-8&lt;/a&gt;.&lt;/p&gt;'><sup>44</sup></a></span> type I diabetes.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Seth A. Sharp et al., &lt;span&gt;“Development and &lt;span&gt;Standardization&lt;/span&gt; of an &lt;span&gt;Improved Type&lt;/span&gt; 1 &lt;span&gt;Diabetes Genetic Risk Score&lt;/span&gt; for &lt;span&gt;Use&lt;/span&gt; in &lt;span&gt;Newborn Screening&lt;/span&gt; and &lt;span&gt;Incident Diagnosis&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Diabetes Care&lt;/em&gt; 42, no. 2 (January 11, 2019): 200–207, &lt;a href="https://doi.org/10.2337/dc18-1785" role="doc-biblioref"&gt;https://doi.org/10.2337/dc18-1785&lt;/a&gt;.&lt;/p&gt;'><sup>45</sup></a></span></p>
<p>PRS scores can further be combined with lifestyle and clinical factors to more accurately predict the risk of cardiovascular disease.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Amit V. Khera et al., &lt;span&gt;“Genome-Wide Polygenic Scores for Common Diseases Identify Individuals with Risk Equivalent to Monogenic Mutations,”&lt;/span&gt; &lt;em&gt;Nature Genetics&lt;/em&gt; 50, no. 9, 9 (September 2018): 1219–24, &lt;a href="https://doi.org/10.1038/s41588-018-0183-z" role="doc-biblioref"&gt;https://doi.org/10.1038/s41588-018-0183-z&lt;/a&gt;; Iftikhar J. Kullo et al., &lt;span&gt;“Incorporating a &lt;span&gt;Genetic Risk Score Into Coronary Heart Disease Risk Estimates&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Circulation&lt;/em&gt; 133, no. 12 (March 22, 2016): 1181–88, &lt;a href="https://doi.org/10.1161/CIRCULATIONAHA.115.020109" role="doc-biblioref"&gt;https://doi.org/10.1161/CIRCULATIONAHA.115.020109&lt;/a&gt;; Pradeep Natarajan et al., &lt;span&gt;“Polygenic &lt;span&gt;Risk Score Identifies Subgroup With Higher Burden&lt;/span&gt; of &lt;span&gt;Atherosclerosis&lt;/span&gt; and &lt;span&gt;Greater Relative Benefit From Statin Therapy&lt;/span&gt; in the &lt;span&gt;Primary Prevention Setting&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Circulation&lt;/em&gt; 135, no. 22 (May 30, 2017): 2091–101, &lt;a href="https://doi.org/10.1161/CIRCULATIONAHA.116.024436" role="doc-biblioref"&gt;https://doi.org/10.1161/CIRCULATIONAHA.116.024436&lt;/a&gt;; Martine Paquette et al., &lt;span&gt;“Polygenic Risk Score Predicts Prevalence of Cardiovascular Disease in Patients with Familial Hypercholesterolemia,”&lt;/span&gt; &lt;em&gt;Journal of Clinical Lipidology&lt;/em&gt; 11, no. 3 (May 1, 2017): 725–732.e5, &lt;a href="https://doi.org/10.1016/j.jacl.2017.03.019" role="doc-biblioref"&gt;https://doi.org/10.1016/j.jacl.2017.03.019&lt;/a&gt;; Emmi Tikkanen et al., &lt;span&gt;“Genetic &lt;span&gt;Risk Prediction&lt;/span&gt; and a 2-&lt;span&gt;Stage Risk Screening Strategy&lt;/span&gt; for &lt;span&gt;Coronary Heart Disease&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Arteriosclerosis, Thrombosis, and Vascular Biology&lt;/em&gt; 33, no. 9 (September 2013): 2261–66, &lt;a href="https://doi.org/10.1161/ATVBAHA.112.301120" role="doc-biblioref"&gt;https://doi.org/10.1161/ATVBAHA.112.301120&lt;/a&gt;.&lt;/p&gt;'><sup>46</sup></a></span></p>
<p>The scores are calculated by summing the trait-associated alleles identified through genome-wide association studies (<strong>GWAS</strong>), as weighted by the alleles’ effect sizes.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;L. Duncan et al., &lt;span&gt;“Analysis of Polygenic Risk Score Usage and Performance in Diverse Human Populations,”&lt;/span&gt; &lt;em&gt;Nature Communications&lt;/em&gt; 10, no. 1, 1 (July 25, 2019): 3328, &lt;a href="https://doi.org/10.1038/s41467-019-11112-0" role="doc-biblioref"&gt;https://doi.org/10.1038/s41467-019-11112-0&lt;/a&gt;.&lt;/p&gt;'><sup>47</sup></a></span> However, most GWAS have been performed with individuals of European ancestry, despite only constituting 16% of the current global population. Although the proportion of participants in GWAS from a non-European background increased from 4% in 2009 to 16 % in 2016),<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Alice B. Popejoy and Stephanie M. Fullerton, &lt;span&gt;“Genomics Is Failing on Diversity,”&lt;/span&gt; &lt;em&gt;Nature&lt;/em&gt; 538, no. 7624, 7624 (October 2016): 161–64, &lt;a href="https://doi.org/10.1038/538161a" role="doc-biblioref"&gt;https://doi.org/10.1038/538161a&lt;/a&gt;.&lt;/p&gt;'><sup>48</sup></a></span> as of 2019, 79% of all GWAS participants recorded in the GWAS Catalog were of European ancestry, and the proportion of non-European individuals has remained the same or reduced since late 2014.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Alicia R. Martin, Masahiro Kanai, et al., &lt;span&gt;“Clinical Use of Current Polygenic Risk Scores May Exacerbate Health Disparities,”&lt;/span&gt; &lt;em&gt;Nature Genetics&lt;/em&gt; 51, no. 4, 4 (April 2019): 584–91, &lt;a href="https://doi.org/10.1038/s41588-019-0379-x" role="doc-biblioref"&gt;https://doi.org/10.1038/s41588-019-0379-x&lt;/a&gt;.&lt;/p&gt;'><sup>49</sup></a></span> This bias also extends to PRS studies, where as of 2019, only 67% of them included only participants of European ancestry, with another 19% including only East Asian ancestry participants, and only 3.8% with cohorts of African, Hispanic, or Indigenous ancestry.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Duncan et al., &lt;span&gt;“Analysis of Polygenic Risk Score Usage and Performance in Diverse Human Populations.”&lt;/span&gt;&lt;/p&gt;"><sup>50</sup></a></span></p>
<p>The consequence of this under-representation of non-European populations is that PRS scores are far better at predicting disease risk in individuals of European ancestry than in those of non-European ancestry.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Alicia R Martin, Christopher R Gignoux, et al., &lt;span&gt;“Human Demographic History Impacts Genetic Risk Prediction Across Diverse Populations,”&lt;/span&gt; &lt;em&gt;The American Journal of Human Genetics&lt;/em&gt; 100, no. 4 (2017): 635–49, &lt;a href="https://doi.org/10.1016/j.ajhg.2017.03.004" role="doc-biblioref"&gt;https://doi.org/10.1016/j.ajhg.2017.03.004&lt;/a&gt;; Martin, Kanai, et al., &lt;span&gt;“Clinical Use of Current Polygenic Risk Scores May Exacerbate Health Disparities.”&lt;/span&gt;&lt;/p&gt;'><sup>51</sup></a></span> For example, <span class="citation">Martin, Gignoux, et al.<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;&lt;span&gt;“Human Demographic History Impacts Genetic Risk Prediction Across Diverse Populations.”&lt;/span&gt;&lt;/p&gt;"><sup>52</sup></a></span> found that height was predicted to decrease with genetic distance from Europeans, despite robust evidence that West Africans are as tall as Europeans on average. Indeed, the predictive accuracy of PRS scores decays with genetic divergence of the GWAS “independent” or “test” sample from the “discovery” - or “training” - sample, as established in both humans,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Martin, Gignoux, et al.; Martin, Kanai, et al., &lt;span&gt;“Clinical Use of Current Polygenic Risk Scores May Exacerbate Health Disparities.”&lt;/span&gt;&lt;/p&gt;"><sup>53</sup></a></span> and livestock.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Samuel A. Clark et al., &lt;span&gt;“The Importance of Information on Relatives for the Prediction of Genomic Breeding Values and the Implications for the Makeup of Reference Data Sets in Livestock Breeding Schemes,”&lt;/span&gt; &lt;em&gt;Genetics Selection Evolution&lt;/em&gt; 44, no. 1 (February 9, 2012): 4, &lt;a href="https://doi.org/10.1186/1297-9686-44-4" role="doc-biblioref"&gt;https://doi.org/10.1186/1297-9686-44-4&lt;/a&gt;; David Habier et al., &lt;span&gt;“The Impact of Genetic Relationship Information on Genomic Breeding Values in &lt;span&gt;German Holstein&lt;/span&gt; Cattle,”&lt;/span&gt; &lt;em&gt;Genetics Selection Evolution&lt;/em&gt; 42, no. 1 (February 19, 2010): 5, &lt;a href="https://doi.org/10.1186/1297-9686-42-5" role="doc-biblioref"&gt;https://doi.org/10.1186/1297-9686-42-5&lt;/a&gt;; M. Pszczola et al., &lt;span&gt;“Reliability of Direct Genomic Values for Animals with Different Relationships Within and to the Reference Population,”&lt;/span&gt; &lt;em&gt;Journal of Dairy Science&lt;/em&gt; 95, no. 1 (January 1, 2012): 389–400, &lt;a href="https://doi.org/10.3168/jds.2011-4338" role="doc-biblioref"&gt;https://doi.org/10.3168/jds.2011-4338&lt;/a&gt;.&lt;/p&gt;'><sup>54</sup></a></span></p>
<p>These differences in representation cause PRS scores to have a lower accuracy for individuals of non-European ancestry. For example, compared to PRS scores for those of European ancestry, PRS scores across multiple traits for individuals of African ancestry are ~64-78% less accurate,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Duncan et al., &lt;span&gt;“Analysis of Polygenic Risk Score Usage and Performance in Diverse Human Populations”&lt;/span&gt;; Martin, Kanai, et al., &lt;span&gt;“Clinical Use of Current Polygenic Risk Scores May Exacerbate Health Disparities.”&lt;/span&gt;&lt;/p&gt;"><sup>55</sup></a></span> and for individuals of South-Asian ancestry ~37% less accurate, and ~50% less accurate for individuals of East-Asian ancestry.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Martin, Kanai, et al., &lt;span&gt;“Clinical Use of Current Polygenic Risk Scores May Exacerbate Health Disparities.”&lt;/span&gt;&lt;/p&gt;"><sup>56</sup></a></span></p>
</div>
<div id="potential-factors-for-prs-non-transferability" class="section level3" number="6.1.2">
<h3>
<span class="header-section-number">6.1.2</span> Potential factors for PRS non-transferability<a class="anchor" aria-label="anchor" href="#potential-factors-for-prs-non-transferability"><i class="fas fa-link"></i></a>
</h3>
<p>What explains this disparity in predictive value? A number of factors may be responsible, including:</p>
<ol style="list-style-type: decimal">
<li><p>GWAS neglect causal variants that do not exist within the “discovery” sample.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Martin, Kanai, et al.&lt;/p&gt;"><sup>57</sup></a></span></p></li>
<li><p>The sample populations may differ in linkage disequilibrium (<strong>LD</strong>) – the correlation structure of the genome – which would change the estimated effect sizes of the causal variants, even when the causal variants themselves are the same.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Martin, Kanai, et al.&lt;/p&gt;"><sup>58</sup></a></span></p></li>
<li><p>Allele frequencies of the causal variants, and the distribution of the effect sizes of the causal variants, may differ between populations.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Martin, Gignoux, et al., &lt;span&gt;“Human Demographic History Impacts Genetic Risk Prediction Across Diverse Populations”&lt;/span&gt;; Marco Scutari, Ian Mackay, and David Balding, &lt;span&gt;“Using &lt;span&gt;Genetic Distance&lt;/span&gt; to &lt;span&gt;Infer&lt;/span&gt; the &lt;span&gt;Accuracy&lt;/span&gt; of &lt;span&gt;Genomic Prediction&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;PLOS Genetics&lt;/em&gt; 12, no. 9 (September 2, 2016): e1006288, &lt;a href="https://doi.org/10.1371/journal.pgen.1006288" role="doc-biblioref"&gt;https://doi.org/10.1371/journal.pgen.1006288&lt;/a&gt;.&lt;/p&gt;'><sup>59</sup></a></span></p></li>
<li><p>The environments and demographies may differ between populations. These differences are often correlated with genetic divergence due to geography, making it difficult to determine whether the associations are driven by the differences between population in their genetics, or their environments.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Martin, Kanai, et al., &lt;span&gt;“Clinical Use of Current Polygenic Risk Scores May Exacerbate Health Disparities”&lt;/span&gt;; Sini Kerminen et al., &lt;span&gt;“Geographic &lt;span&gt;Variation&lt;/span&gt; and &lt;span&gt;Bias&lt;/span&gt; in the &lt;span&gt;Polygenic Scores&lt;/span&gt; of &lt;span&gt;Complex Diseases&lt;/span&gt; and &lt;span&gt;Traits&lt;/span&gt; in &lt;span&gt;Finland&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;The American Journal of Human Genetics&lt;/em&gt; 104, no. 6 (June 6, 2019): 1169–81, &lt;a href="https://doi.org/10.1016/j.ajhg.2019.05.001" role="doc-biblioref"&gt;https://doi.org/10.1016/j.ajhg.2019.05.001&lt;/a&gt;.&lt;/p&gt;'><sup>60</sup></a></span></p></li>
</ol>
<p>The first three factors can degrade predictive performance even in the absence of biological and environmental differences, whereas environmental and demographic differences can drive forces of natural selection can in turn drive differences in causal genetic architecture.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Martin, Kanai, et al., &lt;span&gt;“Clinical Use of Current Polygenic Risk Scores May Exacerbate Health Disparities.”&lt;/span&gt;&lt;/p&gt;"><sup>61</sup></a></span></p>
<p>Differences in LD and allele frequencies between populations can explain 70-80% of the loss of PRS relative accuracy for traits like body mass index and type 2 diabetes.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Ying Wang et al., &lt;span&gt;“Theoretical and Empirical Quantification of the Accuracy of Polygenic Scores in Ancestry Divergent Populations,”&lt;/span&gt; &lt;em&gt;Nature Communications&lt;/em&gt; 11, no. 1, 1 (July 31, 2020): 3865, &lt;a href="https://doi.org/10.1038/s41467-020-17719-y" role="doc-biblioref"&gt;https://doi.org/10.1038/s41467-020-17719-y&lt;/a&gt;.&lt;/p&gt;'><sup>62</sup></a></span></p>
<p>I discuss each of these factors in turn</p>
<div id="failure-to-discover-causal-variants-in-diverse-populations" class="section level4" number="6.1.2.1">
<h4>
<span class="header-section-number">6.1.2.1</span> Failure to discover causal variants in diverse populations<a class="anchor" aria-label="anchor" href="#failure-to-discover-causal-variants-in-diverse-populations"><i class="fas fa-link"></i></a>
</h4>
<p>The power to discover a causal variant through GWAS depends on the variant’s effect size and frequency in the study population.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Martin, Kanai, et al., &lt;span&gt;“Clinical Use of Current Polygenic Risk Scores May Exacerbate Health Disparities”&lt;/span&gt;; P. C. Sham et al., &lt;span&gt;“Power of &lt;span&gt;Linkage&lt;/span&gt; Versus &lt;span&gt;Association Analysis&lt;/span&gt; of &lt;span&gt;Quantitative Traits&lt;/span&gt;, by &lt;span&gt;Use&lt;/span&gt; of &lt;span&gt;Variance-Components Models&lt;/span&gt;, for &lt;span&gt;Sibship Data&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;The American Journal of Human Genetics&lt;/em&gt; 66, no. 5 (May 1, 2000): 1616–30, &lt;a href="https://doi.org/10.1086/302891" role="doc-biblioref"&gt;https://doi.org/10.1086/302891&lt;/a&gt;.&lt;/p&gt;'><sup>63</sup></a></span> That is to say, the greater the effect of the variant, or the more common it is, the more likely it is to be discovered.</p>
<p>Many SNP arrays used for GWAS do not sequence the whole genome, but rather a selection (from several hundred thousand to millions) of genetic markers intended to present common genomic variation.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Eleonora Porcu et al., &lt;span&gt;“Genotype &lt;span&gt;Imputation&lt;/span&gt; in &lt;span&gt;Genome-Wide Association Studies&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Current Protocols in Human Genetics&lt;/em&gt; 78, no. 1 (2013): 1.25.1–14, &lt;a href="https://doi.org/10.1002/0471142905.hg0125s78" role="doc-biblioref"&gt;https://doi.org/10.1002/0471142905.hg0125s78&lt;/a&gt;.&lt;/p&gt;'><sup>64</sup></a></span> To increase the density of genotypes, which would increase the likelihood of refining the association signal and identifying causal variants, researchers often “impute” variants that aren’t sequenced directly.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Porcu et al.&lt;/p&gt;"><sup>65</sup></a></span> The imputation process involves “phasing” the study genotypes onto the genotypes of a “reference panel.”<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Shane McCarthy et al., &lt;span&gt;“A Reference Panel of 64,976 Haplotypes for Genotype Imputation,”&lt;/span&gt; &lt;em&gt;Nature Genetics&lt;/em&gt; 48, no. 10, 10 (October 2016): 1279–83, &lt;a href="https://doi.org/10.1038/ng.3643" role="doc-biblioref"&gt;https://doi.org/10.1038/ng.3643&lt;/a&gt;.&lt;/p&gt;'><sup>66</sup></a></span> However, if the reference panel does not sufficiently represent the populations in the study sample, they are likely to miss or incorrectly impute those genotypes.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Martin, Kanai, et al., &lt;span&gt;“Clinical Use of Current Polygenic Risk Scores May Exacerbate Health Disparities.”&lt;/span&gt;&lt;/p&gt;"><sup>67</sup></a></span></p>
</div>
<div id="differences-in-ld-between-populations" class="section level4" number="6.1.2.2">
<h4>
<span class="header-section-number">6.1.2.2</span> Differences in LD between populations<a class="anchor" aria-label="anchor" href="#differences-in-ld-between-populations"><i class="fas fa-link"></i></a>
</h4>
<p>To illustrate the problem, if a SNP has an LD <span class="math inline">\(r^2\)</span> with a causal variant of 0.8 in the discovery population and 0.6 in the target population, it would explain 25% = (1 - 0.6/0.8) less trait variation in the target population, and would therefore be less predictive.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Wang et al., &lt;span&gt;“Theoretical and Empirical Quantification of the Accuracy of Polygenic Scores in Ancestry Divergent Populations.”&lt;/span&gt;&lt;/p&gt;"><sup>68</sup></a></span> Differences in effect-size estimates may typically be small for most regions of the genome, but PRS sum across these effects, therefore aggregating these population differences.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Martin, Kanai, et al., &lt;span&gt;“Clinical Use of Current Polygenic Risk Scores May Exacerbate Health Disparities.”&lt;/span&gt;&lt;/p&gt;"><sup>69</sup></a></span> Previous empirical and simulation studies have shown that accruacy of PRS scores decay wtih increased genetic differentation (<span class="math inline">\(F_{ST}\)</span>) and LD differences between populations.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Habier et al., &lt;span&gt;“The Impact of Genetic Relationship Information on Genomic Breeding Values in &lt;span&gt;German Holstein&lt;/span&gt; Cattle”&lt;/span&gt;; Pszczola et al., &lt;span&gt;“Reliability of Direct Genomic Values for Animals with Different Relationships Within and to the Reference Population”&lt;/span&gt;; Scutari, Mackay, and Balding, &lt;span&gt;“Using &lt;span&gt;Genetic Distance&lt;/span&gt; to &lt;span&gt;Infer&lt;/span&gt; the &lt;span&gt;Accuracy&lt;/span&gt; of &lt;span&gt;Genomic Prediction&lt;/span&gt;”&lt;/span&gt;; Wang et al., &lt;span&gt;“Theoretical and Empirical Quantification of the Accuracy of Polygenic Scores in Ancestry Divergent Populations.”&lt;/span&gt;&lt;/p&gt;"><sup>70</sup></a></span></p>
<p>It was demonstrated in simulations that using LD information from an external reference panel as a prior to infer the posterior mean effect size of a genetic variant can improve PRS predictive accuracy.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Bjarni J Vilhjálmsson et al., &lt;span&gt;“Modeling Linkage Disequilibrium Increases Accuracy of Polygenic Risk Scores,”&lt;/span&gt; &lt;em&gt;The American Journal of Human Genetics&lt;/em&gt; 97, no. 4 (2015): 576–92.&lt;/p&gt;"><sup>71</sup></a></span></p>
<p>Genetic divergence between popoulations can be measured by <span class="math inline">\(F_{ST}\)</span>, and the correlation between true and predicted phenotypic valaues decays approximately linearly with respect to <span class="math inline">\(F_{ST}\)</span>.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Scutari, Mackay, and Balding, &lt;span&gt;“Using &lt;span&gt;Genetic Distance&lt;/span&gt; to &lt;span&gt;Infer&lt;/span&gt; the &lt;span&gt;Accuracy&lt;/span&gt; of &lt;span&gt;Genomic Prediction&lt;/span&gt;.”&lt;/span&gt;&lt;/p&gt;"><sup>72</sup></a></span></p>
</div>
<div id="differences-in-allele-frequencies-between-populations" class="section level4" number="6.1.2.3">
<h4>
<span class="header-section-number">6.1.2.3</span> Differences in allele frequencies between populations<a class="anchor" aria-label="anchor" href="#differences-in-allele-frequencies-between-populations"><i class="fas fa-link"></i></a>
</h4>
<p>Causal variants can differ in both frequency and effect size between different ancestry groups (e.g. for lactase persistence)</p>
<p>As <span class="math inline">\(F_{ST}\)</span> does not take into account the effect size or direction of the effect of the trait-associated allele, for highly-polygenic traits like the ones shown here, <span class="math inline">\(F_{ST}\)</span> is almost entirely decoupled from the mean additive genetic value (or polygenic risk score) between populations.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Jeremy J. Berg and Graham Coop, &lt;span&gt;“A &lt;span&gt;Population Genetic Signal&lt;/span&gt; of &lt;span&gt;Polygenic Adaptation&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;PLOS Genetics&lt;/em&gt; 10, no. 8 (August 7, 2014): e1004412, &lt;a href="https://doi.org/10.1371/journal.pgen.1004412" role="doc-biblioref"&gt;https://doi.org/10.1371/journal.pgen.1004412&lt;/a&gt;.&lt;/p&gt;'><sup>73</sup></a></span></p>
</div>
</div>
</div>
<div id="Fst-analysis-chap" class="section level2" number="6.2">
<h2>
<span class="header-section-number">6.2</span> Analysis<a class="anchor" aria-label="anchor" href="#Fst-analysis-chap"><i class="fas fa-link"></i></a>
</h2>
<p>In this chapter I illustrate the distribution of <span class="math inline">\(F_{ST}\)</span> scores for loci associated with 587 traits, a subset of the GWAS Catalog that passed our criteria for suitable polygenic traits (see Chapter <a href="Fst-chap.html#Fst-analysis-chap">6.2</a>). Our analysis is empirical, in that we do not explicitly formulate a statistical test for drift or selection, differential or otherwise. Using high-coverage sequence data for 2,504 individuals from the 1000 Genomes Project phase 3 release, for each trait in the GWAS Catalog we calculated the distribution of <span class="math inline">\(F_{ST}\)</span> across all approximately-unlinked SNPs associated with it (trait SNPs), and compared these <span class="math inline">\(F_{ST}\)</span> distributions with the <span class="math inline">\(F_{ST}\)</span> distributions of random-selected SNPs that were matched to the trait SNPs by their allele frequencies in European populations (control SNPs). Our results show that traits related to the physical correlates of “race” (such as skin-pigmentation, eye colour, and hair shape) tend to have relatively high <span class="math inline">\(F_{ST}\)</span> values – signifying relatively high variance in allele frequencies between populations – whereas traits related to intelligence (such as self-reported EA, mathematical ability, and cognitive function measurement) tend to have lower <span class="math inline">\(F_{ST}\)</span> values that are similar to those of most polygenic traits such as height and body mass index.</p>
<div id="genomes" class="section level3" number="6.2.1">
<h3>
<span class="header-section-number">6.2.1</span> 1000 Genomes<a class="anchor" aria-label="anchor" href="#genomes"><i class="fas fa-link"></i></a>
</h3>
<p>As the reference for human genomic variation across diverse populations, we used the New York Genome Center high-coverage, phased .vcf files<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;&lt;span&gt;“Index of /Vol1/Ftp/Data_collections/&lt;span&gt;1000g&lt;/span&gt;_2504_high_coverage/Working/20201028_3202_phased/,”&lt;/span&gt; accessed March 24, 2022, &lt;a href="http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/data_collections/1000G_2504_high_coverage/working/20201028_3202_phased/" role="doc-biblioref"&gt;http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/data_collections/1000G_2504_high_coverage/working/20201028_3202_phased/&lt;/a&gt;.&lt;/p&gt;'><sup>74</sup></a></span> for the 2,504 individuals described in the 1000 Genomes phase 3 release.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;1000 Genomes Project Consortium et al., &lt;span&gt;“A Global Reference for Human Genetic Variation,”&lt;/span&gt; &lt;em&gt;Nature&lt;/em&gt; 526, no. 7571 (2015): 68.&lt;/p&gt;"><sup>75</sup></a></span> We then annotated those .vcf files with human SNP IDs from dbSNP release 9606.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Elizabeth M. Smigielski et al., &lt;span&gt;“&lt;span class="nocase"&gt;dbSNP&lt;/span&gt;: A Database of Single Nucleotide Polymorphisms,”&lt;/span&gt; &lt;em&gt;Nucleic Acids Research&lt;/em&gt; 28, no. 1 (January 1, 2000): 352–55, &lt;a href="https://doi.org/10.1093/nar/28.1.352" role="doc-biblioref"&gt;https://doi.org/10.1093/nar/28.1.352&lt;/a&gt;.&lt;/p&gt;'><sup>76</sup></a></span></p>
</div>
<div id="gwas-catalog" class="section level3" number="6.2.2">
<h3>
<span class="header-section-number">6.2.2</span> GWAS Catalog<a class="anchor" aria-label="anchor" href="#gwas-catalog"><i class="fas fa-link"></i></a>
</h3>
<p>We used the R package <em>gwasrapidd</em><span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Ramiro Magno and Ana-Teresa Maia, &lt;span&gt;“Gwasrapidd: An &lt;span&gt;R&lt;/span&gt; Package to Query, Download and Wrangle &lt;span&gt;GWAS&lt;/span&gt; Catalog Data,”&lt;/span&gt; &lt;em&gt;Bioinformatics&lt;/em&gt; 36, no. 2 (January 15, 2020): 649–50, &lt;a href="https://doi.org/10.1093/bioinformatics/btz605" role="doc-biblioref"&gt;https://doi.org/10.1093/bioinformatics/btz605&lt;/a&gt;.&lt;/p&gt;'><sup>77</sup></a></span> to query all traits in the GWAS Catalog<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Jacqueline MacArthur et al., &lt;span&gt;“The New &lt;span&gt;NHGRI-EBI Catalog&lt;/span&gt; of Published Genome-Wide Association Studies (&lt;span&gt;GWAS Catalog&lt;/span&gt;),”&lt;/span&gt; &lt;em&gt;Nucleic Acids Research&lt;/em&gt; 45, no. D1 (January 4, 2017): D896–901, &lt;a href="https://doi.org/10.1093/nar/gkw1133" role="doc-biblioref"&gt;https://doi.org/10.1093/nar/gkw1133&lt;/a&gt;.&lt;/p&gt;'><sup>78</sup></a></span> as of 9 August 2021 (<span class="math inline">\(N_{TRAITS}\)</span> = 3,459). For 541 of these traits, no matching variant IDs could be pulled out from the 1000 Genomes VCFs, leaving <span class="math inline">\(N_{TRAITS}\)</span> = 3,008.</p>
</div>
<div id="linkage-disequilibrium" class="section level3" number="6.2.3">
<h3>
<span class="header-section-number">6.2.3</span> Linkage disequilibrium<a class="anchor" aria-label="anchor" href="#linkage-disequilibrium"><i class="fas fa-link"></i></a>
</h3>
<p>To obtain the “trait SNP” dataset, for each trait, we sought to isolate the SNP closest to each of its true causal variants, and exclude the SNPs in linkage disequilibrium (LD) with them. To this end, we used PLINK 1.9<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Chang et al., &lt;span&gt;“Second-Generation &lt;span&gt;PLINK&lt;/span&gt;”&lt;/span&gt;; Purcell and Chang, &lt;em&gt;&lt;span&gt;PLINK&lt;/span&gt; 1.9&lt;/em&gt;.&lt;/p&gt;"><sup>79</sup></a></span> to “clump” the SNPs associated with each of the remaining 3,008 traits, using an index variant p-value threshold of <span class="math inline">\(10^{-8}\)</span>,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Orestis A. Panagiotou, John P. A. Ioannidis, and Genome-Wide Significance Project, &lt;span&gt;“What Should the Genome-Wide Significance Threshold Be? &lt;span&gt;Empirical&lt;/span&gt; Replication of Borderline Genetic Associations,”&lt;/span&gt; &lt;em&gt;International Journal of Epidemiology&lt;/em&gt; 41, no. 1 (February 2012): 273–86, &lt;a href="https://doi.org/10.1093/ije/dyr178" role="doc-biblioref"&gt;https://doi.org/10.1093/ije/dyr178&lt;/a&gt;.&lt;/p&gt;'><sup>80</sup></a></span> <span class="math inline">\(r^2\)</span> threshold of 0.1,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;W. G. Hill and Alan Robertson, &lt;span&gt;“Linkage Disequilibrium in Finite Populations,”&lt;/span&gt; &lt;em&gt;Theoretical and Applied Genetics&lt;/em&gt; 38, no. 6 (June 1, 1968): 226–31, &lt;a href="https://doi.org/10.1007/BF01245622" role="doc-biblioref"&gt;https://doi.org/10.1007/BF01245622&lt;/a&gt;.&lt;/p&gt;'><sup>81</sup></a></span> and base window size of 1 Mb. This process left us with 2,045 traits with at least one index SNP that met the p-value threshold. The index SNPs for each trait formed our set of trait SNPs. In order to target relatively polygenic traits, we further filtered out traits with fewer than 10 trait SNPs, leaving <span class="math inline">\(N_{TRAITS}\)</span> = 587. Supplementary Fig. 4 shows the counts of unique SNP IDs associated with each trait before and after clumping, and an interactive version is available in the notebook cited above.</p>
</div>
<div id="control-snps" class="section level3" number="6.2.4">
<h3>
<span class="header-section-number">6.2.4</span> Control SNPs<a class="anchor" aria-label="anchor" href="#control-snps"><i class="fas fa-link"></i></a>
</h3>
<p>To obtain our “control SNP” dataset, we assigned each trait SNP to one of 20 bins based on its minor allele frequency in European populations (as provided in the original 1000 Genomes .vcf files under the column header ‘INFO/AC_EUR’). For example, if a trait SNP had a minor allele frequency of 0.08 in European populations, it was assigned to the <span class="math inline">\((0.05, 0.1]\)</span> bin. We did the same for all (unassociated) SNPs in the .vcf files, then paired each trait SNP with a random SNP from the .vcf file in the equivalent bin. These allele-frequency-paired random SNPs formed our set of “control SNPs”, which we used to infer the <span class="math inline">\(F_{ST}\)</span> distribution of a random set of SNPs with the same allele frequencies as the trait SNPs, and against which we could compare the <span class="math inline">\(F_{ST}\)</span> distribution of the trait SNPs.</p>
</div>
<div id="f_st-and-ranking-traits-by-signed-kolmogorov-smirnov-d-statistic" class="section level3" number="6.2.5">
<h3>
<span class="header-section-number">6.2.5</span> <span class="math inline">\(F_{ST}\)</span> and ranking traits by signed Kolmogorov-Smirnov <span class="math inline">\(D\)</span> statistic<a class="anchor" aria-label="anchor" href="#f_st-and-ranking-traits-by-signed-kolmogorov-smirnov-d-statistic"><i class="fas fa-link"></i></a>
</h3>
<p>We then calculated <span class="math inline">\(F_{ST}\)</span> for each of the trait SNPs and their matched control SNPs using the Weir and Cockerham method,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;B. S. Weir and C. Clark Cockerham, &lt;span&gt;“Estimating &lt;span&gt;F-Statistics&lt;/span&gt; for the &lt;span&gt;Analysis&lt;/span&gt; of &lt;span&gt;Population Structure&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Evolution&lt;/em&gt; 38, no. 6 (1984): 1358–70, &lt;a href="https://doi.org/10.2307/2408641" role="doc-biblioref"&gt;https://doi.org/10.2307/2408641&lt;/a&gt;.&lt;/p&gt;'><sup>82</sup></a></span> as implemented in the R package <em>pegas</em>.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Emmanuel Paradis, &lt;span&gt;“Pegas: An &lt;span&gt;R&lt;/span&gt; Package for Population Genetics with an Integrated–Modular Approach,”&lt;/span&gt; &lt;em&gt;Bioinformatics&lt;/em&gt; 26, no. 3 (February 1, 2010): 419–20, &lt;a href="https://doi.org/10.1093/bioinformatics/btp696" role="doc-biblioref"&gt;https://doi.org/10.1093/bioinformatics/btp696&lt;/a&gt;.&lt;/p&gt;'><sup>83</sup></a></span> To rank all traits based on the directional difference in <span class="math inline">\(F_{ST}\)</span> distributions between trait and control SNPs, we ran three Kolmogorov-Smirnov (KS) tests for each trait <span class="math inline">\(t\)</span> with <span class="math inline">\(x_t\)</span> = <span class="math inline">\(F_{ST, trait SNPs}\)</span> and <span class="math inline">\(y_t\)</span> = <span class="math inline">\(F_{ST, control SNPs}\)</span>:</p>
<ol style="list-style-type: decimal">
<li><p>two-sided (<span class="math inline">\(D_t\)</span>) ;</p></li>
<li><p>one-sided “greater” (<span class="math inline">\(D_t^+\)</span>) ; and</p></li>
<li><p>one-sided “less” (<span class="math inline">\(D_t^-\)</span>).</p></li>
</ol>
<p>I note that <span class="math inline">\(D_{t} = max(D_t^+, D_t^-)\)</span>, where <span class="math inline">\(D_t^+\)</span> is the greatest vertical distance attained by the eCDF of <span class="math inline">\(x_t\)</span> over the eCDF of <span class="math inline">\(y_t\)</span>, and <span class="math inline">\(D_t^-\)</span> is the greatest vertical distance attained by the eCDF of <span class="math inline">\(y_t\)</span> over the eCDF of <span class="math inline">\(x_t\)</span>.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;W. J. Conover, &lt;em&gt;Practical &lt;span&gt;Nonparametric Statistics&lt;/span&gt;&lt;/em&gt; (&lt;span&gt;John Wiley &amp;amp; Sons&lt;/span&gt;, 1999), &lt;a href="https://books.google.com?id=n_39DwAAQBAJ" role="doc-biblioref"&gt;https://books.google.com?id=n_39DwAAQBAJ&lt;/a&gt;; J. Durbin, &lt;em&gt;Distribution &lt;span&gt;Theory&lt;/span&gt; for &lt;span&gt;Tests Based&lt;/span&gt; on &lt;span&gt;Sample Distribution Function&lt;/span&gt;&lt;/em&gt; (&lt;span&gt;SIAM&lt;/span&gt;, 1973), &lt;a href="https://books.google.com?id=zAryCrT1IUYC" role="doc-biblioref"&gt;https://books.google.com?id=zAryCrT1IUYC&lt;/a&gt;.&lt;/p&gt;'><sup>84</sup></a></span> Accordingly, we used a comparison of <span class="math inline">\(D_t^+\)</span> and <span class="math inline">\(D_t^-\)</span> to created a signed D statistic (<span class="math inline">\(D\)</span>), based on the logic that trait SNPs with a lower overall <span class="math inline">\(F_{ST}\)</span> than control SNPs tend to have a higher <span class="math inline">\(D\)</span> under the “greater” test than the “less” test, and vice versa.</p>
<p>Therefore, <span class="math inline">\({D_t^S}\)</span>:</p>
<p><span class="math display">\[
\begin{aligned}
D_t^- &gt; D_t^+ &amp;: &amp;-D_t \\
D_t^- = D_t^+ &amp;: &amp;0 \\
D_t^- &lt; D_t^+ &amp;: &amp;D_t \\
\end{aligned}
\]</span></p>
<p>In <strong>Figure <a href="Fst-chap.html#fig:FstMain">6.1</a></strong> we present the <span class="math inline">\(F_{ST}\)</span> distributions of trait SNPs for an illustrative subset of 28 human traits, ranked by <span class="math inline">\({D_t^S}\)</span> when compared with their matched control SNPs. <strong>Figure <a href="Fst-chap.html#fig:FstMain">6.1</a>A</strong> shows the densities of SNPs as a function of <span class="math inline">\(F_{ST}\)</span>, and <strong>Figure <a href="Fst-chap.html#fig:FstMain">6.1</a>B and C</strong> show their empirical Cumulative Distribution Functions (eCDFs). <strong>Figure <a href="Fst-chap.html#fig:FstMain">6.1</a>B</strong> includes the eCDFs of control SNPs in grey. eCDF figures for all 587 traits that passed our filters (Methods) are provided in Supplementary Fig. 1.</p>

<div class="figure">
<span style="display:block;" id="fig:FstMain"></span>
<img src="figs/fst/20220314_final_0.1_1000.png" alt="Distributions of \(F_{ST}\) across 28 illustrative human traits, ranked by signed-D (Kolmogorov-Smirnov test) comparing trait and control SNPs. A. \(F_{ST}\) density ridge plots with SNP markers. B. Empirical Cumulative Distribution Functions (eCDFs) of \(F_{ST}\) for trait-associated (colour) and random control (grey) SNPs, faceted by trait. C. Consolidated eCDFs of trait-associated SNPs from (B). eCDFs for all traits are included in Supplementary Figure 1." width="100%"><p class="caption">
Figure 6.1: Distributions of <span class="math inline">\(F_{ST}\)</span> across 28 illustrative human traits, ranked by signed-D (Kolmogorov-Smirnov test) comparing trait and control SNPs. <strong>A</strong>. <span class="math inline">\(F_{ST}\)</span> density ridge plots with SNP markers. <strong>B</strong>. Empirical Cumulative Distribution Functions (eCDFs) of <span class="math inline">\(F_{ST}\)</span> for trait-associated (colour) and random control (grey) SNPs, faceted by trait. <strong>C</strong>. Consolidated eCDFs of trait-associated SNPs from (B). eCDFs for all traits are included in Supplementary Figure 1.
</p>
</div>

</div>
</div>
</div>

  <div class="chapter-nav">
<div class="prev"><a href="MIKK-F2-chap.html"><span class="header-section-number">5</span> Bold/shy behaviours in the MIKK panel</a></div>
<div class="next"><a href="references.html">References</a></div>
</div></main><div class="col-md-3 col-lg-2 d-none d-md-block sidebar sidebar-chapter">
    <nav id="toc" data-toggle="toc" aria-label="On this page"><h2>On this page</h2>
      <ul class="nav navbar-nav">
<li><a class="nav-link" href="#Fst-chap"><span class="header-section-number">6</span> Variation in the frequency of trait-associated alleles across global human populations</a></li>
<li>
<a class="nav-link" href="#Fst-background"><span class="header-section-number">6.1</span> Background</a><ul class="nav navbar-nav">
<li><a class="nav-link" href="#polygenic-risk-scores-prs-and-genome-wide-association-studies-gwas"><span class="header-section-number">6.1.1</span> Polygenic Risk Scores (PRS) and Genome-Wide Association Studies (GWAS)</a></li>
<li><a class="nav-link" href="#potential-factors-for-prs-non-transferability"><span class="header-section-number">6.1.2</span> Potential factors for PRS non-transferability</a></li>
</ul>
</li>
<li>
<a class="nav-link" href="#Fst-analysis-chap"><span class="header-section-number">6.2</span> Analysis</a><ul class="nav navbar-nav">
<li><a class="nav-link" href="#genomes"><span class="header-section-number">6.2.1</span> 1000 Genomes</a></li>
<li><a class="nav-link" href="#gwas-catalog"><span class="header-section-number">6.2.2</span> GWAS Catalog</a></li>
<li><a class="nav-link" href="#linkage-disequilibrium"><span class="header-section-number">6.2.3</span> Linkage disequilibrium</a></li>
<li><a class="nav-link" href="#control-snps"><span class="header-section-number">6.2.4</span> Control SNPs</a></li>
<li><a class="nav-link" href="#f_st-and-ranking-traits-by-signed-kolmogorov-smirnov-d-statistic"><span class="header-section-number">6.2.5</span> \(F_{ST}\) and ranking traits by signed Kolmogorov-Smirnov \(D\) statistic</a></li>
</ul>
</li>
</ul>

      <div class="book-extra">
        <ul class="list-unstyled">
<li><a id="book-source" href="https://github.com/brettellebi/PhD-thesis/blob/master/book/06-Fst.Rmd">View source <i class="fab fa-github"></i></a></li>
          <li><a id="book-edit" href="https://github.com/brettellebi/PhD-thesis/edit/master/book/06-Fst.Rmd">Edit this page <i class="fab fa-github"></i></a></li>
        </ul>
</div>
    </nav>
</div>

</div>
</div> <!-- .container -->

<footer class="bg-primary text-light mt-5"><div class="container"><div class="row">

  <div class="col-12 col-md-6 mt-3">
    <p>"<strong>Genetic analysis of quantitative traits in medaka fish and humans</strong>" was written by Ian Brettell. It was last built on 2022-06-15.</p>
  </div>

  <div class="col-12 col-md-6 mt-3">
    <p>This book was built by the <a class="text-light" href="https://bookdown.org">bookdown</a> R package.</p>
  </div>

</div></div>
</footer><!-- dynamically load mathjax for compatibility with self-contained --><script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script><script type="text/x-mathjax-config">const popovers = document.querySelectorAll('a.footnote-ref[data-toggle="popover"]');
for (let popover of popovers) {
  const div = document.createElement('div');
  div.setAttribute('style', 'position: absolute; top: 0, left:0; width:0, height:0, overflow: hidden; visibility: hidden;');
  div.innerHTML = popover.getAttribute('data-content');

  var has_math = div.querySelector("span.math");
  if (has_math) {
    document.body.appendChild(div);
    MathJax.Hub.Queue(["Typeset", MathJax.Hub, div]);
    MathJax.Hub.Queue(function() {
      popover.setAttribute('data-content', div.innerHTML);
      document.body.removeChild(div);
    })
  }
}
</script>
</body>
</html>
