<!DOCTYPE html>
<html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<title>Chapter 6 Variation in the frequency of trait-associated alleles across global human populations | Genetic analysis of quantitative traits in medaka fish and humans</title>
<meta name="author" content="Ian Brettell">
<meta name="description" content="In this chapter I illustrate the distribution of \(F_{ST}\) scores for loci associated with 587 traits, a subset of the GWAS Catalog which passed our criteria for suitable polygenic traits...">
<meta name="generator" content="bookdown 0.26 with bs4_book()">
<meta property="og:title" content="Chapter 6 Variation in the frequency of trait-associated alleles across global human populations | Genetic analysis of quantitative traits in medaka fish and humans">
<meta property="og:type" content="book">
<meta property="og:description" content="In this chapter I illustrate the distribution of \(F_{ST}\) scores for loci associated with 587 traits, a subset of the GWAS Catalog which passed our criteria for suitable polygenic traits...">
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="Chapter 6 Variation in the frequency of trait-associated alleles across global human populations | Genetic analysis of quantitative traits in medaka fish and humans">
<meta name="twitter:description" content="In this chapter I illustrate the distribution of \(F_{ST}\) scores for loci associated with 587 traits, a subset of the GWAS Catalog which passed our criteria for suitable polygenic traits...">
<!-- JS --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://kit.fontawesome.com/6ecbd6c532.js" crossorigin="anonymous"></script><script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="libs/bootstrap-4.6.0/bootstrap.min.css" rel="stylesheet">
<script src="libs/bootstrap-4.6.0/bootstrap.bundle.min.js"></script><script src="libs/bs3compat-0.3.1/transition.js"></script><script src="libs/bs3compat-0.3.1/tabs.js"></script><script src="libs/bs3compat-0.3.1/bs3compat.js"></script><link href="libs/bs4_book-1.0.0/bs4_book.css" rel="stylesheet">
<script src="libs/bs4_book-1.0.0/bs4_book.js"></script><style>
    #content {
        font-size: 0.9rem;
    }
    </style>
<script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- CSS --><style type="text/css">
    
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  </style>
<style type="text/css">
    /* Used with Pandoc 2.11+ new --citeproc when CSL is used */
    div.csl-bib-body { }
    div.csl-entry {
      clear: both;
        }
    .hanging div.csl-entry {
      margin-left:2em;
      text-indent:-2em;
    }
    div.csl-left-margin {
      min-width:2em;
      float:left;
    }
    div.csl-right-inline {
      margin-left:2em;
      padding-left:1em;
    }
    div.csl-indent {
      margin-left: 2em;
    }
  </style>
<link rel="stylesheet" href="style.css">
</head>
<body data-spy="scroll" data-target="#toc">

<div class="container-fluid">
<div class="row">
  <header class="col-sm-12 col-lg-3 sidebar sidebar-book"><a class="sr-only sr-only-focusable" href="#content">Skip to main content</a>

    <div class="d-flex align-items-start justify-content-between">
      <h1>
        <a href="index.html" title="">Genetic analysis of quantitative traits in medaka fish and humans</a>
      </h1>
      <button class="btn btn-outline-primary d-lg-none ml-2 mt-1" type="button" data-toggle="collapse" data-target="#main-nav" aria-expanded="true" aria-controls="main-nav"><i class="fas fa-bars"></i><span class="sr-only">Show table of contents</span></button>
    </div>

    <div id="main-nav" class="collapse-lg">
      <form role="search">
        <input id="search" class="form-control" type="search" placeholder="Search" aria-label="Search">
</form>

      <nav aria-label="Table of contents"><h2>Table of contents</h2>
        <ul class="book-toc list-unstyled">
<li><a class="" href="index.html"><span class="header-section-number">1</span> About</a></li>
<li><a class="" href="introduction.html"><span class="header-section-number">2</span> Introduction</a></li>
<li><a class="" href="MIKK-genomes-chap.html"><span class="header-section-number">3</span> Genomic variations in the MIKK panel</a></li>
<li><a class="" href="Pilot-chap.html"><span class="header-section-number">4</span> Classification of bold/shy behaviours in 5 inbred medaka lines</a></li>
<li><a class="" href="MIKK-F2-chap.html"><span class="header-section-number">5</span> Bold/shy behaviours in the MIKK panel</a></li>
<li><a class="active" href="Fst-chap.html"><span class="header-section-number">6</span> Variation in the frequency of trait-associated alleles across global human populations</a></li>
<li><a class="" href="references.html">References</a></li>
</ul>

        <div class="book-extra">
          <p><a id="book-repo" href="https://github.com/brettellebi/PhD-thesis">View book source <i class="fab fa-github"></i></a></p>
        </div>
      </nav>
</div>
  </header><main class="col-sm-12 col-md-9 col-lg-7" id="content"><div id="Fst-chap" class="section level1" number="6">
<h1>
<span class="header-section-number">6</span> Variation in the frequency of trait-associated alleles across global human populations<a class="anchor" aria-label="anchor" href="#Fst-chap"><i class="fas fa-link"></i></a>
</h1>
<p>In this chapter I illustrate the distribution of <span class="math inline">\(F_{ST}\)</span> scores for loci associated with 587 traits, a subset of the GWAS Catalog which passed our criteria for suitable polygenic traits (Methods). Our analysis is empirical, in that we do not explicitly formulate a statistical test for drift or selection, differential or otherwise. Using high-coverage sequence data for 2,504 individuals from the 1000 Genomes Project phase 3 release, for each trait in the GWAS Catalog we calculated the distribution of <span class="math inline">\(F_{ST}\)</span> across all approximately-unlinked SNPs associated with it (trait SNPs), and compared these <span class="math inline">\(F_{ST}\)</span> distributions with the <span class="math inline">\(F_{ST}\)</span> distributions of random-selected SNPs that were matched to the trait SNPs by their allele frequencies in European populations (control SNPs). Our results show that traits related to the physical correlates of “race” (such as skin-pigmentation, eye colour, and hair shape) tend to have relatively high <span class="math inline">\(F_{ST}\)</span> values – signifying relatively high variance in allele frequencies between populations – whereas traits related to intelligence (such as self-reported EA, mathematical ability, and cognitive function measurement) tend to have lower <span class="math inline">\(F_{ST}\)</span> values that are similar to those of most polygenic traits such as height and body mass index.</p>
<div id="background" class="section level2" number="6.1">
<h2>
<span class="header-section-number">6.1</span> Background<a class="anchor" aria-label="anchor" href="#background"><i class="fas fa-link"></i></a>
</h2>
<div id="polygenic-risk-scores-prs-and-genome-wide-association-studies-gwas" class="section level3" number="6.1.1">
<h3>
<span class="header-section-number">6.1.1</span> Polygenic Risk Scores (PRS) and Genome-Wide Association Studies (GWAS)<a class="anchor" aria-label="anchor" href="#polygenic-risk-scores-prs-and-genome-wide-association-studies-gwas"><i class="fas fa-link"></i></a>
</h3>
<p>As seen in previous chapters, an individual’s phenotypic value at a given point in time is the product of complex interactions between their genome and their environment, accumulating during their initial development and continuing throughout their lifetimes.</p>
<p>A richer understanding of the cumulative effect of genetic variants on any trait allows for the prediction of the value that an individual is most likely to have for that trait. Of all human traits, diseases are particularly salient – in 2018, the global healthcare industry was valued at US$8 trillion, and predicted to increase to US$12 trillion by 2022 <span class="citation">(<span>“The $11.9 <span>Trillion Global Healthcare Market</span>: <span>Key Opportunities</span> &amp; <span>Strategies</span> (2014-2022) - <span>ResearchAndMarkets</span>.com”</span> 2019)</span>. This strong financial imperative complements the moral imperative to reduce suffering, together driving many to determine how to use genetic information to improve human health.</p>
<p>Once it became possible to sequence human genomes at scale, researchers have sought to use genetic information to predict an individual’s likely value for a given trait <span class="citation">(Wray, Goddard, and Visscher 2007)</span>. It is thought that by combining genetic information with with other environmental and phenotypic information, clinicians could work towards applying “precision medicine”, where their patients’ medications, diet, and other lifestyle factors could be tailored to their unique needs. The genetic component of this pursuit is now commonly known as polygenic risk scores (<strong>PRS</strong>), also known as genetic risk profiling, which seeks to predict the likelihood of a given trait based on an individual’s genomic variants. To date, PRS studies have shown reliable yet modest improvement above clinical models to predictions for a variety of health-related traits including blood pressure, breast cancer <span class="citation">(Maas et al. 2016)</span>, prostate cancer, <span class="citation">(Schumacher et al. 2018)</span>, type I diabetes <span class="citation">(Sharp et al. 2019)</span>.</p>
<p>PRS scores can further be combined with lifestyle and clinical factors to more accurately predict the risk of cardiovascular disease <span class="citation">(Khera et al. 2018; Kullo et al. 2016; Natarajan et al. 2017; Paquette et al. 2017; Tikkanen et al. 2013)</span>.</p>
<p>The scores are calculated by summing the trait-associated alleles identified through genome-wide association studies (<strong>GWAS</strong>), weighted by the alleles’ effect sizes.<span class="citation">(Duncan et al. 2019)</span> However, most GWAS have been performed with individuals of European ancestry. Although the proportion of participants in GWAS from a non-European background is increasing (from 4% in 20019 to 16 % in 2016 <span class="citation">(Popejoy and Fullerton 2016)</span>), as of 2019, 79% of all GWAS participants recorded in the GWAS Catalog were of European ancestry.<span class="citation">(A. R. Martin et al. 2019)</span> The consequence of this is that PRS scores are far better at predicting disease risk in individuals of European ancestry <span class="citation">(A. R. Martin et al. 2017, 2019)</span>. For example, <span class="citation">A. R. Martin et al. (2017)</span> found that height was predicted to decrease with genetic distance from Europeans, despite robust evidence that West Africans are as tall as Europeans on average. Indeed, the predictive accuracy of PRS scores decays with genetic divergence of the GWAS “independent” or “test” sample from the “discovery” - or “training” - sample, as established in both humans <span class="citation">(A. R. Martin et al. 2017, 2019)</span>, and livestock <span class="citation">(Clark et al. 2012; Habier et al. 2010; Pszczola et al. 2012)</span>.</p>
<p>These differences in representation cause PRS scores to have a lower accuracy for individuals of non-European ancestry. For example, compared to PRS scores for those of European ancestry, PRS scores across multiple traits for individuals of African ancestry are ~64-78% less accurate <span class="citation">(Duncan et al. 2019; A. R. Martin et al. 2019)</span>, and for indiviauls of South-Asian acnestry ~37% less accurace, and ~50% less accurate for individuals of East-Asian ancestry <span class="citation">(A. R. Martin et al. 2019)</span>.</p>
<p>What explains this disparity in predictive value? A number of factors may be responsible including:</p>
<ol style="list-style-type: decimal">
<li><p>GWAS neglect causal variants that do not exist within the “discovery” sample <span class="citation">(A. R. Martin et al. 2019)</span>.</p></li>
<li><p>The sample populations may differ in linkage disequilibrium (<strong>LD</strong>) – the correlation structure of the genome – which would change the estimated effect sizes of the causal variants, even when the causal variants themselves are the same <span class="citation">(A. R. Martin et al. 2019)</span>. To illusatrate, if a SNP has an LD <span class="math inline">\(r^2\)</span> with a causal variant of 0.8 in the discovery population and 0.6 in the target population, it would explain 25% = (1 - 0.6/0.8) less trait variation in the target population and therefore would be less predictive <span class="citation">(Wang et al. 2020)</span>. Differences in effect-size estimates may typically be small for most regions of the genome, but PRS sum across these effects, therefore aggregating these population differences <span class="citation">(A. R. Martin et al. 2019)</span>. Previous empirical and simulation studies have shown that accruacy of PRS scores decay wtih increased genetic differentation (<span class="math inline">\(F_{ST}\)</span>) and LD differences between populations <span class="citation">(Habier et al. 2010; Pszczola et al. 2012; Scutari, Mackay, and Balding 2016; Wang et al. 2020)</span>.</p></li>
<li><p>Allele frequencies of the causal variants, and the distribution of the effect sizes of the causal variants, may differ between populations <span class="citation">(A. R. Martin et al. 2017; Scutari, Mackay, and Balding 2016)</span>.</p></li>
<li><p>The environments and demographies may differ between populations. These differences are often correlated with genetic divergence due to geography, making it difficult to determine whether the associations are driven by the differences between population in their genetics, or their environments <span class="citation">(A. R. Martin et al. 2019; Kerminen et al. 2019)</span>.</p></li>
</ol>
<p>The first three factors can degrade predictive performance even in the absence of biological and environmental differences, whereas environmental and demographic differences can drive forces of natural selection can in turn drive differences in causal genetic architecture <span class="citation">(A. R. Martin et al. 2019)</span>.</p>
<p>Differences in LD and allele frequencies between populations can explain 70-80% of the loss of PRS relative accuracy for traits like body mass index and type 2 diabetes <span class="citation">(Wang et al. 2020)</span>.</p>
</div>
<div id="effect-of-the-environment-on" class="section level3" number="6.1.2">
<h3>
<span class="header-section-number">6.1.2</span> Effect of the environment on<a class="anchor" aria-label="anchor" href="#effect-of-the-environment-on"><i class="fas fa-link"></i></a>
</h3>
</div>
<div id="discovery-of-causal-variants" class="section level3" number="6.1.3">
<h3>
<span class="header-section-number">6.1.3</span> Discovery of causal variants<a class="anchor" aria-label="anchor" href="#discovery-of-causal-variants"><i class="fas fa-link"></i></a>
</h3>
<p>The power to discover a causal variant through GWAS depends on the variant’s effect size and frequency in the study population <span class="citation">(A. R. Martin et al. 2019; Sham et al. 2000)</span>. That is to say, the greater the effect of the variant, or the more common it is, the more likely it is to be discovered.</p>
<p>Many SNP arrays used for GWAS do not sequence the whole genome, but rather a selection (from several hundred thousand to millions) of genetic markers intended to present common genomic variation <span class="citation">(Porcu et al. 2013)</span>. To increase the density of genotypes, which would increase the likelihood of refining the association signal and identifying causal variants, researchers often “impute” variants that aren’t sequenced directly <span class="citation">(Porcu et al. 2013)</span>. The imputation process involves “phasing” the study genotypes onto the genotypes of a “reference panel” <span class="citation">(McCarthy et al. 2016)</span>. However, if the reference panel does not sufficiently represent the populations in the study sample, they are likely to miss or incorrectly impute genotypes from those under-represented populations <span class="citation">(A. R. Martin et al. 2019)</span>.</p>
<p>Genetic divergence between popoulations can be measured by <span class="math inline">\(F_{ST}\)</span>, and the correlation between true and predicted phenotypic vlaues decays approximately linearly with respect to <span class="math inline">\(F_{ST}\)</span> <span class="citation">(Scutari, Mackay, and Balding 2016)</span>.</p>
<p>It was demonstrated in simulations that using LD information from an external reference panel as a prior to infer the posterior mean effect size of a genetic variant can improve PRS predictive accuracy <span class="citation">(Vilhjálmsson et al. 2015)</span>.</p>
<p>Causal variants can differ in both frequency and effect size between different ancestry groups (e.g. for lactase persistence)</p>
<p>As noted in the 2019 review of PRS scores by <span class="citation">Duncan et al. (2019)</span>, 67% of PRS studies included only participants of European ancestry, with another 19% including only East Asian ancestry participants, and only 3.8% with cohorts of African, Hispanic, or Indigenous ancestry.</p>
<p>As <span class="math inline">\(F_{ST}\)</span> does not take into account the effect size or direction of the effect of the trait-associated allele, for highly-polygenic traits like the ones shown here, <span class="math inline">\(F_{ST}\)</span> is almost entirely decoupled from the mean additive genetic value (or polygenic risk score) between populations, <span class="citation">(Berg and Coop 2014)</span></p>
</div>
</div>
<div id="datasets" class="section level2" number="6.2">
<h2>
<span class="header-section-number">6.2</span> Datasets<a class="anchor" aria-label="anchor" href="#datasets"><i class="fas fa-link"></i></a>
</h2>
<div id="genomes" class="section level3" number="6.2.1">
<h3>
<span class="header-section-number">6.2.1</span> 1000 Genomes<a class="anchor" aria-label="anchor" href="#genomes"><i class="fas fa-link"></i></a>
</h3>
<p>As the reference for human genomic variation across diverse populations, we used the New York Genome Center high-coverage, phased .vcf files<span class="citation">(<span>“Index of /Vol1/Ftp/Data_collections/<span>1000g</span>_2504_high_coverage/Working/20201028_3202_phased/”</span> n.d.)</span> for the 2,504 individuals described in the 1000 Genomes phase 3 release.<span class="citation">(Consortium et al. 2015)</span> We then annotated those .vcf files with human SNP IDs from dbSNP release 9606.<span class="citation">(Smigielski et al. 2000)</span></p>
</div>
<div id="gwas-catalog" class="section level3" number="6.2.2">
<h3>
<span class="header-section-number">6.2.2</span> GWAS Catalog<a class="anchor" aria-label="anchor" href="#gwas-catalog"><i class="fas fa-link"></i></a>
</h3>
<p>We used the R package <em>gwasrapidd</em><span class="citation">(Magno and Maia 2020)</span> to query all traits in the GWAS Catalog<span class="citation">(MacArthur et al. 2017)</span> as of 9 August 2021 (<span class="math inline">\(N_{TRAITS}\)</span> = 3,459). For 541 of these traits, no matching variant IDs could be pulled out from the 1000 Genomes VCFs, leaving <span class="math inline">\(N_{TRAITS}\)</span> = 3,008.</p>
</div>
<div id="linkage-disequilibrium" class="section level3" number="6.2.3">
<h3>
<span class="header-section-number">6.2.3</span> Linkage disequilibrium<a class="anchor" aria-label="anchor" href="#linkage-disequilibrium"><i class="fas fa-link"></i></a>
</h3>
<p>To obtain the “trait SNP” dataset, for each trait, we sought to isolate the SNP closest to each of its true causal variants, and exclude the SNPs in linkage disequilibrium (LD) with them. To this end, we used PLINK 1.9<span class="citation">(Chang et al. 2015; Purcell and Chang, n.d.)</span> to “clump” the SNPs associated with each of the remaining 3,008 traits, using an index variant p-value threshold of <span class="math inline">\(10^{-8}\)</span>,<span class="citation">(Panagiotou, Ioannidis, and Genome-Wide Significance Project 2012)</span> <span class="math inline">\(r^2\)</span> threshold of 0.1,<span class="citation">(Hill and Robertson 1968)</span> and base window size of 1 Mb. This process left us with 2,045 traits with at least one index SNP that met the p-value threshold. The index SNPs for each trait formed our set of trait SNPs. In order to target relatively polygenic traits, we further filtered out traits with fewer than 10 trait SNPs, leaving <span class="math inline">\(N_{TRAITS}\)</span> = 587. Supplementary Fig. 4 shows the counts of unique SNP IDs associated with each trait before and after clumping, and an interactive version is available in the notebook cited above.</p>
</div>
<div id="control-snps" class="section level3" number="6.2.4">
<h3>
<span class="header-section-number">6.2.4</span> Control SNPs<a class="anchor" aria-label="anchor" href="#control-snps"><i class="fas fa-link"></i></a>
</h3>
<p>To obtain our “control SNP” dataset, we assigned each trait SNP to one of 20 bins based on its minor allele frequency in European populations (as provided in the original 1000 Genomes .vcf files under the column header ‘INFO/AC_EUR’). For example, if a trait SNP had a minor allele frequency of 0.08 in European populations, it was assigned to the <span class="math inline">\((0.05, 0.1]\)</span> bin. We did the same for all (unassociated) SNPs in the .vcf files, then paired each trait SNP with a random SNP from the .vcf file in the equivalent bin. These allele-frequency-paired random SNPs formed our set of “control SNPs”, which we used to infer the <span class="math inline">\(F_{ST}\)</span> distribution of a random set of SNPs with the same allele frequencies as the trait SNPs, and against which we could compare the <span class="math inline">\(F_{ST}\)</span> distribution of the trait SNPs.</p>
</div>
<div id="f_st-and-ranking-traits-by-signed-kolmogorov-smirnov-d-statistic" class="section level3" number="6.2.5">
<h3>
<span class="header-section-number">6.2.5</span> <span class="math inline">\(F_{ST}\)</span> and ranking traits by signed Kolmogorov-Smirnov <span class="math inline">\(D\)</span> statistic<a class="anchor" aria-label="anchor" href="#f_st-and-ranking-traits-by-signed-kolmogorov-smirnov-d-statistic"><i class="fas fa-link"></i></a>
</h3>
<p>We then calculated <span class="math inline">\(F_{ST}\)</span> for each of the trait SNPs and their matched control SNPs using the Weir and Cockerham method,<span class="citation">(Weir and Cockerham 1984)</span> as implemented in the R package <em>pegas</em>.<span class="citation">(Paradis 2010)</span> To rank all traits based on the directional difference in <span class="math inline">\(F_{ST}\)</span> distributions between trait and control SNPs, we ran three Kolmogorov-Smirnov (KS) tests for each trait <span class="math inline">\(t\)</span> with <span class="math inline">\(x_t\)</span> = <span class="math inline">\(F_{ST, trait SNPs}\)</span> and <span class="math inline">\(y_t\)</span> = <span class="math inline">\(F_{ST, control SNPs}\)</span>:</p>
<ol style="list-style-type: decimal">
<li><p>two-sided (<span class="math inline">\(D_t\)</span>) ;</p></li>
<li><p>one-sided “greater” (<span class="math inline">\(D_t^+\)</span>) ; and</p></li>
<li><p>one-sided “less” (<span class="math inline">\(D_t^-\)</span>).</p></li>
</ol>
<p>I note that <span class="math inline">\(D_{t} = max(D_t^+, D_t^-)\)</span>, where <span class="math inline">\(D_t^+\)</span> is the greatest vertical distance attained by the eCDF of <span class="math inline">\(x_t\)</span> over the eCDF of <span class="math inline">\(y_t\)</span>, and <span class="math inline">\(D_t^-\)</span> is the greatest vertical distance attained by the eCDF of <span class="math inline">\(y_t\)</span> over the eCDF of <span class="math inline">\(x_t\)</span>.<span class="citation">(Conover 1999; Durbin 1973)</span> Accordingly, we used a comparison of <span class="math inline">\(D_t^+\)</span> and <span class="math inline">\(D_t^-\)</span> to created a signed D statistic (<span class="math inline">\(D\)</span>), based on the logic that trait SNPs with a lower overall <span class="math inline">\(F_{ST}\)</span> than control SNPs tend to have a higher <span class="math inline">\(D\)</span> under the “greater” test than the “less” test, and vice versa.</p>
<p>Therefore, <span class="math inline">\({D_t^S}\)</span>:</p>
<p><span class="math display">\[
\begin{aligned}
D_t^- &gt; D_t^+ &amp;: &amp;-D_t \\
D_t^- = D_t^+ &amp;: &amp;0 \\
D_t^- &lt; D_t^+ &amp;: &amp;D_t \\
\end{aligned}
\]</span></p>
<p>In <strong>Figure <a href="Fst-chap.html#fig:FstMain">6.1</a></strong> we present the <span class="math inline">\(F_{ST}\)</span> distributions of trait SNPs for an illustrative subset of 28 human traits, ranked by <span class="math inline">\({D_t^S}\)</span> when compared with their matched control SNPs. <strong>Figure <a href="Fst-chap.html#fig:FstMain">6.1</a>A</strong> shows the densities of SNPs as a function of <span class="math inline">\(F_{ST}\)</span>, and <strong>Figure <a href="Fst-chap.html#fig:FstMain">6.1</a>B and C</strong> show their empirical Cumulative Distribution Functions (eCDFs). <strong>Figure <a href="Fst-chap.html#fig:FstMain">6.1</a>B</strong> includes the eCDFs of control SNPs in grey. eCDF figures for all 587 traits that passed our filters (Methods) are provided in Supplementary Fig. 1.</p>

<div class="figure">
<span style="display:block;" id="fig:FstMain"></span>
<img src="figs/fst/20220314_final_0.1_1000.png" alt="Distributions of \(F_{ST}\) across 28 illustrative human traits, ranked by signed-D (Kolmogorov-Smirnov test) comparing trait and control SNPs. A. \(F_{ST}\) density ridge plots with SNP markers. B. Empirical Cumulative Distribution Functions (eCDFs) of \(F_{ST}\) for trait-associated (colour) and random control (grey) SNPs, faceted by trait. C. Consolidated eCDFs of trait-associated SNPs from (B). eCDFs for all traits are included in Supplementary Figure 1." width="100%"><p class="caption">
Figure 6.1: Distributions of <span class="math inline">\(F_{ST}\)</span> across 28 illustrative human traits, ranked by signed-D (Kolmogorov-Smirnov test) comparing trait and control SNPs. <strong>A</strong>. <span class="math inline">\(F_{ST}\)</span> density ridge plots with SNP markers. <strong>B</strong>. Empirical Cumulative Distribution Functions (eCDFs) of <span class="math inline">\(F_{ST}\)</span> for trait-associated (colour) and random control (grey) SNPs, faceted by trait. <strong>C</strong>. Consolidated eCDFs of trait-associated SNPs from (B). eCDFs for all traits are included in Supplementary Figure 1.
</p>
</div>

</div>
</div>
</div>
  <div class="chapter-nav">
<div class="prev"><a href="MIKK-F2-chap.html"><span class="header-section-number">5</span> Bold/shy behaviours in the MIKK panel</a></div>
<div class="next"><a href="references.html">References</a></div>
</div></main><div class="col-md-3 col-lg-2 d-none d-md-block sidebar sidebar-chapter">
    <nav id="toc" data-toggle="toc" aria-label="On this page"><h2>On this page</h2>
      <ul class="nav navbar-nav">
<li><a class="nav-link" href="#Fst-chap"><span class="header-section-number">6</span> Variation in the frequency of trait-associated alleles across global human populations</a></li>
<li>
<a class="nav-link" href="#background"><span class="header-section-number">6.1</span> Background</a><ul class="nav navbar-nav">
<li><a class="nav-link" href="#polygenic-risk-scores-prs-and-genome-wide-association-studies-gwas"><span class="header-section-number">6.1.1</span> Polygenic Risk Scores (PRS) and Genome-Wide Association Studies (GWAS)</a></li>
<li><a class="nav-link" href="#effect-of-the-environment-on"><span class="header-section-number">6.1.2</span> Effect of the environment on</a></li>
<li><a class="nav-link" href="#discovery-of-causal-variants"><span class="header-section-number">6.1.3</span> Discovery of causal variants</a></li>
</ul>
</li>
<li>
<a class="nav-link" href="#datasets"><span class="header-section-number">6.2</span> Datasets</a><ul class="nav navbar-nav">
<li><a class="nav-link" href="#genomes"><span class="header-section-number">6.2.1</span> 1000 Genomes</a></li>
<li><a class="nav-link" href="#gwas-catalog"><span class="header-section-number">6.2.2</span> GWAS Catalog</a></li>
<li><a class="nav-link" href="#linkage-disequilibrium"><span class="header-section-number">6.2.3</span> Linkage disequilibrium</a></li>
<li><a class="nav-link" href="#control-snps"><span class="header-section-number">6.2.4</span> Control SNPs</a></li>
<li><a class="nav-link" href="#f_st-and-ranking-traits-by-signed-kolmogorov-smirnov-d-statistic"><span class="header-section-number">6.2.5</span> \(F_{ST}\) and ranking traits by signed Kolmogorov-Smirnov \(D\) statistic</a></li>
</ul>
</li>
</ul>

      <div class="book-extra">
        <ul class="list-unstyled">
<li><a id="book-source" href="https://github.com/brettellebi/PhD-thesis/blob/master/book/06-Fst.Rmd">View source <i class="fab fa-github"></i></a></li>
          <li><a id="book-edit" href="https://github.com/brettellebi/PhD-thesis/edit/master/book/06-Fst.Rmd">Edit this page <i class="fab fa-github"></i></a></li>
        </ul>
</div>
    </nav>
</div>

</div>
</div> <!-- .container -->

<footer class="bg-primary text-light mt-5"><div class="container"><div class="row">

  <div class="col-12 col-md-6 mt-3">
    <p>"<strong>Genetic analysis of quantitative traits in medaka fish and humans</strong>" was written by Ian Brettell. It was last built on 2022-06-14.</p>
  </div>

  <div class="col-12 col-md-6 mt-3">
    <p>This book was built by the <a class="text-light" href="https://bookdown.org">bookdown</a> R package.</p>
  </div>

</div></div>
</footer><!-- dynamically load mathjax for compatibility with self-contained --><script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script><script type="text/x-mathjax-config">const popovers = document.querySelectorAll('a.footnote-ref[data-toggle="popover"]');
for (let popover of popovers) {
  const div = document.createElement('div');
  div.setAttribute('style', 'position: absolute; top: 0, left:0; width:0, height:0, overflow: hidden; visibility: hidden;');
  div.innerHTML = popover.getAttribute('data-content');

  var has_math = div.querySelector("span.math");
  if (has_math) {
    document.body.appendChild(div);
    MathJax.Hub.Queue(["Typeset", MathJax.Hub, div]);
    MathJax.Hub.Queue(function() {
      popover.setAttribute('data-content', div.innerHTML);
      document.body.removeChild(div);
    })
  }
}
</script>
</body>
</html>
