<!DOCTYPE html>
<html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<title>Chapter 2 Genomic variations in the MIKK panel | Genetic analysis of quantitative traits in medaka fish and humans</title>
<meta name="author" content="Ian Brettell">
<meta name="description" content="This project was carried out in collaboration with Felix Loosli’s group at the Karlsruhe Institute of Technology (KIT), and Jochen Wittbrodt’s group in the Centre for Organismal Studies (COS) at...">
<meta name="generator" content="bookdown 0.26 with bs4_book()">
<meta property="og:title" content="Chapter 2 Genomic variations in the MIKK panel | Genetic analysis of quantitative traits in medaka fish and humans">
<meta property="og:type" content="book">
<meta property="og:description" content="This project was carried out in collaboration with Felix Loosli’s group at the Karlsruhe Institute of Technology (KIT), and Jochen Wittbrodt’s group in the Centre for Organismal Studies (COS) at...">
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="Chapter 2 Genomic variations in the MIKK panel | Genetic analysis of quantitative traits in medaka fish and humans">
<meta name="twitter:description" content="This project was carried out in collaboration with Felix Loosli’s group at the Karlsruhe Institute of Technology (KIT), and Jochen Wittbrodt’s group in the Centre for Organismal Studies (COS) at...">
<!-- JS --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://kit.fontawesome.com/6ecbd6c532.js" crossorigin="anonymous"></script><script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="libs/bootstrap-4.6.0/bootstrap.min.css" rel="stylesheet">
<script src="libs/bootstrap-4.6.0/bootstrap.bundle.min.js"></script><script src="libs/bs3compat-0.3.1/transition.js"></script><script src="libs/bs3compat-0.3.1/tabs.js"></script><script src="libs/bs3compat-0.3.1/bs3compat.js"></script><link href="libs/bs4_book-1.0.0/bs4_book.css" rel="stylesheet">
<script src="libs/bs4_book-1.0.0/bs4_book.js"></script><style>
    #content {
        font-size: 0.9rem;
    }
    </style>
<script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- CSS --><style type="text/css">
    
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  </style>
<style type="text/css">
    /* Used with Pandoc 2.11+ new --citeproc when CSL is used */
    div.csl-bib-body { }
    div.csl-entry {
      clear: both;
        }
    .hanging div.csl-entry {
      margin-left:2em;
      text-indent:-2em;
    }
    div.csl-left-margin {
      min-width:2em;
      float:left;
    }
    div.csl-right-inline {
      margin-left:2em;
      padding-left:1em;
    }
    div.csl-indent {
      margin-left: 2em;
    }
  </style>
<link rel="stylesheet" href="style.css">
</head>
<body data-spy="scroll" data-target="#toc">

<div class="container-fluid">
<div class="row">
  <header class="col-sm-12 col-lg-3 sidebar sidebar-book"><a class="sr-only sr-only-focusable" href="#content">Skip to main content</a>

    <div class="d-flex align-items-start justify-content-between">
      <h1>
        <a href="index.html" title="">Genetic analysis of quantitative traits in medaka fish and humans</a>
      </h1>
      <button class="btn btn-outline-primary d-lg-none ml-2 mt-1" type="button" data-toggle="collapse" data-target="#main-nav" aria-expanded="true" aria-controls="main-nav"><i class="fas fa-bars"></i><span class="sr-only">Show table of contents</span></button>
    </div>

    <div id="main-nav" class="collapse-lg">
      <form role="search">
        <input id="search" class="form-control" type="search" placeholder="Search" aria-label="Search">
</form>

      <nav aria-label="Table of contents"><h2>Table of contents</h2>
        <ul class="book-toc list-unstyled"><li><a class="" href="index.html"><span class="header-section-number">Chapter 2</span> Genomic variations in the MIKK panel</a></li></ul>

        <div class="book-extra">
          <p><a id="book-repo" href="https://github.com/brettellebi/PhD-thesis">View book source <i class="fab fa-github"></i></a></p>
        </div>
      </nav>
</div>
  </header><main class="col-sm-12 col-md-9 col-lg-7" id="content"><div id="genomic-variations-in-the-mikk-panel" class="section level1" number="2">
<h1>
<span class="header-section-number">2</span> Genomic variations in the MIKK panel<a class="anchor" aria-label="anchor" href="#genomic-variations-in-the-mikk-panel"><i class="fas fa-link"></i></a>
</h1>
<p>This project was carried out in collaboration with Felix Loosli’s group at the Karlsruhe Institute of Technology (KIT), and Jochen Wittbrodt’s group in the Centre for Organismal Studies (COS) at the University of Heidelberg.</p>
<p>This chapter will set out my contributions to the the following pair of papers published in <em>Genome Biology</em>, on both of which I am joint-first author:</p>
<ul>
<li><p><span class="citation">Tomas Fitzgerald et al.<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;&lt;a href="references.html#ref-fitzgerald2022" role="doc-biblioref"&gt;&lt;span&gt;“The Medaka Inbred Kiyosu-Karlsruhe (MIKK) Panel,”&lt;/span&gt; &lt;em&gt;Genome Biology&lt;/em&gt; 23, no. 1 (February 21, 2022): 59, &lt;/a&gt;&lt;a href="https://doi.org/10.1186/s13059-022-02623-z" role="doc-biblioref"&gt;https://doi.org/10.1186/s13059-022-02623-z&lt;/a&gt;.&lt;/p&gt;'><sup>1</sup></a></span></p></li>
<li><p><span class="citation">Adrien Leger et al.<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;&lt;a href="references.html#ref-leger2022" role="doc-biblioref"&gt;&lt;span&gt;“Genomic Variations and Epigenomic Landscape of the Medaka Inbred Kiyosu-Karlsruhe (MIKK) Panel,”&lt;/span&gt; &lt;em&gt;Genome Biology&lt;/em&gt; 23, no. 1 (February 21, 2022): 58, &lt;/a&gt;&lt;a href="https://doi.org/10.1186/s13059-022-02602-4" role="doc-biblioref"&gt;https://doi.org/10.1186/s13059-022-02602-4&lt;/a&gt;.&lt;/p&gt;'><sup>2</sup></a></span></p></li>
</ul>
<div id="genomic-characterisation-of-the-mikk-panel" class="section level2" number="2.1">
<h2>
<span class="header-section-number">2.1</span> Genomic characterisation of the MIKK panel<a class="anchor" aria-label="anchor" href="#genomic-characterisation-of-the-mikk-panel"><i class="fas fa-link"></i></a>
</h2>
<div id="non-sib-calls" class="section level3" number="2.1.1">
<h3>
<span class="header-section-number">2.1.1</span> The MIKK panel Illumina callset<a class="anchor" aria-label="anchor" href="#non-sib-calls"><i class="fas fa-link"></i></a>
</h3>
<p>The MIKK Illumina callset contained SNP and INDEL calls for 79 of the 80 extant MIKK panel lines. In order to avoid allele frequency biases introduced by the 16 pairs/triplets of “sibling lines”, I removed each pair’s arbitrarily labelled second sibling line from the variant call set, leaving 63 MIKK panel lines for this analysis (<strong>MIKK non-sibling calls</strong>), and used only those calls for the analyses in Chapters <a href="genomic-variations-in-the-mikk-panel.html#nuc-div">2.1.4</a>, <a href="genomic-variations-in-the-mikk-panel.html#ld-decay-sec">2.1.5</a> and xxx.</p>
</div>
<div id="assessing-the-inbreeding-trajectory-of-the-mikk-panel" class="section level3" number="2.1.2">
<h3>
<span class="header-section-number">2.1.2</span> Assessing the inbreeding trajectory of the MIKK panel<a class="anchor" aria-label="anchor" href="#assessing-the-inbreeding-trajectory-of-the-mikk-panel"><i class="fas fa-link"></i></a>
</h3>
<p>The Medaka Inbred Kiyosu-Karlsruhe (MIKK) panel was bred from a wild population of medaka found in the Kiyosu area near Toyohashi, Aichi Prefecture, in southern Japan.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;&lt;a href="references.html#ref-spivakov2014" role="doc-biblioref"&gt;Mikhail Spivakov et al., &lt;span&gt;“Genomic and Phenotypic Characterization of a Wild Medaka Population: Towards the Establishment of an Isogenic Population Genetic Resource in Fish,”&lt;/span&gt; &lt;em&gt;G3 Genes|Genomes|Genetics&lt;/em&gt; 4, no. 3 (March 1, 2014): 433–45, &lt;/a&gt;&lt;a href="https://doi.org/10.1534/g3.113.008722" role="doc-biblioref"&gt;https://doi.org/10.1534/g3.113.008722&lt;/a&gt;.&lt;/p&gt;'><sup>3</sup></a></span> From this wild population, the Loosli Group at KIT set up random crosses of single mating pairs to create 115 ‘founder families’. For each founder family, they then set up between two and five single full-sibling-pair inbreeding crosses, which resulted in 253 F1 lines. Lines derived from the same founder family are referred to as ‘sibling lines’. Over the course of the next eight generations of inbreeding, they used only one mating pair per line. I generated Fig. <a href="genomic-variations-in-the-mikk-panel.html#fig:InbreedingFigure">2.1</a>A and B from the inbreeding data provided by the Loosli Group. Fig. <a href="genomic-variations-in-the-mikk-panel.html#fig:InbreedingFigure">2.1</a>A shows the number of lines that survived over the course of the first 14 generations of the inbreeding program, and the various causes for the termination of other lines. Fig. <a href="genomic-variations-in-the-mikk-panel.html#fig:InbreedingFigure">2.1</a>B shows the average fecundity levels of the surviving lines at generation F16. In addition, the Birney Group at EMBL-EBI generated morphometric data for the MIKK panel lines to demonstrate the distribution of physical phenotypes across the MIKK panel. I used this data on relative eye diameters to generate Fig. <a href="genomic-variations-in-the-mikk-panel.html#fig:InbreedingFigure">2.1</a>C.</p>

<div class="figure">
<span style="display:block;" id="fig:InbreedingFigure"></span>
<img src="figs/mikk_genome/20211213_final_figure.png" alt="Inbreeding,fecundity and eye size in the MIKK panel lines. A: Status of all MIKK panel lines during the first 14 generations of inbreeding, showing cause of death for non-extant lines. B: Average fecundity of MIKK panel lines in generation F16, as measured during peak egg production in July 2020. C: Distribution of mean relative eye size for female and male medaka across all MIKK panel lines." width="100%"><p class="caption">
Figure 2.1: Inbreeding,fecundity and eye size in the MIKK panel lines. <strong>A</strong>: Status of all MIKK panel lines during the first 14 generations of inbreeding, showing cause of death for non-extant lines. <strong>B</strong>: Average fecundity of MIKK panel lines in generation F16, as measured during peak egg production in July 2020. <strong>C</strong>: Distribution of mean relative eye size for female and male medaka across all MIKK panel lines.
</p>
</div>
</div>
<div id="introgression-with-northern-japanese-and-korean-medaka-populations" class="section level3" number="2.1.3">
<h3>
<span class="header-section-number">2.1.3</span> Introgression with northern Japanese and Korean medaka populations<a class="anchor" aria-label="anchor" href="#introgression-with-northern-japanese-and-korean-medaka-populations"><i class="fas fa-link"></i></a>
</h3>
<p>To explore the evolutionary history of the MIKK panel’s founding population, we sought to determine whether there was evidence of introgression between that southern Japanese population, and northern Japanese and Korean medaka populations. [Refer to background on northern medaka speciating away from southern medaka.] To this end, I used the 50-fish multiple alignment from Ensembl release 102 to obtain the aligned genome sequences for the established medaka inbred lines <em>HdrR</em> (southern Japan), <em>HNI</em> (northern Japan), and <em>HSOK</em> (Korea), as well as the most recent common ancestor of all three strains.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;&lt;a href="references.html#ref-IndexPubRelease102" role="doc-biblioref"&gt;&lt;span&gt;“Index of /Pub/Release-102/Emf/Ensembl-Compara/Multiple_alignments/50_fish.epo/,”&lt;/span&gt; accessed January 25, 2022, &lt;/a&gt;&lt;a href="https://ftp.ensembl.org/pub/release-102/emf/ensembl-compara/multiple_alignments/50_fish.epo/" role="doc-biblioref"&gt;https://ftp.ensembl.org/pub/release-102/emf/ensembl-compara/multiple_alignments/50_fish.epo/&lt;/a&gt;.&lt;/p&gt;'><sup>4</sup></a></span> I then carried out an ABBA BABA analysis to calculate a modified ‘admixture proportion’ statistic <span class="math inline">\(\hat{f}_d\)</span><span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;&lt;a href="references.html#ref-martinEvaluatingUseABBA2015" role="doc-biblioref"&gt;Simon H. Martin, John W. Davey, and Chris D. Jiggins, &lt;span&gt;“Evaluating the &lt;span&gt;Use&lt;/span&gt; of &lt;span&gt;ABBA&lt;/span&gt;–&lt;span&gt;BABA Statistics&lt;/span&gt; to &lt;span&gt;Locate Introgressed Loci&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Molecular Biology and Evolution&lt;/em&gt; 32, no. 1 (January 1, 2015): 244–57, &lt;/a&gt;&lt;a href="https://doi.org/10.1093/molbev/msu269" role="doc-biblioref"&gt;https://doi.org/10.1093/molbev/msu269&lt;/a&gt;.&lt;/p&gt;'><sup>5</sup></a></span> as a measure of the proportion of shared genome in 500-kb sliding windows between the MIKK panel and either <em>iCab</em>, <em>HNI</em>, or <em>HSOK</em> (<a href="genomic-variations-in-the-mikk-panel.html#fig:ABBABABA">2.2</a>.</p>

<div class="figure">
<span style="display:block;" id="fig:ABBABABA"></span>
<img src="figs/mikk_genome/07_introgression.png" alt="Figure 2: ABBA-BABA analysis. A. Phylogenetic tree generated from the Ensembl release 102 50-fish multiple alignment, showing only the medaka lines used in the ABBA-BABA analysis. B. Schema of the comparisons carried out in the ABBA-BABA analysis. C. Circos plot comparing introgression (\(\hat{f}_d\)) between the MIKK panel and either iCab (yellow), HNI (orange), or HSOK (purple), calculated within 500-kb sliding windows using a minimum of 250 SNPs per window." width="100%"><p class="caption">
Figure 2.2: <strong>Figure 2</strong>: ABBA-BABA analysis. <strong>A</strong>. Phylogenetic tree generated from the Ensembl release 102 50-fish multiple alignment, showing only the medaka lines used in the ABBA-BABA analysis. <strong>B</strong>. Schema of the comparisons carried out in the ABBA-BABA analysis. <strong>C</strong>. Circos plot comparing introgression (<span class="math inline">\(\hat{f}_d\)</span>) between the MIKK panel and either <em>iCab</em> (yellow), <em>HNI</em> (orange), or <em>HSOK</em> (purple), calculated within 500-kb sliding windows using a minimum of 250 SNPs per window.
</p>
</div>
<p>Based on the genome-wide mean <span class="math inline">\(\hat{f}_d\)</span>, the MIKK panel shares approximately 25% of its genome with <em>iCab</em>, 9% with <em>HNI</em>, and 12% with <em>HSOK</em>. These results provide evidence that the MIKK panel’s originating population has more recently introgressed with medaka from Korea than with medaka from northern Japan. This supports the findings in <span class="citation">Spivakov et al.<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;&lt;a href="references.html#ref-spivakov2014" role="doc-biblioref"&gt;&lt;span&gt;“Genomic and Phenotypic Characterization of a Wild Medaka Population”&lt;/span&gt;&lt;/a&gt;.&lt;/p&gt;'><sup>6</sup></a></span>, where the authors found little evidence of significant interbreeding between southern and northern Japanese medaka since the populations diverged.</p>
</div>
<div id="nuc-div" class="section level3" number="2.1.4">
<h3>
<span class="header-section-number">2.1.4</span> Nucleotide diversity<a class="anchor" aria-label="anchor" href="#nuc-div"><i class="fas fa-link"></i></a>
</h3>
<p>As an additional means of assessing genetic diversity in the MIKK panel, I calculated nucleotide diversity (<span class="math inline">\(\pi\)</span>) within 500-kb non-overlapping windows across the genome of 63 of the 80 MIKK panel lines (having excluded one line from each pair of sibling lines), and compared this to the nucleotide diversity in 7 wild medaka from the same Kiyosu population from which the MIKK panel was derived. Mean and median nucleotide diversity in both the MIKK panel and wild Kiyosu medaka were close to 0, and slightly higher in the MIKK panel (mean: MIKK = 0.0038, wild = 0.0037; median: MIKK = 0.0033, wild = 0.0031). The patterns of varying nucleotide diversity across the genome are shared between the MIKK panel and wild Kiyosu medaka, where regions with high levels of repeat content tend to have higher nucleotide diversity (r = 0.386, p &lt; 0.001) (<a href="genomic-variations-in-the-mikk-panel.html#fig:NucleotideDiversity">2.3</a>).</p>

<div class="figure">
<span style="display:block;" id="fig:NucleotideDiversity"></span>
<img src="figs/mikk_genome/supp_01_pi_circos.png" alt="Circos plot with nucleotide diversity (\(\pi\)) calculated within 500-kb non-overlapping windows for 63 non-sibling lines from the MIKK panel (green) and 7 wild Kiyosu medaka samples from the same originating population (purple); proportion of sequence classified as repeats by RepeatMasker (blue); and mean mapping quality (pink)." width="100%"><p class="caption">
Figure 2.3: Circos plot with nucleotide diversity (<span class="math inline">\(\pi\)</span>) calculated within 500-kb non-overlapping windows for 63 non-sibling lines from the MIKK panel (<em>green</em>) and 7 wild Kiyosu medaka samples from the same originating population (<em>purple</em>); proportion of sequence classified as repeats by RepeatMasker (<em>blue</em>); and mean mapping quality (<em>pink</em>).
</p>
</div>
<p>The higher level of <span class="math inline">\(\hat{\pi}\)</span> observed within specific regions on several chromosomes – such as chromosomes 2, 11, and 18 – correspond closely to the regions we identified as containing large (&gt;250 kb) inversions that appear to be shared across at least some of the MIKK panel (Fig. <a href="genomic-variations-in-the-mikk-panel.html#fig:SVInvs">2.4</a>). These regions are also enriched for large deletions and duplications.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;&lt;a href="references.html#ref-leger2022" role="doc-biblioref"&gt;Leger et al., &lt;span&gt;“Genomic Variations and Epigenomic Landscape of the Medaka Inbred Kiyosu-Karlsruhe (MIKK) Panel”&lt;/span&gt;&lt;/a&gt;.&lt;/p&gt;'><sup>7</sup></a></span> Inversions cause permanent heterozygosity,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;&lt;a href="references.html#ref-hoffmannChromosomalInversionPolymorphisms2004" role="doc-biblioref"&gt;Ary A. Hoffmann, Carla M. Sgr‘o, and Andrew R. Weeks, &lt;span&gt;“Chromosomal Inversion Polymorphisms and Adaptation,”&lt;/span&gt; &lt;em&gt;Trends in Ecology &amp;amp; Evolution&lt;/em&gt; 19, no. 9 (September 1, 2004): 482–88, &lt;/a&gt;&lt;a href="https://doi.org/10.1016/j.tree.2004.06.013" role="doc-biblioref"&gt;https://doi.org/10.1016/j.tree.2004.06.013&lt;/a&gt;.&lt;/p&gt;'><sup>8</sup></a></span> and duplications and deletions may have increased the density of called SNPs in these regions,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;&lt;a href="references.html#ref-fredmanComplexSNPrelatedSequence2004" role="doc-biblioref"&gt;David Fredman et al., &lt;span&gt;“Complex &lt;span class="nocase"&gt;SNP-related&lt;/span&gt; Sequence Variation in Segmental Genome Duplications,”&lt;/span&gt; &lt;em&gt;Nature Genetics&lt;/em&gt; 36, no. 8, 8 (August 2004): 861–66, &lt;/a&gt;&lt;a href="https://doi.org/10.1038/ng1401" role="doc-biblioref"&gt;https://doi.org/10.1038/ng1401&lt;/a&gt;.&lt;/p&gt;'><sup>9</sup></a></span> so the observed depressions in homozygosity at these loci may be the result of such large structural variants that are present in the MIKK panel’s genomes.</p>

<div class="figure">
<span style="display:block;" id="fig:SVInvs"></span>
<img src="figs/mikk_genome/20210224_sv_invs_lines.png" alt="Inversions identified in 9 MIKK panel lines using a combination of Oxford Nanopore Technologies long-read and Illumina short-read sequences (see Chapter 2.2 below)." width="100%"><p class="caption">
Figure 2.4: Inversions identified in 9 MIKK panel lines using a combination of Oxford Nanopore Technologies long-read and Illumina short-read sequences (see Chapter <a href="genomic-variations-in-the-mikk-panel.html#mikksv-sec">2.2</a> below).
</p>
</div>
</div>
<div id="ld-decay-sec" class="section level3" number="2.1.5">
<h3>
<span class="header-section-number">2.1.5</span> LD decay<a class="anchor" aria-label="anchor" href="#ld-decay-sec"><i class="fas fa-link"></i></a>
</h3>
<p>I analysed the MIKK panel’s allele frequency distribution and linkage disequilibrium (LD) structure to assess their likely effects on genetic mapping. To remove allele-frequency biases introduced by the presence of sibling lines in the MIKK panel, I used only the MIKK non-sibling calls (see Chapter <a href="genomic-variations-in-the-mikk-panel.html#non-sib-calls">2.1.1</a>).</p>
<p>To assess how accurately one may be able to map genetic variants using the MIKK panel relative to a human dataset, I compared the MIKK panel’s minor allele frequency (MAF) distribution and LD structure against that of the 2504 humans in the 1KG Phase 3 release.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;&lt;a href="references.html#ref-GlobalReferenceHuman2015" role="doc-biblioref"&gt;&lt;span&gt;“A Global Reference for Human Genetic Variation,”&lt;/span&gt; &lt;em&gt;Nature&lt;/em&gt; 526, no. 7571 (2015): 68–74, &lt;/a&gt;&lt;a href="https://doi.org/10.1038/nature15393" role="doc-biblioref"&gt;https://doi.org/10.1038/nature15393&lt;/a&gt;.&lt;/p&gt;'><sup>10</sup></a></span> To prepare the “1KG calls”, we first downloaded the VCFs for each autosome from the project’s FTP site (<a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/release/20130502/" class="uri">ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/release/20130502/</a>), then merged them into a single VCF using GATK.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;&lt;a href="references.html#ref-mckennaGenomeAnalysisToolkit2010" role="doc-biblioref"&gt;Aaron McKenna et al., &lt;span&gt;“The &lt;span&gt;Genome Analysis Toolkit&lt;/span&gt;: &lt;span&gt;A MapReduce&lt;/span&gt; Framework for Analyzing Next-Generation &lt;span&gt;DNA&lt;/span&gt; Sequencing Data,”&lt;/span&gt; &lt;em&gt;Genome Research&lt;/em&gt; 20, no. 9 (January 9, 2010): 1297–1303, &lt;/a&gt;&lt;a href="https://doi.org/10.1101/gr.107524.110" role="doc-biblioref"&gt;https://doi.org/10.1101/gr.107524.110&lt;/a&gt;.&lt;/p&gt;'><sup>11</sup></a></span></p>
<p>I then used PLINK<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;&lt;a href="references.html#ref-changSecondgenerationPLINKRising2015" role="doc-biblioref"&gt;Christopher C Chang et al., &lt;span&gt;“Second-Generation &lt;span&gt;PLINK&lt;/span&gt;: Rising to the Challenge of Larger and Richer Datasets,”&lt;/span&gt; &lt;em&gt;GigaScience&lt;/em&gt; 4, no. 1 (December 2015): 7, &lt;/a&gt;&lt;a href="https://doi.org/10.1186/s13742-015-0047-8" role="doc-biblioref"&gt;https://doi.org/10.1186/s13742-015-0047-8&lt;/a&gt;; &lt;a href="references.html#ref-purcellPLINK" role="doc-biblioref"&gt;Shaun M Purcell and Christopher C Chang, &lt;em&gt;&lt;span&gt;PLINK&lt;/span&gt; 1.9&lt;/em&gt;, n.d., &lt;/a&gt;&lt;a href="https://www.cog-genomics.org/plink/1.9/" role="doc-biblioref"&gt;www.cog-genomics.org/plink/1.9/&lt;/a&gt;.&lt;/p&gt;'><sup>12</sup></a></span> to calculate the minor allele frequencies for all non-missing, biallelic SNPs in both the MIKK non-sibling and IKG calls (<em>N</em> SNPs = 16,395,558 and 81,042,381 respectively), and then used R<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;&lt;a href="references.html#ref-rcoreteamLanguageEnvironmentStatistical2021" role="doc-biblioref"&gt;R Core Team, &lt;em&gt;R: &lt;span&gt;A&lt;/span&gt; Language and Environment for Statistical Computing&lt;/em&gt;, version 4.1.0 (&lt;span&gt;Vienna, Austria&lt;/span&gt;: &lt;span&gt;R Foundation for Statistical Computing&lt;/span&gt;, 2021), &lt;/a&gt;&lt;a href="https://www.R-project.org/" role="doc-biblioref"&gt;https://www.R-project.org/&lt;/a&gt;.&lt;/p&gt;'><sup>13</sup></a></span> and the <em>tidyverse</em> suite of R packages<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;&lt;a href="references.html#ref-wickhamWelcomeTidyverse2019" role="doc-biblioref"&gt;Hadley Wickham et al., &lt;span&gt;“Welcome to the &lt;span&gt;Tidyverse&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Journal of Open Source Software&lt;/em&gt; 4, no. 43 (November 21, 2019): 1686, &lt;/a&gt;&lt;a href="https://doi.org/10.21105/joss.01686" role="doc-biblioref"&gt;https://doi.org/10.21105/joss.01686&lt;/a&gt;.&lt;/p&gt;'><sup>14</sup></a></span> to produce Fig. <a href="genomic-variations-in-the-mikk-panel.html#fig:LDdecay">2.5</a>. To visualize the MIKK panel’s LD structure, I first used PLINK to filter the MIKK non-sibling calls for non-missing, biallelic SNPs with MAF ≥ 0.10, leaving 2,968,786 SNPs. I then used PLINK to take a random sample of 3000 SNPs per chromosome and recode them for analysis in Haploview.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;&lt;a href="references.html#ref-barrettHaploviewAnalysisVisualization2005" role="doc-biblioref"&gt;J. C. Barrett et al., &lt;span&gt;“Haploview: Analysis and Visualization of &lt;span&gt;LD&lt;/span&gt; and Haplotype Maps,”&lt;/span&gt; &lt;em&gt;Bioinformatics&lt;/em&gt; 21, no. 2 (January 15, 2005): 263–65, &lt;/a&gt;&lt;a href="https://doi.org/10.1093/bioinformatics/bth457" role="doc-biblioref"&gt;https://doi.org/10.1093/bioinformatics/bth457&lt;/a&gt;.&lt;/p&gt;'><sup>15</sup></a></span> We used Haploview to generate LD plots covering the length of each of the 24 chromosomes in the default color scheme, showing <span class="math inline">\(r^2\)</span> values for each pair of SNPs within 1 Mb of each other.</p>
<p>To determine the rate of LD decay in the MIKK panel and compare it to that in the 1KG sample, for both the MIKK non-sibling calls and the 1KG calls, we used PLINK to compute r2 on each autosome for all pairs of non-missing, biallelic SNPs with MAF &gt; 0.10 within 10 kb of one another (for 1KG and the MIKK panel respectively ~ 5.5M and ~ 3M SNPs, with a total number of pairwise r2 observations of 204,152,922 and 146,785,673). I then used R and the tidyverse suite of R packages to group the <span class="math inline">\(r^2\)</span> observations for each pair of SNPs based on their distance from one another into non-overlapping bins of 100 bp in length, then calculated the mean <span class="math inline">\(r^2\)</span> in each of those bins and generated the plots in Fig. <a href="genomic-variations-in-the-mikk-panel.html#fig:LDdecay">2.5</a> using the mean <span class="math inline">\(r^2\)</span> and left boundary of each bin. All code used for this section can be found here: <a href="https://birneylab.github.io/MIKK_genome_main_paper/02_LD_decay.html" class="uri">https://birneylab.github.io/MIKK_genome_main_paper/02_LD_decay.html</a>.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;&lt;a href="references.html#ref-MIKKPanelGenome2021" role="doc-biblioref"&gt;&lt;em&gt;&lt;span&gt;MIKK&lt;/span&gt; Panel Genome Analysis&lt;/em&gt; (2021; repr., &lt;span&gt;birneylab&lt;/span&gt;, 2021), &lt;/a&gt;&lt;a href="https://github.com/birneylab/MIKK_genome_main_paper" role="doc-biblioref"&gt;https://github.com/birneylab/MIKK_genome_main_paper&lt;/a&gt;.&lt;/p&gt;'><sup>16</sup></a></span></p>
<p>Fig. <a href="genomic-variations-in-the-mikk-panel.html#fig:LDdecay">2.5</a> compares the allele frequency distribution for the 16.4M biallelic SNPs identified in those filtered calls, with the allele frequency distribution of the 81M biallelic SNPs in the 1000 Genomes Project Phase 3 release (N = 2,504) (<strong>1KG</strong>) in human. As expected, the 1KG and MIKK panel calls are similarly enriched for low-frequency variants, albeit to a lesser extent in the MIKK panel, which is likely due to its smaller sample size.</p>

<div class="figure">
<span style="display:block;" id="fig:LDdecay"></span>
<img src="figs/mikk_genome/08_ld_decay.png" alt="Linkage disequilibrium on chr 17, and LD decay in the MIKK panel and 1000 Genomes human dataset. A: Minor allele frequency distributions and LD decay for biallelic, non-missing SNPs in the 1000 Genomes Phase 3 variant calls (N = 2,504) (1KG), and the MIKK panel Illumina-based calls excluding one of each pair of sibling lines (N = 63), across all autosomes (1KG: chrs 1-22; MIKK: chrs 1-24); Density of allele frequencies in the 1KG and MIKK panel calls. B: LD decay for each autosome, calculated by taking the mean r2 of pairs of SNPs with MAF &gt; 0.1 within non-overlapping 100 bp windows of distance from one another, up to a maximum of 10 kb. Inset: mean r2 within 100 bp windows, up to a maximum of 1 kb. LD decays faster on chromosome 2 for the MIKK panel due to its higher recombination rate, consistent with the genetic linkage map described in (6). B: Heatmap showing the genotypes of 63 non-sibling MIKK panel lines across the region of high LD on chr17. C: Figure generated by Haploview17 of a large region on chr17 with high LD." width="100%"><p class="caption">
Figure 2.5: Linkage disequilibrium on chr 17, and LD decay in the MIKK panel and 1000 Genomes human dataset. <strong>A</strong>: Minor allele frequency distributions and LD decay for biallelic, non-missing SNPs in the 1000 Genomes Phase 3 variant calls (N = 2,504) (1KG), and the MIKK panel Illumina-based calls excluding one of each pair of sibling lines (N = 63), across all autosomes (1KG: chrs 1-22; MIKK: chrs 1-24); Density of allele frequencies in the 1KG and MIKK panel calls. <strong>B</strong>: LD decay for each autosome, calculated by taking the mean r2 of pairs of SNPs with MAF &gt; 0.1 within non-overlapping 100 bp windows of distance from one another, up to a maximum of 10 kb. <strong>Inset</strong>: mean r2 within 100 bp windows, up to a maximum of 1 kb. LD decays faster on chromosome 2 for the MIKK panel due to its higher recombination rate, consistent with the genetic linkage map described in (6). B: Heatmap showing the genotypes of 63 non-sibling MIKK panel lines across the region of high LD on chr17. <strong>C</strong>: Figure generated by Haploview<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;&lt;a href="references.html#ref-barrettHaploviewAnalysisVisualization2005" role="doc-biblioref"&gt;Barrett et al., &lt;span&gt;“Haploview”&lt;/span&gt;&lt;/a&gt;.&lt;/p&gt;'><sup>17</sup></a></span> of a large region on chr17 with high LD.
</p>
</div>
<p>To review the MIKK panel’s LD structure, for each autosome in humans (chromosomes 1-22) and each chromosome in medaka (chromosomes 1-24), we calculated <span class="math inline">\(r^2\)</span> between all pairs of biallelic SNPs with a minor allele frequency (MAF) greater than 0.10, within 10 kb of one another [elaborate from Methods].</p>
<p>We then grouped the paired SNPs by distance from one another into non-overlapping 100 bp windows, and calculated the mean <span class="math inline">\(r^2\)</span> for each window in order to represent how LD decays with distance between loci. Fig <em><a href="genomic-variations-in-the-mikk-panel.html#fig:LDdecay">2.5</a>B</em> compares the LD decay between the 1KG and MIKK panel SNPs. Based on the 1KG calls under these parameters, LD decays in humans to a mean <span class="math inline">\(r^2\)</span> of around 0.2-0.35 at a distance of 10 kb, whereas the MIKK panel reaches this level within 1 kb, with a mean <span class="math inline">\(r^2\)</span> of 0.3-0.4 at a distance of ~100 bp. This implies that when a causal variant is present in at least two lines in the MIKK panel, one may be able to map causal variants at a higher resolution than in humans. We note that LD decays faster in chromosome 2 of the MIKK panel relative to the other chromosomes. This suggests that it has a much higher recombination rate, which is consistent with the linkage map described in <a href="https://www.zotero.org/google-docs/?6iGiFi">(6)</a> showing a higher genetic distance per Mb for this chromosome. This higher recombination rate in chromosome 2 may in turn be caused by its relatively high proportion of repeat content (<strong>Fig. <a href="genomic-variations-in-the-mikk-panel.html#fig:repeats">2.6</a></strong>).</p>

<div class="figure">
<span style="display:block;" id="fig:repeats"></span>
<img src="figs/mikk_genome/20210401_hdrr_repeats_final.png" alt="Repeat content in the HdrR genome based on RepeatMasker results obtained by Jack Monahan. A. Proportion of repeat content per-chromosome. B. log10 of repeat lengths and counts per repeat class. “Misc” includes all repeats assigned to their own specific class, for example “(GAG)n” or “(GATCCA)n”. C. Circos plot showing repeat length (radial axes) by locus (angular axis) and repeat class (track)." width="100%"><p class="caption">
Figure 2.6: Repeat content in the <em>HdrR</em> genome based on RepeatMasker results obtained by Jack Monahan. <strong>A</strong>. Proportion of repeat content per-chromosome. <strong>B</strong>. log10 of repeat lengths and counts per repeat class. “Misc” includes all repeats assigned to their own specific class, for example “(GAG)n” or “(GATCCA)n”. <strong>C</strong>. Circos plot showing repeat length (radial axes) by locus (angular axis) and repeat class (track).
</p>
</div>
</div>
</div>
<div id="mikksv-sec" class="section level2" number="2.2">
<h2>
<span class="header-section-number">2.2</span> Genomic variations and epigenetic landscape of the MIKK panel<a class="anchor" aria-label="anchor" href="#mikksv-sec"><i class="fas fa-link"></i></a>
</h2>
<p>As an alternative to the variation pangenome approach described in this paper,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;&lt;a href="references.html#ref-leger2022" role="doc-biblioref"&gt;Leger et al., &lt;span&gt;“Genomic Variations and Epigenomic Landscape of the Medaka Inbred Kiyosu-Karlsruhe (MIKK) Panel”&lt;/span&gt;&lt;/a&gt;.&lt;/p&gt;'><sup>18</sup></a></span> I explored structural variation (SVs) in a reference-anchored manner, similar to many human studies. Differences in SVs between panel lines is another important class of genetic variation that could cause or contribute to significant phenotypic differences. Here we used Nanopore sequencing data obtained for 9 of the 12 selected lines allowing us to characterise larger SVs in the MIKK panel and to create a more extensive picture of genomic rearrangements compared to available medaka reference genomes. The Birney Group first called structural variants using only the ONT long reads, producing a set of structural variants classified into five types: deletions (DEL), insertions (INS), translocations (TRA), duplications (DUP) and inversions (INV). I then “polished” the called DEL and INS variants with Illumina short reads to improve their accuracy. The polishing process filtered out 7.4% of DEL and 12.8% of INS variants, and adjusted the breakpoints (i.e. start and end positions) for 75-77% of DEL and INS variants in each sample by a mean of 23 bp for the start position, and 33 bp for the end position. This process produced a total of 143,326 filtered SVs.</p>
<p>The 9 “polished” samples contained a mean per-sample count of approximately 37K DEL variants (12% singletons), 29.5K INS variants (14%), 3.5K TRA variants (9%), 2.5K DUP (7%) and 600 INV (7%) (Fig. 4D). DEL variants were up to 494 kb in length, with 90% of unique DEL variants shorter than 3.8 kb. INS variants were only up to 13.8 kb in length, with 90% of unique INS variants shorter than 2 kb. DUP and INV variants tended to be longer, with a mean length of 19 and 70.5 kb respectively (Fig. 4A). Fig. 4E shows the per-sample distribution of DEL variants across the genome. Most large DEL variants over 250 kb in length were common among the MIKK panel lines. A number of large DEL variants appear to have accumulated within the 0-10 Mb region of chromosome 2, which is enriched for repeats in the HdrR reference genome (Fig. 6A).</p>
<p>SVs were generally enriched in regions covered by repeats. While only 16% of bases in the HdrR reference were classified as repeats (irrespective of strand), those bases overlapped with 72% of DEL, 63% of DUP, 81% of INV and 35% of TRA variant regions. However, repeat bases only overlapped with 21% of INS variants. We also assessed each SV’s probability of being loss-of-function (pLI) (39) by calculating the logarithm of odds (LOD) for the pLI scores of all genes overlapping the variant (<strong>Fig. <a href="genomic-variations-in-the-mikk-panel.html#fig:SV-main">2.7</a>B,C</strong>). 30,357 out of 134,088 DEL, INS, DUP and INV variants overlapped at least one gene, and 9% of those had a score greater than 10, indicating a high probability that the SV would cause a loss of function. Two INS variants on chr2 had an outlying LOD score of 57 as a result of overlapping medaka gene ENSORLG00000003411, which has a pLI score of 1 – the highest intolerance to variants causing a loss of function. This gene is homologous with human genes SCN1A, SCN2A and SCN3A, which encode sodium channels and have been associated with neuronal and sleep disorders. We did not find evidence that longer SVs tended to have a higher probability of causing a loss of function (<strong>Fig. <a href="genomic-variations-in-the-mikk-panel.html#fig:SV-main">2.7</a>B</strong>).</p>

<div class="figure">
<span style="display:block;" id="fig:SV-main"></span>
<img src="figs/mikk_genome/20210408_sv_main.png" alt="Polished SVs in 9 MIKK panel lines sequenced with ONT. DEL: deletion; INS: insertion; TRA: translocation; DUP: duplication; INV: inversion. A. Aggregate log10 counts and lengths of distinct SVs by type, excluding TRA. B. pLI LOD scores in distinct SVs by SV type. C. Histogram of LOD scores by SV type. D. Total and singleton counts of SV types per sample. E. Circos plot showing per-sample distribution and lengths of DEL variants across the genome." width="100%"><p class="caption">
Figure 2.7: Polished SVs in 9 MIKK panel lines sequenced with ONT. DEL: deletion; INS: insertion; TRA: translocation; DUP: duplication; INV: inversion. <strong>A</strong>. Aggregate log10 counts and lengths of distinct SVs by type, excluding TRA. <strong>B</strong>. pLI LOD scores in distinct SVs by SV type. <strong>C</strong>. Histogram of LOD scores by SV type. <strong>D</strong>. Total and singleton counts of SV types per sample. <strong>E</strong>. Circos plot showing per-sample distribution and lengths of DEL variants across the genome.
</p>
</div>
<p>We compared these polished INS and DEL calls with the high-quality graph-based alternative paths and large-scale deletions, respectively. We found that 2 of the 19 regions covered by graph-based alternative paths, and 4 of the 16 regions covered by graph-based deletions (see section titled <em>Novel genetic sequences and large-scale insertions and deletions in the MIKK panel</em> in <span class="citation">Leger et al.<a href="#fn19" class="footnote-ref" id="fnref19"><sup>19</sup></a></span>), had no SVs that overlapped those regions at all, which suggests they would have been missed entirely when using a reference-anchored approach alone.</p>
<p>With the exception of one alternative path on chromosome 20, the alternative paths were not captured by INS variants, which only covered up to 63% of the bases in each region, and in many cases substantially less. On the other hand, for 8 of the 16 graph-based deletions, the DEL variants covered at least 85% of the bases in those regions. The other 8 graph-based deletions were either not at all covered by DEL variants, or only slightly. This indicates that the reference-based approach is better at detecting large-scale deletions than alternative paths (“insertions”), but still misses around half of such variants relative to the graph-based approach.</p>

</div>
</div>

  <div class="chapter-nav">
<div class="empty"></div>
<div class="empty"></div>
</div></main><div class="col-md-3 col-lg-2 d-none d-md-block sidebar sidebar-chapter">
    <nav id="toc" data-toggle="toc" aria-label="On this page"><h2>On this page</h2>
      <div id="book-on-this-page"></div>

      <div class="book-extra">
        <ul class="list-unstyled">
          
        </ul>
</div>
    </nav>
</div>

</div>
</div> <!-- .container -->

<footer class="bg-primary text-light mt-5"><div class="container"><div class="row">

  <div class="col-12 col-md-6 mt-3">
    <p>"<strong>Genetic analysis of quantitative traits in medaka fish and humans</strong>" was written by Ian Brettell. It was last built on 2022-05-26.</p>
  </div>

  <div class="col-12 col-md-6 mt-3">
    <p>This book was built by the <a class="text-light" href="https://bookdown.org">bookdown</a> R package.</p>
  </div>

</div></div>
</footer><!-- dynamically load mathjax for compatibility with self-contained --><script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script><script type="text/x-mathjax-config">const popovers = document.querySelectorAll('a.footnote-ref[data-toggle="popover"]');
for (let popover of popovers) {
  const div = document.createElement('div');
  div.setAttribute('style', 'position: absolute; top: 0, left:0; width:0, height:0, overflow: hidden; visibility: hidden;');
  div.innerHTML = popover.getAttribute('data-content');

  var has_math = div.querySelector("span.math");
  if (has_math) {
    document.body.appendChild(div);
    MathJax.Hub.Queue(["Typeset", MathJax.Hub, div]);
    MathJax.Hub.Queue(function() {
      popover.setAttribute('data-content', div.innerHTML);
      document.body.removeChild(div);
    })
  }
}
</script>
</body>
</html>
