<!DOCTYPE html>
<html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<title>Chapter 2 Genomic variations in the MIKK panel | Genetic analysis of quantitative traits in medaka fish and humans</title>
<meta name="author" content="Ian Brettell">
<meta name="description" content="This project was carried out in collaboration with Felix Loosli’s group at the Karlsruhe Institute of Technology (KIT), and Jochen Wittbrodt’s group in the Centre for Organismal Studies (COS) at...">
<meta name="generator" content="bookdown 0.26 with bs4_book()">
<meta property="og:title" content="Chapter 2 Genomic variations in the MIKK panel | Genetic analysis of quantitative traits in medaka fish and humans">
<meta property="og:type" content="book">
<meta property="og:description" content="This project was carried out in collaboration with Felix Loosli’s group at the Karlsruhe Institute of Technology (KIT), and Jochen Wittbrodt’s group in the Centre for Organismal Studies (COS) at...">
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="Chapter 2 Genomic variations in the MIKK panel | Genetic analysis of quantitative traits in medaka fish and humans">
<meta name="twitter:description" content="This project was carried out in collaboration with Felix Loosli’s group at the Karlsruhe Institute of Technology (KIT), and Jochen Wittbrodt’s group in the Centre for Organismal Studies (COS) at...">
<!-- JS --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://kit.fontawesome.com/6ecbd6c532.js" crossorigin="anonymous"></script><script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="libs/bootstrap-4.6.0/bootstrap.min.css" rel="stylesheet">
<script src="libs/bootstrap-4.6.0/bootstrap.bundle.min.js"></script><script src="libs/bs3compat-0.3.1/transition.js"></script><script src="libs/bs3compat-0.3.1/tabs.js"></script><script src="libs/bs3compat-0.3.1/bs3compat.js"></script><link href="libs/bs4_book-1.0.0/bs4_book.css" rel="stylesheet">
<script src="libs/bs4_book-1.0.0/bs4_book.js"></script><style>
    #content {
        font-size: 0.9rem;
    }
    </style>
<script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- CSS --><style type="text/css">
    
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  </style>
<style type="text/css">
    /* Used with Pandoc 2.11+ new --citeproc when CSL is used */
    div.csl-bib-body { }
    div.csl-entry {
      clear: both;
        }
    .hanging div.csl-entry {
      margin-left:2em;
      text-indent:-2em;
    }
    div.csl-left-margin {
      min-width:2em;
      float:left;
    }
    div.csl-right-inline {
      margin-left:2em;
      padding-left:1em;
    }
    div.csl-indent {
      margin-left: 2em;
    }
  </style>
<link rel="stylesheet" href="style.css">
</head>
<body data-spy="scroll" data-target="#toc">

<div class="container-fluid">
<div class="row">
  <header class="col-sm-12 col-lg-3 sidebar sidebar-book"><a class="sr-only sr-only-focusable" href="#content">Skip to main content</a>

    <div class="d-flex align-items-start justify-content-between">
      <h1>
        <a href="index.html" title="">Genetic analysis of quantitative traits in medaka fish and humans</a>
      </h1>
      <button class="btn btn-outline-primary d-lg-none ml-2 mt-1" type="button" data-toggle="collapse" data-target="#main-nav" aria-expanded="true" aria-controls="main-nav"><i class="fas fa-bars"></i><span class="sr-only">Show table of contents</span></button>
    </div>

    <div id="main-nav" class="collapse-lg">
      <form role="search">
        <input id="search" class="form-control" type="search" placeholder="Search" aria-label="Search">
</form>

      <nav aria-label="Table of contents"><h2>Table of contents</h2>
        <ul class="book-toc list-unstyled">
<li><a class="" href="index.html"><span class="header-section-number">1</span> About</a></li>
<li><a class="active" href="genomic-variations-in-the-mikk-panel.html"><span class="header-section-number">2</span> Genomic variations in the MIKK panel</a></li>
<li><a class="" href="references.html">References</a></li>
</ul>

        <div class="book-extra">
          <p><a id="book-repo" href="https://github.com/brettellebi/PhD-thesis">View book source <i class="fab fa-github"></i></a></p>
        </div>
      </nav>
</div>
  </header><main class="col-sm-12 col-md-9 col-lg-7" id="content"><div id="genomic-variations-in-the-mikk-panel" class="section level1" number="2">
<h1>
<span class="header-section-number">2</span> Genomic variations in the MIKK panel<a class="anchor" aria-label="anchor" href="#genomic-variations-in-the-mikk-panel"><i class="fas fa-link"></i></a>
</h1>
<p>This project was carried out in collaboration with Felix Loosli’s group at the Karlsruhe Institute of Technology (KIT), and Jochen Wittbrodt’s group in the Centre for Organismal Studies (COS) at the University of Heidelberg.</p>
<p>This chapter will set out my contributions to the the following pair of papers published in <em>Genome Biology</em>:</p>
<ul>
<li><p><span class="citation">Tomas Fitzgerald et al.<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;&lt;span&gt;“The Medaka Inbred Kiyosu-Karlsruhe (MIKK) Panel,”&lt;/span&gt; &lt;em&gt;Genome Biology&lt;/em&gt; 23, no. 1 (February 21, 2022): 59, &lt;a href="https://doi.org/10.1186/s13059-022-02623-z" role="doc-biblioref"&gt;https://doi.org/10.1186/s13059-022-02623-z&lt;/a&gt;.&lt;/p&gt;'><sup>1</sup></a></span></p></li>
<li><p><span class="citation">Adrien Leger et al.<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;&lt;span&gt;“Genomic Variations and Epigenomic Landscape of the Medaka Inbred Kiyosu-Karlsruhe (MIKK) Panel,”&lt;/span&gt; &lt;em&gt;Genome Biology&lt;/em&gt; 23, no. 1 (February 21, 2022): 58, &lt;a href="https://doi.org/10.1186/s13059-022-02602-4" role="doc-biblioref"&gt;https://doi.org/10.1186/s13059-022-02602-4&lt;/a&gt;.&lt;/p&gt;'><sup>2</sup></a></span></p></li>
</ul>
<div id="genomic-characterisation-of-the-medaka-inbred-karlsruhe-kiyosu-mikk-panel" class="section level2" number="2.1">
<h2>
<span class="header-section-number">2.1</span> Genomic characterisation of the Medaka Inbred Karlsruhe-Kiyosu (MIKK) panel<a class="anchor" aria-label="anchor" href="#genomic-characterisation-of-the-medaka-inbred-karlsruhe-kiyosu-mikk-panel"><i class="fas fa-link"></i></a>
</h2>
<div id="assessing-the-inbreeding-trajectory-of-the-mikk-panel" class="section level3" number="2.1.1">
<h3>
<span class="header-section-number">2.1.1</span> Assessing the inbreeding trajectory of the MIKK panel<a class="anchor" aria-label="anchor" href="#assessing-the-inbreeding-trajectory-of-the-mikk-panel"><i class="fas fa-link"></i></a>
</h3>
<p>The Medaka Inbred Kiyosu-Karlsruhe (MIKK) panel was bred from a wild population of medaka found in the Kiyosu area near Toyohashi, Aichi Prefecture, in southern Japan. <span class="citation">Mikhail Spivakov et al.<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;&lt;span&gt;“Genomic and Phenotypic Characterization of a Wild Medaka Population: Towards the Establishment of an Isogenic Population Genetic Resource in Fish,”&lt;/span&gt; &lt;em&gt;G3 Genes|Genomes|Genetics&lt;/em&gt; 4, no. 3 (March 1, 2014): 433–45, &lt;a href="https://doi.org/10.1534/g3.113.008722" role="doc-biblioref"&gt;https://doi.org/10.1534/g3.113.008722&lt;/a&gt;.&lt;/p&gt;'><sup>3</sup></a></span> From this wild population, the Loosli Group at KIT set up random crosses of single mating pairs to create 115 ‘founder families’. For each founder family, they then set up between two and five single full-sibling-pair inbreeding crosses, which resulted in 253 F1 lines. Lines derived from the same founder family are referred to as ‘sibling lines’. Over the course of the next eight generations of inbreeding, they used only one mating pair per line. I generated Fig. <a href="genomic-variations-in-the-mikk-panel.html#fig:InbreedingFigure">2.1</a>A and B from the inbreeding data provided by the Loosli Group. Fig. <a href="genomic-variations-in-the-mikk-panel.html#fig:InbreedingFigure">2.1</a>A shows the number of lines that survived over the course of the first 14 generations of the inbreeding program, and the various causes for the termination of other lines. Fig. <a href="genomic-variations-in-the-mikk-panel.html#fig:InbreedingFigure">2.1</a>B shows the average fecundity levels of the surviving lines at generation F16. In addition, the Birney Group at EMBL-EBI generated morphometric data for the MIKK panel lines to demonstrate the distribution of physical phenotypes across the MIKK panel. I used this data on relative eye diameters to generate Fig. <a href="genomic-variations-in-the-mikk-panel.html#fig:InbreedingFigure">2.1</a>C.</p>

<div class="figure">
<span style="display:block;" id="fig:InbreedingFigure"></span>
<img src="figs/mikk_genome/20211213_final_figure.png" alt="Inbreeding,fecundity and eye size in the MIKK panel lines. A: Status of all MIKK panel lines during the first 14 generations of inbreeding, showing cause of death for non-extant lines. B: Average fecundity of MIKK panel lines in generation F16, as measured during peak egg production in July 2020. C: Distribution of mean relative eye size for female and male medaka across all MIKK panel lines." width="100%"><p class="caption">
Figure 2.1: Inbreeding,fecundity and eye size in the MIKK panel lines. <strong>A</strong>: Status of all MIKK panel lines during the first 14 generations of inbreeding, showing cause of death for non-extant lines. <strong>B</strong>: Average fecundity of MIKK panel lines in generation F16, as measured during peak egg production in July 2020. <strong>C</strong>: Distribution of mean relative eye size for female and male medaka across all MIKK panel lines.
</p>
</div>
</div>
<div id="introgression-with-northern-japanese-and-korean-medaka-populations" class="section level3" number="2.1.2">
<h3>
<span class="header-section-number">2.1.2</span> Introgression with northern Japanese and Korean medaka populations<a class="anchor" aria-label="anchor" href="#introgression-with-northern-japanese-and-korean-medaka-populations"><i class="fas fa-link"></i></a>
</h3>
<p>To explore the evolutionary history of the MIKK panel’s founding population, we sought to determine whether there was evidence of introgression between that southern Japanese population, and northern Japanese and Korean medaka populations. [Refer to background on northern medaka speciating away from southern medaka.] To this end, I used the 50-fish multiple alignment from Ensembl release 102 to obtain the aligned genome sequences for the established medaka inbred lines HdrR (southern Japan), HNI (northern Japan), and HSOK (Korea), as well as the most recent common ancestor of all three strains.<a href="https://www.zotero.org/google-docs/?RA3Ett">(2)</a> I then carried out an ABBA BABA analysis to calculate a modified ‘admixture proportion’ statistic fd <a href="https://www.zotero.org/google-docs/?9loUI7">(3)</a> as a measure of the proportion of shared genome in 500-kb sliding windows between the MIKK panel and either iCab, HNI, or HSOK (<strong>Fig. 2</strong>).</p>

<div class="figure">
<span style="display:block;" id="fig:ABBABABA"></span>
<img src="figs/mikk_genome/07_introgression.png" alt="Figure 2: ABBA-BABA analysis. A. Phylogenetic tree generated from the Ensembl release 102 50-fish multiple alignment, showing only the medaka lines used in the ABBA-BABA analysis. B. Schema of the comparisons carried out in the ABBA-BABA analysis. C. Circos plot comparing introgression (fd ) between the MIKK panel and either iCab (yellow), HNI (orange), or HSOK (purple), calculated within 500-kb sliding windows using a minimum of 250 SNPs per window." width="100%"><p class="caption">
Figure 2.2: <strong>Figure 2</strong>: ABBA-BABA analysis. <strong>A</strong>. Phylogenetic tree generated from the Ensembl release 102 50-fish multiple alignment, showing only the medaka lines used in the ABBA-BABA analysis. B. Schema of the comparisons carried out in the ABBA-BABA analysis. C. Circos plot comparing introgression (fd ) between the MIKK panel and either iCab (yellow), HNI (orange), or HSOK (purple), calculated within 500-kb sliding windows using a minimum of 250 SNPs per window.
</p>
</div>
<p>Based on the genome-wide mean <span class="math inline">\(\hat{f}_d\)</span>, the MIKK panel shares approximately 25% of its genome with iCab, 9% with HNI, and 12% with HSOK. These results provide evidence that the MIKK panel’s originating population has more recently introgressed with medaka from Korea than with medaka from northern Japan. This supports the findings in <span class="citation">Spivakov et al.<a href="#fn4" class="footnote-ref" id="fnref4"><sup>4</sup></a></span>, where the authors found little evidence of significant interbreeding between southern and northern Japanese medaka since the populations diverged.</p>
</div>
<div id="nucleotide-diversity" class="section level3" number="2.1.3">
<h3>
<span class="header-section-number">2.1.3</span> Nucleotide diversity<a class="anchor" aria-label="anchor" href="#nucleotide-diversity"><i class="fas fa-link"></i></a>
</h3>
<p>As an additional means of assessing genetic diversity in the MIKK panel, I calculated nucleotide diversity (π) within 500-kb non-overlapping windows across the genome of 63 of the 80 MIKK panel lines (having excluded one line from each pair of sibling lines), and compared this to the nucleotide diversity in 7 wild medaka from the same Kiyosu population from which the MIKK panel was derived. Mean and median nucleotide diversity in both the MIKK panel and wild Kiyosu medaka were close to 0, and slightly higher in the MIKK panel (mean : MIKK = 0.0038, wild = 0.0037; median : MIKK = 0.0033, wild = 0.0031). The patterns of varying nucleotide diversity across the genome are shared between the MIKK panel and wild Kiyosu medaka, where regions with high levels of repeat content tend to have higher nucleotide diversity (r = 0.386, p &lt; 0.001) <a href="genomic-variations-in-the-mikk-panel.html#fig:NucleotideDiversity">2.3</a>.</p>
<div class="figure">
<span style="display:block;" id="fig:NucleotideDiversity"></span>
<img src="figs/mikk_genome/supp_01_pi_circos.png" alt="(ref:NucleotideDiversity)" width="100%"><p class="caption">
Figure 2.3: (ref:NucleotideDiversity)
</p>
</div>

</div>
</div>
</div>

  <div class="chapter-nav">
<div class="prev"><a href="index.html"><span class="header-section-number">1</span> About</a></div>
<div class="next"><a href="references.html">References</a></div>
</div></main><div class="col-md-3 col-lg-2 d-none d-md-block sidebar sidebar-chapter">
    <nav id="toc" data-toggle="toc" aria-label="On this page"><h2>On this page</h2>
      <ul class="nav navbar-nav">
<li><a class="nav-link" href="#genomic-variations-in-the-mikk-panel"><span class="header-section-number">2</span> Genomic variations in the MIKK panel</a></li>
<li>
<a class="nav-link" href="#genomic-characterisation-of-the-medaka-inbred-karlsruhe-kiyosu-mikk-panel"><span class="header-section-number">2.1</span> Genomic characterisation of the Medaka Inbred Karlsruhe-Kiyosu (MIKK) panel</a><ul class="nav navbar-nav">
<li><a class="nav-link" href="#assessing-the-inbreeding-trajectory-of-the-mikk-panel"><span class="header-section-number">2.1.1</span> Assessing the inbreeding trajectory of the MIKK panel</a></li>
<li><a class="nav-link" href="#introgression-with-northern-japanese-and-korean-medaka-populations"><span class="header-section-number">2.1.2</span> Introgression with northern Japanese and Korean medaka populations</a></li>
<li><a class="nav-link" href="#nucleotide-diversity"><span class="header-section-number">2.1.3</span> Nucleotide diversity</a></li>
</ul>
</li>
</ul>

      <div class="book-extra">
        <ul class="list-unstyled">
<li><a id="book-source" href="https://github.com/brettellebi/PhD-thesis/blob/master/book/02-MIKK_genome.Rmd">View source <i class="fab fa-github"></i></a></li>
          <li><a id="book-edit" href="https://github.com/brettellebi/PhD-thesis/edit/master/book/02-MIKK_genome.Rmd">Edit this page <i class="fab fa-github"></i></a></li>
        </ul>
</div>
    </nav>
</div>

</div>
</div> <!-- .container -->

<footer class="bg-primary text-light mt-5"><div class="container"><div class="row">

  <div class="col-12 col-md-6 mt-3">
    <p>"<strong>Genetic analysis of quantitative traits in medaka fish and humans</strong>" was written by Ian Brettell. It was last built on 2022-05-25.</p>
  </div>

  <div class="col-12 col-md-6 mt-3">
    <p>This book was built by the <a class="text-light" href="https://bookdown.org">bookdown</a> R package.</p>
  </div>

</div></div>
</footer><!-- dynamically load mathjax for compatibility with self-contained --><script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script><script type="text/x-mathjax-config">const popovers = document.querySelectorAll('a.footnote-ref[data-toggle="popover"]');
for (let popover of popovers) {
  const div = document.createElement('div');
  div.setAttribute('style', 'position: absolute; top: 0, left:0; width:0, height:0, overflow: hidden; visibility: hidden;');
  div.innerHTML = popover.getAttribute('data-content');

  var has_math = div.querySelector("span.math");
  if (has_math) {
    document.body.appendChild(div);
    MathJax.Hub.Queue(["Typeset", MathJax.Hub, div]);
    MathJax.Hub.Queue(function() {
      popover.setAttribute('data-content', div.innerHTML);
      document.body.removeChild(div);
    })
  }
}
</script>
</body>
</html>
