<!DOCTYPE html>
<html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<title>Chapter 1 Introduction | Japanese courage: a genetic analysis of complex traits in medaka fish and humans</title>
<meta name="author" content="Ian Brettell">
<meta name="description" content="Humankind has long sought to understand the basis of biological variation. What gives rise to the wondrous variety of life forms on Earth? Why do individuals of a particular species differ from...">
<meta name="generator" content="bookdown 0.28 with bs4_book()">
<meta property="og:title" content="Chapter 1 Introduction | Japanese courage: a genetic analysis of complex traits in medaka fish and humans">
<meta property="og:type" content="book">
<meta property="og:description" content="Humankind has long sought to understand the basis of biological variation. What gives rise to the wondrous variety of life forms on Earth? Why do individuals of a particular species differ from...">
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="Chapter 1 Introduction | Japanese courage: a genetic analysis of complex traits in medaka fish and humans">
<meta name="twitter:description" content="Humankind has long sought to understand the basis of biological variation. What gives rise to the wondrous variety of life forms on Earth? Why do individuals of a particular species differ from...">
<!-- JS --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://kit.fontawesome.com/6ecbd6c532.js" crossorigin="anonymous"></script><script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="libs/bootstrap-4.6.0/bootstrap.min.css" rel="stylesheet">
<script src="libs/bootstrap-4.6.0/bootstrap.bundle.min.js"></script><script src="libs/bs3compat-0.4.0/transition.js"></script><script src="libs/bs3compat-0.4.0/tabs.js"></script><script src="libs/bs3compat-0.4.0/bs3compat.js"></script><link href="libs/bs4_book-1.0.0/bs4_book.css" rel="stylesheet">
<script src="libs/bs4_book-1.0.0/bs4_book.js"></script><style>
    #content {
        font-size: 0.9rem;
    }
    </style>
<script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- CSS --><style type="text/css">
    
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  </style>
<style type="text/css">
    /* Used with Pandoc 2.11+ new --citeproc when CSL is used */
    div.csl-bib-body { }
    div.csl-entry {
      clear: both;
        }
    .hanging div.csl-entry {
      margin-left:2em;
      text-indent:-2em;
    }
    div.csl-left-margin {
      min-width:2em;
      float:left;
    }
    div.csl-right-inline {
      margin-left:2em;
      padding-left:1em;
    }
    div.csl-indent {
      margin-left: 2em;
    }
  </style>
<link rel="stylesheet" href="style.css">
</head>
<body data-spy="scroll" data-target="#toc">

<div class="container-fluid">
<div class="row">
  <header class="col-sm-12 col-lg-3 sidebar sidebar-book"><a class="sr-only sr-only-focusable" href="#content">Skip to main content</a>

    <div class="d-flex align-items-start justify-content-between">
      <h1>
        <a href="index.html" title="">Japanese courage: a genetic analysis of complex traits in medaka fish and humans</a>
      </h1>
      <button class="btn btn-outline-primary d-lg-none ml-2 mt-1" type="button" data-toggle="collapse" data-target="#main-nav" aria-expanded="true" aria-controls="main-nav"><i class="fas fa-bars"></i><span class="sr-only">Show table of contents</span></button>
    </div>

    <div id="main-nav" class="collapse-lg">
      <form role="search">
        <input id="search" class="form-control" type="search" placeholder="Search" aria-label="Search">
</form>

      <nav aria-label="Table of contents"><h2>Table of contents</h2>
        <ul class="book-toc list-unstyled">
<li><a class="" href="index.html">About</a></li>
<li><a class="active" href="Introduction.html"><span class="header-section-number">1</span> Introduction</a></li>
<li><a class="" href="MIKK-genomes-chap.html"><span class="header-section-number">2</span> Genomic variations in the MIKK panel</a></li>
<li><a class="" href="Pilot-chap.html"><span class="header-section-number">3</span> Classification of bold/shy behaviours in 5 inbred medaka lines</a></li>
<li><a class="" href="Somite-chap.html"><span class="header-section-number">4</span> Genetic loci associated with somite development periodicity</a></li>
<li><a class="" href="MIKK-F2-chap.html"><span class="header-section-number">5</span> Genetic linkage study of bold/shy behaviours in the MIKK panel</a></li>
<li><a class="" href="Fst-chap.html"><span class="header-section-number">6</span> Variation in the frequency of trait-associated alleles across global human populations</a></li>
<li><a class="" href="references.html">References</a></li>
<li class="book-part">Appendix</li>
<li><a class="" href="hmm-states-with-0.05-second-interval.html"><span class="header-section-number">A</span> 15 HMM states with 0.05 second interval</a></li>
<li><a class="" href="four-panel-app.html"><span class="header-section-number">B</span> Four-panel plots for MIKK lines selected for the F2 cross</a></li>
<li><a class="" href="hmm-state-time-dependence-for-all-mikk-panel-lines.html"><span class="header-section-number">C</span> HMM state time dependence for all MIKK panel lines</a></li>
<li><a class="" href="f2-recombination-karyoplot-with-missing-calls.html"><span class="header-section-number">D</span> F2 recombination karyoplot with missing calls</a></li>
<li><a class="" href="loco-grm-for-chromosome-1.html"><span class="header-section-number">E</span> LOCO GRM for chromosome 1</a></li>
<li><a class="" href="ecdf-of-all-polygenic-traits-in-the-gwas-catalog-ranked-by-d_ts.html"><span class="header-section-number">F</span> eCDF of all polygenic traits in the GWAS Catalog ranked by \({D_t^S}\)</a></li>
</ul>

        <div class="book-extra">
          <p><a id="book-repo" href="https://github.com/brettellebi/PhD-thesis">View book source <i class="fab fa-github"></i></a></p>
        </div>
      </nav>
</div>
  </header><main class="col-sm-12 col-md-9 col-lg-7" id="content"><div id="Introduction" class="section level1" number="1">
<h1>
<span class="header-section-number">1</span> Introduction<a class="anchor" aria-label="anchor" href="#Introduction"><i class="fas fa-link"></i></a>
</h1>
<p>Humankind has long sought to understand the basis of biological variation. What gives rise to the wondrous variety of life forms on Earth? Why do individuals of a particular species differ from one another? How do children inherit traits that are similar to those of their parents, yet on the whole remain distinct from both their parents and their siblings? And are the traits we care about – our health, our BEHAVIOUR [NEED TO GET THE SOCIAL ENVRIONMENT OUT] intelligence, our ability to thrive in a changing world – pre-determined from birth, or continuously pliable throughout our lives? These questions fundamentally concern the natural laws of inheritance, which until recently remained mysterious and obscure. Rapidly-improving technologies are making these questions increasingly tractable, yet the interplay between genes and environment are still the subject of controversy, especially in relation to human traits such as intelligence.</p>
<p>In this thesis, I primarily explore the extent to which phenotypic variation is determined by an individual’s own genes, and how much it is mediated by the genes of one’s social companions, with a particular focus on the phenotype of bold-type behaviours in the Japanese rice paddy fish <em>Oryzias latipes</em>. I first provide a brief history of the field, describing some of the work of the scientists on whose shoulders this research stands, from Charles Darwin’s theory of evolution, Mendel’s identification of the units of inheritance, and the study of continuous variation by Francis Galton, Karl Pearson, Ronald Fisher, and Sewall Wright, but also touching on their other legacies.[BRING EUGENICS INTO MAIN TEXT.] I then outline the traditional animal crossing methods and modern DNA sequencing and statistical techniques that I used to parse the respective contributions of genes and environment to variation in complex traits, and ultimately identify the genetic variants that drive those differences. [I’M ALSO GOING TO GIVE YOU AN INTRODUCTION THTE PHENOTYPES I STUDY IN MEDAKA]</p>
<div id="a-brief-history-of-genetics" class="section level2" number="1.1">
<h2>
<span class="header-section-number">1.1</span> A brief history of genetics<a class="anchor" aria-label="anchor" href="#a-brief-history-of-genetics"><i class="fas fa-link"></i></a>
</h2>
<p>[MUCH OF THIS INTRODUCTION AHS BEEN INFORMED BY MUKHERJEE, RUTHERFORD, ETC. WHERE POSSIBLE I HAVE QUOTED FROM THE ORIGINAL SOURCES.]</p>
<p>Much of this section has been informed by <span class="citation">Siddhartha Mukherjee<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;&lt;em&gt;The &lt;span&gt;Gene&lt;/span&gt;: &lt;span&gt;An Intimate History&lt;/span&gt;&lt;/em&gt; (&lt;span&gt;Simon and Schuster&lt;/span&gt;, 2016), &lt;a href="https://books.google.com?id=XvAsDAAAQBAJ" role="doc-biblioref"&gt;https://books.google.com?id=XvAsDAAAQBAJ&lt;/a&gt;.&lt;/p&gt;'><sup>1</sup></a></span>, <span class="citation">Adam Rutherford<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;&lt;em&gt;How to Argue with a Racist: &lt;span&gt;History&lt;/span&gt;, Science, Race and Reality&lt;/em&gt; (&lt;span&gt;Hachette UK&lt;/span&gt;, 2020).&lt;/p&gt;"><sup>2</sup></a></span>, and <span class="citation">James Tabery<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;&lt;em&gt;Beyond Versus: &lt;span&gt;The&lt;/span&gt; Struggle to Understand the Interaction of Nature and Nurture&lt;/em&gt; (&lt;span&gt;mit Press&lt;/span&gt;, 2014).&lt;/p&gt;"><sup>3</sup></a></span>. Where possible, I have cited the original sources.</p>
<div id="ancient-greece" class="section level3" number="1.1.1">
<h3>
<span class="header-section-number">1.1.1</span> Ancient Greece<a class="anchor" aria-label="anchor" href="#ancient-greece"><i class="fas fa-link"></i></a>
</h3>
<p>Throughout ancient history, the sources of biological variation were completely unknown. With limited technologies available to them, the Ancient Greek philosophers proposed theories for the inheritance of traits that they observed in humans. Around 500 BC, Pythagoras applied his knowledge of triangles to the question of how traits are inherited from one’s parents. He proposed the theory known as “spermism”, positing that hereditary information was passed down from parent to child via male sperm, with the female providing the nutrients that would allow it to grow. Like the theorem that bears his name, he supposed that these two sides of the “triangle” would determine the length of the third side: the characteristics of the child.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee, &lt;em&gt;The &lt;span&gt;Gene&lt;/span&gt;&lt;/em&gt;.&lt;/p&gt;"><sup>4</sup></a></span> Over a century later, in 380 BC, Plato extended this metaphor in <em>The Republic</em><span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Alain Badiou, &lt;em&gt;Plato’s &lt;span&gt;Republic&lt;/span&gt;&lt;/em&gt; (&lt;span&gt;John Wiley &amp;amp; Sons&lt;/span&gt;, 2013).&lt;/p&gt;"><sup>5</sup></a></span> to argue that this principle could be applied to perfect humanity by breeding perfect combinations of parents at perfect times.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee, &lt;em&gt;The &lt;span&gt;Gene&lt;/span&gt;&lt;/em&gt;.&lt;/p&gt;"><sup>6</sup></a></span> [THE FIRST EUGENICIST.]</p>
<p>Aristotle later joined the discussion with his treatise <em>Generation of Animals</em>,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Aristotle, &lt;em&gt;On the &lt;span&gt;Generation&lt;/span&gt; of &lt;span&gt;Animals&lt;/span&gt;&lt;/em&gt; (&lt;span&gt;Good Press&lt;/span&gt;, 2021), &lt;a href="https://books.google.com?id=aLUqEAAAQBAJ" role="doc-biblioref"&gt;https://books.google.com?id=aLUqEAAAQBAJ&lt;/a&gt;.&lt;/p&gt;'><sup>7</sup></a></span> where he noted that children inherited features from their mothers as well as their fathers, raising cases where human skin colour and other traits from maternal ancestors could skip generations, and thus sperm could not be the only vessel of hereditary information. He suggested an idea of “movement” – the transmission of information – from the father’s sperm, which sculpts the mother’s menstrual blood in the same way a carpenter carves a piece of wood.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee, &lt;em&gt;The &lt;span&gt;Gene&lt;/span&gt;&lt;/em&gt;.&lt;/p&gt;"><sup>8</sup></a></span> It was, however, impossible for Aristotle to deduce the form in which the information was conveyed.</p>
</div>
<div id="middle-ages" class="section level3" number="1.1.2">
<h3>
<span class="header-section-number">1.1.2</span> Middle Ages<a class="anchor" aria-label="anchor" href="#middle-ages"><i class="fas fa-link"></i></a>
</h3>
<p>From medieval times through to the 1800s, the prevailing theory of heredity was that a tiny human – a homunculus – sat within the sperm, waiting to be inflated upon its introduction to a woman’s uterus. But from where did each previous homunculus originate? Logically, the theory would require each homunculus to hold another homunculus, <em>ad infinitum</em>.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee.&lt;/p&gt;"><sup>9</sup></a></span> This theory deceived the inventor of the microscope, Nicolaas Hartsoeker, into thinking he observed a homunculus in a sperm he was studying (<strong>Figure <a href="Introduction.html#fig:homunculus-pic">1.1</a></strong>). But a fundamental question remained: what triggered the expansion of the human form, causing the embryo to develop new parts on its road to becoming fetus? The answer could only have been some instruction, blueprint, or code, but any specifics remained out of reach.</p>

<div class="figure" style="text-align: right">
<span style="display:block;" id="fig:homunculus-pic"></span>
<img src="figs/introduction/homunculus.png" alt="Preformation, drawn by Nicolaas Hartsoeker in 1695. Image adapted from Wikimedia Commons10." width="100%"><p class="caption">
Figure 1.1: Preformation, drawn by Nicolaas Hartsoeker in 1695. Image adapted from <span class="citation">Wikimedia Commons<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;&lt;span&gt;“File:preformation.GIF — Wikimedia Commons&lt;span&gt;,&lt;/span&gt; the Free Media Repository,”&lt;/span&gt; 2019, &lt;a href="%5Curl%7Bhttps://commons.wikimedia.org/w/index.php?title=File:Preformation.GIF&amp;amp;oldid=372547016%7D" role="doc-biblioref"&gt;\url{https://commons.wikimedia.org/w/index.php?title=File:Preformation.GIF&amp;amp;oldid=372547016}&lt;/a&gt;.&lt;/p&gt;'><sup>10</sup></a></span>.
</p>
</div>
</div>
<div id="charles-darwin" class="section level3" number="1.1.3">
<h3>
<span class="header-section-number">1.1.3</span> Charles Darwin<a class="anchor" aria-label="anchor" href="#charles-darwin"><i class="fas fa-link"></i></a>
</h3>
<p>During the early 1800s, the prevailing doctrine on the origins of biological variation was Creationism, based on Christianity’s literal interpretation of the Bible’s Book of Genesis.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Gordon Campbell, &lt;em&gt;King &lt;span&gt;James Bible&lt;/span&gt;: 400th &lt;span&gt;Anniversary Edition&lt;/span&gt;&lt;/em&gt; (&lt;span&gt;OUP Oxford&lt;/span&gt;, 2010).&lt;/p&gt;"><sup>11</sup></a></span> Any mechanistic description of how species – and individuals within the same species – differed from one another was thought to threaten this doctrine, making inquiries of this kind potentially blasphemous, and therefore dangerous.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee, &lt;em&gt;The &lt;span&gt;Gene&lt;/span&gt;&lt;/em&gt;.&lt;/p&gt;"><sup>12</sup></a></span></p>
<p>It was in this context that a 22-year-old English clergyman named Charles Darwin (<strong>Figure <a href="Introduction.html#fig:charles-darwin-young-portrait">1.2</a></strong>) boarded the <em>HMS Beagle</em> in 1831 to commence a voyage around the world that would last for almost five years. Darwin had previously studied theology at the University of Cambridge, but was drawn to study the natural world. He had apprenticed with his fellow clergyman John Henslow, a botanist and geologist who curated the Cambridge Botanic Garden, who suggested that he join the <em>Beagle</em>’s exploratory survey of South America as the “gentleman scientist” they were seeking to assist with the collection of specimens.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee.&lt;/p&gt;"><sup>13</sup></a></span></p>

<div class="figure" style="text-align: center">
<span style="display:block;" id="fig:charles-darwin-young-portrait"></span>
<img src="figs/introduction/Charles_Darwin_by_G._Richmond.png" alt="Portrait of Charles Darwin from the late 1830s by George Richmond.14" width="70%"><p class="caption">
Figure 1.2: Portrait of Charles Darwin from the late 1830s by George Richmond.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Wikimedia Commons, &lt;span&gt;“File:charles Darwin by g. Richmond.png — Wikimedia Commons&lt;span&gt;,&lt;/span&gt; the Free Media Repository,”&lt;/span&gt; 2017, &lt;a href="%5Curl%7Bhttps://commons.wikimedia.org/w/index.php?title=File:Charles_Darwin_by_G._Richmond.png&amp;amp;oldid=260695102%7D" role="doc-biblioref"&gt;\url{https://commons.wikimedia.org/w/index.php?title=File:Charles_Darwin_by_G._Richmond.png&amp;amp;oldid=260695102}&lt;/a&gt;.&lt;/p&gt;'><sup>14</sup></a></span>
</p>
</div>
<p>After rounding Cape Horn and moving northward along the western coast of South America, the <em>HMS Beagle</em> eventually reached the Galápagos Islands on the coast of Peru, an archipelago of 18 islands formed from volcanic lava.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee, &lt;em&gt;The &lt;span&gt;Gene&lt;/span&gt;&lt;/em&gt;.&lt;/p&gt;"><sup>15</sup></a></span> Over the course of five weeks, Darwin collected carcasses of birds, lizards, and plants.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee.&lt;/p&gt;"><sup>16</sup></a></span> Upon his return to England, he was hailed as a minor celebrity among natural historians due to the collections of specimens he had gathered and shipped back.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee.&lt;/p&gt;"><sup>17</sup></a></span> John Gould – the ornithologist who lent his (wife’s) name to the Gouldian finch (<strong>Figure <a href="Introduction.html#fig:gouldian-finch">1.3</a></strong>) – told him that the various birds that Darwin thought were a variety of wrens, warblers, blackbirds, and “gros-beaks” were in fact all 13 different species of finches.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee.&lt;/p&gt;"><sup>18</sup></a></span></p>

<div class="figure" style="text-align: right">
<span style="display:block;" id="fig:gouldian-finch"></span>
<img src="figs/introduction/gouldian_finch.png" alt="The Gouldian Finch, an Australian native bird described by British ornithological artist John Gould in 1844 and named after his deceased wife Elizabeth.19 Photograph by Sarah R. Pryke, published in Sarah R Pryke and Simon C Griffith20." width="100%"><p class="caption">
Figure 1.3: The Gouldian Finch, an Australian native bird described by British ornithological artist John Gould in 1844 and named after his deceased wife Elizabeth.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Paul Bancroft, &lt;span&gt;“Part 1: &lt;span&gt;An&lt;/span&gt; Introduction to the &lt;span&gt;Gouldian&lt;/span&gt; Finch - &lt;span&gt;Planet Aviary&lt;/span&gt;,”&lt;/span&gt; accessed August 22, 2022, &lt;a href="https://planetaviary.com/part-1-an-introduction-to-the-gouldian-finch/" role="doc-biblioref"&gt;https://planetaviary.com/part-1-an-introduction-to-the-gouldian-finch/&lt;/a&gt;.&lt;/p&gt;'><sup>19</sup></a></span> Photograph by Sarah R. Pryke, published in <span class="citation">Sarah R Pryke and Simon C Griffith<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;&lt;span&gt;“Red Dominates Black: Agonistic Signalling Among Head Morphs in the Colour Polymorphic &lt;span&gt;Gouldian&lt;/span&gt; Finch,”&lt;/span&gt; &lt;em&gt;Proceedings of the Royal Society B: Biological Sciences&lt;/em&gt; 273, no. 1589 (April 22, 2006): 949–57, &lt;a href="https://doi.org/10.1098/rspb.2005.3362" role="doc-biblioref"&gt;https://doi.org/10.1098/rspb.2005.3362&lt;/a&gt;.&lt;/p&gt;'><sup>20</sup></a></span>.
</p>
</div>
<p>Each island had produced its own variant (<strong>Figure <a href="Introduction.html#fig:darwin-finches">1.4</a></strong>), and this caused Darwin to consider whether they had all arisen from a common ancestral finch, branching off like the boughs of a tree over time [QUOTE ORIGIN OF SPECIES].<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee, &lt;em&gt;The &lt;span&gt;Gene&lt;/span&gt;&lt;/em&gt;.&lt;/p&gt;"><sup>21</sup></a></span> He understood that animal breeders took advantage of the natural variation in populations to select for desired traits, but he questioned what force had guided the development of these different varieties of finches in the wild.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee.&lt;/p&gt;"><sup>22</sup></a></span></p>

<div class="figure" style="text-align: left">
<span style="display:block;" id="fig:darwin-finches"></span>
<img src="figs/introduction/Darwin's_finches_by_Gould.png" alt="Illustration of variation in Galapagos finches by John Gould, published in Charles Darwin23. Image from Wikimedia Commons24." width="100%"><p class="caption">
Figure 1.4: Illustration of variation in Galapagos finches by John Gould, published in <span class="citation">Charles Darwin<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;&lt;em&gt;A &lt;span&gt;Naturalist&lt;/span&gt;’s &lt;span&gt;Voyage&lt;/span&gt;: &lt;span&gt;Journal&lt;/span&gt; of &lt;span&gt;Researches Into&lt;/span&gt; the &lt;span&gt;Natural History&lt;/span&gt; and &lt;span&gt;Geology&lt;/span&gt; of the &lt;span&gt;Countries Visited During&lt;/span&gt; the &lt;span&gt;Voyage&lt;/span&gt; of &lt;span&gt;H&lt;/span&gt;.&lt;span&gt;M&lt;/span&gt;.&lt;span&gt;S&lt;/span&gt;. &lt;span&gt;Beagle&lt;/span&gt; : With &lt;span&gt;Maps&lt;/span&gt; and &lt;span&gt;Illustrations&lt;/span&gt;&lt;/em&gt; (&lt;span&gt;Murray&lt;/span&gt;, 1882), &lt;a href="https://books.google.com?id=JyR7HY3FMtkC" role="doc-biblioref"&gt;https://books.google.com?id=JyR7HY3FMtkC&lt;/a&gt;.&lt;/p&gt;'><sup>23</sup></a></span>. Image from <span class="citation">Wikimedia Commons<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;&lt;span&gt;“File:darwin’s Finches by Gould.jpg — Wikimedia Commons&lt;span&gt;,&lt;/span&gt; the Free Media Repository,”&lt;/span&gt; 2022, &lt;a href="%5Curl%7Bhttps://commons.wikimedia.org/w/index.php?title=File:Darwin%27s_finches_by_Gould.jpg&amp;amp;oldid=643906085%7D" role="doc-biblioref"&gt;\url{https://commons.wikimedia.org/w/index.php?title=File:Darwin%27s_finches_by_Gould.jpg&amp;amp;oldid=643906085}&lt;/a&gt;.&lt;/p&gt;'><sup>24</sup></a></span>.
</p>
</div>
<p>Well prior to Darwin’s voyage, Thomas Malthus, a curate and amateur economist, had published a paper in titled <em>An Essay on the Principle of Population</em>,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Thomas Robert Malthus, &lt;em&gt;An &lt;span&gt;Essay&lt;/span&gt; on the &lt;span&gt;Principle&lt;/span&gt; of &lt;span&gt;Population&lt;/span&gt;..&lt;/em&gt;, 1872.&lt;/p&gt;"><sup>25</sup></a></span> in which he argued that the human population was in constant struggle with its limited resource pool, which in turn was affected by droughts, floods, epidemics, and diseases.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee, &lt;em&gt;The &lt;span&gt;Gene&lt;/span&gt;&lt;/em&gt;.&lt;/p&gt;"><sup>26</sup></a></span> Darwin read the paper and identified this struggle for resources as the natural hand that selected those who possessed traits favourable for survival.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee.&lt;/p&gt;"><sup>27</sup></a></span> He continued to gestate these ideas throughout the period from his return from the voyage in 1836 through to 1858, when he read a draft paper that had been sent to him by the author, Alfred Russel Wallace (<strong>Figure <a href="Introduction.html#fig:alfred-wallace">1.5</a></strong>).</p>
<p>Like Darwin, Wallace had set off on a voyage to distant lands, and had observed the stunning variation across the Malay Archipelago in populations separated by channels of water.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee.&lt;/p&gt;"><sup>28</sup></a></span> And like Darwin, he also derived his theory on the basis of this variation from Malthus’s paper.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee.&lt;/p&gt;"><sup>29</sup></a></span> The draft paper he sent to Darwin outlined his general theory of evolution and natural selection.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee.&lt;/p&gt;"><sup>30</sup></a></span> In a panic about being “scooped”, Darwin sent both papers to his geologist friend Charles Lyell, who advised Darwin to have both papers presented simultaneously at the meeting of the Linnean Society so that they could both be credited for the discovery.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee.&lt;/p&gt;"><sup>31</sup></a></span></p>

<div class="figure" style="text-align: left">
<span style="display:block;" id="fig:alfred-wallace"></span>
<img src="figs/introduction/Alfred-Russel-Wallace.png" alt="Alfred Russel Wallace, taken around 1895. Image from Wikimedia Commons32." width="100%"><p class="caption">
Figure 1.5: Alfred Russel Wallace, taken around 1895. Image from <span class="citation">Wikimedia Commons<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;&lt;span&gt;“File:alfred-Russel-Wallace-C1895.jpg — Wikimedia Commons&lt;span&gt;,&lt;/span&gt; the Free Media Repository,”&lt;/span&gt; 2022, &lt;a href="https://commons.wikimedia.org/w/index.php?title=File:Alfred-Russel-Wallace-c1895.jpg&amp;amp;oldid=642853479" role="doc-biblioref"&gt;https://commons.wikimedia.org/w/index.php?title=File:Alfred-Russel-Wallace-c1895.jpg&amp;amp;oldid=642853479&lt;/a&gt;.&lt;/p&gt;'><sup>32</sup></a></span>.
</p>
</div>
<p>The presentation made few waves at the time, but Darwin proceeded to complete his opus, <em>On the Origin of Species by Means of Natural Selection</em>.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Charles Darwin, &lt;em&gt;On the &lt;span&gt;Origin&lt;/span&gt; of &lt;span&gt;Species&lt;/span&gt; by &lt;span&gt;Means&lt;/span&gt; of &lt;span&gt;Natural Selection&lt;/span&gt;, &lt;span&gt;Or&lt;/span&gt;, &lt;span&gt;The Preservation&lt;/span&gt; of &lt;span&gt;Favoured Races&lt;/span&gt; in the &lt;span&gt;Struggle&lt;/span&gt; for &lt;span&gt;Life&lt;/span&gt;&lt;/em&gt; (&lt;span&gt;J. Murray&lt;/span&gt;, 1859), &lt;a href="https://books.google.com?id=gGUYt2PsDJcC" role="doc-biblioref"&gt;https://books.google.com?id=gGUYt2PsDJcC&lt;/a&gt;.&lt;/p&gt;'><sup>33</sup></a></span> [WHAT DID HE ACTUALLY SAY? EVOLTUION THROUGH NATURAL SELECTION NOW ESTABLISHED AND HAS STOOD THE TEST OF TIME] The book was unexpectedly met with enthusiastic reviews, and sold well.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee, &lt;em&gt;The &lt;span&gt;Gene&lt;/span&gt;&lt;/em&gt;.&lt;/p&gt;"><sup>34</sup></a></span> However two crucial questions yet remained: how was the variation within species generated in the first place, and how were the traits transmitted to future generations?<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee.&lt;/p&gt;"><sup>35</sup></a></span></p>
</div>
<div id="gregor-mendel" class="section level3" number="1.1.4">
<h3>
<span class="header-section-number">1.1.4</span> Gregor Mendel<a class="anchor" aria-label="anchor" href="#gregor-mendel"><i class="fas fa-link"></i></a>
</h3>
<p>Around the time the <em>Origin of Species</em> was published, the prevailing theory of heredity was promoted by the French biologist Jean-Baptiste de Lamarck (<strong>Figure <a href="Introduction.html#fig:lamarck">1.6</a></strong>).<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee.&lt;/p&gt;"><sup>36</sup></a></span> Lamarck’s theory involved the transmission of traits that were strengthened or weakened by environmental pressures in one generation – such as a giraffe having to stretch its neck to reach the leaves on a canopy – which was then passed down to subsequent generations as a form of “pre-adaptation.”<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee.&lt;/p&gt;"><sup>37</sup></a></span></p>

<div class="figure" style="text-align: left">
<span style="display:block;" id="fig:lamarck"></span>
<img src="figs/introduction/Jean-Baptiste_de_Lamarck.png" alt="Portrait of Jean Baptiste de Lamarck from 1802-1803 by Charles Thévenin.38" width="100%"><p class="caption">
Figure 1.6: Portrait of Jean Baptiste de Lamarck from 1802-1803 by Charles Thévenin.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Wikimedia Commons, &lt;span&gt;“File:jean-Baptiste de Lamarck.jpg — Wikimedia Commons&lt;span&gt;,&lt;/span&gt; the Free Media Repository,”&lt;/span&gt; 2022, &lt;a href="%5Curl%7Bhttps://commons.wikimedia.org/w/index.php?title=File:Jean-Baptiste_de_Lamarck.jpg&amp;amp;oldid=641428959%7D" role="doc-biblioref"&gt;\url{https://commons.wikimedia.org/w/index.php?title=File:Jean-Baptiste_de_Lamarck.jpg&amp;amp;oldid=641428959}&lt;/a&gt;.&lt;/p&gt;'><sup>38</sup></a></span>
</p>
</div>
<p>But by extending that logic, if a human had their arm amputated, their children would be born with shortened arms.<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;This theory was experimentally tested by the German embryologist August Weismann, who amputated the tails of mice to determine whether the offspring would be born tailless. The offspring were all born with tails intact &lt;span class="citation"&gt; (Mukherjee, &lt;em&gt;The &lt;span&gt;Gene&lt;/span&gt;&lt;/em&gt;)&lt;/span&gt;.&lt;/p&gt;'><sup>39</sup></a> Darwin was convinced that selection rather acted upon <em>pre-existing</em> variation in a population, and proposed that that variation was transmitted by hereditary particles he called <em>gemmules</em>.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee.&lt;/p&gt;"><sup>40</sup></a></span> During conception, he posited that the gemmules of each parent were blended like paints. But such blending would result in a monochrome population, preventing the creation of the unique, outlying traits required to drive the variation he observed. Around this time Darwin had recorded notes on an obscure paper titled <em>Experiments in Plant Hybridization</em>, but he appeared to have inadvertently skipped the page containing a description of the experiments on pea hybrids performed by an Augustine monk called Gregor Mendel (<strong>Figure <a href="Introduction.html#fig:mendel">1.7</a></strong>).<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee.&lt;/p&gt;"><sup>41</sup></a></span></p>

<div class="figure" style="text-align: right">
<span style="display:block;" id="fig:mendel"></span>
<img src="figs/introduction/Gregor_Mendel_2.png" alt="Photograph from Wikimedia Commons42." width="100%"><p class="caption">
Figure 1.7: Photograph from <span class="citation">Wikimedia Commons<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;&lt;span&gt;“File:gregor Mendel 2.jpg — Wikimedia Commons&lt;span&gt;,&lt;/span&gt; the Free Media Repository,”&lt;/span&gt; 2021, &lt;a href="%5Curl%7Bhttps://commons.wikimedia.org/w/index.php?title=File:Gregor_Mendel_2.jpg&amp;amp;oldid=534728192%7D" role="doc-biblioref"&gt;\url{https://commons.wikimedia.org/w/index.php?title=File:Gregor_Mendel_2.jpg&amp;amp;oldid=534728192}&lt;/a&gt;.&lt;/p&gt;'><sup>42</sup></a></span>.
</p>
</div>
<p>Mendel had aspired to become a teacher, but had failed the teacher’s exam multiple times, so by 1853 he had returned to the monastery in Brno, Morvia (present day Czech Republic), and planted a crop of peas which he had been breeding for about three years.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee, &lt;em&gt;The &lt;span&gt;Gene&lt;/span&gt;&lt;/em&gt;.&lt;/p&gt;"><sup>43</sup></a></span> He had collected 34 strains that bred “true”, meaning that the offspring were identical to the parents in seven famous traits:<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee.&lt;/p&gt;"><sup>44</sup></a></span></p>
<ol style="list-style-type: decimal">
<li><p>the texture of the seed (smooth versus wrinkled)</p></li>
<li><p>the colour of seeds (yellow versus green)</p></li>
<li><p>the colour of the flower (white versus violet)</p></li>
<li><p>the position of the flower (at the tip of the plant versus the branches)</p></li>
<li><p>the colour of the pea pod (green versus yellow)</p></li>
<li><p>the shape of the pea pod (smooth versus crumpled)</p></li>
<li><p>the height of the plant (tall versus short)</p></li>
</ol>
<p>Mendel referred to these alternative versions of a trait as <em>forms</em> – biologists in the 1900s would later refer to them as <em>alleles</em>, from the Greek <em>allos</em>, referring to different subtypes of the same thing.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee.&lt;/p&gt;"><sup>45</sup></a></span> Mendel produced hybrids of plants with different alleles to determine which allele the consequent offspring would possess. Over the course of eight years (1857-1864), he created hybrid crosses between the F0 strains (known as the filial 1 hybrid (<strong>F1</strong>) generation), and crossed those with each other too (known as the <strong>F2</strong> generation), recording their values for the above traits as he went.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee.&lt;/p&gt;"><sup>46</sup></a></span> He could soon identify patterns in the reams of data he generated:<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Based around 28,000 plants, 40,000 flowers, and almost 400,000 seeds.&lt;/p&gt;"><sup>47</sup></a> for each of the above 7 traits, the filial 1 hybrid (<strong>F1</strong>) generation expressed only a single allele, which he referred to as the <em>dominant</em>; the hidden allele he referred to as <em>recessive</em>. Curiously, however, in the F2 generation the recessive allele reappeared at a ratio of 1:3.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee, &lt;em&gt;The &lt;span&gt;Gene&lt;/span&gt;&lt;/em&gt;.&lt;/p&gt;"><sup>48</sup></a></span></p>
<p>From these results, Mendel inferred that the F1 hybrids must retain the information from both parents, while only expressing one version of them.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee.&lt;/p&gt;"><sup>49</sup></a></span> Then when the F1 generation was inter-crossed to produce the F2 generation, the recessive allele would only be expressed when inherited with another recessive allele.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee.&lt;/p&gt;"><sup>50</sup></a></span> In 1865, Mendel presented his paper <em>Versuche über Pflanzenhybriden</em> (Experiments on Plant Hybridization) to a group of farmers, botanists and biologists at the Natural Science Society in Brno,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee.&lt;/p&gt;"><sup>51</sup></a></span> and it was later published in the annual journal <em>Proceedings of the Brno Natural Science Society</em>.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Gregor Mendel, &lt;span&gt;“Versuche Uber Pflanzen-Hybriden,”&lt;/span&gt; &lt;em&gt;Verhandlungen Des Naturforschenden Vereins in Brunn Fur&lt;/em&gt; 4 (1866): 3–47.&lt;/p&gt;"><sup>52</sup></a></span> He mailed out 40 copies of the paper to various scientific societies, but it was only cited four times between 1866 and 1900, and virtually disappeared from scientific literature.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee, &lt;em&gt;The &lt;span&gt;Gene&lt;/span&gt;&lt;/em&gt;.&lt;/p&gt;"><sup>53</sup></a></span></p>
<p>In 1900, a Dutch botanist named Hugo de Vries chanced upon Mendel’s paper while working on plant hybrids.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee.&lt;/p&gt;"><sup>54</sup></a></span> Likely driven by self-interest in being credited for the discovery, he proceeded to publish his own findings without mentioning the work of Mendel.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee.&lt;/p&gt;"><sup>55</sup></a></span> He noted a patch of primroses named <em>Oenothera lamarckiana</em><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;An ironic twist given the plant was named after Lamarck.&lt;/p&gt;"><sup>56</sup></a>, which spontaneously generated variants. He termed these variants <em>mutants</em>.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee, &lt;em&gt;The &lt;span&gt;Gene&lt;/span&gt;&lt;/em&gt;.&lt;/p&gt;"><sup>57</sup></a></span> It became clear to de Vries that variation in populations arose spontaneously, at random, rather than in the Lamarckian mode of contemporaneous reaction to environmental pressures.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee.&lt;/p&gt;"><sup>58</sup></a></span> De Vries’s paper was read by Carl Correns, a botanist in Tübingen, Germany, who had also come across the work of Mendel, and pressured de Vries into acknowledging Mendel’s work in the subsequent version of his publication.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee.&lt;/p&gt;"><sup>59</sup></a></span></p>
<p>In 1894, the English botanist William Bateson (<strong>Figure <a href="Introduction.html#fig:bateson">1.8</a></strong>) had published a book titled <em>Materials for the study of variation</em>,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;William Bateson, &lt;em&gt;Materials for the &lt;span&gt;Study&lt;/span&gt; of &lt;span&gt;Variation&lt;/span&gt;: &lt;span&gt;Treated&lt;/span&gt; with &lt;span&gt;Especial Regard&lt;/span&gt; to &lt;span&gt;Discontinuity&lt;/span&gt; in the &lt;span&gt;Origin&lt;/span&gt; of &lt;span&gt;Species&lt;/span&gt;&lt;/em&gt; (&lt;span&gt;Macmillan and Company&lt;/span&gt;, 1894), &lt;a href="https://books.google.com?id=_HIZAAAAYAAJ" role="doc-biblioref"&gt;https://books.google.com?id=_HIZAAAAYAAJ&lt;/a&gt;.&lt;/p&gt;'><sup>60</sup></a></span> in which he noted that biological variation occurs continuously for some traits, and dimorphically for others.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee, &lt;em&gt;The &lt;span&gt;Gene&lt;/span&gt;&lt;/em&gt;.&lt;/p&gt;"><sup>61</sup></a></span> In 1900, while on a train from Cambridge to London, Bateson read the second version of de Vries’s paper, and was immediately struck by the significance of Mendel’s original study.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee.&lt;/p&gt;"><sup>62</sup></a></span> He independently confirmed Mendel’s findings, and began to promote Mendel’s work, including by publishing an English translation of Mendel’s original paper.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;C. T. Druery and William Bateson, &lt;span&gt;“Experiments in Plant Hybridization,”&lt;/span&gt; &lt;em&gt;Journal of the Royal Horticultural Society&lt;/em&gt; 26 (1901): 1–32.&lt;/p&gt;"><sup>63</sup></a></span> In 1905, Bateson coined a word for the study of these units of inheritance: <em>genetics</em>, from the Greek <em>genno</em>, meaning “to give birth.”<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee, &lt;em&gt;The &lt;span&gt;Gene&lt;/span&gt;&lt;/em&gt;.&lt;/p&gt;"><sup>64</sup></a></span> A few years later, in 1909, the botanist Wilhelm Johannsen coined a distinct word to denote the unit of inheritance: the <em>gene</em>.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee.&lt;/p&gt;"><sup>65</sup></a></span></p>

<div class="figure" style="text-align: right">
<span style="display:block;" id="fig:bateson"></span>
<img src="figs/introduction/Bateson2.png" alt="Portrait of William Bateson, date and author unknown.66" width="100%"><p class="caption">
Figure 1.8: Portrait of William Bateson, date and author unknown.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Wikimedia Commons, &lt;span&gt;“File:Bateson2.jpg — Wikimedia Commons&lt;span&gt;,&lt;/span&gt; the Free Media Repository,”&lt;/span&gt; 2017, &lt;a href="%5Curl%7Bhttps://commons.wikimedia.org/w/index.php?title=File:Bateson2.jpg&amp;amp;oldid=255958176%7D" role="doc-biblioref"&gt;\url{https://commons.wikimedia.org/w/index.php?title=File:Bateson2.jpg&amp;amp;oldid=255958176}&lt;/a&gt;.&lt;/p&gt;'><sup>66</sup></a></span>
</p>
</div>
</div>
<div id="quantitative-genetics-francis-galton-karl-pearson-ronald-fisher-and-sewall-wright" class="section level3" number="1.1.5">
<h3>
<span class="header-section-number">1.1.5</span> Quantitative genetics: Francis Galton, Karl Pearson, Ronald Fisher and Sewall Wright<a class="anchor" aria-label="anchor" href="#quantitative-genetics-francis-galton-karl-pearson-ronald-fisher-and-sewall-wright"><i class="fas fa-link"></i></a>
</h3>
<p>The rediscovery of Mendel’s work triggered a vociferous debate between Bateson and a cohort of other scientists who sought to reconcile Mendelian inheritance with the continuous variation observed in natural populations.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Nicholas H Barton, &lt;span&gt;“Sewall &lt;span&gt;Wright&lt;/span&gt; on &lt;span&gt;Evolution&lt;/span&gt; in &lt;span&gt;Mendelian Populations&lt;/span&gt; and the &lt;span&gt;‘&lt;span&gt;Shifting Balance&lt;/span&gt;’,&lt;/span&gt;”&lt;/span&gt; &lt;em&gt;Genetics&lt;/em&gt; 202, no. 1 (January 1, 2016): 3–4, &lt;a href="https://doi.org/10.1534/genetics.115.184796" role="doc-biblioref"&gt;https://doi.org/10.1534/genetics.115.184796&lt;/a&gt;.&lt;/p&gt;'><sup>67</sup></a></span> Francis Galton (<strong>Figure <a href="Introduction.html#fig:galton">1.9</a></strong>) was the cousin of Charles Darwin, born in the same year as Mendel (1822), and travelled to Egypt and Sudan in 1844, spurring his lifelong obsession with the differences between human races.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee, &lt;em&gt;The &lt;span&gt;Gene&lt;/span&gt;&lt;/em&gt;.&lt;/p&gt;"><sup>68</sup></a></span> His reading of Darwin’s <em>Origin of Species</em> in 1859 galvanised him to explore the measurement and variance of heredity in humans, with an emphasis on height, intelligence, temperament, and physical prowess.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee.&lt;/p&gt;"><sup>69</sup></a></span><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;One of Galton’s more eccentric activities involved strolling through England and Scotland secretly tabulating beauty by ranking the women he met as “attractive”, “indifferent,” or “repellent” using pinpricks on a card hidden in his pocket &lt;span class="citation"&gt; (Mukherjee)&lt;/span&gt;.&lt;/p&gt;'><sup>70</sup></a> He coined the phrase <em>nature versus nurture</em> to distinguish between hereditary and environmental influences.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Francis Galton, &lt;span&gt;“On &lt;span&gt;Men&lt;/span&gt; of &lt;span&gt;Science&lt;/span&gt;, Their &lt;span&gt;Nature&lt;/span&gt; and Their &lt;span&gt;Nurture&lt;/span&gt;,”&lt;/span&gt; in &lt;em&gt;Proceedings of the &lt;span&gt;Royal Institution&lt;/span&gt; of &lt;span&gt;Great Britain&lt;/span&gt;&lt;/em&gt;, vol. 7, 1874, 227–36.&lt;/p&gt;"><sup>71</sup></a></span> [GALTON WAS INTERESTED IN THIS TO IMPROVE THE HUMNAN STOCK. REMOVING THE POOR PEPOLE, RACISM IS HIDDEN INSIDE THERE. ]</p>

<div class="figure" style="text-align: left">
<span style="display:block;" id="fig:galton"></span>
<img src="figs/introduction/Francis_Galton_1850s.png" alt="Portrait of Francis Galton, taken in the 1850s or early 1860s, originally scanned from Karl Pearson72." width="100%"><p class="caption">
Figure 1.9: Portrait of Francis Galton, taken in the 1850s or early 1860s, originally scanned from <span class="citation">Karl Pearson<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;&lt;em&gt;The &lt;span&gt;Life&lt;/span&gt;, &lt;span&gt;Letters&lt;/span&gt; and &lt;span&gt;Labours&lt;/span&gt; of &lt;span&gt;Francis Galton&lt;/span&gt;&lt;/em&gt; (&lt;span&gt;Cambridge University Press&lt;/span&gt;, 2011), &lt;a href="https://books.google.com?id=sz7vSOwOGzgC" role="doc-biblioref"&gt;https://books.google.com?id=sz7vSOwOGzgC&lt;/a&gt;; Wikimedia Commons, &lt;span&gt;“File:francis Galton 1850s.jpg — Wikimedia Commons&lt;span&gt;,&lt;/span&gt; the Free Media Repository,”&lt;/span&gt; 2021, &lt;a href="%5Curl%7Bhttps://commons.wikimedia.org/w/index.php?title=File:Francis_Galton_1850s.jpg&amp;amp;oldid=569302173%7D" role="doc-biblioref"&gt;\url{https://commons.wikimedia.org/w/index.php?title=File:Francis_Galton_1850s.jpg&amp;amp;oldid=569302173}&lt;/a&gt;.&lt;/p&gt;'><sup>72</sup></a></span>.
</p>
</div>
<p>Through surveys he sent out to men and women in the mid-1880s, he requested they mail him detailed measurements on the height, weight, eye colour, intelligence, and artistic abilities of parents, grandparents, and children in return for a substantial fee.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee, &lt;em&gt;The &lt;span&gt;Gene&lt;/span&gt;&lt;/em&gt;.&lt;/p&gt;"><sup>73</sup></a></span> With this data he discovered that tall parents indeed tend to have tall children, albeit on average, although the distribution of heights within a generation fit the shape of a normal distribution, or ‘bell-curve.’<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee.&lt;/p&gt;"><sup>74</sup></a></span> In a surprising twist, he also discovered that the mean height of the sons of the tallest fathers tended to be slightly lower than the father’s height, and closer to the population’s average – a phenomenon he described as <em>regression to the mean</em>.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee.&lt;/p&gt;"><sup>75</sup></a></span> The <em>ancestral law of heredity</em> sparked an intellectual war with Bateson, through which they attempted to reconcile the dominant/recessive theory with the quantitative theory proposed by Galton.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee.&lt;/p&gt;"><sup>76</sup></a></span></p>
<p>Spurred by a desire to apply this new science to the advancement of humankind, and by a fear the the “unfit” were outbreeding the “fit,”<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Tabery, &lt;em&gt;Beyond Versus&lt;/em&gt;.&lt;/p&gt;"><sup>77</sup></a></span> in 1883 Galton coined the term <em>eugenics</em> for his project to improve the human population through selective breeding.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;James D. Watson and Andrew Berry, &lt;em&gt;&lt;span&gt;DNA&lt;/span&gt;: &lt;span&gt;The&lt;/span&gt; Secret of Life&lt;/em&gt; (&lt;span&gt;Knopf&lt;/span&gt;, 2009).&lt;/p&gt;"><sup>78</sup></a></span> It was his vision for a “science which deals with all influences that improve the inborn qualities of race.”<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Francis Galton, &lt;span&gt;“Eugenics: &lt;span&gt;Its Definition&lt;/span&gt;, &lt;span&gt;Scope&lt;/span&gt;, and &lt;span&gt;Aims&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;American Journal of Sociology&lt;/em&gt; 10, no. 1 (July 1904): 1–25, &lt;a href="https://doi.org/10.1086/211280" role="doc-biblioref"&gt;https://doi.org/10.1086/211280&lt;/a&gt;; Tabery, &lt;em&gt;Beyond Versus&lt;/em&gt;.&lt;/p&gt;'><sup>79</sup></a></span><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Driven by fear of a world overrun by imbeciles, Galton wrote: “What nature does blindly, slowly and ruthlessly, man may do providently, quickly, and kindly. As it lies within his power, so it becomes his duty to work in that direction.”&lt;/p&gt;"><sup>80</sup></a></p>
<p>In the 1890s, in an attempt to tease out the respective contributions of nature and nurture, Galton proposed the first study on human twins. Since twins share identical genetic material, he reasoned, they represent a natural experiment where any substantial similarities between them could be attributed to genes, while any differences were the consequence of environment.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee, &lt;em&gt;The &lt;span&gt;Gene&lt;/span&gt;&lt;/em&gt;.&lt;/p&gt;"><sup>81</sup></a></span> However, these studies were hampered by Galton’s failure to distinguish between identical and non-identical twins.</p>
<p>[PEARSON WANTED TO IMRPOVE HUMAN INTELLIGENCE. CREATES PCA TO EXPLORE THAT.]</p>
<p>In Galton’s later years, he adopted the English mathematician Karl Pearson (<strong>Figure <a href="Introduction.html#fig:pearson">1.10</a></strong>) as his protégé. From 1893 to 1904, Pearson built upon the work of his mentor, continuing to develop a number of statistical techniques for biometry including the chi-squared test,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Karl Pearson, &lt;span&gt;“X. &lt;span&gt;On&lt;/span&gt; the Criterion That a Given System of Deviations from the Probable in the Case of a Correlated System of Variables Is Such That It Can Be Reasonably Supposed to Have Arisen from Random Sampling,”&lt;/span&gt; &lt;em&gt;The London, Edinburgh, and Dublin Philosophical Magazine and Journal of Science&lt;/em&gt; 50, no. 302 (July 1, 1900): 157–75, &lt;a href="https://doi.org/10.1080/14786440009463897" role="doc-biblioref"&gt;https://doi.org/10.1080/14786440009463897&lt;/a&gt;.&lt;/p&gt;'><sup>82</sup></a></span> the term “standard deviation,”<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Karl Pearson, &lt;span&gt;“On the Dissection of Asymmetrical Frequency Curves,”&lt;/span&gt; &lt;em&gt;Phil. Trans. Roy. Soc&lt;/em&gt; 185 (1894): 71–110.&lt;/p&gt;"><sup>83</sup></a></span> and correlation and regression coefficients,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Karl Pearson, &lt;span&gt;“Mathematical Contributions to the Theory of Evolution, &lt;span&gt;On&lt;/span&gt; the Law of Ancestral Heredity,”&lt;/span&gt; &lt;em&gt;Proceedings of the Royal Society of London&lt;/em&gt; 62, no. 379-387 (January 1898): 386–412, &lt;a href="https://doi.org/10.1098/rspl.1897.0128" role="doc-biblioref"&gt;https://doi.org/10.1098/rspl.1897.0128&lt;/a&gt;.&lt;/p&gt;'><sup>84</sup></a></span> and establishing the foundations of principal components analysis.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Karl Pearson, &lt;span&gt;“&lt;span&gt;LIII&lt;/span&gt;. &lt;span&gt;On&lt;/span&gt; Lines and Planes of Closest Fit to Systems of Points in Space,”&lt;/span&gt; &lt;em&gt;The London, Edinburgh, and Dublin Philosophical Magazine and Journal of Science&lt;/em&gt; 2, no. 11 (November 1, 1901): 559–72, &lt;a href="https://doi.org/10.1080/14786440109462720" role="doc-biblioref"&gt;https://doi.org/10.1080/14786440109462720&lt;/a&gt;.&lt;/p&gt;'><sup>85</sup></a></span><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Pearson also inherited his Galton’s enthusiasm for eugenics. When Galton died in 1911, he bequeathed money to University College London for a Galton Eugenics Professorship, a position that was given to Pearson, who was also the head of the newly created Department of Applied Statistics, the world’s first university statistics department &lt;span class="citation"&gt; (Daniel J. Kevles, &lt;em&gt;In the &lt;span&gt;Name&lt;/span&gt; of &lt;span&gt;Eugenics&lt;/span&gt;: &lt;span&gt;Genetics&lt;/span&gt; and the &lt;span&gt;Uses&lt;/span&gt; of &lt;span&gt;Human Heredity&lt;/span&gt;&lt;/em&gt; (&lt;span&gt;Harvard University Press&lt;/span&gt;, 1995), &lt;a href="https://books.google.com?id=NWBi9kyb8xUC" role="doc-biblioref"&gt;https://books.google.com?id=NWBi9kyb8xUC&lt;/a&gt;; Kathryn Paige Harden, &lt;em&gt;The &lt;span&gt;Genetic Lottery&lt;/span&gt;: &lt;span&gt;Why DNA Matters&lt;/span&gt; for &lt;span&gt;Social Equality&lt;/span&gt;&lt;/em&gt; (&lt;span&gt;Princeton University Press&lt;/span&gt;, 2021), &lt;a href="https://books.google.com?id=R9oiEAAAQBAJ" role="doc-biblioref"&gt;https://books.google.com?id=R9oiEAAAQBAJ&lt;/a&gt;)&lt;/span&gt;.&lt;/p&gt;'><sup>86</sup></a></p>
<p>The corrected standard deviation <span class="math inline">\(s\)</span> for a sample is defined as:</p>
<p><span class="math display" id="eq:sd">\[\begin{equation}
s = \sqrt{\frac{1}{N-1}{\sum_{i=1} ^n (x_i - \bar{x})^2}} \tag{1.1}
\end{equation}\]</span></p>
<p>The Pearson correlation coefficient for a sample is defined as:</p>
<p><span class="math display" id="eq:cor">\[\begin{equation}
r_{X,Y} = \frac{\mathrm{cov}(X,Y)}{s_X s_Y} \tag{1.2}
\end{equation}\]</span>
<span class="math display">\[\begin{equation}
\mathrm{cov} = \sum_{i=1} ^n (x_i - \bar{x})(y_i - \bar{y})
\end{equation}\]</span></p>

<div class="figure" style="text-align: right">
<span style="display:block;" id="fig:pearson"></span>
<img src="figs/introduction/Karl_Pearson_1910.png" alt="Portrait of Karl Pearson in 1910.87" width="100%"><p class="caption">
Figure 1.10: Portrait of Karl Pearson in 1910.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Wikimedia Commons, &lt;span&gt;“File:karl Pearson, 1910.jpg — Wikimedia Commons&lt;span&gt;,&lt;/span&gt; the Free Media Repository,”&lt;/span&gt; 2020, &lt;a href="%5Curl%7Bhttps://commons.wikimedia.org/w/index.php?title=File:Karl_Pearson,_1910.jpg&amp;amp;oldid=477532248%7D" role="doc-biblioref"&gt;\url{https://commons.wikimedia.org/w/index.php?title=File:Karl_Pearson,_1910.jpg&amp;amp;oldid=477532248}&lt;/a&gt;.&lt;/p&gt;'><sup>87</sup></a></span>
</p>
</div>
<p>Following in the footsteps of Galton and Pearson, the mathematician Ronald Fisher (<strong>Figure <a href="Introduction.html#fig:fisher">1.11</a></strong>) of Caius College at the University of Cambridge began to apply his skills to elucidating how continuous traits, like height, could be driven by genetic variation.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee, &lt;em&gt;The &lt;span&gt;Gene&lt;/span&gt;&lt;/em&gt;.&lt;/p&gt;"><sup>88</sup></a></span> In 1918, Fisher published his analysis in a landmark paper entitled <em>The Correlation between Relatives on the Supposition of Mendelian Inheritance</em>,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;R. A. Fisher, &lt;span&gt;“&lt;span&gt;XV&lt;/span&gt;.—&lt;span&gt;The Correlation&lt;/span&gt; Between &lt;span&gt;Relatives&lt;/span&gt; on the &lt;span&gt;Supposition&lt;/span&gt; of &lt;span&gt;Mendelian Inheritance&lt;/span&gt;.”&lt;/span&gt; January 1, 1919, &lt;a href="https://doi.org/10.1017/s0080456800012163" role="doc-biblioref"&gt;https://doi.org/10.1017/s0080456800012163&lt;/a&gt;.&lt;/p&gt;'><sup>89</sup></a></span> where he described how the combination of a large number of Mendelian alleles acting on the same trait would result in a normal distribution. This reconciliation between Mendelian inheritance and observed continuous traits was the beginning of what was later referred to by Julian Huxley as the “<strong>modern evolutionary synthesis</strong>.”<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Julian Huxley, &lt;span&gt;“Evolution. &lt;span&gt;The&lt;/span&gt; Modern Synthesis.”&lt;/span&gt; &lt;em&gt;Evolution. The Modern Synthesis.&lt;/em&gt;, 1942; Tabery, &lt;em&gt;Beyond Versus&lt;/em&gt;.&lt;/p&gt;"><sup>90</sup></a></span></p>

<div class="figure" style="text-align: left">
<span style="display:block;" id="fig:fisher"></span>
<img src="figs/introduction/Youngronaldfisher2.png" alt="Photo of a young Ronald Fisher taken in 1913. Image from Wikimedia Commons91." width="100%"><p class="caption">
Figure 1.11: Photo of a young Ronald Fisher taken in 1913. Image from <span class="citation">Wikimedia Commons<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;&lt;span&gt;“File:Youngronaldfisher2.JPG — Wikimedia Commons&lt;span&gt;,&lt;/span&gt; the Free Media Repository,”&lt;/span&gt; 2021, &lt;a href="%5Curl%7Bhttps://commons.wikimedia.org/w/index.php?title=File:Youngronaldfisher2.JPG&amp;amp;oldid=614548490%7D" role="doc-biblioref"&gt;\url{https://commons.wikimedia.org/w/index.php?title=File:Youngronaldfisher2.JPG&amp;amp;oldid=614548490}&lt;/a&gt;.&lt;/p&gt;'><sup>91</sup></a></span>.
</p>
</div>
<p>In the same paper, Fisher also introduced the concept of <em>variance</em> as the square of the standard deviation developed by Galton and Pearson (Equation <a href="Introduction.html#eq:sd">(1.1)</a> above), and used it to partition the <em>causes</em> of variation.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Tabery, &lt;em&gt;Beyond Versus&lt;/em&gt;.&lt;/p&gt;"><sup>92</sup></a></span><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Fisher: “For stature the coefficient of correlation between brothers is about .54, which we may interpret by saying that 54% of their variance is accounted for by ancestry alone, and that 46% must have some other explanation.”&lt;/p&gt;"><sup>93</sup></a> In 1921 he published his first application of this “analysis of variance” (<strong>ANOVA</strong>),<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Ronald Aylmer Fisher, &lt;span&gt;“On the&lt;span&gt;"&lt;/span&gt; &lt;span&gt;Probable Error&lt;/span&gt;&lt;span&gt;"&lt;/span&gt; of a &lt;span&gt;Coefficient&lt;/span&gt; of &lt;span&gt;Correlation Deduced&lt;/span&gt; from a &lt;span&gt;Small Sample&lt;/span&gt;.”&lt;/span&gt; 1921.&lt;/p&gt;'><sup>94</sup></a></span> which became widely used following its inclusion in his widely influential book <em>Statistical Methods for Research Workers</em>.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Ronald Aylmer Fisher, &lt;em&gt;Statistical &lt;span&gt;Methods&lt;/span&gt; for &lt;span&gt;Research Workers&lt;/span&gt;&lt;/em&gt; (&lt;span&gt;Oliver and Boyd&lt;/span&gt;, 1925), &lt;a href="https://books.google.com?id=1NApzwEACAAJ" role="doc-biblioref"&gt;https://books.google.com?id=1NApzwEACAAJ&lt;/a&gt;.&lt;/p&gt;'><sup>95</sup></a></span><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Fisher was also an ardent eugenicist. He helped create the Cambridge University Eugenics Society in 1911, hosting meetings in his rooms, organising public lectures by well-known eugenicists, assisting at the First International Eugenics Congress, and even delivering his own eugenic lectures &lt;span class="citation"&gt; (Pauline Mazumdar, &lt;em&gt;Eugenics, Human Genetics and Human Failings: The &lt;span&gt;Eugenics Society&lt;/span&gt;, Its Sources and Its Critics in &lt;span&gt;Britain&lt;/span&gt;&lt;/em&gt; (&lt;span&gt;Routledge&lt;/span&gt;, 2005); Tabery, &lt;em&gt;Beyond Versus&lt;/em&gt;)&lt;/span&gt;. He was also a member of the British Eugenics Society, and helped establish the Society’s Committee for Legalising Sterilisation. His statistical innovations were developed in part to make eugenic assessments of the relative importance of nature and nurture when it came to evaluating traits like feeble-mindedness &lt;span class="citation"&gt; (Tabery, &lt;em&gt;Beyond Versus&lt;/em&gt;)&lt;/span&gt;. Eugenics policies were implemented in the United States, Sweden, and other countries, which resulted in the forced sterilisations and deaths of millions &lt;span class="citation"&gt; (Rutherford, &lt;em&gt;How to Argue with a Racist&lt;/em&gt;)&lt;/span&gt;. Ultimately, eugenics was adopted as a policy by the Nazi regime, resulting in the genocide of around a million Jews, Poles, and Gypsies. The association between eugenics and the horrors of World War II became inextricable, and the eugenics movement was all but finished &lt;span class="citation"&gt; (Mukherjee, &lt;em&gt;The &lt;span&gt;Gene&lt;/span&gt;&lt;/em&gt;)&lt;/span&gt;.&lt;/p&gt;'><sup>96</sup></a> Fisher later moved to the Rothamsted Agricultural Research Station in Harpenden, where he created many of the statistical methodologies – such as tests of significance and the design of experiments – that continue to be used by statisticians today.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Tabery, &lt;em&gt;Beyond Versus&lt;/em&gt;.&lt;/p&gt;"><sup>97</sup></a></span><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;With Winifred A. Mackenzie, Fisher studied the response of 12 different potato varieties to different manure-based fertilizer treatments, where they set up a plot of two treatments (treatment/no treatment), with three replicate plots within each treatment, and three rows within each replicate plots with different manure treatments (either manure, manure + potassium sulfate, or manure + potassium chloride) &lt;span class="citation"&gt; (R. A. Fisher and W. A. Mackenzie, &lt;span&gt;“Studies in Crop Variation. &lt;span&gt;II&lt;/span&gt;. &lt;span&gt;The&lt;/span&gt; Manurial Response of Different Potato Varieties,”&lt;/span&gt; &lt;em&gt;The Journal of Agricultural Science&lt;/em&gt; 13, no. 3 (July 1923): 311–20, &lt;a href="https://doi.org/10.1017/S0021859600003592" role="doc-biblioref"&gt;https://doi.org/10.1017/S0021859600003592&lt;/a&gt;)&lt;/span&gt;. From this data Fisher generated the first ANOVA table, listing the various causes of variation along with their respective contribution to total variation in crop yield &lt;span class="citation"&gt; (Tabery, &lt;em&gt;Beyond Versus&lt;/em&gt;)&lt;/span&gt;.&lt;/p&gt;'><sup>98</sup></a></p>
<p>Around the time Fisher was working on his landmark 1919 paper, the American geneticist Sewall Wright (<strong>Figure <a href="Introduction.html#fig:wright">1.12</a></strong>) was using his work on guinea pigs to reconcile Mendelian inheritance with continuous variation.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Sewall Wright, &lt;span&gt;“The &lt;span&gt;Genetical Theory&lt;/span&gt; of &lt;span&gt;Natural Selection&lt;/span&gt;: &lt;span&gt;A Review&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Journal of Heredity&lt;/em&gt; 21, no. 8 (August 1, 1930): 349–56, &lt;a href="https://doi.org/10.1093/oxfordjournals.jhered.a103361" role="doc-biblioref"&gt;https://doi.org/10.1093/oxfordjournals.jhered.a103361&lt;/a&gt;.&lt;/p&gt;'><sup>99</sup></a></span> He developed a mathematical theory of inbreeding, eventually introducing the inbreeding coefficient <span class="math inline">\(F\)</span> as the correlation between uniting gametes in 1922.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Sewall Wright, &lt;span&gt;“Coefficients of &lt;span&gt;Inbreeding&lt;/span&gt; and &lt;span&gt;Relationship&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;The American Naturalist&lt;/em&gt; 56, no. 645 (July 1922): 330–38, &lt;a href="https://doi.org/10.1086/279872" role="doc-biblioref"&gt;https://doi.org/10.1086/279872&lt;/a&gt;.&lt;/p&gt;'><sup>100</sup></a></span> Over the course of an esteemed career, together with Fisher and J. B. S. Haldane,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;William B. Provine, &lt;em&gt;The Origins of Theoretical Population Genetics: With a New Afterword&lt;/em&gt; (&lt;span&gt;University of Chicago Press&lt;/span&gt;, 2001); Tabery, &lt;em&gt;Beyond Versus&lt;/em&gt;.&lt;/p&gt;"><sup>101</sup></a></span> he advanced the modern evolutionary synthesis, and founded the field of population genetics with his papers on mating systems<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Sewall Wright, &lt;span&gt;“Isolation by &lt;span&gt;Distance&lt;/span&gt; Under &lt;span&gt;Diverse Systems&lt;/span&gt; of &lt;span&gt;Mating&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Genetics&lt;/em&gt; 31, no. 1 (January 25, 1946): 39–59, &lt;a href="https://doi.org/10.1093/genetics/31.1.39" role="doc-biblioref"&gt;https://doi.org/10.1093/genetics/31.1.39&lt;/a&gt;.&lt;/p&gt;'><sup>102</sup></a></span> and genetic drift<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Sewall Wright, &lt;span&gt;“On the &lt;span&gt;Roles&lt;/span&gt; of &lt;span&gt;Directed&lt;/span&gt; and &lt;span&gt;Random Changes&lt;/span&gt; in &lt;span&gt;Gene Frequency&lt;/span&gt; in the &lt;span&gt;Genetics&lt;/span&gt; of &lt;span&gt;Populations&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Evolution&lt;/em&gt; 2, no. 4 (1948): 279–94, &lt;a href="https://doi.org/10.1111/j.1558-5646.1948.tb02746.x" role="doc-biblioref"&gt;https://doi.org/10.1111/j.1558-5646.1948.tb02746.x&lt;/a&gt;.&lt;/p&gt;'><sup>103</sup></a></span> .</p>

<div class="figure" style="text-align: right">
<span style="display:block;" id="fig:wright"></span>
<img src="figs/introduction/Sewall_Wright.png" alt="Photo of Sewall Wright in 1965. Image from Charles Goodnight104." width="100%"><p class="caption">
Figure 1.12: Photo of Sewall Wright in 1965. Image from <span class="citation">Charles Goodnight<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;&lt;span&gt;“Sewall &lt;span&gt;Wright&lt;/span&gt;’s &lt;span&gt;Seven Generalizations&lt;/span&gt; about &lt;span&gt;Populations&lt;/span&gt;”&lt;/span&gt; (&lt;span&gt;Evolution in Structured Populations&lt;/span&gt;, May 22, 2014), &lt;a href="https://blog.uvm.edu/cgoodnig/2014/05/22/sewall-wrights-seven-generalizations-about-populations/" role="doc-biblioref"&gt;https://blog.uvm.edu/cgoodnig/2014/05/22/sewall-wrights-seven-generalizations-about-populations/&lt;/a&gt;.&lt;/p&gt;'><sup>104</sup></a></span>.
</p>
</div>
</div>
<div id="genetic-mapping-thomas-morgan-tetsuo-aida-and-theodosius-dobzhansky" class="section level3" number="1.1.6">
<h3>
<span class="header-section-number">1.1.6</span> Genetic mapping: Thomas Morgan, Tetsuo Aida and Theodosius Dobzhansky<a class="anchor" aria-label="anchor" href="#genetic-mapping-thomas-morgan-tetsuo-aida-and-theodosius-dobzhansky"><i class="fas fa-link"></i></a>
</h3>
<p>During this fertile period in statistical genetics research, the physical location of the gene was still a mystery. In the 1890s, Theodor Boveri, a German embryologist working with sea urchins in Naples, identified the chromosome as the location of the gene.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee, &lt;em&gt;The &lt;span&gt;Gene&lt;/span&gt;&lt;/em&gt;.&lt;/p&gt;"><sup>105</sup></a></span> In 1907, on a trip to the United States to give talks on Mendel’s discovery, he met the American cell biologist Thomas Hunt Morgan (<strong>Figure <a href="Introduction.html#fig:morgan">1.13</a></strong>),<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee.&lt;/p&gt;"><sup>106</sup></a></span> who sought to extend Boveri’s work by exploring the architecture of genes using the fruit fly, <em>Drosophila melanogaster</em>.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee.&lt;/p&gt;"><sup>107</sup></a></span></p>

<div class="figure" style="text-align: left">
<span style="display:block;" id="fig:morgan"></span>
<img src="figs/introduction/Thomas_Morgan.png" alt="Thomas Hunt Morgan in 1920. Portrait taken by A. F. Huettner. Image from Diana E Kenney and Gary G Borisy108." width="100%"><p class="caption">
Figure 1.13: Thomas Hunt Morgan in 1920. Portrait taken by A. F. Huettner. Image from <span class="citation">Diana E Kenney and Gary G Borisy<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;&lt;span&gt;“Thomas &lt;span&gt;Hunt Morgan&lt;/span&gt; at the &lt;span&gt;Marine Biological Laboratory&lt;/span&gt;: &lt;span&gt;Naturalist&lt;/span&gt; and &lt;span&gt;Experimentalist&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Genetics&lt;/em&gt; 181, no. 3 (March 1, 2009): 841–46, &lt;a href="https://doi.org/10.1534/genetics.109.101659" role="doc-biblioref"&gt;https://doi.org/10.1534/genetics.109.101659&lt;/a&gt;.&lt;/p&gt;'><sup>108</sup></a></span>.
</p>
</div>
<p>Around 1905 Morgan began to breed <em>Drosophila</em>, identifying visible variants that he could track over generations, including white versus red eyes, forked versus straight bristles, sable-coloured bodies, curved legs; bent, bat-like wings; disjointed abdomens, and deformed eyes.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee, &lt;em&gt;The &lt;span&gt;Gene&lt;/span&gt;&lt;/em&gt;.&lt;/p&gt;"><sup>109</sup></a></span> Through repeated crossing experiments, Morgan discovered that some genes were transmitted together at a higher rate than chance alone. He proposed that these genes were physically “linked” to one another, implying that they were situated on some string within the chromosome.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee.&lt;/p&gt;"><sup>110</sup></a></span> This advanced the understanding of a gene from a purely theoretical unit of inheritance to a physical unit.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee.&lt;/p&gt;"><sup>111</sup></a></span> He further discovered that, occasionally, a gene that was otherwise linked to another could “cross over” from one parental strand to the other, generating offspring with a mixture of parental alleles.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee.&lt;/p&gt;"><sup>112</sup></a></span></p>
<p>While Morgan was carrying out his experiments with <em>Drosophila</em>, Japanese researchers were also studying Mendelian inheritance with the Japanese medaka fish, <em>Oryzias latipes</em>. Following three articles published by three separate professors in Japanese on the recessive inheritance of medaka colour variants,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Shoji Fukamachi and Kiyoshi Naruse, &lt;span&gt;“100 Years Since the Medaka’s International Debut: &lt;span&gt;Aida&lt;/span&gt;’s Legacy”&lt;/span&gt; (&lt;span&gt;Genes to Genomes&lt;/span&gt;, August 20, 2021), &lt;a href="https://genestogenomes.org/100-years-since-the-medakas-international-debut-aidas-legacy/" role="doc-biblioref"&gt;https://genestogenomes.org/100-years-since-the-medakas-international-debut-aidas-legacy/&lt;/a&gt;.&lt;/p&gt;'><sup>113</sup></a></span> a teacher named Tatsuo Aida (<strong>Figure <a href="Introduction.html#fig:aida">1.14</a></strong>) published the first article in English about medaka in 1921<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Tatuo Aida, &lt;span&gt;“On the &lt;span&gt;Inheritance&lt;/span&gt; of &lt;span&gt;Color&lt;/span&gt; in a &lt;span&gt;Fresh-Water Fish&lt;/span&gt;, &lt;span&gt;Aplocheilus Latipes Temmick&lt;/span&gt; and &lt;span&gt;Schlegel&lt;/span&gt;, with &lt;span&gt;Special Reference&lt;/span&gt; to &lt;span&gt;Sex-Linked Inheritance&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Genetics&lt;/em&gt; 6, no. 6 (November 1921): 554–73, &lt;a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1200522/" role="doc-biblioref"&gt;https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1200522/&lt;/a&gt;.&lt;/p&gt;'><sup>114</sup></a></span> (<strong>Figure <a href="Introduction.html#fig:aida-fig">1.15</a></strong>). The study involved a total of 22 crosses of two generations, and among several other findings, Aida discovered recombination between the X and Y chromosome – the world’s first demonstration of PSEUDO-AUTOSOMAL SEX CROSSOVER inheritance in any species.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Kiyoshi Naruse, Minoru Tanaka, and Hiroyuki Takeda, &lt;em&gt;Medaka: A Model for Organogenesis, Human Disease, and Evolution&lt;/em&gt; (&lt;span&gt;Springer Science &amp;amp; Business Media&lt;/span&gt;, 2011).&lt;/p&gt;"><sup>115</sup></a></span><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;The editor, William Castle, was so impressed by this last finding that he added the following footnote to the paper: “It should be pointed out, in justice to the author, that his observations go beyond those of Schmidt in the important respect of showing the occurrence of crossing over between the X and the Y chromosome.”&lt;/p&gt;"><sup>116</sup></a></p>

<div class="figure" style="text-align: right">
<span style="display:block;" id="fig:aida"></span>
<img src="figs/introduction/Aida.png" alt="Photo of Tetsuo Aida from.117" width="100%"><p class="caption">
Figure 1.14: Photo of Tetsuo Aida from.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Fukamachi and Naruse, &lt;span&gt;“100 Years Since the Medaka’s International Debut.”&lt;/span&gt;&lt;/p&gt;"><sup>117</sup></a></span>
</p>
</div>

<div class="figure">
<span style="display:block;" id="fig:aida-fig"></span>
<img src="figs/introduction/Aida_1921_fig.png" alt="Figure from Aida118 showing the different medaka colour variants used in the study." width="100%"><p class="caption">
Figure 1.15: Figure from <span class="citation">Aida<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;&lt;span&gt;“On the &lt;span&gt;Inheritance&lt;/span&gt; of &lt;span&gt;Color&lt;/span&gt; in a &lt;span&gt;Fresh-Water Fish&lt;/span&gt;, &lt;span&gt;Aplocheilus Latipes Temmick&lt;/span&gt; and &lt;span&gt;Schlegel&lt;/span&gt;, with &lt;span&gt;Special Reference&lt;/span&gt; to &lt;span&gt;Sex-Linked Inheritance&lt;/span&gt;.”&lt;/span&gt;&lt;/p&gt;"><sup>118</sup></a></span> showing the different medaka colour variants used in the study.
</p>
</div>
<p>In the 1930s, Theodosius Dobzhansky (<strong>Figure <a href="Introduction.html#fig:dobzhansky">1.16</a></strong>), a Ukrainian biologist who had trained with Morgan, used <em>Drosophila pseudoobscura</em> to investigate how genetic variation drove evolution.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee, &lt;em&gt;The &lt;span&gt;Gene&lt;/span&gt;&lt;/em&gt;.&lt;/p&gt;"><sup>119</sup></a></span> Using a single population of flies that he raised in different temperatures while controlling for all other environmental variables, he found that after four months, the genetic ratios in the two sub-populations had changed.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee.&lt;/p&gt;"><sup>120</sup></a></span> Through this experiment, he determined that genetic variation was the norm across biology; that the adaptive benefit of a given variant would depend on the particular environment that an individual found itself in, and that most phenotypes were driven by many genes interacting with each other and the environment, while also subject to chance.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee.&lt;/p&gt;"><sup>121</sup></a></span></p>

<div class="figure" style="text-align: left">
<span style="display:block;" id="fig:dobzhansky"></span>
<img src="figs/introduction/Dobzhansky.png" alt="Undated photo of Theodosius Dobzhansky from Jerry A Coyne122, reprinted courtesy of UW-Madison Archives, #S05461." width="100%"><p class="caption">
Figure 1.16: Undated photo of Theodosius Dobzhansky from <span class="citation">Jerry A Coyne<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;&lt;span&gt;“Theodosius &lt;span&gt;Dobzhansky&lt;/span&gt; on &lt;span&gt;Hybrid Sterility&lt;/span&gt; and &lt;span&gt;Speciation&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Genetics&lt;/em&gt; 202, no. 1 (January 1, 2016): 5–7, &lt;a href="https://doi.org/10.1534/genetics.115.184770" role="doc-biblioref"&gt;https://doi.org/10.1534/genetics.115.184770&lt;/a&gt;.&lt;/p&gt;'><sup>122</sup></a></span>, reprinted courtesy of UW-Madison Archives, #S05461.
</p>
</div>
</div>
<div id="the-discovery-of-the-structure-of-dna" class="section level3" number="1.1.7">
<h3>
<span class="header-section-number">1.1.7</span> The discovery of the structure of DNA<a class="anchor" aria-label="anchor" href="#the-discovery-of-the-structure-of-dna"><i class="fas fa-link"></i></a>
</h3>
<p>By the early 1940s, it was known that genes resided in chromatin, the mixture of proteins and nucleic acids that compose chromosomes. After some flirtation with proteins as the molecule of inheritance, Oswald Avery finally proved that it was in fact deoxyribonucleic acid (<strong>DNA</strong>) that was “the material substance of the gene” – the “cloth from which genes were cut.”<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee, &lt;em&gt;The &lt;span&gt;Gene&lt;/span&gt;&lt;/em&gt;.&lt;/p&gt;"><sup>123</sup></a></span> In 1953, Rosalind Franklin, James Watson, and Francis Crick, and Maurice Wilkins (<strong>Figure <a href="Introduction.html#fig:all-four">1.17</a></strong>) discovered the structure of DNA using X-ray crystallography.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee.&lt;/p&gt;"><sup>124</sup></a></span> [EXPAND THIS SECTION. WATSON EUGENICIST? ENGAGE WITH HIS COMMENTS. CRICK SAYS SOMETHING TOO, BUT LESS PUBLICISED.]</p>

<div class="figure">
<span style="display:block;" id="fig:all-four"></span>
<img src="figs/introduction/all_four.png" alt="Composite image of Rosalind Franklin (1956), James Watson (1980s), Francis Crick (1980s) and Maurice Wilkins (early 1990s) from125 reprinted courtesy of Cold Spring Harbor Laboratory Library and Archive, James D. Watson Collection; Wilkins photo courtesy of TVNZ." width="100%"><p class="caption">
Figure 1.17: Composite image of Rosalind Franklin (1956), James Watson (1980s), Francis Crick (1980s) and Maurice Wilkins (early 1990s) from<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;&lt;span&gt;“&lt;span&gt;DNA&lt;/span&gt; Then and Now,”&lt;/span&gt; accessed August 30, 2022, &lt;a href="https://undsci.berkeley.edu/article/0_0_0/dna_14" role="doc-biblioref"&gt;https://undsci.berkeley.edu/article/0_0_0/dna_14&lt;/a&gt;.&lt;/p&gt;'><sup>125</sup></a></span> reprinted courtesy of Cold Spring Harbor Laboratory Library and Archive, James D. Watson Collection; Wilkins photo courtesy of TVNZ.
</p>
</div>
</div>
<div id="the-development-of-dna-sequencing" class="section level3" number="1.1.8">
<h3>
<span class="header-section-number">1.1.8</span> The development of DNA sequencing<a class="anchor" aria-label="anchor" href="#the-development-of-dna-sequencing"><i class="fas fa-link"></i></a>
</h3>
<p>Over the course of the 1970s, Frederick Sanger (<strong>Figure <a href="Introduction.html#fig:sanger">1.18</a></strong>), a biochemist at the University of Cambridge, developed a method identifying each nucleotide base as it was added to the strand during DNA replication.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mukherjee, &lt;em&gt;The &lt;span&gt;Gene&lt;/span&gt;&lt;/em&gt;.&lt;/p&gt;"><sup>126</sup></a></span> In 1977 he published what became known as the “Sanger method”, using radio-labelled nucleic acids to identify the precise order and type of nucleotides that made up a DNA sequence,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Frederick Sanger, Steven Nicklen, and Alan R. Coulson, &lt;span&gt;“&lt;span&gt;DNA&lt;/span&gt; Sequencing with Chain-Terminating Inhibitors,”&lt;/span&gt; &lt;em&gt;Proceedings of the National Academy of Sciences&lt;/em&gt; 74, no. 12 (1977): 5463–67.&lt;/p&gt;"><sup>127</sup></a></span> earning him a second Nobel prize in Chemistry in 1980.[<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;&lt;span&gt;“The &lt;span&gt;Nobel Prize&lt;/span&gt; in &lt;span&gt;Chemistry&lt;/span&gt; 1980”&lt;/span&gt; (&lt;span&gt;NobelPrize.org&lt;/span&gt;), accessed August 30, 2022, &lt;a href="https://www.nobelprize.org/prizes/chemistry/1980/summary/" role="doc-biblioref"&gt;https://www.nobelprize.org/prizes/chemistry/1980/summary/&lt;/a&gt;.&lt;/p&gt;'><sup>128</sup></a></span>DO I KNOW HOW SEQUENCING WORKS? MAKE SURE I KNOW.]</p>

<div class="figure" style="text-align: right">
<span style="display:block;" id="fig:sanger"></span>
<img src="figs/introduction/sanger.png" alt="Undated photo of Frederick Sanger from.129" width="100%"><p class="caption">
Figure 1.18: Undated photo of Frederick Sanger from.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;&lt;span&gt;“Obituary: &lt;span&gt;Fred Sanger&lt;/span&gt; the Scientist and Twice &lt;span&gt;Nobel Prize&lt;/span&gt; Winner Dies”&lt;/span&gt; (&lt;span&gt;Express.co.uk&lt;/span&gt;, November 23, 2013), &lt;a href="https://www.express.co.uk/news/obituaries/444612/Obituary-Fred-Sanger-the-scientist-and-twice-Nobel-Prize-winner-dies-aged-95" role="doc-biblioref"&gt;https://www.express.co.uk/news/obituaries/444612/Obituary-Fred-Sanger-the-scientist-and-twice-Nobel-Prize-winner-dies-aged-95&lt;/a&gt;.&lt;/p&gt;'><sup>129</sup></a></span>
</p>
</div>
<p>These technological breakthroughs eventually led to the first draft of the human genome being published in 2001 by a large international consortium.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;International Human Genome Sequencing Consortium, &lt;span&gt;“Initial Sequencing and Analysis of the Human Genome,”&lt;/span&gt; &lt;em&gt;Nature&lt;/em&gt; 409, no. 6822, 6822 (February 2001): 860–921, &lt;a href="https://doi.org/10.1038/35057062" role="doc-biblioref"&gt;https://doi.org/10.1038/35057062&lt;/a&gt;.&lt;/p&gt;'><sup>130</sup></a></span><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;The full sequence (including heterochromatic regions) was only completed in 2022 &lt;span class="citation"&gt; (Sergey Nurk et al., &lt;span&gt;“The Complete Sequence of a Human Genome,”&lt;/span&gt; &lt;em&gt;Science&lt;/em&gt; 376, no. 6588 (April 2022): 44–53, &lt;a href="https://doi.org/10.1126/science.abj6987" role="doc-biblioref"&gt;https://doi.org/10.1126/science.abj6987&lt;/a&gt;)&lt;/span&gt;.&lt;/p&gt;'><sup>131</sup></a> The improvement in sequencing technologies has continued to progess to the stage where a $100 human genome is now on the horizon.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Elizabeth Pennisi, &lt;span&gt;“A $100 Genome? &lt;span&gt;New DNA&lt;/span&gt; Sequencers Could Be a &lt;span&gt;‘Game Changer’&lt;/span&gt; for Biology, Medicine | &lt;span&gt;Science&lt;/span&gt; | &lt;span&gt;AAAS&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Science&lt;/em&gt;, June 15, 2022, &lt;a href="https://www.science.org/content/article/100-genome-new-dna-sequencers-could-be-game-changer-biology-medicine" role="doc-biblioref"&gt;https://www.science.org/content/article/100-genome-new-dna-sequencers-could-be-game-changer-biology-medicine&lt;/a&gt;.&lt;/p&gt;'><sup>132</sup></a></span> Long-read technologies developed by Pacific Biosciences and Oxford Nanopore Technologies can also generate single reads of over 2 megabases in length<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Alexander Payne et al., &lt;span&gt;“&lt;span&gt;BulkVis&lt;/span&gt;: A Graphical Viewer for &lt;span&gt;Oxford&lt;/span&gt; Nanopore Bulk &lt;span&gt;Fast5&lt;/span&gt; Files,”&lt;/span&gt; &lt;em&gt;Bioinformatics&lt;/em&gt; 35, no. 13 (July 1, 2019): 2193–98, &lt;a href="https://doi.org/10.1093/bioinformatics/bty841" role="doc-biblioref"&gt;https://doi.org/10.1093/bioinformatics/bty841&lt;/a&gt;.&lt;/p&gt;'><sup>133</sup></a></span> and directly sequence DNA modifications.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Yunhao Wang et al., &lt;span&gt;“Nanopore Sequencing Technology, Bioinformatics and Applications,”&lt;/span&gt; &lt;em&gt;Nature Biotechnology&lt;/em&gt; 39, no. 11, 11 (November 2021): 1348–65, &lt;a href="https://doi.org/10.1038/s41587-021-01108-x" role="doc-biblioref"&gt;https://doi.org/10.1038/s41587-021-01108-x&lt;/a&gt;.&lt;/p&gt;'><sup>134</sup></a></span> These developments are making genetic technologies ever more accessible and informative, paving the way for greater insights into the genetic basis of phenotypic differences. Yet despite these technological advances, the relative contribution of genetics and environment to differences in complex traits is still not fully understood – especially in humans – and therefore remains an active area of research.</p>
</div>
</div>
<div id="genetic-and-environmental-causes-of-variation-in-complex-traits" class="section level2" number="1.2">
<h2>
<span class="header-section-number">1.2</span> Genetic and environmental causes of variation in complex traits<a class="anchor" aria-label="anchor" href="#genetic-and-environmental-causes-of-variation-in-complex-traits"><i class="fas fa-link"></i></a>
</h2>
<p>Individual differences in human traits have now been studied for more than a century, yet the causes of variation in human traits remain uncertain and COMPLEX.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Tinca J. C. Polderman et al., &lt;span&gt;“Meta-Analysis of the Heritability of Human Traits Based on Fifty Years of Twin Studies,”&lt;/span&gt; &lt;em&gt;Nature Genetics&lt;/em&gt; 47, no. 7, 7 (July 2015): 702–9, &lt;a href="https://doi.org/10.1038/ng.3285" role="doc-biblioref"&gt;https://doi.org/10.1038/ng.3285&lt;/a&gt;.&lt;/p&gt;'><sup>135</sup></a></span> Specifically, the partitioning of observed variability into underlying genetic and environmental sources, and the relative importance of additive and non-additive genetic variation, continue to be debated.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Polderman et al.&lt;/p&gt;"><sup>136</sup></a></span></p>
<p>[START WITH THE SIMPLE END. HERITABILITY. GWAS. ANIMAL BREEDING. FAMILIES. ]</p>
<p>A simplified causal model for the inheritance of traits from one’s parents is presented in <strong>Figure <a href="Introduction.html#fig:pheno-causal-model">1.19</a></strong>, adapted from <span class="citation">Tim T. Morris et al.<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;&lt;span&gt;“Population Phenomena Inflate Genetic Associations of Complex Social Traits,”&lt;/span&gt; &lt;em&gt;Science Advances&lt;/em&gt; 6, no. 16 (April 15, 2020): eaay0328, &lt;a href="https://doi.org/10.1126/sciadv.aay0328" role="doc-biblioref"&gt;https://doi.org/10.1126/sciadv.aay0328&lt;/a&gt;.&lt;/p&gt;'><sup>137</sup></a></span> The ancestry of each parent determines both their genotypes and phenotypes (yellow lines). The parents’ genotypes directly affect their own phenotypes (black arrows genotype → phenotype). The parents transmit half of each of their alleles <em>directly</em> to their offspring (black arrows from genotype → genotype), while the non-transmitted alleles also affect their offspring <em>indirectly</em> through their effect on the parents’ phenotypes (red lines), a phenomenon known as “<strong>dynastic effects</strong>.”<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Augustine Kong et al., &lt;span&gt;“The Nature of Nurture: &lt;span&gt;Effects&lt;/span&gt; of Parental Genotypes,”&lt;/span&gt; &lt;em&gt;Science&lt;/em&gt; 359, no. 6374 (January 26, 2018): 424–28, &lt;a href="https://doi.org/10.1126/science.aan6877" role="doc-biblioref"&gt;https://doi.org/10.1126/science.aan6877&lt;/a&gt;.&lt;/p&gt;'><sup>138</sup></a></span> In addition, the parents’ phenotypes can be correlated through “<strong>assortative mating</strong>” (green line), which also has the consequence of creating correlations between the genotypes that affect those phenotypes.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Laurence J. Howe et al., &lt;span&gt;“Genetic Evidence for Assortative Mating on Alcohol Consumption in the &lt;span&gt;UK Biobank&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Nature Communications&lt;/em&gt; 10, no. 1, 1 (November 19, 2019): 5039, &lt;a href="https://doi.org/10.1038/s41467-019-12424-x" role="doc-biblioref"&gt;https://doi.org/10.1038/s41467-019-12424-x&lt;/a&gt;.&lt;/p&gt;'><sup>139</sup></a></span><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;The model has been substantiated by &lt;span class="citation"&gt;Liang-Dar Hwang et al., &lt;span&gt;“Estimating Indirect Parental Genetic Effects on Offspring Phenotypes Using Virtual Parental Genotypes Derived from Sibling and Half Sibling Pairs,”&lt;/span&gt; &lt;em&gt;PLOS Genetics&lt;/em&gt; 16, no. 10 (October 26, 2020): e1009154, &lt;a href="https://doi.org/10.1371/journal.pgen.1009154" role="doc-biblioref"&gt;https://doi.org/10.1371/journal.pgen.1009154&lt;/a&gt;&lt;/span&gt;, who discovered that indirect parental genetic effects could influence offspring phenotypes through the alleles that they &lt;em&gt;do not&lt;/em&gt; pass on to their children, but which drive the parents’ own behaviour, ultimately influencing the child’s outcomes. Their strategy involved creating “virtual” mothers and fathers by estimating the genotypic dosages of parental genotypes using physically genotyped data from relative pairs. They applied the approach to 19,066 sibling pairs from the UK Biobank and showed a polygenic score consisting of imputed parental educational attainment SNP dosages is strongly related to offspring educational attainment even after correcting for offspring genotype at the same loci.&lt;/p&gt;'><sup>140</sup></a></p>

<div class="figure">
<span style="display:block;" id="fig:pheno-causal-model"></span>
<img src="figs/introduction/pheno_causal_model.png" alt="Causal model of offspring phenotypes. Ancestry causes differences in both genotypes and phenotypes of the parents (yellow lines). Dynastic effects are transmitted through parental parental phenotypes (red lines). Assortative mating creates associations between the phenotypes of the parents (green line), which will also lead to genotypic correlations. Arrows represent direction of effect. Figure adapted from Morris et al.141" width="100%"><p class="caption">
Figure 1.19: Causal model of offspring phenotypes. Ancestry causes differences in both genotypes and phenotypes of the parents (yellow lines). Dynastic effects are transmitted through parental parental phenotypes (red lines). Assortative mating creates associations between the phenotypes of the parents (green line), which will also lead to genotypic correlations. Arrows represent direction of effect. Figure adapted from <span class="citation">Morris et al.<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;&lt;span&gt;“Population Phenomena Inflate Genetic Associations of Complex Social Traits.”&lt;/span&gt;&lt;/p&gt;"><sup>141</sup></a></span>
</p>
</div>
<p>This model only includes environmental factors related to the parents’ phenotypes, yet the phenotype of an individual is continuously affected by all other environmental influences it is exposed to throughout its lifetime.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Robert Plomin and Kathryn Asbury, &lt;span&gt;“Nature and &lt;span&gt;Nurture&lt;/span&gt;: &lt;span&gt;Genetic&lt;/span&gt; and &lt;span&gt;Environmental Influences&lt;/span&gt; on &lt;span&gt;Behavior&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;The ANNALS of the American Academy of Political and Social Science&lt;/em&gt; 600, no. 1 (July 1, 2005): 86–98, &lt;a href="https://doi.org/10.1177/0002716205277184" role="doc-biblioref"&gt;https://doi.org/10.1177/0002716205277184&lt;/a&gt;.&lt;/p&gt;'><sup>142</sup></a></span> It is impossible to parse out the respective contributions of each of these factors at the individual level:<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Valerie S. Knopik et al., &lt;em&gt;Behavioral &lt;span&gt;Genetics&lt;/span&gt;&lt;/em&gt; (&lt;span&gt;Macmillan Learning&lt;/span&gt;, 2019), &lt;a href="https://books.google.com?id=mtDJvQEACAAJ" role="doc-biblioref"&gt;https://books.google.com?id=mtDJvQEACAAJ&lt;/a&gt;.&lt;/p&gt;'><sup>143</sup></a></span> to what extent does the width or the length of a rectangle determine its area? It is determined by the product of the two. But by examining a population of rectangles, one can determine how much of the variance within the population depends on either width or length. The same principle applies to genotypic and environmental variation within groups of individuals.</p>
<div id="heritability" class="section level3" number="1.2.1">
<h3>
<span class="header-section-number">1.2.1</span> Heritability<a class="anchor" aria-label="anchor" href="#heritability"><i class="fas fa-link"></i></a>
</h3>
<p>Fisher’s analysis of variance methodology allows for the partitioning of variance into components that are attributable to different factors. If the total observed phenotypic variation within a population is represented by <span class="math inline">\(V_{P}\)</span>, then it can be decomposed into a simple model that includes genetic and environmental factors:<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;D. S. Falconer and T. F. C. Mackay, &lt;em&gt;Introduction to Quantitative Genetics&lt;/em&gt; (&lt;span&gt;UK: Longman Group&lt;/span&gt;, 1996).&lt;/p&gt;"><sup>144</sup></a></span></p>
<p><span class="math display" id="eq:pge">\[\begin{equation}
V_{P} = V_{G} + V_{E} \tag{1.3}
\end{equation}\]</span></p>
<p><span class="math inline">\(V_{G}\)</span> is the total proportion of phenotypic variance attributable to genetic factors, and <span class="math inline">\(V_{E}\)</span> is that attributable to environmental factors (i.e. all non-genetic factors). <em>Broad-sense heritability</em> (<span class="math inline">\(H^2\)</span>) measures the proportion of phenotypic variance that is attributable to genetic factors:<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Falconer and Mackay.&lt;/p&gt;"><sup>145</sup></a></span></p>
<p><span class="math display">\[
H^2 = \frac{V_G}{V_P}
\]</span></p>
<p><em>Narrow-sense heritability</em> (<span class="math inline">\(h^2\)</span>) measures the proportion of phenotypic variance attributable to <em>additive</em> genetic variance, excluding variance caused by Mendelian dominant/recessive alleles, and epistasis (the interaction of alleles at different loci):<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;William Bateson, &lt;span&gt;“Mendel’s &lt;span&gt;Principles&lt;/span&gt; of &lt;span&gt;Heredity&lt;/span&gt;: &lt;span&gt;Cambridge University Press&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;März 1909; 2nd Impr&lt;/em&gt; 3 (1909): 1913; Falconer and Mackay, &lt;em&gt;Introduction to Quantitative Genetics&lt;/em&gt;; Daniëlle Posthuma et al., &lt;span&gt;“Theory and &lt;span&gt;Practice&lt;/span&gt; in &lt;span&gt;Quantitative Genetics&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Twin Research and Human Genetics&lt;/em&gt; 6, no. 5 (October 2003): 361–76, &lt;a href="https://doi.org/10.1375/twin.6.5.361" role="doc-biblioref"&gt;https://doi.org/10.1375/twin.6.5.361&lt;/a&gt;.&lt;/p&gt;'><sup>146</sup></a></span></p>
<p><span class="math display" id="eq:heritns">\[\begin{equation}
h^2 = \frac{V_A}{V_P} \tag{1.4}
\end{equation}\]</span></p>
<p>Environmental variance cannot be removed experimentally because it includes by definition all non-genetic variance, and much of this is beyond experimental control. However, elimination of genotypic variance can be achieved experimentally<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Falconer and Mackay, &lt;em&gt;Introduction to Quantitative Genetics&lt;/em&gt;.&lt;/p&gt;"><sup>147</sup></a></span> by either using human or animal identical twins, or by creating highly inbred lines. Pursuant to Equation <a href="Introduction.html#eq:pge">(1.3)</a>, the environmental variance (<span class="math inline">\(V_E\)</span>) can be subtracted from the total phenotypic variance (<span class="math inline">\(V_P\)</span>) to give an estimate of the trait’s heritability <span class="math inline">\(V_G\)</span>.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Falconer and Mackay.&lt;/p&gt;"><sup>148</sup></a></span><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;It is important to note that as heritability is measured as a fraction of total phenotypic variance, it is liable to change depending on the level of environmental variation present in the study population &lt;span class="citation"&gt; (Peter M. Visscher, William G. Hill, and Naomi R. Wray, &lt;span&gt;“Heritability in the Genomics Era — Concepts and Misconceptions,”&lt;/span&gt; &lt;em&gt;Nature Reviews Genetics&lt;/em&gt; 9, no. 4, 4 (April 2008): 255–66, &lt;a href="https://doi.org/10.1038/nrg2322" role="doc-biblioref"&gt;https://doi.org/10.1038/nrg2322&lt;/a&gt;)&lt;/span&gt;.&lt;/p&gt;'><sup>149</sup></a></p>
<p>In humans, the twin study design has been used widely to disentangle the relative contributions of genes and environment for a variety of traits. The classical twin design is based on contrasting the trait resemblance of monozygotic (<strong>MZ</strong>) and dizygotic (<strong>DZ</strong>) twin pairs.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Polderman et al., &lt;span&gt;“Meta-Analysis of the Heritability of Human Traits Based on Fifty Years of Twin Studies.”&lt;/span&gt;&lt;/p&gt;"><sup>150</sup></a></span> MZ twins share approximately 100% of genotypes, whereas DZ twins, like siblings, share on average 50% of their genotypes.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Knopik et al., &lt;em&gt;Behavioral &lt;span&gt;Genetics&lt;/span&gt;&lt;/em&gt;.&lt;/p&gt;"><sup>151</sup></a></span> Assuming they share the same environment (an point I return to below), a rough estimate of heritability can be obtained by comparing the respective correlations of MZ twins and DZ twins with the phenotype of interest. If <span class="math inline">\(r_{MZ}\)</span> and <span class="math inline">\(r_{DZ}\)</span> are the correlation coefficients observed between pairs of MZ and DZ twins for a trait of interest (Equation <a href="Introduction.html#eq:cor">(1.2)</a> above), <span class="math inline">\(h^2\)</span> = additive genetic influences (Equation <a href="Introduction.html#eq:heritns">(1.4)</a> above), and <span class="math inline">\(c^2\)</span> = common environmental influences, then Falconer’s formula tells us:</p>
<p><span class="math display">\[\begin{equation}
r_{MZ} = h^2 + c^2
\end{equation}\]</span></p>
<p><span class="math display">\[\begin{equation}
r_{DZ} = \tfrac{1}{2}h^2 + c^2
\end{equation}\]</span></p>
<p><span class="math display">\[\begin{equation}
\tfrac{1}{2}h^2 = r_{MZ} - r_{DZ}
\end{equation}\]</span></p>
<p>Therefore:
<span class="math display">\[\begin{equation}
h^2 = 2\times(r_{MZ} - r_{DZ})
\end{equation}\]</span></p>
<p>Narrow-sense heritability <span class="math inline">\(h^2\)</span> can therefore be estimated by doubling the difference between the correlation coefficients of MZ versus DZ twins. A 2015 meta-analysis of virtually all human twin studies that had been performed to that date included 2,748 twin studies assessing ~18,000 traits and ~14.5M twin pairs. The results showed that across all traits, <span class="math inline">\(h^2\)</span> is 0.488 and <span class="math inline">\(c^2\)</span> is 0.174.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Polderman et al., &lt;span&gt;“Meta-Analysis of the Heritability of Human Traits Based on Fifty Years of Twin Studies.”&lt;/span&gt;&lt;/p&gt;"><sup>152</sup></a></span> Moreover, in 69% of studies <span class="math inline">\(2r_{DZ} = r_{MZ}\)</span> (implying that the shared environment <span class="math inline">\(c^2\)</span> is equivalent across both MZ and DZ twin pairs). This suggests that for most traits, shared environmental or non-additive genetic variation do not make a substantial contribution to phenotypic variation.</p>
<p>[COULD BE THE INTRO TO: THIS SIMPLE MODEL DOESN’T WORK] However, that study found that several behavioural traits were inconsistent with this model. These traits included conduct disorder, higher-level cognitive functions, mental and behavioural disorders due to the use of alcohol or tobacco, anxiety disorder, and weight maintenance. For these traits, either or both non-additive genetic influences or shared environmental influences are therefore needed to explain the observed patterns of twin correlations.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Polderman et al.&lt;/p&gt;"><sup>153</sup></a></span><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;&lt;span class="citation"&gt;Kong et al., &lt;span&gt;“The Nature of Nurture”&lt;/span&gt;&lt;/span&gt; also showed that parents’ non-transmitted alleles can affect a child through their impacts on the parents and other relatives, a phenomenon the authors call “genetic nurture”, comprising 29.9% of the transmitted polygenic score.&lt;/p&gt;'><sup>154</sup></a></p>
<p>Beyond twin-studies, the advent of next-generation sequencing technologies has enabled the dense genotyping of large samples of individuals, leading to the development of new methods for analysing the genetic influences on human traits. The most widely used method is known as the Genome-Wide Association Analysis (<strong>GWAS</strong>).</p>
</div>
<div id="genome-wide-association-analysis-gwas" class="section level3" number="1.2.2">
<h3>
<span class="header-section-number">1.2.2</span> Genome-Wide Association Analysis (GWAS)<a class="anchor" aria-label="anchor" href="#genome-wide-association-analysis-gwas"><i class="fas fa-link"></i></a>
</h3>
<p>GWAS aim to identify genetic variants associated with traits by comparing the allele frequencies of individuals who share similar ancestries, but differ in values for the trait in question.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Emil Uffelmann et al., &lt;span&gt;“Genome-Wide Association Studies,”&lt;/span&gt; &lt;em&gt;Nature Reviews Methods Primers&lt;/em&gt; 1, no. 1, 1 (August 26, 2021): 1–21, &lt;a href="https://doi.org/10.1038/s43586-021-00056-9" role="doc-biblioref"&gt;https://doi.org/10.1038/s43586-021-00056-9&lt;/a&gt;.&lt;/p&gt;'><sup>155</sup></a></span> As of 2021, over 5,700 GWAS have been performed for more than 3,330 human traits.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Uffelmann et al.&lt;/p&gt;"><sup>156</sup></a></span></p>
<p>The standard GWAS process involves first genotyping individuals using SNP microarrays – which target common genetic variants – or by sequencing their full exomes<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;The coding regions of the genome.&lt;/p&gt;"><sup>157</sup></a> (Whole Exome Sequencing, <strong>WES</strong>) or genomes (Whole-Genome Sequencing, <strong>WGS</strong>). It is common to then use principal components analysis (<strong>PCA</strong>) [AND OTHER METHODS?] to restrict the analysis individuals who share common ancestry, in order to avoid population stratification effects.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Uffelmann et al., &lt;span&gt;“Genome-Wide Association Studies.”&lt;/span&gt;&lt;/p&gt;"><sup>158</sup></a></span> Missing genotypes can be imputed using a genetic reference panel such as the 1000 Genomes Project.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;1000 Genomes Project Consortium et al., &lt;span&gt;“A Global Reference for Human Genetic Variation,”&lt;/span&gt; &lt;em&gt;Nature&lt;/em&gt; 526, no. 7571 (2015): 68.&lt;/p&gt;"><sup>159</sup></a></span> Finally, an association test can be run, which includes an individual-specific random effect term to account for genetic relatedness among individuals.</p>
<p>Since it became possible to sequence the genotypes of individuals at scale, it has been an ongoing point of debate as to how best to model the effects that genetic variants have on a trait of interest. The standard linear regression model currently used for GWAS can be written as follows:<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Uffelmann et al., &lt;span&gt;“Genome-Wide Association Studies.”&lt;/span&gt;&lt;/p&gt;"><sup>160</sup></a></span></p>
<p><span class="math display">\[\begin{equation}
\textbf{Y} \sim \textbf{W} \alpha ~+~{X_s}{\beta_s} + g + e
\end{equation}\]</span>
<span class="math display">\[\begin{equation}
g \sim N(0,\sigma^2_A\psi)
\end{equation}\]</span>
<span class="math display">\[\begin{equation}
e \sim N(0,\sigma^2_e \textbf{I})
\end{equation}\]</span></p>
<p>where, for each individual, <span class="math inline">\(\textbf{Y}\)</span> is a vector of phenotype values, <span class="math inline">\(\textbf{W}\)</span> is a matrix of covariates including an intercept term, <span class="math inline">\(\alpha\)</span> is a corresponding vector of effect sizes, <span class="math inline">\(X_s\)</span> is a vector of genotype values for all individuals at SNP <span class="math inline">\(s\)</span>, <span class="math inline">\(\beta_s\)</span> is the corresponding fixed effect size of genetic variant <span class="math inline">\(s\)</span> (also known as the SNP effect size), <span class="math inline">\(g\)</span> is a random effect that captures the polygenic effect of other SNPs, <span class="math inline">\(e\)</span> is a random effect of residual errors, <span class="math inline">\(\sigma^2_{A}\)</span> measures additive genetic variation of the phenotype, <span class="math inline">\(\psi\)</span> is the standard genetic relationship matrix, <span class="math inline">\(\sigma^2_e\)</span> measures residual variance, and <span class="math inline">\(\textbf{I}\)</span> is an identity matrix.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Uffelmann et al.&lt;/p&gt;"><sup>161</sup></a></span></p>
<p>[EXPAND ON GRM]</p>
<p>Population structure and unequal relatedness among individuals in a given cohort can lead to false discoveries.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;W J Ewens and R S Spielman, &lt;span&gt;“The Transmission/Disequilibrium Test: History, Subdivision, and Admixture.”&lt;/span&gt; &lt;em&gt;American Journal of Human Genetics&lt;/em&gt; 57, no. 2 (August 1995): 455–64, &lt;a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1801556/" role="doc-biblioref"&gt;https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1801556/&lt;/a&gt;; Members of the Complex Trait Consortium, &lt;span&gt;“The Nature and Identification of Quantitative Trait Loci: A Community’s View,”&lt;/span&gt; &lt;em&gt;Nature Reviews Genetics&lt;/em&gt; 4, no. 11, 11 (November 2003): 911–16, &lt;a href="https://doi.org/10.1038/nrg1206" role="doc-biblioref"&gt;https://doi.org/10.1038/nrg1206&lt;/a&gt;.&lt;/p&gt;'><sup>162</sup></a></span> This is because individuals who share common ancestries will share both variants that do affect the trait of interest, and variants that do not. Both causal and non-causal variants will be correlated with one other due to that shared ancestry. Therefore, if an association is found between the causal variants and a trait of interest, the non-causal variants that are correlated with the causal variants will also be found to be statistically associated with the trait.</p>
<p>GWAS output a <span class="math inline">\(p\)</span>-value and directional effect size (commonly denoted as <span class="math inline">\(\beta\)</span>) for each tested SNP. Significance of SNPs are determined by either Bonferroni correction (<span class="math inline">\(\alpha / M_{SNPs}\)</span>, commonly set as <span class="math inline">\(p &lt; 5 \times 10^{-8}\)</span> for human studies), [NO!!! ARGUMENT MAD EIN THE CASE-CONTROL CONSORTIUM. CLOSER TO A FALSE DISCOVERY RATE ARGUMENT. BASED ON THE QQPLOT. PERMUTATIONS FROM FARM ANIMALS. IN HUMANS THE POPULATION IS ALWAYS SORT FO THE SAME. WITH FARM ANIMALS WE ALWAYS HAVE COMPLICATED STRUCTURE. NEED TO MENTION IN LINKAGE STUDIES OF LOD &gt; 3 IE 1/1000 CRITERIA. ALSO A BIT ARBITRARY. AT WHAT POINT DOES A LOCUS BECOME INTERESTING TO LOOK AT? ] or by running permutations of the phenotype (and covariates) and selecting the lowest <span class="math inline">\(p\)</span>-value from those permutations as the significance threshold.<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;The summed effect sizes of significant SNPs should theoretically reach the heritability estimates from twin studies, however until recently they fell far short, leading to a discussion on the sources of this “missing heritability” &lt;span class="citation"&gt; (Teri A. Manolio et al., &lt;span&gt;“Finding the Missing Heritability of Complex Diseases,”&lt;/span&gt; &lt;em&gt;Nature&lt;/em&gt; 461, no. 7265, 7265 (October 2009): 747–53, &lt;a href="https://doi.org/10.1038/nature08494" role="doc-biblioref"&gt;https://doi.org/10.1038/nature08494&lt;/a&gt;)&lt;/span&gt;. GWAS are limited by sample size, as extremely large samples are required to detect rare variants, and variants with small effect sizes. Short-read sequencing technologies are also unable to detect large “structural” variants, which may have large effects on the phenotype of interest. However, this issue has largely been resolved through improvements in genotyping that can capture rare variants &lt;span class="citation"&gt; (Jian Yang et al., &lt;span&gt;“Common &lt;span&gt;SNPs&lt;/span&gt; Explain a Large Proportion of the Heritability for Human Height,”&lt;/span&gt; &lt;em&gt;Nature Genetics&lt;/em&gt; 42, no. 7, 7 (July 2010): 565–69, &lt;a href="https://doi.org/10.1038/ng.608" role="doc-biblioref"&gt;https://doi.org/10.1038/ng.608&lt;/a&gt;)&lt;/span&gt;, and tools such as GCTA which estimates the variance explained by &lt;em&gt;all&lt;/em&gt; SNPs in the genome, rather than testing the association of individual SNPs &lt;span class="citation"&gt; (Jian Yang et al., &lt;span&gt;“&lt;span&gt;GCTA&lt;/span&gt;: &lt;span&gt;A Tool&lt;/span&gt; for &lt;span class="nocase"&gt;Genome-wide Complex Trait Analysis&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;The American Journal of Human Genetics&lt;/em&gt; 88, no. 1 (January 7, 2011): 76–82, &lt;a href="https://doi.org/10.1016/j.ajhg.2010.11.011" role="doc-biblioref"&gt;https://doi.org/10.1016/j.ajhg.2010.11.011&lt;/a&gt;)&lt;/span&gt;.&lt;/p&gt;'><sup>163</sup></a></p>
</div>
<div id="animal-models" class="section level3" number="1.2.3">
<h3>
<span class="header-section-number">1.2.3</span> Animal models<a class="anchor" aria-label="anchor" href="#animal-models"><i class="fas fa-link"></i></a>
</h3>
<p>Twin studies and GWAS have led to great advances in our understanding of the genetic basis of complex traits. However they are hampered by the inability to experimentally manipulate either genes or environment in humans. Since the rediscovery of Mendel’s work, researchers have resorted to using model organisms, with which it is possible to control for both.</p>
<p>The genetics of model organisms may be controlled to a degree by establishing inbred strains. Mendel used self-fertilising plants, which without sexual reproduction are effectively inbred strains. At that time, inbred strains of mammals were unknown. But through the repeated mating of siblings over successive generations, as the individuals within each line inherit the same same haplotype from their related parents, they become almost genetically identical to one another, with the added benefit that their genotypes can be replicated across time in subsequent generations.</p>
<p>In the early 1900s, researchers began to develop inbred lines of guinea pigs and mice for the purpose of genetic research. In 1906, an inbreeding experiment involving guinea-pigs was started by George M. Rommel of the Animal Husbandry Division of the United States Department of Agriculture.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Orson N. Eaton, &lt;span&gt;“A Quarter-Century of Inbreeding in Guinea-Pigs,”&lt;/span&gt; &lt;em&gt;Journal of Experimental Zoology&lt;/em&gt; 63, no. 2 (1932): 261–90, &lt;a href="https://doi.org/10.1002/jez.1400630202" role="doc-biblioref"&gt;https://doi.org/10.1002/jez.1400630202&lt;/a&gt;.&lt;/p&gt;'><sup>164</sup></a></span> The experiment was taken over in 1915 by Sewall Wright, who used them to develop his vast contributions to the field.</p>
<p>Contemporaneously, the Harvard undergraduate Clarence Cook Little and his supervisor William Ernest Little collaborated with Abbie Lathrop, a breeder of fancy mice and rats which she marketed to rodent hobbyists and keepers of exotic pets, and later began selling in large numbers to scientific researchers.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;David P. Steensma, Robert A. Kyle, and Marc A. Shampo, &lt;span&gt;“Abbie &lt;span&gt;Lathrop&lt;/span&gt;, the &lt;span&gt;‘&lt;span&gt;Mouse Woman&lt;/span&gt; of &lt;span&gt;Granby&lt;/span&gt;’&lt;/span&gt;: &lt;span&gt;Rodent Fancier&lt;/span&gt; and &lt;span&gt;Accidental Genetics Pioneer&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Mayo Clinic Proceedings&lt;/em&gt; 85, no. 11 (November 1, 2010): e83, &lt;a href="https://doi.org/10.4065/mcp.2010.0647" role="doc-biblioref"&gt;https://doi.org/10.4065/mcp.2010.0647&lt;/a&gt;.&lt;/p&gt;'><sup>165</sup></a></span> In 1902, Castle bought some of Lathrop’s mice for his laboratory.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Steensma, Kyle, and Shampo.&lt;/p&gt;"><sup>166</sup></a></span> The most frequently used laboratory mouse strain for the past 80 years, C57BL/6J (“Black J”), is derived from one of Lathrop’s animals – mouse number 57 – bred by Little.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Steensma, Kyle, and Shampo.&lt;/p&gt;"><sup>167</sup></a></span></p>
<p>The utility of inbred strains eventually led to the establishment of “panels” of inbred strains for several model organisms including the thale cress (<em>Arabidopsis thaliana</em>),<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Joy Bergelson and Fabrice Roux, &lt;span&gt;“Towards Identifying Genes Underlying Ecologically Relevant Traits in &lt;span&gt;Arabidopsis&lt;/span&gt; Thaliana,”&lt;/span&gt; &lt;em&gt;Nature Reviews Genetics&lt;/em&gt; 11, no. 12, 12 (December 2010): 867–79, &lt;a href="https://doi.org/10.1038/nrg2896" role="doc-biblioref"&gt;https://doi.org/10.1038/nrg2896&lt;/a&gt;.&lt;/p&gt;'><sup>168</sup></a></span> common bean (<em>Phaseolus vulgaris L</em>),<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;William C. Johnson and Paul Gepts, &lt;span&gt;“Segregation for Performance in Recombinant Inbred Populations Resulting from Inter-Gene Pool Crosses of Common Bean ( &lt;span&gt;Phaseolus&lt;/span&gt; Vulgaris &lt;span&gt;L&lt;/span&gt;.),”&lt;/span&gt; &lt;em&gt;Euphytica&lt;/em&gt; 106, no. 1 (March 1, 1999): 45–56, &lt;a href="https://doi.org/10.1023/A:1003541201923" role="doc-biblioref"&gt;https://doi.org/10.1023/A:1003541201923&lt;/a&gt;.&lt;/p&gt;'><sup>169</sup></a></span> tomato (<em>Lycopersicon esculentum</em>),<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Vera Saliba-Colombani et al., &lt;span&gt;“Efficiency of &lt;span&gt;RFLP&lt;/span&gt;, &lt;span&gt;RAPD&lt;/span&gt;, and &lt;span&gt;AFLP&lt;/span&gt; Markers for the Construction of an Intraspecific Map of the Tomato Genome,”&lt;/span&gt; &lt;em&gt;Genome&lt;/em&gt; 43, no. 1 (February 2000): 29–40, &lt;a href="https://doi.org/10.1139/g99-096" role="doc-biblioref"&gt;https://doi.org/10.1139/g99-096&lt;/a&gt;.&lt;/p&gt;'><sup>170</sup></a></span> maize (<em>Zea mays</em>),<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Anis M. Limami et al., &lt;span&gt;“Genetic and &lt;span&gt;Physiological Analysis&lt;/span&gt; of &lt;span&gt;Germination Efficiency&lt;/span&gt; in &lt;span&gt;Maize&lt;/span&gt; in &lt;span&gt;Relation&lt;/span&gt; to &lt;span&gt;Nitrogen Metabolism Reveals&lt;/span&gt; the &lt;span&gt;Importance&lt;/span&gt; of &lt;span&gt;Cytosolic Glutamine Synthetase&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Plant Physiology&lt;/em&gt; 130, no. 4 (December 1, 2002): 1860–70, &lt;a href="https://doi.org/10.1104/pp.009647" role="doc-biblioref"&gt;https://doi.org/10.1104/pp.009647&lt;/a&gt;.&lt;/p&gt;'><sup>171</sup></a></span> nematode (<em>Caenorhabditis elegans</em>),<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Kathryn S. Evans et al., &lt;span&gt;“From &lt;span&gt;QTL&lt;/span&gt; to Gene: &lt;span&gt;C&lt;/span&gt;. Elegans Facilitates Discoveries of the Genetic Mechanisms Underlying Natural Variation,”&lt;/span&gt; &lt;em&gt;Trends in Genetics&lt;/em&gt; 37, no. 10 (October 1, 2021): 933–47, &lt;a href="https://doi.org/10.1016/j.tig.2021.06.005" role="doc-biblioref"&gt;https://doi.org/10.1016/j.tig.2021.06.005&lt;/a&gt;.&lt;/p&gt;'><sup>172</sup></a></span> fruit fly (<em>Drosophila melanogaster</em>),<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Trudy F. C. Mackay and Wen Huang, &lt;span&gt;“Charting the Genotype–Phenotype Map: Lessons from the &lt;span&gt;Drosophila&lt;/span&gt; Melanogaster &lt;span&gt;Genetic Reference Panel&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;WIREs Developmental Biology&lt;/em&gt; 7, no. 1 (2018): e289, &lt;a href="https://doi.org/10.1002/wdev.289" role="doc-biblioref"&gt;https://doi.org/10.1002/wdev.289&lt;/a&gt;.&lt;/p&gt;'><sup>173</sup></a></span> and mouse (<em>Mus musculus</em>).<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Michael C. Saul et al., &lt;span&gt;“High-&lt;span&gt;Diversity Mouse Populations&lt;/span&gt; for &lt;span&gt;Complex Traits&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Trends in Genetics&lt;/em&gt; 35, no. 7 (July 1, 2019): 501–14, &lt;a href="https://doi.org/10.1016/j.tig.2019.04.003" role="doc-biblioref"&gt;https://doi.org/10.1016/j.tig.2019.04.003&lt;/a&gt;.&lt;/p&gt;'><sup>174</sup></a></span></p>
<p>Although the mouse is an appropriate model for humans due to their orthologous mammalian organ systems and cell types, inbred strains of this organism descend from individuals that had already been domesticated, and therefore do not represent the genetic variation present in wild populations. Furthermore, the large panels of inbred mice such as the Collaborative Cross (<strong>CC</strong>),<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;David W. Threadgill et al., &lt;span&gt;“The &lt;span&gt;Collaborative Cross&lt;/span&gt;: &lt;span&gt;A Recombinant Inbred Mouse Population&lt;/span&gt; for the &lt;span&gt;Systems Genetic Era&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;ILAR Journal&lt;/em&gt; 52, no. 1 (January 1, 2011): 24–31, &lt;a href="https://doi.org/10.1093/ilar.52.1.24" role="doc-biblioref"&gt;https://doi.org/10.1093/ilar.52.1.24&lt;/a&gt;.&lt;/p&gt;'><sup>175</sup></a></span> Diversity Outcross (<strong>DO</strong>)<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Karen L Svenson et al., &lt;span&gt;“High-&lt;span&gt;Resolution Genetic Mapping Using&lt;/span&gt; the &lt;span&gt;Mouse Diversity Outbred Population&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Genetics&lt;/em&gt; 190, no. 2 (February 1, 2012): 437–47, &lt;a href="https://doi.org/10.1534/genetics.111.132597" role="doc-biblioref"&gt;https://doi.org/10.1534/genetics.111.132597&lt;/a&gt;.&lt;/p&gt;'><sup>176</sup></a></span> and B6-by-D2 (<strong>BXD</strong>)<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Jeremy L. Peirce et al., &lt;span&gt;“A New Set of &lt;span&gt;BXD&lt;/span&gt; Recombinant Inbred Lines from Advanced Intercross Populations in Mice,”&lt;/span&gt; &lt;em&gt;BMC Genetics&lt;/em&gt; 5, no. 1 (April 29, 2004): 7, &lt;a href="https://doi.org/10.1186/1471-2156-5-7" role="doc-biblioref"&gt;https://doi.org/10.1186/1471-2156-5-7&lt;/a&gt;.&lt;/p&gt;'><sup>177</sup></a></span> are derived from only a small number of individuals. [THEY’VE COME FROM THIS COMPLEX HYBRIDISATION FROM THREE DIFFERENT SPECIES OF MICE. HAPPENED IN JAPAN – FANCY MICE.] As gene-environment studies seek to ultimately understand their effects on traits “in the wild” (such as with humans), there is accordingly a need for a panel of inbred vertebrates that represents the genetic variation present in natural populations.</p>
</div>
<div id="the-medaka-inbred-kiyosu-karlsruhe-mikk-panel" class="section level3" number="1.2.4">
<h3>
<span class="header-section-number">1.2.4</span> The Medaka Inbred Kiyosu-Karlsruhe (MIKK) panel<a class="anchor" aria-label="anchor" href="#the-medaka-inbred-kiyosu-karlsruhe-mikk-panel"><i class="fas fa-link"></i></a>
</h3>
<p>[REMIND THAT ITS THE SAME FISH THAT AIDA USED IN 1921]
The medaka fish (<em>Oryzias latipes</em>) has been studied as a model organism in Japan for over a century,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Joachim Wittbrodt, Akihiro Shima, and Manfred Schartl, &lt;span&gt;“Medaka — a Model Organism from the Far East,”&lt;/span&gt; &lt;em&gt;Nature Reviews Genetics&lt;/em&gt; 3, no. 1, 1 (January 2002): 53–64, &lt;a href="https://doi.org/10.1038/nrg704" role="doc-biblioref"&gt;https://doi.org/10.1038/nrg704&lt;/a&gt;.&lt;/p&gt;'><sup>178</sup></a></span> and is gaining recognition elsewhere as a powerful genetic model for vertebrates.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Mikhail Spivakov et al., &lt;span&gt;“Genomic and &lt;span&gt;Phenotypic Characterization&lt;/span&gt; of a &lt;span&gt;Wild Medaka Population&lt;/span&gt;: &lt;span&gt;Towards&lt;/span&gt; the &lt;span&gt;Establishment&lt;/span&gt; of an &lt;span&gt;Isogenic Population Genetic Resource&lt;/span&gt; in &lt;span&gt;Fish&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;G3 Genes|Genomes|Genetics&lt;/em&gt; 4, no. 3 (March 1, 2014): 433–45, &lt;a href="https://doi.org/10.1534/g3.113.008722" role="doc-biblioref"&gt;https://doi.org/10.1534/g3.113.008722&lt;/a&gt;.&lt;/p&gt;'><sup>179</sup></a></span> In addition to possessing a number of desirable traits that are characteristic of model organisms (including their small-size, short reproduction time, and high fertility), medaka are also – uniquely among vertebrates – resilient to inbreeding from the wild.</p>
<p>This resilience to inbreeding has allowed (predominantly Japanese) researchers to develop several inbred strains that have been maintained for over 100 generations. One of the most famous inbred medaka strains, <em>HdrR</em>, was derived from Tatsuo Aida’s collection of southern Japanese medaka fish.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Fukamachi and Naruse, &lt;span&gt;“100 Years Since the Medaka’s International Debut.”&lt;/span&gt;&lt;/p&gt;"><sup>180</sup></a></span> Other prominent strains include <em>HO5</em> and <em>iCab</em> from southern Japan, and <em>Kaga</em> and <em>HNI</em> from northern Japan. The locations of the originating populations of these strains are set out in <strong>Figure <a href="Introduction.html#fig:line-locations">1.20</a></strong>.</p>

<div class="figure">
<span style="display:block;" id="fig:line-locations"></span>
<img src="figs/pilot/line_locations.png" alt="Image adapted from,181 showing the locations of the originating populations of the 5 inbred medaka lines used in this study." width="100%"><p class="caption">
Figure 1.20: Image adapted from,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Spivakov et al., &lt;span&gt;“Genomic and &lt;span&gt;Phenotypic Characterization&lt;/span&gt; of a &lt;span&gt;Wild Medaka Population&lt;/span&gt;.”&lt;/span&gt;&lt;/p&gt;"><sup>181</sup></a></span> showing the locations of the originating populations of the 5 inbred medaka lines used in this study.
</p>
</div>
<p>Since 2010, the Birney Group at EMBL-EBI, in collaboration with the Wittbrodt Group at COS, University of Heidelberg and the Loosli Group at the Karlsruhe Institute of Technology (KIT), have been working to establish the world’s first panel of vertebrate inbred strains FROM THE WILD – now known as the Medaka Inbred Kiyosu-Karlsruhe Panel (<strong>MIKK panel</strong>). The MIKK Panel was bred from a wild population caught near Kiyosu in Southern Japan (<strong>Figure <a href="Introduction.html#fig:line-locations">1.20</a></strong>), and now comprises 80 inbred, near-isogenic “lines.”<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Tomas Fitzgerald et al., &lt;span&gt;“The &lt;span&gt;Medaka Inbred Kiyosu-Karlsruhe&lt;/span&gt; (&lt;span&gt;MIKK&lt;/span&gt;) Panel,”&lt;/span&gt; &lt;em&gt;Genome Biology&lt;/em&gt; 23, no. 1 (February 21, 2022): 59, &lt;a href="https://doi.org/10.1186/s13059-022-02623-z" role="doc-biblioref"&gt;https://doi.org/10.1186/s13059-022-02623-z&lt;/a&gt;.&lt;/p&gt;'><sup>182</sup></a></span></p>
<p>The MIKK Panel was created to map genetic variants associated with quantitative traits at a high resolution, and to explore the interactions between those variants and any environmental variables of interest. In <strong>Chapter <a href="MIKK-genomes-chap.html#MIKK-genomes-chap">2</a></strong> of this thesis,<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;The contents of this Chapter &lt;a href="MIKK-genomes-chap.html#MIKK-genomes-chap"&gt;2&lt;/a&gt; have been published in &lt;span class="citation"&gt;Fitzgerald et al.&lt;span&gt;&lt;/span&gt;&lt;/span&gt; and &lt;span class="citation"&gt;Adrien Leger et al., &lt;span&gt;“Genomic Variations and Epigenomic Landscape of the &lt;span&gt;Medaka Inbred Kiyosu-Karlsruhe&lt;/span&gt; (&lt;span&gt;MIKK&lt;/span&gt;) Panel,”&lt;/span&gt; &lt;em&gt;Genome Biology&lt;/em&gt; 23, no. 1 (February 21, 2022): 58, &lt;a href="https://doi.org/10.1186/s13059-022-02602-4" role="doc-biblioref"&gt;https://doi.org/10.1186/s13059-022-02602-4&lt;/a&gt;&lt;/span&gt;.&lt;/p&gt;'><sup>183</sup></a> I describe several genomic characteristics of the MIKK panel that are relevant to its utility for genetic mapping of complex traits, including:</p>
<ol style="list-style-type: decimal">
<li><p>The inbreeding trajectory of the MIKK panel (section <a href="MIKK-genomes-chap.html#inbreeding-sec">2.2.2</a>);</p></li>
<li><p>The evolutionary history of the MIKK panel’s founding population (section <a href="MIKK-genomes-chap.html#introgression-sec">2.2.3</a>);</p></li>
<li><p>Levels of homozygosity across the MIKK panel (section <a href="MIKK-genomes-chap.html#nuc-div">2.2.4</a>);</p></li>
<li><p>The allele-frequency distribution and rate of linkage disequilibrium (LD) decay (section <a href="MIKK-genomes-chap.html#ld-decay-sec">2.2.5</a>); and</p></li>
<li><p>The structural variants present in a smaller sample of lines using Oxford Nanopore long-read sequencing data (Chapter <a href="MIKK-genomes-chap.html#mikksv-sec">2.3</a>).</p></li>
</ol>
<p>The fundamental utility of the MIKK panel arises from: a) its representation of the genetic variation that exists in a wild population; b) the ability to replicate individuals with near-identical genotypes, allowing one to modify environmental variables while keeping the genetics constant to determine the strength of those environmental effects; and c) the presence of numerous inbred lines that can by placed in the same environment, allowing one to infer the strength of the genetic effect. The overarching purpose of the MIKK panel is to provide a resource for researchers who seek to combine the traditional F2 cross technique used by Mendel with modern genetic sequencing and statistical methods, in order to identify genetic variants associated with complex traits. I describe this process below.</p>
</div>
<div id="the-f2-cross" class="section level3" number="1.2.5">
<h3>
<span class="header-section-number">1.2.5</span> The F2 cross<a class="anchor" aria-label="anchor" href="#the-f2-cross"><i class="fas fa-link"></i></a>
</h3>
<p>The F2 cross is the same method used by Mendel in his historic paper, but here applied to identifying genetic variants associated with traits of interest. A schema for the method is presented in <strong>Figure <a href="Introduction.html#fig:F2-cross-schema">1.21</a></strong>. In essence, it involves starting with two inbred strains that diverge for the trait of interest (the ‘parental strains’, or F0 generation F0). One then crosses the parental strains to create a generation of F1 hybrid individuals who each possess, for every pair of their chromosomes, one chromosome from each of their parents. The individuals in this F1 generation are genetically identical to their parents with respect to their germ line. Finally, one inter-crosses the F1 generation to create a set of F2 individuals that share unique combinations of the original F0 strains’ genotypes, and tend to display values for the trait of interest that span across the spectrum between the extreme values of their parents.</p>

<div class="figure">
<span style="display:block;" id="fig:F2-cross-schema"></span>
<img src="figs/mikk_behaviour/F2-cross-schema.png" alt="Schema of the F2-cross experimental setup. The F0 generation comprises two medaka strains that have extreme, opposing values for a trait of interest, represented by the colours red and blue. Below them is an illustrative single pair of chromosomes. The chromosomes within each pair are depicted as the same colour, as the strains are homozygous through successive generations of inbreeding. Their F1 offspring is heterozygous for each pair of their 24 chromosomes, and all F1 individuals are therefore almost genetically identical to one another (with the exception of somatic mutations and the regions of the genome that were not homozygous in the parental generations). The F1 individuals are then inter-crossed with one another to produce the F2 generation, which, due to recombination events during gamete formation, have unique combinations of the parental strains’ genotypes, and tend to span the phenotypic spectrum between the extremes of their F0 parental strains, represented by their colours." width="100%"><p class="caption">
Figure 1.21: Schema of the F2-cross experimental setup. The F0 generation comprises two medaka strains that have extreme, opposing values for a trait of interest, represented by the colours red and blue. Below them is an illustrative single pair of chromosomes. The chromosomes within each pair are depicted as the same colour, as the strains are homozygous through successive generations of inbreeding. Their F1 offspring is heterozygous for each pair of their 24 chromosomes, and all F1 individuals are therefore almost genetically identical to one another (with the exception of somatic mutations and the regions of the genome that were not homozygous in the parental generations). The F1 individuals are then inter-crossed with one another to produce the F2 generation, which, due to recombination events during gamete formation, have unique combinations of the parental strains’ genotypes, and tend to span the phenotypic spectrum between the extremes of their F0 parental strains, represented by their colours.
</p>
</div>
<p>By combining this traditional crossing method with modern sequencing technologies, researchers are not only able to map genetic variants associated with a phenotype of interest at a higher resolution than in humans, but also functionally validate those variants by carrying out targeted gene editing in an embryo to confirm that the developed organism shows the expected change in phenotype. Many studies have already made use of this combined method (CITE), known as a “Genome-Wide Linkage Analysis”.</p>
</div>
</div>
<div id="studied-phenotypes" class="section level2" number="1.3">
<h2>
<span class="header-section-number">1.3</span> Studied phenotypes<a class="anchor" aria-label="anchor" href="#studied-phenotypes"><i class="fas fa-link"></i></a>
</h2>
<p>In this thesis I apply the Genome-Wide Linkage Analysis method to identifying the genetic variants associated with two traits: the periodicity of somite development (<strong>Chapter <a href="Somite-chap.html#Somite-chap">4</a></strong>), and bold-type behaviours (<strong>Chapter <a href="MIKK-F2-chap.html#MIKK-F2-chap">5</a></strong>). I discuss each of these phenotypes in turn.</p>
<div id="genetic-control-of-somite-period-development" class="section level3" number="1.3.1">
<h3>
<span class="header-section-number">1.3.1</span> Genetic control of somite period development<a class="anchor" aria-label="anchor" href="#genetic-control-of-somite-period-development"><i class="fas fa-link"></i></a>
</h3>
<p>During the development of an embryo, somites are the earliest primitive segmental structures that form from presomatic mesoderm cells (<strong>PSM</strong>).<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Woong Kim et al., &lt;span&gt;“The Period of the Somite Segmentation Clock Is Sensitive to &lt;span&gt;Notch&lt;/span&gt; Activity,”&lt;/span&gt; &lt;em&gt;Molecular Biology of the Cell&lt;/em&gt; 22, no. 18 (September 15, 2011): 3541–49, &lt;a href="https://doi.org/10.1091/mbc.e11-02-0139" role="doc-biblioref"&gt;https://doi.org/10.1091/mbc.e11-02-0139&lt;/a&gt;.&lt;/p&gt;'><sup>184</sup></a></span> They later differentiate into vertebrae, ribs, and skeletal muscles, thereby establishing the body’s anterior-posterior axis. <strong>Figure <a href="Introduction.html#fig:mouse-embryo">1.22</a></strong> depicts a number of formed somites in a 9.5-day-old mouse embryo.
Image of a mouse embryo at day 9.5 from <span class="citation">Gridley<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;&lt;span&gt;“The Long and Short of It.”&lt;/span&gt;&lt;/p&gt;"><sup>314</sup></a></span>, showing somites in darker colours. Image of a mouse embryo at day 9.5 from <span class="citation">Thomas Gridley<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;&lt;span&gt;“The Long and Short of It: &lt;span&gt;Somite&lt;/span&gt; Formation in Mice,”&lt;/span&gt; &lt;em&gt;Developmental Dynamics&lt;/em&gt; 235, no. 9 (2006): 2330–36, &lt;a href="https://doi.org/10.1002/dvdy.20850" role="doc-biblioref"&gt;https://doi.org/10.1002/dvdy.20850&lt;/a&gt;.&lt;/p&gt;'><sup>185</sup></a></span>, showing somites in darker colours.</p>
<div class="figure" style="text-align: right">
<span style="display:block;" id="fig:mouse-embryo"></span>
<img src="figs/somites/mouse_embryo_gridley.png" alt="Image of a mouse embryo at day 9.5 from Gridley314, showing somites in darker colours." width="100%"><p class="caption">
Figure 1.22: Image of a mouse embryo at day 9.5 from <span class="citation">Gridley<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;&lt;span&gt;“The Long and Short of It.”&lt;/span&gt;&lt;/p&gt;"><sup>314</sup></a></span>, showing somites in darker colours.
</p>
</div>
<p>Somite formation occurs rhythmically and sequentially, with the time between the formation of each pair of somites referred to as the “period”. The period of somite formation varies greatly between species: ~30 minutes for zebrafish, 90 minutes for chickens, 2-3 hours for mice, and 5-6 hours for humans.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Alexis Hubaud and Olivier Pourqui’e, &lt;span&gt;“Signalling Dynamics in Vertebrate Segmentation,”&lt;/span&gt; &lt;em&gt;Nature Reviews Molecular Cell Biology&lt;/em&gt; 15, no. 11, 11 (November 2014): 709–21, &lt;a href="https://doi.org/10.1038/nrm3891" role="doc-biblioref"&gt;https://doi.org/10.1038/nrm3891&lt;/a&gt;; Mitsuhiro Matsuda et al., &lt;span&gt;“Species-Specific Segmentation Clock Periods Are Due to Differential Biochemical Reaction Speeds,”&lt;/span&gt; &lt;em&gt;Science&lt;/em&gt; 369, no. 6510 (2020): 1450–55.&lt;/p&gt;'><sup>186</sup></a></span> <strong>Figure <a href="Introduction.html#fig:somite-seg-ali">1.23</a></strong> shows the a series of time-stamped images of somite segmentation in medaka fish, generated by my collaborator, Ali Seleit.</p>

<div class="figure" style="text-align: left">
<span style="display:block;" id="fig:somite-seg-ali"></span>
<img src="figs/somites/ali_fish_seg_compiled.png" alt="Time-stamped images of somite segmentation in medaka, generated by Ali Seleit." width="100%"><p class="caption">
Figure 1.23: Time-stamped images of somite segmentation in medaka, generated by Ali Seleit.
</p>
</div>
<p>The period of somite formation is controlled by a molecular oscillator, known as the ‘segmentation clock’, which drives waves of gene expression in the Notch, fibroblast growth factor (FGF), and Wnt pathways, forming a signalling gradient that regresses towards the tail in concert with axis elongation.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;C’eline Gomez et al., &lt;span&gt;“Control of Segment Number in Vertebrate Embryos,”&lt;/span&gt; &lt;em&gt;Nature&lt;/em&gt; 454, no. 7202, 7202 (July 2008): 335–39, &lt;a href="https://doi.org/10.1038/nature07020" role="doc-biblioref"&gt;https://doi.org/10.1038/nature07020&lt;/a&gt;.&lt;/p&gt;'><sup>187</sup></a></span> Over the course of elongation, the wave period increases (i.e. each somite takes longer to form), and the PSM progressively shrinks until it is exhausted, eventually terminating somite formation.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Gomez et al.&lt;/p&gt;"><sup>188</sup></a></span></p>
<p>It is not fully understood how the phase waves of the segmentation clock are initially established.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Henning J Falk et al., &lt;span&gt;“Imaging the Onset of Oscillatory Signaling Dynamics During Mouse Embryo Gastrulation,”&lt;/span&gt; &lt;em&gt;Development (Cambridge, England)&lt;/em&gt; 149, no. 13 (July 1, 2022): dev200083, &lt;a href="https://doi.org/10.1242/dev.200083" role="doc-biblioref"&gt;https://doi.org/10.1242/dev.200083&lt;/a&gt;.&lt;/p&gt;'><sup>189</sup></a></span> <span class="citation">Matsuda et al.<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;&lt;span&gt;“Species-Specific Segmentation Clock Periods Are Due to Differential Biochemical Reaction Speeds.”&lt;/span&gt;&lt;/p&gt;"><sup>190</sup></a></span> found that period differences between mouse and human occur at the single-cell level (i.e. not due to intercellular communication), and are driven by biochemical reaction speeds - specifically, mRNA and protein degradation rates, transcription and translation delays, and intron and splicing delays. To identify the genetic basis of these biochemical differences, our collaborators Ali Seleit and Alexander Aulehla at EMBL-Heidelberg used a CRISPR-Case9 knock-in approach<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Ali Seleit, Alexander Aulehla, and Alexandre Paix, &lt;span&gt;“Endogenous Protein Tagging in Medaka Using a Simplified &lt;span&gt;CRISPR&lt;/span&gt;/&lt;span&gt;Cas9&lt;/span&gt; Knock-in Approach.”&lt;/span&gt; &lt;em&gt;eLife&lt;/em&gt; 10 (December 1, 2021): e75050, &lt;a href="https://doi.org/10.7554/elife.75050" role="doc-biblioref"&gt;https://doi.org/10.7554/elife.75050&lt;/a&gt;.&lt;/p&gt;'><sup>191</sup></a></span> to establish a medaka <em>Cab</em> strain with an endogenous, fluorescing reporter gene (Her7-Venus) for the oscillation signalling pathway. This method allows them to image somite formation and extract quantitative measures for segmentation clock dynamics.</p>
<p>In medaka, it is known that the southern Japanese <em>Cab</em> strain and the northern Japanese <em>Kaga</em> strain have divergent somite periodicity, where <em>Kaga</em>’s tends to be faster, and <em>Cab</em>’s slower (<strong>Figure <a href="Introduction.html#fig:F0-Cab-Kaga-HdrR">1.24</a></strong>).</p>

<div class="figure" style="text-align: right">
<span style="display:block;" id="fig:F0-Cab-Kaga-HdrR"></span>
<img src="figs/somites/ali_period_F0_Cab_Kaga.png" alt="Comparison of period for three inbred medaka strains (Cab, Kaga and HdrR). Kaga’s period is lower, and therefore it takes less time to form each somite than Cab. Figure generated by Ali Seleit." width="100%"><p class="caption">
Figure 1.24: Comparison of period for three inbred medaka strains (<em>Cab</em>, <em>Kaga</em> and <em>HdrR</em>). Kaga’s period is lower, and therefore it takes less time to form each somite than <em>Cab</em>. Figure generated by Ali Seleit.
</p>
</div>
<p>Our collaborators accordingly set up a one-way F2 cross experiment as described in Chapter <a href="#MIKK-F2-cross"><strong>??</strong></a>, using the reporter-carrying <em>Cab</em> strain and the <em>Kaga</em> strain as the parental F0 strains, in order to identify genetic loci associated with these differences in clock dynamics. They inter-crossed the hybrid F1 generation to create a sample of 622 F2 individuals, imaged the developing embryos of these F2 samples, and used pyBOAT<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Christoph Schmal, Gregor Mönke, and Adri’an E. Granada, &lt;span&gt;“Analysis of &lt;span&gt;Complex Circadian Time Series Data Using Wavelets&lt;/span&gt;,”&lt;/span&gt; in &lt;em&gt;Circadian &lt;span&gt;Regulation&lt;/span&gt;: &lt;span&gt;Methods&lt;/span&gt; and &lt;span&gt;Protocols&lt;/span&gt;&lt;/em&gt;, ed. Guiomar Solanas and Patrick -Simon Welz, Methods in &lt;span&gt;Molecular Biology&lt;/span&gt; (&lt;span&gt;New York, NY&lt;/span&gt;: &lt;span&gt;Springer US&lt;/span&gt;, 2022), 35–54, &lt;a href="https://doi.org/10.1007/978-1-0716-2249-0_3" role="doc-biblioref"&gt;https://doi.org/10.1007/978-1-0716-2249-0_3&lt;/a&gt;.&lt;/p&gt;'><sup>192</sup></a></span> to extract the oscillation features during somite development. <strong>Figure <a href="Introduction.html#fig:somite-period-ali">1.25</a></strong> shows a series of raw images used by pyBOAT to track the elongation of a medaka tail during somitogenesis, with the identified posterior tip of the embryo labelled with a blue circle.</p>

<div class="figure" style="text-align: left">
<span style="display:block;" id="fig:somite-period-ali"></span>
<img src="figs/somites/ali_compiled_somite_elong.png" alt="Screenshots of vertebral elongation in an F2 individual captured by Ali Seleit during imaging. The blue circle represents the point tracked by pyBOAT over time, generating the quantitative phenotype data on period development used in this study." width="100%"><p class="caption">
Figure 1.25: Screenshots of vertebral elongation in an F2 individual captured by Ali Seleit during imaging. The blue circle represents the point tracked by pyBOAT over time, generating the quantitative phenotype data on period development used in this study.
</p>
</div>
</div>
<div id="behaviour" class="section level3" number="1.3.2">
<h3>
<span class="header-section-number">1.3.2</span> Behaviour<a class="anchor" aria-label="anchor" href="#behaviour"><i class="fas fa-link"></i></a>
</h3>
<p>In <strong>Chapters <a href="Pilot-chap.html#Pilot-chap">3</a></strong> and <strong><a href="MIKK-F2-chap.html#MIKK-F2-chap">5</a></strong> I use medaka as a model organism to explore social genetic effects on bold-type behaviours. Behaviour is a complex trait that is affected by both genes and environment, and for social animals such as humans and medaka, one’s social environment is considered likely to constitute a large component of the environmental effect.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;D. E. Ruzzante and R. W. Doyle, &lt;span&gt;“Behavioural and Growth Responses to the Intensity of Intraspecific Social Interaction Among Medaka, &lt;span&gt;Oryzias&lt;/span&gt; Latipes (&lt;span&gt;Temminck&lt;/span&gt; and &lt;span&gt;Schlegel&lt;/span&gt;) (&lt;span&gt;Pisces&lt;/span&gt;, &lt;span&gt;Cyprinodontidae&lt;/span&gt;),”&lt;/span&gt; &lt;em&gt;Journal of Fish Biology&lt;/em&gt; 37, no. 5 (1990): 663–73, &lt;a href="https://doi.org/10.1111/j.1095-8649.1990.tb02531.x" role="doc-biblioref"&gt;https://doi.org/10.1111/j.1095-8649.1990.tb02531.x&lt;/a&gt;; Simon N. Young, &lt;span&gt;“The Neurobiology of Human Social Behaviour: An Important but Neglected Topic,”&lt;/span&gt; &lt;em&gt;Journal of Psychiatry and Neuroscience&lt;/em&gt; 33, no. 5 (September 1, 2008): 391–92, &lt;a href="https://www.jpn.ca/content/33/5/391" role="doc-biblioref"&gt;https://www.jpn.ca/content/33/5/391&lt;/a&gt;.&lt;/p&gt;'><sup>193</sup></a></span> Apart from social aspects, an organism must face many “hostile forces of nature” throughout its life,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;David M. Buss, &lt;span&gt;“Evolutionary Personality Psychology,”&lt;/span&gt; &lt;em&gt;Annual Review of Psychology&lt;/em&gt; 42 (1991): 459–91, &lt;a href="https://doi.org/10.1146/annurev.ps.42.020191.002331" role="doc-biblioref"&gt;https://doi.org/10.1146/annurev.ps.42.020191.002331&lt;/a&gt;; Darwin, &lt;em&gt;On the &lt;span&gt;Origin&lt;/span&gt; of &lt;span&gt;Species&lt;/span&gt; by &lt;span&gt;Means&lt;/span&gt; of &lt;span&gt;Natural Selection&lt;/span&gt;, &lt;span&gt;Or&lt;/span&gt;, &lt;span&gt;The Preservation&lt;/span&gt; of &lt;span&gt;Favoured Races&lt;/span&gt; in the &lt;span&gt;Struggle&lt;/span&gt; for &lt;span&gt;Life&lt;/span&gt;&lt;/em&gt;.&lt;/p&gt;'><sup>194</sup></a></span> such as food shortages, predation, harsh climate, and diseases. Adaptive behaviours allow individuals to navigate such dangers and maximise the likelihood of their survival at both the individual and population level.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Steven L. Lima and Lawrence M. Dill, &lt;span&gt;“Behavioral Decisions Made Under the Risk of Predation: A Review and Prospectus,”&lt;/span&gt; &lt;em&gt;Canadian Journal of Zoology&lt;/em&gt; 68, no. 4 (April 1990): 619–40, &lt;a href="https://doi.org/10.1139/z90-092" role="doc-biblioref"&gt;https://doi.org/10.1139/z90-092&lt;/a&gt;.&lt;/p&gt;'><sup>195</sup></a></span></p>
<p>Boldness-shyness is thought to be a fundamental axis of behavioural variation in many species, with an obvious causal relationship to an individual’s likelihood of survival, and consequently with natural selection at the population level.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;David Sloan Wilson et al., &lt;span&gt;“Shyness and Boldness in Humans and Other Animals,”&lt;/span&gt; &lt;em&gt;Trends in Ecology &amp;amp; Evolution&lt;/em&gt; 9, no. 11 (November 1, 1994): 442–46, &lt;a href="https://doi.org/10.1016/0169-5347(94)90134-1" role="doc-biblioref"&gt;https://doi.org/10.1016/0169-5347(94)90134-1&lt;/a&gt;.&lt;/p&gt;'><sup>196</sup></a></span> It represents an evolutionary trade-off between acquiring benefits (in terms of food or mates) and avoiding harms (in terms of predators or conspecific competitors), with each situation accompanied by its own optimal degree of risk.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Lima and Dill, &lt;span&gt;“Behavioral Decisions Made Under the Risk of Predation.”&lt;/span&gt;&lt;/p&gt;"><sup>197</sup></a></span> It is both heritable,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Kenth Svartberg, &lt;span&gt;“Shyness–Boldness Predicts Performance in Working Dogs,”&lt;/span&gt; &lt;em&gt;Applied Animal Behaviour Science&lt;/em&gt; 79, no. 2 (October 20, 2002): 157–74, &lt;a href="https://doi.org/10.1016/S0168-1591(02)00120-X" role="doc-biblioref"&gt;https://doi.org/10.1016/S0168-1591(02)00120-X&lt;/a&gt;; Culum Brown, Fiona Burgess, and Victoria A. Braithwaite, &lt;span&gt;“Heritable and Experiential Effects on Boldness in a Tropical Poeciliid,”&lt;/span&gt; &lt;em&gt;Behavioral Ecology and Sociobiology&lt;/em&gt; 62, no. 2 (December 1, 2007): 237–43, &lt;a href="https://doi.org/10.1007/s00265-007-0458-3" role="doc-biblioref"&gt;https://doi.org/10.1007/s00265-007-0458-3&lt;/a&gt;.&lt;/p&gt;'><sup>198</sup></a></span> and subject to change following different life experiences or under different environmental conditions.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Brown, Burgess, and Braithwaite, &lt;span&gt;“Heritable and Experiential Effects on Boldness in a Tropical Poeciliid.”&lt;/span&gt;&lt;/p&gt;"><sup>199</sup></a></span></p>
<p>[OPEN FIELD AND NOVEL OBJECT IS A GENERIC PARADIGM, NOT JUST FISH. CITE BAUD.]</p>
<p>Boldness-shyness has been studied extensively with fish. Shy individuals tend to react to novelty by reducing their activity and becoming more vigilant, whereas bold individuals show higher levels of activity and exploratory behaviour.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;C. Brown, F. Jones, and V. A. Braithwaite, &lt;span&gt;“Correlation Between Boldness and Body Mass in Natural Populations of the Poeciliid &lt;span&gt;Brachyrhaphis&lt;/span&gt; Episcopi,”&lt;/span&gt; &lt;em&gt;Journal of Fish Biology&lt;/em&gt; 71, no. 6 (2007): 1590–1601, &lt;a href="https://doi.org/10.1111/j.1095-8649.2007.01627.x" role="doc-biblioref"&gt;https://doi.org/10.1111/j.1095-8649.2007.01627.x&lt;/a&gt;.&lt;/p&gt;'><sup>200</sup></a></span> One assay commonly used to measure this behavioural domain is referred to as the ‘open field’ assay, where fishes are observed while swimming freely in an experimental setting.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Brown, Jones, and Braithwaite; Kevin Laland, Jens Krause, and Culum Brown, &lt;em&gt;Fish &lt;span&gt;Cognition&lt;/span&gt; and &lt;span&gt;Behavior&lt;/span&gt;&lt;/em&gt; (&lt;span&gt;John Wiley &amp;amp; Sons&lt;/span&gt;, 2011), &lt;a href="https://books.google.com?id=cI9gbVyH6lsC" role="doc-biblioref"&gt;https://books.google.com?id=cI9gbVyH6lsC&lt;/a&gt;; Tyrone Lucon-Xiccato et al., &lt;span&gt;“Comparison of Anxiety-Like and Social Behaviour in Medaka and Zebrafish,”&lt;/span&gt; &lt;em&gt;Scientific Reports&lt;/em&gt; 12, no. 1, 1 (June 28, 2022): 10926, &lt;a href="https://doi.org/10.1038/s41598-022-14978-1" role="doc-biblioref"&gt;https://doi.org/10.1038/s41598-022-14978-1&lt;/a&gt;; Tyrone Lucon-Xiccato et al., &lt;span&gt;“Development of &lt;span&gt;Open-Field Behaviour&lt;/span&gt; in the &lt;span&gt;Medaka&lt;/span&gt;, &lt;span&gt;Oryzias&lt;/span&gt; Latipes,”&lt;/span&gt; &lt;em&gt;Biology&lt;/em&gt; 9, no. 11, 11 (November 2020): 389, &lt;a href="https://doi.org/10.3390/biology9110389" role="doc-biblioref"&gt;https://doi.org/10.3390/biology9110389&lt;/a&gt;; Tyrone Lucon-Xiccato and Angelo Bisazza, &lt;span&gt;“Individual Differences in Cognition Among Teleost Fishes,”&lt;/span&gt; &lt;em&gt;Behavioural Processes&lt;/em&gt;, The &lt;span&gt;Cognition&lt;/span&gt; of &lt;span&gt;Fish&lt;/span&gt;, 141 (August 1, 2017): 184–95, &lt;a href="https://doi.org/10.1016/j.beproc.2017.01.015" role="doc-biblioref"&gt;https://doi.org/10.1016/j.beproc.2017.01.015&lt;/a&gt;; Wataru Matsunaga and Eiji Watanabe, &lt;span&gt;“Habituation of Medaka (&lt;span&gt;Oryzias&lt;/span&gt; Latipes) Demonstrated by Open-Field Testing,”&lt;/span&gt; &lt;em&gt;Behavioural Processes&lt;/em&gt; 85, no. 2 (October 1, 2010): 142–50, &lt;a href="https://doi.org/10.1016/j.beproc.2010.06.019" role="doc-biblioref"&gt;https://doi.org/10.1016/j.beproc.2010.06.019&lt;/a&gt;.&lt;/p&gt;'><sup>201</sup></a></span></p>
<p>Another is the ‘novel object’ assay, where a novel object is introduced to the fishes’ environment to simulate a threat.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Brown, Jones, and Braithwaite, &lt;span&gt;“Correlation Between Boldness and Body Mass in Natural Populations of the Poeciliid &lt;span&gt;Brachyrhaphis&lt;/span&gt; Episcopi”&lt;/span&gt;; Joachim Schjolden, Argaudas Stoskhus, and Svante Winberg, &lt;span&gt;“Does &lt;span&gt;Individual Variation&lt;/span&gt; in &lt;span&gt;Stress Responses&lt;/span&gt; and &lt;span&gt;Agonistic Behavior Reflect Divergent Stress Coping Strategies&lt;/span&gt; in &lt;span&gt;Juvenile Rainbow Trout&lt;/span&gt;?”&lt;/span&gt; &lt;em&gt;Physiological and Biochemical Zoology&lt;/em&gt; 78, no. 5 (September 2005): 715–23, &lt;a href="https://doi.org/10.1086/432153" role="doc-biblioref"&gt;https://doi.org/10.1086/432153&lt;/a&gt;; David S. Wilson et al., &lt;span&gt;“Shy-Bold Continuum in Pumpkinseed Sunfish (&lt;span&gt;Lepomis&lt;/span&gt; Gibbosus): &lt;span&gt;An&lt;/span&gt; Ecological Study of a Psychological Trait,”&lt;/span&gt; &lt;em&gt;Journal of Comparative Psychology&lt;/em&gt; 107, no. 3 (1993): 250–60, &lt;a href="https://doi.org/10.1037/0735-7036.107.3.250" role="doc-biblioref"&gt;https://doi.org/10.1037/0735-7036.107.3.250&lt;/a&gt;; Dominic Wright, Roger K. Butlin, and Örjan Carlborg, &lt;span&gt;“Epistatic Regulation of Behavioural and Morphological Traits in the Zebrafish (&lt;span&gt;Danio&lt;/span&gt; Rerio),”&lt;/span&gt; &lt;em&gt;Behavior Genetics&lt;/em&gt; 36, no. 6 (November 1, 2006): 914–22, &lt;a href="https://doi.org/10.1007/s10519-006-9080-9" role="doc-biblioref"&gt;https://doi.org/10.1007/s10519-006-9080-9&lt;/a&gt;; D. Wright et al., &lt;span&gt;“Inter and Intra-Population Variation in Shoaling and Boldness in the Zebrafish (&lt;span&gt;Danio&lt;/span&gt; Rerio),”&lt;/span&gt; &lt;em&gt;Journal of Fish Biology&lt;/em&gt; 63, no. s1 (2003): 258–59, &lt;a href="https://doi.org/10.1111/j.1095-8649.2003.216bw.x" role="doc-biblioref"&gt;https://doi.org/10.1111/j.1095-8649.2003.216bw.x&lt;/a&gt;.&lt;/p&gt;'><sup>202</sup></a></span> Where both assays were performed on the same fish, the behaviours exhibited were found to be correlated across assays, indicating that both were measuring the same boldness-shyness axis.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Brown, Jones, and Braithwaite, &lt;span&gt;“Correlation Between Boldness and Body Mass in Natural Populations of the Poeciliid &lt;span&gt;Brachyrhaphis&lt;/span&gt; Episcopi.”&lt;/span&gt;&lt;/p&gt;"><sup>203</sup></a></span> Both the open field and novel object assay also permit the measurement of habituation, where the response of the fish may change over time after growing accustomed to the new environment or object. Variation in the length of time the fish take to return to a more “normal” level of movement may indicate different levels of boldness.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Laland, Krause, and Brown, &lt;em&gt;Fish &lt;span&gt;Cognition&lt;/span&gt; and &lt;span&gt;Behavior&lt;/span&gt;&lt;/em&gt;.&lt;/p&gt;"><sup>204</sup></a></span></p>
<div id="social-genetic-effects" class="section level4" number="1.3.2.1">
<h4>
<span class="header-section-number">1.3.2.1</span> Social genetic effects<a class="anchor" aria-label="anchor" href="#social-genetic-effects"><i class="fas fa-link"></i></a>
</h4>
<p>The experiments described in <strong>Chapters <a href="Pilot-chap.html#Pilot-chap">3</a></strong> and <strong><a href="MIKK-F2-chap.html#MIKK-F2-chap">5</a></strong> are designed to simulataneously measure both the direct effect of an individual’s genes on their behaviour, as we as the indirect effect of their social partner’s genes on the focal individual’s behaviour, which is also an environmental factor.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Amelie Baud et al., &lt;span&gt;“Genetic &lt;span&gt;Variation&lt;/span&gt; in the &lt;span&gt;Social Environment Contributes&lt;/span&gt; to &lt;span&gt;Health&lt;/span&gt; and &lt;span&gt;Disease&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;PLOS Genetics&lt;/em&gt; 13, no. 1 (January 25, 2017): e1006498, &lt;a href="https://doi.org/10.1371/journal.pgen.1006498" role="doc-biblioref"&gt;https://doi.org/10.1371/journal.pgen.1006498&lt;/a&gt;.&lt;/p&gt;'><sup>205</sup></a></span> Social genetic effects have been shown to exert influence on various traits in mice including anxiety, wound healing, immune function, and body weight,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Baud et al.&lt;/p&gt;"><sup>206</sup></a></span> and various traits related to development and survival in many species of livestock.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Esther D. Ellen et al., &lt;span&gt;“The Prospects of Selection for Social Genetic Effects to Improve Welfare and Productivity in Livestock,”&lt;/span&gt; &lt;em&gt;Frontiers in Genetics&lt;/em&gt; 5 (2014), &lt;a href="https://www.frontiersin.org/articles/10.3389/fgene.2014.00377" role="doc-biblioref"&gt;https://www.frontiersin.org/articles/10.3389/fgene.2014.00377&lt;/a&gt;.&lt;/p&gt;'><sup>207</sup></a></span> However, in those studies the social interactions were maintained throughout development, and it is unclear whether social genetic effects can still exert influence on adaptive behaviours during discrete, time-limited interactions.</p>
<p>In <strong>Chapter <a href="Pilot-chap.html#Pilot-chap">3</a></strong> I used a combined open-field and novel-object assay involving assaying pairs for fish, to determine: a) whether there were consistent differences in bold-shy behaviours exhibited by 5 established inbred strains of medaka fish (<em>iCab</em>, <em>HdrR</em>, and <em>HO5</em> from southern Japan, and <em>Kaga</em> and <em>HNI</em> from northern Japan); and b) whether there were consistent differences in bold-shy behaviours exhibited by a given strain (<em>iCab</em>) dependent on the strain that it was partnered with. The former was intended to measure the effect of an individual’s own genes on its behaviour (a direct genetic effect), and the latter to measure the effect of the genes of the focal fish’s tank partner on the behaviour of the focal fish (social genetic effect). Our experimental design allowed us to assess both direct and indirect effects on behaviour simultaneously, and to infer the degree to which variation in bold-shy behaviours is attributable to the differences in an individual’s own genetics, the differences in the genetics of their social companions, and stochastic variation. A schema for the experimental plan is shown in <strong>Figure <a href="Introduction.html#fig:behavioural-schema">1.26</a></strong>.</p>

<div class="figure">
<span style="display:block;" id="fig:behavioural-schema"></span>
<img src="figs/pilot/experimental_schema.png" alt="Schema for experimental plan. iCab-iCab pairings are the control condition. To explore direct genetic effects on behaviour, we compare the behaviours of test fishes from different lines, and infer that the differences between lines are caused by the differences in their genetics (direct genetic effects). To explore social genetic effects, we use the same data but turn our focus to the reference fish, and infer that the differences we observe between their behaviours are caused by the differences in their social environments, which are in turn driven by the different genetics in the test fish lines (social genetic effects)." width="100%"><p class="caption">
Figure 1.26: Schema for experimental plan. <em>iCab</em>-<em>iCab</em> pairings are the control condition. To explore direct genetic effects on behaviour, we compare the behaviours of test fishes from different lines, and infer that the differences between lines are caused by the differences in their genetics (direct genetic effects). To explore social genetic effects, we use the same data but turn our focus to the reference fish, and infer that the differences we observe between their behaviours are caused by the differences in their social environments, which are in turn driven by the different genetics in the test fish lines (social genetic effects).
</p>
</div>
<p>In <strong>Chapter <a href="MIKK-F2-chap.html#MIKK-F2-chap">5</a></strong> I extend the behavioural analysis described in Chapter <a href="Pilot-chap.html#Pilot-chap">3</a> over the MIKK panel. I then identify the lines that diverged in both (a) their own behaviour; and (b) the level of transmission of their behaviour onto their <em><span style="color: #424B4D;">iCab</span></em> reference tank partner, and then use them as the parental strains in an F2 cross to attempt to identify the genetic variants associated with those differences.</p>
</div>
</div>
</div>
<div id="poor-transferability-of-polygenic-scores-to-diverse-human-populations" class="section level2" number="1.4">
<h2>
<span class="header-section-number">1.4</span> Poor transferability of polygenic scores to diverse human populations<a class="anchor" aria-label="anchor" href="#poor-transferability-of-polygenic-scores-to-diverse-human-populations"><i class="fas fa-link"></i></a>
</h2>
<p>[CITE MIKE INOUYE]</p>
<p><strong>Chapter <a href="Fst-chap.html#Fst-chap">6</a></strong> – the final chapter of this thesis – relates to the poor transferability of polygenic scores (<strong>PRS</strong>) derived from GWAS results when applied to non-European populations. Humans have long sought to use genetic information to predict an individual’s likely value for a given trait, in our own species and in other organisms. As already discussed, an individual’s phenotypic value at a given point in time is the product of complex interactions between their genome and their environment, beginning from embryonic development and continuing throughout their lifetimes. It is now clear that “complex” traits such as height, intelligence, and behaviour are highly polygenic, meaning that they are genetically influenced by hundreds or thousands of genetic variants, each exerting a small effect in one or the other direction along the trait’s spectrum.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;G. Sella and N. Barton, &lt;span&gt;“Thinking &lt;span&gt;About&lt;/span&gt; the &lt;span&gt;Evolution&lt;/span&gt; of &lt;span&gt;Complex Traits&lt;/span&gt; in the &lt;span&gt;Era&lt;/span&gt; of &lt;span&gt;Genome-Wide Association Studies&lt;/span&gt;.”&lt;/span&gt; &lt;em&gt;Annual Review of Genomics and Human Genetics&lt;/em&gt;, 2019, &lt;a href="https://doi.org/10.1146/annurev-genom-083115-022316" role="doc-biblioref"&gt;https://doi.org/10.1146/annurev-genom-083115-022316&lt;/a&gt;.&lt;/p&gt;'><sup>208</sup></a></span></p>
<p>A richer understanding of the cumulative effect of genetic variants on any trait allows for the prediction of the value that an individual is most likely to have for that trait. Of all human traits, diseases are particularly salient; in 2018, the global healthcare industry was valued at US$8 trillion, and predicted to increase to US$12 trillion by 2022.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;&lt;span&gt;“The $11.9 &lt;span&gt;Trillion Global Healthcare Market&lt;/span&gt;: &lt;span&gt;Key Opportunities&lt;/span&gt; &amp;amp; &lt;span&gt;Strategies&lt;/span&gt; (2014-2022) - &lt;span&gt;ResearchAndMarkets&lt;/span&gt;.com,”&lt;/span&gt; June 25, 2019, &lt;a href="https://www.businesswire.com/news/home/20190625005862/en/The-11.9-Trillion-Global-Healthcare-Market-Key-Opportunities-Strategies-2014-2022---ResearchAndMarkets.com" role="doc-biblioref"&gt;https://www.businesswire.com/news/home/20190625005862/en/The-11.9-Trillion-Global-Healthcare-Market-Key-Opportunities-Strategies-2014-2022---ResearchAndMarkets.com&lt;/a&gt;.&lt;/p&gt;'><sup>209</sup></a></span> This strong financial imperative complements the moral imperative to reduce suffering, together driving the question of how to use genetic information to improve human health.</p>
<p>Recent technological developments have made it possible to sequence human genomes at scale, and it is thought that by combining detailed genetic information with with other environmental and phenotypic information (such as lifestyle or clinical factors), clinicians could move towards the practice of “precision medicine”, where interventions could be tailored to their patients’ unique risk profiles.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Naomi R. Wray, Michael E. Goddard, and Peter M. Visscher, &lt;span&gt;“Prediction of Individual Genetic Risk to Disease from Genome-Wide Association Studies,”&lt;/span&gt; &lt;em&gt;Genome Research&lt;/em&gt; 17, no. 10 (January 10, 2007): 1520–28, &lt;a href="https://doi.org/10.1101/gr.6665407" role="doc-biblioref"&gt;https://doi.org/10.1101/gr.6665407&lt;/a&gt;.&lt;/p&gt;'><sup>210</sup></a></span> The use of genetic information to predict individuals’ values for a trait of interest entails the construction of metrics known as “polygenic scores” (<strong>PGS</strong>). When the trait is a disease, PGS is commonly known as polygenic risk scores (<strong>PRS</strong>), or genetic risk profiling, but I use the term PGS to encompass both disease and non-disease traits.</p>
<div id="polygenic-scores-pgs-and-gwas" class="section level3" number="1.4.1">
<h3>
<span class="header-section-number">1.4.1</span> Polygenic Scores (PGS) and GWAS<a class="anchor" aria-label="anchor" href="#polygenic-scores-pgs-and-gwas"><i class="fas fa-link"></i></a>
</h3>
<div id="pgs" class="section level4" number="1.4.1.1">
<h4>
<span class="header-section-number">1.4.1.1</span> PGS<a class="anchor" aria-label="anchor" href="#pgs"><i class="fas fa-link"></i></a>
</h4>
<p>PGS using genetic information alone show modest yet reliable accuracy for the prediction of complex traits:<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Alicia R. Martin et al., &lt;span&gt;“Clinical Use of Current Polygenic Risk Scores May Exacerbate Health Disparities,”&lt;/span&gt; &lt;em&gt;Nature Genetics&lt;/em&gt; 51, no. 4, 4 (April 2019): 584–91, &lt;a href="https://doi.org/10.1038/s41588-019-0379-x" role="doc-biblioref"&gt;https://doi.org/10.1038/s41588-019-0379-x&lt;/a&gt;.&lt;/p&gt;'><sup>211</sup></a></span> the correlations between PGS and the trait value as measured by <span class="math inline">\(R^2\)</span> have reached 0.24 for height,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Loic Yengo et al., &lt;span&gt;“Meta-Analysis of Genome-Wide Association Studies for Height and Body Mass Index in  700000 Individuals of &lt;span&gt;European&lt;/span&gt; Ancestry,”&lt;/span&gt; &lt;em&gt;Human Molecular Genetics&lt;/em&gt; 27, no. 20 (October 15, 2018): 3641–49, &lt;a href="https://doi.org/10.1093/hmg/ddy271" role="doc-biblioref"&gt;https://doi.org/10.1093/hmg/ddy271&lt;/a&gt;.&lt;/p&gt;'><sup>212</sup></a></span> and 0.12-0.16 for educational attainment.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Aysu Okbay et al., &lt;span&gt;“Polygenic Prediction of Educational Attainment Within and Between Families from Genome-Wide Association Analyses in 3 Million Individuals,”&lt;/span&gt; &lt;em&gt;Nature Genetics&lt;/em&gt;, March 31, 2022, 1–13, &lt;a href="https://doi.org/10.1038/s41588-022-01016-z" role="doc-biblioref"&gt;https://doi.org/10.1038/s41588-022-01016-z&lt;/a&gt;.&lt;/p&gt;'><sup>213</sup></a></span> PGS also improve predictions beyond non-genetic clinical models for a variety of health-related traits, including breast cancer,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Paige Maas et al., &lt;span&gt;“Breast &lt;span&gt;Cancer Risk From Modifiable&lt;/span&gt; and &lt;span&gt;Nonmodifiable Risk Factors Among White Women&lt;/span&gt; in the &lt;span&gt;United States&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;JAMA Oncology&lt;/em&gt; 2, no. 10 (October 1, 2016): 1295–1302, &lt;a href="https://doi.org/10.1001/jamaoncol.2016.1025" role="doc-biblioref"&gt;https://doi.org/10.1001/jamaoncol.2016.1025&lt;/a&gt;.&lt;/p&gt;'><sup>214</sup></a></span> prostate cancer,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Fredrick R. Schumacher et al., &lt;span&gt;“Association Analyses of More Than 140,000 Men Identify 63 New Prostate Cancer Susceptibility Loci,”&lt;/span&gt; &lt;em&gt;Nature Genetics&lt;/em&gt; 50, no. 7, 7 (July 2018): 928–36, &lt;a href="https://doi.org/10.1038/s41588-018-0142-8" role="doc-biblioref"&gt;https://doi.org/10.1038/s41588-018-0142-8&lt;/a&gt;.&lt;/p&gt;'><sup>215</sup></a></span> and type I diabetes.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Seth A. Sharp et al., &lt;span&gt;“Development and &lt;span&gt;Standardization&lt;/span&gt; of an &lt;span&gt;Improved Type&lt;/span&gt; 1 &lt;span&gt;Diabetes Genetic Risk Score&lt;/span&gt; for &lt;span&gt;Use&lt;/span&gt; in &lt;span&gt;Newborn Screening&lt;/span&gt; and &lt;span&gt;Incident Diagnosis&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Diabetes Care&lt;/em&gt; 42, no. 2 (January 11, 2019): 200–207, &lt;a href="https://doi.org/10.2337/dc18-1785" role="doc-biblioref"&gt;https://doi.org/10.2337/dc18-1785&lt;/a&gt;.&lt;/p&gt;'><sup>216</sup></a></span> The predictive accuracy of PGS scores can be further improved by combining genetic information with lifestyle and clinical factors, as seen with cardiovascular disease.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Amit V. Khera et al., &lt;span&gt;“Genome-Wide Polygenic Scores for Common Diseases Identify Individuals with Risk Equivalent to Monogenic Mutations,”&lt;/span&gt; &lt;em&gt;Nature Genetics&lt;/em&gt; 50, no. 9, 9 (September 2018): 1219–24, &lt;a href="https://doi.org/10.1038/s41588-018-0183-z" role="doc-biblioref"&gt;https://doi.org/10.1038/s41588-018-0183-z&lt;/a&gt;; Iftikhar J. Kullo et al., &lt;span&gt;“Incorporating a &lt;span&gt;Genetic Risk Score Into Coronary Heart Disease Risk Estimates&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Circulation&lt;/em&gt; 133, no. 12 (March 22, 2016): 1181–88, &lt;a href="https://doi.org/10.1161/CIRCULATIONAHA.115.020109" role="doc-biblioref"&gt;https://doi.org/10.1161/CIRCULATIONAHA.115.020109&lt;/a&gt;; Pradeep Natarajan et al., &lt;span&gt;“Polygenic &lt;span&gt;Risk Score Identifies Subgroup With Higher Burden&lt;/span&gt; of &lt;span&gt;Atherosclerosis&lt;/span&gt; and &lt;span&gt;Greater Relative Benefit From Statin Therapy&lt;/span&gt; in the &lt;span&gt;Primary Prevention Setting&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Circulation&lt;/em&gt; 135, no. 22 (May 30, 2017): 2091–101, &lt;a href="https://doi.org/10.1161/CIRCULATIONAHA.116.024436" role="doc-biblioref"&gt;https://doi.org/10.1161/CIRCULATIONAHA.116.024436&lt;/a&gt;; Martine Paquette et al., &lt;span&gt;“Polygenic Risk Score Predicts Prevalence of Cardiovascular Disease in Patients with Familial Hypercholesterolemia,”&lt;/span&gt; &lt;em&gt;Journal of Clinical Lipidology&lt;/em&gt; 11, no. 3 (May 1, 2017): 725–732.e5, &lt;a href="https://doi.org/10.1016/j.jacl.2017.03.019" role="doc-biblioref"&gt;https://doi.org/10.1016/j.jacl.2017.03.019&lt;/a&gt;; Emmi Tikkanen et al., &lt;span&gt;“Genetic &lt;span&gt;Risk Prediction&lt;/span&gt; and a 2-&lt;span&gt;Stage Risk Screening Strategy&lt;/span&gt; for &lt;span&gt;Coronary Heart Disease&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Arteriosclerosis, Thrombosis, and Vascular Biology&lt;/em&gt; 33, no. 9 (September 2013): 2261–66, &lt;a href="https://doi.org/10.1161/ATVBAHA.112.301120" role="doc-biblioref"&gt;https://doi.org/10.1161/ATVBAHA.112.301120&lt;/a&gt;; Luanluan Sun et al., &lt;span&gt;“Polygenic Risk Scores in Cardiovascular Risk Prediction: &lt;span&gt;A&lt;/span&gt; Cohort Study and Modelling Analyses,”&lt;/span&gt; &lt;em&gt;PLOS Medicine&lt;/em&gt; 18, no. 1 (January 14, 2021): e1003498, &lt;a href="https://doi.org/10.1371/journal.pmed.1003498" role="doc-biblioref"&gt;https://doi.org/10.1371/journal.pmed.1003498&lt;/a&gt;.&lt;/p&gt;'><sup>217</sup></a></span></p>
</div>
<div id="gwas" class="section level4" number="1.4.1.2">
<h4>
<span class="header-section-number">1.4.1.2</span> GWAS<a class="anchor" aria-label="anchor" href="#gwas"><i class="fas fa-link"></i></a>
</h4>
<p>Most GWAS have been performed with individuals of European ancestry, despite only constituting around 16% of the present global population. Although the proportion of participants in GWAS from a non-European background increased from 4% in 2009 to 16% in 2016),<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Alice B. Popejoy and Stephanie M. Fullerton, &lt;span&gt;“Genomics Is Failing on Diversity,”&lt;/span&gt; &lt;em&gt;Nature&lt;/em&gt; 538, no. 7624, 7624 (October 2016): 161–64, &lt;a href="https://doi.org/10.1038/538161a" role="doc-biblioref"&gt;https://doi.org/10.1038/538161a&lt;/a&gt;.&lt;/p&gt;'><sup>218</sup></a></span> as of 2019, 79% of all GWAS participants recorded in the GWAS Catalog were of European ancestry, and the proportion of non-European individuals has remained the same or reduced since late 2014.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Martin et al., &lt;span&gt;“Clinical Use of Current Polygenic Risk Scores May Exacerbate Health Disparities.”&lt;/span&gt;&lt;/p&gt;"><sup>219</sup></a></span> This bias extends to PGS studies, where as of 2019, only 67% of them included only participants of European ancestry, with another 19% including only East Asian ancestry participants, and only 3.8% with cohorts of African, Hispanic, or Indigenous ancestry.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;L. Duncan et al., &lt;span&gt;“Analysis of Polygenic Risk Score Usage and Performance in Diverse Human Populations,”&lt;/span&gt; &lt;em&gt;Nature Communications&lt;/em&gt; 10, no. 1, 1 (July 25, 2019): 3328, &lt;a href="https://doi.org/10.1038/s41467-019-11112-0" role="doc-biblioref"&gt;https://doi.org/10.1038/s41467-019-11112-0&lt;/a&gt;.&lt;/p&gt;'><sup>220</sup></a></span></p>
<p>It is therefore unsurprising that PGS scores are far better at predicting disease risk in individuals of European ancestry than in those of non-European ancestry.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Alicia R Martin et al., &lt;span&gt;“Human Demographic History Impacts Genetic Risk Prediction Across Diverse Populations,”&lt;/span&gt; &lt;em&gt;The American Journal of Human Genetics&lt;/em&gt; 100, no. 4 (2017): 635–49, &lt;a href="https://doi.org/10.1016/j.ajhg.2017.03.004" role="doc-biblioref"&gt;https://doi.org/10.1016/j.ajhg.2017.03.004&lt;/a&gt;; Martin et al., &lt;span&gt;“Clinical Use of Current Polygenic Risk Scores May Exacerbate Health Disparities.”&lt;/span&gt;&lt;/p&gt;'><sup>221</sup></a></span> Indeed, the predictive accuracy of PGS scores decays with genetic divergence of the GWAS “independent” or “test” sample, from the “discovery” or “training” sample, as established in both humans,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Martin et al., &lt;span&gt;“Human Demographic History Impacts Genetic Risk Prediction Across Diverse Populations”&lt;/span&gt;; Martin et al., &lt;span&gt;“Clinical Use of Current Polygenic Risk Scores May Exacerbate Health Disparities.”&lt;/span&gt;&lt;/p&gt;"><sup>222</sup></a></span> and livestock.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Samuel A. Clark et al., &lt;span&gt;“The Importance of Information on Relatives for the Prediction of Genomic Breeding Values and the Implications for the Makeup of Reference Data Sets in Livestock Breeding Schemes,”&lt;/span&gt; &lt;em&gt;Genetics Selection Evolution&lt;/em&gt; 44, no. 1 (February 9, 2012): 4, &lt;a href="https://doi.org/10.1186/1297-9686-44-4" role="doc-biblioref"&gt;https://doi.org/10.1186/1297-9686-44-4&lt;/a&gt;; David Habier et al., &lt;span&gt;“The Impact of Genetic Relationship Information on Genomic Breeding Values in &lt;span&gt;German Holstein&lt;/span&gt; Cattle,”&lt;/span&gt; &lt;em&gt;Genetics Selection Evolution&lt;/em&gt; 42, no. 1 (February 19, 2010): 5, &lt;a href="https://doi.org/10.1186/1297-9686-42-5" role="doc-biblioref"&gt;https://doi.org/10.1186/1297-9686-42-5&lt;/a&gt;; M. Pszczola et al., &lt;span&gt;“Reliability of Direct Genomic Values for Animals with Different Relationships Within and to the Reference Population,”&lt;/span&gt; &lt;em&gt;Journal of Dairy Science&lt;/em&gt; 95, no. 1 (January 1, 2012): 389–400, &lt;a href="https://doi.org/10.3168/jds.2011-4338" role="doc-biblioref"&gt;https://doi.org/10.3168/jds.2011-4338&lt;/a&gt;.&lt;/p&gt;'><sup>223</sup></a></span></p>
<p>Compared to PGS scores for those of European ancestry, PGS scores across multiple traits are ~64-78% less accurate for individuals of African ancestry,,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Duncan et al., &lt;span&gt;“Analysis of Polygenic Risk Score Usage and Performance in Diverse Human Populations”&lt;/span&gt;; Martin et al., &lt;span&gt;“Clinical Use of Current Polygenic Risk Scores May Exacerbate Health Disparities.”&lt;/span&gt;&lt;/p&gt;"><sup>224</sup></a></span> ~50% less accurate for individuals of East-Asian ancestry, and ~37% less accurate for individuals of South-Asian ancestry.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Martin et al., &lt;span&gt;“Clinical Use of Current Polygenic Risk Scores May Exacerbate Health Disparities.”&lt;/span&gt;&lt;/p&gt;"><sup>225</sup></a></span></p>
</div>
</div>
<div id="contributors-to-pgs-non-transferability" class="section level3" number="1.4.2">
<h3>
<span class="header-section-number">1.4.2</span> Contributors to PGS non-transferability<a class="anchor" aria-label="anchor" href="#contributors-to-pgs-non-transferability"><i class="fas fa-link"></i></a>
</h3>
<p>What explains this disparity in predictive value? A number of factors may be responsible, including:</p>
<ol style="list-style-type: decimal">
<li><p>The failure of GWAS to identify causal variants that either do not exist or are not identifiable within the “discovery” sample, for both technological and methodological reasons;<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Martin et al.&lt;/p&gt;"><sup>226</sup></a></span></p></li>
<li><p>The sample populations may differ in linkage disequilibrium (<strong>LD</strong>) – the correlation structure of the genome – which would change the estimated effect sizes of the causal variants, even when the causal variants themselves are the same;<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Martin et al.&lt;/p&gt;"><sup>227</sup></a></span></p></li>
<li><p>Allele frequencies of the causal variants, and the distribution of the effect sizes of the causal variants, may differ between populations;<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Martin et al., &lt;span&gt;“Human Demographic History Impacts Genetic Risk Prediction Across Diverse Populations”&lt;/span&gt;; Marco Scutari, Ian Mackay, and David Balding, &lt;span&gt;“Using &lt;span&gt;Genetic Distance&lt;/span&gt; to &lt;span&gt;Infer&lt;/span&gt; the &lt;span&gt;Accuracy&lt;/span&gt; of &lt;span&gt;Genomic Prediction&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;PLOS Genetics&lt;/em&gt; 12, no. 9 (September 2, 2016): e1006288, &lt;a href="https://doi.org/10.1371/journal.pgen.1006288" role="doc-biblioref"&gt;https://doi.org/10.1371/journal.pgen.1006288&lt;/a&gt;.&lt;/p&gt;'><sup>228</sup></a></span> and</p></li>
<li><p>The environments and demographies of populations tend to differ. Such differences are often correlated with genetic divergence due to geography, making it difficult to determine whether the associations are driven by the differences between population in their genetics, or their environments.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Martin et al., &lt;span&gt;“Clinical Use of Current Polygenic Risk Scores May Exacerbate Health Disparities”&lt;/span&gt;; Sini Kerminen et al., &lt;span&gt;“Geographic &lt;span&gt;Variation&lt;/span&gt; and &lt;span&gt;Bias&lt;/span&gt; in the &lt;span&gt;Polygenic Scores&lt;/span&gt; of &lt;span&gt;Complex Diseases&lt;/span&gt; and &lt;span&gt;Traits&lt;/span&gt; in &lt;span&gt;Finland&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;The American Journal of Human Genetics&lt;/em&gt; 104, no. 6 (June 6, 2019): 1169–81, &lt;a href="https://doi.org/10.1016/j.ajhg.2019.05.001" role="doc-biblioref"&gt;https://doi.org/10.1016/j.ajhg.2019.05.001&lt;/a&gt;.&lt;/p&gt;'><sup>229</sup></a></span></p></li>
</ol>
<p>The first three factors can degrade predictive performance even in the absence of biological and environmental differences. On the other hand, environmental and demographic differences can drive forces of natural selection can in turn drive differences in causal genetic architecture.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Martin et al., &lt;span&gt;“Clinical Use of Current Polygenic Risk Scores May Exacerbate Health Disparities.”&lt;/span&gt;&lt;/p&gt;"><sup>230</sup></a></span></p>
<p>I will discuss each of these factors in turn before addressing point (3) in this analysis.</p>
<div id="fst-discovery-sec" class="section level4" number="1.4.2.1">
<h4>
<span class="header-section-number">1.4.2.1</span> Technological and methodological limitations of GWAS<a class="anchor" aria-label="anchor" href="#fst-discovery-sec"><i class="fas fa-link"></i></a>
</h4>
<p>The power to discover a causal variant through GWAS depends on the variant’s effect size and frequency in the study population.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Martin et al.; P. C. Sham et al., &lt;span&gt;“Power of &lt;span&gt;Linkage&lt;/span&gt; Versus &lt;span&gt;Association Analysis&lt;/span&gt; of &lt;span&gt;Quantitative Traits&lt;/span&gt;, by &lt;span&gt;Use&lt;/span&gt; of &lt;span&gt;Variance-Components Models&lt;/span&gt;, for &lt;span&gt;Sibship Data&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;The American Journal of Human Genetics&lt;/em&gt; 66, no. 5 (May 1, 2000): 1616–30, &lt;a href="https://doi.org/10.1086/302891" role="doc-biblioref"&gt;https://doi.org/10.1086/302891&lt;/a&gt;.&lt;/p&gt;'><sup>231</sup></a></span> That is to say, the stronger the variant’s effect, or the more common it is, the more likely it is to be discovered. Rare variants tend to have stronger effect sizes,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Kyoko Watanabe et al., &lt;span&gt;“A Global Overview of Pleiotropy and Genetic Architecture in Complex Traits,”&lt;/span&gt; &lt;em&gt;Nature Genetics&lt;/em&gt; 51, no. 9, 9 (September 2019): 1339–48, &lt;a href="https://doi.org/10.1038/s41588-019-0481-0" role="doc-biblioref"&gt;https://doi.org/10.1038/s41588-019-0481-0&lt;/a&gt;.&lt;/p&gt;'><sup>232</sup></a></span> likely due to purifying selection,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Ju-Hyun Park et al., &lt;span&gt;“Distribution of Allele Frequencies and Effect Sizes and Their Interrelationships for Common Genetic Susceptibility Variants,”&lt;/span&gt; &lt;em&gt;Proceedings of the National Academy of Sciences&lt;/em&gt; 108, no. 44 (November 2011): 18026–31, &lt;a href="https://doi.org/10.1073/pnas.1114759108" role="doc-biblioref"&gt;https://doi.org/10.1073/pnas.1114759108&lt;/a&gt;.&lt;/p&gt;'><sup>233</sup></a></span> and tend not to be shared across populations.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Simon Gravel et al., &lt;span&gt;“Demographic History and Rare Allele Sharing Among Human Populations,”&lt;/span&gt; &lt;em&gt;Proceedings of the National Academy of Sciences&lt;/em&gt; 108, no. 29 (2011): 11983–88, &lt;a href="https://doi.org/10.1073/pnas.1019276108" role="doc-biblioref"&gt;https://doi.org/10.1073/pnas.1019276108&lt;/a&gt;; Consortium et al., &lt;span&gt;“A Global Reference for Human Genetic Variation.”&lt;/span&gt;&lt;/p&gt;'><sup>234</sup></a></span> This is particularly relevant for African populations, as they have a much greater level of genetic variance than other populations due to the human species having originated on that continent.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Consortium et al., &lt;span&gt;“A Global Reference for Human Genetic Variation.”&lt;/span&gt;&lt;/p&gt;"><sup>235</sup></a></span> Therefore, if GWAS aren’t performed on diverse populations, PGS can’t take into account the rare variants present in non-European populations that are likely to exert stronger effects on the trait of interest. There are also several other issues that can affect the discoverability of causal variants through GWAS, including the technology used for genotyping, the selection of the cohort, and the necessary exclusion of genotypic outliers.
With respect to genotyping technologies, GWAS often use data from SNP microarrays. These do not sequence the whole genome, but rather a selection (from several hundred thousand to millions) of genetic markers intended to present <em>common</em> genetic variation,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Eleonora Porcu et al., &lt;span&gt;“Genotype &lt;span&gt;Imputation&lt;/span&gt; in &lt;span&gt;Genome-Wide Association Studies&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Current Protocols in Human Genetics&lt;/em&gt; 78, no. 1 (2013): 1.25.1–14, &lt;a href="https://doi.org/10.1002/0471142905.hg0125s78" role="doc-biblioref"&gt;https://doi.org/10.1002/0471142905.hg0125s78&lt;/a&gt;.&lt;/p&gt;'><sup>236</sup></a></span> which accordingly tend to neglect rare genetic variants.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Uffelmann et al., &lt;span&gt;“Genome-Wide Association Studies.”&lt;/span&gt;&lt;/p&gt;"><sup>237</sup></a></span> To increase the density of genotypes, which would increase the likelihood of refining the association signal and identifying causal variants, researchers often “impute” variants that aren’t sequenced directly.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Porcu et al., &lt;span&gt;“Genotype &lt;span&gt;Imputation&lt;/span&gt; in &lt;span&gt;Genome-Wide Association Studies&lt;/span&gt;.”&lt;/span&gt;&lt;/p&gt;"><sup>238</sup></a></span> The imputation process involves “phasing” the study genotypes onto the genotypes of a “reference panel.”<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Shane McCarthy et al., &lt;span&gt;“A Reference Panel of 64,976 Haplotypes for Genotype Imputation,”&lt;/span&gt; &lt;em&gt;Nature Genetics&lt;/em&gt; 48, no. 10, 10 (October 2016): 1279–83, &lt;a href="https://doi.org/10.1038/ng.3643" role="doc-biblioref"&gt;https://doi.org/10.1038/ng.3643&lt;/a&gt;.&lt;/p&gt;'><sup>239</sup></a></span> However, if the reference panel does not sufficiently represent the population in the study sample, they are likely to miss or incorrectly impute those genotypes.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Martin et al., &lt;span&gt;“Clinical Use of Current Polygenic Risk Scores May Exacerbate Health Disparities.”&lt;/span&gt;&lt;/p&gt;"><sup>240</sup></a></span> Again, this is particularly problematic for African populations.</p>
<p>The lack of representation of rare variants in SNP microarrays can be overcome by using next-generation sequencing technologies such as whole-genome sequencing (<strong>WGS</strong>) and whole-exome sequencing (<strong>WES</strong>). (The former seeks to sequence the full genome, and the latter of only targets the coding regions of the genome.) These methods are more expensive than SNP microarrays, which hinders their widespread use at scale, and although their costs are continuing to decrease rapidly, there is a question as to whether they return a proportionate benefit in all use cases.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Katharina Schwarze et al., &lt;span&gt;“Are Whole-Exome and Whole-Genome Sequencing Approaches Cost-Effective? &lt;span&gt;A&lt;/span&gt; Systematic Review of the Literature,”&lt;/span&gt; &lt;em&gt;Genetics in Medicine&lt;/em&gt; 20, no. 10 (October 1, 2018): 1122–30, &lt;a href="https://doi.org/10.1038/gim.2017.247" role="doc-biblioref"&gt;https://doi.org/10.1038/gim.2017.247&lt;/a&gt;.&lt;/p&gt;'><sup>241</sup></a></span></p>
<p>A second limitation is the selection of GWAS cohorts, which can introduce selection and collider biases.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Uffelmann et al., &lt;span&gt;“Genome-Wide Association Studies.”&lt;/span&gt;&lt;/p&gt;"><sup>242</sup></a></span> For instance, the UK Biobank, which contains genetic and phenotypic data on 500,000 participants who volunteered for inclusion between 2006 and 2010, tend to be older, female, healthier, and wealthier than non-participants.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Anna Fry et al., &lt;span&gt;“Comparison of &lt;span&gt;Sociodemographic&lt;/span&gt; and &lt;span&gt;Health-Related Characteristics&lt;/span&gt; of &lt;span&gt;UK Biobank Participants With Those&lt;/span&gt; of the &lt;span&gt;General Population&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;American Journal of Epidemiology&lt;/em&gt; 186, no. 9 (November 1, 2017): 1026–34, &lt;a href="https://doi.org/10.1093/aje/kwx246" role="doc-biblioref"&gt;https://doi.org/10.1093/aje/kwx246&lt;/a&gt;.&lt;/p&gt;'><sup>243</sup></a></span> This creaties the possibility of confounding genetic associations with environmental factors, which I discuss further in <a href="Introduction.html#fst-env-sec">1.4.2.4</a> below.</p>
<p>A third limitation is the “quality control” step that is required during the GWAS process.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Uffelmann et al., &lt;span&gt;“Genome-Wide Association Studies.”&lt;/span&gt;&lt;/p&gt;"><sup>244</sup></a></span> To avoid confounding from population stratification, which can lead to overestimated heritability and biased PGS, GWAS cohorts are filtered to include only those with similar ancestries – or relative genetic homogeneity – by clustering individuals through principal component analysis (<strong>PCA</strong>) of their genotypes, and excluding outliers. I elaborate on the issue of population stratification in section <a href="Introduction.html#fst-env-sec">1.4.2.4</a> below, but at present, a statistical model for GWAS that can include cohorts with diverse ancestries without the risk of serious confounding is yet to be developed.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Jeremy J Berg et al., &lt;span&gt;“Reduced Signal for Polygenic Adaptation of Height in &lt;span&gt;UK Biobank&lt;/span&gt;,”&lt;/span&gt; ed. Magnus Nordborg et al., &lt;em&gt;eLife&lt;/em&gt; 8 (March 21, 2019): e39725, &lt;a href="https://doi.org/10.7554/eLife.39725" role="doc-biblioref"&gt;https://doi.org/10.7554/eLife.39725&lt;/a&gt;.&lt;/p&gt;'><sup>245</sup></a></span></p>
</div>
<div id="differences-in-ld" class="section level4" number="1.4.2.2">
<h4>
<span class="header-section-number">1.4.2.2</span> Differences in LD<a class="anchor" aria-label="anchor" href="#differences-in-ld"><i class="fas fa-link"></i></a>
</h4>
<p>Because GWAS SNP markers are often not the causal variants themselves, but merely in physical proximity to them, the estimated effect size of a SNP marker depends on the extent to which it is in LD with the causal variant.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Hakhamanesh Mostafavi et al., &lt;span&gt;“Variable Prediction Accuracy of Polygenic Scores Within an Ancestry Group,”&lt;/span&gt; ed. Ruth Loos, Michael B Eisen, and Paul O’Reilly, &lt;em&gt;eLife&lt;/em&gt; 9 (January 30, 2020): e48376, &lt;a href="https://doi.org/10.7554/eLife.48376" role="doc-biblioref"&gt;https://doi.org/10.7554/eLife.48376&lt;/a&gt;; Jonathan K. Pritchard and Molly Przeworski, &lt;span&gt;“Linkage &lt;span&gt;Disequilibrium&lt;/span&gt; in &lt;span&gt;Humans&lt;/span&gt;: &lt;span&gt;Models&lt;/span&gt; and &lt;span&gt;Data&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;The American Journal of Human Genetics&lt;/em&gt; 69, no. 1 (July 1, 2001): 1–14, &lt;a href="https://doi.org/10.1086/321275" role="doc-biblioref"&gt;https://doi.org/10.1086/321275&lt;/a&gt;.&lt;/p&gt;'><sup>246</sup></a></span> To illustrate the problem, if a SNP has an LD <span class="math inline">\(r^2\)</span> with a causal variant of 0.8 in the discovery population and 0.6 in the target population, it would explain 25% = (1 - 0.6/0.8) less trait variation in the target population, and would therefore be less predictive.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Ying Wang et al., &lt;span&gt;“Theoretical and Empirical Quantification of the Accuracy of Polygenic Scores in Ancestry Divergent Populations,”&lt;/span&gt; &lt;em&gt;Nature Communications&lt;/em&gt; 11, no. 1, 1 (July 31, 2020): 3865, &lt;a href="https://doi.org/10.1038/s41467-020-17719-y" role="doc-biblioref"&gt;https://doi.org/10.1038/s41467-020-17719-y&lt;/a&gt;.&lt;/p&gt;'><sup>247</sup></a></span></p>
<p>These differences in effect-size estimates may typically be small for most regions of the genome, but as PGS sum across all such effects, they aggregate these population differences.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Martin et al., &lt;span&gt;“Clinical Use of Current Polygenic Risk Scores May Exacerbate Health Disparities”&lt;/span&gt;; Berg et al., &lt;span&gt;“Reduced Signal for Polygenic Adaptation of Height in &lt;span&gt;UK Biobank&lt;/span&gt;.”&lt;/span&gt;&lt;/p&gt;"><sup>248</sup></a></span> Previous empirical and simulation studies have shown that accuracy of PGS scores decay with increased genetic differentiation (<span class="math inline">\(F_{ST}\)</span> – described below in <a href="Introduction.html#Fst-descr">1.4.3</a>) and LD differences between populations.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Habier et al., &lt;span&gt;“The Impact of Genetic Relationship Information on Genomic Breeding Values in &lt;span&gt;German Holstein&lt;/span&gt; Cattle”&lt;/span&gt;; Pszczola et al., &lt;span&gt;“Reliability of Direct Genomic Values for Animals with Different Relationships Within and to the Reference Population”&lt;/span&gt;; Scutari, Mackay, and Balding, &lt;span&gt;“Using &lt;span&gt;Genetic Distance&lt;/span&gt; to &lt;span&gt;Infer&lt;/span&gt; the &lt;span&gt;Accuracy&lt;/span&gt; of &lt;span&gt;Genomic Prediction&lt;/span&gt;”&lt;/span&gt;; Wang et al., &lt;span&gt;“Theoretical and Empirical Quantification of the Accuracy of Polygenic Scores in Ancestry Divergent Populations.”&lt;/span&gt;&lt;/p&gt;"><sup>249</sup></a></span> The issue may be addressed to a degree by using LD information from an external reference panel as a prior to infer the posterior mean effect size of a genetic variant – <span class="citation">Bjarni J Vilhjálmsson et al.<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;&lt;span&gt;“Modeling Linkage Disequilibrium Increases Accuracy of Polygenic Risk Scores,”&lt;/span&gt; &lt;em&gt;The American Journal of Human Genetics&lt;/em&gt; 97, no. 4 (2015): 576–92.&lt;/p&gt;"><sup>250</sup></a></span> demonstrated through simulations that this could improve PGS predictive accuracy. Yet the most appropriate means of deal with differences between populations in LD remains an active area of research.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Duncan et al., &lt;span&gt;“Analysis of Polygenic Risk Score Usage and Performance in Diverse Human Populations.”&lt;/span&gt;&lt;/p&gt;"><sup>251</sup></a></span></p>
</div>
<div id="differences-in-allele-frequencies" class="section level4" number="1.4.2.3">
<h4>
<span class="header-section-number">1.4.2.3</span> Differences in allele frequencies<a class="anchor" aria-label="anchor" href="#differences-in-allele-frequencies"><i class="fas fa-link"></i></a>
</h4>
<p>Causal variants can differ in both frequency and effect size between different ancestry groups, e.g. for lactase persistence,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Laure S’egurel and C’eline Bon, &lt;span&gt;“On the &lt;span&gt;Evolution&lt;/span&gt; of &lt;span&gt;Lactase Persistence&lt;/span&gt; in &lt;span&gt;Humans&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Annual Review of Genomics and Human Genetics&lt;/em&gt; 18 (August 1, 2017): 297–319, &lt;a href="https://doi.org/10.1146/annurev-genom-091416-035340" role="doc-biblioref"&gt;https://doi.org/10.1146/annurev-genom-091416-035340&lt;/a&gt;.&lt;/p&gt;'><sup>252</sup></a></span> or skin pigmentation.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Kaustubh Adhikari et al., &lt;span&gt;“A &lt;span&gt;GWAS&lt;/span&gt; in &lt;span&gt;Latin Americans&lt;/span&gt; Highlights the Convergent Evolution of Lighter Skin Pigmentation in &lt;span&gt;Eurasia&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Nature Communications&lt;/em&gt; 10, no. 1, 1 (January 21, 2019): 358, &lt;a href="https://doi.org/10.1038/s41467-018-08147-0" role="doc-biblioref"&gt;https://doi.org/10.1038/s41467-018-08147-0&lt;/a&gt;.&lt;/p&gt;'><sup>253</sup></a></span> If a causal allele is rare in the GWAS discovery population, even if it is discovered (see <a href="Introduction.html#fst-discovery-sec">1.4.2.1</a>), it is likely to have noisy effect size estimates, and therefore likely to inaccurately estimate its effect size in a different population where it exists at a higher frequency.</p>
<p>Differences in allele frequencies between populations can arise through random genetic drift, or be driven by selective pressures towards the trait optima for a given environment.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Arbel Harpak and Molly Przeworski, &lt;span&gt;“The Evolution of Group Differences in Changing Environments,”&lt;/span&gt; &lt;em&gt;PLOS Biology&lt;/em&gt; 19, no. 1 (January 25, 2021): e3001072, &lt;a href="https://doi.org/10.1371/journal.pbio.3001072" role="doc-biblioref"&gt;https://doi.org/10.1371/journal.pbio.3001072&lt;/a&gt;.&lt;/p&gt;'><sup>254</sup></a></span> However, evolutionary biologists have found that differences between populations in the mean values for traits tend to occur through small, coordinated shifts in their allele frequencies.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Berg et al., &lt;span&gt;“Reduced Signal for Polygenic Adaptation of Height in &lt;span&gt;UK Biobank&lt;/span&gt;”&lt;/span&gt;; Michael D Edge and Graham Coop, &lt;span&gt;“Reconstructing the &lt;span&gt;History&lt;/span&gt; of &lt;span&gt;Polygenic Scores Using Coalescent Trees&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Genetics&lt;/em&gt; 211, no. 1 (January 1, 2019): 235–62, &lt;a href="https://doi.org/10.1534/genetics.118.301687" role="doc-biblioref"&gt;https://doi.org/10.1534/genetics.118.301687&lt;/a&gt;.&lt;/p&gt;'><sup>255</sup></a></span> In Chapter <a href="Fst-chap.html#Fst-chap">6</a>, I explore the differences in allele frequencies across populations for all polygenic traits in the GWAS Catalog, and confirm that with few exceptions – including skin pigmentation, and HIV viral load – the differences in allele frequencies between populations tends to be small.</p>
</div>
<div id="fst-env-sec" class="section level4" number="1.4.2.4">
<h4>
<span class="header-section-number">1.4.2.4</span> Differences in environment<a class="anchor" aria-label="anchor" href="#fst-env-sec"><i class="fas fa-link"></i></a>
</h4>
<p>Genes continuously interact with each other (GxG, or “epistasis”),<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Pierre-Alexis Gros, Herv’e Le Nagard, and Olivier Tenaillon, &lt;span&gt;“The &lt;span&gt;Evolution&lt;/span&gt; of &lt;span&gt;Epistasis&lt;/span&gt; and &lt;span&gt;Its Links With Genetic Robustness&lt;/span&gt;, &lt;span&gt;Complexity&lt;/span&gt; and &lt;span&gt;Drift&lt;/span&gt; in a &lt;span&gt;Phenotypic Model&lt;/span&gt; of &lt;span&gt;Adaptation&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Genetics&lt;/em&gt; 182, no. 1 (May 1, 2009): 277–93, &lt;a href="https://doi.org/10.1534/genetics.108.099127" role="doc-biblioref"&gt;https://doi.org/10.1534/genetics.108.099127&lt;/a&gt;.&lt;/p&gt;'><sup>256</sup></a></span> the genes of one’s parents (“genetic nurture”, <span class="citation">Kong et al.<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;&lt;span&gt;“The Nature of Nurture.”&lt;/span&gt;&lt;/p&gt;"><sup>257</sup></a></span>) or social companions (“social genetic effects”),<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Benjamin W. Domingue et al., &lt;span&gt;“The Social Genome of Friends and Schoolmates in the &lt;span&gt;National Longitudinal Study&lt;/span&gt; of &lt;span&gt;Adolescent&lt;/span&gt; to &lt;span&gt;Adult Health&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Proceedings of the National Academy of Sciences&lt;/em&gt; 115, no. 4 (January 23, 2018): 702–7, &lt;a href="https://doi.org/10.1073/pnas.1711803115" role="doc-biblioref"&gt;https://doi.org/10.1073/pnas.1711803115&lt;/a&gt;; Baud et al., &lt;span&gt;“Genetic &lt;span&gt;Variation&lt;/span&gt; in the &lt;span&gt;Social Environment Contributes&lt;/span&gt; to &lt;span&gt;Health&lt;/span&gt; and &lt;span&gt;Disease&lt;/span&gt;.”&lt;/span&gt;&lt;/p&gt;'><sup>258</sup></a></span><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;As also explored in Chapters &lt;a href="Pilot-chap.html#Pilot-chap"&gt;3&lt;/a&gt; and &lt;a href="MIKK-F2-chap.html#MIKK-F2-chap"&gt;5&lt;/a&gt;&lt;/p&gt;'><sup>259</sup></a> and the wider non-genetic environment (GxE).</p>
<p>The respective contributions of genetics and environment to traits with social value – such as intelligence – is highly contentious, especially when there are apparent differences between populations in the mean values for those traits. PGS measure the proportion of variance within a population that is explained by genetics. Because PRS summarises a <em>proportion</em> of the total variance, when studying a population that is subject to greater environmental variation, the variance attributable to genetic factors will proportionately reduce. The corollary being that when studying a population where the environment is held constant, the proportion of variance for that trait that is explained by genetic factors will approach 1. Therefore, increases in the amount of environmental variance that a population is exposed to will reduce the accuracy of PGS predictions when applied to that population.</p>
<p>Different environments are also often correlated with population structure.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Berg et al., &lt;span&gt;“Reduced Signal for Polygenic Adaptation of Height in &lt;span&gt;UK Biobank&lt;/span&gt;.”&lt;/span&gt;&lt;/p&gt;"><sup>260</sup></a></span> For example, in East Asia, there is a greater proportion of individuals of East-Asian ancestry than there is of European ancestry, and <em>vice versa</em> in Europe. Those East-Asian individuals will therefore tend to share more of their genetic background with each other than with Europeans, and that population structure will be correlated with the different environments that exist in East Asia compared to Europe. This makes it difficult to determine whether it is the differences in their environments or the differences in their genetics that is driving the discrepancies between the mean values for traits between those populations. These complexities are unlikely to be resolved in the near future, which makes it attractive to turn to model organisms to address more basic biological questions regarding GxE in relation to complex traits,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Leif Andersson and Michel Georges, &lt;span&gt;“Domestic-Animal Genomics: Deciphering the Genetics of Complex Traits,”&lt;/span&gt; &lt;em&gt;Nature Reviews Genetics&lt;/em&gt; 5, no. 3, 3 (March 2004): 202–12, &lt;a href="https://doi.org/10.1038/nrg1294" role="doc-biblioref"&gt;https://doi.org/10.1038/nrg1294&lt;/a&gt;.&lt;/p&gt;'><sup>261</sup></a></span> as we have done with respect to behaviour in Chapters <a href="Pilot-chap.html#Pilot-chap">3</a> and <a href="MIKK-F2-chap.html#MIKK-F2-chap">5</a>.</p>
</div>
</div>
<div id="Fst-descr" class="section level3" number="1.4.3">
<h3>
<span class="header-section-number">1.4.3</span> <span class="math inline">\(F_{ST}\)</span><a class="anchor" aria-label="anchor" href="#Fst-descr"><i class="fas fa-link"></i></a>
</h3>
<p>The widely-used fixation index (<span class="math inline">\(F_{ST}\)</span>) was introduced independently by Sewall Wright<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Sewall Wright, &lt;span&gt;“The &lt;span&gt;Genetical Structure&lt;/span&gt; of &lt;span&gt;Populations&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Annals of Eugenics&lt;/em&gt; 15, no. 1 (1949): 323–54, &lt;a href="https://doi.org/10.1111/j.1469-1809.1949.tb02451.x" role="doc-biblioref"&gt;https://doi.org/10.1111/j.1469-1809.1949.tb02451.x&lt;/a&gt;.&lt;/p&gt;'><sup>262</sup></a></span> and Gustave Malécot<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Gustave Mal’ecot, &lt;span&gt;“Mathématiques de l’hérédité,”&lt;/span&gt; 1948.&lt;/p&gt;"><sup>263</sup></a></span> as a metric for measuring the genetic diversity between populations.<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;In Wright’s notation, &lt;span class="math inline"&gt;\(F\)&lt;/span&gt; refers to “fixation” of an allele, and &lt;span class="math inline"&gt;\(_{ST}\)&lt;/span&gt; refers to “subpopulations within the total population”.&lt;/p&gt;'><sup>264</sup></a> It quantifies the relative variance in allele frequency between groups compared to within groups, reflecting the combined effects of genetic drift, migration, mutation, and selection.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Kent E. Holsinger and Bruce S. Weir, &lt;span&gt;“Genetics in Geographically Structured Populations: Defining, Estimating and Interpreting &lt;span&gt;FST&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Nature Reviews Genetics&lt;/em&gt; 10, no. 9, 9 (September 2009): 639–50, &lt;a href="https://doi.org/10.1038/nrg2611" role="doc-biblioref"&gt;https://doi.org/10.1038/nrg2611&lt;/a&gt;.&lt;/p&gt;'><sup>265</sup></a></span> The metric ranges from 0 to 1, where loci with high <span class="math inline">\(F_{ST}\)</span> values – that is, loci with a large relative between-group variance in allele frequencies – may have been subject to selection or different demographic processes.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Holsinger and Weir.&lt;/p&gt;"><sup>266</sup></a></span> The metric has customarily been used to identify regions of the genome have been subject to diversifying selection.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Joshua M. Akey et al., &lt;span&gt;“Interrogating a &lt;span&gt;High-Density SNP Map&lt;/span&gt; for &lt;span&gt;Signatures&lt;/span&gt; of &lt;span&gt;Natural Selection&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Genome Research&lt;/em&gt; 12, no. 12 (January 12, 2002): 1805–14, &lt;a href="https://doi.org/10.1101/gr.631202" role="doc-biblioref"&gt;https://doi.org/10.1101/gr.631202&lt;/a&gt;; Feng Guo, Dipak K. Dey, and Kent E. Holsinger, &lt;span&gt;“A &lt;span&gt;Bayesian Hierarchical Model&lt;/span&gt; for &lt;span&gt;Analysis&lt;/span&gt; of &lt;span&gt;Single-Nucleotide Polymorphisms Diversity&lt;/span&gt; in &lt;span&gt;Multilocus&lt;/span&gt;, &lt;span&gt;Multipopulation Samples&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Journal of the American Statistical Association&lt;/em&gt; 104, no. 485 (March 1, 2009): 142–54, &lt;a href="https://doi.org/10.1198/jasa.2009.0010" role="doc-biblioref"&gt;https://doi.org/10.1198/jasa.2009.0010&lt;/a&gt;; Bruce S. Weir et al., &lt;span&gt;“Measures of Human Population Structure Show Heterogeneity Among Genomic Regions,”&lt;/span&gt; &lt;em&gt;Genome Research&lt;/em&gt; 15, no. 11 (January 11, 2005): 1468–76, &lt;a href="https://doi.org/10.1101/gr.4398405" role="doc-biblioref"&gt;https://doi.org/10.1101/gr.4398405&lt;/a&gt;.&lt;/p&gt;'><sup>267</sup></a></span></p>

</div>
</div>
</div>

  <div class="chapter-nav">
<div class="prev"><a href="index.html">About</a></div>
<div class="next"><a href="MIKK-genomes-chap.html"><span class="header-section-number">2</span> Genomic variations in the MIKK panel</a></div>
</div></main><div class="col-md-3 col-lg-2 d-none d-md-block sidebar sidebar-chapter">
    <nav id="toc" data-toggle="toc" aria-label="On this page"><h2>On this page</h2>
      <ul class="nav navbar-nav">
<li><a class="nav-link" href="#Introduction"><span class="header-section-number">1</span> Introduction</a></li>
<li>
<a class="nav-link" href="#a-brief-history-of-genetics"><span class="header-section-number">1.1</span> A brief history of genetics</a><ul class="nav navbar-nav">
<li><a class="nav-link" href="#ancient-greece"><span class="header-section-number">1.1.1</span> Ancient Greece</a></li>
<li><a class="nav-link" href="#middle-ages"><span class="header-section-number">1.1.2</span> Middle Ages</a></li>
<li><a class="nav-link" href="#charles-darwin"><span class="header-section-number">1.1.3</span> Charles Darwin</a></li>
<li><a class="nav-link" href="#gregor-mendel"><span class="header-section-number">1.1.4</span> Gregor Mendel</a></li>
<li><a class="nav-link" href="#quantitative-genetics-francis-galton-karl-pearson-ronald-fisher-and-sewall-wright"><span class="header-section-number">1.1.5</span> Quantitative genetics: Francis Galton, Karl Pearson, Ronald Fisher and Sewall Wright</a></li>
<li><a class="nav-link" href="#genetic-mapping-thomas-morgan-tetsuo-aida-and-theodosius-dobzhansky"><span class="header-section-number">1.1.6</span> Genetic mapping: Thomas Morgan, Tetsuo Aida and Theodosius Dobzhansky</a></li>
<li><a class="nav-link" href="#the-discovery-of-the-structure-of-dna"><span class="header-section-number">1.1.7</span> The discovery of the structure of DNA</a></li>
<li><a class="nav-link" href="#the-development-of-dna-sequencing"><span class="header-section-number">1.1.8</span> The development of DNA sequencing</a></li>
</ul>
</li>
<li>
<a class="nav-link" href="#genetic-and-environmental-causes-of-variation-in-complex-traits"><span class="header-section-number">1.2</span> Genetic and environmental causes of variation in complex traits</a><ul class="nav navbar-nav">
<li><a class="nav-link" href="#heritability"><span class="header-section-number">1.2.1</span> Heritability</a></li>
<li><a class="nav-link" href="#genome-wide-association-analysis-gwas"><span class="header-section-number">1.2.2</span> Genome-Wide Association Analysis (GWAS)</a></li>
<li><a class="nav-link" href="#animal-models"><span class="header-section-number">1.2.3</span> Animal models</a></li>
<li><a class="nav-link" href="#the-medaka-inbred-kiyosu-karlsruhe-mikk-panel"><span class="header-section-number">1.2.4</span> The Medaka Inbred Kiyosu-Karlsruhe (MIKK) panel</a></li>
<li><a class="nav-link" href="#the-f2-cross"><span class="header-section-number">1.2.5</span> The F2 cross</a></li>
</ul>
</li>
<li>
<a class="nav-link" href="#studied-phenotypes"><span class="header-section-number">1.3</span> Studied phenotypes</a><ul class="nav navbar-nav">
<li><a class="nav-link" href="#genetic-control-of-somite-period-development"><span class="header-section-number">1.3.1</span> Genetic control of somite period development</a></li>
<li><a class="nav-link" href="#behaviour"><span class="header-section-number">1.3.2</span> Behaviour</a></li>
</ul>
</li>
<li>
<a class="nav-link" href="#poor-transferability-of-polygenic-scores-to-diverse-human-populations"><span class="header-section-number">1.4</span> Poor transferability of polygenic scores to diverse human populations</a><ul class="nav navbar-nav">
<li><a class="nav-link" href="#polygenic-scores-pgs-and-gwas"><span class="header-section-number">1.4.1</span> Polygenic Scores (PGS) and GWAS</a></li>
<li><a class="nav-link" href="#contributors-to-pgs-non-transferability"><span class="header-section-number">1.4.2</span> Contributors to PGS non-transferability</a></li>
<li><a class="nav-link" href="#Fst-descr"><span class="header-section-number">1.4.3</span> \(F_{ST}\)</a></li>
</ul>
</li>
</ul>

      <div class="book-extra">
        <ul class="list-unstyled">
<li><a id="book-source" href="https://github.com/brettellebi/PhD-thesis/blob/master/book/01-Introduction.Rmd">View source <i class="fab fa-github"></i></a></li>
          <li><a id="book-edit" href="https://github.com/brettellebi/PhD-thesis/edit/master/book/01-Introduction.Rmd">Edit this page <i class="fab fa-github"></i></a></li>
        </ul>
</div>
    </nav>
</div>

</div>
</div> <!-- .container -->

<footer class="bg-primary text-light mt-5"><div class="container"><div class="row">

  <div class="col-12 col-md-6 mt-3">
    <p>"<strong>Japanese courage: a genetic analysis of complex traits in medaka fish and humans</strong>" was written by Ian Brettell. It was last built on 2022-09-06.</p>
  </div>

  <div class="col-12 col-md-6 mt-3">
    <p>This book was built by the <a class="text-light" href="https://bookdown.org">bookdown</a> R package.</p>
  </div>

</div></div>
</footer><!-- dynamically load mathjax for compatibility with self-contained --><script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.9/latest.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script><script type="text/x-mathjax-config">const popovers = document.querySelectorAll('a.footnote-ref[data-toggle="popover"]');
for (let popover of popovers) {
  const div = document.createElement('div');
  div.setAttribute('style', 'position: absolute; top: 0, left:0; width:0, height:0, overflow: hidden; visibility: hidden;');
  div.innerHTML = popover.getAttribute('data-content');

  var has_math = div.querySelector("span.math");
  if (has_math) {
    document.body.appendChild(div);
    MathJax.Hub.Queue(["Typeset", MathJax.Hub, div]);
    MathJax.Hub.Queue(function() {
      popover.setAttribute('data-content', div.innerHTML);
      document.body.removeChild(div);
    })
  }
}
</script>
</body>
</html>
