<!DOCTYPE html>
<html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<title>Chapter 4 Genetic linkage study of bold/shy behaviours in the MIKK panel | Japanese courage: a genetic analysis of complex traits in medaka fish and humans</title>
<meta name="author" content="Ian Brettell">
<meta name="description" content="The purpose of the study described in this chapter was to run the behavioural analysis described in Chapter 3 over the MIKK panel described in Chapter 2, identify the lines that diverged in both...">
<meta name="generator" content="bookdown 0.26 with bs4_book()">
<meta property="og:title" content="Chapter 4 Genetic linkage study of bold/shy behaviours in the MIKK panel | Japanese courage: a genetic analysis of complex traits in medaka fish and humans">
<meta property="og:type" content="book">
<meta property="og:description" content="The purpose of the study described in this chapter was to run the behavioural analysis described in Chapter 3 over the MIKK panel described in Chapter 2, identify the lines that diverged in both...">
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="Chapter 4 Genetic linkage study of bold/shy behaviours in the MIKK panel | Japanese courage: a genetic analysis of complex traits in medaka fish and humans">
<meta name="twitter:description" content="The purpose of the study described in this chapter was to run the behavioural analysis described in Chapter 3 over the MIKK panel described in Chapter 2, identify the lines that diverged in both...">
<!-- JS --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://kit.fontawesome.com/6ecbd6c532.js" crossorigin="anonymous"></script><script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="libs/bootstrap-4.6.0/bootstrap.min.css" rel="stylesheet">
<script src="libs/bootstrap-4.6.0/bootstrap.bundle.min.js"></script><script src="libs/bs3compat-0.4.0/transition.js"></script><script src="libs/bs3compat-0.4.0/tabs.js"></script><script src="libs/bs3compat-0.4.0/bs3compat.js"></script><link href="libs/bs4_book-1.0.0/bs4_book.css" rel="stylesheet">
<script src="libs/bs4_book-1.0.0/bs4_book.js"></script><style>
    #content {
        font-size: 0.9rem;
    }
    </style>
<script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- CSS --><style type="text/css">
    
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  </style>
<style type="text/css">
    /* Used with Pandoc 2.11+ new --citeproc when CSL is used */
    div.csl-bib-body { }
    div.csl-entry {
      clear: both;
        }
    .hanging div.csl-entry {
      margin-left:2em;
      text-indent:-2em;
    }
    div.csl-left-margin {
      min-width:2em;
      float:left;
    }
    div.csl-right-inline {
      margin-left:2em;
      padding-left:1em;
    }
    div.csl-indent {
      margin-left: 2em;
    }
  </style>
<link rel="stylesheet" href="style.css">
</head>
<body data-spy="scroll" data-target="#toc">

<div class="container-fluid">
<div class="row">
  <header class="col-sm-12 col-lg-3 sidebar sidebar-book"><a class="sr-only sr-only-focusable" href="#content">Skip to main content</a>

    <div class="d-flex align-items-start justify-content-between">
      <h1>
        <a href="index.html" title="">Japanese courage: a genetic analysis of complex traits in medaka fish and humans</a>
      </h1>
      <button class="btn btn-outline-primary d-lg-none ml-2 mt-1" type="button" data-toggle="collapse" data-target="#main-nav" aria-expanded="true" aria-controls="main-nav"><i class="fas fa-bars"></i><span class="sr-only">Show table of contents</span></button>
    </div>

    <div id="main-nav" class="collapse-lg">
      <form role="search">
        <input id="search" class="form-control" type="search" placeholder="Search" aria-label="Search">
</form>

      <nav aria-label="Table of contents"><h2>Table of contents</h2>
        <ul class="book-toc list-unstyled">
<li><a class="" href="index.html">About</a></li>
<li><a class="" href="Introduction.html"><span class="header-section-number">1</span> Introduction</a></li>
<li><a class="" href="MIKK-genomes-chap.html"><span class="header-section-number">2</span> Genomic variations in the MIKK panel</a></li>
<li><a class="" href="Pilot-chap.html"><span class="header-section-number">3</span> Classification of bold/shy behaviours in 5 inbred medaka lines</a></li>
<li><a class="active" href="MIKK-F2-chap.html"><span class="header-section-number">4</span> Genetic linkage study of bold/shy behaviours in the MIKK panel</a></li>
<li><a class="" href="genetic-loci-associated-with-somite-development-periodicity.html"><span class="header-section-number">5</span> Genetic loci associated with somite development periodicity</a></li>
<li><a class="" href="Fst-chap.html"><span class="header-section-number">6</span> Variation in the frequency of trait-associated alleles across global human populations</a></li>
<li><a class="" href="references.html">References</a></li>
<li class="book-part">Appendix</li>
<li><a class="" href="ecdf-of-all-polygenic-traits-in-the-gwas-catalog-ranked-by-d_ts.html"><span class="header-section-number">A</span> eCDF of all polygenic traits in the GWAS Catalog ranked by \({D_t^S}\)</a></li>
<li><a class="" href="hmm-states-with-0.05-second-interval.html"><span class="header-section-number">B</span> 15 HMM states with 0.05 second interval</a></li>
<li><a class="" href="hmm-state-time-dependence-for-all-mikk-panel-lines.html"><span class="header-section-number">C</span> HMM state time dependence for all MIKK panel lines</a></li>
</ul>

        <div class="book-extra">
          <p><a id="book-repo" href="https://github.com/brettellebi/PhD-thesis">View book source <i class="fab fa-github"></i></a></p>
        </div>
      </nav>
</div>
  </header><main class="col-sm-12 col-md-9 col-lg-7" id="content"><div id="MIKK-F2-chap" class="section level1" number="4">
<h1>
<span class="header-section-number">4</span> Genetic linkage study of bold/shy behaviours in the MIKK panel<a class="anchor" aria-label="anchor" href="#MIKK-F2-chap"><i class="fas fa-link"></i></a>
</h1>
<p>The purpose of the study described in this chapter was to run the behavioural analysis described in Chapter <a href="Pilot-chap.html#Pilot-chap">3</a> over the MIKK panel described in Chapter <a href="MIKK-genomes-chap.html#MIKK-genomes-chap">2</a>, identify the lines that diverged in both their behaviour, and the level of transmission of their behaviour onto their <em>iCab</em> reference tank partner, and then use them as the parental strains in an F2 cross to attempt to identify the genetic variants associated with those differences.</p>
<div id="MIKK-F2-cross" class="section level2" number="4.1">
<h2>
<span class="header-section-number">4.1</span> The F2 cross experimental setup<a class="anchor" aria-label="anchor" href="#MIKK-F2-cross"><i class="fas fa-link"></i></a>
</h2>
<p>The F2 cross is a traditional method for mapping genetic variants associated with traits of interest. A schema for the method is presented in <strong>Figure <a href="MIKK-F2-chap.html#fig:F2-cross-schema">4.1</a></strong>. In essence, it involves starting with two inbred strains that diverge for the trait of interest (the ‘parental strains’, or F0 generation F0). One then crosses the parental strains to create a generation of F1 hybrid individuals who each possess, for every pair of their chromosomes, one chromosome from each of their parents. The individuals in this F1 generation are genetically identical to their parents with respect to their germline. Finally, one inter-crosses the F1 generation to create a set of F2 individuals that share unique combinations of the original F0 strains’ genotypes, and tend to display values for the trait of interest that span across the spectrum between the extreme values of their parents.</p>

<div class="figure">
<span style="display:block;" id="fig:F2-cross-schema"></span>
<img src="figs/mikk_behaviour/F2-cross-schema.png" alt="Schema of the F2-cross experimental setup. The F0 generation comprises two medaka strains that have extreme, opposing values for a trait of interest, represented by the colours red and blue. Below them is an illustrative single pair of chromosomes. The chromosomes within each pair are depicted as the same colour, as the strains are homozygous through successive generations of inbreeding. Their F1 offspring is heterozygous for each pair of their 24 chromosomes, and all F1 individuals are therefore almost genetically identical to one another (with the exception of somatic mutations and the regions of the genome that were not homozygous in the parental generations). The F1 individuals are then inter-crossed with one another to produce the F2 generation, which, due to recombination events during gamete formation, have unqiue combinations of the parental strains’ genotypes, and tend to span the phenotypic spectrum between the extremes of their F0 parental strains, represented by their colours." width="100%"><p class="caption">
Figure 4.1: Schema of the F2-cross experimental setup. The F0 generation comprises two medaka strains that have extreme, opposing values for a trait of interest, represented by the colours red and blue. Below them is an illustrative single pair of chromosomes. The chromosomes within each pair are depicted as the same colour, as the strains are homozygous through successive generations of inbreeding. Their F1 offspring is heterozygous for each pair of their 24 chromosomes, and all F1 individuals are therefore almost genetically identical to one another (with the exception of somatic mutations and the regions of the genome that were not homozygous in the parental generations). The F1 individuals are then inter-crossed with one another to produce the F2 generation, which, due to recombination events during gamete formation, have unqiue combinations of the parental strains’ genotypes, and tend to span the phenotypic spectrum between the extremes of their F0 parental strains, represented by their colours.
</p>
</div>
</div>
<div id="data-collection---f0-generation" class="section level2" number="4.2">
<h2>
<span class="header-section-number">4.2</span> Data collection - F0 generation<a class="anchor" aria-label="anchor" href="#data-collection---f0-generation"><i class="fas fa-link"></i></a>
</h2>
<p>In November 2019 I traveled to the fish facility managed by our collaborator, Felix Loosli at KIT in Karlsruhe, and over the course of 11 days from 11 to 21 November 2019, I ran the behavioural assay described in Chapter <a href="Pilot-chap.html#pilot-data-collection">3.4.1</a> another 206 times. I again used the <em>iCab</em> strain as the reference fish, and for the test fish I used either an individual from one of the MIKK panel lines, individuals captured from the same Kiyosu population as the MIKK panel but permitted to breed freely within a separate tank in the facility (‘Kiyosu closed-capture’, or ‘Kiyosu CC’), or individuals from a related but different species of medaka from the Philippines, <em>Oryzias luzonensis</em>. I ensured that I performed at least 2 assay runs for each MIKK panel line that was available, generating a minimum of 8 test fish replicates per line. As there were four pairs of fish in the test tank during each run, the complete dataset comprises 824 videos of pairs of fish, which I further divided by assay component (open field and novel object) to create 1648 videos.</p>
<p>I again used the software <em>idtrackerai</em><span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Romero-Ferrero et al., &lt;span&gt;“Idtracker.ai.”&lt;/span&gt;&lt;/p&gt;"><sup>81</sup></a></span> to track the movement of the fishes across frames of each video. After adjusting the software parameters for each video to maximise the number of frames that were successfully tracked, I was left with 1610 out of the 1648 videos (~97.7%) where both fishes were tracked over at least 85% of frames, and I only included these 1610 videos in the downstream analysis. The first question to address was whether the MIKK panel lines differed in their behaviours. I therefore computed each individual fish’s mean speed (measured as the distance traveled in pixels per 0.08 seconds) over the course of the full 20-minute video, grouped them by line, and plotted the results presented in <strong>Figure <a href="MIKK-F2-chap.html#fig:mikk-mean-speed">4.2</a></strong>. I continue to use the same order and colour palette for the MIKK panel lines as in this Figure throughout the rest of this Chapter.</p>

<div class="figure">
<span style="display:block;" id="fig:mikk-mean-speed"></span>
<img src="figs/mikk_behaviour/F0_line_mean_speed_0.08.png" alt="Mean speed of the MIKK panel and other strains over the course of the entire 20-minute video (measured as the distance traveled in pixels per 0.05 seconds). iCab fishes in the iCab-iCab control condition are at the top, the MIKK panel lines are sorted by their group median, and the Kiyosu closed capture and O. luzonensis fishes are at the bottom." width="100%"><p class="caption">
Figure 4.2: Mean speed of the MIKK panel and other strains over the course of the entire 20-minute video (measured as the distance traveled in pixels per 0.05 seconds). <em>iCab</em> fishes in the <em>iCab</em>-<em>iCab</em> control condition are at the top, the MIKK panel lines are sorted by their group median, and the Kiyosu closed capture and <em>O. luzonensis</em> fishes are at the bottom.
</p>
</div>
<p>This figure shows that there are clear differences between some MIKK panel lines at the extremes, and that the lines differ in the amount of within-line variance observed. This figure acted as a guide to determine which lines to select as the parental strains in the F2 cross. To identify genetic variants directly associated with bold-shy behaviours, I sought to select lines that showed either high or low levels of movement, and preferably low within-line variance.</p>

<div class="figure">
<span style="display:block;" id="fig:mikk-mean-speed-variance"></span>
<img src="figs/mikk_behaviour/line_mean_speed_variance_0.05_all.png" alt="Line median (vertical axis) and line variance (horizontal axis) for individual mean speed across the full 20-minute video (i.e. both the open field and novel object assay components)." width="100%"><p class="caption">
Figure 4.3: Line median (vertical axis) and line variance (horizontal axis) for individual mean speed across the full 20-minute video (i.e. both the open field and novel object assay components).
</p>
</div>
</div>
<div id="hmm-states" class="section level2" number="4.3">
<h2>
<span class="header-section-number">4.3</span> HMM states<a class="anchor" aria-label="anchor" href="#hmm-states"><i class="fas fa-link"></i></a>
</h2>
<p>To examine these behaviours at a finer resolution, as for the pilot study described in <a href="Pilot-chap.html#Pilot-chap">3</a>, I again applied a hidden markov model (<strong>HMM</strong>) to classify the fishes’ movements based on their distance and angle of travel between time intervals. I used the same method to select the best choice of time interval and number of states (<strong>Figure <a href="#mikk-param-comp"><strong>??</strong></a></strong>). Here I observed the same phenomenon where the parameter combinations that performed the best showed an asymmetry between some states that would make interpretation difficult. For example, a time interval of 0.08 seconds combined with a state space of 17 caused state 4 to appear to get carved out of state 3 (<strong>Figure <a href="MIKK-F2-chap.html#fig:mikk-hmm-asym">4.5</a></strong>).</p>
<div class="figure">
<span style="display:block;" id="fig:mikk-param-comp"></span>
<img src="figs/mikk_behaviour/compare_params.png" alt="(ref:mikk-param-comp)" width="100%"><p class="caption">
Figure 4.4: (ref:mikk-param-comp)
</p>
</div>

<div class="figure">
<span style="display:block;" id="fig:mikk-hmm-asym"></span>
<img src="figs/mikk_behaviour/0.08_17_polar_all_dge.png" alt="The best apparent combination of parameters (0.08 time interval with a 17-state space) created an asymmetry between states 3 and 4, which would causes difficulties in interpreting their biological relevance." width="100%"><p class="caption">
Figure 4.5: The best apparent combination of parameters (0.08 time interval with a 17-state space) created an asymmetry between states 3 and 4, which would causes difficulties in interpreting their biological relevance.
</p>
</div>
<p>The best combination of parameters without this asymmetry was a time interval of 0.05 seconds with a state space of 15 (see the polar plots for the states in <strong>Appendix @ref(fig:hmm-states-0.05)</strong>). However, due to a glitch in the video recording software, several videos recorded on 13 November 2019 were incorrectly recorded with a frame rate of 14 instead of the desired 30. The insufficient number of frames for those videos meant that it was impossible to measure the distance and angle of travel between a time interval as low as 0.05 seconds. So that these videos could be included in the dataset, I accordingly selected the combination of 15 states and a 0.08-second interval for all downstream analyses.</p>

<div class="figure">
<span style="display:block;" id="fig:mikk-hmm-sym"></span>
<img src="figs/mikk_behaviour/0.08_15_polar_all_dge.png" alt="The HMM states used for the downstream analysis, with the model classified based on the distance of travel (log10 pixels, radial axis) and angle of travel (angle). A straight forward movement would sit around 0°, a left movement around 270°, and a right movement around 90°." width="100%"><p class="caption">
Figure 4.6: The HMM states used for the downstream analysis, with the model classified based on the distance of travel (log<sub>10</sub> pixels, radial axis) and angle of travel (angle). A straight forward movement would sit around 0°, a left movement around 270°, and a right movement around 90°.
</p>
</div>
</div>
<div id="social-genetic-effects-1" class="section level2" number="4.4">
<h2>
<span class="header-section-number">4.4</span> Social genetic effects<a class="anchor" aria-label="anchor" href="#social-genetic-effects-1"><i class="fas fa-link"></i></a>
</h2>
<p>As discussed above, in this project our traits of interest include not only direct genetic behaviours, but also social genetic behaviours. We therefore sought to identify the MIKK panel lines that transmitted their behaviour onto the reference <em>iCab</em> tank partners either to the largest or smallest degrees. I formulated two methods to measure this, referred to as a) HMM state co-occupancy; and b) reference deviation. The first, HMM state co-occupancy, measured the proportions of time that the <em>iCab</em> reference fish spent in the same HMM state as its tank partner. The second, deviation of the reference fishes’ deviation from the behaviour exhibited in the control condition, seeks to quantify the extent to which the <em>iCab</em>’s behaviour changes when partnered with</p>
<div id="state-co-occupancy" class="section level3" number="4.4.1">
<h3>
<span class="header-section-number">4.4.1</span> State co-occupancy<a class="anchor" aria-label="anchor" href="#state-co-occupancy"><i class="fas fa-link"></i></a>
</h3>
<p><strong>Figure <a href="MIKK-F2-chap.html#fig:F0-sge-cooc-box">4.7</a></strong> sets out the proportions of total time for each assay sub-component that each pair of individual fish spent in the same HMM state, grouped by line and ranked in the same order as their group median for individual mean speed as shown above in <strong>Figure <a href="MIKK-F2-chap.html#fig:mikk-mean-speed">4.2</a></strong>. <strong><a href="MIKK-F2-chap.html#fig:F0-sge-cooc-box">4.7</a>A</strong> shows the data as boxplots, with <span class="math inline">\(p\)</span>-values from the Kruskal-Wallis test comparing all groups. <strong>Figure <a href="MIKK-F2-chap.html#fig:F0-sge-cooc-box">4.7</a>B</strong> shows the same data but with each group’s median represented by columns to make it easier to compare group medians. Of the slower-moving lines, <span style="color:#FF699C">8-2</span> and 18-2 tend to show relatively higher state co-occupancy in the open field component, but during the novel object component, the slow-moving line 139-4 has the highest median co-occupancy of all lines. Of the faster-moving lines, 43-2 and 13-2 showed the highest state co-occupancy during the open field assay component. However, the moderate-to-fast line 21-2 had relatively high state co-occupancy during both assay components.</p>

<div class="figure">
<span style="display:block;" id="fig:F0-sge-cooc-box"></span>
<img src="figs/mikk_behaviour/0.08_15_cooc_box_all.png" alt="Frequency of HMM state co-occupancy" width="100%"><p class="caption">
Figure 4.7: Frequency of HMM state co-occupancy
</p>
</div>
<p>To visualise which states are driving the higher co-occupancy measures, for a selection of lines I generated a heatmap of the states occupied simultaneously by the <em>iCab</em> reference and MIKK test fishes, combining the observations for all individuals within each test fish group (<strong>Figure <a href="#F0-sge-cooc-heat"><strong>??</strong></a></strong>). When <em>iCab</em> is paired with 18-2 or 8-2, the fishes most frequently occupy states 3 or 1 in both open field and novel object components. In pairings with line 139-4, while the test fishes remain in the still states 1 or 3 during the open field assay, <em>iCab</em> tends to be moving much faster in states 11 and 13. However, during the novel object component, they co-occupy state 3 more than in any other combination. This general pattern is observed with line 14-2 as well, albeit to a lesser extent. For 38-2, the fishes tend not to show a strong preference for co-occupying a particular state for either assay component, but the diagonal spread indicates that they tend to move at similar speeds. When paired with the faster moving 21-2, the novel object component appears to accentuate the co-occupancy of state 3 that is also observed in the open field component. Finally, when paired with line 40-1, in both assay components, both fishes show a strong preference for the faster-moving states.</p>

<div class="figure">
<span style="display:block;" id="fig:F0-sge-cooc-heat"></span>
<img src="figs/mikk_behaviour/0.08_15_cooc_heatmap.png" alt="Heatmaps for a selection of MIKK panel lines (including those ultimately selected as the parental strains in the F2 cross) showing the frequency of HMM states simultaneously occupied by the reference (x-axis) and test (y-axis) fishes, aggregated over all replicates in each line." width="100%"><p class="caption">
Figure 4.8: Heatmaps for a selection of MIKK panel lines (including those ultimately selected as the parental strains in the F2 cross) showing the frequency of HMM states simultaneously occupied by the reference (x-axis) and test (y-axis) fishes, aggregated over all replicates in each line.
</p>
</div>
</div>
<div id="deviation-of-icab-from-its-control-condition" class="section level3" number="4.4.2">
<h3>
<span class="header-section-number">4.4.2</span> Deviation of <em>iCab</em> from its control condition<a class="anchor" aria-label="anchor" href="#deviation-of-icab-from-its-control-condition"><i class="fas fa-link"></i></a>
</h3>
<p>The second method for quantifying the level of behavioural transmission from test fish to <em>iCab</em> reference fish was to determine the proportion of time that the <em>iCab</em> spent in a particular state when paired with another <em>iCab</em>, and then quantify the degree to which those proportions change when in the presence of a fish from another line (<strong>Figure <a href="MIKK-F2-chap.html#fig:F0-sge-deviation">4.9</a></strong>). <strong>Figure <a href="MIKK-F2-chap.html#fig:F0-sge-deviation">4.9</a>A</strong> presents boxplots for state frequencies for all <em>iCab</em> individuals in <em>iCab</em>-<em>iCab</em> pairings. I further calculated the state frequencies for all <em>iCab</em> reference fishes for all other MIKK line pairings. For each combination of assay component, line-pairing, and HMM state, I then ran Welch’s t-test<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Graeme D. Ruxton, &lt;span&gt;“The Unequal Variance t-Test Is an Underused Alternative to &lt;span&gt;Student&lt;/span&gt;’s t-Test and the &lt;span&gt;Mann&lt;/span&gt;–&lt;span&gt;Whitney U&lt;/span&gt; Test,”&lt;/span&gt; &lt;em&gt;Behavioral Ecology&lt;/em&gt; 17, no. 4 (July 1, 2006): 688–90, &lt;a href="https://doi.org/10.1093/beheco/ark016" role="doc-biblioref"&gt;https://doi.org/10.1093/beheco/ark016&lt;/a&gt;.&lt;/p&gt;'><sup>82</sup></a></span> comparing the proportions of time the <em>iCab</em> individuals spent in that state when paired with another <em>iCab</em>, against the proportions of time the <em>iCab</em> reference individuals spent in that state when paired with a different MIKK line. I then summed the t-statistics across states to generate a single metric for each combination of line and assay component, and plotted the results in <strong>Figure <a href="MIKK-F2-chap.html#fig:F0-sge-deviation">4.9</a>B</strong>.</p>

<div class="figure">
<span style="display:block;" id="fig:F0-sge-deviation"></span>
<img src="figs/mikk_behaviour/0.08_15_deviation.png" alt="Deviation of state frequency for iCab reference fishes when paired with MIKK panel lines relative to when paired with another iCab. A: Boxplots of HMM state frequency for iCab individuals when paired with another iCab. B:" width="100%"><p class="caption">
Figure 4.9: Deviation of state frequency for <em>iCab</em> reference fishes when paired with MIKK panel lines relative to when paired with another <em>iCab</em>. <strong>A</strong>: Boxplots of HMM state frequency for <em>iCab</em> individuals when paired with another <em>iCab</em>. <strong>B</strong>:
</p>
</div>
<p>The first thing to note in this Figure is that <em>iCab</em>’s deviations from its behaviour exhibited in the control condition tend to be smaller when paired with faster-moving fishes. This was expected given <em>iCab</em> is also a faster-moving fish, but it strengthens the observation that when paired with slower-moving fishes, those tank partners are causing the <em>iCab</em> reference fish to move slower than they would otherwise. Of the slower-moving MIKK lines, 4-2 and 14-2 causing the largest deviations of the <em>iCab</em> reference fishes’ behaviours from their control condition behaviour in the open field and novel object components respectively. I also observe large deviations for the moderately-moving 106-2 and 71-1 during the novel object components. There are no clear outliers for either assay component for any of the faster moving lines.</p>
</div>
</div>
<div id="selection-of-lines-for-the-f2-cross" class="section level2" number="4.5">
<h2>
<span class="header-section-number">4.5</span> Selection of lines for the F2 cross<a class="anchor" aria-label="anchor" href="#selection-of-lines-for-the-f2-cross"><i class="fas fa-link"></i></a>
</h2>
<p>On the basis of the above findings, I selected 6 MIKK panel lines as the parental lines for the F2 cross (<strong>Figure @ref(F0-line-mean-speed-select</strong>). Conceptually, I sought to select lines that diverged on two measures: a) bold-shy behaviours; and b) the extent to which the lines transmitted their behaviours onto their tank partners. As <em>iCab</em> is a fast-moving line, it was more difficult to detect instances where fast-moving MIKK panel lines were influencing its behaviour. I was therefore more confident of identifying slow-movement/high-charisma lines, and so to increase the likelihood of identifying genetic variants that are responsible for a stronger transmission of slow-moving behaviours, I chose two slow-movement/high-charisma lines for the F2 cross: 18-2 and 8-2. In the event that both lines possessed the same genetic variants that influence this behavioural transmission trait, it would vastly increase the power of detecting it during the genetic linkage analysis. Both these lines were one of the most slow-moving lines, had high levels of state co-occupancy during both assay components, and 18-2 caused a high level of deviation from the <em>iCab</em> control condition.</p>
<div class="figure">
<span style="display:block;" id="fig:F0-line-mean-speed-select"></span>
<img src="figs/mikk_behaviour/line_mean_speed_0.05_selected.png" alt="(ref:F0-line-mean-speed-select)" width="100%"><p class="caption">
Figure 4.10: (ref:F0-line-mean-speed-select)
</p>
</div>
<p>For the slow-movement/low-charisma line, I selected 50-2 which exhibited a moderately slow level of movement, low within-line variance for mean speed, and low measures for both state co-occupancy and <em>iCab</em> deviation. For the high-movement/high-charisma line, I selected 22-1. Despite its high within line variance (<strong>Figure <a href="MIKK-F2-chap.html#fig:F2-time-dge-of">4.12</a></strong>), this potentially made it easier to detect its social genetic effects, as the slower-moving individuals appeared to transmit those behaviours strongly to their <em>iCab</em> tank partners, giving it a high score among fast-moving lines for state co-occupancy across both assay components. For the high-movement/low-charisma line I selected 40-1, as it has low within-line variance, and low-to-moderate metrics for state co-occupancy and <em>iCab</em> deviation. However, I note that these measures may be confounded by the possibility that 40-1 behaves in a similar way to <em>iCab</em>, which would be difficult to determine whether it was behaving differently.</p>
<p>In addition to these extreme lines, I also selected a line that was intermediate for both traits in an attempt to avoid breeding incompatibilities that might arise from attempting to cross lines with such divergent behavioural traits. For this purpose I selected line 38-2 for its intermediate speed, low within-line variance for mean speed, and intermediate measures for HMM state co-occupancy and <em>iCab</em> deviation.</p>

<div class="figure">
<span style="display:block;" id="fig:F0-line-mean-speed-var-select"></span>
<img src="figs/mikk_behaviour/line_mean_speed_variance_selected.png" alt="Line median (vertical axis) and line variance (horizontal axis) for individual mean speed across the full 20-minute video (i.e. both the open field and novel object assay components), coloured for the lines selected as the parental strains for the F2 cross." width="100%"><p class="caption">
Figure 4.11: Line median (vertical axis) and line variance (horizontal axis) for individual mean speed across the full 20-minute video (i.e. both the open field and novel object assay components), coloured for the lines selected as the parental strains for the F2 cross.
</p>
</div>
</div>
<div id="direct-genetic-effects-1" class="section level2" number="4.6">
<h2>
<span class="header-section-number">4.6</span> Direct genetic effects<a class="anchor" aria-label="anchor" href="#direct-genetic-effects-1"><i class="fas fa-link"></i></a>
</h2>
<p>With these lines selected, I ran a similar analysis to what I described in <a href="Pilot-chap.html#Pilot-chap">3</a>, where I ran multi-way ANOVAs to determine whether certain lines differed in the proportions of time they spent in these HMM states, while including the date of assay, time of assay, tank quadrant and tank side as covariates. <strong>Table <a href="MIKK-F2-chap.html#tab:mikk-dge-F0">4.1</a></strong> sets out the states which showed a significant difference between these 6 lines, with p-values adjusted for the False Discovery Rate (<strong>FDR</strong>).</p>
<div class="inline-table"><table class="table table-sm">
<caption>
<span id="tab:mikk-dge-F0">Table 4.1: </span>Significant differences in the proportion of time spent in each HMM state across test fish lines selected for the F2 cross for the open field and novel object assay components.</caption>
<thead><tr class="header">
<th align="left">Assay</th>
<th align="left">State</th>
<th align="left">Variance explained (%)</th>
<th align="left">p-value (FDR-adjusted)</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">open field</td>
<td align="left">3</td>
<td align="left">21.53</td>
<td align="left">2.67e-03</td>
</tr>
<tr class="even">
<td align="left">open field</td>
<td align="left">4</td>
<td align="left">23.28</td>
<td align="left">1.60e-02</td>
</tr>
<tr class="odd">
<td align="left">open field</td>
<td align="left">5</td>
<td align="left">20.61</td>
<td align="left">3.15e-02</td>
</tr>
<tr class="even">
<td align="left">open field</td>
<td align="left">10</td>
<td align="left">29.06</td>
<td align="left">1.41e-03</td>
</tr>
<tr class="odd">
<td align="left">open field</td>
<td align="left">12</td>
<td align="left">24.91</td>
<td align="left">1.45e-02</td>
</tr>
<tr class="even">
<td align="left">open field</td>
<td align="left">14</td>
<td align="left">29.52</td>
<td align="left">1.29e-03</td>
</tr>
<tr class="odd">
<td align="left">novel object</td>
<td align="left">1</td>
<td align="left">16.21</td>
<td align="left">1.90e-02</td>
</tr>
<tr class="even">
<td align="left">novel object</td>
<td align="left">2</td>
<td align="left">15.14</td>
<td align="left">4.86e-02</td>
</tr>
<tr class="odd">
<td align="left">novel object</td>
<td align="left">4</td>
<td align="left">26.90</td>
<td align="left">5.21e-03</td>
</tr>
<tr class="even">
<td align="left">novel object</td>
<td align="left">5</td>
<td align="left">27.84</td>
<td align="left">4.27e-03</td>
</tr>
<tr class="odd">
<td align="left">novel object</td>
<td align="left">10</td>
<td align="left">23.73</td>
<td align="left">2.81e-02</td>
</tr>
</tbody>
</table></div>
<p><strong>Figures <a href="MIKK-F2-chap.html#fig:F2-time-dge-of">4.12</a></strong> and <strong><a href="MIKK-F2-chap.html#fig:F2-time-dge-no">4.13</a></strong> highlight the states that showed significant differences in the proportions of time that these lines spent in those states for the open field and novel object assay components respectively. In both figures, <strong>A</strong> highlights the significant states, <strong>B</strong> shows how the individuals moved through those states over the course of the 10-minute assay component, and <strong>C</strong> shows the densities of the significant states within each line. For the open field component, although the tile plots show a notable level of variance within each line, the density plots clarify how the three slow-moving lines (8-2, 18-2 and 50-2) spent little time in the fast-moving states 10, 12 and 14 relative to the fast-moving lines. Interestingly, the intermediate line 38-2 tended to transition into these states around the middle of the video, presumably a period of habituating to the new environment.</p>

<div class="figure">
<span style="display:block;" id="fig:F2-time-dge-of"></span>
<img src="figs/mikk_behaviour/select_0.08_15_dge_of.png" alt="Differences between MIKK F0 lines in the HMM states they occupied during the open field assay component. A: 15 HMM states with panels coloured red to indicate significant differences between MIKK F0 lines in the proportion of time spent in those states. B: Transitions between HMM states across time for each individual test fish, grouped by MIKK line Tiles are coloured by the state most frequently occupied by each fish within 2-second intervals. C: Densities within each MIKK line for the occupation of states that significantly differed between strains (colour), with other states consolidated (grey)." width="100%"><p class="caption">
Figure 4.12: Differences between MIKK F0 lines in the HMM states they occupied during the open field assay component. <strong>A</strong>: 15 HMM states with panels coloured red to indicate significant differences between MIKK F0 lines in the proportion of time spent in those states. <strong>B</strong>: Transitions between HMM states across time for each individual test fish, grouped by MIKK line Tiles are coloured by the state most frequently occupied by each fish within 2-second intervals. <strong>C</strong>: Densities within each MIKK line for the occupation of states that significantly differed between strains (colour), with other states consolidated (grey).
</p>
</div>
<p>In contrast, during the novel object component the HMM states that showed significant differences between lines</p>

<div class="figure">
<span style="display:block;" id="fig:F2-time-dge-no"></span>
<img src="figs/mikk_behaviour/select_0.08_15_dge_no.png" alt="Differences between MIKK F0 lines in the HMM states they occupied during the novel object assay component. A: 15 HMM states with panels coloured red to indicate significant differences between MIKK F0 lines in the proportion of time spent in those states. B: Transitions between HMM states across time for each individual test fish, grouped by MIKK line Tiles are coloured by the state most frequently occupied by each fish within 2-second intervals. C: Densities within each MIKK line for the occupation of states that significantly differed between strains (colour), with other states consolidated (grey)." width="100%"><p class="caption">
Figure 4.13: Differences between MIKK F0 lines in the HMM states they occupied during the novel object assay component. <strong>A</strong>: 15 HMM states with panels coloured red to indicate significant differences between MIKK F0 lines in the proportion of time spent in those states. <strong>B</strong>: Transitions between HMM states across time for each individual test fish, grouped by MIKK line Tiles are coloured by the state most frequently occupied by each fish within 2-second intervals. <strong>C</strong>: Densities within each MIKK line for the occupation of states that significantly differed between strains (colour), with other states consolidated (grey).
</p>
</div>
<div id="social-genetic-effects-2" class="section level3" number="4.6.1">
<h3>
<span class="header-section-number">4.6.1</span> Social genetic effects<a class="anchor" aria-label="anchor" href="#social-genetic-effects-2"><i class="fas fa-link"></i></a>
</h3>

<div class="figure">
<span style="display:block;" id="fig:F2-time-sge-of"></span>
<img src="figs/mikk_behaviour/select_0.08_15_sge_of.png" alt="Differences between MIKK F0 lines in the HMM states they occupied during the open field assay component. A: 15 HMM states with panels coloured red to indicate significant differences between MIKK F0 lines in the proportion of time spent in those states. B: Transitions between HMM states across time for each individual test fish, grouped by MIKK line Tiles are coloured by the state most frequently occupied by each fish within 2-second intervals. C: Densities within each MIKK line for the occupation of states that significantly differed between strains (colour), with other states consolidated (grey)." width="100%"><p class="caption">
Figure 4.14: Differences between MIKK F0 lines in the HMM states they occupied during the open field assay component. <strong>A</strong>: 15 HMM states with panels coloured red to indicate significant differences between MIKK F0 lines in the proportion of time spent in those states. <strong>B</strong>: Transitions between HMM states across time for each individual test fish, grouped by MIKK line Tiles are coloured by the state most frequently occupied by each fish within 2-second intervals. <strong>C</strong>: Densities within each MIKK line for the occupation of states that significantly differed between strains (colour), with other states consolidated (grey).
</p>
</div>

<div class="figure">
<span style="display:block;" id="fig:F2-time-sge-no"></span>
<img src="figs/mikk_behaviour/select_0.08_15_sge_no.png" alt="Differences between MIKK F0 lines in the HMM states they occupied during the novel object assay component. A: 15 HMM states with panels coloured red to indicate significant differences between MIKK F0 lines in the proportion of time spent in those states. B: Transitions between HMM states across time for each individual test fish, grouped by MIKK line Tiles are coloured by the state most frequently occupied by each fish within 2-second intervals. C: Densities within each MIKK line for the occupation of states that significantly differed between strains (colour), with other states consolidated (grey)." width="100%"><p class="caption">
Figure 4.15: Differences between MIKK F0 lines in the HMM states they occupied during the novel object assay component. <strong>A</strong>: 15 HMM states with panels coloured red to indicate significant differences between MIKK F0 lines in the proportion of time spent in those states. <strong>B</strong>: Transitions between HMM states across time for each individual test fish, grouped by MIKK line Tiles are coloured by the state most frequently occupied by each fish within 2-second intervals. <strong>C</strong>: Densities within each MIKK line for the occupation of states that significantly differed between strains (colour), with other states consolidated (grey).
</p>
</div>
</div>
</div>
<div id="f2-generation" class="section level2" number="4.7">
<h2>
<span class="header-section-number">4.7</span> F2 generation<a class="anchor" aria-label="anchor" href="#f2-generation"><i class="fas fa-link"></i></a>
</h2>
<div id="behavioural-data-collection" class="section level3" number="4.7.1">
<h3>
<span class="header-section-number">4.7.1</span> Behavioural data collection<a class="anchor" aria-label="anchor" href="#behavioural-data-collection"><i class="fas fa-link"></i></a>
</h3>
<p>Around August 2019, our collaborators in the Loosli Group at KIT commenced the breeding program for this experiment with the 6 MIKK panel lines I had selected above.<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Although the breeding program began in 2019, we understand from our collaborators that from around mid-2020 the F1 generation was producing poorly. After lengthy investigations, they discovered that the Covid pandemic had caused disruptions to the supply chains of their fish food manufacturer, which had compelled them to change the recipe. This change in the fish food recipe was the cause of the poor breeding. While the issue has since been resolved, it has resulted in a much smaller number of F2 individuals than anticipated.&lt;/p&gt;"><sup>83</sup></a></p>
</div>
<div id="genotyping" class="section level3" number="4.7.2">
<h3>
<span class="header-section-number">4.7.2</span> Genotyping<a class="anchor" aria-label="anchor" href="#genotyping"><i class="fas fa-link"></i></a>
</h3>
</div>
<div id="genetic-linkage-analysis" class="section level3" number="4.7.3">
<h3>
<span class="header-section-number">4.7.3</span> Genetic linkage analysis<a class="anchor" aria-label="anchor" href="#genetic-linkage-analysis"><i class="fas fa-link"></i></a>
</h3>
</div>
<div id="candidate-snps-for-crispr-knockouts" class="section level3" number="4.7.4">
<h3>
<span class="header-section-number">4.7.4</span> Candidate SNPs for CRISPR-knockouts<a class="anchor" aria-label="anchor" href="#candidate-snps-for-crispr-knockouts"><i class="fas fa-link"></i></a>
</h3>
</div>
</div>
<div id="discussion-1" class="section level2" number="4.8">
<h2>
<span class="header-section-number">4.8</span> Discussion<a class="anchor" aria-label="anchor" href="#discussion-1"><i class="fas fa-link"></i></a>
</h2>
</div>
<div id="future-directions" class="section level2" number="4.9">
<h2>
<span class="header-section-number">4.9</span> Future directions<a class="anchor" aria-label="anchor" href="#future-directions"><i class="fas fa-link"></i></a>
</h2>
</div>
<div id="lessons" class="section level2" number="4.10">
<h2>
<span class="header-section-number">4.10</span> Lessons<a class="anchor" aria-label="anchor" href="#lessons"><i class="fas fa-link"></i></a>
</h2>
<ul>
<li><p>Describe using a different MIKK panel line as the reference, preferring a moderate-moving line to enable easier detection of social genetic effects from fast-moving lines, i.e. which lines cause <em>iCab</em> to move more boldly?</p></li>
<li><p>Chosen different strains for the F2 cross. 139-4 had low within-line variance for mean speed, and the highest median state co-occupancy in the novel object component. From the conclusions of Chapter <a href="Pilot-chap.html#Pilot-chap">3</a>, which were carried out after the F2 cross lines were already selected, it became apparent that the novel object assay was particularly useful for revealing social genetic effected. I hypothesise that at times of higher stress or predation threat, the individual fish take more behavioural cues from their tank partners. It therefore would have been a good candidate for the slow-moving, high-charisma line.</p></li>
<li><p>Explain why I didn’t choose 22-1 or 10-1.</p></li>
</ul>
</div>
</div>

  <div class="chapter-nav">
<div class="prev"><a href="Pilot-chap.html"><span class="header-section-number">3</span> Classification of bold/shy behaviours in 5 inbred medaka lines</a></div>
<div class="next"><a href="genetic-loci-associated-with-somite-development-periodicity.html"><span class="header-section-number">5</span> Genetic loci associated with somite development periodicity</a></div>
</div></main><div class="col-md-3 col-lg-2 d-none d-md-block sidebar sidebar-chapter">
    <nav id="toc" data-toggle="toc" aria-label="On this page"><h2>On this page</h2>
      <ul class="nav navbar-nav">
<li><a class="nav-link" href="#MIKK-F2-chap"><span class="header-section-number">4</span> Genetic linkage study of bold/shy behaviours in the MIKK panel</a></li>
<li><a class="nav-link" href="#MIKK-F2-cross"><span class="header-section-number">4.1</span> The F2 cross experimental setup</a></li>
<li><a class="nav-link" href="#data-collection---f0-generation"><span class="header-section-number">4.2</span> Data collection - F0 generation</a></li>
<li><a class="nav-link" href="#hmm-states"><span class="header-section-number">4.3</span> HMM states</a></li>
<li>
<a class="nav-link" href="#social-genetic-effects-1"><span class="header-section-number">4.4</span> Social genetic effects</a><ul class="nav navbar-nav">
<li><a class="nav-link" href="#state-co-occupancy"><span class="header-section-number">4.4.1</span> State co-occupancy</a></li>
<li><a class="nav-link" href="#deviation-of-icab-from-its-control-condition"><span class="header-section-number">4.4.2</span> Deviation of iCab from its control condition</a></li>
</ul>
</li>
<li><a class="nav-link" href="#selection-of-lines-for-the-f2-cross"><span class="header-section-number">4.5</span> Selection of lines for the F2 cross</a></li>
<li>
<a class="nav-link" href="#direct-genetic-effects-1"><span class="header-section-number">4.6</span> Direct genetic effects</a><ul class="nav navbar-nav"><li><a class="nav-link" href="#social-genetic-effects-2"><span class="header-section-number">4.6.1</span> Social genetic effects</a></li></ul>
</li>
<li>
<a class="nav-link" href="#f2-generation"><span class="header-section-number">4.7</span> F2 generation</a><ul class="nav navbar-nav">
<li><a class="nav-link" href="#behavioural-data-collection"><span class="header-section-number">4.7.1</span> Behavioural data collection</a></li>
<li><a class="nav-link" href="#genotyping"><span class="header-section-number">4.7.2</span> Genotyping</a></li>
<li><a class="nav-link" href="#genetic-linkage-analysis"><span class="header-section-number">4.7.3</span> Genetic linkage analysis</a></li>
<li><a class="nav-link" href="#candidate-snps-for-crispr-knockouts"><span class="header-section-number">4.7.4</span> Candidate SNPs for CRISPR-knockouts</a></li>
</ul>
</li>
<li><a class="nav-link" href="#discussion-1"><span class="header-section-number">4.8</span> Discussion</a></li>
<li><a class="nav-link" href="#future-directions"><span class="header-section-number">4.9</span> Future directions</a></li>
<li><a class="nav-link" href="#lessons"><span class="header-section-number">4.10</span> Lessons</a></li>
</ul>

      <div class="book-extra">
        <ul class="list-unstyled">
<li><a id="book-source" href="https://github.com/brettellebi/PhD-thesis/blob/master/book/04-MIKK_F2.Rmd">View source <i class="fab fa-github"></i></a></li>
          <li><a id="book-edit" href="https://github.com/brettellebi/PhD-thesis/edit/master/book/04-MIKK_F2.Rmd">Edit this page <i class="fab fa-github"></i></a></li>
        </ul>
</div>
    </nav>
</div>

</div>
</div> <!-- .container -->

<footer class="bg-primary text-light mt-5"><div class="container"><div class="row">

  <div class="col-12 col-md-6 mt-3">
    <p>"<strong>Japanese courage: a genetic analysis of complex traits in medaka fish and humans</strong>" was written by Ian Brettell. It was last built on 2022-08-16.</p>
  </div>

  <div class="col-12 col-md-6 mt-3">
    <p>This book was built by the <a class="text-light" href="https://bookdown.org">bookdown</a> R package.</p>
  </div>

</div></div>
</footer><!-- dynamically load mathjax for compatibility with self-contained --><script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script><script type="text/x-mathjax-config">const popovers = document.querySelectorAll('a.footnote-ref[data-toggle="popover"]');
for (let popover of popovers) {
  const div = document.createElement('div');
  div.setAttribute('style', 'position: absolute; top: 0, left:0; width:0, height:0, overflow: hidden; visibility: hidden;');
  div.innerHTML = popover.getAttribute('data-content');

  var has_math = div.querySelector("span.math");
  if (has_math) {
    document.body.appendChild(div);
    MathJax.Hub.Queue(["Typeset", MathJax.Hub, div]);
    MathJax.Hub.Queue(function() {
      popover.setAttribute('data-content', div.innerHTML);
      document.body.removeChild(div);
    })
  }
}
</script>
</body>
</html>
