<!DOCTYPE html>
<html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<title>Chapter 2 Genomic variations in the MIKK panel | Japanese courage: a genetic analysis of complex traits in medaka fish and humans</title>
<meta name="author" content="Ian Brettell">
<meta name="description" content="This project was carried out in collaboration with Felix Loosli’s group at the Karlsruhe Institute of Technology (KIT), and Joachim Wittbrodt’s group in the Centre for Organismal Studies (COS) at...">
<meta name="generator" content="bookdown 0.28 with bs4_book()">
<meta property="og:title" content="Chapter 2 Genomic variations in the MIKK panel | Japanese courage: a genetic analysis of complex traits in medaka fish and humans">
<meta property="og:type" content="book">
<meta property="og:description" content="This project was carried out in collaboration with Felix Loosli’s group at the Karlsruhe Institute of Technology (KIT), and Joachim Wittbrodt’s group in the Centre for Organismal Studies (COS) at...">
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="Chapter 2 Genomic variations in the MIKK panel | Japanese courage: a genetic analysis of complex traits in medaka fish and humans">
<meta name="twitter:description" content="This project was carried out in collaboration with Felix Loosli’s group at the Karlsruhe Institute of Technology (KIT), and Joachim Wittbrodt’s group in the Centre for Organismal Studies (COS) at...">
<!-- JS --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://kit.fontawesome.com/6ecbd6c532.js" crossorigin="anonymous"></script><script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="libs/bootstrap-4.6.0/bootstrap.min.css" rel="stylesheet">
<script src="libs/bootstrap-4.6.0/bootstrap.bundle.min.js"></script><script src="libs/bs3compat-0.4.0/transition.js"></script><script src="libs/bs3compat-0.4.0/tabs.js"></script><script src="libs/bs3compat-0.4.0/bs3compat.js"></script><link href="libs/bs4_book-1.0.0/bs4_book.css" rel="stylesheet">
<script src="libs/bs4_book-1.0.0/bs4_book.js"></script><style>
    #content {
        font-size: 0.9rem;
    }
    </style>
<script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- CSS --><style type="text/css">
    
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  </style>
<style type="text/css">
    /* Used with Pandoc 2.11+ new --citeproc when CSL is used */
    div.csl-bib-body { }
    div.csl-entry {
      clear: both;
        }
    .hanging div.csl-entry {
      margin-left:2em;
      text-indent:-2em;
    }
    div.csl-left-margin {
      min-width:2em;
      float:left;
    }
    div.csl-right-inline {
      margin-left:2em;
      padding-left:1em;
    }
    div.csl-indent {
      margin-left: 2em;
    }
  </style>
<link rel="stylesheet" href="style.css">
</head>
<body data-spy="scroll" data-target="#toc">

<div class="container-fluid">
<div class="row">
  <header class="col-sm-12 col-lg-3 sidebar sidebar-book"><a class="sr-only sr-only-focusable" href="#content">Skip to main content</a>

    <div class="d-flex align-items-start justify-content-between">
      <h1>
        <a href="index.html" title="">Japanese courage: a genetic analysis of complex traits in medaka fish and humans</a>
      </h1>
      <button class="btn btn-outline-primary d-lg-none ml-2 mt-1" type="button" data-toggle="collapse" data-target="#main-nav" aria-expanded="true" aria-controls="main-nav"><i class="fas fa-bars"></i><span class="sr-only">Show table of contents</span></button>
    </div>

    <div id="main-nav" class="collapse-lg">
      <form role="search">
        <input id="search" class="form-control" type="search" placeholder="Search" aria-label="Search">
</form>

      <nav aria-label="Table of contents"><h2>Table of contents</h2>
        <ul class="book-toc list-unstyled">
<li><a class="" href="index.html">About</a></li>
<li><a class="" href="Introduction.html"><span class="header-section-number">1</span> Introduction</a></li>
<li><a class="active" href="MIKK-genomes-chap.html"><span class="header-section-number">2</span> Genomic variations in the MIKK panel</a></li>
<li><a class="" href="Pilot-chap.html"><span class="header-section-number">3</span> Classification of bold/shy behaviours in 5 inbred medaka lines</a></li>
<li><a class="" href="MIKK-F2-chap.html"><span class="header-section-number">4</span> Genetic linkage study of bold/shy behaviours in the MIKK panel</a></li>
<li><a class="" href="Somite-chap.html"><span class="header-section-number">5</span> Genetic loci associated with somite development periodicity</a></li>
<li><a class="" href="Fst-chap.html"><span class="header-section-number">6</span> Variation in the frequency of trait-associated alleles across global human populations</a></li>
<li><a class="" href="references.html">References</a></li>
<li class="book-part">Appendix</li>
<li><a class="" href="chapter-refmikk-f2-chap-supplementary-information.html"><span class="header-section-number">A</span> Chapter @ref(MIKK-F2-chap): supplementary information</a></li>
<li><a class="" href="supplmentary-information-for-chapter-reffst-chap.html"><span class="header-section-number">B</span> Supplmentary information for Chapter @ref(Fst-chap)</a></li>
</ul>

        <div class="book-extra">
          <p><a id="book-repo" href="https://github.com/brettellebi/PhD-thesis">View book source <i class="fab fa-github"></i></a></p>
        </div>
      </nav>
</div>
  </header><main class="col-sm-12 col-md-9 col-lg-7" id="content"><div id="MIKK-genomes-chap" class="section level1" number="2">
<h1>
<span class="header-section-number">2</span> Genomic variations in the MIKK panel<a class="anchor" aria-label="anchor" href="#MIKK-genomes-chap"><i class="fas fa-link"></i></a>
</h1>
<p>This project was carried out in collaboration with Felix Loosli’s group at the Karlsruhe Institute of Technology (KIT), and Joachim Wittbrodt’s group in the Centre for Organismal Studies (COS) at the University of Heidelberg.</p>
<p>This chapter sets out my contributions to the the following pair of papers published in the journal <em>Genome Biology</em>, on both of which I am joint-first author:</p>
<ul>
<li><p><span class="citation">Tomas Fitzgerald et al.<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;&lt;span&gt;“The &lt;span&gt;Medaka Inbred Kiyosu-Karlsruhe&lt;/span&gt; (&lt;span&gt;MIKK&lt;/span&gt;) Panel,”&lt;/span&gt; &lt;em&gt;Genome Biology&lt;/em&gt; 23, no. 1 (February 21, 2022): 59, &lt;a href="https://doi.org/10.1186/s13059-022-02623-z" role="doc-biblioref"&gt;https://doi.org/10.1186/s13059-022-02623-z&lt;/a&gt;.&lt;/p&gt;'><sup>14</sup></a></span></p></li>
<li><p><span class="citation">Adrien Leger et al.<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;&lt;span&gt;“Genomic Variations and Epigenomic Landscape of the &lt;span&gt;Medaka Inbred Kiyosu-Karlsruhe&lt;/span&gt; (&lt;span&gt;MIKK&lt;/span&gt;) Panel,”&lt;/span&gt; &lt;em&gt;Genome Biology&lt;/em&gt; 23, no. 1 (February 21, 2022): 58, &lt;a href="https://doi.org/10.1186/s13059-022-02602-4" role="doc-biblioref"&gt;https://doi.org/10.1186/s13059-022-02602-4&lt;/a&gt;.&lt;/p&gt;'><sup>15</sup></a></span></p></li>
</ul>
<div id="the-medaka-inbred-kiyosu-karlruhe-mikk-panel" class="section level2" number="2.1">
<h2>
<span class="header-section-number">2.1</span> The Medaka Inbred Kiyosu-Karlruhe (MIKK) panel<a class="anchor" aria-label="anchor" href="#the-medaka-inbred-kiyosu-karlruhe-mikk-panel"><i class="fas fa-link"></i></a>
</h2>
<p>Biological traits are the product of an interaction between an organism’s genes and its environment, often described as the relationship between “nature and nurture”.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Robert Plomin and Kathryn Asbury, &lt;span&gt;“Nature and &lt;span&gt;Nurture&lt;/span&gt;: &lt;span&gt;Genetic&lt;/span&gt; and &lt;span&gt;Environmental Influences&lt;/span&gt; on &lt;span&gt;Behavior&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;The ANNALS of the American Academy of Political and Social Science&lt;/em&gt; 600, no. 1 (July 1, 2005): 86–98, &lt;a href="https://doi.org/10.1177/0002716205277184" role="doc-biblioref"&gt;https://doi.org/10.1177/0002716205277184&lt;/a&gt;.&lt;/p&gt;'><sup>16</sup></a></span> This is especially true for complex traits such as behaviour, which I investigate in Chapters <a href="Pilot-chap.html#Pilot-chap">3</a> and <a href="MIKK-F2-chap.html#MIKK-F2-chap">4</a>.</p>
<p>It is unfeasible to explore the relationship between genes and environment experimentally in humans due to the insufficient ability to manipulate either set of variables. Researchers accordingly resort to using model organisms, with which it is possible to control for both. The genetics of model organisms may be controlled to a degree by establishing inbred strains through the repeated mating of siblings over successive generations. Eventually, as the individuals within each line inherit the same same haplotype from their related parents, they become almost genetically identical to one another, with the added benefit that their genotypes can be replicated across time in subsequent generations. This utility has led to the establishment of “panels” of inbred strains for several model organisms including the thale cress (<em>Arabidopsis thaliana</em>),<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Joy Bergelson and Fabrice Roux, &lt;span&gt;“Towards Identifying Genes Underlying Ecologically Relevant Traits in &lt;span&gt;Arabidopsis&lt;/span&gt; Thaliana,”&lt;/span&gt; &lt;em&gt;Nature Reviews Genetics&lt;/em&gt; 11, no. 12, 12 (December 2010): 867–79, &lt;a href="https://doi.org/10.1038/nrg2896" role="doc-biblioref"&gt;https://doi.org/10.1038/nrg2896&lt;/a&gt;.&lt;/p&gt;'><sup>17</sup></a></span> common bean (<em>Phaseolus vulgaris L</em>),<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;William C. Johnson and Paul Gepts, &lt;span&gt;“Segregation for Performance in Recombinant Inbred Populations Resulting from Inter-Gene Pool Crosses of Common Bean ( &lt;span&gt;Phaseolus&lt;/span&gt; Vulgaris &lt;span&gt;L&lt;/span&gt;.),”&lt;/span&gt; &lt;em&gt;Euphytica&lt;/em&gt; 106, no. 1 (March 1, 1999): 45–56, &lt;a href="https://doi.org/10.1023/A:1003541201923" role="doc-biblioref"&gt;https://doi.org/10.1023/A:1003541201923&lt;/a&gt;.&lt;/p&gt;'><sup>18</sup></a></span> tomato (<em>Lycopersicon esculentum</em>),<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Vera Saliba-Colombani et al., &lt;span&gt;“Efficiency of &lt;span&gt;RFLP&lt;/span&gt;, &lt;span&gt;RAPD&lt;/span&gt;, and &lt;span&gt;AFLP&lt;/span&gt; Markers for the Construction of an Intraspecific Map of the Tomato Genome,”&lt;/span&gt; &lt;em&gt;Genome&lt;/em&gt; 43, no. 1 (February 2000): 29–40, &lt;a href="https://doi.org/10.1139/g99-096" role="doc-biblioref"&gt;https://doi.org/10.1139/g99-096&lt;/a&gt;.&lt;/p&gt;'><sup>19</sup></a></span> maize (<em>Zea mays</em>),<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Anis M. Limami et al., &lt;span&gt;“Genetic and &lt;span&gt;Physiological Analysis&lt;/span&gt; of &lt;span&gt;Germination Efficiency&lt;/span&gt; in &lt;span&gt;Maize&lt;/span&gt; in &lt;span&gt;Relation&lt;/span&gt; to &lt;span&gt;Nitrogen Metabolism Reveals&lt;/span&gt; the &lt;span&gt;Importance&lt;/span&gt; of &lt;span&gt;Cytosolic Glutamine Synthetase&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Plant Physiology&lt;/em&gt; 130, no. 4 (December 1, 2002): 1860–70, &lt;a href="https://doi.org/10.1104/pp.009647" role="doc-biblioref"&gt;https://doi.org/10.1104/pp.009647&lt;/a&gt;.&lt;/p&gt;'><sup>20</sup></a></span> nematode (<em>Caenorhabditis elegans</em>),<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Kathryn S. Evans et al., &lt;span&gt;“From &lt;span&gt;QTL&lt;/span&gt; to Gene: &lt;span&gt;C&lt;/span&gt;. Elegans Facilitates Discoveries of the Genetic Mechanisms Underlying Natural Variation,”&lt;/span&gt; &lt;em&gt;Trends in Genetics&lt;/em&gt; 37, no. 10 (October 1, 2021): 933–47, &lt;a href="https://doi.org/10.1016/j.tig.2021.06.005" role="doc-biblioref"&gt;https://doi.org/10.1016/j.tig.2021.06.005&lt;/a&gt;.&lt;/p&gt;'><sup>21</sup></a></span> fruit fly (<em>Drosophila melanogaster</em>),<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Trudy F. C. Mackay and Wen Huang, &lt;span&gt;“Charting the Genotype–Phenotype Map: Lessons from the &lt;span&gt;Drosophila&lt;/span&gt; Melanogaster &lt;span&gt;Genetic Reference Panel&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;WIREs Developmental Biology&lt;/em&gt; 7, no. 1 (2018): e289, &lt;a href="https://doi.org/10.1002/wdev.289" role="doc-biblioref"&gt;https://doi.org/10.1002/wdev.289&lt;/a&gt;.&lt;/p&gt;'><sup>22</sup></a></span> and mouse (<em>Mus musculus</em>).<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Michael C. Saul et al., &lt;span&gt;“High-&lt;span&gt;Diversity Mouse Populations&lt;/span&gt; for &lt;span&gt;Complex Traits&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Trends in Genetics&lt;/em&gt; 35, no. 7 (July 1, 2019): 501–14, &lt;a href="https://doi.org/10.1016/j.tig.2019.04.003" role="doc-biblioref"&gt;https://doi.org/10.1016/j.tig.2019.04.003&lt;/a&gt;.&lt;/p&gt;'><sup>23</sup></a></span></p>
<p>Although the mouse is an appropriate model for humans due to their orthologous mammalian organ systems and cell types, inbred strains of this organism descend from individuals that had already been domesticated, and therefore do not represent the genetic variation present in wild populations. Furthermore, the large panels of inbred mice such as the Collaborative Cross (CC),<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;David W. Threadgill et al., &lt;span&gt;“The &lt;span&gt;Collaborative Cross&lt;/span&gt;: &lt;span&gt;A Recombinant Inbred Mouse Population&lt;/span&gt; for the &lt;span&gt;Systems Genetic Era&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;ILAR Journal&lt;/em&gt; 52, no. 1 (January 1, 2011): 24–31, &lt;a href="https://doi.org/10.1093/ilar.52.1.24" role="doc-biblioref"&gt;https://doi.org/10.1093/ilar.52.1.24&lt;/a&gt;.&lt;/p&gt;'><sup>24</sup></a></span> Diversity Outcross (DO)<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Karen L Svenson et al., &lt;span&gt;“High-&lt;span&gt;Resolution Genetic Mapping Using&lt;/span&gt; the &lt;span&gt;Mouse Diversity Outbred Population&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Genetics&lt;/em&gt; 190, no. 2 (February 1, 2012): 437–47, &lt;a href="https://doi.org/10.1534/genetics.111.132597" role="doc-biblioref"&gt;https://doi.org/10.1534/genetics.111.132597&lt;/a&gt;.&lt;/p&gt;'><sup>25</sup></a></span> and B6-by-D2 (BXD)<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Jeremy L. Peirce et al., &lt;span&gt;“A New Set of &lt;span&gt;BXD&lt;/span&gt; Recombinant Inbred Lines from Advanced Intercross Populations in Mice,”&lt;/span&gt; &lt;em&gt;BMC Genetics&lt;/em&gt; 5, no. 1 (April 29, 2004): 7, &lt;a href="https://doi.org/10.1186/1471-2156-5-7" role="doc-biblioref"&gt;https://doi.org/10.1186/1471-2156-5-7&lt;/a&gt;.&lt;/p&gt;'><sup>26</sup></a></span> are derived from only a small number of individuals. As gene-environment studies seek to ultimately understand their effects on traits “in the wild” (such as with humans), there is accordingly a need for a panel of inbred vertebrates that represents the genetic variation present in natural populations.</p>
<p>The medaka fish (<em>Oryzias latipes</em>) has been studied as a model organism in Japan for over a century,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Joachim Wittbrodt, Akihiro Shima, and Manfred Schartl, &lt;span&gt;“Medaka — a Model Organism from the Far East,”&lt;/span&gt; &lt;em&gt;Nature Reviews Genetics&lt;/em&gt; 3, no. 1, 1 (January 2002): 53–64, &lt;a href="https://doi.org/10.1038/nrg704" role="doc-biblioref"&gt;https://doi.org/10.1038/nrg704&lt;/a&gt;.&lt;/p&gt;'><sup>27</sup></a></span> and is gaining recognition elsewhere as a powerful genetic model for vertebrates.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Mikhail Spivakov et al., &lt;span&gt;“Genomic and &lt;span&gt;Phenotypic Characterization&lt;/span&gt; of a &lt;span&gt;Wild Medaka Population&lt;/span&gt;: &lt;span&gt;Towards&lt;/span&gt; the &lt;span&gt;Establishment&lt;/span&gt; of an &lt;span&gt;Isogenic Population Genetic Resource&lt;/span&gt; in &lt;span&gt;Fish&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;G3 Genes|Genomes|Genetics&lt;/em&gt; 4, no. 3 (March 1, 2014): 433–45, &lt;a href="https://doi.org/10.1534/g3.113.008722" role="doc-biblioref"&gt;https://doi.org/10.1534/g3.113.008722&lt;/a&gt;.&lt;/p&gt;'><sup>28</sup></a></span> In addition to possessing a number of desirable traits that are characteristic of model organisms (including their small-size, short reproduction time, and high fertility), medaka are also – uniquely among vertebrates – resilient to inbreeding from the wild.</p>
<p>Since 2010, the Birney Group at EMBL-EBI, in collaboration with the Wittbrodt Group at COS, University of Heidelberg and the Loosli Group at the Karlsruhe Institute of Technology (KIT), have been working to establish the world’s first panel of vertebrate inbred strains – now known as the Medaka Inbred Kiyosu-Karlsruhe Panel (<strong>MIKK panel</strong>). The MIKK Panel was bred from a wild population caught near Kiyosu in Southern Japan, and now comprises 80 inbred, near-isogenic “lines”.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Fitzgerald et al., &lt;span&gt;“The &lt;span&gt;Medaka Inbred Kiyosu-Karlsruhe&lt;/span&gt; (&lt;span&gt;MIKK&lt;/span&gt;) Panel.”&lt;/span&gt;&lt;/p&gt;"><sup>29</sup></a></span></p>
<p>The MIKK Panel was created to map genetic variants associated with quantitative traits at a high resolution, and to explore the interactions between those variants and any environmental variables of interest. The purpose of the companion papers <span class="citation">Fitzgerald et al.<a href="#fn30" class="footnote-ref" id="fnref30"><sup>30</sup></a></span> and <span class="citation">Leger et al.<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;&lt;span&gt;“Genomic Variations and Epigenomic Landscape of the &lt;span&gt;Medaka Inbred Kiyosu-Karlsruhe&lt;/span&gt; (&lt;span&gt;MIKK&lt;/span&gt;) Panel.”&lt;/span&gt;&lt;/p&gt;"><sup>31</sup></a></span> was to introduce the MIKK panel to the scientific community, and describe the genetic characteristics of the MIKK panel that would make it a useful resource for other researchers who wish to explore the genetics of quantitative traits in vertebrates. My contributions to these papers involved visualising the inbreeding trajectory of the panel (Chapter <a href="MIKK-genomes-chap.html#inbreeding-sec">2.2.2</a>),
exploring the evolutionary history of the MIKK panel’s founding population (Chapter <a href="MIKK-genomes-chap.html#introgression-sec">2.2.3</a>), measuring the levels of homozygosity across the panel (Chapter <a href="MIKK-genomes-chap.html#nuc-div">2.2.4</a>), assessing its allele-frequency distribution and rate of linkage disequilibrium (LD) decay (Chapter <a href="MIKK-genomes-chap.html#ld-decay-sec">2.2.5</a>), and characterising the structural variants present in a smaller sample of lines using Oxford Nanopore long-read sequencing data (Chapter <a href="MIKK-genomes-chap.html#mikksv-sec">2.3</a>).</p>
</div>
<div id="genomic-characterisation-of-the-mikk-panel" class="section level2" number="2.2">
<h2>
<span class="header-section-number">2.2</span> Genomic characterisation of the MIKK panel<a class="anchor" aria-label="anchor" href="#genomic-characterisation-of-the-mikk-panel"><i class="fas fa-link"></i></a>
</h2>
<div id="non-sib-calls" class="section level3" number="2.2.1">
<h3>
<span class="header-section-number">2.2.1</span> MIKK panel DNA sequence dataset<a class="anchor" aria-label="anchor" href="#non-sib-calls"><i class="fas fa-link"></i></a>
</h3>
<p>For the preparation of <span class="citation">Fitzgerald et al.<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;&lt;span&gt;“The &lt;span&gt;Medaka Inbred Kiyosu-Karlsruhe&lt;/span&gt; (&lt;span&gt;MIKK&lt;/span&gt;) Panel.”&lt;/span&gt;&lt;/p&gt;"><sup>32</sup></a></span>, 79 of the 80 extant MIKK panel lines – together with several wild Kiyosu samples and individuals from the established <em>iCab</em> medaka strain – had their DNA sequenced from brain samples using Illumina short-read sequencing technology. Tomas Fitzgerald from the Birney Group at EMBL-EBI then aligned these sequences to the <em>HdrR</em> medaka reference and called variants to produce the <strong>MIKK Illumina call set</strong> in the form of a .vcf file containing single nucleotide polymorphism (SNP) and small insertion-deletion (INDEL) calls for each line. To avoid allele frequency biases introduced by the 16 pairs/triplets of “sibling lines” (see <a href="MIKK-genomes-chap.html#inbreeding-sec">2.2.2</a>), I removed each pair’s arbitrarily-labelled second sibling line from the variant call set, leaving 63 MIKK panel lines (<strong>MIKK non-sibling call set</strong>), and used only those calls for the analyses in Chapters <a href="MIKK-genomes-chap.html#nuc-div">2.2.4</a> and <a href="MIKK-genomes-chap.html#ld-decay-sec">2.2.5</a>.</p>
<p>For the preparation of <span class="citation">Leger et al.<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;&lt;span&gt;“Genomic Variations and Epigenomic Landscape of the &lt;span&gt;Medaka Inbred Kiyosu-Karlsruhe&lt;/span&gt; (&lt;span&gt;MIKK&lt;/span&gt;) Panel.”&lt;/span&gt;&lt;/p&gt;"><sup>33</sup></a></span>, 12 MIKK panel lines had their DNA sequenced from brain samples using Oxford Nanopore Technologies (ONT) long-read sequencing technology. Adrien Leger from the Birney Group at EMBL-EBI then aligned these sequences to the <em>HdrR</em> medaka reference, and called variants to produce the <strong>MIKK ONT call set</strong> in the form of a .vcf file containing structural variants calls for each line with tags for insertions (INS), deletions (DEL), duplications (SUP), inversions (INV) and translocations (TRA). The work described below used these variant call sets as the primary datasets.</p>
</div>
<div id="inbreeding-sec" class="section level3" number="2.2.2">
<h3>
<span class="header-section-number">2.2.2</span> Assessing the inbreeding trajectory of the MIKK panel<a class="anchor" aria-label="anchor" href="#inbreeding-sec"><i class="fas fa-link"></i></a>
</h3>
<p>The MIKK panel was bred from a wild population of medaka found in the Kiyosu area near Toyohashi, Aichi Prefecture, in southern Japan.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Spivakov et al., &lt;span&gt;“Genomic and &lt;span&gt;Phenotypic Characterization&lt;/span&gt; of a &lt;span&gt;Wild Medaka Population&lt;/span&gt;.”&lt;/span&gt;&lt;/p&gt;"><sup>34</sup></a></span> From this wild population, the Loosli Group at KIT set up random crosses of single mating pairs to create 115 ‘founder families’. For each founder family, they then set up between two and five single full-sibling-pair inbreeding crosses, which resulted in 253 F1 lines. Lines derived from the same founder family are referred to as ‘sibling lines’. Over the course of the next eight generations of inbreeding, they used only one mating pair per line. I generated <strong>Fig. <a href="MIKK-genomes-chap.html#fig:InbreedingFigure">2.1</a>A</strong> and <strong>B</strong> from the inbreeding data provided by the Loosli Group. <strong>Fig. <a href="MIKK-genomes-chap.html#fig:InbreedingFigure">2.1</a>A</strong> shows the number of lines that survived over the course of the first 14 generations of the inbreeding program, and the various causes for the termination of other lines. <strong>Fig. <a href="MIKK-genomes-chap.html#fig:InbreedingFigure">2.1</a>B</strong> shows the average fecundity levels of the surviving lines at generation F16. In addition, the Birney Group at EMBL-EBI generated morphometric data for the MIKK panel lines to demonstrate the distribution of physical phenotypes across the MIKK panel. I used this data on relative eye diameters to generate <strong>Fig. <a href="MIKK-genomes-chap.html#fig:InbreedingFigure">2.1</a>C</strong>.</p>

<div class="figure">
<span style="display:block;" id="fig:InbreedingFigure"></span>
<img src="figs/mikk_genome/20211213_final_figure.png" alt="Inbreeding, fecundity and eye size in the MIKK panel lines. A: Status of all MIKK panel lines during the first 14 generations of inbreeding, showing cause of death for non-extant lines. B: Average fecundity of MIKK panel lines in generation F16, as measured during peak egg production in July 2020. C: Distribution of mean relative eye size for female and male medaka across all MIKK panel lines." width="100%"><p class="caption">
Figure 2.1: Inbreeding, fecundity and eye size in the MIKK panel lines. <strong>A</strong>: Status of all MIKK panel lines during the first 14 generations of inbreeding, showing cause of death for non-extant lines. <strong>B</strong>: Average fecundity of MIKK panel lines in generation F16, as measured during peak egg production in July 2020. <strong>C</strong>: Distribution of mean relative eye size for female and male medaka across all MIKK panel lines.
</p>
</div>
</div>
<div id="introgression-sec" class="section level3" number="2.2.3">
<h3>
<span class="header-section-number">2.2.3</span> Introgression with northern Japanese and Korean medaka populations<a class="anchor" aria-label="anchor" href="#introgression-sec"><i class="fas fa-link"></i></a>
</h3>
<p>To explore the evolutionary history of the MIKK panel’s founding population, we sought to determine whether there was evidence of introgression between that southern Japanese population, and northern Japanese and Korean medaka populations. To this end, I used the 50-fish multiple alignment from Ensembl release 102 to obtain the aligned genome sequences for the established medaka inbred lines <em>HdrR</em> (southern Japan), <em>HNI</em> (northern Japan), and <em>HSOK</em> (Korea), as well as the most recent common ancestor of all three strains.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;&lt;span&gt;“Index of /Pub/Release-102/Emf/Ensembl-Compara/Multiple_alignments/50_fish.epo/,”&lt;/span&gt; accessed January 25, 2022, &lt;a href="https://ftp.ensembl.org/pub/release-102/emf/ensembl-compara/multiple_alignments/50_fish.epo/" role="doc-biblioref"&gt;https://ftp.ensembl.org/pub/release-102/emf/ensembl-compara/multiple_alignments/50_fish.epo/&lt;/a&gt;.&lt;/p&gt;'><sup>35</sup></a></span> Using the phylogenetic tree provided with the dataset, and the <em>ape</em> R package,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Emmanuel Paradis and Klaus Schliep, &lt;span&gt;“Ape 5.0: An Environment for Modern Phylogenetics and Evolutionary Analyses in &lt;span&gt;R&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Bioinformatics&lt;/em&gt; 35, no. 3 (February 1, 2019): 526–28, &lt;a href="https://doi.org/10.1093/bioinformatics/bty633" role="doc-biblioref"&gt;https://doi.org/10.1093/bioinformatics/bty633&lt;/a&gt;.&lt;/p&gt;'><sup>36</sup></a></span> I identified the most recent common ancestor of those three strains. For each locus with a non-missing base for <em>HdrR</em>, I assigned the allele in that ancestral sequence as the ‘ancestral’ allele, and the alternative allele as the ‘derived’ allele, and then combined that dataset with the MIKK Illumina call set and variant calls for the southern Japanese <em>iCab</em> strain (see <a href="MIKK-genomes-chap.html#non-sib-calls">2.2.1</a>).</p>
<p>I then carried out an ABBA BABA analysis to calculate a modified ‘admixture proportion’ statistic <span class="math inline">\(\hat{f}_d\)</span><span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Simon H. Martin, John W. Davey, and Chris D. Jiggins, &lt;span&gt;“Evaluating the &lt;span&gt;Use&lt;/span&gt; of &lt;span&gt;ABBA&lt;/span&gt;–&lt;span&gt;BABA Statistics&lt;/span&gt; to &lt;span&gt;Locate Introgressed Loci&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Molecular Biology and Evolution&lt;/em&gt; 32, no. 1 (January 1, 2015): 244–57, &lt;a href="https://doi.org/10.1093/molbev/msu269" role="doc-biblioref"&gt;https://doi.org/10.1093/molbev/msu269&lt;/a&gt;.&lt;/p&gt;'><sup>37</sup></a></span> as a measure of the proportion of shared genome in 500-kb sliding windows between the MIKK panel and either <em>iCab</em>, <em>HNI</em>, or <em>HSOK</em> (<strong>Fig. <a href="MIKK-genomes-chap.html#fig:ABBABABA">2.2</a></strong>), using the scripts provided by the first author of <span class="citation">Martin, Davey, and Jiggins<a href="#fn38" class="footnote-ref" id="fnref38"><sup>38</sup></a></span> on their GitHub page.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Simon martin, &lt;em&gt;Simonhmartin/Genomics_general&lt;/em&gt;, 2022, &lt;a href="https://github.com/simonhmartin/genomics_general" role="doc-biblioref"&gt;https://github.com/simonhmartin/genomics_general&lt;/a&gt;.&lt;/p&gt;'><sup>39</sup></a></span></p>

<div class="figure">
<span style="display:block;" id="fig:ABBABABA"></span>
<img src="figs/mikk_genome/07_introgression.png" alt="Figure 2: ABBA-BABA analysis. A. Phylogenetic tree generated from the Ensembl release 102 50-fish multiple alignment, showing only the medaka lines used in the ABBA-BABA analysis. B. Schema of the comparisons carried out in the ABBA-BABA analysis. C. Circos plot comparing introgression (\(\hat{f}_d\)) between the MIKK panel and either iCab (yellow), HNI (orange), or HSOK (purple), calculated within 500-kb sliding windows using a minimum of 250 SNPs per window." width="100%"><p class="caption">
Figure 2.2: <strong>Figure 2</strong>: ABBA-BABA analysis. <strong>A</strong>. Phylogenetic tree generated from the Ensembl release 102 50-fish multiple alignment, showing only the medaka lines used in the ABBA-BABA analysis. <strong>B</strong>. Schema of the comparisons carried out in the ABBA-BABA analysis. <strong>C</strong>. Circos plot comparing introgression (<span class="math inline">\(\hat{f}_d\)</span>) between the MIKK panel and either <em>iCab</em> (yellow), <em>HNI</em> (orange), or <em>HSOK</em> (purple), calculated within 500-kb sliding windows using a minimum of 250 SNPs per window.
</p>
</div>
<p>Based on the genome-wide mean <span class="math inline">\(\hat{f}_d\)</span>, the MIKK panel shares approximately 25% of its genome with <em>iCab</em>, 9% with <em>HNI</em>, and 12% with <em>HSOK</em>. These results provide evidence that the MIKK panel’s originating population has more recently introgressed with medaka from Korea than with medaka from northern Japan. This supports the findings in <span class="citation">Spivakov et al.<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;&lt;span&gt;“Genomic and &lt;span&gt;Phenotypic Characterization&lt;/span&gt; of a &lt;span&gt;Wild Medaka Population&lt;/span&gt;.”&lt;/span&gt;&lt;/p&gt;"><sup>40</sup></a></span>, where the authors found little evidence of significant interbreeding between southern and northern Japanese medaka since the populations diverged. Although the proportional difference between <em>HNI</em> and <em>HSOK</em> is small, this further supports the general finding that northern and southern Japanese medaka strains show low levels of interbreeding that may be a result of geographical isolation or genome divergence.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Takafumi Katsumura et al., &lt;span&gt;“Medaka &lt;span&gt;Population Genome Structure&lt;/span&gt; and &lt;span&gt;Demographic History Described&lt;/span&gt; via &lt;span class="nocase"&gt;Genotyping-by-Sequencing&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;G3 Genes|Genomes|Genetics&lt;/em&gt; 9, no. 1 (January 1, 2019): 217–28, &lt;a href="https://doi.org/10.1534/g3.118.200779" role="doc-biblioref"&gt;https://doi.org/10.1534/g3.118.200779&lt;/a&gt;.&lt;/p&gt;'><sup>41</sup></a></span></p>
</div>
<div id="nuc-div" class="section level3" number="2.2.4">
<h3>
<span class="header-section-number">2.2.4</span> Nucleotide diversity<a class="anchor" aria-label="anchor" href="#nuc-div"><i class="fas fa-link"></i></a>
</h3>
<p>As a means of assessing genetic diversity in the MIKK panel, I calculated nucleotide diversity (<span class="math inline">\(\hat{\pi}\)</span>) within 500-kb non-overlapping windows across the genome of the 63 lines in the MIKK non-sibling call set (see <a href="MIKK-genomes-chap.html#non-sib-calls">2.2.1</a>), and compared this to the nucleotide diversity in 7 wild medaka from the same Kiyosu population from which the MIKK panel was derived. Mean and median nucleotide diversity in both the MIKK panel and wild Kiyosu medaka were close to 0, and slightly higher in the MIKK panel (mean: MIKK = 0.0038, wild = 0.0037; median: MIKK = 0.0033, wild = 0.0031). The patterns of varying nucleotide diversity across the genome are shared between the MIKK panel and wild Kiyosu medaka, where regions with high levels of repeat content tend to have higher nucleotide diversity (r = 0.386, p &lt; 0.001) (<strong>Fig. <a href="MIKK-genomes-chap.html#fig:NucleotideDiversity">2.3</a></strong>). I also calculated <span class="math inline">\(\hat{\pi}\)</span> for each line individually, and as expected, levels of <span class="math inline">\(\hat{\pi}\)</span> around the (XX/XY) sex determination region of 1:~16-17 Mb are elevated in all lines relative to the consistently low levels found in most other chromosomes.</p>

<div class="figure">
<span style="display:block;" id="fig:NucleotideDiversity"></span>
<img src="figs/mikk_genome/supp_01_pi_circos.png" alt="Circos plot with nucleotide diversity (\(\hat{\pi}\)) calculated within 500-kb non-overlapping windows for 63 non-sibling lines from the MIKK panel (green) and 7 wild Kiyosu medaka samples from the same originating population (purple); proportion of sequence classified as repeats by RepeatMasker (blue); and mean mapping quality (pink)." width="100%"><p class="caption">
Figure 2.3: Circos plot with nucleotide diversity (<span class="math inline">\(\hat{\pi}\)</span>) calculated within 500-kb non-overlapping windows for 63 non-sibling lines from the MIKK panel (<em>green</em>) and 7 wild Kiyosu medaka samples from the same originating population (<em>purple</em>); proportion of sequence classified as repeats by RepeatMasker (<em>blue</em>); and mean mapping quality (<em>pink</em>).
</p>
</div>
<p>The higher level of <span class="math inline">\(\hat{\pi}\)</span> observed within specific regions on several chromosomes – such as chromosomes 2, 11, and 18 – correspond closely to the regions we identified as containing large (&gt;250 kb) inversions that appear to be shared across at least some of the MIKK panel (<strong>Fig. <a href="MIKK-genomes-chap.html#fig:SVInvs">2.4</a></strong>). These regions are also enriched for large deletions and duplications.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Leger et al., &lt;span&gt;“Genomic Variations and Epigenomic Landscape of the &lt;span&gt;Medaka Inbred Kiyosu-Karlsruhe&lt;/span&gt; (&lt;span&gt;MIKK&lt;/span&gt;) Panel.”&lt;/span&gt;&lt;/p&gt;"><sup>42</sup></a></span> Inversions cause permanent heterozygosity,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Ary A. Hoffmann, Carla M. Sgr‘o, and Andrew R. Weeks, &lt;span&gt;“Chromosomal Inversion Polymorphisms and Adaptation,”&lt;/span&gt; &lt;em&gt;Trends in Ecology &amp;amp; Evolution&lt;/em&gt; 19, no. 9 (September 1, 2004): 482–88, &lt;a href="https://doi.org/10.1016/j.tree.2004.06.013" role="doc-biblioref"&gt;https://doi.org/10.1016/j.tree.2004.06.013&lt;/a&gt;.&lt;/p&gt;'><sup>43</sup></a></span> and duplications and deletions may have increased the density of called SNPs in these regions,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;David Fredman et al., &lt;span&gt;“Complex &lt;span class="nocase"&gt;SNP-related&lt;/span&gt; Sequence Variation in Segmental Genome Duplications,”&lt;/span&gt; &lt;em&gt;Nature Genetics&lt;/em&gt; 36, no. 8, 8 (August 2004): 861–66, &lt;a href="https://doi.org/10.1038/ng1401" role="doc-biblioref"&gt;https://doi.org/10.1038/ng1401&lt;/a&gt;.&lt;/p&gt;'><sup>44</sup></a></span> so the observed depressions in homozygosity at these loci may be the result of such large structural variants that are present in the MIKK panel’s genomes.</p>
<p>Overall, this analysis confirms that the MIKK panel shows similar levels of homozygosity compared to classical laboratory inbred medaka strains, and possesses a strong increase in isogenic genotypes compared to wild medaka from the original wild population.</p>

<div class="figure">
<span style="display:block;" id="fig:SVInvs"></span>
<img src="figs/mikk_genome/20210224_sv_invs_lines.png" alt="Inversions identified in 9 MIKK panel lines using a combination of Oxford Nanopore Technologies long-read and Illumina short-read sequences (see Chapter 2.3 below)." width="100%"><p class="caption">
Figure 2.4: Inversions identified in 9 MIKK panel lines using a combination of Oxford Nanopore Technologies long-read and Illumina short-read sequences (see Chapter <a href="MIKK-genomes-chap.html#mikksv-sec">2.3</a> below).
</p>
</div>
</div>
<div id="ld-decay-sec" class="section level3" number="2.2.5">
<h3>
<span class="header-section-number">2.2.5</span> LD decay<a class="anchor" aria-label="anchor" href="#ld-decay-sec"><i class="fas fa-link"></i></a>
</h3>
<p>I analysed the MIKK panel’s allele frequency distribution and linkage disequilibrium (LD) structure to assess their likely effects on genetic mapping. To remove allele-frequency biases introduced by the presence of sibling lines in the MIKK panel, I used only the MIKK non-sibling call set (see Chapter <a href="MIKK-genomes-chap.html#non-sib-calls">2.2.1</a>).</p>
<p>To assess how accurately one may be able to map genetic variants using the MIKK panel relative to a human dataset, I compared the MIKK panel’s minor allele frequency (MAF) distribution and LD structure against that of the 2,504 humans in the 1KG Phase 3 release.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;&lt;span&gt;“A Global Reference for Human Genetic Variation,”&lt;/span&gt; &lt;em&gt;Nature&lt;/em&gt; 526, no. 7571 (2015): 68–74, &lt;a href="https://doi.org/10.1038/nature15393" role="doc-biblioref"&gt;https://doi.org/10.1038/nature15393&lt;/a&gt;.&lt;/p&gt;'><sup>45</sup></a></span> To prepare the <strong>“1KG call set”</strong>, I first downloaded the .vcf files for each autosome from the project’s FTP site (<a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/release/20130502/" class="uri">ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/release/20130502/</a>), then merged them into a single VCF using GATK.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Aaron McKenna et al., &lt;span&gt;“The &lt;span&gt;Genome Analysis Toolkit&lt;/span&gt;: &lt;span&gt;A MapReduce&lt;/span&gt; Framework for Analyzing Next-Generation &lt;span&gt;DNA&lt;/span&gt; Sequencing Data,”&lt;/span&gt; &lt;em&gt;Genome Research&lt;/em&gt; 20, no. 9 (January 9, 2010): 1297–1303, &lt;a href="https://doi.org/10.1101/gr.107524.110" role="doc-biblioref"&gt;https://doi.org/10.1101/gr.107524.110&lt;/a&gt;.&lt;/p&gt;'><sup>46</sup></a></span> I then used PLINK<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Christopher C Chang et al., &lt;span&gt;“Second-Generation &lt;span&gt;PLINK&lt;/span&gt;: Rising to the Challenge of Larger and Richer Datasets,”&lt;/span&gt; &lt;em&gt;GigaScience&lt;/em&gt; 4, no. 1 (December 2015): 7, &lt;a href="https://doi.org/10.1186/s13742-015-0047-8" role="doc-biblioref"&gt;https://doi.org/10.1186/s13742-015-0047-8&lt;/a&gt;; Shaun M Purcell and Christopher C Chang, &lt;em&gt;&lt;span&gt;PLINK&lt;/span&gt; 1.9&lt;/em&gt;, n.d., &lt;a href="https://www.cog-genomics.org/plink/1.9/" role="doc-biblioref"&gt;www.cog-genomics.org/plink/1.9/&lt;/a&gt;.&lt;/p&gt;'><sup>47</sup></a></span> to calculate the minor allele frequencies for all non-missing, biallelic SNPs in both the MIKK non-sibling and IKG call sets (<em>N</em> SNPs = 16,395,558 and 81,042,381 respectively) (<strong>Fig. <a href="MIKK-genomes-chap.html#fig:LDdecay">2.5</a>A</strong>). As expected, the 1KG and MIKK panel calls are similarly enriched for low-frequency variants, albeit to a lesser extent in the MIKK panel, which is likely due to its smaller sample size.</p>
<p>To determine the rate of LD decay in the MIKK panel and compare it to that in the 1KG sample, for both the MIKK non-sibling and 1KG call sets, I used PLINK to compute <span class="math inline">\(r^2\)</span> on each autosome for all pairs of non-missing, biallelic SNPs with MAF <span class="math inline">\(&gt;\)</span> 0.10 within 10 kb of one another (for 1KG and the MIKK panel respectively ~ 5.5M and ~ 3M SNPs, with a total number of pairwise <span class="math inline">\(r^2\)</span> observations of 204,152,922 and 146,785,673). I then grouped the <span class="math inline">\(r^2\)</span> observations for each pair of SNPs based on their distance from one another into non-overlapping bins of 100 bp in length, and calculated the mean <span class="math inline">\(r^2\)</span> in each of those bins to generate <strong>Fig. <a href="MIKK-genomes-chap.html#fig:LDdecay">2.5</a>B</strong> using the mean <span class="math inline">\(r^2\)</span> and left boundary of each bin.</p>

<div class="figure">
<span style="display:block;" id="fig:LDdecay"></span>
<img src="figs/mikk_genome/08_ld_decay.png" alt="Minor allele frequency distributions and LD decay for biallelic, non-missing SNPs in the 1000 Genomes Phase 3 variant calls (N = 2,504) (1KG), and the MIKK panel Illumina-based calls excluding one of each pair of sibling lines (N = 63), across all autosomes (1KG: chrs 1-22; MIKK: chrs 1-24). A: Histogram of allele frequencies in the 1KG and MIKK panel calls. B: LD decay for each autosome, calculated by taking the mean \(r^2\) of pairs of SNPs with MAF \(&gt;\) 0.1 within non-overlapping 100 bp windows of distance from one another, up to a maximum of 10 kb. LD decays faster on chromosome 2 for the MIKK panel due to its higher recombination rate." width="100%"><p class="caption">
Figure 2.5: Minor allele frequency distributions and LD decay for biallelic, non-missing SNPs in the 1000 Genomes Phase 3 variant calls (N = 2,504) (1KG), and the MIKK panel Illumina-based calls excluding one of each pair of sibling lines (N = 63), across all autosomes (1KG: chrs 1-22; MIKK: chrs 1-24). <strong>A</strong>: Histogram of allele frequencies in the 1KG and MIKK panel calls. <strong>B</strong>: LD decay for each autosome, calculated by taking the mean <span class="math inline">\(r^2\)</span> of pairs of SNPs with MAF <span class="math inline">\(&gt;\)</span> 0.1 within non-overlapping 100 bp windows of distance from one another, up to a maximum of 10 kb. LD decays faster on chromosome 2 for the MIKK panel due to its higher recombination rate.
</p>
</div>
<p>Based on the 1KG calls under these parameters, LD decays in humans to a mean <span class="math inline">\(r^2\)</span> of around 0.2-0.35 at a distance of 10 kb, whereas the MIKK panel reaches this level within 1 kb, with a mean <span class="math inline">\(r^2\)</span> of 0.3-0.4 at a distance of ~100 bp. This implies that when a causal variant is present in at least two lines in the MIKK panel, one may be able to map causal variants at a higher resolution than in humans. We note that LD decays faster in chromosome 2 of the MIKK panel relative to the other chromosomes. This suggests that it has a much higher recombination rate, which is consistent with the linkage map described in <span class="citation">Kiyoshi Naruse et al.<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;&lt;span&gt;“A &lt;span&gt;Detailed Linkage Map&lt;/span&gt; of &lt;span&gt;Medaka&lt;/span&gt;, &lt;span&gt;Oryzias&lt;/span&gt; Latipes: &lt;span&gt;Comparative Genomics&lt;/span&gt; and &lt;span&gt;Genome Evolution&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Genetics&lt;/em&gt; 154, no. 4 (April 1, 2000): 1773–84, &lt;a href="https://www.genetics.org/content/154/4/1773" role="doc-biblioref"&gt;https://www.genetics.org/content/154/4/1773&lt;/a&gt;.&lt;/p&gt;'><sup>48</sup></a></span>, showing a higher genetic distance per Mb for this chromosome. This higher recombination rate in chromosome 2 may in turn be caused by its relatively high proportion of repeat content (<strong>Fig. <a href="MIKK-genomes-chap.html#fig:repeats">2.6</a></strong>).</p>

<div class="figure">
<span style="display:block;" id="fig:repeats"></span>
<img src="figs/mikk_genome/20210401_hdrr_repeats_final.png" alt="Repeat content in the HdrR genome based on RepeatMasker results obtained by Jack Monahan. A. Proportion of repeat content per-chromosome. B. log10 of repeat lengths and counts per repeat class. “Misc” includes all repeats assigned to their own specific class, for example “(GAG)n” or “(GATCCA)n”. C. Circos plot showing repeat length (radial axes) by locus (angular axis) and repeat class (track)." width="100%"><p class="caption">
Figure 2.6: Repeat content in the <em>HdrR</em> genome based on RepeatMasker results obtained by Jack Monahan. <strong>A</strong>. Proportion of repeat content per-chromosome. <strong>B</strong>. log10 of repeat lengths and counts per repeat class. “Misc” includes all repeats assigned to their own specific class, for example “(GAG)n” or “(GATCCA)n”. <strong>C</strong>. Circos plot showing repeat length (radial axes) by locus (angular axis) and repeat class (track).
</p>
</div>
</div>
</div>
<div id="mikksv-sec" class="section level2" number="2.3">
<h2>
<span class="header-section-number">2.3</span> Structural variation in the MIKK panel<a class="anchor" aria-label="anchor" href="#mikksv-sec"><i class="fas fa-link"></i></a>
</h2>
<p>As an alternative to the variation pangenome approach described in <span class="citation">Leger et al.<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;&lt;span&gt;“Genomic Variations and Epigenomic Landscape of the &lt;span&gt;Medaka Inbred Kiyosu-Karlsruhe&lt;/span&gt; (&lt;span&gt;MIKK&lt;/span&gt;) Panel.”&lt;/span&gt;&lt;/p&gt;"><sup>49</sup></a></span>, I explored the structural variants (SVs) present in 9 of the MIKK panel lines in a reference-anchored manner, similar to many human studies. Differences in SVs between panel lines is another important class of genetic variation that could cause or contribute to significant phenotypic differences. Here we used ONT data obtained for 9 of the 12 selected lines allowing us to characterise larger SVs in the MIKK panel and to create a more extensive picture of genomic rearrangements compared to available medaka reference genomes. Adrien Leger from the Birney Group at EMBL-EBI first called structural variants using only the ONT long reads, producing a set of structural variants classified into five types: deletions (DEL), insertions (INS), translocations (TRA), duplications (DUP) and inversions (INV). I then “polished” the called DEL and INS variants with Illumina short reads to improve their accuracy. The polishing process filtered out 7.4% of DEL and 12.8% of INS variants, and adjusted the breakpoints (i.e. start and end positions) for 75-77% of DEL and INS variants in each sample by a mean of 23 bp for the start position, and 33 bp for the end position. This process produced a total of 143,326 filtered SVs.</p>
<p>The 9 “polished” samples contained a mean per-sample count of approximately 37K DEL variants (12% singletons), 29.5K INS variants (14%), 3.5K TRA variants (9%), 2.5K DUP (7%) and 600 INV (7%) (<strong>Fig. <a href="MIKK-genomes-chap.html#fig:SV-main">2.7</a>D</strong>). DEL variants were up to 494 kb in length, with 90% of unique DEL variants shorter than 3.8 kb. INS variants were only up to 13.8 kb in length, with 90% of unique INS variants shorter than 2 kb. DUP and INV variants tended to be longer, with a mean length of 19 and 70.5 kb respectively (<strong>Fig. <a href="MIKK-genomes-chap.html#fig:SV-main">2.7</a>A</strong>). <strong>Fig. <a href="MIKK-genomes-chap.html#fig:SV-main">2.7</a>E</strong> shows the per-sample distribution of DEL variants across the genome. Most large DEL variants over 250 kb in length were common among the MIKK panel lines. A number of large DEL variants appear to have accumulated within the 0-10 Mb region of chromosome 2, which is enriched for repeats in the HdrR reference genome (<strong>Fig. <a href="MIKK-genomes-chap.html#fig:repeats">2.6</a></strong>).</p>
<p>SVs were generally enriched in regions covered by repeats. While only 16% of bases in the <em>HdrR</em> reference were classified as repeats (irrespective of strand), those bases overlapped with 72% of DEL, 63% of DUP, 81% of INV and 35% of TRA variant regions. However, repeat bases only overlapped with 21% of INS variants. We also assessed each SV’s probability of being loss-of-function (pLI)<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Monkol Lek et al., &lt;span&gt;“Analysis of Protein-Coding Genetic Variation in 60,706 Humans,”&lt;/span&gt; &lt;em&gt;Nature&lt;/em&gt; 536, no. 7616, 7616 (August 2016): 285–91, &lt;a href="https://doi.org/10.1038/nature19057" role="doc-biblioref"&gt;https://doi.org/10.1038/nature19057&lt;/a&gt;.&lt;/p&gt;'><sup>50</sup></a></span> by calculating the logarithm of odds (LOD) for the pLI scores of all genes overlapping the variant (<strong>Fig. <a href="MIKK-genomes-chap.html#fig:SV-main">2.7</a>B,C</strong>). 30,357 out of 134,088 DEL, INS, DUP and INV variants overlapped at least one gene, and 9% of those had a score greater than 10, indicating a high probability that the SV would cause a loss of function. Two INS variants on chr2 had an outlying LOD score of 57 as a result of overlapping medaka gene ENSORLG00000003411, which has a pLI score of 1 – the highest intolerance to variants causing a loss of function. This gene is homologous with human genes <em>SCN1A</em>, <em>SCN2A</em> and <em>SCN3A</em>, which encode sodium channels and have been associated with neuronal and sleep disorders. We did not find evidence that longer SVs tended to have a higher probability of causing a loss of function (<strong>Fig. <a href="MIKK-genomes-chap.html#fig:SV-main">2.7</a>B</strong>).</p>

<div class="figure">
<span style="display:block;" id="fig:SV-main"></span>
<img src="figs/mikk_genome/20210408_sv_main.png" alt="Polished SVs in 9 MIKK panel lines sequenced with ONT. DEL: deletion; INS: insertion; TRA: translocation; DUP: duplication; INV: inversion. A. Aggregate log10 counts and lengths of distinct SVs by type, excluding TRA. B. pLI LOD scores in distinct SVs by SV type. C. Histogram of LOD scores by SV type. D. Total and singleton counts of SV types per sample. E. Circos plot showing per-sample distribution and lengths of DEL variants across the genome." width="100%"><p class="caption">
Figure 2.7: Polished SVs in 9 MIKK panel lines sequenced with ONT. DEL: deletion; INS: insertion; TRA: translocation; DUP: duplication; INV: inversion. <strong>A</strong>. Aggregate log10 counts and lengths of distinct SVs by type, excluding TRA. <strong>B</strong>. pLI LOD scores in distinct SVs by SV type. <strong>C</strong>. Histogram of LOD scores by SV type. <strong>D</strong>. Total and singleton counts of SV types per sample. <strong>E</strong>. Circos plot showing per-sample distribution and lengths of DEL variants across the genome.
</p>
</div>
<p>We compared these polished INS and DEL calls with the high-quality graph-based alternative paths and large-scale deletions, respectively (see section titled <em>Novel genetic sequences and large-scale insertions and deletions in the MIKK panel</em> in <span class="citation">Leger et al.<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;&lt;span&gt;“Genomic Variations and Epigenomic Landscape of the &lt;span&gt;Medaka Inbred Kiyosu-Karlsruhe&lt;/span&gt; (&lt;span&gt;MIKK&lt;/span&gt;) Panel.”&lt;/span&gt;&lt;/p&gt;"><sup>51</sup></a></span>). We found that 2 of the 19 regions covered by graph-based alternative paths, and 4 of the 16 regions covered by graph-based deletions, had no SVs that overlapped those regions at all, which suggests they would have been missed entirely when using a reference-anchored approach alone.</p>
<p>With the exception of one alternative path on chromosome 20, the alternative paths were not captured by INS variants, which only covered up to 63% of the bases in each region, and in many cases substantially less. On the other hand, for 8 of the 16 graph-based deletions, the DEL variants covered at least 85% of the bases in those regions. The other 8 graph-based deletions were either not at all covered by DEL variants, or only slightly. This indicates that the reference-based approach is better at detecting large-scale deletions than alternative paths (“insertions”), but still misses around half of such variants relative to the graph-based approach.</p>
</div>
<div id="conclusions" class="section level2" number="2.4">
<h2>
<span class="header-section-number">2.4</span> Conclusions<a class="anchor" aria-label="anchor" href="#conclusions"><i class="fas fa-link"></i></a>
</h2>
<p>Taken together, these analyses show that the MIKK panel is highly homozygous, with LD characteristics that will favour high-resolution genetic mapping relative to humans. In the future, the SV analysis performed on a subset of the MIKK panel will be expanded across the entire panel, which will permit the inclusion of both large- and small-scale variants in genetic linkage studies. I proceeded to use the MIKK panel to analyse bold/shy behaviours, as I describe in Chapter <a href="MIKK-F2-chap.html#MIKK-F2-chap">4</a>, with a view to carrying out an F2-cross linkage study to identify genetic variants associated with differences in the behaviours of an individual, and the extent to which they transmit those behaviours to their social companions. However, before carrying out this study, we first ran a “pilot” study on 5 previously-established inbred lines to validate our behavioural assay. This is the subject of the following chapter.</p>

</div>
</div>

  <div class="chapter-nav">
<div class="prev"><a href="Introduction.html"><span class="header-section-number">1</span> Introduction</a></div>
<div class="next"><a href="Pilot-chap.html"><span class="header-section-number">3</span> Classification of bold/shy behaviours in 5 inbred medaka lines</a></div>
</div></main><div class="col-md-3 col-lg-2 d-none d-md-block sidebar sidebar-chapter">
    <nav id="toc" data-toggle="toc" aria-label="On this page"><h2>On this page</h2>
      <ul class="nav navbar-nav">
<li><a class="nav-link" href="#MIKK-genomes-chap"><span class="header-section-number">2</span> Genomic variations in the MIKK panel</a></li>
<li><a class="nav-link" href="#the-medaka-inbred-kiyosu-karlruhe-mikk-panel"><span class="header-section-number">2.1</span> The Medaka Inbred Kiyosu-Karlruhe (MIKK) panel</a></li>
<li>
<a class="nav-link" href="#genomic-characterisation-of-the-mikk-panel"><span class="header-section-number">2.2</span> Genomic characterisation of the MIKK panel</a><ul class="nav navbar-nav">
<li><a class="nav-link" href="#non-sib-calls"><span class="header-section-number">2.2.1</span> MIKK panel DNA sequence dataset</a></li>
<li><a class="nav-link" href="#inbreeding-sec"><span class="header-section-number">2.2.2</span> Assessing the inbreeding trajectory of the MIKK panel</a></li>
<li><a class="nav-link" href="#introgression-sec"><span class="header-section-number">2.2.3</span> Introgression with northern Japanese and Korean medaka populations</a></li>
<li><a class="nav-link" href="#nuc-div"><span class="header-section-number">2.2.4</span> Nucleotide diversity</a></li>
<li><a class="nav-link" href="#ld-decay-sec"><span class="header-section-number">2.2.5</span> LD decay</a></li>
</ul>
</li>
<li><a class="nav-link" href="#mikksv-sec"><span class="header-section-number">2.3</span> Structural variation in the MIKK panel</a></li>
<li><a class="nav-link" href="#conclusions"><span class="header-section-number">2.4</span> Conclusions</a></li>
</ul>

      <div class="book-extra">
        <ul class="list-unstyled">
<li><a id="book-source" href="https://github.com/brettellebi/PhD-thesis/blob/master/book/02-MIKK_genome.Rmd">View source <i class="fab fa-github"></i></a></li>
          <li><a id="book-edit" href="https://github.com/brettellebi/PhD-thesis/edit/master/book/02-MIKK_genome.Rmd">Edit this page <i class="fab fa-github"></i></a></li>
        </ul>
</div>
    </nav>
</div>

</div>
</div> <!-- .container -->

<footer class="bg-primary text-light mt-5"><div class="container"><div class="row">

  <div class="col-12 col-md-6 mt-3">
    <p>"<strong>Japanese courage: a genetic analysis of complex traits in medaka fish and humans</strong>" was written by Ian Brettell. It was last built on 2022-08-20.</p>
  </div>

  <div class="col-12 col-md-6 mt-3">
    <p>This book was built by the <a class="text-light" href="https://bookdown.org">bookdown</a> R package.</p>
  </div>

</div></div>
</footer><!-- dynamically load mathjax for compatibility with self-contained --><script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.9/latest.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script><script type="text/x-mathjax-config">const popovers = document.querySelectorAll('a.footnote-ref[data-toggle="popover"]');
for (let popover of popovers) {
  const div = document.createElement('div');
  div.setAttribute('style', 'position: absolute; top: 0, left:0; width:0, height:0, overflow: hidden; visibility: hidden;');
  div.innerHTML = popover.getAttribute('data-content');

  var has_math = div.querySelector("span.math");
  if (has_math) {
    document.body.appendChild(div);
    MathJax.Hub.Queue(["Typeset", MathJax.Hub, div]);
    MathJax.Hub.Queue(function() {
      popover.setAttribute('data-content', div.innerHTML);
      document.body.removeChild(div);
    })
  }
}
</script>
</body>
</html>
