<!DOCTYPE html>
<html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<title>Chapter 5 Genetic linkage study of bold/shy behaviours in the MIKK panel | Japanese courage: a genetic analysis of complex traits in medaka fish and humans</title>
<meta name="author" content="Ian Brettell">
<meta name="description" content="The purpose of the study described in this chapter was to run the behavioural analysis described in Chapter 3 over the MIKK panel described in Chapter 2, identify the lines that diverged in both...">
<meta name="generator" content="bookdown 0.28 with bs4_book()">
<meta property="og:title" content="Chapter 5 Genetic linkage study of bold/shy behaviours in the MIKK panel | Japanese courage: a genetic analysis of complex traits in medaka fish and humans">
<meta property="og:type" content="book">
<meta property="og:description" content="The purpose of the study described in this chapter was to run the behavioural analysis described in Chapter 3 over the MIKK panel described in Chapter 2, identify the lines that diverged in both...">
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="Chapter 5 Genetic linkage study of bold/shy behaviours in the MIKK panel | Japanese courage: a genetic analysis of complex traits in medaka fish and humans">
<meta name="twitter:description" content="The purpose of the study described in this chapter was to run the behavioural analysis described in Chapter 3 over the MIKK panel described in Chapter 2, identify the lines that diverged in both...">
<!-- JS --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://kit.fontawesome.com/6ecbd6c532.js" crossorigin="anonymous"></script><script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="libs/bootstrap-4.6.0/bootstrap.min.css" rel="stylesheet">
<script src="libs/bootstrap-4.6.0/bootstrap.bundle.min.js"></script><script src="libs/bs3compat-0.4.0/transition.js"></script><script src="libs/bs3compat-0.4.0/tabs.js"></script><script src="libs/bs3compat-0.4.0/bs3compat.js"></script><link href="libs/bs4_book-1.0.0/bs4_book.css" rel="stylesheet">
<script src="libs/bs4_book-1.0.0/bs4_book.js"></script><style>
    #content {
        font-size: 0.9rem;
    }
    </style>
<script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- CSS --><style type="text/css">
    
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  </style>
<style type="text/css">
    /* Used with Pandoc 2.11+ new --citeproc when CSL is used */
    div.csl-bib-body { }
    div.csl-entry {
      clear: both;
        }
    .hanging div.csl-entry {
      margin-left:2em;
      text-indent:-2em;
    }
    div.csl-left-margin {
      min-width:2em;
      float:left;
    }
    div.csl-right-inline {
      margin-left:2em;
      padding-left:1em;
    }
    div.csl-indent {
      margin-left: 2em;
    }
  </style>
<link rel="stylesheet" href="style.css">
</head>
<body data-spy="scroll" data-target="#toc">

<div class="container-fluid">
<div class="row">
  <header class="col-sm-12 col-lg-3 sidebar sidebar-book"><a class="sr-only sr-only-focusable" href="#content">Skip to main content</a>

    <div class="d-flex align-items-start justify-content-between">
      <h1>
        <a href="index.html" title="">Japanese courage: a genetic analysis of complex traits in medaka fish and humans</a>
      </h1>
      <button class="btn btn-outline-primary d-lg-none ml-2 mt-1" type="button" data-toggle="collapse" data-target="#main-nav" aria-expanded="true" aria-controls="main-nav"><i class="fas fa-bars"></i><span class="sr-only">Show table of contents</span></button>
    </div>

    <div id="main-nav" class="collapse-lg">
      <form role="search">
        <input id="search" class="form-control" type="search" placeholder="Search" aria-label="Search">
</form>

      <nav aria-label="Table of contents"><h2>Table of contents</h2>
        <ul class="book-toc list-unstyled">
<li><a class="" href="index.html">About</a></li>
<li><a class="" href="Introduction.html"><span class="header-section-number">1</span> Introduction</a></li>
<li><a class="" href="MIKK-genomes-chap.html"><span class="header-section-number">2</span> Genomic variations in the MIKK panel</a></li>
<li><a class="" href="Pilot-chap.html"><span class="header-section-number">3</span> Classification of bold/shy behaviours in 5 inbred medaka lines</a></li>
<li><a class="" href="Somite-chap.html"><span class="header-section-number">4</span> Genetic loci associated with somite development periodicity</a></li>
<li><a class="active" href="MIKK-F2-chap.html"><span class="header-section-number">5</span> Genetic linkage study of bold/shy behaviours in the MIKK panel</a></li>
<li><a class="" href="Fst-chap.html"><span class="header-section-number">6</span> Variation in the frequency of trait-associated alleles across global human populations</a></li>
<li><a class="" href="references.html">References</a></li>
<li class="book-part">Appendix</li>
<li><a class="" href="hmm-states-with-0.05-second-interval.html"><span class="header-section-number">A</span> 15 HMM states with 0.05 second interval</a></li>
<li><a class="" href="four-panel-app.html"><span class="header-section-number">B</span> Four-panel plots for MIKK lines selected for the F2 cross</a></li>
<li><a class="" href="hmm-state-time-dependence-for-all-mikk-panel-lines.html"><span class="header-section-number">C</span> HMM state time dependence for all MIKK panel lines</a></li>
<li><a class="" href="f2-recombination-karyoplot-with-missing-calls.html"><span class="header-section-number">D</span> F2 recombination karyoplot with missing calls</a></li>
<li><a class="" href="loco-grm-for-chromosome-1.html"><span class="header-section-number">E</span> LOCO GRM for chromosome 1</a></li>
<li><a class="" href="ecdf-of-all-polygenic-traits-in-the-gwas-catalog-ranked-by-d_ts.html"><span class="header-section-number">F</span> eCDF of all polygenic traits in the GWAS Catalog ranked by \({D_t^S}\)</a></li>
</ul>

        <div class="book-extra">
          <p><a id="book-repo" href="https://github.com/brettellebi/PhD-thesis">View book source <i class="fab fa-github"></i></a></p>
        </div>
      </nav>
</div>
  </header><main class="col-sm-12 col-md-9 col-lg-7" id="content"><div id="MIKK-F2-chap" class="section level1" number="5">
<h1>
<span class="header-section-number">5</span> Genetic linkage study of bold/shy behaviours in the MIKK panel<a class="anchor" aria-label="anchor" href="#MIKK-F2-chap"><i class="fas fa-link"></i></a>
</h1>
<p>The purpose of the study described in this chapter was to run the behavioural analysis described in Chapter <a href="Pilot-chap.html#Pilot-chap">3</a> over the MIKK panel described in Chapter <a href="MIKK-genomes-chap.html#MIKK-genomes-chap">2</a>, identify the lines that diverged in both (a) their own behaviour; and (b) the level of transmission of their behaviour onto their <em><span style="color: #424B4D;">iCab</span></em> reference tank partner, and then use them as the parental strains in an F2 cross to attempt to identify the genetic variants associated with those differences.</p>
<div id="MIKK-F2-cross" class="section level2" number="5.1">
<h2>
<span class="header-section-number">5.1</span> The F2 cross experimental setup<a class="anchor" aria-label="anchor" href="#MIKK-F2-cross"><i class="fas fa-link"></i></a>
</h2>
<p>The F2 cross is a traditional method for mapping genetic variants associated with traits of interest. A schema for the method is presented in <strong>Figure <a href="Introduction.html#fig:F2-cross-schema">1.20</a></strong>. In essence, it involves starting with two inbred strains that diverge for the trait of interest (the ‘parental strains’, or F0 generation F0). One then crosses the parental strains to create a generation of F1 hybrid individuals who each possess, for every pair of their chromosomes, one chromosome from each of their parents. The individuals in this F1 generation are genetically identical to their parents with respect to their germ line. Finally, one inter-crosses the F1 generation to create a set of F2 individuals that share unique combinations of the original F0 strains’ genotypes, and tend to display values for the trait of interest that span across the spectrum between the extreme values of their parents.</p>
<p>Schema of the F2-cross experimental setup. The F0 generation comprises two medaka strains that have extreme, opposing values for a trait of interest, represented by the colours red and blue. Below them is an illustrative single pair of chromosomes. The chromosomes within each pair are depicted as the same colour, as the strains are homozygous through successive generations of inbreeding. Their F1 offspring is heterozygous for each pair of their 24 chromosomes, and all F1 individuals are therefore almost genetically identical to one another (with the exception of somatic mutations and the regions of the genome that were not homozygous in the parental generations). The F1 individuals are then inter-crossed with one another to produce the F2 generation, which, due to recombination events during gamete formation, have unique combinations of the parental strains’ genotypes, and tend to span the phenotypic spectrum between the extremes of their F0 parental strains, represented by their colours. Schema of the F2-cross experimental setup. The F0 generation comprises two medaka strains that have extreme, opposing values for a trait of interest, represented by the colours red and blue. Below them is an illustrative single pair of chromosomes. The chromosomes within each pair are depicted as the same colour, as the strains are homozygous through successive generations of inbreeding. Their F1 offspring is heterozygous for each pair of their 24 chromosomes, and all F1 individuals are therefore almost genetically identical to one another (with the exception of somatic mutations and the regions of the genome that were not homozygous in the parental generations). The F1 individuals are then inter-crossed with one another to produce the F2 generation, which, due to recombination events during gamete formation, have unique combinations of the parental strains’ genotypes, and tend to span the phenotypic spectrum between the extremes of their F0 parental strains, represented by their colours.</p>
<div class="figure">
<span style="display:block;" id="fig:F2-cross-schema"></span>
<img src="figs/mikk_behaviour/F2-cross-schema.png" alt="Schema of the F2-cross experimental setup. The F0 generation comprises two medaka strains that have extreme, opposing values for a trait of interest, represented by the colours red and blue. Below them is an illustrative single pair of chromosomes. The chromosomes within each pair are depicted as the same colour, as the strains are homozygous through successive generations of inbreeding. Their F1 offspring is heterozygous for each pair of their 24 chromosomes, and all F1 individuals are therefore almost genetically identical to one another (with the exception of somatic mutations and the regions of the genome that were not homozygous in the parental generations). The F1 individuals are then inter-crossed with one another to produce the F2 generation, which, due to recombination events during gamete formation, have unique combinations of the parental strains’ genotypes, and tend to span the phenotypic spectrum between the extremes of their F0 parental strains, represented by their colours." width="100%"><p class="caption">
Figure 1.20: Schema of the F2-cross experimental setup. The F0 generation comprises two medaka strains that have extreme, opposing values for a trait of interest, represented by the colours red and blue. Below them is an illustrative single pair of chromosomes. The chromosomes within each pair are depicted as the same colour, as the strains are homozygous through successive generations of inbreeding. Their F1 offspring is heterozygous for each pair of their 24 chromosomes, and all F1 individuals are therefore almost genetically identical to one another (with the exception of somatic mutations and the regions of the genome that were not homozygous in the parental generations). The F1 individuals are then inter-crossed with one another to produce the F2 generation, which, due to recombination events during gamete formation, have unique combinations of the parental strains’ genotypes, and tend to span the phenotypic spectrum between the extremes of their F0 parental strains, represented by their colours.
</p>
</div>
</div>
<div id="data-collection---f0-generation" class="section level2" number="5.2">
<h2>
<span class="header-section-number">5.2</span> Data collection - F0 generation<a class="anchor" aria-label="anchor" href="#data-collection---f0-generation"><i class="fas fa-link"></i></a>
</h2>
<p>In November 2019 I traveled to the fish facility managed by our collaborator, Felix Loosli at KIT in Karlsruhe, and over the course of 11 days from 11 to 21 November 2019, I ran the behavioural assay described in Chapter <a href="Pilot-chap.html#pilot-data-collection">3.1.1</a> another 206 times. I again used the <em><span style="color: #424B4D;">iCab</span></em> strain as the reference fish, and for the test fish I used either an individual from one of the MIKK panel lines, individuals captured from the same Kiyosu population as the MIKK panel but permitted to breed freely within a separate tank in the facility (‘Kiyosu closed-capture’, or ‘Kiyosu CC’), or individuals from a related but different species of medaka from the Philippines, <em>Oryzias luzonensis</em>. I ensured that I performed at least 2 assay runs for each MIKK panel line that was available, generating a minimum of 8 test fish replicates per line. As there were four pairs of fish in the test tank during each run, the complete dataset comprises 824 videos of pairs of fish, which I further divided by assay component (open field and novel object) to create 1648 videos.</p>
<p>I again used the software <em>idtrackerai</em><span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Romero-Ferrero et al., &lt;span&gt;“Idtracker.ai.”&lt;/span&gt;&lt;/p&gt;"><sup>273</sup></a></span> to track the movement of the fishes across frames of each video. After adjusting the software parameters for each video to maximise the number of frames that were successfully tracked, I was left with 1610 out of the 1648 videos (~97.7%) where both fishes were tracked over at least 85% of frames, and I only included these 1610 videos in the downstream analysis. The first question to address was whether the MIKK panel lines differed in their behaviours. I therefore computed each individual fish’s mean speed (measured as the distance traveled in pixels per 0.08 seconds) over the course of the full 20-minute video, grouped them by line, and plotted the results presented in <strong>Figure <a href="MIKK-F2-chap.html#fig:mikk-mean-speed">5.1</a></strong>. I continue to use the same order and colour palette for the MIKK panel lines as in this Figure throughout the rest of this Chapter.</p>

<div class="figure">
<span style="display:block;" id="fig:mikk-mean-speed"></span>
<img src="figs/mikk_behaviour/F0_line_mean_speed_0.08.png" alt="Mean speed of the MIKK panel and other strains over the course of the entire 20-minute video (measured as the distance traveled in pixels per 0.05 seconds). iCab fishes in the iCab-iCab control condition are at the top, the MIKK panel lines are sorted by their group median, and the Kiyosu closed capture and O. luzonensis fishes are at the bottom." width="100%"><p class="caption">
Figure 5.1: Mean speed of the MIKK panel and other strains over the course of the entire 20-minute video (measured as the distance traveled in pixels per 0.05 seconds). <em><span style="color: #424B4D;">iCab</span></em> fishes in the <em><span style="color: #424B4D;">iCab</span></em>-<em><span style="color: #424B4D;">iCab</span></em> control condition are at the top, the MIKK panel lines are sorted by their group median, and the Kiyosu closed capture and <em>O. luzonensis</em> fishes are at the bottom.
</p>
</div>
<p>This figure shows that there are clear differences between some MIKK panel lines at the extremes, and that the lines differ in the amount of within-line variance observed (shown plotted against the lines’ median speed in <strong>Figure <a href="MIKK-F2-chap.html#fig:mikk-mean-speed-variance">5.2</a></strong>). These figures acted as a general guide to determine which lines to select as the parental strains in the F2 cross. To identify genetic variants directly associated with bold-shy behaviours, I sought to select lines that showed either high or low levels of movement, and preferably low within-line variance.</p>

<div class="figure">
<span style="display:block;" id="fig:mikk-mean-speed-variance"></span>
<img src="figs/mikk_behaviour/line_mean_speed_variance_0.05_all.png" alt="Line median (vertical axis) and line variance (horizontal axis) for individual mean speed across the full 20-minute video (i.e. both the open field and novel object assay components)." width="100%"><p class="caption">
Figure 5.2: Line median (vertical axis) and line variance (horizontal axis) for individual mean speed across the full 20-minute video (i.e. both the open field and novel object assay components).
</p>
</div>
<p>To illustrate the differences between certain lines in terms of both direct and social genetic effects, in <strong>Figure <a href="MIKK-F2-chap.html#fig:extreme-paths">5.3</a></strong> I have plotted the tracked paths for 3 lines 5 minutes into the 10-minute open field assay component: the slowest line <span style="color: #FB737A;">22-1</span>, one of the other slowest lines <span style="color: #FF66A6;">18-2</span>, and the fastest line <span style="color: #F8766D;">10-1</span>. The fishes are coloured by their line, with <em><span style="color: #424B4D;">iCab</span></em> in dark grey. There appears to be a social genetic effect when comparing the behaviour of <em><span style="color: #424B4D;">iCab</span></em> when paired with the similarly slow-moving lines <span style="color: #FB737A;">22-1</span> and <span style="color: #FF66A6;">18-2</span>. At the other extreme, line <span style="color: #F8766D;">10-1</span> has moved extremely quickly, spending much of its time moving along the boundaries of their test tanks. I understand from our collaborator Felix Loosli, a fish behaviour expert, that this movement along the boundaries of a tank is typical of medaka when introduced to a novel environment, and we observe it too with <em><span style="color: #424B4D;">iCab</span></em> when it is paired with line <span style="color: #FB737A;">22-1</span>.</p>

<div class="figure">
<span style="display:block;" id="fig:extreme-paths"></span>
<img src="figs/mikk_behaviour/path_plot_22-1_18-2_10-1_300.png" alt="Path plots for lines 22-1 (A), 18-2 (B) and 10-1 (C) 5 minutes into the open field assay for the first (left) and second (right) run with each line. The paths of iCab individuals are coloured in dark grey, with the other lines depicted in their representative colours." width="100%"><p class="caption">
Figure 5.3: Path plots for lines 22-1 (<strong>A</strong>), 18-2 (<strong>B</strong>) and 10-1 (<strong>C</strong>) 5 minutes into the open field assay for the first (left) and second (right) run with each line. The paths of <em><span style="color: #424B4D;">iCab</span></em> individuals are coloured in dark grey, with the other lines depicted in their representative colours.
</p>
</div>
<p>Although lines <span style="color: #FB737A;">22-1</span> and <span style="color: #F8766D;">10-1</span> were the most extreme in terms of mean speed, I ruled them out of selection for the F2 cross for the following reasons. Our collaborators informed us that through a separate analysis on heartbeat phenotypes across the MIKK panel, they discovered that the heart of line <span style="color: #FB737A;">22-1</span> often stops beating for up to minutes at a time. This may explain the lack of movement observed during our assay. The behaviours they exhibit may therefore not represent a phenotype related to the boldness-shyness axis, but rather an organ failure.</p>
<p>On the other hand, line <span style="color: #F8766D;">10-1</span> appeared to habituate to the open field assay component by slowing down, rather than speeding up (<strong>Figure <a href="MIKK-F2-chap.html#fig:10-1-dens">5.4</a></strong>). That is to say, as the assay progressed, they slowed down their movement, and this suggests that their typical response to stress is to move faster rather than slower. However, as I am using speed as a proxy for boldness (where a quicker habituation to the assay, indicated by an increase in movement, suggests greater boldness), this would create difficulties in attributing a functional interpretation of behaviours in the F2 individuals, and for this reason I excluded line <span style="color: #F8766D;">10-1</span>.</p>

<div class="figure">
<span style="display:block;" id="fig:10-1-dens"></span>
<img src="figs/mikk_behaviour/select_0.08_15_10-1_dge.png" alt="Tile and density plots for line 10-1, showing the reduction in the frequency of higher-movement states over the course of the open field assay component (A and B), with the novel object assay component (C and D) for comparison." width="100%"><p class="caption">
Figure 5.4: Tile and density plots for line <span style="color: #F8766D;">10-1</span>, showing the reduction in the frequency of higher-movement states over the course of the open field assay component (<strong>A</strong> and <strong>B</strong>), with the novel object assay component (<strong>C</strong> and <strong>D</strong>) for comparison.
</p>
</div>
</div>
<div id="hmm-states" class="section level2" number="5.3">
<h2>
<span class="header-section-number">5.3</span> HMM states<a class="anchor" aria-label="anchor" href="#hmm-states"><i class="fas fa-link"></i></a>
</h2>
<p>To explore the behaviours of the MIKK panel at a finer resolution, as for the pilot study described in <a href="Pilot-chap.html#Pilot-chap">3</a>, I again applied a hidden markov model (<strong>HMM</strong>) to classify the fishes’ movements based on their distance and angle of travel between time intervals. I used the same method to select the best choice of time interval and number of states (<strong>Figure <a href="MIKK-F2-chap.html#fig:mikk-param-comp">5.5</a></strong>).</p>

<div class="figure">
<span style="display:block;" id="fig:mikk-param-comp"></span>
<img src="figs/mikk_behaviour/compare_params.png" alt="Comparison between HMM parameters. Horizontal axis: Mean concordance between states assigned by HMMs through a 2-fold cross-validation process. Vertical axis: Kruskal-Wallis statistic comparing strains based on the proportion of time spent in each HMM state, summed across all states. Size of points correspond to the interval, in seconds, between which the distance and angle of travel was calculated (Methods)." width="100%"><p class="caption">
Figure 5.5: Comparison between HMM parameters. Horizontal axis: Mean concordance between states assigned by HMMs through a 2-fold cross-validation process. Vertical axis: Kruskal-Wallis statistic comparing strains based on the proportion of time spent in each HMM state, summed across all states. Size of points correspond to the interval, in seconds, between which the distance and angle of travel was calculated (Methods).
</p>
</div>
<p>Here I observed the same phenomenon where the parameter combinations that performed the best showed an asymmetry between some states that would make interpretation difficult. For example, a time interval of 0.08 seconds combined with a state space of 17 caused state 4 to appear to get carved out of state 3 (<strong>Figure <a href="MIKK-F2-chap.html#fig:mikk-hmm-asym">5.6</a></strong>).</p>

<div class="figure">
<span style="display:block;" id="fig:mikk-hmm-asym"></span>
<img src="figs/mikk_behaviour/0.08_17_polar_all_dge.png" alt="The best apparent combination of parameters (0.08 time interval with a 17-state space) created an asymmetry between states 3 and 4, which would causes difficulties in interpreting their biological relevance." width="100%"><p class="caption">
Figure 5.6: The best apparent combination of parameters (0.08 time interval with a 17-state space) created an asymmetry between states 3 and 4, which would causes difficulties in interpreting their biological relevance.
</p>
</div>
<p>The best combination of parameters without this asymmetry was a time interval of 0.05 seconds with a state space of 15 (see the polar plots for the states in <strong>Appendix <a href="hmm-states-with-0.html#fig:hmm-states-05">A.1</a></strong>). However, due to a glitch in the video recording software, several videos recorded on 13 November 2019 were incorrectly recorded with a frame rate of 14 instead of the desired 30. The insufficient number of frames for those videos meant that it was impossible to measure the distance and angle of travel between a time interval as low as 0.05 seconds. So that these videos could be included in the dataset, I accordingly selected the combination of 15 states and a 0.08-second interval for all downstream analyses.</p>

<div class="figure">
<span style="display:block;" id="fig:mikk-hmm-sym"></span>
<img src="figs/mikk_behaviour/0.08_15_polar_all_dge.png" alt="The HMM states used for the downstream analysis, with the model classified based on the distance of travel (log10 pixels, radial axis) and angle of travel (angle). A straight forward movement would sit around 0°, a left movement around 270°, and a right movement around 90°." width="100%"><p class="caption">
Figure 5.7: The HMM states used for the downstream analysis, with the model classified based on the distance of travel (log<sub>10</sub> pixels, radial axis) and angle of travel (angle). A straight forward movement would sit around 0°, a left movement around 270°, and a right movement around 90°.
</p>
</div>
</div>
<div id="social-genetic-effects-1" class="section level2" number="5.4">
<h2>
<span class="header-section-number">5.4</span> Social genetic effects<a class="anchor" aria-label="anchor" href="#social-genetic-effects-1"><i class="fas fa-link"></i></a>
</h2>
<p>As discussed above, in this project our traits of interest include not only direct genetic behaviours, but also social genetic behaviours. I therefore sought to identify the MIKK panel lines that transmitted their behaviour onto the reference <em><span style="color: #424B4D;">iCab</span></em> tank partners either to the greatest or least extent. I formulated two methods to measure this, referred to as a) HMM state co-occupancy; and b) reference deviation. The first, HMM state co-occupancy, measured the proportions of time that the <em><span style="color: #424B4D;">iCab</span></em> reference fish spent in the same HMM state as its tank partner. The second, deviation of the reference fishes’ deviation from the behaviour exhibited in the control condition, seeks to quantify the extent to which the <em><span style="color: #424B4D;">iCab</span></em>’s behaviour changes when partnered with</p>
<div id="state-co-occupancy" class="section level3" number="5.4.1">
<h3>
<span class="header-section-number">5.4.1</span> State co-occupancy<a class="anchor" aria-label="anchor" href="#state-co-occupancy"><i class="fas fa-link"></i></a>
</h3>
<p><strong>Figure <a href="MIKK-F2-chap.html#fig:F0-sge-cooc-box">5.8</a></strong> sets out the proportions of total time for each assay sub-component that each pair of individual fish spent in the same HMM state, grouped by line and ranked in the same order as their group median for individual mean speed as shown above in <strong>Figure <a href="MIKK-F2-chap.html#fig:mikk-mean-speed">5.1</a></strong>. <strong><a href="MIKK-F2-chap.html#fig:F0-sge-cooc-box">5.8</a>A</strong> shows the data as boxplots, with <span class="math inline">\(p\)</span>-values from the Kruskal-Wallis test comparing all groups. <strong>Figure <a href="MIKK-F2-chap.html#fig:F0-sge-cooc-box">5.8</a>B</strong> shows the same data but with each group’s median represented by columns to make it easier to compare group medians. Of the slower-moving lines, <span style="color: #FF699C;">8-2</span> and <span style="color: #FF66A6;">18-2</span> tend to show relatively higher state co-occupancy in the open field component, but during the novel object component, the slow-moving line <span style="color: #FF61CC;">139-4</span> has the highest median co-occupancy of all lines. Of the faster-moving lines, <span style="color: #F17D50;">43-2</span> and <span style="color: #F57A5F;">13-2</span> showed the highest state co-occupancy during the open field assay component. However, the moderate-to-fast line <span style="color: #49B500;">21-2</span> had relatively high state co-occupancy during both assay components.</p>

<div class="figure">
<span style="display:block;" id="fig:F0-sge-cooc-box"></span>
<img src="figs/mikk_behaviour/0.08_15_cooc_box_all.png" alt="Frequency of HMM state co-occupancy" width="100%"><p class="caption">
Figure 5.8: Frequency of HMM state co-occupancy
</p>
</div>
<p>To visualise which states are driving the higher co-occupancy measures, for a selection of lines I generated a heatmap of the states occupied simultaneously by the <em><span style="color: #424B4D;">iCab</span></em> reference and MIKK test fishes, combining the observations for all individuals within each test fish group (<strong>Figure <a href="MIKK-F2-chap.html#fig:F0-sge-cooc-heat">5.9</a></strong>). When <em><span style="color: #424B4D;">iCab</span></em> is paired with <span style="color: #FF66A6;">18-2</span> or <span style="color: #FF699C;">8-2</span>, the fishes most frequently occupy states 3 or 1 in both open field and novel object components. In pairings with line <span style="color: #FF61CC;">139-4</span>, while the test fishes remain in the still states 1 or 3 during the open field assay, <em><span style="color: #424B4D;">iCab</span></em> tends to be moving much faster in states 11 and 13. However, during the novel object component, they co-occupy state 3 more than in any other combination. This general pattern is observed with line <span style="color: #F066EA;">14-2</span> as well, albeit to a lesser extent. For <span style="color: #00C08B;">38-2</span>, the fishes tend not to show a strong preference for co-occupying a particular state for either assay component, but the diagonal spread indicates that they tend to move at similar speeds. When paired with the faster moving <span style="color: #49B500;">21-2</span>, the novel object component appears to accentuate the co-occupancy of state 3 that is also observed in the open field component. Finally, when paired with line <span style="color: #93AA00;">40-1</span>, in both assay components, both fishes show a strong preference for the faster-moving states.</p>

<div class="figure">
<span style="display:block;" id="fig:F0-sge-cooc-heat"></span>
<img src="figs/mikk_behaviour/0.08_15_cooc_heatmap.png" alt="Heatmaps for a selection of MIKK panel lines (including those ultimately selected as the parental strains in the F2 cross) showing the frequency of HMM states simultaneously occupied by the reference (x-axis) and test (y-axis) fishes, aggregated over all replicates in each line." width="100%"><p class="caption">
Figure 5.9: Heatmaps for a selection of MIKK panel lines (including those ultimately selected as the parental strains in the F2 cross) showing the frequency of HMM states simultaneously occupied by the reference (x-axis) and test (y-axis) fishes, aggregated over all replicates in each line.
</p>
</div>
</div>
<div id="deviation-of-icab-from-its-control-condition" class="section level3" number="5.4.2">
<h3>
<span class="header-section-number">5.4.2</span> Deviation of <em><span style="color: #424B4D;">iCab</span></em> from its control condition<a class="anchor" aria-label="anchor" href="#deviation-of-icab-from-its-control-condition"><i class="fas fa-link"></i></a>
</h3>
<p>The second method for quantifying the level of behavioural transmission from test fish to <em><span style="color: #424B4D;">iCab</span></em> reference fish was to determine the proportion of time that the <em><span style="color: #424B4D;">iCab</span></em> spent in a particular state when paired with another <em><span style="color: #424B4D;">iCab</span></em>, and then quantify the degree to which those proportions change when in the presence of a fish from another line (<strong>Figure <a href="MIKK-F2-chap.html#fig:F0-sge-deviation">5.10</a></strong>). <strong>Figure <a href="MIKK-F2-chap.html#fig:F0-sge-deviation">5.10</a>A</strong> presents boxplots for state frequencies for all <em><span style="color: #424B4D;">iCab</span></em> individuals in <em><span style="color: #424B4D;">iCab</span></em>-<em><span style="color: #424B4D;">iCab</span></em> pairings. I further calculated the state frequencies for all <em><span style="color: #424B4D;">iCab</span></em> reference fishes for all other MIKK line pairings. For each combination of assay component, line-pairing, and HMM state, I then ran Welch’s t-test<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Graeme D. Ruxton, &lt;span&gt;“The Unequal Variance t-Test Is an Underused Alternative to &lt;span&gt;Student&lt;/span&gt;’s t-Test and the &lt;span&gt;Mann&lt;/span&gt;–&lt;span&gt;Whitney U&lt;/span&gt; Test,”&lt;/span&gt; &lt;em&gt;Behavioral Ecology&lt;/em&gt; 17, no. 4 (July 1, 2006): 688–90, &lt;a href="https://doi.org/10.1093/beheco/ark016" role="doc-biblioref"&gt;https://doi.org/10.1093/beheco/ark016&lt;/a&gt;.&lt;/p&gt;'><sup>274</sup></a></span> comparing the proportions of time the <em><span style="color: #424B4D;">iCab</span></em> individuals spent in that state when paired with another <em><span style="color: #424B4D;">iCab</span></em>, against the proportions of time the <em><span style="color: #424B4D;">iCab</span></em> reference individuals spent in that state when paired with a different MIKK line. I then summed the t-statistics across states to generate a single metric for each combination of line and assay component, and plotted the results in <strong>Figure <a href="MIKK-F2-chap.html#fig:F0-sge-deviation">5.10</a>B</strong>.</p>

<div class="figure">
<span style="display:block;" id="fig:F0-sge-deviation"></span>
<img src="figs/mikk_behaviour/0.08_15_deviation.png" alt="Deviation of state frequency for iCab reference fishes when paired with MIKK panel lines relative to when paired with another iCab. A: Boxplots of HMM state frequency for iCab individuals when paired with another iCab. B:" width="100%"><p class="caption">
Figure 5.10: Deviation of state frequency for <em><span style="color: #424B4D;">iCab</span></em> reference fishes when paired with MIKK panel lines relative to when paired with another <em><span style="color: #424B4D;">iCab</span></em>. <strong>A</strong>: Boxplots of HMM state frequency for <em><span style="color: #424B4D;">iCab</span></em> individuals when paired with another <em><span style="color: #424B4D;">iCab</span></em>. <strong>B</strong>:
</p>
</div>
<p>The first thing to note in this Figure is that <em><span style="color: #424B4D;">iCab</span></em>’s deviations from its behaviour exhibited in the control condition tend to be smaller when paired with faster-moving fishes. This was expected given <em><span style="color: #424B4D;">iCab</span></em> is also a faster-moving fish, but it strengthens the observation that when paired with slower-moving fishes, those tank partners are causing the <em><span style="color: #424B4D;">iCab</span></em> reference fish to move slower than they would otherwise. Of the slower-moving MIKK lines, <span style="color: #FC61D4;">4-2</span> and <span style="color: #F066EA;">14-2</span> causing the largest deviations of the <em><span style="color: #424B4D;">iCab</span></em> reference fishes’ behaviours from their control condition behaviour in the open field and novel object components respectively. I also observe large deviations for the moderately-moving <span style="color: #00B9E3;">106-2</span> and <span style="color: #00BECD;">71-1</span> during the novel object components. There are no clear outliers for either assay component for any of the faster moving lines.</p>
</div>
</div>
<div id="selection-of-lines-for-the-f2-cross" class="section level2" number="5.5">
<h2>
<span class="header-section-number">5.5</span> Selection of lines for the F2 cross<a class="anchor" aria-label="anchor" href="#selection-of-lines-for-the-f2-cross"><i class="fas fa-link"></i></a>
</h2>
<p>On the basis of the above findings, I selected 6 MIKK panel lines as the parental lines for the F2 cross (<strong>Figure <a href="MIKK-F2-chap.html#fig:F0-line-mean-speed-select">5.11</a></strong>). Conceptually, I sought to select lines that diverged on two measures: a) bold-shy behaviours; and b) the extent to which the lines transmitted their behaviours onto their tank partners.</p>
<p>As <em><span style="color: #424B4D;">iCab</span></em> is a fast-moving line, it was more difficult to detect instances where fast-moving MIKK panel lines were influencing its behaviour. I was therefore more confident of identifying slow-movement/high-charisma lines, and so to increase the likelihood of identifying genetic variants that are responsible for a stronger transmission of slow-moving behaviours, I chose two slow-movement/high-charisma lines for the F2 cross: <span style="color: #FF66A6;">18-2</span> and <span style="color: #FF699C;">8-2</span>. In the event that both lines possessed the same genetic variants that influence this behavioural transmission trait, it would vastly increase the power of detecting it during the genetic linkage analysis. Both these lines were one of the most slow-moving lines, had high levels of state co-occupancy during both assay components, and <span style="color: #FF66A6;">18-2</span> caused a high level of deviation from the <em><span style="color: #424B4D;">iCab</span></em> control condition.</p>

<div class="figure">
<span style="display:block;" id="fig:F0-line-mean-speed-select"></span>
<img src="figs/mikk_behaviour/line_mean_speed_0.05_selected.png" alt="Mean speed of the MIKK panel and other strains over the course of the entire 20-minute video (measured as the distance traveled in pixels per 0.05 seconds), as shown above in Figure 5.1 but now highlighting the MIKK lines that were selected for the F2 cross." width="100%"><p class="caption">
Figure 5.11: Mean speed of the MIKK panel and other strains over the course of the entire 20-minute video (measured as the distance traveled in pixels per 0.05 seconds), as shown above in Figure <a href="MIKK-F2-chap.html#fig:mikk-mean-speed">5.1</a> but now highlighting the MIKK lines that were selected for the F2 cross.
</p>
</div>
<p>For the slow-movement/low-charisma line, I selected <span style="color: #BB81FF;">50-2</span> which exhibited a moderately slow level of movement, low within-line variance for mean speed, and low measures for both state co-occupancy and <em><span style="color: #424B4D;">iCab</span></em> deviation. For the high-movement/high-charisma line, I selected <span style="color: #49B500;">21-2</span>. Despite its high within line variance (<strong>Figure <a href="MIKK-F2-chap.html#fig:F2-time-dge-of">5.14</a></strong>), this potentially made it easier to detect its social genetic effects, as the slower-moving individuals appeared to transmit those behaviours strongly to their <em><span style="color: #424B4D;">iCab</span></em> tank partners, giving it a high score among fast-moving lines for state co-occupancy across both assay components. For the high-movement/low-charisma line I selected <span style="color: #93AA00;">40-1</span>, as it has low within-line variance, and low-to-moderate metrics for state co-occupancy and <em><span style="color: #424B4D;">iCab</span></em> deviation. However, I note that these measures may be confounded by the possibility that <span style="color: #93AA00;">40-1</span> behaves in a similar way to <em><span style="color: #424B4D;">iCab</span></em>, which would be difficult to determine whether it was behaving differently.</p>
<p>In addition to these extreme lines, I also selected a line that was intermediate for both traits in an attempt to avoid breeding incompatibilities that might arise from attempting to cross lines with such divergent behavioural traits. For this purpose I selected line <span style="color: #00C08B;">38-2</span> for its intermediate speed, low within-line variance for mean speed, and intermediate measures for HMM state co-occupancy and <em><span style="color: #424B4D;">iCab</span></em> deviation.</p>

<div class="figure">
<span style="display:block;" id="fig:F0-line-mean-speed-var-select"></span>
<img src="figs/mikk_behaviour/line_mean_speed_variance_selected.png" alt="Line median (vertical axis) and line variance (horizontal axis) for individual mean speed across the full 20-minute video (i.e. both the open field and novel object assay components) as shown above in Figure 5.2, now coloured only for the lines selected as the parental strains for the F2 cross." width="100%"><p class="caption">
Figure 5.12: Line median (vertical axis) and line variance (horizontal axis) for individual mean speed across the full 20-minute video (i.e. both the open field and novel object assay components) as shown above in Figure <a href="MIKK-F2-chap.html#fig:mikk-mean-speed-variance">5.2</a>, now coloured only for the lines selected as the parental strains for the F2 cross.
</p>
</div>
<p>The selection of parental lines for the F2 cross indicating their putative position across the two axes of interest (movement, and behavioural transmission or ‘charisma’) are set out below in <strong>Figure <a href="MIKK-F2-chap.html#fig:F0-line-select-schema">5.13</a></strong>.</p>

<div class="figure">
<span style="display:block;" id="fig:F0-line-select-schema"></span>
<img src="figs/mikk_behaviour/line_selection_schema.png" alt="Schema of the MIKK panel lines selected as the parental strains for the multi-way F2 cross, indicating their intended position across the two axes of interest (movement, and behavioural transmission or ‘charisma’)." width="100%"><p class="caption">
Figure 5.13: Schema of the MIKK panel lines selected as the parental strains for the multi-way F2 cross, indicating their intended position across the two axes of interest (movement, and behavioural transmission or ‘charisma’).
</p>
</div>
</div>
<div id="direct-genetic-effects-1" class="section level2" number="5.6">
<h2>
<span class="header-section-number">5.6</span> Direct genetic effects<a class="anchor" aria-label="anchor" href="#direct-genetic-effects-1"><i class="fas fa-link"></i></a>
</h2>
<p>With these lines selected, I ran a similar analysis to what I described in <a href="Pilot-chap.html#Pilot-chap">3</a>, where I ran multi-way ANOVAs to determine whether certain lines differed in the proportions of time they spent in these HMM states, while including the date of assay, time of assay, tank quadrant and tank side as covariates. <strong>Table <a href="MIKK-F2-chap.html#tab:mikk-dge-F0">5.1</a></strong> sets out the states which showed a significant difference between these 6 lines, with p-values adjusted for the False Discovery Rate (<strong>FDR</strong>).</p>
<div class="inline-table"><table class="table table-sm">
<caption>
<span id="tab:mikk-dge-F0">Table 5.1: </span>Significant differences in the proportion of time spent in each HMM state across test fish lines selected for the F2 cross for the open field and novel object assay components.</caption>
<thead><tr class="header">
<th align="left">Assay</th>
<th align="left">State</th>
<th align="left">Variance explained (%)</th>
<th align="left">p-value (FDR-adjusted)</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">open field</td>
<td align="left">3</td>
<td align="left">21.53</td>
<td align="left">2.67e-03</td>
</tr>
<tr class="even">
<td align="left">open field</td>
<td align="left">4</td>
<td align="left">23.28</td>
<td align="left">1.60e-02</td>
</tr>
<tr class="odd">
<td align="left">open field</td>
<td align="left">5</td>
<td align="left">20.61</td>
<td align="left">3.15e-02</td>
</tr>
<tr class="even">
<td align="left">open field</td>
<td align="left">10</td>
<td align="left">29.06</td>
<td align="left">1.41e-03</td>
</tr>
<tr class="odd">
<td align="left">open field</td>
<td align="left">12</td>
<td align="left">24.91</td>
<td align="left">1.45e-02</td>
</tr>
<tr class="even">
<td align="left">open field</td>
<td align="left">14</td>
<td align="left">29.52</td>
<td align="left">1.29e-03</td>
</tr>
<tr class="odd">
<td align="left">novel object</td>
<td align="left">1</td>
<td align="left">16.21</td>
<td align="left">1.90e-02</td>
</tr>
<tr class="even">
<td align="left">novel object</td>
<td align="left">2</td>
<td align="left">15.14</td>
<td align="left">4.86e-02</td>
</tr>
<tr class="odd">
<td align="left">novel object</td>
<td align="left">4</td>
<td align="left">26.90</td>
<td align="left">5.21e-03</td>
</tr>
<tr class="even">
<td align="left">novel object</td>
<td align="left">5</td>
<td align="left">27.84</td>
<td align="left">4.27e-03</td>
</tr>
<tr class="odd">
<td align="left">novel object</td>
<td align="left">10</td>
<td align="left">23.73</td>
<td align="left">2.81e-02</td>
</tr>
</tbody>
</table></div>
<p><strong>Figures <a href="MIKK-F2-chap.html#fig:F2-time-dge-of">5.14</a></strong> and <strong><a href="MIKK-F2-chap.html#fig:F2-time-dge-no">5.15</a></strong> highlight the states that showed significant differences in the proportions of time that these lines spent in those states for the open field and novel object assay components respectively. In both figures, <strong>A</strong> highlights the significant states, <strong>B</strong> shows how the individuals moved through those states over the course of the 10-minute assay component, and <strong>C</strong> shows the densities of the significant states within each line. For the open field component, although the tile plots show a notable level of variance within each line, the density plots clarify how the three slow-moving lines (8-2, 18-2 and 50-2) spent little time in the fast-moving states 10, 12 and 14 relative to the fast-moving lines. Interestingly, the intermediate line 38-2 tended to transition into these states around the middle of the video, presumably a period of habituating to the new environment.</p>

<div class="figure">
<span style="display:block;" id="fig:F2-time-dge-of"></span>
<img src="figs/mikk_behaviour/select_0.08_15_dge_of.png" alt="Differences between MIKK F0 lines in the HMM states they occupied during the open field assay component. A: 15 HMM states with panels coloured red to indicate significant differences between MIKK F0 lines in the proportion of time spent in those states. B: Transitions between HMM states across time for each individual test fish, grouped by MIKK line Tiles are coloured by the state most frequently occupied by each fish within 2-second intervals. C: Densities within each MIKK line for the occupation of states that significantly differed between strains (colour), with other states consolidated (grey)." width="100%"><p class="caption">
Figure 5.14: Differences between MIKK F0 lines in the HMM states they occupied during the open field assay component. <strong>A</strong>: 15 HMM states with panels coloured red to indicate significant differences between MIKK F0 lines in the proportion of time spent in those states. <strong>B</strong>: Transitions between HMM states across time for each individual test fish, grouped by MIKK line Tiles are coloured by the state most frequently occupied by each fish within 2-second intervals. <strong>C</strong>: Densities within each MIKK line for the occupation of states that significantly differed between strains (colour), with other states consolidated (grey).
</p>
</div>
<p>During the novel object component the HMM states that showed significant differences between lines were mostly restricted to the slow-moving states with the exception of state 10, which most clearly distinguishes the slow-moving lines from the intermediate- and fast-moving lines. Again, the intermediate line <span style="color: #00C08B;">38-2</span> shows a sharp drop in the occupation of the slow moving states after a period of habituation.</p>

<div class="figure">
<span style="display:block;" id="fig:F2-time-dge-no"></span>
<img src="figs/mikk_behaviour/select_0.08_15_dge_no.png" alt="Differences between MIKK F0 lines in the HMM states they occupied during the novel object assay component. A: 15 HMM states with panels coloured red to indicate significant differences between MIKK F0 lines in the proportion of time spent in those states. B: Transitions between HMM states across time for each individual test fish, grouped by MIKK line Tiles are coloured by the state most frequently occupied by each fish within 2-second intervals. C: Densities within each MIKK line for the occupation of states that significantly differed between strains (colour), with other states consolidated (grey)." width="100%"><p class="caption">
Figure 5.15: Differences between MIKK F0 lines in the HMM states they occupied during the novel object assay component. <strong>A</strong>: 15 HMM states with panels coloured red to indicate significant differences between MIKK F0 lines in the proportion of time spent in those states. <strong>B</strong>: Transitions between HMM states across time for each individual test fish, grouped by MIKK line Tiles are coloured by the state most frequently occupied by each fish within 2-second intervals. <strong>C</strong>: Densities within each MIKK line for the occupation of states that significantly differed between strains (colour), with other states consolidated (grey).
</p>
</div>
<div id="social-genetic-effects-2" class="section level3" number="5.6.1">
<h3>
<span class="header-section-number">5.6.1</span> Social genetic effects<a class="anchor" aria-label="anchor" href="#social-genetic-effects-2"><i class="fas fa-link"></i></a>
</h3>
<p>To confirm whether the <em><span style="color: #424B4D;">iCab</span></em> reference fishes altered their behaviour depending on the MIKK F0 line of their tank partner, we carried out the same analysis and model as above using only data from the <em><span style="color: #424B4D;">iCab</span></em> reference fishes. The states that showed a significant difference across any of the variables included in the model are set out in <strong>Table <a href="MIKK-F2-chap.html#tab:mikk-sge-F0">5.2</a></strong>. The iCab reference fishes differed significantly in the proportion of time they spent in a given state depending on the strain of their tank partner (<span class="math inline">\(p\)</span> &lt; 0.05, FDR-adjusted) for 4 out of 15 states in the open field assay (1.26x10<sup>-2</sup> &lt; <span class="math inline">\(p\)</span> &lt; 4x10<sup>-2</sup>), and 4 out of 15 states for the novel object assay (7.71x10<sup>-3</sup> &lt; <span class="math inline">\(p\)</span> &lt; 1.34x10<sup>-2</sup>). The line of the tank partner explained up to ~29% of the variance in the proportion of time the <em><span style="color: #424B4D;">iCab</span></em> reference spent in a given state. Full tables for all states and variables are provided in the Supplementary Material.</p>
<div class="inline-table"><table class="table table-sm">
<caption>
<span id="tab:mikk-sge-F0">Table 5.2: </span>Significant differences in the proportion of time spent in each HMM state by iCab reference fishes depending on the MIKK F0 line of their tank partner during the open field and novel object assay components.</caption>
<thead><tr class="header">
<th align="left">Assay</th>
<th align="left">State</th>
<th align="left">Variance explained (%)</th>
<th align="left">p-value (FDR-adjusted)</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">open field</td>
<td align="left">3</td>
<td align="left">14.93</td>
<td align="left">2.10e-02</td>
</tr>
<tr class="even">
<td align="left">open field</td>
<td align="left">4</td>
<td align="left">13.60</td>
<td align="left">2.28e-02</td>
</tr>
<tr class="odd">
<td align="left">open field</td>
<td align="left">5</td>
<td align="left">15.33</td>
<td align="left">1.26e-02</td>
</tr>
<tr class="even">
<td align="left">open field</td>
<td align="left">7</td>
<td align="left">14.26</td>
<td align="left">4.00e-02</td>
</tr>
<tr class="odd">
<td align="left">novel object</td>
<td align="left">4</td>
<td align="left">14.33</td>
<td align="left">1.23e-02</td>
</tr>
<tr class="even">
<td align="left">novel object</td>
<td align="left">5</td>
<td align="left">14.38</td>
<td align="left">1.15e-02</td>
</tr>
<tr class="odd">
<td align="left">novel object</td>
<td align="left">9</td>
<td align="left">24.61</td>
<td align="left">1.34e-02</td>
</tr>
<tr class="even">
<td align="left">novel object</td>
<td align="left">11</td>
<td align="left">28.77</td>
<td align="left">7.71e-03</td>
</tr>
</tbody>
</table></div>
<div class="figure">
<span style="display:block;" id="fig:F2-time-sge-of"></span>
<img src="figs/mikk_behaviour/select_0.08_15_sge_of.png" alt="Differences between MIKK F0 lines in the HMM states they occupied during the open field assay component. A: 15 HMM states with panels coloured red to indicate significant differences between MIKK F0 lines in the proportion of time spent in those states. B: Transitions between HMM states across time for each individual test fish, grouped by MIKK line Tiles are coloured by the state most frequently occupied by each fish within 2-second intervals. C: Densities within each MIKK line for the occupation of states that significantly differed between strains (colour), with other states consolidated (grey)." width="100%"><p class="caption">
Figure 5.16: Differences between MIKK F0 lines in the HMM states they occupied during the open field assay component. <strong>A</strong>: 15 HMM states with panels coloured red to indicate significant differences between MIKK F0 lines in the proportion of time spent in those states. <strong>B</strong>: Transitions between HMM states across time for each individual test fish, grouped by MIKK line Tiles are coloured by the state most frequently occupied by each fish within 2-second intervals. <strong>C</strong>: Densities within each MIKK line for the occupation of states that significantly differed between strains (colour), with other states consolidated (grey).
</p>
</div>

<div class="figure">
<span style="display:block;" id="fig:F2-time-sge-no"></span>
<img src="figs/mikk_behaviour/select_0.08_15_sge_no.png" alt="Differences between MIKK F0 lines in the HMM states they occupied during the novel object assay component. A: 15 HMM states with panels coloured red to indicate significant differences between MIKK F0 lines in the proportion of time spent in those states. B: Transitions between HMM states across time for each individual test fish, grouped by MIKK line Tiles are coloured by the state most frequently occupied by each fish within 2-second intervals. C: Densities within each MIKK line for the occupation of states that significantly differed between strains (colour), with other states consolidated (grey)." width="100%"><p class="caption">
Figure 5.17: Differences between MIKK F0 lines in the HMM states they occupied during the novel object assay component. <strong>A</strong>: 15 HMM states with panels coloured red to indicate significant differences between MIKK F0 lines in the proportion of time spent in those states. <strong>B</strong>: Transitions between HMM states across time for each individual test fish, grouped by MIKK line Tiles are coloured by the state most frequently occupied by each fish within 2-second intervals. <strong>C</strong>: Densities within each MIKK line for the occupation of states that significantly differed between strains (colour), with other states consolidated (grey).
</p>
</div>
<p>To assist with interpretation of the relationships between the HMM states and the types of behaviours exhibited, I have provided in <strong>Appendix <a href="four-panel-app.html#four-panel-app">B</a></strong> four-panel plots showing a) a frame grab from the raw four-quadrant video; b) path plots for both fishes; c) path plots for the test fishes coloured by HMM state; and d) path plots for the reference fishes coloured by HMM state, all taken at the 300-second mark (5 minutes) of each 10-minute assay component.</p>
</div>
</div>
<div id="f2-generation" class="section level2" number="5.7">
<h2>
<span class="header-section-number">5.7</span> F2 generation<a class="anchor" aria-label="anchor" href="#f2-generation"><i class="fas fa-link"></i></a>
</h2>
<div id="behavioural-data-collection" class="section level3" number="5.7.1">
<h3>
<span class="header-section-number">5.7.1</span> Behavioural data collection<a class="anchor" aria-label="anchor" href="#behavioural-data-collection"><i class="fas fa-link"></i></a>
</h3>
<p>Around August 2019, our collaborators in the Loosli Group at KIT commenced the breeding program for this experiment with the 6 MIKK panel lines I had selected above.<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Although the breeding program began in 2019, I understand from our collaborators that from around mid-2020 the F1 generation was producing poorly. After lengthy investigations, they discovered that the Covid pandemic had caused disruptions to the supply chains of their fish food manufacturer, which had compelled the manufacturer to modify the recipe. This change in the fish food recipe was the cause of the poor breeding. While the issue has since been resolved, it has resulted in a much smaller number of F2 individuals to be produced than was originally anticipated.&lt;/p&gt;"><sup>275</sup></a> From 17 November 2021 to 5 May 2022, a Research Assistant in Prof. Loosli’s lab, Alicia Günthel, performed the assay 69 time with F2 individuals from the 12 crosses they had bred, producing a total of 271 videos of pairs of fish. The counts for the 12 crosses used to generate these 271 test fishes are set out in <strong>Table <a href="MIKK-F2-chap.html#tab:F2-cross-counts">5.3</a></strong>.</p>
<div class="inline-table"><table class="table table-sm">
<caption>
<span id="tab:F2-cross-counts">Table 5.3: </span>Significant differences in the proportion of time spent in each HMM state by iCab reference fishes depending on the MIKK F0 line of their tank partner during the open field and novel object assay components.</caption>
<thead><tr class="header">
<th align="left">paternal line</th>
<th align="left">maternal line</th>
<th align="right">count</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">21-2</td>
<td align="left">40-1</td>
<td align="right">60</td>
</tr>
<tr class="even">
<td align="left">38-2</td>
<td align="left">40-1</td>
<td align="right">57</td>
</tr>
<tr class="odd">
<td align="left">38-2</td>
<td align="left">18-2</td>
<td align="right">35</td>
</tr>
<tr class="even">
<td align="left">8-2</td>
<td align="left">40-1</td>
<td align="right">24</td>
</tr>
<tr class="odd">
<td align="left">50-2</td>
<td align="left">18-2</td>
<td align="right">23</td>
</tr>
<tr class="even">
<td align="left">38-2</td>
<td align="left">21-2</td>
<td align="right">19</td>
</tr>
<tr class="odd">
<td align="left">8-2</td>
<td align="left">38-2</td>
<td align="right">15</td>
</tr>
<tr class="even">
<td align="left">50-2</td>
<td align="left">38-2</td>
<td align="right">12</td>
</tr>
<tr class="odd">
<td align="left">18-2</td>
<td align="left">21-2</td>
<td align="right">7</td>
</tr>
<tr class="even">
<td align="left">21-2</td>
<td align="left">50-2</td>
<td align="right">7</td>
</tr>
<tr class="odd">
<td align="left">40-1</td>
<td align="left">50-2</td>
<td align="right">6</td>
</tr>
<tr class="even">
<td align="left">50-2</td>
<td align="left">8-2</td>
<td align="right">6</td>
</tr>
</tbody>
</table></div>
<p>I again used <em>idtrackerai</em><span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Romero-Ferrero et al., &lt;span&gt;“Idtracker.ai.”&lt;/span&gt;&lt;/p&gt;"><sup>276</sup></a></span> to track the F2 individuals across frames. When splitting the 271 videos of pairs of fish into their open field and novel object assay components for a total of 542 videos, for 526 of them both fish were tracked across at least 85% of frames. I only used these 526 videos in the downstream analysis, but prior to publication I will seek to improve the tracking process so that these individuals can be included.</p>
</div>
<div id="behavioural-phenotyping" class="section level3" number="5.7.2">
<h3>
<span class="header-section-number">5.7.2</span> Behavioural phenotyping<a class="anchor" aria-label="anchor" href="#behavioural-phenotyping"><i class="fas fa-link"></i></a>
</h3>
<p>To ensure that the predicted HMM states for behaviour were consistent across both F0 and F2 generations, I had included these F2 individuals for training and prediction in the models described above. I again calculated the proportions of time that every individual spent in each state (“<strong>state frequency</strong>”), then inverse-normalised the values within each combination of assay component (open field or novel object) and state (1 to 15). Inverse-normalisation is a rank-based normalisation approach which involves replacing the values in the phenotype vector with their rank (where ties are averaged), then converting the ranks into a normal distribution with the quantile function.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Michael J. Wichura, &lt;span&gt;“Algorithm &lt;span&gt;AS&lt;/span&gt; 241: &lt;span&gt;The Percentage Points&lt;/span&gt; of the &lt;span&gt;Normal Distribution&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Journal of the Royal Statistical Society. Series C (Applied Statistics)&lt;/em&gt; 37, no. 3 (1988): 477–84, &lt;a href="https://doi.org/10.2307/2347330" role="doc-biblioref"&gt;https://doi.org/10.2307/2347330&lt;/a&gt;.&lt;/p&gt;'><sup>277</sup></a></span> The inverse-normalisation function I used for this analysis is set out in the following <code>R</code> code:</p>
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">invnorm</span> <span class="op">=</span> <span class="kw">function</span><span class="op">(</span><span class="va">x</span><span class="op">)</span> <span class="op">{</span></span>
<span>  <span class="va">res</span> <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/rank.html">rank</a></span><span class="op">(</span><span class="va">x</span><span class="op">)</span></span>
<span>  <span class="co"># The arbitrary 0.5 value is added to the denominator below </span></span>
<span>  <span class="co"># to avoid `qnorm()` returning 'Inf' for the last-ranked value</span></span>
<span>  <span class="va">res</span> <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/stats/Normal.html">qnorm</a></span><span class="op">(</span><span class="va">res</span><span class="op">/</span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/length.html">length</a></span><span class="op">(</span><span class="va">res</span><span class="op">)</span><span class="op">+</span><span class="fl">0.5</span><span class="op">)</span><span class="op">)</span></span>
<span>  <span class="kw"><a href="https://rdrr.io/r/base/function.html">return</a></span><span class="op">(</span><span class="va">res</span><span class="op">)</span></span>
<span><span class="op">}</span></span></code></pre></div>
<p><strong>Figures <a href="MIKK-F2-chap.html#fig:F2-state-freq-dge">5.18</a></strong> and <strong><a href="MIKK-F2-chap.html#fig:F2-state-freq-sge">5.19</a></strong> compares the phenotype pre- and post-transformation with this normalisation approach. For the higher-movement states there are increasing numbers of individuals who spent no time in those states, which are responsible for the apparently non-normal distributions observed for the skewed distributions post-transformation. States 1, 3, 6, 8, 9 and 11 already appear to have a large amount of variation between individuals, but this normalisation process will increase the variance for states where there is small, yet potentially meaningful, variation between individuals. One exception may perhaps be state 15, which involves very large distances of travel in all directions, and therefore may correspond to tracking errors.</p>

<div class="figure">
<span style="display:block;" id="fig:F2-state-freq-dge"></span>
<img src="figs/mikk_behaviour/0.08_15_state_freq_F2_dge.png" alt="Effect of inverse-normalisation on the HMM state frequency of the F2 test fishes." width="100%"><p class="caption">
Figure 5.18: Effect of inverse-normalisation on the HMM state frequency of the F2 test fishes.
</p>
</div>

<div class="figure">
<span style="display:block;" id="fig:F2-state-freq-sge"></span>
<img src="figs/mikk_behaviour/0.08_15_state_freq_F2_sge.png" alt="Effect of inverse-normalisation on the HMM state frequency of the F2 iCab reference fishes." width="100%"><p class="caption">
Figure 5.19: Effect of inverse-normalisation on the HMM state frequency of the F2 <em><span style="color: #424B4D;">iCab</span></em> reference fishes.
</p>
</div>
</div>
<div id="genotyping" class="section level3" number="5.7.3">
<h3>
<span class="header-section-number">5.7.3</span> Genotyping<a class="anchor" aria-label="anchor" href="#genotyping"><i class="fas fa-link"></i></a>
</h3>
<p>After phenotyping the F2 samples, our collaborators in the Loosli Group at KIT took finclips from the F2 individuals, extracted their DNA, and arranged for them to be shallow-sequenced on the Illumina short-read platform at a coverage of ~1x per sample. Using a similar method to what I described in Chapter <a href="Somite-chap.html#Somite-chap">4</a>, I aligned these sequences to the <em>HdrR</em> reference with BWA-MEM2,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Vasimuddin et al., &lt;span&gt;“Efficient Architecture-Aware Acceleration of &lt;span&gt;BWA-MEM&lt;/span&gt; for Multicore Systems.”&lt;/span&gt;&lt;/p&gt;"><sup>278</sup></a></span> sorted the reads and marked duplicates with Picard,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;&lt;span&gt;“Picard Toolkit.”&lt;/span&gt;&lt;/p&gt;"><sup>279</sup></a></span> then indexed the resulting BAM files with samtools.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Danecek et al., &lt;span&gt;“Twelve Years of &lt;span&gt;SAMtools&lt;/span&gt; and &lt;span&gt;BCFtools&lt;/span&gt;.”&lt;/span&gt;&lt;/p&gt;"><sup>280</sup></a></span> I then used <em>bam-readcount</em><span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Khanna et al., &lt;span&gt;“Bam-Readcount - Rapid Generation of Basepair-Resolution Sequence Metrics.”&lt;/span&gt;&lt;/p&gt;"><sup>281</sup></a></span> to count the reads that supported either the paternal or the maternal allele for all biallelic SNPs that were homozygous-divergent in the given sample’s two parental strains (i.e. homozgyous reference in the paternal line, and homozygous alternative in the maternal line, or <em>vice versa</em>), summed the read counts within 5 kb blocks, and calculated the frequency of reads within each bin that supported the maternal allele. This generated a value for each bin between 0 and 1, where 0 signified that all reads within that bin supported the paternal allele, and 1 signified that all reads within that bin supported the maternal allele. Bins containing no reads were imputed with a value of 0.5.</p>
<p>I then used these values for all F2 individuals as the input to a Hidden Markov Model (HMM) with the software package <em>hmmlearn</em>,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;&lt;em&gt;Hmmlearn/Hmmlearn&lt;/em&gt;.&lt;/p&gt;"><sup>282</sup></a></span> which I applied to classify each bin as one of three states, with state 0 corresponding to homozygous for the paternal allele, 1 corresponding to heterozygous, and 2 corresponding to homozygous for the maternal allele. Across each chromosome of every sample, the output of the HMM was expected to produce a sequence of states. Based on previous analyses described in Chapter <a href="Somite-chap.html#Somite-chap">4</a>, I used the same HMM parameters as I did there, and used the HMM state outputs to generate the recombination blocks shown in <strong>Figure <a href="MIKK-F2-chap.html#fig:F2-recomb-blocks">5.20</a></strong>. Missing genotype state calls arose from a sample having insufficient reads within a bin, for which I imputed the missing state calls within each sample’s chromosome based on the previous state call on that chromosome. A karyoplot retaining the missing blocks is provided in <strong>Appendix <a href="f2-recombination-karyoplot-with-missing-calls.html#fig:F2-recomb-blocks-missing">D.1</a></strong>.</p>

<div class="figure">
<span style="display:block;" id="fig:F2-recomb-blocks"></span>
<img src="figs/mikk_behaviour/karyoplot_no_missing.png" alt="Karyoplot for F2 samples, coloured by genotype. Samples are sorted in the order in which they were phenotyped. Blocks are filled with the colour of the paternal F0 line for the homozygous paternal haplotype block, black for heterozygous, and the colour of the maternal F0 line for the homozygous maternal haplotype block. Missing calls were imputed based on the previous successful call on a given chromosome." width="100%"><p class="caption">
Figure 5.20: Karyoplot for F2 samples, coloured by genotype. Samples are sorted in the order in which they were phenotyped. Blocks are filled with the colour of the paternal F0 line for the homozygous paternal haplotype block, black for heterozygous, and the colour of the maternal F0 line for the homozygous maternal haplotype block. Missing calls were imputed based on the previous successful call on a given chromosome.
</p>
</div>
<p>I then took these HMM-generated haplotype block calls and used them to impute each sample’s SNP-level genotypes using the homozygous biallelic SNP calls in the high-coverage .vcf file for the MIKK panel F0 lines described in Chapter <a href="MIKK-genomes-chap.html#MIKK-genomes-chap">2</a>. This set of variants included a total of ~20.7M SNPs, which I used to generate a Plink-format .bed file, forming the genotype input for the genetic linkage analysis.</p>
</div>
<div id="genetic-linkage-analysis" class="section level3" number="5.7.4">
<h3>
<span class="header-section-number">5.7.4</span> Genetic linkage analysis<a class="anchor" aria-label="anchor" href="#genetic-linkage-analysis"><i class="fas fa-link"></i></a>
</h3>
<p>For the purpose of using the <em>GCTA</em> software package<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Yang et al., &lt;span&gt;“&lt;span&gt;GCTA&lt;/span&gt;.”&lt;/span&gt;&lt;/p&gt;"><sup>283</sup></a></span> for the genetic linkage analysis, That software requires the construction of a genetic relationship matrix (<strong>GRM</strong>) <span class="math inline">\(\textbf{A} = \textbf{WW}'/N\)</span>. <span class="math inline">\(\textbf{W}\)</span> is a standardised genotype matrix with the <span class="math inline">\(ij^{th}\)</span> element <span class="math inline">\(w_{ij} = (x_{ij} - 2p_i) / \sqrt{(2p_i(1-p_i)}\)</span>, where <span class="math inline">\(x_{ij}\)</span> is the number of copies of the reference allele for the <span class="math inline">\(i^{th}\)</span> SNP of the <span class="math inline">\(j^{th}\)</span> individual and <span class="math inline">\(p_i\)</span> is the frequency of the reference allele.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Yang et al.&lt;/p&gt;"><sup>284</sup></a></span></p>
<p>For the GRM, I first filtered the .bed for SNPs that had no missing calls for any samples (<span class="math inline">\(M_{SNPs}\)</span> = 44,360). I then used these SNPs to construct a “leave-one-chromosome-out” (<strong>LOCO</strong>) genetic relationship matrix for each chromosome – that is, if the “focal” chromosome was chr1, I would exclude the SNPs on that chromosome before constructing the GRM. To illustrate, <strong>Figure <a href="MIKK-F2-chap.html#fig:F2-grm">5.21</a></strong> is a GRM constructed using all 44,360 non-missing SNPs. However, given the relatively small amount of non-missing SNPs on each chromosome, the number of SNPs on the focal chromosome that were excluded was small, resulting in GRMs that appear almost identical by eye – see the LOCO-GRM for chromosome 1 in <strong>Appendix <a href="loco-grm-for-chromosome-1.html#fig:loco-grm-chr1">E.1</a></strong>. The individuals from each cross neatly cluster together, and the individuals that share one parental strain cluster nearby.</p>

<div class="figure">
<span style="display:block;" id="fig:F2-grm"></span>
<img src="figs/mikk_behaviour/grm_man_0.8.png" alt="Genetic relationship matrix for 271 F2 samples based on 44,360 non-missing SNPs." width="100%"><p class="caption">
Figure 5.21: Genetic relationship matrix for 271 F2 samples based on 44,360 non-missing SNPs.
</p>
</div>
<p>I used these GRMs in the mixed linear model based association analysis (<strong>MLMA</strong>) implemented in GCTA,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Yang et al.&lt;/p&gt;"><sup>285</sup></a></span> where the model was generally specified as follows:</p>
<p><span class="math display">\[
y = a + bx + g + e
\]</span></p>
<p><span class="math inline">\(y\)</span> was the phenotype, <span class="math inline">\(a\)</span> was the mean term, <span class="math inline">\(b\)</span> was the additive effect (fixed effect) of the candidate SNP tested for association, <span class="math inline">\(x\)</span> was the SNP genotype indicator variable coded as 0, 1 or 2, <span class="math inline">\(g\)</span> was the polygenic effect (random effect) i.e. the accumulated effect of all SNPs (as captured by the GRM calculated using all SNPs) and <span class="math inline">\(e\)</span> was the residual.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Yang et al.&lt;/p&gt;"><sup>286</sup></a></span> For <span class="math inline">\(y\)</span>, I used the inverse-normalised state frequencies described above, and the LOCO-GRM as <span class="math inline">\(g\)</span> for all SNPs on the focal chromosome. For example, for all SNPs on chr18, I used the LOCO-GRM that excluded all SNPs from that chromosome. In addition, I included the time of assay and the tank quadrant as covariates, which the software regresses out from the phenotype prior to running the MLMA. I excluded the date of assay and the tank side as covariates because individuals from the same cross tended to be assayed in the same test tank on the same day, and are therefore confounded with their genetics.</p>
<p>To set a significance threshold, for each combination of HMM state and assay, I ran 10 MLMA tests over the same dataset where I had permuted the phenotype and covariates using a different random seed. The logic behind this method is to determine the lowest p-value that one could expect when there is no true relationship between the individuals’ genetics and their phenotype. I then extracted the smallest p-value from all 10 results, and used this as the significance threshold. I additionally calculated the Bonferroni threshold as <span class="math inline">\(\alpha / M\)</span>, where <span class="math inline">\(\alpha\)</span> is set to 0.05 and <span class="math inline">\(M\)</span> is the total number of SNPs in the dataset (2,726,797) = 1.83x10<sup>-8</sup>.</p>
<div id="direct-genetic-effects-2" class="section level4" number="5.7.4.1">
<h4>
<span class="header-section-number">5.7.4.1</span> Direct genetic effects<a class="anchor" aria-label="anchor" href="#direct-genetic-effects-2"><i class="fas fa-link"></i></a>
</h4>
<p>For the DGE phenotypes (comparing the state frequencies across test fishes), I sought to identify the genetic loci in the F2 individuals that were associated with differences in their own behaviour. For neither the open field nor novel object component did the p-values exceed the Bonferroni threshold, however a number of loci across several chromosomes exceeded the thresholds set by the permutations. I plotted the p-values in Manhattan plots for each combination of state and assay component, and provide them all in the Appendix. Here I showcase the Manhattan plot for state 3, containing a number of significant loci that I discuss further below.</p>

<div class="figure">
<span style="display:block;" id="fig:F2-man-dge-no-3"></span>
<img src="figs/mikk_behaviour/manhattans/dge_no_3_time-quadrant.png" alt="Manhattan plot for inverse-normalised HMM state 3 frequency of F2 test fishes during the novel object assay." width="100%"><p class="caption">
Figure 5.22: Manhattan plot for inverse-normalised HMM state 3 frequency of F2 test fishes during the novel object assay.
</p>
</div>
</div>
<div id="social-genetic-effects-3" class="section level4" number="5.7.4.2">
<h4>
<span class="header-section-number">5.7.4.2</span> Social genetic effects<a class="anchor" aria-label="anchor" href="#social-genetic-effects-3"><i class="fas fa-link"></i></a>
</h4>
<p>For social genetic effects, I was attempting to discover genetic variants in the F2 individuals that caused differences in the behaviour of their <em><span style="color: #424B4D;">iCab</span></em> tank partners. As expected, fewer loci reached significance for this transmitted indirect genetic effect than for the direct genetic effects, however I still found several significant loci based on the permutations threshold for states 4 (chr4), 7 (chr1), 12 (chr11), and 13 (chr12) during the open field assay component; but only one, barely significant SNP for state 5 (chr 13) during the novel object assay. This difference was somewhat surprising given our previous suppositions (DISCUSS) that the novel object component drew out stronger social genetic effects.</p>

<div class="figure">
<span style="display:block;" id="fig:F2-man-sge-of-13"></span>
<img src="figs/mikk_behaviour/manhattans/sge_of_13_time-quadrant.png" alt="Manhattan plot for inverse-normalised HMM state 3 frequency of iCab reference fishes during the novel object assay component." width="100%"><p class="caption">
Figure 5.23: Manhattan plot for inverse-normalised HMM state 3 frequency of <em><span style="color: #424B4D;">iCab</span></em> reference fishes during the novel object assay component.
</p>
</div>
</div>
</div>
<div id="candidate-snps-for-crispr-knockouts" class="section level3" number="5.7.5">
<h3>
<span class="header-section-number">5.7.5</span> Candidate SNPs for CRISPR-knockouts<a class="anchor" aria-label="anchor" href="#candidate-snps-for-crispr-knockouts"><i class="fas fa-link"></i></a>
</h3>
<p>The next goal was to attempt to identify the SNPs that are mostly likely to be the causal variants, or closest in proximity to the causal variants, responsible for differences in the phenotypes of interest. I therefore proceeded to extract the significant SNPs and use Ensembl’s Variant Effect Predictor (<strong>VEP</strong>)<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;McLaren et al., &lt;span&gt;“The &lt;span&gt;Ensembl Variant Effect Predictor&lt;/span&gt;.”&lt;/span&gt;&lt;/p&gt;"><sup>287</sup></a></span> to identify the SNPs that are most likely to have a functional consequence, and determine whether the genes they reside in have already been identified as being involved in biological pathways that could be related to behaviour. <strong>Table <a href="MIKK-F2-chap.html#tab:F2-sig-snps-consq-counts">5.4</a></strong> shows the counts for the different potential consequences of the unique significant SNPs, based on Ensembl’s annotation of the <em>HdrR</em> reference. I note that as multiple genes can overlap the same locus (including those transcribed from opposing strands), one SNP may map to multiple genes and therefore be counted as having more than one type of consequence.</p>
<div class="inline-table"><table class="table table-sm">
<caption>
<span id="tab:F2-sig-snps-consq-counts">Table 5.4: </span>Counts for consequences of significant SNPs.</caption>
<thead><tr class="header">
<th align="left">consequence</th>
<th align="right">count</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">intron_variant</td>
<td align="right">349</td>
</tr>
<tr class="even">
<td align="left">intergenic_variant</td>
<td align="right">178</td>
</tr>
<tr class="odd">
<td align="left">upstream_gene_variant</td>
<td align="right">92</td>
</tr>
<tr class="even">
<td align="left">downstream_gene_variant</td>
<td align="right">75</td>
</tr>
<tr class="odd">
<td align="left">synonymous_variant</td>
<td align="right">9</td>
</tr>
<tr class="even">
<td align="left">3_prime_UTR_variant</td>
<td align="right">6</td>
</tr>
<tr class="odd">
<td align="left">missense_variant</td>
<td align="right">6</td>
</tr>
<tr class="even">
<td align="left">5_prime_UTR_variant</td>
<td align="right">3</td>
</tr>
<tr class="odd">
<td align="left">splice_acceptor_variant</td>
<td align="right">2</td>
</tr>
<tr class="even">
<td align="left">splice_region_variant,intron_variant</td>
<td align="right">2</td>
</tr>
<tr class="odd">
<td align="left">splice_region_variant,synonymous_variant</td>
<td align="right">2</td>
</tr>
<tr class="even">
<td align="left">splice_donor_variant</td>
<td align="right">1</td>
</tr>
</tbody>
</table></div>
<p>As expected, most variants reside in non-coding regions [CITE], but in an attempt to isolate the most likely causative SNPs, I extracted those that are predicted to cause a missense mutation and therefore most likely to have a functional consequence (<strong>Table <a href="MIKK-F2-chap.html#tab:F2-sig-snps-missense">5.5</a></strong>).</p>
<div class="inline-table"><table class="table table-sm">
<caption>
<span id="tab:F2-sig-snps-missense">Table 5.5: </span>Missense SNPs.</caption>
<colgroup>
<col width="12%">
<col width="10%">
<col width="4%">
<col width="3%">
<col width="7%">
<col width="3%">
<col width="3%">
<col width="5%">
<col width="15%">
<col width="33%">
</colgroup>
<thead><tr class="header">
<th align="left">Genetic effect</th>
<th align="left">Assay</th>
<th align="right">State</th>
<th align="right">Chr</th>
<th align="right">Pos</th>
<th align="left">Ref</th>
<th align="left">Alt</th>
<th align="left">Allele</th>
<th align="left">Gene</th>
<th align="left">Description</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">DGE</td>
<td align="left">open field</td>
<td align="right">1</td>
<td align="right">1</td>
<td align="right">22508288</td>
<td align="left">A</td>
<td align="left">C</td>
<td align="left">C</td>
<td align="left">ENSORLG00000009385</td>
<td align="left">vav guanine nucleotide exchange factor 1</td>
</tr>
<tr class="even">
<td align="left">DGE</td>
<td align="left">open field</td>
<td align="right">7</td>
<td align="right">13</td>
<td align="right">33714439</td>
<td align="left">G</td>
<td align="left">T</td>
<td align="left">T</td>
<td align="left">ENSORLG00000028050</td>
<td align="left">uncharacterized LOC105356481</td>
</tr>
<tr class="odd">
<td align="left">DGE</td>
<td align="left">open field</td>
<td align="right">10</td>
<td align="right">3</td>
<td align="right">28342550</td>
<td align="left">A</td>
<td align="left">G</td>
<td align="left">G</td>
<td align="left">ENSORLG00000024879</td>
<td align="left"></td>
</tr>
<tr class="even">
<td align="left">DGE</td>
<td align="left">novel object</td>
<td align="right">3</td>
<td align="right">1</td>
<td align="right">31428698</td>
<td align="left">G</td>
<td align="left">A</td>
<td align="left">A</td>
<td align="left">ENSORLG00000013962</td>
<td align="left">ecto-ADP-ribosyltransferase 5-like</td>
</tr>
<tr class="odd">
<td align="left">DGE</td>
<td align="left">novel object</td>
<td align="right">3</td>
<td align="right">10</td>
<td align="right">15319434</td>
<td align="left">C</td>
<td align="left">T</td>
<td align="left">T</td>
<td align="left">ENSORLG00000029574</td>
<td align="left"></td>
</tr>
<tr class="even">
<td align="left">DGE</td>
<td align="left">novel object</td>
<td align="right">3</td>
<td align="right">10</td>
<td align="right">15319434</td>
<td align="left">C</td>
<td align="left">T</td>
<td align="left">T</td>
<td align="left">ENSORLG00000024866</td>
<td align="left">protocadherin alpha-C2-like</td>
</tr>
<tr class="odd">
<td align="left">DGE</td>
<td align="left">novel object</td>
<td align="right">10</td>
<td align="right">10</td>
<td align="right">15319434</td>
<td align="left">C</td>
<td align="left">T</td>
<td align="left">T</td>
<td align="left">ENSORLG00000029574</td>
<td align="left"></td>
</tr>
<tr class="even">
<td align="left">DGE</td>
<td align="left">novel object</td>
<td align="right">10</td>
<td align="right">10</td>
<td align="right">15319434</td>
<td align="left">C</td>
<td align="left">T</td>
<td align="left">T</td>
<td align="left">ENSORLG00000024866</td>
<td align="left">protocadherin alpha-C2-like</td>
</tr>
<tr class="odd">
<td align="left">SGE</td>
<td align="left">open field</td>
<td align="right">13</td>
<td align="right">12</td>
<td align="right">5136828</td>
<td align="left">G</td>
<td align="left">A</td>
<td align="left">A</td>
<td align="left">ENSORLG00000002961</td>
<td align="left">complement C9</td>
</tr>
</tbody>
</table></div>
<p>One SNP, 10:15,319,434, was significant for DGE for frequencies in both state 3 (<strong>Figure <a href="MIKK-F2-chap.html#fig:sig-snp-10-15mb">5.24</a></strong>) and 10 during the novel object assay (see <strong>Figure <a href="MIKK-F2-chap.html#fig:F2-man-dge-no-3">5.22</a></strong> above for state 3), and maps to two genes: ENSORLG00000029574 and ENSORLG00000024866. ENSORLG00000029574 is a novel gene on the forward strand with no recorded phenotypes. However, ENSORLG00000024866 is a gene on the reverse strand for protocadherin alpha-C2-like, a well-known protein involved in mammalian synapse formation.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Dirk Junghans et al., &lt;span&gt;“Postsynaptic and Differential Localization to Neuronal Subtypes of Protocadherin Β16 in the Mammalian Central Nervous System,”&lt;/span&gt; &lt;em&gt;European Journal of Neuroscience&lt;/em&gt; 27, no. 3 (2008): 559–71, &lt;a href="https://doi.org/10.1111/j.1460-9568.2008.06052.x" role="doc-biblioref"&gt;https://doi.org/10.1111/j.1460-9568.2008.06052.x&lt;/a&gt;; Greg R. Phillips et al., &lt;span&gt;“Γ-&lt;span&gt;Protocadherins Are Targeted&lt;/span&gt; to &lt;span&gt;Subsets&lt;/span&gt; of &lt;span&gt;Synapses&lt;/span&gt; and &lt;span&gt;Intracellular Organelles&lt;/span&gt; in &lt;span&gt;Neurons&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Journal of Neuroscience&lt;/em&gt; 23, no. 12 (June 15, 2003): 5096–5104, &lt;a href="https://doi.org/10.1523/JNEUROSCI.23-12-05096.2003" role="doc-biblioref"&gt;https://doi.org/10.1523/JNEUROSCI.23-12-05096.2003&lt;/a&gt;.&lt;/p&gt;'><sup>288</sup></a></span></p>

<div class="figure">
<span style="display:block;" id="fig:sig-snp-10-15mb"></span>
<img src="figs/mikk_behaviour/sig_snps_boxplots/3-10:15319434.png" alt="State 3 frequency during the novel object assay for counts of the alternative allele (T) at SNP 10:15,319,434. The boxes are coloured on the left by the paternal line, and on the right by the maternal line." width="100%"><p class="caption">
Figure 5.24: State 3 frequency during the novel object assay for counts of the alternative allele (T) at SNP 10:15,319,434. The boxes are coloured on the left by the paternal line, and on the right by the maternal line.
</p>
</div>
<p>The second predicted missense variant of note was detected for SGE in the open field assay for frequencies in state 13. The locus 12:5,136,828 maps to gene ENSORLG00000002961, which is described as a complement <em>C9</em>. Discovered phenotypes for species orthologues include reduced prepulse inhibition and heart phenotypes in mice,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Mouse Genome Database, Judith A Blake et al., &lt;span&gt;“Mouse &lt;span&gt;Genome Database&lt;/span&gt; (&lt;span&gt;MGD&lt;/span&gt;): &lt;span&gt;Knowledgebase&lt;/span&gt; for Mouse–Human Comparative Biology,”&lt;/span&gt; &lt;em&gt;Nucleic Acids Research&lt;/em&gt; 49, no. D1 (January 8, 2021): D981–87, &lt;a href="https://doi.org/10.1093/nar/gkaa1083" role="doc-biblioref"&gt;https://doi.org/10.1093/nar/gkaa1083&lt;/a&gt;.&lt;/p&gt;'><sup>289</sup></a></span> and neurodevelopmental disorders in rats.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Rat Genome Database, Jennifer R Smith et al., &lt;span&gt;“The &lt;span&gt;Year&lt;/span&gt; of the &lt;span&gt;Rat&lt;/span&gt;: &lt;span&gt;The Rat Genome Database&lt;/span&gt; at 20: A Multi-Species Knowledgebase and Analysis Platform,”&lt;/span&gt; &lt;em&gt;Nucleic Acids Research&lt;/em&gt; 48, no. D1 (January 8, 2020): D731–42, &lt;a href="https://doi.org/10.1093/nar/gkz1041" role="doc-biblioref"&gt;https://doi.org/10.1093/nar/gkz1041&lt;/a&gt;.&lt;/p&gt;'><sup>290</sup></a></span> Here, in the F2 individuals, it appears to have an inconsistent effect in the behaviour of the test fishes, but apparently causes the reference fish to increase the proportion of time it spends in the fast-moving state 13 (<strong>Figure <a href="MIKK-F2-chap.html#fig:F2-man-sge-of-13">5.23</a></strong>).</p>

<div class="figure">
<span style="display:block;" id="fig:sig-snp-12-5mb"></span>
<img src="figs/mikk_behaviour/sig_snps_boxplots/13-12:5136828.png" alt="State 13 frequency during the open field assay for counts of the alternative allele (A) at SNP 12:5,136,828. The boxes are coloured on the left by the paternal line, and on the right by the maternal line." width="100%"><p class="caption">
Figure 5.25: State 13 frequency during the open field assay for counts of the alternative allele (A) at SNP 12:5,136,828. The boxes are coloured on the left by the paternal line, and on the right by the maternal line.
</p>
</div>
<p>There are additional significant loci of note shown in <strong>Figure <a href="MIKK-F2-chap.html#fig:F2-man-dge-no-3">5.22</a></strong> above that are not predicted to alter proteins, but nevertheless appear to affect genes related to neurological development (<strong>Table <a href="MIKK-F2-chap.html#tab:tbl-sig-snp-chr-9-10">5.6</a></strong>).</p>
<div class="inline-table"><table class="table table-sm">
<caption>
<span id="tab:tbl-sig-snp-chr-9-10">Table 5.6: </span>Significant loci on chromosome 9 for direct genetic effects on state 3 frequency during the novel object assay.</caption>
<colgroup>
<col width="10%">
<col width="8%">
<col width="4%">
<col width="2%">
<col width="6%">
<col width="2%">
<col width="2%">
<col width="4%">
<col width="12%">
<col width="34%">
<col width="10%">
</colgroup>
<thead><tr class="header">
<th align="left">Genetic effect</th>
<th align="left">Assay</th>
<th align="right">State</th>
<th align="right">Chr</th>
<th align="right">Pos</th>
<th align="left">Ref</th>
<th align="left">Alt</th>
<th align="left">Allele</th>
<th align="left">Gene</th>
<th align="left">Description</th>
<th align="left">Human homologue</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">DGE</td>
<td align="left">novel object</td>
<td align="right">3</td>
<td align="right">9</td>
<td align="right">9802754</td>
<td align="left">C</td>
<td align="left">A</td>
<td align="left">A</td>
<td align="left">ENSORLG00000024663</td>
<td align="left">glutamate ionotropic receptor delta type subunit 2</td>
<td align="left">GRID2</td>
</tr>
<tr class="even">
<td align="left">DGE</td>
<td align="left">novel object</td>
<td align="right">3</td>
<td align="right">10</td>
<td align="right">18537719</td>
<td align="left">C</td>
<td align="left">A</td>
<td align="left">A</td>
<td align="left">ENSORLG00000006464</td>
<td align="left">neuroligin 3a</td>
<td align="left">NLGN3</td>
</tr>
</tbody>
</table></div>
<p>The first is located around 9:9,802,754 (<strong>Figure <a href="MIKK-F2-chap.html#fig:sig-snp-9-9mb">5.26</a></strong>), and maps to a gene for glutamate ionotropic receptor delta type subunit 2, an orthologue of the human gene <em>GRID2</em>. Deletions in this gene have been found to cause ataxia in humans,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;L. Benjamin Hills et al., &lt;span&gt;“Deletions in &lt;span&gt;Grid2&lt;/span&gt; Lead to a Recessive Syndrome of Cerebellar Ataxia and Tonic Upgaze in Humans,”&lt;/span&gt; &lt;em&gt;Neurology&lt;/em&gt; 81, no. 16 (October 15, 2013): 1378–86, &lt;a href="https://doi.org/10.1212/WNL.0b013e3182a841a3" role="doc-biblioref"&gt;https://doi.org/10.1212/WNL.0b013e3182a841a3&lt;/a&gt;; G. Eda Utine et al., &lt;span&gt;“A &lt;span&gt;Homozygous Deletion&lt;/span&gt; in &lt;span&gt;Grid2 Causes&lt;/span&gt; a &lt;span&gt;Human Phenotype With Cerebellar Ataxia&lt;/span&gt; and &lt;span&gt;Atrophy&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Journal of Child Neurology&lt;/em&gt; 28, no. 7 (July 1, 2013): 926–32, &lt;a href="https://doi.org/10.1177/0883073813484967" role="doc-biblioref"&gt;https://doi.org/10.1177/0883073813484967&lt;/a&gt;.&lt;/p&gt;'><sup>291</sup></a></span> and other mutations in this gene have been found to cause various neurological disorders in mice.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Mouse Genome Database, Blake et al., &lt;span&gt;“Mouse &lt;span&gt;Genome Database&lt;/span&gt; (&lt;span&gt;MGD&lt;/span&gt;).”&lt;/span&gt;&lt;/p&gt;"><sup>292</sup></a></span></p>

<div class="figure">
<span style="display:block;" id="fig:sig-snp-9-9mb"></span>
<img src="figs/mikk_behaviour/sig_snps_boxplots/3-9:9802754.png" alt="State 3 frequency during the novel object assay for counts of the alternative allele (A) at SNP 9:9,802,754. The boxes are coloured on the left by the paternal line, and on the right by the maternal line." width="100%"><p class="caption">
Figure 5.26: State 3 frequency during the novel object assay for counts of the alternative allele (A) at SNP 9:9,802,754. The boxes are coloured on the left by the paternal line, and on the right by the maternal line.
</p>
</div>
<p>The second locus resides around 10:18,537,719 (<strong>Figure <a href="MIKK-F2-chap.html#fig:sig-snp-10-18mb">5.27</a></strong>) within the neuroligin 3 gene, an orthologue of the human gene <em>NLGN3</em>, which has been linked with autism in humans,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;St’ephane Jamain et al., &lt;span&gt;“Mutations of the &lt;span class="nocase"&gt;X-linked&lt;/span&gt; Genes Encoding Neuroligins &lt;span&gt;Nlgn3&lt;/span&gt; and &lt;span&gt;Nlgn4&lt;/span&gt; Are Associated with Autism,”&lt;/span&gt; &lt;em&gt;Nature Genetics&lt;/em&gt; 34, no. 1, 1 (May 2003): 27–29, &lt;a href="https://doi.org/10.1038/ng1136" role="doc-biblioref"&gt;https://doi.org/10.1038/ng1136&lt;/a&gt;.&lt;/p&gt;'><sup>293</sup></a></span> and various neurological in mice and rats.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Blake et al., &lt;span&gt;“Mouse &lt;span&gt;Genome Database&lt;/span&gt; (&lt;span&gt;MGD&lt;/span&gt;)”&lt;/span&gt;; Smith et al., &lt;span&gt;“The &lt;span&gt;Year&lt;/span&gt; of the &lt;span&gt;Rat&lt;/span&gt;.”&lt;/span&gt;&lt;/p&gt;"><sup>294</sup></a></span></p>

<div class="figure">
<span style="display:block;" id="fig:sig-snp-10-18mb"></span>
<img src="figs/mikk_behaviour/sig_snps_boxplots/3-10:18537719.png" alt="State 3 frequency during the novel object assay for counts of the alternative allele (A) at SNP 10:18,537,719. The boxes are coloured on the left by the paternal line, and on the right by the maternal line." width="100%"><p class="caption">
Figure 5.27: State 3 frequency during the novel object assay for counts of the alternative allele (A) at SNP 10:18,537,719. The boxes are coloured on the left by the paternal line, and on the right by the maternal line.
</p>
</div>
<p>These results provide promising evidence that the assay and methods described above can identify genetic loci associated with differences in behaviours - and potentially difference in the transmission of behaviour onto social companions - with functional relevance to humans. However, before seeking to functionally validate these variants, there are a number of steps that we seek to take to refine the analysis and thereby increase our confidence in the variants we ultimately select for validation.</p>
</div>
</div>
<div id="discussion-1" class="section level2" number="5.8">
<h2>
<span class="header-section-number">5.8</span> Discussion<a class="anchor" aria-label="anchor" href="#discussion-1"><i class="fas fa-link"></i></a>
</h2>
<p>With the benefit of hindsight, there are a several aspects of this analysis that I would have performed differently. The first relates to the choices of certain parental lines for the F2 cross, which were too heavily reliant on the aggregate measures of speed and charisma, without due regard to the level of behavioural variance observed within each line. As evident in <strong>Figures <a href="MIKK-F2-chap.html#fig:F2-time-dge-of">5.14</a></strong> and <strong><a href="MIKK-F2-chap.html#fig:F2-time-dge-no">5.15</a></strong> above, lines <span style="color: #FF699C;">8-2</span> and <span style="color: #49B500;">21-2</span> showed large within-line variances. For <span style="color: #FF699C;">8-2</span> in particular, the differences appear to correspond to the date on which they were assayed - specifically, in the first run the four <span style="color: #FF699C;">8-2</span> individuals showed almost no movement, whereas in the second run they showed much higher levels of movement. The different within-line variances are an interesting phenotype to explore further, as they suggest that different lines show different degrees of sensitivity to environmental changes. However, for this analysis, high within-line variances make it difficult to ascertain the line’s “true” phenotype, and therefore make them unsuitable for selection based on our axes of interest for this study (bold vs shy; high vs low behavioural transmission).</p>
<p>An appropriate substitute for <span style="color: #FF699C;">8-2</span> may be line <span style="color: #FF61CC;">139-4</span>, which had low within-line variance for mean speed, and the highest median state co-occupancy in the novel object assay component. From the analysis of Chapter <a href="Pilot-chap.html#Pilot-chap">3</a>, which concluded after the lines were already selected for the F2 cross, it became apparent that the novel object assay was particularly useful for revealing social genetic effects. I hypothesise that at times of higher stress or predation threat, the fish take more behavioural cues from their tank partners. Line <span style="color: #FF61CC;">139-4</span> therefore would have been a good candidate for the slow-moving, high-charisma line.</p>
<p>Similarly, lines <span style="color: #F57A5F;">13-2</span> or <span style="color: #D39200;">94-1</span> would be preferred substitutes for line <span style="color: #49B500;">21-2</span> as the fast-moving, high charisma line. They are faster than <span style="color: #49B500;">21-2</span>, but unlike the fastest line <span style="color: #F8766D;">10-1</span>, they show a degree of habituation (where most individuals moved slowly at the beginning of the assay before eventually speeding up). They also showed lower within-line variance than <span style="color: #49B500;">21-2</span>, but comparable measures for the SGE measures of state co-occupancy and reference deviation.</p>
<p>One way of avoiding these issues would have been to used a more quantitative, objective approach to selecting the lines. For example, I could have used Mann-Whitney tests to quantify the levels of differences between pairs of lines for a trait of interest (as I did for <strong>Figure <a href="MIKK-F2-chap.html#fig:mikk-param-comp">5.5</a></strong>), and then select the lines that maximised that statistic. Such metrics would still perhaps need to be qualitatively weighted against other relevant traits, such as within-line variance. Nevertheless, my downstream analyses show that despite not having potentially selected the ideal lines for the F2 cross, the differences between them were sufficient to identify genetic loci associated with our phenotypes of interest, even with a relatively small sample size of 271 F2 individuals.</p>
<p>Another aspect of the analysis that could have been improved was to use one or more MIKK panel lines as the reference line instead of <em><span style="color: #424B4D;">iCab</span></em>. As previously mentioned, <em><span style="color: #424B4D;">iCab</span></em> is a relatively fast line, which made it difficult to detect social genetic effects when paired with similarly fast-moving lines. Instead, we could have made use of the behavioural variance across the MIKK panel to choose a reference line that exhibits moderate levels of movement. However, we would not be able to assess the levels of movement across the MIKK panel without first assaying the panel with a single reference strain, and we chose <em><span style="color: #424B4D;">iCab</span></em> because we not only had a good understanding of its behavioural traits based on our findings from Chapter <a href="Pilot-chap.html#Pilot-chap">3</a>, but also because it has high fecundity, which made many replicates available and in turn allowed us to avoid using the same reference fish more than once within 24 hours. Now that we have carried out this analysis, we would be able to select a more appropriate line as the reference in any future analyses.</p>
<p>The other benefit from using a MIKK panel line as the reference fish would be to open up the possibility of exploring social genetic effect interactions. For example, if we used two different MIKK lines as the reference, we could explore how those lines differ in the degree to which they are influenced by their tank partner. To state the issue inversely, it would reveal differences in the extent to which different lines transmit their behaviour onto their (reference) tank partner, depending on both of their genetics. A question for further studies could therefore be framed as, ‘given line <span style="color: #FF66A6;">18-2</span> appears to possess a high level of charisma, is that level consistent when they are paired with different MIKK panel lines, and if it differs across lines, then why?’ Due to the relative onerousness of running this behavioural assay, such experiments would involve a substantial time commitment, but in light of the findings presented here, the investigations could be restricted to a subset of the MIKK panel that display interesting behavioural phenotypes.</p>
</div>
<div id="future-directions" class="section level2" number="5.9">
<h2>
<span class="header-section-number">5.9</span> Future directions<a class="anchor" aria-label="anchor" href="#future-directions"><i class="fas fa-link"></i></a>
</h2>
<p>Our collaborators are in the process of breeding more F2 individuals to increase the sample size for this analysis. For the subsequent breeding program, I have suggested that we replace line <span style="color: #49B500;">21-2</span> with either <span style="color: #F57A5F;">13-2</span> or <span style="color: #AFA200;">23-1</span>, which both show lower within-line variance in mean speed, but similarly high levels of speed and behavioural transmission (as measured by state co-occupancy and <em><span style="color: #424B4D;">iCab</span></em> reference deviation). Any additional F2 individuals will still need to be phenotyped and sequenced, and the phenotype data collection (i.e. video recording the behavioural assay) can be relatively laborious, so this process will likely take some time.</p>
<p>In the meantime, I have 96 Kiyosu CC individuals whose videos have already been processed, and whose DNA has been shallow-sequenced (~1x). It was not feasible to include that data in this analysis due to time constraints. As they have been allowed to breed freely from the same Kiyosu population as the MIKK panel, they are not strongly homozygous, and they are likely to possess many additional genetic variants that are not represented in the MIKK panel. I am therefore not able to impute their genotypes from high-coverage parental strains, as I did with the F2 individuals. The alignments to the reference are also likely to include many mapping errors due to the lack of sufficient coverage. The additional information they provide will therefore not be as useful as additional F2 individuals, but it should nevertheless improve the power to detect casual variants. The amount of additional information we obtain from the Kiyosu CC individuals will also inform whether we include them in this way in future F2 crosses.</p>
<p>A second issue I plan to address is the failure of F2 reads to align to chromosomes 22 and 23 (see <strong>Figure <a href="MIKK-F2-chap.html#fig:F2-recomb-blocks">5.20</a></strong> above). As the issue occurs across most of the sample, and the coverage does not look skewed for those chromosomes (<strong>Figure <a href="MIKK-F2-chap.html#fig:F2-coverage">5.28</a></strong>), there is likely to a be an error in the bioinformatic pipeline that I constructed.</p>

<div class="figure">
<span style="display:block;" id="fig:F2-coverage"></span>
<img src="figs/mikk_behaviour/coverage_F2.png" alt="Mean depth of sequencing coverage per chromosome for the 271 F2 samples used in this analysis." width="100%"><p class="caption">
Figure 5.28: Mean depth of sequencing coverage per chromosome for the 271 F2 samples used in this analysis.
</p>
</div>
<p>[TO DO: homozygous-divergent SNP counts for each cross, across chromosomes]</p>
<p>A third issue is the matter of selecting the optimal HMM parameters for calling the recombination blocks, as described in Chapter <a href="Somite-chap.html#Somite-chap">4</a>. For the analysis in this present chapter I used the same HMM parameters as I had used there. In that chapter, I used the <span class="math inline">\(p\)</span>-values from the resulting genetic linkage analysis as a rough measure of genotyping quality (assuming that lower <span class="math inline">\(p\)</span>-values were driving by more accurate genotypes), together with a qualitative assessment of how many crossovers I expected to observe in a particular chromosome for a subset of the sample. The issue can be resolved by obtaining “ground truth” genotype calls through high-coverage sequence data for a set of samples that have also been shallow-sequenced. This would allow one to compare the HMM recombination block calls with the ground truth genotypes, and thereby train the model to optimise the accuracy of its predictions for the locations of the recombination sites (using a test dataset to validate the parameters). I expect the improved genotype calls will further improve the power to detect genetic variants.</p>
<p>With respect to the genetic linkage analysis, due to the method I used to calculate the genetic relationship matrix (<strong>GRM</strong>), I was compelled to exclude all SNPs that were missing calls in any of the samples, reducing the number of SNPs to merely ~40K out of the ~20M in the full dataset. Not only does this reduce the accuracy of the GRM as a means of capturing the relative relatedness between samples, but the problem will only be exacerbated with the inclusion of additional samples (as the likelihood of a SNP having at least one sample with a missing call will increase with the number of samples). I will therefore adapt my method to include SNPs with missing calls, which should further increase statistical power.</p>
<p>Finally, I applied a separate genetic linkage association test for the state frequency of each HMM state, where it may be beneficial to apply a method that can combine the information across all states into a single phenotype (if not fewer phenotypes), and then run an association test on that reduced phenotype. There are a number of alternatives to consider, including principal components analysis (PCA) of the state frequencies, or the software MTAG<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Multi-Trait Analysis of Genome-wide summary statistics , Patrick Turley et al., &lt;span&gt;“Multi-Trait Analysis of Genome-Wide Association Summary Statistics Using &lt;span&gt;MTAG&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Nature Genetics&lt;/em&gt; 50, no. 2, 2 (February 2018): 229–37, &lt;a href="https://doi.org/10.1038/s41588-017-0009-4" role="doc-biblioref"&gt;https://doi.org/10.1038/s41588-017-0009-4&lt;/a&gt;.&lt;/p&gt;'><sup>295</sup></a></span> which can combine the summary statistics from multiple genetic association tests. Ultimately, the hope is that by leveraging the information about the frequencies in which the fishes occupy each state, combined across <em>all</em> states, while appropriately taking into account the interdependence of those frequencies (e.g. the more time a fish spends in state 3, the less time it is likely to spend in state 13), it will further strengthen these results.</p>
<p>Aside from modifications designed to increase statistical power, there is also capacity to improve the way that I phenotype the individuals, and in particular how I characterise social genetic effects. In this analysis I have only used two behavioural variables: distance and angle of travel. This behavioural dataset is extremely rich, and myriad variables could be extracted from the videos. Shoaling traits would have special relevance to the study of social genetic effects. For instance, one could extract three variables that would be associated with shoaling behaviours: a) distance between tank partners; b) synchrony in speed; and c) parallelism in direction of travel. These three variables could then be reduced down to two with an HMM (i.e. one state representing “shoaling”, and another representing “not shoaling”). The relative frequencies that the different MIKK lines spend in these states would allow one to rank them by the degree of shoaling behaviour they exhibit, and this would likely be a far more accurate representation of social genetics than what I have shown here.</p>

</div>
</div>

  <div class="chapter-nav">
<div class="prev"><a href="Somite-chap.html"><span class="header-section-number">4</span> Genetic loci associated with somite development periodicity</a></div>
<div class="next"><a href="Fst-chap.html"><span class="header-section-number">6</span> Variation in the frequency of trait-associated alleles across global human populations</a></div>
</div></main><div class="col-md-3 col-lg-2 d-none d-md-block sidebar sidebar-chapter">
    <nav id="toc" data-toggle="toc" aria-label="On this page"><h2>On this page</h2>
      <ul class="nav navbar-nav">
<li><a class="nav-link" href="#MIKK-F2-chap"><span class="header-section-number">5</span> Genetic linkage study of bold/shy behaviours in the MIKK panel</a></li>
<li><a class="nav-link" href="#MIKK-F2-cross"><span class="header-section-number">5.1</span> The F2 cross experimental setup</a></li>
<li><a class="nav-link" href="#data-collection---f0-generation"><span class="header-section-number">5.2</span> Data collection - F0 generation</a></li>
<li><a class="nav-link" href="#hmm-states"><span class="header-section-number">5.3</span> HMM states</a></li>
<li>
<a class="nav-link" href="#social-genetic-effects-1"><span class="header-section-number">5.4</span> Social genetic effects</a><ul class="nav navbar-nav">
<li><a class="nav-link" href="#state-co-occupancy"><span class="header-section-number">5.4.1</span> State co-occupancy</a></li>
<li><a class="nav-link" href="#deviation-of-icab-from-its-control-condition"><span class="header-section-number">5.4.2</span> Deviation of iCab from its control condition</a></li>
</ul>
</li>
<li><a class="nav-link" href="#selection-of-lines-for-the-f2-cross"><span class="header-section-number">5.5</span> Selection of lines for the F2 cross</a></li>
<li>
<a class="nav-link" href="#direct-genetic-effects-1"><span class="header-section-number">5.6</span> Direct genetic effects</a><ul class="nav navbar-nav"><li><a class="nav-link" href="#social-genetic-effects-2"><span class="header-section-number">5.6.1</span> Social genetic effects</a></li></ul>
</li>
<li>
<a class="nav-link" href="#f2-generation"><span class="header-section-number">5.7</span> F2 generation</a><ul class="nav navbar-nav">
<li><a class="nav-link" href="#behavioural-data-collection"><span class="header-section-number">5.7.1</span> Behavioural data collection</a></li>
<li><a class="nav-link" href="#behavioural-phenotyping"><span class="header-section-number">5.7.2</span> Behavioural phenotyping</a></li>
<li><a class="nav-link" href="#genotyping"><span class="header-section-number">5.7.3</span> Genotyping</a></li>
<li><a class="nav-link" href="#genetic-linkage-analysis"><span class="header-section-number">5.7.4</span> Genetic linkage analysis</a></li>
<li><a class="nav-link" href="#candidate-snps-for-crispr-knockouts"><span class="header-section-number">5.7.5</span> Candidate SNPs for CRISPR-knockouts</a></li>
</ul>
</li>
<li><a class="nav-link" href="#discussion-1"><span class="header-section-number">5.8</span> Discussion</a></li>
<li><a class="nav-link" href="#future-directions"><span class="header-section-number">5.9</span> Future directions</a></li>
</ul>

      <div class="book-extra">
        <ul class="list-unstyled">
<li><a id="book-source" href="https://github.com/brettellebi/PhD-thesis/blob/master/book/05-MIKK_F2.Rmd">View source <i class="fab fa-github"></i></a></li>
          <li><a id="book-edit" href="https://github.com/brettellebi/PhD-thesis/edit/master/book/05-MIKK_F2.Rmd">Edit this page <i class="fab fa-github"></i></a></li>
        </ul>
</div>
    </nav>
</div>

</div>
</div> <!-- .container -->

<footer class="bg-primary text-light mt-5"><div class="container"><div class="row">

  <div class="col-12 col-md-6 mt-3">
    <p>"<strong>Japanese courage: a genetic analysis of complex traits in medaka fish and humans</strong>" was written by Ian Brettell. It was last built on 2022-08-31.</p>
  </div>

  <div class="col-12 col-md-6 mt-3">
    <p>This book was built by the <a class="text-light" href="https://bookdown.org">bookdown</a> R package.</p>
  </div>

</div></div>
</footer><!-- dynamically load mathjax for compatibility with self-contained --><script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.9/latest.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script><script type="text/x-mathjax-config">const popovers = document.querySelectorAll('a.footnote-ref[data-toggle="popover"]');
for (let popover of popovers) {
  const div = document.createElement('div');
  div.setAttribute('style', 'position: absolute; top: 0, left:0; width:0, height:0, overflow: hidden; visibility: hidden;');
  div.innerHTML = popover.getAttribute('data-content');

  var has_math = div.querySelector("span.math");
  if (has_math) {
    document.body.appendChild(div);
    MathJax.Hub.Queue(["Typeset", MathJax.Hub, div]);
    MathJax.Hub.Queue(function() {
      popover.setAttribute('data-content', div.innerHTML);
      document.body.removeChild(div);
    })
  }
}
</script>
</body>
</html>
