<!DOCTYPE html>
<html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<title>Chapter 4 Genetic linkage study of bold/shy behaviours in the MIKK panel | Genetic analysis of quantitative traits in medaka fish and humans</title>
<meta name="author" content="Ian Brettell">
<meta name="description" content="The purpose of the study described in this chapter was to run the behavioural analysis described in Chapter 3 over the MIKK panel described in Chapter 2, identify the lines that diverged in both...">
<meta name="generator" content="bookdown 0.26 with bs4_book()">
<meta property="og:title" content="Chapter 4 Genetic linkage study of bold/shy behaviours in the MIKK panel | Genetic analysis of quantitative traits in medaka fish and humans">
<meta property="og:type" content="book">
<meta property="og:description" content="The purpose of the study described in this chapter was to run the behavioural analysis described in Chapter 3 over the MIKK panel described in Chapter 2, identify the lines that diverged in both...">
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="Chapter 4 Genetic linkage study of bold/shy behaviours in the MIKK panel | Genetic analysis of quantitative traits in medaka fish and humans">
<meta name="twitter:description" content="The purpose of the study described in this chapter was to run the behavioural analysis described in Chapter 3 over the MIKK panel described in Chapter 2, identify the lines that diverged in both...">
<!-- JS --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://kit.fontawesome.com/6ecbd6c532.js" crossorigin="anonymous"></script><script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="libs/bootstrap-4.6.0/bootstrap.min.css" rel="stylesheet">
<script src="libs/bootstrap-4.6.0/bootstrap.bundle.min.js"></script><script src="libs/bs3compat-0.4.0/transition.js"></script><script src="libs/bs3compat-0.4.0/tabs.js"></script><script src="libs/bs3compat-0.4.0/bs3compat.js"></script><link href="libs/bs4_book-1.0.0/bs4_book.css" rel="stylesheet">
<script src="libs/bs4_book-1.0.0/bs4_book.js"></script><style>
    #content {
        font-size: 0.9rem;
    }
    </style>
<script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- CSS --><style type="text/css">
    
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  </style>
<style type="text/css">
    /* Used with Pandoc 2.11+ new --citeproc when CSL is used */
    div.csl-bib-body { }
    div.csl-entry {
      clear: both;
        }
    .hanging div.csl-entry {
      margin-left:2em;
      text-indent:-2em;
    }
    div.csl-left-margin {
      min-width:2em;
      float:left;
    }
    div.csl-right-inline {
      margin-left:2em;
      padding-left:1em;
    }
    div.csl-indent {
      margin-left: 2em;
    }
  </style>
<link rel="stylesheet" href="style.css">
</head>
<body data-spy="scroll" data-target="#toc">

<div class="container-fluid">
<div class="row">
  <header class="col-sm-12 col-lg-3 sidebar sidebar-book"><a class="sr-only sr-only-focusable" href="#content">Skip to main content</a>

    <div class="d-flex align-items-start justify-content-between">
      <h1>
        <a href="index.html" title="">Genetic analysis of quantitative traits in medaka fish and humans</a>
      </h1>
      <button class="btn btn-outline-primary d-lg-none ml-2 mt-1" type="button" data-toggle="collapse" data-target="#main-nav" aria-expanded="true" aria-controls="main-nav"><i class="fas fa-bars"></i><span class="sr-only">Show table of contents</span></button>
    </div>

    <div id="main-nav" class="collapse-lg">
      <form role="search">
        <input id="search" class="form-control" type="search" placeholder="Search" aria-label="Search">
</form>

      <nav aria-label="Table of contents"><h2>Table of contents</h2>
        <ul class="book-toc list-unstyled">
<li><a class="" href="index.html">About</a></li>
<li><a class="" href="Introduction.html"><span class="header-section-number">1</span> Introduction</a></li>
<li><a class="" href="MIKK-genomes-chap.html"><span class="header-section-number">2</span> Genomic variations in the MIKK panel</a></li>
<li><a class="" href="Pilot-chap.html"><span class="header-section-number">3</span> Classification of bold/shy behaviours in 5 inbred medaka lines</a></li>
<li><a class="active" href="MIKK-F2-chap.html"><span class="header-section-number">4</span> Genetic linkage study of bold/shy behaviours in the MIKK panel</a></li>
<li><a class="" href="genetic-loci-associated-with-somite-development-periodicity.html"><span class="header-section-number">5</span> Genetic loci associated with somite development periodicity</a></li>
<li><a class="" href="Fst-chap.html"><span class="header-section-number">6</span> Variation in the frequency of trait-associated alleles across global human populations</a></li>
<li><a class="" href="references.html">References</a></li>
<li class="book-part">Appendix</li>
<li><a class="" href="ecdf-of-all-polygenic-traits-in-the-gwas-catalog-ranked-by-d_ts.html"><span class="header-section-number">A</span> eCDF of all polygenic traits in the GWAS Catalog ranked by \({D_t^S}\)</a></li>
<li><a class="" href="hmm-state-time-dependence-for-all-mikk-panel-lines.html"><span class="header-section-number">B</span> HMM state time dependence for all MIKK panel lines</a></li>
</ul>

        <div class="book-extra">
          <p><a id="book-repo" href="https://github.com/brettellebi/PhD-thesis">View book source <i class="fab fa-github"></i></a></p>
        </div>
      </nav>
</div>
  </header><main class="col-sm-12 col-md-9 col-lg-7" id="content"><div id="MIKK-F2-chap" class="section level1" number="4">
<h1>
<span class="header-section-number">4</span> Genetic linkage study of bold/shy behaviours in the MIKK panel<a class="anchor" aria-label="anchor" href="#MIKK-F2-chap"><i class="fas fa-link"></i></a>
</h1>
<p>The purpose of the study described in this chapter was to run the behavioural analysis described in Chapter <a href="Pilot-chap.html#Pilot-chap">3</a> over the MIKK panel described in Chapter <a href="MIKK-genomes-chap.html#MIKK-genomes-chap">2</a>, identify the lines that diverged in both their behaviour, and the level of transmission of their behaviour onto their <em>iCab</em> reference tank partner, and then use them as the parental strains in an F2 cross to attempt to identify the genetic variants associated with those differences.</p>
<div id="MIKK-F2-cross" class="section level2" number="4.1">
<h2>
<span class="header-section-number">4.1</span> F2 cross<a class="anchor" aria-label="anchor" href="#MIKK-F2-cross"><i class="fas fa-link"></i></a>
</h2>
<p>The F2 cross is a traditional method for mapping genetic variants associated with traits of interest. A schema for the method is presented in <strong>Figure <a href="MIKK-F2-chap.html#fig:F2-cross-schema">4.1</a></strong>. In essence, it involves starting with two inbred strains that diverge for the trait of interest (the ‘parental strains’, or F0 generation F0). One then crosses the parental strains to create a generation of F1 hybrid individuals who each possess, for every pair of their chromosomes, one chromosome from each of their parents. The individuals in this F1 generation are genetically identical to their parents with respect to their germline. Finally, one inter-crosses the F1 generation to create a set of F2 individuals that share unique combinations of the original F0 strains’ genotypes, and tend to display values for the trait of interest that span across the spectrum between the extreme values of their parents.</p>

<div class="figure">
<span style="display:block;" id="fig:F2-cross-schema"></span>
<img src="figs/mikk_behaviour/F2-cross-schema.png" alt="Schema of the F2-cross experimental setup. The F0 generation comprises two medaka strains that have extreme, opposing values for a trait of interest, represented by the colours red and blue. Below them is an illustrative single pair of chromosomes. The chromosomes within each pair are depicted as the same colour, as the strains are homozygous through successive generations of inbreeding. Their F1 offspring is heterozygous for each pair of their 24 chromosomes, and all F1 individuals are therefore almost genetically identical to one another (with the exception of somatic mutations and the regions of the genome that were not homozygous in the parental generations). The F1 individuals are then inter-crossed with one another to produce the F2 generation, which, due to recombination events during gamete formation, have unqiue combinations of the parental strains’ genotypes, and tend to span the phenotypic spectrum between the extremes of their F0 parental strains, represented by their colours." width="100%"><p class="caption">
Figure 4.1: Schema of the F2-cross experimental setup. The F0 generation comprises two medaka strains that have extreme, opposing values for a trait of interest, represented by the colours red and blue. Below them is an illustrative single pair of chromosomes. The chromosomes within each pair are depicted as the same colour, as the strains are homozygous through successive generations of inbreeding. Their F1 offspring is heterozygous for each pair of their 24 chromosomes, and all F1 individuals are therefore almost genetically identical to one another (with the exception of somatic mutations and the regions of the genome that were not homozygous in the parental generations). The F1 individuals are then inter-crossed with one another to produce the F2 generation, which, due to recombination events during gamete formation, have unqiue combinations of the parental strains’ genotypes, and tend to span the phenotypic spectrum between the extremes of their F0 parental strains, represented by their colours.
</p>
</div>
</div>
<div id="data-collection---f0-generation" class="section level2" number="4.2">
<h2>
<span class="header-section-number">4.2</span> Data collection - F0 generation<a class="anchor" aria-label="anchor" href="#data-collection---f0-generation"><i class="fas fa-link"></i></a>
</h2>
<p>In November 2019 I traveled to the fish facility managed by our collaborator, Felix Loosli at KIT in Karlsruhe, and over the course of 11 days from 11 to 21 November 2019, I ran the behavioural assay described in Chapter <a href="Pilot-chap.html#pilot-data-collection">3.4.1</a> another 206 times. I again used the <em>iCab</em> strain as the reference fish, and for the test fish I used either an individual from one of the MIKK panel lines, individuals captured from the same Kiyosu population as the MIKK panel but permitted to breed freely within a separate tank in the facility (‘Kiyosu closed-capture’, or ‘Kiyosu CC’), or individuals from a related but different species of medaka from the Philippines, <em>Oryzias luzonensis</em>. I ensured that I performed at least 2 assay runs for each MIKK panel line that was available, generating a minimum of 8 test fish replicates per line. As there were four pairs of fish in the test tank during each run, the complete dataset comprises 824 videos of pairs of fish, which I further divided by assay component (open field and novel object) to create 1648 videos.</p>
<p>I again used the software <em>idtrackerai</em><span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Romero-Ferrero et al., &lt;span&gt;“Idtracker.ai.”&lt;/span&gt;&lt;/p&gt;"><sup>78</sup></a></span> to track the movement of the fishes across frames of each video. After adjusting the software parameters for each video to maximise the number of frames that were successfully tracked, I was left with 1610 out of the 1648 videos (~97.7%) where both fishes were tracked over at least 85% of frames. Only these 1610 videos were used in the downstream analysis. The first question to address was whether the MIKK panel lines differed in their behaviours. I therefore computed each individual fish’s mean speed (measured as the distance traveled in pixels per 0.05 seconds) over the course of the full 20-minute video, grouped them by line, and plotted the results presented in <strong>Figure <a href="MIKK-F2-chap.html#fig:mikk-mean-speed">4.2</a></strong>.</p>

</div>
<div id="mean-speed-for-all-mikk-lines" class="section level2" number="4.3">
<h2>
<span class="header-section-number">4.3</span> Mean speed for all MIKK lines<a class="anchor" aria-label="anchor" href="#mean-speed-for-all-mikk-lines"><i class="fas fa-link"></i></a>
</h2>
<div class="figure">
<span style="display:block;" id="fig:mikk-mean-speed"></span>
<img src="figs/mikk_behaviour/F0_line_mean_speed_0.08.png" alt="Mean speed of the MIKK panel and other strains over the course of the entire 20-minute video (measured as the distance traveled in pixels per 0.05 seconds). iCab fishes in the iCab-iCab control condition are at the top, the MIKK panel lines are sorted by their group median, and the Kiyosu closed capture and O. luzonensis fishes are at the bottom." width="100%"><p class="caption">
Figure 4.2: Mean speed of the MIKK panel and other strains over the course of the entire 20-minute video (measured as the distance traveled in pixels per 0.05 seconds). <em>iCab</em> fishes in the <em>iCab</em>-<em>iCab</em> control condition are at the top, the MIKK panel lines are sorted by their group median, and the Kiyosu closed capture and <em>O. luzonensis</em> fishes are at the bottom.
</p>
</div>
<p>This figure shows that there are clear differences between some MIKK panel lines at the extremes, and that the lines differ in the amount of within-line variance observed. This figure acted as a guide to determine which lines to select as the parental strains in the F2 cross. To identify genetic variants directly associated with bold-shy behaviours, I sought to select lines that showed either high or low levels of movement, and preferably low within-line variance.</p>
<div class="figure">
<span style="display:block;" id="fig:mikk-mean-speed-variance"></span>
<img src="figs/mikk_behaviour/line_mean_speed_variance_0.05_all.png" alt="(ref:mikk-mean-speed-variance)" width="100%"><p class="caption">
Figure 4.3: (ref:mikk-mean-speed-variance)
</p>
</div>
</div>
<div id="hmm-states" class="section level2" number="4.4">
<h2>
<span class="header-section-number">4.4</span> HMM states<a class="anchor" aria-label="anchor" href="#hmm-states"><i class="fas fa-link"></i></a>
</h2>
<p>To examine these behaviours at a finer resolution, as for the pilot study described in <a href="Pilot-chap.html#Pilot-chap">3</a>, I again applied a hidden markov model (<strong>HMM</strong>) to classify the fishes’ movements based on their distance and angle of travel between time intervals. I used the same method to select the best choice of time interval and number of states (<strong>Figure <a href="#mikk-param-comp"><strong>??</strong></a></strong>). Here we observed the same phenomenon where the parameter combinations that performed the best showed an asymmetry between some states that would make interpretation difficult. For example, a time interval of 0.08 seconds combined with a state space of 17 caused state 4 to appear to get carved out of state 3 (<strong>Figure <a href="MIKK-F2-chap.html#fig:mikk-hmm-asym">4.5</a></strong>).</p>
<div class="figure">
<span style="display:block;" id="fig:mikk-param-comp"></span>
<img src="figs/mikk_behaviour/compare_params.png" alt="(ref:mikk-param-comp)" width="100%"><p class="caption">
Figure 4.4: (ref:mikk-param-comp)
</p>
</div>

<div class="figure">
<span style="display:block;" id="fig:mikk-hmm-asym"></span>
<img src="figs/mikk_behaviour/0.08_17_polar_all_dge.png" alt="The best apparent combination of parameters (0.08 time interval with a 17-state space) created an asymmetry between states 3 and 4, which would causes difficulties in interpreting their biological relevance." width="100%"><p class="caption">
Figure 4.5: The best apparent combination of parameters (0.08 time interval with a 17-state space) created an asymmetry between states 3 and 4, which would causes difficulties in interpreting their biological relevance.
</p>
</div>
<p>I therefore selected the best combination of parameters without this asymmetry, which in this case was a time interval of 0.05 seconds with a state space of 15 (<strong>Figure <a href="MIKK-F2-chap.html#fig:mikk-hmm-sym">4.6</a></strong>), and used these states for all downstream analysis.</p>

<div class="figure">
<span style="display:block;" id="fig:mikk-hmm-sym"></span>
<img src="figs/mikk_behaviour/0.05_15_polar_all_dge.png" alt="The HMM states used for the downstream analysis, with the model classified based on the distance of travel (log10 pixels, radial axis) and angle of travel (angle). A straight forward movement would sit around 0°, a left movement around 270°, and a right movement around 90°." width="100%"><p class="caption">
Figure 4.6: The HMM states used for the downstream analysis, with the model classified based on the distance of travel (log<sub>10</sub> pixels, radial axis) and angle of travel (angle). A straight forward movement would sit around 0°, a left movement around 270°, and a right movement around 90°.
</p>
</div>
<div class="figure">
<span style="display:block;" id="fig:F0-line-mean-speed-select"></span>
<img src="figs/mikk_behaviour/line_mean_speed_0.05_selected.png" alt="(ref:F0-line-mean-speed-select)" width="100%"><p class="caption">
Figure 4.7: (ref:F0-line-mean-speed-select)
</p>
</div>
<div class="figure">
<img src="figs/mikk_behaviour/line_mean_speed_variance_selected.png" alt="(ref:F0-line-mean-speed-var_select)" width="100%"><p class="caption">
(#fig:F0-line-mean-speed-var_select)(ref:F0-line-mean-speed-var_select)
</p>
</div>

<div class="figure">
<span style="display:block;" id="fig:F2-time-dge-of"></span>
<img src="figs/mikk_behaviour/select_0.05_15_dge_of.png" alt="HMM states that significantly differed between" width="100%"><p class="caption">
Figure 4.8: HMM states that significantly differed between
</p>
</div>

</div>
</div>

  <div class="chapter-nav">
<div class="prev"><a href="Pilot-chap.html"><span class="header-section-number">3</span> Classification of bold/shy behaviours in 5 inbred medaka lines</a></div>
<div class="next"><a href="genetic-loci-associated-with-somite-development-periodicity.html"><span class="header-section-number">5</span> Genetic loci associated with somite development periodicity</a></div>
</div></main><div class="col-md-3 col-lg-2 d-none d-md-block sidebar sidebar-chapter">
    <nav id="toc" data-toggle="toc" aria-label="On this page"><h2>On this page</h2>
      <ul class="nav navbar-nav">
<li><a class="nav-link" href="#MIKK-F2-chap"><span class="header-section-number">4</span> Genetic linkage study of bold/shy behaviours in the MIKK panel</a></li>
<li><a class="nav-link" href="#MIKK-F2-cross"><span class="header-section-number">4.1</span> F2 cross</a></li>
<li><a class="nav-link" href="#data-collection---f0-generation"><span class="header-section-number">4.2</span> Data collection - F0 generation</a></li>
<li><a class="nav-link" href="#mean-speed-for-all-mikk-lines"><span class="header-section-number">4.3</span> Mean speed for all MIKK lines</a></li>
<li><a class="nav-link" href="#hmm-states"><span class="header-section-number">4.4</span> HMM states</a></li>
</ul>

      <div class="book-extra">
        <ul class="list-unstyled">
<li><a id="book-source" href="https://github.com/brettellebi/PhD-thesis/blob/master/book/04-MIKK_F2.Rmd">View source <i class="fab fa-github"></i></a></li>
          <li><a id="book-edit" href="https://github.com/brettellebi/PhD-thesis/edit/master/book/04-MIKK_F2.Rmd">Edit this page <i class="fab fa-github"></i></a></li>
        </ul>
</div>
    </nav>
</div>

</div>
</div> <!-- .container -->

<footer class="bg-primary text-light mt-5"><div class="container"><div class="row">

  <div class="col-12 col-md-6 mt-3">
    <p>"<strong>Genetic analysis of quantitative traits in medaka fish and humans</strong>" was written by Ian Brettell. It was last built on 2022-08-08.</p>
  </div>

  <div class="col-12 col-md-6 mt-3">
    <p>This book was built by the <a class="text-light" href="https://bookdown.org">bookdown</a> R package.</p>
  </div>

</div></div>
</footer><!-- dynamically load mathjax for compatibility with self-contained --><script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script><script type="text/x-mathjax-config">const popovers = document.querySelectorAll('a.footnote-ref[data-toggle="popover"]');
for (let popover of popovers) {
  const div = document.createElement('div');
  div.setAttribute('style', 'position: absolute; top: 0, left:0; width:0, height:0, overflow: hidden; visibility: hidden;');
  div.innerHTML = popover.getAttribute('data-content');

  var has_math = div.querySelector("span.math");
  if (has_math) {
    document.body.appendChild(div);
    MathJax.Hub.Queue(["Typeset", MathJax.Hub, div]);
    MathJax.Hub.Queue(function() {
      popover.setAttribute('data-content', div.innerHTML);
      document.body.removeChild(div);
    })
  }
}
</script>
</body>
</html>
