% Options for packages loaded elsewhere
\PassOptionsToPackage{unicode}{hyperref}
\PassOptionsToPackage{hyphens}{url}
%
\documentclass[
]{book}
\usepackage{amsmath,amssymb}
\usepackage{lmodern}
\usepackage{iftex}
\ifPDFTeX
  \usepackage[T1]{fontenc}
  \usepackage[utf8]{inputenc}
  \usepackage{textcomp} % provide euro and other symbols
\else % if luatex or xetex
  \usepackage{unicode-math}
  \defaultfontfeatures{Scale=MatchLowercase}
  \defaultfontfeatures[\rmfamily]{Ligatures=TeX,Scale=1}
  \setmainfont[]{Libre Baskerville}
\fi
% Use upquote if available, for straight quotes in verbatim environments
\IfFileExists{upquote.sty}{\usepackage{upquote}}{}
\IfFileExists{microtype.sty}{% use microtype if available
  \usepackage[]{microtype}
  \UseMicrotypeSet[protrusion]{basicmath} % disable protrusion for tt fonts
}{}
\makeatletter
\@ifundefined{KOMAClassName}{% if non-KOMA class
  \IfFileExists{parskip.sty}{%
    \usepackage{parskip}
  }{% else
    \setlength{\parindent}{0pt}
    \setlength{\parskip}{6pt plus 2pt minus 1pt}}
}{% if KOMA class
  \KOMAoptions{parskip=half}}
\makeatother
\usepackage{xcolor}
\usepackage[left=4cm, right=3cm, top=2.5cm, bottom=2.5cm]{geometry}
\usepackage{listings}
\newcommand{\passthrough}[1]{#1}
\lstset{defaultdialect=[5.3]Lua}
\lstset{defaultdialect=[x86masm]Assembler}
\usepackage{longtable,booktabs,array}
\usepackage{calc} % for calculating minipage widths
% Correct order of tables after \paragraph or \subparagraph
\usepackage{etoolbox}
\makeatletter
\patchcmd\longtable{\par}{\if@noskipsec\mbox{}\fi\par}{}{}
\makeatother
% Allow footnotes in longtable head/foot
\IfFileExists{footnotehyper.sty}{\usepackage{footnotehyper}}{\usepackage{footnote}}
\makesavenoteenv{longtable}
\usepackage{graphicx}
\makeatletter
\def\maxwidth{\ifdim\Gin@nat@width>\linewidth\linewidth\else\Gin@nat@width\fi}
\def\maxheight{\ifdim\Gin@nat@height>\textheight\textheight\else\Gin@nat@height\fi}
\makeatother
% Scale images if necessary, so that they will not overflow the page
% margins by default, and it is still possible to overwrite the defaults
% using explicit options in \includegraphics[width, height, ...]{}
\setkeys{Gin}{width=\maxwidth,height=\maxheight,keepaspectratio}
% Set default figure placement to htbp
\makeatletter
\def\fps@figure{htbp}
\makeatother
\setlength{\emergencystretch}{3em} % prevent overfull lines
\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}
\setcounter{secnumdepth}{5}
\newlength{\cslhangindent}
\setlength{\cslhangindent}{1.5em}
\newlength{\csllabelwidth}
\setlength{\csllabelwidth}{3em}
\newlength{\cslentryspacingunit} % times entry-spacing
\setlength{\cslentryspacingunit}{\parskip}
\newenvironment{CSLReferences}[2] % #1 hanging-ident, #2 entry spacing
 {% don't indent paragraphs
  \setlength{\parindent}{0pt}
  % turn on hanging indent if param 1 is 1
  \ifodd #1
  \let\oldpar\par
  \def\par{\hangindent=\cslhangindent\oldpar}
  \fi
  % set entry spacing
  \setlength{\parskip}{#2\cslentryspacingunit}
 }%
 {}
\usepackage{calc}
\newcommand{\CSLBlock}[1]{#1\hfill\break}
\newcommand{\CSLLeftMargin}[1]{\parbox[t]{\csllabelwidth}{#1}}
\newcommand{\CSLRightInline}[1]{\parbox[t]{\linewidth - \csllabelwidth}{#1}\break}
\newcommand{\CSLIndent}[1]{\hspace{\cslhangindent}#1}
\usepackage{float}
%\usepackage{newtxmath}
\usepackage{amsmath}
\usepackage{listings}
%\floatplacement{figure}{H}
\usepackage{booktabs}
\usepackage{setspace}
\usepackage{fontspec}
\onehalfspacing
\usepackage{xcolor}
\usepackage{titlesec}
\usepackage{xfrac}
% Make caption labels bold: https://bookdown.org/yihui/rmarkdown-cookbook/latex-extra.html
\usepackage[labelfont={bf}]{caption} 
% force floats forward: https://bookdown.org/yihui/rmarkdown-cookbook/figure-placement.html
\usepackage{flafter}
% https://tex.stackexchange.com/questions/279/how-do-i-ensure-that-figures-appear-in-the-section-theyre-associated-with
\usepackage{placeins}
\lstset{
  breaklines=true
}
\usepackage{wrapfig}
\usepackage{lipsum}
\usepackage{caption}
\captionsetup[figure]{font=small}
% following code copied from https://bookdown.org/yihui/rmarkdown-cookbook/figure-placement.html#fnref10 
\renewcommand{\topfraction}{.85}
\renewcommand{\bottomfraction}{.7}
\renewcommand{\textfraction}{.15}
\renewcommand{\floatpagefraction}{.66}
\setcounter{topnumber}{3}
\setcounter{bottomnumber}{3}
\setcounter{totalnumber}{4}

%% https://stackoverflow.com/questions/3275770/modifying-section-to-make-it-colorful-with-latex
%\usepackage{color}
%\titleformat{\section}
%{\color{red}}


% https://tex.stackexchange.com/questions/10320/section-and-subsection-colors-using-titlesec
%\makeatletter
%\newcommand*\@secondofsix[6]{#2}
%\newcommand{\addtotitleformat}{%
%  \@ifstar{\addtotitleformat@star}{\addtotitleformat@nostar}}
%\newcommand\addtotitleformat@nostar[2]{%
%  \PackageError{titlesec}{non starred form of \string\addtotitleformat\space not supported}{}}
%\newcommand\addtotitleformat@star[2]{%
%  \expandafter\expandafter\expandafter\expandafter
%  \expandafter\expandafter\expandafter\def
%  \expandafter\expandafter\expandafter\expandafter
%  \expandafter\expandafter\expandafter\@currentsection@font
%  \expandafter\expandafter\expandafter\expandafter
%  \expandafter\expandafter\expandafter{%
%    \expandafter\expandafter\expandafter\@secondofsix
%       \csname ttlf@\expandafter\@gobble\string#1\endcsname}%
%  \titleformat*{#1}{\@currentsection@font#2}%
%}
%\makeatother
%
%\addtotitleformat*{\section}{\Huge\color{red}}
%\addtotitleformat*{\subsection}{\sffamily\color{blue}}
\usepackage{titling}
\usepackage{subcaption}
\pretitle{\begin{center} \includegraphics[width=2in,height=2in]{/Users/brettell/Documents/Repositories/PhD-thesis/book/figs/title/Arms_PembrokeCollege_Cambridge.pdf}\LARGE\\ \bigskip \bigskip \bigskip }
\posttitle{\end{center}}
\predate{\begin{center} Pembroke College \\ \bigskip }
\postdate{\end{center} \begin{center} \bigskip \bigskip \bigskip This thesis is submitted for the degree of Doctor of Philosophy. \end{center} \centering \begin{figure}[!tpb] \centering \begin{subfigure}{0.49 \linewidth} \centering \vfill \includegraphics[height=0.45in]{/Users/brettell/Documents/Repositories/PhD-thesis/book/figs/title/cambridge_university2.pdf} \end{subfigure} \hfill \begin{subfigure}{0.49 \linewidth} \centering \includegraphics[height=0.55in]{/Users/brettell/Documents/Repositories/PhD-thesis/book/figs/title/EMBL_EBI_Logo_black.pdf} \end{subfigure} \end{figure} }
\usepackage{float}
\usepackage{booktabs}
\usepackage{longtable}
\usepackage{array}
\usepackage{multirow}
\usepackage{wrapfig}
\usepackage{colortbl}
\usepackage{pdflscape}
\usepackage{tabu}
\usepackage{threeparttable}
\usepackage{threeparttablex}
\usepackage[normalem]{ulem}
\usepackage{makecell}
\usepackage{xcolor}
\ifLuaTeX
  \usepackage{selnolig}  % disable illegal ligatures
\fi
\IfFileExists{bookmark.sty}{\usepackage{bookmark}}{\usepackage{hyperref}}
\IfFileExists{xurl.sty}{\usepackage{xurl}}{} % add URL line breaks if available
\urlstyle{same} % disable monospaced font for URLs
\hypersetup{
  pdftitle={Japanese courage: a genetic analysis of complex traits in medaka fish and humans},
  pdfauthor={Ian Narain Brettell},
  hidelinks,
  pdfcreator={LaTeX via pandoc}}

\title{Japanese courage: a genetic analysis of complex traits in medaka fish and humans}
\author{Ian Narain Brettell}
\date{30 September 2022}

\begin{document}
\maketitle

{
\setcounter{tocdepth}{1}
\tableofcontents
}
\hypertarget{acknowledgements}{%
\chapter*{Acknowledgements}\label{acknowledgements}}
\addcontentsline{toc}{chapter}{Acknowledgements}

\hypertarget{preface}{%
\chapter*{Preface}\label{preface}}
\addcontentsline{toc}{chapter}{Preface}

This thesis is the result of my own work and includes nothing which is the outcome of work done in collaboration except as declared in the preface and specified in the text.

It is not substantially the same as any work that has already been submitted before for any degree or other qualification except as declared in the preface and specified in the text.

It does not exceed the prescribed word limit for the Degree Committee for the Faculty of Biology.

\hypertarget{abstract}{%
\chapter*{Abstract}\label{abstract}}
\addcontentsline{toc}{chapter}{Abstract}

This thesis primarily explores how an individual's genes interact with the genes of their social companions to create differences in behaviour, using the Japanese medaka fish as a model organism. Chapter 1 sets out the introduction to the diverse topics covered in this thesis, followed by five substantive chapters.

Chapter 2 describes several genomic characteristics of the Medaka Inbred Kiyosu-Karlsruhe (MIKK) panel, which comprises 80 inbred lines of medaka that were bred from a wild population from the city of Kiyosu, southern Japan. In this chapter I plot the inbreeding trajectory of the MIKK panel and analyse a number of genomic characteristics relevant to its utility for the genetic mapping of complex traits, including: the panel's evolutionary relationship with other previously-established inbred medaka strains; the degree of homozygosity in the inbred lines; the rate of linkage disequilibrium decay across the panel; and the genomic repeats and structural variation present in their genomes.

In Chapter 3, I use a custom behavioural assay to characterise and classify bold-shy behaviours in 5 previously established inbred medaka strains. I describe the assay, assess its robustness against confounding factors, and apply a hidden markov model (HMM) to classify the fishes' behaviours across a spectrum of boldness-shyness based on the individuals' distance and angle of travel between pre-defined time intervals. I describe how the strains differ in their behaviours over the course of the assay (a ``direct genetic effect'') and how the behaviour of a single ``reference'' strain (\emph{iCab}) differs in the presence of different strains (a ``social genetic effect'').

In Chapter 4, I describe the bioinformatic processes and genetic association models that I used to map the variants associated with variation in the period of somite development, based on an F\textsubscript{2} cross between the southern Japanese \emph{iCab} strain, and the northern Japanese \emph{Kaga} strain.

In Chapter 5, I explain how I ran the custom behavioural assay described in Chapter 3 over the MIKK panel to identify lines that diverge in both their own bold-shy behaviours (the direct genetic effect) and the extent to which they transmit those behaviours onto their tank partners (the social genetic effect). I then describe how I used those divergent lines as the parental lines in a multi-way F\textsubscript{2} cross to identify the genetic variants associated with both direct and social genetic effects.

Finally, in Chapter 6, I turn to humans to compare and rank all complex traits in the GWAS Catalog based on the extent to which their associated alleles vary across global populations, using the Fixation Index (\(F_{ST}\)) as a metric, and the 1000 Genomes dataset as a sample of global genetic variation. In this chapter I set out the bioinformatic pipelines used to process the data, present the distributions of \(F_{ST}\) for trait-associated alleles across the genome, and use the Kolmogorov-Smirnov test to compare the distributions of \(F_{ST}\) across different traits.

Altogether, this thesis describes some of the genomic characteristics of both medaka fish and humans, and how those variations relate to differences in complex traits, with a particular focus on the genetic causes of adaptive behaviours and the transmission of those behaviours onto one's social companions.

\hypertarget{Introduction}{%
\chapter{Introduction}\label{Introduction}}

Humankind has long sought to understand the basis of biological variation. What gives rise to the wondrous variety of life forms on Earth? Why do individuals of a particular species differ from one another? How do children inherit traits that are similar to those of their parents, yet on the whole remain distinct from both their parents and their siblings? Are the traits we care about -- our health, our intelligence, our behaviour, our ability to thrive in a changing world -- pre-determined from birth, or continuously pliable throughout our lives? And in particular, as social creatures, to what extent does our social environment affect these outcomes? Such questions fundamentally concern the natural laws of inheritance, which until recently remained obscure. Rapidly-improving technologies are making these inquiries increasingly tractable, yet the interplay between genes and environment are still the subject of controversy, especially in relation to human traits such as intelligence.

In this thesis, I primarily explore the extent to which phenotypic variation is determined by an individual's own genes, and how much it is mediated by the genes of one's social companions. As a model for these dynamics, I focus on the phenotype of bold-type behaviours in the Japanese rice paddy fish medaka (\emph{Oryzias latipes}). In this introduction I first provide a brief history of the field of genetics, describing some of the work of the scientists on whose shoulders this research stands, from Charles Darwin's theory of evolution, Mendel's identification of the units of inheritance, and the study of continuous variation by Francis Galton, Karl Pearson, Ronald Fisher, and others, while also touching on their other legacies. I then outline the traditional animal crossing methods and modern DNA sequencing and statistical techniques that I have used to parse the respective contributions of genes and environment to variation in complex (polygenic) traits, and introduce the developmental and behavioural phenotypes that I study in medaka. Finally, I provide some background to the issue of the non-portability of human polygenic scores to diverse populations, which I explore in the final chapter of this thesis.

\hypertarget{a-brief-history-of-genetics}{%
\section{A brief history of genetics}\label{a-brief-history-of-genetics}}

Much of this section has been informed by Mukherjee (2016), Rutherford (2020), and Tabery (2014). Where possible, I have cited the original sources.

\hypertarget{ancient-greece}{%
\subsection{Ancient Greece}\label{ancient-greece}}

Throughout ancient history, the sources of biological variation were completely unknown. With limited technologies available to them, the Ancient Greek philosophers proposed theories for the inheritance of traits that they observed in humans. Around 500 BC, Pythagoras applied his knowledge of triangles to the question of how traits are inherited from one's parents. He proposed the theory known as ``spermism'', positing that hereditary information was passed down from parent to child via male sperm, with the female providing the nutrients that would allow it to grow. Like the theorem that bears his name, he supposed that these two sides of the ``triangle'' would determine the length of the third side: the characteristics of the child. Over a century later, in 380 BC, Plato extended this metaphor in \emph{The Republic} (Plato 2013) to argue that this principle could be applied to perfect humanity by breeding the ideal combinations of parents at perfect times, which is possibly the first recorded expression of eugenic intent.

Aristotle later joined the discussion with his treatise \emph{Generation of Animals} (Aristotle 2021), where he noted that children inherited features from their mothers as well as their fathers, raising cases where human skin colour and other traits from maternal ancestors could skip generations, and thus sperm could not be the only vessel of hereditary information. He suggested an idea of ``movement'' -- the transmission of information -- from the father's sperm, which sculpts the mother's menstrual blood in the same way a carpenter carves a piece of wood. It was, however, impossible at this time for Aristotle to deduce the form in which the information was conveyed.

\hypertarget{middle-ages}{%
\subsection{Middle Ages}\label{middle-ages}}

From the Middle Ages through to the 1800s, the prevailing theory of heredity was that a tiny human -- a homunculus -- sat within the sperm, waiting to be inflated upon its introduction to a woman's uterus. But from where did each previous homunculus originate? Logically, the theory would require each homunculus to hold within it another homunculus, \emph{ad infinitum}. This theory deceived the inventor of the microscope, Nicolaas Hartsoeker, into thinking he observed a homunculus in a sperm he was studying (\textbf{Figure \ref{fig:homunculus-pic}}) (Hartsoeker 1694). But the theory posed a fundamental question: what triggered the expansion of the human form, causing the embryo to develop new parts on its road to becoming a fetus? The answer could only have been some instruction or code, but any specifics remained out of reach.



\begin{figure}

{\centering \includegraphics[width=0.5\linewidth]{figs/introduction/homunculus} 

}

\caption{Preformation, drawn by Nicolaas Hartsoeker in 1695. Image adapted from Wikimedia Commons (2019).}\label{fig:homunculus-pic}
\end{figure}

\hypertarget{charles-darwin}{%
\subsection{Charles Darwin}\label{charles-darwin}}

During the early 1800s, the prevailing doctrine on the origins of biological variation was Creationism, based on Christianity's literal interpretation of the Bible's Book of Genesis (Campbell 2010). Any mechanistic description of how species -- and individuals within the same species -- differed from one another was thought to threaten this doctrine, making inquiries of this kind potentially blasphemous, and therefore dangerous (Armstrong 2000).

It was in this context that a 22-year-old English clergyman named Charles Darwin (\textbf{Figure \ref{fig:charles-darwin-young-portrait}}) boarded the \emph{HMS Beagle} in 1831 to commence a voyage around the world that would last for almost five years (Darwin 2019). Darwin had previously studied theology at the University of Cambridge, but was drawn to study the natural world. He had apprenticed with his fellow clergyman John Henslow, a botanist and geologist who curated the Cambridge Botanic Garden (Isely 2002), who suggested that he join the \emph{Beagle}'s exploratory survey of South America as the ``gentleman scientist'' they were seeking to assist with the collection of specimens (Henslow 1831).



\begin{figure}

{\centering \includegraphics[width=0.7\linewidth]{figs/introduction/Charles_Darwin_by_G._Richmond} 

}

\caption{Portrait of Charles Darwin from the late 1830s by George Richmond (Wikimedia Commons 2017b).}\label{fig:charles-darwin-young-portrait}
\end{figure}

After rounding Cape Horn and moving northward along the western coast of South America, the \emph{HMS Beagle} eventually reached the Galápagos Islands on the coast of Peru, an archipelago of 18 islands formed from volcanic lava (Darwin 1998). Over the course of five weeks, Darwin collected carcasses of birds, lizards, and plants. Upon his return to England, he was hailed as a minor celebrity among natural historians due to the collections of specimens he had gathered and shipped back. John Gould -- the ornithologist who lent his (wife's) name to the Gouldian finch (\textbf{Figure \ref{fig:gouldian-finch}}) -- told him that the various birds that Darwin thought were a variety of wrens, warblers, blackbirds, and ``gros-beaks'' were all in fact 13 different species of finches.



\begin{figure}

{\centering \includegraphics[width=1\linewidth]{figs/introduction/gouldian_finch} 

}

\caption{The Gouldian Finch, an Australian native bird described by British ornithological artist John Gould in 1844 and named after his deceased wife Elizabeth (Bancroft 2016). Photograph by Sarah R. Pryke, published in Pryke and Griffith (2006).}\label{fig:gouldian-finch}
\end{figure}

Each island had produced its own variant (\textbf{Figure \ref{fig:darwin-finches}}), and this caused Darwin to consider whether they had all arisen from a common ancestral finch, branching off like the boughs of a tree over time. As he wrote in the second edition of his account of his voyage (Darwin 1845):

\begin{quote}
\emph{Seeing this gradation and diversity of structure in one small, intimately related group of birds, one might really fancy that from an original paucity of birds in this archipelago, one species had been taken and modified for different ends.}
\end{quote}

He understood that animal breeders took advantage of the natural variation in populations to select for desired traits, but he questioned what force had guided the development of these different varieties of finches in the wild.



\begin{figure}

{\centering \includegraphics[width=1\linewidth]{figs/introduction/Darwin's_finches_by_Gould} 

}

\caption{Illustration of variation in Galapagos finches by John Gould, published in Darwin (1882). Image from Wikimedia Commons (2022b).}\label{fig:darwin-finches}
\end{figure}

Well prior to Darwin's voyage, Thomas Malthus, a curate and amateur economist, had published a paper titled \emph{An Essay on the Principle of Population} (Malthus 1872), in which he argued that the human population was in constant struggle with its limited resource pool, which in turn was affected by droughts, floods, epidemics, and diseases. Darwin read the paper and identified this struggle for resources as the force that selected those with the traits that favoured their survival. He continued to gestate these ideas over the period from his return from the voyage in 1836 through to 1858, when he read a draft paper that had been sent to him by the author, Alfred Russel Wallace (\textbf{Figure \ref{fig:alfred-wallace}}) (Marchant and Wallace 1916).

Like Darwin, Wallace had set off on a voyage to distant lands, and had observed the stunning variation across the Malay Archipelago in populations separated by channels of water (C. H. Smith and Beccaloni 2010). And like Darwin, he also derived his theory on the basis of this variation from Malthus's paper. The draft paper he sent to Darwin outlined his theory of evolution and natural selection (Wallace 1855). In a panic about being ``scooped'', Darwin sent both papers to his geologist friend Charles Lyell, who advised Darwin to have both papers presented simultaneously at the meeting of the Linnean Society so that they could both be credited for the discovery (Darwin et al. 2002).



\begin{figure}

{\centering \includegraphics[width=0.5\linewidth]{figs/introduction/Alfred-Russel-Wallace} 

}

\caption{Alfred Russel Wallace, taken around 1895. Image from Wikimedia Commons (2022a).}\label{fig:alfred-wallace}
\end{figure}

The presentation made few waves at the time, but Darwin proceeded to complete his opus, \emph{On the Origin of Species by Means of Natural Selection} (Darwin 1859), where he provided the famous quote:

\begin{quote}
\emph{Thus, from the war of nature, from famine and death, the most exalted object which we are capable of conceiving, namely, the production of the higher animals, directly follows. There is grandeur in this view of life, with its several powers, having been originally breathed into a few forms or into one; and that, whilst this planet has gone cycling on according to the fixed law of gravity, from so simple a beginning endless forms most beautiful and most wonderful have been, and are being, evolved.}
\end{quote}

The book was unexpectedly met with enthusiastic reviews (\emph{The {Saturday Review} of {Politics}, {Literature}, {Science} and {Art}} 1859), and sold well (Darwin 1887). Darwin's theory of evolution through natural selection has since stood the test of time, and is now established as the core process by which life on Earth acquires its grand variety of forms. At the time of publication of the \emph{Origin of Species}, however, two crucial questions still remained unanswered: how was the variation within species generated in the first place, and how were the traits transmitted to future generations?

\hypertarget{gregor-mendel}{%
\subsection{Gregor Mendel}\label{gregor-mendel}}

Around the time the \emph{Origin of Species} was published, the prevailing theory of heredity was promoted by the French biologist Jean-Baptiste de Lamarck (\textbf{Figure \ref{fig:lamarck}}). Lamarck's theory involved the transmission of traits that were strengthened or weakened by environmental pressures in one generation -- such as a giraffe having to stretch its neck to reach the leaves on a canopy -- which was then passed down to subsequent generations as a form of ``pre-adaptation'' (Strickberger 1990).



\begin{figure}

{\centering \includegraphics[width=0.5\linewidth]{figs/introduction/Jean-Baptiste_de_Lamarck} 

}

\caption{Portrait of Jean Baptiste de Lamarck from 1802-1803 by Charles Thévenin (Wikimedia Commons 2022c).}\label{fig:lamarck}
\end{figure}

But by extending that logic, if a human had their arm amputated, their children would be born with shortened arms.\footnote{This theory was experimentally tested by the German embryologist August Weismann, who amputated the tails of mice to determine whether the offspring would be born tailless (Schwartz 2008). The offspring were all born with tails intact.} Darwin was convinced that selection rather acted upon \emph{pre-existing} variation in a population, and proposed that that variation was transmitted by hereditary particles he called \emph{gemmules} (Schwartz 2008). During conception, he proposed that the gemmules of each parent were blended like paints (B. Charlesworth and Charlesworth 2009). But such blending would result in a monochrome population, preventing the creation of the unique, outlying traits required to drive the variation he observed (Jenkin 1867). Around this time Darwin had recorded notes on an obscure paper titled \emph{Experiments in Plant Hybridization} (Mendel 1866), but he appeared to have inadvertently skipped the page containing a description of the experiments on pea hybrids performed by an Augustine monk named Gregor Mendel (\textbf{Figure \ref{fig:mendel}}).



\begin{figure}

{\centering \includegraphics[width=0.6\linewidth]{figs/introduction/Gregor_Mendel_2} 

}

\caption{Photograph from Wikimedia Commons (2021b).}\label{fig:mendel}
\end{figure}

Mendel had aspired to become a teacher, but had failed the teacher's exam multiple times, so by 1853 he had returned to the monastery in Brno, Morvia (present day Czech Republic), and planted a crop of peas which he had been breeding for about three years (Henig 2000). He had collected 34 strains that bred ``true'', meaning that the offspring were identical to the parents in seven famous traits (Reeve 2014):

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\item
  the texture of the seed (smooth versus wrinkled)
\item
  the colour of seeds (yellow versus green)
\item
  the colour of the flower (white versus violet)
\item
  the position of the flower (at the tip of the plant versus the branches)
\item
  the colour of the pea pod (green versus yellow)
\item
  the shape of the pea pod (smooth versus crumpled)
\item
  the height of the plant (tall versus short)
\end{enumerate}

Mendel referred to these alternative versions of a trait as \emph{forms} -- biologists in the 1900s would later refer to them as \emph{alleles}, from the Greek \emph{allos}, referring to different subtypes of the same thing (Mukherjee 2016). Mendel produced hybrids of plants with different alleles to determine which allele the consequent offspring would possess. Over the course of eight years (1857-1864), he crossed the true-breeding strains (termed ``filial 0'', or \textbf{F\textsubscript{0}}) to create the hybrid \textbf{F\textsubscript{1}} generation, and crossed those with each other too (the \textbf{F\textsubscript{2}} generation), recording their values for the above traits as he went. He could soon identify patterns in the reams of data he generated:\footnote{Based around 28,000 plants, 40,000 flowers, and almost 400,000 seeds.} for each of the above 7 traits, the F\textsubscript{1} generation expressed only a single allele, which he referred to as the \emph{dominant}; the hidden allele he referred to as \emph{recessive} (Henig 2000). Curiously, however, in the F\textsubscript{2} generation the recessive allele reappeared at a ratio of 1:3 (Henig 2000).

From these results, Mendel inferred that the F\textsubscript{1} hybrids must retain the information from both parents, while only expressing one version of them. Then when the F\textsubscript{1} generation was inter-crossed to produce the F\textsubscript{2} generation, the recessive allele would only be expressed when inherited with another recessive allele (Mukherjee 2016). In 1865, Mendel presented his paper \emph{Versuche über Pflanzenhybriden} (Experiments on Plant Hybridization) to a group of farmers, botanists and biologists at the Natural Science Society in Brno, and it was later published in the annual journal \emph{Proceedings of the Brno Natural Science Society} (Mendel 1866). He mailed out 40 copies of the paper to various scientific societies, but it was only cited four times between 1866 and 1900, and virtually disappeared from scientific literature (Dunn 1991).

In 1900, a Dutch botanist named Hugo de Vries chanced upon Mendel's paper while working on plant hybrids (Sandler and Sandler 1985). Likely driven by self-interest in being credited for the discovery, he proceeded to publish his own findings without mentioning the work of Mendel (Larson, History, and Law Edward J. Larson PH.D J. D. 2004). He noted a patch of primroses named \emph{Oenothera lamarckiana}\footnote{An ironic twist given the plant was named after Lamarck.}, which spontaneously generated variants (Schwartz 2008), and he termed these variants \emph{mutants} (de Vries 1909). It became clear to de Vries that variation in populations arose spontaneously, at random, rather than in the Lamarckian mode of contemporaneous reaction to environmental pressures. De Vries's paper was read by Carl Correns, a botanist in Tübingen, Germany, who had also come across the work of Mendel, and pressured de Vries into acknowledging Mendel's work in the subsequent version of his publication (Rheinberger 2000).

In 1894, the English biologist William Bateson (\textbf{Figure \ref{fig:bateson}}) had published a book titled \emph{Materials for the study of variation} (Bateson 1894), in which he noted that biological variation occurs continuously for some traits, and dimorphically for others. In 1900, while on a train from Cambridge to London, Bateson read the second version of de Vries's paper, and was immediately struck by the significance of Mendel's original study (Malone 2002). He independently confirmed Mendel's findings (Punnett 1905), and began to promote Mendel's work (Cock and Forsdyke 2008), including by publishing an English translation of Mendel's original paper (Druery and Bateson 1901). In 1905, Bateson coined a word for the study of these units of inheritance: \emph{genetics}, from the Greek \emph{genno}, meaning ``to give birth'' (Schwartz 2008). A few years later, in 1909, the botanist Wilhelm Johannsen coined a distinct word to denote the unit of inheritance: the \emph{gene} (Wanscher 1975).



\begin{figure}

{\centering \includegraphics[width=1\linewidth]{figs/introduction/Bateson2} 

}

\caption{Portrait of William Bateson, date and author unknown (Wikimedia Commons 2017a).}\label{fig:bateson}
\end{figure}

Mendel's work has been distilled into what are now called \emph{Mendel's Laws}:\footnote{Marks (2008) offers a fascinating description of how ``Mendel's Laws'' developed through a series of pedagogical processes, achieving something close to its final form in Thomas Hunt Morgan's \emph{A Critique of the Theory of Evolution} (Morgan 1919).}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\item
  \textbf{Segregation}: One allele of each parent is randomly and independently selected, with a probability of 0.5, for transmission to the offspring, and the single alleles from each parent unite randomly to form the offspring's genotype. To illustrate the principle, after having crossed two F\textsubscript{0} inbred strains that are homozygous for different alleles (denoted as \emph{AA} and \emph{aa}), thus generating the F\textsubscript{1} hybrid with the genotype \emph{Aa}, the resultant F\textsubscript{2} individuals from the F\textsubscript{1} intercross will have a genotypic ratio of 1:2:1 respectively for the genotypes \emph{AA} (homozygous for the first F\textsubscript{0} parent's allele), \emph{Aa} (heterozygous), and \emph{aa} (homozygous for the second F\textsubscript{0} parent's allele). If the alleles are dominant/recessive, as was the case for Mendel's pea plant phenotypes, these genotypes would correspond to the 3:1 phenotypic ratio that he observed (Laird and Lange 2010).
\item
  \textbf{Independent assortment}: The alleles of separate traits are passed independently of one another, based on Mendel's dihybrid crosses which showed a 9:3:3:1 phenotypic ratio, and a 3:1 phenotypic ratio for each gene. However, it was later discovered that this ``law'' does not apply to genes that are linked to one another by physical proximity on the chromosome, which I expand upon below in section \ref{mapping-sec}.
\end{enumerate}

\hypertarget{quantitative-genetics-francis-galton-karl-pearson-ronald-fisher-and-sewall-wright}{%
\subsection{Quantitative genetics: Francis Galton, Karl Pearson, Ronald Fisher and Sewall Wright}\label{quantitative-genetics-francis-galton-karl-pearson-ronald-fisher-and-sewall-wright}}

Francis Galton (\textbf{Figure \ref{fig:galton}}) was the cousin of Charles Darwin, born in the same year as Mendel (1822), and travelled to Egypt and Sudan in 1844, spurring his lifelong obsession with the differences between human races (Simonton 1999). Driven by a fear the the ``unfit'' were outbreeding the ``fit'' (Tabery 2014), Galton's reading of Darwin's \emph{Origin of Species} in 1859 galvanised him to apply this new science to improve the human population through selective breeding (James D. Watson and Berry 2009). With a view to improving the human stock, he began to explore the measurement and variance of heredity in humans, with an emphasis on height, intelligence, temperament, and physical prowess (Fairbanks 2009).\footnote{One of Galton's more eccentric activities involved strolling through England and Scotland secretly tabulating beauty by ranking the women he met as ``attractive'', ``indifferent,'' or ``repellent'' using pinpricks on a card hidden in his pocket (Mukherjee 2016).} From Galton (1883):

\begin{quote}
\emph{We greatly want a brief word to express the science of improving stock\ldots{} to give to the more suitable races or strains of blood a better chance of prevailing speedily over the less suitable\ldots{}}
\end{quote}

For this purpose, he coined the term \emph{eugenics} (James D. Watson and Berry 2009). It was his vision for a ``science which deals with all influences that improve the inborn qualities of race'' (Galton 1904):

\begin{quote}
\emph{What nature does blindly, slowly and ruthlessly, man may do providently, quickly, and kindly. As it lies within his power, so it becomes his duty to work in that direction.}
\end{quote}



\begin{figure}

{\centering \includegraphics[width=0.7\linewidth]{figs/introduction/Francis_Galton_1850s} 

}

\caption{Portrait of Francis Galton, taken in the 1850s or early 1860s, originally scanned from Pearson (2011; Wikimedia Commons 2021a).}\label{fig:galton}
\end{figure}

Through surveys Galton sent out to men and women in the mid-1880s, he requested they mail him detailed measurements on the height, weight, eye colour, intelligence, and artistic abilities of their relatives, in return for a substantial fee. With this data he discovered that tall parents indeed tend to have tall children, albeit on average, and that the distribution of heights within a generation fit the shape of a normal distribution. In a surprising twist, he also discovered that the mean height of the sons of the tallest fathers tended to be slightly lower than the father's height, and closer to the population's average -- a phenomenon he described as \emph{regression to the mean} (Galton 1886).

Having earlier coined the phrase \emph{nature versus nurture} to distinguish between hereditary and environmental influences (Galton 1874), in the 1890s Galton sought to tease out their respective contributions to the traits he deemed important by proposing the first study on human twins. He reasoned that given twins share identical genetic material, they represent a natural experiment where their similarities could be attributed to genes, while any differences were caused by differences in their environments (Tabery 2014).\footnote{These studies, however, were hampered by Galton's failure to distinguish between identical and non-identical twins.}

Galton also proposed an \emph{ancestral law of heredity} (Galton 1889) -- whereby parents each contributed half of the content a feature to their offspring, the grandparents a quarter, and so on -- sparking a vociferous debate between Bateson and a cohort of other scientists who sought to reconcile Mendelian inheritance with the continuous variation observed in natural populations (Barton 2016). The debate would not be resolved for some time.

In Galton's later years, he adopted the English mathematician Karl Pearson (\textbf{Figure \ref{fig:pearson}}) as his protégé. Like his mentor, Pearson applied his formidable talents to advancing social Darwinism (Karl 1901):

\begin{quote}
\emph{My view -- and I think it may be called the scientific view of a nation, is that of an organized whole, kept up to a high pitch of internal efficiency by insuring that its numbers are substantially recruited from the better stocks, and kept up to a high pitch of external efficiency by contest, chiefly by way of war with inferior races.}
\end{quote}

From 1893 to 1904, Pearson built upon the work of Galton by developing a number of statistical techniques for biometry including the chi-squared test (Pearson 1900), standard deviation (Pearson 1894), correlation and regression coefficients (Pearson 1898), and the foundations of principal components analysis (Pearson 1901). When Galton died in 1911, he bequeathed money to University College London for a Galton Eugenics Professorship, a position that was given to Pearson, who was also the head of the newly established Department of Applied Statistics, the world's first university statistics department (Kevles 1995; Harden 2021).

To set out some equations relevant to this thesis, Pearson's corrected standard deviation \(s\) for a sample is defined as:

\begin{equation}
s = \sqrt{\frac{1}{N-1}{\sum_{i=1} ^n (x_i - \bar{x})^2}} \label{eq:sd}
\end{equation}

And the Pearson correlation coefficient for a sample is defined as:

\begin{equation}
r_{X,Y} = \frac{\mathrm{cov}(X,Y)}{s_X s_Y} \label{eq:cor}
\end{equation}
\begin{equation}
\mathrm{cov} = \sum_{i=1} ^n (x_i - \bar{x})(y_i - \bar{y})
\end{equation}



\begin{figure}

{\centering \includegraphics[width=0.7\linewidth]{figs/introduction/Karl_Pearson_1910} 

}

\caption{Portrait of Karl Pearson in 1910 (Wikimedia Commons 2020).}\label{fig:pearson}
\end{figure}

Following in the footsteps of Galton and Pearson, the mathematician Ronald Fisher (\textbf{Figure \ref{fig:fisher}}) of Caius College at the University of Cambridge began to apply his skills to elucidating how continuous traits, like height, could be driven by genetic variation (Reeve and Black 2001). In 1918, Fisher published his analysis in a landmark paper titled \emph{The Correlation between Relatives on the Supposition of Mendelian Inheritance} (R. A. Fisher 1918), where he described how the combination of a large number of Mendelian alleles acting on the same trait would result in a normal distribution. This reconciliation between Mendelian inheritance and observed continuous traits was the beginning of what was later referred to by Julian Huxley as the ``\textbf{modern evolutionary synthesis}'' (Huxley 1942; Tabery 2014).



\begin{figure}

{\centering \includegraphics[width=0.6\linewidth]{figs/introduction/Youngronaldfisher2} 

}

\caption{Photo of a young Ronald Fisher taken in 1913. Image from Wikimedia Commons (2021c).}\label{fig:fisher}
\end{figure}

In the same paper, Fisher also introduced the concept of \emph{variance} as the square of the standard deviation developed by Galton and Pearson (Equation \eqref{eq:sd} above), and used it to partition the \emph{causes} of variation (Tabery 2014).\footnote{R. A. Fisher (1918): ``For stature the coefficient of correlation between brothers is about .54, which we may interpret by saying that 54\% of their variance is accounted for by ancestry alone, and that 46\% must have some other explanation.''} In 1921 he published his first application of this ``analysis of variance'' (\textbf{ANOVA}) (Ronald Aylmer Fisher 1921), which became widely used following its inclusion in his widely influential book \emph{Statistical Methods for Research Workers} (Ronald Aylmer Fisher 1925).

Fisher was also an ardent eugenicist, and his statistical innovations were developed in part to assess the relative importance of nature and nurture for traits like feeble-mindedness (Tabery 2014). He helped create the Cambridge University Eugenics Society in 1911, hosting meetings in his rooms, organising public lectures by well-known eugenicists, assisting at the First International Eugenics Congress, and delivering his own eugenic lectures. Fisher also helped establish the Society's Committee for Legalising Sterilisation (Mazumdar 2005; Tabery 2014).

Through the sustained advocacy of scientists led by Galton, Pearson, and Fisher, the concept of eugenics took a firm hold on Western thought (Hawkins 1997), leading to the implementation of eugenics policies in a number of American and European countries including the United States (Barrett and Kurzman 2004), Canada (A. McLaren 2015), Sweden (Björkman and Widmalm 2010), and Nazi Germany (Bartov 2015). From the early 1900s through to the end of World War II, these policies resulted in the forced sterilisation and murder of millions (Bashford and Levine 2010). The eugenics movement was inextricably linked to these horrors, and subsequently fell into decline (Black 2012).

Prior to the war, around the time Fisher was working on his 1919 paper, the American geneticist Sewall Wright (\textbf{Figure \ref{fig:wright}}) was using his work on guinea pigs to reconcile Mendelian inheritance with continuous variation (S. Wright 1930). He developed a mathematical theory of inbreeding, eventually introducing the inbreeding coefficient \(F\) as the correlation between uniting gametes in 1922 (S. Wright 1922). Over the course of an esteemed career, together with Fisher and John B. S. Haldane, he advanced the modern evolutionary synthesis (Provine 2001; Tabery 2014), and founded the field of population genetics with his papers on mating systems (S. Wright 1946) and genetic drift (S. Wright 1948).



\begin{figure}

{\centering \includegraphics[width=0.8\linewidth]{figs/introduction/Sewall_Wright} 

}

\caption{Photo of Sewall Wright in 1965. Image from Goodnight (2014).}\label{fig:wright}
\end{figure}

\hypertarget{mapping-sec}{%
\subsection{Genetic mapping: Thomas Morgan, Tetsuo Aida and Theodosius Dobzhansky}\label{mapping-sec}}

During this fertile period in statistical genetics research, the physical location of the gene was still a mystery. In the 1890s, Theodor Boveri, a German embryologist working with sea urchins in Naples, identified the chromosome as the location of the gene (Laubichler and Davidson 2008). Inspired by Boveri's work, the American cell biologist Thomas Hunt Morgan (\textbf{Figure \ref{fig:morgan}}) sought to extend it by exploring the architecture of genes using the fruit fly, \emph{Drosophila melanogaster}, as a model organism (Allen 1978).



\begin{figure}

{\centering \includegraphics[width=0.8\linewidth]{figs/introduction/Thomas_Morgan} 

}

\caption{Thomas Hunt Morgan in 1920. Portrait taken by A. F. Huettner. Image from Kenney and Borisy (2009).}\label{fig:morgan}
\end{figure}

Around 1905, Morgan began to breed \emph{Drosophila}, identifying visible variants that he could track over generations, such as white versus red eyes, forked versus straight bristles, disjointed abdomens, and deformed eyes (Hodge 2009). Through repeated crossing experiments, Morgan discovered that some genes were transmitted together at a higher rate than chance alone. He proposed that these genes were physically ``linked'' to one another, implying that they were situated within the chromosome like strings on a bead (Morgan 1972). This advanced the understanding of a gene from a purely theoretical unit of inheritance to a physical unit (Morgan 1935). He further discovered that, occasionally, a gene that was otherwise linked to another could ``cross over'' from one parental strand to the other, generating offspring with a mixture of parental alleles (Morgan 1916).

While Morgan was carrying out his experiments with \emph{Drosophila}, Japanese researchers were also studying Mendelian inheritance with the Japanese medaka fish (\emph{Oryzias latipes}), the main subject of this thesis. Following three articles published by three separate professors in Japanese on the recessive inheritance of medaka colour variants (Fukamachi and Naruse 2021), a teacher named Tatsuo Aida (\textbf{Figure \ref{fig:aida}}) published the first article in English about medaka in 1921 (Aida 1921) (\textbf{Figure \ref{fig:aida-fig}}). The study involved a total of 22 crosses of two generations, and among several other findings, Aida discovered recombination between the X and Y chromosome -- the world's first demonstration of crossovers between pseudo-autosomal sex chromosomes in any species (Naruse, Tanaka, and Takeda 2011).\footnote{The editor, William Castle, was so impressed by this last finding that he added the following footnote to the paper: ``It should be pointed out, in justice to the author, that his observations go beyond those of Schmidt in the important respect of showing the occurrence of crossing over between the X and the Y chromosome.''}



\begin{figure}

{\centering \includegraphics[width=0.8\linewidth]{figs/introduction/Aida} 

}

\caption{Photo of Tetsuo Aida from (Fukamachi and Naruse 2021).}\label{fig:aida}
\end{figure}



\begin{figure}

{\centering \includegraphics[width=1\linewidth]{figs/introduction/Aida_1921_fig} 

}

\caption{Figure from Aida (1921) showing the different medaka colour variants used in the study.}\label{fig:aida-fig}
\end{figure}

In the 1930s, Theodosius Dobzhansky (\textbf{Figure \ref{fig:dobzhansky}}), a Ukrainian biologist who had trained with Morgan, used \emph{Drosophila pseudoobscura} to investigate how genetic variation drove evolution (Mukherjee 2016). Using a single population of flies that he raised in different temperatures while controlling for all other environmental variables, he found that after four months, the genetic ratios in the two sub-populations had changed (Dobzhansky 1943). Through this experiment, he determined that genetic variation was the norm across biology; that the adaptive benefit of a given variant would depend on the particular environment that an individual found itself in; and that most phenotypes were driven by many genes interacting with each other and the environment.



\begin{figure}

{\centering \includegraphics[width=0.5\linewidth]{figs/introduction/Dobzhansky} 

}

\caption{Undated photo of Theodosius Dobzhansky from Coyne (2016).}\label{fig:dobzhansky}
\end{figure}

\hypertarget{the-discovery-of-the-structure-of-dna}{%
\subsection{The discovery of the structure of DNA}\label{the-discovery-of-the-structure-of-dna}}

By the early 1940s, it was known that genes resided in chromatin, the mixture of proteins and nucleic acids that compose chromosomes. After some flirtation with proteins as the molecule of inheritance, Avery, MacLeod, and McCarty (1944) finally proved that it was in fact deoxyribonucleic acid (\textbf{DNA}) that was ``the material substance of the gene'' -- the ``cloth from which genes were cut'' (Gorner and Lyon 1996).

From 1951, the British biochemist Rosalind Franklin had been working at King's College in London, using X-ray crystallography to photograph DNA (Maddox 2003). In 1952, her student, Raymond Gosling, took the now famous ``Photo 51'' (\textbf{Figure '\ref{fig:photo51}}), and upon Franklin's decision to depart from King's College, she suggested he show the photo to Maurice Wilkins, a New Zealand-born biophysicist who was also working at King's College on the structure of DNA (Williams 2019).



\begin{figure}

{\centering \includegraphics[width=0.5\linewidth]{figs/introduction/Photo_51} 

}

\caption{Photo 51 from Franklin and Gosling (1953).}\label{fig:photo51}
\end{figure}

Without Franklin's permission, Wilkins brought photo 51 to James Watson, a molecular biologist working at the University of Cambridge, who immediately saw that ``the black cross could arise only from a helical structure'' (Marx 2004). Watson and his colleague Francis Crick, a molecular biologist from England, used this observation -- and a report by Franklin and Wilkins on their most recent measurements of the outer backbone -- to develop the established model of DNA structure: a double helix of a sugar-phosphate backbone and the nucleotide bases facing inwards and paired with one another in the conformation of A\(\rightarrow\)T and G\(\rightarrow\)C (James D. Watson, Gann, and Witkowski 2012).

Watson and Crick built their complete model in the first week of March 1953, and, following consultations with Wilkins and Franklin, who both approved of the model, published their famous paper on 25 April titled \emph{Molecular structure of nucleic acids: a structure for deoxyribose nucleic acid} (J. D. Watson and Crick 1953). Theirs was accompanied by two others: one by Gosling and Franklin with crystallographic evidence for the double-helical structure including photo 51 (Franklin and Gosling 1953); and another by Wilkins corroborating the evidence further with experimental data from DNA crystals (Wilkins, Stokes, and Wilson 1953). For their discovery, Watson, Crick, and Wilkins were awarded the Nobel Prize in 1962. Franklin had died two years earlier from ovarian cancer, so was not nominated.

It is worth noting that both Watson and Crick have since made a number of controversial comments about eugenics, race and intelligence. In 1970, while reading Karl Pearson's biography of Francis Galton in 1970, Crick wrote to Bernard Davis of Harvard University and said:

\begin{quote}
\emph{My other suggestion is in an attempt to solve the problem of irresponsible people and especially those who are poorly endowed genetically having large numbers of unnecessary children. Because of their irresponsibility, it seems to me that for them, sterilization is the only answer and I would do this by bribery. It would probably pay society to offer such individuals something like £1,000 down and a pension of £5 a week over the age of 60. As you probably know, the bribe in India is a transistor radio and apparently there are plenty of takers.} (Ridley 2006)
\end{quote}

In an interview with David Pearson of the Wellcome Library for the History and Understanding of Medicine, Crick also said:

\begin{quote}
\emph{In the long run, it is unavoidable that society will begin to worry about the character of the next generation \ldots{} It is not a subject at the moment which we can tackle easily because people have so many religious beliefs and until we have a more uniform view of ourselves I think it would be risky to try and do anything in the way of eugenics \ldots{} I would be astonished if, in the next 100 or 200 years, society did not come round to the view that they would have to try to improve the next generation in some extent or one way or another.} (Wellcome 2003)
\end{quote}

And in 2015, Watson said:

\begin{quote}
\emph{All our social policies are based on the fact that their (blacks) intelligence is the same as ours (whites) -- whereas all the testing says not really \ldots{} people who have to deal with black employees find this not true.} (Wagenseil 2015)
\end{quote}

During a speaking tour, Watson again claimed that black people were less intelligent than white people, and the idea that ``equal powers of reason'' were shared across racial groups was a delusion (Milmo 2007). Long after the tragedies of World War II, the spectre of eugenics remains present among the genetics community, complicating the legacies of Watson and Crick, who were responsible for one of the 20th century's most remarkable discoveries.

\hypertarget{the-development-of-dna-sequencing}{%
\subsection{The development of DNA sequencing}\label{the-development-of-dna-sequencing}}

In the early 1950s, Frederick Sanger (\textbf{Figure \ref{fig:sanger}}), a biochemist at the University of Cambridge, developed methods of ``sequencing by separation'', first used to determine the first protein sequence of insulin (F. Sanger and Tuppy 1951a, 1951b) -- winning him his first Nobel Prize in Chemistry in 1958 ({``The {Nobel Prize} in {Chemistry} 1958''} n.d.) -- and later to decipher RNA sequences, with the first being alanine tRNA (Shendure et al. 2017). Following these achievements, Sanger turned to sequencing DNA, flipping his approach to one of ``sequencing by synthesis''. In 1975 he and Alan Coulson published a preliminary method using radio-labelled nucleic acids to sequence short DNA strands (F. Sanger and Coulson 1975). The major breakthrough, however, occurred in 1977, when he and his colleagues developed what is now known as the ``Sanger method'' (Frederick Sanger, Nicklen, and Coulson 1977), involving fluorescently-labelled ``dideoxy'' nucleosides that terminate the chain after being added to the strand. This resulted in strands of different lengths that could be visualised on a four-lane polyacrylamide slab gel, with the location of each fragment indicating the identity of the base at that position.



\begin{figure}

{\centering \includegraphics[width=0.5\linewidth]{figs/introduction/sanger} 

}

\caption{Undated photo of Frederick Sanger from {``Obituary: {Fred Sanger} the Scientist and Twice {Nobel Prize} Winner Dies''} (2013).}\label{fig:sanger}
\end{figure}

The Sanger method was highly accurate and efficient, and further refinements of this approach, together with advances in DNA cloning (O'Connor, Peifer, and Bender 1989), led to the development of a method known as ``shotgun sequencing''. The method involves randomly breaking up genomes into short (100-1000 base pair (\textbf{bp})) strands, sequencing them, and then assembling them based on overlapping regions (Staden 1979). This made it possible, for the first time, to sequence whole, large-scale genomes (Shendure et al. 2017). The Human Genome Project was established in 1990 as a large, international consortium, and through the application of shotgun sequencing, published the first draft of the human genome in 2001 (I. H. G. S. Consortium 2001; Venter et al. 2001), with an ``essentially complete'' draft announced in 2003 (I. H. G. S. Consortium 2003).\footnote{The full sequence (including heterochromatic regions) was only completed in 2022 (Nurk et al. 2022).}

Shortly after this milestone, the Sanger method was superseded by ``second generation'' DNA sequencing, also called ``massively parallel'' or ``next generation'' sequencing (\textbf{NGS}). The first NGS platforms arrived in 2005, involving the dense fixation of a library of millions or billions of DNA templates onto a two-dimensional surface. This allowed for \emph{in vitro} amplification and imaging through the detection of fluorescence from dideoxynucleotides over repeated cycles of biochemistry (Shendure et al. 2017). In this method, each cycle incorporates a single nucleotide, and after imaging to determine which of four colours was incorporated by each template on the surface, the blocking and fluorescent groups are removed to set up the next extension (Shendure et al. 2017). Read lengths from NGS tend to be shorter than the Sanger method (in the low hundreds of bases, compared to 600-700 for Sanger), but with a similar accuracy of over 99.9\% (Shendure et al. 2017). The intense competition between several NGS companies including 454 (acquired by Roche), Solexa (acquired by Illumina) and Complete Genomics resulted in the reduction in the cost of DNA sequencing per-megabase from \$1,000 to \$0.10 between 2007 and 2012 (Wetterstrand 2021).

Sanger sequencing and NGS both come with drawbacks. They require template amplification, which can create copying errors, sequencing-dependent biases, and information loss (for example, DNA modifications) (Shendure et al. 2017). In addition, the short read length can make genome assembly difficult for regions with high repeat content or structural variation (Amarasinghe et al. 2020). New technologies pioneered by PacBio and Oxford Nanopore Technologies (\textbf{ONT}) overcome both issues by sequencing single molecules in real time (Mikheyev and Tin 2014; Rhoads and Au 2015), which removes the amplification step and generates reads from kilobases (PacBio) up to megabases in length (ONT) (Payne et al. 2019). These ``third-generation'' or ``long-read'' sequencing technologies formerly had a much higher error rate than NGS platforms (Laver et al. 2015), but are now quickly closing the gap (Jain et al. 2018), and recent ONT platforms can also directly sequence DNA modifications (Yunhao Wang et al. 2021). The long reads produced by these technologies have two primary benefits. First, they allow for a more straightforward \emph{de novo} assembly of genomes, making it more feasible to construct pan-genomes in a graph-based format that represent a wider spectrum of diversity in a species, rather than calling only short variants based on a single reference genome (Sherman and Salzberg 2020). Second, they can capture complex, large-scale structural variation that has remained hidden until now, and which can have significant effects on phenotypes including autism and schizophrenia in humans (Weischenfeldt et al. 2013). In Chapter \ref{MIKK-genomes-chap} I use the long-read ONT platform to detect structural variants in 9 medaka fish genomes, and compare the traditional reference-based approach to the graph-based approach in respect of their ability to detect those variants.

Altogether, the rapid improvement in sequencing technologies has now reached the stage where a \$100 human genome is on the horizon (Pennisi 2022). These developments are making genetic technologies ever more accessible and informative, paving the way for greater insights into the genetic basis of phenotypic differences. Yet despite these technological advances, the relative contribution of genetics and environment to differences in complex traits is not fully understood -- especially in humans -- and therefore remains an active area of research.

\hypertarget{causes-of-variation-in-complex-traits}{%
\section{Causes of variation in complex traits}\label{causes-of-variation-in-complex-traits}}

Individual differences in human traits have now been studied for more than a century, yet the causes of variation in human traits remain uncertain and complex (Polderman et al. 2015). Specifically, the partitioning of observed variability into underlying genetic and environmental sources, and the relative importance of additive and non-additive genetic variation, continue to be debated to this day (Polderman et al. 2015).

The most simple causal model for phenotypes is shown in \textbf{Figure \ref{fig:pge-simp}}.



\begin{figure}

{\centering \includegraphics[width=0.5\linewidth]{figs/introduction/pge-simp} 

}

\caption{Simplified causal model of phenotypic differences.}\label{fig:pge-simp}
\end{figure}

This model could be described mathematically as follows, where \(P\) represents an individual's phenotype, \(G\) represents their genetics, and \(E\) represents the environment, being all other (non-genetic) factors that could have influenced their phenotype up to the time of measurement:

\begin{equation}
P = G + E \label{eq:pge-simp}
\end{equation}

It is impossible to parse out the respective contributions of each of these factors at the individual level, because to what extent does the width or the length of a rectangle determine its area? It is determined by the product of the two. But by examining a population of rectangles, one can determine how much of the \emph{variance} within the population depends on either width or length (Knopik et al. 2019). The same principle applies to genotypic and environmental variation within groups of individuals. Fisher's ANOVA methodology allows for the partitioning of variance into components that are attributable to different factors. If the total observed phenotypic variation within a population is represented by \(V_{P}\), then it can be decomposed into a simple model that includes genetic and environmental factors (Falconer and Mackay 1996):

\begin{equation}
V_{P} = V_{G} + V_{E} \label{eq:pge}
\end{equation}

\(V_{G}\) is the total proportion of phenotypic variance attributable to genetic factors, and \(V_{E}\) is that attributable to environmental factors (i.e.~all non-genetic factors). Environmental variance cannot be removed experimentally because it includes by definition all non-genetic variance, and much of this is beyond experimental control. However, elimination of genotypic variance can be achieved experimentally (Falconer and Mackay 1996) by either using human or animal identical twins, or by creating highly inbred lines. Pursuant to Equation \eqref{eq:pge}, the environmental variance (\(V_E\)) can be subtracted from the total phenotypic variance (\(V_P\)) to give an estimate of the trait's heritability \(V_G\) (Falconer and Mackay 1996). \emph{Broad-sense heritability} (\(H^2\)) measures the proportion of phenotypic variance that is attributable to genetic factors (Falconer and Mackay 1996):

\begin{equation}
H^2 = \frac{V_G}{V_P} \label{eq:heritbs}
\end{equation}

\(V_{G}\) can be further decomposed into additive, dominance, and interaction (``epistatic'') variances. Additive variance, \(V_A\), also known as the variance of ``breeding values'', measures the degree to which offspring resemble their parents due to the alleles they have inherited from them. Dominance variance, \(V_D\), measures the dominance deviation, being the extent to which the phenotype deviates from the additive linear relationship across genotypes as a result of interactions between alleles of the same gene. Interaction variance, \(V_I\), measures the degree to which the phenotype deviates from the additive linear relationship across genotypes as a result of interactions between genes at different loci (Falconer and Mackay 1996).

\begin{equation}
V_G = V_A + V_D + V_I \label{eq:gendecomp}
\end{equation}

Research has traditionally focused on \(V_A\) due to its importance for predicting phenotypes between relatives, and opportunities for change through natural or artificial selection (William G. Hill, Goddard, and Visscher 2008). Recent evidence has supported the proposition that most genetic variance is additive (William G. Hill, Goddard, and Visscher 2008; Polderman et al. 2015), although I expand on this point below. \emph{Narrow-sense heritability} (\(h^2\)) measures the proportion of phenotypic variance attributable to additive genetic variance, and therefore expresses the extent to which phenotypes are determined by genes transmitted from one's parents (Bateson 1909; Falconer and Mackay 1996; Posthuma et al. 2003).

\begin{equation}
h^2 = \frac{V_A}{V_P} \label{eq:heritns}
\end{equation}

It is important to note, however, that as heritability (both in its narrow and broad sense) is measured as a fraction of total phenotypic variance, it is liable to change depending on the level of environmental variation present in the study population (Visscher, Hill, and Wray 2008).

\hypertarget{heritability-and-twin-studies}{%
\subsection{Heritability and twin studies}\label{heritability-and-twin-studies}}

In humans, the twin study design has been used widely to disentangle the relative contributions of genes and environment for a variety of traits. The classical twin design is based on contrasting the trait resemblance of monozygotic (\textbf{MZ}) and dizygotic (\textbf{DZ}) twin pairs (Polderman et al. 2015). MZ twins share approximately 100\% of genotypes, whereas DZ twins, like siblings, share on average 50\% of their genotypes (Knopik et al. 2019). Assuming they share the same environment (a point I return to below), a rough estimate of heritability can be obtained by comparing the respective correlations of MZ twins and DZ twins with the phenotype of interest. If \(r_{MZ}\) and \(r_{DZ}\) are the correlation coefficients observed between pairs of MZ and DZ twins for a trait of interest (Equation \eqref{eq:cor} above), \(h^2\) = additive genetic influences (Equation \eqref{eq:heritns} above), and \(c^2\) = common environmental influences, then Falconer's formula tells us:

\begin{equation}
r_{MZ} = h^2 + c^2
\end{equation}

\begin{equation}
r_{DZ} = \tfrac{1}{2}h^2 + c^2
\end{equation}

\begin{equation}
\tfrac{1}{2}h^2 = r_{MZ} - r_{DZ}
\end{equation}

Therefore:
\begin{equation}
h^2 = 2\times(r_{MZ} - r_{DZ})
\end{equation}

Narrow-sense heritability \(h^2\) can therefore be estimated by doubling the difference between the correlation coefficients of MZ versus DZ twins. A 2015 meta-analysis of virtually all human twin studies that had been performed to that date included 2,748 twin studies assessing \textasciitilde18,000 traits and \textasciitilde14.5M twin pairs. The results showed that across all traits, \(h^2\) is 0.488 and \(c^2\) is 0.174 (Polderman et al. 2015). Moreover, in 69\% of studies \(2r_{DZ} = r_{MZ}\), implying that the shared environment \(c^2\) is equivalent across both MZ and DZ twin pairs, and that for most traits, shared environmental or non-additive genetic variation do not make a substantial contribution to phenotypic variation. This was most prominent for neurological, ear, nose and throat, cardiovascular and ophthalmological domains. On the other hand, that study found that several behavioural traits were inconsistent with this model. These traits included conduct disorder, higher-level cognitive functions, mental and behavioural disorders due to the use of alcohol or tobacco, anxiety disorder, and weight maintenance. For these traits, either or both non-additive genetic influences or shared environmental influences are therefore needed to explain the observed patterns of twin correlations (Polderman et al. 2015). Leaving to one side the non-additive genetic influences on a phenotype -- which are outside the scope of this thesis -- the findings of this study make it clear that environmental influences contribute substantially to differences in behaviours in humans, which may be further complicated by interactions between the two sources of variation. As social creatures, our social environment is a particularly salient environmental factor, which I discuss below in section \ref{SGE}.

\hypertarget{gene-environment-interactions}{%
\subsection{Gene-environment interactions}\label{gene-environment-interactions}}

The model presented above in Equation \eqref{eq:pge-simp} assumes that the environment has the same effect on different genotypes. When this is not true, and the variance components of genes and environment do not add up -- that is, where \(V_{P} \neq V_{G} + V_{E}\) -- there is said to be an interaction between genes and environment (\textbf{GxE}) (\textbf{Figure \ref{fig:pgxe}}) (Falconer and Mackay 1996).



\begin{figure}

{\centering \includegraphics[width=0.5\linewidth]{figs/introduction/pgxe} 

}

\caption{Schema for a causal model of phenotypic differences, including GxE.}\label{fig:pgxe}
\end{figure}

It can be expressed mathematically as:

\begin{equation}
P = G + E + I_{G\times E} \label{eq:gbye}
\end{equation}

Where the variance decomposition is:

\begin{equation}
V_P = V_G + V_E + V_{G\times E} \label{eq:gbyedecomp}
\end{equation}

\textbf{Figure \ref{fig:schema-gxe}} illustrates the general concept behind GxE. Two different genotypes, \(G_A\) and \(G_B\), are measured for the phenotype \(P\) under two different environments, \(E_1\) and \(E_2\). Figure \ref{fig:schema-gxe}A represents a situation where \(P = G + E\) holds. Figure \ref{fig:schema-gxe}B represents a situation where there is an interaction between genes and environment. GxE is common in nature (Garcia de Le'aniz, Verspoor, and Hawkins 1989; Sgr`o and Hoffmann 2004), and is likely to be particularly relevant to social behaviours, which I expand upon below in section \ref{SGE}.



\begin{figure}

{\centering \includegraphics[width=1\linewidth]{figs/introduction/schema_gxe} 

}

\caption{Two different genotypes, \(G_A\) and \(G_B\) are measured for the phenotype \(P\) under two different environments, \(E_1\) and \(E_2\). Figure adapted from (Caballero 2020).}\label{fig:schema-gxe}
\end{figure}

\hypertarget{animal-breeding}{%
\subsection{Animal breeding}\label{animal-breeding}}

Humans have been manipulating the traits of animals for thousands of years, and this observation greatly influenced Darwin's insights into the power of selection (William G. Hill 2014). The application of the science of genetics to animal breeding was greatly advanced by the work of Jay L. Lush (1896-1982) (Caballero 2020), who developed the well-known `breeder's equation' for predicting an animal population's response to artificial selection depending on: a) the heritability of the trait; and b) the selection pressure exerted (Lush 1937).

Animal breeding involves identifying the animals with the highest breeding value and selecting them for reproduction, then replacing them as better ones come along (William G. Hill 2014). It therefore requires a prediction of the likely effects of using particular individuals for breeding stock in order to maximise the desired phenotype, such as milk yield for cows. Charles Henderson, a student of Lush, recognised that in predicting future animal breeding values, there were the ``fixed'' effects of the parents' breeding values; fixed effects such as herd, year, or season, which had to be included but not estimated; and then ``random'' effects, such as litter size, which were samples from a distribution (William G. Hill 2014). In his PhD thesis (Henderson 1948), he formulated a methodology known as Best Linear Unbiased Prediction (\textbf{BLUP}), which incorporates the genetic additive covariances between individuals in the model as a ``relationship matrix''. Henderson's BLUP method was extended into Restricted (i.e.~residual) Maximum Likelihood (\textbf{REML}) (Patterson and Thompson 1971), which is now the standard method used to fit the linear mixed models in Genome-Wide Association Studies (\textbf{GWAS}). The genetic relationship matrix is also now a widely-used method for correcting against population structure, which I expand upon in the following section.

\hypertarget{GWAS}{%
\subsection{From global variance to individual loci -- family genetic linkage studies and GWAS}\label{GWAS}}

Twin studies focus on the global partitioning of variances attributable to genetic and environmental facts, but since Morgan's work, researchers have also sought to identify the specific genetic variants responsible for Mendelian phenotypes in humans. From the 1980s, advances in DNA sequencing enabled the application of genetic linkage studies -- such as those previously performed in model organisms like \emph{Drosophila} -- to human families, in order to trace inheritances and thereby discover thousands of genes for rare Mendelian diseases (Altshuler, Daly, and Lander 2008). The process involved sequencing known polymorphic regions, and using them as markers to trace the transmission of chromosomal regions in families (Altshuler, Daly, and Lander 2008). The feasibility of this method was first demonstrated in 1983 with Huntington disease (Gusella et al. 1983), and over the following decades demonstrated that Mendelian disease-causing mutations often cause major changes in encoded proteins. The utility of the approach, however, reached its limit when applied to mapping common diseases such as heart disease, breast cancer, and diabetes, even those with familial clustering (Altshuler, Daly, and Lander 2008).

Instead of tracing genes through families, an alternative approach is to map genes by comparing the frequencies of genetic variants between affected and unaffected individuals, known as a ``case-control'' approach. Early applications focused on candidate genes and found correlations between blood-group antigens and peptic ulcer disease in the 1950s (Aird et al. 1954); the human leukocyte antigen (HLA) locus and autoimmune and infectious diseases in the 1960s and 1970s (Klein and Sato 2000); and apolipoprotein E with Alzheimer's disease in the 1980s (Strittmatter and Roses 1996). These early studies were more the exception than the rule, as candidate genes were a ``shot in the dark'' (as established from the family linkage studies, which often found variants that were previously unsuspected). They were susceptible to false positives caused by population structure (Altshuler, Daly, and Lander 2008).

It became clear that Mendelian diseases were rare, caused by highly-penetrant, monogenic variants that remain rare in the population likely as a consequence of purifying selection (Di Rienzo 2006; Jonathan K. Pritchard and Cox 2002; Quintana-Murci and Barreiro 2010; Reich and Lander 2001). In contrast, common diseases tend to be late onset, therefore leaving reproductive fitness unaffected. This logic led to the ``common disease - common variant'' hypothesis, driving a push to expand association analyses genome-wide (Collins, Guyer, and Chakravarti 1997; Lander 1996; Risch and Merikangas 1996).

Next-generation sequencing technologies enabled the inclusion of thousands of SNP markers to be tested simultaneously for their association with a phenotype of interest, with the first Genome-Wide Association Study (\textbf{GWAS}) published in 2001 (Ozaki et al. 2002). GWAS aim to identify genetic variants associated with traits by comparing the allele frequencies of individuals who share similar ancestries, but differ in values for the trait in question (Uffelmann et al. 2021). As of 2021, over 5,700 GWAS have been performed for more than 3,330 human traits (Uffelmann et al. 2021).

In its simple form, GWAS construct a linear model for each SNP marker of the form (Caballero 2020):

\begin{equation}
\textbf{y} = \alpha + \pmb{\beta} \textbf{x} + \textbf{e} \label{eq:gwassimp}
\end{equation}

Where \(\textbf{y}\) is a vector of the sample phenotypes, \(\alpha\) is the population mean, \(\pmb{\beta}\) is a vector of the effect sizes of the marker, \(\textbf{x}\) is the number of copies of an allele for each marker (usually 0, 1, or 2), and \(\textbf{e}\) is a vector of random residual errors.

However, GWAS must always grapple with population structure and unequal relatedness among individuals, which in a given cohort can lead to false discoveries (Ewens and Spielman 1995; Members of the Complex Trait Consortium 2003). This phenomenon is caused by the fact that individuals who share common ancestries will share both variants that do affect the trait of interest, and variants that do not. The non-causal variants that are correlated with the causal variants due to this shared ancestry will therefore also be found to be statistically associated with the trait. Recent methods for dealing with this issue include admixture inference (to identify related clusters, Jonathan K. Pritchard, Stephens, and Donnelly 2000), and principal components analysis (to include the first set of principal components as covariates in the model, Price et al. 2006). A popular method, which can also be combined with the above methods, is to use a linear mixed model that includes the random effect \(g\) to capture the polygenic effect of all SNPs other than the one being tested. \(g\) in turn has a variance that is adjusted by a ``kinship matrix'' or Genetic Relationship Matrix (\textbf{GRM}), \(\pmb{\psi}\), which measures the degree of relatedness between individuals in the sample (Yu et al. 2006; Yang, Lee, et al. 2011; Loh et al. 2015). The standard linear regression model currently used for GWAS can be written as follows (Uffelmann et al. 2021):

\begin{equation}
\textbf{Y} \sim \textbf{W} \alpha ~+~\textbf{X}_s\pmb{\beta}_s + g + e \label{eq:gwaslmm}
\end{equation}
\begin{equation}
g \sim N(0,\sigma^2_A\pmb{\psi}) \label{eq:gwaslmmg}
\end{equation}
\begin{equation}
e \sim N(0,\sigma^2_e \textbf{I}) \label{eq:gwaslmme}
\end{equation}

Where, for each individual, \(\textbf{Y}\) is a vector of phenotype values, \(\textbf{W}\) is a matrix of covariates including an intercept term, \(\alpha\) is a corresponding vector of effect sizes, \(X_s\) is a vector of genotype values for all individuals at SNP \(s\), \(\pmb{\beta}_s\) is the corresponding fixed effect size of genetic variant \(s\) (also known as the SNP effect size), \(g\) is a random effect that captures the polygenic effect of other SNPs, \(e\) is a random effect of residual errors, \(\sigma^2_{A}\) measures additive genetic variation of the phenotype, \(\pmb{\psi}\) is the standard GRM, \(\sigma^2_e\) measures residual variance, and \(\textbf{I}\) is an identity matrix (Uffelmann et al. 2021).

The standard GRM (\(\pmb{\psi}\)) is calculated by first standardising each genotype \(k_{ij}\) for the \(i\)\textsuperscript{th} SNP of the \(j\)\textsuperscript{th} individual (Yang, Lee, et al. 2011):

\begin{equation}
k_{ij} = (x_{ij} - 2p_i)/\sqrt{2p_i(1-p_i)} \label{eq:grmstd}
\end{equation}

Where \(x_{ij}\) is the number of copies of the alternative allele (0, 1, 2), and \(p_i\) is the frequency of the alternative allele.

The matrix of standardised genotypes \(\textbf{K}\) is then multiplied by its transpose and divided by the number of samples \(N\) to produce \(\pmb{\psi}\) (Yang, Lee, et al. 2011):

\begin{equation}
\pmb{\psi} = \textbf{K}\textbf{K}'/ N \label{eq:grm}
\end{equation}

GWAS output a \(p\)-value and directional effect size (\(\beta\)) for each tested SNP. As the method tends to test millions of SNPs, there need to be corrections for multiple testing. The preferred method for setting the \(p\)-value threshold for significance varies depending on the study, but in essence it indicates the proportion of false positives that the investigator is willing to tolerate in their study (Clarke et al. 2011). By convention, in human studies the threshold is set at \(p < 5 \times 10^{-8}\) (Barsh et al. 2012; Bush and Moore 2012; Clarke et al. 2011). The most conservative method, Bonferroni correction, adjusts the type I error rate \(\alpha\) by the number of tests performed, so if \(\alpha = 0.05\) and the number of loci tested is 500,000, the corrected significance threshold is \(0.05 / 500,000 = 1\times10^{-6}\). Bonferroni correction tends to be conservative because it assumes that all loci are independent, which is untrue due to genetic linkage. A less conservative method is a correction for the false discovery rate (\textbf{FDR}), which provides an estimate of the proportion of significant results that are actually true (Hochberg and Benjamini 1990). It is widely applied to GWAS (van den Oord 2008), but also suffers from that lack of independence between loci. The `gold standard' method for accurate detection is permutation testing (Westfall and Young 1993), where the phenotype is randomly permuted across samples and a separate GWAS is run \(N\) times, representing \(N\) possible samplings of individuals under the null hypothesis. Although more computationally intensive, it generates an empirical distribution with resolution \(N\), such that an \(N\) of 10 gives an empirical \(p\)-value of 1/10\textsuperscript{th} of a decimal place (Bush and Moore 2012).

\hypertarget{animal-models}{%
\subsection{Animal models}\label{animal-models}}

The methods described above have led to great advances in our understanding of the genetic basis of complex traits. However, when applied to humans, they are hampered by the inability to experimentally manipulate either genes or environment. Since the rediscovery of Mendel's work, researchers have resorted to using model organisms, with which it is possible to control for both. The genetics of model organisms may be controlled to a degree by establishing inbred strains. Mendel used self-fertilising plants, which without sexual reproduction are effectively inbred strains. Given their utility for investigating fundamental biological principles, ``panels'' of inbred strains for several plant and insect model organisms have since been established, including the thale cress (\emph{Arabidopsis thaliana}) (Bergelson and Roux 2010), common bean (\emph{Phaseolus vulgaris L}) (Johnson and Gepts 1999), tomato (\emph{Lycopersicon esculentum}) (Saliba-Colombani et al. 2000), maize (\emph{Zea mays}) (Limami et al. 2002), nematode (\emph{Caenorhabditis elegans}) (Evans et al. 2021), and fruit fly (\emph{Drosophila melanogaster}) (Mackay and Huang 2018).

In Mendel's time, inbred strains of mammals were unknown, but it is in principle possible to create them through the repeated mating of siblings over successive generations, provided they survive the inbreeding depression (D. Charlesworth and Willis 2009). As the individuals within each line inherit the same haplotype from their related parents, they become almost genetically identical to one another, with the added benefit that their genotypes can be replicated across time in subsequent generations. In the early 1900s, researchers began to develop inbred lines of guinea pigs and mice for the purpose of genetic research. In 1906, an inbreeding experiment involving guinea-pigs was started by George M. Rommel of the Animal Husbandry Division of the United States Department of Agriculture. The experiment was taken over in 1915 by Sewall Wright, who used them to develop his vast contributions to the field (Eaton 1932).

Contemporaneously, the Harvard undergraduate Clarence Cook Little and his supervisor William Ernest Little collaborated with Abbie Lathrop, a breeder of fancy mice and rats which she marketed to rodent hobbyists and keepers of exotic pets, and later began selling in large numbers to scientific researchers (Steensma, Kyle, and Shampo 2010). In 1902, Castle bought some of Lathrop's mice for his laboratory (Steensma, Kyle, and Shampo 2010). The most frequently used laboratory mouse strain for the past 80 years, C57BL/6J (``Black J''), is derived from one of Lathrop's animals -- mouse number 57 -- bred by Little (Steensma, Kyle, and Shampo 2010). Large panels of inbred mouse lines have since been established, such as the Collaborative Cross (\textbf{CC}) (Threadgill et al. 2011), Diversity Outcross (\textbf{DO}) (Svenson et al. 2012) and B6-by-D2 (\textbf{BXD}) (Peirce et al. 2004). However, inbred strains of this organism are descended from only a small number of individuals, originating from three separate species that were bred in captivity, and then undergoing complex domestication before the laboratory lines were established (Morse 2012; Wade and Daly 2005).

Farm animals have also led to advances in understanding the contributions of genes and environment to complex phenotypes (L. Andersson 2001; L. Andersson and Georges 2004; Goddard and Hayes 2009). Their organ systems may be more similar to humans than mouse (L. Andersson 2001); their high genetic homogeneity increases the power to detect causal variants; and they tend to produce large numbers of offspring which allows for the testing of progeny (L. Andersson and Georges 2004).

Although these mammal species are appropriate models for humans due to their orthologous mammalian organ systems and cell types, they are highly domesticated, and therefore do not represent the kind or scope of genetic variation present in wild populations. As gene-environment studies seek to ultimately understand their effects on traits out outbred populations such as humans, there is accordingly a need for a panel of inbred vertebrates that represents the genetic variation that is present in natural populations.

\hypertarget{MIKK-background}{%
\subsection{The Medaka Inbred Kiyosu-Karlsruhe (MIKK) panel}\label{MIKK-background}}

The medaka fish (\emph{Oryzias latipes}) is the model organism studied by Tatsuo Aida, which I introduced above in section \ref{mapping-sec}. Medaka has been studied as a model organism in Japan for over a century (Wittbrodt, Shima, and Schartl 2002), and is gaining recognition elsewhere as a powerful genetic model for vertebrates (Spivakov et al. 2014). In addition to possessing a number of desirable traits that are characteristic of model organisms (including their small-size, short reproduction time, high fertility, and concise genome), medaka are also -- uniquely among vertebrates -- resilient to inbreeding \emph{from the wild}.

This resilience to inbreeding has allowed (predominantly Japanese) researchers to develop several inbred strains that have been maintained for over 100 generations (Murata et al. 2019). One of the most famous inbred medaka strains, \emph{HdrR}, was derived from Tatsuo Aida's collection of southern Japanese medaka fish (Fukamachi and Naruse 2021). Other prominent strains include \emph{HO5} and \emph{iCab} from southern Japan, and \emph{Kaga} and \emph{HNI} from northern Japan. The locations of the originating populations of these strains are set out in \textbf{Figure \ref{fig:line-locations}}.



\begin{figure}

{\centering \includegraphics[width=1\linewidth]{figs/pilot/line_locations} 

}

\caption{Image adapted from Spivakov et al. (2014), showing the locations of the originating populations of the inbred medaka strains used in the studies described in Chapters \ref{Pilot-chap} and \ref{Somite-chap}, as well as the Kiyosu population from which the MIKK panel was derived (Chapters \ref{MIKK-genomes-chap} and \ref(MIKK-F2-chap)).}\label{fig:line-locations}
\end{figure}

Since 2010, the Birney Group at EMBL-EBI, in collaboration with the Wittbrodt Group at COS, University of Heidelberg and the Loosli Group at the Karlsruhe Institute of Technology (\textbf{KIT}), have been working to establish the world's first panel of vertebrate inbred lines from the wild -- now known as the Medaka Inbred Kiyosu-Karlsruhe Panel (\textbf{MIKK panel}). The MIKK Panel was bred from a wild population caught near Kiyosu in Southern Japan (\textbf{Figure \ref{fig:line-locations}}), and now comprises 80 inbred, near-isogenic ``lines'' (Fitzgerald et al. 2022).

The MIKK Panel was created to map genetic variants associated with quantitative traits at a high resolution, and to explore the interactions between those variants and any environmental variables of interest. In \textbf{Chapter \ref{MIKK-genomes-chap}} of this thesis,\footnote{The contents of this Chapter \ref{MIKK-genomes-chap} have been published in Fitzgerald et al. (2022) and Leger et al. (2022).} I describe several genomic characteristics of the MIKK panel that are relevant to its utility for genetic mapping of complex traits, including:

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\item
  The inbreeding trajectory of the MIKK panel;
\item
  Levels of homozygosity across the MIKK panel;
\item
  The evolutionary history of the MIKK panel's founding population;
\item
  The allele-frequency distribution and rate of linkage disequilibrium (LD) decay; and
\item
  The structural variants present in 9 lines using ONT long-read sequencing data.
\end{enumerate}

The fundamental utility of the MIKK panel arises from: a) its representation of the genetic variation that exists in a wild population; b) the ability to replicate individuals with near-identical genotypes, allowing one to modify environmental variables while keeping the genetics constant to determine the strength of those \emph{environmental} effects; and c) the presence of numerous inbred lines that can by placed in the same environment, allowing one to infer the strength of the \emph{genetic} effect. The overarching purpose of the MIKK panel is to provide a resource for researchers who seek to combine the traditional F\textsubscript{2} cross technique pioneered by Mendel with modern genetic sequencing and statistical methods, in order to identify genetic variants associated with complex traits. I describe this process below. For the sake of clarity, in this thesis I refer to the previously established inbred medaka strains \emph{iCab}, \emph{HdrR}, \emph{HO5}, \emph{HNI} and \emph{Kaga} as ``strains'', and the MIKK panel inbred lines as ``lines'', although in effect the two terms are synonymous.

\hypertarget{the-f2-cross}{%
\subsection{\texorpdfstring{The F\textsubscript{2} cross}{The F2 cross}}\label{the-f2-cross}}

The F\textsubscript{2} cross is the same method used by Mendel in his historic paper, but here applied to identifying genetic variants associated with polygenic traits -- traits that are associated with a number of genes. A schema for the method is presented in \textbf{Figure \ref{fig:F2-cross-schema}}. To restate the process, one starts with two inbred strains that diverge for the trait of interest (the `parental strains', or F\textsubscript{0} generation). One then crosses the parental strains to create a generation of F\textsubscript{1} hybrid individuals who each possess, for every pair of their chromosomes, one chromosome from each of their parents. The individuals in this F\textsubscript{1} generation are genetically identical to their parents with respect to their germ line. Finally, one inter-crosses the F\textsubscript{1} generation to create a set of F\textsubscript{2} individuals that share unique combinations of the original F\textsubscript{0} strains' genotypes, and tend to display values for the trait of interest that span across the spectrum between the extreme values of their parents.



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/F2-cross-schema} \caption{Schema of the F\textsubscript{2}-cross experimental setup. The F\textsubscript{0} generation comprises two medaka strains that have extreme, opposing values for a trait of interest, represented by the colours red and blue. Below them is an illustrative single pair of chromosomes. The chromosomes within each pair are depicted as the same colour, as the strains are homozygous through successive generations of inbreeding. Their F\textsubscript{1} offspring is heterozygous for each pair of their 24 chromosomes, and all F\textsubscript{1} individuals are therefore almost genetically identical to one another (with the exception of somatic mutations and the regions of the genome that were not homozygous in the parental generations). The F\textsubscript{1} individuals are then inter-crossed with one another to produce the F\textsubscript{2} generation, which, due to recombination events during gamete formation, have unique combinations of the parental strains' genotypes, and tend to span the phenotypic spectrum between the extremes of their F\textsubscript{0} parental strains, represented by their colours.}\label{fig:F2-cross-schema}
\end{figure}

By combining this traditional crossing method with modern sequencing technologies, researchers are not only able to map genetic variants associated with a phenotype of interest at a higher resolution than in humans, but also functionally validate those variants by carrying out targeted gene editing in an embryo to confirm that the developed organism shows the expected change in phenotype. Many studies have already made use of this combined method (Blaj et al. 2022; Falker-Gieske et al. 2019; X. Gu et al. 2011; Stratz et al. 2018), here referred to as an ``\textbf{F\textsubscript{2}-cross GWAS}''.

\hypertarget{studied-phenotypes}{%
\section{Studied phenotypes}\label{studied-phenotypes}}

In this thesis I use the F\textsubscript{2}-cross GWAS method to identify the genetic variants associated with two traits of primary interest: the periodicity of somite development (\textbf{Chapter \ref{Somite-chap}}), and bold-type behaviours (\textbf{Chapter \ref{MIKK-F2-chap}}). I introduce each of these phenotypes in turn.

\hypertarget{genetic-control-of-somite-period-development}{%
\subsection{Genetic control of somite period development}\label{genetic-control-of-somite-period-development}}

During the development of an embryo, somites are the earliest primitive segmental structures that form from presomatic mesoderm cells (\textbf{PSM}) (Kim et al. 2011). They later differentiate into vertebrae, ribs, and skeletal muscles, thereby establishing the body's anterior-posterior axis. \textbf{Figure \ref{fig:mouse-embryo}} depicts a number of formed somites in a 9.5-day-old mouse embryo.



\begin{figure}

{\centering \includegraphics[width=0.6\linewidth]{figs/somites/mouse_embryo_gridley} 

}

\caption{Image of a mouse embryo at day 9.5 from Gridley (2006), showing somites in darker colours.}\label{fig:mouse-embryo}
\end{figure}

Somite formation occurs rhythmically and sequentially, with the time between the formation of each pair of somites referred to as the ``period''. The period of somite formation varies greatly between species: \textasciitilde30 minutes for zebrafish, 90 minutes for chickens, 2-3 hours for mice, and 5-6 hours for humans (Hubaud and Pourqui'e 2014; Matsuda et al. 2020). \textbf{Figure \ref{fig:somite-seg-ali}} shows the a series of time-stamped images of somite segmentation in medaka fish, generated by my collaborator, Ali Seleit.



\begin{figure}

\includegraphics[width=1\linewidth]{figs/somites/ali_fish_seg_compiled} \hfill{}

\caption{Time-stamped images of somite segmentation in medaka, generated by Ali Seleit.}\label{fig:somite-seg-ali}
\end{figure}

The period of somite formation is controlled by a molecular oscillator, known as the `segmentation clock', which drives waves of gene expression in the Notch, fibroblast growth factor (FGF), and Wnt pathways, forming a signalling gradient that regresses towards the tail in concert with axis elongation (Gomez et al. 2008). Over the course of elongation, the wave period increases (i.e.~each somite takes longer to form), and the PSM progressively shrinks until it is exhausted, eventually terminating somite formation (Gomez et al. 2008).

It is not fully understood how the phase waves of the segmentation clock are initially established (Falk et al. 2022). Matsuda et al. (2020) found that period differences between mouse and human occur at the single-cell level (i.e.~not due to intercellular communication), and are driven by biochemical reaction speeds - specifically, mRNA and protein degradation rates, transcription and translation delays, and intron and splicing delays. To identify the genetic basis of these biochemical differences, our collaborators Ali Seleit and Alexander Aulehla at EMBL-Heidelberg used a CRISPR-Cas9 knock-in approach (Seleit, Aulehla, and Paix 2021) to establish a medaka \emph{Cab} strain with an endogenous, fluorescing reporter gene (Her7-Venus) for the oscillation signalling pathway. This method allows them to image somite formation and extract quantitative measures for segmentation clock dynamics, and they determined that the southern Japanese \emph{Cab} strain and the northern Japanese \emph{Kaga} strain have divergent somite periodicity, where \emph{Kaga}'s tends to be faster, and \emph{Cab}'s slower (\textbf{Figure \ref{fig:F0-Cab-Kaga-HdrR}}).



\begin{figure}

{\centering \includegraphics[width=0.5\linewidth]{figs/somites/ali_period_F0_Cab_Kaga} 

}

\caption{Comparison of period for three inbred medaka strains (\emph{Cab}, \emph{Kaga} and \emph{HdrR}). Kaga's period is lower, and therefore it takes less time to form each somite than \emph{Cab}. Figure generated by Ali Seleit.}\label{fig:F0-Cab-Kaga-HdrR}
\end{figure}

Our collaborators accordingly set up an F\textsubscript{2} cross experiment as described above, using the reporter-carrying \emph{Cab} strain and the \emph{Kaga} strain as the parental F\textsubscript{0} strains, in order to identify genetic loci associated with these differences in clock dynamics. They inter-crossed the hybrid F\textsubscript{1} generation to create a sample of 622 F\textsubscript{2} individuals, imaged the developing embryos of these F\textsubscript{2} samples, and used pyBOAT (Schmal, Mönke, and Granada 2022) to extract the oscillation features during somite development. \textbf{Figure \ref{fig:somite-period-ali}} shows a series of raw images used by pyBOAT to track the elongation of a medaka tail during somitogenesis, with the identified posterior tip of the embryo labelled with a blue circle. In \textbf{Chapter \ref{Somite-chap}}, I describe the bioinformatic analyses I performed to map the genetic variants associated with this period phenotype.



\begin{figure}

{\centering \includegraphics[width=0.8\linewidth]{figs/somites/ali_compiled_somite_elong} 

}

\caption{Screenshots of vertebral elongation in an F\textsubscript{2} individual captured by Ali Seleit during imaging. The blue circle represents the point tracked by pyBOAT over time, generating the quantitative phenotype data on period development used in this study.}\label{fig:somite-period-ali}
\end{figure}

\hypertarget{behaviour}{%
\subsection{Behaviour}\label{behaviour}}

In \textbf{Chapters \ref{Pilot-chap}} and \textbf{\ref{MIKK-F2-chap}} I use medaka as a model organism to explore social genetic effects on bold-type behaviours. Behaviour is a complex trait that is affected by both genes and environment, and for social animals such as humans and medaka, one's social environment is considered likely to constitute a large component of the environmental effect (Ruzzante and Doyle 1990; Young 2008). Apart from social aspects, an organism must face many ``hostile forces of nature'' throughout its life (Buss 1991; Darwin 1859), such as food shortages, predation, harsh climate, and diseases. Adaptive behaviours allow individuals to navigate such dangers and maximise the likelihood of their survival at both the individual and population level (Lima and Dill 1990).

Boldness-shyness is thought to be a fundamental axis of behavioural variation in many species, with an obvious causal relationship to an individual's likelihood of survival, and consequently with natural selection at the population level (Sloan Wilson et al. 1994). It represents an evolutionary trade-off between acquiring benefits (in terms of food or mates) and avoiding harms (in terms of predators or conspecific competitors), with each situation accompanied by its own optimal degree of risk (Lima and Dill 1990). It is both heritable (Svartberg 2002; Culum Brown, Burgess, and Braithwaite 2007), and subject to change following different life experiences or under different environmental conditions (Culum Brown, Burgess, and Braithwaite 2007).

This axis of behaviour has been studied extensively in many organisms. Two generic paradigms for measuring boldness include the \emph{open field test} and the \emph{novel object} test. The open field test involves observing the test subject while it moves freely in an experimental setting, and has been performed on the bullfrog tadpole (Carlson and Langkilde 2013), gecko (Nordberg, Denny, and Schwarzkopf 2021), mouse (Garfield et al. 2011; Yuen et al. 2017), rat (Baud et al. 2014), rabbit (Meijsser et al. 1989), common vole (Herde and Eccard 2013), and Siberian dwarf hamster (Kanda, Louon, and Straley 2012). It is particularly favoured with fish, where the general interpretation is that shy individuals tend to react to novelty by reducing their activity and becoming more vigilant, whereas bold individuals show higher levels of activity and exploratory behaviour (C. Brown, Jones, and Braithwaite 2007; Matsunaga and Watanabe 2010; Laland, Krause, and Brown 2011; Dahlbom et al. 2011; Lucon-Xiccato and Bisazza 2017; Alfonso et al. 2019; Lucon-Xiccato et al. 2020, 2022; Hamilton et al. 2021).

The second paradigm is the novel object test, where a novel object is introduced to the test subject's environment as a way of simulating a potential threat. The test has been used with birds (de Azevedo and Young 2006), squirrels (Uchida et al. 2019), rabbits (A. Andersson, Laikre, and Bergvall 2014), baboons (Carter et al. 2012), vervet monkeys (Blaszczyk 2017), and grey mouse lemurs (Dammhahn and Almeling 2012). Like the open field test, it has also been extensively used with fish (C. Brown, Jones, and Braithwaite 2007; Schjolden, Stoskhus, and Winberg 2005; Wilson et al. 1993; Dominic Wright, Butlin, and Carlborg 2006; D. Wright et al. 2003; Hamilton et al. 2021).

Where both test paradigms are performed on the same fish, the behaviours exhibited were found to be correlated across assays, indicating that both were measuring the same boldness-shyness axis (C. Brown, Jones, and Braithwaite 2007). Both the open field and novel object test also permit the measurement of habituation, where the response of the fish may change over time after growing accustomed to the new environment or object. Variation in the length of time the fish take to return to a more ``normal'' level of movement may indicate different levels of boldness (Laland, Krause, and Brown 2011).

\hypertarget{SGE}{%
\subsection{Social genetic effects}\label{SGE}}

The experiments I describe in \textbf{Chapters \ref{Pilot-chap}} and \textbf{\ref{MIKK-F2-chap}} are designed to simultaneously measure both the direct effect of an individual's genes on their behaviour, as well as the indirect effect of their social partner's genes on the focal individual's behaviour, which is also an environmental factor (Baud et al. 2017). Social genetic effects have been shown to exert influence on various traits in mice including anxiety, wound healing, immune function, and body weight (Baud et al. 2017), and various traits related to development and survival in many species of livestock (Ellen et al. 2014). However, in those studies the social interactions were maintained throughout development, and it is unclear whether social genetic effects can still exert influence on adaptive behaviours during discrete, time-limited interactions.

In \textbf{Chapter \ref{Pilot-chap}} I apply a combined open field and novel object assay to determine the respective contributions of direct and social genetic effects on bold-shy behaviours in medaka fish, using 5 previously-established inbred strains. This ``pilot study'' was intended to validate the assay by ensuring that it could detect both direct and social genetic effects, and to confirm that it isn't unduly affected by confounding factors. In \textbf{Chapter \ref{MIKK-F2-chap}} I extend this analysis over the MIKK panel. I identify the lines that diverged in both (a) their own behaviour; and (b) the level of transmission of their behaviour onto their \emph{\definecolor{iCab_424B4D}{HTML}{424B4D}\textcolor{iCab_424B4D}{iCab}} tank partner, and then use them as the parental strains in an F\textsubscript{2}-cross to identify the genetic variants associated with those differences.

\hypertarget{poor-transferability-of-polygenic-scores-to-diverse-human-populations}{%
\section{Poor transferability of polygenic scores to diverse human populations}\label{poor-transferability-of-polygenic-scores-to-diverse-human-populations}}

\textbf{Chapter \ref{Fst-chap}} -- the final chapter of this thesis -- relates to the poor transferability of polygenic scores derived from GWAS results when applied to non-European populations. Humans have long sought to use genetic information to predict an individual's likely value for a given trait, in our own species and in other organisms. It is now clear that complex traits such as height, intelligence, and behaviour are highly polygenic, meaning that they are genetically influenced by hundreds or thousands of genetic variants, each exerting a small effect in one or the other direction along the trait's spectrum (Sella and Barton 2019). A richer understanding of the cumulative effect of genetic variants on any trait allows for the prediction of the value that an individual is most likely to have for that trait. Of all human traits, diseases are particularly salient; in 2018, the global healthcare industry was valued at US\$8 trillion, and predicted to increase to US\$12 trillion by 2022 ({``The \$11.9 {Trillion Global Healthcare Market}: {Key Opportunities} \& {Strategies} (2014-2022) - {ResearchAndMarkets}.com''} 2019). This strong financial imperative complements the moral imperative to reduce suffering, together driving the question of how to use genetic information to improve human health.

Recent technological developments have made it possible to sequence human genomes at scale, and it is thought that by combining detailed genetic information with with other environmental and phenotypic information (such as lifestyle or clinical factors), clinicians could move towards the practice of ``precision medicine'', where interventions could be tailored to their patients' unique risk profiles (Wray, Goddard, and Visscher 2007). The use of genetic information to predict individuals' values for a trait of interest entails the construction of metrics known as ``polygenic scores'' (\textbf{PGS}) (Lambert, Abraham, and Inouye 2019). When the trait is a disease, PGS is commonly known as polygenic risk scores (\textbf{PRS}), or genetic risk profiling, but I use the term PGS to encompass both disease and non-disease traits.

\hypertarget{pgs-and-gwas}{%
\subsection{PGS and GWAS}\label{pgs-and-gwas}}

\hypertarget{pgs}{%
\subsubsection{PGS}\label{pgs}}

PGS using genetic information alone show modest yet reliable accuracy for the prediction of complex traits (Alicia R. Martin et al. 2019): the correlations between PGS and the trait value as measured by \(R^2\) have reached 0.24 for height (Yengo et al. 2018), and 0.12-0.16 for educational attainment (Okbay et al. 2022). PGS also improve predictions beyond non-genetic clinical models for a variety of health-related traits, including breast cancer (Maas et al. 2016), prostate cancer (Schumacher et al. 2018), and type I diabetes (Sharp et al. 2019). The predictive accuracy of PGS scores can be further improved by combining genetic information with lifestyle and clinical factors, as seen with cardiovascular disease (Khera et al. 2018; Kullo et al. 2016; Natarajan et al. 2017; Paquette et al. 2017; Tikkanen et al. 2013; Sun et al. 2021).

\hypertarget{gwas}{%
\subsubsection{GWAS}\label{gwas}}

Most GWAS have been performed with individuals of European ancestry, despite only constituting around 16\% of the present global population. Although the proportion of participants in GWAS from a non-European background increased from 4\% in 2009 to 16\% in 2016 (Popejoy and Fullerton 2016)), as of 2019, 79\% of all GWAS participants recorded in the GWAS Catalog were of European ancestry, and the proportion of non-European individuals has remained the same or reduced since late 2014 (Alicia R. Martin et al. 2019). This bias extends to PGS studies, where as of 2019, only 67\% of them included only participants of European ancestry, with another 19\% including only East Asian ancestry participants, and only 3.8\% with cohorts of African, Hispanic, or Indigenous ancestry (Duncan et al. 2019).

It is therefore unsurprising that PGS scores are far better at predicting disease risk in individuals of European ancestry than in those of non-European ancestry (Alicia R. Martin et al. 2017; Alicia R. Martin et al. 2019). Indeed, the predictive accuracy of PGS scores decays with genetic divergence of the GWAS ``independent'' or ``test'' sample, from the ``discovery'' or ``training'' sample, as established in both humans (Alicia R. Martin et al. 2017; Alicia R. Martin et al. 2019), and livestock (Clark et al. 2012; Habier et al. 2010; Pszczola et al. 2012). Compared to PGS scores for those of European ancestry, PGS scores across multiple traits are \textasciitilde64-78\% less accurate for individuals of African ancestry, (Duncan et al. 2019; Alicia R. Martin et al. 2019), \textasciitilde50\% less accurate for individuals of East-Asian ancestry, and \textasciitilde37\% less accurate for individuals of South-Asian ancestry (Alicia R. Martin et al. 2019).

\hypertarget{contributors-to-pgs-non-transferability}{%
\subsection{Contributors to PGS non-transferability}\label{contributors-to-pgs-non-transferability}}

What explains this disparity in predictive value? A number of factors may be responsible, including:

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\item
  The failure of GWAS to identify causal variants that either do not exist or are not identifiable within the ``discovery'' sample, for both technological and methodological reasons (Alicia R. Martin et al. 2019);
\item
  The sample populations may differ in linkage disequilibrium (\textbf{LD}) -- the correlation structure of the genome -- which would change the estimated effect sizes of the causal variants, even when the causal variants themselves are the same (Alicia R. Martin et al. 2019);
\item
  Allele frequencies of the causal variants, and the distribution of the effect sizes of the causal variants, may differ between populations (Alicia R. Martin et al. 2017; Scutari, Mackay, and Balding 2016); and
\item
  The environments and demographies of populations tend to differ. Such differences are often correlated with genetic divergence due to geography, making it difficult to determine whether the associations are driven by the differences between population in their genetics, or their environments (Alicia R. Martin et al. 2019; Kerminen et al. 2019).
\end{enumerate}

The first three factors can degrade predictive performance even in the absence of biological and environmental differences. On the other hand, environmental and demographic differences can drive forces of natural selection which can in turn drive differences in causal genetic architecture (Alicia R. Martin et al. 2019).

I discuss each of these factors in turn before addressing point (3) in \textbf{Chapter \ref{Fst-chap}}.

\hypertarget{fst-discovery-sec}{%
\subsubsection{Technological and methodological limitations of GWAS}\label{fst-discovery-sec}}

The power to discover a causal variant through GWAS depends on the variant's effect size and frequency in the study population (Alicia R. Martin et al. 2019; Sham et al. 2000). That is to say, the stronger the variant's effect, or the more common it is, the more likely it is to be discovered. Rare variants tend to have stronger effect sizes (Watanabe et al. 2019), likely due to purifying selection (Park et al. 2011), and tend not to be shared across populations (Gravel et al. 2011; 1000. G. P. Consortium et al. 2015). This is particularly relevant for African populations, as they have a much greater level of genetic variance than other populations due to the human species having originated on that continent (1000. G. P. Consortium et al. 2015). Therefore, if GWAS aren't performed on diverse populations, PGS can't take into account the rare variants present in non-European populations that are likely to exert stronger effects on the trait of interest. There are also several other issues that can affect the discoverability of causal variants through GWAS, including the technology used for genotyping, the selection of the cohort, and the necessary exclusion of genotypic outliers.

With respect to genotyping technologies, GWAS often use data from SNP microarrays. These do not sequence the whole genome, but rather a selection (from several hundred thousand to millions) of genetic markers intended to present \emph{common} genetic variation (Porcu et al. 2013), which accordingly tend to neglect rare genetic variants (Uffelmann et al. 2021). To increase the density of genotypes, which would increase the likelihood of refining the association signal and identifying causal variants, researchers often ``impute'' variants that aren't sequenced directly (Porcu et al. 2013). The imputation process involves ``phasing'' the study genotypes onto the genotypes of a ``reference panel'' (McCarthy et al. 2016). However, if the reference panel does not sufficiently represent the population in the study sample, they are likely to miss or incorrectly impute those genotypes (Alicia R. Martin et al. 2019). Again, this is particularly problematic for under-represented African populations.

The lack of representation of rare variants in SNP microarrays can be overcome by using next-generation sequencing technologies such as whole-genome sequencing (\textbf{WGS}) and whole-exome sequencing (\textbf{WES}). (The former seeks to sequence the full genome, and the latter of only targets the coding regions of the genome.) These methods are more expensive than SNP microarrays, which hinders their widespread use at scale, and although their costs are continuing to decrease rapidly, there is a question as to whether they return a proportionate benefit in all use cases (Schwarze et al. 2018).

A second limitation is the selection of GWAS cohorts, which can introduce selection and collider biases (Uffelmann et al. 2021). For instance, the UK Biobank, which contains genetic and phenotypic data on 500,000 participants who volunteered for inclusion between 2006 and 2010, tend to be older, female, healthier, and wealthier than non-participants (Fry et al. 2017). This creates the possibility of confounding genetic associations with environmental factors, which I discuss further in section \ref{fst-env-sec} of Chapter \ref{Fst-chap}

A third limitation is the ``quality control'' step that is required during the GWAS process (Uffelmann et al. 2021). To avoid confounding from population stratification, which can lead to overestimated heritability and biased PGS, GWAS cohorts are filtered to include only those with similar ancestries -- or relative genetic homogeneity -- by clustering individuals through principal component analysis (\textbf{PCA}) of their genotypes, and excluding outliers. I elaborate on the issue of population stratification in section \ref{fst-env-sec} below, but at present, a statistical model for GWAS that can include cohorts with diverse ancestries without the risk of serious confounding is yet to be developed (Jeremy J. Berg et al. 2019).

\hypertarget{differences-in-ld}{%
\subsubsection{Differences in LD}\label{differences-in-ld}}

Because GWAS SNP markers are often not the causal variants themselves, but merely in physical proximity to them, the estimated effect size of a SNP marker depends on the extent to which it is in LD with the causal variant (Mostafavi et al. 2020; Jonathan K. Pritchard and Przeworski 2001). To illustrate the problem, if a SNP has an LD \(r^2\) with a causal variant of 0.8 in the discovery population and 0.6 in the target population, it would explain 25\% = (1 - 0.6/0.8) less trait variation in the target population, and would therefore be less predictive (Ying Wang et al. 2020).

These differences in effect-size estimates may typically be small for most regions of the genome, but as PGS sum across all such effects, they aggregate these population differences (Alicia R. Martin et al. 2019; Jeremy J. Berg et al. 2019). Previous empirical and simulation studies have shown that accuracy of PGS scores decay with increased genetic differentiation (\(F_{ST}\) -- described below in \ref{Fst-descr}) and LD differences between populations (Habier et al. 2010; Pszczola et al. 2012; Scutari, Mackay, and Balding 2016; Ying Wang et al. 2020). The issue may be addressed to a degree by using LD information from an external reference panel as a prior to infer the posterior mean effect size of a genetic variant -- Vilhjálmsson et al. (2015) demonstrated through simulations that this could improve PGS predictive accuracy. Yet the most appropriate means of deal with differences between populations in LD remains an active area of research (Duncan et al. 2019).

\hypertarget{differences-in-allele-frequencies}{%
\subsubsection{Differences in allele frequencies}\label{differences-in-allele-frequencies}}

Causal variants can differ in both frequency and effect size between different ancestry groups, e.g.~for lactase persistence (S'egurel and Bon 2017), or skin pigmentation (Adhikari et al. 2019). If a causal allele is rare in the GWAS discovery population, even if it is discovered (see section \ref{fst-discovery-sec} above), it is likely to have noisy effect size estimates, and therefore likely to inaccurately estimate its effect size in a different population where it exists at a higher frequency.

Differences in allele frequencies between populations can arise through random genetic drift, or be driven by selective pressures towards the trait optima for a given environment (Harpak and Przeworski 2021). However, evolutionary biologists have found that differences between populations in the mean values for traits tend to occur through small, coordinated shifts in their allele frequencies (Jeremy J. Berg et al. 2019; Edge and Coop 2019). In Chapter \ref{Fst-chap}, I explore the differences in allele frequencies across populations for all polygenic traits in the GWAS Catalog, and confirm that with few exceptions -- including skin pigmentation, and HIV viral load -- the differences in allele frequencies between populations tends to be small.

\hypertarget{fst-env-sec}{%
\subsubsection{Differences in environment}\label{fst-env-sec}}

Genes continuously interact with each other (GxG, or ``epistasis'' (Gros, Le Nagard, and Tenaillon 2009)), the genes of one's parents (``genetic nurture'', Kong et al. (2018)) or social companions (``social genetic effects'') (Domingue et al. 2018; Baud et al. 2017),\footnote{Which I also explore in Chapters \ref{Pilot-chap} and \ref{MIKK-F2-chap}} and the wider non-genetic environment (GxE).

The respective contributions of genetics and environment to traits with social value -- such as intelligence -- is highly contentious, especially when there are apparent differences between populations in the mean values for those traits. PGS measure the proportion of variance within a population that is explained by genetics. Because PRS summarises a \emph{proportion} of the total variance, when studying a population that is subject to greater environmental variation, the variance attributable to genetic factors will proportionately reduce. The corollary being that when studying a population where the environment is held constant, the proportion of variance for that trait that is explained by genetic factors will approach 1. Therefore, increases in the amount of environmental variance that a population is exposed to will reduce the accuracy of PGS predictions when applied to that population.

Different environments are also often correlated with population structure (Jeremy J. Berg et al. 2019). For example, in East Asia, there is a greater proportion of individuals of East-Asian ancestry than there is of European ancestry, and \emph{vice versa} in Europe. Those East-Asian individuals will therefore tend to share more of their genetic background with each other than with Europeans, and that population structure will be correlated with the different environments that exist in East Asia compared to Europe. This makes it difficult to determine whether it is the differences in their environments or the differences in their genetics that is driving the discrepancies between the mean values for traits between those populations. These complexities are unlikely to be resolved in the near future, which makes it attractive to turn to model organisms to address more basic biological questions regarding GxE in relation to complex traits (L. Andersson and Georges 2004), as I have done with respect to behaviour in Chapters \ref{Pilot-chap} and \ref{MIKK-F2-chap}.

\hypertarget{Fst-descr}{%
\subsection{\texorpdfstring{\(F_{ST}\)}{F\_\{ST\}}}\label{Fst-descr}}

The widely-used fixation index (\(F_{ST}\)) was introduced independently by Sewall Wright (S. Wright 1949) and Gustave Malécot (Mal'ecot 1948) as a metric for measuring the genetic diversity between populations. In Wright's notation, \(F\) refers to ``fixation'' of an allele, and \(_{ST}\) refers to ``subpopulations within the total population''. \(F_{ST}\) quantifies the relative variance in allele frequency between groups compared to within groups, reflecting the combined effects of genetic drift, migration, mutation, and selection (Holsinger and Weir 2009). The metric ranges from 0 to 1, where loci with high \(F_{ST}\) values -- that is, loci with a large relative between-group variance in allele frequencies -- may have been subject to selection or different demographic processes (Holsinger and Weir 2009). The metric has customarily been used to identify regions of the genome have been subject to diversifying selection (Akey et al. 2002; Guo, Dey, and Holsinger 2009; Bruce S. Weir et al. 2005).

In Chapter \ref{Fst-chap}, I extract all approximately-unlinked, trait-associated SNPs in the GWAS Catalog (MacArthur et al. 2017), and use \(F_{ST}\) to measure the differences in allele frequencies between the 26 diverse populations in the 1000 Genomes Project phase 3 release dataset (1000. G. P. Consortium et al. 2015). I then rank the traits based on the differences in the distributions of \(F_{ST}\) relative to an allele-frequency matched set of control SNPs to identify the traits for which there is substantial variation in their associated alleles across global populations.

\hypertarget{MIKK-genomes-chap}{%
\chapter{Genomic variations in the MIKK panel}\label{MIKK-genomes-chap}}

\hypertarget{the-medaka-inbred-kiyosu-karlruhe-mikk-panel}{%
\section[The Medaka Inbred Kiyosu-Karlruhe (MIKK) panel]{\texorpdfstring{The Medaka Inbred Kiyosu-Karlruhe (MIKK) panel\footnote{This project was carried out in collaboration with Felix Loosli's group at the Karlsruhe Institute of Technology (KIT), and Joachim Wittbrodt's group in the Centre for Organismal Studies (COS) at the University of Heidelberg. This chapter sets out my contributions to Fitzgerald et al. (2022) and Leger et al. (2022), which were published as companion papers in the journal \emph{Genome Biology} to introduce the MIKK panel to the scientific community, and describe the genetic characteristics of the MIKK panel that would make it a useful resource for other researchers who wish to explore the genetics of quantitative traits in vertebrates. I was granted joint-first authorship on both papers.}}{The Medaka Inbred Kiyosu-Karlruhe (MIKK) panel}}\label{the-medaka-inbred-kiyosu-karlruhe-mikk-panel}}

I introduced the MIKK panel in section \ref{MIKK-background} of the previous chapter, and here I describe several of its genomic and phenotypic characteristics that are relevant to its utility for the genetic mapping of complex traits. These include a visualisation of the inbreeding trajectory of the panel (section \ref{inbreeding-sec}), an exploration of the evolutionary history of the MIKK panel's founding population (section \ref{introgression-sec}), measurement of the levels of homozygosity across the panel (section \ref{nuc-div}), an assessment of the panel's allele-frequency distribution and rate of linkage disequilibrium (LD) decay (section \ref{ld-decay-sec}), and a characterisation of the structural variants present in a smaller sample of lines using Oxford Nanopore long-read sequencing data (section \ref{mikksv-sec}).

\hypertarget{genomic-characterisation-of-the-mikk-panel}{%
\section{Genomic characterisation of the MIKK panel}\label{genomic-characterisation-of-the-mikk-panel}}

\hypertarget{non-sib-calls}{%
\subsection{MIKK panel DNA sequence dataset}\label{non-sib-calls}}

In 2018, our collaborators sequenced 79 of the 80 extant MIKK panel lines -- together with several wild Kiyosu samples and individuals from the established \emph{iCab} medaka strain -- from male brain samples using Illumina short-read sequencing technology. Tomas Fitzgerald from the Birney Group at EMBL-EBI then aligned these sequences to the \emph{HdrR} medaka reference and called variants to produce the \textbf{MIKK Illumina call set}, in the form of a .vcf file containing single nucleotide polymorphism (SNP) and small insertion-deletion (INDEL) calls for each line. To avoid allele frequency biases introduced by the 16 pairs/triplets of ``sibling lines'' (see \ref{inbreeding-sec}), I removed each pair's arbitrarily-labelled second sibling line from the variant call set, leaving 63 MIKK panel lines (\textbf{MIKK non-sibling call set}), and used only those calls for the analyses in sections \ref{nuc-div} and \ref{ld-decay-sec} below.

I also performed a pilot study on 9 MIKK panel lines using Oxford Nanopore Technologies (\textbf{ONT}) long-read sequence data. Adrien Leger, then of the Birney Group at EMBL-EBI, aligned these sequences to the \emph{HdrR} medaka reference (Yates et al. 2020), and called variants to produce the \textbf{MIKK ONT call set} in the form of a .vcf file containing structural variants calls for each line with tags for insertions (INS), deletions (DEL), duplications (SUP), inversions (INV) and translocations (TRA). In the work described below, I used these variant call sets as the primary datasets.

\hypertarget{inbreeding-sec}{%
\subsection{Assessing the inbreeding trajectory of the MIKK panel}\label{inbreeding-sec}}

The MIKK panel was bred from a wild population of medaka found in the Kiyosu area near Toyohashi, Aichi Prefecture, in southern Japan (Spivakov et al. 2014). From this wild population, the Loosli Group at KIT set up random crosses of single mating pairs to create 115 `founder families'. For each founder family, they then set up between two and five single full-sibling-pair inbreeding crosses, which resulted in 253 F1 lines. Lines derived from the same founder family are referred to as `sibling lines'. Over the course of the next eight generations of inbreeding, they used only one mating pair per line. I generated \textbf{Figure \ref{fig:InbreedingFigure}A} and \textbf{B} from the inbreeding data provided by the Loosli Group. \textbf{Figure \ref{fig:InbreedingFigure}A} shows the number of lines that survived over the course of the first 14 generations of the inbreeding program, and the various causes for the termination of other lines. \textbf{Figure \ref{fig:InbreedingFigure}B} shows the average fecundity levels of the surviving lines at generation F\textsubscript{16}.



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_genome/20211213_final_figure} \caption{Inbreeding, fecundity and eye size in the MIKK panel lines. \textbf{A}: Status of all MIKK panel lines during the first 14 generations of inbreeding, showing cause of death for non-extant lines. \textbf{B}: Average fecundity of MIKK panel lines in generation F16, as measured during peak egg production in July 2020. \textbf{C}: Distribution of mean relative eye size for female and male medaka across all MIKK panel lines.}\label{fig:InbreedingFigure}
\end{figure}

In addition, the Birney Group at EMBL-EBI generated morphometric data for the MIKK panel lines to demonstrate the distribution of physical phenotypes across the MIKK panel. I used this data on relative eye diameters to generate \textbf{Fig. \ref{fig:InbreedingFigure}C}, where the trait appears to have a roughly normal distribution, with minimal differences in the mean between sexes. I also computed this trait's broad-sense heritability \(\eta^2_{(partial)} = \hat{H^2} = SS_{line} / (SS_{line} + SS_{residuals}) =\) 0.56. This value is on the higher range of heritability estimates for morphological traits across different species inluding mouse and human (Visscher, Hill, and Wray 2008; Claes et al. 2018; Weinberg, Cornell, and Leslie 2018), and indicates that the majority of variance observed across the panel is driven by differences between lines in their genetics. This suggests the MIKK panel will be a promising resource for discovering the genetic variants that are responsible for such differences.

\hypertarget{levels-of-homozygosity}{%
\subsection{Levels of homozygosity}\label{levels-of-homozygosity}}

To assess whether the inbreeding process resulted in high levels of homozygosity in the MIKK panel, for each line I extracted from the MIKK Illumina call set all non-missing, biallelic calls (both SNPs and indels) with \emph{bcftools} (Danecek et al. 2021), counted the number of SNPs within non-overlapping, 5-kb bins, and calculated the proportion of SNPs within each bin that were homozgyous. To provide an example of the levels of homozygosity in the MIKK panel, \textbf{Figure \ref{fig:circos-18-2}} is a circos plot generated with circlize (Z. Gu et al. 2014) for the MIKK panel line 18-2, featuring the proportion of homozygous SNPs per 5-kb bin (pink), and the total number of SNPs in each bin (yellow). This plot is representative of the plots generated for all MIKK panel lines. With few exceptions, almost all 5-kb bins across the genome contain a very high proportion of homozygous variants, and all other MIKK panel lines show similar levels of homozygosity. Although there are more sophisticated means of measuring homozygosity -- for example, by using a Hidden Markov Model to separate out regions of residual homozygosity\footnote{As described by Fitzgerald et al. (2022).} -- this confirms that the breeding program was successful in establishing a panel of inbred lines with levels of homozygosity comparable to established medaka strains that have been inbred for dozens of generations, such as the \emph{iCab} strain at generation F\textsubscript{24} which was sequenced with the MIKK panel (\textbf{Figure \ref{fig:circos-iCab-F24}}).



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_genome/circos/18-2} \caption{Proportion of homozygous SNPs within 5 kb bins in the MIKK panel line 18-2 (pink), and number of SNPs in each bin (yellow).}\label{fig:circos-18-2}
\end{figure}



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_genome/circos/iCab_F24} \caption{Proportion of homozygous SNPs within 5 kb bins in the previously established \emph{iCab} strain at generation F24 (dark grey), and number of SNPs in each bin (yellow).}\label{fig:circos-iCab-F24}
\end{figure}

Medaka has a male heterogametic XX-XY sex determination system, but because the sex chromosomes are karyotypically similar, they have traditionally been assigned the label of ``chromosome 1'' (Fitzgerald et al. 2022). The authors there identified the sex determination region as residing within the locus of 1:15.9-16.5 Mb. As the individuals who were sequenced were male and therefore heterozygous at that locus, one would expect to observe a sharp decrease in homozygosity around that region. However, in this figure -- which was generated from the same data as Fitzgerald et al. (2022) -- such a narrow region is barely visible due to both the thickness of the plot lines, and the non-uniformity of low-homozygosity across that region.

\hypertarget{nuc-div}{%
\subsection{Nucleotide diversity}\label{nuc-div}}

I also sought to assess the levels of genetic diversity in the MIKK panel, and to compare them with the levels of genetic diversity present in the wild Kiyosu population. If the levels were similar, it would confirm that the MIKK panel provided a reasonable representation of the variation present in its originating population, and that any selection for loci associated with domestication was not widespread across the genome. As a means of assessing genetic diversity in the MIKK panel, I calculated the widely-used measure of `nucleotide diversity' (\(\hat{\pi}\)) (Nei and Li 1979) within 500-kb non-overlapping windows across the genome of the 63 lines in the MIKK non-sibling call set (see \ref{non-sib-calls}), and compared this to the nucleotide diversity in 7 wild medaka from the same Kiyosu population from which the MIKK panel was derived. Mean and median nucleotide diversity in both the MIKK panel and wild Kiyosu medaka were similar, and slightly higher in the MIKK panel (mean: MIKK = 0.0038, wild = 0.0037; median: MIKK = 0.0033, wild = 0.0031). This may have been caused by the smaller sample size of wild Kiyosu medaka, as mean \(\hat{\pi}\) for a random sample of 7 MIKK lines dropped to 0.0037, although it is unexpected that the highly-inbred MIKK lines have similar levels of \(\hat{\pi}\) to wild Kiyosu individuals. The patterns of varying nucleotide diversity across the genome are shared between the MIKK panel and wild Kiyosu medaka, where regions with high levels of repeat content tend to have higher nucleotide diversity (r = 0.386, \(p\) \textless{} 0.001) (\textbf{Fig. \ref{fig:NucleotideDiversity}}). I also calculated \(\hat{\pi}\) for each line individually, and as expected, levels of \(\hat{\pi}\) around the (XX/XY) sex determination region of 1:\textasciitilde16-17 Mb are elevated in all lines relative to the consistently low levels found in most other chromosomes.



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_genome/supp_01_pi_circos} \caption{Circos plot with nucleotide diversity (\(\hat{\pi}\)) calculated within 500-kb non-overlapping windows for 63 non-sibling lines from the MIKK panel (\emph{green}) and 7 wild Kiyosu medaka samples from the same originating population (\emph{purple}); proportion of sequence classified as repeats by RepeatMasker (\emph{blue}); mean mapping quality (\emph{pink}); and sex determination locus (\emph{red arrow}).}\label{fig:NucleotideDiversity}
\end{figure}

As expected, there is a peak of \(\hat{\pi}\) around the sex determination locus (as indicated by the red arrow). However, there are far higher peaks on several chromosomes -- most notably for chromosomes 2, 11, and 18 -- which correspond closely to the regions I identified as containing large (\textgreater250 kb) inversions that appear to be shared across at least some of the MIKK panel (\textbf{Fig. \ref{fig:SVInvs}}). These regions are also enriched for large deletions and duplications.(Leger et al. 2022) Inversions cause permanent heterozygosity (Hoffmann, Sgr`o, and Weeks 2004), and duplications and deletions may have increased the density of called SNPs in these regions (Fredman et al. 2004), so the observed depressions in homozygosity at these loci may be the result of such large structural variants that are present in the MIKK panel's genomes. Overall, this analysis confirms that the MIKK panel shows similar levels of homozygosity compared to classical laboratory inbred medaka strains.



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_genome/20210224_sv_invs_lines} \caption{Inversions identified in 9 MIKK panel lines using a combination of ONT long-read and Illumina short-read sequences (see Chapter \ref{mikksv-sec} below).}\label{fig:SVInvs}
\end{figure}

\hypertarget{introgression-sec}{%
\subsection{Introgression with northern Japanese and eastern Korean medaka populations}\label{introgression-sec}}

There is an open question as to whether the Northern and Southern Japanese medaka populations have been able to interbreed since being separated on either side of the Hida and Akaishi Mountains (commonly known in the West as the `Japanese Alps') (Naruse, Tanaka, and Takeda 2011; Spivakov et al. 2014). I therefore sought to determine whether there was evidence of `introgression' -- that is, the transfer of genes through interbreeding -- between the southern Japanese, northern Japanese and eastern Korean medaka populations. To this end, I used the 50-fish multiple alignment from Ensembl release 102 to obtain the aligned genome sequences for the established medaka inbred lines \emph{HdrR} (southern Japan), \emph{HNI} (northern Japan), and \emph{HSOK} (eastern Korea), as well as the most recent common ancestor of all three strains (Howe et al. 2021) (\textbf{Figure \ref{fig:ABBABABA}A}). Using the phylogenetic tree provided with the dataset and the \emph{ape} R package (Paradis and Schliep 2019), I identified the ancestral base of those three strains. For each locus with a non-missing base for \emph{HdrR}, I assigned the allele in that ancestral base as the `ancestral' allele, and the alternative allele as the `derived' allele, and then combined that dataset with the MIKK Illumina call set and variant calls for the southern Japanese \emph{iCab} strain (see \ref{non-sib-calls}).

I chose to use the ABBA-BABA test to explore the levels of introgression between the southern Japanese Kiyosu population from which the MIKK panel was derived, and its ancestrally-related northern Japanese and eastern Korean medaka populations. The ABBA-BABA test is used to measure the proportion of shared genome between populations, and was initially developed to determine whether humans had interbred with neanderthals (Durand et al. 2011). It involves comparing three ingroup populations, denoted as P\textsubscript{1}, P\textsubscript{2} and P\textsubscript{3}, with an outgroup population, denoted as O, which is intended to define the ancestral state. The populations are presumed to have the evolutionary relationship (((P\textsubscript{1}, P\textsubscript{2}), P\textsubscript{3}), O). The letters in \emph{ABBA-BABA} represent the alleles possessed respectively by those populations. Therefore \emph{ABBA} represents a locus where P\textsubscript{1} and O share the \emph{A} allele, and P\textsubscript{2} and P\textsubscript{3} share the \emph{B} allele. Likewise, \emph{BABA} represents a locus where P\textsubscript{1} and P\textsubscript{3} share the \emph{B} allele, and P\textsubscript{2} and O share the \emph{A} allele. In circumstances where the populations have not significantly interbred after deviation,\footnote{Assuming there are no large deviations created by recurrent mutations, ancestral population structure, or variable substitution rates (Durand et al. 2011).} the ratio of \emph{ABBA} to \emph{BABA} sites is expected to be equal. On the other hand, interbreeding between P\textsubscript{2} and P\textsubscript{3} would create a statistically significant deviation from this ratio.

Using the scripts provided by the first author of S. H. Martin, Davey, and Jiggins (2015) on their GitHub page (martin {[}2016{]} 2022), I applied this ABBA-BABA test to measure of the proportion of shared genome in 500-kb sliding windows between the MIKK panel and either \emph{iCab}, \emph{HNI}, or \emph{HSOK} (\textbf{Figure \ref{fig:ABBABABA}}), with a modified `admixture proportion' statistic \(\hat{f}_d\) (S. H. Martin, Davey, and Jiggins 2015). Like the inbred lines of the MIKK panel, \emph{iCab} is strain derived from a southern medaka population, and represents a population for which I expect a substantial mixing between the two. The focus of the exploration here is rather on the degrees of mixing between the MIKK panel and either the \emph{HNI} strain from northern Japan, and the \emph{HSOK} strain from eastern Korea.



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_genome/07_introgression} \caption{\textbf{Figure 2}: ABBA-BABA analysis. \textbf{A}. Phylogenetic tree generated from the Ensembl release 102 50-fish multiple alignment, showing only the medaka lines used in the ABBA-BABA analysis. \textbf{B}. Schema of the comparisons carried out in the ABBA-BABA analysis. \textbf{C}. Circos plot comparing introgression (\(\hat{f}_d\)) between the MIKK panel and either \emph{iCab} (yellow), \emph{HNI} (orange), or \emph{HSOK} (purple), calculated within 500-kb sliding windows using a minimum of 250 SNPs per window.}\label{fig:ABBABABA}
\end{figure}

Based on the genome-wide mean \(\hat{f}_d\), the MIKK panel shares approximately 25\% of its genome with \emph{iCab}, 9\% with \emph{HNI}, and 12\% with \emph{HSOK}. These results provide evidence that the MIKK panel's originating population has more recently interbred with medaka from eastern Korea than with medaka from northern Japan, which is surprising given Japan and Korea are separated by the Korean Strait, whereas the founding populations of the MIKK panel and the \emph{HNI} strain are separated by a ``mere'' mountain range. This supports the findings in Spivakov et al. (2014), where the authors found little evidence of significant interbreeding between southern and northern Japanese medaka since the populations diverged. Although the proportional difference between \emph{HNI} and \emph{HSOK} is small, this further supports the general finding that northern and southern Japanese medaka strains show low levels of interbreeding that may be a result of geographical isolation or genome divergence through a process of speciation (Katsumura et al. 2019).

\hypertarget{ld-decay-sec}{%
\subsection{LD decay}\label{ld-decay-sec}}

Linkage disequilibrium (\textbf{LD}) describes the ``non-independent association of two alleles in a population'' (Uffelmann et al. 2021). It usually occurs between loci that are physically proximate to one another on the chromosome, as a recombination event is less likely to occur between them, causing a tendency for them to be inherited together. Genetic mapping requires variants to segregate in order to determine how they differ in their effects, and so regions with high LD make it difficult to identify the causal variants that reside in those regions, because they tend to be inherited together. If the MIKK panel's LD structure was relatively low compared to humans, it would indicate that one would be able to map variants at a higher resolution, in turn making it easier to identify variants that cause differences in traits of interest. For this purpose I analysed the MIKK panel's allele frequency distribution and LD structure to assess their likely effects on genetic mapping. To remove allele-frequency biases introduced by the presence of sibling lines in the MIKK panel, I used only the MIKK non-sibling call set (see section \ref{non-sib-calls} above).

To assess how accurately one may be able to map genetic variants using the MIKK panel relative to a human dataset, I compared the MIKK panel's minor allele frequency (MAF) distribution and LD structure against that of the 2,504 humans in the 1KG Phase 3 release.({``A Global Reference for Human Genetic Variation''} 2015) To prepare the \textbf{``1KG call set''}, I first downloaded the .vcf files for each autosome from the project's FTP site, then merged them into a single VCF using GATK (McKenna et al. 2010). I then used PLINK (Chang et al. 2015; Purcell and Chang, n.d.) to calculate the minor allele frequencies for all non-missing, biallelic SNPs in both the MIKK non-sibling and 1KG call sets (\emph{N} SNPs = 16,395,558 and 81,042,381 respectively) (\textbf{Fig. \ref{fig:LDdecay}A}). As expected, the 1KG and MIKK panel calls are similarly enriched for low-frequency variants, albeit to a lesser extent in the MIKK panel, which is expected due to its smaller sample size.

To determine the rate of LD decay in the MIKK panel and compare it to that in the 1KG sample, for both the MIKK non-sibling and 1KG call sets, I used PLINK to compute \(r^2\) on each autosome for all pairs of non-missing, biallelic SNPs with MAF \(>\) 0.10 within 10 kb of one another (for 1KG and the MIKK panel respectively \textasciitilde{} 5.5M and \textasciitilde{} 3M SNPs, with a total number of pairwise \(r^2\) observations of 204,152,922 and 146,785,673). I then grouped the \(r^2\) observations for each pair of SNPs based on their distance from one another into non-overlapping bins of 100 bp in length, and calculated the mean \(r^2\) in each of those bins to generate \textbf{Fig. \ref{fig:LDdecay}B} using the mean \(r^2\) and left boundary of each bin.



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_genome/08_ld_decay} \caption{Minor allele frequency distributions and LD decay for biallelic, non-missing SNPs in the 1000 Genomes Phase 3 variant calls (N = 2,504) (1KG), and the MIKK panel Illumina-based calls excluding one of each pair of sibling lines (N = 63), across all autosomes (1KG: chrs 1-22; MIKK: chrs 1-24). \textbf{A}: Histogram of allele frequencies in the 1KG and MIKK panel calls. \textbf{B}: LD decay for each autosome, calculated by taking the mean \(r^2\) of pairs of SNPs with MAF \(>\) 0.1 within non-overlapping 100 bp windows of distance from one another, up to a maximum of 10 kb. LD decays faster on chromosome 2 for the MIKK panel due to its higher recombination rate.}\label{fig:LDdecay}
\end{figure}

Based on the 1KG calls under these parameters, LD decays in humans to a mean \(r^2\) of around 0.2-0.35 at a distance of 10 kb, whereas the MIKK panel reaches this level within 1 kb, with a mean \(r^2\) of 0.3-0.4 at a distance of \textasciitilde100 bp. This implies that when a causal variant is present in at least two lines in the MIKK panel, one may be able to map causal variants at a higher resolution than in humans. I note that LD decays faster in chromosome 2 of the MIKK panel relative to the other chromosomes. This suggests that it has a much higher recombination rate, which is consistent with the linkage map described in Naruse et al. (2000), showing a higher genetic distance per Mb for this chromosome. This higher recombination rate in chromosome 2 may in turn be caused by its relatively high proportion of repeat content (\textbf{Fig. \ref{fig:repeats}A}).



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_genome/20210401_hdrr_repeats_final} \caption{Repeat content in the \emph{HdrR} genome based on RepeatMasker results obtained by Jack Monahan. \textbf{A}. Proportion of repeat content per-chromosome. \textbf{B}. log10 of repeat lengths and counts per repeat class. ``Misc'' includes all repeats assigned to their own specific class, for example ``(GAG)n'' or ``(GATCCA)n''. \textbf{C}. Circos plot showing repeat length (radial axes) by locus (angular axis) and repeat class (track).}\label{fig:repeats}
\end{figure}

\hypertarget{mikksv-sec}{%
\section{Structural variation in the MIKK panel}\label{mikksv-sec}}

As an alternative to the variation pangenome approach described in Leger et al. (2022), I explored the structural variants (SVs) present in 9 of the MIKK panel lines in a reference-anchored manner, similar to many human studies. Differences in SVs between panel lines is another important class of genetic variation that could cause or contribute to significant phenotypic differences. Here I used ONT long-read data obtained for 9 of the 12 selected lines, allowing me to characterise larger SVs in the MIKK panel and to create a more extensive picture of genomic rearrangements compared to available medaka reference genomes. Adrien Leger from the Birney Group at EMBL-EBI first called structural variants using only the ONT long reads, producing a set of structural variants classified into five types: deletions (\textbf{DEL}), insertions (\textbf{INS}), translocations (\textbf{TRA}), duplications (\textbf{DUP}) and inversions (\textbf{INV}). I then ``polished'' the called DEL and INS variants with Illumina short reads to improve their accuracy, using the software \emph{SViper} v2.0.0 (Mehringer 2017). The polishing process filtered out 7.4\% of DEL and 12.8\% of INS variants, and adjusted the breakpoints (i.e.~start and end positions) for 75-77\% of DEL and INS variants in each sample by a mean of 23 bp for the start position, and 33 bp for the end position. This process produced a total of 143,326 filtered SVs.

The 9 ``polished'' samples contained a mean per-sample count of approximately 37K DEL variants (12\% singletons), 29.5K INS variants (14\%), 3.5K TRA variants (9\%), 2.5K DUP (7\%) and 600 INV (7\%) (\textbf{Fig. \ref{fig:SV-main}D}). DEL variants were up to 494 kb in length, with 90\% of unique DEL variants shorter than 3.8 kb. INS variants were only up to 13.8 kb in length, with 90\% of unique INS variants shorter than 2 kb. DUP and INV variants tended to be longer, with a mean length of 19 and 70.5 kb respectively (\textbf{Fig. \ref{fig:SV-main}A}). \textbf{Fig. \ref{fig:SV-main}E} shows the per-sample distribution of DEL variants across the genome. Most large DEL variants over 250 kb in length were common among the MIKK panel lines. A number of large DEL variants appear to have accumulated within the 0-10 Mb region of chromosome 2, which is enriched for repeats in the HdrR reference genome (\textbf{Fig. \ref{fig:repeats}}).

SVs were generally enriched in regions covered by repeats. While only 16\% of bases in the \emph{HdrR} reference were classified as repeats (irrespective of strand), those bases overlapped with 72\% of DEL, 63\% of DUP, 81\% of INV and 35\% of TRA variant regions. However, repeat bases only overlapped with 21\% of INS variants. I also assessed each SV's probability of Loss-of-function Intolerance (\textbf{pLI}) (Lek et al. 2016) by calculating the logarithm of odds (LOD) for the pLI scores of all human orthologous genes overlapping the variant (based on the Exome Aggregation Consortium (ExAC) database (Fuller et al. 2019)) (\textbf{Fig. \ref{fig:SV-main}B,C}). 30,357 out of 134,088 DEL, INS, DUP and INV variants overlapped at least one gene, and 9\% of those had a score greater than 10, indicating a high probability that the SV would cause a loss of function. Two INS variants on chr2 had an outlying LOD score of 57 as a result of overlapping medaka gene ENSORLG00000003411, which has a pLI score of 1 -- the highest intolerance to variants causing a loss of function. This gene is homologous with human genes \emph{SCN1A}, \emph{SCN2A} and \emph{SCN3A}, which encode sodium channels and have been associated with neuronal and sleep disorders. I did not find evidence that longer SVs tended to have a higher probability of causing a loss of function (\textbf{Fig. \ref{fig:SV-main}B}).



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_genome/20210408_sv_main} \caption{Polished SVs in 9 MIKK panel lines sequenced with ONT. DEL: deletion; INS: insertion; TRA: translocation; DUP: duplication; INV: inversion. \textbf{A}. Aggregate log10 counts and lengths of distinct SVs by type, excluding TRA. \textbf{B}. pLI LOD scores in distinct SVs by SV type. \textbf{C}. Histogram of LOD scores by SV type. \textbf{D}. Total and singleton counts of SV types per sample. \textbf{E}. Circos plot showing per-sample distribution and lengths of DEL variants across the genome.}\label{fig:SV-main}
\end{figure}

I compared these polished INS and DEL calls with the high-quality graph-based alternative paths and large-scale deletions, respectively (see section titled \emph{Novel genetic sequences and large-scale insertions and deletions in the MIKK panel} in Leger et al. (2022)). I found that 2 of the 19 regions covered by graph-based alternative paths, and 4 of the 16 regions covered by graph-based deletions, had no SVs that overlapped those regions at all, which suggests they would have been missed entirely when using a reference-anchored approach alone.

With the exception of one alternative path on chromosome 20, the alternative paths were not captured by INS variants, which only covered up to 63\% of the bases in each region, and in many cases substantially less. On the other hand, for 8 of the 16 graph-based deletions, the DEL variants covered at least 85\% of the bases in those regions. The other 8 graph-based deletions were either not at all covered by DEL variants, or only slightly. This indicates, as expected, that the reference-based approach is better at detecting large-scale deletions than alternative paths (``insertions''), but still misses around half of such variants relative to the graph-based approach.

\hypertarget{Pilot-chap}{%
\chapter{Classification of bold/shy behaviours in 5 established inbred medaka strains}\label{Pilot-chap}}

\chaptermark{Pilot behaviour study}

To take advantage of the rich genetic resources offered by medaka, my collaborators (primarily the Loosli Group at KIT) and I developed a behavioural assay to examine the performance of isogenic inbred strains for an inter-strain comparative analysis of bold-shy behaviours in medaka. In this chapter I describe how I used this assay -- which combines both an ``open-field'' and ``novel object'' component with pairs for fish -- to determine: a) whether there were consistent differences in bold-shy behaviours exhibited by 5 established inbred strains of medaka fish (\emph{iCab}, \emph{HdrR}, and \emph{HO5} from southern Japan, and \emph{Kaga} and \emph{HNI} from northern Japan); and b) whether there were consistent differences in bold-shy behaviours exhibited by a given strain (\emph{iCab}) dependent on the strain that it was partnered with. The former was intended to measure the effect of an individual's own genes on its behaviour (a direct genetic effect), and the latter to measure the effect of the genes of the focal fish's tank partner on the behaviour of the focal fish (a social genetic effect). This experimental design allowed me to assess both direct and indirect effects on behaviour simultaneously, and to infer the degree to which variation in bold-shy behaviours is attributable to the differences in an individual's own genetics, the differences in the genetics of their social companions, and stochastic variation. A schema for the experimental plan is shown in \textbf{Figure \ref{fig:behavioural-schema}}.



\begin{figure}
\includegraphics[width=1\linewidth]{figs/pilot/experimental_schema} \caption{Schema for experimental plan. \emph{iCab}-\emph{iCab} pairings are the control condition. To explore direct genetic effects on behaviour, I compare the behaviours of test fishes from different strains, and infer that the differences between strains are caused by the differences in their genetics (direct genetic effects). To explore social genetic effects, I use the same data but turn our focus to the reference fish, and infer that the differences I observe between their behaviours are caused by the differences in their social environments, which are in turn driven by the different genetics in the test fish strains (social genetic effects).}\label{fig:behavioural-schema}
\end{figure}

\hypertarget{results}{%
\section{Results}\label{results}}

\hypertarget{pilot-data-collection}{%
\subsection{Data collection}\label{pilot-data-collection}}

Our behavioural assay is 20 minutes long, comprising two consecutively-run 10-minute components: a) an `open field' component, where the fishes are introduced to the test tank and left to swim around freely; and b) a `novel object' component, where a small black plastic cylinder is added to the tank at the beginning of the second 10-minute period, after which the fishes are again left to swim around freely. The assay is run on pairs of fish. Medaka is a seasonal breeder in which photoperiod has a strong effect on physiology and behaviour (L'opez-Olmeda et al. 2021). In this study I tested fish that were acclimated to summer conditions. To avoid confounding mating behaviours between males and females and associated aggressive interactions between males I used only female fish in all experiments. To increase the throughput of the assay, the test tank was divided into four quadrants with barriers, allowing me to run the assay on four pairs of fish simultaneously. In addition, I used two test tanks situated side-by-side, allowing me to run 8 assays concurrently. The experimental setup used is shown in \textbf{Figure \ref{fig:assay-setup}}.



\begin{figure}
\includegraphics[width=1\linewidth]{figs/pilot/setup_pic_with_frame_110} \caption{\textbf{A}: Experimental setup, with two test boxes side-by-side (denoted as ``L'' for left and ``R'' for right). Each test box contains one test tank, separated by removable barriers into quadrants, allowing for the simultaneous assay of four pairs of fish per test tank. The interior of the box is ambiently illuminated by LED lights, and a camera is suspended over the centre of each test tank to record the videos. \textbf{B}: Four pairs of fishes in a test tank with labelled quadrants (I, II, III, IV) and strains (\emph{iCab} or \emph{HdrR}). C: Paths of \emph{iCab} reference fish and \emph{HdrR} test fish from the video at panel (B) at 110 seconds.}\label{fig:assay-setup}
\end{figure}

Between 11 and 16 June 2019, I assayed a total of 307 pairs of fish, comprising the following counts for each strain pairing: 68 \emph{iCab/iCab}, 60 \emph{iCab/HdrR}, 76 \emph{iCab/HNI}, 47 \emph{iCab/Kaga}, and 56 \emph{iCab/HO5}. The locations of the strains' originating populations were shown in the Introduction chapter, \textbf{Figure \ref{fig:line-locations}}. The fish from the iCab strain was denoted as the ``reference fish'', and was introduced to the test tank first. The ``test fish'' which was either another iCab fish (for the control condition), or a fish from one of the other four strains that were assayed in this experiment (\emph{HdrR}, \emph{HNI}, \emph{Kaga}, \emph{HO5}). The order in which the strains were assayed across the six days was randomly determined prior to the collection of the data. The test tanks were also rinsed between runs to remove any substances released by subjects during previous runs that could influence the behaviour of the subjects that followed.

\hypertarget{tracking}{%
\subsection{Tracking}\label{tracking}}

I split the videos by quadrant and assay component, and tracked the movement of the fishes with the open-access software package \emph{idtrackerai} (Romero-Ferrero et al. 2019). Each individual fish was tracked across at least 85\% of frames in each video, with 78\% of videos tracked for over 99\% of frames. The failure to track both fishes across all frames was usually caused by instances where a fish was situated behind the novel object or quadrant divider, rendering it invisible to the camera. A random selection of 20 videos was reviewed to search for instances of mislabeled fishes due to software errors. I found none, which is consistent with the reliability of the software as reported by its authors. I therefore concluded that such instances would be absent or very rare and thus negligible.

To identify which fish was the reference or test fish, I could generally distinguish between the strains based on their colour, as \emph{iCab} pigmentation is light-brown, \emph{HO5} are light-brown with an orange hue, \emph{HdrR} females are white, and \emph{HNI} and \emph{Kaga} are grey. Where the reference fish was indistinguishable in colour from the test fish, I identified the fish that was introduced to the test tank first, and followed it by eye through to the first frame of the assay. In cases of \emph{iCab}-\emph{iCab} pairings, I randomly assigned the individuals to either the ``reference'' or ``test'' fish. I then used the coordinates of the fishes in each frame to calculate the distance and angle travelled by each fish between a fixed time interval, for example every 0.05 seconds. Distance was calculated between points B\(\rightarrow\)C, and angle was calculated between points A\(\rightarrow\)B\(\rightarrow\)C. I used these distance and angle variables to train a HMM with the \emph{hmmlearn} Python package (\emph{Hmmlearn/Hmmlearn} {[}2014{]} 2022) in order to classify the movements as discrete behaviours or ``modes of movement''.

\hypertarget{effect-of-covariates}{%
\subsection{Effect of covariates}\label{effect-of-covariates}}

I examined the effects of several covariates, including date of assay, time of assay, tank quadrant, and tank side. To achieve this, I calculated the mean speed of individuals in \emph{iCab}-\emph{iCab} pairings (\(N\) = 136) over the course of the entire 20-minute video (including both open field and novel object assays), and ran a multi-way ANOVA with all covariates (\textbf{Figure \ref{fig:covariate-effects}}). I found significant differences for date of assay and and tank quadrant (\(p\) = 0.0139 and 0.0108), but not for time of assay or tank side. This may have been driven by a difference in the way the assay was performed on the first day of the experiment (11 June 2019), where I used a thick fabric sheet to cover the front of the box rather than the wooden doors shown in \textbf{Figure \ref{fig:assay-setup}}, as they were only installed the following day. The greater level of external light and sound permeating through the fabric may have caused the fishes to exhibit slower movement on that first day, and when the data from that day is excluded, the \(p\)-values for date of assay and tank quadrant increase to 0.0477 and 0.0469 respectively.



\begin{figure}
\includegraphics[width=1\linewidth]{figs/pilot/covariate_effects_no-split-by-assay} \caption{Effect of covariates on mean speed of iCab individuals when paired with another iCab over the course of the full video (including both open field and novel object assays). \emph{P}-values were calculated from a multi-way ANOVA with all four covariates included as terms, and adjusted for false discovery rate.}\label{fig:covariate-effects}
\end{figure}

\hypertarget{pilot-hmm-params}{%
\subsection{Choice of time interval and number of HMM states}\label{pilot-hmm-params}}

I then sought to convert the distance and angle of travel measures into a more biologically relevant phenotype. Given the data is a time series, there are several methods available to achieve this, including change detection (Jonietz and Bucher 2018). However, I selected HMMs because they are well-suited for classifying the hidden states that generate stochastic, sequential observations, and because they use expectation maximisation to learn the optimal parameters for a given HMM state. It is nevertheless left to the researcher to decide how many HMM states to use and, in this case, the time interval between which the distance and angle of travel are calculated. To assess which combination of HMM state space and time interval would be best to use, I extracted the distance and angle measurements as input variables, and trained a number of HMMs on the full dataset with different combinations of a defined state space (either 5, 10, 12, 14, 15, 16, 17, 18, or 20 states) and time interval (0.05, 0.08, 0.1, 0.15, 0.2, 0.3, 0.5 and 1 seconds). I then sought to determine the combination of these parameters that would strike the optimal balance between: a) the ability to detect differences between strains; b) a minimisation of model overfitting; and c) the retention of left-right symmetry between states that would facilitate biological interpretation.

To quantify each model's ability to detect differences between strains, I ran a Kruskal-Wallis test (Hollander, Wolfe, and Chicken 2013) for each HMM state, comparing the frequencies of time spent in that state between the five strains, and then summed the statistics over all HMM states in the given model's state space. Next, to quantify the level of overfitting by each model, I carried out a two-fold cross-validation process, where I randomly assigned the videos to either group A or group B. For each group, I computed the concordance between the states assigned by the model that was trained on itself (e.g.~A), and by the model that was trained on the other dataset (e.g.~B), and then calculated the mean of those two measures (concordance between A trained on A and A trained on B; and concordance between B trained on B and B trained on A).

\textbf{Figure \ref{fig:compare-params}} sets out the comparison of different combinations of HMM state space and time interval on these two measures. The horizontal axis represents the level of overfitting (`mean concordance between cross-validated HMM states'), and the vertical axis represents the quantification of differences between strains (`summed Kruskal-Wallis statistic comparing frequency of time spent in each HMM state across medaka strains').



\begin{figure}
\includegraphics[width=1\linewidth]{figs/pilot/compare_params_with_point} \caption{Comparison between different combinations of HMM state space and time interval. Horizontal axis: Mean concordance between states assigned by HMMs through a 2-fold cross-validation process. Vertical axis: Kruskal-Wallis statistic comparing strains based on the proportion of time spent in each HMM state, summed across all states. The size of the points correspond to the interval, in seconds, between which the distance and angle of travel was calculated. The top right of the figure represents the better models, indicated by the pink arrow. The highlighted circle represents the best combination of HMM state space and time interval that also had favourable symmetrical properties (see below).}\label{fig:compare-params}
\end{figure}

The most optimal combination based on these metrics was a state space of 16 and a time interval of 0.08. The distances and angles of travel for each of the 16 states are shown in \textbf{Figure \ref{fig:pilot-polar-asym}}, generated from 10,000 randomly-selected data points (i.e.~movements). Each point represents the radial distance in log\textsubscript{10}(pixels) that a fish travelled from its previous location (B\(\rightarrow\)C), and the polar angle travelled from A\(\rightarrow\)B \(\rightarrow\)C, where A\(\rightarrow\)B is always aligned vertically along the 0-180° axis. For example, a point at 45° far from the pole represents a fast, forward movement to the right, and a point at 260° close to the pole represents a slow, backward movement to the left. The states are ranked by mean distance, so state 1 represents slow movements in any direction, and state 8 represents faster forward movements.



\begin{figure}
\includegraphics[width=1\linewidth]{figs/pilot/polar_all_dge_0.05_16} \caption{Classification of movements by the 16-state HMM, based on distance (in log\textsubscript{10}(pixels)) and angle of travel between a time interval of 0.05 seconds from points B\(\rightarrow\)C for distance, and points A\(\rightarrow\)B\(\rightarrow\)C for angle. Each point represents the distance and angle at point C, and A\(\rightarrow\)B is aligned vertically along the 0-180° radial axis. The figure shows an illustrative 10,000 data points (movements), randomly selected from the full dataset. States are sorted in ascending order by mean distance.}\label{fig:pilot-polar-asym}
\end{figure}

The figure above shows an asymmetry between states 1 and 3, where state 3 appears to have been ``carved'' out of state 1, and to a lesser extent state 5. This asymmetry makes it difficult to interpret their biological relevance, and this phenomenon was consistent across all more-optimal combinations with 15 or more states. Accordingly, for the downstream analysis I selected the combination of 14 states with a 0.08-second interval between time points (\textbf{Figure \ref{fig:pilot-polar}}), because out of the remaining combinations it appeared to optimally balance the level of overfitting and detection of differences between strains.



\begin{figure}
\includegraphics[width=1\linewidth]{figs/pilot/polar_all_dge} \caption{Classification of movements by the 14-state HMM, based on distance (in log\textsubscript{10}(pixels)) and angle of travel between a time interval of 0.08 seconds from points B\(\rightarrow\)C for distance, and points A\(\rightarrow\)B\(\rightarrow\)C for angle. Each point represents the distance and angle at point C, and A\(\rightarrow\)B is aligned vertically along the 0-180° radial axis. The figure shows an illustrative 10,000 data points (movements), randomly selected from the full dataset. States are sorted in ascending order by mean distance.}\label{fig:pilot-polar}
\end{figure}

I next sought to determine whether there was a difference in the proportion of time spent in each state between the 5 inbred strains (implying the presence of direct genetic effects), and between the \emph{iCab} reference fishes depending on the inbred strains of their tank partner (implying the presence of social genetic effects).

\hypertarget{direct-and-social-genetic-effects-on-boldshy-behaviour}{%
\subsection{Direct and social genetic effects on bold/shy behaviour}\label{direct-and-social-genetic-effects-on-boldshy-behaviour}}

\hypertarget{direct-genetic-effects}{%
\subsubsection{Direct genetic effects}\label{direct-genetic-effects}}

To determine whether the test fish strains differed in the proportions of time they spent in each state, I ran separate multi-way ANOVAs for each combination of assay component (open field or novel object) and state (states 1 to 14):

\[
frequency_{A,S} = \beta_{1}(test~fish~strain) + \beta_{2}(date) + \beta_{3}(time) + \beta_{4}(quadrant) + \beta_{5}(tank~side)
\]
where \(A\) = assay component (open field or novel object), and \(S\) = HMM state (1 to 14).

The proportion of time each individual spent within a state was first inverse-normalised within each combination of assay and state, and the date of assay, time of assay, tank quadrant, and tank side were included as covariates. P-values were then adjusted for the False Discovery Rate (\textbf{FDR}).

The states that showed a significant difference between strains are set out in \textbf{Table \ref{tab:pilot-dge-tbl}}. The test fish strains differed significantly in the proportion of time spent in a given state (p \textless{} 0.05, FDR-adjusted) for 11 out of 14 states in the open field assay (4.45x10\textsuperscript{-24} \textless{} \(p\) \textless{} 1.06x10\textsuperscript{-2}), and 7 out of 14 states for the novel object assay (5.53x10\textsuperscript{-16} \textless{} \(p\) \textless{} 4.2x10\textsuperscript{-3}), with the strain of the test fish explaining up to \textasciitilde28\% of the variance in the proportion of time spent in a given state. For some states, there was also a significant -- but weaker -- difference between quadrants, date of assay, and tank side (open field: 1.03x10\textsuperscript{-9} \textless{} \(p\) \textless{} 1.18x10\textsuperscript{-2}; novel object: 1.75x10\textsuperscript{-8} \textless{} \(p\) \textless{} 4.9x10\textsuperscript{-2}).

\begin{table}

\caption{\label{tab:pilot-dge-tbl}Significant differences in the proportion of time spent in each HMM state across test fish strains for the open field and novel object assay components.}
\centering
\begin{tabular}[t]{llll}
\toprule
Assay & State & Variance explained (\%) & p-value (FDR-adjusted)\\
\midrule
\cellcolor{gray!6}{open field} & \cellcolor{gray!6}{1} & \cellcolor{gray!6}{26.62} & \cellcolor{gray!6}{3.60e-22}\\
open field & 2 & 21.94 & 6.18e-18\\
\cellcolor{gray!6}{open field} & \cellcolor{gray!6}{3} & \cellcolor{gray!6}{28.21} & \cellcolor{gray!6}{4.45e-24}\\
open field & 4 & 17.73 & 2.12e-15\\
\cellcolor{gray!6}{open field} & \cellcolor{gray!6}{5} & \cellcolor{gray!6}{7.48} & \cellcolor{gray!6}{2.78e-05}\\
\addlinespace
open field & 6 & 5.60 & 2.48e-04\\
\cellcolor{gray!6}{open field} & \cellcolor{gray!6}{7} & \cellcolor{gray!6}{5.46} & \cellcolor{gray!6}{3.11e-04}\\
open field & 8 & 5.79 & 1.08e-04\\
\cellcolor{gray!6}{open field} & \cellcolor{gray!6}{9} & \cellcolor{gray!6}{7.36} & \cellcolor{gray!6}{8.96e-06}\\
open field & 10 & 3.78 & 1.06e-02\\
\addlinespace
\cellcolor{gray!6}{open field} & \cellcolor{gray!6}{13} & \cellcolor{gray!6}{4.66} & \cellcolor{gray!6}{3.42e-03}\\
novel object & 1 & 17.19 & 7.28e-14\\
\cellcolor{gray!6}{novel object} & \cellcolor{gray!6}{2} & \cellcolor{gray!6}{14.28} & \cellcolor{gray!6}{2.92e-11}\\
novel object & 3 & 19.30 & 5.53e-16\\
\cellcolor{gray!6}{novel object} & \cellcolor{gray!6}{4} & \cellcolor{gray!6}{13.12} & \cellcolor{gray!6}{1.26e-10}\\
\addlinespace
novel object & 6 & 4.12 & 4.20e-03\\
\cellcolor{gray!6}{novel object} & \cellcolor{gray!6}{7} & \cellcolor{gray!6}{8.93} & \cellcolor{gray!6}{1.84e-06}\\
novel object & 9 & 10.68 & 9.87e-08\\
\bottomrule
\end{tabular}
\end{table}

\textbf{Figure \ref{fig:pilot-dge-time}} depicts the time dependence of HMM states over the course of the video, and the regions of the test tank that were most frequently occupied by the different strains. Figures A and H show the HMM states with panels coloured red to indicate the states that were found to significantly differ between test fish strains. \textbf{Figures \ref{fig:pilot-dge-time}B} and \textbf{E} show how each individual test fish, grouped by inbred strain, moved through HMM states throughout the course of the video, with each tile coloured by the state most frequently-occupied by the individual within 2-second intervals. Figures \textbf{\ref{fig:pilot-dge-time}C} and \textbf{F} show the same data, recalculated as densities within each strain, with only the states that showed significant differences between strains in colour, and the remaining states consolidated and coloured grey. There is an especially clear difference between strains in the proportions of time spent in the slowest-moving states (i.e.~states 1 to 3) at the beginning of each assay component, with an increase across all strains in the novel object assay, likely as a consequence of having less room to move, as well as the fear response that the novel object was designed to elicit. The differences are clearest when comparing the southern Japanese medaka strains (\emph{iCab}, \emph{HdrR}, and \emph{HO5}) against the northern Japanese strains (\emph{Kaga} and \emph{HNI}). In addition, I note that \emph{Kaga} tends to spend more time in the fast- and forward-moving state 13 at the beginning of the assay than at other times, which suggests that for that strain, such movements may be the manifestation of a stress response. \textbf{Figures \ref{fig:pilot-dge-time}D} and \textbf{G} show, as densities, the regions of the tank that were most frequently occupied by each strain. Although the northern Japanese strains \emph{Kaga} and \emph{HNI} were similarly fast-moving relative to the southern Japanese strains, in the open field assay component they appear to favour different regions of the tank -- where \emph{HNI} occupied the central regions of the tank with more frequency, \emph{Kaga} tended to prefer swimming along the boundaries of the tank.



\begin{figure}
\includegraphics[width=1\linewidth]{figs/pilot/0.08_14_dge} \caption{Differences between test fish strains in the HMM states they occupied during the open field (\emph{top}) and novel object (\emph{bottom}) assay components. \textbf{A and H}: 14 HMM states with panels coloured red to indicate significant differences between test fish strains in the proportion of time spent in those states during the separate assay components. \textbf{B and E}: Transitions between HMM states across time for each individual test fish, grouped by strain. Tiles are coloured by the state most frequently occupied by each fish within 2-second intervals. \textbf{C and F}: Densities within each strain for the occupation of states that significantly differed between strains (colour), with other states consolidated (grey). \textbf{D and G}: Densities of the test tank locations occupied by each strain, calculated within 900 grid points (30x30).}\label{fig:pilot-dge-time}
\end{figure}

\clearpage

\hypertarget{pilot-sge}{%
\subsubsection{Social genetic effects}\label{pilot-sge}}

To determine whether the \emph{iCab} reference fishes altered their behaviour depending on the inbred strain of their tank partner, I carried out the same analysis and model as above using only data from the \emph{iCab} reference fishes. The states that showed a significant difference are set out in \textbf{Table \ref{tab:pilot-sge-tbl}}. The \emph{iCab} reference fishes differed significantly in the proportion of time they spent in a given state depending on the strain of their tank partner (\(p\) \textless{} 0.05, FDR-adjusted) for 5 out of 14 states in the open field assay (2.04x10\textsuperscript{-6} \textless{} \(p\) \textless{} 2.11x10\textsuperscript{-2}), and 7 out of 14 states for the novel object assay (4.76x10\textsuperscript{-7} \textless{} \(p\) \textless{} 4.06x10\textsuperscript{-2}). The strain of the tank partner explained up to \textasciitilde9\% of the variance in the proportion of time the \emph{iCab} reference spent in a given state. Full tables for all states and variables are provided in the Supplementary Material.

\begin{table}

\caption{\label{tab:pilot-sge-tbl}Significant differences in the proportion of time spent in each HMM state by iCab reference fishes depending on the strain of their tank partner during the open field and novel object assay components.}
\centering
\begin{tabular}[t]{llll}
\toprule
Assay & State & Variance explained (\%) & p-value (FDR-adjusted)\\
\midrule
\cellcolor{gray!6}{open field} & \cellcolor{gray!6}{1} & \cellcolor{gray!6}{8.52} & \cellcolor{gray!6}{5.88e-06}\\
open field & 2 & 9.09 & 2.04e-06\\
\cellcolor{gray!6}{open field} & \cellcolor{gray!6}{3} & \cellcolor{gray!6}{7.97} & \cellcolor{gray!6}{1.77e-05}\\
open field & 5 & 3.92 & 4.29e-03\\
\cellcolor{gray!6}{open field} & \cellcolor{gray!6}{12} & \cellcolor{gray!6}{2.97} & \cellcolor{gray!6}{2.11e-02}\\
\addlinespace
novel object & 1 & 7.94 & 8.73e-06\\
\cellcolor{gray!6}{novel object} & \cellcolor{gray!6}{2} & \cellcolor{gray!6}{9.38} & \cellcolor{gray!6}{4.76e-07}\\
novel object & 3 & 8.73 & 1.58e-06\\
\cellcolor{gray!6}{novel object} & \cellcolor{gray!6}{6} & \cellcolor{gray!6}{4.76} & \cellcolor{gray!6}{1.23e-03}\\
novel object & 8 & 2.80 & 4.06e-02\\
\addlinespace
\cellcolor{gray!6}{novel object} & \cellcolor{gray!6}{10} & \cellcolor{gray!6}{3.81} & \cellcolor{gray!6}{8.43e-03}\\
\bottomrule
\end{tabular}
\end{table}

I observe that the proportions of time the \emph{iCab} reference fishes spent in different states do differ based on the strains of their tank partner, and that their behavioural patterns tend to reflect those of the test fish strains they are paired with (\textbf{Figure \ref{fig:pilot-sge-time}}). Thus, the \emph{iCab} reference fishes spend less time in the slower-moving states when in the presence of the faster-moving northern Japanese strains \emph{Kaga} and \emph{HNI}, which in turn shows that the test fishes are transmitting their behaviours to some degree to their \emph{iCab} tank partners.The differences in the behaviours of \emph{iCab} reference fishes based on the strain of their tank partners are not as large as those between the test fish strains. However, unlike what was observed with the test fishes, here there is a greater number of states that show significant differences during the novel object component of the assay compared to the open field component. This suggests that when movement is restricted, or when in the presence of a potential threat, the behaviour of the test fish tank partner has more of an influence on the \emph{iCab} reference fish's behaviour than otherwise.



\begin{figure}
\includegraphics[width=0.78\linewidth]{figs/pilot/0.08_14_sge} \caption{Differences between HMM states occupied by the reference fish when paired with different test fish strains during the open field (top) and novel object (bottom) assay components. \textbf{A and H}: 14 HMM states with panels coloured red to indicate significant differences between the reference fishes under different strain-pairings in the proportion of time spent in those states during the separate assay components. The HMM states are the same as those in \textbf{Figure \ref{fig:pilot-dge-time}}, but coloured with a different palette. \textbf{B and E}: Transitions between HMM states across time for each individual \emph{iCab} reference fish, grouped by the strain of its tank partner. Tiles are coloured by the state most frequently occupied by each fish within 2-second intervals. \textbf{C and F}: Densities within each strain-pairing for the occupation of states that significantly differed between strain-pairings (colour), with other states consolidated (grey). \textbf{D and G}: Densities of the test tank locations occupied by the iCab reference fishes when paired with different strain, calculated within 900 grid points (30x30).}\label{fig:pilot-sge-time}
\end{figure}

Finally, to develop a metric to quantify the degree to which the \emph{iCab} reference fishes' behaviour is influenced by the strain of its tank partner, I calculated the proportions of time that the pairs of fish spent simultaneously occupying the same HMM state. \textbf{Figure \ref{fig:pilot-cooc-heat}} shows that when paired with the slowest-moving \emph{HdrR} strain, the \emph{iCab} reference fishes spent more time co-occupying the slow- and forward-moving state 2, whereas they tended to co-occupy the fast- and forward-moving state 11 when paired with the faster-moving northern \emph{Kaga} and \emph{HNI} strains. When paired with another \emph{iCab}, they preferred to co-occupy the slow-moving and pan-directional state 4. For each combination of assay component and state, I then ran a Kruskal-Wallis test to determine whether there were differences in the frequencies of state co-occupancy under different strain pairings (\(p\) \textless{} 0.05, FDR-adjusted), and found significant differences for states 1 to 6 for the open field component (1.1x10\textsuperscript{-8} \textless{} \(p\) \textless{} 4.1x10\textsuperscript{-2}), and states 1 to 4 and 7 for the novel object component (1.6x10\textsuperscript{-6} \textless{} \(p\) \textless{} 2.5x10\textsuperscript{-2}). The heatmaps show that \emph{iCab} tends to co-occupy state 4 at the highest frequency when paired with another \emph{iCab}, which suggests that this pan-directional, slow-moving state is a more comfortable or natural state for \emph{iCab} to occupy, particularly when under stress.



\begin{figure}
\includegraphics[width=1\linewidth]{figs/pilot/0.08_14_cooc_heatmap} \caption{Frequency of HMM state co-occupancy between pairs of fish, calculated across all all videos per strain-pairing. MAKE TEXT BIGGER}\label{fig:pilot-cooc-heat}
\end{figure}

\hypertarget{Somite-chap}{%
\chapter{Genetic loci associated with somite development periodicity}\label{Somite-chap}}

\chaptermark{Somite development periodicity}

Somites are the earliest primitive segmental structures that form from presomatic mesoderm cells (\textbf{PSM}) in the embryo (Kim et al. 2011), later differentiating into vertebrae, ribs, and skeletal muscles. Somite formation occurs rhythmically and sequentially, with the time between the formation of each pair of somites referred to as the ``period''. The period of somite formation varies greatly between species (Matsuda et al. 2020), and is controlled by a molecular oscillator, known as the `segmentation clock', which drives waves of gene expression in developmental pathways.

It is not fully understood how the phase waves of the segmentation clock are initially established (Falk et al. 2022). To identify the genetic basis of these phase waves, our collaborators Ali Seleit and Alexander Aulehla at EMBL-Heidelberg used a CRISPR-Cas9 knock-in approach (Seleit, Aulehla, and Paix 2021) to establish a medaka \emph{Cab} strain with an endogenous, fluorescing reporter gene (Her7-Venus) for the oscillation signalling pathway. This method allows them to image somite formation and extract quantitative measures for segmentation clock dynamics. They determined that the southern Japanese \emph{Cab} strain and the northern Japanese \emph{Kaga} strain have divergent somite periodicity, where \emph{Kaga}'s tends to be faster, and \emph{Cab}'s slower (\textbf{Figure \ref{fig:F0-Cab-Kaga-HdrR}}), and selected them as the parental strains in an F\textsubscript{2} cross. This chapter sets out my bioinformatic contributions to the project by identifying the genetic variations associated with differences in somite periodicity.

\hypertarget{somite-phenotype}{%
\section{Phenotypes of interest}\label{somite-phenotype}}

\hypertarget{somite-development-period}{%
\subsection{Somite development period}\label{somite-development-period}}

\textbf{Figure \ref{fig:ali-somite-period-lines}} shows the period data generated by pyBOAT for this study, for 100 illustrative F\textsubscript{2} samples over 300 minutes. The same data can be represented by boxplots as shown in \textbf{Figure \ref{fig:ali-somite-period-box}}. I experimented with using the F\textsubscript{2} individuals' mean period and period intercept as the phenotype of interest. The two measures are highly correlated (\(Pearson's~r =\) 0.84, \(p\) \textless{} 2.2 x 10\textsuperscript{-16}), so after displaying the distributions for both measures in Figure \ref{fig:somite-phenos}, I proceed to only discuss the analysis of period intercept, as it would appear to potentially be more robust to the changes in slope that can be observed in Figure \ref{fig:ali-somite-period-lines}.



\begin{figure}
\includegraphics[width=1\linewidth]{figs/somites/ali_period_lines_100fish300mins} \caption{PyBOAT results for 100 illustrative F\textsubscript{2} samples, showing the period length in minutes over the course of 300 minutes. Period tends to increase over time, meaning that as the embryo develops, each successive somite takes longer to form. Figure generated by Ali Seleit.}\label{fig:ali-somite-period-lines}
\end{figure}



\begin{figure}
\includegraphics[width=1\linewidth]{figs/somites/ali_F2_mean_period} \caption{Period measurements for 70 F\textsubscript{2} individuals displayed as boxplots with each individual's median and interquartile range. Figure generated by Ali Seleit.}\label{fig:ali-somite-period-box}
\end{figure}

\clearpage

\hypertarget{unsegmented-presomitic-mesoderm-area-psm}{%
\subsection{Unsegmented presomitic mesoderm area (PSM)}\label{unsegmented-presomitic-mesoderm-area-psm}}

In the proceeding analyses, I also included a second phenotype of interest: the total area of the unsegmented tissue at the stage where 10-11 somites had been formed (\textbf{PSM area}). As the measure is simply based on the total number of pixels covered by the embryo object, I considered it to be potentially more robust than the period measurements, and therefore included it as a type of positive control for the genetic association analyses on the period phenotype. The measurements for PSM area comparing F\textsubscript{0} \emph{Cab} and \emph{Kaga} strains are set out in \textbf{Figure \ref{fig:ali-psm-F0}}.



\begin{figure}
\includegraphics[width=0.6\linewidth]{figs/somites/ali_PSM_Cab_Kaga} \caption{Measurements of unsegmented PSM area in pixels for the F\textsubscript{0} individuals from the \emph{Cab} strain (\(N = 19\)) and \emph{Kaga} strain (\(N = 10\)). \emph{Kaga} tends to have a smaller PSM than \emph{Cab}. Figure generated by Ali Seleit.}\label{fig:ali-psm-F0}
\end{figure}

\clearpage

\hypertarget{comparisons-between-f0-f1-and-f2-generations}{%
\subsection{\texorpdfstring{Comparisons between F\textsubscript{0}, F\textsubscript{1} and F\textsubscript{2} generations}{Comparisons between F0, F1 and F2 generations}}\label{comparisons-between-f0-f1-and-f2-generations}}

The distributions across the F\textsubscript{0}, F\textsubscript{1} and F\textsubscript{2} generations are unexpected (\textbf{Figure \ref{fig:somite-phenos}}). I rather expected to observe an F\textsubscript{2} distribution with a similar median to the F\textsubscript{1}, and a variance that spanned across the extremes of the F\textsubscript{0} strains. Instead, I observed that for the period phenotypes, the F\textsubscript{2} generation had a mean that was slightly higher than the median of the higher-period F\textsubscript{0} \emph{Cab} strain, and many F\textsubscript{2} samples exceed the period values in those F\textsubscript{0} samples. Our collaborators assured me that these observations were unlikely to be caused by technical issues. A possible biological explanation of this is that there are more genetic combinations that slow down the clock rather than speed it up (Sanchez et al. 2022; Schröter and Oates 2010). This phenomenon could be exacerbated by the fact that the \emph{Cab} and \emph{Kaga} strains originate from different Japanese medaka populations (southern and northern respectively) that are understood to be at the point of speciation (see Chapter \ref{MIKK-genomes-chap}), so this slower period may be driven by a biological incompatibility between their genomes in cases where they do not have a complete chromosome from each parent (as the F\textsubscript{1} generation does). I nevertheless proceeded with the genetic analysis with a view to potentially discovering the reason for this unusual distribution.



\begin{figure}
\includegraphics[width=1\linewidth]{figs/somites/phenotypes} \caption{Comparisons between the F\textsubscript{0}, F\textsubscript{1} and F\textsubscript{2} generations for period intercept, mean, and unsegmented PSM area. Here, the F\textsubscript{0} only includes \emph{Cab} individuals. \textbf{A}: period intercept. \textbf{B}: period mean. \textbf{C}: unsegmented PSM area. \(P\)-values are derived from Kruskal-Wallis tests comparing the F\textsubscript{2} individuals across microscopes. For PSM area, data was only collected for the F\textsubscript{2} generation.}\label{fig:somite-phenos}
\end{figure}

Another important issue to note is that the F\textsubscript{2} individuals were sequenced using different microscopes of the same model (Zeiss LSM 780) but with different temperature control units and incubator boxes, denoted as `AU' and `DB'.\footnote{`AU' for the Aulehla Lab microscope, and `DB' for EMBL-Heidelberg's Developmental Biology Unit microscope.} Our collaborators noticed that there was a difference between the microscopes in their temperatures of 0.7-0.8°C, translating to a 4-minute difference in the F\textsubscript{2} means for the period intercept measure (Kruskal-Wallis = 177.97, \(p\) = 1.34 x 10\textsuperscript{-40}), and a 3.5-minute difference in the F\textsubscript{2} means for the period mean measure (Kruskal-Wallis = 141.79, \(p\) = 1.08 x 10\textsuperscript{-32}). This difference would need to be accounted for in the downstream analysis through either adjusting the phenotype prior to running the genetic association model, or by including microscope as a covariate in the model. No significant difference was found for the PSM area.

\hypertarget{inverse-norm-sec}{%
\subsubsection{Inverse-normalisation}\label{inverse-norm-sec}}

To resolve this difference between microscopes for the period intercept data, I elected to transform it for the F\textsubscript{2} generation by ``inverse-normalising'' the period intercept within each microscope (\textbf{Figure \ref{fig:invnorm-intercept}}), and used this transformed phenotype for the downstream analysis. Inverse-normalisation is a rank-based normalisation approach which involves replacing the values in the phenotype vector with their rank (where ties are averaged), then converting the ranks into a normal distribution with the quantile function (Wichura 1988). The inverse-normalisation function I used for this analysis is set out in the following \passthrough{\lstinline!R!} code:

\definecolor{codegreen}{rgb}{0,0.6,0}
\definecolor{codegray}{rgb}{0.5,0.5,0.5}
\definecolor{codepurple}{rgb}{0.58,0,0.82}
\definecolor{backcolour}{rgb}{0.95,0.95,0.92}
\lstdefinestyle{mystyle}{
    backgroundcolor=\color{backcolour},   
    commentstyle=\color{codegreen},
    keywordstyle=\color{magenta},
    numberstyle=\tiny\color{codegray},
    stringstyle=\color{codepurple},
    basicstyle=\ttfamily\footnotesize,
    breakatwhitespace=false,         
    breaklines=true,                 
    captionpos=b,                    
    keepspaces=true,                 
    numbers=left,                    
    numbersep=5pt,                  
    showspaces=false,                
    showstringspaces=false,
    showtabs=false,                  
    tabsize=2
}
\lstset{style=mystyle}
\begin{lstlisting}[language=R, caption=Inverse-normalisation function ]
invnorm = function(x) {
  res = rank(x)
  # The arbitrary 0.5 value is added to the denominator below 
  # to avoid `qnorm()` returning 'Inf' for the last-ranked value
  res = qnorm(res/(length(res)+0.5))
  return(res)
}
\end{lstlisting}



\begin{figure}
\includegraphics[width=1\linewidth]{figs/somites/invnorm_intercept} \caption{Comparison of the period intercept phenotype data for the F\textsubscript{2} generation before (\textbf{A}) and after (\textbf{B}) inverse-normalisation, with vertical lines marking the mean of each group.}\label{fig:invnorm-intercept}
\end{figure}

\clearpage

\hypertarget{genetic-sequencing-data}{%
\section{Genetic sequencing data}\label{genetic-sequencing-data}}

Our collaborators extracted DNA from the F\textsubscript{0}, F\textsubscript{1}, and F\textsubscript{2}, and sequenced the F\textsubscript{0} and F\textsubscript{1} samples with the Illumina platform at high coverage (\textasciitilde26x for \emph{Cab} and \textasciitilde29x for \emph{Kaga}), as measured by samtools (Danecek et al. 2021). \textbf{Figure \ref{fig:F0-coverage}} sets out the mean sequencing depth within each chromosome and across the whole genome for the \emph{Cab} and \emph{Kaga} F\textsubscript{0} samples. Our collaborators then sequenced the F\textsubscript{2} samples at low coverage (\textasciitilde1x), which would be sufficient to map their genotypes back to the genotypes of their parental strains (see section \ref{somite-f2-sequencing} below for further details).



\begin{figure}
\includegraphics[width=1\linewidth]{figs/somites/F0_coverage} \caption{Mean sequencing depth per chromosome for \emph{Cab} and \emph{Kaga} F\textsubscript{0} strains, with genome-wide mean depth across all chromosomes shown under the subtitles.}\label{fig:F0-coverage}
\end{figure}

\clearpage

\hypertarget{f0-homozygosity-and-f1-heterozygosity}{%
\section{\texorpdfstring{F\textsubscript{0} homozygosity and F\textsubscript{1} heterozygosity}{F0 homozygosity and F1 heterozygosity}}\label{f0-homozygosity-and-f1-heterozygosity}}

Before proceeding to map the F\textsubscript{2} sequences to the genotypes of the F\textsubscript{0} generation, I first investigated the levels of homozygosity in the F\textsubscript{0} \emph{Cab} and \emph{Kaga} strains, as this would affect our ability to accurately call the F\textsubscript{2} generation. That is to say, for regions where either F\textsubscript{0} parent is consistently heterozygous, it would be difficult to determine the parent from which a particular F\textsubscript{2} individual derived its chromosomes at that locus. I therefore aligned the high-coverage sequencing data for the F\textsubscript{0} \emph{Cab} and \emph{Kaga} strains to the medaka \emph{HdrR} reference (Ensembl release 104, build ASM223467v1) using \emph{BWA-MEM2} (Vasimuddin et al. 2019), sorted the aligned .sam files, marked duplicate reads, and merged the paired reads with picard ({``Picard Toolkit''} 2019), and indexed the .bam files with \emph{Samtools} (Li et al. 2009).

To call variants, I followed the GATK best practices (to the extent they were applicable) (McKenna et al. 2010; DePristo et al. 2011; Van der Auwera and O'Connor 2020) with GATK's HaplotypeCaller and GenotypeGVCFs tools (Poplin et al. 2018), then merged all calls into a single .vcf file with picard ({``Picard Toolkit''} 2019). Finally, I extracted the biallelic calls for \emph{Cab} and \emph{Kaga} with bcftools (Danecek et al. 2021), counted the number of SNPs within non-overlapping, 5-kb bins, and calculated the proportion of SNPs within each bin that were homozgyous.

\textbf{Figure \ref{fig:somite-f0-cab}} is a circos plot generated with circlize (Z. Gu et al. 2014) for the \emph{Cab} F\textsubscript{0} strain used in this experiment, featuring the proportion of homozygous SNPs per 5-kb bin (green), and the total number of SNPs in each bin (yellow). As expected for a strain that has been inbred for over 10 generations, the mean homozygosity for this strain is high, with a mean proportion of homozygosity across all bins of 83\%.



\begin{figure}
\includegraphics[width=1\linewidth]{figs/somites/Cab} \caption{Proportion of homozygous SNPs within 5 kb bins in the \emph{Cab} F\textsubscript{0} generation genome (green), and number of SNPs in each bin (yellow).}\label{fig:somite-f0-cab}
\end{figure}

\clearpage

However, the levels of homozygosity in the \emph{Kaga} strain used in this experiment was far lower, with a mean homozygosity across all bins of only 31\% (\textbf{Figure \ref{fig:somite-f0-kaga}}). This was a surprise, as it is an established strain that has been extant for decades, and we therefore expected the level of homozygosity to be commensurate with that observed in the \emph{Cab} strain. An obvious exception is chr22, for which \emph{Kaga} appears to be homozygous across its entire length.



\begin{figure}
\includegraphics[width=1\linewidth]{figs/somites/Kaga} \caption{Proportion of homozygous SNPs within 5 kb bins in the \emph{Kaga} F\textsubscript{0} generation genome (red), and number of SNPs in each bin (yellow).}\label{fig:somite-f0-kaga}
\end{figure}

To determine whether the low levels of observed homozygosity in Kaga was affected by its alignments to the southern Japanese \emph{HdrR} reference, we also aligned the F\textsubscript{0} samples to the northern Japanese \emph{HNI} reference (\textbf{Figure \ref{fig:somite-f0-kaga-hni}}. This did not make differences to the levels of observed homozygosity in either sample, which gave us confidence that the low homozygosity observed in \emph{Kaga} was not driven by reference bias. I understand from our collaborators that the low homozygosity of this \emph{Kaga} individual must have resulted from the strain having been contaminated at some stage by breeding with a different inbred strain prior to when they received the individuals.



\begin{figure}
\includegraphics[width=1\linewidth]{figs/somites/Kaga_HNI} \caption{Proportion of homozygous SNPs within 5 kb bins in the \emph{Kaga} F\textsubscript{0} generation genome when aligned to the \emph{HNI} reference (red), and number of SNPs in each bin (yellow).}\label{fig:somite-f0-kaga-hni}
\end{figure}

\hypertarget{f1-homozygosity}{%
\section{\texorpdfstring{F\textsubscript{1} homozygosity}{F1 homozygosity}}\label{f1-homozygosity}}

I next examined the level of heterozygosity in the F\textsubscript{1} generation from the \emph{Cab}-\emph{Kaga} cross. \textbf{Figure \ref{fig:somite-f1}} shows the level of heterozygosity across the genome of the F\textsubscript{1} hybrid in brown measured by the proportion of heterozygous SNPs within 5-kb bins (brown), and the number of SNPs in each bin (yellow). Approximately half the chromosomes show inconsistent heterozygosity, with a mean heterozygosity across all bins of 67\%. This lower level of apparent heterozygosity than expected was likely caused by the low levels of homozygosity in the \emph{Kaga} F\textsubscript{0} parent.



\begin{figure}
\includegraphics[width=1\linewidth]{figs/somites/F1} \caption{Proportion of heterozygous SNPs within 5 kb bins in the \emph{Cab}-\emph{Kaga} F\textsubscript{1} cross (brown), and number of SNPs in each bin (yellow).}\label{fig:somite-f1}
\end{figure}

For the purpose of mapping the F\textsubscript{2} sample sequences to the genomes of their parental strains, I selected only biallelic SNPs that were homozygous-divergent in the F\textsubscript{0} generation (i.e.~homozygous reference allele in \emph{Cab} and homozygous alternative allele in \emph{Kaga} or vice versa) \emph{and} heterozygous in the F\textsubscript{1} generation. The number of SNPs that met these criteria per chromosome are set out in \textbf{Figure \ref{fig:snp-counts-per-chrom}}. The strong homozygosity of \emph{Kaga} on chr22 is likely responsible for the much greater number of loci on that chromosome that can be used for calling genoytpes in the F\textsubscript{2} generation, and highlights the importance of the parental strains being highly homozygous when used in experimental crosses such as this.



\begin{figure}
\includegraphics[width=1\linewidth]{figs/somites/snp_counts_per_chr_hdrr} \caption{Number of SNPs per chromosome that were homozygous-divergent in the F\textsubscript{0} \emph{Cab} and \emph{Kaga} generations, and heterozygous in the F\textsubscript{1} generation.}\label{fig:snp-counts-per-chrom}
\end{figure}

\clearpage

\hypertarget{somite-f2-sequencing}{%
\section{\texorpdfstring{F\textsubscript{2} genotyping}{F2 genotyping}}\label{somite-f2-sequencing}}

To maximise the efficiency of our sequencing runs, our collaborators ``shallow-sequenced'' the F\textsubscript{2} generation with the short-read Illumina platform at a depth of \textasciitilde1x. We then aligned these sequences to the \emph{HdrR} reference with BWA-MEM2 (Vasimuddin et al. 2019), sorted the reads and marked duplicates with Picard ({``Picard Toolkit''} 2019), then indexed the resulting BAM files with samtools (Danecek et al. 2021). Genotyping these shallow sequences with the same method as used for the high-coverage sequences for the F\textsubscript{0} and F\textsubscript{1} generation would be inappropriate. We therefore used a different method whereby we used \emph{bam-readcount} (Khanna et al. 2022) to count the reads that supported either the \emph{Cab} or the \emph{Kaga} allele for all SNPs that met the criteria described above in section \ref{f1-homozygosity}, summed the read counts within 5 kb blocks, and calculated the frequency of reads within each bin that supported the \emph{Kaga} allele. This generated a value for each bin between 0 and 1, where 0 signified that all reads within that bin supported the \emph{Cab} allele, and 1 signified that all reads within that bin supported the \emph{Kaga} allele. Bins containing no reads were imputed with a value of 0.5.

I then used these values for all F\textsubscript{2} individuals as the input to a Hidden Markov Model (HMM) with the software package \emph{hmmlearn} (\emph{Hmmlearn/Hmmlearn} {[}2014{]} 2022), which I applied to classify each bin as one of three states, with state 0 corresponding to homozygous-\emph{Cab}, 1 corresponding to heterozygous, and 2 corresponding to homozygous-\emph{Kaga}. Across each chromosome of every sample, the output of the HMM was expected to produce a sequence of states. Based on previous biological knowledge that crossover events occur on average less than once per chromosome (Haenel et al. 2018) (see \textbf{Figure \ref{fig:zebrafish-co-per-chrom}} for the average crossover rates per chromosome in zebrafish), I expected to observe the same state persisting for long stretches of the chromosome, only changing to another state between 0 and 3 times, and rarely more.



\begin{figure}
\includegraphics[width=1\linewidth]{04-somites_files/figure-latex/zebrafish-co-per-chrom-1} \caption{Crossovers per chromosome based on data provided in Haenel et al. (2018), where ``crossovers per chromosome'' for each chromosome \(c\) was calculated by \(\frac{crossover~rate_{c}(cM / Mb) \times length_{c}(Mb)} {100}\). The medaka genome is shorter in length than zebrafish genome (\textasciitilde800 Mb compared to \textasciitilde1,300 Mb), which according to the authors would suggest that medaka likely has a higher average crossover rate than what is presented in this figure.}\label{fig:zebrafish-co-per-chrom}
\end{figure}

\textbf{Figure \ref{fig:hmm-scat}} shows how adjusting the HMM parameters changed the called genotypes for 10 F\textsubscript{2} samples on chromosome 18, by way of an example. Allowing the HMM to train itself for the transition probabilities and emission variances, the HMM produced an apparently noisy output (\textbf{Figure \ref{fig:hmm-scat}A}). Fixing the transition probabilities to make it very likely for a state to transition back to itself rather than to another state did not appear to change the output (\textbf{Figure \ref{fig:hmm-scat}A}). However, increasing the Gaussian emission variances created the ``blocks'' that I expected, and the higher the emission variances, the ``tighter'' the blocks (\textbf{Figure \ref{fig:hmm-scat}C-F}).



\begin{figure}
\includegraphics[width=1\linewidth]{figs/somites/scatter_collage} \caption{HMM states called for each bin across chr18 for 10 F\textsubscript{2} samples, to illustrate the effect of changing the HMM's transition probabilities and emission variances. States 0, 1, and 2 correspond to homozygous \emph{Cab}, heterozygous, and homozygous \emph{Kaga}. Each point represents a 5-kb bin. Y-axis is the proportion of reads within each bin that align to the \emph{Kaga} allele. X-axis is the bp location of the start of each bin. \textbf{A}: Standard HMM with all model parameters trained on the data. \textbf{B}. HMM with fixed transition probabilities of 0 \(\rightarrow\) 0 or 1 \(\rightarrow\) 1 or 2 \(\rightarrow\) 2 = 0.999; 0 \(\rightarrow\) 1 or 2 \(\rightarrow\) 1 = 0.00066; 0 \(\rightarrow\) 2 or 2 \(\rightarrow\) 0 = 0.000333; 1 \(\rightarrow\) 0 or 1 \(\rightarrow\) 2 = 0.0005. \textbf{C}-\textbf{F} retain those transition probabilities but with different fixed emission variances of 0.01 (\textbf{C}), 0.33 (\textbf{D}), 0.8 (\textbf{E}), and 1 (\textbf{F}).}\label{fig:hmm-scat}
\end{figure}

\clearpage

I used these genotype-block calls to generate the recombination karyoplot shown in \textbf{Figure \ref{fig:karyo-wi-missing}}, with homozygous-\emph{Cab} blocks in green, heterozoygous blocks in navy blue, and homozygous \emph{Kaga} blocks in red. Missing calls are blank, where the vertical blank lines indicate that the region could not be called for any F\textsubscript{2} individuals, likely due to an insufficient number of informative SNPs residing in those 5-kb blocks; and horizontal blank lines indicate that the sample could not be called, likely due to low sequencing coverage for that sample.



\begin{figure}

{\centering \includegraphics[height=0.95\textheight]{figs/somites/karyoplot_wi_missing_cropped} 

}

\caption{Recombination blocks in 622 F\textsubscript{2} samples based on the ratio of reads mapping to either the \emph{Cab} or \emph{Kaga} allele within 5-kb bins, with homozygous-\emph{Cab} blocks in green, heterozoygous blocks in navy blue, and homozygous \emph{Kaga} blocks in red. Most blocks show 0-2 crossover events, as expected, with some regions showing higher numbers of crossovers interpreted as noise. Unfilled regions are those with no state called by the HMM due to a lack of reads mapping to SNPs within those 5-kb bins.}\label{fig:karyo-wi-missing}
\end{figure}

In the downstream analysis, I excluded the 22 samples that showed poor coverage across the genome. For the remaining samples, I ``filled'' the bins with missing genotypes based on the call of the previous called bin, or if unavailable (e.g.~the missing bin was at the start of the chromosome), then the next called bin (\textbf{Figure \ref{fig:karyo-no-missing}}; note that this figure retains the low-coverage samples (horizontal blank lines) that were excluded from further analysis to allow for a direct comparison with \textbf{Figure \ref{fig:karyo-wi-missing}}). I used these filled genotype calls for the association tests described below in section \textbf{\ref{somite-assoc-tests}}. As a rough way to estimate the accuracy of this genotyping method, we checked the genotypes called by the HMM for the reporter region on chr16 \textasciitilde28.7Mb against our collaborators' manual recording of reporter gene counts based on the intensity of the \emph{Her7}-Venus reporter's fluorescence. The confusion matrix is set out in \textbf{Table \ref{tab:reporter-conc-tbl}}, showing that 83\% of genotypes called by the HMM were consistent with the reporter fluorescence.

\begin{table}

\caption{\label{tab:reporter-conc-tbl}Confusion matrix for reporter genotype as called by the HMM, and as inferred from the fluorescence brightness.}
\centering
\begin{tabular}[t]{llr}
\toprule
HMM state & Reporter phenotype & Count\\
\midrule
\cellcolor{gray!6}{0} & \cellcolor{gray!6}{0} & \cellcolor{gray!6}{252}\\
0 & 1 & 91\\
\cellcolor{gray!6}{1} & \cellcolor{gray!6}{0} & \cellcolor{gray!6}{9}\\
1 & 1 & 245\\
\cellcolor{gray!6}{2} & \cellcolor{gray!6}{1} & \cellcolor{gray!6}{1}\\
\addlinespace
Total & - & 598\\
\bottomrule
\end{tabular}
\end{table}

These karyoplots show interesting recombination patterns for several chromosomes. Given the F\textsubscript{2} individuals were selected for the reporter gene on chr16, as expected, there appears to be a strong strong skew towards those genotypes across the whole chromosome. On chr3, most samples are homozygous-\emph{Cab} for the second half of the chromosome, with a consistent breakpoint around \textasciitilde22 Mb. However, the final fifth of samples which show a different recombination pattern. The samples are sorted based on the order that they were phenotyped and sequenced, so this difference could have been caused by their being generated from different F\textsubscript{1} individuals with distinct haplotypes.



\begin{figure}
\includegraphics[width=1\linewidth]{figs/somites/karyoplot_no_missing_cropped} \caption{Recombination blocks in 622 F\textsubscript{2} samples based on the ratio of reads mapping to either the \emph{Cab} or \emph{Kaga} allele within 5-kb bins, with homozygous-\emph{Cab} blocks in green, heterozoygous blocks in navy blue, and homozygous \emph{Kaga} blocks in red. Most blocks show 0-2 crossover events, as expected, with some regions showing higher numbers of crossovers interpreted as noise. Bins with missing genotypes were ``filled'' based on the call of the previous called bin, or if unavailable (e.g.~the missing bin was at the start of the chromosome), then the next called bin.}\label{fig:karyo-no-missing}
\end{figure}

\textbf{Figure \ref{fig:prop-sites-total}} shows the proportion of 5-kb bins called as either homozygous-\emph{Cab}, heterozygous, or homozygous-\emph{Kaga} within each F\textsubscript{2} sample (points). The ordinary expectation for the ratios would be 0.25, 0.5, and 0.25 respectively. However, we observe a skew towards homozygous-\emph{Cab} and away from homozygous \emph{Kaga}. This was likely caused by the hybrid incompatibility between \emph{Cab} and \emph{Kaga}, given the two strains were derived from populations that are thought to be at the point of speciation (see the previous analysis in \textbf{Chapter \ref{MIKK-genomes-chap}}, section \textbf{\ref{introgression-sec}}).



\begin{figure}
\includegraphics[width=1\linewidth]{figs/somites/prop_sites_total} \caption{Proportions of 5-kb blocks called as either homozygous-\emph{Cab}, heterozygous, or homozygous-\emph{Kaga}.}\label{fig:prop-sites-total}
\end{figure}

\clearpage

\hypertarget{somite-assoc-tests}{%
\section{GWAS}\label{somite-assoc-tests}}

Finally, I used the called recombination blocks as pseudo-SNPs in a genetic linkage analysis. To detect associations between the pseudo-SNPs and the three phenotypes of interest, I used a linear mixed model (\textbf{LMM}) as implemented in GCTA (Yang, Lee, et al. 2011), which follows the form set out in the Introduction in section \ref{GWAS}, equations \eqref{eq:gwaslmm} to \eqref{eq:gwaslmme}. For the genetic relationship matrix (\textbf{GRM}), I additionally used the leave-one-chromosome-out implementation of GCTA's LMM, with excludes the chromosome on which the candidate SNP is located when calculating the GRM. A GRM constructed from the entire genome is presented in \textbf{Figure \ref{fig:somite-grm}}. The square in the top right-hand corner is created by samples \textasciitilde550-648, which, based on Figure \ref{fig:karyo-no-missing} above, clearly have distinct genotypes to the rest of the samples due to their having been bred from different F\textsubscript{1} parents.



\begin{figure}
\includegraphics[width=1\linewidth]{figs/somites/grm_man_0.8} \caption{GRM generated for 600 F\textsubscript{2} samples using Equations \eqref{eq:grmstd} and \eqref{eq:grm}.}\label{fig:somite-grm}
\end{figure}

As described above in \textbf{section \ref{somite-phenotype}}, the microscope used to image the embryos (either AU or DB) differed by several degrees in heat, which likely caused differences in the measurements observed. We accordingly experimented with including microscope as a covariate, either alone or together with the genotype for the reporter locus (either homozygous or heterozygous), or excluding it altogether. In an attempt to avoid complications resulting from its inclusion, Id also tried inverse-normalising the period phenotype within each microscope group, transforming the phenotype to fit a normal distribution across both microscopes.

To set the significance threshold, I permuted the phenotype across samples using 10 different random seeds, together with all covariates when included, and ran a separate linkage model for each permutation. I then set the lowest \(p\)-value from all 10 permutation as the significance threshold for the non-permuted model. I additionally applied a Bonferroni correction to our \(p\)-values by dividing \(\alpha\) (0.05) by the number of pseudo-SNPs in the model, and set this as a secondary threshold.

\hypertarget{period-intercept}{%
\subsection{Period intercept}\label{period-intercept}}

\textbf{Figure \ref{fig:somite-manhattan}} is a Manhattan plot of the genetic linkage results for the period intercept phenotype, inverse-normalised across microscopes. The regions found to be significant based on the permutations' minimum \(p\)-value are set out in \textbf{Table \ref{tab:somite-sig-int-tbl}}.



\begin{figure}
\includegraphics[width=1\linewidth]{figs/somites/manhattan_intercept} \caption{Manhattan plot of the genetic linkage results for the period intercept phenotype, inverse-normalised across microscopes. Pseudo-SNPs with \(p\)-values lower than the permutation significance threshold are highlighted in red.}\label{fig:somite-manhattan}
\end{figure}

\begin{table}

\caption{\label{tab:somite-sig-int-tbl}Significant 5-kb bin ranges for period intercept below the minimum p-value from 10 permutations.}
\centering
\begin{tabular}[t]{rrrr}
\toprule
Chromosome & Bin start & Bin end & Length (kb)\\
\midrule
\cellcolor{gray!6}{3} & \cellcolor{gray!6}{31880001} & \cellcolor{gray!6}{35420000} & \cellcolor{gray!6}{3540}\\
4 & 18090001 & 18095000 & 5\\
\cellcolor{gray!6}{10} & \cellcolor{gray!6}{2995001} & \cellcolor{gray!6}{3690000} & \cellcolor{gray!6}{695}\\
\bottomrule
\end{tabular}
\end{table}

These regions contained a total of 46,872 SNPs imputed from the genotype of the F\textsubscript{0} parental strains.
I ran Ensembl's Variant Effect Predictor (W. McLaren et al. 2016) over these SNPs to identify those that would be most likely to have functional consequences. The full counts of SNPs falling into each category of `consequence' are set out in \textbf{Table \ref{tab:int-consequence-tbl}}. From this process I identified 38 genes that included a missense variant, 1 that included a missense variant and a start lost (ENSORLG00000014616), ane 1 that included a missense variant and a stop lost (ENSORLG00000015149).

\begin{table}

\caption{\label{tab:int-consequence-tbl}Variant Effect Predictor results for SNPs in the bins.}
\centering
\begin{tabular}[t]{lr}
\toprule
Consequence & Count\\
\midrule
\cellcolor{gray!6}{intron variant} & \cellcolor{gray!6}{47211}\\
intergenic variant & 20045\\
\cellcolor{gray!6}{upstream gene variant} & \cellcolor{gray!6}{7304}\\
downstream gene variant & 5229\\
\cellcolor{gray!6}{3 prime UTR variant} & \cellcolor{gray!6}{1082}\\
\addlinespace
synonymous variant & 694\\
\cellcolor{gray!6}{missense variant} & \cellcolor{gray!6}{383}\\
5 prime UTR variant & 201\\
\cellcolor{gray!6}{splice region variant,intron variant} & \cellcolor{gray!6}{126}\\
missense variant,splice region variant & 19\\
\addlinespace
\cellcolor{gray!6}{splice region variant,synonymous variant} & \cellcolor{gray!6}{17}\\
stop gained & 3\\
\cellcolor{gray!6}{splice donor variant} & \cellcolor{gray!6}{1}\\
start lost & 1\\
\cellcolor{gray!6}{stop lost} & \cellcolor{gray!6}{1}\\
\addlinespace
stop lost,splice region variant & 1\\
\bottomrule
\end{tabular}
\end{table}

Our collaborators then combined these results with bulk RNA-seq that they had performed on F\textsubscript{0} \emph{Cab} and \emph{Kaga} individuals, to determine which of these genes are expressed in the tail during embryogenesis. This allowed them to reduce to the list to 29 genes, and a gene ontology analysis of this found that the list of genes was enriched for body axis, somitogenesis, and segmentation (\textbf{Table \ref{tab:psm-final-genes}}). For this list of genes, our collaborators are now in the process of knocking out the protein-altering \emph{Cab} allele into \emph{Kaga} embryos, and \emph{vice versa}, to functionally validate these variants.

\begin{landscape}\begin{table}

\caption{\label{tab:psm-final-genes}Target genes for functional validation expressed in the unsegmented PSM and containing protein alterations. Table generated by Ali Seleit.}
\centering
\resizebox{\linewidth}{!}{
\begin{tabular}[t]{rlll}
\toprule
Chromosome & Ensembl gene ID & Description & Role\\
\midrule
\cellcolor{gray!6}{3} & \cellcolor{gray!6}{ENSORLG00000014656} & \cellcolor{gray!6}{mesoderm posterior protein 2-like} & \cellcolor{gray!6}{Somitogenesis}\\
3 & ENSORLG00000014659 & mesp & Somitogenesis\\
\cellcolor{gray!6}{10} & \cellcolor{gray!6}{ENSORLG00000020474} & \cellcolor{gray!6}{protocadherin 10b} & \cellcolor{gray!6}{Somitogenesis}\\
3 & ENSORLG00000014616 & NA & Possible role in Somitogenesis (MAP-kinase)\\
\cellcolor{gray!6}{10} & \cellcolor{gray!6}{ENSORLG00000020551} & \cellcolor{gray!6}{FAT atypical cadherin 4} & \cellcolor{gray!6}{Possible role in Somitogenesis (PCP, Yap1 regulator)}\\
\addlinespace
10 & ENSORLG00000020531 & neurogenin 1 & Possible role in Somitogenesis (BHLH-TF regulation of Wnt)\\
\cellcolor{gray!6}{3} & \cellcolor{gray!6}{ENSORLG00000015149} & \cellcolor{gray!6}{ADAM metallopeptidase with thrombospondin type 1 motif 18} & \cellcolor{gray!6}{Extracellular matrix}\\
3 & ENSORLG00000015418 & matrix metallopeptidase 15 & Extracellular matrix\\
\cellcolor{gray!6}{10} & \cellcolor{gray!6}{ENSORLG00000020488} & \cellcolor{gray!6}{transforming growth factor beta induced} & \cellcolor{gray!6}{Extracellular matrix}\\
3 & ENSORLG00000028055 & NA &  Signal trasduction (Rho)\\
\addlinespace
\cellcolor{gray!6}{10} & \cellcolor{gray!6}{ENSORLG00000020481} & \cellcolor{gray!6}{ArfGAP with RhoGAP domain, ankyrin repeat and PH domain 3} & \cellcolor{gray!6}{ Signal transduction (GTPase activator RhoGap)}\\
10 & ENSORLG00000020525 & TBC1 domain family member 2A-like & Signal transduction Gtpase activator, cadherin recycling\\
\cellcolor{gray!6}{3} & \cellcolor{gray!6}{ENSORLG00000015260} & \cellcolor{gray!6}{synapse associated protein 1} & \cellcolor{gray!6}{TOR acitivity}\\
10 & ENSORLG00000028553 & NA & NA\\
\cellcolor{gray!6}{3} & \cellcolor{gray!6}{ENSORLG00000015460} & \cellcolor{gray!6}{dpy-19 like C-mannosyltransferase 3} & \cellcolor{gray!6}{glycosylation}\\
\addlinespace
10 & ENSORLG00000022010 & beta-1,4-galactosyltransferase 1 & glycosylation\\
\cellcolor{gray!6}{10} & \cellcolor{gray!6}{ENSORLG00000020498} & \cellcolor{gray!6}{protein NipSnap homolog 3A} & \cellcolor{gray!6}{Mitochondria}\\
10 & ENSORLG00000020494 & nitric oxide associated 1 & Mitochondria\\
\cellcolor{gray!6}{10} & \cellcolor{gray!6}{ENSORLG00000020504} & \cellcolor{gray!6}{ATP-binding cassette sub-family A member 1} & \cellcolor{gray!6}{Metabolism (cholestrol efflux)}\\
3 & ENSORLG00000015118 & phosphorylase kinase regulatory subunit beta & Metabolism\\
\addlinespace
\cellcolor{gray!6}{10} & \cellcolor{gray!6}{ENSORLG00000020493} & \cellcolor{gray!6}{RE1 silencing transcription factor} & \cellcolor{gray!6}{RNA pol II (transcritpion regulation}\\
3 & ENSORLG00000015365 & solute carrier family 7 member 6 opposite strand & RNA pol II (nuclear export)\\
\cellcolor{gray!6}{10} & \cellcolor{gray!6}{ENSORLG00000029052} & \cellcolor{gray!6}{exosome component 3} & \cellcolor{gray!6}{rRNA + RNA binding}\\
3 & ENSORLG00000015278 & adhesion G protein-coupled receptor G3 & G-protein coupled receptor\\
\cellcolor{gray!6}{3} & \cellcolor{gray!6}{ENSORLG00000015287} & \cellcolor{gray!6}{adhesion G-protein coupled receptor G5} & \cellcolor{gray!6}{G-protein coupled receptor}\\
\addlinespace
10 & ENSORLG00000025674 & matrin-3 & inner nuclear protein (chromatin architecture\\
\cellcolor{gray!6}{10} & \cellcolor{gray!6}{ENSORLG00000023325} & \cellcolor{gray!6}{matrin-3} & \cellcolor{gray!6}{inner nuclear protein (chromatin architecture}\\
10 & ENSORLG00000022388 & poly(A) binding protein interacting protein 2 & Translation repressor\\
\cellcolor{gray!6}{3} & \cellcolor{gray!6}{ENSORLG00000015096} & \cellcolor{gray!6}{integrin alpha FG-GAP repeat containing 1} & \cellcolor{gray!6}{NA possibly T cell activation}\\
\bottomrule
\end{tabular}}
\end{table}
\end{landscape}

\hypertarget{psm-area}{%
\subsection{PSM area}\label{psm-area}}

\textbf{Figure \ref{fig:psm-manhattan}} is a Manhattan plot of the genetic linkage results for the PSM area phenotype. The regions found to be significant based on the permutations' minimum \(p\)-value are set out in \textbf{Table \ref{tab:somite-sig-psm-tbl}}, although they exceed the Bonferroni correction threshold as well. I note that this \textasciitilde6 Mb significant region on chromosome 3 does not overlap at all with the significant region discovered for the period intercept phenotype.



\begin{figure}
\includegraphics[width=1\linewidth]{figs/somites/manhattan_psm_no-covariates} \caption{Manhattan plot of the genetic linkage results for the PSM area phenotype. Pseudo-SNPs with \(p\)-values lower than the permutation significance threshold are highlighted in yellow.}\label{fig:psm-manhattan}
\end{figure}

\begin{table}

\caption{\label{tab:somite-sig-psm-tbl}Significant 5-kb bin range for PSM area below the minimum p-value from 10 permutations.}
\centering
\begin{tabular}[t]{rrrr}
\toprule
Chromosome & Bin start & Bin end & Length (kb)\\
\midrule
\cellcolor{gray!6}{3} & \cellcolor{gray!6}{20375001} & \cellcolor{gray!6}{26285000} & \cellcolor{gray!6}{5910}\\
\bottomrule
\end{tabular}
\end{table}

This region contained a total of 29,096 SNPs imputed from the genotype of the F\textsubscript{0} parental strains.
I ran Ensembl's Variant Effect Predictor (W. McLaren et al. 2016) over these SNPs to identify those that would be most likely to have functional consequences. The full counts of SNPs falling into each category of `consequence' are set out in \textbf{Table \ref{tab:psm-consequence-tbl}}. From this process I identified 114 genes that included a missense variant, and 1 that included a both a missense variant and a start lost (ENSORLG00000010863, a centriole, cilia and spindle-associated protein).

\begin{table}

\caption{\label{tab:psm-consequence-tbl}Variant Effect Predictor results for SNPs in the bins.}
\centering
\begin{tabular}[t]{lr}
\toprule
Consequence & Count\\
\midrule
\cellcolor{gray!6}{intron variant} & \cellcolor{gray!6}{23189}\\
intergenic variant & 9171\\
\cellcolor{gray!6}{downstream gene variant} & \cellcolor{gray!6}{8894}\\
upstream gene variant & 8491\\
\cellcolor{gray!6}{3 prime UTR variant} & \cellcolor{gray!6}{2104}\\
\addlinespace
synonymous variant & 1141\\
\cellcolor{gray!6}{missense variant} & \cellcolor{gray!6}{716}\\
5 prime UTR variant & 433\\
\cellcolor{gray!6}{splice region variant,intron variant} & \cellcolor{gray!6}{184}\\
splice region variant,synonymous variant & 18\\
\addlinespace
\cellcolor{gray!6}{missense variant,splice region variant} & \cellcolor{gray!6}{7}\\
stop gained & 2\\
\cellcolor{gray!6}{splice donor variant} & \cellcolor{gray!6}{1}\\
start lost & 1\\
\bottomrule
\end{tabular}
\end{table}

Our collaborators then combined these results with bulk RNA-seq that they had performed on F\textsubscript{0} \emph{Cab} and \emph{Kaga} individuals, to determine which of these genes are expressed in the unsegmented tail during embryogenesis. This allowed them to reduce to the list to 96 genes, although they were not apparently associated with a specific gene ontology. As with the period intercept phenotype, our collaborators are now in the process of knocking in the \emph{Cab} allele into \emph{Kaga} embryos, and \emph{vice versa}, to functionally validate these variants.

\hypertarget{MIKK-F2-chap}{%
\chapter{Genetic linkage study of bold/shy behaviours in the MIKK panel}\label{MIKK-F2-chap}}

\chaptermark{MIKK panel behaviour}

The purpose of the study I set out in this chapter was to run the behavioural analysis described in Chapter \ref{Pilot-chap} over the MIKK panel, identify the lines that diverged in both (a) their own behaviour; and (b) the level of transmission of their behaviour onto their \emph{\definecolor{iCab_424B4D}{HTML}{424B4D}\textcolor{iCab_424B4D}{iCab}} reference tank partner, and then use them as the parental strains in an F\textsubscript{2} cross to attempt to identify the specific genetic loci associated with those differences.

\hypertarget{data-collection---f0-generation}{%
\section{\texorpdfstring{Data collection - F\textsubscript{0} generation}{Data collection - F0 generation}}\label{data-collection---f0-generation}}

In November 2019 I traveled to the fish facility managed by our collaborators, the Loosli Group at KIT in Karlsruhe, Germany, and over the course of 11 days from 11 to 21 November 2019, I ran the behavioural assay described in Chapter \ref{pilot-data-collection} another 206 times. I again used the \emph{\definecolor{iCab_424B4D}{HTML}{424B4D}\textcolor{iCab_424B4D}{iCab}} strain as the reference fish, and for the test fish I used either an individual from one of the MIKK panel lines, individuals captured from the same Kiyosu population as the MIKK panel but permitted to breed freely within a separate tank in the facility (`Kiyosu closed-capture', or `\textbf{Kiyosu CC}'), or individuals from a related but different species of medaka from the Philippines, \emph{Oryzias luzonensis}. I ensured that I performed at least 2 assay runs of 4 individuals each on two different days for each MIKK panel line that was available, generating a minimum of 8 test fish replicates per line. As there were four pairs of fish in the test tank during each run, the complete dataset comprises 824 videos of pairs of fish, which I further divided by assay component (open field and novel object) to create 1648 videos.

I again used the software \emph{idtrackerai} (Romero-Ferrero et al. 2019) to track the movement of the fishes across frames of each video. After adjusting the software parameters for each video to maximise the number of frames that were successfully tracked, I was left with 1610 out of the 1648 videos (\textasciitilde97.7\%) where both fishes were tracked over at least 85\% of frames, and I only included these 1610 videos in the downstream analysis. The first question to address was whether the MIKK panel lines differed in their behaviours. I therefore computed each individual fish's mean speed (measured as the distance traveled in pixels per 0.08 seconds) over the course of the full 20-minute video, grouped them by line, and plotted the results presented in \textbf{Figure \ref{fig:mikk-mean-speed}}. I continue to use the same order and colour palette for the MIKK panel lines as in this Figure throughout the rest of this Chapter.



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/line_mean_speed_0.08_all} \caption{Mean speed of the MIKK panel and other strains over the course of the entire 20-minute video (measured as the distance traveled in pixels per 0.08 seconds). \emph{\definecolor{iCab_424B4D}{HTML}{424B4D}\textcolor{iCab_424B4D}{iCab}} fishes in the \emph{\definecolor{iCab_424B4D}{HTML}{424B4D}\textcolor{iCab_424B4D}{iCab}}-\emph{\definecolor{iCab_424B4D}{HTML}{424B4D}\textcolor{iCab_424B4D}{iCab}} control condition are at the top, the MIKK panel lines are sorted by their group median, and the Kiyosu closed capture and \emph{O. luzonensis} fishes are at the bottom.}\label{fig:mikk-mean-speed}
\end{figure}

This figure shows that there are clear differences between some MIKK panel lines at the extremes, and that the lines differ in the amount of within-line variance observed (shown plotted against the lines' median speed in \textbf{Figure \ref{fig:mikk-mean-speed-variance}}). These figures acted as a general guide to determine which lines to select as the parental strains in the F\textsubscript{2} cross. To identify genetic variants directly associated with bold-shy behaviours, I sought to select lines that showed either high or low levels of movement, and preferably low within-line variance.



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/line_mean_speed_variance_0.08_all} \caption{Line median (vertical axis) and line variance (horizontal axis) for individual mean speed (pixels per 0.08 seconds) across the full 20-minute video (i.e.~both the open field and novel object assay components).}\label{fig:mikk-mean-speed-variance}
\end{figure}

To illustrate the differences between certain lines in terms of both direct and social genetic effects, in \textbf{Figure \ref{fig:extreme-paths}} I have plotted the tracked paths for 3 lines 5 minutes into the 10-minute open field assay component: the slowest line \definecolor{22-1 (‘David’)_FB737A}{HTML}{FB737A}\textcolor{22-1 (‘David’)_FB737A}{22-1 (‘David’)},\footnote{In almost all cases, for the entirety of the assay, David remained as still (and captivating) as Michelangelo's statue.} one of the other slowest lines \definecolor{18-2 (‘Elsa’)_FF66A6}{HTML}{FF66A6}\textcolor{18-2 (‘Elsa’)_FF66A6}{18-2 (‘Elsa’)},\footnote{Elsa was Frozen like David, but far more charismatic.} and the fastest line . The fishes are coloured by their line, with \emph{\definecolor{iCab_424B4D}{HTML}{424B4D}\textcolor{iCab_424B4D}{iCab}} in dark grey. There appears to be a social genetic effect when comparing the behaviour of \emph{\definecolor{iCab_424B4D}{HTML}{424B4D}\textcolor{iCab_424B4D}{iCab}} when paired with the similarly slow-moving lines and \definecolor{18-2 (‘Elsa’)_FF66A6}{HTML}{FF66A6}\textcolor{18-2 (‘Elsa’)_FF66A6}{18-2 (‘Elsa’)}. At the other extreme, line \definecolor{10-1 (‘Janeway’)_F8766D}{HTML}{F8766D}\textcolor{10-1 (‘Janeway’)_F8766D}{10-1 (‘Janeway’)} has moved extremely quickly,\footnote{Janeway moved so fast it appeared she was approaching warp speed.} spending much of its time moving along the boundaries of their test tanks. I understand from our collaborator Felix Loosli, a fish behaviour expert, that this movement along the boundaries of a tank is typical of medaka when introduced to a novel environment, and I observe it too with \emph{\definecolor{iCab_424B4D}{HTML}{424B4D}\textcolor{iCab_424B4D}{iCab}} when it is paired with line \definecolor{22-1 (‘David’)_FB737A}{HTML}{FB737A}\textcolor{22-1 (‘David’)_FB737A}{22-1 (‘David’)}.



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/path_plot_22-1_18-2_10-1_300} \caption{Path plots for lines 22-1 (\textbf{A}), 18-2 (\textbf{B}) and 10-1 (\textbf{C}) 5 minutes into the open field assay for the first (left) and second (right) run with each line. The paths of \emph{\definecolor{iCab_424B4D}{HTML}{424B4D}\textcolor{iCab_424B4D}{iCab}} individuals are coloured in dark grey, with the other lines depicted in their representative colours.}\label{fig:extreme-paths}
\end{figure}

Although lines \definecolor{22-1 (‘David’)_FB737A}{HTML}{FB737A}\textcolor{22-1 (‘David’)_FB737A}{22-1 (‘David’)} and \definecolor{10-1 (‘Janeway’)_F8766D}{HTML}{F8766D}\textcolor{10-1 (‘Janeway’)_F8766D}{10-1 (‘Janeway’)} were the most extreme in terms of mean speed, I ruled them out of selection for the F\textsubscript{2} cross for the following reasons. Our collaborators informed us that through a separate analysis on heartbeat phenotypes across the MIKK panel, they discovered that the heart of line \definecolor{22-1 (‘David’)_FB737A}{HTML}{FB737A}\textcolor{22-1 (‘David’)_FB737A}{22-1 (‘David’)} often stops beating for up to minutes at a time. This may explain the lack of movement observed during our assay. The behaviours they exhibit may therefore not represent a phenotype related to the boldness-shyness axis, but rather an extreme phenotype of a particular organ system. Our collaborators are, however, using this line in a separate F\textsubscript{2} cross investigating heart phenotypes.

On the other hand, line \definecolor{10-1 (‘Janeway’)_F8766D}{HTML}{F8766D}\textcolor{10-1 (‘Janeway’)_F8766D}{10-1 (‘Janeway’)} appeared to habituate to the open field assay component by slowing down, rather than speeding up (\textbf{Figure \ref{fig:10-1-dens}}). That is to say, as the assay progressed, they slowed down their movement, and this suggests that their typical response to stress is to move faster rather than slower. However, as I am using speed as a proxy for boldness (where a quicker habituation to the assay, indicated by an increase in movement, suggests greater boldness), this would create difficulties in attributing a functional interpretation of behaviours in the F\textsubscript{2} individuals, and for this reason I excluded line \definecolor{10-1 (‘Janeway’)_F8766D}{HTML}{F8766D}\textcolor{10-1 (‘Janeway’)_F8766D}{10-1 (‘Janeway’)}.



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/select_0.08_15_10-1_dge} \caption{Tile and density plots for line \definecolor{10-1 (‘Janeway’)_F8766D}{HTML}{F8766D}\textcolor{10-1 (‘Janeway’)_F8766D}{10-1 (‘Janeway’)}, showing the reduction in the frequency of higher-movement states over the course of the open field assay component (\textbf{A} and \textbf{B}), with the novel object assay component (\textbf{C} and \textbf{D}) for comparison.}\label{fig:10-1-dens}
\end{figure}

\clearpage

\hypertarget{hmm-states}{%
\section{HMM states}\label{hmm-states}}

To explore the behaviours of the MIKK panel at a finer resolution, as for the pilot study described in Chapter \ref{Pilot-chap}, I again applied a Hidden Markov Model (\textbf{HMM}) to classify the fishes' movements based on their distance and angle of travel between time intervals. I used the same method to select the best choice of time interval and number of states (\textbf{Figure \ref{fig:mikk-param-comp}}).



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/compare_params_with_arrows} \caption{Comparison between HMM parameters. Horizontal axis: Mean concordance between states assigned by HMMs through a 2-fold cross-validation process. Vertical axis: Kruskal-Wallis statistic comparing strains based on the proportion of time spent in each HMM state, summed across all states. Size of points correspond to the interval, in seconds, between which the distance and angle of travel was calculated (Methods). The top right of the figure represents the better models, indicated by the pink arrow. The highlighted circle represents the best combination of HMM state space and time interval that also had favourable symmetrical properties (see below).}\label{fig:mikk-param-comp}
\end{figure}

Here I observed the same phenomenon where the parameter combinations that performed the best showed an asymmetry between some states that would make interpretation difficult. For example, a time interval of 0.08 seconds combined with a state space of 17 caused state 4 to appear to get carved out of state 3 (\textbf{Figure \ref{fig:mikk-hmm-asym}}).



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/0.08_17_polar_all_dge} \caption{Based on the concordance analysis results set out in Figure \ref{fig:mikk-param-comp} above, the best apparent combination of parameters -- 0.08 time interval with a 17-state space -- created an asymmetry between states 3 and 4, which would cause difficulties in interpreting their biological relevance.}\label{fig:mikk-hmm-asym}
\end{figure}

The best combination of parameters \emph{without} this asymmetry was a time interval of 0.05 seconds with a state space of 15 (see the polar plots for the states in \textbf{Appendix \ref{fig:hmm-states-05}}). However, due to a glitch in the video recording software, several videos recorded on 13 November 2019 were incorrectly recorded with a frame rate of 14 instead of the desired 30. The insufficient number of frames for those videos meant that it was impossible to measure the distance and angle of travel between a time interval as low as 0.05 seconds. So that these videos could be included in the dataset, I accordingly selected the combination of 15 states and a 0.08-second interval for all downstream analyses (\textbf{Figure \ref{fig:mikk-hmm-sym}}).



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/0.08_15_polar_all_dge} \caption{The HMM states used for the downstream analysis, with the model classified based on the distance of travel (log\textsubscript{10} pixels, radial axis) and angle of travel (angle). A straight forward movement would sit around 0°, a left movement around 270°, and a right movement around 90°.}\label{fig:mikk-hmm-sym}
\end{figure}

\clearpage

\hypertarget{social-genetic-effects}{%
\section{Social genetic effects}\label{social-genetic-effects}}

As discussed above, in this project our traits of interest include not only direct genetic behaviours, but also social genetic behaviours. I therefore sought to identify the MIKK panel lines that transmit their behaviour onto the reference \emph{\definecolor{iCab_424B4D}{HTML}{424B4D}\textcolor{iCab_424B4D}{iCab}} tank partners either to the greatest or least extent. I refer to this transmission of behaviour as ``\textbf{\emph{charisma}}'', to capture the element of behavioural influence. I formulated two methods to measure charisma, which I refer to as a) HMM state co-occupancy; and b) reference deviation. The first, HMM state co-occupancy, measures the proportions of time that the \emph{\definecolor{iCab_424B4D}{HTML}{424B4D}\textcolor{iCab_424B4D}{iCab}} reference fish spends in the same HMM state as its tank partner. The second, deviation of the reference fishes' behaviour from the behaviour exhibited in the control condition, seeks to quantify the extent to which the \emph{\definecolor{iCab_424B4D}{HTML}{424B4D}\textcolor{iCab_424B4D}{iCab}}'s behaviour changes when partnered with each MIKK panel line.

\hypertarget{state-co-occupancy}{%
\subsection{State co-occupancy}\label{state-co-occupancy}}

\textbf{Figure \ref{fig:F0-sge-cooc-box}} sets out the proportions of total time for each assay sub-component that each pair of individual fish spent in the same HMM state, grouped by line and ranked in the same order as their group median for individual mean speed as shown above in \textbf{Figure \ref{fig:mikk-mean-speed}}. \textbf{\ref{fig:F0-sge-cooc-box}A} shows the data as boxplots, with \(p\)-values from the Kruskal-Wallis test comparing all groups. \textbf{Figure \ref{fig:F0-sge-cooc-box}B} shows the same data but with each group's median represented by columns to make it easier to compare group medians. Of the slower-moving lines, \definecolor{8-2 (‘Gail’)_FF699C}{HTML}{FF699C}\textcolor{8-2 (‘Gail’)_FF699C}{8-2 (‘Gail’)}\footnote{Named after Gail the Snail, the slow but curiously captivating character from \emph{It's Always Sunny in Philadelphia}.} and \definecolor{18-2 (‘Elsa’)_FF66A6}{HTML}{FF66A6}\textcolor{18-2 (‘Elsa’)_FF66A6}{18-2 (‘Elsa’)} tend to show relatively higher state co-occupancy in the open field component, but during the novel object component, the slow-moving line \definecolor{139-4_FF61CC}{HTML}{FF61CC}\textcolor{139-4_FF61CC}{139-4} has the highest median co-occupancy of all lines. Of the faster-moving lines, \definecolor{43-2_F17D50}{HTML}{F17D50}\textcolor{43-2_F17D50}{43-2} and \definecolor{13-2_F57A5F}{HTML}{F57A5F}\textcolor{13-2_F57A5F}{13-2} showed the highest state co-occupancy during the open field assay component. However, the moderate-to-fast line \definecolor{21-2 (‘Amelia’)_49B500}{HTML}{49B500}\textcolor{21-2 (‘Amelia’)_49B500}{21-2 (‘Amelia’)}\footnote{Named after the speedy and charismatic Amelia Earhart.} had relatively high state co-occupancy during both assay components.



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/0.08_15_cooc_box_all} \caption{Frequency of HMM state co-occupancy}\label{fig:F0-sge-cooc-box}
\end{figure}

To visualise which states are driving the higher co-occupancy measures, for a selection of lines I generated a heatmap of the states occupied simultaneously by the \emph{\definecolor{iCab_424B4D}{HTML}{424B4D}\textcolor{iCab_424B4D}{iCab}} reference and MIKK test fishes, combining the observations for all individuals within each test fish group (\textbf{Figure \ref{fig:F0-sge-cooc-heat}}). When \emph{\definecolor{iCab_424B4D}{HTML}{424B4D}\textcolor{iCab_424B4D}{iCab}} is paired with \definecolor{18-2 (‘Elsa’)_FF66A6}{HTML}{FF66A6}\textcolor{18-2 (‘Elsa’)_FF66A6}{18-2 (‘Elsa’)} or \definecolor{8-2 (‘Gail’)_FF699C}{HTML}{FF699C}\textcolor{8-2 (‘Gail’)_FF699C}{8-2 (‘Gail’)}, the fishes most frequently occupy states 3 or 1 in both open field and novel object components. In pairings with line \definecolor{139-4_FF61CC}{HTML}{FF61CC}\textcolor{139-4_FF61CC}{139-4}, while the test fishes remain in the still states 1 or 3 during the open field assay, \emph{\definecolor{iCab_424B4D}{HTML}{424B4D}\textcolor{iCab_424B4D}{iCab}} tends to be moving much faster in states 11 and 13. However, during the novel object component, they co-occupy state 3 more than in any other combination. This general pattern is observed with line \definecolor{14-2_F066EA}{HTML}{F066EA}\textcolor{14-2_F066EA}{14-2} as well, albeit to a lesser extent. For \definecolor{38-2 (‘Mark’)_00C08B}{HTML}{00C08B}\textcolor{38-2 (‘Mark’)_00C08B}{38-2 (‘Mark’)},\footnote{Line 38-2 appears to have intermediate measures for both movement and charisma, causing us to affectionately refer to it as the ``boring'' line, and later christening it after Mark Corrigan from \emph{Peep Show}.} the fishes tend not to show a strong preference for co-occupying a particular state for either assay component, but the diagonal spread indicates that they tend to move at similar speeds. When paired with the faster moving \definecolor{21-2 (‘Amelia’)_49B500}{HTML}{49B500}\textcolor{21-2 (‘Amelia’)_49B500}{21-2 (‘Amelia’)}, the novel object component appears to accentuate the co-occupancy of state 3 that is also observed in the open field component. Finally, when paired with line \definecolor{40-1 (‘Max’)_93AA00}{HTML}{93AA00}\textcolor{40-1 (‘Max’)_93AA00}{40-1 (‘Max’)},\footnote{Line 40-1 and its sibling line 40-2 both appeared preternaturally bold, just like the siblings Max and Moritz from the eoponymous story by Wilhelm Busch (Busch 2015).} in both assay components, both fishes show a strong preference for the faster-moving states.



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/0.08_15_cooc_heatmap} \caption{Heatmaps for a selection of MIKK panel lines (including those ultimately selected as the parental strains in the F\textsubscript{2} cross) showing the frequency of HMM states simultaneously occupied by the reference (x-axis) and test (y-axis) fishes during the open field (\textbf{A}) and novel object (\textbf{B}) assay components, aggregated over all replicates for each line.}\label{fig:F0-sge-cooc-heat}
\end{figure}

\hypertarget{deviation-of-from-its-control-condition}{%
\subsection{\texorpdfstring{Deviation of \emph{\definecolor{iCab_424B4D}{HTML}{424B4D}\textcolor{iCab_424B4D}{iCab}} from its control condition}{Deviation of  from its control condition}}\label{deviation-of-from-its-control-condition}}

The second method for quantifying the level of behavioural transmission from test fish to \emph{\definecolor{iCab_424B4D}{HTML}{424B4D}\textcolor{iCab_424B4D}{iCab}} reference fish was to determine the proportion of time that the \emph{\definecolor{iCab_424B4D}{HTML}{424B4D}\textcolor{iCab_424B4D}{iCab}} spent in a particular state when paired with another \emph{\definecolor{iCab_424B4D}{HTML}{424B4D}\textcolor{iCab_424B4D}{iCab}}, and then quantify the degree to which those proportions change when in the presence of a fish from another line (\textbf{Figure \ref{fig:F0-sge-deviation}}). \textbf{Figure \ref{fig:F0-sge-deviation}A} presents boxplots for state frequencies for all \emph{\definecolor{iCab_424B4D}{HTML}{424B4D}\textcolor{iCab_424B4D}{iCab}} individuals in \emph{\definecolor{iCab_424B4D}{HTML}{424B4D}\textcolor{iCab_424B4D}{iCab}}-\emph{\definecolor{iCab_424B4D}{HTML}{424B4D}\textcolor{iCab_424B4D}{iCab}} pairings. I further calculated the state frequencies for all \emph{\definecolor{iCab_424B4D}{HTML}{424B4D}\textcolor{iCab_424B4D}{iCab}} reference fishes for all other MIKK line pairings. For each combination of assay component, line-pairing, and HMM state, I then ran Welch's t-test (Ruxton 2006) comparing the proportions of time the \emph{\definecolor{iCab_424B4D}{HTML}{424B4D}\textcolor{iCab_424B4D}{iCab}} individuals spent in that state when paired with another \emph{\definecolor{iCab_424B4D}{HTML}{424B4D}\textcolor{iCab_424B4D}{iCab}}, against the proportions of time the \emph{\definecolor{iCab_424B4D}{HTML}{424B4D}\textcolor{iCab_424B4D}{iCab}} reference individuals spent in that state when paired with a different MIKK line. I then summed the t-statistics across states to generate a single metric for each combination of line and assay component, and plotted the results in \textbf{Figure \ref{fig:F0-sge-deviation}B}.



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/0.08_15_deviation} \caption{Deviation of state frequency for \emph{\definecolor{iCab_424B4D}{HTML}{424B4D}\textcolor{iCab_424B4D}{iCab}} reference fishes when paired with MIKK panel lines relative to when paired with another \emph{\definecolor{iCab_424B4D}{HTML}{424B4D}\textcolor{iCab_424B4D}{iCab}}. \textbf{A}: Boxplots of HMM state frequency for \emph{\definecolor{iCab_424B4D}{HTML}{424B4D}\textcolor{iCab_424B4D}{iCab}} individuals when paired with another \emph{\definecolor{iCab_424B4D}{HTML}{424B4D}\textcolor{iCab_424B4D}{iCab}}. \textbf{B}: Summed absolute t-statistics (Welch's t-test) comparing the proportions of time \emph{\definecolor{iCab_424B4D}{HTML}{424B4D}\textcolor{iCab_424B4D}{iCab}} spent in each state when paired with another \emph{\definecolor{iCab_424B4D}{HTML}{424B4D}\textcolor{iCab_424B4D}{iCab}}, against when paired with a MIKK panel line.}\label{fig:F0-sge-deviation}
\end{figure}

The first thing to note in this Figure is that \emph{\definecolor{iCab_424B4D}{HTML}{424B4D}\textcolor{iCab_424B4D}{iCab}}'s deviations from its behaviour exhibited in the control condition tend to be smaller when paired with faster-moving fishes. This was expected given \emph{\definecolor{iCab_424B4D}{HTML}{424B4D}\textcolor{iCab_424B4D}{iCab}} is also a faster-moving fish, but it strengthens the observation that when paired with slower-moving fishes, those tank partners are causing the \emph{\definecolor{iCab_424B4D}{HTML}{424B4D}\textcolor{iCab_424B4D}{iCab}} reference fish to move slower than they would otherwise. Of the slower-moving MIKK lines, \definecolor{4-2_FC61D4}{HTML}{FC61D4}\textcolor{4-2_FC61D4}{4-2} and \definecolor{14-2_F066EA}{HTML}{F066EA}\textcolor{14-2_F066EA}{14-2} causing the largest deviations of the \emph{\definecolor{iCab_424B4D}{HTML}{424B4D}\textcolor{iCab_424B4D}{iCab}} reference fishes' behaviours from their control condition behaviour in the open field and novel object components respectively. I also observe large deviations for the moderately-moving \definecolor{106-2_00B9E3}{HTML}{00B9E3}\textcolor{106-2_00B9E3}{106-2} and \definecolor{71-1_00BECD}{HTML}{00BECD}\textcolor{71-1_00BECD}{71-1} during the novel object components. There are no clear outliers for either assay component for any of the faster moving lines.

\hypertarget{selection-of-lines-for-the-f2-cross}{%
\section{\texorpdfstring{Selection of lines for the F\textsubscript{2} cross}{Selection of lines for the F2 cross}}\label{selection-of-lines-for-the-f2-cross}}

On the basis of the above findings, I selected 6 MIKK panel lines as the parental lines for the F\textsubscript{2} cross (\textbf{Figure \ref{fig:F0-line-mean-speed-var-select}}). Conceptually, I sought to select lines that diverged on two measures: a) bold-shy behaviours; and b) the extent to which the lines transmitted their behaviours onto their tank partners.

As \emph{\definecolor{iCab_424B4D}{HTML}{424B4D}\textcolor{iCab_424B4D}{iCab}} is a fast-moving line, it was more difficult to detect instances where fast-moving MIKK panel lines were influencing its behaviour. I was therefore more confident of identifying slow-movement/high-charisma lines, and so to increase the likelihood of identifying genetic variants that are responsible for a stronger transmission of slow-moving behaviours, I chose two slow-movement/high-charisma lines for the F\textsubscript{2} cross: \definecolor{18-2 (‘Elsa’)_FF66A6}{HTML}{FF66A6}\textcolor{18-2 (‘Elsa’)_FF66A6}{18-2 (‘Elsa’)} and \definecolor{8-2 (‘Gail’)_FF699C}{HTML}{FF699C}\textcolor{8-2 (‘Gail’)_FF699C}{8-2 (‘Gail’)}. In the event that both lines possessed the same genetic variants that influence this behavioural transmission trait, it would vastly increase the power of detecting it during the genetic linkage analysis. Both these lines were one of the most slow-moving lines, had high levels of state co-occupancy during both assay components, and \definecolor{18-2 (‘Elsa’)_FF66A6}{HTML}{FF66A6}\textcolor{18-2 (‘Elsa’)_FF66A6}{18-2 (‘Elsa’)} caused a high level of deviation from the \emph{\definecolor{iCab_424B4D}{HTML}{424B4D}\textcolor{iCab_424B4D}{iCab}} control condition.



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/line_mean_speed_variance_selected} \caption{Line median (vertical axis) and line variance (horizontal axis) for individual mean speed across the full 20-minute video (i.e.~both the open field and novel object assay components) as shown above in Figure \ref{fig:mikk-mean-speed-variance}, now coloured only for the lines selected as the parental strains for the F\textsubscript{2} cross.}\label{fig:F0-line-mean-speed-var-select}
\end{figure}

For the slow-movement/low-charisma line, I selected \definecolor{50-2 (‘Patrick’)_BB81FF}{HTML}{BB81FF}\textcolor{50-2 (‘Patrick’)_BB81FF}{50-2 (‘Patrick’)} which exhibited a moderately slow level of movement, low within-line variance for mean speed, and low measures for both state co-occupancy and \emph{\definecolor{iCab_424B4D}{HTML}{424B4D}\textcolor{iCab_424B4D}{iCab}} deviation.{[}\^{}Named after Spongebob's slow and unassuming best friend, Patrick Star.{]} For the high-movement/high-charisma line, I selected \definecolor{21-2 (‘Amelia’)_49B500}{HTML}{49B500}\textcolor{21-2 (‘Amelia’)_49B500}{21-2 (‘Amelia’)}. Despite its high within line variance (\textbf{Figure \ref{fig:F2-time-dge-of}}), this potentially made it easier to detect its social genetic effects, as the slower-moving individuals appeared to transmit those behaviours strongly to their \emph{\definecolor{iCab_424B4D}{HTML}{424B4D}\textcolor{iCab_424B4D}{iCab}} tank partners, giving it a high score among fast-moving lines for state co-occupancy across both assay components. For the high-movement/low-charisma line I selected \definecolor{40-1 (‘Max’)_93AA00}{HTML}{93AA00}\textcolor{40-1 (‘Max’)_93AA00}{40-1 (‘Max’)}, as it has low within-line variance, and low-to-moderate metrics for state co-occupancy and \emph{\definecolor{iCab_424B4D}{HTML}{424B4D}\textcolor{iCab_424B4D}{iCab}} deviation. However, I note that these measures may be confounded by the possibility that \definecolor{40-1 (‘Max’)_93AA00}{HTML}{93AA00}\textcolor{40-1 (‘Max’)_93AA00}{40-1 (‘Max’)} behaves in a similar way to \emph{\definecolor{iCab_424B4D}{HTML}{424B4D}\textcolor{iCab_424B4D}{iCab}}, which would be difficult to determine whether it was behaving differently.

In addition to these extreme lines, I also selected a line that was intermediate for both traits in an attempt to avoid breeding incompatibilities that might arise from attempting to cross lines with such divergent behavioural traits. For this purpose I selected line \definecolor{38-2 (‘Mark’)_00C08B}{HTML}{00C08B}\textcolor{38-2 (‘Mark’)_00C08B}{38-2 (‘Mark’)} for its intermediate speed, low within-line variance for mean speed, and intermediate measures for HMM state co-occupancy and \emph{\definecolor{iCab_424B4D}{HTML}{424B4D}\textcolor{iCab_424B4D}{iCab}} deviation.

The selection of parental lines for the F\textsubscript{2} cross indicating their putative position across the two axes of interest (movement, and behavioural transmission or `charisma') are set out below in \textbf{Figure \ref{fig:F0-line-select-schema}}.



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/line_selection_schema} \caption{Schema of the MIKK panel lines selected as the parental strains for the multi-way F\textsubscript{2} cross, indicating their intended position across the two axes of interest (movement, and behavioural transmission or `charisma').}\label{fig:F0-line-select-schema}
\end{figure}

\clearpage

\hypertarget{direct-genetic-effects-1}{%
\section{Direct genetic effects}\label{direct-genetic-effects-1}}

With these lines selected, I ran a similar analysis to what I described in \ref{Pilot-chap}, where I ran multi-way ANOVAs to determine whether certain lines differed in the proportions of time they spent in these HMM states, while including the date of assay, time of assay, tank quadrant and tank side as covariates. \textbf{Table \ref{tab:mikk-dge-F0}} sets out the states which showed a significant difference between these 6 lines, with p-values adjusted for the False Discovery Rate (\textbf{FDR}).

\begin{table}

\caption{\label{tab:mikk-dge-F0}Significant differences in the proportion of time spent in each HMM state across test fish lines selected for the F~2~ cross for the open field and novel object assay components.}
\centering
\begin{tabular}[t]{llll}
\toprule
Assay & State & Variance explained (\%) & p-value (FDR-adjusted)\\
\midrule
\cellcolor{gray!6}{open field} & \cellcolor{gray!6}{3} & \cellcolor{gray!6}{21.53} & \cellcolor{gray!6}{2.67e-03}\\
open field & 4 & 23.28 & 1.60e-02\\
\cellcolor{gray!6}{open field} & \cellcolor{gray!6}{5} & \cellcolor{gray!6}{20.61} & \cellcolor{gray!6}{3.15e-02}\\
open field & 10 & 29.06 & 1.41e-03\\
\cellcolor{gray!6}{open field} & \cellcolor{gray!6}{12} & \cellcolor{gray!6}{24.91} & \cellcolor{gray!6}{1.45e-02}\\
\addlinespace
open field & 14 & 29.52 & 1.29e-03\\
\cellcolor{gray!6}{novel object} & \cellcolor{gray!6}{1} & \cellcolor{gray!6}{16.21} & \cellcolor{gray!6}{1.90e-02}\\
novel object & 2 & 15.14 & 4.86e-02\\
\cellcolor{gray!6}{novel object} & \cellcolor{gray!6}{4} & \cellcolor{gray!6}{26.90} & \cellcolor{gray!6}{5.21e-03}\\
novel object & 5 & 27.84 & 4.27e-03\\
\addlinespace
\cellcolor{gray!6}{novel object} & \cellcolor{gray!6}{10} & \cellcolor{gray!6}{23.73} & \cellcolor{gray!6}{2.81e-02}\\
\bottomrule
\end{tabular}
\end{table}

\textbf{Figures \ref{fig:F2-time-dge-of}} and \textbf{\ref{fig:F2-time-dge-no}} highlight the states that showed significant differences in the proportions of time that these lines spent in those states for the open field and novel object assay components respectively. In both figures, \textbf{A} highlights the significant states, \textbf{B} shows how the individuals moved through those states over the course of the 10-minute assay component, and \textbf{C} shows the densities of the significant states within each line. For the open field component, although the tile plots show a notable level of variance within each line, the density plots clarify how the three slow-moving lines (\definecolor{8-2 (‘Gail’)_FF699C}{HTML}{FF699C}\textcolor{8-2 (‘Gail’)_FF699C}{8-2 (‘Gail’)}, \definecolor{18-2 (‘Elsa’)_FF66A6}{HTML}{FF66A6}\textcolor{18-2 (‘Elsa’)_FF66A6}{18-2 (‘Elsa’)} and \definecolor{50-2 (‘Patrick’)_BB81FF}{HTML}{BB81FF}\textcolor{50-2 (‘Patrick’)_BB81FF}{50-2 (‘Patrick’)}) spent little time in the fast-moving states 10, 12 and 14 relative to the fast-moving lines. Interestingly, the intermediate line \definecolor{38-2 (‘Mark’)_00C08B}{HTML}{00C08B}\textcolor{38-2 (‘Mark’)_00C08B}{38-2 (‘Mark’)} tended to transition into these states around the middle of the video, presumably a period of habituating to the new environment.



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/select_0.08_15_dge_of} \caption{Differences between MIKK F\textsubscript{0} lines in the HMM states they occupied during the \emph{open field} assay component. \textbf{A}: 15 HMM states with panels coloured red to indicate significant differences between MIKK F\textsubscript{0} lines in the proportion of time spent in those states. \textbf{B}: Transitions between HMM states across time for each individual test fish, grouped by MIKK line Tiles are coloured by the state most frequently occupied by each fish within 2-second intervals. \textbf{C}: Densities within each MIKK line for the occupation of states that significantly differed between strains (colour), with other states consolidated (grey).}\label{fig:F2-time-dge-of}
\end{figure}

During the novel object component the HMM states that showed significant differences between lines were mostly restricted to the slow-moving states with the exception of state 10, which most clearly distinguishes the slow-moving lines from the intermediate- and fast-moving lines. Again, the intermediate line \definecolor{38-2 (‘Mark’)_00C08B}{HTML}{00C08B}\textcolor{38-2 (‘Mark’)_00C08B}{38-2 (‘Mark’)} shows a sharp drop in the occupation of the slow moving states after a period of habituation.



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/select_0.08_15_dge_no} \caption{Differences between MIKK F\textsubscript{0} lines in the HMM states they occupied during the \emph{novel object} assay component. \textbf{A}: 15 HMM states with panels coloured red to indicate significant differences between MIKK F\textsubscript{0} lines in the proportion of time spent in those states. \textbf{B}: Transitions between HMM states across time for each individual test fish, grouped by MIKK line Tiles are coloured by the state most frequently occupied by each fish within 2-second intervals. \textbf{C}: Densities within each MIKK line for the occupation of states that significantly differed between strains (colour), with other states consolidated (grey).}\label{fig:F2-time-dge-no}
\end{figure}

\hypertarget{social-genetic-effects-1}{%
\subsection{Social genetic effects}\label{social-genetic-effects-1}}

To confirm whether the \emph{\definecolor{iCab_424B4D}{HTML}{424B4D}\textcolor{iCab_424B4D}{iCab}} reference fishes altered their behaviour depending on the MIKK F\textsubscript{0} line of their tank partner, I carried out the same analysis and model as above using only data from the \emph{\definecolor{iCab_424B4D}{HTML}{424B4D}\textcolor{iCab_424B4D}{iCab}} reference fishes. The states that showed a significant difference across any of the variables included in the model are set out in \textbf{Table \ref{tab:mikk-sge-F0}}. The iCab reference fishes differed significantly in the proportion of time they spent in a given state depending on the strain of their tank partner (\(p\) \textless{} 0.05, FDR-adjusted) for 4 out of 15 states in the open field assay (1.26x10\textsuperscript{-2} \textless{} \(p\) \textless{} 4x10\textsuperscript{-2}), and 4 out of 15 states for the novel object assay (7.71x10\textsuperscript{-3} \textless{} \(p\) \textless{} 1.34x10\textsuperscript{-2}). The line of the tank partner explained up to \textasciitilde29\% of the variance in the proportion of time the \emph{\definecolor{iCab_424B4D}{HTML}{424B4D}\textcolor{iCab_424B4D}{iCab}} reference spent in a given state. Full tables for all states and variables are provided in the Supplementary Material.

\begin{table}

\caption{\label{tab:mikk-sge-F0}Significant differences in the proportion of time spent in each HMM state by iCab reference fishes depending on the MIKK F0 line of their tank partner during the open field and novel object assay components.}
\centering
\begin{tabular}[t]{llll}
\toprule
Assay & State & Variance explained (\%) & p-value (FDR-adjusted)\\
\midrule
\cellcolor{gray!6}{open field} & \cellcolor{gray!6}{3} & \cellcolor{gray!6}{14.93} & \cellcolor{gray!6}{2.10e-02}\\
open field & 4 & 13.60 & 2.28e-02\\
\cellcolor{gray!6}{open field} & \cellcolor{gray!6}{5} & \cellcolor{gray!6}{15.33} & \cellcolor{gray!6}{1.26e-02}\\
open field & 7 & 14.26 & 4.00e-02\\
\cellcolor{gray!6}{novel object} & \cellcolor{gray!6}{4} & \cellcolor{gray!6}{14.33} & \cellcolor{gray!6}{1.23e-02}\\
\addlinespace
novel object & 5 & 14.38 & 1.15e-02\\
\cellcolor{gray!6}{novel object} & \cellcolor{gray!6}{9} & \cellcolor{gray!6}{24.61} & \cellcolor{gray!6}{1.34e-02}\\
novel object & 11 & 28.77 & 7.71e-03\\
\bottomrule
\end{tabular}
\end{table}



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/select_0.08_15_sge_of} \caption{Differences between MIKK F\textsubscript{0} lines in the HMM states they occupied during the \emph{open field} assay component. \textbf{A}: 15 HMM states with panels coloured red to indicate significant differences between MIKK F\textsubscript{0} lines in the proportion of time spent in those states. \textbf{B}: Transitions between HMM states across time for each individual test fish, grouped by MIKK line Tiles are coloured by the state most frequently occupied by each fish within 2-second intervals. \textbf{C}: Densities within each MIKK line for the occupation of states that significantly differed between strains (colour), with other states consolidated (grey).}\label{fig:F2-time-sge-of}
\end{figure}



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/select_0.08_15_sge_no} \caption{Differences between MIKK F\textsubscript{0} lines in the HMM states they occupied during the \emph{novel object} assay component. \textbf{A}: 15 HMM states with panels coloured red to indicate significant differences between MIKK F\textsubscript{0} lines in the proportion of time spent in those states. \textbf{B}: Transitions between HMM states across time for each individual test fish, grouped by MIKK line Tiles are coloured by the state most frequently occupied by each fish within 2-second intervals. \textbf{C}: Densities within each MIKK line for the occupation of states that significantly differed between strains (colour), with other states consolidated (grey).}\label{fig:F2-time-sge-no}
\end{figure}

To assist with interpretation of the relationships between the HMM states and the types of behaviours exhibited, I have provided in \textbf{Appendix \ref{four-panel-app}} four-panel plots showing a) a frame grab from the raw four-quadrant video; b) path plots for both fishes; c) path plots for the test fishes coloured by HMM state; and d) path plots for the reference fishes coloured by HMM state, all taken at the 300-second mark (5 minutes) of each 10-minute assay component.

\clearpage

\hypertarget{f2-generation}{%
\section{\texorpdfstring{F\textsubscript{2} generation}{F2 generation}}\label{f2-generation}}

\hypertarget{behavioural-data-collection}{%
\subsection{Behavioural data collection}\label{behavioural-data-collection}}

Around August 2019, our collaborators in the Loosli Group at KIT commenced the breeding program for this experiment with the 6 MIKK panel lines I had selected above. The goal was to generate around 100 F\textsubscript{2} individuals per cross, for a total F\textsubscript{2} sample size of around 1000, and to run genetic association analyses for each cross separately, as well as all crosses together. However, from around mid-2020, the F\textsubscript{1} generation began to reproduce very poorly, and after lengthy investigations, our collaborators discovered that the Covid pandemic had caused disruptions to the supply chains of their fish food manufacturer, which had compelled the manufacturer to modify the recipe, and this modification was the cause of the poor breeding. While the issue has since been resolved, it has resulted in a much smaller number of F\textsubscript{2} individuals to be produced than was originally anticipated. As a consequence, I was compelled to analysis the dataset in the style of a joint pedigree (that is, a multi-way F\textsubscript{2} cross), as has been performed with farm animals such as the pig (Tortereau et al. 2010; Tang et al. 2017).

From 17 November 2021 to 5 May 2022, a Research Assistant in Prof.~Loosli's lab, Alicia Günthel, performed the assay 69 time with F\textsubscript{2} individuals from the 12 crosses they had bred, producing a total of 271 videos of pairs of fish. The counts for the 12 crosses used to generate these 271 test fishes are set out in \textbf{Table \ref{tab:F2-cross-counts}}. This much smaller F\textsubscript{2} sample size prevented us from analysing each cross separately (as one would do for a traditional F\textsubscript{2} cross), and so I was restricted to performing only the joint analysis in the similar way to a farm animal pedigree (REFER TO RELEVANT INTRO SECTION).

\begin{table}

\caption{\label{tab:F2-cross-counts}Significant differences in the proportion of time spent in each HMM state by iCab reference fishes depending on the MIKK F~0~ line of their tank partner during the open field and novel object assay components.}
\centering
\begin{tabular}[t]{llr}
\toprule
paternal line & maternal line & count\\
\midrule
\cellcolor{gray!6}{21-2} & \cellcolor{gray!6}{40-1} & \cellcolor{gray!6}{60}\\
38-2 & 40-1 & 57\\
\cellcolor{gray!6}{38-2} & \cellcolor{gray!6}{18-2} & \cellcolor{gray!6}{35}\\
8-2 & 40-1 & 24\\
\cellcolor{gray!6}{50-2} & \cellcolor{gray!6}{18-2} & \cellcolor{gray!6}{23}\\
\addlinespace
38-2 & 21-2 & 19\\
\cellcolor{gray!6}{8-2} & \cellcolor{gray!6}{38-2} & \cellcolor{gray!6}{15}\\
50-2 & 38-2 & 12\\
\cellcolor{gray!6}{18-2} & \cellcolor{gray!6}{21-2} & \cellcolor{gray!6}{7}\\
21-2 & 50-2 & 7\\
\addlinespace
\cellcolor{gray!6}{40-1} & \cellcolor{gray!6}{50-2} & \cellcolor{gray!6}{6}\\
50-2 & 8-2 & 6\\
\bottomrule
\end{tabular}
\end{table}

I again used \emph{idtrackerai} (Romero-Ferrero et al. 2019) to track the F\textsubscript{2} individuals across frames. When splitting the 271 videos of pairs of fish into their open field and novel object assay components for a total of 542 videos, for 526 of them both fish were tracked across at least 85\% of frames. I only used these 526 videos in the downstream analysis, but prior to publication I will seek to improve the tracking process so that these individuals can be included.

\hypertarget{behavioural-phenotyping}{%
\subsection{Behavioural phenotyping}\label{behavioural-phenotyping}}

To ensure that the predicted HMM states for behaviour were consistent across both F\textsubscript{0} and F\textsubscript{2} generations, I had included these F\textsubscript{2} individuals for training and prediction in the models described above. I again calculated the proportions of time that every individual spent in each state (``\textbf{state frequency}''), then inverse-normalised\footnote{See section \ref{inverse-norm-sec} in Chapter \ref{Somite-chap} for a description of this process.} the values within each combination of assay component (open field or novel object) and state (1 to 15).

\textbf{Figures \ref{fig:F2-state-freq-dge}} and \textbf{\ref{fig:F2-state-freq-sge}} compares the phenotype pre- and post-transformation with this normalisation approach. For the higher-movement states there are increasing numbers of individuals who spent no time in those states, which are responsible for the apparently non-normal distributions observed for the skewed distributions post-transformation. States 1, 3, 6, 8, 9 and 11 already appear to have a large amount of variation between individuals, but this normalisation process will increase the variance for states where there is small, yet potentially meaningful, variation between individuals. One exception may perhaps be state 15, which involves very large distances of travel in all directions, and therefore may correspond to tracking errors.



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/0.08_15_state_freq_F2_dge} \caption{Effect of inverse-normalisation on the HMM state frequency of the F\textsubscript{2} test fishes.}\label{fig:F2-state-freq-dge}
\end{figure}



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/0.08_15_state_freq_F2_sge} \caption{Effect of inverse-normalisation on the HMM state frequency of the F\textsubscript{2} \emph{\definecolor{iCab_424B4D}{HTML}{424B4D}\textcolor{iCab_424B4D}{iCab}} reference fishes.}\label{fig:F2-state-freq-sge}
\end{figure}

\hypertarget{genotyping}{%
\subsection{Genotyping}\label{genotyping}}

After phenotyping the F\textsubscript{2} samples, our collaborators in the Loosli Group at KIT took finclips from the F\textsubscript{2} individuals, extracted their DNA, and arranged for them to be shallow-sequenced on the Illumina short-read platform at a coverage of \textasciitilde1x per sample. Using a similar method to what I described in Chapter \ref{Somite-chap}, I aligned these sequences to the \emph{HdrR} reference with BWA-MEM2 (Vasimuddin et al. 2019), sorted the reads and marked duplicates with Picard ({``Picard Toolkit''} 2019), then indexed the resulting BAM files with samtools (Danecek et al. 2021). I then used \emph{bam-readcount} (Khanna et al. 2022) to count the reads that supported either the paternal or the maternal allele for all biallelic SNPs that were homozygous-divergent in the given sample's two parental strains (i.e.~homozgyous reference in the paternal line, and homozygous alternative in the maternal line, or \emph{vice versa}), summed the read counts within 5 kb blocks, and calculated the frequency of reads within each bin that supported the maternal allele. This generated a value for each bin between 0 and 1, where 0 signified that all reads within that bin supported the paternal allele, and 1 signified that all reads within that bin supported the maternal allele. Bins containing no reads were imputed with a value of 0.5.

I then used these values for all F\textsubscript{2} individuals as the input to a Hidden Markov Model (HMM) with the software package \emph{hmmlearn} (\emph{Hmmlearn/Hmmlearn} {[}2014{]} 2022), which I applied to classify each bin as one of three states, with state 0 corresponding to homozygous for the paternal allele, 1 corresponding to heterozygous, and 2 corresponding to homozygous for the maternal allele. Across each chromosome of every sample, the output of the HMM was expected to produce a sequence of states. Based on previous analyses described in Chapter \ref{Somite-chap}, I used the same HMM parameters as I did there, and used the HMM state outputs to generate the recombination blocks shown in \textbf{Figure \ref{fig:F2-recomb-blocks}}. Missing genotype state calls arose from a sample having insufficient reads within a bin, for which I imputed the missing state calls within each sample's chromosome based on the previous state call on that chromosome. A karyoplot retaining the missing blocks is provided in \textbf{Appendix \ref{fig:F2-recomb-blocks-missing}}. Unfortunately, there appear to be many missing HMM state calls for chromosomes 22 and 23, and this may be caused by the small number of homozygous-divergent variants that reside in those chromosomes, as I explore below in the Conclusions chapter of this thesis.



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/karyoplot_no_missing} \caption{Karyoplot for F\textsubscript{2} samples, coloured by genotype. Samples are sorted in the order in which they were phenotyped. Blocks are filled with the colour of the paternal F\textsubscript{0} line for the homozygous paternal haplotype block, black for heterozygous, and the colour of the maternal F\textsubscript{0} line for the homozygous maternal haplotype block. Missing calls were imputed based on the previous successful call on a given chromosome.}\label{fig:F2-recomb-blocks}
\end{figure}

\textbf{Figure \ref{fig:F2-prop-sites}} sets out the genome-wide frequency of bins that were called as either homozygous paternal, heterozygous, or homozygous maternal for each individual, grouped by cross. In accordance with Mendel's law of segregation, the expected frequencies are 0.25, 0.5, and 0.25 respectively. Despite the relatively low sample sizes for each cross, the observed frequencies are generally consistent with these expected frequencies, with notable exceptions for the crosses involving \definecolor{21-2 (‘Amelia’)_49B500}{HTML}{49B500}\textcolor{21-2 (‘Amelia’)_49B500}{21-2 (‘Amelia’)}, where that line contributed a frequency of nearly 0 bins in the four crosses it was used for. This may have been caused by some hybrid incompatibility between line \definecolor{21-2 (‘Amelia’)_49B500}{HTML}{49B500}\textcolor{21-2 (‘Amelia’)_49B500}{21-2 (‘Amelia’)} and the other MIKK panel lines used in these crosses, where an excess of genes from line \definecolor{21-2 (‘Amelia’)_49B500}{HTML}{49B500}\textcolor{21-2 (‘Amelia’)_49B500}{21-2 (‘Amelia’)} rendered the individual non-viable, and therefore did not reach the developmental stage where they could be assayed. There also appears to be an issue with the cross between \definecolor{50-2 (‘Patrick’)_BB81FF}{HTML}{BB81FF}\textcolor{50-2 (‘Patrick’)_BB81FF}{50-2 (‘Patrick’)} and \definecolor{8-2 (‘Gail’)_FF699C}{HTML}{FF699C}\textcolor{8-2 (‘Gail’)_FF699C}{8-2 (‘Gail’)}, where the genotypes are skewed away from \definecolor{50-2 (‘Patrick’)_BB81FF}{HTML}{BB81FF}\textcolor{50-2 (‘Patrick’)_BB81FF}{50-2 (‘Patrick’)}, although the small sample size for that cross makes it difficult to confirm.



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/prop_sites} \caption{Frequencies of bins within each individual called as either homozygous for the paternal allele, heterozygous, or homozygous for the maternal allele.}\label{fig:F2-prop-sites}
\end{figure}

I was nevertheless satisfied that the concordance between our observed genotypes and their expected frequencies indicated that our HMM genotyping was sufficiently robust. I therefore proceeded to use these HMM-generated haplotype block calls to impute each sample's SNP-level genotypes using the homozygous biallelic SNP calls in the high-coverage .vcf file for the MIKK panel F\textsubscript{0} lines described in Chapter \ref{MIKK-genomes-chap}. This set of variants included a total of \textasciitilde20.7M SNPs, which I used to generate a Plink-format .bed file, forming the genotype input for the genetic linkage analysis.

\hypertarget{f2-cross-gwas}{%
\subsection{\texorpdfstring{F\textsubscript{2}-cross GWAS}{F2-cross GWAS}}\label{f2-cross-gwas}}

\hypertarget{grm}{%
\subsubsection{GRM}\label{grm}}

For the purpose of using the \emph{GCTA} software package (Yang, Lee, et al. 2011) for the genetic association testing, I was required to construct a genetic relationship matrix (\textbf{GRM}) in the manner described in the Introduction (Equations \eqref{eq:grmstd} and \eqref{eq:grm}). I first filtered the .bed for SNPs that had no missing calls for any samples (\(M_{SNPs}\) = 44,360). I then used these SNPs to construct a ``leave-one-chromosome-out'' (\textbf{LOCO}) genetic relationship matrix for each chromosome -- that is, if the ``focal'' chromosome was chr1, I would exclude the SNPs on that chromosome before constructing the GRM. To illustrate, \textbf{Figure \ref{fig:F2-grm}} is a GRM constructed using all 44,360 non-missing SNPs. However, given the relatively small amount of non-missing SNPs on each chromosome, the number of SNPs on the focal chromosome that were excluded was small, resulting in GRMs that appear almost identical by eye -- see the LOCO-GRM for chromosome 1 in \textbf{Appendix \ref{fig:loco-grm-chr1}}. In contrast to the GRM presented in the previous Chapter (Figure \ref{fig:somite-grm}), which was based on a single F\textsubscript{2} cross and looked relatively homogenous, here the individuals from each separate cross neatly cluster together, and the individuals that share one parental strain cluster nearby.



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/grm_man_0.8} \caption{Genetic relationship matrix for 271 F\textsubscript{2} samples based on 44,360 non-missing SNPs.}\label{fig:F2-grm}
\end{figure}

\hypertarget{heritability}{%
\subsubsection{Heritability}\label{heritability}}

I used the above GRM with GCTA (Yang, Lee, et al. 2011) to calculate the narrow-sense heritability (\(h^2\)) of the inverse-normalised proportions of time the fish spent in each state (direct genetic effects), and for the proportions of time their tank partners spent in each state (social genetic effects) (\textbf{Figure \ref{fig:F2-heritability}}). For direct genetic effects (\textbf{Figure \ref{fig:F2-heritability}A}), the maximum estimates are smaller than expected, with a maximum \(h^2\) of 0.173 for state 2 during the open field assay component. There is also higher heritability detected through the open field assay than the novel object assay. Interestingly, the somewhat wider- and forward-moving state 6 shows no heritability, which indicates that the proportions of time spent in that state is not affected by genetics. The heritability estimates for social genetic effects (\textbf{Figure \ref{fig:F2-heritability}B}) are near 0, with a slight exception for the straighter- and forward-moving state 8 in the novel object assay, which also showed significant differences between the F\textsubscript{0} lines.



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/heritability} \caption{Narrow-sense heritability (\(h^2\)) computed by GCTA (Yang, Lee, et al. 2011; Yang, Manolio, et al. 2011) based on all non-missing SNPs used to generate the GRM (\(N\) = 44,360), for direct genetic effects (\textbf{A}) and social genetic effects (\textbf{B}).}\label{fig:F2-heritability}
\end{figure}

\hypertarget{gwas-1}{%
\subsubsection{GWAS}\label{gwas-1}}

For the association testing, I used a linear mixed model (\textbf{LMM}) as implemented in GCTA (Yang, Lee, et al. 2011). For the phenotype, I used the inverse-normalised state frequencies described above, and the LOCO-GRM for all SNPs on the focal chromosome. That is to say, for all SNPs on chr18, I used the LOCO-GRM that excluded all SNPs from that chromosome. In addition, I included the time of assay and the tank quadrant as covariates, which the software regresses out from the phenotype prior to running the LMM I excluded the date of assay and the tank side as covariates because individuals from the same cross tended to be assayed in the same test tank on the same day, and are therefore confounded with their genetics.

To set a significance threshold, for each combination of HMM state and assay, I ran 10 LMM tests over the same dataset where I had permuted the phenotype and covariates using a different random seed. The logic behind this method is to determine the lowest p-value that one could expect when there is no true relationship between the individuals' genetics and their phenotype. I then extracted the smallest p-value from all 10 results, and used this as the significance threshold. I additionally calculated the Bonferroni threshold as \(\alpha / M\), where \(\alpha\) is set to 0.05 and \(M\) is the total number of SNPs in the dataset (2,726,797) = 1.83 x 10\textsuperscript{-8}.

\hypertarget{direct-genetic-effects-2}{%
\subsubsection{Direct genetic effects}\label{direct-genetic-effects-2}}

For the DGE phenotypes (comparing the state frequencies across test fishes), I sought to identify the genetic loci in the F\textsubscript{2} individuals that were associated with differences in their own behaviour. For neither the open field nor novel object component did the p-values exceed the Bonferroni threshold, however a number of loci across several chromosomes exceeded the thresholds set by the permutations. I plotted the p-values in Manhattan plots for each combination of state and assay component, and provide them all in the Appendix. Here I showcase the Manhattan plot for state 3, containing a number of significant loci that I discuss further below.



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/manhattans/dge_no_3_time-quadrant} \caption{Manhattan plot for inverse-normalised HMM state 3 frequency of F\textsubscript{2} test fishes during the novel object assay.}\label{fig:F2-man-dge-no-3}
\end{figure}

\hypertarget{social-genetic-effects-2}{%
\subsubsection{Social genetic effects}\label{social-genetic-effects-2}}

For social genetic effects, I was attempting to discover genetic variants in the F\textsubscript{2} individuals that caused differences in the behaviour of their \emph{\definecolor{iCab_424B4D}{HTML}{424B4D}\textcolor{iCab_424B4D}{iCab}} tank partners. As expected, fewer loci reached significance for this transmitted indirect genetic effect than for the direct genetic effects, however I still found several significant loci based on the permutations threshold for states 4 (chr4), 7 (chr1), 12 (chr11), and 13 (chr12) during the open field assay component; but only one, barely significant SNP for state 5 (chr 13) during the novel object assay. This difference was somewhat surprising given our previous suppositions in Chapter \ref{Pilot-chap} that the novel object component drew out stronger social genetic effects. One possible explanation is that even though the differences between line pairings appear more pronounced in the novel object assay component, it is more difficult to map variants associated with these differences because a greater proportion of the variance is driven by non-genetic sources.



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/manhattans/sge_of_13_time-quadrant} \caption{Manhattan plot for inverse-normalised HMM state 3 frequency of \emph{\definecolor{iCab_424B4D}{HTML}{424B4D}\textcolor{iCab_424B4D}{iCab}} reference fishes during the novel object assay component.}\label{fig:F2-man-sge-of-13}
\end{figure}

\hypertarget{candidate-loci-for-crispr-knockouts}{%
\subsection{Candidate loci for CRISPR-knockouts}\label{candidate-loci-for-crispr-knockouts}}

\hypertarget{missense-variants}{%
\subsubsection{Missense variants}\label{missense-variants}}

The next goal was to attempt to identify the SNPs that are mostly likely to be the causal variants, or closest in proximity to the causal variants, that are responsible for differences in the phenotypes of interest. I therefore proceeded to extract the significant SNPs and use Ensembl's Variant Effect Predictor (\textbf{VEP}) (W. McLaren et al. 2016) to identify the SNPs that are most likely to have a functional consequence, and determine whether the genes they reside in have already been identified as being involved in biological pathways that could be related to behaviour. \textbf{Table \ref{tab:F2-sig-snps-consq-counts}} shows the counts for the different potential consequences of the unique significant SNPs, based on Ensembl's annotation of the \emph{HdrR} reference genome. I note that as multiple genes can overlap the same locus (including those transcribed from opposing strands), one SNP may map to multiple genes and therefore be counted as having more than one type of consequence.

\begin{table}

\caption{\label{tab:F2-sig-snps-consq-counts}Counts for consequences of significant SNPs.}
\centering
\begin{tabular}[t]{lr}
\toprule
consequence & count\\
\midrule
\cellcolor{gray!6}{intron\_variant} & \cellcolor{gray!6}{349}\\
intergenic\_variant & 178\\
\cellcolor{gray!6}{upstream\_gene\_variant} & \cellcolor{gray!6}{92}\\
downstream\_gene\_variant & 75\\
\cellcolor{gray!6}{synonymous\_variant} & \cellcolor{gray!6}{9}\\
\addlinespace
3\_prime\_UTR\_variant & 6\\
\cellcolor{gray!6}{missense\_variant} & \cellcolor{gray!6}{6}\\
5\_prime\_UTR\_variant & 3\\
\cellcolor{gray!6}{splice\_acceptor\_variant} & \cellcolor{gray!6}{2}\\
splice\_region\_variant,intron\_variant & 2\\
\addlinespace
\cellcolor{gray!6}{splice\_region\_variant,synonymous\_variant} & \cellcolor{gray!6}{2}\\
splice\_donor\_variant & 1\\
\bottomrule
\end{tabular}
\end{table}

As expected, most variants reside in non-coding regions {[}CITE{]}, but in an attempt to isolate the most likely causative SNPs, I extracted those that are predicted to cause a missense mutation and therefore most likely to have a functional consequence (\textbf{Table \ref{tab:F2-sig-snps-missense}}).

\begin{table}

\caption{\label{tab:F2-sig-snps-missense}Missense SNPs.}
\centering
\resizebox{\linewidth}{!}{
\begin{tabular}[t]{llrrrlllll}
\toprule
Genetic effect & Assay & State & Chr & Pos & Ref & Alt & Allele & Gene & Description\\
\midrule
\cellcolor{gray!6}{DGE} & \cellcolor{gray!6}{open field} & \cellcolor{gray!6}{1} & \cellcolor{gray!6}{1} & \cellcolor{gray!6}{22508288} & \cellcolor{gray!6}{A} & \cellcolor{gray!6}{C} & \cellcolor{gray!6}{C} & \cellcolor{gray!6}{ENSORLG00000009385} & \cellcolor{gray!6}{vav guanine nucleotide exchange factor 1}\\
DGE & open field & 7 & 13 & 33714439 & G & T & T & ENSORLG00000028050 & uncharacterized LOC105356481\\
\cellcolor{gray!6}{DGE} & \cellcolor{gray!6}{open field} & \cellcolor{gray!6}{10} & \cellcolor{gray!6}{3} & \cellcolor{gray!6}{28342550} & \cellcolor{gray!6}{A} & \cellcolor{gray!6}{G} & \cellcolor{gray!6}{G} & \cellcolor{gray!6}{ENSORLG00000024879} & \cellcolor{gray!6}{}\\
DGE & novel object & 3 & 1 & 31428698 & G & A & A & ENSORLG00000013962 & ecto-ADP-ribosyltransferase 5-like\\
\cellcolor{gray!6}{DGE} & \cellcolor{gray!6}{novel object} & \cellcolor{gray!6}{3} & \cellcolor{gray!6}{10} & \cellcolor{gray!6}{15319434} & \cellcolor{gray!6}{C} & \cellcolor{gray!6}{T} & \cellcolor{gray!6}{T} & \cellcolor{gray!6}{ENSORLG00000029574} & \cellcolor{gray!6}{}\\
\addlinespace
DGE & novel object & 3 & 10 & 15319434 & C & T & T & ENSORLG00000024866 & protocadherin alpha-C2-like\\
\cellcolor{gray!6}{DGE} & \cellcolor{gray!6}{novel object} & \cellcolor{gray!6}{10} & \cellcolor{gray!6}{10} & \cellcolor{gray!6}{15319434} & \cellcolor{gray!6}{C} & \cellcolor{gray!6}{T} & \cellcolor{gray!6}{T} & \cellcolor{gray!6}{ENSORLG00000029574} & \cellcolor{gray!6}{}\\
DGE & novel object & 10 & 10 & 15319434 & C & T & T & ENSORLG00000024866 & protocadherin alpha-C2-like\\
\cellcolor{gray!6}{SGE} & \cellcolor{gray!6}{open field} & \cellcolor{gray!6}{13} & \cellcolor{gray!6}{12} & \cellcolor{gray!6}{5136828} & \cellcolor{gray!6}{G} & \cellcolor{gray!6}{A} & \cellcolor{gray!6}{A} & \cellcolor{gray!6}{ENSORLG00000002961} & \cellcolor{gray!6}{complement C9}\\
\bottomrule
\end{tabular}}
\end{table}

One SNP, 10:15,319,434, was significant in respect of DGE for frequencies in both state 3 (\textbf{Figure \ref{fig:sig-snp-10-15mb}}) and 10 during the novel object assay (see \textbf{Figure \ref{fig:F2-man-dge-no-3}} above for state 3), and maps to two genes: ENSORLG00000029574 and ENSORLG00000024866. ENSORLG00000029574 is a novel gene on the forward strand with no recorded phenotypes. Based on discussions with an expert on genetic prediction, it is most likely that this was a mis-prediction. However, on the reverse strand of this locus is ENSORLG00000024866, a gene for protocadherin alpha-C2-like, a well-known protein involved in mammalian synapse formation (Junghans et al. 2008; Phillips et al. 2003).



\begin{figure}

{\centering \includegraphics[width=0.5\linewidth]{figs/mikk_behaviour/sig_snps_boxplots/3-10:15319434} 

}

\caption{State 3 frequency during the novel object assay for counts of the alternative allele (T) at SNP 10:15,319,434 in the protocadherin gene. The boxes are coloured on the left by the paternal line, and on the right by the maternal line.}\label{fig:sig-snp-10-15mb}
\end{figure}

The second predicted missense variant of note was detected for SGE in the open field assay for frequencies in state 13. The locus 12:5,136,828 maps to gene ENSORLG00000002961, which is described as a complement \emph{C9} (\textbf{Figure \ref{fig:sig-snp-12-5mb}}). Discovered phenotypes for species orthologues include reduced prepulse inhibition and heart phenotypes in mice (Mouse Genome Database, Blake et al. 2021), and neurodevelopmental disorders in rats (Rat Genome Database, J. R. Smith et al. 2020). Here, in the F\textsubscript{2} individuals, it appears to have a complex effect on the behaviour of the test fishes between the crosses, where it apparently causes the reference fish to increase the proportion of time it spends in the fast-moving state 13 (\textbf{Figure \ref{fig:F2-man-sge-of-13}}).



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/sig_snps_boxplots/13-12:5136828} \caption{State 13 frequency during the open field assay for counts of the alternative allele (A) at SNP 12:5,136,828. The boxes are coloured on the left by the paternal line, and on the right by the maternal line.}\label{fig:sig-snp-12-5mb}
\end{figure}

The evidence from these loci is perhaps not so persuasive. For the first missense SNP at 10:15,319,434, only one cross has data for more than one genotype, and is therefore informative (DGE, 38-2 x 18-2). For the second missense SNP at 12:5,136,828, although three crosses are informative, there are few data points for two of them (18-2 x 21-2 and 21-2 x 50-2), the latter of which is showing an effect in the opposite direction to the other two. Like the first missense SNP at 10:15,319,434, many significant SNPs are missing calls for a number of crosses. With the exception of chromosomes 22 and 23 (which are often missing HMM state calls, making it impossible to genotype those individuals), for all other chromosomes I imputed the genotypes directly from the high-coverage MIKK panel .vcf file, so if any crosses are missing genotypes, it must be because there are missing genotype calls for either of their two parental strains in that .vcf file. This in turn may have been caused by a failure to detect larger genomic variation with the short-read Illumina platform that was used to sequence them, and may therefore be overcome by combining those short reads with Oxford Nanopore Technologies long-read sequences as I did in Chapter \ref{mikksv-sec}.

\hypertarget{other-potentially-causal-variants}{%
\subsubsection{Other potentially causal variants}\label{other-potentially-causal-variants}}

There are additional significant loci of note shown in \textbf{Figure \ref{fig:F2-man-dge-no-3}} above that are not predicted to alter proteins, but nevertheless appear to affect genes related to neurological development. \textbf{Table \ref{tab:tbl-sig-snp-chr-9-10}} sets out five of these loci, followed by a brief description of each.

\begin{table}

\caption{\label{tab:tbl-sig-snp-chr-9-10}Notable significant SNPs for direct genetic effects.}
\centering
\resizebox{\linewidth}{!}{
\begin{tabular}[t]{llrrrlllll}
\toprule
Genetic effect & Assay & State & Chr & Pos & Ref & Alt & Gene & Description & Human homologue\\
\midrule
\cellcolor{gray!6}{DGE} & \cellcolor{gray!6}{open field} & \cellcolor{gray!6}{1} & \cellcolor{gray!6}{18} & \cellcolor{gray!6}{15178810} & \cellcolor{gray!6}{G} & \cellcolor{gray!6}{T} & \cellcolor{gray!6}{ENSORLG00000006940} & \cellcolor{gray!6}{RELT like 1} & \cellcolor{gray!6}{RELL1}\\
DGE & open field & 8 & 5 & 24433800 & C & A & ENSORLG00000012418 & claudin 19 & CLDN19\\
\cellcolor{gray!6}{DGE} & \cellcolor{gray!6}{novel object} & \cellcolor{gray!6}{3} & \cellcolor{gray!6}{9} & \cellcolor{gray!6}{9802754} & \cellcolor{gray!6}{C} & \cellcolor{gray!6}{A} & \cellcolor{gray!6}{ENSORLG00000024663} & \cellcolor{gray!6}{glutamate ionotropic receptor delta type subunit 2} & \cellcolor{gray!6}{GRID2}\\
DGE & novel object & 3 & 10 & 18537719 & C & A & ENSORLG00000006464 & neuroligin 3a & NLGN3\\
\cellcolor{gray!6}{DGE} & \cellcolor{gray!6}{novel object} & \cellcolor{gray!6}{9} & \cellcolor{gray!6}{6} & \cellcolor{gray!6}{30807013} & \cellcolor{gray!6}{T} & \cellcolor{gray!6}{A} & \cellcolor{gray!6}{ENSORLG00000016329} & \cellcolor{gray!6}{pannexin 2} & \cellcolor{gray!6}{PANX2}\\
\bottomrule
\end{tabular}}
\end{table}

\hypertarget{rell1}{%
\paragraph{\texorpdfstring{\emph{RELL1}}{RELL1}}\label{rell1}}

The locus around 18:15,178,810 is situated within in the RELT like 1 (\emph{RELL1}) gene (ENSORLG00000006940). Values for the frequency of time spent in HMM state 2 during the open field assay based on the genotypes for this locus is set out in \textbf{Figure \ref{fig:sig-snp-18-15mb}}. The orthologue for this gene has been found to be associated with abnormal auditory brainstem response and hyperactivity in mouse (Mouse Genome Database, Blake et al. 2021).



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/sig_snps_boxplots/2-18:15178810} \caption{State 2 frequency during the open field assay for counts of the alternative allele (T) at 18:15,178,810. The boxes are coloured on the left by the paternal line, and on the right by the maternal line.}\label{fig:sig-snp-18-15mb}
\end{figure}

\hypertarget{cldn19}{%
\paragraph{\texorpdfstring{\emph{CLDN19}}{CLDN19}}\label{cldn19}}

The locus around 5:24,433,800 maps to the claudin 19 gene (\emph{cldn19}, ENSORLG00000012418), a projected zinc finger. Values for the frequency of time spent in HMM state 2 during the open field assay based on the genotypes for this locus is set out in \textbf{Figure \ref{fig:sig-snp-5-24mb}}. Orthologues of this gene have been associated with neurological traits including impaired coordination and abnormal gait in mice (Mouse Genome Database, Blake et al. 2021).



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/sig_snps_boxplots/8-5:24433800} \caption{State 8 frequency during the open field assay for counts of the alternative allele (A) at 5:24,433,800. The boxes are coloured on the left by the paternal line, and on the right by the maternal line.}\label{fig:sig-snp-5-24mb}
\end{figure}

\hypertarget{grid2}{%
\paragraph{\texorpdfstring{\emph{GRID2}}{GRID2}}\label{grid2}}

The locus around 9:9,802,754 (\textbf{Figure \ref{fig:sig-snp-9-9mb}}) maps to a gene for glutamate ionotropic receptor delta type subunit 2 (\emph{grid2}, ENSORLG00000024663), an orthologue of the human gene \emph{GRID2}. Deletions in this gene have been found to cause ataxia in humans (Hills et al. 2013; Utine et al. 2013), and other mutations in this gene have been found to cause various neurological disorders in mice (Mouse Genome Database, Blake et al. 2021).



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/sig_snps_boxplots/3-9:9802754} \caption{State 3 frequency during the novel object assay for counts of the alternative allele (A) at SNP 9:9,802,754. The boxes are coloured on the left by the paternal line, and on the right by the maternal line.}\label{fig:sig-snp-9-9mb}
\end{figure}

\hypertarget{nlgn3}{%
\paragraph{\texorpdfstring{\emph{NLGN3}}{NLGN3}}\label{nlgn3}}

The locus around 10:18,537,719 (\textbf{Figure \ref{fig:sig-snp-10-18mb}}) sits within the neuroligin 3 gene (\emph{nlgn3a}, ENSORLG00000006464), an orthologue of the human gene \emph{NLGN3}, which has been linked with autism in humans (Jamain et al. 2003), and various neurological in mice and rats (Blake et al. 2021; J. R. Smith et al. 2020).



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/sig_snps_boxplots/3-10:18537719} \caption{State 3 frequency during the novel object assay for counts of the alternative allele (A) at SNP 10:18,537,719. The boxes are coloured on the left by the paternal line, and on the right by the maternal line.}\label{fig:sig-snp-10-18mb}
\end{figure}

\hypertarget{panx2}{%
\paragraph{\texorpdfstring{\emph{PANX2}}{PANX2}}\label{panx2}}

The locus around 6:30,807,013 in the pannexin 2 (\emph{panx2}) gene (ENSORLG00000016329), a projected zinc finger orthologue with the human gene \emph{PANX2} (\textbf{Figure \ref{fig:sig-snp-6-30mb}}). Its orthologues have been associated with decreased prepulse inhibition in mice (Blake et al. 2021), and intellectual disability and autism spectrum disorder in rats (J. R. Smith et al. 2020).



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/sig_snps_boxplots/9-6:30807013} \caption{State 9 frequency during the novel object assay for counts of the alternative allele (A) at 6:30,807,013. The boxes are coloured on the left by the paternal line, and on the right by the maternal line.}\label{fig:sig-snp-6-30mb}
\end{figure}

\hypertarget{discussion}{%
\section{Discussion}\label{discussion}}

These results of this chapter confirm that the F\textsubscript{2} cross method applied to the MIKK panel is a powerful technique for the discovery of loci associated with complex traits. I applied modern computational methods to first develop a data-driven analysis of fish behaviour, and then select a subset of inbred lines that diverged in both their own behaviour, and the extent to which they transmitted their behaviours to their social partners. I then performed an F\textsubscript{2} cross with those lines to map the genetic variants associated with those differences. Although this analysis was vastly under-powered, I nevertheless discovered a number of statistically significant loci in orthologue genes that have been found to be related to neurological phenotypes in mice, rats, and humans. Future work can functionally validate these loci with targeted CRISPR-Cas9 knockouts, expanding on our knowledge about the genetic and environmental contributions to trait variation through an improved ability to control both genes and social environment with inbred lines that are representative of the kinds of variation that is naturally occurring in the wild.

\hypertarget{Fst-chap}{%
\chapter{Variation in the frequency of trait-associated alleles across global human populations}\label{Fst-chap}}

\chaptermark{Human $F_{ST}$ distributions}

Thousands of GWAS have now been performed for more than 3,330 human traits (Uffelmann et al. 2021), identifying tens of thousands of loci that are associated with variation in these traits, together with the strengths of their effects (MacArthur et al. 2017). Researchers have used these data to develop polygenic scores (\textbf{PGS}), which can predict the values that individuals will have for a trait based on their genotypes (Lambert, Abraham, and Inouye 2019). However, as most GWAS have been performed on European populations, PGS predictions haven proven to be less robust when applied to non-European populations. The likely causes of this poor transferability of PGS are proposed to be some combination of differences between global populations in linkage disequilibrium patterns, trait-associated allele frequencies, and environment.
The purpose of the analysis in this Chapter was to explore the extent to which differences in trait-associated allele frequencies between diverse human populations was responsible for the poor portability of Polygenic Scores (\textbf{PGS}) to non-European populations. To achieve this, I sougght to extract all approximately-unlinked, trait-associated SNPs in the GWAS Catalog (MacArthur et al. 2017), and use \(F_{ST}\) to measure the differences in allele frequencies between the 26 diverse populations in the 1000 Genomes Project phase 3 release dataset (1000. G. P. Consortium et al. 2015). I then rank the traits based on the differences in the distributions of \(F_{ST}\) relative to an allele-frequency matched set of control SNPs, and find that with few exceptions (most notably, HIV viral load and race-related traits such as skin pigmentation), the diverse human populations do not differ substantially in their allele frequencies for these traits. This work provides evidence that the poor portability of PGS to non-European populations may not be attributable to differences in their allele frequencies, and may rather be attributable to differences in LD and environment.

In this Chapter, I describe my analysis of the distribution of \(F_{ST}\) for trait-associated alleles across diverse human populations, to explore the extent to which different allele frequencies contribute to the poor transferability of PGS to non-European populations.

\hypertarget{f_st-distribution-for-all-snps-in-the-human-genome}{%
\section{\texorpdfstring{\(F_{ST}\) distribution for all SNPs in the human genome}{F\_\{ST\} distribution for all SNPs in the human genome}}\label{f_st-distribution-for-all-snps-in-the-human-genome}}

I first sought to explore the distribution of \(F_{ST}\) for SNPs across the human genome. As the reference for human genomic variation across diverse populations, I used the New York Genome Center high-coverage, phased .vcf files for the 2,504 individuals from 26 populations from across the globe, as described in the 1000 Genomes phase 3 release (1000. G. P. Consortium et al. 2015). I then annotated those .vcf files with human SNP IDs from dbSNP release 9606 (Smigielski et al. 2000), and calculated \(F_{ST}\) for each of the \textasciitilde69M SNPs in that dataset with \emph{PLINK 1.9} (Chang et al. 2015; Purcell and Chang, n.d.). Figure \ref{fig:fst-circos} shows the location and \(F_{ST}\) value for all SNPs in the 1000 Genomes dataset, and Figure \ref{fig:fst-histo} shows their distribution across the range of \(F_{ST}\) values.



\begin{figure}
\includegraphics[width=1\linewidth]{figs/fst/circos_all_fst} \caption{Circos plot showing \(F_{ST}\) values for all SNPs across the human genome.}\label{fig:fst-circos}
\end{figure}



\begin{figure}
\includegraphics[width=1\linewidth]{figs/fst/histo_fst_all} \caption{Histogram of \(F_{ST}\) values for all SNPs across the human genome.}\label{fig:fst-histo}
\end{figure}

\textbf{Figure \ref{fig:fst-circos}} shows that with very few exceptions, \(F_{ST}\) remains below 0.6, although specific regions do appear to be differentially selected, as indicated be the peaks on chromosomes 1, 2, and 17. \textbf{Figure \ref{fig:fst-histo}} makes it clear that the vast majority of SNPs have very low \(F_{ST}\), with a genome-wide mean of 0.019. The question then is whether the SNPs that have a detectable effect on traits tend to have higher \(F_{ST}\) values -- and therefore greater variation in allele frequencies between populations -- than random SNPs with similar allele frequencies in European populations. If so, this would suggest that variation in allele frequencies might contribute significantly to the non-transferability of PRS scores derived from European-focused GWAS.

\clearpage

\hypertarget{Fst-analysis-chap}{%
\section{Analysis}\label{Fst-analysis-chap}}

In this analysis I explore the distribution of \(F_{ST}\) scores for loci associated with 587 traits, a subset of the GWAS Catalog that passed our criteria for suitable polygenic traits (see section \ref{Fst-analysis-chap}). Using high-coverage sequence data for 2,504 individuals from the 1000 Genomes Project phase 3 release, for each trait in the GWAS Catalog I calculated the distribution of \(F_{ST}\) across all approximately-unlinked SNPs associated with it (\textbf{trait SNPs}), and compared these \(F_{ST}\) distributions with the \(F_{ST}\) distributions of random-selected SNPs that were matched to the trait SNPs by their allele frequencies in European populations (\textbf{control SNPs}).

\hypertarget{gwas-catalog}{%
\subsubsection{GWAS Catalog}\label{gwas-catalog}}

I used the R package \emph{gwasrapidd} (Magno and Maia 2020) to query all traits in the GWAS Catalog (MacArthur et al. 2017) as of 9 August 2021 (\(N_{TRAITS}\) = 3,459). For 541 of these traits, no matching variant IDs could be pulled out from the 1000 Genomes VCFs, leaving \(N_{TRAITS}\) = 3,008.

\hypertarget{linkage-disequilibrium}{%
\subsection{Linkage disequilibrium}\label{linkage-disequilibrium}}

To obtain the ``trait SNP'' dataset, for each trait, I sought to isolate the SNP closest to each of its true causal variants, and exclude the SNPs in LD with them. To this end, I used PLINK 1.9 (Chang et al. 2015; Purcell and Chang, n.d.) to ``clump'' the SNPs associated with each of the remaining 3,008 traits, using an ``index'' SNP p-value threshold of \(10^{-8}\) (Panagiotou, Ioannidis, and Genome-Wide Significance Project 2012), \(r^2\) threshold of 0.1 (W. G. Hill and Robertson 1968), and base window size of 1 Mb. This process revealed 2,045 traits with at least one index SNP that met the p-value threshold. The index SNPs for each trait formed the set of trait SNPs, and \textbf{Figure \ref{fig:FstSnpCount}} shows the counts of unique SNP IDs associated with each trait before and after clumping. In order to target relatively polygenic traits, I further filtered out traits with fewer than 10 trait SNPs, leaving \(N_{TRAITS}\) = 587.



\begin{figure}
\includegraphics[width=1\linewidth]{figs/fst/0.1_1000_20220314_snp_counts} \caption{log10 counts of associated SNPs for each trait before and after the clumping process, which involved: a) excluding all SNPs with a \emph{p}-value greater than \(10^{-8}\); and b) starting with the SNPs with the lowest \emph{p}-values (``index'' SNPs), excluding all other SNPs within a 1 Mb region of the index SNP with an LD \(r^2\) of more than 0.1.}\label{fig:FstSnpCount}
\end{figure}

\hypertarget{control-snps}{%
\subsection{Control SNPs}\label{control-snps}}

To obtain the ``control SNP'' dataset, I assigned each trait SNP to one of 20 bins based on its minor allele frequency in European populations (as provided in the original 1000 Genomes .vcf files under the column header `INFO/AC\_EUR'). For example, if a trait SNP had a minor allele frequency of 0.08 in European populations, it was assigned to the \((0.05, 0.1]\) bin. I did the same for all (un-associated) SNPs in the .vcf files, then paired each trait SNP with a random SNP from the .vcf file in the equivalent bin. These allele-frequency-paired random SNPs formed the set of ``control SNPs'', which I used to infer the \(F_{ST}\) distribution of a random set of SNPs with the same allele frequencies as the trait SNPs, and against which I could compare the \(F_{ST}\) distribution of the trait SNPs.

\hypertarget{f_st-and-ranking-traits-by-signed-kolmogorov-smirnov-d-statistic}{%
\subsection{\texorpdfstring{\(F_{ST}\) and ranking traits by signed Kolmogorov-Smirnov \(D\) statistic}{F\_\{ST\} and ranking traits by signed Kolmogorov-Smirnov D statistic}}\label{f_st-and-ranking-traits-by-signed-kolmogorov-smirnov-d-statistic}}

I then calculated \(F_{ST}\) for each of the trait SNPs and their matched control SNPs using the Weir and Cockerham method (B. S. Weir and Cockerham 1984), as implemented in the R package \emph{pegas} (Paradis 2010). I then sought to rank all traits based on the directional difference in \(F_{ST}\) distributions between trait and control SNPs. The Kolmogorov-Smirnov (KS) test is an appropriate test to measure the differences between distributions (Conover 1999), but the \emph{p}-values are strongly affected by the number of samples in each distribution (here, the number of loci in each trait), and it does not measure whether one distribution is overall higher or lower than the other distribution. These limitations made it impossible to use the KS test \emph{p}-values to compare the directional distances between trait and control SNPs across traits that have very different numbers of SNPs. I accordingly formulated the following method for ranking the traits based on the directional distance of the \(F_{ST}\) distributions of the trait SNPs compared to the control SNPs:

First, I ran three KS tests for each trait \(t\) with \(x_t\) = \(F_{ST, trait SNPs}\) and \(y_t\) = \(F_{ST, control SNPs}\):

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\item
  two-sided (\(D_t\)) ;
\item
  one-sided ``greater'' (\(D_t^+\)) ; and
\item
  one-sided ``less'' (\(D_t^-\)).
\end{enumerate}

I note that \(D_{t} = max(D_t^+, D_t^-)\), where \(D_t^+\) is the greatest vertical distance attained by the eCDF of \(x_t\) over the eCDF of \(y_t\), and \(D_t^-\) is the greatest vertical distance attained by the eCDF of \(y_t\) over the eCDF of \(x_t\) (Conover 1999; Durbin 1973). I then used a comparison of \(D_t^+\) and \(D_t^-\) to formulate a ``signed D statistic'' (\(D_t^S\)), based on the logic that trait SNPs with a lower overall \(F_{ST}\) than control SNPs tend to have a higher \(D\) under the ``greater'' test than the ``less'' test, and vice versa.

Therefore, \({D_t^S}\):

\[
\begin{aligned}
D_t^- > D_t^+ &: &-D_t \\
D_t^- = D_t^+ &: &0 \\
D_t^- < D_t^+ &: &D_t \\
\end{aligned}
\]

In \textbf{Figure \ref{fig:FstMain}} I present the \(F_{ST}\) distributions of trait SNPs for an illustrative subset of 28 human traits, ranked by \({D_t^S}\) when compared with their matched control SNPs. \textbf{Figure \ref{fig:FstMain}A} shows the densities of SNPs as a function of \(F_{ST}\), and \textbf{Figure \ref{fig:FstMain}B and C} show their empirical Cumulative Distribution Functions (eCDFs). \textbf{Figure \ref{fig:FstMain}B} includes the eCDFs of control SNPs in grey. eCDF figures for all 587 traits that passed our filters (Methods) are provided in the Appendix \ref{fig:eCDFall}.



\begin{figure}
\includegraphics[width=1\linewidth]{figs/fst/0.1_1000_20220314_final} \caption{Distributions of \(F_{ST}\) across 28 illustrative human traits, ranked by signed-D (Kolmogorov-Smirnov test) comparing trait and control SNPs. \textbf{A}. \(F_{ST}\) density ridge plots with SNP markers. \textbf{B}. Empirical Cumulative Distribution Functions (eCDFs) of \(F_{ST}\) for trait-associated (colour) and random control (grey) SNPs, faceted by trait. \textbf{C}. Consolidated eCDFs of trait-associated SNPs from (B). eCDFs for all traits are included in the Appendix.}\label{fig:FstMain}
\end{figure}

These results show that for most traits, especially those that are highly polygenic, the \(F_{ST}\) of their associated alleles tend to be low, and to differ little from their matched control SNPs. This suggests that the causal variants of polygenic traits are generally shared across global populations at similar frequencies. Ying Wang et al. (2020) determined through simulations that differences in LD and allele frequencies between populations can explain 70-80\% of the loss of PGS relative accuracy for traits like body mass index and type 2 diabetes. As most of these variants have been discovered in European populations, the poor transferability of PGS between populations does not appear to be primarily driven by differences in the allele frequencies of these discovered variants, but perhaps rather by as-yet undiscovered variants of larger effect sizes in non-European populations, as well as differences between these populations in LD structure and environment.

There is a question as to whether the higher distributions of \(F_{ST}\) observed for hair shape, eye colour, skin pigmentation, and HIV infection, are due to those variants being discovered in diverse populations, such as those in Africa. For instance, if the alleles were predominantly discovered through GWAS on African populations, which contain an outsized proportion of genetic variation, those alleles would be more likely to show lower frequencies in other populations, and consequently higher \(F_{ST}\). To examine the number of individuals from different ancestries that were used in these studies, I generated Figure \ref{fig:fst-n-indivs}. This figure shows that GWAS performed for the more comprehensively-studied phenotypes such as body height, BMI, and educational attainment, included individuals from many diverse populations, whereas the GWAS on pigmentation-related traits tended to focus on European or Latin American populations, as well as those of African ancestry in some cases. As expected, the GWAS on HIV-related traits included a higher proportion of individuals of African ancestry, which may explain the higher levels of \(F_{ST}\) observed there. I note that the GWAS Catalog data did not allow me to determine whether different studies on the same trait used the same cohorts (e.g.~from UK Biobank), which would have the effect of inflating the counts shown in Figure \ref{fig:fst-n-indivs}.



\begin{figure}
\includegraphics[width=1\linewidth]{figs/fst/20220324_n_individuals} \caption{Counts of individuals from different populations used in GWAS studies on select traits.}\label{fig:fst-n-indivs}
\end{figure}

\hypertarget{conclusions}{%
\chapter*{Conclusions}\label{conclusions}}
\addcontentsline{toc}{chapter}{Conclusions}

\hypertarget{chapter-refmikk-genomes-chap}{%
\section*{Chapter \ref{MIKK-genomes-chap}}\label{chapter-refmikk-genomes-chap}}
\addcontentsline{toc}{section}{Chapter \ref{MIKK-genomes-chap}}

In this first substantive chapter of the thesis, I explored several genomic traits in the MIKK panel that are relevant to its utility for the genetic mapping of complex traits, and analysed the structural variants in a subset of its genomes that could be discovered through the use of Oxford Nanopore Technologies' (\textbf{ONT}) long-read sequencing platform. I displayed the inbreeding trajectory of the MIKK panel, confirming that medaka have a remarkable resilience to inbreeding from the wild, and that despite the spread of a bacterial infection during the inbreeding process that destroyed a number of lines, 30\% of them nevertheless survived by generation F\textsubscript{15}. I showed the MIKK panel possesses a normally-distributed morphology phenotype (mean relative eye size) as an example of the distribution of complex traits across the panel.

I confirmed that the inbreeding process resulted in inbred lines that were highly homozygous across almost the entire genome, comparable to the levels of homozygosity that were present in established inbred strains that have been inbreed for over 100 generations, and that regions of residual heterozygosity are possibly related to regions that have high levels of repeat content and structural variation. I explored the evolutionary relationship between southern Japanese, northern Japanese, and east Korean medaka populations, providing evidence that despite having speciated earlier, the southern Japanese and east Korean populations have interbred more recently with each other than the southern and northern Japanese populations, despite the former pair being presently separated by the Korean Strait. With particular relevance to genetic mapping, I showed that the MIKK panel has a rate of LD decay that would permit a mapping resolution up to 10 times higher than in humans, which in turn would make it more likely to identify the variants responsible for differences in complex traits.

Finally, I demonstrated the power of ONT long-read sequencing technologies to uncover large-scale structural variation that would otherwise have been hidden from NGS short-read sequencing. In addition, I showed that the traditional reference-anchored approach of genomic alignment can miss large-scale insertions that can otherwise be detected by the emerging graph-based approach. In the future, the ONT sequencing that was performed here on a subset of the MIKK panel will be expanded across the entire panel, which will permit the inclusion of both large- and small-scale variants in F\textsubscript{2} cross GWAS studies.

\hypertarget{chapter-refpilot-chap}{%
\section*{Chapter \ref{Pilot-chap}}\label{chapter-refpilot-chap}}
\addcontentsline{toc}{section}{Chapter \ref{Pilot-chap}}

In this chapter I described a custom assay for measuring bold-shy behaviours in medaka fish that can reliably detect differences: (a) between individuals from different inbred strains -- allowing for the quantification of direct genetic effects on behaviour; and (b) between the behaviour of individuals when paired with tank partners from a different strain -- allowing for the quantification of social genetic effects on behaviour. I found that behaviours exhibited during the assay show minimal variance across the covariates of date of assay, time of assay, tank quadrant, and tank side. This creates confidence that the genetic effects that I observed -- both direct and indirect -- are not unduly affected by these variables.

I showed that a Hidden Markov Model (\textbf{HMM}) adequately classifies movements based on the direction and angle of travel within set time intervals. The HMM method has previously been used to classify maternal behaviour patterns in mice (Carola, Mirabeau, and Gross 2011), and I have extended the application here to bold-shy behaviours in medaka fish. In principle, this method can be expanded to include additional behavioural features such as proximity to the wall or other objects, inwards/outwards orientation, proximity to the tank partner, and other metrics related to leader-follower dynamics, a possibility that will be tested in future studies.

The HMM classification of behaviours also facilitated the ranking of behaviours by speed and angle of movement, which may be further interpreted in terms of biology and adaptation styles. However, even though the HMM classifies the slowest-moving states as separate modes of movement, one should be cautious when attributing biological meaning to their differences, because even when a fish is almost completely still, the fish object's centroid (upon which the variables of distance and angle of travel are calculated) will tend to move by one or several pixels through minor changes in the segmentation of the object, and are thus subject to greater noise than the faster-moving states.

I found that inbred strains of medaka fish can be distinguished by the proportion of time they spend in certain states. The slowest states, capturing no or minimal movement, were the states that most clearly separated the strains, and these differences appeared most clearly at the beginning of the open field and novel object assays. This suggests that for southern Japanese strains such as \emph{iCab}, \emph{HdrR}, and \emph{HO5}, the ``freeze'' reflex is caused by anxiety, which eventually dissipates over time, indicating habituation. Similar observations showing initial anxiety as shown by freezing behaviour and subsequent habituation have been made in other studies using open field assays with the \emph{iCab} strain (Lucon-Xiccato et al. 2022).

On the other hand, the northern Japanese strains \emph{Kaga} and \emph{HNI} spent little time in the slow-moving states at the beginning of the video, which indicates either that their habituation sets in earlier, or that their stress and anxiety is expressed to a lesser degree as freezing behaviour. The former appears to be more likely for \emph{Kaga}, as in the open field assay, it spends more time in the faster-moving states at the beginning of the video and then slows down thereafter, which suggests that its higher level of movement may be induced by stress. It is interesting then that once the novel object is introduced, \emph{Kaga} tends to move slowly like the other strains. Obviously, introduction to a novel environment and exposure to a potentially dangerous object elicits different behavioural patterns in this strain. Taken together with the overt thigmotaxis behaviour (that is, moving along the sides of the tank during the open field assay component) relative to other strains, its behaviour may suggest an ``escape-seeking'' response. It will be interesting to carry out further experiments to determine under which fear-inducing conditions \emph{Kaga} reacts with this more frantic style of movement rather than the freeze response that it displays in the novel object component in common with the other strains, and to explore its physiological basis.

With respect to social genetic effects, I have found that a fish's behaviour is affected by the differential behaviour of its tank partner, although the effect is less powerful than the direct genetic effects exerted on a fish's own biology. These social genetic effects are detectable when observing the proportions of time the reference fishes spend in certain states over the course of the video, and can be quantified by measuring the frequency of state co-occupancy with their tank partners.

It is also interesting to find that \emph{iCab} prefers to co-occupy the slow, pan-directional state 4 when paired with another individual from the same strain. As the strains are raised exclusively in the company of individuals from the same strain, that type of behaviour may represent a more comfortable -- yet still cautious -- mode of movement, or a type of mimicry that tends to occur when in the company of individuals that are more genetically similar, or socially familiar.

In this study I only used one strain as the reference fish, but future experiments can expand on this analysis by using different strains as the reference, thereby exploring how a strain's genetics influence the degree of their behavioural plasticity, and how the distinct behaviours of strains may interact in intriguing ways -- or in other words, expanding the analysis to incorporate gene-environment interactions. In summary, I showed that the custom combined open-field/novel-object experimental setup in combination with the HMM analysis can reliably detect both direct and social genetic effects. This opened up the opportunity to carry out a similar study on a larger panel of inbred strains, which I did with the MIKK panel in Chapter \ref{MIKK-F2-chap}.

\hypertarget{chapter-refsomite-chap}{%
\section*{Chapter \ref{Somite-chap}}\label{chapter-refsomite-chap}}
\addcontentsline{toc}{section}{Chapter \ref{Somite-chap}}

In this chapter, I performed the bioinformatic analyses required to map genetic variants associated with somite development periodicity. I first explored the distributions across the F\textsubscript{0}, F\textsubscript{1} and F\textsubscript{2} generations of the primary phenotype of interest -- somite periodicity -- as well as the positive control phenotype of unsegmented presomatic mesoderm (\textbf{PSM}) area. I discovered that there was a significant difference in phenotype based on the microscope that was used to measure somite periodicity (but not PSM area), and controlled for this downstream.

I then assessed the levels of homozygosity in the F\textsubscript{0} generation, and discovered that the \emph{Kaga} strain that was used had unexpectedly low levels of homozygosity, likely as a result of contamination with another strain. This created many region of low heterozygosity in the F\textsubscript{1}, which restricted out ability to map variants in the F\textsubscript{2} generation, with the exception of chr22, the reason for which warrants further investigation. To map the shallow-sequenced F\textsubscript{2} samples back to the the F\textsubscript{0} parental haplotypes, I refined the parameters of an HMM to call the recombination blocks based on the expected number of crossovers per chromosome. I discovered a bias away from homozygous-\emph{Kaga} genotypes, likely due to a combination of its lower original homozygosity, and possible genetic incompatibility with the \emph{Cab} strain as a consequence of their ancestral divergence.

I then used the F\textsubscript{2} recombination blocks as pseudo-SNPs in a GWAS for both the somite periodicity and PSM area phenotypes, and discovered a strongly-significant locus for each phenotype on separate regions of chr3, which I refined down to discrete regions using a tool for the effects of variant prediction, and provided a set of candidate loci for a CRISPR-Cas 9 knockout screen, which my collaborators are currently performing. Together with the ongoing work of my collaborators, this analysis will shed light on the genetic control of somite periodicity, expanding our knowledge of how this crucial developmental process occurs, and why it varies between different organisms.

\hypertarget{chapter-refmikk-f2-chap}{%
\section*{Chapter \ref{MIKK-F2-chap}}\label{chapter-refmikk-f2-chap}}
\addcontentsline{toc}{section}{Chapter \ref{MIKK-F2-chap}}

The results from this chapter provide promising evidence that the assay and methods described can allow researchers to identify genetic loci associated with differences in behaviours - and potentially difference in the transmission of behaviour onto social companions - with functional relevance to humans. However, before seeking to functionally validate the variants, there are a number of steps that we could take to refine the analysis, and thereby increase our confidence in the variants we ultimately select for validation.

With the benefit of hindsight, there are a several aspects of the analysis described in \textbf{Chapter \ref{MIKK-F2-chap}} that I would have performed differently. The first relates to the choices of certain parental lines for the F\textsubscript{2} cross, which were too heavily reliant on the aggregate measures of speed and charisma, without due regard to the level of behavioural variance observed within each line. As evident in \textbf{Figures \ref{fig:F2-time-dge-of}} and \textbf{\ref{fig:F2-time-dge-no}} above, lines \definecolor{8-2 (‘Gail’)_FF699C}{HTML}{FF699C}\textcolor{8-2 (‘Gail’)_FF699C}{8-2 (‘Gail’)} and \definecolor{21-2 (‘Amelia’)_49B500}{HTML}{49B500}\textcolor{21-2 (‘Amelia’)_49B500}{21-2 (‘Amelia’)} showed large within-line variances. For \definecolor{8-2 (‘Gail’)_FF699C}{HTML}{FF699C}\textcolor{8-2 (‘Gail’)_FF699C}{8-2 (‘Gail’)} in particular, the differences appear to correspond to the date on which they were assayed - specifically, in the first run the four \definecolor{8-2 (‘Gail’)_FF699C}{HTML}{FF699C}\textcolor{8-2 (‘Gail’)_FF699C}{8-2 (‘Gail’)} individuals showed almost no movement, whereas in the second run they showed much higher levels of movement. The different within-line variances are an interesting phenotype to explore further, as they suggest that different lines show different degrees of sensitivity to environmental changes. However, for this analysis, high within-line variances make it difficult to ascertain the line's ``true'' phenotype, and therefore make them unsuitable for selection based on our axes of interest for this study (bold vs shy; high vs low behavioural transmission).

An appropriate substitute for \definecolor{8-2 (‘Gail’)_FF699C}{HTML}{FF699C}\textcolor{8-2 (‘Gail’)_FF699C}{8-2 (‘Gail’)} may be line \definecolor{139-4_FF61CC}{HTML}{FF61CC}\textcolor{139-4_FF61CC}{139-4}, which had low within-line variance for mean speed, and the highest median state co-occupancy in the novel object assay component. From the analysis of Chapter \ref{Pilot-chap}, which concluded after the lines were already selected for the F\textsubscript{2} cross, it became apparent that the novel object assay was particularly useful for revealing social genetic effects. I hypothesise that at times of higher stress or predation threat, the fish take more behavioural cues from their tank partners. Line \definecolor{139-4_FF61CC}{HTML}{FF61CC}\textcolor{139-4_FF61CC}{139-4} therefore would have been a good candidate for the slow-moving, high-charisma line.

Similarly, lines \definecolor{13-2_F57A5F}{HTML}{F57A5F}\textcolor{13-2_F57A5F}{13-2} or \definecolor{94-1_D39200}{HTML}{D39200}\textcolor{94-1_D39200}{94-1} would be preferred substitutes for line \definecolor{21-2 (‘Amelia’)_49B500}{HTML}{49B500}\textcolor{21-2 (‘Amelia’)_49B500}{21-2 (‘Amelia’)} as the fast-moving, high charisma line. They are faster than \definecolor{21-2 (‘Amelia’)_49B500}{HTML}{49B500}\textcolor{21-2 (‘Amelia’)_49B500}{21-2 (‘Amelia’)}, but unlike the fastest line \definecolor{10-1 (‘Janeway’)_F8766D}{HTML}{F8766D}\textcolor{10-1 (‘Janeway’)_F8766D}{10-1 (‘Janeway’)}, they show a degree of habituation (where most individuals moved slowly at the beginning of the assay before eventually speeding up). They also showed lower within-line variance than \definecolor{21-2 (‘Amelia’)_49B500}{HTML}{49B500}\textcolor{21-2 (‘Amelia’)_49B500}{21-2 (‘Amelia’)}, but comparable measures for the SGE measures of state co-occupancy and reference deviation.

One way of avoiding these issues would have been to used a more quantitative, objective approach to selecting the lines. For example, I could have used Mann-Whitney tests to quantify the levels of differences between pairs of lines for a trait of interest (as I did for \textbf{Figure \ref{fig:mikk-param-comp}}), and then select the lines that maximised that statistic. Such metrics would still perhaps need to be qualitatively weighted against other relevant traits, such as within-line variance. Nevertheless, my downstream analyses show that despite not having potentially selected the ideal lines for the F\textsubscript{2} cross, the differences between them were sufficient to identify genetic loci associated with our phenotypes of interest, even with a relatively small sample size of 271 F\textsubscript{2} individuals.

Another aspect of the analysis that could have been improved was to use one or more MIKK panel lines as the reference line instead of \emph{\definecolor{iCab_424B4D}{HTML}{424B4D}\textcolor{iCab_424B4D}{iCab}}. As previously mentioned, \emph{\definecolor{iCab_424B4D}{HTML}{424B4D}\textcolor{iCab_424B4D}{iCab}} is a relatively fast line, which made it difficult to detect social genetic effects when paired with similarly fast-moving lines. Instead, we could have made use of the behavioural variance across the MIKK panel to choose a reference line that exhibits moderate levels of movement. However, we would not be able to assess the levels of movement across the MIKK panel without first assaying the panel with a single reference strain, and we chose \emph{\definecolor{iCab_424B4D}{HTML}{424B4D}\textcolor{iCab_424B4D}{iCab}} because we not only had a good understanding of its behavioural traits based on our findings from Chapter \ref{Pilot-chap}, but also because it has high fecundity, which made many replicates available and in turn allowed us to avoid using the same reference fish more than once within 24 hours. Now that we have carried out this analysis, we would be able to select a more appropriate line as the reference in any future analyses.

The other benefit from using a MIKK panel line as the reference fish would be to open up the possibility of exploring social genetic effect interactions. For example, if we used two different MIKK lines as the reference, we could explore how those lines differ in the degree to which they are influenced by their tank partner. To state the issue inversely, it would reveal differences in the extent to which different lines transmit their behaviour onto their (reference) tank partner, depending on both of their genetics. A question for further studies could therefore be framed as, `given line \definecolor{18-2 (‘Elsa’)_FF66A6}{HTML}{FF66A6}\textcolor{18-2 (‘Elsa’)_FF66A6}{18-2 (‘Elsa’)} appears to possess a high level of charisma, is that level consistent when they are paired with different MIKK panel lines, and if it differs across lines, then why?' Due to the relative onerousness of running this behavioural assay, such experiments would involve a substantial time commitment, but in light of the findings presented here, the investigations could be restricted to a subset of the MIKK panel that display interesting behavioural phenotypes.

\hypertarget{future-directions-for-the-f2-cross-gwas-for-bold-type-behaviours-in-the-mikk-panel}{%
\subsection{\texorpdfstring{Future directions for the F\textsubscript{2} cross GWAS for bold-type behaviours in the MIKK panel}{Future directions for the F2 cross GWAS for bold-type behaviours in the MIKK panel}}\label{future-directions-for-the-f2-cross-gwas-for-bold-type-behaviours-in-the-mikk-panel}}

Our collaborators are in the process of breeding more F\textsubscript{2} individuals to increase the sample size for this analysis. For the subsequent breeding program, I have suggested that we replace line \definecolor{21-2 (‘Amelia’)_49B500}{HTML}{49B500}\textcolor{21-2 (‘Amelia’)_49B500}{21-2 (‘Amelia’)} with either \definecolor{13-2_F57A5F}{HTML}{F57A5F}\textcolor{13-2_F57A5F}{13-2} or \definecolor{23-1_AFA200}{HTML}{AFA200}\textcolor{23-1_AFA200}{23-1}, which both show lower within-line variance in mean speed, but similarly high levels of speed and behavioural transmission (as measured by state co-occupancy and \emph{\definecolor{iCab_424B4D}{HTML}{424B4D}\textcolor{iCab_424B4D}{iCab}} reference deviation). Any additional F\textsubscript{2} individuals will still need to be phenotyped and sequenced, and the phenotype data collection (i.e.~video recording the behavioural assay) can be relatively laborious, so this process will likely take some time.

In the meantime, I have 96 Kiyosu CC individuals whose videos have already been processed, and whose DNA has been shallow-sequenced (\textasciitilde1x). It was not feasible to include that data in this analysis due to time constraints. As they have been allowed to breed freely from the same Kiyosu population as the MIKK panel, they are not strongly homozygous, and they are likely to possess many additional genetic variants that are not represented in the MIKK panel. Unlike the F\textsubscript{2} samples, which I could impute directly from their F\textsubscript{0} parental lines, I will need to use the full high-coverage variant call set from the MIKK panel to impute their genotypes.

A second issue I plan to address is the failure of F\textsubscript{2} reads to align to chromosomes 22 and 23 (see \textbf{Figure \ref{fig:F2-recomb-blocks}} above). The problem occurs across most of the sample, the coverage does not look skewed for those chromosomes (\textbf{Figure \ref{fig:F2-coverage}}), and nor do the counts of homozygous-divergent SNPs between the F\textsubscript{0} parents do not look consistently smaller for those chromosomes (\textbf{Figure \ref{fig:homo-div-snp-counts}}). There is therefore likely to a be an error at some stage of the bioinformatic pipeline that I constructed.



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/coverage_F2} \caption{Mean depth of sequencing coverage per chromosome for the 271 F\textsubscript{2} samples used in this analysis.}\label{fig:F2-coverage}
\end{figure}



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/homo_div_snp_counts} \caption{Counts of SNPs that are homozygous-divergent between the parental F\textsubscript{0} lines in each of the 12 crosses used.}\label{fig:homo-div-snp-counts}
\end{figure}

A third issue is the matter of selecting the optimal HMM parameters for calling the recombination blocks, as described in Chapter \ref{Somite-chap}. For the analysis in this present chapter I used the same HMM parameters as I had used there. In that chapter, I used the \(p\)-values from the resulting genetic linkage analysis as a rough measure of genotyping quality (assuming that lower \(p\)-values were driving by more accurate genotypes), together with a qualitative assessment of how many crossovers I expected to observe in a particular chromosome for a subset of the sample. The issue can be resolved by obtaining ``ground truth'' genotype calls through high-coverage sequence data for a set of samples that have also been shallow-sequenced. This would allow one to compare the HMM recombination block calls with the ground truth genotypes, and thereby train the model to optimise the accuracy of its predictions for the locations of the recombination sites (using a test dataset to validate the parameters). I expect the improved genotype calls will further improve the power to detect genetic variants.

With respect to the genetic linkage analysis, due to the method I used to calculate the genetic relationship matrix (\textbf{GRM}), I was compelled to exclude all SNPs that were missing calls in any of the samples, reducing the number of SNPs to merely \textasciitilde40K out of the \textasciitilde20M in the full dataset. Not only does this reduce the accuracy of the GRM as a means of capturing the relative relatedness between samples, but the problem will only be exacerbated with the inclusion of additional samples (as the likelihood of a SNP having at least one sample with a missing call will increase with the number of samples). I will therefore adapt my method to include SNPs with missing calls, which should further increase statistical power.

Finally, I applied a separate genetic linkage association test for the state frequency of each HMM state, where it may be beneficial to apply a method that can combine the information across all states into a single phenotype (if not fewer phenotypes), and then run an association test on that reduced phenotype. There are a number of alternatives to consider, including principal components analysis (PCA) of the state frequencies, or the software MTAG (Multi-Trait Analysis of Genome-wide summary statistics , Turley et al. 2018) which can combine the summary statistics from multiple genetic association tests. Ultimately, the hope is that by leveraging the information about the frequencies in which the fishes occupy each state, combined across \emph{all} states, while appropriately taking into account the interdependence of those frequencies (e.g.~the more time a fish spends in state 3, the less time it is likely to spend in state 13), it will further strengthen these results.

Aside from modifications designed to increase statistical power, there is also capacity to improve the way that I phenotype the individuals, and in particular how I characterise social genetic effects. In this analysis I have only used two behavioural variables: distance and angle of travel. This behavioural dataset is extremely rich, and myriad variables could be extracted from the videos. Shoaling traits would have special relevance to the study of social genetic effects. For instance, one could extract three variables that would be associated with shoaling behaviours: a) distance between tank partners; b) synchrony in speed; and c) parallelism in direction of travel. These three variables could then be reduced down to two with an HMM (i.e.~one state representing ``shoaling'', and another representing ``not shoaling''). The relative frequencies that the different MIKK lines spend in these states would allow one to rank them by the degree of shoaling behaviour they exhibit, and this would likely be a far more accurate representation of social genetics than what I have shown here.

\hypertarget{chapter-reffst-chap}{%
\section*{Chapter \ref{Fst-chap}}\label{chapter-reffst-chap}}
\addcontentsline{toc}{section}{Chapter \ref{Fst-chap}}

The purpose of this final chapter was to determine the extent to which differences in allele frequencies between diverse human populations may contribute to the poor transferability of polygenic scores (\textbf{PGS}) to non-European populations. I used \(F_{ST}\) as a measure for differences in allele frequencies between the 26 populations in the 1000 Genomes dataset. I then compared the distributions of \(F_{ST}\) across all polygenic traits in the GWAS Catalog using a signed \(D\) statistic from the Kolmogorov-Smirnov test, \({D_t^S}\), to rank the traits by the distance of their \(F_{ST}\) distributions from a random set of allele-frequency-matched control SNPs. I found that for most traits, the \(F_{ST}\) distributions of associated alleles differed little from the control SNPs, indicating that the allele frequencies of trait-associated alleles do not differ greatly from non-associated (i.e.~background) SNPs. The notable exceptions were race-associated traits including skin pigmentation, eye colour, and hair shape, which showed large differences from their control SNPs, suggesting that -- as would be expected -- the frequencies of the alleles associated with those traits do differ greatly between populations. These results provide evidence that differences in allele frequencies for most traits may not contribute significantly to the poor transferability of PGS to non-European populations.

It would be convenient to extrapolate these findings to conclude that global populations do not differ significantly in the traits themselves, but this is not possible due to two limitations of the \(F_{ST}\) measure. The first is that \(F_{ST}\) does not correspond to similarities or differences between populations in \emph{trait values}. This is because \(F_{ST}\) does not take into account the effect size or the direction of effect of the trait-associated allele, so for highly-polygenic traits like the ones shown here, \(F_{ST}\) is almost entirely decoupled from the mean additive genetic value between populations (Jeremy J. Berg and Coop 2014a). This means that even when a trait's associated alleles have low \(F_{ST}\), as we see for most traits here, it does not follow that the population mean values for that trait would be the same due to genetic similarities between those populations. As Kremer and Le Corre (2012) determined by through simulations, for highly polygenic traits, mean trait differences are driven by small, coordinated shifts in allele frequencies, which would not be detected by \(F_{ST}\) (Le Corre and Kremer 2012).

The second relevant limitation is that \(F_{ST}\) is not sufficiently powered to detect selection for polygenic traits (Jeremy J. Berg and Coop 2014b), which means that these results cannot be used to determine the extent to which alleles are under selection in a given population. This is apparent in our analysis, where in contrast with Racimo, Berg, and Pickrell (2018), height and BMI, which have been found to be under strong differential selection, appear similar to other polygenic traits.

Nevertheless, it is a worthwhile and instructive finding that \(F_{ST}\) distributions for trait-associated alleles do not differ greatly from those of non-associated (``background'') alleles. In terms of its implications for the poor transferability of PGS to non-European populations, the obvious solution is to increase the representation of those populations in GWAS and PGS studies, as has often been proposed elsewhere (Alicia R. Martin et al. 2017; Alicia R. Martin et al. 2018; Bien et al. 2019; Alicia R. Martin et al. 2019; Sirugo, Williams, and Tishkoff 2019; Wojcik et al. 2019).

\hypertarget{references}{%
\chapter*{References}\label{references}}
\addcontentsline{toc}{chapter}{References}

\hypertarget{refs}{}
\begin{CSLReferences}{1}{0}
\leavevmode\vadjust pre{\hypertarget{ref-GlobalReferenceHuman2015}{}}%
{``A Global Reference for Human Genetic Variation.''} 2015. \emph{Nature} 526 (7571): 68--74. \url{https://doi.org/10.1038/nature15393}.

\leavevmode\vadjust pre{\hypertarget{ref-adhikariGWASLatinAmericans2019}{}}%
Adhikari, Kaustubh, Javier Mendoza-Revilla, Anood Sohail, Macarena Fuentes-Guajardo, Jodie Lampert, Juan Camilo Chac'on-Duque, Malena Hurtado, et al. 2019. {``A {GWAS} in {Latin Americans} Highlights the Convergent Evolution of Lighter Skin Pigmentation in {Eurasia}.''} \emph{Nature Communications} 10 (1, 1): 358. \url{https://doi.org/10.1038/s41467-018-08147-0}.

\leavevmode\vadjust pre{\hypertarget{ref-aidaInheritanceColorFreshWater1921}{}}%
Aida, Tatuo. 1921. {``On the {Inheritance} of {Color} in a {Fresh-Water Fish}, {Aplocheilus Latipes Temmick} and {Schlegel}, with {Special Reference} to {Sex-Linked Inheritance}.''} \emph{Genetics} 6 (6): 554--73. \url{https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1200522/}.

\leavevmode\vadjust pre{\hypertarget{ref-airdBloodGroupsRelation1954}{}}%
Aird, Ian, H. H. Bentall, J. A. Mehigan, and J. A. Fraser Roberts. 1954. {``The {Blood Groups} in {Relation} to {Peptic Ulceration} and {Carcinoma} of {Colon}, {Rectum}, {Breast}, and {Bronchus}.''} \emph{Br Med J} 2 (4883): 315--21. \url{https://doi.org/10.1136/bmj.2.4883.315}.

\leavevmode\vadjust pre{\hypertarget{ref-akeyInterrogatingHighDensitySNP2002}{}}%
Akey, Joshua M., Ge Zhang, Kun Zhang, Li Jin, and Mark D. Shriver. 2002. {``Interrogating a {High-Density SNP Map} for {Signatures} of {Natural Selection}.''} \emph{Genome Research} 12 (12): 1805--14. \url{https://doi.org/10.1101/gr.631202}.

\leavevmode\vadjust pre{\hypertarget{ref-alfonsoCopingStylesEuropean2019}{}}%
Alfonso, S'ebastien, Bastien Sadoul, Manuel Gesto, Lucette Joassard, B'eatrice Chatain, Benjamin Geffroy, and Marie-Laure B'egout. 2019. {``Coping Styles in {European} Sea Bass: {The} Link Between Boldness, Stress Response and Neurogenesis.''} \emph{Physiology \& Behavior} 207 (August): 76--85. \url{https://doi.org/10.1016/j.physbeh.2019.04.020}.

\leavevmode\vadjust pre{\hypertarget{ref-allenThomasHuntMorgan1978}{}}%
Allen, Garland E. 1978. \emph{Thomas {Hunt Morgan}: {The Man} and {His Science}}. {Princeton University Press}. \url{https://books.google.com?id=MdVOAQAAIAAJ}.

\leavevmode\vadjust pre{\hypertarget{ref-altshulerGeneticMappingHuman2008}{}}%
Altshuler, David, Mark J. Daly, and Eric S. Lander. 2008. {``Genetic {Mapping} in {Human Disease}.''} \emph{Science} 322 (5903): 881--88. \url{https://doi.org/10.1126/science.1156409}.

\leavevmode\vadjust pre{\hypertarget{ref-amarasingheOpportunitiesChallengesLongread2020}{}}%
Amarasinghe, Shanika L., Shian Su, Xueyi Dong, Luke Zappia, Matthew E. Ritchie, and Quentin Gouil. 2020. {``Opportunities and Challenges in Long-Read Sequencing Data Analysis.''} \emph{Genome Biology} 21 (1): 30. \url{https://doi.org/10.1186/s13059-020-1935-5}.

\leavevmode\vadjust pre{\hypertarget{ref-anderssonTwoShadesBoldness2014}{}}%
Andersson, Anastasia, Linda Laikre, and Ulrika A. Bergvall. 2014. {``Two Shades of Boldness: Novel Object and Anti-Predator Behavior Reflect Different Personality Dimensions in Domestic Rabbits.''} \emph{Journal of Ethology} 32 (3): 123--36. \url{https://doi.org/10.1007/s10164-014-0401-9}.

\leavevmode\vadjust pre{\hypertarget{ref-anderssonGeneticDissectionPhenotypic2001}{}}%
Andersson, Leif. 2001. {``Genetic Dissection of Phenotypic Diversity in Farm Animals.''} \emph{Nature Reviews Genetics} 2 (2, 2): 130--38. \url{https://doi.org/10.1038/35052563}.

\leavevmode\vadjust pre{\hypertarget{ref-anderssonDomesticanimalGenomicsDeciphering2004}{}}%
Andersson, Leif, and Michel Georges. 2004. {``Domestic-Animal Genomics: Deciphering the Genetics of Complex Traits.''} \emph{Nature Reviews Genetics} 5 (3, 3): 202--12. \url{https://doi.org/10.1038/nrg1294}.

\leavevmode\vadjust pre{\hypertarget{ref-aristotleGenerationAnimals2021}{}}%
Aristotle. 2021. \emph{On the {Generation} of {Animals}}. {Good Press}. \url{https://books.google.com?id=aLUqEAAAQBAJ}.

\leavevmode\vadjust pre{\hypertarget{ref-armstrongEnglishParsonnaturalistCompanionship2000}{}}%
Armstrong, Patrick. 2000. \emph{The {English Parson-naturalist}: {A Companionship Between Science} and {Religion}}. {Gracewing Publishing}. \url{https://books.google.com?id=hB0hEc4CN3wC}.

\leavevmode\vadjust pre{\hypertarget{ref-averyStudiesChemicalNature1944}{}}%
Avery, Oswald T., Colin M. MacLeod, and Maclyn McCarty. 1944. {``Studies on the Chemical Nature of the Substance Inducing Transformation of Pneumococcal Types: {Induction} of Transformation by a Desoxyribonucleic Acid Fraction Isolated from Pneumococcus Type {II}.''} \emph{Journal of Experimental Medicine} 79 (2): 137--58. \url{https://doi.org/10.1084/jem.79.2.137}.

\leavevmode\vadjust pre{\hypertarget{ref-bancroftPartIntroductionGouldian2016}{}}%
Bancroft, Paul. 2016. {``Part 1: {An} Introduction to the {Gouldian} Finch - {Planet Aviary}.''} February 17, 2016. \url{https://planetaviary.com/part-1-an-introduction-to-the-gouldian-finch/}.

\leavevmode\vadjust pre{\hypertarget{ref-barrettGlobalizingSocialMovement2004}{}}%
Barrett, Deborah, and Charles Kurzman. 2004. {``Globalizing Social Movement Theory: {The} Case of Eugenics.''} \emph{Theory and Society} 33 (5): 487--527. \url{https://doi.org/10.1023/B:RYSO.0000045719.45687.aa}.

\leavevmode\vadjust pre{\hypertarget{ref-barshGuidelinesGenomeWideAssociation2012}{}}%
Barsh, Gregory S., Gregory P. Copenhaver, Greg Gibson, and Scott M. Williams. 2012. {``Guidelines for {Genome-Wide Association Studies}.''} \emph{PLOS Genetics} 8 (7): e1002812. \url{https://doi.org/10.1371/journal.pgen.1002812}.

\leavevmode\vadjust pre{\hypertarget{ref-bartonSewallWrightEvolution2016}{}}%
Barton, Nicholas H. 2016. {``Sewall {Wright} on {Evolution} in {Mendelian Populations} and the {`{Shifting Balance}'}.''} \emph{Genetics} 202 (1): 3--4. \url{https://doi.org/10.1534/genetics.115.184796}.

\leavevmode\vadjust pre{\hypertarget{ref-bartovHolocaustOriginsImplementation2015}{}}%
Bartov, Omer. 2015. \emph{The {Holocaust}: {Origins}, {Implementation}, {Aftermath}}. {Routledge}. \url{https://books.google.com?id=m84JSgAACAAJ}.

\leavevmode\vadjust pre{\hypertarget{ref-bashfordOxfordHandbookHistory2010a}{}}%
Bashford, Alison, and Philippa Levine. 2010. \emph{The {Oxford Handbook} of the {History} of {Eugenics}}. {Oxford University Press, USA}. \url{https://books.google.com?id=Ml4vDwAAQBAJ}.

\leavevmode\vadjust pre{\hypertarget{ref-batesonMaterialsStudyVariation1894}{}}%
Bateson, William. 1894. \emph{Materials for the {Study} of {Variation}: {Treated} with {Especial Regard} to {Discontinuity} in the {Origin} of {Species}}. {Macmillan and Company}. \url{https://books.google.com?id=_HIZAAAAYAAJ}.

\leavevmode\vadjust pre{\hypertarget{ref-batesonMendelPrinciplesHeredity1909}{}}%
---------. 1909. {``Mendel's {Principles} of {Heredity}: {Cambridge University Press}.''} \emph{März 1909; 2nd Impr} 3: 1913.

\leavevmode\vadjust pre{\hypertarget{ref-baudGenomesPhenomesPopulation2014}{}}%
Baud, Amelie, Victor Guryev, Oliver Hummel, Martina Johannesson, and Jonathan Flint. 2014. {``Genomes and Phenomes of a Population of Outbred Rats and Its Progenitors.''} \emph{Scientific Data} 1 (1, 1): 140011. \url{https://doi.org/10.1038/sdata.2014.11}.

\leavevmode\vadjust pre{\hypertarget{ref-baudGeneticVariationSocial2017}{}}%
Baud, Amelie, Megan K. Mulligan, Francesco Paolo Casale, Jesse F. Ingels, Casey J. Bohl, Jacques Callebert, Jean-Marie Launay, et al. 2017. {``Genetic {Variation} in the {Social Environment Contributes} to {Health} and {Disease}.''} \emph{PLOS Genetics} 13 (1): e1006498. \url{https://doi.org/10.1371/journal.pgen.1006498}.

\leavevmode\vadjust pre{\hypertarget{ref-bergPopulationGeneticSignal2014a}{}}%
Berg, Jeremy J., and Graham Coop. 2014a. {``A {Population Genetic Signal} of {Polygenic Adaptation}.''} \emph{PLOS Genetics} 10 (8): e1004412. \url{https://doi.org/10.1371/journal.pgen.1004412}.

\leavevmode\vadjust pre{\hypertarget{ref-bergPopulationGeneticSignal2014}{}}%
---------. 2014b. {``A {Population Genetic Signal} of {Polygenic Adaptation}.''} \emph{PLOS Genetics} 10 (8): e1004412. \url{https://doi.org/10.1371/journal.pgen.1004412}.

\leavevmode\vadjust pre{\hypertarget{ref-bergReducedSignalPolygenic2019}{}}%
Berg, Jeremy J, Arbel Harpak, Nasa Sinnott-Armstrong, Anja Moltke Joergensen, Hakhamanesh Mostafavi, Yair Field, Evan August Boyle, et al. 2019. {``Reduced Signal for Polygenic Adaptation of Height in {UK Biobank}.''} Edited by Magnus Nordborg, Mark I McCarthy, Magnus Nordborg, Nicholas H Barton, and Joachim Hermisson. \emph{eLife} 8 (March): e39725. \url{https://doi.org/10.7554/eLife.39725}.

\leavevmode\vadjust pre{\hypertarget{ref-bergelsonIdentifyingGenesUnderlying2010}{}}%
Bergelson, Joy, and Fabrice Roux. 2010. {``Towards Identifying Genes Underlying Ecologically Relevant Traits in {Arabidopsis} Thaliana.''} \emph{Nature Reviews Genetics} 11 (12, 12): 867--79. \url{https://doi.org/10.1038/nrg2896}.

\leavevmode\vadjust pre{\hypertarget{ref-bienFutureGenomicStudies2019}{}}%
Bien, Stephanie A., Genevieve L. Wojcik, Chani J. Hodonsky, Christopher R. Gignoux, Iona Cheng, Tara C. Matise, Ulrike Peters, Eimear E. Kenny, and Kari E. North. 2019. {``The {Future} of {Genomic Studies Must Be Globally Representative}: {Perspectives} from {PAGE}.''} \emph{Annual Review of Genomics and Human Genetics} 20 (1): 181--200. \url{https://doi.org/10.1146/annurev-genom-091416-035517}.

\leavevmode\vadjust pre{\hypertarget{ref-bjorkmanSellingEugenicsCase2010}{}}%
Björkman, Maria, and Sven Widmalm. 2010. {``Selling Eugenics: The Case of {Sweden}.''} \emph{Notes and Records of the Royal Society} 64 (4): 379--400. \url{https://doi.org/10.1098/rsnr.2010.0009}.

\leavevmode\vadjust pre{\hypertarget{ref-blackWarWeakEugenics2012}{}}%
Black, Edwin. 2012. \emph{War {Against} the {Weak}: {Eugenics} and {America}'s {Campaign} to {Create} a {Master Race}, {Expanded Edition}}. {Dialog Press}. \url{https://books.google.com?id=xzwTEAAAQBAJ}.

\leavevmode\vadjust pre{\hypertarget{ref-blajStructuralVariantsTandem2022}{}}%
Blaj, Iulia, Jens Tetens, Jörn Bennewitz, Georg Thaller, and Clemens Falker-Gieske. 2022. {``Structural Variants and Tandem Repeats in the Founder Individuals of Four {F2} Pig Crosses and Implications to {F2 GWAS} Results.''} \emph{BMC Genomics} 23 (1): 631. \url{https://doi.org/10.1186/s12864-022-08716-0}.

\leavevmode\vadjust pre{\hypertarget{ref-blakeMouseGenomeDatabase2021}{}}%
Blake, Judith A, Richard Baldarelli, James A Kadin, Joel E Richardson, Cynthia L Smith, Carol J Bult, and the Mouse Genome Database Group. 2021. {``Mouse {Genome Database} ({MGD}): {Knowledgebase} for Mouse--Human Comparative Biology.''} \emph{Nucleic Acids Research} 49 (D1): D981--87. \url{https://doi.org/10.1093/nar/gkaa1083}.

\leavevmode\vadjust pre{\hypertarget{ref-blaszczykBoldnessNovelObjects2017}{}}%
Blaszczyk, Maryjka B. 2017. {``Boldness Towards Novel Objects Predicts Predator Inspection in Wild Vervet Monkeys.''} \emph{Animal Behaviour} 123 (January): 91--100. \url{https://doi.org/10.1016/j.anbehav.2016.10.017}.

\leavevmode\vadjust pre{\hypertarget{ref-brownCorrelationBoldnessBody2007}{}}%
Brown, C., F. Jones, and V. A. Braithwaite. 2007. {``Correlation Between Boldness and Body Mass in Natural Populations of the Poeciliid {Brachyrhaphis} Episcopi.''} \emph{Journal of Fish Biology} 71 (6): 1590--1601. \url{https://doi.org/10.1111/j.1095-8649.2007.01627.x}.

\leavevmode\vadjust pre{\hypertarget{ref-brownHeritableExperientialEffects2007}{}}%
Brown, Culum, Fiona Burgess, and Victoria A. Braithwaite. 2007. {``Heritable and Experiential Effects on Boldness in a Tropical Poeciliid.''} \emph{Behavioral Ecology and Sociobiology} 62 (2): 237--43. \url{https://doi.org/10.1007/s00265-007-0458-3}.

\leavevmode\vadjust pre{\hypertarget{ref-buschMaxUndMoritz2015}{}}%
Busch, Wilhelm. 2015. \emph{Max und Moritz: Eine Bubengeschichte in sieben Streichen}. {Esslinger}. \url{https://books.google.com?id=U9nGrQEACAAJ}.

\leavevmode\vadjust pre{\hypertarget{ref-bushChapter11GenomeWide2012}{}}%
Bush, William S., and Jason H. Moore. 2012. {``Chapter 11: {Genome-Wide Association Studies}.''} \emph{PLOS Computational Biology} 8 (12): e1002822. \url{https://doi.org/10.1371/journal.pcbi.1002822}.

\leavevmode\vadjust pre{\hypertarget{ref-bussEvolutionaryPersonalityPsychology1991}{}}%
Buss, David M. 1991. {``Evolutionary Personality Psychology.''} \emph{Annual Review of Psychology} 42: 459--91. \url{https://doi.org/10.1146/annurev.ps.42.020191.002331}.

\leavevmode\vadjust pre{\hypertarget{ref-caballeroQuantitativeGenetics2020}{}}%
Caballero, Armando. 2020. \emph{Quantitative {Genetics}}. {Cambridge University Press}. \url{https://books.google.com?id=yOfWDwAAQBAJ}.

\leavevmode\vadjust pre{\hypertarget{ref-campbellKingJamesBible2010}{}}%
Campbell, Gordon. 2010. \emph{King {James Bible}: 400th {Anniversary Edition}}. {OUP Oxford}.

\leavevmode\vadjust pre{\hypertarget{ref-carlsonPersonalityTraitsAre2013}{}}%
Carlson, Bradley E., and Tracy Langkilde. 2013. {``Personality {Traits Are Expressed} in {Bullfrog Tadpoles} During {Open-Field Trials}.''} \emph{Journal of Herpetology} 47 (2): 378--83. \url{https://doi.org/10.1670/12-061}.

\leavevmode\vadjust pre{\hypertarget{ref-carolaHiddenMarkovModel2011}{}}%
Carola, Valeria, Olivier Mirabeau, and Cornelius T. Gross. 2011. {``Hidden {Markov Model Analysis} of {Maternal Behavior Patterns} in {Inbred} and {Reciprocal Hybrid Mice}.''} \emph{PLOS ONE} 6 (3): e14753. \url{https://doi.org/10.1371/journal.pone.0014753}.

\leavevmode\vadjust pre{\hypertarget{ref-carterHowNotMeasure2012}{}}%
Carter, Alecia J., Harry H. Marshall, Robert Heinsohn, and Guy Cowlishaw. 2012. {``How Not to Measure Boldness: Novel Object and Antipredator Responses Are Not the Same in Wild Baboons.''} \emph{Animal Behaviour} 84 (3): 603--9. \url{https://doi.org/10.1016/j.anbehav.2012.06.015}.

\leavevmode\vadjust pre{\hypertarget{ref-changSecondgenerationPLINKRising2015}{}}%
Chang, Christopher C, Carson C Chow, Laurent CAM Tellier, Shashaank Vattikuti, Shaun M Purcell, and James J Lee. 2015. {``Second-Generation {PLINK}: Rising to the Challenge of Larger and Richer Datasets.''} \emph{GigaScience} 4 (1): 7. \url{https://doi.org/10.1186/s13742-015-0047-8}.

\leavevmode\vadjust pre{\hypertarget{ref-charlesworthDarwinGenetics2009}{}}%
Charlesworth, Brian, and Deborah Charlesworth. 2009. {``Darwin and {Genetics}.''} \emph{Genetics} 183 (3): 757--66. \url{https://doi.org/10.1534/genetics.109.109991}.

\leavevmode\vadjust pre{\hypertarget{ref-charlesworthGeneticsInbreedingDepression2009}{}}%
Charlesworth, Deborah, and John H. Willis. 2009. {``The Genetics of Inbreeding Depression.''} \emph{Nature Reviews Genetics} 10 (11, 11): 783--96. \url{https://doi.org/10.1038/nrg2664}.

\leavevmode\vadjust pre{\hypertarget{ref-claesGenomewideMappingGlobaltolocal2018}{}}%
Claes, Peter, Jasmien Roosenboom, Julie D. White, Tomek Swigut, Dzemila Sero, Jiarui Li, Myoung Keun Lee, et al. 2018. {``Genome-Wide Mapping of Global-to-Local Genetic Effects on Human Facial Shape.''} \emph{Nature Genetics} 50 (3, 3): 414--23. \url{https://doi.org/10.1038/s41588-018-0057-4}.

\leavevmode\vadjust pre{\hypertarget{ref-clarkImportanceInformationRelatives2012}{}}%
Clark, Samuel A., John M. Hickey, Hans D. Daetwyler, and Julius HJ van der Werf. 2012. {``The Importance of Information on Relatives for the Prediction of Genomic Breeding Values and the Implications for the Makeup of Reference Data Sets in Livestock Breeding Schemes.''} \emph{Genetics Selection Evolution} 44 (1): 4. \url{https://doi.org/10.1186/1297-9686-44-4}.

\leavevmode\vadjust pre{\hypertarget{ref-clarkeBasicStatisticalAnalysis2011}{}}%
Clarke, Geraldine M., Carl A. Anderson, Fredrik H. Pettersson, Lon R. Cardon, Andrew P. Morris, and Krina T. Zondervan. 2011. {``Basic Statistical Analysis in Genetic Case-Control Studies.''} \emph{Nature Protocols} 6 (2, 2): 121--33. \url{https://doi.org/10.1038/nprot.2010.182}.

\leavevmode\vadjust pre{\hypertarget{ref-cockTreasureYourExceptions2008}{}}%
Cock, Alan, and Donald R. Forsdyke. 2008. \emph{Treasure {Your Exceptions}: {The Science} and {Life} of {William Bateson}}. {Springer Science \& Business Media}. \url{https://books.google.com?id=FDUi1xg26gsC}.

\leavevmode\vadjust pre{\hypertarget{ref-collinsVariationsThemeCataloging1997}{}}%
Collins, Francis S., Mark S. Guyer, and Aravinda Chakravarti. 1997. {``Variations on a {Theme}: {Cataloging Human DNA Sequence Variation}.''} \emph{Science} 278 (5343): 1580--81. \url{https://doi.org/10.1126/science.278.5343.1580}.

\leavevmode\vadjust pre{\hypertarget{ref-conoverPracticalNonparametricStatistics1999}{}}%
Conover, W. J. 1999. \emph{Practical {Nonparametric Statistics}}. {John Wiley \& Sons}. \url{https://books.google.com?id=n_39DwAAQBAJ}.

\leavevmode\vadjust pre{\hypertarget{ref-10002015global}{}}%
Consortium, 1000 Genomes Project et al. 2015. {``A Global Reference for Human Genetic Variation.''} \emph{Nature} 526 (7571): 68.

\leavevmode\vadjust pre{\hypertarget{ref-internationalhumangenomesequencingconsortiumInitialSequencingAnalysis2001a}{}}%
Consortium, International Human Genome Sequencing. 2001. {``Initial Sequencing and Analysis of the Human Genome.''} \emph{Nature} 409 (6822, 6822): 860--921. \url{https://doi.org/10.1038/35057062}.

\leavevmode\vadjust pre{\hypertarget{ref-internationalhumangenomesequencingconsortiumInternationalConsortiumCompletes2003}{}}%
---------. 2003. {``International {Consortium Completes Human Genome Project}.''} {Genome.gov}. April 14, 2003. \url{https://www.genome.gov/11006929/2003-release-international-consortium-completes-hgp}.

\leavevmode\vadjust pre{\hypertarget{ref-coyneTheodosiusDobzhanskyHybrid2016}{}}%
Coyne, Jerry A. 2016. {``Theodosius {Dobzhansky} on {Hybrid Sterility} and {Speciation}.''} \emph{Genetics} 202 (1): 5--7. \url{https://doi.org/10.1534/genetics.115.184770}.

\leavevmode\vadjust pre{\hypertarget{ref-dahlbomBoldnessPredictsSocial2011}{}}%
Dahlbom, S. Josefin, David Lagman, Katrin Lundstedt-Enkel, L. Fredrik Sundström, and Svante Winberg. 2011. {``Boldness {Predicts Social Status} in {Zebrafish} ({Danio} Rerio).''} \emph{PLOS ONE} 6 (8): e23565. \url{https://doi.org/10.1371/journal.pone.0023565}.

\leavevmode\vadjust pre{\hypertarget{ref-dammhahnRiskTakingForaging2012}{}}%
Dammhahn, Melanie, and Laura Almeling. 2012. {``Is Risk Taking During Foraging a Personality Trait? {A} Field Test for Cross-Context Consistency in Boldness.''} \emph{Animal Behaviour} 84 (5): 1131--39. \url{https://doi.org/10.1016/j.anbehav.2012.08.014}.

\leavevmode\vadjust pre{\hypertarget{ref-danecekTwelveYearsSAMtools2021}{}}%
Danecek, Petr, James K Bonfield, Jennifer Liddle, John Marshall, Valeriu Ohan, Martin O Pollard, Andrew Whitwham, et al. 2021. {``Twelve Years of {SAMtools} and {BCFtools}.''} \emph{GigaScience} 10 (2): giab008. \url{https://doi.org/10.1093/gigascience/giab008}.

\leavevmode\vadjust pre{\hypertarget{ref-darwinJournalResearchesNatural1845}{}}%
Darwin, Charles. 1845. \emph{Journal of {Researches Into} the {Natural History} and {Geology} of the {Countries Visited During} the {Voyage} of {H}.{M}.{S}. {Beagle Round} the {World}: {Under} the {Command} of {Capt}. {Fitz Roy}, {R}.{N}}. 2nd ed. {J. Murray}. \url{https://books.google.com?id=nXm9vEpV8BAC}.

\leavevmode\vadjust pre{\hypertarget{ref-darwinOriginSpeciesMeans1859}{}}%
---------. 1859. \emph{On the {Origin} of {Species} by {Means} of {Natural Selection}, {Or}, {The Preservation} of {Favoured Races} in the {Struggle} for {Life}}. {J. Murray}. \url{https://books.google.com?id=gGUYt2PsDJcC}.

\leavevmode\vadjust pre{\hypertarget{ref-darwinNaturalistVoyageJournal1882}{}}%
---------. 1882. \emph{A {Naturalist}'s {Voyage}: {Journal} of {Researches Into} the {Natural History} and {Geology} of the {Countries Visited During} the {Voyage} of {H}.{M}.{S}. {Beagle} : With {Maps} and {Illustrations}}. {Murray}. \url{https://books.google.com?id=JyR7HY3FMtkC}.

\leavevmode\vadjust pre{\hypertarget{ref-darwinLifeLettersCharles1887}{}}%
---------. 1887. \emph{The {Life} and {Letters} of {Charles Darwin}: {Including} an {Autobiographical Chapter}}. {D. Appleton}. \url{https://books.google.com?id=iodTqIRzVukC}.

\leavevmode\vadjust pre{\hypertarget{ref-darwinCharlesDarwinLetters1998}{}}%
---------. 1998. \emph{Charles {Darwin}'s {Letters}: {A Selection}, 1825-1859}. {Cambridge University Press}. \url{https://books.google.com?id=E2TTcP6KUHgC}.

\leavevmode\vadjust pre{\hypertarget{ref-darwinAutobiographyCharlesDarwin2019}{}}%
---------. 2019. \emph{The {Autobiography} of {Charles Darwin}}. {BoD -- Books on Demand}. \url{https://books.google.com?id=t_SxDwAAQBAJ}.

\leavevmode\vadjust pre{\hypertarget{ref-darwinCorrespondenceCharlesDarwin2002}{}}%
Darwin, Charles, Duncan M. Porter, Sheila Ann Dean, Samantha Evans, Shelley Innes, and Alison M. Pearn. 2002. \emph{The {Correspondence} of {Charles Darwin}: 1821-1860}. {Cambridge University Press}. \url{https://books.google.com?id=oCWGX18UrhEC}.

\leavevmode\vadjust pre{\hypertarget{ref-azevedoShynessBoldnessGreater2006}{}}%
de Azevedo, Cristiano S., and Robert J. Young. 2006. {``Shyness and Boldness in Greater Rheas {Rhea} Americana {Linnaeus} ({Rheiformes}, {Rheidae}): The Effects of Antipredator Training on the Personality of the Birds.''} \emph{Revista Brasileira de Zoologia} 23 (March): 202--10. \url{https://doi.org/10.1590/S0101-81752006000100012}.

\leavevmode\vadjust pre{\hypertarget{ref-vriesMutationTheory19091909}{}}%
de Vries, Hugo. 1909. \emph{The {Mutation} Theory v. 1, 1909}. {Open Court Publishing Company}. \url{https://books.google.com?id=cdOhB5p3HkIC}.

\leavevmode\vadjust pre{\hypertarget{ref-depristoFrameworkVariationDiscovery2011}{}}%
DePristo, Mark A., Eric Banks, Ryan Poplin, Kiran V. Garimella, Jared R. Maguire, Christopher Hartl, Anthony A. Philippakis, et al. 2011. {``A Framework for Variation Discovery and Genotyping Using Next-Generation {DNA} Sequencing Data.''} \emph{Nature Genetics} 43 (5, 5): 491--98. \url{https://doi.org/10.1038/ng.806}.

\leavevmode\vadjust pre{\hypertarget{ref-dirienzoPopulationGeneticsModels2006}{}}%
Di Rienzo, Anna. 2006. {``Population Genetics Models of Common Diseases.''} \emph{Current Opinion in Genetics \& Development}, Genomes and evolution, 16 (6): 630--36. \url{https://doi.org/10.1016/j.gde.2006.10.002}.

\leavevmode\vadjust pre{\hypertarget{ref-dobzhanskyGeneticsNaturalPopulations1943}{}}%
Dobzhansky, Th. 1943. {``Genetics of Natural Populations {IX}. {Temporal} Changes in the Composition of Populations of Drosophila Pseudoobscura.''} \emph{Genetics} 28 (2): 162--86. \url{https://doi.org/10.1093/genetics/28.2.162}.

\leavevmode\vadjust pre{\hypertarget{ref-domingueSocialGenomeFriends2018}{}}%
Domingue, Benjamin W., Daniel W. Belsky, Jason M. Fletcher, Dalton Conley, Jason D. Boardman, and Kathleen Mullan Harris. 2018. {``The Social Genome of Friends and Schoolmates in the {National Longitudinal Study} of {Adolescent} to {Adult Health}.''} \emph{Proceedings of the National Academy of Sciences} 115 (4): 702--7. \url{https://doi.org/10.1073/pnas.1711803115}.

\leavevmode\vadjust pre{\hypertarget{ref-drueryExperimentsPlantHybridization1901}{}}%
Druery, C. T., and William Bateson. 1901. {``Experiments in Plant Hybridization.''} \emph{Journal of the Royal Horticultural Society} 26: 1--32.

\leavevmode\vadjust pre{\hypertarget{ref-duncanAnalysisPolygenicRisk2019}{}}%
Duncan, L., H. Shen, B. Gelaye, J. Meijsen, K. Ressler, M. Feldman, R. Peterson, and B. Domingue. 2019. {``Analysis of Polygenic Risk Score Usage and Performance in Diverse Human Populations.''} \emph{Nature Communications} 10 (1, 1): 3328. \url{https://doi.org/10.1038/s41467-019-11112-0}.

\leavevmode\vadjust pre{\hypertarget{ref-dunnShortHistoryGenetics1991}{}}%
Dunn, Leslie Clarence. 1991. \emph{A {Short History} of {Genetics}: {The Development} of {Some} of the {Main Lines} of {Thought}, 1864-1939}. {Iowa State University Press}. \url{https://books.google.com?id=MpEiAQAAMAAJ}.

\leavevmode\vadjust pre{\hypertarget{ref-durandTestingAncientAdmixture2011}{}}%
Durand, Eric Y., Nick Patterson, David Reich, and Montgomery Slatkin. 2011. {``Testing for {Ancient Admixture} Between {Closely Related Populations}.''} \emph{Molecular Biology and Evolution} 28 (8): 2239--52. \url{https://doi.org/10.1093/molbev/msr048}.

\leavevmode\vadjust pre{\hypertarget{ref-durbinDistributionTheoryTests1973}{}}%
Durbin, J. 1973. \emph{Distribution {Theory} for {Tests Based} on {Sample Distribution Function}}. {SIAM}. \url{https://books.google.com?id=zAryCrT1IUYC}.

\leavevmode\vadjust pre{\hypertarget{ref-eatonQuartercenturyInbreedingGuineapigs1932}{}}%
Eaton, Orson N. 1932. {``A Quarter-Century of Inbreeding in Guinea-Pigs.''} \emph{Journal of Experimental Zoology} 63 (2): 261--90. \url{https://doi.org/10.1002/jez.1400630202}.

\leavevmode\vadjust pre{\hypertarget{ref-edgeReconstructingHistoryPolygenic2019}{}}%
Edge, Michael D, and Graham Coop. 2019. {``Reconstructing the {History} of {Polygenic Scores Using Coalescent Trees}.''} \emph{Genetics} 211 (1): 235--62. \url{https://doi.org/10.1534/genetics.118.301687}.

\leavevmode\vadjust pre{\hypertarget{ref-ellenProspectsSelectionSocial2014}{}}%
Ellen, Esther D., T. Bas Rodenburg, Gerard A. A. Albers, J. Elizabeth Bolhuis, Irene Camerlink, Naomi Duijvesteijn, Egbert F. Knol, et al. 2014. {``The Prospects of Selection for Social Genetic Effects to Improve Welfare and Productivity in Livestock.''} \emph{Frontiers in Genetics} 5. \url{https://www.frontiersin.org/articles/10.3389/fgene.2014.00377}.

\leavevmode\vadjust pre{\hypertarget{ref-evansQTLGeneElegans2021}{}}%
Evans, Kathryn S., Marijke H. van Wijk, Patrick T. McGrath, Erik C. Andersen, and Mark G. Sterken. 2021. {``From {QTL} to Gene: {C}. Elegans Facilitates Discoveries of the Genetic Mechanisms Underlying Natural Variation.''} \emph{Trends in Genetics} 37 (10): 933--47. \url{https://doi.org/10.1016/j.tig.2021.06.005}.

\leavevmode\vadjust pre{\hypertarget{ref-ewensTransmissionDisequilibriumTest1995}{}}%
Ewens, W J, and R S Spielman. 1995. {``The Transmission/Disequilibrium Test: History, Subdivision, and Admixture.''} \emph{American Journal of Human Genetics} 57 (2): 455--64. \url{https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1801556/}.

\leavevmode\vadjust pre{\hypertarget{ref-fairbanksRelicsEdenPowerful2009}{}}%
Fairbanks, Daniel J. 2009. \emph{Relics of {Eden}: {The Powerful Evidence} of {Evolution} in {Human DNA}}. {Prometheus Books}. \url{https://books.google.com?id=O1mFjWzznWkC}.

\leavevmode\vadjust pre{\hypertarget{ref-falconerIntroductionQuantitativeGenetics1996}{}}%
Falconer, D. S., and T. F. C. Mackay. 1996. \emph{Introduction to Quantitative Genetics}. {UK: Longman Group}.

\leavevmode\vadjust pre{\hypertarget{ref-falkImagingOnsetOscillatory2022}{}}%
Falk, Henning J, Takehito Tomita, Gregor Mönke, Katie McDole, and Alexander Aulehla. 2022. {``Imaging the Onset of Oscillatory Signaling Dynamics During Mouse Embryo Gastrulation.''} \emph{Development (Cambridge, England)} 149 (13): dev200083. \url{https://doi.org/10.1242/dev.200083}.

\leavevmode\vadjust pre{\hypertarget{ref-falker-gieskeGWASMeatCarcass2019}{}}%
Falker-Gieske, Clemens, Iulia Blaj, Siegfried Preuß, Jörn Bennewitz, Georg Thaller, and Jens Tetens. 2019. {``{GWAS} for {Meat} and {Carcass Traits Using Imputed Sequence Level Genotypes} in {Pooled F2-Designs} in {Pigs}.''} \emph{G3 Genes\textbar Genomes\textbar Genetics} 9 (9): 2823--34. \url{https://doi.org/10.1534/g3.119.400452}.

\leavevmode\vadjust pre{\hypertarget{ref-fisherXVCorrelationRelatives1918}{}}%
Fisher, R. A. 1918. {``{XV}.---{The Correlation} Between {Relatives} on the {Supposition} of {Mendelian Inheritance}.''} October. \url{https://doi.org/10.1017/s0080456800012163}.

\leavevmode\vadjust pre{\hypertarget{ref-fisherProbableErrorCoefficient1921}{}}%
Fisher, Ronald Aylmer. 1921. {``On the{"} {Probable Error}{"} of a {Coefficient} of {Correlation Deduced} from a {Small Sample}.''}

\leavevmode\vadjust pre{\hypertarget{ref-fisherStatisticalMethodsResearch1925}{}}%
---------. 1925. \emph{Statistical {Methods} for {Research Workers}}. {Oliver and Boyd}. \url{https://books.google.com?id=1NApzwEACAAJ}.

\leavevmode\vadjust pre{\hypertarget{ref-fitzgeraldMedakaInbredKiyosuKarlsruhe2022}{}}%
Fitzgerald, Tomas, Ian Brettell, Adrien Leger, Nadeshda Wolf, Natalja Kusminski, Jack Monahan, Carl Barton, et al. 2022. {``The {Medaka Inbred Kiyosu-Karlsruhe} ({MIKK}) Panel.''} \emph{Genome Biology} 23 (1): 59. \url{https://doi.org/10.1186/s13059-022-02623-z}.

\leavevmode\vadjust pre{\hypertarget{ref-franklinMolecularConfigurationSodium1953}{}}%
Franklin, Rosalind E., and Raymond G. Gosling. 1953. {``Molecular Configuration in Sodium Thymonucleate.''} \emph{Nature} 171 (4356): 740--41.

\leavevmode\vadjust pre{\hypertarget{ref-fredmanComplexSNPrelatedSequence2004}{}}%
Fredman, David, Stefan J. White, Susanna Potter, Evan E. Eichler, Johan T. Den Dunnen, and Anthony J. Brookes. 2004. {``Complex {SNP-related} Sequence Variation in Segmental Genome Duplications.''} \emph{Nature Genetics} 36 (8, 8): 861--66. \url{https://doi.org/10.1038/ng1401}.

\leavevmode\vadjust pre{\hypertarget{ref-fryComparisonSociodemographicHealthRelated2017}{}}%
Fry, Anna, Thomas J Littlejohns, Cathie Sudlow, Nicola Doherty, Ligia Adamska, Tim Sprosen, Rory Collins, and Naomi E Allen. 2017. {``Comparison of {Sociodemographic} and {Health-Related Characteristics} of {UK Biobank Participants With Those} of the {General Population}.''} \emph{American Journal of Epidemiology} 186 (9): 1026--34. \url{https://doi.org/10.1093/aje/kwx246}.

\leavevmode\vadjust pre{\hypertarget{ref-fukamachi100YearsMedaka2021}{}}%
Fukamachi, Shoji, and Kiyoshi Naruse. 2021. {``100 Years Since the Medaka's International Debut: {Aida}'s Legacy.''} {Genes to Genomes}. August 20, 2021. \url{https://genestogenomes.org/100-years-since-the-medakas-international-debut-aidas-legacy/}.

\leavevmode\vadjust pre{\hypertarget{ref-fullerMeasuringIntoleranceMutation2019}{}}%
Fuller, Zachary L., Jeremy J. Berg, Hakhamanesh Mostafavi, Guy Sella, and Molly Przeworski. 2019. {``Measuring Intolerance to Mutation in Human Genetics.''} \emph{Nature Genetics} 51 (5, 5): 772--76. \url{https://doi.org/10.1038/s41588-019-0383-1}.

\leavevmode\vadjust pre{\hypertarget{ref-galtonMenScienceTheir1874a}{}}%
Galton, Francis. 1874. {``On {Men} of {Science}, Their {Nature} and Their {Nurture}.''} In \emph{Proceedings of the {Royal Institution} of {Great Britain}}, 7:227--36.

\leavevmode\vadjust pre{\hypertarget{ref-galtonInquiriesHumanFaculty1883}{}}%
---------. 1883. \emph{Inquiries {Into Human Faculty} and {Its Development}}. {Macmillan}. \url{https://books.google.com?id=GGAZAAAAYAAJ}.

\leavevmode\vadjust pre{\hypertarget{ref-galtonRegressionMediocrityHereditary1886}{}}%
---------. 1886. {``Regression {Towards Mediocrity} in {Hereditary Stature}.''} \emph{The Journal of the Anthropological Institute of Great Britain and Ireland} 15: 246--63. \url{https://doi.org/10.2307/2841583}.

\leavevmode\vadjust pre{\hypertarget{ref-galtonNaturalInheritance1889}{}}%
---------. 1889. \emph{Natural Inheritance}. {Macmillan and Company}.

\leavevmode\vadjust pre{\hypertarget{ref-galtonEugenicsItsDefinition1904}{}}%
---------. 1904. {``Eugenics: {Its Definition}, {Scope}, and {Aims}.''} \emph{American Journal of Sociology} 10 (1): 1--25. \url{https://doi.org/10.1086/211280}.

\leavevmode\vadjust pre{\hypertarget{ref-garciadeleanizGeneticDeterminationContribution1989}{}}%
Garcia de Le'aniz, C., E. Verspoor, and A. D. Hawkins. 1989. {``Genetic Determination of the Contribution of Stocked and Wild {Atlantic} Salmon, {Salmo} Salar {L}., To the Angling Fisheries in Two {Spanish} Rivers.''} \emph{Journal of Fish Biology} 35: 261--70. \url{https://doi.org/10.1111/j.1095-8649.1989.tb03069.x}.

\leavevmode\vadjust pre{\hypertarget{ref-garfieldDistinctPhysiologicalBehavioural2011}{}}%
Garfield, Alastair S., Michael Cowley, Florentia M. Smith, Kim Moorwood, Joanne E. Stewart-Cox, Kerry Gilroy, Sian Baker, et al. 2011. {``Distinct Physiological and Behavioural Functions for Parental Alleles of Imprinted {Grb10}.''} \emph{Nature} 469 (7331, 7331): 534--38. \url{https://doi.org/10.1038/nature09651}.

\leavevmode\vadjust pre{\hypertarget{ref-goddardMappingGenesComplex2009}{}}%
Goddard, Michael E., and Ben J. Hayes. 2009. {``Mapping Genes for Complex Traits in Domestic Animals and Their Use in Breeding Programmes.''} \emph{Nature Reviews Genetics} 10 (6, 6): 381--91. \url{https://doi.org/10.1038/nrg2575}.

\leavevmode\vadjust pre{\hypertarget{ref-gomezControlSegmentNumber2008}{}}%
Gomez, C'eline, Ertuğrul M. Özbudak, Joshua Wunderlich, Diana Baumann, Julian Lewis, and Olivier Pourqui'e. 2008. {``Control of Segment Number in Vertebrate Embryos.''} \emph{Nature} 454 (7202, 7202): 335--39. \url{https://doi.org/10.1038/nature07020}.

\leavevmode\vadjust pre{\hypertarget{ref-goodnightSewallWrightSeven2014}{}}%
Goodnight, Charles. 2014. {``Sewall {Wright}'s {Seven Generalizations} about {Populations}.''} {Evolution in Structured Populations}. May 22, 2014. \url{https://blog.uvm.edu/cgoodnig/2014/05/22/sewall-wrights-seven-generalizations-about-populations/}.

\leavevmode\vadjust pre{\hypertarget{ref-gornerAlteredFatesGenetic1996}{}}%
Gorner, Peter, and Jeff Lyon. 1996. \emph{Altered {Fates}: {The Genetic Re-Engineering} of {Human Life}}. {W. W. Norton, Incorporated}. \url{https://books.google.com?id=MrhEzwEACAAJ}.

\leavevmode\vadjust pre{\hypertarget{ref-gravel2011demographic}{}}%
Gravel, Simon, Brenna M Henn, Ryan N Gutenkunst, Amit R Indap, Gabor T Marth, Andrew G Clark, Fuli Yu, et al. 2011. {``Demographic History and Rare Allele Sharing Among Human Populations.''} \emph{Proceedings of the National Academy of Sciences} 108 (29): 11983--88. \url{https://doi.org/10.1073/pnas.1019276108}.

\leavevmode\vadjust pre{\hypertarget{ref-gridleyLongShortIt2006}{}}%
Gridley, Thomas. 2006. {``The Long and Short of It: {Somite} Formation in Mice.''} \emph{Developmental Dynamics} 235 (9): 2330--36. \url{https://doi.org/10.1002/dvdy.20850}.

\leavevmode\vadjust pre{\hypertarget{ref-grosEvolutionEpistasisIts2009}{}}%
Gros, Pierre-Alexis, Herv'e Le Nagard, and Olivier Tenaillon. 2009. {``The {Evolution} of {Epistasis} and {Its Links With Genetic Robustness}, {Complexity} and {Drift} in a {Phenotypic Model} of {Adaptation}.''} \emph{Genetics} 182 (1): 277--93. \url{https://doi.org/10.1534/genetics.108.099127}.

\leavevmode\vadjust pre{\hypertarget{ref-guGenomeWideAssociationStudy2011}{}}%
Gu, Xiaorong, Chungang Feng, Li Ma, Chi Song, Yanqiang Wang, Yang Da, Huifang Li, et al. 2011. {``Genome-{Wide Association Study} of {Body Weight} in {Chicken F2 Resource Population}.''} \emph{PLOS ONE} 6 (7): e21872. \url{https://doi.org/10.1371/journal.pone.0021872}.

\leavevmode\vadjust pre{\hypertarget{ref-guCirclizeImplementsEnhances2014}{}}%
Gu, Zuguang, Lei Gu, Roland Eils, Matthias Schlesner, and Benedikt Brors. 2014. {``Circlize Implements and Enhances Circular Visualization in {R}.''} \emph{Bioinformatics} 30 (19): 2811--12. \url{https://doi.org/10.1093/bioinformatics/btu393}.

\leavevmode\vadjust pre{\hypertarget{ref-guoBayesianHierarchicalModel2009}{}}%
Guo, Feng, Dipak K. Dey, and Kent E. Holsinger. 2009. {``A {Bayesian Hierarchical Model} for {Analysis} of {Single-Nucleotide Polymorphisms Diversity} in {Multilocus}, {Multipopulation Samples}.''} \emph{Journal of the American Statistical Association} 104 (485): 142--54. \url{https://doi.org/10.1198/jasa.2009.0010}.

\leavevmode\vadjust pre{\hypertarget{ref-gusellaPolymorphicDNAMarker1983}{}}%
Gusella, James F., Nancy S. Wexler, P. Michael Conneally, Susan L. Naylor, Mary Anne Anderson, Rudolph E. Tanzi, Paul C. Watkins, et al. 1983. {``A Polymorphic {DNA} Marker Genetically Linked to {Huntington}'s Disease.''} \emph{Nature} 306 (5940, 5940): 234--38. \url{https://doi.org/10.1038/306234a0}.

\leavevmode\vadjust pre{\hypertarget{ref-habierImpactGeneticRelationship2010}{}}%
Habier, David, Jens Tetens, Franz-Reinhold Seefried, Peter Lichtner, and Georg Thaller. 2010. {``The Impact of Genetic Relationship Information on Genomic Breeding Values in {German Holstein} Cattle.''} \emph{Genetics Selection Evolution} 42 (1): 5. \url{https://doi.org/10.1186/1297-9686-42-5}.

\leavevmode\vadjust pre{\hypertarget{ref-haenelMetaanalysisChromosomescaleCrossover2018}{}}%
Haenel, Quiterie, Telma G. Laurentino, Marius Roesti, and Daniel Berner. 2018. {``Meta-Analysis of Chromosome-Scale Crossover Rate Variation in Eukaryotes and Its Significance to Evolutionary Genomics.''} \emph{Molecular Ecology} 27 (11): 2477--97. \url{https://doi.org/10.1111/mec.14699}.

\leavevmode\vadjust pre{\hypertarget{ref-hamiltonShoalingBoldnessAnxietylike2021}{}}%
Hamilton, Trevor J., Jeffrey Krook, Joshua Szaszkiewicz, and Warren Burggren. 2021. {``Shoaling, Boldness, Anxiety-Like Behavior and Locomotion in Zebrafish ({Danio} Rerio) Are Altered by Acute Benzo{[}a{]}pyrene Exposure.''} \emph{Science of The Total Environment} 774 (June): 145702. \url{https://doi.org/10.1016/j.scitotenv.2021.145702}.

\leavevmode\vadjust pre{\hypertarget{ref-hardenGeneticLotteryWhy2021}{}}%
Harden, Kathryn Paige. 2021. \emph{The {Genetic Lottery}: {Why DNA Matters} for {Social Equality}}. {Princeton University Press}. \url{https://books.google.com?id=R9oiEAAAQBAJ}.

\leavevmode\vadjust pre{\hypertarget{ref-harpakEvolutionGroupDifferences2021}{}}%
Harpak, Arbel, and Molly Przeworski. 2021. {``The Evolution of Group Differences in Changing Environments.''} \emph{PLOS Biology} 19 (1): e3001072. \url{https://doi.org/10.1371/journal.pbio.3001072}.

\leavevmode\vadjust pre{\hypertarget{ref-hartsoekerEssayDioptrique1694}{}}%
Hartsoeker, Nicolas. 1694. \emph{Essay de dioptrique}. {J. Anisson}. \url{https://books.google.com?id=IzYVAAAAQAAJ}.

\leavevmode\vadjust pre{\hypertarget{ref-hawkinsSocialDarwinismEuropean1997}{}}%
Hawkins, Mike. 1997. \emph{Social darwinism in European and American thought, 1860-1945: nature as model and nature as threat}. {Cambridge University Press}. \url{https://books.google.com?id=aoXJoAEACAAJ}.

\leavevmode\vadjust pre{\hypertarget{ref-hendersonEstimationGeneralSpecific1948}{}}%
Henderson, Charles Roy. 1948. {``Estimation of General, Specific and Maternal Combining Abilities in Crosses Among Inbred Lines of Swine.''} PhD thesis, {United States -- Iowa}: {Iowa State University}. \url{https://www.proquest.com/docview/301840571/abstract/4D5A65EDC31B4110PQ/1}.

\leavevmode\vadjust pre{\hypertarget{ref-henigMonkGardenLost2000}{}}%
Henig, Robin Marantz. 2000. \emph{The {Monk} in the {Garden}: {The Lost} and {Found Genius} of {Gregor Mendel}, the {Father} of {Genetics}}. {Houghton Mifflin Harcourt}.

\leavevmode\vadjust pre{\hypertarget{ref-henslowLetter1051831}{}}%
Henslow, John. 1831. {``Letter 105.''} {Darwin Correspondence Project}. 1831. \url{https://www.darwinproject.ac.uk/letter/DCP-LETT-105.xml}.

\leavevmode\vadjust pre{\hypertarget{ref-herdeConsistencyBoldnessActivity2013}{}}%
Herde, Antje, and Jana A. Eccard. 2013. {``Consistency in Boldness, Activity and Exploration at Different Stages of Life.''} \emph{BMC Ecology} 13 (1): 49. \url{https://doi.org/10.1186/1472-6785-13-49}.

\leavevmode\vadjust pre{\hypertarget{ref-hillLinkageDisequilibriumFinite1968}{}}%
Hill, W. G., and Alan Robertson. 1968. {``Linkage Disequilibrium in Finite Populations.''} \emph{Theoretical and Applied Genetics} 38 (6): 226--31. \url{https://doi.org/10.1007/BF01245622}.

\leavevmode\vadjust pre{\hypertarget{ref-hillApplicationsPopulationGenetics2014}{}}%
Hill, William G. 2014. {``Applications of {Population Genetics} to {Animal Breeding}, from {Wright}, {Fisher} and {Lush} to {Genomic Prediction}.''} \emph{Genetics} 196 (1): 1--16. \url{https://doi.org/10.1534/genetics.112.147850}.

\leavevmode\vadjust pre{\hypertarget{ref-hillDataTheoryPoint2008}{}}%
Hill, William G., Michael E. Goddard, and Peter M. Visscher. 2008. {``Data and {Theory Point} to {Mainly Additive Genetic Variance} for {Complex Traits}.''} \emph{PLOS Genetics} 4 (2): e1000008. \url{https://doi.org/10.1371/journal.pgen.1000008}.

\leavevmode\vadjust pre{\hypertarget{ref-hillsDeletionsGRID2Lead2013}{}}%
Hills, L. Benjamin, Amira Masri, Kotaro Konno, Wataru Kakegawa, Anh-Thu N. Lam, Elizabeth Lim-Melia, Nandini Chandy, et al. 2013. {``Deletions in {Grid2} Lead to a Recessive Syndrome of Cerebellar Ataxia and Tonic Upgaze in Humans.''} \emph{Neurology} 81 (16): 1378--86. \url{https://doi.org/10.1212/WNL.0b013e3182a841a3}.

\leavevmode\vadjust pre{\hypertarget{ref-HmmlearnHmmlearn2022}{}}%
\emph{Hmmlearn/Hmmlearn}. (2014) 2022. {hmmlearn}. \url{https://github.com/hmmlearn/hmmlearn}.

\leavevmode\vadjust pre{\hypertarget{ref-hochbergMorePowerfulProcedures1990}{}}%
Hochberg, Yosef, and Yoav Benjamini. 1990. {``More Powerful Procedures for Multiple Significance Testing.''} \emph{Statistics in Medicine} 9 (7): 811--18. \url{https://doi.org/10.1002/sim.4780090710}.

\leavevmode\vadjust pre{\hypertarget{ref-hodgeGeneticEngineeringManipulating2009}{}}%
Hodge, Russ. 2009. \emph{Genetic {Engineering}: {Manipulating} the {Mechanisms} of {Life}}. {Infobase Publishing}. \url{https://books.google.com?id=Fqa6lmQ_YVMC}.

\leavevmode\vadjust pre{\hypertarget{ref-hoffmannChromosomalInversionPolymorphisms2004}{}}%
Hoffmann, Ary A., Carla M. Sgr`o, and Andrew R. Weeks. 2004. {``Chromosomal Inversion Polymorphisms and Adaptation.''} \emph{Trends in Ecology \& Evolution} 19 (9): 482--88. \url{https://doi.org/10.1016/j.tree.2004.06.013}.

\leavevmode\vadjust pre{\hypertarget{ref-hollanderNonparametricStatisticalMethods2013}{}}%
Hollander, Myles, Douglas A. Wolfe, and Eric Chicken. 2013. \emph{Nonparametric {Statistical Methods}}. {John Wiley \& Sons}. \url{https://books.google.com?id=Y5s3AgAAQBAJ}.

\leavevmode\vadjust pre{\hypertarget{ref-holsingerGeneticsGeographicallyStructured2009}{}}%
Holsinger, Kent E., and Bruce S. Weir. 2009. {``Genetics in Geographically Structured Populations: Defining, Estimating and Interpreting {FST}.''} \emph{Nature Reviews Genetics} 10 (9, 9): 639--50. \url{https://doi.org/10.1038/nrg2611}.

\leavevmode\vadjust pre{\hypertarget{ref-howeEnsembl20212021}{}}%
Howe, Kevin L, Premanand Achuthan, James Allen, Jamie Allen, Jorge Alvarez-Jarreta, M Ridwan Amode, Irina M Armean, et al. 2021. {``Ensembl 2021.''} \emph{Nucleic Acids Research} 49 (D1): D884--91. \url{https://doi.org/10.1093/nar/gkaa942}.

\leavevmode\vadjust pre{\hypertarget{ref-hubaudSignallingDynamicsVertebrate2014}{}}%
Hubaud, Alexis, and Olivier Pourqui'e. 2014. {``Signalling Dynamics in Vertebrate Segmentation.''} \emph{Nature Reviews Molecular Cell Biology} 15 (11, 11): 709--21. \url{https://doi.org/10.1038/nrm3891}.

\leavevmode\vadjust pre{\hypertarget{ref-huxleyEvolutionModernSynthesis1942}{}}%
Huxley, Julian. 1942. {``Evolution. {The} Modern Synthesis.''} \emph{Evolution. The Modern Synthesis.}

\leavevmode\vadjust pre{\hypertarget{ref-iselyOneHundredOne2002}{}}%
Isely, Duane. 2002. \emph{One {Hundred} and {One Botanists}}. {Purdue University Press}. \url{https://books.google.com?id=an6r8m0JfV8C}.

\leavevmode\vadjust pre{\hypertarget{ref-jainNanoporeSequencingAssembly2018}{}}%
Jain, Miten, Sergey Koren, Karen H. Miga, Josh Quick, Arthur C. Rand, Thomas A. Sasani, John R. Tyson, et al. 2018. {``Nanopore Sequencing and Assembly of a Human Genome with Ultra-Long Reads.''} \emph{Nature Biotechnology} 36 (4, 4): 338--45. \url{https://doi.org/10.1038/nbt.4060}.

\leavevmode\vadjust pre{\hypertarget{ref-jamainMutationsXlinkedGenes2003}{}}%
Jamain, St'ephane, H'el`ene Quach, Catalina Betancur, Maria Råstam, Catherine Colineaux, I. Carina Gillberg, Henrik Soderstrom, et al. 2003. {``Mutations of the {X-linked} Genes Encoding Neuroligins {Nlgn3} and {Nlgn4} Are Associated with Autism.''} \emph{Nature Genetics} 34 (1, 1): 27--29. \url{https://doi.org/10.1038/ng1136}.

\leavevmode\vadjust pre{\hypertarget{ref-jenkinOriginSpecies1867}{}}%
Jenkin, Fleeming. 1867. {``The Origin of Species.''} \emph{North British Review} 46 (92): 277--318.

\leavevmode\vadjust pre{\hypertarget{ref-johnsonSegregationPerformanceRecombinant1999}{}}%
Johnson, William C., and Paul Gepts. 1999. {``Segregation for Performance in Recombinant Inbred Populations Resulting from Inter-Gene Pool Crosses of Common Bean ( {Phaseolus} Vulgaris {L}.).''} \emph{Euphytica} 106 (1): 45--56. \url{https://doi.org/10.1023/A:1003541201923}.

\leavevmode\vadjust pre{\hypertarget{ref-jonietzContinuousTrajectoryPattern2018}{}}%
Jonietz, David, and Dominik Bucher. 2018. {``Continuous {Trajectory Pattern Mining} for {Mobility Behaviour Change Detection}.''} In \emph{Progress in {Location Based Services} 2018}, edited by Peter Kiefer, Haosheng Huang, Nico Van de Weghe, and Martin Raubal, 211--30. Lecture {Notes} in {Geoinformation} and {Cartography}. {Cham}: {Springer International Publishing}. \url{https://doi.org/10.1007/978-3-319-71470-7_11}.

\leavevmode\vadjust pre{\hypertarget{ref-junghansPostsynapticDifferentialLocalization2008}{}}%
Junghans, Dirk, Matthias Heidenreich, Iris Hack, Verdon Taylor, Michael Frotscher, and Rolf Kemler. 2008. {``Postsynaptic and Differential Localization to Neuronal Subtypes of Protocadherin Β16 in the Mammalian Central Nervous System.''} \emph{European Journal of Neuroscience} 27 (3): 559--71. \url{https://doi.org/10.1111/j.1460-9568.2008.06052.x}.

\leavevmode\vadjust pre{\hypertarget{ref-kandaStabilityActivityBoldness2012}{}}%
Kanda, L. Leann, Laura Louon, and Katherine Straley. 2012. {``Stability in {Activity} and {Boldness Across Time} and {Context} in {Captive Siberian Dwarf Hamsters}.''} \emph{Ethology} 118 (6): 518--33. \url{https://doi.org/10.1111/j.1439-0310.2012.02038.x}.

\leavevmode\vadjust pre{\hypertarget{ref-karlNationalLifeStandpoint1901a}{}}%
Karl, Pearson. 1901. \emph{National {Life} from the Standpoint of {Science}}. {London}.

\leavevmode\vadjust pre{\hypertarget{ref-katsumuraMedakaPopulationGenome2019}{}}%
Katsumura, Takafumi, Shoji Oda, Hiroshi Mitani, and Hiroki Oota. 2019. {``Medaka {Population Genome Structure} and {Demographic History Described} via {Genotyping-by-Sequencing}.''} \emph{G3 Genes\textbar Genomes\textbar Genetics} 9 (1): 217--28. \url{https://doi.org/10.1534/g3.118.200779}.

\leavevmode\vadjust pre{\hypertarget{ref-kenneyThomasHuntMorgan2009}{}}%
Kenney, Diana E, and Gary G Borisy. 2009. {``Thomas {Hunt Morgan} at the {Marine Biological Laboratory}: {Naturalist} and {Experimentalist}.''} \emph{Genetics} 181 (3): 841--46. \url{https://doi.org/10.1534/genetics.109.101659}.

\leavevmode\vadjust pre{\hypertarget{ref-kerminenGeographicVariationBias2019}{}}%
Kerminen, Sini, Alicia R. Martin, Jukka Koskela, Sanni E. Ruotsalainen, Aki S. Havulinna, Ida Surakka, Aarno Palotie, et al. 2019. {``Geographic {Variation} and {Bias} in the {Polygenic Scores} of {Complex Diseases} and {Traits} in {Finland}.''} \emph{The American Journal of Human Genetics} 104 (6): 1169--81. \url{https://doi.org/10.1016/j.ajhg.2019.05.001}.

\leavevmode\vadjust pre{\hypertarget{ref-kevlesNameEugenicsGenetics1995}{}}%
Kevles, Daniel J. 1995. \emph{In the {Name} of {Eugenics}: {Genetics} and the {Uses} of {Human Heredity}}. {Harvard University Press}. \url{https://books.google.com?id=NWBi9kyb8xUC}.

\leavevmode\vadjust pre{\hypertarget{ref-khannaBamreadcountRapidGeneration2022}{}}%
Khanna, Ajay, David E. Larson, Sridhar Nonavinkere Srivatsan, Matthew Mosior, Travis E. Abbott, Susanna Kiwala, Timothy J. Ley, et al. 2022. {``Bam-Readcount - Rapid Generation of Basepair-Resolution Sequence Metrics.''} \emph{Journal of Open Source Software} 7 (69): 3722. \url{https://doi.org/10.21105/joss.03722}.

\leavevmode\vadjust pre{\hypertarget{ref-kheraGenomewidePolygenicScores2018}{}}%
Khera, Amit V., Mark Chaffin, Krishna G. Aragam, Mary E. Haas, Carolina Roselli, Seung Hoan Choi, Pradeep Natarajan, et al. 2018. {``Genome-Wide Polygenic Scores for Common Diseases Identify Individuals with Risk Equivalent to Monogenic Mutations.''} \emph{Nature Genetics} 50 (9, 9): 1219--24. \url{https://doi.org/10.1038/s41588-018-0183-z}.

\leavevmode\vadjust pre{\hypertarget{ref-kimPeriodSomiteSegmentation2011}{}}%
Kim, Woong, Takaaki Matsui, Masataka Yamao, Makoto Ishibashi, Kota Tamada, Toru Takumi, Kenji Kohno, et al. 2011. {``The Period of the Somite Segmentation Clock Is Sensitive to {Notch} Activity.''} \emph{Molecular Biology of the Cell} 22 (18): 3541--49. \url{https://doi.org/10.1091/mbc.e11-02-0139}.

\leavevmode\vadjust pre{\hypertarget{ref-kleinHLASystem2000}{}}%
Klein, Jan, and Akie Sato. 2000. {``The {HLA System}.''} \emph{New England Journal of Medicine} 343 (11): 782--86. \url{https://doi.org/10.1056/NEJM200009143431106}.

\leavevmode\vadjust pre{\hypertarget{ref-knopikBehavioralGenetics2019}{}}%
Knopik, Valerie S., Jenae M. Neiderhiser, John C. DeFries, and Robert Plomin. 2019. \emph{Behavioral {Genetics}}. {Macmillan Learning}. \url{https://books.google.com?id=mtDJvQEACAAJ}.

\leavevmode\vadjust pre{\hypertarget{ref-kongNatureNurtureEffects2018}{}}%
Kong, Augustine, Gudmar Thorleifsson, Michael L. Frigge, Bjarni J. Vilhjalmsson, Alexander I. Young, Thorgeir E. Thorgeirsson, Stefania Benonisdottir, et al. 2018. {``The Nature of Nurture: {Effects} of Parental Genotypes.''} \emph{Science} 359 (6374): 424--28. \url{https://doi.org/10.1126/science.aan6877}.

\leavevmode\vadjust pre{\hypertarget{ref-kremerDecouplingDifferentiationTraits2012}{}}%
Kremer, A., and V. Le Corre. 2012. {``Decoupling of Differentiation Between Traits and Their Underlying Genes in Response to Divergent Selection.''} \emph{Heredity} 108 (4, 4): 375--85. \url{https://doi.org/10.1038/hdy.2011.81}.

\leavevmode\vadjust pre{\hypertarget{ref-kulloIncorporatingGeneticRisk2016}{}}%
Kullo, Iftikhar J., Hayan Jouni, Erin E. Austin, Sherry-Ann Brown, Teresa M. Kruisselbrink, Iyad N. Isseh, Raad A. Haddad, et al. 2016. {``Incorporating a {Genetic Risk Score Into Coronary Heart Disease Risk Estimates}.''} \emph{Circulation} 133 (12): 1181--88. \url{https://doi.org/10.1161/CIRCULATIONAHA.115.020109}.

\leavevmode\vadjust pre{\hypertarget{ref-lopez-olmedaLongPhotoperiodImpairs2021}{}}%
L'opez-Olmeda, Jose Fernando, Haiyu Zhao, Markus Reischl, Christian Pylatiuk, Tyrone Lucon-Xiccato, Felix Loosli, and Nicholas S. Foulkes. 2021. {``Long Photoperiod Impairs Learning in Male but Not Female Medaka.''} \emph{iScience} 24 (7): 102784. \url{https://doi.org/10.1016/j.isci.2021.102784}.

\leavevmode\vadjust pre{\hypertarget{ref-lairdFundamentalsModernStatistical2010}{}}%
Laird, Nan M., and Christoph Lange. 2010. \emph{The {Fundamentals} of {Modern Statistical Genetics}}. {Springer New York}. \url{https://books.google.com?id=T7GqDAEACAAJ}.

\leavevmode\vadjust pre{\hypertarget{ref-lalandFishCognitionBehavior2011}{}}%
Laland, Kevin, Jens Krause, and Culum Brown. 2011. \emph{Fish {Cognition} and {Behavior}}. {John Wiley \& Sons}. \url{https://books.google.com?id=cI9gbVyH6lsC}.

\leavevmode\vadjust pre{\hypertarget{ref-lambertClinicalUtilityPolygenic2019}{}}%
Lambert, Samuel A, Gad Abraham, and Michael Inouye. 2019. {``Towards Clinical Utility of Polygenic Risk Scores.''} \emph{Human Molecular Genetics} 28 (R2): R133--42. \url{https://doi.org/10.1093/hmg/ddz187}.

\leavevmode\vadjust pre{\hypertarget{ref-landerNewGenomicsGlobal1996}{}}%
Lander, Eric S. 1996. {``The {New Genomics}: {Global Views} of {Biology}.''} \emph{Science} 274 (5287): 536--39. \url{https://doi.org/10.1126/science.274.5287.536}.

\leavevmode\vadjust pre{\hypertarget{ref-larsonEvolutionRemarkableHistory2004}{}}%
Larson, Edward John, Richard B. Russell Professor of History, and Talmadge Professor of Law Edward J. Larson PH.D J. D. 2004. \emph{Evolution: {The Remarkable History} of a {Scientific Theory}}. {Modern Library}. \url{https://books.google.com?id=wSwWjYZqkAEC}.

\leavevmode\vadjust pre{\hypertarget{ref-laubichlerBoveriLongExperiment2008}{}}%
Laubichler, Manfred D., and Eric H. Davidson. 2008. {``Boveri's Long Experiment: {Sea} Urchin Merogones and the Establishment of the Role of Nuclear Chromosomes in Development.''} \emph{Developmental Biology} 314 (1): 1--11. \url{https://doi.org/10.1016/j.ydbio.2007.11.024}.

\leavevmode\vadjust pre{\hypertarget{ref-laverAssessingPerformanceOxford2015}{}}%
Laver, T., J. Harrison, P. A. O'NANeill, K. Moore, A. Farbos, K. Paszkiewicz, and D. J. Studholm. 2015. {``Assessing the Performance of the {Oxford Nanopore Technologies MinION}.''} \emph{Biomolecular Detection and Quantification} 3 (March): 1--8. \url{https://doi.org/10.1016/j.bdq.2015.02.001}.

\leavevmode\vadjust pre{\hypertarget{ref-lecorreGeneticDifferentiationQuantitative2012}{}}%
Le Corre, Val'erie, and Antoine Kremer. 2012. {``The Genetic Differentiation at Quantitative Trait Loci Under Local Adaptation.''} \emph{Molecular Ecology} 21 (7): 1548--66. \url{https://doi.org/10.1111/j.1365-294X.2012.05479.x}.

\leavevmode\vadjust pre{\hypertarget{ref-legerGenomicVariationsEpigenomic2022}{}}%
Leger, Adrien, Ian Brettell, Jack Monahan, Carl Barton, Nadeshda Wolf, Natalja Kusminski, Cathrin Herder, et al. 2022. {``Genomic Variations and Epigenomic Landscape of the {Medaka Inbred Kiyosu-Karlsruhe} ({MIKK}) Panel.''} \emph{Genome Biology} 23 (1): 58. \url{https://doi.org/10.1186/s13059-022-02602-4}.

\leavevmode\vadjust pre{\hypertarget{ref-lekAnalysisProteincodingGenetic2016}{}}%
Lek, Monkol, Konrad J. Karczewski, Eric V. Minikel, Kaitlin E. Samocha, Eric Banks, Timothy Fennell, Anne H. O'NADonnell-Luria, et al. 2016. {``Analysis of Protein-Coding Genetic Variation in 60,706 Humans.''} \emph{Nature} 536 (7616, 7616): 285--91. \url{https://doi.org/10.1038/nature19057}.

\leavevmode\vadjust pre{\hypertarget{ref-liSequenceAlignmentMap2009}{}}%
Li, Heng, Bob Handsaker, Alec Wysoker, Tim Fennell, Jue Ruan, Nils Homer, Gabor Marth, Goncalo Abecasis, Richard Durbin, and 1000 Genome Project Data Processing Subgroup. 2009. {``The {Sequence} Alignment/Map ({SAM}) Format and {SAMtools}.''} \emph{Bioinformatics} 25 (16): 2078--79.

\leavevmode\vadjust pre{\hypertarget{ref-limaBehavioralDecisionsMade1990}{}}%
Lima, Steven L., and Lawrence M. Dill. 1990. {``Behavioral Decisions Made Under the Risk of Predation: A Review and Prospectus.''} \emph{Canadian Journal of Zoology} 68 (4): 619--40. \url{https://doi.org/10.1139/z90-092}.

\leavevmode\vadjust pre{\hypertarget{ref-limamiGeneticPhysiologicalAnalysis2002}{}}%
Limami, Anis M., Clothilde Rouillon, Gaëlle Glevarec, Andr'e Gallais, and Bertrand Hirel. 2002. {``Genetic and {Physiological Analysis} of {Germination Efficiency} in {Maize} in {Relation} to {Nitrogen Metabolism Reveals} the {Importance} of {Cytosolic Glutamine Synthetase}.''} \emph{Plant Physiology} 130 (4): 1860--70. \url{https://doi.org/10.1104/pp.009647}.

\leavevmode\vadjust pre{\hypertarget{ref-lohEfficientBayesianMixedmodel2015}{}}%
Loh, Po-Ru, George Tucker, Brendan K. Bulik-Sullivan, Bjarni J. Vilhj'almsson, Hilary K. Finucane, Rany M. Salem, Daniel I. Chasman, et al. 2015. {``Efficient {Bayesian} Mixed-Model Analysis Increases Association Power in Large Cohorts.''} \emph{Nature Genetics} 47 (3, 3): 284--90. \url{https://doi.org/10.1038/ng.3190}.

\leavevmode\vadjust pre{\hypertarget{ref-lucon-xiccatoIndividualDifferencesCognition2017}{}}%
Lucon-Xiccato, Tyrone, and Angelo Bisazza. 2017. {``Individual Differences in Cognition Among Teleost Fishes.''} \emph{Behavioural Processes}, The {Cognition} of {Fish}, 141 (August): 184--95. \url{https://doi.org/10.1016/j.beproc.2017.01.015}.

\leavevmode\vadjust pre{\hypertarget{ref-lucon-xiccatoDevelopmentOpenFieldBehaviour2020}{}}%
Lucon-Xiccato, Tyrone, Francesca Conti, Felix Loosli, Nicholas S. Foulkes, and Cristiano Bertolucci. 2020. {``Development of {Open-Field Behaviour} in the {Medaka}, {Oryzias} Latipes.''} \emph{Biology} 9 (11, 11): 389. \url{https://doi.org/10.3390/biology9110389}.

\leavevmode\vadjust pre{\hypertarget{ref-lucon-xiccatoComparisonAnxietylikeSocial2022}{}}%
Lucon-Xiccato, Tyrone, Felix Loosli, Francesca Conti, Nicholas S. Foulkes, and Cristiano Bertolucci. 2022. {``Comparison of Anxiety-Like and Social Behaviour in Medaka and Zebrafish.''} \emph{Scientific Reports} 12 (1, 1): 10926. \url{https://doi.org/10.1038/s41598-022-14978-1}.

\leavevmode\vadjust pre{\hypertarget{ref-lushAnimalBreedingPlans1937}{}}%
Lush, Jay L. 1937. \emph{Animal Breeding Plans. {Ames}}. {Collegiate Press, Inc}.

\leavevmode\vadjust pre{\hypertarget{ref-maasBreastCancerRisk2016}{}}%
Maas, Paige, Myrto Barrdahl, Amit D. Joshi, Paul L. Auer, Mia M. Gaudet, Roger L. Milne, Fredrick R. Schumacher, et al. 2016. {``Breast {Cancer Risk From Modifiable} and {Nonmodifiable Risk Factors Among White Women} in the {United States}.''} \emph{JAMA Oncology} 2 (10): 1295--1302. \url{https://doi.org/10.1001/jamaoncol.2016.1025}.

\leavevmode\vadjust pre{\hypertarget{ref-macarthurNewNHGRIEBICatalog2017}{}}%
MacArthur, Jacqueline, Emily Bowler, Maria Cerezo, Laurent Gil, Peggy Hall, Emma Hastings, Heather Junkins, et al. 2017. {``The New {NHGRI-EBI Catalog} of Published Genome-Wide Association Studies ({GWAS Catalog}).''} \emph{Nucleic Acids Research} 45 (D1): D896--901. \url{https://doi.org/10.1093/nar/gkw1133}.

\leavevmode\vadjust pre{\hypertarget{ref-mackayChartingGenotypePhenotype2018}{}}%
Mackay, Trudy F. C., and Wen Huang. 2018. {``Charting the Genotype--Phenotype Map: Lessons from the {Drosophila} Melanogaster {Genetic Reference Panel}.''} \emph{WIREs Developmental Biology} 7 (1): e289. \url{https://doi.org/10.1002/wdev.289}.

\leavevmode\vadjust pre{\hypertarget{ref-maddoxDoubleHelixWronged2003}{}}%
Maddox, Brenda. 2003. {``The Double Helix and the 'Wronged Heroine'.''} \emph{Nature} 421 (6921, 6921): 407--8. \url{https://doi.org/10.1038/nature01399}.

\leavevmode\vadjust pre{\hypertarget{ref-magnoGwasrapiddPackageQuery2020}{}}%
Magno, Ramiro, and Ana-Teresa Maia. 2020. {``Gwasrapidd: An {R} Package to Query, Download and Wrangle {GWAS} Catalog Data.''} \emph{Bioinformatics} 36 (2): 649--50. \url{https://doi.org/10.1093/bioinformatics/btz605}.

\leavevmode\vadjust pre{\hypertarget{ref-malecotMathematiquesHeredite1948}{}}%
Mal'ecot, Gustave. 1948. {``Mathématiques de l'hérédité.''}

\leavevmode\vadjust pre{\hypertarget{ref-maloneItDoesnTake2002}{}}%
Malone, John. 2002. \emph{It {Doesn}'t {Take} a {Rocket Scientist}: {Great Amateurs} of {Science}}. {Wiley}. \url{https://books.google.com?id=OcG9NAEACAAJ}.

\leavevmode\vadjust pre{\hypertarget{ref-malthusEssayPrinciplePopulation1872}{}}%
Malthus, Thomas Robert. 1872. \emph{An {Essay} on the {Principle} of {Population}..}

\leavevmode\vadjust pre{\hypertarget{ref-marchantAlfredRusselWallace1916}{}}%
Marchant, Sir James, and Alfred Russel Wallace. 1916. \emph{Alfred {Russel Wallace}: {Letters} and {Reminiscences} ({Complete})}. {Library of Alexandria}. \url{https://books.google.com?id=0IF1K9IYQYMC}.

\leavevmode\vadjust pre{\hypertarget{ref-marksConstructionMendelLaws2008}{}}%
Marks, Jonathan. 2008. {``The Construction of {Mendel}'s Laws.''} \emph{Evolutionary Anthropology: Issues, News, and Reviews} 17 (6): 250--53. \url{https://doi.org/10.1002/evan.20192}.

\leavevmode\vadjust pre{\hypertarget{ref-martinHumanDemographicHistory2017}{}}%
Martin, Alicia R, Christopher R Gignoux, Raymond K Walters, Genevieve L Wojcik, Benjamin M Neale, Simon Gravel, Mark J Daly, Carlos D Bustamante, and Eimear E Kenny. 2017. {``Human Demographic History Impacts Genetic Risk Prediction Across Diverse Populations.''} \emph{The American Journal of Human Genetics} 100 (4): 635--49. \url{https://doi.org/10.1016/j.ajhg.2017.03.004}.

\leavevmode\vadjust pre{\hypertarget{ref-martinClinicalUseCurrent2019}{}}%
Martin, Alicia R., Masahiro Kanai, Yoichiro Kamatani, Yukinori Okada, Benjamin M. Neale, and Mark J. Daly. 2019. {``Clinical Use of Current Polygenic Risk Scores May Exacerbate Health Disparities.''} \emph{Nature Genetics} 51 (4, 4): 584--91. \url{https://doi.org/10.1038/s41588-019-0379-x}.

\leavevmode\vadjust pre{\hypertarget{ref-martinCriticalNeedsChallenges2018}{}}%
Martin, Alicia R, Solomon Teferra, Marlo Möller, Eileen G Hoal, and Mark J Daly. 2018. {``The Critical Needs and Challenges for Genetic Architecture Studies in {Africa}.''} \emph{Current Opinion in Genetics \& Development}, Genetics of {Human Origins}, 53 (December): 113--20. \url{https://doi.org/10.1016/j.gde.2018.08.005}.

\leavevmode\vadjust pre{\hypertarget{ref-martinSimonhmartinGenomicsGeneral2022}{}}%
martin, Simon. (2016) 2022. \emph{Simonhmartin/Genomics\_general}. \url{https://github.com/simonhmartin/genomics_general}.

\leavevmode\vadjust pre{\hypertarget{ref-martinEvaluatingUseABBA2015}{}}%
Martin, Simon H., John W. Davey, and Chris D. Jiggins. 2015. {``Evaluating the {Use} of {ABBA}--{BABA Statistics} to {Locate Introgressed Loci}.''} \emph{Molecular Biology and Evolution} 32 (1): 244--57. \url{https://doi.org/10.1093/molbev/msu269}.

\leavevmode\vadjust pre{\hypertarget{ref-marxWatsonCrickDNA2004}{}}%
Marx, Christy. 2004. \emph{Watson and {Crick} and {DNA}}. {Rosen Central Primary Source}. \url{https://books.google.com?id=hLcDn6j7I8gC}.

\leavevmode\vadjust pre{\hypertarget{ref-matsudaSpeciesspecificSegmentationClock2020}{}}%
Matsuda, Mitsuhiro, Hanako Hayashi, Jordi Garcia-Ojalvo, Kumiko Yoshioka-Kobayashi, Ryoichiro Kageyama, Yoshihiro Yamanaka, Makoto Ikeya, Junya Toguchida, Cantas Alev, and Miki Ebisuya. 2020. {``Species-Specific Segmentation Clock Periods Are Due to Differential Biochemical Reaction Speeds.''} \emph{Science} 369 (6510): 1450--55.

\leavevmode\vadjust pre{\hypertarget{ref-matsunagaHabituationMedakaOryzias2010}{}}%
Matsunaga, Wataru, and Eiji Watanabe. 2010. {``Habituation of Medaka ({Oryzias} Latipes) Demonstrated by Open-Field Testing.''} \emph{Behavioural Processes} 85 (2): 142--50. \url{https://doi.org/10.1016/j.beproc.2010.06.019}.

\leavevmode\vadjust pre{\hypertarget{ref-mazumdarEugenicsHumanGenetics2005}{}}%
Mazumdar, Pauline. 2005. \emph{Eugenics, Human Genetics and Human Failings: The {Eugenics Society}, Its Sources and Its Critics in {Britain}}. {Routledge}.

\leavevmode\vadjust pre{\hypertarget{ref-mccarthyReferencePanel642016}{}}%
McCarthy, Shane, Sayantan Das, Warren Kretzschmar, Olivier Delaneau, Andrew R Wood, Alexander Teumer, Hyun Min Kang, et al. 2016. {``A Reference Panel of 64,976 Haplotypes for Genotype Imputation.''} \emph{Nature Genetics} 48 (10, 10): 1279--83. \url{https://doi.org/10.1038/ng.3643}.

\leavevmode\vadjust pre{\hypertarget{ref-mckennaGenomeAnalysisToolkit2010}{}}%
McKenna, Aaron, Matthew Hanna, Eric Banks, Andrey Sivachenko, Kristian Cibulskis, Andrew Kernytsky, Kiran Garimella, et al. 2010. {``The {Genome Analysis Toolkit}: {A MapReduce} Framework for Analyzing Next-Generation {DNA} Sequencing Data.''} \emph{Genome Research} 20 (9): 1297--1303. \url{https://doi.org/10.1101/gr.107524.110}.

\leavevmode\vadjust pre{\hypertarget{ref-mclarenOurOwnMaster2015}{}}%
McLaren, Angus. 2015. \emph{Our {Own Master Race}: {Eugenics} in {Canada}, 1885-1945}. {University of Toronto Press}. \url{https://books.google.com?id=4rP3rQEACAAJ}.

\leavevmode\vadjust pre{\hypertarget{ref-mclarenEnsemblVariantEffect2016}{}}%
McLaren, William, Laurent Gil, Sarah E. Hunt, Harpreet Singh Riat, Graham R. S. Ritchie, Anja Thormann, Paul Flicek, and Fiona Cunningham. 2016. {``The {Ensembl Variant Effect Predictor}.''} \emph{Genome Biology} 17 (1): 122. \url{https://doi.org/10.1186/s13059-016-0974-4}.

\leavevmode\vadjust pre{\hypertarget{ref-mehringerSmehringerSViper2017}{}}%
Mehringer, Svenja. 2017. \emph{Smehringer/{SViper}}. \url{https://github.com/smehringer/SViper}.

\leavevmode\vadjust pre{\hypertarget{ref-meijsserAnalysisOpenfieldPerformance1989}{}}%
Meijsser, F. M., A. M. P. Kersten, P. R. Wiepkema, and J. H. M. Metz. 1989. {``An Analysis of the Open-Field Performance of Sub-Adult Rabbits.''} \emph{Applied Animal Behaviour Science} 24 (2): 147--55. \url{https://doi.org/10.1016/0168-1591(89)90042-7}.

\leavevmode\vadjust pre{\hypertarget{ref-membersofthecomplextraitconsortiumNatureIdentificationQuantitative2003}{}}%
Members of the Complex Trait Consortium. 2003. {``The Nature and Identification of Quantitative Trait Loci: A Community's View.''} \emph{Nature Reviews Genetics} 4 (11, 11): 911--16. \url{https://doi.org/10.1038/nrg1206}.

\leavevmode\vadjust pre{\hypertarget{ref-mendelVersucheUberPflanzenhybriden1866}{}}%
Mendel, Gregor. 1866. {``Versuche Uber Pflanzen-Hybriden.''} \emph{Verhandlungen Des Naturforschenden Vereins in Brunn Fur} 4: 3--47.

\leavevmode\vadjust pre{\hypertarget{ref-mikheyevFirstLookOxford2014}{}}%
Mikheyev, Alexander S., and Mandy M. Y. Tin. 2014. {``A First Look at the {Oxford Nanopore MinION} Sequencer.''} \emph{Molecular Ecology Resources} 14 (6): 1097--1102. \url{https://doi.org/10.1111/1755-0998.12324}.

\leavevmode\vadjust pre{\hypertarget{ref-milmoFuryDNAPioneer2007}{}}%
Milmo, Cahal. 2007. {``Fury at {DNA} Pioneer's Theory: {Africans} Are Less Intelligent Than {Westerners}.''} \emph{The Independent: News}, October 16, 2007. \url{https://www.independent.co.uk/news/science/fury-at-dna-pioneer-s-theory-africans-are-less-intelligent-than-westerners-394898.html}.

\leavevmode\vadjust pre{\hypertarget{ref-morganCritiqueTheoryEvolution1916}{}}%
Morgan, Thomas Hunt. 1916. \emph{A {Critique} of the {Theory} of {Evolution}. {Lectures Delivered} at {Princeton University} ... 1916}. \url{https://books.google.com?id=tGUJwwEACAAJ}.

\leavevmode\vadjust pre{\hypertarget{ref-morganCritiqueTheoryEvolution1919}{}}%
---------. 1919. \emph{A {Critique} of the {Theory} of {Evolution}}. {Princeton University Press}.

\leavevmode\vadjust pre{\hypertarget{ref-morganRelationGeneticsPhysiology1935}{}}%
---------. 1935. {``The {Relation} of {Genetics} to {Physiology} and {Medicine}.''} \emph{The Scientific Monthly} 41 (1): 5--18. \url{https://www.jstor.org/stable/15918}.

\leavevmode\vadjust pre{\hypertarget{ref-morganMechanismMendelianHeredity1972}{}}%
---------. 1972. \emph{The {Mechanism} of {Mendelian Heredity}}. {Johnson Reprint Corporation}.

\leavevmode\vadjust pre{\hypertarget{ref-morseOriginsInbredMice2012}{}}%
Morse, Herbert C. III. 2012. \emph{Origins of {Inbred Mice}}. {Elsevier}. \url{https://books.google.com?id=mRyp1vKI3fsC}.

\leavevmode\vadjust pre{\hypertarget{ref-mostafaviVariablePredictionAccuracy2020}{}}%
Mostafavi, Hakhamanesh, Arbel Harpak, Ipsita Agarwal, Dalton Conley, Jonathan K Pritchard, and Molly Przeworski. 2020. {``Variable Prediction Accuracy of Polygenic Scores Within an Ancestry Group.''} Edited by Ruth Loos, Michael B Eisen, and Paul O'Reilly. \emph{eLife} 9 (January): e48376. \url{https://doi.org/10.7554/eLife.48376}.

\leavevmode\vadjust pre{\hypertarget{ref-mukherjeeGeneIntimateHistory2016}{}}%
Mukherjee, Siddhartha. 2016. \emph{The {Gene}: {An Intimate History}}. {Simon and Schuster}. \url{https://books.google.com?id=XvAsDAAAQBAJ}.

\leavevmode\vadjust pre{\hypertarget{ref-murataMedakaBiologyManagement2019}{}}%
Murata, Kenji, Masato Kinoshita, Kiyoshi Naruse, Minoru Tanaka, and Yasuhiro Kamei. 2019. \emph{Medaka: {Biology}, {Management}, and {Experimental Protocols}}. {John Wiley \& Sons, Incorporated}. \url{https://books.google.com?id=V5BAzQEACAAJ}.

\leavevmode\vadjust pre{\hypertarget{ref-naruseDetailedLinkageMap2000}{}}%
Naruse, Kiyoshi, Shoji Fukamachi, Hiroshi Mitani, Mariko Kondo, Tomoko Matsuoka, Shu Kondo, Nana Hanamura, et al. 2000. {``A {Detailed Linkage Map} of {Medaka}, {Oryzias} Latipes: {Comparative Genomics} and {Genome Evolution}.''} \emph{Genetics} 154 (4): 1773--84. \url{https://www.genetics.org/content/154/4/1773}.

\leavevmode\vadjust pre{\hypertarget{ref-naruseMedakaModelOrganogenesis2011a}{}}%
Naruse, Kiyoshi, Minoru Tanaka, and Hiroyuki Takeda. 2011. \emph{Medaka: A Model for Organogenesis, Human Disease, and Evolution}. {Springer Science \& Business Media}.

\leavevmode\vadjust pre{\hypertarget{ref-natarajanPolygenicRiskScore2017}{}}%
Natarajan, Pradeep, Robin Young, Nathan O. Stitziel, Sandosh Padmanabhan, Usman Baber, Roxana Mehran, Samantha Sartori, et al. 2017. {``Polygenic {Risk Score Identifies Subgroup With Higher Burden} of {Atherosclerosis} and {Greater Relative Benefit From Statin Therapy} in the {Primary Prevention Setting}.''} \emph{Circulation} 135 (22): 2091--101. \url{https://doi.org/10.1161/CIRCULATIONAHA.116.024436}.

\leavevmode\vadjust pre{\hypertarget{ref-neiMathematicalModelStudying1979}{}}%
Nei, M, and W H Li. 1979. {``Mathematical Model for Studying Genetic Variation in Terms of Restriction Endonucleases.''} \emph{Proceedings of the National Academy of Sciences} 76 (10): 5269--73. \url{https://doi.org/10.1073/pnas.76.10.5269}.

\leavevmode\vadjust pre{\hypertarget{ref-nordbergTestingMeasuresBoldness2021}{}}%
Nordberg, Eric, Rheanne Denny, and Lin Schwarzkopf. 2021. {``Testing Measures of Boldness and Exploratory Activity in Native Versus Invasive Species: Geckos as a Model System.''} \emph{Animal Behaviour} 177 (July): 215--22. \url{https://doi.org/10.1016/j.anbehav.2021.05.013}.

\leavevmode\vadjust pre{\hypertarget{ref-nurkCompleteSequenceHuman2022}{}}%
Nurk, Sergey, Sergey Koren, Arang Rhie, Mikko Rautiainen, Andrey V. Bzikadze, Alla Mikheenko, Mitchell R. Vollger, et al. 2022. {``The Complete Sequence of a Human Genome.''} \emph{Science} 376 (6588): 44--53. \url{https://doi.org/10.1126/science.abj6987}.

\leavevmode\vadjust pre{\hypertarget{ref-oconnorConstructionLargeDNA1989}{}}%
O'Connor, Michael, Mark Peifer, and Welcome Bender. 1989. {``Construction of {Large DNA Segments} in {Escherichia} Coli.''} \emph{Science} 244 (4910): 1307--12. \url{https://doi.org/10.1126/science.2660262}.

\leavevmode\vadjust pre{\hypertarget{ref-ObituaryFredSanger2013}{}}%
{``Obituary: {Fred Sanger} the Scientist and Twice {Nobel Prize} Winner Dies.''} 2013. {Express.co.uk}. November 23, 2013. \url{https://www.express.co.uk/news/obituaries/444612/Obituary-Fred-Sanger-the-scientist-and-twice-Nobel-Prize-winner-dies-aged-95}.

\leavevmode\vadjust pre{\hypertarget{ref-okbayPolygenicPredictionEducational2022}{}}%
Okbay, Aysu, Yeda Wu, Nancy Wang, Hariharan Jayashankar, Michael Bennett, Seyed Moeen Nehzati, Julia Sidorenko, et al. 2022. {``Polygenic Prediction of Educational Attainment Within and Between Families from Genome-Wide Association Analyses in 3 Million Individuals.''} \emph{Nature Genetics}, March, 1--13. \url{https://doi.org/10.1038/s41588-022-01016-z}.

\leavevmode\vadjust pre{\hypertarget{ref-ozakiFunctionalSNPsLymphotoxina2002}{}}%
Ozaki, Kouichi, Yozo Ohnishi, Aritoshi Iida, Akihiko Sekine, Ryo Yamada, Tatsuhiko Tsunoda, Hiroshi Sato, et al. 2002. {``Functional {SNPs} in the Lymphotoxin-α Gene That Are Associated with Susceptibility to Myocardial Infarction.''} \emph{Nature Genetics} 32 (4, 4): 650--54. \url{https://doi.org/10.1038/ng1047}.

\leavevmode\vadjust pre{\hypertarget{ref-panagiotouWhatShouldGenomewide2012}{}}%
Panagiotou, Orestis A., John P. A. Ioannidis, and Genome-Wide Significance Project. 2012. {``What Should the Genome-Wide Significance Threshold Be? {Empirical} Replication of Borderline Genetic Associations.''} \emph{International Journal of Epidemiology} 41 (1): 273--86. \url{https://doi.org/10.1093/ije/dyr178}.

\leavevmode\vadjust pre{\hypertarget{ref-paquettePolygenicRiskScore2017}{}}%
Paquette, Martine, Michael Chong, S'ebastien Th'eriault, Robert Dufour, Guillaume Par'e, and Alexis Baass. 2017. {``Polygenic Risk Score Predicts Prevalence of Cardiovascular Disease in Patients with Familial Hypercholesterolemia.''} \emph{Journal of Clinical Lipidology} 11 (3): 725--732.e5. \url{https://doi.org/10.1016/j.jacl.2017.03.019}.

\leavevmode\vadjust pre{\hypertarget{ref-paradisPegasPackagePopulation2010}{}}%
Paradis, Emmanuel. 2010. {``Pegas: An {R} Package for Population Genetics with an Integrated--Modular Approach.''} \emph{Bioinformatics} 26 (3): 419--20. \url{https://doi.org/10.1093/bioinformatics/btp696}.

\leavevmode\vadjust pre{\hypertarget{ref-paradisApeEnvironmentModern2019}{}}%
Paradis, Emmanuel, and Klaus Schliep. 2019. {``Ape 5.0: An Environment for Modern Phylogenetics and Evolutionary Analyses in {R}.''} \emph{Bioinformatics} 35 (3): 526--28. \url{https://doi.org/10.1093/bioinformatics/bty633}.

\leavevmode\vadjust pre{\hypertarget{ref-parkDistributionAlleleFrequencies2011}{}}%
Park, Ju-Hyun, Mitchell H. Gail, Clarice R. Weinberg, Raymond J. Carroll, Charles C. Chung, Zhaoming Wang, Stephen J. Chanock, Joseph F. Fraumeni, and Nilanjan Chatterjee. 2011. {``Distribution of Allele Frequencies and Effect Sizes and Their Interrelationships for Common Genetic Susceptibility Variants.''} \emph{Proceedings of the National Academy of Sciences} 108 (44): 18026--31. \url{https://doi.org/10.1073/pnas.1114759108}.

\leavevmode\vadjust pre{\hypertarget{ref-pattersonRecoveryInterblockInformation1971}{}}%
Patterson, H. D., and R. Thompson. 1971. {``Recovery of Inter-Block Information When Block Sizes Are Unequal.''} \emph{Biometrika} 58 (3): 545--54. \url{https://doi.org/10.1093/biomet/58.3.545}.

\leavevmode\vadjust pre{\hypertarget{ref-payneBulkVisGraphicalViewer2019}{}}%
Payne, Alexander, Nadine Holmes, Vardhman Rakyan, and Matthew Loose. 2019. {``{BulkVis}: A Graphical Viewer for {Oxford} Nanopore Bulk {Fast5} Files.''} \emph{Bioinformatics} 35 (13): 2193--98. \url{https://doi.org/10.1093/bioinformatics/bty841}.

\leavevmode\vadjust pre{\hypertarget{ref-pearsonDissectionAsymmetricalFrequency1894}{}}%
Pearson, Karl. 1894. {``On the Dissection of Asymmetrical Frequency Curves.''} \emph{Phil. Trans. Roy. Soc} 185: 71--110.

\leavevmode\vadjust pre{\hypertarget{ref-pearsonMathematicalContributionsTheory1898}{}}%
---------. 1898. {``Mathematical Contributions to the Theory of Evolution, {On} the Law of Ancestral Heredity.''} \emph{Proceedings of the Royal Society of London} 62 (379-387): 386--412. \url{https://doi.org/10.1098/rspl.1897.0128}.

\leavevmode\vadjust pre{\hypertarget{ref-pearsonCriterionThatGiven1900}{}}%
---------. 1900. {``X. {On} the Criterion That a Given System of Deviations from the Probable in the Case of a Correlated System of Variables Is Such That It Can Be Reasonably Supposed to Have Arisen from Random Sampling.''} \emph{The London, Edinburgh, and Dublin Philosophical Magazine and Journal of Science} 50 (302): 157--75. \url{https://doi.org/10.1080/14786440009463897}.

\leavevmode\vadjust pre{\hypertarget{ref-pearsonLIIILinesPlanes1901}{}}%
---------. 1901. {``{LIII}. {On} Lines and Planes of Closest Fit to Systems of Points in Space.''} \emph{The London, Edinburgh, and Dublin Philosophical Magazine and Journal of Science} 2 (11): 559--72. \url{https://doi.org/10.1080/14786440109462720}.

\leavevmode\vadjust pre{\hypertarget{ref-pearsonLifeLettersLabours2011}{}}%
---------. 2011. \emph{The {Life}, {Letters} and {Labours} of {Francis Galton}}. {Cambridge University Press}. \url{https://books.google.com?id=sz7vSOwOGzgC}.

\leavevmode\vadjust pre{\hypertarget{ref-peirceNewSetBXD2004}{}}%
Peirce, Jeremy L., Lu Lu, Jing Gu, Lee M. Silver, and Robert W. Williams. 2004. {``A New Set of {BXD} Recombinant Inbred Lines from Advanced Intercross Populations in Mice.''} \emph{BMC Genetics} 5 (1): 7. \url{https://doi.org/10.1186/1471-2156-5-7}.

\leavevmode\vadjust pre{\hypertarget{ref-pennisi100GenomeNew2022}{}}%
Pennisi, Elizabeth. 2022. {``A \$100 Genome? {New DNA} Sequencers Could Be a {`Game Changer'} for Biology, Medicine \textbar{} {Science} \textbar{} {AAAS}.''} \emph{Science}, June 15, 2022. \url{https://www.science.org/content/article/100-genome-new-dna-sequencers-could-be-game-changer-biology-medicine}.

\leavevmode\vadjust pre{\hypertarget{ref-phillipsGProtocadherinsAreTargeted2003}{}}%
Phillips, Greg R., Hidekazu Tanaka, Marcus Frank, Alice Elste, Lazar Fidler, Deanna L. Benson, and David R. Colman. 2003. {``Γ-{Protocadherins Are Targeted} to {Subsets} of {Synapses} and {Intracellular Organelles} in {Neurons}.''} \emph{Journal of Neuroscience} 23 (12): 5096--5104. \url{https://doi.org/10.1523/JNEUROSCI.23-12-05096.2003}.

\leavevmode\vadjust pre{\hypertarget{ref-Picard2019toolkit}{}}%
{``Picard Toolkit.''} 2019. \emph{Broad Institute, GitHub Repository}. \url{https://broadinstitute.github.io/picard/}; Broad Institute.

\leavevmode\vadjust pre{\hypertarget{ref-platoPlatoRepublic2013}{}}%
Plato. 2013. \emph{Plato's {Republic}}. Translated by Alain Badiou. {John Wiley \& Sons}.

\leavevmode\vadjust pre{\hypertarget{ref-poldermanMetaanalysisHeritabilityHuman2015}{}}%
Polderman, Tinca J. C., Beben Benyamin, Christiaan A. de Leeuw, Patrick F. Sullivan, Arjen van Bochoven, Peter M. Visscher, and Danielle Posthuma. 2015. {``Meta-Analysis of the Heritability of Human Traits Based on Fifty Years of Twin Studies.''} \emph{Nature Genetics} 47 (7, 7): 702--9. \url{https://doi.org/10.1038/ng.3285}.

\leavevmode\vadjust pre{\hypertarget{ref-popejoyGenomicsFailingDiversity2016}{}}%
Popejoy, Alice B., and Stephanie M. Fullerton. 2016. {``Genomics Is Failing on Diversity.''} \emph{Nature} 538 (7624, 7624): 161--64. \url{https://doi.org/10.1038/538161a}.

\leavevmode\vadjust pre{\hypertarget{ref-poplinScalingAccurateGenetic2018}{}}%
Poplin, Ryan, Valentin Ruano-Rubio, Mark A. DePristo, Tim J. Fennell, Mauricio O. Carneiro, Geraldine A. Van der Auwera, David E. Kling, et al. 2018. {``Scaling Accurate Genetic Variant Discovery to Tens of Thousands of Samples.''} {bioRxiv}. \url{https://doi.org/10.1101/201178}.

\leavevmode\vadjust pre{\hypertarget{ref-porcuGenotypeImputationGenomeWide2013}{}}%
Porcu, Eleonora, Serena Sanna, Christian Fuchsberger, and Lars G. Fritsche. 2013. {``Genotype {Imputation} in {Genome-Wide Association Studies}.''} \emph{Current Protocols in Human Genetics} 78 (1): 1.25.1--14. \url{https://doi.org/10.1002/0471142905.hg0125s78}.

\leavevmode\vadjust pre{\hypertarget{ref-posthumaTheoryPracticeQuantitative2003}{}}%
Posthuma, Daniëlle, A. Leo Beem, Eco J. C. de Geus, G. Caroline M. van Baal, Jacob B. von Hjelmborg, Ivan Iachine, and Dorret I. Boomsma. 2003. {``Theory and {Practice} in {Quantitative Genetics}.''} \emph{Twin Research and Human Genetics} 6 (5): 361--76. \url{https://doi.org/10.1375/twin.6.5.361}.

\leavevmode\vadjust pre{\hypertarget{ref-pricePrincipalComponentsAnalysis2006}{}}%
Price, Alkes L., Nick J. Patterson, Robert M. Plenge, Michael E. Weinblatt, Nancy A. Shadick, and David Reich. 2006. {``Principal Components Analysis Corrects for Stratification in Genome-Wide Association Studies.''} \emph{Nature Genetics} 38 (8, 8): 904--9. \url{https://doi.org/10.1038/ng1847}.

\leavevmode\vadjust pre{\hypertarget{ref-pritchardAllelicArchitectureHuman2002}{}}%
Pritchard, Jonathan K., and Nancy J. Cox. 2002. {``The Allelic Architecture of Human Disease Genes: Common Disease--Common Variant\ldots{}{\(\mkern1mu\)}or Not?''} \emph{Human Molecular Genetics} 11 (20): 2417--23. \url{https://doi.org/10.1093/hmg/11.20.2417}.

\leavevmode\vadjust pre{\hypertarget{ref-pritchardLinkageDisequilibriumHumans2001}{}}%
Pritchard, Jonathan K., and Molly Przeworski. 2001. {``Linkage {Disequilibrium} in {Humans}: {Models} and {Data}.''} \emph{The American Journal of Human Genetics} 69 (1): 1--14. \url{https://doi.org/10.1086/321275}.

\leavevmode\vadjust pre{\hypertarget{ref-pritchardInferencePopulationStructure2000}{}}%
Pritchard, Jonathan K, Matthew Stephens, and Peter Donnelly. 2000. {``Inference of {Population Structure Using Multilocus Genotype Data}.''} \emph{Genetics} 155 (2): 945--59. \url{https://doi.org/10.1093/genetics/155.2.945}.

\leavevmode\vadjust pre{\hypertarget{ref-provineOriginsTheoreticalPopulation2001}{}}%
Provine, William B. 2001. \emph{The Origins of Theoretical Population Genetics: With a New Afterword}. {University of Chicago Press}.

\leavevmode\vadjust pre{\hypertarget{ref-prykeRedDominatesBlack2006}{}}%
Pryke, Sarah R, and Simon C Griffith. 2006. {``Red Dominates Black: Agonistic Signalling Among Head Morphs in the Colour Polymorphic {Gouldian} Finch.''} \emph{Proceedings of the Royal Society B: Biological Sciences} 273 (1589): 949--57. \url{https://doi.org/10.1098/rspb.2005.3362}.

\leavevmode\vadjust pre{\hypertarget{ref-pszczolaReliabilityDirectGenomic2012}{}}%
Pszczola, M., T. Strabel, H. A. Mulder, and M. P. L. Calus. 2012. {``Reliability of Direct Genomic Values for Animals with Different Relationships Within and to the Reference Population.''} \emph{Journal of Dairy Science} 95 (1): 389--400. \url{https://doi.org/10.3168/jds.2011-4338}.

\leavevmode\vadjust pre{\hypertarget{ref-punnettMendelism1905}{}}%
Punnett, Reginald Crundall. 1905. \emph{Mendelism}. {Macmillan \& Bowes}.

\leavevmode\vadjust pre{\hypertarget{ref-purcellPLINK}{}}%
Purcell, Shaun M, and Christopher C Chang. n.d. \emph{{PLINK} 1.9}. \href{https://www.cog-genomics.org/plink/1.9/}{www.cog-genomics.org/plink/1.9/}.

\leavevmode\vadjust pre{\hypertarget{ref-quintana-murciRolePlayedNatural2010}{}}%
Quintana-Murci, Lluis, and Luis B. Barreiro. 2010. {``The Role Played by Natural Selection on {Mendelian} Traits in Humans.''} \emph{Annals of the New York Academy of Sciences} 1214 (1): 1--17. \url{https://doi.org/10.1111/j.1749-6632.2010.05856.x}.

\leavevmode\vadjust pre{\hypertarget{ref-racimoDetectingPolygenicAdaptation2018}{}}%
Racimo, Fernando, Jeremy J Berg, and Joseph K Pickrell. 2018. {``Detecting {Polygenic Adaptation} in {Admixture Graphs}.''} \emph{Genetics} 208 (4): 1565--84. \url{https://doi.org/10.1534/genetics.117.300489}.

\leavevmode\vadjust pre{\hypertarget{ref-reeveEncyclopediaGenetics2014}{}}%
Reeve, Eric C. R. 2014. \emph{Encyclopedia of {Genetics}}. {Routledge}. \url{https://books.google.com?id=BGqYAgAAQBAJ}.

\leavevmode\vadjust pre{\hypertarget{ref-reeveEncyclopediaGenetics2001}{}}%
Reeve, Eric C. R., and Isobel Black. 2001. \emph{Encyclopedia of {Genetics}}. {Taylor \& Francis}. \url{https://books.google.com?id=JjLWYKqehRsC}.

\leavevmode\vadjust pre{\hypertarget{ref-reichAllelicSpectrumHuman2001}{}}%
Reich, David E, and Eric S Lander. 2001. {``On the Allelic Spectrum of Human Disease.''} \emph{Trends in Genetics} 17 (9): 502--10. \url{https://doi.org/10.1016/S0168-9525(01)02410-6}.

\leavevmode\vadjust pre{\hypertarget{ref-rheinbergerMendelianInheritanceGermany2000}{}}%
Rheinberger, Hans-Jörg. 2000. {``Mendelian Inheritance in {Germany} Between 1900 and 1910. {The} Case of {Carl Correns} (1864--1933).''} \emph{Comptes Rendus de l'Académie Des Sciences-Séries III-Sciences de La Vie} 323 (12): 1089--96.

\leavevmode\vadjust pre{\hypertarget{ref-rhoadsPacBioSequencingIts2015}{}}%
Rhoads, Anthony, and Kin Fai Au. 2015. {``{PacBio Sequencing} and {Its Applications}.''} \emph{Genomics, Proteomics \& Bioinformatics}, {SI}: {Metagenomics} of {Marine Environments}, 13 (5): 278--89. \url{https://doi.org/10.1016/j.gpb.2015.08.002}.

\leavevmode\vadjust pre{\hypertarget{ref-ridleyFrancisCrickDiscoverer2006}{}}%
Ridley, Matt. 2006. \emph{Francis {Crick}: Discoverer of the Genetic Code}. {Atlas Books}.

\leavevmode\vadjust pre{\hypertarget{ref-rischFutureGeneticStudies1996}{}}%
Risch, Neil, and Kathleen Merikangas. 1996. {``The {Future} of {Genetic Studies} of {Complex Human Diseases}.''} \emph{Science} 273 (5281): 1516--17. \url{https://doi.org/10.1126/science.273.5281.1516}.

\leavevmode\vadjust pre{\hypertarget{ref-romero-ferreroIdtrackerAiTracking2019}{}}%
Romero-Ferrero, Francisco, Mattia G. Bergomi, Robert C. Hinz, Francisco J. H. Heras, and Gonzalo G. de Polavieja. 2019. {``Idtracker.ai: Tracking All Individuals in Small or Large Collectives of Unmarked Animals.''} \emph{Nature Methods} 16 (2, 2): 179--82. \url{https://doi.org/10.1038/s41592-018-0295-5}.

\leavevmode\vadjust pre{\hypertarget{ref-rutherfordHowArgueRacist2020}{}}%
Rutherford, Adam. 2020. \emph{How to Argue with a Racist: {History}, Science, Race and Reality}. {Hachette UK}.

\leavevmode\vadjust pre{\hypertarget{ref-ruxtonUnequalVarianceTtest2006}{}}%
Ruxton, Graeme D. 2006. {``The Unequal Variance t-Test Is an Underused Alternative to {Student}'s t-Test and the {Mann}--{Whitney U} Test.''} \emph{Behavioral Ecology} 17 (4): 688--90. \url{https://doi.org/10.1093/beheco/ark016}.

\leavevmode\vadjust pre{\hypertarget{ref-ruzzanteBehaviouralGrowthResponses1990}{}}%
Ruzzante, D. E., and R. W. Doyle. 1990. {``Behavioural and Growth Responses to the Intensity of Intraspecific Social Interaction Among Medaka, {Oryzias} Latipes ({Temminck} and {Schlegel}) ({Pisces}, {Cyprinodontidae}).''} \emph{Journal of Fish Biology} 37 (5): 663--73. \url{https://doi.org/10.1111/j.1095-8649.1990.tb02531.x}.

\leavevmode\vadjust pre{\hypertarget{ref-segurelEvolutionLactasePersistence2017a}{}}%
S'egurel, Laure, and C'eline Bon. 2017. {``On the {Evolution} of {Lactase Persistence} in {Humans}.''} \emph{Annual Review of Genomics and Human Genetics} 18 (August): 297--319. \url{https://doi.org/10.1146/annurev-genom-091416-035340}.

\leavevmode\vadjust pre{\hypertarget{ref-saliba-colombaniEfficiencyRFLPRAPD2000}{}}%
Saliba-Colombani, Vera, Mathilde Causse, Laurent Gervais, and Jacqueline Philouze. 2000. {``Efficiency of {RFLP}, {RAPD}, and {AFLP} Markers for the Construction of an Intraspecific Map of the Tomato Genome.''} \emph{Genome} 43 (1): 29--40. \url{https://doi.org/10.1139/g99-096}.

\leavevmode\vadjust pre{\hypertarget{ref-sanchezArnoldTongueEntrainment2022}{}}%
Sanchez, Paul Gerald Layague, Victoria Mochulska, Christian Mauffette Denis, Gregor Mönke, Takehito Tomita, Nobuko Tsuchida-Straeten, Yvonne Petersen, Katharina F. Sonnen, Paul François, and Alexander Aulehla. 2022. {``Arnold Tongue Entrainment Reveals Dynamical Principles of the Embryonic Segmentation Clock.''} {bioRxiv}. \url{https://doi.org/10.1101/2021.10.20.465101}.

\leavevmode\vadjust pre{\hypertarget{ref-sandlerConceptualAmbiguityThat1985}{}}%
Sandler, Iris, and Laurence Sandler. 1985. {``A {Conceptual Ambiguity} That {Contributed} to the {Neglect} of {Mendel}'s {Paper}.''} \emph{History and Philosophy of the Life Sciences} 7 (1): 3--70. \url{https://www.jstor.org/stable/23328361}.

\leavevmode\vadjust pre{\hypertarget{ref-sangerRapidMethodDetermining1975}{}}%
Sanger, F., and A. R. Coulson. 1975. {``A Rapid Method for Determining Sequences in {DNA} by Primed Synthesis with {DNA} Polymerase.''} \emph{Journal of Molecular Biology} 94 (3): 441--48. \url{https://doi.org/10.1016/0022-2836(75)90213-2}.

\leavevmode\vadjust pre{\hypertarget{ref-sangerDNASequencingChainterminating1977}{}}%
Sanger, Frederick, Steven Nicklen, and Alan R. Coulson. 1977. {``{DNA} Sequencing with Chain-Terminating Inhibitors.''} \emph{Proceedings of the National Academy of Sciences} 74 (12): 5463--67.

\leavevmode\vadjust pre{\hypertarget{ref-sangerAminoacidSequencePhenylalanyl1951}{}}%
Sanger, F., and H. Tuppy. 1951a. {``The Amino-Acid Sequence in the Phenylalanyl Chain of Insulin. 1. {The} Identification of Lower Peptides from Partial Hydrolysates.''} \emph{Biochemical Journal} 49 (4): 463--81. \url{https://doi.org/10.1042/bj0490463}.

\leavevmode\vadjust pre{\hypertarget{ref-sangerAminoacidSequencePhenylalanyl1951a}{}}%
---------. 1951b. {``The Amino-Acid Sequence in the Phenylalanyl Chain of Insulin. 2. {The} Investigation of Peptides from Enzymic Hydrolysates.''} \emph{Biochemical Journal} 49 (4): 481--90. \url{https://doi.org/10.1042/bj0490481}.

\leavevmode\vadjust pre{\hypertarget{ref-schjoldenDoesIndividualVariation2005}{}}%
Schjolden, Joachim, Argaudas Stoskhus, and Svante Winberg. 2005. {``Does {Individual Variation} in {Stress Responses} and {Agonistic Behavior Reflect Divergent Stress Coping Strategies} in {Juvenile Rainbow Trout}?''} \emph{Physiological and Biochemical Zoology} 78 (5): 715--23. \url{https://doi.org/10.1086/432153}.

\leavevmode\vadjust pre{\hypertarget{ref-schmalAnalysisComplexCircadian2022}{}}%
Schmal, Christoph, Gregor Mönke, and Adri'an E. Granada. 2022. {``Analysis of {Complex Circadian Time Series Data Using Wavelets}.''} In \emph{Circadian {Regulation}: {Methods} and {Protocols}}, edited by Guiomar Solanas and Patrick -Simon Welz, 35--54. Methods in {Molecular Biology}. {New York, NY}: {Springer US}. \url{https://doi.org/10.1007/978-1-0716-2249-0_3}.

\leavevmode\vadjust pre{\hypertarget{ref-schroterSegmentNumberAxial2010}{}}%
Schröter, Christian, and Andrew C. Oates. 2010. {``Segment {Number} and {Axial Identity} in a {Segmentation Clock Period Mutant}.''} \emph{Current Biology} 20 (14): 1254--58. \url{https://doi.org/10.1016/j.cub.2010.05.071}.

\leavevmode\vadjust pre{\hypertarget{ref-schumacherAssociationAnalysesMore2018}{}}%
Schumacher, Fredrick R., Ali Amin Al Olama, Sonja I. Berndt, Sara Benlloch, Mahbubl Ahmed, Edward J. Saunders, Tokhir Dadaev, et al. 2018. {``Association Analyses of More Than 140,000 Men Identify 63 New Prostate Cancer Susceptibility Loci.''} \emph{Nature Genetics} 50 (7, 7): 928--36. \url{https://doi.org/10.1038/s41588-018-0142-8}.

\leavevmode\vadjust pre{\hypertarget{ref-schwartzPursuitGene2008}{}}%
Schwartz, James. 2008. \emph{In {Pursuit} of the {Gene}}. {Harvard University Press}. \url{https://books.google.com?id=bl2KvwEACAAJ}.

\leavevmode\vadjust pre{\hypertarget{ref-schwarzeAreWholeexomeWholegenome2018}{}}%
Schwarze, Katharina, James Buchanan, Jenny C. Taylor, and Sarah Wordsworth. 2018. {``Are Whole-Exome and Whole-Genome Sequencing Approaches Cost-Effective? {A} Systematic Review of the Literature.''} \emph{Genetics in Medicine} 20 (10): 1122--30. \url{https://doi.org/10.1038/gim.2017.247}.

\leavevmode\vadjust pre{\hypertarget{ref-scutariUsingGeneticDistance2016}{}}%
Scutari, Marco, Ian Mackay, and David Balding. 2016. {``Using {Genetic Distance} to {Infer} the {Accuracy} of {Genomic Prediction}.''} \emph{PLOS Genetics} 12 (9): e1006288. \url{https://doi.org/10.1371/journal.pgen.1006288}.

\leavevmode\vadjust pre{\hypertarget{ref-seleitEndogenousProteinTagging2021}{}}%
Seleit, Ali, Alexander Aulehla, and Alexandre Paix. 2021. {``Endogenous Protein Tagging in Medaka Using a Simplified {CRISPR}/{Cas9} Knock-in Approach.''} \emph{eLife} 10 (December): e75050. \url{https://doi.org/10.7554/elife.75050}.

\leavevmode\vadjust pre{\hypertarget{ref-sellaThinkingEvolutionComplex2019}{}}%
Sella, G., and N. Barton. 2019. {``Thinking {About} the {Evolution} of {Complex Traits} in the {Era} of {Genome-Wide Association Studies}.''} \emph{Annual Review of Genomics and Human Genetics}. \url{https://doi.org/10.1146/annurev-genom-083115-022316}.

\leavevmode\vadjust pre{\hypertarget{ref-sgroGeneticCorrelationsTradeoffs2004}{}}%
Sgr`o, C. M., and A. A. Hoffmann. 2004. {``Genetic Correlations, Tradeoffs and Environmental Variation.''} \emph{Heredity} 93 (3, 3): 241--48. \url{https://doi.org/10.1038/sj.hdy.6800532}.

\leavevmode\vadjust pre{\hypertarget{ref-shamPowerLinkageAssociation2000}{}}%
Sham, P. C., S. S. Cherny, S. Purcell, and J. K. Hewitt. 2000. {``Power of {Linkage} Versus {Association Analysis} of {Quantitative Traits}, by {Use} of {Variance-Components Models}, for {Sibship Data}.''} \emph{The American Journal of Human Genetics} 66 (5): 1616--30. \url{https://doi.org/10.1086/302891}.

\leavevmode\vadjust pre{\hypertarget{ref-sharpDevelopmentStandardizationImproved2019}{}}%
Sharp, Seth A., Stephen S. Rich, Andrew R. Wood, Samuel E. Jones, Robin N. Beaumont, James W. Harrison, Darius A. Schneider, et al. 2019. {``Development and {Standardization} of an {Improved Type} 1 {Diabetes Genetic Risk Score} for {Use} in {Newborn Screening} and {Incident Diagnosis}.''} \emph{Diabetes Care} 42 (2): 200--207. \url{https://doi.org/10.2337/dc18-1785}.

\leavevmode\vadjust pre{\hypertarget{ref-shendureDNASequencing402017}{}}%
Shendure, Jay, Shankar Balasubramanian, George M. Church, Walter Gilbert, Jane Rogers, Jeffery A. Schloss, and Robert H. Waterston. 2017. {``{DNA} Sequencing at 40: Past, Present and Future.''} \emph{Nature} 550 (7676, 7676): 345--53. \url{https://doi.org/10.1038/nature24286}.

\leavevmode\vadjust pre{\hypertarget{ref-shermanPangenomicsHumanGenome2020}{}}%
Sherman, Rachel M., and Steven L. Salzberg. 2020. {``Pan-Genomics in the Human Genome Era.''} \emph{Nature Reviews Genetics} 21 (4, 4): 243--54. \url{https://doi.org/10.1038/s41576-020-0210-7}.

\leavevmode\vadjust pre{\hypertarget{ref-simontonOriginsGeniusDarwinian1999}{}}%
Simonton, Dean Keith. 1999. \emph{Origins of {Genius}: {Darwinian Perspectives} on {Creativity}}. {Oxford University Press}. \url{https://books.google.com?id=ChNnDAAAQBAJ}.

\leavevmode\vadjust pre{\hypertarget{ref-sirugoMissingDiversityHuman2019}{}}%
Sirugo, Giorgio, Scott M. Williams, and Sarah A. Tishkoff. 2019. {``The {Missing Diversity} in {Human Genetic Studies}.''} \emph{Cell} 177 (1): 26--31. \url{https://doi.org/10.1016/j.cell.2019.02.048}.

\leavevmode\vadjust pre{\hypertarget{ref-sloanwilsonShynessBoldnessHumans1994}{}}%
Sloan Wilson, David, Anne B. Clark, Kristine Coleman, and Ted Dearstyne. 1994. {``Shyness and Boldness in Humans and Other Animals.''} \emph{Trends in Ecology \& Evolution} 9 (11): 442--46. \url{https://doi.org/10.1016/0169-5347(94)90134-1}.

\leavevmode\vadjust pre{\hypertarget{ref-smigielskiDbSNPDatabaseSingle2000}{}}%
Smigielski, Elizabeth M., Karl Sirotkin, Minghong Ward, and Stephen T. Sherry. 2000. {``{dbSNP}: A Database of Single Nucleotide Polymorphisms.''} \emph{Nucleic Acids Research} 28 (1): 352--55. \url{https://doi.org/10.1093/nar/28.1.352}.

\leavevmode\vadjust pre{\hypertarget{ref-smithNaturalSelectionIntellectual2010}{}}%
Smith, Charles Hyde, and George Beccaloni. 2010. \emph{Natural {Selection} and {Beyond}: {The Intellectual Legacy} of {Alfred Russel Wallace}}. {Oxford University Press}. \url{https://books.google.com?id=xkYVDAAAQBAJ}.

\leavevmode\vadjust pre{\hypertarget{ref-smithYearRatRat2020}{}}%
Smith, Jennifer R, G Thomas Hayman, Shur-Jen Wang, Stanley J F Laulederkind, Matthew J Hoffman, Mary L Kaldunski, Monika Tutaj, et al. 2020. {``The {Year} of the {Rat}: {The Rat Genome Database} at 20: A Multi-Species Knowledgebase and Analysis Platform.''} \emph{Nucleic Acids Research} 48 (D1): D731--42. \url{https://doi.org/10.1093/nar/gkz1041}.

\leavevmode\vadjust pre{\hypertarget{ref-spivakovGenomicPhenotypicCharacterization2014}{}}%
Spivakov, Mikhail, Thomas O Auer, Ravindra Peravali, Ian Dunham, Dirk Dolle, Asao Fujiyama, Atsushi Toyoda, et al. 2014. {``Genomic and {Phenotypic Characterization} of a {Wild Medaka Population}: {Towards} the {Establishment} of an {Isogenic Population Genetic Resource} in {Fish}.''} \emph{G3 Genes\textbar Genomes\textbar Genetics} 4 (3): 433--45. \url{https://doi.org/10.1534/g3.113.008722}.

\leavevmode\vadjust pre{\hypertarget{ref-stadenStrategyDNASequencing1979}{}}%
Staden, R. 1979. {``A Strategy of {DNA} Sequencing Employing Computer Programs.''} \emph{Nucleic Acids Research} 6 (7): 2601--10. \url{https://doi.org/10.1093/nar/6.7.2601}.

\leavevmode\vadjust pre{\hypertarget{ref-steensmaAbbieLathropMouse2010}{}}%
Steensma, David P., Robert A. Kyle, and Marc A. Shampo. 2010. {``Abbie {Lathrop}, the {`{Mouse Woman} of {Granby}'}: {Rodent Fancier} and {Accidental Genetics Pioneer}.''} \emph{Mayo Clinic Proceedings} 85 (11): e83. \url{https://doi.org/10.4065/mcp.2010.0647}.

\leavevmode\vadjust pre{\hypertarget{ref-stratzLinkageDisequilibriumPattern2018}{}}%
Stratz, P., M. Schmid, R. Wellmann, S. Preuß, I. Blaj, J. Tetens, G. Thaller, and J. Bennewitz. 2018. {``Linkage Disequilibrium Pattern and Genome-Wide Association Mapping for Meat Traits in Multiple Porcine {F2} Crosses.''} \emph{Animal Genetics} 49 (5): 403--12. \url{https://doi.org/10.1111/age.12684}.

\leavevmode\vadjust pre{\hypertarget{ref-strickbergerEvolution1990}{}}%
Strickberger, Monroe W. 1990. \emph{Evolution}. {Jones and Bartlett}. \url{https://books.google.com?id=qU_wAAAAMAAJ}.

\leavevmode\vadjust pre{\hypertarget{ref-strittmatterApolipoproteinAlzheimerDisease1996}{}}%
Strittmatter, Warren J., and Allen D. Roses. 1996. {``Apolipoprotein {E} and {Alzheimer}'s {Disease}.''} \emph{Annual Review of Neuroscience} 19 (1): 53--77. \url{https://doi.org/10.1146/annurev.ne.19.030196.000413}.

\leavevmode\vadjust pre{\hypertarget{ref-sunPolygenicRiskScores2021}{}}%
Sun, Luanluan, Lisa Pennells, Stephen Kaptoge, Christopher P. Nelson, Scott C. Ritchie, Gad Abraham, Matthew Arnold, et al. 2021. {``Polygenic Risk Scores in Cardiovascular Risk Prediction: {A} Cohort Study and Modelling Analyses.''} \emph{PLOS Medicine} 18 (1): e1003498. \url{https://doi.org/10.1371/journal.pmed.1003498}.

\leavevmode\vadjust pre{\hypertarget{ref-svartbergShynessBoldnessPredicts2002}{}}%
Svartberg, Kenth. 2002. {``Shyness--Boldness Predicts Performance in Working Dogs.''} \emph{Applied Animal Behaviour Science} 79 (2): 157--74. \url{https://doi.org/10.1016/S0168-1591(02)00120-X}.

\leavevmode\vadjust pre{\hypertarget{ref-svensonHighResolutionGeneticMapping2012}{}}%
Svenson, Karen L, Daniel M Gatti, William Valdar, Catherine E Welsh, Riyan Cheng, Elissa J Chesler, Abraham A Palmer, Leonard McMillan, and Gary A Churchill. 2012. {``High-{Resolution Genetic Mapping Using} the {Mouse Diversity Outbred Population}.''} \emph{Genetics} 190 (2): 437--47. \url{https://doi.org/10.1534/genetics.111.132597}.

\leavevmode\vadjust pre{\hypertarget{ref-taberyStruggleUnderstandInteraction2014}{}}%
Tabery, James. 2014. \emph{Beyond Versus: {The} Struggle to Understand the Interaction of Nature and Nurture}. {mit Press}.

\leavevmode\vadjust pre{\hypertarget{ref-tangIdentificationLociAffecting2017}{}}%
Tang, Jianhong, Zhiyan Zhang, Bin Yang, Yuanmei Guo, Huashui Ai, Yi Long, Ying Su, et al. 2017. {``Identification of Loci Affecting Teat Number by Genome-Wide Association Studies on Three Pig Populations.''} \emph{Asian-Australasian Journal of Animal Sciences} 30 (1): 1--7. \url{https://doi.org/10.5713/ajas.15.0980}.

\leavevmode\vadjust pre{\hypertarget{ref-11TrillionGlobal2019}{}}%
{``The \$11.9 {Trillion Global Healthcare Market}: {Key Opportunities} \& {Strategies} (2014-2022) - {ResearchAndMarkets}.com.''} 2019. June 25, 2019. \url{https://www.businesswire.com/news/home/20190625005862/en/The-11.9-Trillion-Global-Healthcare-Market-Key-Opportunities-Strategies-2014-2022---ResearchAndMarkets.com}.

\leavevmode\vadjust pre{\hypertarget{ref-NobelPrizeChemistrya}{}}%
{``The {Nobel Prize} in {Chemistry} 1958.''} n.d. {NobelPrize.org}. Accessed September 28, 2022. \url{https://www.nobelprize.org/prizes/chemistry/1958/summary/}.

\leavevmode\vadjust pre{\hypertarget{ref-SaturdayReviewPolitics1859}{}}%
\emph{The {Saturday Review} of {Politics}, {Literature}, {Science} and {Art}}. 1859. {Saturday Review}. \url{https://books.google.com?id=8UZ4NwHp9c4C}.

\leavevmode\vadjust pre{\hypertarget{ref-threadgillCollaborativeCrossRecombinant2011}{}}%
Threadgill, David W., Darla R. Miller, Gary A. Churchill, and Fernando Pardo-Manuel de Villena. 2011. {``The {Collaborative Cross}: {A Recombinant Inbred Mouse Population} for the {Systems Genetic Era}.''} \emph{ILAR Journal} 52 (1): 24--31. \url{https://doi.org/10.1093/ilar.52.1.24}.

\leavevmode\vadjust pre{\hypertarget{ref-tikkanenGeneticRiskPrediction2013}{}}%
Tikkanen, Emmi, Aki S. Havulinna, Aarno Palotie, Veikko Salomaa, and Samuli Ripatti. 2013. {``Genetic {Risk Prediction} and a 2-{Stage Risk Screening Strategy} for {Coronary Heart Disease}.''} \emph{Arteriosclerosis, Thrombosis, and Vascular Biology} 33 (9): 2261--66. \url{https://doi.org/10.1161/ATVBAHA.112.301120}.

\leavevmode\vadjust pre{\hypertarget{ref-tortereauCombiningTwoMeishan2010}{}}%
Tortereau, Flavie, H'el`ene Gilbert, Henri CM Heuven, Jean-Pierre Bidanel, Martien AM Groenen, and Juliette Riquet. 2010. {``Combining Two {Meishan F2} Crosses Improves the Detection of {QTL} on Pig Chromosomes 2, 4 and 6.''} \emph{Genetics Selection Evolution} 42 (1): 42. \url{https://doi.org/10.1186/1297-9686-42-42}.

\leavevmode\vadjust pre{\hypertarget{ref-turleyMultitraitAnalysisGenomewide2018}{}}%
Turley, Patrick, Raymond K. Walters, Omeed Maghzian, Aysu Okbay, James J. Lee, Mark Alan Fontana, Tuan Anh Nguyen-Viet, et al. 2018. {``Multi-Trait Analysis of Genome-Wide Association Summary Statistics Using {MTAG}.''} \emph{Nature Genetics} 50 (2, 2): 229--37. \url{https://doi.org/10.1038/s41588-017-0009-4}.

\leavevmode\vadjust pre{\hypertarget{ref-uchidaDecreasedVigilanceHabituation2019}{}}%
Uchida, Kenta, Kei K Suzuki, Tatsuki Shimamoto, Hisashi Yanagawa, and Itsuro Koizumi. 2019. {``Decreased Vigilance or Habituation to Humans? {Mechanisms} on Increased Boldness in Urban Animals.''} \emph{Behavioral Ecology} 30 (6): 1583--90. \url{https://doi.org/10.1093/beheco/arz117}.

\leavevmode\vadjust pre{\hypertarget{ref-uffelmannGenomewideAssociationStudies2021}{}}%
Uffelmann, Emil, Qin Qin Huang, Nchangwi Syntia Munung, Jantina de Vries, Yukinori Okada, Alicia R. Martin, Hilary C. Martin, Tuuli Lappalainen, and Danielle Posthuma. 2021. {``Genome-Wide Association Studies.''} \emph{Nature Reviews Methods Primers} 1 (1, 1): 1--21. \url{https://doi.org/10.1038/s43586-021-00056-9}.

\leavevmode\vadjust pre{\hypertarget{ref-utineHomozygousDeletionGRID22013}{}}%
Utine, G. Eda, Göknur Haliloğlu, Bilge Salancı, Arda Çetinkaya, P. Özlem Kiper, Yasemin Alanay, Dilek Aktaş, Koray Boduroğlu, and Mehmet Alikaşifoğlu. 2013. {``A {Homozygous Deletion} in {Grid2 Causes} a {Human Phenotype With Cerebellar Ataxia} and {Atrophy}.''} \emph{Journal of Child Neurology} 28 (7): 926--32. \url{https://doi.org/10.1177/0883073813484967}.

\leavevmode\vadjust pre{\hypertarget{ref-vandenoordControllingFalseDiscoveries2008}{}}%
van den Oord, Edwin J. C. G. 2008. {``Controlling False Discoveries in Genetic Studies.''} \emph{American Journal of Medical Genetics Part B: Neuropsychiatric Genetics} 147B (5): 637--44. \url{https://doi.org/10.1002/ajmg.b.30650}.

\leavevmode\vadjust pre{\hypertarget{ref-vanderauweraGenomicsCloudUsing2020}{}}%
Van der Auwera, Geraldine A., and Brian D. O'Connor. 2020. \emph{Genomics in the Cloud: Using {Docker}, {GATK}, and {WDL} in {Terra}}. {O'Reilly Media}.

\leavevmode\vadjust pre{\hypertarget{ref-vasimuddinEfficientArchitectureawareAcceleration2019}{}}%
Vasimuddin, Md, Sanchit Misra, Heng Li, and Srinivas Aluru. 2019. {``Efficient Architecture-Aware Acceleration of {BWA-MEM} for Multicore Systems.''} In \emph{2019 {IEEE International Parallel} and {Distributed Processing Symposium} ({IPDPS})}, 314--24. {IEEE}.

\leavevmode\vadjust pre{\hypertarget{ref-venterSequenceHumanGenome2001}{}}%
Venter, J. Craig, Mark D. Adams, Eugene W. Myers, Peter W. Li, Richard J. Mural, Granger G. Sutton, Hamilton O. Smith, et al. 2001. {``The {Sequence} of the {Human Genome}.''} \emph{Science} 291 (5507): 1304--51. \url{https://doi.org/10.1126/science.1058040}.

\leavevmode\vadjust pre{\hypertarget{ref-vilhjalmsson2015modeling}{}}%
Vilhjálmsson, Bjarni J, Jian Yang, Hilary K Finucane, Alexander Gusev, Sara Lindström, Stephan Ripke, Giulio Genovese, et al. 2015. {``Modeling Linkage Disequilibrium Increases Accuracy of Polygenic Risk Scores.''} \emph{The American Journal of Human Genetics} 97 (4): 576--92.

\leavevmode\vadjust pre{\hypertarget{ref-visscherHeritabilityGenomicsEra2008}{}}%
Visscher, Peter M., William G. Hill, and Naomi R. Wray. 2008. {``Heritability in the Genomics Era --- Concepts and Misconceptions.''} \emph{Nature Reviews Genetics} 9 (4, 4): 255--66. \url{https://doi.org/10.1038/nrg2322}.

\leavevmode\vadjust pre{\hypertarget{ref-wadeGeneticVariationLaboratory2005}{}}%
Wade, Claire M., and Mark J. Daly. 2005. {``Genetic Variation in Laboratory Mice.''} \emph{Nature Genetics} 37 (11, 11): 1175--80. \url{https://doi.org/10.1038/ng1666}.

\leavevmode\vadjust pre{\hypertarget{ref-wagenseilDNADiscovererBlacks2015}{}}%
Wagenseil, Paul. 2015. {``{DNA Discoverer}: {Blacks Less Intelligent Than Whites}.''} Text.Article. {Fox News}; {Fox News}. March 25, 2015. \url{https://www.foxnews.com/story/dna-discoverer-blacks-less-intelligent-than-whites}.

\leavevmode\vadjust pre{\hypertarget{ref-wallaceXVIIILawWhich1855}{}}%
Wallace, Alfred R. 1855. {``{XVIII}.---{On} the Law Which Has Regulated the Introduction of New Species.''} \emph{Annals and Magazine of Natural History} 16 (93): 184--96.

\leavevmode\vadjust pre{\hypertarget{ref-wangTheoreticalEmpiricalQuantification2020}{}}%
Wang, Ying, Jing Guo, Guiyan Ni, Jian Yang, Peter M. Visscher, and Loic Yengo. 2020. {``Theoretical and Empirical Quantification of the Accuracy of Polygenic Scores in Ancestry Divergent Populations.''} \emph{Nature Communications} 11 (1, 1): 3865. \url{https://doi.org/10.1038/s41467-020-17719-y}.

\leavevmode\vadjust pre{\hypertarget{ref-wangNanoporeSequencingTechnology2021}{}}%
Wang, Yunhao, Yue Zhao, Audrey Bollas, Yuru Wang, and Kin Fai Au. 2021. {``Nanopore Sequencing Technology, Bioinformatics and Applications.''} \emph{Nature Biotechnology} 39 (11, 11): 1348--65. \url{https://doi.org/10.1038/s41587-021-01108-x}.

\leavevmode\vadjust pre{\hypertarget{ref-wanscherHistoryWilhelmJohannsen1975}{}}%
Wanscher, Johan Henrik. 1975. {``The History of {Wilhelm Johannsen}'s Genetical Terms and Concepts from the Period 1903 to 1926.''} \emph{Centaurus} 19 (2): 125--47.

\leavevmode\vadjust pre{\hypertarget{ref-watanabeGlobalOverviewPleiotropy2019}{}}%
Watanabe, Kyoko, Sven Stringer, Oleksandr Frei, Maša Umi'cevi'c Mirkov, Christiaan de Leeuw, Tinca J. C. Polderman, Sophie van der Sluis, Ole A. Andreassen, Benjamin M. Neale, and Danielle Posthuma. 2019. {``A Global Overview of Pleiotropy and Genetic Architecture in Complex Traits.''} \emph{Nature Genetics} 51 (9, 9): 1339--48. \url{https://doi.org/10.1038/s41588-019-0481-0}.

\leavevmode\vadjust pre{\hypertarget{ref-watsonMolecularStructureNucleic1953}{}}%
Watson, J. D., and F. H. C. Crick. 1953. {``Molecular {Structure} of {Nucleic Acids}: {A Structure} for {Deoxyribose Nucleic Acid}.''} \emph{Nature} 171 (4356, 4356): 737--38. \url{https://doi.org/10.1038/171737a0}.

\leavevmode\vadjust pre{\hypertarget{ref-watsonDNASecretLife2009}{}}%
Watson, James D., and Andrew Berry. 2009. \emph{{DNA}: {The} Secret of Life}. {Knopf}.

\leavevmode\vadjust pre{\hypertarget{ref-watsonAnnotatedIllustratedDouble2012}{}}%
Watson, James D., Alexander Gann, and Jan Witkowski. 2012. \emph{The Annotated and Illustrated Double Helix}. {Simon and Schuster}.

\leavevmode\vadjust pre{\hypertarget{ref-weinbergCraniofacialGeneticsWhere2018}{}}%
Weinberg, Seth M., Robert Cornell, and Elizabeth J. Leslie. 2018. {``Craniofacial Genetics: {Where} Have We Been and Where Are We Going?''} \emph{PLOS Genetics} 14 (6): e1007438. \url{https://doi.org/10.1371/journal.pgen.1007438}.

\leavevmode\vadjust pre{\hypertarget{ref-weirEstimatingFStatisticsAnalysis1984}{}}%
Weir, B. S., and C. Clark Cockerham. 1984. {``Estimating {F-Statistics} for the {Analysis} of {Population Structure}.''} \emph{Evolution} 38 (6): 1358--70. \url{https://doi.org/10.2307/2408641}.

\leavevmode\vadjust pre{\hypertarget{ref-weirMeasuresHumanPopulation2005}{}}%
Weir, Bruce S., Lon R. Cardon, Amy D. Anderson, Dahlia M. Nielsen, and William G. Hill. 2005. {``Measures of Human Population Structure Show Heterogeneity Among Genomic Regions.''} \emph{Genome Research} 15 (11): 1468--76. \url{https://doi.org/10.1101/gr.4398405}.

\leavevmode\vadjust pre{\hypertarget{ref-weischenfeldtPhenotypicImpactGenomic2013}{}}%
Weischenfeldt, Joachim, Orsolya Symmons, François Spitz, and Jan O. Korbel. 2013. {``Phenotypic Impact of Genomic Structural Variation: Insights from and for Human Disease.''} \emph{Nature Reviews Genetics} 14 (2, 2): 125--38. \url{https://doi.org/10.1038/nrg3373}.

\leavevmode\vadjust pre{\hypertarget{ref-wellcomeFrancisCrickControversial2003}{}}%
Wellcome. 2003. {``Francis {Crick}'s Controversial Archive on First Public Display.''} {Wellcome}. March 31, 2003. \url{https://wellcome.org/press-release/francis-cricks-controversial-archive-first-public-display}.

\leavevmode\vadjust pre{\hypertarget{ref-wengerAccurateCircularConsensus2019}{}}%
Wenger, Aaron M., Paul Peluso, William J. Rowell, Pi-Chuan Chang, Richard J. Hall, Gregory T. Concepcion, Jana Ebler, et al. 2019. {``Accurate Circular Consensus Long-Read Sequencing Improves Variant Detection and Assembly of a Human Genome.''} \emph{Nature Biotechnology} 37 (10, 10): 1155--62. \url{https://doi.org/10.1038/s41587-019-0217-9}.

\leavevmode\vadjust pre{\hypertarget{ref-westfallResamplingBasedMultipleTesting1993}{}}%
Westfall, Peter H., and S. Stanley Young. 1993. \emph{Resampling-{Based Multiple Testing}: {Examples} and {Methods} for p-{Value Adjustment}}. {John Wiley \& Sons}. \url{https://books.google.com?id=nuQXORVGI1QC}.

\leavevmode\vadjust pre{\hypertarget{ref-wetterstrandDNASequencingCosts2021}{}}%
Wetterstrand, Kris A. 2021. {``{DNA Sequencing Costs}: {Data}.''} {Genome.gov}. November 1, 2021. \url{https://www.genome.gov/about-genomics/fact-sheets/DNA-Sequencing-Costs-Data}.

\leavevmode\vadjust pre{\hypertarget{ref-wichuraAlgorithm241Percentage1988}{}}%
Wichura, Michael J. 1988. {``Algorithm {AS} 241: {The Percentage Points} of the {Normal Distribution}.''} \emph{Journal of the Royal Statistical Society. Series C (Applied Statistics)} 37 (3): 477--84. \url{https://doi.org/10.2307/2347330}.

\leavevmode\vadjust pre{\hypertarget{ref-batesonPhoto}{}}%
Wikimedia Commons. 2017a. {``File:Bateson2.jpg --- Wikimedia Commons{,} the Free Media Repository.''} \url{/url\%7Bhttps://commons.wikimedia.org/w/index.php?title=File:Bateson2.jpg\&oldid=255958176\%7D}.

\leavevmode\vadjust pre{\hypertarget{ref-charlesDarwinYoung}{}}%
---------. 2017b. {``File:charles Darwin by g. Richmond.png --- Wikimedia Commons{,} the Free Media Repository.''} \url{/url\%7Bhttps://commons.wikimedia.org/w/index.php?title=File:Charles_Darwin_by_G._Richmond.png\&oldid=260695102\%7D}.

\leavevmode\vadjust pre{\hypertarget{ref-homunculusImage}{}}%
---------. 2019. {``File:preformation.GIF --- Wikimedia Commons{,} the Free Media Repository.''} \url{/url\%7Bhttps://commons.wikimedia.org/w/index.php?title=File:Preformation.GIF\&oldid=372547016\%7D}.

\leavevmode\vadjust pre{\hypertarget{ref-pearsonPhoto}{}}%
---------. 2020. {``File:karl Pearson, 1910.jpg --- Wikimedia Commons{,} the Free Media Repository.''} \url{/url\%7Bhttps://commons.wikimedia.org/w/index.php?title=File:Karl_Pearson,_1910.jpg\&oldid=477532248\%7D}.

\leavevmode\vadjust pre{\hypertarget{ref-galtonPhoto}{}}%
---------. 2021a. {``File:francis Galton 1850s.jpg --- Wikimedia Commons{,} the Free Media Repository.''} \url{/url\%7Bhttps://commons.wikimedia.org/w/index.php?title=File:Francis_Galton_1850s.jpg\&oldid=569302173\%7D}.

\leavevmode\vadjust pre{\hypertarget{ref-mendelPhoto}{}}%
---------. 2021b. {``File:gregor Mendel 2.jpg --- Wikimedia Commons{,} the Free Media Repository.''} \url{/url\%7Bhttps://commons.wikimedia.org/w/index.php?title=File:Gregor_Mendel_2.jpg\&oldid=534728192\%7D}.

\leavevmode\vadjust pre{\hypertarget{ref-fisherPhoto}{}}%
---------. 2021c. {``File:Youngronaldfisher2.JPG --- Wikimedia Commons{,} the Free Media Repository.''} \url{/url\%7Bhttps://commons.wikimedia.org/w/index.php?title=File:Youngronaldfisher2.JPG\&oldid=614548490\%7D}.

\leavevmode\vadjust pre{\hypertarget{ref-alfredWallace1895}{}}%
---------. 2022a. {``File:alfred-Russel-Wallace-C1895.jpg --- Wikimedia Commons{,} the Free Media Repository.''} \url{https://commons.wikimedia.org/w/index.php?title=File:Alfred-Russel-Wallace-c1895.jpg\&oldid=642853479}.

\leavevmode\vadjust pre{\hypertarget{ref-darwinFinches}{}}%
---------. 2022b. {``File:darwin's Finches by Gould.jpg --- Wikimedia Commons{,} the Free Media Repository.''} \url{/url\%7Bhttps://commons.wikimedia.org/w/index.php?title=File:Darwin\%27s_finches_by_Gould.jpg\&oldid=643906085\%7D}.

\leavevmode\vadjust pre{\hypertarget{ref-lamarck1802}{}}%
---------. 2022c. {``File:jean-Baptiste de Lamarck.jpg --- Wikimedia Commons{,} the Free Media Repository.''} \url{/url\%7Bhttps://commons.wikimedia.org/w/index.php?title=File:Jean-Baptiste_de_Lamarck.jpg\&oldid=641428959\%7D}.

\leavevmode\vadjust pre{\hypertarget{ref-wilkinsMolecularStructureNucleic1953}{}}%
Wilkins, M. H. F., A. R. Stokes, and H. R. Wilson. 1953. {``Molecular {Structure} of {Nucleic Acids}: {Molecular Structure} of {Deoxypentose Nucleic Acids}.''} \emph{Nature} 171 (4356, 4356): 738--40. \url{https://doi.org/10.1038/171738a0}.

\leavevmode\vadjust pre{\hypertarget{ref-williamsUnravellingDoubleHelix2019}{}}%
Williams, Gareth. 2019. \emph{Unravelling the {Double Helix}: {The Lost Heroes} of {DNA}}. {Hachette UK}. \url{https://books.google.com?id=uFRrDwAAQBAJ}.

\leavevmode\vadjust pre{\hypertarget{ref-wilsonShyboldContinuumPumpkinseed1993}{}}%
Wilson, David S., Kristine Coleman, Anne B. Clark, and Laurence Biederman. 1993. {``Shy-Bold Continuum in Pumpkinseed Sunfish ({Lepomis} Gibbosus): {An} Ecological Study of a Psychological Trait.''} \emph{Journal of Comparative Psychology} 107 (3): 250--60. \url{https://doi.org/10.1037/0735-7036.107.3.250}.

\leavevmode\vadjust pre{\hypertarget{ref-wittbrodtMedakaModelOrganism2002}{}}%
Wittbrodt, Joachim, Akihiro Shima, and Manfred Schartl. 2002. {``Medaka --- a Model Organism from the Far East.''} \emph{Nature Reviews Genetics} 3 (1, 1): 53--64. \url{https://doi.org/10.1038/nrg704}.

\leavevmode\vadjust pre{\hypertarget{ref-wojcikGeneticAnalysesDiverse2019}{}}%
Wojcik, Genevieve L., Mariaelisa Graff, Katherine K. Nishimura, Ran Tao, Jeffrey Haessler, Christopher R. Gignoux, Heather M. Highland, et al. 2019. {``Genetic Analyses of Diverse Populations Improves Discovery for Complex Traits.''} \emph{Nature} 570 (7762, 7762): 514--18. \url{https://doi.org/10.1038/s41586-019-1310-4}.

\leavevmode\vadjust pre{\hypertarget{ref-wrayPredictionIndividualGenetic2007}{}}%
Wray, Naomi R., Michael E. Goddard, and Peter M. Visscher. 2007. {``Prediction of Individual Genetic Risk to Disease from Genome-Wide Association Studies.''} \emph{Genome Research} 17 (10): 1520--28. \url{https://doi.org/10.1101/gr.6665407}.

\leavevmode\vadjust pre{\hypertarget{ref-wrightEpistaticRegulationBehavioural2006}{}}%
Wright, Dominic, Roger K. Butlin, and Örjan Carlborg. 2006. {``Epistatic Regulation of Behavioural and Morphological Traits in the Zebrafish ({Danio} Rerio).''} \emph{Behavior Genetics} 36 (6): 914--22. \url{https://doi.org/10.1007/s10519-006-9080-9}.

\leavevmode\vadjust pre{\hypertarget{ref-wrightInterIntrapopulationVariation2003}{}}%
Wright, D., L. B. Rimmer, V. L. Pritchard, R. K. Butlin, and J. Krause. 2003. {``Inter and Intra-Population Variation in Shoaling and Boldness in the Zebrafish ({Danio} Rerio).''} \emph{Journal of Fish Biology} 63 (s1): 258--59. \url{https://doi.org/10.1111/j.1095-8649.2003.216bw.x}.

\leavevmode\vadjust pre{\hypertarget{ref-wrightCoefficientsInbreedingRelationship1922}{}}%
Wright, Sewall. 1922. {``Coefficients of {Inbreeding} and {Relationship}.''} \emph{The American Naturalist} 56 (645): 330--38. \url{https://doi.org/10.1086/279872}.

\leavevmode\vadjust pre{\hypertarget{ref-wrightGeneticalTheoryNatural1930}{}}%
---------. 1930. {``The {Genetical Theory} of {Natural Selection}: {A Review}.''} \emph{Journal of Heredity} 21 (8): 349--56. \url{https://doi.org/10.1093/oxfordjournals.jhered.a103361}.

\leavevmode\vadjust pre{\hypertarget{ref-wrightIsolationDistanceDiverse1946}{}}%
---------. 1946. {``Isolation by {Distance} Under {Diverse Systems} of {Mating}.''} \emph{Genetics} 31 (1): 39--59. \url{https://doi.org/10.1093/genetics/31.1.39}.

\leavevmode\vadjust pre{\hypertarget{ref-wrightRolesDirectedRandom1948}{}}%
---------. 1948. {``On the {Roles} of {Directed} and {Random Changes} in {Gene Frequency} in the {Genetics} of {Populations}.''} \emph{Evolution} 2 (4): 279--94. \url{https://doi.org/10.1111/j.1558-5646.1948.tb02746.x}.

\leavevmode\vadjust pre{\hypertarget{ref-wrightGeneticalStructurePopulations1949}{}}%
---------. 1949. {``The {Genetical Structure} of {Populations}.''} \emph{Annals of Eugenics} 15 (1): 323--54. \url{https://doi.org/10.1111/j.1469-1809.1949.tb02451.x}.

\leavevmode\vadjust pre{\hypertarget{ref-yangGCTAToolGenomewide2011}{}}%
Yang, Jian, S. Hong Lee, Michael E. Goddard, and Peter M. Visscher. 2011. {``{GCTA}: {A Tool} for {Genome-wide Complex Trait Analysis}.''} \emph{The American Journal of Human Genetics} 88 (1): 76--82. \url{https://doi.org/10.1016/j.ajhg.2010.11.011}.

\leavevmode\vadjust pre{\hypertarget{ref-yangGenomePartitioningGenetic2011}{}}%
Yang, Jian, Teri A. Manolio, Louis R. Pasquale, Eric Boerwinkle, Neil Caporaso, Julie M. Cunningham, Mariza de Andrade, et al. 2011. {``Genome Partitioning of Genetic Variation for Complex Traits Using Common {SNPs}.''} \emph{Nature Genetics} 43 (6, 6): 519--25. \url{https://doi.org/10.1038/ng.823}.

\leavevmode\vadjust pre{\hypertarget{ref-yatesEnsembl20202020}{}}%
Yates, Andrew D, Premanand Achuthan, Wasiu Akanni, James Allen, Jamie Allen, Jorge Alvarez-Jarreta, M Ridwan Amode, et al. 2020. {``Ensembl 2020.''} \emph{Nucleic Acids Research} 48 (D1): D682--88. \url{https://doi.org/10.1093/nar/gkz966}.

\leavevmode\vadjust pre{\hypertarget{ref-yengoMetaanalysisGenomewideAssociation2018}{}}%
Yengo, Loic, Julia Sidorenko, Kathryn E Kemper, Zhili Zheng, Andrew R Wood, Michael N Weedon, Timothy M Frayling, et al. 2018. {``Meta-Analysis of Genome-Wide Association Studies for Height and Body Mass Index in ~700000 Individuals of {European} Ancestry.''} \emph{Human Molecular Genetics} 27 (20): 3641--49. \url{https://doi.org/10.1093/hmg/ddy271}.

\leavevmode\vadjust pre{\hypertarget{ref-youngNeurobiologyHumanSocial2008}{}}%
Young, Simon N. 2008. {``The Neurobiology of Human Social Behaviour: An Important but Neglected Topic.''} \emph{Journal of Psychiatry and Neuroscience} 33 (5): 391--92. \url{https://www.jpn.ca/content/33/5/391}.

\leavevmode\vadjust pre{\hypertarget{ref-yuUnifiedMixedmodelMethod2006}{}}%
Yu, Jianming, Gael Pressoir, William H. Briggs, Irie Vroh Bi, Masanori Yamasaki, John F. Doebley, Michael D. McMullen, et al. 2006. {``A Unified Mixed-Model Method for Association Mapping That Accounts for Multiple Levels of Relatedness.''} \emph{Nature Genetics} 38 (2, 2): 203--8. \url{https://doi.org/10.1038/ng1702}.

\leavevmode\vadjust pre{\hypertarget{ref-yuenBoldnessAreOpen2017}{}}%
Yuen, C. H., I. Schoepf, C. Schradin, and N. Pillay. 2017. {``Boldness: Are Open Field and Startle Tests Measuring the Same Personality Trait?''} \emph{Animal Behaviour} 128 (June): 143--51. \url{https://doi.org/10.1016/j.anbehav.2017.04.009}.

\end{CSLReferences}

\hypertarget{appendix-appendix}{%
\appendix}


\hypertarget{chapter-2-supplementary-information}{%
\chapter{Chapter 2 supplementary information}\label{chapter-2-supplementary-information}}



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_genome/supp_02_pi_chrs_1_22} \caption{(ref:uc-div-per-indiv)}\label{fig:nuc-div-per-indiv}
\end{figure}

\hypertarget{chapter-5-supplementary-information}{%
\chapter{Chapter 5 supplementary information}\label{chapter-5-supplementary-information}}

\hypertarget{hmm-states-with-0.05-second-interval}{%
\section{15 HMM states with 0.05 second interval}\label{hmm-states-with-0.05-second-interval}}



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/0.05_15_polar_all_dge} \caption{HMM states predicted for the F0 dataset, using the combination of a 0.05-second interval between which the distance and angle variables were calculated.}\label{fig:hmm-states-05}
\end{figure}

\hypertarget{four-panel-app}{%
\section{Four-panel plots for MIKK lines selected for the F2 cross}\label{four-panel-app}}

All frame grabs were taken at the 300th second (5-minute mark) of each 10-minute assay component.

\hypertarget{open-field}{%
\subsection{Open field}\label{open-field}}

\hypertarget{section}{%
\subsubsection{8-2}\label{section}}



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/four_panel_plots/open_field_20191114_1040_8-2_L_A_300} \caption{Line 8-2, open field, run A.}\label{fig:4p-8-2-of-A}
\end{figure}



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/four_panel_plots/open_field_20191118_1224_8-2_R_B_300} \caption{Line 8-2, open field, run B.}\label{fig:4p-8-2-of-B}
\end{figure}

\hypertarget{section-1}{%
\subsubsection{18-2}\label{section-1}}



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/four_panel_plots/open_field_20191112_1236_18-2_L_A_300} \caption{Line 18-2, open field, run A.}\label{fig:4p-18-2-of-A}
\end{figure}



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/four_panel_plots/open_field_20191116_1039_18-2_L_B_300} \caption{Line 18-2, open field, run B.}\label{fig:4p-18-2-of-B}
\end{figure}



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/four_panel_plots/open_field_20191119_1459_18-2_R_C_300} \caption{Line 18-2, open field, run C.}\label{fig:4p-18-2-of-C}
\end{figure}

\hypertarget{section-2}{%
\subsubsection{50-2}\label{section-2}}



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/four_panel_plots/open_field_20191114_1305_50-2_R_A_300} \caption{Line 50-2, open field, run A.}\label{fig:4p-50-2-of-A}
\end{figure}



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/four_panel_plots/open_field_20191116_1438_50-2_R_B_300} \caption{Line 50-2, open field, run B.}\label{fig:4p-50-2-of-B}
\end{figure}



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/four_panel_plots/open_field_20191120_1150_50-2_R_C_300} \caption{Line 50-2, open field, run C.}\label{fig:4p-50-2-of-C}
\end{figure}

\hypertarget{section-3}{%
\subsubsection{38-2}\label{section-3}}



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/four_panel_plots/open_field_20191112_1456_38-2_L_A_300} \caption{Line 38-2, open field, run A.}\label{fig:4p-38-2-of-A}
\end{figure}



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/four_panel_plots/open_field_20191116_1203_38-2_L_B_300} \caption{Line 38-2, open field, run B.}\label{fig:4p-38-2-of-B}
\end{figure}

\hypertarget{section-4}{%
\subsubsection{21-2}\label{section-4}}



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/four_panel_plots/open_field_20191113_1527_21-2_L_A_300} \caption{Line 38-2, open field, run A.}\label{fig:4p-21-2-of-A}
\end{figure}



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/four_panel_plots/open_field_20191116_1309_21-2_R_B_300} \caption{Line 38-2, open field, run B.}\label{fig:4p-21-2-of-B}
\end{figure}

\hypertarget{section-5}{%
\subsubsection{40-1}\label{section-5}}



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/four_panel_plots/open_field_20191114_1217_40-1_L_A_300} \caption{Line 40-1, open field, run A.}\label{fig:4p-40-1-of-A}
\end{figure}



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/four_panel_plots/open_field_20191118_1129_40-1_R_B_300} \caption{Line 40-1, open field, run B.}\label{fig:4p-40-1-of-B}
\end{figure}

\hypertarget{novel-object}{%
\subsection{Novel object}\label{novel-object}}

\hypertarget{section-6}{%
\subsubsection{8-2}\label{section-6}}



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/four_panel_plots/novel_object_20191114_1040_8-2_L_A_300} \caption{Line 8-2, novel object, run A.}\label{fig:4p-8-2-no-A}
\end{figure}



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/four_panel_plots/novel_object_20191118_1224_8-2_R_B_300} \caption{Line 8-2, novel object, run B.}\label{fig:4p-8-2-no-B}
\end{figure}

\hypertarget{section-7}{%
\subsubsection{18-2}\label{section-7}}



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/four_panel_plots/novel_object_20191112_1236_18-2_L_A_300} \caption{Line 18-2, novel object, run A.}\label{fig:4p-18-2-no-A}
\end{figure}



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/four_panel_plots/novel_object_20191116_1039_18-2_L_B_300} \caption{Line 18-2, novel object, run B.}\label{fig:4p-18-2-no-B}
\end{figure}



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/four_panel_plots/novel_object_20191119_1459_18-2_R_C_300} \caption{Line 18-2, novel object, run C.}\label{fig:4p-18-2-no-C}
\end{figure}

\hypertarget{section-8}{%
\subsubsection{50-2}\label{section-8}}



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/four_panel_plots/novel_object_20191114_1305_50-2_R_A_300} \caption{Line 50-2, novel object, run A.}\label{fig:4p-50-2-no-A}
\end{figure}



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/four_panel_plots/novel_object_20191116_1438_50-2_R_B_300} \caption{Line 50-2, novel object, run B.}\label{fig:4p-50-2-no-B}
\end{figure}



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/four_panel_plots/novel_object_20191120_1150_50-2_R_C_300} \caption{Line 50-2, novel object, run C.}\label{fig:4p-50-2-no-C}
\end{figure}

\hypertarget{section-9}{%
\subsubsection{38-2}\label{section-9}}



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/four_panel_plots/novel_object_20191112_1456_38-2_L_A_300} \caption{Line 38-2, novel object, run A.}\label{fig:4p-38-2-no-A}
\end{figure}



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/four_panel_plots/novel_object_20191116_1203_38-2_L_B_300} \caption{Line 38-2, novel object, run B.}\label{fig:4p-38-2-no-B}
\end{figure}

\hypertarget{section-10}{%
\subsubsection{21-2}\label{section-10}}



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/four_panel_plots/novel_object_20191113_1527_21-2_L_A_300} \caption{Line 38-2, novel object, run A.}\label{fig:4p-21-2-no-A}
\end{figure}



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/four_panel_plots/novel_object_20191116_1309_21-2_R_B_300} \caption{Line 38-2, novel object, run B.}\label{fig:4p-21-2-no-B}
\end{figure}

\hypertarget{section-11}{%
\subsubsection{40-1}\label{section-11}}



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/four_panel_plots/novel_object_20191114_1217_40-1_L_A_300} \caption{Line 40-1, novel object, run A.}\label{fig:4p-40-1-no-A}
\end{figure}



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/four_panel_plots/novel_object_20191118_1129_40-1_R_B_300} \caption{Line 40-1, novel object, run B.}\label{fig:4p-40-1-no-B}
\end{figure}

\hypertarget{hmm-state-time-dependence-for-all-mikk-panel-lines}{%
\section{HMM state time dependence for all MIKK panel lines}\label{hmm-state-time-dependence-for-all-mikk-panel-lines}}

\hypertarget{direct-genetic-effects-3}{%
\subsection{Direct genetic effects}\label{direct-genetic-effects-3}}

\hypertarget{tile-plot}{%
\subsubsection{Tile plot}\label{tile-plot}}



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/0.08_15_tile_dge} \caption{Tile plot for all 1610 test fishes included in the MIKK panel behaviour analysis, ordered by each line's group median for individual mean speed over the course of the 20-minute video (open field and novel object combined). The order of lines is identical to that shown in \textbf{Figure \ref{fig:mikk-mean-speed}}.}\label{fig:mikk-all-tile-dge}
\end{figure}

\hypertarget{density-plot}{%
\subsubsection{Density plot}\label{density-plot}}



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/0.08_15_dens_dge} \caption{Tile plot for all 1610 test fishes included in the MIKK panel behaviour analysis, ordered by each line's group median for individual mean speed over the course of the 20-minute video (open field and novel object combined). The order of lines is identical to that shown in \textbf{Figure \ref{fig:mikk-mean-speed}}.}\label{fig:mikk-all-dens-dge}
\end{figure}

\hypertarget{social-genetic-effects-3}{%
\subsection{Social genetic effects}\label{social-genetic-effects-3}}

\hypertarget{tile-plot-1}{%
\subsubsection{Tile plot}\label{tile-plot-1}}



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/0.08_15_tile_sge} \caption{Tile plot for all 1610 reference fishes included in the MIKK panel behaviour analysis, ordered by each line's group median for individual mean speed over the course of the 20-minute video (open field and novel object combined). The order of lines is identical to that shown in \textbf{Figure \ref{fig:mikk-mean-speed}}.}\label{fig:mikk-all-tile-sge}
\end{figure}

\hypertarget{density-plot-1}{%
\subsubsection{Density plot}\label{density-plot-1}}



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/0.08_15_dens_sge} \caption{Tile plot for all 1610 reference fishes included in the MIKK panel behaviour analysis, ordered by each line's group median for individual mean speed over the course of the 20-minute video (open field and novel object combined). The order of lines is identical to that shown in \textbf{Figure \ref{fig:mikk-mean-speed}}.}\label{fig:mikk-all-dens-sge}
\end{figure}

\hypertarget{f2-recombination-karyoplot-with-missing-calls}{%
\section{F2 recombination karyoplot with missing calls}\label{f2-recombination-karyoplot-with-missing-calls}}



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/karyoplot_wi_missing} \caption{Karyoplot for F2 samples, coloured by genotype. Samples are sorted in the order in which they were phenotyped. Blocks are filled with the colour of the paternal F0 line for the homozygous paternal haplotype block, black for heterozygous, and the colour of the maternal F0 line for the homozygous maternal haplotype block.}\label{fig:F2-recomb-blocks-missing}
\end{figure}

\hypertarget{loco-grm-for-chromosome-1}{%
\section{LOCO GRM for chromosome 1}\label{loco-grm-for-chromosome-1}}



\begin{figure}
\includegraphics[width=1\linewidth]{figs/mikk_behaviour/grm_loco_man_chr1} \caption{``Leave-one-chromosome-out'' genetic relationship matrix for 271 F2 samples based on 44,284 non-missing SNPs, having excluded 76 SNPs on chromosome 1.}\label{fig:loco-grm-chr1}
\end{figure}

\hypertarget{chapter-6-supplementary-information}{%
\chapter{Chapter 6 supplementary information}\label{chapter-6-supplementary-information}}

\hypertarget{ecdf-of-all-polygenic-traits-in-the-gwas-catalog-ranked-by-d_ts}{%
\section{\texorpdfstring{eCDF of all polygenic traits in the GWAS Catalog ranked by \({D_t^S}\)}{eCDF of all polygenic traits in the GWAS Catalog ranked by \{D\_t\^{}S\}}}\label{ecdf-of-all-polygenic-traits-in-the-gwas-catalog-ranked-by-d_ts}}



\begin{figure}
\includegraphics[width=1\linewidth]{figs/fst/0.1_1000_20220314_ecdf_all_faceted_with_controls_d_rank_long} \caption{587 traits from the GWAS Catalog that passed our filters for polygenic traits, ranked by \({D_t^S}\).}\label{fig:eCDFall}
\end{figure}

\end{document}
