<!DOCTYPE html>
<html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<title>Chapter 3 Classification of bold/shy behaviours in 5 inbred medaka lines | Japanese courage: a genetic analysis of complex traits in medaka fish and humans</title>
<meta name="author" content="Ian Brettell">
<meta name="description" content="Taking advantage of the rich genetic resources offered by medaka we established an OFT to examine the performance of isogenic inbred strains for an inter-strain comparative analysis of the...">
<meta name="generator" content="bookdown 0.28 with bs4_book()">
<meta property="og:title" content="Chapter 3 Classification of bold/shy behaviours in 5 inbred medaka lines | Japanese courage: a genetic analysis of complex traits in medaka fish and humans">
<meta property="og:type" content="book">
<meta property="og:description" content="Taking advantage of the rich genetic resources offered by medaka we established an OFT to examine the performance of isogenic inbred strains for an inter-strain comparative analysis of the...">
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="Chapter 3 Classification of bold/shy behaviours in 5 inbred medaka lines | Japanese courage: a genetic analysis of complex traits in medaka fish and humans">
<meta name="twitter:description" content="Taking advantage of the rich genetic resources offered by medaka we established an OFT to examine the performance of isogenic inbred strains for an inter-strain comparative analysis of the...">
<!-- JS --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://kit.fontawesome.com/6ecbd6c532.js" crossorigin="anonymous"></script><script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="libs/bootstrap-4.6.0/bootstrap.min.css" rel="stylesheet">
<script src="libs/bootstrap-4.6.0/bootstrap.bundle.min.js"></script><script src="libs/bs3compat-0.4.0/transition.js"></script><script src="libs/bs3compat-0.4.0/tabs.js"></script><script src="libs/bs3compat-0.4.0/bs3compat.js"></script><link href="libs/bs4_book-1.0.0/bs4_book.css" rel="stylesheet">
<script src="libs/bs4_book-1.0.0/bs4_book.js"></script><style>
    #content {
        font-size: 0.9rem;
    }
    </style>
<script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- CSS --><style type="text/css">
    
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  </style>
<style type="text/css">
    /* Used with Pandoc 2.11+ new --citeproc when CSL is used */
    div.csl-bib-body { }
    div.csl-entry {
      clear: both;
        }
    .hanging div.csl-entry {
      margin-left:2em;
      text-indent:-2em;
    }
    div.csl-left-margin {
      min-width:2em;
      float:left;
    }
    div.csl-right-inline {
      margin-left:2em;
      padding-left:1em;
    }
    div.csl-indent {
      margin-left: 2em;
    }
  </style>
<link rel="stylesheet" href="style.css">
</head>
<body data-spy="scroll" data-target="#toc">

<div class="container-fluid">
<div class="row">
  <header class="col-sm-12 col-lg-3 sidebar sidebar-book"><a class="sr-only sr-only-focusable" href="#content">Skip to main content</a>

    <div class="d-flex align-items-start justify-content-between">
      <h1>
        <a href="index.html" title="">Japanese courage: a genetic analysis of complex traits in medaka fish and humans</a>
      </h1>
      <button class="btn btn-outline-primary d-lg-none ml-2 mt-1" type="button" data-toggle="collapse" data-target="#main-nav" aria-expanded="true" aria-controls="main-nav"><i class="fas fa-bars"></i><span class="sr-only">Show table of contents</span></button>
    </div>

    <div id="main-nav" class="collapse-lg">
      <form role="search">
        <input id="search" class="form-control" type="search" placeholder="Search" aria-label="Search">
</form>

      <nav aria-label="Table of contents"><h2>Table of contents</h2>
        <ul class="book-toc list-unstyled">
<li><a class="" href="index.html">About</a></li>
<li><a class="" href="Introduction.html"><span class="header-section-number">1</span> Introduction</a></li>
<li><a class="" href="MIKK-genomes-chap.html"><span class="header-section-number">2</span> Genomic variations in the MIKK panel</a></li>
<li><a class="active" href="Pilot-chap.html"><span class="header-section-number">3</span> Classification of bold/shy behaviours in 5 inbred medaka lines</a></li>
<li><a class="" href="Somite-chap.html"><span class="header-section-number">4</span> Genetic loci associated with somite development periodicity</a></li>
<li><a class="" href="MIKK-F2-chap.html"><span class="header-section-number">5</span> Genetic linkage study of bold/shy behaviours in the MIKK panel</a></li>
<li><a class="" href="Fst-chap.html"><span class="header-section-number">6</span> Variation in the frequency of trait-associated alleles across global human populations</a></li>
<li><a class="" href="references.html">References</a></li>
<li class="book-part">Appendix</li>
<li><a class="" href="hmm-states-with-0.05-second-interval.html"><span class="header-section-number">A</span> 15 HMM states with 0.05 second interval</a></li>
<li><a class="" href="four-panel-app.html"><span class="header-section-number">B</span> Four-panel plots for MIKK lines selected for the F2 cross</a></li>
<li><a class="" href="hmm-state-time-dependence-for-all-mikk-panel-lines.html"><span class="header-section-number">C</span> HMM state time dependence for all MIKK panel lines</a></li>
<li><a class="" href="f2-recombination-karyoplot-with-missing-calls.html"><span class="header-section-number">D</span> F2 recombination karyoplot with missing calls</a></li>
<li><a class="" href="loco-grm-for-chromosome-1.html"><span class="header-section-number">E</span> LOCO GRM for chromosome 1</a></li>
<li><a class="" href="ecdf-of-all-polygenic-traits-in-the-gwas-catalog-ranked-by-d_ts.html"><span class="header-section-number">F</span> eCDF of all polygenic traits in the GWAS Catalog ranked by \({D_t^S}\)</a></li>
</ul>

        <div class="book-extra">
          <p><a id="book-repo" href="https://github.com/brettellebi/PhD-thesis">View book source <i class="fab fa-github"></i></a></p>
        </div>
      </nav>
</div>
  </header><main class="col-sm-12 col-md-9 col-lg-7" id="content"><div id="Pilot-chap" class="section level1" number="3">
<h1>
<span class="header-section-number">3</span> Classification of bold/shy behaviours in 5 inbred medaka lines<a class="anchor" aria-label="anchor" href="#Pilot-chap"><i class="fas fa-link"></i></a>
</h1>
<p>Taking advantage of the rich genetic resources offered by medaka we established an OFT to examine the performance of isogenic inbred strains for an inter-strain comparative analysis of the boldness-shyness behaviour of medaka. We used a combined open-field and novel-object assay involving assaying pairs for fish, to determine: a) whether there were consistent differences in bold-shy behaviours exhibited by 5 established inbred strains of medaka fish (<em>iCab</em>, <em>HdrR</em>, and <em>HO5</em> from southern Japan, and <em>Kaga</em> and <em>HNI</em> from northern Japan); and b) whether there were consistent differences in bold-shy behaviours exhibited by a given strain (<em>iCab</em>) dependent on the strain that it was partnered with. The former was intended to measure the effect of an individual’s own genes on its behaviour (a direct genetic effect), and the latter to measure the effect of the genes of the focal fish’s tank partner on the behaviour of the focal fish (social genetic effect). Our experimental design allowed us to assess both direct and indirect effects on behaviour simultaneously, and to infer the degree to which variation in bold-shy behaviours is attributable to the differences in an individual’s own genetics, the differences in the genetics of their social companions, and stochastic variation. A schema for the experimental plan is shown in <strong>Figure <a href="Pilot-chap.html#fig:behavioural-schema">3.1</a></strong>.</p>

<div class="figure">
<span style="display:block;" id="fig:behavioural-schema"></span>
<img src="figs/pilot/experimental_schema.png" alt="Schema for experimental plan. iCab-iCab pairings are the control condition. To explore direct genetic effects on behaviour, we compare the behaviours of test fishes from different lines, and infer that the differences between lines are caused by the differences in their genetics (direct genetic effects). To explore social genetic effects, we use the same data but turn our focus to the reference fish, and infer that the differences we observe between their behaviours are caused by the differences in their social environments, which are in turn driven by the different genetics in the test fish lines (social genetic effects)." width="100%"><p class="caption">
Figure 3.1: Schema for experimental plan. <em>iCab</em>-<em>iCab</em> pairings are the control condition. To explore direct genetic effects on behaviour, we compare the behaviours of test fishes from different lines, and infer that the differences between lines are caused by the differences in their genetics (direct genetic effects). To explore social genetic effects, we use the same data but turn our focus to the reference fish, and infer that the differences we observe between their behaviours are caused by the differences in their social environments, which are in turn driven by the different genetics in the test fish lines (social genetic effects).
</p>
</div>
<p>[Wrap up the findings in one or two sentences:
We find inter-strain differences that are statistically significant
Social genetic effects that influence behaviour of a strain depending on the partner (or genetic configuration of the test pair, not sure whether this sounds ok)]</p>
<p>To detect</p>
<div id="results" class="section level2" number="3.1">
<h2>
<span class="header-section-number">3.1</span> Results<a class="anchor" aria-label="anchor" href="#results"><i class="fas fa-link"></i></a>
</h2>
<div id="pilot-data-collection" class="section level3" number="3.1.1">
<h3>
<span class="header-section-number">3.1.1</span> Data collection<a class="anchor" aria-label="anchor" href="#pilot-data-collection"><i class="fas fa-link"></i></a>
</h3>
<p>Our behavioural assay is 20 minutes long, comprising two consecutively-run 10-minute components: a) an ‘open field’ component, where the fishes are introduced to the test tank and left to swim around freely; and b) a ‘novel object’ component, where a small black plastic cylinder is added to the tank at the beginning of the second 10-minute period, after which the fishes are again left to swim around freely. The assay is run on pairs of fish. Medaka is a seasonal breeder in which photoperiod has a strong effect on physiology and behaviour.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Jose Fernando L’opez-Olmeda et al., &lt;span&gt;“Long Photoperiod Impairs Learning in Male but Not Female Medaka,”&lt;/span&gt; &lt;em&gt;iScience&lt;/em&gt; 24, no. 7 (July 23, 2021): 102784, &lt;a href="https://doi.org/10.1016/j.isci.2021.102784" role="doc-biblioref"&gt;https://doi.org/10.1016/j.isci.2021.102784&lt;/a&gt;.&lt;/p&gt;'><sup>239</sup></a></span> In this study we tested fish that were acclimated to summer conditions. To avoid confounding mating behaviours between males and females and associated aggressive interactions between males we used only female fish in all experiments. To increase the throughput of the assay, the test tank was divided into four quadrants with barriers, allowing us to run the assay on four pairs of fish simultaneously. Two test tanks situated side-by-side were used, allowing to run 8 concurrent assays. The experimental setup used is shown in Figure <a href="Pilot-chap.html#fig:assay-setup">3.2</a>.</p>

<div class="figure">
<span style="display:block;" id="fig:assay-setup"></span>
<img src="figs/pilot/setup_pic_with_frame_110.png" alt="A: Experimental setup, with two test boxes side-by-side (denoted as “L” for left and “R” for right). Each test box contains one test tank, separated by removable barriers into quadrants, allowing for the simultaneous assay of four pairs of fish per test tank. The interior of the box is ambiently illuminated by LED lights, and a camera is suspended over the centre of each test tank to record the videos. B: Four pairs of fishes in a test tank with labelled quadrants (I, II, III, IV) and strains (iCab or HdrR). C: Paths of iCab reference fish and HdrR test fish from the video at panel (B) at 110 seconds." width="100%"><p class="caption">
Figure 3.2: <strong>A</strong>: Experimental setup, with two test boxes side-by-side (denoted as “L” for left and “R” for right). Each test box contains one test tank, separated by removable barriers into quadrants, allowing for the simultaneous assay of four pairs of fish per test tank. The interior of the box is ambiently illuminated by LED lights, and a camera is suspended over the centre of each test tank to record the videos. <strong>B</strong>: Four pairs of fishes in a test tank with labelled quadrants (I, II, III, IV) and strains (<em>iCab</em> or <em>HdrR</em>). C: Paths of <em>iCab</em> reference fish and <em>HdrR</em> test fish from the video at panel (B) at 110 seconds.
</p>
</div>
<p>Between 11 and 16 June 2019, I assayed a total of 307 pairs of fish, comprising the following counts for each strain pairing: 68 <em>iCab/iCab</em>, 60 <em>iCab/HdrR</em>, 76 <em>iCab/HNI</em>, 47 <em>iCab/Kaga</em>, and 56 <em>iCab/HO5</em>. The locations of the lines’ originating populations are shown in <strong>Figure <a href="Introduction.html#fig:line-locations">1.19</a></strong>. The fish from the iCab strain was denoted as the “reference fish”, and was introduced to the test tank first. The “test fish” which was either another iCab fish (for the control condition), or a fish from one of the other four strains that were assayed in this experiment (<em>HdrR</em>, <em>HNI</em>, <em>Kaga</em>, <em>HO5</em>). The order in which the strains were assayed across the six days was randomly determined prior to the collection of the data. The test tanks were also rinsed between runs to remove any substances released by subjects during previous runs that could influence the behaviour of the subjects that followed.</p>
<p>Image adapted from,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Spivakov et al., &lt;span&gt;“Genomic and &lt;span&gt;Phenotypic Characterization&lt;/span&gt; of a &lt;span&gt;Wild Medaka Population&lt;/span&gt;.”&lt;/span&gt;&lt;/p&gt;"><sup>172</sup></a></span> showing the locations of the originating populations of the 5 inbred medaka lines used in this study. Image adapted from,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Spivakov et al., &lt;span&gt;“Genomic and &lt;span&gt;Phenotypic Characterization&lt;/span&gt; of a &lt;span&gt;Wild Medaka Population&lt;/span&gt;.”&lt;/span&gt;&lt;/p&gt;"><sup>240</sup></a></span> showing the locations of the originating populations of the 5 inbred medaka lines used in this study.</p>
<div class="figure">
<span style="display:block;" id="fig:line-locations"></span>
<img src="figs/pilot/line_locations.png" alt="Image adapted from,172 showing the locations of the originating populations of the 5 inbred medaka lines used in this study." width="100%"><p class="caption">
Figure 1.19: Image adapted from,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Spivakov et al., &lt;span&gt;“Genomic and &lt;span&gt;Phenotypic Characterization&lt;/span&gt; of a &lt;span&gt;Wild Medaka Population&lt;/span&gt;.”&lt;/span&gt;&lt;/p&gt;"><sup>172</sup></a></span> showing the locations of the originating populations of the 5 inbred medaka lines used in this study.
</p>
</div>
</div>
<div id="tracking" class="section level3" number="3.1.2">
<h3>
<span class="header-section-number">3.1.2</span> Tracking<a class="anchor" aria-label="anchor" href="#tracking"><i class="fas fa-link"></i></a>
</h3>
<p>We split the videos by quadrant and assay component, and tracked the movement of the fishes with the open-access software package <em>idtrackerai</em>.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Francisco Romero-Ferrero et al., &lt;span&gt;“Idtracker.ai: Tracking All Individuals in Small or Large Collectives of Unmarked Animals,”&lt;/span&gt; &lt;em&gt;Nature Methods&lt;/em&gt; 16, no. 2, 2 (February 2019): 179–82, &lt;a href="https://doi.org/10.1038/s41592-018-0295-5" role="doc-biblioref"&gt;https://doi.org/10.1038/s41592-018-0295-5&lt;/a&gt;.&lt;/p&gt;'><sup>241</sup></a></span> Each individual fish was tracked across at least 85% of frames in each video, with 78% of videos tracked for over 99% of frames. The failure to track both fishes across all frames was usually caused by instances where a fish was situated behind the novel object or quadrant divider, rendering it invisible to the camera. A random selection of 20 videos was reviewed to search for instances of mislabeled fishes due to software errors. We found none, which is consistent with the reliability of the software as reported by its authors. We therefore concluded that such instances would be absent or very rare and thus negligible.</p>
<p>To identify which fish was the reference or test fish, we could generally distinguish between the strains based on their colour, as <em>iCab</em> pigmentation is light-brown, <em>HO5</em> are light-brown with an orange hue, <em>HdrR</em> females are white, and <em>HNI</em> and <em>Kaga</em> are grey. Where the reference fish was indistinguishable in colour from the test fish, we identified the fish that was introduced to the test tank first, and followed it by eye through to the first frame of the assay. In cases of <em>iCab</em>-<em>iCab</em> pairings, we randomly assigned the individuals to either the “reference” or “test” fish.</p>
<p>We then used the coordinates of the fishes in each frame to calculate the distance and angle travelled by each fish between a fixed time interval, for example every 0.05 seconds. Distance was calculated between points B → C, and angle was calculated between points A → B → C. We used these distance and angle variables to train a HMM with the <em>hmmlearn</em> Python package<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;&lt;em&gt;Hmmlearn/Hmmlearn&lt;/em&gt; (2014; repr., &lt;span&gt;hmmlearn&lt;/span&gt;, 2022), &lt;a href="https://github.com/hmmlearn/hmmlearn" role="doc-biblioref"&gt;https://github.com/hmmlearn/hmmlearn&lt;/a&gt;.&lt;/p&gt;'><sup>242</sup></a></span> in order to classify the movements as discrete behaviours or “modes of movement”.</p>
</div>
<div id="effect-of-covariates" class="section level3" number="3.1.3">
<h3>
<span class="header-section-number">3.1.3</span> Effect of covariates<a class="anchor" aria-label="anchor" href="#effect-of-covariates"><i class="fas fa-link"></i></a>
</h3>
<p>We examined the effects of several covariates, including date of assay, time of assay, tank quadrant, and tank side. To achieve this, we calculated the mean speed of individuals in <em>iCab</em>-<em>iCab</em> pairings (<span class="math inline">\(N\)</span> = 136) over the course of the entire 20-minute video (including both open field and novel object assays), and ran a multi-way ANOVA with all covariates (<strong>Figure <a href="Pilot-chap.html#fig:covariate-effects">3.3</a></strong>). We found significant differences for date of assay and and tank quadrant (p = 0.0139 and 0.0108), but not for time of assay or tank side. This may have been driven by a difference in the way the assay was performed on the first day of the experiment (11 June 2019), where we used a thick fabric sheet to cover the front of the box rather than the wooden doors shown in <strong>Figure <a href="Pilot-chap.html#fig:assay-setup">3.2</a></strong>, as they were only installed the following day. The greater level of external light and sound permeating through the fabric may have caused the fishes to exhibit slower movement on that first day, and when the data from that day is excluded, the p-values for date of assay and tank quadrant increase to 0.0477 and 0.0469 respectively.</p>

<div class="figure">
<span style="display:block;" id="fig:covariate-effects"></span>
<img src="figs/pilot/covariate_effects_no-split-by-assay.png" alt="Effect of covariates on mean speed of iCab individuals when paired with another iCab over the course of the full video (including both open field and novel object assays). P-values were calculated from a multi-way ANOVA with all four covariates included as terms, and adjusted for false discovery rate." width="100%"><p class="caption">
Figure 3.3: Effect of covariates on mean speed of iCab individuals when paired with another iCab over the course of the full video (including both open field and novel object assays). <em>P</em>-values were calculated from a multi-way ANOVA with all four covariates included as terms, and adjusted for false discovery rate.
</p>
</div>
</div>
<div id="pilot-hmm-params" class="section level3" number="3.1.4">
<h3>
<span class="header-section-number">3.1.4</span> Choice of time interval and number of HMM states<a class="anchor" aria-label="anchor" href="#pilot-hmm-params"><i class="fas fa-link"></i></a>
</h3>
<p>HMMs are well-suited for classifying the hidden states that generate stochastic, sequential observations. To determine the optimal parameters for the HMM’s classification of behaviours, we sought to reduce overfitting (which tends to favour using a lower number of HMM states) while maximising the ability to distinguish between strains based on the relative time they spent in each HMM state (which tends to favour a higher number of HMM states). We additionally considered the time interval between which the distance and angle of travel was measured.</p>
<p><strong>Figure <a href="Pilot-chap.html#fig:compare-params">3.4</a></strong> sets out the comparison of HMM parameters on two measures designed to quantify, respectively, the level of overfitting (‘mean concordance between cross-validated HMM states’), and the quantification of differences between strains (’summed Kruskal-Wallis statistic comparing frequency of time spent in each HMM state across medaka strains) (Methods).</p>

<div class="figure">
<span style="display:block;" id="fig:compare-params"></span>
<img src="figs/pilot/compare_params.png" alt="Comparison between HMM parameters. Horizontal axis: Mean concordance between states assigned by HMMs through a 2-fold cross-validation process. Vertical axis: Kruskal-Wallis statistic comparing strains based on the proportion of time spent in each HMM state, summed across all states. Size of points correspond to the interval, in seconds, between which the distance and angle of travel was calculated (Methods)." width="100%"><p class="caption">
Figure 3.4: Comparison between HMM parameters. Horizontal axis: Mean concordance between states assigned by HMMs through a 2-fold cross-validation process. Vertical axis: Kruskal-Wallis statistic comparing strains based on the proportion of time spent in each HMM state, summed across all states. Size of points correspond to the interval, in seconds, between which the distance and angle of travel was calculated (Methods).
</p>
</div>
<p>Based on these results and a visualisation of the polar plots for each combination of state number and time interval, we excluded combinations with 15 or more states due to an asymmetry across states that would create difficulties for interpreting their biological meaning (Supplementary material). For the downstream analysis we selected the combination of 14 states with a 0.08-second interval between time points, because out of the remaining combinations it appeared to optimally balance the level of overfitting and detection of differences between strains.</p>
<p>The distances and angles of travel for the 14 states are shown in Figure 4, generated from 10,000 randomly-selected data points (i.e. movements). Each point represents the radial distance in log10(pixels) that a fish travelled from its previous location (B → C), and the polar angle travelled from A → B → C, where A → B is always aligned vertically along the 0-180° axis. For example, a point at 45° far from the pole represents a fast, forward movement to the right, and a point at 260° close to the pole represents a slow, backward movement to the left.</p>

<div class="figure">
<span style="display:block;" id="fig:pilot-polar"></span>
<img src="figs/pilot/polar_all_dge.png" alt="Classification of movements by the 14-state HMM, based on distance (in log10(pixels)) and angle of travel between a time interval of 0.08 seconds from points B → C for distance, and points A → B → C for angle. Each point represents the distance and angle at point C, and A → B is aligned vertically along the 0-180° radial axis. The figure shows an illustrative 10,000 data points (movements), randomly selected from the full dataset. States are sorted in ascending order by mean distance." width="100%"><p class="caption">
Figure 3.5: Classification of movements by the 14-state HMM, based on distance (in log<sub>10</sub>(pixels)) and angle of travel between a time interval of 0.08 seconds from points B → C for distance, and points A → B → C for angle. Each point represents the distance and angle at point C, and A → B is aligned vertically along the 0-180° radial axis. The figure shows an illustrative 10,000 data points (movements), randomly selected from the full dataset. States are sorted in ascending order by mean distance.
</p>
</div>
<p>We next sought to determine whether there was a difference in the proportion of time spent in each state between the 5 inbred strains (implying the presence of direct genetic effects), and between the <em>iCab</em> reference fishes depending on the inbred strains of their tank partner (implying the presence of social genetic effects).</p>
</div>
<div id="direct-and-social-genetic-effects-on-boldshy-behaviour" class="section level3" number="3.1.5">
<h3>
<span class="header-section-number">3.1.5</span> Direct and social genetic effects on bold/shy behaviour<a class="anchor" aria-label="anchor" href="#direct-and-social-genetic-effects-on-boldshy-behaviour"><i class="fas fa-link"></i></a>
</h3>
<div id="direct-genetic-effects" class="section level4" number="3.1.5.1">
<h4>
<span class="header-section-number">3.1.5.1</span> Direct genetic effects<a class="anchor" aria-label="anchor" href="#direct-genetic-effects"><i class="fas fa-link"></i></a>
</h4>
<p>To determine whether the test fish strains differed in the proportions of time they spent in each state, we ran separate multi-way ANOVAs for each combination of assay component (open field or novel object) and state (states 1 to 14):</p>
<p><span class="math display">\[
frequency_{assay component,state} = \beta_{1}(test fish strain) + \beta_{2}(date) + \beta_{3}(time) + \beta_{4}(quadrant) + \beta_{5}(tank side)
\]</span></p>
<p>The proportion of time each individual spent within a state was first inverse-normalised within each combination of assay and state, and the date of assay, time of assay, tank quadrant, and tank side were included as covariates. P-values were then adjusted for the False Discovery Rate (<strong>FDR</strong>).</p>
<p>The states that showed a significant difference are set out in <strong>Table <a href="Pilot-chap.html#tab:pilot-dge-tbl">3.1</a></strong>. The test fish strains differed significantly in the proportion of time spent in a given state (p &lt; 0.05, FDR-adjusted) for 11 out of 14 states in the open field assay (4.45x10<sup>-24</sup> &lt; p &lt; 1.06x10-2), and 7 out of 14 states for the novel object assay (5.53x10<sup>-16</sup> &lt; p &lt; 4.2x10-3), with the strain of the test fish explaining up to ~28% of the variance in the proportion of time spent in a given state. For some states, there was also a significant difference between quadrants, date of assay, and tank side (open field: 1.03e-09 &lt; p &lt; 1.18x10-2; novel object: 1.75x10-8 &lt; p &lt; 4.9x10-2). Full tables for all states and variables are provided in the Supplementary Material.</p>
<div class="inline-table"><table class="table table-sm">
<caption>
<span id="tab:pilot-dge-tbl">Table 3.1: </span>Significant differences in the proportion of time spent in each HMM state across test fish strains for the open field and novel object assay components.</caption>
<thead><tr class="header">
<th align="left">Assay</th>
<th align="left">State</th>
<th align="left">Variance explained (%)</th>
<th align="left">p-value (FDR-adjusted)</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">open field</td>
<td align="left">1</td>
<td align="left">26.62</td>
<td align="left">3.60e-22</td>
</tr>
<tr class="even">
<td align="left">open field</td>
<td align="left">2</td>
<td align="left">21.94</td>
<td align="left">6.18e-18</td>
</tr>
<tr class="odd">
<td align="left">open field</td>
<td align="left">3</td>
<td align="left">28.21</td>
<td align="left">4.45e-24</td>
</tr>
<tr class="even">
<td align="left">open field</td>
<td align="left">4</td>
<td align="left">17.73</td>
<td align="left">2.12e-15</td>
</tr>
<tr class="odd">
<td align="left">open field</td>
<td align="left">5</td>
<td align="left">7.48</td>
<td align="left">2.78e-05</td>
</tr>
<tr class="even">
<td align="left">open field</td>
<td align="left">6</td>
<td align="left">5.60</td>
<td align="left">2.48e-04</td>
</tr>
<tr class="odd">
<td align="left">open field</td>
<td align="left">7</td>
<td align="left">5.46</td>
<td align="left">3.11e-04</td>
</tr>
<tr class="even">
<td align="left">open field</td>
<td align="left">8</td>
<td align="left">5.79</td>
<td align="left">1.08e-04</td>
</tr>
<tr class="odd">
<td align="left">open field</td>
<td align="left">9</td>
<td align="left">7.36</td>
<td align="left">8.96e-06</td>
</tr>
<tr class="even">
<td align="left">open field</td>
<td align="left">10</td>
<td align="left">3.78</td>
<td align="left">1.06e-02</td>
</tr>
<tr class="odd">
<td align="left">open field</td>
<td align="left">13</td>
<td align="left">4.66</td>
<td align="left">3.42e-03</td>
</tr>
<tr class="even">
<td align="left">novel object</td>
<td align="left">1</td>
<td align="left">17.19</td>
<td align="left">7.28e-14</td>
</tr>
<tr class="odd">
<td align="left">novel object</td>
<td align="left">2</td>
<td align="left">14.28</td>
<td align="left">2.92e-11</td>
</tr>
<tr class="even">
<td align="left">novel object</td>
<td align="left">3</td>
<td align="left">19.30</td>
<td align="left">5.53e-16</td>
</tr>
<tr class="odd">
<td align="left">novel object</td>
<td align="left">4</td>
<td align="left">13.12</td>
<td align="left">1.26e-10</td>
</tr>
<tr class="even">
<td align="left">novel object</td>
<td align="left">6</td>
<td align="left">4.12</td>
<td align="left">4.20e-03</td>
</tr>
<tr class="odd">
<td align="left">novel object</td>
<td align="left">7</td>
<td align="left">8.93</td>
<td align="left">1.84e-06</td>
</tr>
<tr class="even">
<td align="left">novel object</td>
<td align="left">9</td>
<td align="left">10.68</td>
<td align="left">9.87e-08</td>
</tr>
</tbody>
</table></div>
<p><strong>Figure <a href="Pilot-chap.html#fig:pilot-dge-time">3.6</a></strong> depicts the time dependence of HMM states over the course of the video, and the regions of the test tank that were most frequently occupied by the different strains. Figures A and H show the HMM states with panels coloured red to indicate the states that were found to significantly differ between test fish strains. <strong>Figures <a href="Pilot-chap.html#fig:pilot-dge-time">3.6</a>B</strong> and <strong>E</strong> show how each individual test fish, grouped by inbred strain, moved through HMM states throughout the course of the video, with each tile coloured by the state most frequently-occupied by the individual within 2-second intervals. Figures <strong><a href="Pilot-chap.html#fig:pilot-dge-time">3.6</a>C</strong> and <strong>F</strong> show the same data, recalculated as densities within each strain, with only the states that showed significant differences between strains in colour, and the remaining states consolidated and coloured grey. There is an especially clear difference between strains in the proportions of time spent in the slowest-moving states (i.e. states 1 to 3) at the beginning of each assay component, with an increase across all strains in the novel object assay, likely as a consequence of having less room to move, as well as the fear response that the novel object was designed to elicit. The differences are clearest when comparing the southern Japanese medaka strains (<em>iCab</em>, <em>HdrR</em>, and <em>HO5</em>) against the northern Japanese strains (<em>Kaga</em> and <em>HNI</em>). In addition, we note that <em>Kaga</em> tends to spend more time in the fast- and forward-moving state 13 at the beginning of the assay than at other times, which suggests that for that strain, such movements may be the manifestation of a stress response. <strong>Figures <a href="Pilot-chap.html#fig:pilot-dge-time">3.6</a>D</strong> and <strong>G</strong> show, as densities, the regions of the tank that were most frequently occupied by each strain. Although the northern Japanese strains <em>Kaga</em> and <em>HNI</em> were similarly fast-moving relative to the southern Japanese strains, in the open field assay component they appear to favour different regions of the tank – where <em>HNI</em> occupied the central regions of the tank with more frequency, <em>Kaga</em> tended to prefer swimming along the boundaries of the tank.</p>

<div class="figure">
<span style="display:block;" id="fig:pilot-dge-time"></span>
<img src="figs/pilot/0.08_14_dge.png" alt="Differences between test fish strains in the HMM states they occupied during the open field (top) and novel object (bottom) assay components. A and H: 14 HMM states with panels coloured red to indicate significant differences between test fish strains in the proportion of time spent in those states during the separate assay components. B and E: Transitions between HMM states across time for each individual test fish, grouped by strain. Tiles are coloured by the state most frequently occupied by each fish within 2-second intervals. C and F: Densities within each strain for the occupation of states that significantly differed between strains (colour), with other states consolidated (grey). D and G: Densities of the test tank locations occupied by each strain, calculated within 900 grid points (30x30)." width="100%"><p class="caption">
Figure 3.6: Differences between test fish strains in the HMM states they occupied during the open field (top) and novel object (bottom) assay components. <strong>A and H</strong>: 14 HMM states with panels coloured red to indicate significant differences between test fish strains in the proportion of time spent in those states during the separate assay components. <strong>B and E</strong>: Transitions between HMM states across time for each individual test fish, grouped by strain. Tiles are coloured by the state most frequently occupied by each fish within 2-second intervals. <strong>C and F</strong>: Densities within each strain for the occupation of states that significantly differed between strains (colour), with other states consolidated (grey). <strong>D and G</strong>: Densities of the test tank locations occupied by each strain, calculated within 900 grid points (30x30).
</p>
</div>
</div>
<div id="pilot-sge" class="section level4" number="3.1.5.2">
<h4>
<span class="header-section-number">3.1.5.2</span> Social genetic effects<a class="anchor" aria-label="anchor" href="#pilot-sge"><i class="fas fa-link"></i></a>
</h4>
<p>To determine whether the <em>iCab</em> reference fishes altered their behaviour depending on the inbred strain of their tank partner, we carried out the same analysis and model as above using only data from the <em>iCab</em> reference fishes. The states that showed a significant difference are set out in <strong>Table <a href="Pilot-chap.html#tab:pilot-sge-tbl">3.2</a></strong>. The <em>iCab</em> reference fishes differed significantly in the proportion of time they spent in a given state depending on the strain of their tank partner (<span class="math inline">\(p\)</span> &lt; 0.05, FDR-adjusted) for 5 out of 14 states in the open field assay (2.04x10<sup>-6</sup> &lt; <span class="math inline">\(p\)</span> &lt; 2.11x10<sup>-2</sup>), and 7 out of 14 states for the novel object assay (4.76x10<sup>-7</sup> &lt; <span class="math inline">\(p\)</span> &lt; 4.06x10<sup>-2</sup>). The strain of the tank partner explained up to ~9% of the variance in the proportion of time the <em>iCab</em> reference spent in a given state. Full tables for all states and variables are provided in the Supplementary Material.</p>
<div class="inline-table"><table class="table table-sm">
<caption>
<span id="tab:pilot-sge-tbl">Table 3.2: </span>Significant differences in the proportion of time spent in each HMM state by iCab reference fishes depending on the strain of their tank partner during the open field and novel object assay components.</caption>
<thead><tr class="header">
<th align="left">Assay</th>
<th align="left">State</th>
<th align="left">Variance explained (%)</th>
<th align="left">p-value (FDR-adjusted)</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">open field</td>
<td align="left">1</td>
<td align="left">8.52</td>
<td align="left">5.88e-06</td>
</tr>
<tr class="even">
<td align="left">open field</td>
<td align="left">2</td>
<td align="left">9.09</td>
<td align="left">2.04e-06</td>
</tr>
<tr class="odd">
<td align="left">open field</td>
<td align="left">3</td>
<td align="left">7.97</td>
<td align="left">1.77e-05</td>
</tr>
<tr class="even">
<td align="left">open field</td>
<td align="left">5</td>
<td align="left">3.92</td>
<td align="left">4.29e-03</td>
</tr>
<tr class="odd">
<td align="left">open field</td>
<td align="left">12</td>
<td align="left">2.97</td>
<td align="left">2.11e-02</td>
</tr>
<tr class="even">
<td align="left">novel object</td>
<td align="left">1</td>
<td align="left">7.94</td>
<td align="left">8.73e-06</td>
</tr>
<tr class="odd">
<td align="left">novel object</td>
<td align="left">2</td>
<td align="left">9.38</td>
<td align="left">4.76e-07</td>
</tr>
<tr class="even">
<td align="left">novel object</td>
<td align="left">3</td>
<td align="left">8.73</td>
<td align="left">1.58e-06</td>
</tr>
<tr class="odd">
<td align="left">novel object</td>
<td align="left">6</td>
<td align="left">4.76</td>
<td align="left">1.23e-03</td>
</tr>
<tr class="even">
<td align="left">novel object</td>
<td align="left">8</td>
<td align="left">2.80</td>
<td align="left">4.06e-02</td>
</tr>
<tr class="odd">
<td align="left">novel object</td>
<td align="left">10</td>
<td align="left">3.81</td>
<td align="left">8.43e-03</td>
</tr>
</tbody>
</table></div>
<p>We observe that the proportions of time the <em>iCab</em> reference fishes spent in different states do differ based on the strains of their tank partner, and that their behavioural patterns tend to reflect those of the test fish strains they are paired with (<strong>Figure <a href="Pilot-chap.html#fig:pilot-sge-time">3.7</a></strong>). Thus, the <em>iCab</em> reference fishes spend less time in the slower-moving states when in the presence of the faster-moving northern Japanese strains <em>Kaga</em> and <em>HNI</em>, which in turn shows that the test fishes are transmitting their behaviours to some degree to their <em>iCab</em> tank partners.The differences in the behaviours of <em>iCab</em> reference fishes based on the strain of their tank partners are not as large as those between the test fish strains. However, unlike what was observed with the test fishes, here there is a greater number of states that show significant differences during the novel object component of the assay compared to the open field component. This suggests that when movement is restricted, or when in the presence of a potential threat, the behaviour of the test fish tank partner has more of an influence on the <em>iCab</em> reference fish’s behaviour than otherwise.</p>

<div class="figure">
<span style="display:block;" id="fig:pilot-sge-time"></span>
<img src="figs/pilot/0.08_14_sge.png" alt="Differences between HMM states occupied by the reference fish when paired with different test fish strains during the open field (top) and novel object (bottom) assay components. A and H: 14 HMM states with panels coloured red to indicate significant differences between the reference fishes under different strain-pairings in the proportion of time spent in those states during the separate assay components. The HMM states are the same as those in Figure 3.6, but coloured with a different palette. B and E: Transitions between HMM states across time for each individual iCab reference fish, grouped by the strain of its tank partner. Tiles are coloured by the state most frequently occupied by each fish within 2-second intervals. C and F: Densities within each strain-pairing for the occupation of states that significantly differed between strain-pairings (colour), with other states consolidated (grey). D and G: Densities of the test tank locations occupied by the iCab reference fishes when paired with different strain, calculated within 900 grid points (30x30)." width="100%"><p class="caption">
Figure 3.7: Differences between HMM states occupied by the reference fish when paired with different test fish strains during the open field (top) and novel object (bottom) assay components. <strong>A and H</strong>: 14 HMM states with panels coloured red to indicate significant differences between the reference fishes under different strain-pairings in the proportion of time spent in those states during the separate assay components. The HMM states are the same as those in <strong>Figure <a href="Pilot-chap.html#fig:pilot-dge-time">3.6</a></strong>, but coloured with a different palette. <strong>B and E</strong>: Transitions between HMM states across time for each individual <em>iCab</em> reference fish, grouped by the strain of its tank partner. Tiles are coloured by the state most frequently occupied by each fish within 2-second intervals. <strong>C and F</strong>: Densities within each strain-pairing for the occupation of states that significantly differed between strain-pairings (colour), with other states consolidated (grey). <strong>D and G</strong>: Densities of the test tank locations occupied by the iCab reference fishes when paired with different strain, calculated within 900 grid points (30x30).
</p>
</div>
<p>Finally, to develop a metric to quantify the degree to which the <em>iCab</em> reference fishes’ behaviour is influenced by the strain of its tank partner, we calculated the proportions of time that the pairs of fish spent simultaneously occupying the same HMM state. <strong>Figure <a href="Pilot-chap.html#fig:pilot-cooc-heat">3.8</a></strong> shows that when paired with the slowest-moving <em>HdrR</em> strain, the <em>iCab</em> reference fishes spent more time co-occupying the slow- and forward-moving state 2, whereas they tended to co-occupy the fast- and forward-moving state 11 when paired with the faster-moving northern <em>Kaga</em> and <em>HNI</em> strains. When paired with another <em>iCab</em>, they preferred to co-occupy the slow-moving and pan-directional state 4. For each combination of assay component and state, we then ran a Kruskal-Wallis test to determine whether there were differences in the frequencies of state co-occupancy under different strain pairings (<span class="math inline">\(p\)</span> &lt; 0.05, FDR-adjusted), and found significant differences for states 1 to 6 for the open field component (1.1x10<sup>-8</sup> &lt; <span class="math inline">\(p\)</span> &lt; 4.1x10<sup>-2</sup>), and states 1 to 4 and 7 for the novel object component (1.6x10<sup>-6</sup> &lt; <span class="math inline">\(p\)</span> &lt; 2.5x10<sup>-2</sup>). The heatmaps show that <em>iCab</em> tends to co-occupy state 4 at the highest frequency when paired with another <em>iCab</em>, which suggests that this pan-directional, slow-moving state is a more comfortable or natural state for <em>iCab</em> to occupy, particularly when under stress.</p>

<div class="figure">
<span style="display:block;" id="fig:pilot-cooc-heat"></span>
<img src="figs/pilot/0.08_14_cooc_heatmap.png" alt="Frequency of HMM state co-occupancy between pairs of fish, calculated across all all videos per strain-pairing." width="100%"><p class="caption">
Figure 3.8: Frequency of HMM state co-occupancy between pairs of fish, calculated across all all videos per strain-pairing.
</p>
</div>
</div>
</div>
</div>
<div id="discussion" class="section level2" number="3.2">
<h2>
<span class="header-section-number">3.2</span> Discussion<a class="anchor" aria-label="anchor" href="#discussion"><i class="fas fa-link"></i></a>
</h2>
<p>In this chapter have described an assay for measuring bold-shy behaviours in medaka fish that can reliably detect differences: (a) between individuals from different inbred strains – allowing for the quantification of direct genetic effects on behaviour; and (b) between the behaviour of individuals when paired with tank partners from a different strain – allowing for the quantification of social genetic effects on behaviour.</p>
<p>I found that behaviours exhibited during the assay show minimal variance across the covariates of date of assay, time of assay, tank quadrant, and tank side. This creates confidence that the genetic effects that we observed – both direct and indirect – are not unduly affected by these variables.</p>
<p>I have shown that a Hidden Markov Model (HMM) allows an adequate classification of movements based on the direction and angle of travel within set time intervals. The HMM method has previously been used to classify maternal behaviour patterns in mice,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Valeria Carola, Olivier Mirabeau, and Cornelius T. Gross, &lt;span&gt;“Hidden &lt;span&gt;Markov Model Analysis&lt;/span&gt; of &lt;span&gt;Maternal Behavior Patterns&lt;/span&gt; in &lt;span&gt;Inbred&lt;/span&gt; and &lt;span&gt;Reciprocal Hybrid Mice&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;PLOS ONE&lt;/em&gt; 6, no. 3 (March 8, 2011): e14753, &lt;a href="https://doi.org/10.1371/journal.pone.0014753" role="doc-biblioref"&gt;https://doi.org/10.1371/journal.pone.0014753&lt;/a&gt;.&lt;/p&gt;'><sup>243</sup></a></span> and I have extended the application here to bold-shy behaviours in medaka fish. In principle, this method can be expanded to include additional behavioural features such as proximity to the wall or other objects, inwards/outwards orientation, proximity to the tank partner, and other metrics related to leader-follower dynamics, a possibility that will be tested in future studies.</p>
<p>The HMM classification of behaviours also facilitated the ranking of behaviours by speed and angle of movement, which may be further interpreted in terms of biology and adaptation styles. However, even though the HMM classifies the slowest-moving states as separate modes of movement, one should be cautious when attributing biological meaning to their differences, because even when a fish is almost completely still, the fish object’s centroid (upon which the variables of distance and angle of travel are calculated) will tend to move by one or several pixels through minor changes in the segmentation of the object, and are thus subject to greater noise than the faster-moving states.</p>
<p>I have found that inbred strains of medaka fish can be distinguished by the proportion of time they spend in certain states. The slowest states, capturing no or minimal movement, were the states that most clearly separated the strains, and these differences appeared most clearly at the beginning of the open field and novel object assays. This suggests that for southern Japanese strains such as <em>iCab</em>, <em>HdrR</em>, and <em>HO5</em>, the “freeze” reflex is caused by anxiety, which eventually dissipates over time indicating habituation. Similar observations showing initial anxiety as shown by freezing behaviour and subsequent habituation have been made in other studies using open field assays with <em>iCab</em> fish.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Lucon-Xiccato et al., &lt;span&gt;“Comparison of Anxiety-Like and Social Behaviour in Medaka and Zebrafish.”&lt;/span&gt;&lt;/p&gt;"><sup>244</sup></a></span></p>
<p>On the other hand, the northern Japanese strains <em>Kaga</em> and <em>HNI</em> spent little time in the slow-moving states at the beginning of the video, which indicates either that their habituation sets in earlier, or that their stress and anxiety is expressed to a lesser degree as freezing behaviour. The former appears to be more likely for <em>Kaga</em>, as in the open field assay, it spends more time in the faster-moving states at the beginning of the video and then slows down thereafter, which suggests that its higher level of movement may be induced by stress. It is interesting then that once the novel object is introduced, <em>Kaga</em> tends to move slowly like the other strains. Obviously introduction to a novel environment and exposure to a potentially dangerous object elicits different behavioural patterns in this strain. Taken together with the overt thigmotaxis behaviour (i.e. moving along the sides of the tank during the open field assay component) relative to other strains, its behaviour may suggest an “escape-seeking” response. It will be interesting to carry out further experiments to determine under which fear-inducing conditions <em>Kaga</em> reacts with this more frantic style of movement rather than the freeze response that it displays in the novel object component in common with the other strains, and to explore its physiological basis.</p>
<p>With respect to social genetic effects, we have found that a fish’s behaviour is affected by the differential behaviour of its tank partner, although the effect is less powerful than the direct genetic effect on a fish’s own biology. These social genetic effects are detectable when observing the proportions of time the reference fishes spend in certain states over the course of the video, and can be quantified by measuring the frequency of state co-occupancy with their tank partners.</p>
<p>It is also interesting to find that <em>iCab</em> prefers to co-occupy the slow, pan-directional state 4 when paired with another individual from the same strain. As the strains are raised exclusively in the company of individuals from the same strain, that type of behaviour may represent a more comfortable – yet still cautious – mode of movement, or a type of mimicry that tends to occur when in the company of individuals that are more genetically or socially familiar.</p>
<p>In this study we only used one strain as the reference fish, but future experiments can expand on this analysis by using different strains as the reference, thereby exploring how a strain’s genetics influence the degree of their behavioural plasticity, and how the distinct behaviours of strains may interact in intriguing ways.</p>
<p>In summary we show that the OFT experimental setup in combination with the HMM analysis can reliably detect both direct and social genetic effects. This creates the opportunity to carry out a similar study on a larger panel of inbred strains, such as the MIKK panel,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Fitzgerald et al., &lt;span&gt;“The &lt;span&gt;Medaka Inbred Kiyosu-Karlsruhe&lt;/span&gt; (&lt;span&gt;MIKK&lt;/span&gt;) Panel.”&lt;/span&gt;&lt;/p&gt;"><sup>245</sup></a></span> with a view to identifying the genetic variants associated with not only these differences in behaviour, but also the differences in the degree to which an individual transmits their behaviour onto their social companions. This in turn will shed light on longstanding biological questions concerning the direct and indirect influences on behaviour, their physiological bases, and their adaptive purposes. This is the subject of the following chapter.</p>

</div>
</div>

  <div class="chapter-nav">
<div class="prev"><a href="MIKK-genomes-chap.html"><span class="header-section-number">2</span> Genomic variations in the MIKK panel</a></div>
<div class="next"><a href="Somite-chap.html"><span class="header-section-number">4</span> Genetic loci associated with somite development periodicity</a></div>
</div></main><div class="col-md-3 col-lg-2 d-none d-md-block sidebar sidebar-chapter">
    <nav id="toc" data-toggle="toc" aria-label="On this page"><h2>On this page</h2>
      <ul class="nav navbar-nav">
<li><a class="nav-link" href="#Pilot-chap"><span class="header-section-number">3</span> Classification of bold/shy behaviours in 5 inbred medaka lines</a></li>
<li>
<a class="nav-link" href="#results"><span class="header-section-number">3.1</span> Results</a><ul class="nav navbar-nav">
<li><a class="nav-link" href="#pilot-data-collection"><span class="header-section-number">3.1.1</span> Data collection</a></li>
<li><a class="nav-link" href="#tracking"><span class="header-section-number">3.1.2</span> Tracking</a></li>
<li><a class="nav-link" href="#effect-of-covariates"><span class="header-section-number">3.1.3</span> Effect of covariates</a></li>
<li><a class="nav-link" href="#pilot-hmm-params"><span class="header-section-number">3.1.4</span> Choice of time interval and number of HMM states</a></li>
<li><a class="nav-link" href="#direct-and-social-genetic-effects-on-boldshy-behaviour"><span class="header-section-number">3.1.5</span> Direct and social genetic effects on bold/shy behaviour</a></li>
</ul>
</li>
<li><a class="nav-link" href="#discussion"><span class="header-section-number">3.2</span> Discussion</a></li>
</ul>

      <div class="book-extra">
        <ul class="list-unstyled">
<li><a id="book-source" href="https://github.com/brettellebi/PhD-thesis/blob/master/book/03-Pilot.Rmd">View source <i class="fab fa-github"></i></a></li>
          <li><a id="book-edit" href="https://github.com/brettellebi/PhD-thesis/edit/master/book/03-Pilot.Rmd">Edit this page <i class="fab fa-github"></i></a></li>
        </ul>
</div>
    </nav>
</div>

</div>
</div> <!-- .container -->

<footer class="bg-primary text-light mt-5"><div class="container"><div class="row">

  <div class="col-12 col-md-6 mt-3">
    <p>"<strong>Japanese courage: a genetic analysis of complex traits in medaka fish and humans</strong>" was written by Ian Brettell. It was last built on 2022-08-31.</p>
  </div>

  <div class="col-12 col-md-6 mt-3">
    <p>This book was built by the <a class="text-light" href="https://bookdown.org">bookdown</a> R package.</p>
  </div>

</div></div>
</footer><!-- dynamically load mathjax for compatibility with self-contained --><script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.9/latest.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script><script type="text/x-mathjax-config">const popovers = document.querySelectorAll('a.footnote-ref[data-toggle="popover"]');
for (let popover of popovers) {
  const div = document.createElement('div');
  div.setAttribute('style', 'position: absolute; top: 0, left:0; width:0, height:0, overflow: hidden; visibility: hidden;');
  div.innerHTML = popover.getAttribute('data-content');

  var has_math = div.querySelector("span.math");
  if (has_math) {
    document.body.appendChild(div);
    MathJax.Hub.Queue(["Typeset", MathJax.Hub, div]);
    MathJax.Hub.Queue(function() {
      popover.setAttribute('data-content', div.innerHTML);
      document.body.removeChild(div);
    })
  }
}
</script>
</body>
</html>
