<!DOCTYPE html>
<html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<title>References | Genetic analysis of quantitative traits in medaka fish and humans</title>
<meta name="author" content="Ian Brettell">
<meta name="description" content="“A Global Reference for Human Genetic Variation.” 2015. Nature 526 (7571): 68–74. https://doi.org/10.1038/nature15393.  Berg, Jeremy J., and Graham Coop. 2014. “A Population Genetic Signal of...">
<meta name="generator" content="bookdown 0.26 with bs4_book()">
<meta property="og:title" content="References | Genetic analysis of quantitative traits in medaka fish and humans">
<meta property="og:type" content="book">
<meta property="og:description" content="“A Global Reference for Human Genetic Variation.” 2015. Nature 526 (7571): 68–74. https://doi.org/10.1038/nature15393.  Berg, Jeremy J., and Graham Coop. 2014. “A Population Genetic Signal of...">
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="References | Genetic analysis of quantitative traits in medaka fish and humans">
<meta name="twitter:description" content="“A Global Reference for Human Genetic Variation.” 2015. Nature 526 (7571): 68–74. https://doi.org/10.1038/nature15393.  Berg, Jeremy J., and Graham Coop. 2014. “A Population Genetic Signal of...">
<!-- JS --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://kit.fontawesome.com/6ecbd6c532.js" crossorigin="anonymous"></script><script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="libs/bootstrap-4.6.0/bootstrap.min.css" rel="stylesheet">
<script src="libs/bootstrap-4.6.0/bootstrap.bundle.min.js"></script><script src="libs/bs3compat-0.3.1/transition.js"></script><script src="libs/bs3compat-0.3.1/tabs.js"></script><script src="libs/bs3compat-0.3.1/bs3compat.js"></script><link href="libs/bs4_book-1.0.0/bs4_book.css" rel="stylesheet">
<script src="libs/bs4_book-1.0.0/bs4_book.js"></script><style>
    #content {
        font-size: 0.9rem;
    }
    </style>
<script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- CSS --><style type="text/css">
    
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  </style>
<style type="text/css">
    /* Used with Pandoc 2.11+ new --citeproc when CSL is used */
    div.csl-bib-body { }
    div.csl-entry {
      clear: both;
        }
    .hanging div.csl-entry {
      margin-left:2em;
      text-indent:-2em;
    }
    div.csl-left-margin {
      min-width:2em;
      float:left;
    }
    div.csl-right-inline {
      margin-left:2em;
      padding-left:1em;
    }
    div.csl-indent {
      margin-left: 2em;
    }
  </style>
<link rel="stylesheet" href="style.css">
</head>
<body data-spy="scroll" data-target="#toc">

<div class="container-fluid">
<div class="row">
  <header class="col-sm-12 col-lg-3 sidebar sidebar-book"><a class="sr-only sr-only-focusable" href="#content">Skip to main content</a>

    <div class="d-flex align-items-start justify-content-between">
      <h1>
        <a href="index.html" title="">Genetic analysis of quantitative traits in medaka fish and humans</a>
      </h1>
      <button class="btn btn-outline-primary d-lg-none ml-2 mt-1" type="button" data-toggle="collapse" data-target="#main-nav" aria-expanded="true" aria-controls="main-nav"><i class="fas fa-bars"></i><span class="sr-only">Show table of contents</span></button>
    </div>

    <div id="main-nav" class="collapse-lg">
      <form role="search">
        <input id="search" class="form-control" type="search" placeholder="Search" aria-label="Search">
</form>

      <nav aria-label="Table of contents"><h2>Table of contents</h2>
        <ul class="book-toc list-unstyled">
<li><a class="" href="index.html"><span class="header-section-number">1</span> About</a></li>
<li><a class="" href="introduction.html"><span class="header-section-number">2</span> Introduction</a></li>
<li><a class="" href="MIKK-genomes-chap.html"><span class="header-section-number">3</span> Genomic variations in the MIKK panel</a></li>
<li><a class="" href="Pilot-chap.html"><span class="header-section-number">4</span> Classification of bold/shy behaviours in 5 inbred medaka lines</a></li>
<li><a class="" href="MIKK-F2-chap.html"><span class="header-section-number">5</span> Bold/shy behaviours in the MIKK panel</a></li>
<li><a class="" href="Fst-chap.html"><span class="header-section-number">6</span> Variation in the frequency of trait-associated alleles across global human populations</a></li>
<li><a class="active" href="references.html">References</a></li>
</ul>

        <div class="book-extra">
          <p><a id="book-repo" href="https://github.com/brettellebi/PhD-thesis">View book source <i class="fab fa-github"></i></a></p>
        </div>
      </nav>
</div>
  </header><main class="col-sm-12 col-md-9 col-lg-7" id="content"><div id="references" class="section level1 unnumbered">
<h1>References<a class="anchor" aria-label="anchor" href="#references"><i class="fas fa-link"></i></a>
</h1>

<div id="refs" class="references csl-bib-body hanging-indent">
<div id="ref-GlobalReferenceHuman2015" class="csl-entry">
<span>“A Global Reference for Human Genetic Variation.”</span> 2015. <em>Nature</em> 526 (7571): 68–74. <a href="https://doi.org/10.1038/nature15393">https://doi.org/10.1038/nature15393</a>.
</div>
<div id="ref-bergPopulationGeneticSignal2014a" class="csl-entry">
Berg, Jeremy J., and Graham Coop. 2014. <span>“A <span>Population Genetic Signal</span> of <span>Polygenic Adaptation</span>.”</span> <em>PLOS Genetics</em> 10 (8): e1004412. <a href="https://doi.org/10.1371/journal.pgen.1004412">https://doi.org/10.1371/journal.pgen.1004412</a>.
</div>
<div id="ref-bergelsonIdentifyingGenesUnderlying2010" class="csl-entry">
Bergelson, Joy, and Fabrice Roux. 2010. <span>“Towards Identifying Genes Underlying Ecologically Relevant Traits in <span>Arabidopsis</span> Thaliana.”</span> <em>Nature Reviews Genetics</em> 11 (12, 12): 867–79. <a href="https://doi.org/10.1038/nrg2896">https://doi.org/10.1038/nrg2896</a>.
</div>
<div id="ref-changSecondgenerationPLINKRising2015" class="csl-entry">
Chang, Christopher C, Carson C Chow, Laurent CAM Tellier, Shashaank Vattikuti, Shaun M Purcell, and James J Lee. 2015. <span>“Second-Generation <span>PLINK</span>: Rising to the Challenge of Larger and Richer Datasets.”</span> <em>GigaScience</em> 4 (1): 7. <a href="https://doi.org/10.1186/s13742-015-0047-8">https://doi.org/10.1186/s13742-015-0047-8</a>.
</div>
<div id="ref-clarkImportanceInformationRelatives2012" class="csl-entry">
Clark, Samuel A., John M. Hickey, Hans D. Daetwyler, and Julius HJ van der Werf. 2012. <span>“The Importance of Information on Relatives for the Prediction of Genomic Breeding Values and the Implications for the Makeup of Reference Data Sets in Livestock Breeding Schemes.”</span> <em>Genetics Selection Evolution</em> 44 (1): 4. <a href="https://doi.org/10.1186/1297-9686-44-4">https://doi.org/10.1186/1297-9686-44-4</a>.
</div>
<div id="ref-conoverPracticalNonparametricStatistics1999" class="csl-entry">
Conover, W. J. 1999. <em>Practical <span>Nonparametric Statistics</span></em>. <span>John Wiley &amp; Sons</span>. <a href="https://books.google.com?id=n_39DwAAQBAJ">https://books.google.com?id=n_39DwAAQBAJ</a>.
</div>
<div id="ref-10002015global" class="csl-entry">
Consortium, 1000 Genomes Project et al. 2015. <span>“A Global Reference for Human Genetic Variation.”</span> <em>Nature</em> 526 (7571): 68.
</div>
<div id="ref-duncanAnalysisPolygenicRisk2019" class="csl-entry">
Duncan, L., H. Shen, B. Gelaye, J. Meijsen, K. Ressler, M. Feldman, R. Peterson, and B. Domingue. 2019. <span>“Analysis of Polygenic Risk Score Usage and Performance in Diverse Human Populations.”</span> <em>Nature Communications</em> 10 (1, 1): 3328. <a href="https://doi.org/10.1038/s41467-019-11112-0">https://doi.org/10.1038/s41467-019-11112-0</a>.
</div>
<div id="ref-durbinDistributionTheoryTests1973" class="csl-entry">
Durbin, J. 1973. <em>Distribution <span>Theory</span> for <span>Tests Based</span> on <span>Sample Distribution Function</span></em>. <span>SIAM</span>. <a href="https://books.google.com?id=zAryCrT1IUYC">https://books.google.com?id=zAryCrT1IUYC</a>.
</div>
<div id="ref-evansQTLGeneElegans2021" class="csl-entry">
Evans, Kathryn S., Marijke H. van Wijk, Patrick T. McGrath, Erik C. Andersen, and Mark G. Sterken. 2021. <span>“From <span>QTL</span> to Gene: <span>C</span>. Elegans Facilitates Discoveries of the Genetic Mechanisms Underlying Natural Variation.”</span> <em>Trends in Genetics</em> 37 (10): 933–47. <a href="https://doi.org/10.1016/j.tig.2021.06.005">https://doi.org/10.1016/j.tig.2021.06.005</a>.
</div>
<div id="ref-fitzgeraldMedakaInbredKiyosuKarlsruhe2022" class="csl-entry">
Fitzgerald, Tomas, Ian Brettell, Adrien Leger, Nadeshda Wolf, Natalja Kusminski, Jack Monahan, Carl Barton, et al. 2022. <span>“The <span>Medaka Inbred Kiyosu-Karlsruhe</span> (<span>MIKK</span>) Panel.”</span> <em>Genome Biology</em> 23 (1): 59. <a href="https://doi.org/10.1186/s13059-022-02623-z">https://doi.org/10.1186/s13059-022-02623-z</a>.
</div>
<div id="ref-fredmanComplexSNPrelatedSequence2004" class="csl-entry">
Fredman, David, Stefan J. White, Susanna Potter, Evan E. Eichler, Johan T. Den Dunnen, and Anthony J. Brookes. 2004. <span>“Complex <span class="nocase">SNP-related</span> Sequence Variation in Segmental Genome Duplications.”</span> <em>Nature Genetics</em> 36 (8, 8): 861–66. <a href="https://doi.org/10.1038/ng1401">https://doi.org/10.1038/ng1401</a>.
</div>
<div id="ref-habierImpactGeneticRelationship2010" class="csl-entry">
Habier, David, Jens Tetens, Franz-Reinhold Seefried, Peter Lichtner, and Georg Thaller. 2010. <span>“The Impact of Genetic Relationship Information on Genomic Breeding Values in <span>German Holstein</span> Cattle.”</span> <em>Genetics Selection Evolution</em> 42 (1): 5. <a href="https://doi.org/10.1186/1297-9686-42-5">https://doi.org/10.1186/1297-9686-42-5</a>.
</div>
<div id="ref-hillLinkageDisequilibriumFinite1968" class="csl-entry">
Hill, W. G., and Alan Robertson. 1968. <span>“Linkage Disequilibrium in Finite Populations.”</span> <em>Theoretical and Applied Genetics</em> 38 (6): 226–31. <a href="https://doi.org/10.1007/BF01245622">https://doi.org/10.1007/BF01245622</a>.
</div>
<div id="ref-hoffmannChromosomalInversionPolymorphisms2004" class="csl-entry">
Hoffmann, Ary A., Carla M. Sgr‘o, and Andrew R. Weeks. 2004. <span>“Chromosomal Inversion Polymorphisms and Adaptation.”</span> <em>Trends in Ecology &amp; Evolution</em> 19 (9): 482–88. <a href="https://doi.org/10.1016/j.tree.2004.06.013">https://doi.org/10.1016/j.tree.2004.06.013</a>.
</div>
<div id="ref-IndexPubRelease102" class="csl-entry">
<span>“Index of /Pub/Release-102/Emf/Ensembl-Compara/Multiple_alignments/50_fish.epo/.”</span> n.d. Accessed January 25, 2022. <a href="https://ftp.ensembl.org/pub/release-102/emf/ensembl-compara/multiple_alignments/50_fish.epo/">https://ftp.ensembl.org/pub/release-102/emf/ensembl-compara/multiple_alignments/50_fish.epo/</a>.
</div>
<div id="ref-IndexVol1Ftp" class="csl-entry">
<span>“Index of /Vol1/Ftp/Data_collections/<span>1000g</span>_2504_high_coverage/Working/20201028_3202_phased/.”</span> n.d. Accessed March 24, 2022. <a href="http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/data_collections/1000G_2504_high_coverage/working/20201028_3202_phased/">http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/data_collections/1000G_2504_high_coverage/working/20201028_3202_phased/</a>.
</div>
<div id="ref-johnsonSegregationPerformanceRecombinant1999" class="csl-entry">
Johnson, William C., and Paul Gepts. 1999. <span>“Segregation for Performance in Recombinant Inbred Populations Resulting from Inter-Gene Pool Crosses of Common Bean ( <span>Phaseolus</span> Vulgaris <span>L</span>.).”</span> <em>Euphytica</em> 106 (1): 45–56. <a href="https://doi.org/10.1023/A:1003541201923">https://doi.org/10.1023/A:1003541201923</a>.
</div>
<div id="ref-katsumuraMedakaPopulationGenome2019" class="csl-entry">
Katsumura, Takafumi, Shoji Oda, Hiroshi Mitani, and Hiroki Oota. 2019. <span>“Medaka <span>Population Genome Structure</span> and <span>Demographic History Described</span> via <span class="nocase">Genotyping-by-Sequencing</span>.”</span> <em>G3 Genes|Genomes|Genetics</em> 9 (1): 217–28. <a href="https://doi.org/10.1534/g3.118.200779">https://doi.org/10.1534/g3.118.200779</a>.
</div>
<div id="ref-kerminenGeographicVariationBias2019" class="csl-entry">
Kerminen, Sini, Alicia R. Martin, Jukka Koskela, Sanni E. Ruotsalainen, Aki S. Havulinna, Ida Surakka, Aarno Palotie, et al. 2019. <span>“Geographic <span>Variation</span> and <span>Bias</span> in the <span>Polygenic Scores</span> of <span>Complex Diseases</span> and <span>Traits</span> in <span>Finland</span>.”</span> <em>The American Journal of Human Genetics</em> 104 (6): 1169–81. <a href="https://doi.org/10.1016/j.ajhg.2019.05.001">https://doi.org/10.1016/j.ajhg.2019.05.001</a>.
</div>
<div id="ref-kheraGenomewidePolygenicScores2018" class="csl-entry">
Khera, Amit V., Mark Chaffin, Krishna G. Aragam, Mary E. Haas, Carolina Roselli, Seung Hoan Choi, Pradeep Natarajan, et al. 2018. <span>“Genome-Wide Polygenic Scores for Common Diseases Identify Individuals with Risk Equivalent to Monogenic Mutations.”</span> <em>Nature Genetics</em> 50 (9, 9): 1219–24. <a href="https://doi.org/10.1038/s41588-018-0183-z">https://doi.org/10.1038/s41588-018-0183-z</a>.
</div>
<div id="ref-kulloIncorporatingGeneticRisk2016" class="csl-entry">
Kullo, Iftikhar J., Hayan Jouni, Erin E. Austin, Sherry-Ann Brown, Teresa M. Kruisselbrink, Iyad N. Isseh, Raad A. Haddad, et al. 2016. <span>“Incorporating a <span>Genetic Risk Score Into Coronary Heart Disease Risk Estimates</span>.”</span> <em>Circulation</em> 133 (12): 1181–88. <a href="https://doi.org/10.1161/CIRCULATIONAHA.115.020109">https://doi.org/10.1161/CIRCULATIONAHA.115.020109</a>.
</div>
<div id="ref-legerGenomicVariationsEpigenomic2022" class="csl-entry">
Leger, Adrien, Ian Brettell, Jack Monahan, Carl Barton, Nadeshda Wolf, Natalja Kusminski, Cathrin Herder, et al. 2022. <span>“Genomic Variations and Epigenomic Landscape of the <span>Medaka Inbred Kiyosu-Karlsruhe</span> (<span>MIKK</span>) Panel.”</span> <em>Genome Biology</em> 23 (1): 58. <a href="https://doi.org/10.1186/s13059-022-02602-4">https://doi.org/10.1186/s13059-022-02602-4</a>.
</div>
<div id="ref-lekAnalysisProteincodingGenetic2016" class="csl-entry">
Lek, Monkol, Konrad J. Karczewski, Eric V. Minikel, Kaitlin E. Samocha, Eric Banks, Timothy Fennell, Anne H. O’NADonnell-Luria, et al. 2016. <span>“Analysis of Protein-Coding Genetic Variation in 60,706 Humans.”</span> <em>Nature</em> 536 (7616, 7616): 285–91. <a href="https://doi.org/10.1038/nature19057">https://doi.org/10.1038/nature19057</a>.
</div>
<div id="ref-limamiGeneticPhysiologicalAnalysis2002" class="csl-entry">
Limami, Anis M., Clothilde Rouillon, Gaëlle Glevarec, Andr’e Gallais, and Bertrand Hirel. 2002. <span>“Genetic and <span>Physiological Analysis</span> of <span>Germination Efficiency</span> in <span>Maize</span> in <span>Relation</span> to <span>Nitrogen Metabolism Reveals</span> the <span>Importance</span> of <span>Cytosolic Glutamine Synthetase</span>.”</span> <em>Plant Physiology</em> 130 (4): 1860–70. <a href="https://doi.org/10.1104/pp.009647">https://doi.org/10.1104/pp.009647</a>.
</div>
<div id="ref-maasBreastCancerRisk2016" class="csl-entry">
Maas, Paige, Myrto Barrdahl, Amit D. Joshi, Paul L. Auer, Mia M. Gaudet, Roger L. Milne, Fredrick R. Schumacher, et al. 2016. <span>“Breast <span>Cancer Risk From Modifiable</span> and <span>Nonmodifiable Risk Factors Among White Women</span> in the <span>United States</span>.”</span> <em>JAMA Oncology</em> 2 (10): 1295–1302. <a href="https://doi.org/10.1001/jamaoncol.2016.1025">https://doi.org/10.1001/jamaoncol.2016.1025</a>.
</div>
<div id="ref-macarthurNewNHGRIEBICatalog2017" class="csl-entry">
MacArthur, Jacqueline, Emily Bowler, Maria Cerezo, Laurent Gil, Peggy Hall, Emma Hastings, Heather Junkins, et al. 2017. <span>“The New <span>NHGRI-EBI Catalog</span> of Published Genome-Wide Association Studies (<span>GWAS Catalog</span>).”</span> <em>Nucleic Acids Research</em> 45 (D1): D896–901. <a href="https://doi.org/10.1093/nar/gkw1133">https://doi.org/10.1093/nar/gkw1133</a>.
</div>
<div id="ref-mackayChartingGenotypePhenotype2018" class="csl-entry">
Mackay, Trudy F. C., and Wen Huang. 2018. <span>“Charting the Genotype–Phenotype Map: Lessons from the <span>Drosophila</span> Melanogaster <span>Genetic Reference Panel</span>.”</span> <em>WIREs Developmental Biology</em> 7 (1): e289. <a href="https://doi.org/10.1002/wdev.289">https://doi.org/10.1002/wdev.289</a>.
</div>
<div id="ref-magnoGwasrapiddPackageQuery2020" class="csl-entry">
Magno, Ramiro, and Ana-Teresa Maia. 2020. <span>“Gwasrapidd: An <span>R</span> Package to Query, Download and Wrangle <span>GWAS</span> Catalog Data.”</span> <em>Bioinformatics</em> 36 (2): 649–50. <a href="https://doi.org/10.1093/bioinformatics/btz605">https://doi.org/10.1093/bioinformatics/btz605</a>.
</div>
<div id="ref-martinHumanDemographicHistory2017" class="csl-entry">
Martin, Alicia R., Christopher R. Gignoux, Raymond K. Walters, Genevieve L. Wojcik, Benjamin M. Neale, Simon Gravel, Mark J. Daly, Carlos D. Bustamante, and Eimear E. Kenny. 2017. <span>“Human <span>Demographic History Impacts Genetic Risk Prediction</span> Across <span>Diverse Populations</span>.”</span> <em>The American Journal of Human Genetics</em> 100 (4): 635–49. <a href="https://doi.org/10.1016/j.ajhg.2017.03.004">https://doi.org/10.1016/j.ajhg.2017.03.004</a>.
</div>
<div id="ref-martinClinicalUseCurrent2019" class="csl-entry">
Martin, Alicia R., Masahiro Kanai, Yoichiro Kamatani, Yukinori Okada, Benjamin M. Neale, and Mark J. Daly. 2019. <span>“Clinical Use of Current Polygenic Risk Scores May Exacerbate Health Disparities.”</span> <em>Nature Genetics</em> 51 (4, 4): 584–91. <a href="https://doi.org/10.1038/s41588-019-0379-x">https://doi.org/10.1038/s41588-019-0379-x</a>.
</div>
<div id="ref-martinSimonhmartinGenomicsGeneral2022" class="csl-entry">
martin, Simon. (2016) 2022. <em>Simonhmartin/Genomics_general</em>. <a href="https://github.com/simonhmartin/genomics_general">https://github.com/simonhmartin/genomics_general</a>.
</div>
<div id="ref-martinEvaluatingUseABBA2015" class="csl-entry">
Martin, Simon H., John W. Davey, and Chris D. Jiggins. 2015. <span>“Evaluating the <span>Use</span> of <span>ABBA</span>–<span>BABA Statistics</span> to <span>Locate Introgressed Loci</span>.”</span> <em>Molecular Biology and Evolution</em> 32 (1): 244–57. <a href="https://doi.org/10.1093/molbev/msu269">https://doi.org/10.1093/molbev/msu269</a>.
</div>
<div id="ref-mccarthyReferencePanel642016" class="csl-entry">
McCarthy, Shane, Sayantan Das, Warren Kretzschmar, Olivier Delaneau, Andrew R Wood, Alexander Teumer, Hyun Min Kang, et al. 2016. <span>“A Reference Panel of 64,976 Haplotypes for Genotype Imputation.”</span> <em>Nature Genetics</em> 48 (10, 10): 1279–83. <a href="https://doi.org/10.1038/ng.3643">https://doi.org/10.1038/ng.3643</a>.
</div>
<div id="ref-mckennaGenomeAnalysisToolkit2010" class="csl-entry">
McKenna, Aaron, Matthew Hanna, Eric Banks, Andrey Sivachenko, Kristian Cibulskis, Andrew Kernytsky, Kiran Garimella, et al. 2010. <span>“The <span>Genome Analysis Toolkit</span>: <span>A MapReduce</span> Framework for Analyzing Next-Generation <span>DNA</span> Sequencing Data.”</span> <em>Genome Research</em> 20 (9): 1297–1303. <a href="https://doi.org/10.1101/gr.107524.110">https://doi.org/10.1101/gr.107524.110</a>.
</div>
<div id="ref-mukherjeeGeneIntimateHistory2016" class="csl-entry">
Mukherjee, Siddhartha. 2016. <em>The <span>Gene</span>: <span>An Intimate History</span></em>. <span>Simon and Schuster</span>. <a href="https://books.google.com?id=XvAsDAAAQBAJ">https://books.google.com?id=XvAsDAAAQBAJ</a>.
</div>
<div id="ref-naruseDetailedLinkageMap2000" class="csl-entry">
Naruse, Kiyoshi, Shoji Fukamachi, Hiroshi Mitani, Mariko Kondo, Tomoko Matsuoka, Shu Kondo, Nana Hanamura, et al. 2000. <span>“A <span>Detailed Linkage Map</span> of <span>Medaka</span>, <span>Oryzias</span> Latipes: <span>Comparative Genomics</span> and <span>Genome Evolution</span>.”</span> <em>Genetics</em> 154 (4): 1773–84. <a href="https://www.genetics.org/content/154/4/1773">https://www.genetics.org/content/154/4/1773</a>.
</div>
<div id="ref-natarajanPolygenicRiskScore2017" class="csl-entry">
Natarajan, Pradeep, Robin Young, Nathan O. Stitziel, Sandosh Padmanabhan, Usman Baber, Roxana Mehran, Samantha Sartori, et al. 2017. <span>“Polygenic <span>Risk Score Identifies Subgroup With Higher Burden</span> of <span>Atherosclerosis</span> and <span>Greater Relative Benefit From Statin Therapy</span> in the <span>Primary Prevention Setting</span>.”</span> <em>Circulation</em> 135 (22): 2091–101. <a href="https://doi.org/10.1161/CIRCULATIONAHA.116.024436">https://doi.org/10.1161/CIRCULATIONAHA.116.024436</a>.
</div>
<div id="ref-panagiotouWhatShouldGenomewide2012" class="csl-entry">
Panagiotou, Orestis A., John P. A. Ioannidis, and Genome-Wide Significance Project. 2012. <span>“What Should the Genome-Wide Significance Threshold Be? <span>Empirical</span> Replication of Borderline Genetic Associations.”</span> <em>International Journal of Epidemiology</em> 41 (1): 273–86. <a href="https://doi.org/10.1093/ije/dyr178">https://doi.org/10.1093/ije/dyr178</a>.
</div>
<div id="ref-paquettePolygenicRiskScore2017" class="csl-entry">
Paquette, Martine, Michael Chong, S’ebastien Th’eriault, Robert Dufour, Guillaume Par’e, and Alexis Baass. 2017. <span>“Polygenic Risk Score Predicts Prevalence of Cardiovascular Disease in Patients with Familial Hypercholesterolemia.”</span> <em>Journal of Clinical Lipidology</em> 11 (3): 725–732.e5. <a href="https://doi.org/10.1016/j.jacl.2017.03.019">https://doi.org/10.1016/j.jacl.2017.03.019</a>.
</div>
<div id="ref-paradisPegasPackagePopulation2010" class="csl-entry">
Paradis, Emmanuel. 2010. <span>“Pegas: An <span>R</span> Package for Population Genetics with an Integrated–Modular Approach.”</span> <em>Bioinformatics</em> 26 (3): 419–20. <a href="https://doi.org/10.1093/bioinformatics/btp696">https://doi.org/10.1093/bioinformatics/btp696</a>.
</div>
<div id="ref-paradisApeEnvironmentModern2019" class="csl-entry">
Paradis, Emmanuel, and Klaus Schliep. 2019. <span>“Ape 5.0: An Environment for Modern Phylogenetics and Evolutionary Analyses in <span>R</span>.”</span> <em>Bioinformatics</em> 35 (3): 526–28. <a href="https://doi.org/10.1093/bioinformatics/bty633">https://doi.org/10.1093/bioinformatics/bty633</a>.
</div>
<div id="ref-peirceNewSetBXD2004" class="csl-entry">
Peirce, Jeremy L., Lu Lu, Jing Gu, Lee M. Silver, and Robert W. Williams. 2004. <span>“A New Set of <span>BXD</span> Recombinant Inbred Lines from Advanced Intercross Populations in Mice.”</span> <em>BMC Genetics</em> 5 (1): 7. <a href="https://doi.org/10.1186/1471-2156-5-7">https://doi.org/10.1186/1471-2156-5-7</a>.
</div>
<div id="ref-plominNatureNurtureGenetic2005" class="csl-entry">
Plomin, Robert, and Kathryn Asbury. 2005. <span>“Nature and <span>Nurture</span>: <span>Genetic</span> and <span>Environmental Influences</span> on <span>Behavior</span>.”</span> <em>The ANNALS of the American Academy of Political and Social Science</em> 600 (1): 86–98. <a href="https://doi.org/10.1177/0002716205277184">https://doi.org/10.1177/0002716205277184</a>.
</div>
<div id="ref-popejoyGenomicsFailingDiversity2016" class="csl-entry">
Popejoy, Alice B., and Stephanie M. Fullerton. 2016. <span>“Genomics Is Failing on Diversity.”</span> <em>Nature</em> 538 (7624, 7624): 161–64. <a href="https://doi.org/10.1038/538161a">https://doi.org/10.1038/538161a</a>.
</div>
<div id="ref-porcuGenotypeImputationGenomeWide2013" class="csl-entry">
Porcu, Eleonora, Serena Sanna, Christian Fuchsberger, and Lars G. Fritsche. 2013. <span>“Genotype <span>Imputation</span> in <span>Genome-Wide Association Studies</span>.”</span> <em>Current Protocols in Human Genetics</em> 78 (1): 1.25.1–14. <a href="https://doi.org/10.1002/0471142905.hg0125s78">https://doi.org/10.1002/0471142905.hg0125s78</a>.
</div>
<div id="ref-pszczolaReliabilityDirectGenomic2012" class="csl-entry">
Pszczola, M., T. Strabel, H. A. Mulder, and M. P. L. Calus. 2012. <span>“Reliability of Direct Genomic Values for Animals with Different Relationships Within and to the Reference Population.”</span> <em>Journal of Dairy Science</em> 95 (1): 389–400. <a href="https://doi.org/10.3168/jds.2011-4338">https://doi.org/10.3168/jds.2011-4338</a>.
</div>
<div id="ref-purcellPLINK" class="csl-entry">
Purcell, Shaun M, and Christopher C Chang. n.d. <em><span>PLINK</span> 1.9</em>. <a href="https://www.cog-genomics.org/plink/1.9/">www.cog-genomics.org/plink/1.9/</a>.
</div>
<div id="ref-saliba-colombaniEfficiencyRFLPRAPD2000" class="csl-entry">
Saliba-Colombani, Vera, Mathilde Causse, Laurent Gervais, and Jacqueline Philouze. 2000. <span>“Efficiency of <span>RFLP</span>, <span>RAPD</span>, and <span>AFLP</span> Markers for the Construction of an Intraspecific Map of the Tomato Genome.”</span> <em>Genome</em> 43 (1): 29–40. <a href="https://doi.org/10.1139/g99-096">https://doi.org/10.1139/g99-096</a>.
</div>
<div id="ref-saulHighDiversityMousePopulations2019" class="csl-entry">
Saul, Michael C., Vivek M. Philip, Laura G. Reinholdt, and Elissa J. Chesler. 2019. <span>“High-<span>Diversity Mouse Populations</span> for <span>Complex Traits</span>.”</span> <em>Trends in Genetics</em> 35 (7): 501–14. <a href="https://doi.org/10.1016/j.tig.2019.04.003">https://doi.org/10.1016/j.tig.2019.04.003</a>.
</div>
<div id="ref-schumacherAssociationAnalysesMore2018" class="csl-entry">
Schumacher, Fredrick R., Ali Amin Al Olama, Sonja I. Berndt, Sara Benlloch, Mahbubl Ahmed, Edward J. Saunders, Tokhir Dadaev, et al. 2018. <span>“Association Analyses of More Than 140,000 Men Identify 63 New Prostate Cancer Susceptibility Loci.”</span> <em>Nature Genetics</em> 50 (7, 7): 928–36. <a href="https://doi.org/10.1038/s41588-018-0142-8">https://doi.org/10.1038/s41588-018-0142-8</a>.
</div>
<div id="ref-scutariUsingGeneticDistance2016" class="csl-entry">
Scutari, Marco, Ian Mackay, and David Balding. 2016. <span>“Using <span>Genetic Distance</span> to <span>Infer</span> the <span>Accuracy</span> of <span>Genomic Prediction</span>.”</span> <em>PLOS Genetics</em> 12 (9): e1006288. <a href="https://doi.org/10.1371/journal.pgen.1006288">https://doi.org/10.1371/journal.pgen.1006288</a>.
</div>
<div id="ref-shamPowerLinkageAssociation2000" class="csl-entry">
Sham, P. C., S. S. Cherny, S. Purcell, and J. K. Hewitt. 2000. <span>“Power of <span>Linkage</span> Versus <span>Association Analysis</span> of <span>Quantitative Traits</span>, by <span>Use</span> of <span>Variance-Components Models</span>, for <span>Sibship Data</span>.”</span> <em>The American Journal of Human Genetics</em> 66 (5): 1616–30. <a href="https://doi.org/10.1086/302891">https://doi.org/10.1086/302891</a>.
</div>
<div id="ref-sharpDevelopmentStandardizationImproved2019" class="csl-entry">
Sharp, Seth A., Stephen S. Rich, Andrew R. Wood, Samuel E. Jones, Robin N. Beaumont, James W. Harrison, Darius A. Schneider, et al. 2019. <span>“Development and <span>Standardization</span> of an <span>Improved Type</span> 1 <span>Diabetes Genetic Risk Score</span> for <span>Use</span> in <span>Newborn Screening</span> and <span>Incident Diagnosis</span>.”</span> <em>Diabetes Care</em> 42 (2): 200–207. <a href="https://doi.org/10.2337/dc18-1785">https://doi.org/10.2337/dc18-1785</a>.
</div>
<div id="ref-smigielskiDbSNPDatabaseSingle2000" class="csl-entry">
Smigielski, Elizabeth M., Karl Sirotkin, Minghong Ward, and Stephen T. Sherry. 2000. <span>“<span class="nocase">dbSNP</span>: A Database of Single Nucleotide Polymorphisms.”</span> <em>Nucleic Acids Research</em> 28 (1): 352–55. <a href="https://doi.org/10.1093/nar/28.1.352">https://doi.org/10.1093/nar/28.1.352</a>.
</div>
<div id="ref-spivakovGenomicPhenotypicCharacterization2014" class="csl-entry">
Spivakov, Mikhail, Thomas O Auer, Ravindra Peravali, Ian Dunham, Dirk Dolle, Asao Fujiyama, Atsushi Toyoda, et al. 2014. <span>“Genomic and <span>Phenotypic Characterization</span> of a <span>Wild Medaka Population</span>: <span>Towards</span> the <span>Establishment</span> of an <span>Isogenic Population Genetic Resource</span> in <span>Fish</span>.”</span> <em>G3 Genes|Genomes|Genetics</em> 4 (3): 433–45. <a href="https://doi.org/10.1534/g3.113.008722">https://doi.org/10.1534/g3.113.008722</a>.
</div>
<div id="ref-svensonHighResolutionGeneticMapping2012" class="csl-entry">
Svenson, Karen L, Daniel M Gatti, William Valdar, Catherine E Welsh, Riyan Cheng, Elissa J Chesler, Abraham A Palmer, Leonard McMillan, and Gary A Churchill. 2012. <span>“High-<span>Resolution Genetic Mapping Using</span> the <span>Mouse Diversity Outbred Population</span>.”</span> <em>Genetics</em> 190 (2): 437–47. <a href="https://doi.org/10.1534/genetics.111.132597">https://doi.org/10.1534/genetics.111.132597</a>.
</div>
<div id="ref-11TrillionGlobal2019" class="csl-entry">
<span>“The $11.9 <span>Trillion Global Healthcare Market</span>: <span>Key Opportunities</span> &amp; <span>Strategies</span> (2014-2022) - <span>ResearchAndMarkets</span>.com.”</span> 2019. June 25, 2019. <a href="https://www.businesswire.com/news/home/20190625005862/en/The-11.9-Trillion-Global-Healthcare-Market-Key-Opportunities-Strategies-2014-2022---ResearchAndMarkets.com">https://www.businesswire.com/news/home/20190625005862/en/The-11.9-Trillion-Global-Healthcare-Market-Key-Opportunities-Strategies-2014-2022---ResearchAndMarkets.com</a>.
</div>
<div id="ref-threadgillCollaborativeCrossRecombinant2011" class="csl-entry">
Threadgill, David W., Darla R. Miller, Gary A. Churchill, and Fernando Pardo-Manuel de Villena. 2011. <span>“The <span>Collaborative Cross</span>: <span>A Recombinant Inbred Mouse Population</span> for the <span>Systems Genetic Era</span>.”</span> <em>ILAR Journal</em> 52 (1): 24–31. <a href="https://doi.org/10.1093/ilar.52.1.24">https://doi.org/10.1093/ilar.52.1.24</a>.
</div>
<div id="ref-tikkanenGeneticRiskPrediction2013" class="csl-entry">
Tikkanen, Emmi, Aki S. Havulinna, Aarno Palotie, Veikko Salomaa, and Samuli Ripatti. 2013. <span>“Genetic <span>Risk Prediction</span> and a 2-<span>Stage Risk Screening Strategy</span> for <span>Coronary Heart Disease</span>.”</span> <em>Arteriosclerosis, Thrombosis, and Vascular Biology</em> 33 (9): 2261–66. <a href="https://doi.org/10.1161/ATVBAHA.112.301120">https://doi.org/10.1161/ATVBAHA.112.301120</a>.
</div>
<div id="ref-vilhjalmsson2015modeling" class="csl-entry">
Vilhjálmsson, Bjarni J, Jian Yang, Hilary K Finucane, Alexander Gusev, Sara Lindström, Stephan Ripke, Giulio Genovese, et al. 2015. <span>“Modeling Linkage Disequilibrium Increases Accuracy of Polygenic Risk Scores.”</span> <em>The American Journal of Human Genetics</em> 97 (4): 576–92.
</div>
<div id="ref-wangTheoreticalEmpiricalQuantification2020" class="csl-entry">
Wang, Ying, Jing Guo, Guiyan Ni, Jian Yang, Peter M. Visscher, and Loic Yengo. 2020. <span>“Theoretical and Empirical Quantification of the Accuracy of Polygenic Scores in Ancestry Divergent Populations.”</span> <em>Nature Communications</em> 11 (1, 1): 3865. <a href="https://doi.org/10.1038/s41467-020-17719-y">https://doi.org/10.1038/s41467-020-17719-y</a>.
</div>
<div id="ref-weirEstimatingFStatisticsAnalysis1984" class="csl-entry">
Weir, B. S., and C. Clark Cockerham. 1984. <span>“Estimating <span>F-Statistics</span> for the <span>Analysis</span> of <span>Population Structure</span>.”</span> <em>Evolution</em> 38 (6): 1358–70. <a href="https://doi.org/10.2307/2408641">https://doi.org/10.2307/2408641</a>.
</div>
<div id="ref-wittbrodtMedakaModelOrganism2002" class="csl-entry">
Wittbrodt, Joachim, Akihiro Shima, and Manfred Schartl. 2002. <span>“Medaka — a Model Organism from the Far East.”</span> <em>Nature Reviews Genetics</em> 3 (1, 1): 53–64. <a href="https://doi.org/10.1038/nrg704">https://doi.org/10.1038/nrg704</a>.
</div>
<div id="ref-wrayPredictionIndividualGenetic2007" class="csl-entry">
Wray, Naomi R., Michael E. Goddard, and Peter M. Visscher. 2007. <span>“Prediction of Individual Genetic Risk to Disease from Genome-Wide Association Studies.”</span> <em>Genome Research</em> 17 (10): 1520–28. <a href="https://doi.org/10.1101/gr.6665407">https://doi.org/10.1101/gr.6665407</a>.
</div>
</div>
</div>
  <div class="chapter-nav">
<div class="prev"><a href="Fst-chap.html"><span class="header-section-number">6</span> Variation in the frequency of trait-associated alleles across global human populations</a></div>
<div class="empty"></div>
</div></main><div class="col-md-3 col-lg-2 d-none d-md-block sidebar sidebar-chapter">
    <nav id="toc" data-toggle="toc" aria-label="On this page"><h2>On this page</h2>
      <ul class="nav navbar-nav"><li><a class="nav-link" href="#references">References</a></li></ul>

      <div class="book-extra">
        <ul class="list-unstyled">
<li><a id="book-source" href="https://github.com/brettellebi/PhD-thesis/blob/master/book/07-references.Rmd">View source <i class="fab fa-github"></i></a></li>
          <li><a id="book-edit" href="https://github.com/brettellebi/PhD-thesis/edit/master/book/07-references.Rmd">Edit this page <i class="fab fa-github"></i></a></li>
        </ul>
</div>
    </nav>
</div>

</div>
</div> <!-- .container -->

<footer class="bg-primary text-light mt-5"><div class="container"><div class="row">

  <div class="col-12 col-md-6 mt-3">
    <p>"<strong>Genetic analysis of quantitative traits in medaka fish and humans</strong>" was written by Ian Brettell. It was last built on 2022-06-14.</p>
  </div>

  <div class="col-12 col-md-6 mt-3">
    <p>This book was built by the <a class="text-light" href="https://bookdown.org">bookdown</a> R package.</p>
  </div>

</div></div>
</footer><!-- dynamically load mathjax for compatibility with self-contained --><script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script><script type="text/x-mathjax-config">const popovers = document.querySelectorAll('a.footnote-ref[data-toggle="popover"]');
for (let popover of popovers) {
  const div = document.createElement('div');
  div.setAttribute('style', 'position: absolute; top: 0, left:0; width:0, height:0, overflow: hidden; visibility: hidden;');
  div.innerHTML = popover.getAttribute('data-content');

  var has_math = div.querySelector("span.math");
  if (has_math) {
    document.body.appendChild(div);
    MathJax.Hub.Queue(["Typeset", MathJax.Hub, div]);
    MathJax.Hub.Queue(function() {
      popover.setAttribute('data-content', div.innerHTML);
      document.body.removeChild(div);
    })
  }
}
</script>
</body>
</html>
