<!DOCTYPE html>
<html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<title>Chapter 5 Genetic loci associated with somite development periodicity | Japanese courage: a genetic analysis of complex traits in medaka fish and humans</title>
<meta name="author" content="Ian Brettell">
<meta name="description" content="5.1 Background During the development of an embryo, somites are the earliest primitive segmental structures that form from presomatic mesoderm cells (PSM).105 They later differentiate into...">
<meta name="generator" content="bookdown 0.28 with bs4_book()">
<meta property="og:title" content="Chapter 5 Genetic loci associated with somite development periodicity | Japanese courage: a genetic analysis of complex traits in medaka fish and humans">
<meta property="og:type" content="book">
<meta property="og:description" content="5.1 Background During the development of an embryo, somites are the earliest primitive segmental structures that form from presomatic mesoderm cells (PSM).105 They later differentiate into...">
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="Chapter 5 Genetic loci associated with somite development periodicity | Japanese courage: a genetic analysis of complex traits in medaka fish and humans">
<meta name="twitter:description" content="5.1 Background During the development of an embryo, somites are the earliest primitive segmental structures that form from presomatic mesoderm cells (PSM).105 They later differentiate into...">
<!-- JS --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://kit.fontawesome.com/6ecbd6c532.js" crossorigin="anonymous"></script><script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="libs/bootstrap-4.6.0/bootstrap.min.css" rel="stylesheet">
<script src="libs/bootstrap-4.6.0/bootstrap.bundle.min.js"></script><script src="libs/bs3compat-0.4.0/transition.js"></script><script src="libs/bs3compat-0.4.0/tabs.js"></script><script src="libs/bs3compat-0.4.0/bs3compat.js"></script><link href="libs/bs4_book-1.0.0/bs4_book.css" rel="stylesheet">
<script src="libs/bs4_book-1.0.0/bs4_book.js"></script><style>
    #content {
        font-size: 0.9rem;
    }
    </style>
<script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- CSS --><style type="text/css">
    
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  </style>
<style type="text/css">
    /* Used with Pandoc 2.11+ new --citeproc when CSL is used */
    div.csl-bib-body { }
    div.csl-entry {
      clear: both;
        }
    .hanging div.csl-entry {
      margin-left:2em;
      text-indent:-2em;
    }
    div.csl-left-margin {
      min-width:2em;
      float:left;
    }
    div.csl-right-inline {
      margin-left:2em;
      padding-left:1em;
    }
    div.csl-indent {
      margin-left: 2em;
    }
  </style>
<link rel="stylesheet" href="style.css">
</head>
<body data-spy="scroll" data-target="#toc">

<div class="container-fluid">
<div class="row">
  <header class="col-sm-12 col-lg-3 sidebar sidebar-book"><a class="sr-only sr-only-focusable" href="#content">Skip to main content</a>

    <div class="d-flex align-items-start justify-content-between">
      <h1>
        <a href="index.html" title="">Japanese courage: a genetic analysis of complex traits in medaka fish and humans</a>
      </h1>
      <button class="btn btn-outline-primary d-lg-none ml-2 mt-1" type="button" data-toggle="collapse" data-target="#main-nav" aria-expanded="true" aria-controls="main-nav"><i class="fas fa-bars"></i><span class="sr-only">Show table of contents</span></button>
    </div>

    <div id="main-nav" class="collapse-lg">
      <form role="search">
        <input id="search" class="form-control" type="search" placeholder="Search" aria-label="Search">
</form>

      <nav aria-label="Table of contents"><h2>Table of contents</h2>
        <ul class="book-toc list-unstyled">
<li><a class="" href="index.html">About</a></li>
<li><a class="" href="Introduction.html"><span class="header-section-number">1</span> Introduction</a></li>
<li><a class="" href="MIKK-genomes-chap.html"><span class="header-section-number">2</span> Genomic variations in the MIKK panel</a></li>
<li><a class="" href="Pilot-chap.html"><span class="header-section-number">3</span> Classification of bold/shy behaviours in 5 inbred medaka lines</a></li>
<li><a class="" href="MIKK-F2-chap.html"><span class="header-section-number">4</span> Genetic linkage study of bold/shy behaviours in the MIKK panel</a></li>
<li><a class="active" href="Somite-chap.html"><span class="header-section-number">5</span> Genetic loci associated with somite development periodicity</a></li>
<li><a class="" href="Fst-chap.html"><span class="header-section-number">6</span> Variation in the frequency of trait-associated alleles across global human populations</a></li>
<li><a class="" href="references.html">References</a></li>
<li class="book-part">Appendix</li>
<li><a class="" href="mikk-behaviour-supplementary-information.html"><span class="header-section-number">A</span> MIKK behaviour supplementary information</a></li>
<li><a class="" href="supplmentary-information-for-chapter-reffst-chap.html"><span class="header-section-number">B</span> Supplmentary information for Chapter @ref(Fst-chap)</a></li>
</ul>

        <div class="book-extra">
          <p><a id="book-repo" href="https://github.com/brettellebi/PhD-thesis">View book source <i class="fab fa-github"></i></a></p>
        </div>
      </nav>
</div>
  </header><main class="col-sm-12 col-md-9 col-lg-7" id="content"><div id="Somite-chap" class="section level1" number="5">
<h1>
<span class="header-section-number">5</span> Genetic loci associated with somite development periodicity<a class="anchor" aria-label="anchor" href="#Somite-chap"><i class="fas fa-link"></i></a>
</h1>
<div id="background" class="section level2" number="5.1">
<h2>
<span class="header-section-number">5.1</span> Background<a class="anchor" aria-label="anchor" href="#background"><i class="fas fa-link"></i></a>
</h2>
<p>During the development of an embryo, somites are the earliest primitive segmental structures that form from presomatic mesoderm cells (<strong>PSM</strong>).<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Woong Kim et al., &lt;span&gt;“The Period of the Somite Segmentation Clock Is Sensitive to &lt;span&gt;Notch&lt;/span&gt; Activity,”&lt;/span&gt; &lt;em&gt;Molecular Biology of the Cell&lt;/em&gt; 22, no. 18 (September 15, 2011): 3541–49, &lt;a href="https://doi.org/10.1091/mbc.e11-02-0139" role="doc-biblioref"&gt;https://doi.org/10.1091/mbc.e11-02-0139&lt;/a&gt;.&lt;/p&gt;'><sup>105</sup></a></span> They later differentiate into vertebrae, ribs, and skeletal muscles, thereby establishing the body’s anterior-posterior axis. <strong>Figure <a href="Somite-chap.html#fig:mouse-embryo">5.1</a></strong> depicts a number of formed somites in a 9.5-day-old mouse embryo.</p>

<div class="figure">
<span style="display:block;" id="fig:mouse-embryo"></span>
<img src="figs/somites/mouse_embryo_gridley.png" alt="Image of a mouse embryo at day 9.5 from Thomas Gridley106, showing somites in darker colours." width="100%"><p class="caption">
Figure 5.1: Image of a mouse embryo at day 9.5 from <span class="citation">Thomas Gridley<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;&lt;span&gt;“The Long and Short of It: &lt;span&gt;Somite&lt;/span&gt; Formation in Mice,”&lt;/span&gt; &lt;em&gt;Developmental Dynamics&lt;/em&gt; 235, no. 9 (2006): 2330–36, &lt;a href="https://doi.org/10.1002/dvdy.20850" role="doc-biblioref"&gt;https://doi.org/10.1002/dvdy.20850&lt;/a&gt;.&lt;/p&gt;'><sup>106</sup></a></span>, showing somites in darker colours.
</p>
</div>
<p>Somite formation occurs rhythmically and sequentially, with the time between the formation of each pair of somites referred to as the “period”. The period of somite formation varies greatly between species: ~30 minutes for zebrafish, 90 minutes for chickens, 2-3 hours for mice, and 5-6 hours for humans.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Alexis Hubaud and Olivier Pourqui’e, &lt;span&gt;“Signalling Dynamics in Vertebrate Segmentation,”&lt;/span&gt; &lt;em&gt;Nature Reviews Molecular Cell Biology&lt;/em&gt; 15, no. 11, 11 (November 2014): 709–21, &lt;a href="https://doi.org/10.1038/nrm3891" role="doc-biblioref"&gt;https://doi.org/10.1038/nrm3891&lt;/a&gt;; Mitsuhiro Matsuda et al., &lt;span&gt;“Species-Specific Segmentation Clock Periods Are Due to Differential Biochemical Reaction Speeds,”&lt;/span&gt; &lt;em&gt;Science&lt;/em&gt; 369, no. 6510 (2020): 1450–55.&lt;/p&gt;'><sup>107</sup></a></span> <strong>Figure <a href="Somite-chap.html#fig:somite-seg-ali">5.2</a></strong> shows the a series of time-stamped images of somite segmentation in medaka fish, generated by Ali Seleit.</p>

<div class="figure">
<span style="display:block;" id="fig:somite-seg-ali"></span>
<img src="figs/somites/ali_fish_seg_compiled.png" alt="Time-stamped images of somite segmentation in medaka, generated by Ali Seleit." width="100%"><p class="caption">
Figure 5.2: Time-stamped images of somite segmentation in medaka, generated by Ali Seleit.
</p>
</div>
<p>The period of somite formation is controlled by a molecular oscillator, known as the ‘segmentation clock’, which drives waves of gene expression in the Notch, fibroblast growth factor (FGF), and Wnt pathways, forming a signalling gradient that regresses towards the tail in concert with axis elongation.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Gomez et al., &lt;span&gt;“Control of Segment Number in Vertebrate Embryos.”&lt;/span&gt;&lt;/p&gt;"><sup>108</sup></a></span> Over the course of elongation, the wave period increases (i.e. each somite takes longer to form), and the PSM progressively shrinks until it is exhausted, eventually terminating somite formation.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Gomez et al.&lt;/p&gt;"><sup>109</sup></a></span></p>
<p>It is not fully understood how the phase waves of the segmentation clock are intially established.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Henning J Falk et al., &lt;span&gt;“Imaging the Onset of Oscillatory Signaling Dynamics During Mouse Embryo Gastrulation,”&lt;/span&gt; &lt;em&gt;Development (Cambridge, England)&lt;/em&gt; 149, no. 13 (July 1, 2022): dev200083, &lt;a href="https://doi.org/10.1242/dev.200083" role="doc-biblioref"&gt;https://doi.org/10.1242/dev.200083&lt;/a&gt;.&lt;/p&gt;'><sup>110</sup></a></span> <span class="citation">Matsuda et al.<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;&lt;span&gt;“Species-Specific Segmentation Clock Periods Are Due to Differential Biochemical Reaction Speeds.”&lt;/span&gt;&lt;/p&gt;"><sup>111</sup></a></span> found that period differences between mouse and human occur at the single-cell level (i.e. not due to intercellular communication), and are driven by biochemical reaction speeds - specifically, mRNA and protein degradation rates, transcription and translation delays, and intron and splicing delays. To identify the genetic basis of these biochemical differences, our collaborators Ali Seleit and Alexander Aulehla at EMBL-Heidelberg used a CRISPR-Case9 knock-in approach<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Ali Seleit, Alexander Aulehla, and Alexandre Paix, &lt;span&gt;“Endogenous Protein Tagging in Medaka Using a Simplified &lt;span&gt;CRISPR&lt;/span&gt;/&lt;span&gt;Cas9&lt;/span&gt; Knock-in Approach.”&lt;/span&gt; &lt;em&gt;eLife&lt;/em&gt; 10 (December 1, 2021): e75050, &lt;a href="https://doi.org/10.7554/elife.75050" role="doc-biblioref"&gt;https://doi.org/10.7554/elife.75050&lt;/a&gt;.&lt;/p&gt;'><sup>112</sup></a></span> to establish a medaka <em>Cab</em> strain with an endogenous, fluorescing reporter gene (Her7-Venus) for the oscillation signalling pathway. This method allows them to image somite formation and extract quantitative measures for segmentation clock dynamics.</p>
<p>In medaka, it is known that the southern Japanese <em>Cab</em> strain and the northern Japanese <em>Kaga</em> strain have divergent somite periodicity, where <em>Kaga</em>’s tends to be faster, and <em>Cab</em>’s slower (<strong>Figure <a href="Somite-chap.html#fig:F0-Cab-Kaga-HdrR">5.3</a></strong>).</p>

<div class="figure">
<span style="display:block;" id="fig:F0-Cab-Kaga-HdrR"></span>
<img src="figs/somites/ali_period_F0_Cab_Kaga.png" alt="Comparison of period for three inbred medaka strains (Cab, Kaga and HdrR). Kaga’s period is lower, and therefore it takes less time to form each somite than Cab. Figure generated by Ali Seleit." width="100%"><p class="caption">
Figure 5.3: Comparison of period for three inbred medaka strains (<em>Cab</em>, <em>Kaga</em> and <em>HdrR</em>). Kaga’s period is lower, and therefore it takes less time to form each somite than <em>Cab</em>. Figure generated by Ali Seleit.
</p>
</div>
<p>Our collaborators accordingly set up a one-way F2 cross experiment as described in Chapter <a href="MIKK-F2-chap.html#MIKK-F2-cross">4.1</a>, using the reporter-carrying <em>Cab</em> strain and the <em>Kaga</em> strain as the parental F0 strains, in order to identify genetic loci associated with these differences in clock dynamics. They inter-crossed the hybrid F1 generation to create a sample of 622 F2 individuals, imaged the developing embryos of these F2 samples, and used pyBOAT<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Christoph Schmal, Gregor Mönke, and Adri’an E. Granada, &lt;span&gt;“Analysis of &lt;span&gt;Complex Circadian Time Series Data Using Wavelets&lt;/span&gt;,”&lt;/span&gt; in &lt;em&gt;Circadian &lt;span&gt;Regulation&lt;/span&gt;: &lt;span&gt;Methods&lt;/span&gt; and &lt;span&gt;Protocols&lt;/span&gt;&lt;/em&gt;, ed. Guiomar Solanas and Patrick -Simon Welz, Methods in &lt;span&gt;Molecular Biology&lt;/span&gt; (&lt;span&gt;New York, NY&lt;/span&gt;: &lt;span&gt;Springer US&lt;/span&gt;, 2022), 35–54, &lt;a href="https://doi.org/10.1007/978-1-0716-2249-0_3" role="doc-biblioref"&gt;https://doi.org/10.1007/978-1-0716-2249-0_3&lt;/a&gt;.&lt;/p&gt;'><sup>113</sup></a></span> to extract the oscillation features during somite development. <strong>Figure <a href="Somite-chap.html#fig:somite-period-ali">5.4</a></strong> shows a series of raw images used by pyBOAT to track the elongation of a medaka tail during somitogenesis, with the identified posterior tip of the embryo labelled with a blue circle.</p>

<div class="figure">
<span style="display:block;" id="fig:somite-period-ali"></span>
<img src="figs/somites/ali_compiled_somite_elong.png" alt="Screenshots of vertebral elongation in an F2 individual captured by Ali Seleit during imaging. The blue circle represents the point tracked by pyBOAT over time, generating the quantitative phenotype data on period development used in this study." width="100%"><p class="caption">
Figure 5.4: Screenshots of vertebral elongation in an F2 individual captured by Ali Seleit during imaging. The blue circle represents the point tracked by pyBOAT over time, generating the quantitative phenotype data on period development used in this study.
</p>
</div>
</div>
<div id="somite-phenotype" class="section level2" number="5.2">
<h2>
<span class="header-section-number">5.2</span> Phenotypes of interest<a class="anchor" aria-label="anchor" href="#somite-phenotype"><i class="fas fa-link"></i></a>
</h2>
<div id="somite-development-period" class="section level3" number="5.2.1">
<h3>
<span class="header-section-number">5.2.1</span> Somite development period<a class="anchor" aria-label="anchor" href="#somite-development-period"><i class="fas fa-link"></i></a>
</h3>
<p><strong>Figure <a href="Somite-chap.html#fig:ali-somite-period-lines">5.5</a></strong> shows the period data generated by pyBOAT for this study, for 100 illustrative F2 samples over 300 minutes. The same data can be represented by boxplots as shown in <strong>Figure <a href="Somite-chap.html#fig:ali-somite-period-box">5.6</a></strong>. I experimented with using the F2 individuals’ mean period and period intercept as the phenotype of interest. The two measures are highly correlated (<span class="math inline">\(Pearson's~r =\)</span> 0.84, <span class="math inline">\(p\)</span> &lt; 2.2x10<sup>-16</sup>), so after displaying the distributions for both measures in Figure <a href="Somite-chap.html#fig:somite-phenos">5.8</a>, I proceed to only discuss the analysis of period intercept, as it would appear to potentially be more robust to the changes in slope that can be observed in Figure <a href="Somite-chap.html#fig:ali-somite-period-lines">5.5</a>.</p>

<div class="figure">
<span style="display:block;" id="fig:ali-somite-period-lines"></span>
<img src="figs/somites/ali_period_lines_100fish300mins.png" alt="PyBOAT results for 100 illustrative F2 samples, showing the period length in minutes over the course of 300 minutes. Period tends to increase over time, meaning that as the embryo develops, each successive somite takes longer to form. Figure generated by Ali Seleit." width="100%"><p class="caption">
Figure 5.5: PyBOAT results for 100 illustrative F2 samples, showing the period length in minutes over the course of 300 minutes. Period tends to increase over time, meaning that as the embryo develops, each successive somite takes longer to form. Figure generated by Ali Seleit.
</p>
</div>

<div class="figure">
<span style="display:block;" id="fig:ali-somite-period-box"></span>
<img src="figs/somites/ali_F2_mean_period.png" alt="Period measurements for 70 F2 individuals displayed as boxplots with each individual’s median and interquartile range. Figure generated by Ali Seleit." width="100%"><p class="caption">
Figure 5.6: Period measurements for 70 F2 individuals displayed as boxplots with each individual’s median and interquartile range. Figure generated by Ali Seleit.
</p>
</div>
</div>
<div id="psm-area" class="section level3" number="5.2.2">
<h3>
<span class="header-section-number">5.2.2</span> PSM area<a class="anchor" aria-label="anchor" href="#psm-area"><i class="fas fa-link"></i></a>
</h3>
<p>In the proceeding analyses, I also included a second phenotype of interest: the total size of the PSM prior to the formation of any somite segments. As the measure is simply based on the total number of pixels covered by the embryo object, I considered it to be potentially more robust than the period measurements, and therefore included it as a type of positive control for the genetic association analyses on the period phenotype. The measurements for PSM area comparing F0 <em>Cab</em> and <em>Kaga</em> strains are set out in <strong>Figure <a href="Somite-chap.html#fig:ali-psm-F0">5.7</a></strong>.</p>

<div class="figure">
<span style="display:block;" id="fig:ali-psm-F0"></span>
<img src="figs/somites/ali_PSM_Cab_Kaga.png" alt="Measurements of unsegmented PSM area in pixels for the F0 individuals from the Cab strain (\(N = 19\)) and Kaga strain (\(N = 10\)). Kaga tends to have a smaller PSM than Cab. Figure generated by Ali Seleit." width="100%"><p class="caption">
Figure 5.7: Measurements of unsegmented PSM area in pixels for the F0 individuals from the <em>Cab</em> strain (<span class="math inline">\(N = 19\)</span>) and <em>Kaga</em> strain (<span class="math inline">\(N = 10\)</span>). <em>Kaga</em> tends to have a smaller PSM than <em>Cab</em>. Figure generated by Ali Seleit.
</p>
</div>
</div>
<div id="comparisons-between-f0-f1-and-f2-generations" class="section level3" number="5.2.3">
<h3>
<span class="header-section-number">5.2.3</span> Comparisons between F0, F1 and F2 generations<a class="anchor" aria-label="anchor" href="#comparisons-between-f0-f1-and-f2-generations"><i class="fas fa-link"></i></a>
</h3>
<p>The distributions across the F0, F1 and F2 generations are curious (<strong>Figure <a href="Somite-chap.html#fig:somite-phenos">5.8</a></strong>). I expected to observe an F2 distribution with a similar median to the F1, and a variance that spanned across the extremes of the F0 strains. Instead, I observed that for the period phenotypes, the F2 generation had a mean that was slightly higher than the median of the higher-period F0 <em>Cab</em> strain, and many F2 samples exceed the period values in those F0 samples. Our collaborators assured me that these observations were unlikely to be caused by technical issues. The <em>Cab</em> and <em>Kaga</em> strains originate from different Japanese medaka populations (southern and northern respectively) that are understood to be at the point of speciation (see Chapter <a href="MIKK-genomes-chap.html#MIKK-genomes-chap">2</a>), so this slower period may be driven by a biological incompatibility between their genomes in cases where they do not have a complete chromosome from each parent (as the F1 generation does). I nevertheless proceeded with the genetic analysis with a view to potentially discovering the reason for this unusual distribution.</p>

<div class="figure">
<span style="display:block;" id="fig:somite-phenos"></span>
<img src="figs/somites/phenotypes.png" alt="Comparisons between the F0, F1 and F2 generations for the three phenotypes of interest. Here, the F0 only includes Cab individuals. A: period intercept. B: period mean. C: unsegmented PSM area. \(P\)-values are derived from Kruskal-Wallis tests comparing the F2 individuals across microscopes." width="100%"><p class="caption">
Figure 5.8: Comparisons between the F0, F1 and F2 generations for the three phenotypes of interest. Here, the F0 only includes <em>Cab</em> individuals. <strong>A</strong>: period intercept. <strong>B</strong>: period mean. <strong>C</strong>: unsegmented PSM area. <span class="math inline">\(P\)</span>-values are derived from Kruskal-Wallis tests comparing the F2 individuals across microscopes.
</p>
</div>
<p>Another important issue to note is that the F2 individuals were sequenced using different microscopes, denoted as ‘AU’ and ‘DB’. Our collaborators noticed that there was a difference between the microscopes in their temperatures of 0.7-0.8°C, translating to a 4-minute difference in the F2 means for the period intercept measure (Kruskal-Wallis = 177.97, <span class="math inline">\(p\)</span> = 1.34x10<sup>-40</sup>), and a 3.5-minute difference in the F2 means for the period mean measure (Kruskal-Wallis = 141.79, <span class="math inline">\(p\)</span> = 1.08x10<sup>-32</sup>). This difference would need to be accounted for in the downstream analysis through either adjusting the phenotype prior to running the genetic association model, or by including microscope as a covariate in the model. No significant was found for the PSM area. To resolve this difference between microscopes for the period intercept data, I elected to transform it for the F2 generation by inverse-normalising the period intercept within each microscope (<strong>Figure <a href="Somite-chap.html#fig:invnorm-intercept">5.9</a></strong>). I then used this transformed data in the downstream analysis.</p>

<div class="figure">
<span style="display:block;" id="fig:invnorm-intercept"></span>
<img src="figs/somites/invnorm_intercept.png" alt="Comparison of the period intercept phenotype data for the F2 generation before (A) and after (B) inverse-normalisation." width="100%"><p class="caption">
Figure 5.9: Comparison of the period intercept phenotype data for the F2 generation before (<strong>A</strong>) and after (<strong>B</strong>) inverse-normalisation.
</p>
</div>
</div>
</div>
<div id="genetic-sequencing-data" class="section level2" number="5.3">
<h2>
<span class="header-section-number">5.3</span> Genetic sequencing data<a class="anchor" aria-label="anchor" href="#genetic-sequencing-data"><i class="fas fa-link"></i></a>
</h2>
<p>Our collaborators extracted DNA from the F0, F1, and F2, and sequenced the F0 and F1 samples with the Illumina platform at high coverage (~26x for <em>Cab</em> and ~29x for <em>Kaga</em>), as measured by samtools.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Danecek et al., &lt;span&gt;“Twelve Years of &lt;span&gt;SAMtools&lt;/span&gt; and &lt;span&gt;BCFtools&lt;/span&gt;.”&lt;/span&gt;&lt;/p&gt;"><sup>114</sup></a></span> <strong>Figure <a href="Somite-chap.html#fig:F0-coverage">5.10</a></strong> sets out the mean sequencing depth within each chromosome and across the whole genome for the <em>Cab</em> and <em>Kaga</em> F0 samples. Our collaborators then sequenced the F2 samples at low coverage (~1x), which would be sufficient to map their genotypes back to the genotypes of their parental strains (see Chapter <a href="Somite-chap.html#somite-f2-sequencing">5.6</a> for further details).</p>
<div class="figure">
<span style="display:block;" id="fig:F0-coverage"></span>
<img src="figs/somites/F0_coverage.png" alt="(ref:F0-coverage)" width="100%"><p class="caption">
Figure 5.10: (ref:F0-coverage)
</p>
</div>
</div>
<div id="f0-homozygosity-and-f1-heterozygosity" class="section level2" number="5.4">
<h2>
<span class="header-section-number">5.4</span> F0 homozygosity and F1 heterozygosity<a class="anchor" aria-label="anchor" href="#f0-homozygosity-and-f1-heterozygosity"><i class="fas fa-link"></i></a>
</h2>
<p>Before proceeding to map the F2 sequences to the genotypes of the F0 generation, I first investigated the levels of homozygosity in the F0 <em>Cab</em> and <em>Kaga</em> strains, as this would have an effect on the resolution of our genetic mapping. That is to say, for regions where either parent is consistently heterozygous, it would be difficult to determine the parent from which a particular F2 individual derived its chromosomes at that locus. I therefore aligned the high-coverage sequencing data for the F0 <em>Cab</em> and <em>Kaga</em> strains to the medaka <em>HdrR</em> reference (Ensembl release 104, build ASM223467v1) using BWA-MEM2, sorted the aligned .sam files, marked duplicate reads, and merged the paired reads with picard,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;&lt;span&gt;“Picard Toolkit.”&lt;/span&gt;&lt;/p&gt;"><sup>115</sup></a></span> and indexed the .bam files with Samtools.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Heng Li et al., &lt;span&gt;“The &lt;span&gt;Sequence&lt;/span&gt; Alignment/Map (&lt;span&gt;SAM&lt;/span&gt;) Format and &lt;span&gt;SAMtools&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Bioinformatics&lt;/em&gt; 25, no. 16 (2009): 2078–79.&lt;/p&gt;"><sup>116</sup></a></span></p>
<p>To call variants, I followed the GATK best practices (to the extent they were applicable)<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;McKenna et al., &lt;span&gt;“The &lt;span&gt;Genome Analysis Toolkit&lt;/span&gt;”&lt;/span&gt;; Mark A. DePristo et al., &lt;span&gt;“A Framework for Variation Discovery and Genotyping Using Next-Generation &lt;span&gt;DNA&lt;/span&gt; Sequencing Data,”&lt;/span&gt; &lt;em&gt;Nature Genetics&lt;/em&gt; 43, no. 5, 5 (May 2011): 491–98, &lt;a href="https://doi.org/10.1038/ng.806" role="doc-biblioref"&gt;https://doi.org/10.1038/ng.806&lt;/a&gt;; Geraldine A. Van der Auwera and Brian D. O’Connor, &lt;em&gt;Genomics in the Cloud: Using &lt;span&gt;Docker&lt;/span&gt;, &lt;span&gt;GATK&lt;/span&gt;, and &lt;span&gt;WDL&lt;/span&gt; in &lt;span&gt;Terra&lt;/span&gt;&lt;/em&gt; (&lt;span&gt;O’Reilly Media&lt;/span&gt;, 2020).&lt;/p&gt;'><sup>117</sup></a></span> with GATK’s HaplotypeCaller and GenotypeGVCFs tools,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Ryan Poplin et al., &lt;span&gt;“Scaling Accurate Genetic Variant Discovery to Tens of Thousands of Samples”&lt;/span&gt; (&lt;span&gt;bioRxiv&lt;/span&gt;, July 24, 2018), &lt;a href="https://doi.org/10.1101/201178" role="doc-biblioref"&gt;https://doi.org/10.1101/201178&lt;/a&gt;.&lt;/p&gt;'><sup>118</sup></a></span> then merged all calls into a single .vcf file with picard.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;&lt;span&gt;“Picard Toolkit.”&lt;/span&gt;&lt;/p&gt;"><sup>119</sup></a></span> Finally, I extracted the biallelic calls for <em>Cab</em> and <em>Kaga</em> with bcftools,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Danecek et al., &lt;span&gt;“Twelve Years of &lt;span&gt;SAMtools&lt;/span&gt; and &lt;span&gt;BCFtools&lt;/span&gt;.”&lt;/span&gt;&lt;/p&gt;"><sup>120</sup></a></span> counted the number of SNPs within non-overlapping, 5-kb bins, and calculated the proportion of SNPs within each bin that were homozgyous.</p>
<p><strong>Figure <a href="Somite-chap.html#fig:somite-f0-cab">5.11</a></strong> is a circos plot generated with circlize<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Zuguang Gu et al., &lt;span&gt;“Circlize Implements and Enhances Circular Visualization in &lt;span&gt;R&lt;/span&gt;,”&lt;/span&gt; &lt;em&gt;Bioinformatics&lt;/em&gt; 30, no. 19 (October 1, 2014): 2811–12, &lt;a href="https://doi.org/10.1093/bioinformatics/btu393" role="doc-biblioref"&gt;https://doi.org/10.1093/bioinformatics/btu393&lt;/a&gt;.&lt;/p&gt;'><sup>121</sup></a></span> for the <em>Cab</em> F0 strain used in this experiment, featuring the proportion of homozygous SNPs per 5-kb bin (green), and the total number of SNPs in each bin (yellow). As expected for a strain that has been inbred for over 10 generations, the mean homozygosity for this strain is high, with a mean proportion of homozygosity across all bins of 83%.</p>

<div class="figure">
<span style="display:block;" id="fig:somite-f0-cab"></span>
<img src="figs/somites/Cab.png" alt="Proportion of homozygous SNPs within 5 kb bins in the Cab F0 generation genome (green), and number of SNPs in each bin (yellow)." width="100%"><p class="caption">
Figure 5.11: Proportion of homozygous SNPs within 5 kb bins in the <em>Cab</em> F0 generation genome (green), and number of SNPs in each bin (yellow).
</p>
</div>
<p>However, the levels of homozygosity in the <em>Kaga</em> strain used in this experiment was far lower, with a mean homozygosity across all bins of only 31%. This was a surprise, as it is an established strain of X generations, and we therefore expected the level of homozygosity to be commensurate with that observed in the <em>Cab</em> strain.</p>

<div class="figure">
<span style="display:block;" id="fig:somite-f0-kaga"></span>
<img src="figs/somites/Kaga.png" alt="Proportion of homozygous SNPs within 5 kb bins in the Kaga F0 generation genome (red), and number of SNPs in each bin (yellow)." width="100%"><p class="caption">
Figure 5.12: Proportion of homozygous SNPs within 5 kb bins in the <em>Kaga</em> F0 generation genome (red), and number of SNPs in each bin (yellow).
</p>
</div>
<p>To determine whether the low levels of observed homozygosity in Kaga was affected by its alignments to the southern Japanese <em>HdrR</em> reference, we also aligned the F0 samples to the northern Japanese <em>HNI</em> reference (<strong>Figure <a href="Somite-chap.html#fig:somite-f0-kaga-hni">5.13</a></strong>. This did not make differences to the levels of observed homozygosity in either sample, which gave us confidence that the low homozygosity observed in <em>Kaga</em> was not driven by reference bias.</p>

<div class="figure">
<span style="display:block;" id="fig:somite-f0-kaga-hni"></span>
<img src="figs/somites/Kaga_HNI.png" alt="Proportion of homozygous SNPs within 5 kb bins in the Kaga F0 generation genome when aligned to the HNI reference (red), and number of SNPs in each bin (yellow)." width="100%"><p class="caption">
Figure 5.13: Proportion of homozygous SNPs within 5 kb bins in the <em>Kaga</em> F0 generation genome when aligned to the <em>HNI</em> reference (red), and number of SNPs in each bin (yellow).
</p>
</div>
</div>
<div id="f1-homozygosity" class="section level2" number="5.5">
<h2>
<span class="header-section-number">5.5</span> F1 homozygosity<a class="anchor" aria-label="anchor" href="#f1-homozygosity"><i class="fas fa-link"></i></a>
</h2>
<p>I next examined the level of heterozygosity in the F1 generation from the <em>Cab</em>-<em>Kaga</em> cross. <strong>Figure <a href="Somite-chap.html#fig:somite-f1">5.14</a></strong> shows the level of heterozygosity across the genome of the F1 hybrid in brown measured by the proportion of heterozygous SNPs within 5-kb bins (brown), and the number of SNPs in each bin (yellow). Approximately half the chromosomes show inconsistent heterozygosity, with a mean heterozygosity across all bins of 67%. This lower level of heterozygosity than expected was likely caused by the low levels of heterozygosity in the <em>Kaga</em> F0 parent.</p>

<div class="figure">
<span style="display:block;" id="fig:somite-f1"></span>
<img src="figs/somites/F1.png" alt="Proportion of heterozygous SNPs within 5 kb bins in the Cab-Kaga F1 cross (brown), and number of SNPs in each bin (yellow)." width="100%"><p class="caption">
Figure 5.14: Proportion of heterozygous SNPs within 5 kb bins in the <em>Cab</em>-<em>Kaga</em> F1 cross (brown), and number of SNPs in each bin (yellow).
</p>
</div>
<p>For the purpose of mapping the F2 sample sequences to the genomes of their parental strains, I selected only biallelic SNPs that were homozygous-divergent in the F0 generation (i.e. homozygous reference allele in <em>Cab</em> and homozygous alternative allele in <em>Kaga</em> or vice versa) <em>and</em> heterozygous in the F1 generation. The number of SNPs that met these criteria per chromosome are set out in <strong>Figure <a href="Somite-chap.html#fig:snp-counts-per-chrom">5.15</a></strong>.</p>

<div class="figure">
<span style="display:block;" id="fig:snp-counts-per-chrom"></span>
<img src="figs/somites/snp_counts_per_chrom.png" alt="Number of SNPs per chromosome that were homozygous-divergent in the F0 Cab and Kaga generations, and heterozygous in the F1 generation." width="100%"><p class="caption">
Figure 5.15: Number of SNPs per chromosome that were homozygous-divergent in the F0 <em>Cab</em> and <em>Kaga</em> generations, and heterozygous in the F1 generation.
</p>
</div>
</div>
<div id="somite-f2-sequencing" class="section level2" number="5.6">
<h2>
<span class="header-section-number">5.6</span> F2 genotyping<a class="anchor" aria-label="anchor" href="#somite-f2-sequencing"><i class="fas fa-link"></i></a>
</h2>
<p>To maximise the efficiency of our sequencing runs, our collaborators “shallow-sequenced” the F2 generation with the short-read Illumina platform at a depth of ~1x. We then aligned these sequences to the <em>HdrR</em> reference with BWA-MEM2,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Vasimuddin et al., &lt;span&gt;“Efficient Architecture-Aware Acceleration of &lt;span&gt;BWA-MEM&lt;/span&gt; for Multicore Systems.”&lt;/span&gt;&lt;/p&gt;"><sup>122</sup></a></span> sorted the reads and marked duplicates with Picard,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;&lt;span&gt;“Picard Toolkit.”&lt;/span&gt;&lt;/p&gt;"><sup>123</sup></a></span> then indexed the resulting BAM files with samtools.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Danecek et al., &lt;span&gt;“Twelve Years of &lt;span&gt;SAMtools&lt;/span&gt; and &lt;span&gt;BCFtools&lt;/span&gt;.”&lt;/span&gt;&lt;/p&gt;"><sup>124</sup></a></span> Genotyping these shallow sequences with the same method as used for the high-coverage sequences for the F0 and F1 generation would be inappropriate. We therefore used a different method whereby we used <em>bam-readcount</em><span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Khanna et al., &lt;span&gt;“Bam-Readcount - Rapid Generation of Basepair-Resolution Sequence Metrics.”&lt;/span&gt;&lt;/p&gt;"><sup>125</sup></a></span> to count the reads that supported either the <em>Cab</em> or the <em>Kaga</em> allele for all SNPs that met the criteria described above in section <a href="Somite-chap.html#f1-homozygosity">5.5</a>, summed the read counts within 5 kb blocks, and calculated the frequency of reads within each bin that supported the <em>Kaga</em> allele. This generated a value for each bin between 0 and 1, where 0 signified that all reads within that bin supported the <em>Cab</em> allele, and 1 signified that all reads within that bin supported the <em>Kaga</em> allele. Bins containing no reads were imputed with a value of 0.5.</p>
<p>I then used these values for all F2 individuals as the input to a Hidden Markov Model (HMM) with the software package <em>hmmlearn</em>,<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;&lt;em&gt;Hmmlearn/Hmmlearn&lt;/em&gt;.&lt;/p&gt;"><sup>126</sup></a></span> which I applied to classify each bin as one of three states, with state 0 corresponding to homozygous-<em>Cab</em>, 1 corresponding to heterozygous, and 2 corresponding to homozygous-<em>Kaga</em>.
Across each chromosome of every sample, the output of the HMM was expected to produce a sequence of states. Based on previous biological knowledge that crossover events occur on average less than once per chromosome<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content='&lt;p&gt;Quiterie Haenel et al., &lt;span&gt;“Meta-Analysis of Chromosome-Scale Crossover Rate Variation in Eukaryotes and Its Significance to Evolutionary Genomics,”&lt;/span&gt; &lt;em&gt;Molecular Ecology&lt;/em&gt; 27, no. 11 (2018): 2477–97, &lt;a href="https://doi.org/10.1111/mec.14699" role="doc-biblioref"&gt;https://doi.org/10.1111/mec.14699&lt;/a&gt;.&lt;/p&gt;'><sup>127</sup></a></span> (see <strong>Figure <a href="Somite-chap.html#fig:zebrafish-co-per-chrom">5.16</a></strong> for the average crossover rates per chromosome in zebrafish), I expected to observe the same state persisting for long stretches of the chromosome, only changing to another state between 0 and 3 times, and rarely more.</p>

<div class="figure">
<span style="display:block;" id="fig:zebrafish-co-per-chrom"></span>
<img src="05-somites_files/figure-html/zebrafish-co-per-chrom-1.png" alt="Crossovers per chromosome based on data provided in Haenel et al.128, where “crossovers per chromosome” for each chromosome \(c\) was calculated by \(\frac{crossover~rate_{c}(cM / Mb) \times length_{c}(Mb)} {100}\). The medaka genome is shorter in length than zebrafish genome (~800 Mb compared to ~1,300 Mb), which according to the authors would suggest that medaka likely has a higher average crossover rate than what is presented in this figure." width="100%"><p class="caption">
Figure 5.16: Crossovers per chromosome based on data provided in <span class="citation">Haenel et al.<a href="#fn128" class="footnote-ref" id="fnref128"><sup>128</sup></a></span>, where “crossovers per chromosome” for each chromosome <span class="math inline">\(c\)</span> was calculated by <span class="math inline">\(\frac{crossover~rate_{c}(cM / Mb) \times length_{c}(Mb)} {100}\)</span>. The medaka genome is shorter in length than zebrafish genome (~800 Mb compared to ~1,300 Mb), which according to the authors would suggest that medaka likely has a higher average crossover rate than what is presented in this figure.
</p>
</div>
<p><strong>Figure @(fig:hmm-standard)</strong> shows how adjusting the HMM parameters changed the called genotypes for 10 F2 samples on chromosome 18. Allowing the HMM to train itself for the transition probabilities and emission variances, the HMM produced an apparently noisy output (<strong>Figure @(fig:hmm-standard)A</strong>). Fixing the transition probabilities to make it very likely for a state to transition back to itself rather than to another state.</p>

<div class="figure">
<span style="display:block;" id="fig:hmm-scatter-diagnoses"></span>
<img src="figs/somites/scatter_collage.png" alt="HMM states called for each bin across chr18 for 10 F2 samples. States 0, 1, and 2 correspond to homozygous Cab, heterozygous, and homozygous Kaga. Each point represents a 5-kb bin. Y-axis is the proportion of reads within each bin that align to the Kaga allele. X-axis is the bp location of the start of each bin. A: Standard HMM with all model parameters trained on the data. B. HMM with fixed transition probabilities of 0→0 or 1→1 or 2→2 = 0.999; 0→1 or 2→1 = 0.00066; 0→2 or 2→0 = 0.000333; 1→0 or 1→2 = 0.0005. C-F retain those transition probabilities but with different fixed emission variances of 0.01 (C), 0.33 (D), 0.8 (E), and 1 (F)." width="100%"><p class="caption">
Figure 5.17: HMM states called for each bin across chr18 for 10 F2 samples. States 0, 1, and 2 correspond to homozygous <em>Cab</em>, heterozygous, and homozygous <em>Kaga</em>. Each point represents a 5-kb bin. Y-axis is the proportion of reads within each bin that align to the <em>Kaga</em> allele. X-axis is the bp location of the start of each bin. <strong>A</strong>: Standard HMM with all model parameters trained on the data. <strong>B</strong>. HMM with fixed transition probabilities of 0→0 or 1→1 or 2→2 = 0.999; 0→1 or 2→1 = 0.00066; 0→2 or 2→0 = 0.000333; 1→0 or 1→2 = 0.0005. <strong>C</strong>-<strong>F</strong> retain those transition probabilities but with different fixed emission variances of 0.01 (<strong>C</strong>), 0.33 (<strong>D</strong>), 0.8 (<strong>E</strong>), and 1 (<strong>F</strong>).
</p>
</div>
<p>I used these genotype-block calls to generate the recombination karyoplots shown in <strong>Figures <a href="Somite-chap.html#fig:karyo-wi-missing">5.18</a></strong> and <strong><a href="Somite-chap.html#fig:karyo-no-missing">5.19</a></strong>.</p>

<div class="figure">
<span style="display:block;" id="fig:karyo-wi-missing"></span>
<img src="figs/somites/karyoplot_wi_missing.png" alt="Recombination blocks in 622 F2 samples based on the ratio of reads mapping to either the Cab or Kaga allele within 5-kb bins, with homozygous-Cab blocks in green, heterozoygous blocks in navy blue, and homozygous Kaga blocks in red. Most blocks show 0-2 crossover events, as expected, with some regions showing higher numbers of crossovers interpreted as noise. Unfilled regions are those with no state called by the HMM due to a lack of reads mapping to SNPs within those 5-kb bins." width="100%"><p class="caption">
Figure 5.18: Recombination blocks in 622 F2 samples based on the ratio of reads mapping to either the <em>Cab</em> or <em>Kaga</em> allele within 5-kb bins, with homozygous-<em>Cab</em> blocks in green, heterozoygous blocks in navy blue, and homozygous <em>Kaga</em> blocks in red. Most blocks show 0-2 crossover events, as expected, with some regions showing higher numbers of crossovers interpreted as noise. Unfilled regions are those with no state called by the HMM due to a lack of reads mapping to SNPs within those 5-kb bins.
</p>
</div>
<div class="figure">
<span style="display:block;" id="fig:karyo-no-missing"></span>
<img src="figs/somites/karyoplot_no_missing.png" alt="(ref:karyo-no-missing)" width="100%"><p class="caption">
Figure 5.19: (ref:karyo-no-missing)
</p>
</div>
<p><strong>Figure <a href="Somite-chap.html#fig:prop-sites-total">5.20</a></strong> shows the proportion of 5-kb bins called as either homozygous-<em>Cab</em>, heterozygous, or homozygous-<em>Kaga</em> within each F2 sample (points). The ordinary expectation for the ratios would be 0.25, 0.5, and 0.25 respectively. However, we observe a skew towards homozygous-<em>Cab</em> and away from homozygous <em>Kaga</em>. This may have been caused by the low homozygosity observed in <em>Kaga</em>.</p>

<div class="figure">
<span style="display:block;" id="fig:prop-sites-total"></span>
<img src="figs/somites/prop_sites_total.png" alt="Proportions of 5-kb blocks called as either homozygous-Cab, heterozygous, or homozygous-Kaga." width="100%"><p class="caption">
Figure 5.20: Proportions of 5-kb blocks called as either homozygous-<em>Cab</em>, heterozygous, or homozygous-<em>Kaga</em>.
</p>
</div>
</div>
<div id="genome-wide-linkage-anlaysis" class="section level2" number="5.7">
<h2>
<span class="header-section-number">5.7</span> Genome-wide linkage anlaysis<a class="anchor" aria-label="anchor" href="#genome-wide-linkage-anlaysis"><i class="fas fa-link"></i></a>
</h2>
<p>Finally, I used the called recombination blocks as pseudo-SNPs in a genetic linkage analysis. To detect associations between the pseudo-SNPs and the three phenotypes of interest, I used a mixed linear model (<strong>MLM</strong>) as implemented in GCTA.<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Yang et al., &lt;span&gt;“&lt;span&gt;GCTA&lt;/span&gt;.”&lt;/span&gt;&lt;/p&gt;"><sup>129</sup></a></span> That paper describes the model as follows:</p>
<p><span class="math display">\[
\textbf{y} = \textbf{X}\beta + \textbf{Wu} + \epsilon \\
var(\textbf{y}) = \textbf{V} = \textbf{WW}'\sigma^2_{u} + \textbf{I}\sigma_{\epsilon}^2
\]</span>
Where <span class="math inline">\(\textbf{y}\)</span> is a <span class="math inline">\(n\)</span> x 1 vector of phenotypes with <span class="math inline">\(n\)</span> being the sample size, <span class="math inline">\(\textbf{W}\)</span> is a standardised genotype matrix, <span class="math inline">\(\textbf{u}\)</span> is a vector of SNP effects, and <span class="math inline">\(\epsilon\)</span> is a vector of residual effects. I additionally used the leave-one-chromosome-out implementation of GCTA’s MLM, with excludes the chromosome on which the candidate SNP is located when calculating the GRM.</p>
<p>As described above in <a href="Somite-chap.html#somite-phenotype">5.2</a>, the microscope used to image the embryos (either AU or DB) differed by several degrees in heat, which likely caused differences in the measurements observed. We accordingly experimented with including microscope as a covariate, either alone or together with the genotype for the reporter locus (either homozygous or heterozygous), or excluding it altogether. In an attempt to avoid complications resulting from its inclusion, we also tried inverse-normalising the period phenotype within each microscope group, transforming the phenotype to fit a normal distribution across both microscopes.</p>
<p>To set the significance threshold, I permuted the phenotype across samples using 10 different random seeds, together with all covariates when included, and ran a separate linkage model for each permutation. I then set the lowest <span class="math inline">\(p\)</span>-value from all 10 permutation as the significance threshold for the non-permuted model. I additionally applied a Bonferroni correction to our <span class="math inline">\(p\)</span>-values by dividing <span class="math inline">\(\alpha\)</span> (0.05) by the number of pseudo-SNPs in the model, and set this as a secondary threshold.</p>
<div id="period-intercept" class="section level3" number="5.7.1">
<h3>
<span class="header-section-number">5.7.1</span> Period intercept<a class="anchor" aria-label="anchor" href="#period-intercept"><i class="fas fa-link"></i></a>
</h3>
<p><strong>Figure <a href="Somite-chap.html#fig:somite-manhattan">5.21</a></strong> is a Manhattan plot of the genetic linkage results for the period intercept phenotype, inverse-normalised across microscopes. The regions found to be significant based on the permutations’ minimum <span class="math inline">\(p\)</span>-value are set out in <strong>Table <a href="Somite-chap.html#tab:somite-sig-int-tbl">5.1</a></strong>.</p>

<div class="figure">
<span style="display:block;" id="fig:somite-manhattan"></span>
<img src="figs/somites/manhattan_intercept.png" alt="Manhattan plot of the genetic linkage results for the period intercept phenotype, inverse-normalised across microscopes. Pseudo-SNPs with \(p\)-values lower than the permutation significance threshold are highlighted in red." width="100%"><p class="caption">
Figure 5.21: Manhattan plot of the genetic linkage results for the period intercept phenotype, inverse-normalised across microscopes. Pseudo-SNPs with <span class="math inline">\(p\)</span>-values lower than the permutation significance threshold are highlighted in red.
</p>
</div>
<div class="inline-table"><table class="table table-sm">
<caption>
<span id="tab:somite-sig-int-tbl">Table 5.1: </span>Significant 5-kb bin ranges for period intercept below the minimum p-value from 10 permutations.</caption>
<thead><tr class="header">
<th align="right">Chromosome</th>
<th align="right">Bin start</th>
<th align="right">Bin end</th>
<th align="right">Length (kb)</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="right">3</td>
<td align="right">31880001</td>
<td align="right">35420000</td>
<td align="right">3540</td>
</tr>
<tr class="even">
<td align="right">4</td>
<td align="right">18090001</td>
<td align="right">18095000</td>
<td align="right">5</td>
</tr>
<tr class="odd">
<td align="right">10</td>
<td align="right">2995001</td>
<td align="right">3690000</td>
<td align="right">695</td>
</tr>
</tbody>
</table></div>
<p>These regions contained a total of 46,872 SNPs imputed from the genotype of the F0 parental strains.
I ran Ensembl’s Variant Effect Predictor<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;McLaren et al., &lt;span&gt;“The &lt;span&gt;Ensembl Variant Effect Predictor&lt;/span&gt;.”&lt;/span&gt;&lt;/p&gt;"><sup>130</sup></a></span> over these SNPs to identify those that would be most likely to have functional consequences. The full counts of SNPs falling into each category of ‘consequence’ are set out in <strong>Table <a href="Somite-chap.html#tab:int-consequence-tbl">5.2</a></strong>. From this process I identified 38 genes that included a missense variant, 1 that included a missense variant and a start lost (ENSORLG00000014616), ane 1 that included a missense variant and a stop lost (ENSORLG00000015149).</p>
<div class="inline-table"><table class="table table-sm">
<caption>
<span id="tab:int-consequence-tbl">Table 5.2: </span>Variant Effect Predictor results for SNPs in the bins.</caption>
<thead><tr class="header">
<th align="left">Consequence</th>
<th align="right">Count</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">intron variant</td>
<td align="right">47211</td>
</tr>
<tr class="even">
<td align="left">intergenic variant</td>
<td align="right">20045</td>
</tr>
<tr class="odd">
<td align="left">upstream gene variant</td>
<td align="right">7304</td>
</tr>
<tr class="even">
<td align="left">downstream gene variant</td>
<td align="right">5229</td>
</tr>
<tr class="odd">
<td align="left">3 prime UTR variant</td>
<td align="right">1082</td>
</tr>
<tr class="even">
<td align="left">synonymous variant</td>
<td align="right">694</td>
</tr>
<tr class="odd">
<td align="left">missense variant</td>
<td align="right">383</td>
</tr>
<tr class="even">
<td align="left">5 prime UTR variant</td>
<td align="right">201</td>
</tr>
<tr class="odd">
<td align="left">splice region variant,intron variant</td>
<td align="right">126</td>
</tr>
<tr class="even">
<td align="left">missense variant,splice region variant</td>
<td align="right">19</td>
</tr>
<tr class="odd">
<td align="left">splice region variant,synonymous variant</td>
<td align="right">17</td>
</tr>
<tr class="even">
<td align="left">stop gained</td>
<td align="right">3</td>
</tr>
<tr class="odd">
<td align="left">splice donor variant</td>
<td align="right">1</td>
</tr>
<tr class="even">
<td align="left">start lost</td>
<td align="right">1</td>
</tr>
<tr class="odd">
<td align="left">stop lost</td>
<td align="right">1</td>
</tr>
<tr class="even">
<td align="left">stop lost,splice region variant</td>
<td align="right">1</td>
</tr>
</tbody>
</table></div>
<p>Our collaborators then combined these results with bulk RNA-seq that they had performed on F0 <em>Cab</em> and <em>Kaga</em> individuals, to determine which of these genes are expressed in the tail during embryogenesis. This allowed them to reduce to the list to 29 genes, and a gene ontology analysis of this found that the list of genes was enriched for body axis, somitogenesis, and segmentation (<strong>Table <a href="Somite-chap.html#tab:psm-final-genes">5.3</a></strong>). For this list of genes, our collaborators are now in the process of knocking in the protein-altering <em>Cab</em> allele into <em>Kaga</em> embryos, and <em>vice versa</em>, to functionally validate these variants.</p>
<div class="inline-table"><table class="table table-sm">
<caption>
<span id="tab:psm-final-genes">Table 5.3: </span>Target genes for functional validation expressed in the unsegmented PSM and containing protein alterations. Table generated by Ali Seleit.</caption>
<colgroup>
<col width="10%">
<col width="12%">
<col width="38%">
<col width="38%">
</colgroup>
<thead><tr class="header">
<th align="right">chromosome_name</th>
<th align="left">ensembl_gene_id</th>
<th align="left">description</th>
<th align="left">Role</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="right">3</td>
<td align="left">ENSORLG00000014656</td>
<td align="left">mesoderm posterior protein 2-like</td>
<td align="left">Somitogenesis</td>
</tr>
<tr class="even">
<td align="right">3</td>
<td align="left">ENSORLG00000014659</td>
<td align="left">mesp</td>
<td align="left">Somitogenesis</td>
</tr>
<tr class="odd">
<td align="right">10</td>
<td align="left">ENSORLG00000020474</td>
<td align="left">protocadherin 10b</td>
<td align="left">Somitogenesis</td>
</tr>
<tr class="even">
<td align="right">3</td>
<td align="left">ENSORLG00000014616</td>
<td align="left">NA</td>
<td align="left">Possible role in Somitogenesis (MAP-kinase)</td>
</tr>
<tr class="odd">
<td align="right">10</td>
<td align="left">ENSORLG00000020551</td>
<td align="left">FAT atypical cadherin 4</td>
<td align="left">Possible role in Somitogenesis (PCP, Yap1 regulator)</td>
</tr>
<tr class="even">
<td align="right">10</td>
<td align="left">ENSORLG00000020531</td>
<td align="left">neurogenin 1</td>
<td align="left">Possible role in Somitogenesis (BHLH-TF regulation of Wnt)</td>
</tr>
<tr class="odd">
<td align="right">3</td>
<td align="left">ENSORLG00000015149</td>
<td align="left">ADAM metallopeptidase with thrombospondin type 1 motif 18</td>
<td align="left">Extracellular matrix</td>
</tr>
<tr class="even">
<td align="right">3</td>
<td align="left">ENSORLG00000015418</td>
<td align="left">matrix metallopeptidase 15</td>
<td align="left">Extracellular matrix</td>
</tr>
<tr class="odd">
<td align="right">10</td>
<td align="left">ENSORLG00000020488</td>
<td align="left">transforming growth factor beta induced</td>
<td align="left">Extracellular matrix</td>
</tr>
<tr class="even">
<td align="right">3</td>
<td align="left">ENSORLG00000028055</td>
<td align="left">NA</td>
<td align="left"> Signal trasduction (Rho)</td>
</tr>
<tr class="odd">
<td align="right">10</td>
<td align="left">ENSORLG00000020481</td>
<td align="left">ArfGAP with RhoGAP domain, ankyrin repeat and PH domain 3</td>
<td align="left"> Signal transduction (GTPase activator RhoGap)</td>
</tr>
<tr class="even">
<td align="right">10</td>
<td align="left">ENSORLG00000020525</td>
<td align="left">TBC1 domain family member 2A-like</td>
<td align="left">Signal transduction Gtpase activator, cadherin recycling</td>
</tr>
<tr class="odd">
<td align="right">3</td>
<td align="left">ENSORLG00000015260</td>
<td align="left">synapse associated protein 1</td>
<td align="left">TOR acitivity</td>
</tr>
<tr class="even">
<td align="right">10</td>
<td align="left">ENSORLG00000028553</td>
<td align="left">NA</td>
<td align="left">NA</td>
</tr>
<tr class="odd">
<td align="right">3</td>
<td align="left">ENSORLG00000015460</td>
<td align="left">dpy-19 like C-mannosyltransferase 3</td>
<td align="left">glycosylation</td>
</tr>
<tr class="even">
<td align="right">10</td>
<td align="left">ENSORLG00000022010</td>
<td align="left">beta-1,4-galactosyltransferase 1</td>
<td align="left">glycosylation</td>
</tr>
<tr class="odd">
<td align="right">10</td>
<td align="left">ENSORLG00000020498</td>
<td align="left">protein NipSnap homolog 3A</td>
<td align="left">Mitochondria</td>
</tr>
<tr class="even">
<td align="right">10</td>
<td align="left">ENSORLG00000020494</td>
<td align="left">nitric oxide associated 1</td>
<td align="left">Mitochondria</td>
</tr>
<tr class="odd">
<td align="right">10</td>
<td align="left">ENSORLG00000020504</td>
<td align="left">ATP-binding cassette sub-family A member 1</td>
<td align="left">Metabolism (cholestrol efflux)</td>
</tr>
<tr class="even">
<td align="right">3</td>
<td align="left">ENSORLG00000015118</td>
<td align="left">phosphorylase kinase regulatory subunit beta</td>
<td align="left">Metabolism</td>
</tr>
<tr class="odd">
<td align="right">10</td>
<td align="left">ENSORLG00000020493</td>
<td align="left">RE1 silencing transcription factor</td>
<td align="left">RNA pol II (transcritpion regulation</td>
</tr>
<tr class="even">
<td align="right">3</td>
<td align="left">ENSORLG00000015365</td>
<td align="left">solute carrier family 7 member 6 opposite strand</td>
<td align="left">RNA pol II (nuclear export)</td>
</tr>
<tr class="odd">
<td align="right">10</td>
<td align="left">ENSORLG00000029052</td>
<td align="left">exosome component 3</td>
<td align="left">rRNA + RNA binding</td>
</tr>
<tr class="even">
<td align="right">3</td>
<td align="left">ENSORLG00000015278</td>
<td align="left">adhesion G protein-coupled receptor G3</td>
<td align="left">G-protein coupled receptor</td>
</tr>
<tr class="odd">
<td align="right">3</td>
<td align="left">ENSORLG00000015287</td>
<td align="left">adhesion G-protein coupled receptor G5</td>
<td align="left">G-protein coupled receptor</td>
</tr>
<tr class="even">
<td align="right">10</td>
<td align="left">ENSORLG00000025674</td>
<td align="left">matrin-3</td>
<td align="left">inner nuclear protein (chromatin architecture</td>
</tr>
<tr class="odd">
<td align="right">10</td>
<td align="left">ENSORLG00000023325</td>
<td align="left">matrin-3</td>
<td align="left">inner nuclear protein (chromatin architecture</td>
</tr>
<tr class="even">
<td align="right">10</td>
<td align="left">ENSORLG00000022388</td>
<td align="left">poly(A) binding protein interacting protein 2</td>
<td align="left">Translation repressor</td>
</tr>
<tr class="odd">
<td align="right">3</td>
<td align="left">ENSORLG00000015096</td>
<td align="left">integrin alpha FG-GAP repeat containing 1</td>
<td align="left">NA possibly T cell activation</td>
</tr>
</tbody>
</table></div>
</div>
<div id="psm-area-1" class="section level3" number="5.7.2">
<h3>
<span class="header-section-number">5.7.2</span> PSM area<a class="anchor" aria-label="anchor" href="#psm-area-1"><i class="fas fa-link"></i></a>
</h3>
<p><strong>Figure <a href="Somite-chap.html#fig:psm-manhattan">5.22</a></strong> is a Manhattan plot of the genetic linkage results for the PSM area phenotype. The regions found to be significant based on the permutations’ minimum <span class="math inline">\(p\)</span>-value are set out in <strong>Table <a href="Somite-chap.html#tab:somite-sig-psm-tbl">5.4</a></strong>, although they exceed the Bonferroni correction threshold as well. I note that this ~6 Mb significant region on chromosome 3 does not overlap at all with the significant region discovered for the period intercept phenotype.</p>

<div class="figure">
<span style="display:block;" id="fig:psm-manhattan"></span>
<img src="figs/somites/manhattan_psm_no-covariates.png" alt="Manhattan plot of the genetic linkage results for the PSM area phenotype. Pseudo-SNPs with \(p\)-values lower than the permutation significance threshold are highlighted in yellow." width="100%"><p class="caption">
Figure 5.22: Manhattan plot of the genetic linkage results for the PSM area phenotype. Pseudo-SNPs with <span class="math inline">\(p\)</span>-values lower than the permutation significance threshold are highlighted in yellow.
</p>
</div>
<div class="inline-table"><table class="table table-sm">
<caption>
<span id="tab:somite-sig-psm-tbl">Table 5.4: </span>Significant 5-kb bin range for PSM area below the minimum p-value from 10 permutations.</caption>
<thead><tr class="header">
<th align="right">CHROM</th>
<th align="right">Bin start</th>
<th align="right">Bin end</th>
<th align="right">Length (kb)</th>
</tr></thead>
<tbody><tr class="odd">
<td align="right">3</td>
<td align="right">20375001</td>
<td align="right">26285000</td>
<td align="right">5910</td>
</tr></tbody>
</table></div>
<p>This region contained a total of 29,096 SNPs imputed from the genotype of the F0 parental strains.
I ran Ensembl’s Variant Effect Predictor<span class="citation"><a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;McLaren et al.&lt;/p&gt;"><sup>131</sup></a></span> over these SNPs to identify those that would be most likely to have functional consequences. The full counts of SNPs falling into each category of ‘consequence’ are set out in <strong>Table <a href="Somite-chap.html#tab:psm-consequence-tbl">5.5</a></strong>. From this process I identified 114 genes that included a missense variant, and 1 that included a both a missense variant and a start lost (ENSORLG00000010863, a centriole, cilia and spindle-associated protein).</p>
<div class="inline-table"><table class="table table-sm">
<caption>
<span id="tab:psm-consequence-tbl">Table 5.5: </span>Variant Effect Predictor results for SNPs in the bins.</caption>
<thead><tr class="header">
<th align="left">Consequence</th>
<th align="right">Count</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">intron variant</td>
<td align="right">23189</td>
</tr>
<tr class="even">
<td align="left">intergenic variant</td>
<td align="right">9171</td>
</tr>
<tr class="odd">
<td align="left">downstream gene variant</td>
<td align="right">8894</td>
</tr>
<tr class="even">
<td align="left">upstream gene variant</td>
<td align="right">8491</td>
</tr>
<tr class="odd">
<td align="left">3 prime UTR variant</td>
<td align="right">2104</td>
</tr>
<tr class="even">
<td align="left">synonymous variant</td>
<td align="right">1141</td>
</tr>
<tr class="odd">
<td align="left">missense variant</td>
<td align="right">716</td>
</tr>
<tr class="even">
<td align="left">5 prime UTR variant</td>
<td align="right">433</td>
</tr>
<tr class="odd">
<td align="left">splice region variant,intron variant</td>
<td align="right">184</td>
</tr>
<tr class="even">
<td align="left">splice region variant,synonymous variant</td>
<td align="right">18</td>
</tr>
<tr class="odd">
<td align="left">missense variant,splice region variant</td>
<td align="right">7</td>
</tr>
<tr class="even">
<td align="left">stop gained</td>
<td align="right">2</td>
</tr>
<tr class="odd">
<td align="left">splice donor variant</td>
<td align="right">1</td>
</tr>
<tr class="even">
<td align="left">start lost</td>
<td align="right">1</td>
</tr>
</tbody>
</table></div>
<p>Our collaborators then combined these results with bulk RNA-seq that they had performed on F0 <em>Cab</em> and <em>Kaga</em> individuals, to determine which of these genes are expressed in the unsegmented tail during embryogenesis. This allowed them to reduce to the list to 96 genes, although they were not apparently associated with a specific gene ontology. As with the period intercept phenotype, our collaborators are now in the process of knocking in the <em>Cab</em> allele into <em>Kaga</em> embryos, and <em>vice versa</em>, to functionally validate these variants.</p>

</div>
</div>
</div>

  <div class="chapter-nav">
<div class="prev"><a href="MIKK-F2-chap.html"><span class="header-section-number">4</span> Genetic linkage study of bold/shy behaviours in the MIKK panel</a></div>
<div class="next"><a href="Fst-chap.html"><span class="header-section-number">6</span> Variation in the frequency of trait-associated alleles across global human populations</a></div>
</div></main><div class="col-md-3 col-lg-2 d-none d-md-block sidebar sidebar-chapter">
    <nav id="toc" data-toggle="toc" aria-label="On this page"><h2>On this page</h2>
      <ul class="nav navbar-nav">
<li><a class="nav-link" href="#Somite-chap"><span class="header-section-number">5</span> Genetic loci associated with somite development periodicity</a></li>
<li><a class="nav-link" href="#background"><span class="header-section-number">5.1</span> Background</a></li>
<li>
<a class="nav-link" href="#somite-phenotype"><span class="header-section-number">5.2</span> Phenotypes of interest</a><ul class="nav navbar-nav">
<li><a class="nav-link" href="#somite-development-period"><span class="header-section-number">5.2.1</span> Somite development period</a></li>
<li><a class="nav-link" href="#psm-area"><span class="header-section-number">5.2.2</span> PSM area</a></li>
<li><a class="nav-link" href="#comparisons-between-f0-f1-and-f2-generations"><span class="header-section-number">5.2.3</span> Comparisons between F0, F1 and F2 generations</a></li>
</ul>
</li>
<li><a class="nav-link" href="#genetic-sequencing-data"><span class="header-section-number">5.3</span> Genetic sequencing data</a></li>
<li><a class="nav-link" href="#f0-homozygosity-and-f1-heterozygosity"><span class="header-section-number">5.4</span> F0 homozygosity and F1 heterozygosity</a></li>
<li><a class="nav-link" href="#f1-homozygosity"><span class="header-section-number">5.5</span> F1 homozygosity</a></li>
<li><a class="nav-link" href="#somite-f2-sequencing"><span class="header-section-number">5.6</span> F2 genotyping</a></li>
<li>
<a class="nav-link" href="#genome-wide-linkage-anlaysis"><span class="header-section-number">5.7</span> Genome-wide linkage anlaysis</a><ul class="nav navbar-nav">
<li><a class="nav-link" href="#period-intercept"><span class="header-section-number">5.7.1</span> Period intercept</a></li>
<li><a class="nav-link" href="#psm-area-1"><span class="header-section-number">5.7.2</span> PSM area</a></li>
</ul>
</li>
</ul>

      <div class="book-extra">
        <ul class="list-unstyled">
<li><a id="book-source" href="https://github.com/brettellebi/PhD-thesis/blob/master/book/05-somites.Rmd">View source <i class="fab fa-github"></i></a></li>
          <li><a id="book-edit" href="https://github.com/brettellebi/PhD-thesis/edit/master/book/05-somites.Rmd">Edit this page <i class="fab fa-github"></i></a></li>
        </ul>
</div>
    </nav>
</div>

</div>
</div> <!-- .container -->

<footer class="bg-primary text-light mt-5"><div class="container"><div class="row">

  <div class="col-12 col-md-6 mt-3">
    <p>"<strong>Japanese courage: a genetic analysis of complex traits in medaka fish and humans</strong>" was written by Ian Brettell. It was last built on 2022-08-21.</p>
  </div>

  <div class="col-12 col-md-6 mt-3">
    <p>This book was built by the <a class="text-light" href="https://bookdown.org">bookdown</a> R package.</p>
  </div>

</div></div>
</footer><!-- dynamically load mathjax for compatibility with self-contained --><script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.9/latest.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script><script type="text/x-mathjax-config">const popovers = document.querySelectorAll('a.footnote-ref[data-toggle="popover"]');
for (let popover of popovers) {
  const div = document.createElement('div');
  div.setAttribute('style', 'position: absolute; top: 0, left:0; width:0, height:0, overflow: hidden; visibility: hidden;');
  div.innerHTML = popover.getAttribute('data-content');

  var has_math = div.querySelector("span.math");
  if (has_math) {
    document.body.appendChild(div);
    MathJax.Hub.Queue(["Typeset", MathJax.Hub, div]);
    MathJax.Hub.Queue(function() {
      popover.setAttribute('data-content', div.innerHTML);
      document.body.removeChild(div);
    })
  }
}
</script>
</body>
</html>
