# Genetic linkage study of bold/shy behaviours in the MIKK panel {#MIKK-F2-chap}

\chaptermark{MIKK panel behaviour}

```{r, include = F}
library(tidyverse)

# Read in line colours
line_cols = readr::read_csv(here::here("results/MIKK_F2/line_colours_0.08.csv"))

# Function for colouring text
colorize <- function(x, color, col_name) {
  if (knitr::is_latex_output()) {
    # remove hash prefix
    color = stringr::str_remove(color, "#")
    sprintf("\\definecolor{%s}{HTML}{%s}\\textcolor{%s}{%s}", col_name, color, col_name, x)
  } else if (knitr::is_html_output()) {
    sprintf("<span style='color: %s;'>%s</span>", color,
      x)
  } else x
}

# Function for pulling the line's matching colour from the data frame, then calling `colorize()`
colour_line = function(LINE){
  COLOUR = line_cols %>% 
    dplyr::filter(line == LINE) %>% 
    dplyr::pull(colour)
  # make colour name
  COL_NAME = paste(LINE,
                   stringr::str_remove(COLOUR, "#"),
                   sep = "_")
  # call colorize function
  colorize(LINE, COLOUR, COL_NAME)
}
```

The purpose of the study described in this chapter was to run the behavioural analysis described in Chapter \@ref(Pilot-chap) over the MIKK panel described in Chapter \@ref(MIKK-genomes-chap), identify the lines that diverged in both (a) their own behaviour; and (b) the level of transmission of their behaviour onto their *`r colour_line("iCab")`* reference tank partner, and then use them as the parental strains in an F2 cross to attempt to identify the specific genetic loci associated with those differences.

## Data collection - F0 generation

In November 2019 I traveled to the fish facility managed by our collaborators, the Loosli Group at KIT in Karlsruhe, Germany, and over the course of 11 days from 11 to 21 November 2019, I ran the behavioural assay described in Chapter \@ref(pilot-data-collection) another 206 times. I again used the *`r colour_line("iCab")`* strain as the reference fish, and for the test fish I used either an individual from one of the MIKK panel lines, individuals captured from the same Kiyosu population as the MIKK panel but permitted to breed freely within a separate tank in the facility ('Kiyosu closed-capture', or '**Kiyosu CC**'), or individuals from a related but different species of medaka from the Philippines, *Oryzias luzonensis*. I ensured that I performed at least 2 assay runs of 4 individuals each on two different days for each MIKK panel line that was available, generating a minimum of 8 test fish replicates per line. As there were four pairs of fish in the test tank during each run, the complete dataset comprises 824 videos of pairs of fish, which I further divided by assay component (open field and novel object) to create 1648 videos. 

I again used the software *idtrackerai* [@romero-ferreroIdtrackerAiTracking2019] to track the movement of the fishes across frames of each video. After adjusting the software parameters for each video to maximise the number of frames that were successfully tracked, I was left with 1610 out of the 1648 videos (~97.7%) where both fishes were tracked over at least 85% of frames, and I only included these 1610 videos in the downstream analysis. The first question to address was whether the MIKK panel lines differed in their behaviours. I therefore computed each individual fish's mean speed (measured as the distance traveled in pixels per 0.08 seconds) over the course of the full 20-minute video, grouped them by line, and plotted the results presented in **Figure \@ref(fig:mikk-mean-speed)**. I continue to use the same order and colour palette for the MIKK panel lines as in this Figure throughout the rest of this Chapter.

(ref:mikk-mean-speed) Mean speed of the MIKK panel and other strains over the course of the entire 20-minute video (measured as the distance traveled in pixels per 0.08 seconds). *`r colour_line("iCab")`* fishes in the *`r colour_line("iCab")`*-*`r colour_line("iCab")`* control condition are at the top, the MIKK panel lines are sorted by their group median, and the Kiyosu closed capture and *O. luzonensis* fishes are at the bottom.

```{r mikk-mean-speed, echo = F, fig.cap = '(ref:mikk-mean-speed)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/line_mean_speed_0.08_all.png"), auto_pdf = T)
```

This figure shows that there are clear differences between some MIKK panel lines at the extremes, and that the lines differ in the amount of within-line variance observed (shown plotted against the lines' median speed in **Figure \@ref(fig:mikk-mean-speed-variance)**). These figures acted as a general guide to determine which lines to select as the parental strains in the F2 cross. To identify genetic variants directly associated with bold-shy behaviours, I sought to select lines that showed either high or low levels of movement, and preferably low within-line variance. 

(ref:mikk-mean-speed-variance) Line median (vertical axis) and line variance (horizontal axis) for individual mean speed (pixels per 0.08 seconds) across the full 20-minute video (i.e. both the open field and novel object assay components).

```{r mikk-mean-speed-variance, echo = F, fig.cap = '(ref:mikk-mean-speed-variance)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/line_mean_speed_variance_0.08_all.png"), auto_pdf = T)
```

To illustrate the differences between certain lines in terms of both direct and social genetic effects, in **Figure \@ref(fig:extreme-paths)** I have plotted the tracked paths for 3 lines 5 minutes into the 10-minute open field assay component: the slowest line `r colour_line("22-1 (‘David’)")`,^[In almost all cases, David remained as still (and captivating) as Michelangelo's statue for the entirety of the assay.] one of the other slowest lines `r colour_line("18-2 (‘Elsa’)")`,^[Similar to David, Elsa was also Frozen, but far more charismatic.] and the fastest line `r colour_line("10-1 ('Janeway')")`. The fishes are coloured by their line, with *`r colour_line("iCab")`* in dark grey. There appears to be a social genetic effect when comparing the behaviour of *`r colour_line("iCab")`* when paired with the similarly slow-moving lines `r colour_line("22-1 (‘David’")` and `r colour_line("18-2 (‘Elsa’)")`. At the other extreme, line `r colour_line("10-1 (‘Janeway’)")`^[Janeway appeared to spend most of its time moving at warp speed.] has moved extremely quickly, spending much of its time moving along the boundaries of their test tanks. I understand from our collaborator Felix Loosli, a fish behaviour expert, that this movement along the boundaries of a tank is typical of medaka when introduced to a novel environment, and we observe it too with *`r colour_line("iCab")`* when it is paired with line `r colour_line("22-1 (‘David’)")`. 

(ref:extreme-paths) Path plots for lines 22-1 (**A**), 18-2 (**B**) and 10-1 (**C**) 5 minutes into the open field assay for the first (left) and second (right) run with each line. The paths of *`r colour_line("iCab")`* individuals are coloured in dark grey, with the other lines depicted in their representative colours.

```{r extreme-paths, echo = F, fig.cap = '(ref:extreme-paths)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/path_plot_22-1_18-2_10-1_300.png"), auto_pdf = T)
```

Although lines `r colour_line("22-1 (‘David’)")` and `r colour_line("10-1 (‘Janeway’)")` were the most extreme in terms of mean speed, I ruled them out of selection for the F2 cross for the following reasons. Our collaborators informed us that through a separate analysis on heartbeat phenotypes across the MIKK panel, they discovered that the heart of line `r colour_line("22-1 (‘David’)")` often stops beating for up to minutes at a time. This may explain the lack of movement observed during our assay. The behaviours they exhibit may therefore not represent a phenotype related to the boldness-shyness axis, but rather an extreme phenotype of a particular organ system. Our collaborators are, however, using this line in a separate F2 cross investigating heart phenotypes.

On the other hand, line `r colour_line("10-1 (‘Janeway’)")` appeared to habituate to the open field assay component by slowing down, rather than speeding up (**Figure \@ref(fig:10-1-dens)**). That is to say, as the assay progressed, they slowed down their movement, and this suggests that their typical response to stress is to move faster rather than slower. However, as I am using speed as a proxy for boldness (where a quicker habituation to the assay, indicated by an increase in movement, suggests greater boldness), this would create difficulties in attributing a functional interpretation of behaviours in the F2 individuals, and for this reason I excluded line `r colour_line("10-1 (‘Janeway’)")`.

(ref:10-1-dens) Tile and density plots for line `r colour_line("10-1 (‘Janeway’)")`, showing the reduction in the frequency of higher-movement states over the course of the open field assay component (**A** and **B**), with the novel object assay component (**C** and **D**) for comparison. 

```{r 10-1-dens, echo = F, fig.cap = '(ref:10-1-dens)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/select_0.08_15_10-1_dge.png"), auto_pdf = T)
```

\clearpage
## HMM states

To explore the behaviours of the MIKK panel at a finer resolution, as for the pilot study described in Chapter \@ref(Pilot-chap), I again applied a Hidden Markov Model (**HMM**) to classify the fishes' movements based on their distance and angle of travel between time intervals. I used the same method to select the best choice of time interval and number of states (**Figure \@ref(fig:mikk-param-comp)**). 

(ref:mikk-param-comp) Comparison between HMM parameters. Horizontal axis: Mean concordance between states assigned by HMMs through a 2-fold cross-validation process. Vertical axis: Kruskal-Wallis statistic comparing strains based on the proportion of time spent in each HMM state, summed across all states. Size of points correspond to the interval, in seconds, between which the distance and angle of travel was calculated (Methods).

```{r mikk-param-comp, echo = F, fig.cap = '(ref:mikk-param-comp)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/compare_params.png"), auto_pdf = T)
```

Here I observed the same phenomenon where the parameter combinations that performed the best showed an asymmetry between some states that would make interpretation difficult. For example, a time interval of 0.08 seconds combined with a state space of 17 caused state 4 to appear to get carved out of state 3 (**Figure \@ref(fig:mikk-hmm-asym)**).

(ref:mikk-hmm-asym) Based on the concordance analysis results set out in Figure \@ref(fig:mikk-param-comp) above, the best apparent combination of parameters -- 0.08 time interval with a 17-state space -- created an asymmetry between states 3 and 4, which would cause difficulties in interpreting their biological relevance. 

```{r mikk-hmm-asym, echo = F, fig.cap = '(ref:mikk-hmm-asym)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/0.08_17_polar_all_dge.png"), auto_pdf = T)
```

The best combination of parameters *without* this asymmetry was a time interval of 0.05 seconds with a state space of 15 (see the polar plots for the states in **Appendix \@ref(fig:hmm-states-05)**). However, due to a glitch in the video recording software, several videos recorded on 13 November 2019 were incorrectly recorded with a frame rate of 14 instead of the desired 30. The insufficient number of frames for those videos meant that it was impossible to measure the distance and angle of travel between a time interval as low as 0.05 seconds. So that these videos could be included in the dataset, I accordingly selected the combination of 15 states and a 0.08-second interval for all downstream analyses (**Figure \@ref(fig:mikk-hmm-sym)**).

(ref:mikk-hmm-sym) The HMM states used for the downstream analysis, with the model classified based on the distance of travel (log~10~ pixels, radial axis) and angle of travel (angle). A straight forward movement would sit around 0°, a left movement around 270°, and a right movement around 90°. 

```{r mikk-hmm-sym, echo = F, fig.cap = '(ref:mikk-hmm-sym)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/0.08_15_polar_all_dge.png"), auto_pdf = T)
```

\clearpage
## Social genetic effects

As discussed above, in this project our traits of interest include not only direct genetic behaviours, but also social genetic behaviours. I therefore sought to identify the MIKK panel lines that transmit their behaviour onto the reference *`r colour_line("iCab")`* tank partners either to the greatest or least extent. I refer to this transmission of behaviour as "***charisma***", to capture the element of behavioural influence. I formulated two methods to measure charisma, which I refer to as a) HMM state co-occupancy; and b) reference deviation. The first, HMM state co-occupancy, measures the proportions of time that the *`r colour_line("iCab")`* reference fish spends in the same HMM state as its tank partner. The second, deviation of the reference fishes' behaviour from the behaviour exhibited in the control condition, seeks to quantify the extent to which the *`r colour_line("iCab")`*'s behaviour changes when partnered with each MIKK panel line.

### State co-occupancy

**Figure \@ref(fig:F0-sge-cooc-box)** sets out the proportions of total time for each assay sub-component that each pair of individual fish spent in the same HMM state, grouped by line and ranked in the same order as their group median for individual mean speed as shown above in **Figure \@ref(fig:mikk-mean-speed)**. **\@ref(fig:F0-sge-cooc-box)A** shows the data as boxplots, with $p$-values from the Kruskal-Wallis test comparing all groups. **Figure \@ref(fig:F0-sge-cooc-box)B** shows the same data but with each group's median represented by columns to make it easier to compare group medians. Of the slower-moving lines, `r colour_line("8-2 (‘Gail’)")`^[Named after Gail the Snail, the slow but curiously captivating character from *It's Always Sunny in Philadelphia*.] and `r colour_line("18-2 (‘Elsa’)")` tend to show relatively higher state co-occupancy in the open field component, but during the novel object component, the slow-moving line `r colour_line("139-4")` has the highest median co-occupancy of all lines. Of the faster-moving lines, `r colour_line("43-2")` and `r colour_line("13-2")` showed the highest state co-occupancy during the open field assay component. However, the moderate-to-fast line `r colour_line("21-2 (‘Amelia’)")`^[Named after the fast and charismatic Amelia Earhart.] had relatively high state co-occupancy during both assay components.

(ref:F0-sge-cooc-box) Frequency of HMM state co-occupancy

```{r F0-sge-cooc-box, echo = F, fig.cap = '(ref:F0-sge-cooc-box)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/0.08_15_cooc_box_all.png"), auto_pdf = T)
```

To visualise which states are driving the higher co-occupancy measures, for a selection of lines I generated a heatmap of the states occupied simultaneously by the *`r colour_line("iCab")`* reference and MIKK test fishes, combining the observations for all individuals within each test fish group (**Figure \@ref(fig:F0-sge-cooc-heat)**). When *`r colour_line("iCab")`* is paired with `r colour_line("18-2 (‘Elsa’)")` or `r colour_line("8-2 (‘Gail’)")`, the fishes most frequently occupy states 3 or 1 in both open field and novel object components. In pairings with line `r colour_line("139-4")`, while the test fishes remain in the still states 1 or 3 during the open field assay, *`r colour_line("iCab")`* tends to be moving much faster in states 11 and 13. However, during the novel object component, they co-occupy state 3 more than in any other combination. This general pattern is observed with line `r colour_line("14-2")` as well, albeit to a lesser extent. For `r colour_line("38-2 (‘Mark’)")`,^[Line 38-2 appears to have intermediate measures for both movement and charisma, causing us to affectionately refer to it as the "boring" line, and then later to christen it after Mark Corrigan from *Peep Show*.] the fishes tend not to show a strong preference for co-occupying a particular state for either assay component, but the diagonal spread indicates that they tend to move at similar speeds. When paired with the faster moving `r colour_line("21-2 (‘Amelia’)")`, the novel object component appears to accentuate the co-occupancy of state 3 that is also observed in the open field component. Finally, when paired with line `r colour_line("40-1 (‘Max’)")`,^[Line 40-1 and its sibling line 40-2 both appeared to be preternaturally bold, like the siblings Max and Moritz from Wilhelm Busch's eponymous story [@buschMaxUndMoritz2015].] in both assay components, both fishes show a strong preference for the faster-moving states.

(ref:F0-sge-cooc-heat) Heatmaps for a selection of MIKK panel lines (including those ultimately selected as the parental strains in the F2 cross) showing the frequency of HMM states simultaneously occupied by the reference (x-axis) and test (y-axis) fishes during the open field (**A**) and novel object (**B**) assay components, aggregated over all replicates for each line. 

```{r F0-sge-cooc-heat, echo = F, fig.cap = '(ref:F0-sge-cooc-heat)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/0.08_15_cooc_heatmap.png"), auto_pdf = T)
```

### Deviation of *`r colour_line("iCab")`* from its control condition

The second method for quantifying the level of behavioural transmission from test fish to *`r colour_line("iCab")`* reference fish was to determine the proportion of time that the *`r colour_line("iCab")`* spent in a particular state when paired with another *`r colour_line("iCab")`*, and then quantify the degree to which those proportions change when in the presence of a fish from another line (**Figure \@ref(fig:F0-sge-deviation)**). **Figure \@ref(fig:F0-sge-deviation)A** presents boxplots for state frequencies for all *`r colour_line("iCab")`* individuals in *`r colour_line("iCab")`*-*`r colour_line("iCab")`* pairings. I further calculated the state frequencies for all *`r colour_line("iCab")`* reference fishes for all other MIKK line pairings. For each combination of assay component, line-pairing, and HMM state, I then ran Welch's t-test [@ruxtonUnequalVarianceTtest2006] comparing the proportions of time the *`r colour_line("iCab")`* individuals spent in that state when paired with another *`r colour_line("iCab")`*, against the proportions of time the *`r colour_line("iCab")`* reference individuals spent in that state when paired with a different MIKK line. I then summed the t-statistics across states to generate a single metric for each combination of line and assay component, and plotted the results in **Figure \@ref(fig:F0-sge-deviation)B**.

(ref:F0-sge-deviation) Deviation of state frequency for *`r colour_line("iCab")`* reference fishes when paired with MIKK panel lines relative to when paired with another *`r colour_line("iCab")`*. **A**: Boxplots of HMM state frequency for *`r colour_line("iCab")`* individuals when paired with another *`r colour_line("iCab")`*. **B**: Summed absolute t-statistics (Welch's t-test) comparing the proportions of time *`r colour_line("iCab")`* spent in each state when paired with another *`r colour_line("iCab")`*, against when paired with a MIKK panel line.

```{r F0-sge-deviation, echo = F, fig.cap = '(ref:F0-sge-deviation)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/0.08_15_deviation.png"), auto_pdf = T)
```

The first thing to note in this Figure is that *`r colour_line("iCab")`*'s deviations from its behaviour exhibited in the control condition tend to be smaller when paired with faster-moving fishes. This was expected given *`r colour_line("iCab")`* is also a faster-moving fish, but it strengthens the observation that when paired with slower-moving fishes, those tank partners are causing the *`r colour_line("iCab")`* reference fish to move slower than they would otherwise. Of the slower-moving MIKK lines, `r colour_line("4-2")` and `r colour_line("14-2")` causing the largest deviations of the *`r colour_line("iCab")`* reference fishes' behaviours from their control condition behaviour in the open field and novel object components respectively. I also observe large deviations for the moderately-moving `r colour_line("106-2")` and `r colour_line("71-1")` during the novel object components. There are no clear outliers for either assay component for any of the faster moving lines.

## Selection of lines for the F2 cross

On the basis of the above findings, I selected 6 MIKK panel lines as the parental lines for the F2 cross (**Figure \@ref(fig:F0-line-mean-speed-var-select)**). Conceptually, I sought to select lines that diverged on two measures: a) bold-shy behaviours; and b) the extent to which the lines transmitted their behaviours onto their tank partners. 

As *`r colour_line("iCab")`* is a fast-moving line, it was more difficult to detect instances where fast-moving MIKK panel lines were influencing its behaviour. I was therefore more confident of identifying slow-movement/high-charisma lines, and so to increase the likelihood of identifying genetic variants that are responsible for a stronger transmission of slow-moving behaviours, I chose two slow-movement/high-charisma lines for the F2 cross: `r colour_line("18-2 (‘Elsa’)")` and `r colour_line("8-2 (‘Gail’)")`. In the event that both lines possessed the same genetic variants that influence this behavioural transmission trait, it would vastly increase the power of detecting it during the genetic linkage analysis. Both these lines were one of the most slow-moving lines, had high levels of state co-occupancy during both assay components, and `r colour_line("18-2 (‘Elsa’)")` caused a high level of deviation from the *`r colour_line("iCab")`* control condition.

(ref:F0-line-mean-speed-var-select) Line median (vertical axis) and line variance (horizontal axis) for individual mean speed across the full 20-minute video (i.e. both the open field and novel object assay components) as shown above in Figure \@ref(fig:mikk-mean-speed-variance), now coloured only for the lines selected as the parental strains for the F2 cross.

```{r F0-line-mean-speed-var-select, echo = F, fig.cap = '(ref:F0-line-mean-speed-var-select)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/line_mean_speed_variance_selected.png"), auto_pdf = T)
```

For the slow-movement/low-charisma line, I selected `r colour_line("50-2 (‘Patrick’)")` which exhibited a moderately slow level of movement, low within-line variance for mean speed, and low measures for both state co-occupancy and *`r colour_line("iCab")`* deviation. For the high-movement/high-charisma line, I selected `r colour_line("21-2 (‘Amelia’)")`. Despite its high within line variance (**Figure \@ref(fig:F2-time-dge-of)**), this potentially made it easier to detect its social genetic effects, as the slower-moving individuals appeared to transmit those behaviours strongly to their *`r colour_line("iCab")`* tank partners, giving it a high score among fast-moving lines for state co-occupancy across both assay components. For the high-movement/low-charisma line I selected `r colour_line("40-1 (‘Max’)")`, as it has low within-line variance, and low-to-moderate metrics for state co-occupancy and *`r colour_line("iCab")`* deviation. However, I note that these measures may be confounded by the possibility that `r colour_line("40-1 (‘Max’)")` behaves in a similar way to *`r colour_line("iCab")`*, which would be difficult to determine whether it was behaving differently. 

In addition to these extreme lines, I also selected a line that was intermediate for both traits in an attempt to avoid breeding incompatibilities that might arise from attempting to cross lines with such divergent behavioural traits. For this purpose I selected line `r colour_line("38-2 (‘Mark’)")` for its intermediate speed, low within-line variance for mean speed, and intermediate measures for HMM state co-occupancy and *`r colour_line("iCab")`* deviation.

The selection of parental lines for the F2 cross indicating their putative position across the two axes of interest (movement, and behavioural transmission or 'charisma') are set out below in **Figure \@ref(fig:F0-line-select-schema)**.

(ref:F0-line-select-schema) Schema of the MIKK panel lines selected as the parental strains for the multi-way F2 cross, indicating their intended position across the two axes of interest (movement, and behavioural transmission or 'charisma').

[FOR EASE OF REFERRING TO THE LINES I GIVE THEM NAMES -- DO THAT AT THE START]

```{r F0-line-select-schema, echo = F, fig.cap = '(ref:F0-line-select-schema)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/line_selection_schema.png"), auto_pdf = T)
```

\clearpage
## Direct genetic effects

With these lines selected, I ran a similar analysis to what I described in \@ref(Pilot-chap), where I ran multi-way ANOVAs to determine whether certain lines differed in the proportions of time they spent in these HMM states, while including the date of assay, time of assay, tank quadrant and tank side as covariates. **Table \@ref(tab:mikk-dge-F0)** sets out the states which showed a significant difference between these 6 lines, with p-values adjusted for the False Discovery Rate (**FDR**).

```{r mikk-dge-F0, echo = F, message = F, warning = F, out.width='100%'}
kableExtra::kbl(
  googlesheets4::read_sheet(ss = "1kD2ndgmTvGlQw8YZFMztO3YoYfOdSw0SI5_BtomDuaI",
                            sheet = "DGE_FINAL"),
  caption = 'Significant differences in the proportion of time spent in each HMM state across test fish lines selected for the F2 cross for the open field and novel object assay components.',
  booktabs = TRUE
) %>% 
  kableExtra::kable_styling(latex_options = "striped")
```

**Figures \@ref(fig:F2-time-dge-of)** and **\@ref(fig:F2-time-dge-no)** highlight the states that showed significant differences in the proportions of time that these lines spent in those states for the open field and novel object assay components respectively. In both figures, **A** highlights the significant states, **B** shows how the individuals moved through those states over the course of the 10-minute assay component, and **C** shows the densities of the significant states within each line. For the open field component, although the tile plots show a notable level of variance within each line, the density plots clarify how the three slow-moving lines (`r colour_line("8-2 (‘Gail’)"`, `r colour_line("18-2 (‘Elsa’)"` and `r colour_line("50-2 (‘Patrick’)"`) spent little time in the fast-moving states 10, 12 and 14 relative to the fast-moving lines. Interestingly, the intermediate line `r colour_line("38-2 (‘Mark’)"` tended to transition into these states around the middle of the video, presumably a period of habituating to the new environment.

(ref:F2-time-dge-of) Differences between MIKK F0 lines in the HMM states they occupied during the *open field* assay component. **A**: 15 HMM states with panels coloured red to indicate significant differences between MIKK F0 lines in the proportion of time spent in those states. **B**: Transitions between HMM states across time for each individual test fish, grouped by MIKK line Tiles are coloured by the state most frequently occupied by each fish within 2-second intervals. **C**: Densities within each MIKK line for the occupation of states that significantly differed between strains (colour), with other states consolidated (grey).

```{r F2-time-dge-of, echo = F, fig.cap = '(ref:F2-time-dge-of)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/select_0.08_15_dge_of.png"), auto_pdf = T)
```

During the novel object component the HMM states that showed significant differences between lines were mostly restricted to the slow-moving states with the exception of state 10, which most clearly distinguishes the slow-moving lines from the intermediate- and fast-moving lines. Again, the intermediate line `r colour_line("38-2 (‘Mark’)")` shows a sharp drop in the occupation of the slow moving states after a period of habituation.

(ref:F2-time-dge-no) Differences between MIKK F0 lines in the HMM states they occupied during the *novel object* assay component. **A**: 15 HMM states with panels coloured red to indicate significant differences between MIKK F0 lines in the proportion of time spent in those states. **B**: Transitions between HMM states across time for each individual test fish, grouped by MIKK line Tiles are coloured by the state most frequently occupied by each fish within 2-second intervals. **C**: Densities within each MIKK line for the occupation of states that significantly differed between strains (colour), with other states consolidated (grey).

```{r F2-time-dge-no, echo = F, fig.cap = '(ref:F2-time-dge-no)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/select_0.08_15_dge_no.png"), auto_pdf = T)
```

### Social genetic effects

To confirm whether the *`r colour_line("iCab")`* reference fishes altered their behaviour depending on the MIKK F0 line of their tank partner, we carried out the same analysis and model as above using only data from the *`r colour_line("iCab")`* reference fishes. The states that showed a significant difference across any of the variables included in the model are set out in **Table \@ref(tab:mikk-sge-F0)**. The iCab reference fishes differed significantly in the proportion of time they spent in a given state depending on the strain of their tank partner ($p$ < 0.05, FDR-adjusted) for 4 out of 15 states in the open field assay (1.26x10^-2^ < $p$ < 4x10^-2^), and 4 out of 15 states for the novel object assay (7.71x10^-3^ < $p$ < 1.34x10^-2^). The line of the tank partner explained up to ~29% of the variance in the proportion of time the *`r colour_line("iCab")`* reference spent in a given state. Full tables for all states and variables are provided in the Supplementary Material. 

```{r mikk-sge-F0, echo = F, message = F, warning = F, out.width='100%'}
kableExtra::kbl(
  googlesheets4::read_sheet(ss = "1kD2ndgmTvGlQw8YZFMztO3YoYfOdSw0SI5_BtomDuaI",
                            sheet = "SGE_FINAL"),
  caption = 'Significant differences in the proportion of time spent in each HMM state by iCab reference fishes depending on the MIKK F0 line of their tank partner during the open field and novel object assay components.',
  booktabs = TRUE
)  %>% 
  kableExtra::kable_styling(latex_options = "striped")
```

(ref:F2-time-sge-of) Differences between HMM states occupied by the reference fish when paired with different MIKK F0 lines during the open field assay component. **A**: 15 HMM states with panels coloured red to indicate significant differences between the reference fishes under different line-pairings in the proportion of time spent in those states. The HMM states are the same as those in **Figure \@ref(fig:F2-time-dge-of)** above, but coloured with a different palette. **B**: Transitions between HMM states across time for each individual *`r colour_line("iCab")`* reference fish, grouped by the strain of its tank partner. Tiles are coloured by the state most frequently occupied by each fish within 2-second intervals. **C**: Densities within each strain-pairing for the occupation of states that significantly differed between strain-pairings (colour), with other states consolidated (grey).

```{r F2-time-sge-of, echo = F, fig.cap = '(ref:F2-time-dge-of)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/select_0.08_15_sge_of.png"), auto_pdf = T)
```

(ref:F2-time-sge-no) Same layout as **Figure \@ref(fig:F2-time-sge-of)** above, but for the novel object assay.

```{r F2-time-sge-no, echo = F, fig.cap = '(ref:F2-time-dge-no)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/select_0.08_15_sge_no.png"), auto_pdf = T)
```

To assist with interpretation of the relationships between the HMM states and the types of behaviours exhibited, I have provided in **Appendix \@ref(four-panel-app)** four-panel plots showing a) a frame grab from the raw four-quadrant video; b) path plots for both fishes; c) path plots for the test fishes coloured by HMM state; and d) path plots for the reference fishes coloured by HMM state, all taken at the 300-second mark (5 minutes) of each 10-minute assay component. 

\clearpage
## F2 generation

### Behavioural data collection

[CLEARLY UNDERPOWERED. GOAL WAS TO GENERATE 100 PER CROSS. STRATEGY HAS CHANGED BECAUSE WE DIDN'T GENERATE ENOUGH. ORIINALLY, WITHOUT THE COVID ISSUE, WE WOULD HAVE ANALYSED EACH CROSS AS WELL AS JOINTLY, NOW WE'RE JUST GOING TO DO THE JOIN ANALYSIS. THIS IS NOT THE TRADITIONAL WAY OF ANALYSING AN F2. TAKING AN F2 SET AND MAKING IT LIKE A FARM ANIMAL PEDIGREE. ADD TO INTRO]

Around August 2019, our collaborators in the Loosli Group at KIT commenced the breeding program for this experiment with the 6 MIKK panel lines I had selected above. Although the breeding program began in 2019, I understand from our collaborators that from around mid-2020 the F1 generation was producing poorly. After lengthy investigations, they discovered that the Covid pandemic had caused disruptions to the supply chains of their fish food manufacturer, which had compelled the manufacturer to modify the recipe. This change in the fish food recipe was the cause of the poor breeding. While the issue has since been resolved, it has resulted in a much smaller number of F2 individuals to be produced than was originally anticipated. CHECK FLOW From 17 November 2021 to 5 May 2022, a Research Assistant in Prof. Loosli's lab, Alicia Günthel, performed the assay 69 time with F2 individuals from the 12 crosses they had bred, producing a total of 271 videos of pairs of fish. The counts for the 12 crosses used to generate these 271 test fishes are set out in **Table \@ref(tab:F2-cross-counts)**.

```{r F2-cross-counts, echo = F, message = F, warning = F, out.width='100%'}
kableExtra::kbl(
  readr::read_csv(here::here("results/MIKK_F2/F2_cross_counts.csv")) %>% 
    dplyr::select(`paternal line` = pat_line,
                  `maternal line` = mat_line,
                  count = n) %>% 
    dplyr::arrange(desc(count)),
  caption = 'Significant differences in the proportion of time spent in each HMM state by iCab reference fishes depending on the MIKK F0 line of their tank partner during the open field and novel object assay components.',
  booktabs = TRUE
) %>% 
  kableExtra::kable_styling(latex_options = "striped")
```

I again used *idtrackerai* [@romero-ferreroIdtrackerAiTracking2019] to track the F2 individuals across frames. When splitting the 271 videos of pairs of fish into their open field and novel object assay components for a total of 542 videos, for 526 of them both fish were tracked across at least 85% of frames. I only used these 526 videos in the downstream analysis, but prior to publication I will seek to improve the tracking process so that these individuals can be included.

### Behavioural phenotyping

To ensure that the predicted HMM states for behaviour were consistent across both F0 and F2 generations, I had included these F2 individuals for training and prediction in the models described above. I again calculated the proportions of time that every individual spent in each state ("**state frequency**"), then inverse-normalised^[See section \@ref(inverse-norm-sec) in Chapter \@ref(Somite-chap) for a description of this process.] the values within each combination of assay component (open field or novel object) and state (1 to 15). 

**Figures \@ref(fig:F2-state-freq-dge)** and **\@ref(fig:F2-state-freq-sge)** compares the phenotype pre- and post-transformation with this normalisation approach. For the higher-movement states there are increasing numbers of individuals who spent no time in those states, which are responsible for the apparently non-normal distributions observed for the skewed distributions post-transformation. States 1, 3, 6, 8, 9 and 11 already appear to have a large amount of variation between individuals, but this normalisation process will increase the variance for states where there is small, yet potentially meaningful, variation between individuals. One exception may perhaps be state 15, which involves very large distances of travel in all directions, and therefore may correspond to tracking errors. 

(ref:F2-state-freq-dge) Effect of inverse-normalisation on the HMM state frequency of the F2 test fishes.

```{r F2-state-freq-dge, echo = F, fig.cap = '(ref:F2-state-freq-dge)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/0.08_15_state_freq_F2_dge.png"), auto_pdf = T)
```

(ref:F2-state-freq-sge) Effect of inverse-normalisation on the HMM state frequency of the F2 *`r colour_line('iCab')`* reference fishes.

```{r F2-state-freq-sge, echo = F, fig.cap = '(ref:F2-state-freq-sge)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/0.08_15_state_freq_F2_sge.png"), auto_pdf = T)
```

### Genotyping

After phenotyping the F2 samples, our collaborators in the Loosli Group at KIT took finclips from the F2 individuals, extracted their DNA, and arranged for them to be shallow-sequenced on the Illumina short-read platform at a coverage of ~1x per sample. Using a similar method to what I described in Chapter \@ref(Somite-chap), I aligned these sequences to the *HdrR* reference with BWA-MEM2 [@vasimuddinEfficientArchitectureawareAcceleration2019], sorted the reads and marked duplicates with Picard [@Picard2019toolkit], then indexed the resulting BAM files with samtools [@danecekTwelveYearsSAMtools2021]. I then used *bam-readcount* [@khannaBamreadcountRapidGeneration2022] to count the reads that supported either the paternal or the maternal allele for all biallelic SNPs that were homozygous-divergent in the given sample's two parental strains (i.e. homozgyous reference in the paternal line, and homozygous alternative in the maternal line, or *vice versa*), summed the read counts within 5 kb blocks, and calculated the frequency of reads within each bin that supported the maternal allele. This generated a value for each bin between 0 and 1, where 0 signified that all reads within that bin supported the paternal allele, and 1 signified that all reads within that bin supported the maternal allele. Bins containing no reads were imputed with a value of 0.5. 

I then used these values for all F2 individuals as the input to a Hidden Markov Model (HMM) with the software package *hmmlearn* [@HmmlearnHmmlearn2022], which I applied to classify each bin as one of three states, with state 0 corresponding to homozygous for the paternal allele, 1 corresponding to heterozygous, and 2 corresponding to homozygous for the maternal allele. Across each chromosome of every sample, the output of the HMM was expected to produce a sequence of states. Based on previous analyses described in Chapter \@ref(Somite-chap), I used the same HMM parameters as I did there, and used the HMM state outputs to generate the recombination blocks shown in **Figure \@ref(fig:F2-recomb-blocks)**. Missing genotype state calls arose from a sample having insufficient reads within a bin, for which I imputed the missing state calls within each sample's chromosome based on the previous state call on that chromosome. A karyoplot retaining the missing blocks is provided in **Appendix \@ref(fig:F2-recomb-blocks-missing)**.

[BE UPFRONT WITH HOW SOMETHING IS HAPPENING IN 23/24]

(ref:F2-recomb-blocks) Karyoplot for F2 samples, coloured by genotype. Samples are sorted in the order in which they were phenotyped. Blocks are filled with the colour of the paternal F0 line for the homozygous paternal haplotype block, black for heterozygous, and the colour of the maternal F0 line for the homozygous maternal haplotype block. Missing calls were imputed based on the previous successful call on a given chromosome. 

```{r F2-recomb-blocks, echo = F, fig.cap = '(ref:F2-recomb-blocks)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/karyoplot_no_missing.png"), auto_pdf = T)
```

[ADD ANOTHER PLOT HERE. PROPOTION OF CALLS/COUNTS FROM HMM. AA/AB/BB.]

I then took these HMM-generated haplotype block calls and used them to impute each sample's SNP-level genotypes using the homozygous biallelic SNP calls in the high-coverage .vcf file for the MIKK panel F0 lines described in Chapter \@ref(MIKK-genomes-chap). This set of variants included a total of ~20.7M SNPs, which I used to generate a Plink-format .bed file, forming the genotype input for the genetic linkage analysis.

### Genetic linkage analysis



For the purpose of using the *GCTA* software package [@yangGCTAToolGenomewide2011] for the genetic linkage analysis,  That software requires the construction of a genetic relationship matrix (**GRM**) $\textbf{A} = \textbf{WW}'/N$. $\textbf{W}$ is a standardised genotype matrix with the $ij^{th}$ element $w_{ij} = (x_{ij} - 2p_i) / \sqrt{(2p_i(1-p_i)}$, where $x_{ij}$ is the number of copies of the reference allele for the $i^{th}$ SNP of the $j^{th}$ individual and $p_i$ is the frequency of the reference allele [@yangGCTAToolGenomewide2011].

For the GRM, I first filtered the .bed for SNPs that had no missing calls for any samples ($M_{SNPs}$ = 44,360). I then used these SNPs to construct a "leave-one-chromosome-out" (**LOCO**) genetic relationship matrix for each chromosome -- that is, if the "focal" chromosome was chr1, I would exclude the SNPs on that chromosome before constructing the GRM. To illustrate, **Figure \@ref(fig:F2-grm)** is a GRM constructed using all 44,360 non-missing SNPs. However, given the relatively small amount of non-missing SNPs on each chromosome, the number of SNPs on the focal chromosome that were excluded was small, resulting in GRMs that appear almost identical by eye -- see the LOCO-GRM for chromosome 1 in **Appendix \@ref(fig:loco-grm-chr1)**. The individuals from each cross neatly cluster together, and the individuals that share one parental strain cluster nearby.

(ref:F2-grm) Genetic relationship matrix for 271 F2 samples based on 44,360 non-missing SNPs.

```{r F2-grm, echo = F, fig.cap = '(ref:F2-grm)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/grm_man_0.8.png"), auto_pdf = T)
```

[INDICATING THAT THE GRM REPRESETNS THE EEXPECTED STRUCTURE FROM THE SET OF F2 CROSSES]

I used these GRMs in the mixed linear model based association analysis (**MLMA**) implemented in GCTA [@yangGCTAToolGenomewide2011], where the model was generally specified as follows:

$$
y = a + bx + g + e
$$

$y$ was the phenotype, $a$ was the mean term, $b$ was the additive effect (fixed effect) of the candidate SNP tested for association, $x$ was the SNP genotype indicator variable coded as 0, 1 or 2, $g$ was the polygenic effect (random effect) i.e. the accumulated effect of all SNPs (as captured by the GRM calculated using all SNPs) and $e$ was the residual [@yangGCTAToolGenomewide2011]. For $y$, I used the inverse-normalised state frequencies described above, and the LOCO-GRM as $g$ for all SNPs on the focal chromosome. For example, for all SNPs on chr18, I used the LOCO-GRM that excluded all SNPs from that chromosome. In addition, I included the time of assay and the tank quadrant as covariates, which the software regresses out from the phenotype prior to running the MLMA. I excluded the date of assay and the tank side as covariates because individuals from the same cross tended to be assayed in the same test tank on the same day, and are therefore confounded with their genetics. 

To set a significance threshold, for each combination of HMM state and assay, I ran 10 MLMA tests over the same dataset where I had permuted the phenotype and covariates using a different random seed. The logic behind this method is to determine the lowest p-value that one could expect when there is no true relationship between the individuals' genetics and their phenotype. I then extracted the smallest p-value from all 10 results, and used this as the significance threshold. I additionally calculated the Bonferroni threshold as $\alpha / M$, where $\alpha$ is set to 0.05 and $M$ is the total number of SNPs in the dataset (2,726,797) = 1.83x10^-8^.

#### Direct genetic effects

For the DGE phenotypes (comparing the state frequencies across test fishes), I sought to identify the genetic loci in the F2 individuals that were associated with differences in their own behaviour. For neither the open field nor novel object component did the p-values exceed the Bonferroni threshold, however a number of loci across several chromosomes exceeded the thresholds set by the permutations. I plotted the p-values in Manhattan plots for each combination of state and assay component, and provide them all in the Appendix. Here I showcase the Manhattan plot for state 3, containing a number of significant loci that I discuss further below.

(ref:F2-man-dge-no-3) Manhattan plot for inverse-normalised HMM state 3 frequency of F2 test fishes during the novel object assay.

```{r F2-man-dge-no-3, echo = F, fig.cap = '(ref:F2-man-dge-no-3)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/manhattans/dge_no_3_time-quadrant.png"), auto_pdf = T)
```

#### Social genetic effects

For social genetic effects, I was attempting to discover genetic variants in the F2 individuals that caused differences in the behaviour of their *`r colour_line("iCab")`* tank partners. As expected, fewer loci reached significance for this transmitted indirect genetic effect than for the direct genetic effects, however I still found several significant loci based on the permutations threshold for states 4 (chr4), 7 (chr1), 12 (chr11), and 13 (chr12) during the open field assay component; but only one, barely significant SNP for state 5 (chr 13) during the novel object assay. This difference was somewhat surprising given our previous suppositions (DISCUSS) that the novel object component drew out stronger social genetic effects. [IT DOESN'T LOOK AS MAPPABLE INT HE NOVEL OBJECT, EVEN THOUGH THE DIFFERENCES LOOK MORE PRONOUNCED. PROBABLY SUGGESTS THAT THERE IS MORE NON-GENETIC VARIANCE IN N.O. THAN O.F.]

(ref:F2-man-sge-of-13) Manhattan plot for inverse-normalised HMM state 3 frequency of *`r colour_line("iCab")`* reference fishes during the novel object assay component.

```{r F2-man-sge-of-13, echo = F, fig.cap = '(ref:F2-man-sge-of-13)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/manhattans/sge_of_13_time-quadrant.png"), auto_pdf = T)
```


### Candidate loci for CRISPR-knockouts

```{r, include = F, message = F, warning = F}
# Get biomaRt annotations
library(biomaRt)
olat_mart = biomaRt::useEnsembl(biomart = "ensembl", dataset = "olatipes_gene_ensembl", mirror = "uswest")

olat_attr = biomaRt::listAttributes(olat_mart)

olat_genes = biomaRt::getBM(attributes = c("ensembl_gene_id",
                                           "hgnc_symbol",
                                           "description"),
                            mart = olat_mart) 

olat_homs = biomaRt::getBM(attributes = c("ensembl_gene_id",
                                          "hsapiens_homolog_associated_gene_name"),
                           mart = olat_mart) 

biom_df = dplyr::left_join(olat_genes,
                           olat_homs,
                           by = "ensembl_gene_id")
```


The next goal was to attempt to identify the SNPs that are mostly likely to be the causal variants, or closest in proximity to the causal variants, responsible for differences in the phenotypes of interest. I therefore proceeded to extract the significant SNPs and use Ensembl's Variant Effect Predictor (**VEP**) [@mclarenEnsemblVariantEffect2016] to identify the SNPs that are most likely to have a functional consequence, and determine whether the genes they reside in have already been identified as being involved in biological pathways that could be related to behaviour. **Table \@ref(tab:F2-sig-snps-consq-counts)** shows the counts for the different potential consequences of the unique significant SNPs, based on Ensembl's annotation of the *HdrR* reference. I note that as multiple genes can overlap the same locus (including those transcribed from opposing strands), one SNP may map to multiple genes and therefore be counted as having more than one type of consequence.

```{r F2-sig-snps-consq-counts, echo = F, message = F, warning = F}
sig = readr::read_csv(here::here("results/MIKK_F2/sigs.csv"))
vep = readr::read_tsv(here::here("results/MIKK_F2/0.08_vep_out.txt"),comment = "##") %>% 
  tidyr::separate(col = "#Uploaded_variation",into = c("CHROM", "POS", "REF_ALT"),sep = "_") %>% 
  tidyr::separate(col = "REF_ALT", into = c("REF", "ALT"), sep = "/") %>% 
  tidyr::unite(col = "SNP", CHROM, POS, sep = ":")

# Bind
df = dplyr::left_join(sig,
                      vep %>% 
                        dplyr::select(SNP, Allele, Gene, Consequence),
                      by = "SNP") 

# Pull out missense SNPs
out_tbl = df %>% 
  # Get only distinct SNP/Gene/Consequence combinations
  dplyr::distinct(SNP, Consequence) %>% 
  dplyr::count(Consequence, name = "count") %>% 
  dplyr::arrange(desc(count)) %>% 
  dplyr::rename(consequence = "Consequence") %>% 
  tidyr::drop_na()

kableExtra::kbl(
  out_tbl,
  caption = 'Counts for consequences of significant SNPs.',
  booktabs = TRUE
) %>% 
  kableExtra::kable_styling(latex_options = "striped")
```

As expected, most variants reside in non-coding regions [CITE], but in an attempt to isolate the most likely causative SNPs, I extracted those that are predicted to cause a missense mutation and therefore most likely to have a functional consequence (**Table \@ref(tab:F2-sig-snps-missense)**). 

```{r F2-sig-snps-missense, echo = F, message = F, warning = F}
sig = readr::read_csv(here::here("results/MIKK_F2/sigs.csv"))
vep = readr::read_tsv(here::here("results/MIKK_F2/0.08_vep_out.txt"),comment = "##") %>% 
  tidyr::separate(col = "#Uploaded_variation",into = c("CHROM", "POS", "REF_ALT"),sep = "_") %>% 
  tidyr::separate(col = "REF_ALT", into = c("REF", "ALT"), sep = "/") %>% 
  tidyr::unite(col = "SNP", CHROM, POS, sep = ":")

# Bind
uniq_df = dplyr::left_join(sig,
                      vep %>% 
                        dplyr::select(SNP, Allele, Gene, Consequence),
                      by = "SNP") %>% 
  # left join to biomart output
  dplyr::left_join(biom_df %>% 
                     dplyr::select(ensembl_gene_id,
                                   description),
                   by = c("Gene" = "ensembl_gene_id")) %>% 
  # get unique SNPs and consequences
  dplyr::distinct(DGE_SGE, ASSAY, STATE, SNP, Gene, Consequence, .keep_all = T) 


# Pull out missense SNPs
missense_tbl = uniq_df %>% 
  dplyr::filter(Consequence == "missense_variant") %>% 
  dplyr::mutate(DGE_SGE = toupper(DGE_SGE),
                ASSAY = stringr::str_replace(ASSAY, "_", " ")) %>% 
  dplyr::select("Genetic effect" = DGE_SGE,
                "Assay" = ASSAY,
                "State" = STATE, 
                "Chr" = CHROM,
                "Pos" = POS,
                "Ref" = REF,
                "Alt" = ALT,
                Allele,
                Gene,
                Description = description) %>% 
  # remove [Source:xxx] from descriptions
  dplyr::mutate(Description = stringr::str_remove_all(Description,
                                                  "\\[.*\\]"))

kableExtra::kbl(
  missense_tbl,
  caption = 'Missense SNPs.',
  booktabs = TRUE
) %>% 
  kableExtra::kable_styling(latex_options = c("striped", "scale_down"))
```

One SNP, 10:15,319,434, was significant for DGE for frequencies in both state 3 (**Figure \@ref(fig:sig-snp-10-15mb)**) and 10 during the novel object assay (see **Figure \@ref(fig:F2-man-dge-no-3)** above for state 3), and maps to two genes: ENSORLG00000029574 and ENSORLG00000024866. ENSORLG00000029574 is a novel gene on the forward strand with no recorded phenotypes. Based on discussions about this issue with an expert on genetic prediction analyses, it is most likely that this was a mis-prediction. However, on the reverse strand of this locus is ENSORLG00000024866, a gene for protocadherin alpha-C2-like, a well-known protein involved in mammalian synapse formation [@junghansPostsynapticDifferentialLocalization2008; @phillipsGProtocadherinsAreTargeted2003].

FIND ONE MORE LOCUS

(ref:sig-snp-10-15mb) State 3 frequency during the novel object assay for counts of the alternative allele (T) at SNP 10:15,319,434 in the protocadherin gene. The boxes are coloured on the left by the paternal line, and on the right by the maternal line.

```{r sig-snp-10-15mb, echo = F, fig.cap = '(ref:sig-snp-10-15mb)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/sig_snps_boxplots/3-10:15319434.png"), auto_pdf = T)
```

The second predicted missense variant of note was detected for SGE in the open field assay for frequencies in state 13. The locus 12:5,136,828 maps to gene ENSORLG00000002961, which is described as a complement *C9*. Discovered phenotypes for species orthologues include reduced prepulse inhibition and heart phenotypes in mice [Mouse Genome Database, @blakeMouseGenomeDatabase2021], and neurodevelopmental disorders in rats [Rat Genome Database, @smithYearRatRat2020]. Here, in the F2 individuals, it appears to have an inconsistent effect [CHANGE TO COMPLEX EFFECT BEHAVIOUR IN THE TEST FISHES BETWEEN THE CROSSES] in the behaviour of the test fishes, but apparently causes the reference fish to increase the proportion of time it spends in the fast-moving state 13 (**Figure \@ref(fig:F2-man-sge-of-13)**).

CONSIDER NOT SHOWING NON-INFORMATIVE CROSSES. ALSO ASK WHY A CERTAIN SNP ISN'T CALLED IN CERTAIN CROSSES. 
IDEALLY WE WANT DIFFERENT CROSSES SHOWING THE SAME DIRECTION OF EFFECT

(ref:sig-snp-12-5mb) State 13 frequency during the open field assay for counts of the alternative allele (A) at SNP 12:5,136,828. The boxes are coloured on the left by the paternal line, and on the right by the maternal line.

```{r sig-snp-12-5mb, echo = F, fig.cap = '(ref:sig-snp-12-5mb)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/sig_snps_boxplots/13-12:5136828.png"), auto_pdf = T)
```

There are additional significant loci of note shown in **Figure \@ref(fig:F2-man-dge-no-3)** above that are not predicted to alter proteins, but nevertheless appear to affect genes related to neurological development (**Table \@ref(tab:tbl-sig-snp-chr-9-10)**). 

```{r tbl-sig-snp-chr-9-10, echo = F, message = F, warning = F}
TARGET_SNPS = c("9:9802754", "10:18537719")

chr9_sigs = df %>% dplyr::filter(SNP %in% TARGET_SNPS) %>% 
  dplyr::left_join(biom_df %>% 
                     dplyr::select(ensembl_gene_id,
                                   description,
                                   hsapiens_homolog_associated_gene_name),
                   by = c("Gene" = "ensembl_gene_id")) %>% 
  dplyr::distinct(CHROM, POS, .keep_all = T) %>% 
  dplyr::mutate(DGE_SGE = toupper(DGE_SGE),
                ASSAY = stringr::str_replace(ASSAY, "_", " ")) %>% 
  dplyr::select("Genetic effect" = DGE_SGE,
                "Assay" = ASSAY,
                "State" = STATE, 
                "Chr" = CHROM,
                "Pos" = POS,
                "Ref" = REF,
                "Alt" = ALT,
                Allele,
                Gene,
                Description = description,
                "Human homologue" = hsapiens_homolog_associated_gene_name) %>% 
  # remove [Source:xxx] from descriptions
  dplyr::mutate(Description = stringr::str_remove_all(Description,
                                                  "\\[.*\\]"))

kableExtra::kbl(
  chr9_sigs,
  caption = 'Significant loci on chromosome 9 for direct genetic effects on state 3 frequency during the novel object assay.',
  booktabs = TRUE
) %>% 
  kableExtra::kable_styling(latex_options = c("striped", "scale_down"))
```

The first is located around 9:9,802,754 (**Figure \@ref(fig:sig-snp-9-9mb)**), and maps to a gene for glutamate ionotropic receptor delta type subunit 2, an orthologue of the human gene *GRID2*. Deletions in this gene have been found to cause ataxia in humans [@hillsDeletionsGRID2Lead2013; @utineHomozygousDeletionGRID22013], and other mutations in this gene have been found to cause various neurological disorders in mice [Mouse Genome Database, @blakeMouseGenomeDatabase2021]. 

(ref:sig-snp-9-9mb) State 3 frequency during the novel object assay for counts of the alternative allele (A) at SNP 9:9,802,754. The boxes are coloured on the left by the paternal line, and on the right by the maternal line.

```{r sig-snp-9-9mb, echo = F, fig.cap = '(ref:sig-snp-9-9mb)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/sig_snps_boxplots/3-9:9802754.png"), auto_pdf = T)
```

The second locus resides around 10:18,537,719 (**Figure \@ref(fig:sig-snp-10-18mb)**) within the neuroligin 3 gene, an orthologue of the human gene *NLGN3*, which has been linked with autism in humans [@jamainMutationsXlinkedGenes2003], and various neurological in mice and rats [@blakeMouseGenomeDatabase2021; @smithYearRatRat2020].

(ref:sig-snp-10-18mb) State 3 frequency during the novel object assay for counts of the alternative allele (A) at SNP 10:18,537,719. The boxes are coloured on the left by the paternal line, and on the right by the maternal line.

```{r sig-snp-10-18mb, echo = F, fig.cap = '(ref:sig-snp-10-18mb)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/sig_snps_boxplots/3-10:18537719.png"), auto_pdf = T)
```

These results provide promising evidence that the assay and methods described above can identify genetic loci associated with differences in behaviours - and potentially difference in the transmission of behaviour onto social companions - with functional relevance to humans. However, before seeking to functionally validate these variants, there are a number of steps that we seek to take to refine the analysis and thereby increase our confidence in the variants we ultimately select for validation.

\clearpage
## Discussion 

ADD SOMETHIN MORE POSITIVE. IT WORKS. LOCI ARE NEUROGLICAL. WITH MORE NUMBERS THIS WILL WORK WELL. MOVE THIS STUFF INTO THE CONCLUSIONS CHAPTER.

With the benefit of hindsight, there are a several aspects of this analysis that I would have performed differently. The first relates to the choices of certain parental lines for the F2 cross, which were too heavily reliant on the aggregate measures of speed and charisma, without due regard to the level of behavioural variance observed within each line. As evident in **Figures \@ref(fig:F2-time-dge-of)** and **\@ref(fig:F2-time-dge-no)** above, lines `r colour_line("8-2 (‘Gail’)")` and `r colour_line("21-2 (‘Amelia’)")` showed large within-line variances. For `r colour_line("8-2 (‘Gail’)")` in particular, the differences appear to correspond to the date on which they were assayed - specifically, in the first run the four `r colour_line("8-2 (‘Gail’)")` individuals showed almost no movement, whereas in the second run they showed much higher levels of movement. The different within-line variances are an interesting phenotype to explore further, as they suggest that different lines show different degrees of sensitivity to environmental changes. However, for this analysis, high within-line variances make it difficult to ascertain the line's "true" phenotype, and therefore make them unsuitable for selection based on our axes of interest for this study (bold vs shy; high vs low behavioural transmission). 

An appropriate substitute for `r colour_line("8-2 (‘Gail’)")` may be line `r colour_line("139-4")`, which had low within-line variance for mean speed, and the highest median state co-occupancy in the novel object assay component. From the analysis of Chapter \@ref(Pilot-chap), which concluded after the lines were already selected for the F2 cross, it became apparent that the novel object assay was particularly useful for revealing social genetic effects. I hypothesise that at times of higher stress or predation threat, the fish take more behavioural cues from their tank partners. Line `r colour_line("139-4")` therefore would have been a good candidate for the slow-moving, high-charisma line. 

Similarly, lines `r colour_line("13-2")` or `r colour_line("94-1")` would be preferred substitutes for line `r colour_line("21-2 (‘Amelia’)")` as the fast-moving, high charisma line. They are faster than `r colour_line("21-2 (‘Amelia’)")`, but unlike the fastest line `r colour_line("10-1 (‘Janeway’)")`, they show a degree of habituation (where most individuals moved slowly at the beginning of the assay before eventually speeding up). They also showed lower within-line variance than `r colour_line("21-2 (‘Amelia’)")`, but comparable measures for the SGE measures of state co-occupancy and reference deviation. 

One way of avoiding these issues would have been to used a more quantitative, objective approach to selecting the lines. For example, I could have used Mann-Whitney tests to quantify the levels of differences between pairs of lines for a trait of interest (as I did for **Figure \@ref(fig:mikk-param-comp)**), and then select the lines that maximised that statistic. Such metrics would still perhaps need to be qualitatively weighted against other relevant traits, such as within-line variance. Nevertheless, my downstream analyses show that despite not having potentially selected the ideal lines for the F2 cross, the differences between them were sufficient to identify genetic loci associated with our phenotypes of interest, even with a relatively small sample size of 271 F2 individuals.

Another aspect of the analysis that could have been improved was to use one or more MIKK panel lines as the reference line instead of *`r colour_line("iCab")`*. As previously mentioned, *`r colour_line("iCab")`* is a relatively fast line, which made it difficult to detect social genetic effects when paired with similarly fast-moving lines. Instead, we could have made use of the behavioural variance across the MIKK panel to choose a reference line that exhibits moderate levels of movement. However, we would not be able to assess the levels of movement across the MIKK panel without first assaying the panel with a single reference strain, and we chose *`r colour_line("iCab")`* because we not only had a good understanding of its behavioural traits based on our findings from Chapter \@ref(Pilot-chap), but also because it has high fecundity, which made many replicates available and in turn allowed us to avoid using the same reference fish more than once within 24 hours. Now that we have carried out this analysis, we would be able to select a more appropriate line as the reference in any future analyses.

The other benefit from using a MIKK panel line as the reference fish would be to open up the possibility of exploring social genetic effect interactions. For example, if we used two different MIKK lines as the reference, we could explore how those lines differ in the degree to which they are influenced by their tank partner. To state the issue inversely, it would reveal differences in the extent to which different lines transmit their behaviour onto their (reference) tank partner, depending on both of their genetics. A question for further studies could therefore be framed as, 'given line `r colour_line("18-2 (‘Elsa’)")` appears to possess a high level of charisma, is that level consistent when they are paired with different MIKK panel lines, and if it differs across lines, then why?' Due to the relative onerousness of running this behavioural assay, such experiments would involve a substantial time commitment, but in light of the findings presented here, the investigations could be restricted to a subset of the MIKK panel that display interesting behavioural phenotypes. 

## Future directions

Our collaborators are in the process of breeding more F2 individuals to increase the sample size for this analysis. For the subsequent breeding program, I have suggested that we replace line `r colour_line("21-2 (‘Amelia’)")` with either `r colour_line("13-2")` or `r colour_line("23-1")`, which both show lower within-line variance in mean speed, but similarly high levels of speed and behavioural transmission (as measured by state co-occupancy and *`r colour_line("iCab")`* reference deviation). Any additional F2 individuals will still need to be phenotyped and sequenced, and the phenotype data collection (i.e. video recording the behavioural assay) can be relatively laborious, so this process will likely take some time.

In the meantime, I have 96 Kiyosu CC individuals whose videos have already been processed, and whose DNA has been shallow-sequenced (~1x). It was not feasible to include that data in this analysis due to time constraints. As they have been allowed to breed freely from the same Kiyosu population as the MIKK panel, they are not strongly homozygous, and they are likely to possess many additional genetic variants that are not represented in the MIKK panel. I am therefore not able to impute their genotypes from high-coverage parental strains, as I did with the F2 individuals. The alignments to the reference are also likely to include many mapping errors due to the lack of sufficient coverage. The additional information they provide will therefore not be as useful as additional F2 individuals, but it should nevertheless improve the power to detect casual variants. The amount of additional information we obtain from the Kiyosu CC individuals will also inform whether we include them in this way in future F2 crosses. [NOT REALLY. THERE ISN'T AN ALIGNMENT TO REFERENCE PROBLEM. BUT WHAT THERE IS IS THAT THINGS WILL HAV ETO BE PROCESSED DIFFERENTLY. IN THE F2S WE ASSUME THAT THE PARENTS ARE HOMOZYGOUS. THIS IS MUCH CLOSER TO THE HUMAN SITUATION. OPTIMAL IS 1X IN THEORY, 4X IN PRACTICE. LIKE 1000 GENOMES. PANEL OF 80 BECOME LIKE THE HIGH COVERAGE DATASET. THIS IS MUCH CLOSER TOT HE OUTBRED SITUTATION WITH HUMANS, WHERE THE PANEL WILL BE THE HIGH COVERAGE PANEL, AND WE'LL HAVE SOME LOW-COVERAGE SAMPLES WHERE WE'LL NEED TO DO SOME IMPUTATION.]

A second issue I plan to address is the failure of F2 reads to align to chromosomes 22 and 23 (see **Figure \@ref(fig:F2-recomb-blocks)** above). As the issue occurs across most of the sample, and the coverage does not look skewed for those chromosomes (**Figure \@ref(fig:F2-coverage)**), there is likely to a be an error in the bioinformatic pipeline that I constructed.

(ref:F2-coverage) Mean depth of sequencing coverage per chromosome for the 271 F2 samples used in this analysis.

```{r F2-coverage, echo = F, fig.cap = '(ref:F2-coverage)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/coverage_F2.png"), auto_pdf = T)
```

[TO DO: homozygous-divergent SNP counts for each cross, across chromosomes]

A third issue is the matter of selecting the optimal HMM parameters for calling the recombination blocks, as described in Chapter \@ref(Somite-chap). For the analysis in this present chapter I used the same HMM parameters as I had used there. In that chapter, I used the $p$-values from the resulting genetic linkage analysis as a rough measure of genotyping quality (assuming that lower $p$-values were driving by more accurate genotypes), together with a qualitative assessment of how many crossovers I expected to observe in a particular chromosome for a subset of the sample. The issue can be resolved by obtaining "ground truth" genotype calls through high-coverage sequence data for a set of samples that have also been shallow-sequenced. This would allow one to compare the HMM recombination block calls with the ground truth genotypes, and thereby train the model to optimise the accuracy of its predictions for the locations of the recombination sites (using a test dataset to validate the parameters). I expect the improved genotype calls will further improve the power to detect genetic variants.

With respect to the genetic linkage analysis, due to the method I used to calculate the genetic relationship matrix (**GRM**), I was compelled to exclude all SNPs that were missing calls in any of the samples, reducing the number of SNPs to merely ~40K out of the ~20M in the full dataset. Not only does this reduce the accuracy of the GRM as a means of capturing the relative relatedness between samples, but the problem will only be exacerbated with the inclusion of additional samples (as the likelihood of a SNP having at least one sample with a missing call will increase with the number of samples). I will therefore adapt my method to include SNPs with missing calls, which should further increase statistical power.

Finally, I applied a separate genetic linkage association test for the state frequency of each HMM state, where it may be beneficial to apply a method that can combine the information across all states into a single phenotype (if not fewer phenotypes), and then run an association test on that reduced phenotype. There are a number of alternatives to consider, including principal components analysis (PCA) of the state frequencies, or the software MTAG [Multi-Trait Analysis of Genome-wide summary statistics , @turleyMultitraitAnalysisGenomewide2018] which can combine the summary statistics from multiple genetic association tests. Ultimately, the hope is that by leveraging the information about the frequencies in which the fishes occupy each state, combined across *all* states, while appropriately taking into account the interdependence of those frequencies (e.g. the more time a fish spends in state 3, the less time it is likely to spend in state 13), it will further strengthen these results.

Aside from modifications designed to increase statistical power, there is also capacity to improve the way that I phenotype the individuals, and in particular how I characterise social genetic effects. In this analysis I have only used two behavioural variables: distance and angle of travel. This behavioural dataset is extremely rich, and myriad variables could be extracted from the videos. Shoaling traits would have special relevance to the study of social genetic effects. For instance, one could extract three variables that would be associated with shoaling behaviours: a) distance between tank partners; b) synchrony in speed; and c) parallelism in direction of travel. These three variables could then be reduced down to two with an HMM (i.e. one state representing "shoaling", and another representing "not shoaling"). The relative frequencies that the different MIKK lines spend in these states would allow one to rank them by the degree of shoaling behaviour they exhibit, and this would likely be a far more accurate representation of social genetics than what I have shown here. 

