# (APPENDIX) Appendix {-}

# Chapter 2 supplementary information

(ref:nuc-div-per-indiv) Nucleotide diversity ($\hat{\pi}$) calculated within 500-kb non-overlapping windows for each of the 63 non-"sibling" lines from the MIKK panel individually, for chromosome 1 (left) and 22 (right). The windows that include the medaka sex determination region (1:~16-17 Mb) show consistent elevated nucleotide diversity across most lines. 

```{r nuc-div-per-indiv, echo = F, fig.cap = '(ref:uc-div-per-indiv)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_genome/supp_02_pi_chrs_1_22.png"), auto_pdf = T)
```

# Chapter 4 supplementary information

## Recombination blocks for F2 individuals showing regions with missing genotype calls

(ref:karyo-wi-missing) Recombination blocks in 622 F2 samples based on the ratio of reads mapping to either the *Cab* or *Kaga* allele within 5-kb bins, with homozygous-*Cab* blocks in green, heterozoygous blocks in navy blue, and homozygous *Kaga* blocks in red. Most blocks show 0-2 crossover events, as expected, with some regions showing higher numbers of crossovers interpreted as noise. Unfilled regions are those with no state called by the HMM due to a lack of reads mapping to SNPs within those 5-kb bins.

```{r karyo-wi-missing, echo = F, fig.cap = '(ref:karyo-wi-missing)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/somites/karyoplot_wi_missing.png"), auto_pdf = T)
```

# Chapter 5 supplementary information

## 15 HMM states with 0.05 second interval

(ref:hmm-states-05) HMM states predicted for the F0 dataset, using the combination of a 0.05-second interval between which the distance and angle variables were calculated.

```{r hmm-states-05, echo = F, fig.cap = '(ref:hmm-states-05)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/0.05_15_polar_all_dge.png"), auto_pdf = T)
```

## Four-panel plots for MIKK lines selected for the F2 cross {#four-panel-app}

All frame grabs were taken at the 300th second (5-minute mark) of each 10-minute assay component.

### Open field

#### 8-2

(ref:4p-8-2-of-A) Line 8-2, open field, run A.

```{r 4p-8-2-of-A, echo = F, fig.cap = '(ref:4p-8-2-of-A)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/four_panel_plots/open_field_20191114_1040_8-2_L_A_300.png"), auto_pdf = T)
```

(ref:4p-8-2-of-B) Line 8-2, open field, run B.

```{r 4p-8-2-of-B, echo = F, fig.cap = '(ref:4p-8-2-of-B)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/four_panel_plots/open_field_20191118_1224_8-2_R_B_300.png"), auto_pdf = T)
```

#### 18-2

(ref:4p-18-2-of-A) Line 18-2, open field, run A.

```{r 4p-18-2-of-A, echo = F, fig.cap = '(ref:4p-18-2-of-A)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/four_panel_plots/open_field_20191112_1236_18-2_L_A_300.png"), auto_pdf = T)
```

(ref:4p-18-2-of-B) Line 18-2, open field, run B.

```{r 4p-18-2-of-B, echo = F, fig.cap = '(ref:4p-18-2-of-B)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/four_panel_plots/open_field_20191116_1039_18-2_L_B_300.png"), auto_pdf = T)
```

(ref:4p-18-2-of-C) Line 18-2, open field, run C.

```{r 4p-18-2-of-C, echo = F, fig.cap = '(ref:4p-18-2-of-C)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/four_panel_plots/open_field_20191119_1459_18-2_R_C_300.png"), auto_pdf = T)
```

#### 50-2

(ref:4p-50-2-of-A) Line 50-2, open field, run A.

```{r 4p-50-2-of-A, echo = F, fig.cap = '(ref:4p-50-2-of-A)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/four_panel_plots/open_field_20191114_1305_50-2_R_A_300.png"), auto_pdf = T)
```

(ref:4p-50-2-of-B) Line 50-2, open field, run B.

```{r 4p-50-2-of-B, echo = F, fig.cap = '(ref:4p-50-2-of-B)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/four_panel_plots/open_field_20191116_1438_50-2_R_B_300.png"), auto_pdf = T)
```

(ref:4p-50-2-of-C) Line 50-2, open field, run C.

```{r 4p-50-2-of-C, echo = F, fig.cap = '(ref:4p-50-2-of-C)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/four_panel_plots/open_field_20191120_1150_50-2_R_C_300.png"), auto_pdf = T)
```

#### 38-2

(ref:4p-38-2-of-A) Line 38-2, open field, run A.

```{r 4p-38-2-of-A, echo = F, fig.cap = '(ref:4p-38-2-of-A)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/four_panel_plots/open_field_20191112_1456_38-2_L_A_300.png"), auto_pdf = T)
```

(ref:4p-38-2-of-B) Line 38-2, open field, run B.

```{r 4p-38-2-of-B, echo = F, fig.cap = '(ref:4p-38-2-of-B)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/four_panel_plots/open_field_20191116_1203_38-2_L_B_300.png"), auto_pdf = T)
```

#### 21-2

(ref:4p-21-2-of-A) Line 38-2, open field, run A.

```{r 4p-21-2-of-A, echo = F, fig.cap = '(ref:4p-21-2-of-A)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/four_panel_plots/open_field_20191113_1527_21-2_L_A_300.png"), auto_pdf = T)
```

(ref:4p-21-2-of-B) Line 38-2, open field, run B.

```{r 4p-21-2-of-B, echo = F, fig.cap = '(ref:4p-21-2-of-B)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/four_panel_plots/open_field_20191116_1309_21-2_R_B_300.png"), auto_pdf = T)
```

#### 40-1

(ref:4p-40-1-of-A) Line 40-1, open field, run A.

```{r 4p-40-1-of-A, echo = F, fig.cap = '(ref:4p-40-1-of-A)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/four_panel_plots/open_field_20191114_1217_40-1_L_A_300.png"), auto_pdf = T)
```

(ref:4p-40-1-of-B) Line 40-1, open field, run B.

```{r 4p-40-1-of-B, echo = F, fig.cap = '(ref:4p-40-1-of-B)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/four_panel_plots/open_field_20191118_1129_40-1_R_B_300.png"), auto_pdf = T)
```

### Novel object

#### 8-2

(ref:4p-8-2-no-A) Line 8-2, novel object, run A.

```{r 4p-8-2-no-A, echo = F, fig.cap = '(ref:4p-8-2-no-A)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/four_panel_plots/novel_object_20191114_1040_8-2_L_A_300.png"), auto_pdf = T)
```

(ref:4p-8-2-no-B) Line 8-2, novel object, run B.

```{r 4p-8-2-no-B, echo = F, fig.cap = '(ref:4p-8-2-no-B)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/four_panel_plots/novel_object_20191118_1224_8-2_R_B_300.png"), auto_pdf = T)
```

#### 18-2

(ref:4p-18-2-no-A) Line 18-2, novel object, run A.

```{r 4p-18-2-no-A, echo = F, fig.cap = '(ref:4p-18-2-no-A)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/four_panel_plots/novel_object_20191112_1236_18-2_L_A_300.png"), auto_pdf = T)
```

(ref:4p-18-2-no-B) Line 18-2, novel object, run B.

```{r 4p-18-2-no-B, echo = F, fig.cap = '(ref:4p-18-2-no-B)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/four_panel_plots/novel_object_20191116_1039_18-2_L_B_300.png"), auto_pdf = T)
```

(ref:4p-18-2-no-C) Line 18-2, novel object, run C.

```{r 4p-18-2-no-C, echo = F, fig.cap = '(ref:4p-18-2-no-C)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/four_panel_plots/novel_object_20191119_1459_18-2_R_C_300.png"), auto_pdf = T)
```

#### 50-2

(ref:4p-50-2-no-A) Line 50-2, novel object, run A.

```{r 4p-50-2-no-A, echo = F, fig.cap = '(ref:4p-50-2-no-A)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/four_panel_plots/novel_object_20191114_1305_50-2_R_A_300.png"), auto_pdf = T)
```

(ref:4p-50-2-no-B) Line 50-2, novel object, run B.

```{r 4p-50-2-no-B, echo = F, fig.cap = '(ref:4p-50-2-no-B)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/four_panel_plots/novel_object_20191116_1438_50-2_R_B_300.png"), auto_pdf = T)
```

(ref:4p-50-2-no-C) Line 50-2, novel object, run C.

```{r 4p-50-2-no-C, echo = F, fig.cap = '(ref:4p-50-2-no-C)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/four_panel_plots/novel_object_20191120_1150_50-2_R_C_300.png"), auto_pdf = T)
```

#### 38-2

(ref:4p-38-2-no-A) Line 38-2, novel object, run A.

```{r 4p-38-2-no-A, echo = F, fig.cap = '(ref:4p-38-2-no-A)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/four_panel_plots/novel_object_20191112_1456_38-2_L_A_300.png"), auto_pdf = T)
```

(ref:4p-38-2-no-B) Line 38-2, novel object, run B.

```{r 4p-38-2-no-B, echo = F, fig.cap = '(ref:4p-38-2-no-B)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/four_panel_plots/novel_object_20191116_1203_38-2_L_B_300.png"), auto_pdf = T)
```

#### 21-2

(ref:4p-21-2-no-A) Line 38-2, novel object, run A.

```{r 4p-21-2-no-A, echo = F, fig.cap = '(ref:4p-21-2-no-A)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/four_panel_plots/novel_object_20191113_1527_21-2_L_A_300.png"), auto_pdf = T)
```

(ref:4p-21-2-no-B) Line 38-2, novel object, run B.

```{r 4p-21-2-no-B, echo = F, fig.cap = '(ref:4p-21-2-no-B)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/four_panel_plots/novel_object_20191116_1309_21-2_R_B_300.png"), auto_pdf = T)
```

#### 40-1

(ref:4p-40-1-no-A) Line 40-1, novel object, run A.

```{r 4p-40-1-no-A, echo = F, fig.cap = '(ref:4p-40-1-no-A)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/four_panel_plots/novel_object_20191114_1217_40-1_L_A_300.png"), auto_pdf = T)
```

(ref:4p-40-1-no-B) Line 40-1, novel object, run B.

```{r 4p-40-1-no-B, echo = F, fig.cap = '(ref:4p-40-1-no-B)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/four_panel_plots/novel_object_20191118_1129_40-1_R_B_300.png"), auto_pdf = T)
```

## HMM state time dependence for all MIKK panel lines

### Direct genetic effects

#### Tile plot

(ref:mikk-all-tile-dge) Tile plot for all 1610 test fishes included in the MIKK panel behaviour analysis, ordered by each line's group median for individual mean speed over the course of the 20-minute video (open field and novel object combined). The order of lines is identical to that shown in **Figure \@ref(fig:mikk-mean-speed)**. 

```{r mikk-all-tile-dge, echo = F, fig.cap = '(ref:mikk-all-tile-dge)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/0.08_15_tile_dge.png"), auto_pdf = T)
```

#### Density plot

(ref:mikk-all-dens-dge) Density plot for all 1610 test fishes included in the MIKK panel behaviour analysis, ordered by each line's group median for individual mean speed over the course of the 20-minute video (open field and novel object combined). The order of lines is identical to that shown in **Figure \@ref(fig:mikk-mean-speed)**. 

```{r mikk-all-dens-dge, echo = F, fig.cap = '(ref:mikk-all-tile-dge)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/0.08_15_dens_dge.png"), auto_pdf = T)
```

### Social genetic effects

#### Tile plot

(ref:mikk-all-tile-sge) Tile plot for all 1610 reference fishes included in the MIKK panel behaviour analysis, ordered by each line's group median for individual mean speed over the course of the 20-minute video (open field and novel object combined). The order of lines is identical to that shown in **Figure \@ref(fig:mikk-mean-speed)**. 

```{r mikk-all-tile-sge, echo = F, fig.cap = '(ref:mikk-all-tile-sge)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/0.08_15_tile_sge.png"), auto_pdf = T)
```

#### Density plot

(ref:mikk-all-dens-sge) Density plot for all 1610 test fishes included in the MIKK panel behaviour analysis, ordered by each line's group median for individual mean speed over the course of the 20-minute video (open field and novel object combined). The order of lines is identical to that shown in **Figure \@ref(fig:mikk-mean-speed)**. 

```{r mikk-all-dens-sge, echo = F, fig.cap = '(ref:mikk-all-tile-sge)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/0.08_15_dens_sge.png"), auto_pdf = T)
```

## F2 recombination karyoplot with missing calls

(ref:F2-recomb-blocks-missing) Karyoplot for F2 samples, coloured by genotype. Samples are sorted in the order in which they were phenotyped. Blocks are filled with the colour of the paternal F0 line for the homozygous paternal haplotype block, black for heterozygous, and the colour of the maternal F0 line for the homozygous maternal haplotype block. 

```{r F2-recomb-blocks-missing, echo = F, fig.cap = '(ref:F2-recomb-blocks-missing)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/karyoplot_wi_missing.png"), auto_pdf = T)
```

## LOCO GRM for chromosome 1 

(ref:loco-grm-chr1) "Leave-one-chromosome-out" genetic relationship matrix for 271 F2 samples based on 44,284 non-missing SNPs, having excluded 76 SNPs on chromosome 1.

```{r loco-grm-chr1, echo = F, fig.cap = '(ref:loco-grm-chr1)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/grm_loco_man_chr1.png"), auto_pdf = T)
```

# Chapter 6 supplementary information

## eCDF of all polygenic traits in the GWAS Catalog ranked by ${D_t^S}$ 

(ref:eCDFall) 587 traits from the GWAS Catalog that passed our filters for polygenic traits, ranked by ${D_t^S}$.

```{r eCDFall, echo = F, fig.cap = '(ref:eCDFall)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/fst/0.1_1000_20220314_ecdf_all_faceted_with_controls_d_rank_long.png"), auto_pdf = T)
```


