# Conclusions {-}

```{r, include = F}
library(tidyverse)

# Read in line colours
line_cols = readr::read_csv(here::here("results/MIKK_F2/line_colours_0.08.csv"))

# Function for colouring text
colorize <- function(x, color, col_name) {
  if (knitr::is_latex_output()) {
    # remove hash prefix
    color = stringr::str_remove(color, "#")
    sprintf("\\definecolor{%s}{HTML}{%s}\\textcolor{%s}{%s}", col_name, color, col_name, x)
  } else if (knitr::is_html_output()) {
    sprintf("<span style='color: %s;'>%s</span>", color,
      x)
  } else x
}

# Function for pulling the line's matching colour from the data frame, then calling `colorize()`
colour_line = function(LINE){
  COLOUR = line_cols %>% 
    dplyr::filter(line == LINE) %>% 
    dplyr::pull(colour)
  # make colour name
  COL_NAME = paste(LINE,
                   stringr::str_remove(COLOUR, "#"),
                   sep = "_")
  # call colorize function
  colorize(LINE, COLOUR, COL_NAME)
}
```

## Chapter \@ref(MIKK-genomes-chap) {-}

In this first substantive chapter of the thesis, I explored several genomic traits in the MIKK panel that are relevant to its utility for the genetic mapping of complex traits, and analysed the structural variants in a subset of its genomes that could be discovered through the use of Oxford Nanopore Technologies' (**ONT**) long-read sequencing platform. I displayed the inbreeding trajectory of the MIKK panel, confirming that medaka have a remarkable resilience to inbreeding from the wild, and that despite the spread of a bacterial infection during the inbreeding process that destroyed a number of lines, 30% of them nevertheless survived by generation F~15~. I showed the MIKK panel possesses a normally-distributed morphology phenotype (mean relative eye size) as an example of the distribution of complex traits across the panel. 

I confirmed that the inbreeding process resulted in inbred lines that were highly homozygous across almost the entire genome, comparable to the levels of homozygosity that were present in established inbred strains that have been inbreed for over 100  generations, and that regions of residual heterozygosity are possibly related to regions that have high levels of repeat content and structural variation. I explored the evolutionary relationship between southern Japanese, northern Japanese, and east Korean medaka populations, providing evidence that despite having speciated earlier, the southern Japanese and east Korean populations have interbred more recently with each other than the southern and northern Japanese populations, despite the former pair being presently separated by the Korean Strait. With particular relevance to genetic mapping, I showed that the MIKK panel has a rate of LD decay that would permit a mapping resolution up to 10 times higher than in humans, which in turn would make it more likely to identify the variants responsible for differences in complex traits. 

Finally, I demonstrated the power of ONT long-read sequencing technologies to uncover large-scale structural variation that would otherwise have been hidden from NGS short-read sequencing. In addition, I showed that the traditional reference-anchored approach of genomic alignment can miss large-scale insertions that can otherwise be detected by the emerging graph-based approach. In the future, the ONT sequencing that was performed here on a subset of the MIKK panel will be expanded across the entire panel, which will permit the inclusion of both large- and small-scale variants in F~2~-cross GWAS studies.

## Chapter \@ref(Pilot-chap) {-}

In this chapter I described a custom assay for measuring bold-shy behaviours in medaka fish that can reliably detect differences: (a) between individuals from different inbred strains – allowing for the quantification of direct genetic effects on behaviour; and (b) between the behaviour of individuals when paired with tank partners from a different strain – allowing for the quantification of indirect genetic effects on behaviour. I found that behaviours exhibited during the assay show minimal variance across the covariates of date of assay, time of assay, tank quadrant, and tank side. This creates confidence that the genetic effects that I observed – both direct and indirect – are not unduly affected by these variables.

I showed that a Hidden Markov Model (**HMM**) adequately classifies movements based on the direction and angle of travel within set time intervals. The HMM method has previously been used to classify maternal behaviour patterns in mice [@carolaHiddenMarkovModel2011], and I have extended the application here to bold-shy behaviours in medaka fish. In principle, this method can be expanded to include additional behavioural features such as proximity to the wall or other objects, inwards/outwards orientation, proximity to the tank partner, and other metrics related to leader-follower dynamics, a possibility that will be tested in future studies.

The HMM classification of behaviours also facilitated the ranking of behaviours by speed and angle of movement, which may be further interpreted in terms of biology and adaptation styles. However, even though the HMM classifies the slowest-moving states as separate modes of movement, one should be cautious when attributing biological meaning to their differences, because even when a fish is almost completely still, the fish object’s centroid (upon which the variables of distance and angle of travel are calculated) will tend to move by one or several pixels through minor changes in the segmentation of the object, and are thus subject to greater noise than the faster-moving states. 

I found that inbred strains of medaka fish can be distinguished by the proportion of time they spend in certain states. The slowest states, capturing no or minimal movement, were the states that most clearly separated the strains, and these differences appeared most clearly at the beginning of the open field and novel object assays. This suggests that for southern Japanese strains such as *iCab*, *HdrR*, and *HO5*, the "freeze" reflex is caused by anxiety, which eventually dissipates over time, indicating habituation. Similar observations showing initial anxiety as shown by freezing behaviour and subsequent habituation have been made in other studies using open field assays with the *iCab* strain [@lucon-xiccatoComparisonAnxietylikeSocial2022].

On the other hand, the northern Japanese strains *Kaga* and *HNI* spent little time in the slow-moving states at the beginning of the video, which indicates either that their habituation sets in far earlier, or that their stress and anxiety is expressed to a lesser degree as freezing behaviour. The latter appears to be more likely for *Kaga*, as in the open field assay, it spends more time in the faster-moving states at the beginning of the video and then slows down thereafter, which suggests that its higher level of movement may be induced by stress. It is interesting then that once the novel object is introduced, *Kaga* tends to move slowly like the other strains. Obviously, introduction to a novel environment and exposure to a potentially dangerous object elicits different behavioural patterns in this strain. Taken together with the overt thigmotaxis behaviour (that is, moving along the sides of the tank during the open field assay component) relative to other strains, its behaviour may suggest an "escape-seeking" response. It will be interesting to carry out further experiments to determine under which fear-inducing conditions *Kaga* reacts with this more frantic style of movement rather than the freeze response that it displays in the novel object component in common with the other strains, and to explore its physiological basis.

With respect to indirect genetic effects, I have found that a fish’s behaviour is affected by the differential behaviour of its tank partner, although the effect is less powerful than the direct genetic effects exerted on a fish's own biology. These indirect genetic effects are detectable when observing the proportions of time the reference fishes spend in certain states over the course of the video, and can be quantified by measuring the frequency of state co-occupancy with their tank partners. 

It is also interesting to find that *iCab* prefers to co-occupy the slow, pan-directional state 4 when paired with another individual from the same strain. As the strains are raised exclusively in the company of individuals from the same strain, that type of behaviour may represent a more comfortable – yet still cautious – mode of movement, or a type of mimicry that tends to occur when in the company of individuals that are more genetically similar, or socially familiar.

In this study I only used one strain as the reference fish, but future experiments can expand on this analysis by using different strains as the reference, thereby exploring how a strain’s genetics influence the degree of their behavioural plasticity, and how the distinct behaviours of strains may interact in intriguing ways -- or in other words, expanding the analysis to incorporate gene-environment interactions. In summary, I showed that the custom combined open-field/novel-object experimental setup in combination with the HMM analysis can reliably detect both direct and indirect genetic effects. This opened up the opportunity to carry out a similar study on a larger panel of inbred strains, which I did with the MIKK panel in Chapter \@ref(MIKK-F2-chap).

## Chapter \@ref(Somite-chap) {-}

In this chapter, I performed the bioinformatic analyses required to map genetic variants associated with somite development periodicity. I first explored the distributions across the F~0~, F~1~ and F~2~ generations of the primary phenotype of interest -- somite periodicity -- as well as the positive control phenotype of unsegmented presomatic mesoderm (**PSM**) area. I discovered that there was a significant difference in phenotype based on the microscope that was used to measure somite periodicity (but not PSM area), and controlled for this downstream.

I then assessed the levels of homozygosity in the F~0~ generation, and discovered that the *Kaga* strain that was used had unexpectedly low levels of homozygosity, likely as a result of contamination with another strain. This created many region of low heterozygosity in the F~1~, which restricted out ability to map variants in the F~2~ generation, with the exception of chr22, the reason for which warrants further investigation. To map the shallow-sequenced F~2~ samples back to the the F~0~ parental haplotypes, I refined the parameters of an HMM to call the recombination blocks based on the expected number of crossovers per chromosome. I discovered a bias away from homozygous-*Kaga* genotypes, likely due to a combination of its lower original homozygosity, and possible genetic incompatibility with the *Cab* strain as a consequence of their ancestral divergence. 

I then used the F~2~ recombination blocks as pseudo-SNPs in a GWAS for both the somite periodicity and PSM area phenotypes, and discovered a strongly-significant locus for each phenotype on separate regions of chr3, which I refined down to discrete regions using a tool for the effects of variant prediction, and provided a set of candidate loci for a CRISPR-Cas 9 knockout screen, which my collaborators are currently performing. Together with the ongoing work of my collaborators, this analysis will shed light on the genetic control of somite periodicity, expanding our knowledge of how this crucial developmental process occurs, and why it varies between different organisms.

## Chapter \@ref(MIKK-F2-chap) {-}

These results of this chapter confirm that the F~2~-cross method applied to the MIKK panel is a powerful technique for the discovery of loci associated with complex traits. I applied modern computational methods to first develop a data-driven analysis of fish behaviour, and then select a subset of inbred lines that diverged in both their own behaviour, and the extent to which they transmitted their behaviours to their social partners. I then performed an F~2~-cross with those lines to map the genetic variants associated with those differences. Although this analysis was clearly under-powered, I nevertheless discovered a number of statistically significant loci in orthologue genes that have been found to be related to neurological phenotypes in mice, rats, and humans. As noted above, my collaborators are in the process of generating another F~2~ generation for these crosses to increase the power of the analysis. Future work can functionally validate these loci with targeted CRISPR-Cas9 knockouts, expanding on our knowledge about the genetic and environmental contributions to trait variation through an improved ability to control both genes and social environment with inbred lines that are representative of the kinds of variation that is naturally occurring in the wild.

## Chapter \@ref(Fst-chap) {-}

The purpose of this final chapter was to determine the extent to which differences in allele frequencies between diverse human populations may contribute to the poor transferability of polygenic scores (**PGS**) to non-European populations. I used $F_{ST}$ as a measure for differences in allele frequencies between the 26 populations in the 1000 Genomes dataset. I then compared the distributions of $F_{ST}$ across all polygenic traits in the GWAS Catalog using a signed $D$ statistic from the Kolmogorov-Smirnov test, ${D_t^S}$, to rank the traits by the distance of their $F_{ST}$ distributions from a random set of allele-frequency-matched control SNPs. I found that for most traits, the $F_{ST}$ distributions of associated alleles differed little from the control SNPs, indicating that the allele frequencies of trait-associated alleles do not differ greatly from non-associated (i.e. background) SNPs. The notable exceptions were HIV viral load, skin pigmentation, eye colour, and hair shape, which showed large differences from their control SNPs, suggesting that -- as would be expected -- the frequencies of the alleles associated with those traits do differ greatly between populations. These results illustrate that most traits have associated alleles that have $F_{ST}$ distributions that do not differ greatly from random SNPs with similar allele frequencies, but there is still a need to quantify how higher $F_{ST}$ distributions might directly affect PGS accuracy.

It would be convenient to extrapolate these findings to conclude that global populations do not differ significantly in the traits themselves, but this is not possible due to two limitations of the $F_{ST}$ measure. The first is that $F_{ST}$ does not correspond to similarities or differences between populations in *trait values*. This is because $F_{ST}$ does not take into account the effect size or the direction of effect of the trait-associated allele, so for highly-polygenic traits like the ones shown here, $F_{ST}$ is almost entirely decoupled from the mean additive genetic value between populations [@bergPopulationGeneticSignal2014a]. This means that even when a trait's associated alleles have low $F_{ST}$, as we see for most traits here, it does not follow that the population mean values for that trait would be the same due to genetic similarities between those populations. As @kremerDecouplingDifferentiationTraits2012 determined by through simulations, for highly polygenic traits, mean trait differences are driven by small, coordinated shifts in allele frequencies, which would not be detected by $F_{ST}$ [@lecorreGeneticDifferentiationQuantitative2012].

The second relevant limitation is that $F_{ST}$ is not sufficiently powered to detect selection for polygenic traits [@bergPopulationGeneticSignal2014], which means that these results cannot be used to determine the extent to which alleles are under selection in a given population. This is apparent in our analysis, where in contrast with @racimoDetectingPolygenicAdaptation2018, height and BMI, which have been found to be under strong differential selection, appear similar to other polygenic traits.

Nevertheless, it is a worthwhile and instructive finding that $F_{ST}$ distributions for trait-associated alleles do not differ greatly from those of non-associated ("background") alleles. In terms of its implications for the poor transferability of PGS to non-European populations, the obvious solution is to increase the representation of those populations in GWAS and PGS studies, as has often been proposed elsewhere [@martinHumanDemographicHistory2017; @martinCriticalNeedsChallenges2018; @bienFutureGenomicStudies2019; @martinClinicalUseCurrent2019;  @sirugoMissingDiversityHuman2019; @wojcikGeneticAnalysesDiverse2019].
