# Conclusions {-}

```{r, include = F}
library(tidyverse)

# Read in line colours
line_cols = readr::read_csv(here::here("results/MIKK_F2/line_colours_0.08.csv"))

# Function for colouring text
colorize <- function(x, color, col_name) {
  if (knitr::is_latex_output()) {
    # remove hash prefix
    color = stringr::str_remove(color, "#")
    sprintf("\\definecolor{%s}{HTML}{%s}\\textcolor{%s}{%s}", col_name, color, col_name, x)
  } else if (knitr::is_html_output()) {
    sprintf("<span style='color: %s;'>%s</span>", color,
      x)
  } else x
}

# Function for pulling the line's matching colour from the data frame, then calling `colorize()`
colour_line = function(LINE){
  COLOUR = line_cols %>% 
    dplyr::filter(line == LINE) %>% 
    dplyr::pull(colour)
  # make colour name
  COL_NAME = paste(LINE,
                   stringr::str_remove(COLOUR, "#"),
                   sep = "_")
  # call colorize function
  colorize(LINE, COLOUR, COL_NAME)
}
```

## Chapter \@ref(MIKK-genomes-chap) {-}

In this first substantive chapter of the thesis, I explored several genomic traits in the MIKK panel that are relevant to its utility for the genetic mapping of complex traits, and analysed the structural variants in a subset of its genomes that could be discovered through the use of Oxford Nanopore Technologies' (**ONT**) long-read sequencing platform. I displayed the inbreeding trajectory of the MIKK panel, confirming that medaka have a remarkable resilience to inbreeding from the wild, and that despite the spread of a bacterial infection during the inbreeding process that destroyed a number of lines, 30% of them nevertheless survived by generation F~15~. I showed the MIKK panel possesses a normally-distributed morphology phenotype (mean relative eye size) as an example of the distribution of complex traits across the panel. 

I confirmed that the inbreeding process resulted in inbred lines that were highly homozygous across almost the entire genome, comparable to the levels of homozygosity that were present in established inbred strains that have been inbreed for over 100  generations, and that regions of residual heterozygosity are possibly related to regions that have high levels of repeat content and structural variation. I explored the evolutionary relationship between southern Japanese, northern Japanese, and east Korean medaka populations, providing evidence that despite having speciated earlier, the southern Japanese and east Korean populations have interbred more recently with each other than the southern and northern Japanese populations, despite the former pair being presently separated by the Korean Strait. With particular relevance to genetic mapping, I showed that the MIKK panel has a rate of LD decay that would permit a mapping resolution up to 10 times higher than in humans, which in turn would make it more likely to identify the variants responsible for differences in complex traits. 

Finally, I demonstrated the power of ONT long-read sequencing technologies to uncover large-scale structural variation that would otherwise have been hidden from NGS short-read sequencing. In addition, I showed that the traditional reference-anchored approach of genomic alignment can miss large-scale insertions that can otherwise be detected by the emerging graph-based approach. In the future, the ONT sequencing that was performed here on a subset of the MIKK panel will be expanded across the entire panel, which will permit the inclusion of both large- and small-scale variants in F~2~-cross GWAS studies.

## Chapter \@ref(Pilot-chap) {-}

In this chapter I described a custom assay for measuring bold-shy behaviours in medaka fish that can reliably detect differences: (a) between individuals from different inbred strains – allowing for the quantification of direct genetic effects on behaviour; and (b) between the behaviour of individuals when paired with tank partners from a different strain – allowing for the quantification of social genetic effects on behaviour. I found that behaviours exhibited during the assay show minimal variance across the covariates of date of assay, time of assay, tank quadrant, and tank side. This creates confidence that the genetic effects that I observed – both direct and indirect – are not unduly affected by these variables.

I showed that a Hidden Markov Model (**HMM**) adequately classifies movements based on the direction and angle of travel within set time intervals. The HMM method has previously been used to classify maternal behaviour patterns in mice [@carolaHiddenMarkovModel2011], and I have extended the application here to bold-shy behaviours in medaka fish. In principle, this method can be expanded to include additional behavioural features such as proximity to the wall or other objects, inwards/outwards orientation, proximity to the tank partner, and other metrics related to leader-follower dynamics, a possibility that will be tested in future studies.

The HMM classification of behaviours also facilitated the ranking of behaviours by speed and angle of movement, which may be further interpreted in terms of biology and adaptation styles. However, even though the HMM classifies the slowest-moving states as separate modes of movement, one should be cautious when attributing biological meaning to their differences, because even when a fish is almost completely still, the fish object’s centroid (upon which the variables of distance and angle of travel are calculated) will tend to move by one or several pixels through minor changes in the segmentation of the object, and are thus subject to greater noise than the faster-moving states. 

I found that inbred strains of medaka fish can be distinguished by the proportion of time they spend in certain states. The slowest states, capturing no or minimal movement, were the states that most clearly separated the strains, and these differences appeared most clearly at the beginning of the open field and novel object assays. This suggests that for southern Japanese strains such as *iCab*, *HdrR*, and *HO5*, the "freeze" reflex is caused by anxiety, which eventually dissipates over time, indicating habituation. Similar observations showing initial anxiety as shown by freezing behaviour and subsequent habituation have been made in other studies using open field assays with the *iCab* strain [@lucon-xiccatoComparisonAnxietylikeSocial2022].

On the other hand, the northern Japanese strains *Kaga* and *HNI* spent little time in the slow-moving states at the beginning of the video, which indicates either that their habituation sets in far earlier, or that their stress and anxiety is expressed to a lesser degree as freezing behaviour. The latter appears to be more likely for *Kaga*, as in the open field assay, it spends more time in the faster-moving states at the beginning of the video and then slows down thereafter, which suggests that its higher level of movement may be induced by stress. It is interesting then that once the novel object is introduced, *Kaga* tends to move slowly like the other strains. Obviously, introduction to a novel environment and exposure to a potentially dangerous object elicits different behavioural patterns in this strain. Taken together with the overt thigmotaxis behaviour (that is, moving along the sides of the tank during the open field assay component) relative to other strains, its behaviour may suggest an "escape-seeking" response. It will be interesting to carry out further experiments to determine under which fear-inducing conditions *Kaga* reacts with this more frantic style of movement rather than the freeze response that it displays in the novel object component in common with the other strains, and to explore its physiological basis.

With respect to social genetic effects, I have found that a fish’s behaviour is affected by the differential behaviour of its tank partner, although the effect is less powerful than the direct genetic effects exerted on a fish's own biology. These social genetic effects are detectable when observing the proportions of time the reference fishes spend in certain states over the course of the video, and can be quantified by measuring the frequency of state co-occupancy with their tank partners. 

It is also interesting to find that *iCab* prefers to co-occupy the slow, pan-directional state 4 when paired with another individual from the same strain. As the strains are raised exclusively in the company of individuals from the same strain, that type of behaviour may represent a more comfortable – yet still cautious – mode of movement, or a type of mimicry that tends to occur when in the company of individuals that are more genetically similar, or socially familiar.

In this study I only used one strain as the reference fish, but future experiments can expand on this analysis by using different strains as the reference, thereby exploring how a strain’s genetics influence the degree of their behavioural plasticity, and how the distinct behaviours of strains may interact in intriguing ways -- or in other words, expanding the analysis to incorporate gene-environment interactions. In summary, I showed that the custom combined open-field/novel-object experimental setup in combination with the HMM analysis can reliably detect both direct and social genetic effects. This opened up the opportunity to carry out a similar study on a larger panel of inbred strains, which I did with the MIKK panel in Chapter \@ref(MIKK-F2-chap).

## Chapter \@ref(Somite-chap) {-}

In this chapter, I performed the bioinformatic analyses required to map genetic variants associated with somite development periodicity. I first explored the distributions across the F~0~, F~1~ and F~2~ generations of the primary phenotype of interest -- somite periodicity -- as well as the positive control phenotype of unsegmented presomatic mesoderm (**PSM**) area. I discovered that there was a significant difference in phenotype based on the microscope that was used to measure somite periodicity (but not PSM area), and controlled for this downstream.

I then assessed the levels of homozygosity in the F~0~ generation, and discovered that the *Kaga* strain that was used had unexpectedly low levels of homozygosity, likely as a result of contamination with another strain. This created many region of low heterozygosity in the F~1~, which restricted out ability to map variants in the F~2~ generation, with the exception of chr22, the reason for which warrants further investigation. To map the shallow-sequenced F~2~ samples back to the the F~0~ parental haplotypes, I refined the parameters of an HMM to call the recombination blocks based on the expected number of crossovers per chromosome. I discovered a bias away from homozygous-*Kaga* genotypes, likely due to a combination of its lower original homozygosity, and possible genetic incompatibility with the *Cab* strain as a consequence of their ancestral divergence. 

I then used the F~2~ recombination blocks as pseudo-SNPs in a GWAS for both the somite periodicity and PSM area phenotypes, and discovered a strongly-significant locus for each phenotype on separate regions of chr3, which I refined down to discrete regions using a tool for the effects of variant prediction, and provided a set of candidate loci for a CRISPR-Cas 9 knockout screen, which my collaborators are currently performing. Together with the ongoing work of my collaborators, this analysis will shed light on the genetic control of somite periodicity, expanding our knowledge of how this crucial developmental process occurs, and why it varies between different organisms.

## Chapter \@ref(MIKK-F2-chap) {-}

The results from this chapter provide promising evidence that the assay and methods described can allow researchers to identify genetic loci associated with differences in behaviours - and potentially difference in the transmission of behaviour onto social companions - with functional relevance to humans. However, before seeking to functionally validate the variants, there are a number of steps that we could take to refine the analysis, and thereby increase our confidence in the variants we ultimately select for validation.

With the benefit of hindsight, there are a several aspects of the analysis described in **Chapter \@ref(MIKK-F2-chap)** that I would have performed differently. The first relates to the choices of certain parental lines for the F~2~-cross, which were too heavily reliant on the aggregate measures of speed and charisma, without due regard to the level of behavioural variance observed within each line. As evident in **Figures \@ref(fig:F2-time-dge-of)** and **\@ref(fig:F2-time-dge-no)** above, lines `r colour_line("8-2 (‘Gail’)")` and `r colour_line("21-2 (‘Amelia’)")` showed large within-line variances. For `r colour_line("8-2 (‘Gail’)")` in particular, the differences appear to correspond to the date on which they were assayed - specifically, in the first run the four `r colour_line("8-2 (‘Gail’)")` individuals showed almost no movement, whereas in the second run they showed much higher levels of movement. The different within-line variances are an interesting phenotype to explore further, as they suggest that different lines show different degrees of sensitivity to environmental changes. However, for this analysis, high within-line variances make it difficult to ascertain the line's "true" phenotype, and therefore make them unsuitable for selection based on our axes of interest for this study (bold vs shy; high vs low behavioural transmission). 

An appropriate substitute for `r colour_line("8-2 (‘Gail’)")` may be line `r colour_line("139-4")`, which had low within-line variance for mean speed, and the highest median state co-occupancy in the novel object assay component. From the analysis of Chapter \@ref(Pilot-chap), which concluded after the lines were already selected for the F~2~-cross, it became apparent that the novel object assay was particularly useful for revealing social genetic effects. I hypothesise that at times of higher stress or predation threat, the fish take more behavioural cues from their tank partners. Line `r colour_line("139-4")` therefore would have been a good candidate for the slow-moving, high-charisma line. 

Similarly, lines `r colour_line("13-2")` or `r colour_line("94-1")` would be preferred substitutes for line `r colour_line("21-2 (‘Amelia’)")` as the fast-moving, high charisma line. They are faster than `r colour_line("21-2 (‘Amelia’)")`, but unlike the fastest line `r colour_line("10-1 (‘Janeway’)")`, they show a degree of habituation (where most individuals moved slowly at the beginning of the assay before eventually speeding up). They also showed lower within-line variance than `r colour_line("21-2 (‘Amelia’)")`, but comparable measures for the SGE measures of state co-occupancy and reference deviation. 

One way of avoiding these issues would have been to used a more quantitative, objective approach to selecting the lines. For example, I could have used Mann-Whitney tests to quantify the levels of differences between pairs of lines for a trait of interest (as I did for **Figure \@ref(fig:mikk-param-comp)**), and then select the lines that maximised that statistic. Such metrics would still perhaps need to be qualitatively weighted against other relevant traits, such as within-line variance. Nevertheless, my downstream analyses show that despite not having potentially selected the ideal lines for the F~2~-cross, the differences between them were sufficient to identify genetic loci associated with our phenotypes of interest, even with a relatively small sample size of 271 F~2~ individuals.

Another aspect of the analysis that could have been improved was to use one or more MIKK panel lines as the reference line instead of *`r colour_line("iCab")`*. As previously mentioned, *`r colour_line("iCab")`* is a relatively fast line, which made it difficult to detect social genetic effects when paired with similarly fast-moving lines. Instead, we could have made use of the behavioural variance across the MIKK panel to choose a reference line that exhibits moderate levels of movement. However, we would not be able to assess the levels of movement across the MIKK panel without first assaying the panel with a single reference strain, and we chose *`r colour_line("iCab")`* because we not only had a good understanding of its behavioural traits based on our findings from Chapter \@ref(Pilot-chap), but also because it has high fecundity, which made many replicates available and in turn allowed us to avoid using the same reference fish more than once within 24 hours. Now that we have carried out this analysis, we would be able to select a more appropriate line as the reference in any future analyses.

The other benefit from using a MIKK panel line as the reference fish would be to open up the possibility of exploring social genetic effect interactions. For example, if we used two different MIKK lines as the reference, we could explore how those lines differ in the degree to which they are influenced by their tank partner. To state the issue inversely, it would reveal differences in the extent to which different lines transmit their behaviour onto their (reference) tank partner, depending on both of their genetics. A question for further studies could therefore be framed as, 'given line `r colour_line("18-2 (‘Elsa’)")` appears to possess a high level of charisma, is that level consistent when they are paired with different MIKK panel lines, and if it differs across lines, then why?' Due to the relative onerousness of running this behavioural assay, such experiments would involve a substantial time commitment, but in light of the findings presented here, the investigations could be restricted to a subset of the MIKK panel that display interesting behavioural phenotypes. 

### Future directions for the F~2~-cross GWAS for bold-type behaviours in the MIKK panel

Our collaborators are in the process of breeding more F~2~ individuals to increase the sample size for this analysis. For the subsequent breeding program, I have suggested that we replace line `r colour_line("21-2 (‘Amelia’)")` with either `r colour_line("13-2")` or `r colour_line("23-1")`, which both show lower within-line variance in mean speed, but similarly high levels of speed and behavioural transmission (as measured by state co-occupancy and *`r colour_line("iCab")`* reference deviation). Any additional F~2~ individuals will still need to be phenotyped and sequenced, and the phenotype data collection (i.e. video recording the behavioural assay) can be relatively laborious, so this process will likely take some time.

In the meantime, I have 96 Kiyosu CC individuals whose videos have already been processed, and whose DNA has been shallow-sequenced (~1x). It was not feasible to include that data in this analysis due to time constraints. As they have been allowed to breed freely from the same Kiyosu population as the MIKK panel, they are not strongly homozygous, and they are likely to possess many additional genetic variants that are not represented in the MIKK panel. Unlike the F~2~ samples, which I could impute directly from their F~0~ parental lines, I will need to use the full high-coverage variant call set from the MIKK panel to impute their genotypes. <!--I am therefore not able to impute their genotypes from high-coverage parental strains, as I did with the F~2~ individuals. The alignments to the reference are also likely to include many mapping errors due to the lack of sufficient coverage. The additional information they provide will therefore not be as useful as additional F~2~ individuals, but it should nevertheless improve the power to detect casual variants. The amount of additional information we obtain from the Kiyosu CC individuals will also inform whether we include them in this way in future F~2~-crosses. <!--[NOT REALLY. THERE ISN'T AN ALIGNMENT TO REFERENCE PROBLEM. BUT WHAT THERE IS IS THAT THINGS WILL HAV ETO BE PROCESSED DIFFERENTLY. IN THE F2S WE ASSUME THAT THE PARENTS ARE HOMOZYGOUS. THIS IS MUCH CLOSER TO THE HUMAN SITUATION. OPTIMAL IS 1X IN THEORY, 4X IN PRACTICE. LIKE 1000 GENOMES. PANEL OF 80 BECOME LIKE THE HIGH COVERAGE DATASET. THIS IS MUCH CLOSER TOT HE OUTBRED SITUTATION WITH HUMANS, WHERE THE PANEL WILL BE THE HIGH COVERAGE PANEL, AND WE'LL HAVE SOME LOW-COVERAGE SAMPLES WHERE WE'LL NEED TO DO SOME IMPUTATION.]-->

A second issue I plan to address is the failure of F~2~ reads to align to chromosomes 22 and 23 (see Chapter \@ref(MIKK-F2-chap) Figure \@ref(fig:F2-recomb-blocks))<!--. The problem occurs across most of the sample, the coverage does not look skewed for those chromosomes (**Figure \@ref(fig:F2-coverage)**), and nor do the counts of homozygous-divergent SNPs between the F~0~ parents do not look consistently smaller for those chromosomes (**Figure \@ref(fig:homo-div-snp-counts)**). There is therefore-->, which may have been caused by an error at some stage of the bioinformatic pipeline that I constructed, and warrants further investigation.
<!--
(ref:F2-coverage) Mean depth of sequencing coverage per chromosome for the 271 F~2~ samples used in this analysis.

```{r F2-coverage, echo = F, fig.cap = '(ref:F2-coverage)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/coverage_F2.png"), auto_pdf = T)
```

(ref:homo-div-snp-counts) Counts of SNPs that are homozygous-divergent between the parental F~0~ lines in each of the 12 crosses used.

```{r homo-div-snp-counts, echo = F, fig.cap = '(ref:homo-div-snp-counts)', out.width='100%'}
knitr::include_graphics(here::here("book/figs/mikk_behaviour/homo_div_snp_counts.png"), auto_pdf = T)
```
-->

A third issue is the matter of selecting the optimal HMM parameters for calling the recombination blocks, as described in Chapter \@ref(Somite-chap). For the analysis in this present chapter I used the same HMM parameters as I had used there. In that chapter, I used the $p$-values from the resulting genetic linkage analysis as a rough measure of genotyping quality (assuming that lower $p$-values were driving by more accurate genotypes), together with a qualitative assessment of how many crossovers I expected to observe in a particular chromosome for a subset of the sample. The issue can be resolved by obtaining "ground truth" genotype calls through high-coverage sequence data for a set of samples that have also been shallow-sequenced. This would allow one to compare the HMM recombination block calls with the ground truth genotypes, and thereby train the model to optimise the accuracy of its predictions for the locations of the recombination sites (using a test dataset to validate the parameters). I expect the improved genotype calls will further improve the power to detect genetic variants.

With respect to the genetic linkage analysis, due to the method I used to calculate the genetic relationship matrix (**GRM**), I was compelled to exclude all SNPs that were missing calls in any of the samples, reducing the number of SNPs to merely ~40K out of the ~20M in the full dataset. Not only does this reduce the accuracy of the GRM as a means of capturing the relative relatedness between samples, but the problem will only be exacerbated with the inclusion of additional samples (as the likelihood of a SNP having at least one sample with a missing call will increase with the number of samples). I will therefore adapt my method to include SNPs with missing calls, which should further increase statistical power.

Finally, I applied a separate genetic linkage association test for the state frequency of each HMM state, where it may be beneficial to apply a method that can combine the information across all states into a single phenotype (if not fewer phenotypes), and then run an association test on that reduced phenotype. There are a number of alternatives to consider, including principal components analysis (PCA) of the state frequencies, or the software MTAG [Multi-Trait Analysis of Genome-wide summary statistics , @turleyMultitraitAnalysisGenomewide2018] which can combine the summary statistics from multiple genetic association tests. Ultimately, the hope is that by leveraging the information about the frequencies in which the fishes occupy each state, combined across *all* states, while appropriately taking into account the interdependence of those frequencies (e.g. the more time a fish spends in state 3, the less time it is likely to spend in state 13), it will further strengthen these results.

Aside from modifications designed to increase statistical power, there is also capacity to improve the way that I phenotype the individuals, and in particular how I characterise social genetic effects. In this analysis I have only used two behavioural variables: distance and angle of travel. This behavioural dataset is extremely rich, and myriad variables could be extracted from the videos. Shoaling traits would have special relevance to the study of social genetic effects. For instance, one could extract three variables that would be associated with shoaling behaviours: a) distance between tank partners; b) synchrony in speed; and c) parallelism in direction of travel. These three variables could then be reduced down to two with an HMM (i.e. one state representing "shoaling", and another representing "not shoaling"). The relative frequencies that the different MIKK lines spend in these states would allow one to rank them by the degree of shoaling behaviour they exhibit, and this would likely be a far more accurate representation of social genetics than what I have shown here. 

## Chapter \@ref(Fst-chap) {-}

The purpose of this final chapter was to determine the extent to which differences in allele frequencies between diverse human populations may contribute to the poor transferability of polygenic scores (**PGS**) to non-European populations. I used $F_{ST}$ as a measure for differences in allele frequencies between the 26 populations in the 1000 Genomes dataset. I then compared the distributions of $F_{ST}$ across all polygenic traits in the GWAS Catalog using a signed $D$ statistic from the Kolmogorov-Smirnov test, ${D_t^S}$, to rank the traits by the distance of their $F_{ST}$ distributions from a random set of allele-frequency-matched control SNPs. I found that for most traits, the $F_{ST}$ distributions of associated alleles differed little from the control SNPs, indicating that the allele frequencies of trait-associated alleles do not differ greatly from non-associated (i.e. background) SNPs. The notable exceptions were race-associated traits including skin pigmentation, eye colour, and hair shape, which showed large differences from their control SNPs, suggesting that -- as would be expected -- the frequencies of the alleles associated with those traits do differ greatly between populations. These results provide evidence that differences in allele frequencies for most traits may not contribute significantly to the poor transferability of PGS to non-European populations. 

It would be convenient to extrapolate these findings to conclude that global populations do not differ significantly in the traits themselves, but this is not possible due to two limitations of the $F_{ST}$ measure. The first is that $F_{ST}$ does not correspond to similarities or differences between populations in *trait values*. This is because $F_{ST}$ does not take into account the effect size or the direction of effect of the trait-associated allele, so for highly-polygenic traits like the ones shown here, $F_{ST}$ is almost entirely decoupled from the mean additive genetic value between populations [@bergPopulationGeneticSignal2014a]. This means that even when a trait's associated alleles have low $F_{ST}$, as we see for most traits here, it does not follow that the population mean values for that trait would be the same due to genetic similarities between those populations. As @kremerDecouplingDifferentiationTraits2012 determined by through simulations, for highly polygenic traits, mean trait differences are driven by small, coordinated shifts in allele frequencies, which would not be detected by $F_{ST}$ [@lecorreGeneticDifferentiationQuantitative2012].

The second relevant limitation is that $F_{ST}$ is not sufficiently powered to detect selection for polygenic traits [@bergPopulationGeneticSignal2014], which means that these results cannot be used to determine the extent to which alleles are under selection in a given population. This is apparent in our analysis, where in contrast with @racimoDetectingPolygenicAdaptation2018, height and BMI, which have been found to be under strong differential selection, appear similar to other polygenic traits.

Nevertheless, it is a worthwhile and instructive finding that $F_{ST}$ distributions for trait-associated alleles do not differ greatly from those of non-associated ("background") alleles. In terms of its implications for the poor transferability of PGS to non-European populations, the obvious solution is to increase the representation of those populations in GWAS and PGS studies, as has often been proposed elsewhere [@martinHumanDemographicHistory2017; @martinCriticalNeedsChallenges2018; @bienFutureGenomicStudies2019; @martinClinicalUseCurrent2019;  @sirugoMissingDiversityHuman2019; @wojcikGeneticAnalysesDiverse2019].
