<!doctype html>
<html lang="en">
<head>
  <meta charset="utf-8" />
  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">

  <title>Japanese courage: a genetic analysis of complex traits in medaka fish and humans</title>

    <meta name="author" content="Ian Brettell" />
  
   <meta name="description" content="<p>Thesis for the award of Doctor of Philosophy from EMBL-EBI and the University of Cambridge.</p>" />
   <meta name="generator" content="placeholder" />
  <meta property="og:title" content="Japanese courage: a genetic analysis of complex traits in medaka fish and humans" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="<p>Thesis for the award of Doctor of Philosophy from EMBL-EBI and the University of Cambridge.</p>" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Japanese courage: a genetic analysis of complex traits in medaka fish and humans" />
  
  <meta name="twitter:description" content="<p>Thesis for the award of Doctor of Philosophy from EMBL-EBI and the University of Cambridge.</p>" />
  
  <!-- JS -->
  <script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script>
  <script src="https://kit.fontawesome.com/6ecbd6c532.js" crossorigin="anonymous"></script>
    <script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
    <link href="libs/bootstrap-4.6.0/bootstrap.min.css" rel="stylesheet" />
    <script src="libs/bootstrap-4.6.0/bootstrap.bundle.min.js"></script>
    <script src="libs/bs3compat-0.4.0/transition.js"></script>
    <script src="libs/bs3compat-0.4.0/tabs.js"></script>
    <script src="libs/bs3compat-0.4.0/bs3compat.js"></script>
    <link href="libs/bs4_book-1.0.0/bs4_book.css" rel="stylesheet" />
    <script src="libs/bs4_book-1.0.0/bs4_book.js"></script>
    <style>
    #content {
        font-size: 0.9rem;
    }
    </style>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script>

  <!-- CSS -->
  <style type="text/css">
    
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  </style>
    <style type="text/css">
    /* Used with Pandoc 2.11+ new --citeproc when CSL is used */
    div.csl-bib-body { }
    div.csl-entry {
      clear: both;
        }
    .hanging div.csl-entry {
      margin-left:2em;
      text-indent:-2em;
    }
    div.csl-left-margin {
      min-width:2em;
      float:left;
    }
    div.csl-right-inline {
      margin-left:2em;
      padding-left:1em;
    }
    div.csl-indent {
      margin-left: 2em;
    }
  </style>
      <link rel="stylesheet" href="style.css" />
  
</head>

<body data-spy="scroll" data-target="#toc">

<div class="container-fluid">
<!--bookdown:title:start-->
<!--bookdown:title:end-->

<!--bookdown:toc:start-->
<div class="row">
  <header class="col-sm-12 col-lg-3 sidebar sidebar-book">
    <a class="sr-only sr-only-focusable" href="#content">Skip to main content</a>

    <div class="d-flex align-items-start justify-content-between">
      <h1>
        <a href="index.html" title="">Japanese courage: a genetic analysis of complex traits in medaka fish and humans</a>
      </h1>
      <button class="btn btn-outline-primary d-lg-none ml-2 mt-1" type="button" data-toggle="collapse" data-target="#main-nav" aria-expanded="true" aria-controls="main-nav"><i class="fas fa-bars"></i><span class="sr-only">Show table of contents</span></button>
    </div>

    <div id="main-nav" class="collapse-lg">
      <form role="search">
        <input id="search" class="form-control" type="search" placeholder="Search" aria-label="Search">
      </form>

      <nav aria-label="Table of contents">
        <h2>Table of contents</h2>
        <div id="book-toc"></div>

        <div class="book-extra">
          <p><a id="book-repo" href="#">View book source <i class="fab fa-github"></i></a></li></p>
        </div>
      </nav>
    </div>
  </header>

  <main class="col-sm-12 col-md-9 col-lg-7" id="content">
<!--bookdown:toc:end-->
<!--bookdown:body:start-->
<div id="about" class="section level1 unnumbered">
<h1 class="unnumbered">About</h1>
<div id="summary" class="section level2" number="0.1">
<h2><span class="header-section-number">0.1</span> Summary</h2>
<p>Japanese courage: a genetic analysis of complex traits in medaka fish and humans</p>
<p>This thesis primarily explores how an individual’s genes interact with the genes of their social companions to create differences in behaviour, using the Japanese medaka fish as a model organism. Chapter 1 sets out the introduction to the diverse topics covered in this thesis.</p>
<p>Chapter 2 describes several genomic characteristics of the Medaka Inbred Kiyosu-Karlsruhe (MIKK) panel, which comprises 80 inbred lines of medaka that were bred from a wild population residing in Kiyosu, southern Japan. In this chapter I plot the inbreeding trajectory of the MIKK panel, and analyse its evolutionary relationship with other previously established inbred medaka strains; degree of homozygosity; rate of linkage disequilibrium decay; repeat content; and structural variation, all which relate to its utility for the genetic mapping of complex traits.</p>
<p>In Chapter 3, I use a custom behavioural assay to characterise and classify bold-shy behaviours in 5 previously established inbred medaka lines. Here I describe the assay, assess its robustness against confounding factors, and apply a hidden markov model (HMM) to classify the fishes’ behaviours across a spectrum of boldness-shyness based on their distance and angle of travel between time points. I describe how the different lines differ in their behaviours over the course of the assay (a direct genetic effect) and how the behaviour of a single “reference” line (<em>iCab</em>) differs in the presence of different lines (a social genetic effect).</p>
<p>In Chapter 4, I explain how I applied this behavioural assay to the MIKK panel in order to identify lines that diverge in both their own bold-shy behaviours (the direct genetic effect) and the extent to which they transmit those behaviours onto their tank partners (the social genetic effect). I then describe how we used those divergent lines as the parental lines in a multi-way F2 cross in an attempt to isolate the genetic variants that are associated with both direct and social genetic effects.</p>
<p>In Chapter 5 I describe the bioinformatic processes and genetic association models used to map the variants associated with differences in the period of somite development, based on a separate F2 cross between the southern Japanese <em>iCab</em> strain, and the northern Japanese <em>Kaga</em> strain.</p>
<p>Finally, in Chapter 6, I compare and rank all complex traits in the GWAS Catalog based on the extent to which their associated alleles vary across global human populations, using the Fixation Index (Fst) as a metric and the 1000 Genomes dataset as a sample of global genetic variation. In this chapter I set out the bioinformatic pipelines used to process the data, present the distributions of Fst for trait-associated alleles across the genome, and use the Kolmogorov-Smirnov test to compare the distributions of Fst across different traits.</p>
<p>Altogether, this thesis describes some of the genomic characteristics of both medaka fish and humans, and how those variations relate to differences in complex traits, with a particular focus on the genetic causes of adaptive behaviours and the transmission of those behaviours onto one’s social companions.</p>
<!--chapter:end:index.Rmd-->
</div>
</div>
<div id="Introduction" class="section level1" number="1">
<h1><span class="header-section-number">1</span> Introduction</h1>
<p>Humankind has long sought to understand the basis of biological variation. What gives rise to the wondrous variety of life forms on Earth? Why do individuals of a particular species differ from one another? How do children inherit traits that are similar to those of their parents, yet on the whole remain distinct from both their parents and their siblings? And are the traits we care about – our health, our intelligence, our ability to thrive in a changing world – pre-determined from birth, or continuously pliable throughout our lives? These questions fundamentally concern the natural laws of inheritance, which until recently remained mysterious and obscure. As a consequence of rapidly-improving technologies, our ability to answer them is becoming increasingly tractable, yet the interplay between genes and environment are still the subject of controversy, especially in relation to human traits such as intelligence.</p>
<p>In this thesis, I explore the extent to which phenotypic variation is determined by an individual’s own genes, and how it may be mediated by the genes of one’s social companions. I provide particular focus on the phenotype of bold-type behaviours. As genetics is a relatively recent area of study, I first provide a brief history of the field, from the first theories of inheritance proposed by the Ancient Greeks, through Darwin’s theory of evolution and Mendel’s identification of the units of inheritance, the study of continuous variation by Galton, Pearson, and Fisher, and the “modern evolutionary synthesis”; early genetic mapping by Thomas Morgan and the extensive use of model organisms – including medaka fish – to eludicate the principles of genetic variation and inheritance; modern DNA sequences methods and Genome-Wide Association Studies (<strong>GWAS</strong>); and finally the combination of these traditional and modern genetic techniques to study direct and social genetic effects on bold-type behaviours in the medaka fish.</p>
<div id="a-brief-history-of-genetics" class="section level2" number="1.1">
<h2><span class="header-section-number">1.1</span> A brief history of genetics</h2>
<div id="ancient-greece" class="section level3" number="1.1.1">
<h3><span class="header-section-number">1.1.1</span> Ancient Greece</h3>
<p>Throughout ancient history, the sources of biological variation were completely unknown. With limited technologies available to them, the Ancient Greek philosophers proposed theories for the inheritance of traits that they observed in humans. Around 500 BC, Pythagoras applied his knowledge of triangles to the question of how traits are inherited from one’s parents. He proposed the theory known as “spermism”, positing that hereditary information was passed down from parent to child via male sperm, with the female only providing the nutrients that would allow it to grow, and, like the theorem that bears his name, that these two sides of the “triangle” would determine the length of the third side: the characteristics of the child.<span class="citation"><a href="#fn1" class="footnote-ref" id="fnref1"><sup>1</sup></a></span> Over a century later, in 380 BC, Plato extended this metaphor in <em>The Republic</em> to argue that this principle could be applied in order to perfect humanity, by breeding perfect combinations of parents at perfect times.<span class="citation"><a href="#fn2" class="footnote-ref" id="fnref2"><sup>2</sup></a></span></p>
<p>Aristotle later joined the discussion with his treatise <em>Generation of Animals</em>,<span class="citation"><a href="#fn3" class="footnote-ref" id="fnref3"><sup>3</sup></a></span> where he noted that children inherited features from their mothers as well as their fathers, raising cases where human skin colour and other traits from maternal ancestors could skip generations, and thus hereditary information must not only be transmitted through sperm. He suggested an idea of “movement” – the transmission of information – from the father’s sperm, which sculpts the mother’s menstrual blood in the same way a carpenter carves a piece of wood.<span class="citation"><a href="#fn4" class="footnote-ref" id="fnref4"><sup>4</sup></a></span> It was, however, impossible for Aristotle to deduce the form in which the information was conveyed.</p>
</div>
<div id="medieval-times" class="section level3" number="1.1.2">
<h3><span class="header-section-number">1.1.2</span> Medieval times</h3>
<p>From medieval times through to the 1800s, the prevailing theory of heredity was that a tiny human – a homunculus – sat within the sperm, waiting to be inflated upon its introduction to a woman’s uterus. But from where did each previous homunculus originate? Logically, the theory would require a homunculus to sit within another homunculus, <em>ad infinitum</em>.<span class="citation"><a href="#fn5" class="footnote-ref" id="fnref5"><sup>5</sup></a></span> The inventor of the microscope, Nicolaas Hartsoeker, even thought he observed a homunculus in a sperm he was studying (<strong>Figure @ref(fig:homunculus-pic)</strong>). But what then triggered the expansion of the human form, involving the development of new parts from embryo to fetus? The answer could only have been some instruction, blueprint, or code, but any specifics remained out of reach.</p>
<p>(ref:homunculus-pic) Preformation, drawn by Nicolaas Hartsoeker in 1695. Image adapted from <span class="citation">Wikimedia Commons<a href="#fn6" class="footnote-ref" id="fnref6"><sup>6</sup></a></span>.</p>
<div class="figure">
<img src="figs/introduction/homunculus.png" alt="(ref:homunculus-pic)" width="50%" />
<p class="caption">
(#fig:homunculus-pic)(ref:homunculus-pic)
</p>
</div>
</div>
<div id="charles-darwin" class="section level3" number="1.1.3">
<h3><span class="header-section-number">1.1.3</span> Charles Darwin</h3>
<p>During the early 1800s, the prevailing doctrine of the origins of biological variation was Creationism, based on Christianity’s literal interpretation of the Bible’s Book of Genesis.<span class="citation"><a href="#fn7" class="footnote-ref" id="fnref7"><sup>7</sup></a></span> Any mechanistic description of how species – and individuals within the same species – differed from one another was thought to threaten this doctrine, making inquiries of this kind potentially blasphemous, and therefore dangerous..<span class="citation"><a href="#fn8" class="footnote-ref" id="fnref8"><sup>8</sup></a></span></p>
<p>In 1831, two months after graduating from the University of Cambridge, a 22-year-old English clergyman named Charles Darwin (<strong>Figure @ref(fig:charles-darwin-young-portrait)</strong>) boarded the <em>HMS Beagle</em> to commence a voyage around the world that would last for almost five years. Darwin had previously studied medicine and theology, but was drawn to study the natural world, and had apprenticed with his fellow clergyman John Henslow, a botanist and geologist who curated the Cambridge Botanic Garden, who suggested that he join the <em>Beagle</em>’s exploratory survey of South America as the “gentleman scientist” they were seeking to assist with the collection of specimens.<span class="citation"><a href="#fn9" class="footnote-ref" id="fnref9"><sup>9</sup></a></span></p>
<p>(ref:charles-darwin-young-portrait) Portrait of Charles Darwin from the late 1830s by George Richmond.<span class="citation"><a href="#fn10" class="footnote-ref" id="fnref10"><sup>10</sup></a></span></p>
<div class="figure">
<img src="figs/introduction/Charles_Darwin_by_G._Richmond.png" alt="(ref:charles-darwin-young-portrait)" width="100%" />
<p class="caption">
(#fig:charles-darwin-young-portrait)(ref:charles-darwin-young-portrait)
</p>
</div>
<p>As the <em>Beagle</em> travelled down the eastern coast of South America, Darwin collected a vast amount of living specimens and ancient fossils – including large extinct mammals such as the megatherium (<strong>Figure @ref(fig:megatherium)</strong>) – and shipped them back to England.<span class="citation"><a href="#fn11" class="footnote-ref" id="fnref11"><sup>11</sup></a></span></p>
<p>(ref:megatherium) Megatherium fossil collected by Charles Darwin, housed in the Natural History Museum, London. Photograph from.<span class="citation"><a href="#fn12" class="footnote-ref" id="fnref12"><sup>12</sup></a></span></p>
<div class="figure">
<img src="figs/introduction/megatherium.png" alt="(ref:megatherium)" width="100%" />
<p class="caption">
(#fig:megatherium)(ref:megatherium)
</p>
</div>
<p>After rounding Cape Horn and moving northward along the western coast of South America, the <em>HMS Beagle</em> eventually reached the Galápagos Islands on the coast of Peru, an archipelago of 18 islands formed from volcanic lava.<span class="citation"><a href="#fn13" class="footnote-ref" id="fnref13"><sup>13</sup></a></span> Over the course of five weeks, Darwin collected carcasses of birds, lizards, and plants.<span class="citation"><a href="#fn14" class="footnote-ref" id="fnref14"><sup>14</sup></a></span> Upon his return to England, Darwin was hailed as a minor celebrity among natural historians due to the collections of specimens he had gathered and shipped back.<span class="citation"><a href="#fn15" class="footnote-ref" id="fnref15"><sup>15</sup></a></span> John Gould – the ornithologist who lent his (wife’s) name to the Gouldian finch (<strong>Figure @ref(fig:gouldian-finch)</strong>) – told him that the various birds that Darwin thought were a variety of wrens, warblers, blackbirds, and “Gross-beaks” were in fact all 13 different species of finches.<span class="citation"><a href="#fn16" class="footnote-ref" id="fnref16"><sup>16</sup></a></span></p>
<p>(ref:gouldian-finch) The Gouldian Finch, an Australian native bird described by British ornithological artist John Gould in 1844 and named after his deceased wife Elizabeth.<span class="citation"><a href="#fn17" class="footnote-ref" id="fnref17"><sup>17</sup></a></span> Photograph by Sarah R. Pryke, published in <span class="citation">Sarah R Pryke and Simon C Griffith<a href="#fn18" class="footnote-ref" id="fnref18"><sup>18</sup></a></span>.</p>
<div class="figure">
<img src="figs/introduction/gouldian_finch.png" alt="(ref:gouldian-finch)" width="100%" />
<p class="caption">
(#fig:gouldian-finch)(ref:gouldian-finch)
</p>
</div>
<p>Each island had produced its own variant, and this caused Darwin to consider whether they had all arisen from a common ancestral finch, branching off like the boughs of a tree over time.<span class="citation"><a href="#fn19" class="footnote-ref" id="fnref19"><sup>19</sup></a></span> He understood that animal breeders took advantage of the natural variation in populations to select for desired traits, but he questioned what force had guided the development of these different varieties of finches in the wild.<span class="citation"><a href="#fn20" class="footnote-ref" id="fnref20"><sup>20</sup></a></span></p>
<p>(ref:darwin-finches) Illustration of variation in Galapagos finches by John Gould, published in <span class="citation">Charles Darwin<a href="#fn21" class="footnote-ref" id="fnref21"><sup>21</sup></a></span>. Image from <span class="citation">Wikimedia Commons<a href="#fn22" class="footnote-ref" id="fnref22"><sup>22</sup></a></span>.</p>
<div class="figure">
<img src="figs/introduction/Darwin's_finches_by_Gould.png" alt="(ref:darwin-finches)" width="100%" />
<p class="caption">
(#fig:darwin-finches)(ref:darwin-finches)
</p>
</div>
<p>In 1798, Thomas Malthus, a curate and amateur economist, had published a paper titled <em>An Essay on the Principle of Population</em>, in which he argued that the human population was in constant struggle with its limited resource pool, which in turn was affected by droughts, floods, epidemics, and diseases.<span class="citation"><a href="#fn23" class="footnote-ref" id="fnref23"><sup>23</sup></a></span> Darwin read the paper and identified this struggle for resources as the natural hand that selected those who possessed traits that would allow them to be more likely to survive.<span class="citation"><a href="#fn24" class="footnote-ref" id="fnref24"><sup>24</sup></a></span></p>
<p>In 1855, Alfred Russel Wallace (<strong>Figure @ref(fig:alfred-wallace)</strong>) published a paper in the <em>Annals and Magazine of Natural History</em>. Like Darwin, he had set off on a voyage to distant lands, and had observed the stunning variation across the Malay Archipelago in populations separated by channels of water.<span class="citation"><a href="#fn25" class="footnote-ref" id="fnref25"><sup>25</sup></a></span> And like Darwin, he also derived his theory on the basis of this variation from Malthus’s paper.<span class="citation"><a href="#fn26" class="footnote-ref" id="fnref26"><sup>26</sup></a></span> In June 1858, he sent a draft paper to Darwin outlining his general theory of evolution and natural selection.<span class="citation"><a href="#fn27" class="footnote-ref" id="fnref27"><sup>27</sup></a></span> Darwin panicked and sent both papers to Lyell, who advised Darwin to have both papers presented simultaneously at the meeting of the Linnean Society so that they could both be credited for the discovery.<span class="citation"><a href="#fn28" class="footnote-ref" id="fnref28"><sup>28</sup></a></span></p>
<p>(ref:alfred-wallace) Alfred Russel Wallace, taken around 1895. Image from <span class="citation">Wikimedia Commons<a href="#fn29" class="footnote-ref" id="fnref29"><sup>29</sup></a></span>.</p>
<div class="figure">
<img src="figs/introduction/Alfred-Russel-Wallace.png" alt="(ref:alfred-wallace)" width="100%" />
<p class="caption">
(#fig:alfred-wallace)(ref:alfred-wallace)
</p>
</div>
<p>The presentation made few waves at the time, but Darwin proceeded to complete his opus, <em>On the Origin of Species by Means of Natural Selection</em>.<span class="citation"><a href="#fn30" class="footnote-ref" id="fnref30"><sup>30</sup></a></span> The book was met with ecstatic reviews.<span class="citation"><a href="#fn31" class="footnote-ref" id="fnref31"><sup>31</sup></a></span> Yet two crucial questions remained: how was the variation within species generated in the first place, and how were the traits transmitted to future generations?<span class="citation"><a href="#fn32" class="footnote-ref" id="fnref32"><sup>32</sup></a></span></p>
</div>
<div id="gregor-mendel" class="section level3" number="1.1.4">
<h3><span class="header-section-number">1.1.4</span> Gregor Mendel</h3>
<p>Around the time the <em>Origin of Species</em> was published, the prevailing theory of heredity was promoted by the French biologist Jean-Baptiste de Lamarck (<strong>Figure @ref(fig:lamarck)</strong>).<span class="citation"><a href="#fn33" class="footnote-ref" id="fnref33"><sup>33</sup></a></span> Lamarck’s theory involved the transmission of traits that were strengthened or weakened by environmental pressures in one generation – such as a giraffe having to stretch its neck to reach the leaves on a canopy – which was then passed down to subsequent generations as a form of “pre-adaptation.”<span class="citation"><a href="#fn34" class="footnote-ref" id="fnref34"><sup>34</sup></a></span></p>
<p>(ref:lamarck) Portrait of Jean Baptiste de Lamarck from 1802-1803 by Charles Thévenin.<span class="citation"><a href="#fn35" class="footnote-ref" id="fnref35"><sup>35</sup></a></span></p>
<div class="figure">
<img src="figs/introduction/Jean-Baptiste_de_Lamarck.png" alt="(ref:lamarck)" width="100%" />
<p class="caption">
(#fig:lamarck)(ref:lamarck)
</p>
</div>
<p>But by extending that logic, if a human had their arm amputated, their children would be born with shortened arms.<a href="#fn36" class="footnote-ref" id="fnref36"><sup>36</sup></a> Darwin was convinced that selection acted upon <em>pre-existing</em> variation in a population, and proposed that that variation was transmitted by hereditary particles called <em>gemmules</em>.<span class="citation"><a href="#fn37" class="footnote-ref" id="fnref37"><sup>37</sup></a></span> During conception, he posited that the gemmules of each parent were blended, like paints. But such blending would result in a monochrome population, preventing the creation of the unique, outlying traits required to drive the variation he observed. Around this time Darwin had recorded notes on an obscure paper titled <em>Experiments in Plant Hybridization</em>, but he appeared to have inadvertently skipped the page containing a description of an obscure paper on pea hybrids by an Augustine monk called Gregor Mendel (<strong>Figure @ref(fig:mendel)</strong>).<span class="citation"><a href="#fn38" class="footnote-ref" id="fnref38"><sup>38</sup></a></span></p>
<p>(ref:mendel) Photograph from <span class="citation">Wikimedia Commons<a href="#fn39" class="footnote-ref" id="fnref39"><sup>39</sup></a></span>.</p>
<div class="figure">
<img src="figs/introduction/Gregor_Mendel_2.png" alt="(ref:mendel)" width="100%" />
<p class="caption">
(#fig:mendel)(ref:mendel)
</p>
</div>
<p>Mendel had aspired to become a teacher, but had failed the teacher’s exam multiple times, so in 1853 he had returned to the monastery in Brno, Morvia (in present day Czech Republic), and planted a crop of peas which he had been breeding for about three years.<span class="citation"><a href="#fn40" class="footnote-ref" id="fnref40"><sup>40</sup></a></span> He had collected 34 strains that bred “true”, meaning that the offspring were identical to the parents in seven traits:<span class="citation"><a href="#fn41" class="footnote-ref" id="fnref41"><sup>41</sup></a></span></p>
<ol style="list-style-type: decimal">
<li><p>the texture of the seed (smooth versus wrinkled)</p></li>
<li><p>the colour of seeds (yellow versus green)</p></li>
<li><p>the colour of the flower (white versus violet)</p></li>
<li><p>the position of the flower (at the tip of the plant versus the branches)</p></li>
<li><p>the colour of the pea pod (green versus yellow)</p></li>
<li><p>the shape of the pea pod (smooth versus crumpled)</p></li>
<li><p>the height of the plant (tall versus short)</p></li>
</ol>
<p>Mendel referred to these alternative versions of a trait as <em>forms</em> – biologists in the 1900s would later refer to them as <em>alleles</em>, from the Greek <em>allos</em>, referring to different subtypes of the same thing.<span class="citation"><a href="#fn42" class="footnote-ref" id="fnref42"><sup>42</sup></a></span> Mendel produced hybrids of plants with different alleles (known as the <strong>F0</strong> generation) to determine which allele the consequent offspring would possess. For eight years (1857-1864), he created hybrid crosses between the F0 strains (known as the filial 1 hybrid (<strong>F1</strong>) generation), and crossed those with each other too (known as the <strong>F2</strong> generation), recording their values for the above traits as we went.<span class="citation"><a href="#fn43" class="footnote-ref" id="fnref43"><sup>43</sup></a></span> He could soon identify patterns in the reams of data he generated:<a href="#fn44" class="footnote-ref" id="fnref44"><sup>44</sup></a> for each of the above 7 traits, the filial 1 hybrid (<strong>F1</strong>) generation expressed only a single allele, which he referred to as the <em>dominant</em>; the hidden allele he referred to as <em>recessive</em>. However, in the F2 generation the recessive allele reappeared at a ratio of 1:3.<span class="citation"><a href="#fn45" class="footnote-ref" id="fnref45"><sup>45</sup></a></span></p>
<p>From these results, Mendel inferred that the F1 hybrids must retain the information from both parents, while only expressing one version of them.<span class="citation"><a href="#fn46" class="footnote-ref" id="fnref46"><sup>46</sup></a></span> Then when the F1 generation was inter-crossed to produce the F2 generation, the recessive allele would only be expressed when inherited with another recessive allele.<span class="citation"><a href="#fn47" class="footnote-ref" id="fnref47"><sup>47</sup></a></span> In 1865, Mendel presented his paper <em>Versuche über Pflanzenhybriden</em> (Experiments on Plant Hybridization) to a group of farmers, botanists and biologists at the Natural Science Society in Brno,<span class="citation"><a href="#fn48" class="footnote-ref" id="fnref48"><sup>48</sup></a></span> and it was later published in the annual <em>Proceedings of the Brno Natural Science Society</em>.<span class="citation"><a href="#fn49" class="footnote-ref" id="fnref49"><sup>49</sup></a></span> He mailed out 40 copies of the paper to various scientific societies, but was only cited four times between 1866 and 1900, and virtually disappeared from scientific literature.<span class="citation"><a href="#fn50" class="footnote-ref" id="fnref50"><sup>50</sup></a></span></p>
<p>In 1900, a Dutch botanist named Hugo de Vries chanced upon Mendel’s paper while working on plant hybrids.<span class="citation"><a href="#fn51" class="footnote-ref" id="fnref51"><sup>51</sup></a></span> Likely driven by self-interest in being credited for the discovery, he proceeded to publish his own findings without mentioning the work of Mendel.<span class="citation"><a href="#fn52" class="footnote-ref" id="fnref52"><sup>52</sup></a></span> He noted a patch of primroses named <em>Oenothera lamarckiana</em><a href="#fn53" class="footnote-ref" id="fnref53"><sup>53</sup></a>, which spontaneously generated variants, which he termed <em>mutants</em>.<span class="citation"><a href="#fn54" class="footnote-ref" id="fnref54"><sup>54</sup></a></span> It became clear to de Vries that variation in populations arose spontaneously, at random, rather than in the Lamarckian mode of contemporaneous reaction to environmental pressures.<span class="citation"><a href="#fn55" class="footnote-ref" id="fnref55"><sup>55</sup></a></span> De Vries’s paper was read by Carl Correns, a botanist in Tübingen, Germany, who had also come across the work of Mendel, and pressured de Vries into acknowledging Mendel’s work in the subsequent version of his publication.<span class="citation"><a href="#fn56" class="footnote-ref" id="fnref56"><sup>56</sup></a></span></p>
<p>A few years earlier, the English botanist William Bateson (<strong>Figure @ref(fig:bateson)</strong>) had published a book titled <em>Materials for the study of variation</em>,<span class="citation"><a href="#fn57" class="footnote-ref" id="fnref57"><sup>57</sup></a></span> in which he noted that biological variation occurs continuously for some traits, and dimorphically for others.<span class="citation"><a href="#fn58" class="footnote-ref" id="fnref58"><sup>58</sup></a></span> In 1900, while on a train from Cambridge to London, Bateson read the second version of de Vries’s paper, and was immediately struck by the significance of Mendel’s original study.<span class="citation"><a href="#fn59" class="footnote-ref" id="fnref59"><sup>59</sup></a></span> He independently confirmed Mendel’s findings, and began to promote Mendel’s work, including by publishing an English translation of Mendel’s original paper.<span class="citation"><a href="#fn60" class="footnote-ref" id="fnref60"><sup>60</sup></a></span> In 1905, Bateson coined a word for the study of these units of inheritance: <em>genetics</em>, from the Greek <em>genno</em>, meaning “to give birth.”<span class="citation"><a href="#fn61" class="footnote-ref" id="fnref61"><sup>61</sup></a></span> A few years later, in 1909, the botanist Wilhelm Johannsen coined a distinct word to denote the unit of inheritance: the <em>gene</em>.<span class="citation"><a href="#fn62" class="footnote-ref" id="fnref62"><sup>62</sup></a></span></p>
<p>(ref:bateson) Portrait of William Bateson, date and author unknown.<span class="citation"><a href="#fn63" class="footnote-ref" id="fnref63"><sup>63</sup></a></span></p>
<div class="figure">
<img src="figs/introduction/Bateson2.png" alt="(ref:bateson)" width="100%" />
<p class="caption">
(#fig:bateson)(ref:bateson)
</p>
</div>
</div>
<div id="statistical-genetics-francis-galton-karl-pearson-and-ronald-fisher" class="section level3" number="1.1.5">
<h3><span class="header-section-number">1.1.5</span> Statistical genetics: Francis Galton, Karl Pearson and Ronald Fisher</h3>
<p>Francis Galton (<strong>Figure @ref(fig:galton)</strong>) was the cousin of Charles Darwin, born in the same year as Mendel (1822), and travelled to Egypt and Sudan in 1844, spurring his lifelong obsession with the differences between human races.<span class="citation"><a href="#fn64" class="footnote-ref" id="fnref64"><sup>64</sup></a></span> His reading of Darwin’s <em>Origin of Species</em> in 1859 galvanised him, and he set out to explore the measurement and variance of heredity in humans, with an emphasis on height, intelligence, temperament, and physical prowess.<span class="citation"><a href="#fn65" class="footnote-ref" id="fnref65"><sup>65</sup></a></span><a href="#fn66" class="footnote-ref" id="fnref66"><sup>66</sup></a> He coined the phrase <em>nature versus nurture</em> to distinguish between hereditary and environmental influences.<span class="citation"><a href="#fn67" class="footnote-ref" id="fnref67"><sup>67</sup></a></span></p>
<p>(ref:galton) Portrait of Francis Galton, taken in the 1850s or early 1860s, originally scanned from <span class="citation">Karl Pearson<a href="#fn68" class="footnote-ref" id="fnref68"><sup>68</sup></a></span>.</p>
<div class="figure">
<img src="figs/introduction/Francis_Galton_1850s.png" alt="(ref:galton)" width="100%" />
<p class="caption">
(#fig:galton)(ref:galton)
</p>
</div>
<p>Through surveys he sent out to men and women in the mid-1880s, he requested they mail him detailed measurements on the height, weight, eye colour, intelligence, and artistic abilities of parents, grandparents, and children in return for a substantial fee.<span class="citation"><a href="#fn69" class="footnote-ref" id="fnref69"><sup>69</sup></a></span> With this data he discovered that tall parents indeed tend to have tall children, albeit on average, although the distribution of heights within a generation fit the shape of a normal distribution, or ‘bell-curve.’<span class="citation"><a href="#fn70" class="footnote-ref" id="fnref70"><sup>70</sup></a></span> In a surprising twist, he also discovered that the mean height of the sons of the tallest fathers tended to be slightly lower than the father’s height, and closer to the population’s average – a phenomenon he described as <em>regression to the mean</em>.<span class="citation"><a href="#fn71" class="footnote-ref" id="fnref71"><sup>71</sup></a></span> The <em>ancestral law of heredity</em> sparked an intellectual war with Bateson, through which they attempted to reconcile the dominant/recessive theory with the quantitative theory proposed by Galton.<span class="citation"><a href="#fn72" class="footnote-ref" id="fnref72"><sup>72</sup></a></span></p>
<p>Spurred by a desire to apply this new science to the advancement of humankind, and by a fear the the “unfit” were outbreeding the “fit,”<span class="citation"><a href="#fn73" class="footnote-ref" id="fnref73"><sup>73</sup></a></span> in 1883 Galton coined the term <em>eugenics</em> for his project to improve the human population through selective breeding.<span class="citation"><a href="#fn74" class="footnote-ref" id="fnref74"><sup>74</sup></a></span> It was his vision for a “science which deals with all influences that improve the inborn qualities of race.”<span class="citation"><a href="#fn75" class="footnote-ref" id="fnref75"><sup>75</sup></a></span> Driven by fear of a world overrun by imbeciles, he wrote:</p>
<blockquote>
<p>What nature does blindly, slowly and ruthlessly, man may do providently, quickly, and kindly. As it lies within his power, so it becomes his duty to work in that direction.</p>
</blockquote>
<p>In the 1890s, in an attempt to tease out the respective contributions of nature and nurture, Galton proposed the first study on human twins. Since twins share identical genetic material, he reasoned, they represent a natural experiment where any substantial similarities between them could be attributed to genes, while any differences were the consequence of environment.<span class="citation"><a href="#fn76" class="footnote-ref" id="fnref76"><sup>76</sup></a></span> However, these studies were hampered by Galton’s failure to distinguish between identical and non-identical twins.</p>
<p>In Galton’s later years, he adopted the English mathematician Karl Pearson (<strong>Figure @ref(fig:pearson)</strong>) as his protégé. From 1893 to 1904, Pearson built upon the work of his mentor, developing a number of statistical techniques for biometry, including the chi-squared test,<span class="citation"><a href="#fn77" class="footnote-ref" id="fnref77"><sup>77</sup></a></span> standard deviation,<span class="citation"><a href="#fn78" class="footnote-ref" id="fnref78"><sup>78</sup></a></span> correlation and regression coefficients,<span class="citation"><a href="#fn79" class="footnote-ref" id="fnref79"><sup>79</sup></a></span> and establishing the foundations of principal components analysis.<span class="citation"><a href="#fn80" class="footnote-ref" id="fnref80"><sup>80</sup></a></span> Pearson extended the application of statistics beyond heredity and into the fields of inheritance, variation, correlation, and natural and sexual selection.<span class="citation"><a href="#fn81" class="footnote-ref" id="fnref81"><sup>81</sup></a></span> Pearson also inherited his mentor’s enthusiasm for eugenics.</p>
<p>(ref:pearson) Portrait of Karl Pearson in 1910.<span class="citation"><a href="#fn82" class="footnote-ref" id="fnref82"><sup>82</sup></a></span></p>
<div class="figure">
<img src="figs/introduction/Karl_Pearson_1910.png" alt="(ref:pearson)" width="100%" />
<p class="caption">
(#fig:pearson)(ref:pearson)
</p>
</div>
<p>When Galton died in 1911, he bequeathed money to University College London for a Galton Eugenics Professorship, a position that was given to his protégé and biographer, the London-born Karl Pearson, who was also the head of the newly created Department of Applied Statistics,<span class="citation"><a href="#fn83" class="footnote-ref" id="fnref83"><sup>83</sup></a></span> the world’s first university statistics department.</p>
<p>The mathematician Ronald Fisher (<strong>Figure @ref(fig:fisher)</strong>) at Caius College in Cambridge began to apply his skills to elucidating how continuous traits, like height, could be driven by genetic variation.<span class="citation"><a href="#fn84" class="footnote-ref" id="fnref84"><sup>84</sup></a></span> In 1918, Fisher published his analysis in a paper entitled “The Correlation between Relatives on the Supposition of Mendelian Inheritance,”<span class="citation"><a href="#fn85" class="footnote-ref" id="fnref85"><sup>85</sup></a></span> where he described how the combination of a large number of Mendelian alleles acting on the same trait would result in a normal distribution, and thereby sparked the field now known as <em>quantitative genetics</em>. This reconciliation between Mendelian inheritance and observed continuous traits was later referred to by Julian Huxley as the “modern evolutionary synthesis.”<span class="citation"><a href="#fn86" class="footnote-ref" id="fnref86"><sup>86</sup></a></span></p>
<ul>
<li><p>Fisher’s work was also later supplemented by that of J. B. S. Haldane and Sewall Wright, who collectively created the field of population genetics.<span class="citation"><a href="#fn87" class="footnote-ref" id="fnref87"><sup>87</sup></a></span></p></li>
<li><p>In <span class="citation">Fisher<a href="#fn88" class="footnote-ref" id="fnref88"><sup>88</sup></a></span> he also developed the concept of <em>variance</em>, as the square of the standard deviation developed by Pearson.</p></li>
</ul>
<div class="figure">
<img src="figs/introduction/Youngronaldfisher2.png" alt="(ref:fisher)" width="100%" />
<p class="caption">
(#fig:fisher)(ref:fisher)
</p>
</div>
<ul>
<li>In <span class="citation">Fisher<a href="#fn89" class="footnote-ref" id="fnref89"><sup>89</sup></a></span>, the author noted:</li>
</ul>
<blockquote>
<p>For stuture the coefficient of correlation between brothers is about .54, which we may interpret by saying that 54% of their variance is accounted for by ancestry alone, and that 46% must have some other explanation.</p>
</blockquote>
<ul>
<li><p>Fisher called this calculation an “analysis of variance”, by which he could partition the <em>causes</em> of variation.<span class="citation"><a href="#fn90" class="footnote-ref" id="fnref90"><sup>90</sup></a></span></p></li>
<li><p>Fisher later moved to the Rothamsted Agricultural Research Station in Harpenden, where he crerated many of the statistical methodologies – such as tests of significance and the design of experiments – that continue to be used by statisticians today.<span class="citation"><a href="#fn91" class="footnote-ref" id="fnref91"><sup>91</sup></a></span></p></li>
<li><p>With Winifred A. Mackenzie, Fisher studies the response of 12 different potato varieties to different manure-based fertilizer treatments, where they set up a plot of two treatments (treatment/no treatment), with three replicate plots within each treatment, and three rows within each replicate plots with different manure treatments (either manure, manure + potassium sulfate, or manure + potassium chloride).<span class="citation"><a href="#fn92" class="footnote-ref" id="fnref92"><sup>92</sup></a></span> From this data Fisher generated the first ANOVA table, listing the various causes of variation along with their respective contribution to total variation in crop yield.<span class="citation"><a href="#fn93" class="footnote-ref" id="fnref93"><sup>93</sup></a></span></p></li>
<li><p>Interaction arose in the ANOVA if there was a deviation from the main effects adding up to fully accounting for the variation, in whcih case there was “non-additivity.”<span class="citation"><a href="#fn94" class="footnote-ref" id="fnref94"><sup>94</sup></a></span></p></li>
<li><p>Fisher was also an ardent eugenicist. He helped create the Cambridge University Eugenics Society in 1911, hosting meetings in his rooms, organising public lectures by well-known eugenicists, assisting at the First Internvaional Eugenics Congress, and even delivering his own eugenic lectures.<span class="citation"><a href="#fn95" class="footnote-ref" id="fnref95"><sup>95</sup></a></span></p></li>
<li><p>He was also a member of the British Eugenics Society, and helped establish the Society’s Committee for Legalising Sterilisation. His statistical innovations were developed in part to make eugenic assessments of the relative importance of nature and nurture when it came to evaluating traits like feeble-mindedness.<span class="citation"><a href="#fn96" class="footnote-ref" id="fnref96"><sup>96</sup></a></span></p></li>
</ul>
<p>Eugenics policies were implemented in the United States, Sweden, Nazi Germany, and other countries, which resulted in the forced sterilisations and deaths of millions.<span class="citation"><a href="#fn97" class="footnote-ref" id="fnref97"><sup>97</sup></a></span></p>
<p>Ultimately, eugenics was adopted as a policy by the Nazi regime, resulting in the genocide of around a million Jews, Poles, and Gypsies. The association between eugenics and the horrors of World War II became inextricable, and the eugenics movement was all but finished.<span class="citation"><a href="#fn98" class="footnote-ref" id="fnref98"><sup>98</sup></a></span></p>
</div>
<div id="genetic-mapping-thomas-morgan-tetsuo-aida-and-theodosius-dobzhansky" class="section level3" number="1.1.6">
<h3><span class="header-section-number">1.1.6</span> Genetic mapping: Thomas Morgan, Tetsuo Aida and Theodosius Dobzhansky</h3>
<p>There remained the question of where the gene resides. In the 1890s, Theodor Boveri, a German embryologist working with sea urchins in Naples, identified the chromosome as the location of the gene.<span class="citation"><a href="#fn99" class="footnote-ref" id="fnref99"><sup>99</sup></a></span> In 1907, on a trip to the United States to give talks on Mendel’s discovery, he met the American cell biologist Thomas Hunt Morgan.<span class="citation"><a href="#fn100" class="footnote-ref" id="fnref100"><sup>100</sup></a></span> Morgan sought to extend Boveri’s work by exploring the architecture of genes, using the fruit fly, <em>Drosophila melanogaster</em>, as a model organism.<span class="citation"><a href="#fn101" class="footnote-ref" id="fnref101"><sup>101</sup></a></span></p>
<p>Around 1905 Morgan began to breed <em>Drosophila</em>, identifying visible variants that he could track over generations, including white versus red eyes, forked versus straight bristles, sable-coloured bodies, curved legs; bent, bat-like wings; disjointed abdomens, and deformed eyes.<span class="citation"><a href="#fn102" class="footnote-ref" id="fnref102"><sup>102</sup></a></span> Through repeated crossing experiments, Morgan discovered that some genes were transmitted together at a higher rate than chance alone. He proposed that these genes were physically “linked” to one another, implying that they were situated on some string within the chromosome.<span class="citation"><a href="#fn103" class="footnote-ref" id="fnref103"><sup>103</sup></a></span> This advanced the understanding of a gene from a purely theoretical unit of inheritance to a physical unit.<span class="citation"><a href="#fn104" class="footnote-ref" id="fnref104"><sup>104</sup></a></span> He further discovered that, occasionally, a gene that was otherwise linked to another could “cross over” from one parental strand to the other, generating offspring with a mixture of parental alleles.<span class="citation"><a href="#fn105" class="footnote-ref" id="fnref105"><sup>105</sup></a></span></p>
<p>While Morgan was carrying out his experiments with <em>Drosophila</em>, Japanese researchers were also studying Mendelian inheritance with the Japanese medaka fish, <em>Oryzias latipes</em>. Following three articles published by three separate professors in Japanese on the recessive inheritance of medaka colour variants,<span class="citation"><a href="#fn106" class="footnote-ref" id="fnref106"><sup>106</sup></a></span> a teacher named Tatsuo Aida (<strong>Figure @ref(fig:aida)</strong>) published the first article in English about medaka in 1921.<span class="citation"><a href="#fn107" class="footnote-ref" id="fnref107"><sup>107</sup></a></span> The study involved a total of 22 crosses of two generations, and among other findings discovered recombination between the X and Y chromosome – the world’s first demonstration in any species of Y-linked inheritance.<span class="citation"><a href="#fn108" class="footnote-ref" id="fnref108"><sup>108</sup></a></span> The editor, William Castle, was so impressed by this last finding that he added the following footnote to the paper:</p>
<blockquote>
<p>It should be pointed out, in justice to the author, that his observations go beyond those of Schmidt in the important respect of showing the occurrence of crossing over between the X and the Y chromosome.</p>
</blockquote>
<div class="figure">
<img src="figs/introduction/Aida.png" alt="(ref:aida)" width="100%" />
<p class="caption">
(#fig:aida)(ref:aida)
</p>
</div>
<p>(ref:aida-fig) Figure from <span class="citation">Aida<a href="#fn109" class="footnote-ref" id="fnref109"><sup>109</sup></a></span> showing the different medaka colour variants used in the study.</p>
<div class="figure">
<img src="figs/introduction/Aida_1921_fig.png" alt="(ref:aida-fig)" width="100%" />
<p class="caption">
(#fig:aida-fig)(ref:aida-fig)
</p>
</div>
<p>In the 1930s, Theodosius Dobzhansky (<strong>Figure @ref(fig:dobzhansky)</strong>), a Ukrainian biologist who had trained with Morgan, used <em>Drosophila pseudoobscura</em> to investigate how genetic variation drove evolution.<span class="citation"><a href="#fn110" class="footnote-ref" id="fnref110"><sup>110</sup></a></span> Using a single population of flies that he raised in different temperatures while controlling for all other environmental variables, he found that after four months, the genetic ratios in the two sub-populations had changed.<span class="citation"><a href="#fn111" class="footnote-ref" id="fnref111"><sup>111</sup></a></span> Through this experiment, he determined that genetic variation was the norm across biology; that the adaptive benefit of a given variant would depend on the particular environment that an individual found itself in, and that most phenotypes were driven by many genes interacting with each other and the environment, while also subject to chance.<span class="citation"><a href="#fn112" class="footnote-ref" id="fnref112"><sup>112</sup></a></span></p>
<p>(ref:dobzhansky) Undated photo of Theodosius Dobzhansky from <span class="citation">Jerry A Coyne<a href="#fn113" class="footnote-ref" id="fnref113"><sup>113</sup></a></span>, reprinted courtesy of UW-Madison Archives, #S05461.</p>
<div class="figure">
<img src="figs/introduction/Dobzhansky.png" alt="(ref:dobzhansky)" width="100%" />
<p class="caption">
(#fig:dobzhansky)(ref:dobzhansky)
</p>
</div>
</div>
<div id="the-discovery-of-the-structure-of-dna" class="section level3" number="1.1.7">
<h3><span class="header-section-number">1.1.7</span> The discovery of the structure of DNA</h3>
<p>By the early 1940s, it was known that genes resided in chromatin, the mixture of proteins and nucleic acids that compose chromosomes. After some flirtation with proteins as the molecule of inheritance, Oswald Avery finally proved that it was in fact deoxyribonucleic acid (<strong>DNA</strong>) that was “the material substance of the gene” – the “cloth from which genes were cut.”<span class="citation"><a href="#fn114" class="footnote-ref" id="fnref114"><sup>114</sup></a></span> After Rosalind Franklin, James Watson, and Francis Crick, and Maurice Wilkins (<strong>Figure @ref(fig:all-four)</strong>) had discovered the structure of DNA using X-ray crystallography in 1953.<span class="citation"><a href="#fn115" class="footnote-ref" id="fnref115"><sup>115</sup></a></span> The double helix also suggested a copying mechanism.<span class="citation"><a href="#fn116" class="footnote-ref" id="fnref116"><sup>116</sup></a></span></p>
<p>(ref:all-four) Composite image of Rosalind Franklin (1956), James Watson (1980s), Francis Crick (1980s) and Maurice Wilkins (early 1990s) from<span class="citation"><a href="#fn117" class="footnote-ref" id="fnref117"><sup>117</sup></a></span> reprinted courtesy of Cold Spring Harbor Laboratory Library and Archive, James D. Watson Collection; Wilkins photo courtesy of TVNZ.</p>
<div class="figure">
<img src="figs/introduction/all_four.png" alt="(ref:all-four)" width="100%" />
<p class="caption">
(#fig:all-four)(ref:all-four)
</p>
</div>
</div>
<div id="the-development-of-dna-sequencing" class="section level3" number="1.1.8">
<h3><span class="header-section-number">1.1.8</span> The development of DNA sequencing</h3>
<p>Frederick Sanger (<strong>Figure @ref(fig:sanger)</strong>), a biochemist at the University of Cambridge, applied a similar methodology as the one he used to decode the sequence of a protein to determine the sequence of a gene.<span class="citation"><a href="#fn118" class="footnote-ref" id="fnref118"><sup>118</sup></a></span> But rather than breaking the molecule down into its components, he reversed the process by identifying each base as it was added to the strand during DNA replication.<span class="citation"><a href="#fn119" class="footnote-ref" id="fnref119"><sup>119</sup></a></span> In 1977 he published what became known as the “Sanger method”, using radio-labelled nucleic acids to identify the precise order and type of nucleotides that made up a DNA sequence,<span class="citation"><a href="#fn120" class="footnote-ref" id="fnref120"><sup>120</sup></a></span> earning him a second Nobel prize in Chemistry in 1980.<span class="citation"><a href="#fn121" class="footnote-ref" id="fnref121"><sup>121</sup></a></span></p>
<p>(ref:sanger) Undated photo of Frederick Sanger from <span class="citation">Wikimedia Commons<a href="#fn122" class="footnote-ref" id="fnref122"><sup>122</sup></a></span>.</p>
<div class="figure">
<img src="figs/introduction/Frederick_Sanger2.png" alt="(ref:sanger)" width="100%" />
<p class="caption">
(#fig:sanger)(ref:sanger)
</p>
</div>
<p>These technological breakthroughs eventually led to the first draft of the human genome being published in 2001 by a large international consortium,<span class="citation"><a href="#fn123" class="footnote-ref" id="fnref123"><sup>123</sup></a></span> while the full sequence (including heterochromatic regions) was only completed in 2022.<span class="citation"><a href="#fn124" class="footnote-ref" id="fnref124"><sup>124</sup></a></span> The improvement in sequencing technologies has continued to progess to the stage where a $100 human genome is on the horizon,<span class="citation"><a href="#fn125" class="footnote-ref" id="fnref125"><sup>125</sup></a></span> and long-read technologies developed by Pacific Biosciences and Oxford Nanopore Technologies are allowing for single reads of over 2 megabases in length<span class="citation"><a href="#fn126" class="footnote-ref" id="fnref126"><sup>126</sup></a></span> and the direct sequencing of DNA modifications.<span class="citation"><a href="#fn127" class="footnote-ref" id="fnref127"><sup>127</sup></a></span> These developments are making genetic technologies ever more accessible and informative, paving the way for greater insights into the genetic basis of phenotypic differences. Yet despite these technological advances, the relative contribution of genetics and environment to differences in complex traits remains unclear – especially in humans – and is therefore an active area of research.</p>
</div>
</div>
<div id="genetic-and-environmental-causes-of-variation-in-complex-traits" class="section level2" number="1.2">
<h2><span class="header-section-number">1.2</span> Genetic and environmental causes of variation in complex traits</h2>
<p>Individual differences in human traits have now been studied for more than a century, yet the causes of variation in human traits remain uncertain and controversial.<span class="citation"><a href="#fn128" class="footnote-ref" id="fnref128"><sup>128</sup></a></span> Specifically, the partitioning of observed variability into underlying genetic and environmental sources and the relative importance of additive and non-additive genetic variation continue to be debated.<span class="citation"><a href="#fn129" class="footnote-ref" id="fnref129"><sup>129</sup></a></span></p>
<p>A simplified causal model for the inheritance of traits from one’s parents is presented in <strong>Figure @ref(fig:pheno-causal-model)</strong>, adapted from <span class="citation">Tim T. Morris et al.<a href="#fn130" class="footnote-ref" id="fnref130"><sup>130</sup></a></span> The ancestry of each parent determines both their genotypes and phenotypes (yellow lines). The parents’ genotypes directly affect their own phenotypes (black arrows genotype → phenotype). The parents then transmit half of each of their alleles <em>directly</em> to their offspring (black arrows from genotype → genotype), while the non-transmitted alleles also affect their offspring <em>indirectly</em> through their effect on the parents’ phenotypes (red lines), a phenomenon known as “<strong>dynastic effects</strong>.”<span class="citation"><a href="#fn131" class="footnote-ref" id="fnref131"><sup>131</sup></a></span> In addition, the parents’ phenotypes can be correlated through “<strong>assortative mating</strong>” (green line), which also has the consequence of creating correlations between the genotypes that affect those phenotypes.<span class="citation"><a href="#fn132" class="footnote-ref" id="fnref132"><sup>132</sup></a></span></p>
<p>(ref:pheno-causal-model) Causal model of offspring phenotypes. Ancestry causes differences in both genotypes and phenotypes of the parents (yellow lines). Dynastic effects are transmitted through parental parental phenotypes (red lines). Assortative mating creates associations between the phenotypes of the parents (green line), which will also lead to genotypic correlations. Arrows represent direction of effect. Figure adapted from <span class="citation">Morris et al.<a href="#fn133" class="footnote-ref" id="fnref133"><sup>133</sup></a></span></p>
<div class="figure">
<img src="figs/introduction/pheno_causal_model.png" alt="(ref:pheno-causal-model)" width="100%" />
<p class="caption">
(#fig:pheno-causal-model)(ref:pheno-causal-model)
</p>
</div>
<p>This model only includes environmental factors related to the parents’ phenotypes, yet the phenotype of an individual is continuously affected by all other environmental influences it is exposed to throughout its lifetime.<span class="citation"><a href="#fn134" class="footnote-ref" id="fnref134"><sup>134</sup></a></span> It is impossible to parse out the respective contributions of each of these factors at the individual level, but the problem becomes tractable by examining genotypic and phenotypic variation within groups of individuals.</p>
<div id="heritability" class="section level3" number="1.2.1">
<h3><span class="header-section-number">1.2.1</span> Heritability</h3>
<p>Fisher’s analysis of variance methodology allowed for the partitioning of variance into components that are attributable to different factors. If the total observed phenotypic variation within a population is represented by <span class="math inline">\(V_{P}\)</span>, then it can be decomposed as follows with respect to genetic and environmental factors:</p>
<p><span class="math display">\[
V_{P} = V_{G} + V_{E}
\]</span>
<span class="math inline">\(V_{G}\)</span> is the total proportion of phenotypic variance attributable to genetic factors, <span class="math inline">\(V_{E}\)</span> is that attributable to environmental factors (i.e. all non-genetic factors). <em>Broad-sense heritability</em> (<span class="math inline">\(H^2\)</span>) measures the proportion of phenotypic variance that is attributable to genetic factors:<span class="citation"><a href="#fn135" class="footnote-ref" id="fnref135"><sup>135</sup></a></span></p>
<p><span class="math display">\[
H^2 = \frac{V_G}{V_P}
\]</span></p>
<p><em>Narrow-sense heritability</em> (<span class="math inline">\(h^2\)</span>) measures the proportion of phenotypic variance attributable to <em>additive</em> genetic variance, excluding variance caused by Mendelian dominant/recessive phenomena, and epistasis (the interaction of alleles at different loci):<span class="citation"><a href="#fn136" class="footnote-ref" id="fnref136"><sup>136</sup></a></span></p>
<p><span class="math display">\[
h^2 = \frac{V_A}{V_P}
\]</span></p>
<p>As heritability is measured as a fraction of total phenotypic variance, it is liable to change depending on the level of environmental variation present in the study population.<span class="citation"><a href="#fn137" class="footnote-ref" id="fnref137"><sup>137</sup></a></span> However, for a given study population, and assuming they individuals are raised under the “normal” range of environmental circumstances, one can subtract the environmental variance (<span class="math inline">\(V_E\)</span>) from the total phenotypic variance (<span class="math inline">\(V_P\)</span>) to give an estimate of the trait’s heritability <span class="math inline">\(V_G\)</span>.<span class="citation"><a href="#fn138" class="footnote-ref" id="fnref138"><sup>138</sup></a></span></p>
<p>Environmental variance cannot be removed experimentally because it includes by definition all non-genetic variance, and much of this is beyond experimental control. However, elimination of genotypic variance can be achieved experimentally<span class="citation"><a href="#fn139" class="footnote-ref" id="fnref139"><sup>139</sup></a></span> by either using human or animal identical twins, or by creating highly inbred lines.</p>
<p>In humans, the twin study design has been used widely to disentangle the relative contributions of genes and environment for a variety of traits. The classical twin design is based on contrasting the trait resemblance of monozygotic (<strong>MZ</strong>) and dizygotic (<strong>DZ</strong>) twin pairs.<span class="citation"><a href="#fn140" class="footnote-ref" id="fnref140"><sup>140</sup></a></span> MZ twins share approximately 100% of genotypes, whereas DZ twins, like siblings, share on average 50% of their genotypes.<span class="citation"><a href="#fn141" class="footnote-ref" id="fnref141"><sup>141</sup></a></span> Assuming they share the same environment (an point I return to below), a rough estimate of heritability can be obtained by comparing the respective correlations of MZ twins and DZ twins with the phenotype of interest. If <span class="math inline">\(r_{MZ}\)</span> and <span class="math inline">\(r_{DZ}\)</span> are the correlation coefficients observed between pairs of MZ and DZ twins for a trait of interest, <span class="math inline">\(h^2\)</span> = additive genetic influences, and <span class="math inline">\(C\)</span> = common environmental influences, then Falconer’s formula tells us:</p>
<p><span class="math display">\[
r_{MZ} = h^2 + C \\
r_{DZ} = \tfrac{1}{2}h^2 + C \\
\tfrac{1}{2}h^2 = r_{MZ} - r_{DZ} \\
\mathrm{Therefore}: \\
h^2 = 2\times(r_{MZ} - r_{DZ})
\]</span>
Narrow-sense heritability <span class="math inline">\(h^2\)</span> can therefore be estimated by doubling the difference between the correlation coefficients of MZ versus DZ twins. A 2015 meta-analysis of virtually all human twin studies that had been performed to that date included 2,748 twin studies assessing ~18,000 traits and ~14.5M twin pairs. The results showed that across all traits, <span class="math inline">\(h^2\)</span> is 0.488 and <span class="math inline">\(C\)</span> is 0.174.<span class="citation"><a href="#fn142" class="footnote-ref" id="fnref142"><sup>142</sup></a></span> Moreover, in 69% of studies <span class="math inline">\(2r_{DZ} = r_{MZ}\)</span> (implying that the shared environment <span class="math inline">\(C\)</span> is equivalent across both MZ and DZ twin pairs). This suggests that for most traits, shared environmental or non-additive genetic variation do not make a substantial contribution to phenotypic variation.</p>
<p>However, that study found that several behavioural traits were inconsistent with this model. These traits included conduct disorder, higher-level cognitive functions, mental and behavioural disorders due to the use of alcohol or tobacco, anxiety disorder, and weight maintenance. For these traits, either or both non-additive genetic influences or shared environmental influences are therefore needed to explain the observed patterns of twin correlations.<span class="citation"><a href="#fn143" class="footnote-ref" id="fnref143"><sup>143</sup></a></span> Indeed, <span class="citation">Kong et al.<a href="#fn144" class="footnote-ref" id="fnref144"><sup>144</sup></a></span> showed that parents’ non-transmitted alleles can affect a child through their impacts on the parents and other relatives, a phenomenon the authors call “genetic nurture”, comprising 29.9% of the transmitted polygenic score.</p>
<p>Beyond twin-studies, the advent of next-generation sequencing technologies has enabled the dense genotyping of large samples of individuals, leading to the development of new methods for analysing the genetic influences on human traits. The most widespread of these methods is known as the Genome-Wide Association Analysis (<strong>GWAS</strong>).</p>
</div>
<div id="genome-wide-association-analysis-gwas" class="section level3" number="1.2.2">
<h3><span class="header-section-number">1.2.2</span> Genome-Wide Association Analysis (GWAS)</h3>
<p>GWAS aim to identify genetic variants associated with traits by comparing the allele frequencies of individuals who share similar ancestries, but differ in values for the trait in question.<span class="citation"><a href="#fn145" class="footnote-ref" id="fnref145"><sup>145</sup></a></span> As of 2021, over 5,700 GWAS have been performed for more than 3,330 human traits.<span class="citation"><a href="#fn146" class="footnote-ref" id="fnref146"><sup>146</sup></a></span></p>
<p>The standard GWAS process involves first genotyping individuals using SNP microarrays, which target common genetic variants, or by sequencing their full exomes or genomes, respectively referred to as Whole Exome Sequencing (<strong>WES</strong>) and Whole-Genome Sequencing (<strong>WGS</strong>). One then uses principal components analysis (<strong>PCA</strong>) to restrict the analysis individuals who share common ancestry, in order to avoid population stratification effects.<span class="citation"><a href="#fn147" class="footnote-ref" id="fnref147"><sup>147</sup></a></span> Missing genotypes can be imputed using a genetic reference panel such as the 1000 Genomes Project.<span class="citation"><a href="#fn148" class="footnote-ref" id="fnref148"><sup>148</sup></a></span> Finally, an association test can be run, which includes an individual-specific random effect term to account for genetic relatedness among individuals.</p>
<p>Since it became possible to sequence the genotypes of individuals at scale, it has been an ongoing point of debate as to how best to model the effects that genetic variants have on a trait of interest. The standard linear regression model now used for GWAS can be written as follows:</p>
<p><span class="math display">\[
\textbf{Y} \sim \textbf{W} \alpha ~+~{X_s}{\beta_s} + g + e \\
g \sim N(0,\sigma^2_A\psi) \\
e \sim N(0,\sigma^2_e \textbf{I})
\]</span>
where, for each individual, <span class="math inline">\(\textbf{Y}\)</span> is a vector of phenotype values, <span class="math inline">\(\textbf{W}\)</span> is a matrix of covariates including an intercept term, <span class="math inline">\(\alpha\)</span> is a corresponding vector of effect sizes, <span class="math inline">\(X_s\)</span> is a vector of genotype values for all individuals at SNP <span class="math inline">\(s\)</span>, <span class="math inline">\(\beta_s\)</span> is the corresponding fixed effect size of genetic variant <span class="math inline">\(s\)</span> (also known as the SNP effect size), <span class="math inline">\(g\)</span> is a random effect that captures the polygenic effect of other SNPs, <span class="math inline">\(e\)</span> is a random effect of residual errors, <span class="math inline">\(\sigma^2_{A}\)</span> measures additive genetic variation of the phenotype, <span class="math inline">\(\psi\)</span> is the standard genetic relationship matrix, <span class="math inline">\(\sigma^2_e\)</span> measures residual variance, and <span class="math inline">\(\textbf{I}\)</span> is an identity matrix.<span class="citation"><a href="#fn149" class="footnote-ref" id="fnref149"><sup>149</sup></a></span></p>
<p>Population structure and unequal relatedness among individuals in a given cohort can lead to false discoveries.<span class="citation"><a href="#fn150" class="footnote-ref" id="fnref150"><sup>150</sup></a></span> This is because individuals who share common ancestries will share both variants that do affect the trait of interest, and variants that do not, and these variants will be correlated with one other due to that shared ancestry. Therefore, if an association is found between the causal variants and a trait of interest, the non-causal variants that are correlated with the causal variants will also be found to be statistically associated with the trait.</p>
<p>GWAS output a <span class="math inline">\(p\)</span>-value and directional effect size (commonly denoted as <span class="math inline">\(\beta\)</span>) for each tested SNP. Significance of SNPs are determined by either Bonferroni correction (<span class="math inline">\(\alpha / M_{SNPs}\)</span>, commonly set as <span class="math inline">\(p &lt; 5 \times 10^{-8}\)</span> for human studies), or by running permutations of the phenotype (and covariates) and selecting the lowest <span class="math inline">\(p\)</span>-value from those permutations as the significance threshold.</p>
<p>The summed effect sizes of significant SNPs should theoretically reach the heritability estimates from twin studies, however until recently they fell far short, leading to a discussion on the sources of this “missing heritability.”<span class="citation"><a href="#fn151" class="footnote-ref" id="fnref151"><sup>151</sup></a></span> GWAS are limited by sample size, as extremely large samples are required to detect rare variants, and variants with small effect sizes. Short-read sequencing technologies are also unable to detect large “structural” variants, which may have large effects on the phenotype of interest. However, this issue has largely been resolved through improvements in genotyping,<span class="citation"><a href="#fn152" class="footnote-ref" id="fnref152"><sup>152</sup></a></span> and tools such as GCTA which estimates the variance explained by <em>all</em> SNPs in the genome, rather than testing the association of individual SNPs that may be in linkage disequilibrium with each other.<span class="citation"><a href="#fn153" class="footnote-ref" id="fnref153"><sup>153</sup></a></span></p>
</div>
<div id="animal-models" class="section level3" number="1.2.3">
<h3><span class="header-section-number">1.2.3</span> Animal models</h3>
<p>It is unfeasible to explore the relationship between genes and environment experimentally in humans due to the insufficient ability to manipulate either class of variables. Since the rediscovery of Mendel’s work, researchers have resorted to using model organisms, with which it is possible to control for both.</p>
<p>The genetics of model organisms may be controlled to a degree by establishing inbred strains. Mendel used self-fertilising plants, which without sexual reproduction are effectively inbred strains. At that time, inbred strains of mammals were unknown. But through the repeated mating of siblings over successive generations, as the individuals within each line inherit the same same haplotype from their related parents, they become almost genetically identical to one another, with the added benefit that their genotypes can be replicated across time in subsequent generations.</p>
<p>In the early 1900s, researcher began to develop inbred lines of guinea pigs and mice for the purpose of genetic research. In 1906, an inbreeding experiment involving guinea-pigs was started by George M. Rommel of the Animal Husbandry Division of the United States Department of Agriculture.<span class="citation"><a href="#fn154" class="footnote-ref" id="fnref154"><sup>154</sup></a></span> The experiment was taken over in 1915 by Sewall Wright, who used them to develop his mathematical theory of inbreeding, eventually introducing the inbreeding coefficient <span class="math inline">\(F\)</span> as the correlation between uniting gametes in 1922.<span class="citation"><a href="#fn155" class="footnote-ref" id="fnref155"><sup>155</sup></a></span></p>
<p>Contemporaneously, the Harvard undergraduate Clarence Cook Little and his supervisor William Ernest Little collaborated with Abbie Lathrop, a breeder of fancy mice and rats which she marketed to rodent hobbyists and keepers of exotic pets, and later began selling in large numbers to scientific researchers.<span class="citation"><a href="#fn156" class="footnote-ref" id="fnref156"><sup>156</sup></a></span> In 1902, Castle bought some of Lathrop’s mice for his laboratory.<span class="citation"><a href="#fn157" class="footnote-ref" id="fnref157"><sup>157</sup></a></span> The most frequently used laboratory mouse strain for the past 80 years, C57BL/6J (“Black J”), is derived from one of Lathrop’s animals – mouse number 57 – bred by Little.<span class="citation"><a href="#fn158" class="footnote-ref" id="fnref158"><sup>158</sup></a></span></p>
<p>The utility of inbred strains eventually led to the establishment of “panels” of inbred strains for several model organisms including the thale cress (<em>Arabidopsis thaliana</em>),<span class="citation"><a href="#fn159" class="footnote-ref" id="fnref159"><sup>159</sup></a></span> common bean (<em>Phaseolus vulgaris L</em>),<span class="citation"><a href="#fn160" class="footnote-ref" id="fnref160"><sup>160</sup></a></span> tomato (<em>Lycopersicon esculentum</em>),<span class="citation"><a href="#fn161" class="footnote-ref" id="fnref161"><sup>161</sup></a></span> maize (<em>Zea mays</em>),<span class="citation"><a href="#fn162" class="footnote-ref" id="fnref162"><sup>162</sup></a></span> nematode (<em>Caenorhabditis elegans</em>),<span class="citation"><a href="#fn163" class="footnote-ref" id="fnref163"><sup>163</sup></a></span> fruit fly (<em>Drosophila melanogaster</em>),<span class="citation"><a href="#fn164" class="footnote-ref" id="fnref164"><sup>164</sup></a></span> and mouse (<em>Mus musculus</em>).<span class="citation"><a href="#fn165" class="footnote-ref" id="fnref165"><sup>165</sup></a></span></p>
<p>Although the mouse is an appropriate model for humans due to their orthologous mammalian organ systems and cell types, inbred strains of this organism descend from individuals that had already been domesticated, and therefore do not represent the genetic variation present in wild populations. Furthermore, the large panels of inbred mice such as the Collaborative Cross (<strong>CC</strong>),<span class="citation"><a href="#fn166" class="footnote-ref" id="fnref166"><sup>166</sup></a></span> Diversity Outcross (<strong>DO</strong>)<span class="citation"><a href="#fn167" class="footnote-ref" id="fnref167"><sup>167</sup></a></span> and B6-by-D2 (<strong>BXD</strong>)<span class="citation"><a href="#fn168" class="footnote-ref" id="fnref168"><sup>168</sup></a></span> are derived from only a small number of individuals. As gene-environment studies seek to ultimately understand their effects on traits “in the wild” (such as with humans), there is accordingly a need for a panel of inbred vertebrates that represents the genetic variation present in natural populations.</p>
</div>
<div id="the-medaka-inbred-kiyosu-karlsruhe-mikk-panel" class="section level3" number="1.2.4">
<h3><span class="header-section-number">1.2.4</span> The Medaka Inbred Kiyosu-Karlsruhe (MIKK) panel</h3>
<p>The medaka fish (<em>Oryzias latipes</em>) has been studied as a model organism in Japan for over a century,<span class="citation"><a href="#fn169" class="footnote-ref" id="fnref169"><sup>169</sup></a></span> and is gaining recognition elsewhere as a powerful genetic model for vertebrates.<span class="citation"><a href="#fn170" class="footnote-ref" id="fnref170"><sup>170</sup></a></span> In addition to possessing a number of desirable traits that are characteristic of model organisms (including their small-size, short reproduction time, and high fertility), medaka are also – uniquely among vertebrates – resilient to inbreeding from the wild.</p>
<p>This resilience to inbreeding has allowed (predominantly Japanese) researchers to develop several inbred strains that have been maintained for over 100 generations. One of the most famous inbred medaka strains, <em>HdrR</em>, was derived from Tatsuo Aida’s collection of southern Japanese medaka fish.<span class="citation"><a href="#fn171" class="footnote-ref" id="fnref171"><sup>171</sup></a></span> Other prominent strains include <em>HO5</em> and <em>iCab</em> from southern Japan, and <em>Kaga</em> and <em>HNI</em> from northern Japan. The locations of the originating populations of these strains are set out in <strong>Figure @ref(fig:line-locations)</strong>.</p>
<p>(ref:line-locations) Image adapted from,<span class="citation"><a href="#fn172" class="footnote-ref" id="fnref172"><sup>172</sup></a></span> showing the locations of the originating populations of the 5 inbred medaka lines used in this study.</p>
<div class="figure">
<img src="figs/pilot/line_locations.png" alt="(ref:line-locations)" width="100%" />
<p class="caption">
(#fig:line-locations)(ref:line-locations)
</p>
</div>
<p>Since 2010, the Birney Group at EMBL-EBI, in collaboration with the Wittbrodt Group at COS, University of Heidelberg and the Loosli Group at the Karlsruhe Institute of Technology (KIT), have been working to establish the world’s first panel of vertebrate inbred strains – now known as the Medaka Inbred Kiyosu-Karlsruhe Panel (<strong>MIKK panel</strong>). The MIKK Panel was bred from a wild population caught near Kiyosu in Southern Japan, and now comprises 80 inbred, near-isogenic “lines.”<span class="citation"><a href="#fn173" class="footnote-ref" id="fnref173"><sup>173</sup></a></span></p>
<p>The MIKK Panel was created to map genetic variants associated with quantitative traits at a high resolution, and to explore the interactions between those variants and any environmental variables of interest. The purpose of the companion papers <span class="citation">Fitzgerald et al.<a href="#fn174" class="footnote-ref" id="fnref174"><sup>174</sup></a></span> and <span class="citation">Adrien Leger et al.<a href="#fn175" class="footnote-ref" id="fnref175"><sup>175</sup></a></span> was to introduce the MIKK panel to the scientific community, and describe the genetic characteristics of the MIKK panel that would make it a useful resource for other researchers who wish to explore the genetics of quantitative traits in vertebrates. My contributions to these papers involved visualising the inbreeding trajectory of the panel (Chapter @ref(inbreeding-sec)), exploring the evolutionary history of the MIKK panel’s founding population (Chapter @ref(introgression-sec)), measuring the levels of homozygosity across the panel (Chapter @ref(nuc-div)), assessing its allele-frequency distribution and rate of linkage disequilibrium (LD) decay (Chapter @ref(ld-decay-sec)), and characterising the structural variants present in a smaller sample of lines using Oxford Nanopore long-read sequencing data (Chapter @ref(mikksv-sec)).</p>
<p>The fundamental utility of the MIKK panel arises from:</p>
<ol style="list-style-type: lower-alpha">
<li><p>its representation of the genetic variation that exists in a wild population.</p></li>
<li><p>the ability to replicate individuals with near-identical genotypes, allowing one to modify environmental variables while keeping the genetics constant to determine the strength of those environmental effects.</p></li>
<li><p>the presence of numerous inbred lines that can by placed in the same environment, allowing one to infer the strength of the genetic effect</p></li>
<li><p>the enrichment of rare variants, which are likely to have a larger effect size,<span class="citation"><a href="#fn176" class="footnote-ref" id="fnref176"><sup>176</sup></a></span> and are therefore more more likely to be discovered.</p></li>
</ol>
</div>
<div id="the-f2-cross" class="section level3" number="1.2.5">
<h3><span class="header-section-number">1.2.5</span> The F2 cross</h3>
<p>The F2 cross is a traditional method for mapping genetic variants associated with traits of interest. A schema for the method is presented in <strong>Figure @ref(fig:F2-cross-schema)</strong>. In essence, it involves starting with two inbred strains that diverge for the trait of interest (the ‘parental strains’, or F0 generation F0). One then crosses the parental strains to create a generation of F1 hybrid individuals who each possess, for every pair of their chromosomes, one chromosome from each of their parents. The individuals in this F1 generation are genetically identical to their parents with respect to their germ line. Finally, one inter-crosses the F1 generation to create a set of F2 individuals that share unique combinations of the original F0 strains’ genotypes, and tend to display values for the trait of interest that span across the spectrum between the extreme values of their parents.</p>
<p>(ref:F2-cross-schema) Schema of the F2-cross experimental setup. The F0 generation comprises two medaka strains that have extreme, opposing values for a trait of interest, represented by the colours red and blue. Below them is an illustrative single pair of chromosomes. The chromosomes within each pair are depicted as the same colour, as the strains are homozygous through successive generations of inbreeding. Their F1 offspring is heterozygous for each pair of their 24 chromosomes, and all F1 individuals are therefore almost genetically identical to one another (with the exception of somatic mutations and the regions of the genome that were not homozygous in the parental generations). The F1 individuals are then inter-crossed with one another to produce the F2 generation, which, due to recombination events during gamete formation, have unique combinations of the parental strains’ genotypes, and tend to span the phenotypic spectrum between the extremes of their F0 parental strains, represented by their colours.</p>
<div class="figure">
<img src="figs/mikk_behaviour/F2-cross-schema.png" alt="(ref:F2-cross-schema)" width="100%" />
<p class="caption">
(#fig:F2-cross-schema)(ref:F2-cross-schema)
</p>
</div>
<p>By combining this traditional genetic mapping method with modern sequencing technologies, researchers are not only able to map genetic variants associated with a phenotype of interest at a higher resolution than in humans, but also functionally validate those variants by “knocking in” (or out) a discovered allele in an embryo to confirm that it has the effect predicted. Many studies have already made use of this combined method (CITE), known as a “Genome-Wide Linkage Analysis”, which applies modern genetic association statistical methods that have been refined over the last decade in humans, known as <em>Genome-Wide Association Analysis</em>, or <strong>GWAS</strong>.</p>
</div>
</div>
<div id="studied-phenotypes" class="section level2" number="1.3">
<h2><span class="header-section-number">1.3</span> Studied phenotypes</h2>
<div id="genetic-control-of-somite-period-development" class="section level3" number="1.3.1">
<h3><span class="header-section-number">1.3.1</span> Genetic control of somite period development</h3>
<p>During the development of an embryo, somites are the earliest primitive segmental structures that form from presomatic mesoderm cells (<strong>PSM</strong>).<span class="citation"><a href="#fn177" class="footnote-ref" id="fnref177"><sup>177</sup></a></span> They later differentiate into vertebrae, ribs, and skeletal muscles, thereby establishing the body’s anterior-posterior axis. <strong>Figure @ref(fig:mouse-embryo)</strong> depicts a number of formed somites in a 9.5-day-old mouse embryo.</p>
<p>(ref:mouse-embryo) Image of a mouse embryo at day 9.5 from <span class="citation">Thomas Gridley<a href="#fn178" class="footnote-ref" id="fnref178"><sup>178</sup></a></span>, showing somites in darker colours.</p>
<div class="figure">
<img src="figs/somites/mouse_embryo_gridley.png" alt="(ref:mouse-embryo)" width="100%" />
<p class="caption">
(#fig:mouse-embryo)(ref:mouse-embryo)
</p>
</div>
<p>Somite formation occurs rhythmically and sequentially, with the time between the formation of each pair of somites referred to as the “period”. The period of somite formation varies greatly between species: ~30 minutes for zebrafish, 90 minutes for chickens, 2-3 hours for mice, and 5-6 hours for humans.<span class="citation"><a href="#fn179" class="footnote-ref" id="fnref179"><sup>179</sup></a></span> <strong>Figure @ref(fig:somite-seg-ali)</strong> shows the a series of time-stamped images of somite segmentation in medaka fish, generated by Ali Seleit.</p>
<p>(ref:somite-seg-ali) Time-stamped images of somite segmentation in medaka, generated by Ali Seleit.</p>
<div class="figure">
<img src="figs/somites/ali_fish_seg_compiled.png" alt="(ref:somite-seg-ali)" width="100%" />
<p class="caption">
(#fig:somite-seg-ali)(ref:somite-seg-ali)
</p>
</div>
<p>The period of somite formation is controlled by a molecular oscillator, known as the ‘segmentation clock’, which drives waves of gene expression in the Notch, fibroblast growth factor (FGF), and Wnt pathways, forming a signalling gradient that regresses towards the tail in concert with axis elongation.<span class="citation"><a href="#fn180" class="footnote-ref" id="fnref180"><sup>180</sup></a></span> Over the course of elongation, the wave period increases (i.e. each somite takes longer to form), and the PSM progressively shrinks until it is exhausted, eventually terminating somite formation.<span class="citation"><a href="#fn181" class="footnote-ref" id="fnref181"><sup>181</sup></a></span></p>
<p>It is not fully understood how the phase waves of the segmentation clock are initially established.<span class="citation"><a href="#fn182" class="footnote-ref" id="fnref182"><sup>182</sup></a></span> <span class="citation">Matsuda et al.<a href="#fn183" class="footnote-ref" id="fnref183"><sup>183</sup></a></span> found that period differences between mouse and human occur at the single-cell level (i.e. not due to intercellular communication), and are driven by biochemical reaction speeds - specifically, mRNA and protein degradation rates, transcription and translation delays, and intron and splicing delays. To identify the genetic basis of these biochemical differences, our collaborators Ali Seleit and Alexander Aulehla at EMBL-Heidelberg used a CRISPR-Case9 knock-in approach<span class="citation"><a href="#fn184" class="footnote-ref" id="fnref184"><sup>184</sup></a></span> to establish a medaka <em>Cab</em> strain with an endogenous, fluorescing reporter gene (Her7-Venus) for the oscillation signalling pathway. This method allows them to image somite formation and extract quantitative measures for segmentation clock dynamics.</p>
<p>In medaka, it is known that the southern Japanese <em>Cab</em> strain and the northern Japanese <em>Kaga</em> strain have divergent somite periodicity, where <em>Kaga</em>’s tends to be faster, and <em>Cab</em>’s slower (<strong>Figure @ref(fig:F0-Cab-Kaga-HdrR)</strong>).</p>
</div>
<div id="behaviour" class="section level3" number="1.3.2">
<h3><span class="header-section-number">1.3.2</span> Behaviour</h3>
<p>Behaviour is a complex trait that is affected by both genes and environment, and for social animals such as humans and medaka, one’s social environment is considered likely to constitute a large component of the environmental effect.<span class="citation"><a href="#fn185" class="footnote-ref" id="fnref185"><sup>185</sup></a></span> Apart from social aspects, an organism must face many “hostile forces of nature” throughout its life,<span class="citation"><a href="#fn186" class="footnote-ref" id="fnref186"><sup>186</sup></a></span> such as food shortages, predation, harsh climate, and diseases. Adaptive behaviours allow individuals to navigate such dangers and maximise the likelihood of their survival at both the individual and population level.<span class="citation"><a href="#fn187" class="footnote-ref" id="fnref187"><sup>187</sup></a></span></p>
<p>Boldness-shyness is thought to be a fundamental axis of behavioural variation in many species, with an obvious causal relationship to an individual’s likelihood of survival, and consequently with natural selection at the population level.<span class="citation"><a href="#fn188" class="footnote-ref" id="fnref188"><sup>188</sup></a></span> It represents an evolutionary trade-off between acquiring benefits (in terms of food or mates) and avoiding harms (in terms of predators or conspecific competitors), with each situation accompanied by its own optimal degree of risk.<span class="citation"><a href="#fn189" class="footnote-ref" id="fnref189"><sup>189</sup></a></span> It is both heritable,<span class="citation"><a href="#fn190" class="footnote-ref" id="fnref190"><sup>190</sup></a></span> and subject to change following different life experiences or under different environmental conditions.<span class="citation"><a href="#fn191" class="footnote-ref" id="fnref191"><sup>191</sup></a></span></p>
<p>Boldness-shyness has been studied extensively with fish. Shy individuals tend to react to novelty by reducing their activity and becoming more vigilant, whereas bold individuals show higher levels of activity and exploratory behaviour.<span class="citation"><a href="#fn192" class="footnote-ref" id="fnref192"><sup>192</sup></a></span> One assay commonly used to measure this behavioural domain is referred to as the ‘open field’ assay, where fishes are observed while swimming freely in an experimental setting.<span class="citation"><a href="#fn193" class="footnote-ref" id="fnref193"><sup>193</sup></a></span></p>
<p>Another is the ‘novel object’ assay, where a novel object is introduced to the fishes’ environment to simulate a threat.<span class="citation"><a href="#fn194" class="footnote-ref" id="fnref194"><sup>194</sup></a></span> Where both assays were performed on the same fish, the behaviours exhibited were found to be correlated across assays, indicating that both were measuring the same boldness-shyness axis.<span class="citation"><a href="#fn195" class="footnote-ref" id="fnref195"><sup>195</sup></a></span> [<strong>there is also an aspect of habituation in both parts of the test that should be mentioned</strong>]</p>
<div id="social-genetic-effects" class="section level4" number="1.3.2.1">
<h4><span class="header-section-number">1.3.2.1</span> Social genetic effects</h4>
<p>The specific environmental factor is the so-called indirect or social genetic effect, which describes the indirect interactions between an individual’s genes and the genes of their companions.<span class="citation"><a href="#fn196" class="footnote-ref" id="fnref196"><sup>196</sup></a></span> Social genetic effects have been shown to exert influence on various traits in mice including anxiety, wound healing, immune function, and body weight,<span class="citation"><a href="#fn197" class="footnote-ref" id="fnref197"><sup>197</sup></a></span> and various traits related to development and survival in many species of livestock.<span class="citation"><a href="#fn198" class="footnote-ref" id="fnref198"><sup>198</sup></a></span> However, in those studies the social interactions were maintained throughout development, and it is unclear whether social genetic effects can still exert influence on adaptive behaviours during discrete, time-limited interactions.</p>
<ul>
<li><p><span class="citation"><a href="#fn199" class="footnote-ref" id="fnref199"><sup>199</sup></a></span> showed that parents’ non-transmitted alleles can affect a child through their impacts on the parents and other relatives, a phenomenon the authors call “genetic nurture”, comprising 29.9% of the transmitted polygenic score.</p></li>
<li><p><span class="citation">Liang-Dar Hwang et al.<a href="#fn200" class="footnote-ref" id="fnref200"><sup>200</sup></a></span> discovered that indirect parental genetic effects could influence offspring phenotypes, through the alleles that they <em>do not</em> pass on to their children, but which drive the parents’ own behaviour, which ultimately influences the child’s outcomes. Their strategy involved creating “virtual” mothers and fathers by estimating the genotypic dosages of parental genotypes using physically genotyped data from relative pairs. They applied the approach to 19,066 sibling pairs from the UK Biobank and showed a polygenic score consisting of imputed parental educational attainment SNP dosages is strongly related to offspring educational attainment even after correcting for offspring genotype at the same loci.</p></li>
</ul>
<!--chapter:end:01-Introduction.Rmd-->
</div>
</div>
</div>
</div>
<div id="MIKK-genomes-chap" class="section level1" number="2">
<h1><span class="header-section-number">2</span> Genomic variations in the MIKK panel</h1>
<p>This project was carried out in collaboration with Felix Loosli’s group at the Karlsruhe Institute of Technology (KIT), and Joachim Wittbrodt’s group in the Centre for Organismal Studies (COS) at the University of Heidelberg.</p>
<p>This chapter sets out my contributions to the the following pair of papers published in the journal <em>Genome Biology</em>, on both of which I am joint-first author:</p>
<ul>
<li><p><span class="citation">Fitzgerald et al.<a href="#fn201" class="footnote-ref" id="fnref201"><sup>201</sup></a></span></p></li>
<li><p><span class="citation">Leger et al.<a href="#fn202" class="footnote-ref" id="fnref202"><sup>202</sup></a></span></p></li>
</ul>
<div id="the-medaka-inbred-kiyosu-karlruhe-mikk-panel" class="section level2" number="2.1">
<h2><span class="header-section-number">2.1</span> The Medaka Inbred Kiyosu-Karlruhe (MIKK) panel</h2>
<p>Biological traits are the product of an interaction between an organism’s genes and its environment, often described as the relationship between “nature and nurture”.<span class="citation"><a href="#fn203" class="footnote-ref" id="fnref203"><sup>203</sup></a></span> This is especially true for complex traits such as behaviour, which I investigate in Chapters @ref(Pilot-chap) and @ref(MIKK-F2-chap).</p>
<p>It is unfeasible to explore the relationship between genes and environment experimentally in humans due to the insufficient ability to manipulate either set of variables. Researchers accordingly resort to using model organisms, with which it is possible to control for both. The genetics of model organisms may be controlled to a degree by establishing inbred strains through the repeated mating of siblings over successive generations. Eventually, as the individuals within each line inherit the same same haplotype from their related parents, they become almost genetically identical to one another, with the added benefit that their genotypes can be replicated across time in subsequent generations. This utility has led to the establishment of “panels” of inbred strains for several model organisms including the thale cress (<em>Arabidopsis thaliana</em>),<span class="citation"><a href="#fn204" class="footnote-ref" id="fnref204"><sup>204</sup></a></span> common bean (<em>Phaseolus vulgaris L</em>),<span class="citation"><a href="#fn205" class="footnote-ref" id="fnref205"><sup>205</sup></a></span> tomato (<em>Lycopersicon esculentum</em>),<span class="citation"><a href="#fn206" class="footnote-ref" id="fnref206"><sup>206</sup></a></span> maize (<em>Zea mays</em>),<span class="citation"><a href="#fn207" class="footnote-ref" id="fnref207"><sup>207</sup></a></span> nematode (<em>Caenorhabditis elegans</em>),<span class="citation"><a href="#fn208" class="footnote-ref" id="fnref208"><sup>208</sup></a></span> fruit fly (<em>Drosophila melanogaster</em>),<span class="citation"><a href="#fn209" class="footnote-ref" id="fnref209"><sup>209</sup></a></span> and mouse (<em>Mus musculus</em>).<span class="citation"><a href="#fn210" class="footnote-ref" id="fnref210"><sup>210</sup></a></span></p>
<p>Although the mouse is an appropriate model for humans due to their orthologous mammalian organ systems and cell types, inbred strains of this organism descend from individuals that had already been domesticated, and therefore do not represent the genetic variation present in wild populations. Furthermore, the large panels of inbred mice such as the Collaborative Cross (CC),<span class="citation"><a href="#fn211" class="footnote-ref" id="fnref211"><sup>211</sup></a></span> Diversity Outcross (DO)<span class="citation"><a href="#fn212" class="footnote-ref" id="fnref212"><sup>212</sup></a></span> and B6-by-D2 (BXD)<span class="citation"><a href="#fn213" class="footnote-ref" id="fnref213"><sup>213</sup></a></span> are derived from only a small number of individuals. As gene-environment studies seek to ultimately understand their effects on traits “in the wild” (such as with humans), there is accordingly a need for a panel of inbred vertebrates that represents the genetic variation present in natural populations.</p>
<p>The medaka fish (<em>Oryzias latipes</em>) has been studied as a model organism in Japan for over a century,<span class="citation"><a href="#fn214" class="footnote-ref" id="fnref214"><sup>214</sup></a></span> and is gaining recognition elsewhere as a powerful genetic model for vertebrates.<span class="citation"><a href="#fn215" class="footnote-ref" id="fnref215"><sup>215</sup></a></span> In addition to possessing a number of desirable traits that are characteristic of model organisms (including their small-size, short reproduction time, and high fertility), medaka are also – uniquely among vertebrates – resilient to inbreeding from the wild.</p>
<p>Since 2010, the Birney Group at EMBL-EBI, in collaboration with the Wittbrodt Group at COS, University of Heidelberg and the Loosli Group at the Karlsruhe Institute of Technology (KIT), have been working to establish the world’s first panel of vertebrate inbred strains – now known as the Medaka Inbred Kiyosu-Karlsruhe Panel (<strong>MIKK panel</strong>). The MIKK Panel was bred from a wild population caught near Kiyosu in Southern Japan, and now comprises 80 inbred, near-isogenic “lines”.<span class="citation"><a href="#fn216" class="footnote-ref" id="fnref216"><sup>216</sup></a></span></p>
<p>The MIKK Panel was created to map genetic variants associated with quantitative traits at a high resolution, and to explore the interactions between those variants and any environmental variables of interest. The purpose of the companion papers <span class="citation">Fitzgerald et al.<a href="#fn217" class="footnote-ref" id="fnref217"><sup>217</sup></a></span> and <span class="citation">Leger et al.<a href="#fn218" class="footnote-ref" id="fnref218"><sup>218</sup></a></span> was to introduce the MIKK panel to the scientific community, and describe the genetic characteristics of the MIKK panel that would make it a useful resource for other researchers who wish to explore the genetics of quantitative traits in vertebrates. My contributions to these papers involved visualising the inbreeding trajectory of the panel (Chapter @ref(inbreeding-sec)),
exploring the evolutionary history of the MIKK panel’s founding population (Chapter @ref(introgression-sec)), measuring the levels of homozygosity across the panel (Chapter @ref(nuc-div)), assessing its allele-frequency distribution and rate of linkage disequilibrium (LD) decay (Chapter @ref(ld-decay-sec)), and characterising the structural variants present in a smaller sample of lines using Oxford Nanopore long-read sequencing data (Chapter @ref(mikksv-sec)).</p>
</div>
<div id="genomic-characterisation-of-the-mikk-panel" class="section level2" number="2.2">
<h2><span class="header-section-number">2.2</span> Genomic characterisation of the MIKK panel</h2>
<div id="non-sib-calls" class="section level3" number="2.2.1">
<h3><span class="header-section-number">2.2.1</span> MIKK panel DNA sequence dataset</h3>
<p>For the preparation of <span class="citation">Fitzgerald et al.<a href="#fn219" class="footnote-ref" id="fnref219"><sup>219</sup></a></span>, 79 of the 80 extant MIKK panel lines – together with several wild Kiyosu samples and individuals from the established <em>iCab</em> medaka strain – had their DNA sequenced from brain samples using Illumina short-read sequencing technology. Tomas Fitzgerald from the Birney Group at EMBL-EBI then aligned these sequences to the <em>HdrR</em> medaka reference and called variants to produce the <strong>MIKK Illumina call set</strong> in the form of a .vcf file containing single nucleotide polymorphism (SNP) and small insertion-deletion (INDEL) calls for each line. To avoid allele frequency biases introduced by the 16 pairs/triplets of “sibling lines” (see @ref(inbreeding-sec)), I removed each pair’s arbitrarily-labelled second sibling line from the variant call set, leaving 63 MIKK panel lines (<strong>MIKK non-sibling call set</strong>), and used only those calls for the analyses in Chapters @ref(nuc-div) and @ref(ld-decay-sec).</p>
<p>For the preparation of <span class="citation">Leger et al.<a href="#fn220" class="footnote-ref" id="fnref220"><sup>220</sup></a></span>, 12 MIKK panel lines had their DNA sequenced from brain samples using Oxford Nanopore Technologies (ONT) long-read sequencing technology. Adrien Leger from the Birney Group at EMBL-EBI then aligned these sequences to the <em>HdrR</em> medaka reference, and called variants to produce the <strong>MIKK ONT call set</strong> in the form of a .vcf file containing structural variants calls for each line with tags for insertions (INS), deletions (DEL), duplications (SUP), inversions (INV) and translocations (TRA). The work described below used these variant call sets as the primary datasets.</p>
</div>
<div id="inbreeding-sec" class="section level3" number="2.2.2">
<h3><span class="header-section-number">2.2.2</span> Assessing the inbreeding trajectory of the MIKK panel</h3>
<p>The MIKK panel was bred from a wild population of medaka found in the Kiyosu area near Toyohashi, Aichi Prefecture, in southern Japan.<span class="citation"><a href="#fn221" class="footnote-ref" id="fnref221"><sup>221</sup></a></span> From this wild population, the Loosli Group at KIT set up random crosses of single mating pairs to create 115 ‘founder families’. For each founder family, they then set up between two and five single full-sibling-pair inbreeding crosses, which resulted in 253 F1 lines. Lines derived from the same founder family are referred to as ‘sibling lines’. Over the course of the next eight generations of inbreeding, they used only one mating pair per line. I generated <strong>Fig. @ref(fig:InbreedingFigure)A</strong> and <strong>B</strong> from the inbreeding data provided by the Loosli Group. <strong>Fig. @ref(fig:InbreedingFigure)A</strong> shows the number of lines that survived over the course of the first 14 generations of the inbreeding program, and the various causes for the termination of other lines. <strong>Fig. @ref(fig:InbreedingFigure)B</strong> shows the average fecundity levels of the surviving lines at generation F16. In addition, the Birney Group at EMBL-EBI generated morphometric data for the MIKK panel lines to demonstrate the distribution of physical phenotypes across the MIKK panel. I used this data on relative eye diameters to generate <strong>Fig. @ref(fig:InbreedingFigure)C</strong>.</p>
<p>(ref:InbreedingFigure) Inbreeding, fecundity and eye size in the MIKK panel lines. <strong>A</strong>: Status of all MIKK panel lines during the first 14 generations of inbreeding, showing cause of death for non-extant lines. <strong>B</strong>: Average fecundity of MIKK panel lines in generation F16, as measured during peak egg production in July 2020. <strong>C</strong>: Distribution of mean relative eye size for female and male medaka across all MIKK panel lines.</p>
<div class="figure">
<img src="figs/mikk_genome/20211213_final_figure.png" alt="(ref:InbreedingFigure)" width="100%" />
<p class="caption">
(#fig:InbreedingFigure)(ref:InbreedingFigure)
</p>
</div>
</div>
<div id="introgression-sec" class="section level3" number="2.2.3">
<h3><span class="header-section-number">2.2.3</span> Introgression with northern Japanese and Korean medaka populations</h3>
<p>To explore the evolutionary history of the MIKK panel’s founding population, we sought to determine whether there was evidence of introgression between that southern Japanese population, and northern Japanese and Korean medaka populations. To this end, I used the 50-fish multiple alignment from Ensembl release 102 to obtain the aligned genome sequences for the established medaka inbred lines <em>HdrR</em> (southern Japan), <em>HNI</em> (northern Japan), and <em>HSOK</em> (Korea), as well as the most recent common ancestor of all three strains.<span class="citation"><a href="#fn222" class="footnote-ref" id="fnref222"><sup>222</sup></a></span> Using the phylogenetic tree provided with the dataset, and the <em>ape</em> R package,<span class="citation"><a href="#fn223" class="footnote-ref" id="fnref223"><sup>223</sup></a></span> I identified the most recent common ancestor of those three strains. For each locus with a non-missing base for <em>HdrR</em>, I assigned the allele in that ancestral sequence as the ‘ancestral’ allele, and the alternative allele as the ‘derived’ allele, and then combined that dataset with the MIKK Illumina call set and variant calls for the southern Japanese <em>iCab</em> strain (see @ref(non-sib-calls)).</p>
<p>I then carried out an ABBA BABA analysis to calculate a modified ‘admixture proportion’ statistic <span class="math inline">\(\hat{f}_d\)</span><span class="citation"><a href="#fn224" class="footnote-ref" id="fnref224"><sup>224</sup></a></span> as a measure of the proportion of shared genome in 500-kb sliding windows between the MIKK panel and either <em>iCab</em>, <em>HNI</em>, or <em>HSOK</em> (<strong>Fig. @ref(fig:ABBABABA)</strong>), using the scripts provided by the first author of <span class="citation">Martin, Davey, and Jiggins<a href="#fn225" class="footnote-ref" id="fnref225"><sup>225</sup></a></span> on their GitHub page.<span class="citation"><a href="#fn226" class="footnote-ref" id="fnref226"><sup>226</sup></a></span></p>
<p>(ref:ABBABABA) <strong>Figure 2</strong>: ABBA-BABA analysis. <strong>A</strong>. Phylogenetic tree generated from the Ensembl release 102 50-fish multiple alignment, showing only the medaka lines used in the ABBA-BABA analysis. <strong>B</strong>. Schema of the comparisons carried out in the ABBA-BABA analysis. <strong>C</strong>. Circos plot comparing introgression (<span class="math inline">\(\hat{f}_d\)</span>) between the MIKK panel and either <em>iCab</em> (yellow), <em>HNI</em> (orange), or <em>HSOK</em> (purple), calculated within 500-kb sliding windows using a minimum of 250 SNPs per window.</p>
<div class="figure">
<img src="figs/mikk_genome/07_introgression.png" alt="(ref:ABBABABA)" width="100%" />
<p class="caption">
(#fig:ABBABABA)(ref:ABBABABA)
</p>
</div>
<p>Based on the genome-wide mean <span class="math inline">\(\hat{f}_d\)</span>, the MIKK panel shares approximately 25% of its genome with <em>iCab</em>, 9% with <em>HNI</em>, and 12% with <em>HSOK</em>. These results provide evidence that the MIKK panel’s originating population has more recently introgressed with medaka from Korea than with medaka from northern Japan. This supports the findings in <span class="citation">Spivakov et al.<a href="#fn227" class="footnote-ref" id="fnref227"><sup>227</sup></a></span>, where the authors found little evidence of significant interbreeding between southern and northern Japanese medaka since the populations diverged. Although the proportional difference between <em>HNI</em> and <em>HSOK</em> is small, this further supports the general finding that northern and southern Japanese medaka strains show low levels of interbreeding that may be a result of geographical isolation or genome divergence.<span class="citation"><a href="#fn228" class="footnote-ref" id="fnref228"><sup>228</sup></a></span></p>
</div>
<div id="nuc-div" class="section level3" number="2.2.4">
<h3><span class="header-section-number">2.2.4</span> Nucleotide diversity</h3>
<p>As a means of assessing genetic diversity in the MIKK panel, I calculated nucleotide diversity (<span class="math inline">\(\hat{\pi}\)</span>) within 500-kb non-overlapping windows across the genome of the 63 lines in the MIKK non-sibling call set (see @ref(non-sib-calls)), and compared this to the nucleotide diversity in 7 wild medaka from the same Kiyosu population from which the MIKK panel was derived. Mean and median nucleotide diversity in both the MIKK panel and wild Kiyosu medaka were close to 0, and slightly higher in the MIKK panel (mean: MIKK = 0.0038, wild = 0.0037; median: MIKK = 0.0033, wild = 0.0031). The patterns of varying nucleotide diversity across the genome are shared between the MIKK panel and wild Kiyosu medaka, where regions with high levels of repeat content tend to have higher nucleotide diversity (r = 0.386, p &lt; 0.001) (<strong>Fig. @ref(fig:NucleotideDiversity)</strong>). I also calculated <span class="math inline">\(\hat{\pi}\)</span> for each line individually, and as expected, levels of <span class="math inline">\(\hat{\pi}\)</span> around the (XX/XY) sex determination region of 1:~16-17 Mb are elevated in all lines relative to the consistently low levels found in most other chromosomes.</p>
<p>(ref:NucleotideDiversity) Circos plot with nucleotide diversity (<span class="math inline">\(\hat{\pi}\)</span>) calculated within 500-kb non-overlapping windows for 63 non-sibling lines from the MIKK panel (<em>green</em>) and 7 wild Kiyosu medaka samples from the same originating population (<em>purple</em>); proportion of sequence classified as repeats by RepeatMasker (<em>blue</em>); and mean mapping quality (<em>pink</em>).</p>
<div class="figure">
<img src="figs/mikk_genome/supp_01_pi_circos.png" alt="(ref:NucleotideDiversity)" width="100%" />
<p class="caption">
(#fig:NucleotideDiversity)(ref:NucleotideDiversity)
</p>
</div>
<p>The higher level of <span class="math inline">\(\hat{\pi}\)</span> observed within specific regions on several chromosomes – such as chromosomes 2, 11, and 18 – correspond closely to the regions we identified as containing large (&gt;250 kb) inversions that appear to be shared across at least some of the MIKK panel (<strong>Fig. @ref(fig:SVInvs)</strong>). These regions are also enriched for large deletions and duplications.<span class="citation"><a href="#fn229" class="footnote-ref" id="fnref229"><sup>229</sup></a></span> Inversions cause permanent heterozygosity,<span class="citation"><a href="#fn230" class="footnote-ref" id="fnref230"><sup>230</sup></a></span> and duplications and deletions may have increased the density of called SNPs in these regions,<span class="citation"><a href="#fn231" class="footnote-ref" id="fnref231"><sup>231</sup></a></span> so the observed depressions in homozygosity at these loci may be the result of such large structural variants that are present in the MIKK panel’s genomes.</p>
<p>Overall, this analysis confirms that the MIKK panel shows similar levels of homozygosity compared to classical laboratory inbred medaka strains, and possesses a strong increase in isogenic genotypes compared to wild medaka from the original wild population.</p>
<p>(ref:SVInvs) Inversions identified in 9 MIKK panel lines using a combination of Oxford Nanopore Technologies long-read and Illumina short-read sequences (see Chapter @ref(mikksv-sec) below).</p>
<div class="figure">
<img src="figs/mikk_genome/20210224_sv_invs_lines.png" alt="(ref:SVInvs)" width="100%" />
<p class="caption">
(#fig:SVInvs)(ref:SVInvs)
</p>
</div>
</div>
<div id="ld-decay-sec" class="section level3" number="2.2.5">
<h3><span class="header-section-number">2.2.5</span> LD decay</h3>
<p>I analysed the MIKK panel’s allele frequency distribution and linkage disequilibrium (LD) structure to assess their likely effects on genetic mapping. To remove allele-frequency biases introduced by the presence of sibling lines in the MIKK panel, I used only the MIKK non-sibling call set (see Chapter @ref(non-sib-calls)).</p>
<p>To assess how accurately one may be able to map genetic variants using the MIKK panel relative to a human dataset, I compared the MIKK panel’s minor allele frequency (MAF) distribution and LD structure against that of the 2,504 humans in the 1KG Phase 3 release.<span class="citation"><a href="#fn232" class="footnote-ref" id="fnref232"><sup>232</sup></a></span> To prepare the <strong>“1KG call set”</strong>, I first downloaded the .vcf files for each autosome from the project’s FTP site (<a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/release/20130502/" class="uri">ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/release/20130502/</a>), then merged them into a single VCF using GATK.<span class="citation"><a href="#fn233" class="footnote-ref" id="fnref233"><sup>233</sup></a></span> I then used PLINK<span class="citation"><a href="#fn234" class="footnote-ref" id="fnref234"><sup>234</sup></a></span> to calculate the minor allele frequencies for all non-missing, biallelic SNPs in both the MIKK non-sibling and IKG call sets (<em>N</em> SNPs = 16,395,558 and 81,042,381 respectively) (<strong>Fig. @ref(fig:LDdecay)A</strong>). As expected, the 1KG and MIKK panel calls are similarly enriched for low-frequency variants, albeit to a lesser extent in the MIKK panel, which is likely due to its smaller sample size.</p>
<p>To determine the rate of LD decay in the MIKK panel and compare it to that in the 1KG sample, for both the MIKK non-sibling and 1KG call sets, I used PLINK to compute <span class="math inline">\(r^2\)</span> on each autosome for all pairs of non-missing, biallelic SNPs with MAF <span class="math inline">\(&gt;\)</span> 0.10 within 10 kb of one another (for 1KG and the MIKK panel respectively ~ 5.5M and ~ 3M SNPs, with a total number of pairwise <span class="math inline">\(r^2\)</span> observations of 204,152,922 and 146,785,673). I then grouped the <span class="math inline">\(r^2\)</span> observations for each pair of SNPs based on their distance from one another into non-overlapping bins of 100 bp in length, and calculated the mean <span class="math inline">\(r^2\)</span> in each of those bins to generate <strong>Fig. @ref(fig:LDdecay)B</strong> using the mean <span class="math inline">\(r^2\)</span> and left boundary of each bin.</p>
<p>(ref:LDdecay) Minor allele frequency distributions and LD decay for biallelic, non-missing SNPs in the 1000 Genomes Phase 3 variant calls (N = 2,504) (1KG), and the MIKK panel Illumina-based calls excluding one of each pair of sibling lines (N = 63), across all autosomes (1KG: chrs 1-22; MIKK: chrs 1-24). <strong>A</strong>: Histogram of allele frequencies in the 1KG and MIKK panel calls. <strong>B</strong>: LD decay for each autosome, calculated by taking the mean <span class="math inline">\(r^2\)</span> of pairs of SNPs with MAF <span class="math inline">\(&gt;\)</span> 0.1 within non-overlapping 100 bp windows of distance from one another, up to a maximum of 10 kb. LD decays faster on chromosome 2 for the MIKK panel due to its higher recombination rate.</p>
<div class="figure">
<img src="figs/mikk_genome/08_ld_decay.png" alt="(ref:LDdecay)" width="100%" />
<p class="caption">
(#fig:LDdecay)(ref:LDdecay)
</p>
</div>
<p>Based on the 1KG calls under these parameters, LD decays in humans to a mean <span class="math inline">\(r^2\)</span> of around 0.2-0.35 at a distance of 10 kb, whereas the MIKK panel reaches this level within 1 kb, with a mean <span class="math inline">\(r^2\)</span> of 0.3-0.4 at a distance of ~100 bp. This implies that when a causal variant is present in at least two lines in the MIKK panel, one may be able to map causal variants at a higher resolution than in humans. We note that LD decays faster in chromosome 2 of the MIKK panel relative to the other chromosomes. This suggests that it has a much higher recombination rate, which is consistent with the linkage map described in <span class="citation">Kiyoshi Naruse et al.<a href="#fn235" class="footnote-ref" id="fnref235"><sup>235</sup></a></span>, showing a higher genetic distance per Mb for this chromosome. This higher recombination rate in chromosome 2 may in turn be caused by its relatively high proportion of repeat content (<strong>Fig. @ref(fig:repeats)</strong>).</p>
<p>(ref:repeats-fig) Repeat content in the <em>HdrR</em> genome based on RepeatMasker results obtained by Jack Monahan. <strong>A</strong>. Proportion of repeat content per-chromosome. <strong>B</strong>. log10 of repeat lengths and counts per repeat class. “Misc” includes all repeats assigned to their own specific class, for example “(GAG)n” or “(GATCCA)n”. <strong>C</strong>. Circos plot showing repeat length (radial axes) by locus (angular axis) and repeat class (track).</p>
<div class="figure">
<img src="figs/mikk_genome/20210401_hdrr_repeats_final.png" alt="(ref:repeats-fig)" width="100%" />
<p class="caption">
(#fig:repeats)(ref:repeats-fig)
</p>
</div>
</div>
</div>
<div id="mikksv-sec" class="section level2" number="2.3">
<h2><span class="header-section-number">2.3</span> Structural variation in the MIKK panel</h2>
<p>As an alternative to the variation pangenome approach described in <span class="citation">Leger et al.<a href="#fn236" class="footnote-ref" id="fnref236"><sup>236</sup></a></span>, I explored the structural variants (SVs) present in 9 of the MIKK panel lines in a reference-anchored manner, similar to many human studies. Differences in SVs between panel lines is another important class of genetic variation that could cause or contribute to significant phenotypic differences. Here we used ONT data obtained for 9 of the 12 selected lines allowing us to characterise larger SVs in the MIKK panel and to create a more extensive picture of genomic rearrangements compared to available medaka reference genomes. Adrien Leger from the Birney Group at EMBL-EBI first called structural variants using only the ONT long reads, producing a set of structural variants classified into five types: deletions (DEL), insertions (INS), translocations (TRA), duplications (DUP) and inversions (INV). I then “polished” the called DEL and INS variants with Illumina short reads to improve their accuracy. The polishing process filtered out 7.4% of DEL and 12.8% of INS variants, and adjusted the breakpoints (i.e. start and end positions) for 75-77% of DEL and INS variants in each sample by a mean of 23 bp for the start position, and 33 bp for the end position. This process produced a total of 143,326 filtered SVs.</p>
<p>The 9 “polished” samples contained a mean per-sample count of approximately 37K DEL variants (12% singletons), 29.5K INS variants (14%), 3.5K TRA variants (9%), 2.5K DUP (7%) and 600 INV (7%) (<strong>Fig. @ref(fig:SV-main)D</strong>). DEL variants were up to 494 kb in length, with 90% of unique DEL variants shorter than 3.8 kb. INS variants were only up to 13.8 kb in length, with 90% of unique INS variants shorter than 2 kb. DUP and INV variants tended to be longer, with a mean length of 19 and 70.5 kb respectively (<strong>Fig. @ref(fig:SV-main)A</strong>). <strong>Fig. @ref(fig:SV-main)E</strong> shows the per-sample distribution of DEL variants across the genome. Most large DEL variants over 250 kb in length were common among the MIKK panel lines. A number of large DEL variants appear to have accumulated within the 0-10 Mb region of chromosome 2, which is enriched for repeats in the HdrR reference genome (<strong>Fig. @ref(fig:repeats)</strong>).</p>
<p>SVs were generally enriched in regions covered by repeats. While only 16% of bases in the <em>HdrR</em> reference were classified as repeats (irrespective of strand), those bases overlapped with 72% of DEL, 63% of DUP, 81% of INV and 35% of TRA variant regions. However, repeat bases only overlapped with 21% of INS variants. We also assessed each SV’s probability of being loss-of-function (pLI)<span class="citation"><a href="#fn237" class="footnote-ref" id="fnref237"><sup>237</sup></a></span> by calculating the logarithm of odds (LOD) for the pLI scores of all genes overlapping the variant (<strong>Fig. @ref(fig:SV-main)B,C</strong>). 30,357 out of 134,088 DEL, INS, DUP and INV variants overlapped at least one gene, and 9% of those had a score greater than 10, indicating a high probability that the SV would cause a loss of function. Two INS variants on chr2 had an outlying LOD score of 57 as a result of overlapping medaka gene ENSORLG00000003411, which has a pLI score of 1 – the highest intolerance to variants causing a loss of function. This gene is homologous with human genes <em>SCN1A</em>, <em>SCN2A</em> and <em>SCN3A</em>, which encode sodium channels and have been associated with neuronal and sleep disorders. We did not find evidence that longer SVs tended to have a higher probability of causing a loss of function (<strong>Fig. @ref(fig:SV-main)B</strong>).</p>
<p>(ref:SV-cap) Polished SVs in 9 MIKK panel lines sequenced with ONT. DEL: deletion; INS: insertion; TRA: translocation; DUP: duplication; INV: inversion. <strong>A</strong>. Aggregate log10 counts and lengths of distinct SVs by type, excluding TRA. <strong>B</strong>. pLI LOD scores in distinct SVs by SV type. <strong>C</strong>. Histogram of LOD scores by SV type. <strong>D</strong>. Total and singleton counts of SV types per sample. <strong>E</strong>. Circos plot showing per-sample distribution and lengths of DEL variants across the genome.</p>
<div class="figure">
<img src="figs/mikk_genome/20210408_sv_main.png" alt="(ref:SV-cap)" width="100%" />
<p class="caption">
(#fig:SV-main)(ref:SV-cap)
</p>
</div>
<p>We compared these polished INS and DEL calls with the high-quality graph-based alternative paths and large-scale deletions, respectively (see section titled <em>Novel genetic sequences and large-scale insertions and deletions in the MIKK panel</em> in <span class="citation">Leger et al.<a href="#fn238" class="footnote-ref" id="fnref238"><sup>238</sup></a></span>). We found that 2 of the 19 regions covered by graph-based alternative paths, and 4 of the 16 regions covered by graph-based deletions, had no SVs that overlapped those regions at all, which suggests they would have been missed entirely when using a reference-anchored approach alone.</p>
<p>With the exception of one alternative path on chromosome 20, the alternative paths were not captured by INS variants, which only covered up to 63% of the bases in each region, and in many cases substantially less. On the other hand, for 8 of the 16 graph-based deletions, the DEL variants covered at least 85% of the bases in those regions. The other 8 graph-based deletions were either not at all covered by DEL variants, or only slightly. This indicates that the reference-based approach is better at detecting large-scale deletions than alternative paths (“insertions”), but still misses around half of such variants relative to the graph-based approach.</p>
</div>
<div id="conclusions" class="section level2" number="2.4">
<h2><span class="header-section-number">2.4</span> Conclusions</h2>
<p>Taken together, these analyses show that the MIKK panel is highly homozygous, with LD characteristics that will favour high-resolution genetic mapping relative to humans. In the future, the SV analysis performed on a subset of the MIKK panel will be expanded across the entire panel, which will permit the inclusion of both large- and small-scale variants in genetic linkage studies. I proceeded to use the MIKK panel to analyse bold/shy behaviours, as I describe in Chapter @ref(MIKK-F2-chap), with a view to carrying out an F2-cross linkage study to identify genetic variants associated with differences in the behaviours of an individual, and the extent to which they transmit those behaviours to their social companions. However, before carrying out this study, we first ran a “pilot” study on 5 previously-established inbred lines to validate our behavioural assay. This is the subject of the following chapter.</p>
<!--chapter:end:02-MIKK_genome.Rmd-->
</div>
</div>
<div id="Pilot-chap" class="section level1" number="3">
<h1><span class="header-section-number">3</span> Classification of bold/shy behaviours in 5 inbred medaka lines</h1>
<p>Taking advantage of the rich genetic resources offered by medaka we established an OFT to examine the performance of isogenic inbred strains for an inter-strain comparative analysis of the boldness-shyness behaviour of medaka. We used a combined open-field and novel-object assay involving assaying pairs for fish, to determine: a) whether there were consistent differences in bold-shy behaviours exhibited by 5 established inbred strains of medaka fish (<em>iCab</em>, <em>HdrR</em>, and <em>HO5</em> from southern Japan, and <em>Kaga</em> and <em>HNI</em> from northern Japan); and b) whether there were consistent differences in bold-shy behaviours exhibited by a given strain (<em>iCab</em>) dependent on the strain that it was partnered with. The former was intended to measure the effect of an individual’s own genes on its behaviour (a direct genetic effect), and the latter to measure the effect of the genes of the focal fish’s tank partner on the behaviour of the focal fish (social genetic effect). Our experimental design allowed us to assess both direct and indirect effects on behaviour simultaneously, and to infer the degree to which variation in bold-shy behaviours is attributable to the differences in an individual’s own genetics, the differences in the genetics of their social companions, and stochastic variation. A schema for the experimental plan is shown in <strong>Figure @ref(fig:behavioural-schema)</strong>.</p>
<p>(ref:behavioural-schema) Schema for experimental plan. <em>iCab</em>-<em>iCab</em> pairings are the control condition. To explore direct genetic effects on behaviour, we compare the behaviours of test fishes from different lines, and infer that the differences between lines are caused by the differences in their genetics (direct genetic effects). To explore social genetic effects, we use the same data but turn our focus to the reference fish, and infer that the differences we observe between their behaviours are caused by the differences in their social environments, which are in turn driven by the different genetics in the test fish lines (social genetic effects).</p>
<div class="figure">
<img src="figs/pilot/experimental_schema.png" alt="(ref:behavioural-schema)" width="100%" />
<p class="caption">
(#fig:behavioural-schema)(ref:behavioural-schema)
</p>
</div>
<p>[Wrap up the findings in one or two sentences:
We find inter-strain differences that are statistically significant
Social genetic effects that influence behaviour of a strain depending on the partner (or genetic configuration of the test pair, not sure whether this sounds ok)]</p>
<p>To detect</p>
<div id="results" class="section level2" number="3.1">
<h2><span class="header-section-number">3.1</span> Results</h2>
<div id="pilot-data-collection" class="section level3" number="3.1.1">
<h3><span class="header-section-number">3.1.1</span> Data collection</h3>
<p>Our behavioural assay is 20 minutes long, comprising two consecutively-run 10-minute components: a) an ‘open field’ component, where the fishes are introduced to the test tank and left to swim around freely; and b) a ‘novel object’ component, where a small black plastic cylinder is added to the tank at the beginning of the second 10-minute period, after which the fishes are again left to swim around freely. The assay is run on pairs of fish. Medaka is a seasonal breeder in which photoperiod has a strong effect on physiology and behaviour.<span class="citation"><a href="#fn239" class="footnote-ref" id="fnref239"><sup>239</sup></a></span> In this study we tested fish that were acclimated to summer conditions. To avoid confounding mating behaviours between males and females and associated aggressive interactions between males we used only female fish in all experiments. To increase the throughput of the assay, the test tank was divided into four quadrants with barriers, allowing us to run the assay on four pairs of fish simultaneously. Two test tanks situated side-by-side were used, allowing to run 8 concurrent assays. The experimental setup used is shown in Figure @ref(fig:assay-setup).</p>
<p>(ref:assay-setup) <strong>A</strong>: Experimental setup, with two test boxes side-by-side (denoted as “L” for left and “R” for right). Each test box contains one test tank, separated by removable barriers into quadrants, allowing for the simultaneous assay of four pairs of fish per test tank. The interior of the box is ambiently illuminated by LED lights, and a camera is suspended over the centre of each test tank to record the videos. <strong>B</strong>: Four pairs of fishes in a test tank with labelled quadrants (I, II, III, IV) and strains (<em>iCab</em> or <em>HdrR</em>). C: Paths of <em>iCab</em> reference fish and <em>HdrR</em> test fish from the video at panel (B) at 110 seconds.</p>
<div class="figure">
<img src="figs/pilot/setup_pic_with_frame_110.png" alt="(ref:assay-setup)" width="100%" />
<p class="caption">
(#fig:assay-setup)(ref:assay-setup)
</p>
</div>
<p>Between 11 and 16 June 2019, I assayed a total of 307 pairs of fish, comprising the following counts for each strain pairing: 68 <em>iCab/iCab</em>, 60 <em>iCab/HdrR</em>, 76 <em>iCab/HNI</em>, 47 <em>iCab/Kaga</em>, and 56 <em>iCab/HO5</em>. The locations of the lines’ originating populations are shown in <strong>Figure @ref(fig:line-locations)</strong>. The fish from the iCab strain was denoted as the “reference fish”, and was introduced to the test tank first. The “test fish” which was either another iCab fish (for the control condition), or a fish from one of the other four strains that were assayed in this experiment (<em>HdrR</em>, <em>HNI</em>, <em>Kaga</em>, <em>HO5</em>). The order in which the strains were assayed across the six days was randomly determined prior to the collection of the data. The test tanks were also rinsed between runs to remove any substances released by subjects during previous runs that could influence the behaviour of the subjects that followed.</p>
<p>(ref:line-locations) Image adapted from,<span class="citation"><a href="#fn240" class="footnote-ref" id="fnref240"><sup>240</sup></a></span> showing the locations of the originating populations of the 5 inbred medaka lines used in this study.</p>
<div class="figure">
<img src="figs/pilot/line_locations.png" alt="(ref:line-locations)" width="100%" />
<p class="caption">
(#fig:line-locations)(ref:line-locations)
</p>
</div>
</div>
<div id="tracking" class="section level3" number="3.1.2">
<h3><span class="header-section-number">3.1.2</span> Tracking</h3>
<p>We split the videos by quadrant and assay component, and tracked the movement of the fishes with the open-access software package <em>idtrackerai</em>.<span class="citation"><a href="#fn241" class="footnote-ref" id="fnref241"><sup>241</sup></a></span> Each individual fish was tracked across at least 85% of frames in each video, with 78% of videos tracked for over 99% of frames. The failure to track both fishes across all frames was usually caused by instances where a fish was situated behind the novel object or quadrant divider, rendering it invisible to the camera. A random selection of 20 videos was reviewed to search for instances of mislabeled fishes due to software errors. We found none, which is consistent with the reliability of the software as reported by its authors. We therefore concluded that such instances would be absent or very rare and thus negligible.</p>
<p>To identify which fish was the reference or test fish, we could generally distinguish between the strains based on their colour, as <em>iCab</em> pigmentation is light-brown, <em>HO5</em> are light-brown with an orange hue, <em>HdrR</em> females are white, and <em>HNI</em> and <em>Kaga</em> are grey. Where the reference fish was indistinguishable in colour from the test fish, we identified the fish that was introduced to the test tank first, and followed it by eye through to the first frame of the assay. In cases of <em>iCab</em>-<em>iCab</em> pairings, we randomly assigned the individuals to either the “reference” or “test” fish.</p>
<p>We then used the coordinates of the fishes in each frame to calculate the distance and angle travelled by each fish between a fixed time interval, for example every 0.05 seconds. Distance was calculated between points B → C, and angle was calculated between points A → B → C. We used these distance and angle variables to train a HMM with the <em>hmmlearn</em> Python package<span class="citation"><a href="#fn242" class="footnote-ref" id="fnref242"><sup>242</sup></a></span> in order to classify the movements as discrete behaviours or “modes of movement”.</p>
</div>
<div id="effect-of-covariates" class="section level3" number="3.1.3">
<h3><span class="header-section-number">3.1.3</span> Effect of covariates</h3>
<p>We examined the effects of several covariates, including date of assay, time of assay, tank quadrant, and tank side. To achieve this, we calculated the mean speed of individuals in <em>iCab</em>-<em>iCab</em> pairings (<span class="math inline">\(N\)</span> = 136) over the course of the entire 20-minute video (including both open field and novel object assays), and ran a multi-way ANOVA with all covariates (<strong>Figure @ref(fig:covariate-effects)</strong>). We found significant differences for date of assay and and tank quadrant (p = 0.0139 and 0.0108), but not for time of assay or tank side. This may have been driven by a difference in the way the assay was performed on the first day of the experiment (11 June 2019), where we used a thick fabric sheet to cover the front of the box rather than the wooden doors shown in <strong>Figure @ref(fig:assay-setup)</strong>, as they were only installed the following day. The greater level of external light and sound permeating through the fabric may have caused the fishes to exhibit slower movement on that first day, and when the data from that day is excluded, the p-values for date of assay and tank quadrant increase to 0.0477 and 0.0469 respectively.</p>
<p>(ref:covariate-effects) Effect of covariates on mean speed of iCab individuals when paired with another iCab over the course of the full video (including both open field and novel object assays). <em>P</em>-values were calculated from a multi-way ANOVA with all four covariates included as terms, and adjusted for false discovery rate.</p>
<div class="figure">
<img src="figs/pilot/covariate_effects_no-split-by-assay.png" alt="(ref:covariate-effects)" width="100%" />
<p class="caption">
(#fig:covariate-effects)(ref:covariate-effects)
</p>
</div>
</div>
<div id="pilot-hmm-params" class="section level3" number="3.1.4">
<h3><span class="header-section-number">3.1.4</span> Choice of time interval and number of HMM states</h3>
<p>HMMs are well-suited for classifying the hidden states that generate stochastic, sequential observations. To determine the optimal parameters for the HMM’s classification of behaviours, we sought to reduce overfitting (which tends to favour using a lower number of HMM states) while maximising the ability to distinguish between strains based on the relative time they spent in each HMM state (which tends to favour a higher number of HMM states). We additionally considered the time interval between which the distance and angle of travel was measured.</p>
<p><strong>Figure @ref(fig:compare-params)</strong> sets out the comparison of HMM parameters on two measures designed to quantify, respectively, the level of overfitting (‘mean concordance between cross-validated HMM states’), and the quantification of differences between strains (’summed Kruskal-Wallis statistic comparing frequency of time spent in each HMM state across medaka strains) (Methods).</p>
<p>(ref:compare-params) Comparison between HMM parameters. Horizontal axis: Mean concordance between states assigned by HMMs through a 2-fold cross-validation process. Vertical axis: Kruskal-Wallis statistic comparing strains based on the proportion of time spent in each HMM state, summed across all states. Size of points correspond to the interval, in seconds, between which the distance and angle of travel was calculated (Methods).</p>
<div class="figure">
<img src="figs/pilot/compare_params.png" alt="(ref:compare-params)" width="100%" />
<p class="caption">
(#fig:compare-params)(ref:compare-params)
</p>
</div>
<p>Based on these results and a visualisation of the polar plots for each combination of state number and time interval, we excluded combinations with 15 or more states due to an asymmetry across states that would create difficulties for interpreting their biological meaning (Supplementary material). For the downstream analysis we selected the combination of 14 states with a 0.08-second interval between time points, because out of the remaining combinations it appeared to optimally balance the level of overfitting and detection of differences between strains.</p>
<p>The distances and angles of travel for the 14 states are shown in Figure 4, generated from 10,000 randomly-selected data points (i.e. movements). Each point represents the radial distance in log10(pixels) that a fish travelled from its previous location (B → C), and the polar angle travelled from A → B → C, where A → B is always aligned vertically along the 0-180° axis. For example, a point at 45° far from the pole represents a fast, forward movement to the right, and a point at 260° close to the pole represents a slow, backward movement to the left.</p>
<p>(ref:pilot-polar) Classification of movements by the 14-state HMM, based on distance (in log<sub>10</sub>(pixels)) and angle of travel between a time interval of 0.08 seconds from points B → C for distance, and points A → B → C for angle. Each point represents the distance and angle at point C, and A → B is aligned vertically along the 0-180° radial axis. The figure shows an illustrative 10,000 data points (movements), randomly selected from the full dataset. States are sorted in ascending order by mean distance.</p>
<div class="figure">
<img src="figs/pilot/polar_all_dge.png" alt="(ref:pilot-polar)" width="100%" />
<p class="caption">
(#fig:pilot-polar)(ref:pilot-polar)
</p>
</div>
<p>We next sought to determine whether there was a difference in the proportion of time spent in each state between the 5 inbred strains (implying the presence of direct genetic effects), and between the <em>iCab</em> reference fishes depending on the inbred strains of their tank partner (implying the presence of social genetic effects).</p>
</div>
<div id="direct-and-social-genetic-effects-on-boldshy-behaviour" class="section level3" number="3.1.5">
<h3><span class="header-section-number">3.1.5</span> Direct and social genetic effects on bold/shy behaviour</h3>
<div id="direct-genetic-effects" class="section level4" number="3.1.5.1">
<h4><span class="header-section-number">3.1.5.1</span> Direct genetic effects</h4>
<p>To determine whether the test fish strains differed in the proportions of time they spent in each state, we ran separate multi-way ANOVAs for each combination of assay component (open field or novel object) and state (states 1 to 14):</p>
<p><span class="math display">\[
frequency_{assay component,state} = \beta_{1}(test fish strain) + \beta_{2}(date) + \beta_{3}(time) + \beta_{4}(quadrant) + \beta_{5}(tank side)
\]</span></p>
<p>The proportion of time each individual spent within a state was first inverse-normalised within each combination of assay and state, and the date of assay, time of assay, tank quadrant, and tank side were included as covariates. P-values were then adjusted for the False Discovery Rate (<strong>FDR</strong>).</p>
<p>The states that showed a significant difference are set out in <strong>Table @ref(tab:pilot-dge-tbl)</strong>. The test fish strains differed significantly in the proportion of time spent in a given state (p &lt; 0.05, FDR-adjusted) for 11 out of 14 states in the open field assay (4.45x10<sup>-24</sup> &lt; p &lt; 1.06x10-2), and 7 out of 14 states for the novel object assay (5.53x10<sup>-16</sup> &lt; p &lt; 4.2x10-3), with the strain of the test fish explaining up to ~28% of the variance in the proportion of time spent in a given state. For some states, there was also a significant difference between quadrants, date of assay, and tank side (open field: 1.03e-09 &lt; p &lt; 1.18x10-2; novel object: 1.75x10-8 &lt; p &lt; 4.9x10-2). Full tables for all states and variables are provided in the Supplementary Material.</p>
<table>
<caption>(#tab:pilot-dge-tbl)Significant differences in the proportion of time spent in each HMM state across test fish strains for the open field and novel object assay components.</caption>
<thead>
<tr class="header">
<th align="left">Assay</th>
<th align="left">State</th>
<th align="left">Variance explained (%)</th>
<th align="left">p-value (FDR-adjusted)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">open field</td>
<td align="left">1</td>
<td align="left">26.62</td>
<td align="left">3.60e-22</td>
</tr>
<tr class="even">
<td align="left">open field</td>
<td align="left">2</td>
<td align="left">21.94</td>
<td align="left">6.18e-18</td>
</tr>
<tr class="odd">
<td align="left">open field</td>
<td align="left">3</td>
<td align="left">28.21</td>
<td align="left">4.45e-24</td>
</tr>
<tr class="even">
<td align="left">open field</td>
<td align="left">4</td>
<td align="left">17.73</td>
<td align="left">2.12e-15</td>
</tr>
<tr class="odd">
<td align="left">open field</td>
<td align="left">5</td>
<td align="left">7.48</td>
<td align="left">2.78e-05</td>
</tr>
<tr class="even">
<td align="left">open field</td>
<td align="left">6</td>
<td align="left">5.60</td>
<td align="left">2.48e-04</td>
</tr>
<tr class="odd">
<td align="left">open field</td>
<td align="left">7</td>
<td align="left">5.46</td>
<td align="left">3.11e-04</td>
</tr>
<tr class="even">
<td align="left">open field</td>
<td align="left">8</td>
<td align="left">5.79</td>
<td align="left">1.08e-04</td>
</tr>
<tr class="odd">
<td align="left">open field</td>
<td align="left">9</td>
<td align="left">7.36</td>
<td align="left">8.96e-06</td>
</tr>
<tr class="even">
<td align="left">open field</td>
<td align="left">10</td>
<td align="left">3.78</td>
<td align="left">1.06e-02</td>
</tr>
<tr class="odd">
<td align="left">open field</td>
<td align="left">13</td>
<td align="left">4.66</td>
<td align="left">3.42e-03</td>
</tr>
<tr class="even">
<td align="left">novel object</td>
<td align="left">1</td>
<td align="left">17.19</td>
<td align="left">7.28e-14</td>
</tr>
<tr class="odd">
<td align="left">novel object</td>
<td align="left">2</td>
<td align="left">14.28</td>
<td align="left">2.92e-11</td>
</tr>
<tr class="even">
<td align="left">novel object</td>
<td align="left">3</td>
<td align="left">19.30</td>
<td align="left">5.53e-16</td>
</tr>
<tr class="odd">
<td align="left">novel object</td>
<td align="left">4</td>
<td align="left">13.12</td>
<td align="left">1.26e-10</td>
</tr>
<tr class="even">
<td align="left">novel object</td>
<td align="left">6</td>
<td align="left">4.12</td>
<td align="left">4.20e-03</td>
</tr>
<tr class="odd">
<td align="left">novel object</td>
<td align="left">7</td>
<td align="left">8.93</td>
<td align="left">1.84e-06</td>
</tr>
<tr class="even">
<td align="left">novel object</td>
<td align="left">9</td>
<td align="left">10.68</td>
<td align="left">9.87e-08</td>
</tr>
</tbody>
</table>
<p><strong>Figure @ref(fig:pilot-dge-time)</strong> depicts the time dependence of HMM states over the course of the video, and the regions of the test tank that were most frequently occupied by the different strains. Figures A and H show the HMM states with panels coloured red to indicate the states that were found to significantly differ between test fish strains. <strong>Figures @ref(fig:pilot-dge-time)B</strong> and <strong>E</strong> show how each individual test fish, grouped by inbred strain, moved through HMM states throughout the course of the video, with each tile coloured by the state most frequently-occupied by the individual within 2-second intervals. Figures <strong>@ref(fig:pilot-dge-time)C</strong> and <strong>F</strong> show the same data, recalculated as densities within each strain, with only the states that showed significant differences between strains in colour, and the remaining states consolidated and coloured grey. There is an especially clear difference between strains in the proportions of time spent in the slowest-moving states (i.e. states 1 to 3) at the beginning of each assay component, with an increase across all strains in the novel object assay, likely as a consequence of having less room to move, as well as the fear response that the novel object was designed to elicit. The differences are clearest when comparing the southern Japanese medaka strains (<em>iCab</em>, <em>HdrR</em>, and <em>HO5</em>) against the northern Japanese strains (<em>Kaga</em> and <em>HNI</em>). In addition, we note that <em>Kaga</em> tends to spend more time in the fast- and forward-moving state 13 at the beginning of the assay than at other times, which suggests that for that strain, such movements may be the manifestation of a stress response. <strong>Figures @ref(fig:pilot-dge-time)D</strong> and <strong>G</strong> show, as densities, the regions of the tank that were most frequently occupied by each strain. Although the northern Japanese strains <em>Kaga</em> and <em>HNI</em> were similarly fast-moving relative to the southern Japanese strains, in the open field assay component they appear to favour different regions of the tank – where <em>HNI</em> occupied the central regions of the tank with more frequency, <em>Kaga</em> tended to prefer swimming along the boundaries of the tank.</p>
<p>(ref:pilot-dge-time) Differences between test fish strains in the HMM states they occupied during the open field (top) and novel object (bottom) assay components. <strong>A and H</strong>: 14 HMM states with panels coloured red to indicate significant differences between test fish strains in the proportion of time spent in those states during the separate assay components. <strong>B and E</strong>: Transitions between HMM states across time for each individual test fish, grouped by strain. Tiles are coloured by the state most frequently occupied by each fish within 2-second intervals. <strong>C and F</strong>: Densities within each strain for the occupation of states that significantly differed between strains (colour), with other states consolidated (grey). <strong>D and G</strong>: Densities of the test tank locations occupied by each strain, calculated within 900 grid points (30x30).</p>
<div class="figure">
<img src="figs/pilot/0.08_14_dge.png" alt="(ref:pilot-dge-time)" width="100%" />
<p class="caption">
(#fig:pilot-dge-time)(ref:pilot-dge-time)
</p>
</div>
</div>
<div id="pilot-sge" class="section level4" number="3.1.5.2">
<h4><span class="header-section-number">3.1.5.2</span> Social genetic effects</h4>
<p>To determine whether the <em>iCab</em> reference fishes altered their behaviour depending on the inbred strain of their tank partner, we carried out the same analysis and model as above using only data from the <em>iCab</em> reference fishes. The states that showed a significant difference are set out in <strong>Table @ref(tab:pilot-sge-tbl)</strong>. The <em>iCab</em> reference fishes differed significantly in the proportion of time they spent in a given state depending on the strain of their tank partner (<span class="math inline">\(p\)</span> &lt; 0.05, FDR-adjusted) for 5 out of 14 states in the open field assay (2.04x10<sup>-6</sup> &lt; <span class="math inline">\(p\)</span> &lt; 2.11x10<sup>-2</sup>), and 7 out of 14 states for the novel object assay (4.76x10<sup>-7</sup> &lt; <span class="math inline">\(p\)</span> &lt; 4.06x10<sup>-2</sup>). The strain of the tank partner explained up to ~9% of the variance in the proportion of time the <em>iCab</em> reference spent in a given state. Full tables for all states and variables are provided in the Supplementary Material.</p>
<table>
<caption>(#tab:pilot-sge-tbl)Significant differences in the proportion of time spent in each HMM state by iCab reference fishes depending on the strain of their tank partner during the open field and novel object assay components.</caption>
<thead>
<tr class="header">
<th align="left">Assay</th>
<th align="left">State</th>
<th align="left">Variance explained (%)</th>
<th align="left">p-value (FDR-adjusted)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">open field</td>
<td align="left">1</td>
<td align="left">8.52</td>
<td align="left">5.88e-06</td>
</tr>
<tr class="even">
<td align="left">open field</td>
<td align="left">2</td>
<td align="left">9.09</td>
<td align="left">2.04e-06</td>
</tr>
<tr class="odd">
<td align="left">open field</td>
<td align="left">3</td>
<td align="left">7.97</td>
<td align="left">1.77e-05</td>
</tr>
<tr class="even">
<td align="left">open field</td>
<td align="left">5</td>
<td align="left">3.92</td>
<td align="left">4.29e-03</td>
</tr>
<tr class="odd">
<td align="left">open field</td>
<td align="left">12</td>
<td align="left">2.97</td>
<td align="left">2.11e-02</td>
</tr>
<tr class="even">
<td align="left">novel object</td>
<td align="left">1</td>
<td align="left">7.94</td>
<td align="left">8.73e-06</td>
</tr>
<tr class="odd">
<td align="left">novel object</td>
<td align="left">2</td>
<td align="left">9.38</td>
<td align="left">4.76e-07</td>
</tr>
<tr class="even">
<td align="left">novel object</td>
<td align="left">3</td>
<td align="left">8.73</td>
<td align="left">1.58e-06</td>
</tr>
<tr class="odd">
<td align="left">novel object</td>
<td align="left">6</td>
<td align="left">4.76</td>
<td align="left">1.23e-03</td>
</tr>
<tr class="even">
<td align="left">novel object</td>
<td align="left">8</td>
<td align="left">2.80</td>
<td align="left">4.06e-02</td>
</tr>
<tr class="odd">
<td align="left">novel object</td>
<td align="left">10</td>
<td align="left">3.81</td>
<td align="left">8.43e-03</td>
</tr>
</tbody>
</table>
<p>We observe that the proportions of time the <em>iCab</em> reference fishes spent in different states do differ based on the strains of their tank partner, and that their behavioural patterns tend to reflect those of the test fish strains they are paired with (<strong>Figure @ref(fig:pilot-sge-time)</strong>). Thus, the <em>iCab</em> reference fishes spend less time in the slower-moving states when in the presence of the faster-moving northern Japanese strains <em>Kaga</em> and <em>HNI</em>, which in turn shows that the test fishes are transmitting their behaviours to some degree to their <em>iCab</em> tank partners.The differences in the behaviours of <em>iCab</em> reference fishes based on the strain of their tank partners are not as large as those between the test fish strains. However, unlike what was observed with the test fishes, here there is a greater number of states that show significant differences during the novel object component of the assay compared to the open field component. This suggests that when movement is restricted, or when in the presence of a potential threat, the behaviour of the test fish tank partner has more of an influence on the <em>iCab</em> reference fish’s behaviour than otherwise.</p>
<p>(ref:pilot-sge-time) Differences between HMM states occupied by the reference fish when paired with different test fish strains during the open field (top) and novel object (bottom) assay components. <strong>A and H</strong>: 14 HMM states with panels coloured red to indicate significant differences between the reference fishes under different strain-pairings in the proportion of time spent in those states during the separate assay components. The HMM states are the same as those in <strong>Figure @ref(fig:pilot-dge-time)</strong>, but coloured with a different palette. <strong>B and E</strong>: Transitions between HMM states across time for each individual <em>iCab</em> reference fish, grouped by the strain of its tank partner. Tiles are coloured by the state most frequently occupied by each fish within 2-second intervals. <strong>C and F</strong>: Densities within each strain-pairing for the occupation of states that significantly differed between strain-pairings (colour), with other states consolidated (grey). <strong>D and G</strong>: Densities of the test tank locations occupied by the iCab reference fishes when paired with different strain, calculated within 900 grid points (30x30).</p>
<div class="figure">
<img src="figs/pilot/0.08_14_sge.png" alt="(ref:pilot-sge-time)" width="100%" />
<p class="caption">
(#fig:pilot-sge-time)(ref:pilot-sge-time)
</p>
</div>
<p>Finally, to develop a metric to quantify the degree to which the <em>iCab</em> reference fishes’ behaviour is influenced by the strain of its tank partner, we calculated the proportions of time that the pairs of fish spent simultaneously occupying the same HMM state. <strong>Figure @ref(fig:pilot-cooc-heat)</strong> shows that when paired with the slowest-moving <em>HdrR</em> strain, the <em>iCab</em> reference fishes spent more time co-occupying the slow- and forward-moving state 2, whereas they tended to co-occupy the fast- and forward-moving state 11 when paired with the faster-moving northern <em>Kaga</em> and <em>HNI</em> strains. When paired with another <em>iCab</em>, they preferred to co-occupy the slow-moving and pan-directional state 4. For each combination of assay component and state, we then ran a Kruskal-Wallis test to determine whether there were differences in the frequencies of state co-occupancy under different strain pairings (<span class="math inline">\(p\)</span> &lt; 0.05, FDR-adjusted), and found significant differences for states 1 to 6 for the open field component (1.1x10<sup>-8</sup> &lt; <span class="math inline">\(p\)</span> &lt; 4.1x10<sup>-2</sup>), and states 1 to 4 and 7 for the novel object component (1.6x10<sup>-6</sup> &lt; <span class="math inline">\(p\)</span> &lt; 2.5x10<sup>-2</sup>). The heatmaps show that <em>iCab</em> tends to co-occupy state 4 at the highest frequency when paired with another <em>iCab</em>, which suggests that this pan-directional, slow-moving state is a more comfortable or natural state for <em>iCab</em> to occupy, particularly when under stress.</p>
<p>(ref:pilot-cooc-heat) Frequency of HMM state co-occupancy between pairs of fish, calculated across all all videos per strain-pairing.</p>
<div class="figure">
<img src="figs/pilot/0.08_14_cooc_heatmap.png" alt="(ref:pilot-cooc-heat)" width="100%" />
<p class="caption">
(#fig:pilot-cooc-heat)(ref:pilot-cooc-heat)
</p>
</div>
</div>
</div>
</div>
<div id="discussion" class="section level2" number="3.2">
<h2><span class="header-section-number">3.2</span> Discussion</h2>
<p>In this chapter have described an assay for measuring bold-shy behaviours in medaka fish that can reliably detect differences: (a) between individuals from different inbred strains – allowing for the quantification of direct genetic effects on behaviour; and (b) between the behaviour of individuals when paired with tank partners from a different strain – allowing for the quantification of social genetic effects on behaviour.</p>
<p>I found that behaviours exhibited during the assay show minimal variance across the covariates of date of assay, time of assay, tank quadrant, and tank side. This creates confidence that the genetic effects that we observed – both direct and indirect – are not unduly affected by these variables.</p>
<p>I have shown that a Hidden Markov Model (HMM) allows an adequate classification of movements based on the direction and angle of travel within set time intervals. The HMM method has previously been used to classify maternal behaviour patterns in mice,<span class="citation"><a href="#fn243" class="footnote-ref" id="fnref243"><sup>243</sup></a></span> and I have extended the application here to bold-shy behaviours in medaka fish. In principle, this method can be expanded to include additional behavioural features such as proximity to the wall or other objects, inwards/outwards orientation, proximity to the tank partner, and other metrics related to leader-follower dynamics, a possibility that will be tested in future studies.</p>
<p>The HMM classification of behaviours also facilitated the ranking of behaviours by speed and angle of movement, which may be further interpreted in terms of biology and adaptation styles. However, even though the HMM classifies the slowest-moving states as separate modes of movement, one should be cautious when attributing biological meaning to their differences, because even when a fish is almost completely still, the fish object’s centroid (upon which the variables of distance and angle of travel are calculated) will tend to move by one or several pixels through minor changes in the segmentation of the object, and are thus subject to greater noise than the faster-moving states.</p>
<p>I have found that inbred strains of medaka fish can be distinguished by the proportion of time they spend in certain states. The slowest states, capturing no or minimal movement, were the states that most clearly separated the strains, and these differences appeared most clearly at the beginning of the open field and novel object assays. This suggests that for southern Japanese strains such as <em>iCab</em>, <em>HdrR</em>, and <em>HO5</em>, the “freeze” reflex is caused by anxiety, which eventually dissipates over time indicating habituation. Similar observations showing initial anxiety as shown by freezing behaviour and subsequent habituation have been made in other studies using open field assays with <em>iCab</em> fish.<span class="citation"><a href="#fn244" class="footnote-ref" id="fnref244"><sup>244</sup></a></span></p>
<p>On the other hand, the northern Japanese strains <em>Kaga</em> and <em>HNI</em> spent little time in the slow-moving states at the beginning of the video, which indicates either that their habituation sets in earlier, or that their stress and anxiety is expressed to a lesser degree as freezing behaviour. The former appears to be more likely for <em>Kaga</em>, as in the open field assay, it spends more time in the faster-moving states at the beginning of the video and then slows down thereafter, which suggests that its higher level of movement may be induced by stress. It is interesting then that once the novel object is introduced, <em>Kaga</em> tends to move slowly like the other strains. Obviously introduction to a novel environment and exposure to a potentially dangerous object elicits different behavioural patterns in this strain. Taken together with the overt thigmotaxis behaviour (i.e. moving along the sides of the tank during the open field assay component) relative to other strains, its behaviour may suggest an “escape-seeking” response. It will be interesting to carry out further experiments to determine under which fear-inducing conditions <em>Kaga</em> reacts with this more frantic style of movement rather than the freeze response that it displays in the novel object component in common with the other strains, and to explore its physiological basis.</p>
<p>With respect to social genetic effects, we have found that a fish’s behaviour is affected by the differential behaviour of its tank partner, although the effect is less powerful than the direct genetic effect on a fish’s own biology. These social genetic effects are detectable when observing the proportions of time the reference fishes spend in certain states over the course of the video, and can be quantified by measuring the frequency of state co-occupancy with their tank partners.</p>
<p>It is also interesting to find that <em>iCab</em> prefers to co-occupy the slow, pan-directional state 4 when paired with another individual from the same strain. As the strains are raised exclusively in the company of individuals from the same strain, that type of behaviour may represent a more comfortable – yet still cautious – mode of movement, or a type of mimicry that tends to occur when in the company of individuals that are more genetically or socially familiar.</p>
<p>In this study we only used one strain as the reference fish, but future experiments can expand on this analysis by using different strains as the reference, thereby exploring how a strain’s genetics influence the degree of their behavioural plasticity, and how the distinct behaviours of strains may interact in intriguing ways.</p>
<p>In summary we show that the OFT experimental setup in combination with the HMM analysis can reliably detect both direct and social genetic effects. This creates the opportunity to carry out a similar study on a larger panel of inbred strains, such as the MIKK panel,<span class="citation"><a href="#fn245" class="footnote-ref" id="fnref245"><sup>245</sup></a></span> with a view to identifying the genetic variants associated with not only these differences in behaviour, but also the differences in the degree to which an individual transmits their behaviour onto their social companions. This in turn will shed light on longstanding biological questions concerning the direct and indirect influences on behaviour, their physiological bases, and their adaptive purposes. This is the subject of the following chapter.</p>
<!--chapter:end:03-Pilot.Rmd-->
</div>
</div>
<div id="Somite-chap" class="section level1" number="4">
<h1><span class="header-section-number">4</span> Genetic loci associated with somite development periodicity</h1>
<div id="background" class="section level2" number="4.1">
<h2><span class="header-section-number">4.1</span> Background</h2>
<p>During the development of an embryo, somites are the earliest primitive segmental structures that form from presomatic mesoderm cells (<strong>PSM</strong>).<span class="citation"><a href="#fn246" class="footnote-ref" id="fnref246"><sup>246</sup></a></span> They later differentiate into vertebrae, ribs, and skeletal muscles, thereby establishing the body’s anterior-posterior axis. <strong>Figure @ref(fig:mouse-embryo)</strong> depicts a number of formed somites in a 9.5-day-old mouse embryo.</p>
<p>(ref:mouse-embryo) Image of a mouse embryo at day 9.5 from <span class="citation">Gridley<a href="#fn247" class="footnote-ref" id="fnref247"><sup>247</sup></a></span>, showing somites in darker colours.</p>
<div class="figure">
<img src="figs/somites/mouse_embryo_gridley.png" alt="(ref:mouse-embryo)" width="100%" />
<p class="caption">
(#fig:mouse-embryo)(ref:mouse-embryo)
</p>
</div>
<p>Somite formation occurs rhythmically and sequentially, with the time between the formation of each pair of somites referred to as the “period”. The period of somite formation varies greatly between species: ~30 minutes for zebrafish, 90 minutes for chickens, 2-3 hours for mice, and 5-6 hours for humans.<span class="citation"><a href="#fn248" class="footnote-ref" id="fnref248"><sup>248</sup></a></span> <strong>Figure @ref(fig:somite-seg-ali)</strong> shows the a series of time-stamped images of somite segmentation in medaka fish, generated by Ali Seleit.</p>
<p>(ref:somite-seg-ali) Time-stamped images of somite segmentation in medaka, generated by Ali Seleit.</p>
<div class="figure">
<img src="figs/somites/ali_fish_seg_compiled.png" alt="(ref:somite-seg-ali)" width="100%" />
<p class="caption">
(#fig:somite-seg-ali)(ref:somite-seg-ali)
</p>
</div>
<p>The period of somite formation is controlled by a molecular oscillator, known as the ‘segmentation clock’, which drives waves of gene expression in the Notch, fibroblast growth factor (FGF), and Wnt pathways, forming a signalling gradient that regresses towards the tail in concert with axis elongation.<span class="citation"><a href="#fn249" class="footnote-ref" id="fnref249"><sup>249</sup></a></span> Over the course of elongation, the wave period increases (i.e. each somite takes longer to form), and the PSM progressively shrinks until it is exhausted, eventually terminating somite formation.<span class="citation"><a href="#fn250" class="footnote-ref" id="fnref250"><sup>250</sup></a></span></p>
<p>It is not fully understood how the phase waves of the segmentation clock are intially established.<span class="citation"><a href="#fn251" class="footnote-ref" id="fnref251"><sup>251</sup></a></span> <span class="citation">Matsuda et al.<a href="#fn252" class="footnote-ref" id="fnref252"><sup>252</sup></a></span> found that period differences between mouse and human occur at the single-cell level (i.e. not due to intercellular communication), and are driven by biochemical reaction speeds - specifically, mRNA and protein degradation rates, transcription and translation delays, and intron and splicing delays. To identify the genetic basis of these biochemical differences, our collaborators Ali Seleit and Alexander Aulehla at EMBL-Heidelberg used a CRISPR-Case9 knock-in approach<span class="citation"><a href="#fn253" class="footnote-ref" id="fnref253"><sup>253</sup></a></span> to establish a medaka <em>Cab</em> strain with an endogenous, fluorescing reporter gene (Her7-Venus) for the oscillation signalling pathway. This method allows them to image somite formation and extract quantitative measures for segmentation clock dynamics.</p>
<p>In medaka, it is known that the southern Japanese <em>Cab</em> strain and the northern Japanese <em>Kaga</em> strain have divergent somite periodicity, where <em>Kaga</em>’s tends to be faster, and <em>Cab</em>’s slower (<strong>Figure @ref(fig:F0-Cab-Kaga-HdrR)</strong>).</p>
<p>(ref:F0-Cab-Kaga-HdrR) Comparison of period for three inbred medaka strains (<em>Cab</em>, <em>Kaga</em> and <em>HdrR</em>). Kaga’s period is lower, and therefore it takes less time to form each somite than <em>Cab</em>. Figure generated by Ali Seleit.</p>
<div class="figure">
<img src="figs/somites/ali_period_F0_Cab_Kaga.png" alt="(ref:F0-Cab-Kaga-HdrR)" width="100%" />
<p class="caption">
(#fig:F0-Cab-Kaga-HdrR)(ref:F0-Cab-Kaga-HdrR)
</p>
</div>
<p>Our collaborators accordingly set up a one-way F2 cross experiment as described in Chapter @ref(MIKK-F2-cross), using the reporter-carrying <em>Cab</em> strain and the <em>Kaga</em> strain as the parental F0 strains, in order to identify genetic loci associated with these differences in clock dynamics. They inter-crossed the hybrid F1 generation to create a sample of 622 F2 individuals, imaged the developing embryos of these F2 samples, and used pyBOAT<span class="citation"><a href="#fn254" class="footnote-ref" id="fnref254"><sup>254</sup></a></span> to extract the oscillation features during somite development. <strong>Figure @ref(fig:somite-period-ali)</strong> shows a series of raw images used by pyBOAT to track the elongation of a medaka tail during somitogenesis, with the identified posterior tip of the embryo labelled with a blue circle.</p>
<p>(ref:somite-period-ali) Screenshots of vertebral elongation in an F2 individual captured by Ali Seleit during imaging. The blue circle represents the point tracked by pyBOAT over time, generating the quantitative phenotype data on period development used in this study.</p>
<div class="figure">
<img src="figs/somites/ali_compiled_somite_elong.png" alt="(ref:somite-period-ali)" width="100%" />
<p class="caption">
(#fig:somite-period-ali)(ref:somite-period-ali)
</p>
</div>
</div>
<div id="somite-phenotype" class="section level2" number="4.2">
<h2><span class="header-section-number">4.2</span> Phenotypes of interest</h2>
<div id="somite-development-period" class="section level3" number="4.2.1">
<h3><span class="header-section-number">4.2.1</span> Somite development period</h3>
<p><strong>Figure @ref(fig:ali-somite-period-lines)</strong> shows the period data generated by pyBOAT for this study, for 100 illustrative F2 samples over 300 minutes. The same data can be represented by boxplots as shown in <strong>Figure @ref(fig:ali-somite-period-box)</strong>. I experimented with using the F2 individuals’ mean period and period intercept as the phenotype of interest. The two measures are highly correlated (<span class="math inline">\(Pearson&#39;s~r =\)</span> 0.84, <span class="math inline">\(p\)</span> &lt; 2.2x10<sup>-16</sup>), so after displaying the distributions for both measures in Figure @ref(fig:somite-phenos), I proceed to only discuss the analysis of period intercept, as it would appear to potentially be more robust to the changes in slope that can be observed in Figure @ref(fig:ali-somite-period-lines).</p>
<p>(ref:ali-somite-period-lines) PyBOAT results for 100 illustrative F2 samples, showing the period length in minutes over the course of 300 minutes. Period tends to increase over time, meaning that as the embryo develops, each successive somite takes longer to form. Figure generated by Ali Seleit.</p>
<div class="figure">
<img src="figs/somites/ali_period_lines_100fish300mins.png" alt="(ref:ali-somite-period-lines)" width="100%" />
<p class="caption">
(#fig:ali-somite-period-lines)(ref:ali-somite-period-lines)
</p>
</div>
<p>(ref:ali-somite-period-box) Period measurements for 70 F2 individuals displayed as boxplots with each individual’s median and interquartile range. Figure generated by Ali Seleit.</p>
<div class="figure">
<img src="figs/somites/ali_F2_mean_period.png" alt="(ref:ali-somite-period-box)" width="100%" />
<p class="caption">
(#fig:ali-somite-period-box)(ref:ali-somite-period-box)
</p>
</div>
</div>
<div id="psm-area" class="section level3" number="4.2.2">
<h3><span class="header-section-number">4.2.2</span> PSM area</h3>
<p>In the proceeding analyses, I also included a second phenotype of interest: the total size of the PSM prior to the formation of any somite segments. As the measure is simply based on the total number of pixels covered by the embryo object, I considered it to be potentially more robust than the period measurements, and therefore included it as a type of positive control for the genetic association analyses on the period phenotype. The measurements for PSM area comparing F0 <em>Cab</em> and <em>Kaga</em> strains are set out in <strong>Figure @ref(fig:ali-psm-F0)</strong>.</p>
<p>(ref:ali-psm-F0) Measurements of unsegmented PSM area in pixels for the F0 individuals from the <em>Cab</em> strain (<span class="math inline">\(N = 19\)</span>) and <em>Kaga</em> strain (<span class="math inline">\(N = 10\)</span>). <em>Kaga</em> tends to have a smaller PSM than <em>Cab</em>. Figure generated by Ali Seleit.</p>
<div class="figure">
<img src="figs/somites/ali_PSM_Cab_Kaga.png" alt="(ref:ali-psm-F0)" width="100%" />
<p class="caption">
(#fig:ali-psm-F0)(ref:ali-psm-F0)
</p>
</div>
</div>
<div id="comparisons-between-f0-f1-and-f2-generations" class="section level3" number="4.2.3">
<h3><span class="header-section-number">4.2.3</span> Comparisons between F0, F1 and F2 generations</h3>
<p>The distributions across the F0, F1 and F2 generations are curious (<strong>Figure @ref(fig:somite-phenos)</strong>). I expected to observe an F2 distribution with a similar median to the F1, and a variance that spanned across the extremes of the F0 strains. Instead, I observed that for the period phenotypes, the F2 generation had a mean that was slightly higher than the median of the higher-period F0 <em>Cab</em> strain, and many F2 samples exceed the period values in those F0 samples. Our collaborators assured me that these observations were unlikely to be caused by technical issues. The <em>Cab</em> and <em>Kaga</em> strains originate from different Japanese medaka populations (southern and northern respectively) that are understood to be at the point of speciation (see Chapter @ref(MIKK-genomes-chap)), so this slower period may be driven by a biological incompatibility between their genomes in cases where they do not have a complete chromosome from each parent (as the F1 generation does). I nevertheless proceeded with the genetic analysis with a view to potentially discovering the reason for this unusual distribution.</p>
<p>(ref:somite-phenos) Comparisons between the F0, F1 and F2 generations for the three phenotypes of interest. Here, the F0 only includes <em>Cab</em> individuals. <strong>A</strong>: period intercept. <strong>B</strong>: period mean. <strong>C</strong>: unsegmented PSM area. <span class="math inline">\(P\)</span>-values are derived from Kruskal-Wallis tests comparing the F2 individuals across microscopes.</p>
<div class="figure">
<img src="figs/somites/phenotypes.png" alt="(ref:somite-phenos)" width="100%" />
<p class="caption">
(#fig:somite-phenos)(ref:somite-phenos)
</p>
</div>
<p>Another important issue to note is that the F2 individuals were sequenced using different microscopes, denoted as ‘AU’ and ‘DB’. Our collaborators noticed that there was a difference between the microscopes in their temperatures of 0.7-0.8°C, translating to a 4-minute difference in the F2 means for the period intercept measure (Kruskal-Wallis = 177.97, <span class="math inline">\(p\)</span> = 1.34x10<sup>-40</sup>), and a 3.5-minute difference in the F2 means for the period mean measure (Kruskal-Wallis = 141.79, <span class="math inline">\(p\)</span> = 1.08x10<sup>-32</sup>). This difference would need to be accounted for in the downstream analysis through either adjusting the phenotype prior to running the genetic association model, or by including microscope as a covariate in the model. No significant was found for the PSM area. To resolve this difference between microscopes for the period intercept data, I elected to transform it for the F2 generation by inverse-normalising the period intercept within each microscope (<strong>Figure @ref(fig:invnorm-intercept)</strong>). I then used this transformed data in the downstream analysis.</p>
<p>(ref:invnorm-intercept) Comparison of the period intercept phenotype data for the F2 generation before (<strong>A</strong>) and after (<strong>B</strong>) inverse-normalisation.</p>
<div class="figure">
<img src="figs/somites/invnorm_intercept.png" alt="(ref:invnorm-intercept)" width="100%" />
<p class="caption">
(#fig:invnorm-intercept)(ref:invnorm-intercept)
</p>
</div>
</div>
</div>
<div id="genetic-sequencing-data" class="section level2" number="4.3">
<h2><span class="header-section-number">4.3</span> Genetic sequencing data</h2>
<p>Our collaborators extracted DNA from the F0, F1, and F2, and sequenced the F0 and F1 samples with the Illumina platform at high coverage (~26x for <em>Cab</em> and ~29x for <em>Kaga</em>), as measured by samtools.<span class="citation"><a href="#fn255" class="footnote-ref" id="fnref255"><sup>255</sup></a></span> <strong>Figure @ref(fig:F0-coverage)</strong> sets out the mean sequencing depth within each chromosome and across the whole genome for the <em>Cab</em> and <em>Kaga</em> F0 samples. Our collaborators then sequenced the F2 samples at low coverage (~1x), which would be sufficient to map their genotypes back to the genotypes of their parental strains (see Chapter @ref(somite-f2-sequencing) for further details).</p>
<div class="figure">
<img src="figs/somites/F0_coverage.png" alt="(ref:F0-coverage)" width="100%" />
<p class="caption">
(#fig:F0-coverage)(ref:F0-coverage)
</p>
</div>
</div>
<div id="f0-homozygosity-and-f1-heterozygosity" class="section level2" number="4.4">
<h2><span class="header-section-number">4.4</span> F0 homozygosity and F1 heterozygosity</h2>
<p>Before proceeding to map the F2 sequences to the genotypes of the F0 generation, I first investigated the levels of homozygosity in the F0 <em>Cab</em> and <em>Kaga</em> strains, as this would have an effect on the resolution of our genetic mapping. That is to say, for regions where either parent is consistently heterozygous, it would be difficult to determine the parent from which a particular F2 individual derived its chromosomes at that locus. I therefore aligned the high-coverage sequencing data for the F0 <em>Cab</em> and <em>Kaga</em> strains to the medaka <em>HdrR</em> reference (Ensembl release 104, build ASM223467v1) using BWA-MEM2, sorted the aligned .sam files, marked duplicate reads, and merged the paired reads with picard,<span class="citation"><a href="#fn256" class="footnote-ref" id="fnref256"><sup>256</sup></a></span> and indexed the .bam files with Samtools.<span class="citation"><a href="#fn257" class="footnote-ref" id="fnref257"><sup>257</sup></a></span></p>
<p>To call variants, I followed the GATK best practices (to the extent they were applicable)<span class="citation"><a href="#fn258" class="footnote-ref" id="fnref258"><sup>258</sup></a></span> with GATK’s HaplotypeCaller and GenotypeGVCFs tools,<span class="citation"><a href="#fn259" class="footnote-ref" id="fnref259"><sup>259</sup></a></span> then merged all calls into a single .vcf file with picard.<span class="citation"><a href="#fn260" class="footnote-ref" id="fnref260"><sup>260</sup></a></span> Finally, I extracted the biallelic calls for <em>Cab</em> and <em>Kaga</em> with bcftools,<span class="citation"><a href="#fn261" class="footnote-ref" id="fnref261"><sup>261</sup></a></span> counted the number of SNPs within non-overlapping, 5-kb bins, and calculated the proportion of SNPs within each bin that were homozgyous.</p>
<p><strong>Figure @ref(fig:somite-f0-cab)</strong> is a circos plot generated with circlize<span class="citation"><a href="#fn262" class="footnote-ref" id="fnref262"><sup>262</sup></a></span> for the <em>Cab</em> F0 strain used in this experiment, featuring the proportion of homozygous SNPs per 5-kb bin (green), and the total number of SNPs in each bin (yellow). As expected for a strain that has been inbred for over 10 generations, the mean homozygosity for this strain is high, with a mean proportion of homozygosity across all bins of 83%.</p>
<p>(ref:somite-f0-cab) Proportion of homozygous SNPs within 5 kb bins in the <em>Cab</em> F0 generation genome (green), and number of SNPs in each bin (yellow).</p>
<div class="figure">
<img src="figs/somites/Cab.png" alt="(ref:somite-f0-cab)" width="100%" />
<p class="caption">
(#fig:somite-f0-cab)(ref:somite-f0-cab)
</p>
</div>
<p>However, the levels of homozygosity in the <em>Kaga</em> strain used in this experiment was far lower, with a mean homozygosity across all bins of only 31%. This was a surprise, as it is an established strain of X generations, and we therefore expected the level of homozygosity to be commensurate with that observed in the <em>Cab</em> strain.</p>
<p>(ref:somite-f0-kaga) Proportion of homozygous SNPs within 5 kb bins in the <em>Kaga</em> F0 generation genome (red), and number of SNPs in each bin (yellow).</p>
<div class="figure">
<img src="figs/somites/Kaga.png" alt="(ref:somite-f0-kaga)" width="100%" />
<p class="caption">
(#fig:somite-f0-kaga)(ref:somite-f0-kaga)
</p>
</div>
<p>To determine whether the low levels of observed homozygosity in Kaga was affected by its alignments to the southern Japanese <em>HdrR</em> reference, we also aligned the F0 samples to the northern Japanese <em>HNI</em> reference (<strong>Figure @ref(fig:somite-f0-kaga-hni)</strong>. This did not make differences to the levels of observed homozygosity in either sample, which gave us confidence that the low homozygosity observed in <em>Kaga</em> was not driven by reference bias.</p>
<p>(ref:somite-f0-kaga-hni) Proportion of homozygous SNPs within 5 kb bins in the <em>Kaga</em> F0 generation genome when aligned to the <em>HNI</em> reference (red), and number of SNPs in each bin (yellow).</p>
<div class="figure">
<img src="figs/somites/Kaga_HNI.png" alt="(ref:somite-f0-kaga-hni)" width="100%" />
<p class="caption">
(#fig:somite-f0-kaga-hni)(ref:somite-f0-kaga-hni)
</p>
</div>
</div>
<div id="f1-homozygosity" class="section level2" number="4.5">
<h2><span class="header-section-number">4.5</span> F1 homozygosity</h2>
<p>I next examined the level of heterozygosity in the F1 generation from the <em>Cab</em>-<em>Kaga</em> cross. <strong>Figure @ref(fig:somite-f1)</strong> shows the level of heterozygosity across the genome of the F1 hybrid in brown measured by the proportion of heterozygous SNPs within 5-kb bins (brown), and the number of SNPs in each bin (yellow). Approximately half the chromosomes show inconsistent heterozygosity, with a mean heterozygosity across all bins of 67%. This lower level of heterozygosity than expected was likely caused by the low levels of heterozygosity in the <em>Kaga</em> F0 parent.</p>
<p>(ref:somite-f1) Proportion of heterozygous SNPs within 5 kb bins in the <em>Cab</em>-<em>Kaga</em> F1 cross (brown), and number of SNPs in each bin (yellow).</p>
<div class="figure">
<img src="figs/somites/F1.png" alt="(ref:somite-f1)" width="100%" />
<p class="caption">
(#fig:somite-f1)(ref:somite-f1)
</p>
</div>
<p>For the purpose of mapping the F2 sample sequences to the genomes of their parental strains, I selected only biallelic SNPs that were homozygous-divergent in the F0 generation (i.e. homozygous reference allele in <em>Cab</em> and homozygous alternative allele in <em>Kaga</em> or vice versa) <em>and</em> heterozygous in the F1 generation. The number of SNPs that met these criteria per chromosome are set out in <strong>Figure @ref(fig:snp-counts-per-chrom)</strong>.</p>
<p>(ref:snp-counts-per-chrom) Number of SNPs per chromosome that were homozygous-divergent in the F0 <em>Cab</em> and <em>Kaga</em> generations, and heterozygous in the F1 generation.</p>
<div class="figure">
<img src="figs/somites/snp_counts_per_chrom.png" alt="(ref:snp-counts-per-chrom)" width="100%" />
<p class="caption">
(#fig:snp-counts-per-chrom)(ref:snp-counts-per-chrom)
</p>
</div>
</div>
<div id="somite-f2-sequencing" class="section level2" number="4.6">
<h2><span class="header-section-number">4.6</span> F2 genotyping</h2>
<p>To maximise the efficiency of our sequencing runs, our collaborators “shallow-sequenced” the F2 generation with the short-read Illumina platform at a depth of ~1x. We then aligned these sequences to the <em>HdrR</em> reference with BWA-MEM2,<span class="citation"><a href="#fn263" class="footnote-ref" id="fnref263"><sup>263</sup></a></span> sorted the reads and marked duplicates with Picard,<span class="citation"><a href="#fn264" class="footnote-ref" id="fnref264"><sup>264</sup></a></span> then indexed the resulting BAM files with samtools.<span class="citation"><a href="#fn265" class="footnote-ref" id="fnref265"><sup>265</sup></a></span> Genotyping these shallow sequences with the same method as used for the high-coverage sequences for the F0 and F1 generation would be inappropriate. We therefore used a different method whereby we used <em>bam-readcount</em><span class="citation"><a href="#fn266" class="footnote-ref" id="fnref266"><sup>266</sup></a></span> to count the reads that supported either the <em>Cab</em> or the <em>Kaga</em> allele for all SNPs that met the criteria described above in section @ref(f1-homozygosity), summed the read counts within 5 kb blocks, and calculated the frequency of reads within each bin that supported the <em>Kaga</em> allele. This generated a value for each bin between 0 and 1, where 0 signified that all reads within that bin supported the <em>Cab</em> allele, and 1 signified that all reads within that bin supported the <em>Kaga</em> allele. Bins containing no reads were imputed with a value of 0.5.</p>
<p>I then used these values for all F2 individuals as the input to a Hidden Markov Model (HMM) with the software package <em>hmmlearn</em>,<span class="citation"><a href="#fn267" class="footnote-ref" id="fnref267"><sup>267</sup></a></span> which I applied to classify each bin as one of three states, with state 0 corresponding to homozygous-<em>Cab</em>, 1 corresponding to heterozygous, and 2 corresponding to homozygous-<em>Kaga</em>.
Across each chromosome of every sample, the output of the HMM was expected to produce a sequence of states. Based on previous biological knowledge that crossover events occur on average less than once per chromosome<span class="citation"><a href="#fn268" class="footnote-ref" id="fnref268"><sup>268</sup></a></span> (see <strong>Figure @ref(fig:zebrafish-co-per-chrom)</strong> for the average crossover rates per chromosome in zebrafish), I expected to observe the same state persisting for long stretches of the chromosome, only changing to another state between 0 and 3 times, and rarely more.</p>
<p>(ref:zebrafish-co-per-chrom) Crossovers per chromosome based on data provided in <span class="citation">Haenel et al.<a href="#fn269" class="footnote-ref" id="fnref269"><sup>269</sup></a></span>, where “crossovers per chromosome” for each chromosome <span class="math inline">\(c\)</span> was calculated by <span class="math inline">\(\frac{crossover~rate_{c}(cM / Mb) \times length_{c}(Mb)} {100}\)</span>. The medaka genome is shorter in length than zebrafish genome (~800 Mb compared to ~1,300 Mb), which according to the authors would suggest that medaka likely has a higher average crossover rate than what is presented in this figure.</p>
<div class="figure">
<img src="04-somites_files/figure-html/zebrafish-co-per-chrom-1.png" alt="(ref:zebrafish-co-per-chrom)" width="100%" />
<p class="caption">
(#fig:zebrafish-co-per-chrom)(ref:zebrafish-co-per-chrom)
</p>
</div>
<p><strong>Figure @(fig:hmm-standard)</strong> shows how adjusting the HMM parameters changed the called genotypes for 10 F2 samples on chromosome 18. Allowing the HMM to train itself for the transition probabilities and emission variances, the HMM produced an apparently noisy output (<strong>Figure @(fig:hmm-standard)A</strong>). Fixing the transition probabilities to make it very likely for a state to transition back to itself rather than to another state.</p>
<p>(ref:hmm-standard) HMM states called for each bin across chr18 for 10 F2 samples. States 0, 1, and 2 correspond to homozygous <em>Cab</em>, heterozygous, and homozygous <em>Kaga</em>. Each point represents a 5-kb bin. Y-axis is the proportion of reads within each bin that align to the <em>Kaga</em> allele. X-axis is the bp location of the start of each bin. <strong>A</strong>: Standard HMM with all model parameters trained on the data. <strong>B</strong>. HMM with fixed transition probabilities of 0→0 or 1→1 or 2→2 = 0.999; 0→1 or 2→1 = 0.00066; 0→2 or 2→0 = 0.000333; 1→0 or 1→2 = 0.0005. <strong>C</strong>-<strong>F</strong> retain those transition probabilities but with different fixed emission variances of 0.01 (<strong>C</strong>), 0.33 (<strong>D</strong>), 0.8 (<strong>E</strong>), and 1 (<strong>F</strong>).</p>
<div class="figure">
<img src="figs/somites/scatter_collage.png" alt="(ref:hmm-standard)" width="100%" />
<p class="caption">
(#fig:hmm-scatter-diagnoses)(ref:hmm-standard)
</p>
</div>
<p>I used these genotype-block calls to generate the recombination karyoplots shown in <strong>Figures @ref(fig:karyo-wi-missing)</strong> and <strong>@ref(fig:karyo-no-missing)</strong>.</p>
<p>(ref:karyo-wi-missing) Recombination blocks in 622 F2 samples based on the ratio of reads mapping to either the <em>Cab</em> or <em>Kaga</em> allele within 5-kb bins, with homozygous-<em>Cab</em> blocks in green, heterozoygous blocks in navy blue, and homozygous <em>Kaga</em> blocks in red. Most blocks show 0-2 crossover events, as expected, with some regions showing higher numbers of crossovers interpreted as noise. Unfilled regions are those with no state called by the HMM due to a lack of reads mapping to SNPs within those 5-kb bins.</p>
<div class="figure">
<img src="figs/somites/karyoplot_wi_missing.png" alt="(ref:karyo-wi-missing)" width="100%" />
<p class="caption">
(#fig:karyo-wi-missing)(ref:karyo-wi-missing)
</p>
</div>
<div class="figure">
<img src="figs/somites/karyoplot_no_missing.png" alt="(ref:karyo-no-missing)" width="100%" />
<p class="caption">
(#fig:karyo-no-missing)(ref:karyo-no-missing)
</p>
</div>
<p><strong>Figure @ref(fig:prop-sites-total)</strong> shows the proportion of 5-kb bins called as either homozygous-<em>Cab</em>, heterozygous, or homozygous-<em>Kaga</em> within each F2 sample (points). The ordinary expectation for the ratios would be 0.25, 0.5, and 0.25 respectively. However, we observe a skew towards homozygous-<em>Cab</em> and away from homozygous <em>Kaga</em>. This may have been caused by the low homozygosity observed in <em>Kaga</em>.</p>
<p>(ref:prop-sites-total) Proportions of 5-kb blocks called as either homozygous-<em>Cab</em>, heterozygous, or homozygous-<em>Kaga</em>.</p>
<div class="figure">
<img src="figs/somites/prop_sites_total.png" alt="(ref:prop-sites-total)" width="100%" />
<p class="caption">
(#fig:prop-sites-total)(ref:prop-sites-total)
</p>
</div>
</div>
<div id="genome-wide-linkage-anlaysis" class="section level2" number="4.7">
<h2><span class="header-section-number">4.7</span> Genome-wide linkage anlaysis</h2>
<p>Finally, I used the called recombination blocks as pseudo-SNPs in a genetic linkage analysis. To detect associations between the pseudo-SNPs and the three phenotypes of interest, I used a mixed linear model (<strong>MLM</strong>) as implemented in GCTA.<span class="citation"><a href="#fn270" class="footnote-ref" id="fnref270"><sup>270</sup></a></span> That paper describes the model as follows:</p>
<p><span class="math display">\[
\textbf{y} = \textbf{X}\beta + \textbf{Wu} + \epsilon \\
var(\textbf{y}) = \textbf{V} = \textbf{WW}&#39;\sigma^2_{u} + \textbf{I}\sigma_{\epsilon}^2
\]</span>
Where <span class="math inline">\(\textbf{y}\)</span> is a <span class="math inline">\(n\)</span> x 1 vector of phenotypes with <span class="math inline">\(n\)</span> being the sample size, <span class="math inline">\(\textbf{W}\)</span> is a standardised genotype matrix, <span class="math inline">\(\textbf{u}\)</span> is a vector of SNP effects, and <span class="math inline">\(\epsilon\)</span> is a vector of residual effects. I additionally used the leave-one-chromosome-out implementation of GCTA’s MLM, with excludes the chromosome on which the candidate SNP is located when calculating the GRM.</p>
<p>As described above in @ref(somite-phenotype), the microscope used to image the embryos (either AU or DB) differed by several degrees in heat, which likely caused differences in the measurements observed. We accordingly experimented with including microscope as a covariate, either alone or together with the genotype for the reporter locus (either homozygous or heterozygous), or excluding it altogether. In an attempt to avoid complications resulting from its inclusion, we also tried inverse-normalising the period phenotype within each microscope group, transforming the phenotype to fit a normal distribution across both microscopes.</p>
<p>To set the significance threshold, I permuted the phenotype across samples using 10 different random seeds, together with all covariates when included, and ran a separate linkage model for each permutation. I then set the lowest <span class="math inline">\(p\)</span>-value from all 10 permutation as the significance threshold for the non-permuted model. I additionally applied a Bonferroni correction to our <span class="math inline">\(p\)</span>-values by dividing <span class="math inline">\(\alpha\)</span> (0.05) by the number of pseudo-SNPs in the model, and set this as a secondary threshold.</p>
<div id="period-intercept" class="section level3" number="4.7.1">
<h3><span class="header-section-number">4.7.1</span> Period intercept</h3>
<p><strong>Figure @ref(fig:somite-manhattan)</strong> is a Manhattan plot of the genetic linkage results for the period intercept phenotype, inverse-normalised across microscopes. The regions found to be significant based on the permutations’ minimum <span class="math inline">\(p\)</span>-value are set out in <strong>Table @ref(tab:somite-sig-int-tbl)</strong>.</p>
<p>(ref:somite-manhattan) Manhattan plot of the genetic linkage results for the period intercept phenotype, inverse-normalised across microscopes. Pseudo-SNPs with <span class="math inline">\(p\)</span>-values lower than the permutation significance threshold are highlighted in red.</p>
<div class="figure">
<img src="figs/somites/manhattan_intercept.png" alt="(ref:somite-manhattan)" width="100%" />
<p class="caption">
(#fig:somite-manhattan)(ref:somite-manhattan)
</p>
</div>
<table>
<caption>(#tab:somite-sig-int-tbl)Significant 5-kb bin ranges for period intercept below the minimum p-value from 10 permutations.</caption>
<thead>
<tr class="header">
<th align="right">Chromosome</th>
<th align="right">Bin start</th>
<th align="right">Bin end</th>
<th align="right">Length (kb)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">3</td>
<td align="right">31880001</td>
<td align="right">35420000</td>
<td align="right">3540</td>
</tr>
<tr class="even">
<td align="right">4</td>
<td align="right">18090001</td>
<td align="right">18095000</td>
<td align="right">5</td>
</tr>
<tr class="odd">
<td align="right">10</td>
<td align="right">2995001</td>
<td align="right">3690000</td>
<td align="right">695</td>
</tr>
</tbody>
</table>
<p>These regions contained a total of 46,872 SNPs imputed from the genotype of the F0 parental strains.
I ran Ensembl’s Variant Effect Predictor<span class="citation"><a href="#fn271" class="footnote-ref" id="fnref271"><sup>271</sup></a></span> over these SNPs to identify those that would be most likely to have functional consequences. The full counts of SNPs falling into each category of ‘consequence’ are set out in <strong>Table @ref(tab:int-consequence-tbl)</strong>. From this process I identified 38 genes that included a missense variant, 1 that included a missense variant and a start lost (ENSORLG00000014616), ane 1 that included a missense variant and a stop lost (ENSORLG00000015149).</p>
<table>
<caption>(#tab:int-consequence-tbl)Variant Effect Predictor results for SNPs in the bins.</caption>
<thead>
<tr class="header">
<th align="left">Consequence</th>
<th align="right">Count</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">intron variant</td>
<td align="right">47211</td>
</tr>
<tr class="even">
<td align="left">intergenic variant</td>
<td align="right">20045</td>
</tr>
<tr class="odd">
<td align="left">upstream gene variant</td>
<td align="right">7304</td>
</tr>
<tr class="even">
<td align="left">downstream gene variant</td>
<td align="right">5229</td>
</tr>
<tr class="odd">
<td align="left">3 prime UTR variant</td>
<td align="right">1082</td>
</tr>
<tr class="even">
<td align="left">synonymous variant</td>
<td align="right">694</td>
</tr>
<tr class="odd">
<td align="left">missense variant</td>
<td align="right">383</td>
</tr>
<tr class="even">
<td align="left">5 prime UTR variant</td>
<td align="right">201</td>
</tr>
<tr class="odd">
<td align="left">splice region variant,intron variant</td>
<td align="right">126</td>
</tr>
<tr class="even">
<td align="left">missense variant,splice region variant</td>
<td align="right">19</td>
</tr>
<tr class="odd">
<td align="left">splice region variant,synonymous variant</td>
<td align="right">17</td>
</tr>
<tr class="even">
<td align="left">stop gained</td>
<td align="right">3</td>
</tr>
<tr class="odd">
<td align="left">splice donor variant</td>
<td align="right">1</td>
</tr>
<tr class="even">
<td align="left">start lost</td>
<td align="right">1</td>
</tr>
<tr class="odd">
<td align="left">stop lost</td>
<td align="right">1</td>
</tr>
<tr class="even">
<td align="left">stop lost,splice region variant</td>
<td align="right">1</td>
</tr>
</tbody>
</table>
<p>Our collaborators then combined these results with bulk RNA-seq that they had performed on F0 <em>Cab</em> and <em>Kaga</em> individuals, to determine which of these genes are expressed in the tail during embryogenesis. This allowed them to reduce to the list to 29 genes, and a gene ontology analysis of this found that the list of genes was enriched for body axis, somitogenesis, and segmentation (<strong>Table @ref(tab:psm-final-genes)</strong>). For this list of genes, our collaborators are now in the process of knocking in the protein-altering <em>Cab</em> allele into <em>Kaga</em> embryos, and <em>vice versa</em>, to functionally validate these variants.</p>
<table>
<caption>(#tab:psm-final-genes)Target genes for functional validation expressed in the unsegmented PSM and containing protein alterations. Table generated by Ali Seleit.</caption>
<colgroup>
<col width="10%" />
<col width="12%" />
<col width="38%" />
<col width="38%" />
</colgroup>
<thead>
<tr class="header">
<th align="right">chromosome_name</th>
<th align="left">ensembl_gene_id</th>
<th align="left">description</th>
<th align="left">Role</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">3</td>
<td align="left">ENSORLG00000014656</td>
<td align="left">mesoderm posterior protein 2-like</td>
<td align="left">Somitogenesis</td>
</tr>
<tr class="even">
<td align="right">3</td>
<td align="left">ENSORLG00000014659</td>
<td align="left">mesp</td>
<td align="left">Somitogenesis</td>
</tr>
<tr class="odd">
<td align="right">10</td>
<td align="left">ENSORLG00000020474</td>
<td align="left">protocadherin 10b</td>
<td align="left">Somitogenesis</td>
</tr>
<tr class="even">
<td align="right">3</td>
<td align="left">ENSORLG00000014616</td>
<td align="left">NA</td>
<td align="left">Possible role in Somitogenesis (MAP-kinase)</td>
</tr>
<tr class="odd">
<td align="right">10</td>
<td align="left">ENSORLG00000020551</td>
<td align="left">FAT atypical cadherin 4</td>
<td align="left">Possible role in Somitogenesis (PCP, Yap1 regulator)</td>
</tr>
<tr class="even">
<td align="right">10</td>
<td align="left">ENSORLG00000020531</td>
<td align="left">neurogenin 1</td>
<td align="left">Possible role in Somitogenesis (BHLH-TF regulation of Wnt)</td>
</tr>
<tr class="odd">
<td align="right">3</td>
<td align="left">ENSORLG00000015149</td>
<td align="left">ADAM metallopeptidase with thrombospondin type 1 motif 18</td>
<td align="left">Extracellular matrix</td>
</tr>
<tr class="even">
<td align="right">3</td>
<td align="left">ENSORLG00000015418</td>
<td align="left">matrix metallopeptidase 15</td>
<td align="left">Extracellular matrix</td>
</tr>
<tr class="odd">
<td align="right">10</td>
<td align="left">ENSORLG00000020488</td>
<td align="left">transforming growth factor beta induced</td>
<td align="left">Extracellular matrix</td>
</tr>
<tr class="even">
<td align="right">3</td>
<td align="left">ENSORLG00000028055</td>
<td align="left">NA</td>
<td align="left"> Signal trasduction (Rho)</td>
</tr>
<tr class="odd">
<td align="right">10</td>
<td align="left">ENSORLG00000020481</td>
<td align="left">ArfGAP with RhoGAP domain, ankyrin repeat and PH domain 3</td>
<td align="left"> Signal transduction (GTPase activator RhoGap)</td>
</tr>
<tr class="even">
<td align="right">10</td>
<td align="left">ENSORLG00000020525</td>
<td align="left">TBC1 domain family member 2A-like</td>
<td align="left">Signal transduction Gtpase activator, cadherin recycling</td>
</tr>
<tr class="odd">
<td align="right">3</td>
<td align="left">ENSORLG00000015260</td>
<td align="left">synapse associated protein 1</td>
<td align="left">TOR acitivity</td>
</tr>
<tr class="even">
<td align="right">10</td>
<td align="left">ENSORLG00000028553</td>
<td align="left">NA</td>
<td align="left">NA</td>
</tr>
<tr class="odd">
<td align="right">3</td>
<td align="left">ENSORLG00000015460</td>
<td align="left">dpy-19 like C-mannosyltransferase 3</td>
<td align="left">glycosylation</td>
</tr>
<tr class="even">
<td align="right">10</td>
<td align="left">ENSORLG00000022010</td>
<td align="left">beta-1,4-galactosyltransferase 1</td>
<td align="left">glycosylation</td>
</tr>
<tr class="odd">
<td align="right">10</td>
<td align="left">ENSORLG00000020498</td>
<td align="left">protein NipSnap homolog 3A</td>
<td align="left">Mitochondria</td>
</tr>
<tr class="even">
<td align="right">10</td>
<td align="left">ENSORLG00000020494</td>
<td align="left">nitric oxide associated 1</td>
<td align="left">Mitochondria</td>
</tr>
<tr class="odd">
<td align="right">10</td>
<td align="left">ENSORLG00000020504</td>
<td align="left">ATP-binding cassette sub-family A member 1</td>
<td align="left">Metabolism (cholestrol efflux)</td>
</tr>
<tr class="even">
<td align="right">3</td>
<td align="left">ENSORLG00000015118</td>
<td align="left">phosphorylase kinase regulatory subunit beta</td>
<td align="left">Metabolism</td>
</tr>
<tr class="odd">
<td align="right">10</td>
<td align="left">ENSORLG00000020493</td>
<td align="left">RE1 silencing transcription factor</td>
<td align="left">RNA pol II (transcritpion regulation</td>
</tr>
<tr class="even">
<td align="right">3</td>
<td align="left">ENSORLG00000015365</td>
<td align="left">solute carrier family 7 member 6 opposite strand</td>
<td align="left">RNA pol II (nuclear export)</td>
</tr>
<tr class="odd">
<td align="right">10</td>
<td align="left">ENSORLG00000029052</td>
<td align="left">exosome component 3</td>
<td align="left">rRNA + RNA binding</td>
</tr>
<tr class="even">
<td align="right">3</td>
<td align="left">ENSORLG00000015278</td>
<td align="left">adhesion G protein-coupled receptor G3</td>
<td align="left">G-protein coupled receptor</td>
</tr>
<tr class="odd">
<td align="right">3</td>
<td align="left">ENSORLG00000015287</td>
<td align="left">adhesion G-protein coupled receptor G5</td>
<td align="left">G-protein coupled receptor</td>
</tr>
<tr class="even">
<td align="right">10</td>
<td align="left">ENSORLG00000025674</td>
<td align="left">matrin-3</td>
<td align="left">inner nuclear protein (chromatin architecture</td>
</tr>
<tr class="odd">
<td align="right">10</td>
<td align="left">ENSORLG00000023325</td>
<td align="left">matrin-3</td>
<td align="left">inner nuclear protein (chromatin architecture</td>
</tr>
<tr class="even">
<td align="right">10</td>
<td align="left">ENSORLG00000022388</td>
<td align="left">poly(A) binding protein interacting protein 2</td>
<td align="left">Translation repressor</td>
</tr>
<tr class="odd">
<td align="right">3</td>
<td align="left">ENSORLG00000015096</td>
<td align="left">integrin alpha FG-GAP repeat containing 1</td>
<td align="left">NA possibly T cell activation</td>
</tr>
</tbody>
</table>
</div>
<div id="psm-area-1" class="section level3" number="4.7.2">
<h3><span class="header-section-number">4.7.2</span> PSM area</h3>
<p><strong>Figure @ref(fig:psm-manhattan)</strong> is a Manhattan plot of the genetic linkage results for the PSM area phenotype. The regions found to be significant based on the permutations’ minimum <span class="math inline">\(p\)</span>-value are set out in <strong>Table @ref(tab:somite-sig-psm-tbl)</strong>, although they exceed the Bonferroni correction threshold as well. I note that this ~6 Mb significant region on chromosome 3 does not overlap at all with the significant region discovered for the period intercept phenotype.</p>
<p>(ref:psm-manhattan) Manhattan plot of the genetic linkage results for the PSM area phenotype. Pseudo-SNPs with <span class="math inline">\(p\)</span>-values lower than the permutation significance threshold are highlighted in yellow.</p>
<div class="figure">
<img src="figs/somites/manhattan_psm_no-covariates.png" alt="(ref:psm-manhattan)" width="100%" />
<p class="caption">
(#fig:psm-manhattan)(ref:psm-manhattan)
</p>
</div>
<table>
<caption>(#tab:somite-sig-psm-tbl)Significant 5-kb bin range for PSM area below the minimum p-value from 10 permutations.</caption>
<thead>
<tr class="header">
<th align="right">CHROM</th>
<th align="right">Bin start</th>
<th align="right">Bin end</th>
<th align="right">Length (kb)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">3</td>
<td align="right">20375001</td>
<td align="right">26285000</td>
<td align="right">5910</td>
</tr>
</tbody>
</table>
<p>This region contained a total of 29,096 SNPs imputed from the genotype of the F0 parental strains.
I ran Ensembl’s Variant Effect Predictor<span class="citation"><a href="#fn272" class="footnote-ref" id="fnref272"><sup>272</sup></a></span> over these SNPs to identify those that would be most likely to have functional consequences. The full counts of SNPs falling into each category of ‘consequence’ are set out in <strong>Table @ref(tab:psm-consequence-tbl)</strong>. From this process I identified 114 genes that included a missense variant, and 1 that included a both a missense variant and a start lost (ENSORLG00000010863, a centriole, cilia and spindle-associated protein).</p>
<table>
<caption>(#tab:psm-consequence-tbl)Variant Effect Predictor results for SNPs in the bins.</caption>
<thead>
<tr class="header">
<th align="left">Consequence</th>
<th align="right">Count</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">intron variant</td>
<td align="right">23189</td>
</tr>
<tr class="even">
<td align="left">intergenic variant</td>
<td align="right">9171</td>
</tr>
<tr class="odd">
<td align="left">downstream gene variant</td>
<td align="right">8894</td>
</tr>
<tr class="even">
<td align="left">upstream gene variant</td>
<td align="right">8491</td>
</tr>
<tr class="odd">
<td align="left">3 prime UTR variant</td>
<td align="right">2104</td>
</tr>
<tr class="even">
<td align="left">synonymous variant</td>
<td align="right">1141</td>
</tr>
<tr class="odd">
<td align="left">missense variant</td>
<td align="right">716</td>
</tr>
<tr class="even">
<td align="left">5 prime UTR variant</td>
<td align="right">433</td>
</tr>
<tr class="odd">
<td align="left">splice region variant,intron variant</td>
<td align="right">184</td>
</tr>
<tr class="even">
<td align="left">splice region variant,synonymous variant</td>
<td align="right">18</td>
</tr>
<tr class="odd">
<td align="left">missense variant,splice region variant</td>
<td align="right">7</td>
</tr>
<tr class="even">
<td align="left">stop gained</td>
<td align="right">2</td>
</tr>
<tr class="odd">
<td align="left">splice donor variant</td>
<td align="right">1</td>
</tr>
<tr class="even">
<td align="left">start lost</td>
<td align="right">1</td>
</tr>
</tbody>
</table>
<p>Our collaborators then combined these results with bulk RNA-seq that they had performed on F0 <em>Cab</em> and <em>Kaga</em> individuals, to determine which of these genes are expressed in the unsegmented tail during embryogenesis. This allowed them to reduce to the list to 96 genes, although they were not apparently associated with a specific gene ontology. As with the period intercept phenotype, our collaborators are now in the process of knocking in the <em>Cab</em> allele into <em>Kaga</em> embryos, and <em>vice versa</em>, to functionally validate these variants.</p>
<!--chapter:end:04-somites.Rmd-->
</div>
</div>
</div>
<div id="MIKK-F2-chap" class="section level1" number="5">
<h1><span class="header-section-number">5</span> Genetic linkage study of bold/shy behaviours in the MIKK panel</h1>
<p>The purpose of the study described in this chapter was to run the behavioural analysis described in Chapter @ref(Pilot-chap) over the MIKK panel described in Chapter @ref(MIKK-genomes-chap), identify the lines that diverged in both (a) their own behaviour; and (b) the level of transmission of their behaviour onto their <em><span style="color: #424B4D;">iCab</span></em> reference tank partner, and then use them as the parental strains in an F2 cross to attempt to identify the genetic variants associated with those differences.</p>
<div id="MIKK-F2-cross" class="section level2" number="5.1">
<h2><span class="header-section-number">5.1</span> The F2 cross experimental setup</h2>
<p>The F2 cross is a traditional method for mapping genetic variants associated with traits of interest. A schema for the method is presented in <strong>Figure @ref(fig:F2-cross-schema)</strong>. In essence, it involves starting with two inbred strains that diverge for the trait of interest (the ‘parental strains’, or F0 generation F0). One then crosses the parental strains to create a generation of F1 hybrid individuals who each possess, for every pair of their chromosomes, one chromosome from each of their parents. The individuals in this F1 generation are genetically identical to their parents with respect to their germ line. Finally, one inter-crosses the F1 generation to create a set of F2 individuals that share unique combinations of the original F0 strains’ genotypes, and tend to display values for the trait of interest that span across the spectrum between the extreme values of their parents.</p>
<p>(ref:F2-cross-schema) Schema of the F2-cross experimental setup. The F0 generation comprises two medaka strains that have extreme, opposing values for a trait of interest, represented by the colours red and blue. Below them is an illustrative single pair of chromosomes. The chromosomes within each pair are depicted as the same colour, as the strains are homozygous through successive generations of inbreeding. Their F1 offspring is heterozygous for each pair of their 24 chromosomes, and all F1 individuals are therefore almost genetically identical to one another (with the exception of somatic mutations and the regions of the genome that were not homozygous in the parental generations). The F1 individuals are then inter-crossed with one another to produce the F2 generation, which, due to recombination events during gamete formation, have unique combinations of the parental strains’ genotypes, and tend to span the phenotypic spectrum between the extremes of their F0 parental strains, represented by their colours.</p>
<div class="figure">
<img src="figs/mikk_behaviour/F2-cross-schema.png" alt="(ref:F2-cross-schema)" width="100%" />
<p class="caption">
(#fig:F2-cross-schema)(ref:F2-cross-schema)
</p>
</div>
</div>
<div id="data-collection---f0-generation" class="section level2" number="5.2">
<h2><span class="header-section-number">5.2</span> Data collection - F0 generation</h2>
<p>In November 2019 I traveled to the fish facility managed by our collaborator, Felix Loosli at KIT in Karlsruhe, and over the course of 11 days from 11 to 21 November 2019, I ran the behavioural assay described in Chapter @ref(pilot-data-collection) another 206 times. I again used the <em><span style="color: #424B4D;">iCab</span></em> strain as the reference fish, and for the test fish I used either an individual from one of the MIKK panel lines, individuals captured from the same Kiyosu population as the MIKK panel but permitted to breed freely within a separate tank in the facility (‘Kiyosu closed-capture’, or ‘Kiyosu CC’), or individuals from a related but different species of medaka from the Philippines, <em>Oryzias luzonensis</em>. I ensured that I performed at least 2 assay runs for each MIKK panel line that was available, generating a minimum of 8 test fish replicates per line. As there were four pairs of fish in the test tank during each run, the complete dataset comprises 824 videos of pairs of fish, which I further divided by assay component (open field and novel object) to create 1648 videos.</p>
<p>I again used the software <em>idtrackerai</em><span class="citation"><a href="#fn273" class="footnote-ref" id="fnref273"><sup>273</sup></a></span> to track the movement of the fishes across frames of each video. After adjusting the software parameters for each video to maximise the number of frames that were successfully tracked, I was left with 1610 out of the 1648 videos (~97.7%) where both fishes were tracked over at least 85% of frames, and I only included these 1610 videos in the downstream analysis. The first question to address was whether the MIKK panel lines differed in their behaviours. I therefore computed each individual fish’s mean speed (measured as the distance traveled in pixels per 0.08 seconds) over the course of the full 20-minute video, grouped them by line, and plotted the results presented in <strong>Figure @ref(fig:mikk-mean-speed)</strong>. I continue to use the same order and colour palette for the MIKK panel lines as in this Figure throughout the rest of this Chapter.</p>
<p>(ref:mikk-mean-speed) Mean speed of the MIKK panel and other strains over the course of the entire 20-minute video (measured as the distance traveled in pixels per 0.05 seconds). <em><span style="color: #424B4D;">iCab</span></em> fishes in the <em><span style="color: #424B4D;">iCab</span></em>-<em><span style="color: #424B4D;">iCab</span></em> control condition are at the top, the MIKK panel lines are sorted by their group median, and the Kiyosu closed capture and <em>O. luzonensis</em> fishes are at the bottom.</p>
<div class="figure">
<img src="figs/mikk_behaviour/F0_line_mean_speed_0.08.png" alt="(ref:mikk-mean-speed)" width="100%" />
<p class="caption">
(#fig:mikk-mean-speed)(ref:mikk-mean-speed)
</p>
</div>
<p>This figure shows that there are clear differences between some MIKK panel lines at the extremes, and that the lines differ in the amount of within-line variance observed (shown plotted against the lines’ median speed in <strong>Figure @ref(fig:mikk-mean-speed-variance)</strong>). These figures acted as a general guide to determine which lines to select as the parental strains in the F2 cross. To identify genetic variants directly associated with bold-shy behaviours, I sought to select lines that showed either high or low levels of movement, and preferably low within-line variance.</p>
<p>(ref:mikk-mean-speed-variance) Line median (vertical axis) and line variance (horizontal axis) for individual mean speed across the full 20-minute video (i.e. both the open field and novel object assay components).</p>
<div class="figure">
<img src="figs/mikk_behaviour/line_mean_speed_variance_0.05_all.png" alt="(ref:mikk-mean-speed-variance)" width="100%" />
<p class="caption">
(#fig:mikk-mean-speed-variance)(ref:mikk-mean-speed-variance)
</p>
</div>
<p>To illustrate the differences between certain lines in terms of both direct and social genetic effects, in <strong>Figure @ref(fig:extreme-paths)</strong> I have plotted the tracked paths for 3 lines 5 minutes into the 10-minute open field assay component: the slowest line <span style="color: #FB737A;">22-1</span>, one of the other slowest lines <span style="color: #FF66A6;">18-2</span>, and the fastest line <span style="color: #F8766D;">10-1</span>. The fishes are coloured by their line, with <em><span style="color: #424B4D;">iCab</span></em> in dark grey. There appears to be a social genetic effect when comparing the behaviour of <em><span style="color: #424B4D;">iCab</span></em> when paired with the similarly slow-moving lines <span style="color: #FB737A;">22-1</span> and <span style="color: #FF66A6;">18-2</span>. At the other extreme, line <span style="color: #F8766D;">10-1</span> has moved extremely quickly, spending much of its time moving along the boundaries of their test tanks. I understand from our collaborator Felix Loosli, a fish behaviour expert, that this movement along the boundaries of a tank is typical of medaka when introduced to a novel environment, and we observe it too with <em><span style="color: #424B4D;">iCab</span></em> when it is paired with line <span style="color: #FB737A;">22-1</span>.</p>
<p>(ref:extreme-paths) Path plots for lines 22-1 (<strong>A</strong>), 18-2 (<strong>B</strong>) and 10-1 (<strong>C</strong>) 5 minutes into the open field assay for the first (left) and second (right) run with each line. The paths of <em><span style="color: #424B4D;">iCab</span></em> individuals are coloured in dark grey, with the other lines depicted in their representative colours.</p>
<div class="figure">
<img src="figs/mikk_behaviour/path_plot_22-1_18-2_10-1_300.png" alt="(ref:extreme-paths)" width="100%" />
<p class="caption">
(#fig:extreme-paths)(ref:extreme-paths)
</p>
</div>
<p>Although lines <span style="color: #FB737A;">22-1</span> and <span style="color: #F8766D;">10-1</span> were the most extreme in terms of mean speed, I ruled them out of selection for the F2 cross for the following reasons. Our collaborators informed us that through a separate analysis on heartbeat phenotypes across the MIKK panel, they discovered that the heart of line <span style="color: #FB737A;">22-1</span> often stops beating for up to minutes at a time. This may explain the lack of movement observed during our assay. The behaviours they exhibit may therefore not represent a phenotype related to the boldness-shyness axis, but rather an organ failure.</p>
<p>On the other hand, line <span style="color: #F8766D;">10-1</span> appeared to habituate to the open field assay component by slowing down, rather than speeding up (<strong>Figure @ref(fig:10-1-dens)</strong>). That is to say, as the assay progressed, they slowed down their movement, and this suggests that their typical response to stress is to move faster rather than slower. However, as I am using speed as a proxy for boldness (where a quicker habituation to the assay, indicated by an increase in movement, suggests greater boldness), this would create difficulties in attributing a functional interpretation of behaviours in the F2 individuals, and for this reason I excluded line <span style="color: #F8766D;">10-1</span>.</p>
<p>(ref:10-1-dens) Tile and density plots for line <span style="color: #F8766D;">10-1</span>, showing the reduction in the frequency of higher-movement states over the course of the open field assay component (<strong>A</strong> and <strong>B</strong>), with the novel object assay component (<strong>C</strong> and <strong>D</strong>) for comparison.</p>
<div class="figure">
<img src="figs/mikk_behaviour/select_0.08_15_10-1_dge.png" alt="(ref:10-1-dens)" width="100%" />
<p class="caption">
(#fig:10-1-dens)(ref:10-1-dens)
</p>
</div>
</div>
<div id="hmm-states" class="section level2" number="5.3">
<h2><span class="header-section-number">5.3</span> HMM states</h2>
<p>To explore the behaviours of the MIKK panel at a finer resolution, as for the pilot study described in @ref(Pilot-chap), I again applied a hidden markov model (<strong>HMM</strong>) to classify the fishes’ movements based on their distance and angle of travel between time intervals. I used the same method to select the best choice of time interval and number of states (<strong>Figure @ref(fig:mikk-param-comp)</strong>).</p>
<p>(ref:mikk-param-comp) Comparison between HMM parameters. Horizontal axis: Mean concordance between states assigned by HMMs through a 2-fold cross-validation process. Vertical axis: Kruskal-Wallis statistic comparing strains based on the proportion of time spent in each HMM state, summed across all states. Size of points correspond to the interval, in seconds, between which the distance and angle of travel was calculated (Methods).</p>
<div class="figure">
<img src="figs/mikk_behaviour/compare_params.png" alt="(ref:mikk-param-comp)" width="100%" />
<p class="caption">
(#fig:mikk-param-comp)(ref:mikk-param-comp)
</p>
</div>
<p>Here I observed the same phenomenon where the parameter combinations that performed the best showed an asymmetry between some states that would make interpretation difficult. For example, a time interval of 0.08 seconds combined with a state space of 17 caused state 4 to appear to get carved out of state 3 (<strong>Figure @ref(fig:mikk-hmm-asym)</strong>).</p>
<p>(ref:mikk-hmm-asym) The best apparent combination of parameters (0.08 time interval with a 17-state space) created an asymmetry between states 3 and 4, which would causes difficulties in interpreting their biological relevance.</p>
<div class="figure">
<img src="figs/mikk_behaviour/0.08_17_polar_all_dge.png" alt="(ref:mikk-hmm-asym)" width="100%" />
<p class="caption">
(#fig:mikk-hmm-asym)(ref:mikk-hmm-asym)
</p>
</div>
<p>The best combination of parameters without this asymmetry was a time interval of 0.05 seconds with a state space of 15 (see the polar plots for the states in <strong>Appendix @ref(fig:hmm-states-05)</strong>). However, due to a glitch in the video recording software, several videos recorded on 13 November 2019 were incorrectly recorded with a frame rate of 14 instead of the desired 30. The insufficient number of frames for those videos meant that it was impossible to measure the distance and angle of travel between a time interval as low as 0.05 seconds. So that these videos could be included in the dataset, I accordingly selected the combination of 15 states and a 0.08-second interval for all downstream analyses.</p>
<p>(ref:mikk-hmm-sym) The HMM states used for the downstream analysis, with the model classified based on the distance of travel (log<sub>10</sub> pixels, radial axis) and angle of travel (angle). A straight forward movement would sit around 0°, a left movement around 270°, and a right movement around 90°.</p>
<div class="figure">
<img src="figs/mikk_behaviour/0.08_15_polar_all_dge.png" alt="(ref:mikk-hmm-sym)" width="100%" />
<p class="caption">
(#fig:mikk-hmm-sym)(ref:mikk-hmm-sym)
</p>
</div>
</div>
<div id="social-genetic-effects-1" class="section level2" number="5.4">
<h2><span class="header-section-number">5.4</span> Social genetic effects</h2>
<p>As discussed above, in this project our traits of interest include not only direct genetic behaviours, but also social genetic behaviours. I therefore sought to identify the MIKK panel lines that transmitted their behaviour onto the reference <em><span style="color: #424B4D;">iCab</span></em> tank partners either to the greatest or least extent. I formulated two methods to measure this, referred to as a) HMM state co-occupancy; and b) reference deviation. The first, HMM state co-occupancy, measured the proportions of time that the <em><span style="color: #424B4D;">iCab</span></em> reference fish spent in the same HMM state as its tank partner. The second, deviation of the reference fishes’ deviation from the behaviour exhibited in the control condition, seeks to quantify the extent to which the <em><span style="color: #424B4D;">iCab</span></em>’s behaviour changes when partnered with</p>
<div id="state-co-occupancy" class="section level3" number="5.4.1">
<h3><span class="header-section-number">5.4.1</span> State co-occupancy</h3>
<p><strong>Figure @ref(fig:F0-sge-cooc-box)</strong> sets out the proportions of total time for each assay sub-component that each pair of individual fish spent in the same HMM state, grouped by line and ranked in the same order as their group median for individual mean speed as shown above in <strong>Figure @ref(fig:mikk-mean-speed)</strong>. <strong>@ref(fig:F0-sge-cooc-box)A</strong> shows the data as boxplots, with <span class="math inline">\(p\)</span>-values from the Kruskal-Wallis test comparing all groups. <strong>Figure @ref(fig:F0-sge-cooc-box)B</strong> shows the same data but with each group’s median represented by columns to make it easier to compare group medians. Of the slower-moving lines, <span style="color: #FF699C;">8-2</span> and <span style="color: #FF66A6;">18-2</span> tend to show relatively higher state co-occupancy in the open field component, but during the novel object component, the slow-moving line <span style="color: #FF61CC;">139-4</span> has the highest median co-occupancy of all lines. Of the faster-moving lines, <span style="color: #F17D50;">43-2</span> and <span style="color: #F57A5F;">13-2</span> showed the highest state co-occupancy during the open field assay component. However, the moderate-to-fast line <span style="color: #49B500;">21-2</span> had relatively high state co-occupancy during both assay components.</p>
<p>(ref:F0-sge-cooc-box) Frequency of HMM state co-occupancy</p>
<div class="figure">
<img src="figs/mikk_behaviour/0.08_15_cooc_box_all.png" alt="(ref:F0-sge-cooc-box)" width="100%" />
<p class="caption">
(#fig:F0-sge-cooc-box)(ref:F0-sge-cooc-box)
</p>
</div>
<p>To visualise which states are driving the higher co-occupancy measures, for a selection of lines I generated a heatmap of the states occupied simultaneously by the <em><span style="color: #424B4D;">iCab</span></em> reference and MIKK test fishes, combining the observations for all individuals within each test fish group (<strong>Figure @ref(fig:F0-sge-cooc-heat)</strong>). When <em><span style="color: #424B4D;">iCab</span></em> is paired with <span style="color: #FF66A6;">18-2</span> or <span style="color: #FF699C;">8-2</span>, the fishes most frequently occupy states 3 or 1 in both open field and novel object components. In pairings with line <span style="color: #FF61CC;">139-4</span>, while the test fishes remain in the still states 1 or 3 during the open field assay, <em><span style="color: #424B4D;">iCab</span></em> tends to be moving much faster in states 11 and 13. However, during the novel object component, they co-occupy state 3 more than in any other combination. This general pattern is observed with line <span style="color: #F066EA;">14-2</span> as well, albeit to a lesser extent. For <span style="color: #00C08B;">38-2</span>, the fishes tend not to show a strong preference for co-occupying a particular state for either assay component, but the diagonal spread indicates that they tend to move at similar speeds. When paired with the faster moving <span style="color: #49B500;">21-2</span>, the novel object component appears to accentuate the co-occupancy of state 3 that is also observed in the open field component. Finally, when paired with line <span style="color: #93AA00;">40-1</span>, in both assay components, both fishes show a strong preference for the faster-moving states.</p>
<p>(ref:F0-sge-cooc-heat) Heatmaps for a selection of MIKK panel lines (including those ultimately selected as the parental strains in the F2 cross) showing the frequency of HMM states simultaneously occupied by the reference (x-axis) and test (y-axis) fishes, aggregated over all replicates in each line.</p>
<div class="figure">
<img src="figs/mikk_behaviour/0.08_15_cooc_heatmap.png" alt="(ref:F0-sge-cooc-heat)" width="100%" />
<p class="caption">
(#fig:F0-sge-cooc-heat)(ref:F0-sge-cooc-heat)
</p>
</div>
</div>
<div id="deviation-of-icab-from-its-control-condition" class="section level3" number="5.4.2">
<h3><span class="header-section-number">5.4.2</span> Deviation of <em><span style="color: #424B4D;">iCab</span></em> from its control condition</h3>
<p>The second method for quantifying the level of behavioural transmission from test fish to <em><span style="color: #424B4D;">iCab</span></em> reference fish was to determine the proportion of time that the <em><span style="color: #424B4D;">iCab</span></em> spent in a particular state when paired with another <em><span style="color: #424B4D;">iCab</span></em>, and then quantify the degree to which those proportions change when in the presence of a fish from another line (<strong>Figure @ref(fig:F0-sge-deviation)</strong>). <strong>Figure @ref(fig:F0-sge-deviation)A</strong> presents boxplots for state frequencies for all <em><span style="color: #424B4D;">iCab</span></em> individuals in <em><span style="color: #424B4D;">iCab</span></em>-<em><span style="color: #424B4D;">iCab</span></em> pairings. I further calculated the state frequencies for all <em><span style="color: #424B4D;">iCab</span></em> reference fishes for all other MIKK line pairings. For each combination of assay component, line-pairing, and HMM state, I then ran Welch’s t-test<span class="citation"><a href="#fn274" class="footnote-ref" id="fnref274"><sup>274</sup></a></span> comparing the proportions of time the <em><span style="color: #424B4D;">iCab</span></em> individuals spent in that state when paired with another <em><span style="color: #424B4D;">iCab</span></em>, against the proportions of time the <em><span style="color: #424B4D;">iCab</span></em> reference individuals spent in that state when paired with a different MIKK line. I then summed the t-statistics across states to generate a single metric for each combination of line and assay component, and plotted the results in <strong>Figure @ref(fig:F0-sge-deviation)B</strong>.</p>
<p>(ref:F0-sge-deviation) Deviation of state frequency for <em><span style="color: #424B4D;">iCab</span></em> reference fishes when paired with MIKK panel lines relative to when paired with another <em><span style="color: #424B4D;">iCab</span></em>. <strong>A</strong>: Boxplots of HMM state frequency for <em><span style="color: #424B4D;">iCab</span></em> individuals when paired with another <em><span style="color: #424B4D;">iCab</span></em>. <strong>B</strong>:</p>
<div class="figure">
<img src="figs/mikk_behaviour/0.08_15_deviation.png" alt="(ref:F0-sge-deviation)" width="100%" />
<p class="caption">
(#fig:F0-sge-deviation)(ref:F0-sge-deviation)
</p>
</div>
<p>The first thing to note in this Figure is that <em><span style="color: #424B4D;">iCab</span></em>’s deviations from its behaviour exhibited in the control condition tend to be smaller when paired with faster-moving fishes. This was expected given <em><span style="color: #424B4D;">iCab</span></em> is also a faster-moving fish, but it strengthens the observation that when paired with slower-moving fishes, those tank partners are causing the <em><span style="color: #424B4D;">iCab</span></em> reference fish to move slower than they would otherwise. Of the slower-moving MIKK lines, <span style="color: #FC61D4;">4-2</span> and <span style="color: #F066EA;">14-2</span> causing the largest deviations of the <em><span style="color: #424B4D;">iCab</span></em> reference fishes’ behaviours from their control condition behaviour in the open field and novel object components respectively. I also observe large deviations for the moderately-moving <span style="color: #00B9E3;">106-2</span> and <span style="color: #00BECD;">71-1</span> during the novel object components. There are no clear outliers for either assay component for any of the faster moving lines.</p>
</div>
</div>
<div id="selection-of-lines-for-the-f2-cross" class="section level2" number="5.5">
<h2><span class="header-section-number">5.5</span> Selection of lines for the F2 cross</h2>
<p>On the basis of the above findings, I selected 6 MIKK panel lines as the parental lines for the F2 cross (<strong>Figure @ref(fig:F0-line-mean-speed-select)</strong>). Conceptually, I sought to select lines that diverged on two measures: a) bold-shy behaviours; and b) the extent to which the lines transmitted their behaviours onto their tank partners.</p>
<p>As <em><span style="color: #424B4D;">iCab</span></em> is a fast-moving line, it was more difficult to detect instances where fast-moving MIKK panel lines were influencing its behaviour. I was therefore more confident of identifying slow-movement/high-charisma lines, and so to increase the likelihood of identifying genetic variants that are responsible for a stronger transmission of slow-moving behaviours, I chose two slow-movement/high-charisma lines for the F2 cross: <span style="color: #FF66A6;">18-2</span> and <span style="color: #FF699C;">8-2</span>. In the event that both lines possessed the same genetic variants that influence this behavioural transmission trait, it would vastly increase the power of detecting it during the genetic linkage analysis. Both these lines were one of the most slow-moving lines, had high levels of state co-occupancy during both assay components, and <span style="color: #FF66A6;">18-2</span> caused a high level of deviation from the <em><span style="color: #424B4D;">iCab</span></em> control condition.</p>
<p>(ref:F0-line-mean-speed-select) Mean speed of the MIKK panel and other strains over the course of the entire 20-minute video (measured as the distance traveled in pixels per 0.05 seconds), as shown above in Figure @ref(fig:mikk-mean-speed) but now highlighting the MIKK lines that were selected for the F2 cross.</p>
<div class="figure">
<img src="figs/mikk_behaviour/line_mean_speed_0.05_selected.png" alt="(ref:F0-line-mean-speed-select)" width="100%" />
<p class="caption">
(#fig:F0-line-mean-speed-select)(ref:F0-line-mean-speed-select)
</p>
</div>
<p>For the slow-movement/low-charisma line, I selected <span style="color: #BB81FF;">50-2</span> which exhibited a moderately slow level of movement, low within-line variance for mean speed, and low measures for both state co-occupancy and <em><span style="color: #424B4D;">iCab</span></em> deviation. For the high-movement/high-charisma line, I selected <span style="color: #49B500;">21-2</span>. Despite its high within line variance (<strong>Figure @ref(fig:F2-time-dge-of)</strong>), this potentially made it easier to detect its social genetic effects, as the slower-moving individuals appeared to transmit those behaviours strongly to their <em><span style="color: #424B4D;">iCab</span></em> tank partners, giving it a high score among fast-moving lines for state co-occupancy across both assay components. For the high-movement/low-charisma line I selected <span style="color: #93AA00;">40-1</span>, as it has low within-line variance, and low-to-moderate metrics for state co-occupancy and <em><span style="color: #424B4D;">iCab</span></em> deviation. However, I note that these measures may be confounded by the possibility that <span style="color: #93AA00;">40-1</span> behaves in a similar way to <em><span style="color: #424B4D;">iCab</span></em>, which would be difficult to determine whether it was behaving differently.</p>
<p>In addition to these extreme lines, I also selected a line that was intermediate for both traits in an attempt to avoid breeding incompatibilities that might arise from attempting to cross lines with such divergent behavioural traits. For this purpose I selected line <span style="color: #00C08B;">38-2</span> for its intermediate speed, low within-line variance for mean speed, and intermediate measures for HMM state co-occupancy and <em><span style="color: #424B4D;">iCab</span></em> deviation.</p>
<p>(ref:F0-line-mean-speed-var-select) Line median (vertical axis) and line variance (horizontal axis) for individual mean speed across the full 20-minute video (i.e. both the open field and novel object assay components) as shown above in Figure @ref(fig:mikk-mean-speed-variance), now coloured only for the lines selected as the parental strains for the F2 cross.</p>
<div class="figure">
<img src="figs/mikk_behaviour/line_mean_speed_variance_selected.png" alt="(ref:F0-line-mean-speed-var-select)" width="100%" />
<p class="caption">
(#fig:F0-line-mean-speed-var-select)(ref:F0-line-mean-speed-var-select)
</p>
</div>
<p>The selection of parental lines for the F2 cross indicating their putative position across the two axes of interest (movement, and behavioural transmission or ‘charisma’) are set out below in <strong>Figure @ref(fig:F0-line-select-schema)</strong>.</p>
<p>(ref:F0-line-select-schema) Schema of the MIKK panel lines selected as the parental strains for the multi-way F2 cross, indicating their intended position across the two axes of interest (movement, and behavioural transmission or ‘charisma’).</p>
<div class="figure">
<img src="figs/mikk_behaviour/line_selection_schema.png" alt="(ref:F0-line-select-schema)" width="100%" />
<p class="caption">
(#fig:F0-line-select-schema)(ref:F0-line-select-schema)
</p>
</div>
</div>
<div id="direct-genetic-effects-1" class="section level2" number="5.6">
<h2><span class="header-section-number">5.6</span> Direct genetic effects</h2>
<p>With these lines selected, I ran a similar analysis to what I described in @ref(Pilot-chap), where I ran multi-way ANOVAs to determine whether certain lines differed in the proportions of time they spent in these HMM states, while including the date of assay, time of assay, tank quadrant and tank side as covariates. <strong>Table @ref(tab:mikk-dge-F0)</strong> sets out the states which showed a significant difference between these 6 lines, with p-values adjusted for the False Discovery Rate (<strong>FDR</strong>).</p>
<table>
<caption>(#tab:mikk-dge-F0)Significant differences in the proportion of time spent in each HMM state across test fish lines selected for the F2 cross for the open field and novel object assay components.</caption>
<thead>
<tr class="header">
<th align="left">Assay</th>
<th align="left">State</th>
<th align="left">Variance explained (%)</th>
<th align="left">p-value (FDR-adjusted)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">open field</td>
<td align="left">3</td>
<td align="left">21.53</td>
<td align="left">2.67e-03</td>
</tr>
<tr class="even">
<td align="left">open field</td>
<td align="left">4</td>
<td align="left">23.28</td>
<td align="left">1.60e-02</td>
</tr>
<tr class="odd">
<td align="left">open field</td>
<td align="left">5</td>
<td align="left">20.61</td>
<td align="left">3.15e-02</td>
</tr>
<tr class="even">
<td align="left">open field</td>
<td align="left">10</td>
<td align="left">29.06</td>
<td align="left">1.41e-03</td>
</tr>
<tr class="odd">
<td align="left">open field</td>
<td align="left">12</td>
<td align="left">24.91</td>
<td align="left">1.45e-02</td>
</tr>
<tr class="even">
<td align="left">open field</td>
<td align="left">14</td>
<td align="left">29.52</td>
<td align="left">1.29e-03</td>
</tr>
<tr class="odd">
<td align="left">novel object</td>
<td align="left">1</td>
<td align="left">16.21</td>
<td align="left">1.90e-02</td>
</tr>
<tr class="even">
<td align="left">novel object</td>
<td align="left">2</td>
<td align="left">15.14</td>
<td align="left">4.86e-02</td>
</tr>
<tr class="odd">
<td align="left">novel object</td>
<td align="left">4</td>
<td align="left">26.90</td>
<td align="left">5.21e-03</td>
</tr>
<tr class="even">
<td align="left">novel object</td>
<td align="left">5</td>
<td align="left">27.84</td>
<td align="left">4.27e-03</td>
</tr>
<tr class="odd">
<td align="left">novel object</td>
<td align="left">10</td>
<td align="left">23.73</td>
<td align="left">2.81e-02</td>
</tr>
</tbody>
</table>
<p><strong>Figures @ref(fig:F2-time-dge-of)</strong> and <strong>@ref(fig:F2-time-dge-no)</strong> highlight the states that showed significant differences in the proportions of time that these lines spent in those states for the open field and novel object assay components respectively. In both figures, <strong>A</strong> highlights the significant states, <strong>B</strong> shows how the individuals moved through those states over the course of the 10-minute assay component, and <strong>C</strong> shows the densities of the significant states within each line. For the open field component, although the tile plots show a notable level of variance within each line, the density plots clarify how the three slow-moving lines (8-2, 18-2 and 50-2) spent little time in the fast-moving states 10, 12 and 14 relative to the fast-moving lines. Interestingly, the intermediate line 38-2 tended to transition into these states around the middle of the video, presumably a period of habituating to the new environment.</p>
<p>(ref:F2-time-dge-of) Differences between MIKK F0 lines in the HMM states they occupied during the open field assay component. <strong>A</strong>: 15 HMM states with panels coloured red to indicate significant differences between MIKK F0 lines in the proportion of time spent in those states. <strong>B</strong>: Transitions between HMM states across time for each individual test fish, grouped by MIKK line Tiles are coloured by the state most frequently occupied by each fish within 2-second intervals. <strong>C</strong>: Densities within each MIKK line for the occupation of states that significantly differed between strains (colour), with other states consolidated (grey).</p>
<div class="figure">
<img src="figs/mikk_behaviour/select_0.08_15_dge_of.png" alt="(ref:F2-time-dge-of)" width="100%" />
<p class="caption">
(#fig:F2-time-dge-of)(ref:F2-time-dge-of)
</p>
</div>
<p>During the novel object component the HMM states that showed significant differences between lines were mostly restricted to the slow-moving states with the exception of state 10, which most clearly distinguishes the slow-moving lines from the intermediate- and fast-moving lines. Again, the intermediate line <span style="color: #00C08B;">38-2</span> shows a sharp drop in the occupation of the slow moving states after a period of habituation.</p>
<p>(ref:F2-time-dge-no) Differences between MIKK F0 lines in the HMM states they occupied during the novel object assay component. <strong>A</strong>: 15 HMM states with panels coloured red to indicate significant differences between MIKK F0 lines in the proportion of time spent in those states. <strong>B</strong>: Transitions between HMM states across time for each individual test fish, grouped by MIKK line Tiles are coloured by the state most frequently occupied by each fish within 2-second intervals. <strong>C</strong>: Densities within each MIKK line for the occupation of states that significantly differed between strains (colour), with other states consolidated (grey).</p>
<div class="figure">
<img src="figs/mikk_behaviour/select_0.08_15_dge_no.png" alt="(ref:F2-time-dge-no)" width="100%" />
<p class="caption">
(#fig:F2-time-dge-no)(ref:F2-time-dge-no)
</p>
</div>
<div id="social-genetic-effects-2" class="section level3" number="5.6.1">
<h3><span class="header-section-number">5.6.1</span> Social genetic effects</h3>
<p>To confirm whether the <em><span style="color: #424B4D;">iCab</span></em> reference fishes altered their behaviour depending on the MIKK F0 line of their tank partner, we carried out the same analysis and model as above using only data from the <em><span style="color: #424B4D;">iCab</span></em> reference fishes. The states that showed a significant difference across any of the variables included in the model are set out in <strong>Table @ref(tab:mikk-sge-F0)</strong>. The iCab reference fishes differed significantly in the proportion of time they spent in a given state depending on the strain of their tank partner (<span class="math inline">\(p\)</span> &lt; 0.05, FDR-adjusted) for 4 out of 15 states in the open field assay (1.26x10<sup>-2</sup> &lt; <span class="math inline">\(p\)</span> &lt; 4x10<sup>-2</sup>), and 4 out of 15 states for the novel object assay (7.71x10<sup>-3</sup> &lt; <span class="math inline">\(p\)</span> &lt; 1.34x10<sup>-2</sup>). The line of the tank partner explained up to ~29% of the variance in the proportion of time the <em><span style="color: #424B4D;">iCab</span></em> reference spent in a given state. Full tables for all states and variables are provided in the Supplementary Material.</p>
<table>
<caption>(#tab:mikk-sge-F0)Significant differences in the proportion of time spent in each HMM state by iCab reference fishes depending on the MIKK F0 line of their tank partner during the open field and novel object assay components.</caption>
<thead>
<tr class="header">
<th align="left">Assay</th>
<th align="left">State</th>
<th align="left">Variance explained (%)</th>
<th align="left">p-value (FDR-adjusted)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">open field</td>
<td align="left">3</td>
<td align="left">14.93</td>
<td align="left">2.10e-02</td>
</tr>
<tr class="even">
<td align="left">open field</td>
<td align="left">4</td>
<td align="left">13.60</td>
<td align="left">2.28e-02</td>
</tr>
<tr class="odd">
<td align="left">open field</td>
<td align="left">5</td>
<td align="left">15.33</td>
<td align="left">1.26e-02</td>
</tr>
<tr class="even">
<td align="left">open field</td>
<td align="left">7</td>
<td align="left">14.26</td>
<td align="left">4.00e-02</td>
</tr>
<tr class="odd">
<td align="left">novel object</td>
<td align="left">4</td>
<td align="left">14.33</td>
<td align="left">1.23e-02</td>
</tr>
<tr class="even">
<td align="left">novel object</td>
<td align="left">5</td>
<td align="left">14.38</td>
<td align="left">1.15e-02</td>
</tr>
<tr class="odd">
<td align="left">novel object</td>
<td align="left">9</td>
<td align="left">24.61</td>
<td align="left">1.34e-02</td>
</tr>
<tr class="even">
<td align="left">novel object</td>
<td align="left">11</td>
<td align="left">28.77</td>
<td align="left">7.71e-03</td>
</tr>
</tbody>
</table>
<p>(ref:F2-time-sge-of) Differences between HMM states occupied by the reference fish when paired with different MIKK F0 lines during the open field assay component. <strong>A</strong>: 15 HMM states with panels coloured red to indicate significant differences between the reference fishes under different line-pairings in the proportion of time spent in those states. The HMM states are the same as those in <strong>Figure @ref(fig:F2-time-dge-of)</strong> above, but coloured with a different palette. <strong>B</strong>: Transitions between HMM states across time for each individual <em><span style="color: #424B4D;">iCab</span></em> reference fish, grouped by the strain of its tank partner. Tiles are coloured by the state most frequently occupied by each fish within 2-second intervals. <strong>C</strong>: Densities within each strain-pairing for the occupation of states that significantly differed between strain-pairings (colour), with other states consolidated (grey).</p>
<div class="figure">
<img src="figs/mikk_behaviour/select_0.08_15_sge_of.png" alt="(ref:F2-time-dge-of)" width="100%" />
<p class="caption">
(#fig:F2-time-sge-of)(ref:F2-time-dge-of)
</p>
</div>
<p>(ref:F2-time-sge-no) Same layout as <strong>Figure @ref(fig:F2-time-sge-of)</strong> above, but for the novel object assay.</p>
<div class="figure">
<img src="figs/mikk_behaviour/select_0.08_15_sge_no.png" alt="(ref:F2-time-dge-no)" width="100%" />
<p class="caption">
(#fig:F2-time-sge-no)(ref:F2-time-dge-no)
</p>
</div>
<p>To assist with interpretation of the relationships between the HMM states and the types of behaviours exhibited, I have provided in <strong>Appendix @ref(four-panel-app)</strong> four-panel plots showing a) a frame grab from the raw four-quadrant video; b) path plots for both fishes; c) path plots for the test fishes coloured by HMM state; and d) path plots for the reference fishes coloured by HMM state, all taken at the 300-second mark (5 minutes) of each 10-minute assay component.</p>
</div>
</div>
<div id="f2-generation" class="section level2" number="5.7">
<h2><span class="header-section-number">5.7</span> F2 generation</h2>
<div id="behavioural-data-collection" class="section level3" number="5.7.1">
<h3><span class="header-section-number">5.7.1</span> Behavioural data collection</h3>
<p>Around August 2019, our collaborators in the Loosli Group at KIT commenced the breeding program for this experiment with the 6 MIKK panel lines I had selected above.<a href="#fn275" class="footnote-ref" id="fnref275"><sup>275</sup></a> From 17 November 2021 to 5 May 2022, a Research Assistant in Prof. Loosli’s lab, Alicia Günthel, performed the assay 69 time with F2 individuals from the 12 crosses they had bred, producing a total of 271 videos of pairs of fish. The counts for the 12 crosses used to generate these 271 test fishes are set out in <strong>Table @ref(tab:F2-cross-counts)</strong>.</p>
<table>
<caption>(#tab:F2-cross-counts)Significant differences in the proportion of time spent in each HMM state by iCab reference fishes depending on the MIKK F0 line of their tank partner during the open field and novel object assay components.</caption>
<thead>
<tr class="header">
<th align="left">paternal line</th>
<th align="left">maternal line</th>
<th align="right">count</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">21-2</td>
<td align="left">40-1</td>
<td align="right">60</td>
</tr>
<tr class="even">
<td align="left">38-2</td>
<td align="left">40-1</td>
<td align="right">57</td>
</tr>
<tr class="odd">
<td align="left">38-2</td>
<td align="left">18-2</td>
<td align="right">35</td>
</tr>
<tr class="even">
<td align="left">8-2</td>
<td align="left">40-1</td>
<td align="right">24</td>
</tr>
<tr class="odd">
<td align="left">50-2</td>
<td align="left">18-2</td>
<td align="right">23</td>
</tr>
<tr class="even">
<td align="left">38-2</td>
<td align="left">21-2</td>
<td align="right">19</td>
</tr>
<tr class="odd">
<td align="left">8-2</td>
<td align="left">38-2</td>
<td align="right">15</td>
</tr>
<tr class="even">
<td align="left">50-2</td>
<td align="left">38-2</td>
<td align="right">12</td>
</tr>
<tr class="odd">
<td align="left">18-2</td>
<td align="left">21-2</td>
<td align="right">7</td>
</tr>
<tr class="even">
<td align="left">21-2</td>
<td align="left">50-2</td>
<td align="right">7</td>
</tr>
<tr class="odd">
<td align="left">40-1</td>
<td align="left">50-2</td>
<td align="right">6</td>
</tr>
<tr class="even">
<td align="left">50-2</td>
<td align="left">8-2</td>
<td align="right">6</td>
</tr>
</tbody>
</table>
<p>I again used <em>idtrackerai</em><span class="citation"><a href="#fn276" class="footnote-ref" id="fnref276"><sup>276</sup></a></span> to track the F2 individuals across frames. When splitting the 271 videos of pairs of fish into their open field and novel object assay components for a total of 542 videos, for 526 of them both fish were tracked across at least 85% of frames. I only used these 526 videos in the downstream analysis, but prior to publication I will seek to improve the tracking process so that these individuals can be included.</p>
</div>
<div id="behavioural-phenotyping" class="section level3" number="5.7.2">
<h3><span class="header-section-number">5.7.2</span> Behavioural phenotyping</h3>
<p>To ensure that the predicted HMM states for behaviour were consistent across both F0 and F2 generations, I had included these F2 individuals for training and prediction in the models described above. I again calculated the proportions of time that every individual spent in each state (“<strong>state frequency</strong>”), then inverse-normalised the values within each combination of assay component (open field or novel object) and state (1 to 15). Inverse-normalisation is a rank-based normalisation approach which involves replacing the values in the phenotype vector with their rank (where ties are averaged), then converting the ranks into a normal distribution with the quantile function.<span class="citation"><a href="#fn277" class="footnote-ref" id="fnref277"><sup>277</sup></a></span> The inverse-normalisation function I used for this analysis is set out in the following <code>R</code> code:</p>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a>invnorm <span class="ot">=</span> <span class="cf">function</span>(x) {</span>
<span id="cb1-2"><a href="#cb1-2" aria-hidden="true" tabindex="-1"></a>  res <span class="ot">=</span> <span class="fu">rank</span>(x)</span>
<span id="cb1-3"><a href="#cb1-3" aria-hidden="true" tabindex="-1"></a>  <span class="co"># The arbitrary 0.5 value is added to the denominator below </span></span>
<span id="cb1-4"><a href="#cb1-4" aria-hidden="true" tabindex="-1"></a>  <span class="co"># to avoid `qnorm()` returning &#39;Inf&#39; for the last-ranked value</span></span>
<span id="cb1-5"><a href="#cb1-5" aria-hidden="true" tabindex="-1"></a>  res <span class="ot">=</span> <span class="fu">qnorm</span>(res<span class="sc">/</span>(<span class="fu">length</span>(res)<span class="sc">+</span><span class="fl">0.5</span>))</span>
<span id="cb1-6"><a href="#cb1-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">return</span>(res)</span>
<span id="cb1-7"><a href="#cb1-7" aria-hidden="true" tabindex="-1"></a>}</span></code></pre></div>
<p><strong>Figures @ref(fig:F2-state-freq-dge)</strong> and <strong>@ref(fig:F2-state-freq-sge)</strong> compares the phenotype pre- and post-transformation with this normalisation approach. For the higher-movement states there are increasing numbers of individuals who spent no time in those states, which are responsible for the apparently non-normal distributions observed for the skewed distributions post-transformation. States 1, 3, 6, 8, 9 and 11 already appear to have a large amount of variation between individuals, but this normalisation process will increase the variance for states where there is small, yet potentially meaningful, variation between individuals. One exception may perhaps be state 15, which involves very large distances of travel in all directions, and therefore may correspond to tracking errors.</p>
<p>(ref:F2-state-freq-dge) Effect of inverse-normalisation on the HMM state frequency of the F2 test fishes.</p>
<div class="figure">
<img src="figs/mikk_behaviour/0.08_15_state_freq_F2_dge.png" alt="(ref:F2-state-freq-dge)" width="100%" />
<p class="caption">
(#fig:F2-state-freq-dge)(ref:F2-state-freq-dge)
</p>
</div>
<p>(ref:F2-state-freq-sge) Effect of inverse-normalisation on the HMM state frequency of the F2 <em><span style="color: #424B4D;">iCab</span></em> reference fishes.</p>
<div class="figure">
<img src="figs/mikk_behaviour/0.08_15_state_freq_F2_sge.png" alt="(ref:F2-state-freq-sge)" width="100%" />
<p class="caption">
(#fig:F2-state-freq-sge)(ref:F2-state-freq-sge)
</p>
</div>
</div>
<div id="genotyping" class="section level3" number="5.7.3">
<h3><span class="header-section-number">5.7.3</span> Genotyping</h3>
<p>After phenotyping the F2 samples, our collaborators in the Loosli Group at KIT took finclips from the F2 individuals, extracted their DNA, and arranged for them to be shallow-sequenced on the Illumina short-read platform at a coverage of ~1x per sample. Using a similar method to what I described in Chapter @ref(Somite-chap), I aligned these sequences to the <em>HdrR</em> reference with BWA-MEM2,<span class="citation"><a href="#fn278" class="footnote-ref" id="fnref278"><sup>278</sup></a></span> sorted the reads and marked duplicates with Picard,<span class="citation"><a href="#fn279" class="footnote-ref" id="fnref279"><sup>279</sup></a></span> then indexed the resulting BAM files with samtools.<span class="citation"><a href="#fn280" class="footnote-ref" id="fnref280"><sup>280</sup></a></span> I then used <em>bam-readcount</em><span class="citation"><a href="#fn281" class="footnote-ref" id="fnref281"><sup>281</sup></a></span> to count the reads that supported either the paternal or the maternal allele for all biallelic SNPs that were homozygous-divergent in the given sample’s two parental strains (i.e. homozgyous reference in the paternal line, and homozygous alternative in the maternal line, or <em>vice versa</em>), summed the read counts within 5 kb blocks, and calculated the frequency of reads within each bin that supported the maternal allele. This generated a value for each bin between 0 and 1, where 0 signified that all reads within that bin supported the paternal allele, and 1 signified that all reads within that bin supported the maternal allele. Bins containing no reads were imputed with a value of 0.5.</p>
<p>I then used these values for all F2 individuals as the input to a Hidden Markov Model (HMM) with the software package <em>hmmlearn</em>,<span class="citation"><a href="#fn282" class="footnote-ref" id="fnref282"><sup>282</sup></a></span> which I applied to classify each bin as one of three states, with state 0 corresponding to homozygous for the paternal allele, 1 corresponding to heterozygous, and 2 corresponding to homozygous for the maternal allele. Across each chromosome of every sample, the output of the HMM was expected to produce a sequence of states. Based on previous analyses described in Chapter @ref(Somite-chap), I used the same HMM parameters as I did there, and used the HMM state outputs to generate the recombination blocks shown in <strong>Figure @ref(fig:F2-recomb-blocks)</strong>. Missing genotype state calls arose from a sample having insufficient reads within a bin, for which I imputed the missing state calls within each sample’s chromosome based on the previous state call on that chromosome. A karyoplot retaining the missing blocks is provided in <strong>Appendix @ref(fig:F2-recomb-blocks-missing)</strong>.</p>
<p>(ref:F2-recomb-blocks) Karyoplot for F2 samples, coloured by genotype. Samples are sorted in the order in which they were phenotyped. Blocks are filled with the colour of the paternal F0 line for the homozygous paternal haplotype block, black for heterozygous, and the colour of the maternal F0 line for the homozygous maternal haplotype block. Missing calls were imputed based on the previous successful call on a given chromosome.</p>
<div class="figure">
<img src="figs/mikk_behaviour/karyoplot_no_missing.png" alt="(ref:F2-recomb-blocks)" width="100%" />
<p class="caption">
(#fig:F2-recomb-blocks)(ref:F2-recomb-blocks)
</p>
</div>
<p>I then took these HMM-generated haplotype block calls and used them to impute each sample’s SNP-level genotypes using the homozygous biallelic SNP calls in the high-coverage .vcf file for the MIKK panel F0 lines described in Chapter @ref(MIKK-genomes-chap). This set of variants included a total of ~20.7M SNPs, which I used to generate a Plink-format .bed file, forming the genotype input for the genetic linkage analysis.</p>
</div>
<div id="genetic-linkage-analysis" class="section level3" number="5.7.4">
<h3><span class="header-section-number">5.7.4</span> Genetic linkage analysis</h3>
<p>For the purpose of using the <em>GCTA</em> software package<span class="citation"><a href="#fn283" class="footnote-ref" id="fnref283"><sup>283</sup></a></span> for the genetic linkage analysis, That software requires the construction of a genetic relationship matrix (<strong>GRM</strong>) <span class="math inline">\(\textbf{A} = \textbf{WW}&#39;/N\)</span>. <span class="math inline">\(\textbf{W}\)</span> is a standardised genotype matrix with the <span class="math inline">\(ij^{th}\)</span> element <span class="math inline">\(w_{ij} = (x_{ij} - 2p_i) / \sqrt{(2p_i(1-p_i)}\)</span>, where <span class="math inline">\(x_{ij}\)</span> is the number of copies of the reference allele for the <span class="math inline">\(i^{th}\)</span> SNP of the <span class="math inline">\(j^{th}\)</span> individual and <span class="math inline">\(p_i\)</span> is the frequency of the reference allele.<span class="citation"><a href="#fn284" class="footnote-ref" id="fnref284"><sup>284</sup></a></span></p>
<p>For the GRM, I first filtered the .bed for SNPs that had no missing calls for any samples (<span class="math inline">\(M_{SNPs}\)</span> = 44,360). I then used these SNPs to construct a “leave-one-chromosome-out” (<strong>LOCO</strong>) genetic relationship matrix for each chromosome – that is, if the “focal” chromosome was chr1, I would exclude the SNPs on that chromosome before constructing the GRM. To illustrate, <strong>Figure @ref(fig:F2-grm)</strong> is a GRM constructed using all 44,360 non-missing SNPs. However, given the relatively small amount of non-missing SNPs on each chromosome, the number of SNPs on the focal chromosome that were excluded was small, resulting in GRMs that appear almost identical by eye – see the LOCO-GRM for chromosome 1 in <strong>Appendix @ref(fig:loco-grm-chr1)</strong>. The individuals from each cross neatly cluster together, and the individuals that share one parental strain cluster nearby.</p>
<p>(ref:F2-grm) Genetic relationship matrix for 271 F2 samples based on 44,360 non-missing SNPs.</p>
<div class="figure">
<img src="figs/mikk_behaviour/grm_man_0.8.png" alt="(ref:F2-grm)" width="100%" />
<p class="caption">
(#fig:F2-grm)(ref:F2-grm)
</p>
</div>
<p>I used these GRMs in the mixed linear model based association analysis (<strong>MLMA</strong>) implemented in GCTA,<span class="citation"><a href="#fn285" class="footnote-ref" id="fnref285"><sup>285</sup></a></span> where the model was generally specified as follows:</p>
<p><span class="math display">\[
y = a + bx + g + e
\]</span></p>
<p><span class="math inline">\(y\)</span> was the phenotype, <span class="math inline">\(a\)</span> was the mean term, <span class="math inline">\(b\)</span> was the additive effect (fixed effect) of the candidate SNP tested for association, <span class="math inline">\(x\)</span> was the SNP genotype indicator variable coded as 0, 1 or 2, <span class="math inline">\(g\)</span> was the polygenic effect (random effect) i.e. the accumulated effect of all SNPs (as captured by the GRM calculated using all SNPs) and <span class="math inline">\(e\)</span> was the residual.<span class="citation"><a href="#fn286" class="footnote-ref" id="fnref286"><sup>286</sup></a></span> For <span class="math inline">\(y\)</span>, I used the inverse-normalised state frequencies described above, and the LOCO-GRM as <span class="math inline">\(g\)</span> for all SNPs on the focal chromosome. For example, for all SNPs on chr18, I used the LOCO-GRM that excluded all SNPs from that chromosome. In addition, I included the time of assay and the tank quadrant as covariates, which the software regresses out from the phenotype prior to running the MLMA. I excluded the date of assay and the tank side as covariates because individuals from the same cross tended to be assayed in the same test tank on the same day, and are therefore confounded with their genetics.</p>
<p>To set a significance threshold, for each combination of HMM state and assay, I ran 10 MLMA tests over the same dataset where I had permuted the phenotype and covariates using a different random seed. The logic behind this method is to determine the lowest p-value that one could expect when there is no true relationship between the individuals’ genetics and their phenotype. I then extracted the smallest p-value from all 10 results, and used this as the significance threshold. I additionally calculated the Bonferroni threshold as <span class="math inline">\(\alpha / M\)</span>, where <span class="math inline">\(\alpha\)</span> is set to 0.05 and <span class="math inline">\(M\)</span> is the total number of SNPs in the dataset (2,726,797) = 1.83x10<sup>-8</sup>.</p>
<div id="direct-genetic-effects-2" class="section level4" number="5.7.4.1">
<h4><span class="header-section-number">5.7.4.1</span> Direct genetic effects</h4>
<p>For the DGE phenotypes (comparing the state frequencies across test fishes), I sought to identify the genetic loci in the F2 individuals that were associated with differences in their own behaviour. For neither the open field nor novel object component did the p-values exceed the Bonferroni threshold, however a number of loci across several chromosomes exceeded the thresholds set by the permutations. I plotted the p-values in Manhattan plots for each combination of state and assay component, and provide them all in the Appendix. Here I showcase the Manhattan plot for state 3, containing a number of significant loci that I discuss further below.</p>
<p>(ref:F2-man-dge-no-3) Manhattan plot for inverse-normalised HMM state 3 frequency of F2 test fishes during the novel object assay.</p>
<div class="figure">
<img src="figs/mikk_behaviour/manhattans/dge_no_3_time-quadrant.png" alt="(ref:F2-man-dge-no-3)" width="100%" />
<p class="caption">
(#fig:F2-man-dge-no-3)(ref:F2-man-dge-no-3)
</p>
</div>
</div>
<div id="social-genetic-effects-3" class="section level4" number="5.7.4.2">
<h4><span class="header-section-number">5.7.4.2</span> Social genetic effects</h4>
<p>For social genetic effects, I was attempting to discover genetic variants in the F2 individuals that caused differences in the behaviour of their <em><span style="color: #424B4D;">iCab</span></em> tank partners. As expected, fewer loci reached significance for this transmitted indirect genetic effect than for the direct genetic effects, however I still found several significant loci based on the permutations threshold for states 4 (chr4), 7 (chr1), 12 (chr11), and 13 (chr12) during the open field assay component; but only one, barely significant SNP for state 5 (chr 13) during the novel object assay. This difference was somewhat surprising given our previous suppositions (DISCUSS) that the novel object component drew out stronger social genetic effects.</p>
<p>(ref:F2-man-sge-of-13) Manhattan plot for inverse-normalised HMM state 3 frequency of <em><span style="color: #424B4D;">iCab</span></em> reference fishes during the novel object assay component.</p>
<div class="figure">
<img src="figs/mikk_behaviour/manhattans/sge_of_13_time-quadrant.png" alt="(ref:F2-man-sge-of-13)" width="100%" />
<p class="caption">
(#fig:F2-man-sge-of-13)(ref:F2-man-sge-of-13)
</p>
</div>
</div>
</div>
<div id="candidate-snps-for-crispr-knockouts" class="section level3" number="5.7.5">
<h3><span class="header-section-number">5.7.5</span> Candidate SNPs for CRISPR-knockouts</h3>
<p>The next goal was to attempt to identify the SNPs that are mostly likely to be the causal variants, or closest in proximity to the causal variants, responsible for differences in the phenotypes of interest. I therefore proceeded to extract the significant SNPs and use Ensembl’s Variant Effect Predictor (<strong>VEP</strong>)<span class="citation"><a href="#fn287" class="footnote-ref" id="fnref287"><sup>287</sup></a></span> to identify the SNPs that are most likely to have a functional consequence, and determine whether the genes they reside in have already been identified as being involved in biological pathways that could be related to behaviour. <strong>Table @ref(tab:F2-sig-snps-consq-counts)</strong> shows the counts for the different potential consequences of the unique significant SNPs, based on Ensembl’s annotation of the <em>HdrR</em> reference. I note that as multiple genes can overlap the same locus (including those transcribed from opposing strands), one SNP may map to multiple genes and therefore be counted as having more than one type of consequence.</p>
<table>
<caption>(#tab:F2-sig-snps-consq-counts)Counts for consequences of significant SNPs.</caption>
<thead>
<tr class="header">
<th align="left">consequence</th>
<th align="right">count</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">intron_variant</td>
<td align="right">349</td>
</tr>
<tr class="even">
<td align="left">intergenic_variant</td>
<td align="right">178</td>
</tr>
<tr class="odd">
<td align="left">upstream_gene_variant</td>
<td align="right">92</td>
</tr>
<tr class="even">
<td align="left">downstream_gene_variant</td>
<td align="right">75</td>
</tr>
<tr class="odd">
<td align="left">synonymous_variant</td>
<td align="right">9</td>
</tr>
<tr class="even">
<td align="left">3_prime_UTR_variant</td>
<td align="right">6</td>
</tr>
<tr class="odd">
<td align="left">missense_variant</td>
<td align="right">6</td>
</tr>
<tr class="even">
<td align="left">5_prime_UTR_variant</td>
<td align="right">3</td>
</tr>
<tr class="odd">
<td align="left">splice_acceptor_variant</td>
<td align="right">2</td>
</tr>
<tr class="even">
<td align="left">splice_region_variant,intron_variant</td>
<td align="right">2</td>
</tr>
<tr class="odd">
<td align="left">splice_region_variant,synonymous_variant</td>
<td align="right">2</td>
</tr>
<tr class="even">
<td align="left">splice_donor_variant</td>
<td align="right">1</td>
</tr>
</tbody>
</table>
<p>As expected, most variants reside in non-coding regions [CITE], but in an attempt to isolate the most likely causative SNPs, I extracted those that are predicted to cause a missense mutation and therefore most likely to have a functional consequence (<strong>Table @ref(tab:F2-sig-snps-missense)</strong>).</p>
<table>
<caption>(#tab:F2-sig-snps-missense)Missense SNPs.</caption>
<colgroup>
<col width="12%" />
<col width="10%" />
<col width="4%" />
<col width="3%" />
<col width="7%" />
<col width="3%" />
<col width="3%" />
<col width="5%" />
<col width="15%" />
<col width="33%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">Genetic effect</th>
<th align="left">Assay</th>
<th align="right">State</th>
<th align="right">Chr</th>
<th align="right">Pos</th>
<th align="left">Ref</th>
<th align="left">Alt</th>
<th align="left">Allele</th>
<th align="left">Gene</th>
<th align="left">Description</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">DGE</td>
<td align="left">open field</td>
<td align="right">1</td>
<td align="right">1</td>
<td align="right">22508288</td>
<td align="left">A</td>
<td align="left">C</td>
<td align="left">C</td>
<td align="left">ENSORLG00000009385</td>
<td align="left">vav guanine nucleotide exchange factor 1</td>
</tr>
<tr class="even">
<td align="left">DGE</td>
<td align="left">open field</td>
<td align="right">7</td>
<td align="right">13</td>
<td align="right">33714439</td>
<td align="left">G</td>
<td align="left">T</td>
<td align="left">T</td>
<td align="left">ENSORLG00000028050</td>
<td align="left">uncharacterized LOC105356481</td>
</tr>
<tr class="odd">
<td align="left">DGE</td>
<td align="left">open field</td>
<td align="right">10</td>
<td align="right">3</td>
<td align="right">28342550</td>
<td align="left">A</td>
<td align="left">G</td>
<td align="left">G</td>
<td align="left">ENSORLG00000024879</td>
<td align="left"></td>
</tr>
<tr class="even">
<td align="left">DGE</td>
<td align="left">novel object</td>
<td align="right">3</td>
<td align="right">1</td>
<td align="right">31428698</td>
<td align="left">G</td>
<td align="left">A</td>
<td align="left">A</td>
<td align="left">ENSORLG00000013962</td>
<td align="left">ecto-ADP-ribosyltransferase 5-like</td>
</tr>
<tr class="odd">
<td align="left">DGE</td>
<td align="left">novel object</td>
<td align="right">3</td>
<td align="right">10</td>
<td align="right">15319434</td>
<td align="left">C</td>
<td align="left">T</td>
<td align="left">T</td>
<td align="left">ENSORLG00000029574</td>
<td align="left"></td>
</tr>
<tr class="even">
<td align="left">DGE</td>
<td align="left">novel object</td>
<td align="right">3</td>
<td align="right">10</td>
<td align="right">15319434</td>
<td align="left">C</td>
<td align="left">T</td>
<td align="left">T</td>
<td align="left">ENSORLG00000024866</td>
<td align="left">protocadherin alpha-C2-like</td>
</tr>
<tr class="odd">
<td align="left">DGE</td>
<td align="left">novel object</td>
<td align="right">10</td>
<td align="right">10</td>
<td align="right">15319434</td>
<td align="left">C</td>
<td align="left">T</td>
<td align="left">T</td>
<td align="left">ENSORLG00000029574</td>
<td align="left"></td>
</tr>
<tr class="even">
<td align="left">DGE</td>
<td align="left">novel object</td>
<td align="right">10</td>
<td align="right">10</td>
<td align="right">15319434</td>
<td align="left">C</td>
<td align="left">T</td>
<td align="left">T</td>
<td align="left">ENSORLG00000024866</td>
<td align="left">protocadherin alpha-C2-like</td>
</tr>
<tr class="odd">
<td align="left">SGE</td>
<td align="left">open field</td>
<td align="right">13</td>
<td align="right">12</td>
<td align="right">5136828</td>
<td align="left">G</td>
<td align="left">A</td>
<td align="left">A</td>
<td align="left">ENSORLG00000002961</td>
<td align="left">complement C9</td>
</tr>
</tbody>
</table>
<p>One SNP, 10:15,319,434, was significant for DGE for frequencies in both state 3 (<strong>Figure @ref(fig:sig-snp-10-15mb)</strong>) and 10 during the novel object assay (see <strong>Figure @ref(fig:F2-man-dge-no-3)</strong> above for state 3), and maps to two genes: ENSORLG00000029574 and ENSORLG00000024866. ENSORLG00000029574 is a novel gene on the forward strand with no recorded phenotypes. However, ENSORLG00000024866 is a gene on the reverse strand for protocadherin alpha-C2-like, a well-known protein involved in mammalian synapse formation.<span class="citation"><a href="#fn288" class="footnote-ref" id="fnref288"><sup>288</sup></a></span></p>
<p>(ref:sig-snp-10-15mb) State 3 frequency during the novel object assay for counts of the alternative allele (T) at SNP 10:15,319,434. The boxes are coloured on the left by the paternal line, and on the right by the maternal line.</p>
<div class="figure">
<img src="figs/mikk_behaviour/sig_snps_boxplots/3-10:15319434.png" alt="(ref:sig-snp-10-15mb)" width="100%" />
<p class="caption">
(#fig:sig-snp-10-15mb)(ref:sig-snp-10-15mb)
</p>
</div>
<p>The second predicted missense variant of note was detected for SGE in the open field assay for frequencies in state 13. The locus 12:5,136,828 maps to gene ENSORLG00000002961, which is described as a complement <em>C9</em>. Discovered phenotypes for species orthologues include reduced prepulse inhibition and heart phenotypes in mice,<span class="citation"><a href="#fn289" class="footnote-ref" id="fnref289"><sup>289</sup></a></span> and neurodevelopmental disorders in rats.<span class="citation"><a href="#fn290" class="footnote-ref" id="fnref290"><sup>290</sup></a></span> Here, in the F2 individuals, it appears to have an inconsistent effect in the behaviour of the test fishes, but apparently causes the reference fish to increase the proportion of time it spends in the fast-moving state 13 (<strong>Figure @ref(fig:F2-man-sge-of-13)</strong>).</p>
<p>(ref:sig-snp-12-5mb) State 13 frequency during the open field assay for counts of the alternative allele (A) at SNP 12:5,136,828. The boxes are coloured on the left by the paternal line, and on the right by the maternal line.</p>
<div class="figure">
<img src="figs/mikk_behaviour/sig_snps_boxplots/13-12:5136828.png" alt="(ref:sig-snp-12-5mb)" width="100%" />
<p class="caption">
(#fig:sig-snp-12-5mb)(ref:sig-snp-12-5mb)
</p>
</div>
<p>There are additional significant loci of note shown in <strong>Figure @ref(fig:F2-man-dge-no-3)</strong> above that are not predicted to alter proteins, but nevertheless appear to affect genes related to neurological development (<strong>Table @ref(tab:tbl-sig-snp-chr-9-10)</strong>).</p>
<table>
<caption>(#tab:tbl-sig-snp-chr-9-10)Significant loci on chromosome 9 for direct genetic effects on state 3 frequency during the novel object assay.</caption>
<colgroup>
<col width="10%" />
<col width="8%" />
<col width="4%" />
<col width="2%" />
<col width="6%" />
<col width="2%" />
<col width="2%" />
<col width="4%" />
<col width="12%" />
<col width="34%" />
<col width="10%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">Genetic effect</th>
<th align="left">Assay</th>
<th align="right">State</th>
<th align="right">Chr</th>
<th align="right">Pos</th>
<th align="left">Ref</th>
<th align="left">Alt</th>
<th align="left">Allele</th>
<th align="left">Gene</th>
<th align="left">Description</th>
<th align="left">Human homologue</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">DGE</td>
<td align="left">novel object</td>
<td align="right">3</td>
<td align="right">9</td>
<td align="right">9802754</td>
<td align="left">C</td>
<td align="left">A</td>
<td align="left">A</td>
<td align="left">ENSORLG00000024663</td>
<td align="left">glutamate ionotropic receptor delta type subunit 2</td>
<td align="left">GRID2</td>
</tr>
<tr class="even">
<td align="left">DGE</td>
<td align="left">novel object</td>
<td align="right">3</td>
<td align="right">10</td>
<td align="right">18537719</td>
<td align="left">C</td>
<td align="left">A</td>
<td align="left">A</td>
<td align="left">ENSORLG00000006464</td>
<td align="left">neuroligin 3a</td>
<td align="left">NLGN3</td>
</tr>
</tbody>
</table>
<p>The first is located around 9:9,802,754 (<strong>Figure @ref(fig:sig-snp-9-9mb)</strong>), and maps to a gene for glutamate ionotropic receptor delta type subunit 2, an orthologue of the human gene <em>GRID2</em>. Deletions in this gene have been found to cause ataxia in humans,<span class="citation"><a href="#fn291" class="footnote-ref" id="fnref291"><sup>291</sup></a></span> and other mutations in this gene have been found to cause various neurological disorders in mice.<span class="citation"><a href="#fn292" class="footnote-ref" id="fnref292"><sup>292</sup></a></span></p>
<p>(ref:sig-snp-9-9mb) State 3 frequency during the novel object assay for counts of the alternative allele (A) at SNP 9:9,802,754. The boxes are coloured on the left by the paternal line, and on the right by the maternal line.</p>
<div class="figure">
<img src="figs/mikk_behaviour/sig_snps_boxplots/3-9:9802754.png" alt="(ref:sig-snp-9-9mb)" width="100%" />
<p class="caption">
(#fig:sig-snp-9-9mb)(ref:sig-snp-9-9mb)
</p>
</div>
<p>The second locus resides around 10:18,537,719 (<strong>Figure @ref(fig:sig-snp-10-18mb)</strong>) within the neuroligin 3 gene, an orthologue of the human gene <em>NLGN3</em>, which has been linked with autism in humans,<span class="citation"><a href="#fn293" class="footnote-ref" id="fnref293"><sup>293</sup></a></span> and various neurological in mice and rats.<span class="citation"><a href="#fn294" class="footnote-ref" id="fnref294"><sup>294</sup></a></span></p>
<p>(ref:sig-snp-10-18mb) State 3 frequency during the novel object assay for counts of the alternative allele (A) at SNP 10:18,537,719. The boxes are coloured on the left by the paternal line, and on the right by the maternal line.</p>
<div class="figure">
<img src="figs/mikk_behaviour/sig_snps_boxplots/3-10:18537719.png" alt="(ref:sig-snp-10-18mb)" width="100%" />
<p class="caption">
(#fig:sig-snp-10-18mb)(ref:sig-snp-10-18mb)
</p>
</div>
<p>These results provide promising evidence that the assay and methods described above can identify genetic loci associated with differences in behaviours - and potentially difference in the transmission of behaviour onto social companions - with functional relevance to humans. However, before seeking to functionally validate these variants, there are a number of steps that we seek to take to refine the analysis and thereby increase our confidence in the variants we ultimately select for validation.</p>
</div>
</div>
<div id="discussion-1" class="section level2" number="5.8">
<h2><span class="header-section-number">5.8</span> Discussion</h2>
<p>With the benefit of hindsight, there are a several aspects of this analysis that I would have performed differently. The first relates to the choices of certain parental lines for the F2 cross, which were too heavily reliant on the aggregate measures of speed and charisma, without due regard to the level of behavioural variance observed within each line. As evident in <strong>Figures @ref(fig:F2-time-dge-of)</strong> and <strong>@ref(fig:F2-time-dge-no)</strong> above, lines <span style="color: #FF699C;">8-2</span> and <span style="color: #49B500;">21-2</span> showed large within-line variances. For <span style="color: #FF699C;">8-2</span> in particular, the differences appear to correspond to the date on which they were assayed - specifically, in the first run the four <span style="color: #FF699C;">8-2</span> individuals showed almost no movement, whereas in the second run they showed much higher levels of movement. The different within-line variances are an interesting phenotype to explore further, as they suggest that different lines show different degrees of sensitivity to environmental changes. However, for this analysis, high within-line variances make it difficult to ascertain the line’s “true” phenotype, and therefore make them unsuitable for selection based on our axes of interest for this study (bold vs shy; high vs low behavioural transmission).</p>
<p>An appropriate substitute for <span style="color: #FF699C;">8-2</span> may be line <span style="color: #FF61CC;">139-4</span>, which had low within-line variance for mean speed, and the highest median state co-occupancy in the novel object assay component. From the analysis of Chapter @ref(Pilot-chap), which concluded after the lines were already selected for the F2 cross, it became apparent that the novel object assay was particularly useful for revealing social genetic effects. I hypothesise that at times of higher stress or predation threat, the fish take more behavioural cues from their tank partners. Line <span style="color: #FF61CC;">139-4</span> therefore would have been a good candidate for the slow-moving, high-charisma line.</p>
<p>Similarly, lines <span style="color: #F57A5F;">13-2</span> or <span style="color: #D39200;">94-1</span> would be preferred substitutes for line <span style="color: #49B500;">21-2</span> as the fast-moving, high charisma line. They are faster than <span style="color: #49B500;">21-2</span>, but unlike the fastest line <span style="color: #F8766D;">10-1</span>, they show a degree of habituation (where most individuals moved slowly at the beginning of the assay before eventually speeding up). They also showed lower within-line variance than <span style="color: #49B500;">21-2</span>, but comparable measures for the SGE measures of state co-occupancy and reference deviation.</p>
<p>One way of avoiding these issues would have been to used a more quantitative, objective approach to selecting the lines. For example, I could have used Mann-Whitney tests to quantify the levels of differences between pairs of lines for a trait of interest (as I did for <strong>Figure @ref(fig:mikk-param-comp)</strong>), and then select the lines that maximised that statistic. Such metrics would still perhaps need to be qualitatively weighted against other relevant traits, such as within-line variance. Nevertheless, my downstream analyses show that despite not having potentially selected the ideal lines for the F2 cross, the differences between them were sufficient to identify genetic loci associated with our phenotypes of interest, even with a relatively small sample size of 271 F2 individuals.</p>
<p>Another aspect of the analysis that could have been improved was to use one or more MIKK panel lines as the reference line instead of <em><span style="color: #424B4D;">iCab</span></em>. As previously mentioned, <em><span style="color: #424B4D;">iCab</span></em> is a relatively fast line, which made it difficult to detect social genetic effects when paired with similarly fast-moving lines. Instead, we could have made use of the behavioural variance across the MIKK panel to choose a reference line that exhibits moderate levels of movement. However, we would not be able to assess the levels of movement across the MIKK panel without first assaying the panel with a single reference strain, and we chose <em><span style="color: #424B4D;">iCab</span></em> because we not only had a good understanding of its behavioural traits based on our findings from Chapter @ref(Pilot-chap), but also because it has high fecundity, which made many replicates available and in turn allowed us to avoid using the same reference fish more than once within 24 hours. Now that we have carried out this analysis, we would be able to select a more appropriate line as the reference in any future analyses.</p>
<p>The other benefit from using a MIKK panel line as the reference fish would be to open up the possibility of exploring social genetic effect interactions. For example, if we used two different MIKK lines as the reference, we could explore how those lines differ in the degree to which they are influenced by their tank partner. To state the issue inversely, it would reveal differences in the extent to which different lines transmit their behaviour onto their (reference) tank partner, depending on both of their genetics. A question for further studies could therefore be framed as, ‘given line <span style="color: #FF66A6;">18-2</span> appears to possess a high level of charisma, is that level consistent when they are paired with different MIKK panel lines, and if it differs across lines, then why?’ Due to the relative onerousness of running this behavioural assay, such experiments would involve a substantial time commitment, but in light of the findings presented here, the investigations could be restricted to a subset of the MIKK panel that display interesting behavioural phenotypes.</p>
</div>
<div id="future-directions" class="section level2" number="5.9">
<h2><span class="header-section-number">5.9</span> Future directions</h2>
<p>Our collaborators are in the process of breeding more F2 individuals to increase the sample size for this analysis. For the subsequent breeding program, I have suggested that we replace line <span style="color: #49B500;">21-2</span> with either <span style="color: #F57A5F;">13-2</span> or <span style="color: #AFA200;">23-1</span>, which both show lower within-line variance in mean speed, but similarly high levels of speed and behavioural transmission (as measured by state co-occupancy and <em><span style="color: #424B4D;">iCab</span></em> reference deviation). Any additional F2 individuals will still need to be phenotyped and sequenced, and the phenotype data collection (i.e. video recording the behavioural assay) can be relatively laborious, so this process will likely take some time.</p>
<p>In the meantime, I have 96 Kiyosu CC individuals whose videos have already been processed, and whose DNA has been shallow-sequenced (~1x). It was not feasible to include that data in this analysis due to time constraints. As they have been allowed to breed freely from the same Kiyosu population as the MIKK panel, they are not strongly homozygous, and they are likely to possess many additional genetic variants that are not represented in the MIKK panel. I am therefore not able to impute their genotypes from high-coverage parental strains, as I did with the F2 individuals. The alignments to the reference are also likely to include many mapping errors due to the lack of sufficient coverage. The additional information they provide will therefore not be as useful as additional F2 individuals, but it should nevertheless improve the power to detect casual variants. The amount of additional information we obtain from the Kiyosu CC individuals will also inform whether we include them in this way in future F2 crosses.</p>
<p>A second issue I plan to address is the failure of F2 reads to align to chromosomes 22 and 23 (see <strong>Figure @ref(fig:F2-recomb-blocks)</strong> above). As the issue occurs across most of the sample, and the coverage does not look skewed for those chromosomes (<strong>Figure @ref(fig:F2-coverage)</strong>), there is likely to a be an error in the bioinformatic pipeline that I constructed.</p>
<p>(ref:F2-coverage) Mean depth of sequencing coverage per chromosome for the 271 F2 samples used in this analysis.</p>
<div class="figure">
<img src="figs/mikk_behaviour/coverage_F2.png" alt="(ref:F2-coverage)" width="100%" />
<p class="caption">
(#fig:F2-coverage)(ref:F2-coverage)
</p>
</div>
<p>[TO DO: homozygous-divergent SNP counts for each cross, across chromosomes]</p>
<p>A third issue is the matter of selecting the optimal HMM parameters for calling the recombination blocks, as described in Chapter @ref(Somite-chap). For the analysis in this present chapter I used the same HMM parameters as I had used there. In that chapter, I used the <span class="math inline">\(p\)</span>-values from the resulting genetic linkage analysis as a rough measure of genotyping quality (assuming that lower <span class="math inline">\(p\)</span>-values were driving by more accurate genotypes), together with a qualitative assessment of how many crossovers I expected to observe in a particular chromosome for a subset of the sample. The issue can be resolved by obtaining “ground truth” genotype calls through high-coverage sequence data for a set of samples that have also been shallow-sequenced. This would allow one to compare the HMM recombination block calls with the ground truth genotypes, and thereby train the model to optimise the accuracy of its predictions for the locations of the recombination sites (using a test dataset to validate the parameters). I expect the improved genotype calls will further improve the power to detect genetic variants.</p>
<p>With respect to the genetic linkage analysis, due to the method I used to calculate the genetic relationship matrix (<strong>GRM</strong>), I was compelled to exclude all SNPs that were missing calls in any of the samples, reducing the number of SNPs to merely ~40K out of the ~20M in the full dataset. Not only does this reduce the accuracy of the GRM as a means of capturing the relative relatedness between samples, but the problem will only be exacerbated with the inclusion of additional samples (as the likelihood of a SNP having at least one sample with a missing call will increase with the number of samples). I will therefore adapt my method to include SNPs with missing calls, which should further increase statistical power.</p>
<p>Finally, I applied a separate genetic linkage association test for the state frequency of each HMM state, where it may be beneficial to apply a method that can combine the information across all states into a single phenotype (if not fewer phenotypes), and then run an association test on that reduced phenotype. There are a number of alternatives to consider, including principal components analysis (PCA) of the state frequencies, or the software MTAG<span class="citation"><a href="#fn295" class="footnote-ref" id="fnref295"><sup>295</sup></a></span> which can combine the summary statistics from multiple genetic association tests. Ultimately, the hope is that by leveraging the information about the frequencies in which the fishes occupy each state, combined across <em>all</em> states, while appropriately taking into account the interdependence of those frequencies (e.g. the more time a fish spends in state 3, the less time it is likely to spend in state 13), it will further strengthen these results.</p>
<p>Aside from modifications designed to increase statistical power, there is also capacity to improve the way that I phenotype the individuals, and in particular how I characterise social genetic effects. In this analysis I have only used two behavioural variables: distance and angle of travel. This behavioural dataset is extremely rich, and myriad variables could be extracted from the videos. Shoaling traits would have special relevance to the study of social genetic effects. For instance, one could extract three variables that would be associated with shoaling behaviours: a) distance between tank partners; b) synchrony in speed; and c) parallelism in direction of travel. These three variables could then be reduced down to two with an HMM (i.e. one state representing “shoaling”, and another representing “not shoaling”). The relative frequencies that the different MIKK lines spend in these states would allow one to rank them by the degree of shoaling behaviour they exhibit, and this would likely be a far more accurate representation of social genetics than what I have shown here.</p>
<!--chapter:end:05-MIKK_F2.Rmd-->
</div>
</div>
<div id="Fst-chap" class="section level1" number="6">
<h1><span class="header-section-number">6</span> Variation in the frequency of trait-associated alleles across global human populations</h1>
<div id="Fst-background" class="section level2" number="6.1">
<h2><span class="header-section-number">6.1</span> Background</h2>
<p>Humans have long sought to use genetic information to predict an individual’s likely value for a given trait, in our own species and in other organisms (Chapter @ref(Introduction)). As seen in previous chapters, an individual’s phenotypic value at a given point in time is the product of complex interactions between their genome and their environment, beginning from embryonic development and continuing throughout their lifetimes. It is now clear that “complex” traits such as height, intelligence, and behaviour are highly polygenic, meaning that they are genetically influenced by hundreds or thousands of genetic variants, each exerting a small effect in one or the other direction along the trait’s spectrum.<span class="citation"><a href="#fn296" class="footnote-ref" id="fnref296"><sup>296</sup></a></span></p>
<p>A richer understanding of the cumulative effect of genetic variants on any trait allows for the prediction of the value that an individual is most likely to have for that trait. Of all human traits, diseases are particularly salient; in 2018, the global healthcare industry was valued at US$8 trillion, and predicted to increase to US$12 trillion by 2022.<span class="citation"><a href="#fn297" class="footnote-ref" id="fnref297"><sup>297</sup></a></span> This strong financial imperative complements the moral imperative to reduce suffering, together driving the question of how to use genetic information to improve human health.</p>
<p>Recent technological developments have made it possible to sequence human genomes at scale, and it is thought that by combining detailed genetic information with with other environmental and phenotypic information (such as lifestyle or clinical factors), clinicians could move towards the practice of “precision medicine”, where interventions could be tailored to their patients’ unique risk profiles.<span class="citation"><a href="#fn298" class="footnote-ref" id="fnref298"><sup>298</sup></a></span> The use of genetic information to predict individuals’ values for a trait of interest entails the construction of metrics known as “polygenic scores” (<strong>PGS</strong>). When the trait is a disease, PGS is commonly known as polygenic risk scores (<strong>PRS</strong>), or genetic risk profiling, but I use the term PGS to encompass both disease and non-disease traits.</p>
<div id="polygenic-scores-pgs-and-genome-wide-association-studies-gwas" class="section level3" number="6.1.1">
<h3><span class="header-section-number">6.1.1</span> Polygenic Scores (PGS) and Genome-Wide Association Studies (GWAS)</h3>
<div id="pgs" class="section level4" number="6.1.1.1">
<h4><span class="header-section-number">6.1.1.1</span> PGS</h4>
<p>PGS using genetic information alone show modest yet reliable accuracy for the prediction of complex traits:<span class="citation"><a href="#fn299" class="footnote-ref" id="fnref299"><sup>299</sup></a></span> the correlations between PGS and the trait value as measured by <span class="math inline">\(R^2\)</span> have reached 0.24 for height,<span class="citation"><a href="#fn300" class="footnote-ref" id="fnref300"><sup>300</sup></a></span> and 0.12-0.16 for educational attainment.<span class="citation"><a href="#fn301" class="footnote-ref" id="fnref301"><sup>301</sup></a></span> PGS also improve predictions beyond non-genetic clinical models for a variety of health-related traits, including breast cancer,<span class="citation"><a href="#fn302" class="footnote-ref" id="fnref302"><sup>302</sup></a></span> prostate cancer,<span class="citation"><a href="#fn303" class="footnote-ref" id="fnref303"><sup>303</sup></a></span> and type I diabetes.<span class="citation"><a href="#fn304" class="footnote-ref" id="fnref304"><sup>304</sup></a></span> The predictive accuracy of PGS scores can be further improved by combining genetic information with lifestyle and clinical factors, as seen with cardiovascular disease.<span class="citation"><a href="#fn305" class="footnote-ref" id="fnref305"><sup>305</sup></a></span></p>
</div>
<div id="gwas" class="section level4" number="6.1.1.2">
<h4><span class="header-section-number">6.1.1.2</span> GWAS</h4>
<p>PGS are calculated for an individual by summing trait-associated alleles identified by genome-wide association studies (<strong>GWAS</strong>), as weighted by the alleles’ effect sizes.<span class="citation"><a href="#fn306" class="footnote-ref" id="fnref306"><sup>306</sup></a></span> GWAS aim to identify genetic variants associated with traits by comparing the allele frequencies of individuals who share similar ancestries, but differ in values for the trait in question.<span class="citation"><a href="#fn307" class="footnote-ref" id="fnref307"><sup>307</sup></a></span> As of 2021, over 5,700 GWAS have been performed for more than 3,330 traits.<span class="citation"><a href="#fn308" class="footnote-ref" id="fnref308"><sup>308</sup></a></span></p>
<p>However, most GWAS have been performed with individuals of European ancestry, despite only constituting around 16% of the present global population. Although the proportion of participants in GWAS from a non-European background increased from 4% in 2009 to 16% in 2016),<span class="citation"><a href="#fn309" class="footnote-ref" id="fnref309"><sup>309</sup></a></span> as of 2019, 79% of all GWAS participants recorded in the GWAS Catalog were of European ancestry, and the proportion of non-European individuals has remained the same or reduced since late 2014.<span class="citation"><a href="#fn310" class="footnote-ref" id="fnref310"><sup>310</sup></a></span> This bias extends to PGS studies, where as of 2019, only 67% of them included only participants of European ancestry, with another 19% including only East Asian ancestry participants, and only 3.8% with cohorts of African, Hispanic, or Indigenous ancestry.<span class="citation"><a href="#fn311" class="footnote-ref" id="fnref311"><sup>311</sup></a></span></p>
<p>It is therefore unsurprising that PGS scores are far better at predicting disease risk in individuals of European ancestry than in those of non-European ancestry.<span class="citation"><a href="#fn312" class="footnote-ref" id="fnref312"><sup>312</sup></a></span> Indeed, the predictive accuracy of PGS scores decays with genetic divergence of the GWAS “independent” or “test” sample, from the “discovery” or “training” sample, as established in both humans,<span class="citation"><a href="#fn313" class="footnote-ref" id="fnref313"><sup>313</sup></a></span> and livestock.<span class="citation"><a href="#fn314" class="footnote-ref" id="fnref314"><sup>314</sup></a></span></p>
<p>Compared to PGS scores for those of European ancestry, PGS scores across multiple traits are ~64-78% less accurate for individuals of African ancestry,,<span class="citation"><a href="#fn315" class="footnote-ref" id="fnref315"><sup>315</sup></a></span> ~50% less accurate for individuals of East-Asian ancestry, and ~37% less accurate for individuals of South-Asian ancestry.<span class="citation"><a href="#fn316" class="footnote-ref" id="fnref316"><sup>316</sup></a></span></p>
</div>
</div>
<div id="contributors-to-pgs-non-transferability" class="section level3" number="6.1.2">
<h3><span class="header-section-number">6.1.2</span> Contributors to PGS non-transferability</h3>
<p>What explains this disparity in predictive value? A number of factors may be responsible, including:</p>
<ol style="list-style-type: decimal">
<li><p>The failure of GWAS to identify causal variants that either do not exist or are not identifiable within the “discovery” sample, for both technological and methodological reasons;<span class="citation"><a href="#fn317" class="footnote-ref" id="fnref317"><sup>317</sup></a></span></p></li>
<li><p>The sample populations may differ in linkage disequilibrium (<strong>LD</strong>) – the correlation structure of the genome – which would change the estimated effect sizes of the causal variants, even when the causal variants themselves are the same;<span class="citation"><a href="#fn318" class="footnote-ref" id="fnref318"><sup>318</sup></a></span></p></li>
<li><p>Allele frequencies of the causal variants, and the distribution of the effect sizes of the causal variants, may differ between populations;<span class="citation"><a href="#fn319" class="footnote-ref" id="fnref319"><sup>319</sup></a></span> and</p></li>
<li><p>The environments and demographies of populations tend to differ. Such differences are often correlated with genetic divergence due to geography, making it difficult to determine whether the associations are driven by the differences between population in their genetics, or their environments.<span class="citation"><a href="#fn320" class="footnote-ref" id="fnref320"><sup>320</sup></a></span></p></li>
</ol>
<p>The first three factors can degrade predictive performance even in the absence of biological and environmental differences. On the other hand, environmental and demographic differences can drive forces of natural selection can in turn drive differences in causal genetic architecture.<span class="citation"><a href="#fn321" class="footnote-ref" id="fnref321"><sup>321</sup></a></span></p>
<p>I will discuss each of these factors in turn before addressing point (3) in this analysis.</p>
<div id="fst-discovery-sec" class="section level4" number="6.1.2.1">
<h4><span class="header-section-number">6.1.2.1</span> Technological and methodological limitations of GWAS</h4>
<p>The power to discover a causal variant through GWAS depends on the variant’s effect size and frequency in the study population.<span class="citation"><a href="#fn322" class="footnote-ref" id="fnref322"><sup>322</sup></a></span> That is to say, the stronger the variant’s effect, or the more common it is, the more likely it is to be discovered. Rare variants tend to have stronger effect sizes,<span class="citation"><a href="#fn323" class="footnote-ref" id="fnref323"><sup>323</sup></a></span> likely due to purifying selection,<span class="citation"><a href="#fn324" class="footnote-ref" id="fnref324"><sup>324</sup></a></span> and tend not to be shared across populations.<span class="citation"><a href="#fn325" class="footnote-ref" id="fnref325"><sup>325</sup></a></span> This is particularly relevant for African populations, as they have a much greater level of genetic variance than other populations due to the human species having originated on that continent.<span class="citation"><a href="#fn326" class="footnote-ref" id="fnref326"><sup>326</sup></a></span> Therefore, if GWAS aren’t performed on diverse populations, PGS can’t take into account the rare variants present in non-European populations that are likely to exert stronger effects on the trait of interest. There are also several other issues that can affect the discoverability of causal variants through GWAS, including the technology used for genotyping, the selection of the cohort, and the necessary exclusion of genotypic outliers.
With respect to genotyping technologies, GWAS often use data from SNP microarrays. These do not sequence the whole genome, but rather a selection (from several hundred thousand to millions) of genetic markers intended to present <em>common</em> genetic variation,<span class="citation"><a href="#fn327" class="footnote-ref" id="fnref327"><sup>327</sup></a></span> which accordingly tend to neglect rare genetic variants.<span class="citation"><a href="#fn328" class="footnote-ref" id="fnref328"><sup>328</sup></a></span> To increase the density of genotypes, which would increase the likelihood of refining the association signal and identifying causal variants, researchers often “impute” variants that aren’t sequenced directly.<span class="citation"><a href="#fn329" class="footnote-ref" id="fnref329"><sup>329</sup></a></span> The imputation process involves “phasing” the study genotypes onto the genotypes of a “reference panel.”<span class="citation"><a href="#fn330" class="footnote-ref" id="fnref330"><sup>330</sup></a></span> However, if the reference panel does not sufficiently represent the population in the study sample, they are likely to miss or incorrectly impute those genotypes.<span class="citation"><a href="#fn331" class="footnote-ref" id="fnref331"><sup>331</sup></a></span> Again, this is particularly problematic for African populations.</p>
<p>The lack of representation of rare variants in SNP microarrays can be overcome by using next-generation sequencing technologies such as whole-genome sequencing (<strong>WGS</strong>) and whole-exome sequencing (<strong>WES</strong>). (The former seeks to sequence the full genome, and the latter of only targets the coding regions of the genome.) These methods are more expensive than SNP microarrays, which hinders their widespread use at scale, and although their costs are continuing to decrease rapidly, there is a question as to whether they return a proportionate benefit in all use cases.<span class="citation"><a href="#fn332" class="footnote-ref" id="fnref332"><sup>332</sup></a></span></p>
<p>A second limitation is the selection of GWAS cohorts, which can introduce selection and collider biases.<span class="citation"><a href="#fn333" class="footnote-ref" id="fnref333"><sup>333</sup></a></span> For instance, the UK Biobank, which contains genetic and phenotypic data on 500,000 participants who volunteered for inclusion between 2006 and 2010, tend to be older, female, healthier, and wealthier than non-participants.<span class="citation"><a href="#fn334" class="footnote-ref" id="fnref334"><sup>334</sup></a></span> This creaties the possibility of confounding genetic associations with environmental factors, which I discuss further in @ref(fst-env-sec) below.</p>
<p>A third limitation is the “quality control” step that is required during the GWAS process.<span class="citation"><a href="#fn335" class="footnote-ref" id="fnref335"><sup>335</sup></a></span> To avoid confounding from population stratification, which can lead to overestimated heritability and biased PGS, GWAS cohorts are filtered to include only those with similar ancestries – or relative genetic homogeneity – by clustering individuals through principal component analysis (<strong>PCA</strong>) of their genotypes, and excluding outliers. I elaborate on the issue of population stratification in section @ref(fst-env-sec) below, but at present, a statistical model for GWAS that can include cohorts with diverse ancestries without the risk of serious confounding is yet to be developed.<span class="citation"><a href="#fn336" class="footnote-ref" id="fnref336"><sup>336</sup></a></span></p>
</div>
<div id="differences-in-ld" class="section level4" number="6.1.2.2">
<h4><span class="header-section-number">6.1.2.2</span> Differences in LD</h4>
<p>Because GWAS SNP markers are often not the causal variants themselves, but merely in physical proximity to them, the estimated effect size of a SNP marker depends on the extent to which it is in LD with the causal variant.<span class="citation"><a href="#fn337" class="footnote-ref" id="fnref337"><sup>337</sup></a></span> To illustrate the problem, if a SNP has an LD <span class="math inline">\(r^2\)</span> with a causal variant of 0.8 in the discovery population and 0.6 in the target population, it would explain 25% = (1 - 0.6/0.8) less trait variation in the target population, and would therefore be less predictive.<span class="citation"><a href="#fn338" class="footnote-ref" id="fnref338"><sup>338</sup></a></span></p>
<p>These differences in effect-size estimates may typically be small for most regions of the genome, but as PGS sum across all such effects, they aggregate these population differences.<span class="citation"><a href="#fn339" class="footnote-ref" id="fnref339"><sup>339</sup></a></span> Previous empirical and simulation studies have shown that accuracy of PGS scores decay with increased genetic differentiation (<span class="math inline">\(F_{ST}\)</span> – described below in @ref(Fst-descr)) and LD differences between populations.<span class="citation"><a href="#fn340" class="footnote-ref" id="fnref340"><sup>340</sup></a></span> The issue may be addressed to a degree by using LD information from an external reference panel as a prior to infer the posterior mean effect size of a genetic variant – <span class="citation">Bjarni J Vilhjálmsson et al.<a href="#fn341" class="footnote-ref" id="fnref341"><sup>341</sup></a></span> demonstrated through simulations that this could improve PGS predictive accuracy. Yet the most appropriate means of deal with differences between populations in LD remains an active area of research.<span class="citation"><a href="#fn342" class="footnote-ref" id="fnref342"><sup>342</sup></a></span></p>
</div>
<div id="differences-in-allele-frequencies" class="section level4" number="6.1.2.3">
<h4><span class="header-section-number">6.1.2.3</span> Differences in allele frequencies</h4>
<p>Causal variants can differ in both frequency and effect size between different ancestry groups, e.g. for lactase persistence,<span class="citation"><a href="#fn343" class="footnote-ref" id="fnref343"><sup>343</sup></a></span> or skin pigmentation.<span class="citation"><a href="#fn344" class="footnote-ref" id="fnref344"><sup>344</sup></a></span> If a causal allele is rare in the GWAS discovery population, even if it is discovered (see @ref(fst-discovery-sec)), it is likely to have noisy effect size estimates, and therefore likely to inaccurately estimate its effect size in a different population where it exists at a higher frequency.</p>
<p>Differences in allele frequencies between populations can arise through random genetic drift, or be driven by selective pressures towards the trait optima for a given environment.<span class="citation"><a href="#fn345" class="footnote-ref" id="fnref345"><sup>345</sup></a></span> However, evolutionary biologists have found that differences between populations in the mean values for traits tend to occur through small, coordinated shifts in their allele frequencies.<span class="citation"><a href="#fn346" class="footnote-ref" id="fnref346"><sup>346</sup></a></span> In Chapter @ref(Fst-chap), I explore the differences in allele frequencies across populations for all polygenic traits in the GWAS Catalog, and confirm that with few exceptions – including skin pigmentation, and HIV viral load – the differences in allele frequencies between populations tends to be small.</p>
</div>
<div id="fst-env-sec" class="section level4" number="6.1.2.4">
<h4><span class="header-section-number">6.1.2.4</span> Differences in environment</h4>
<p>Genes continuously interact with each other (GxG, or “epistasis”),<span class="citation"><a href="#fn347" class="footnote-ref" id="fnref347"><sup>347</sup></a></span> the genes of one’s parents (“genetic nurture”, <span class="citation">Kong et al.<a href="#fn348" class="footnote-ref" id="fnref348"><sup>348</sup></a></span>) or social companions (“social genetic effects”),<span class="citation"><a href="#fn349" class="footnote-ref" id="fnref349"><sup>349</sup></a></span><a href="#fn350" class="footnote-ref" id="fnref350"><sup>350</sup></a> and the wider non-genetic environment (GxE).</p>
<p>The respective contributions of genetics and environment to traits with social value – such as intelligence – is highly contentious, especially when there are apparent differences between populations in the mean values for those traits. PGS measure the proportion of variance within a population that is explained by genetics. Because PRS summarises a <em>proportion</em> of the total variance, when studying a population that is subject to greater environmental variation, the variance attributable to genetic factors will proportionately reduce. The corollary being that when studying a population where the environment is held constant, the proportion of variance for that trait that is explained by genetic factors will approach 1. Therefore, increases in the amount of environmental variance that a population is exposed to will reduce the accuracy of PGS predictions when applied to that population.</p>
<p>Different environments are also often correlated with population structure.<span class="citation"><a href="#fn351" class="footnote-ref" id="fnref351"><sup>351</sup></a></span> For example, in East Asia, there is a greater proportion of individuals of East-Asian ancestry than there is of European ancestry, and <em>vice versa</em> in Europe. Those East-Asian individuals will therefore tend to share more of their genetic background with each other than with Europeans, and that population structure will be correlated with the different environments that exist in East Asia compared to Europe. This makes it difficult to determine whether it is the differences in their environments or the differences in their genetics that is driving the discrepancies between the mean values for traits between those populations. These complexities are unlikely to be resolved in the near future, which makes it attractive to turn to model organisms to address more basic biological questions regarding GxE in relation to complex traits,<span class="citation"><a href="#fn352" class="footnote-ref" id="fnref352"><sup>352</sup></a></span> as we have done with respect to behaviour in Chapters @ref(Pilot-chap) and @ref(MIKK-F2-chap).</p>
</div>
</div>
<div id="Fst-descr" class="section level3" number="6.1.3">
<h3><span class="header-section-number">6.1.3</span> <span class="math inline">\(F_{ST}\)</span></h3>
<p>The widely-used fixation index (<span class="math inline">\(F_{ST}\)</span>) was introduced independently by Sewall Wright<span class="citation"><a href="#fn353" class="footnote-ref" id="fnref353"><sup>353</sup></a></span> and Gustave Malécot<span class="citation"><a href="#fn354" class="footnote-ref" id="fnref354"><sup>354</sup></a></span> as a metric for measuring the genetic diversity between populations.<a href="#fn355" class="footnote-ref" id="fnref355"><sup>355</sup></a> It quantifies the relative variance in allele frequency between groups compared to within groups, reflecting the combined effects of genetic drift, migration, mutation, and selection.<span class="citation"><a href="#fn356" class="footnote-ref" id="fnref356"><sup>356</sup></a></span> The metric ranges from 0 to 1, where loci with high <span class="math inline">\(F_{ST}\)</span> values – that is, loci with a large relative between-group variance in allele frequencies – may have been subject to selection or different demographic processes.<span class="citation"><a href="#fn357" class="footnote-ref" id="fnref357"><sup>357</sup></a></span> The metric has customarily been used to identify regions of the genome have been subject to diversifying selection.<span class="citation"><a href="#fn358" class="footnote-ref" id="fnref358"><sup>358</sup></a></span></p>
<p>I first sought to explore the distribution of <span class="math inline">\(F_{ST}\)</span> for SNPs across the human genome. As the reference for human genomic variation across diverse populations, I used the New York Genome Center high-coverage, phased .vcf files<span class="citation"><a href="#fn359" class="footnote-ref" id="fnref359"><sup>359</sup></a></span> for the 2,504 individuals from 26 populations from across the globe, as described in the 1000 Genomes phase 3 release.<span class="citation"><a href="#fn360" class="footnote-ref" id="fnref360"><sup>360</sup></a></span> I then annotated those .vcf files with human SNP IDs from dbSNP release 9606,<span class="citation"><a href="#fn361" class="footnote-ref" id="fnref361"><sup>361</sup></a></span> and calculated <span class="math inline">\(F_{ST}\)</span> for each of the ~69M SNPs in that dataset with PLINK 1.9.<span class="citation"><a href="#fn362" class="footnote-ref" id="fnref362"><sup>362</sup></a></span> Figure @ref(fig:fst-circos) shows the location and <span class="math inline">\(F_{ST}\)</span> value for all SNPs in the 1000 Genomes dataset, and Figure @ref(fig:fst-histo) shows their distribution across the range of <span class="math inline">\(F_{ST}\)</span> values.</p>
<div class="figure">
<img src="figs/fst/circos_all_fst.png" alt="(ref:fst-circos)" width="100%" />
<p class="caption">
(#fig:fst-circos)(ref:fst-circos)
</p>
</div>
<div class="figure">
<img src="figs/fst/histo_fst_all.png" alt="(ref:fst-histo)" width="100%" />
<p class="caption">
(#fig:fst-histo)(ref:fst-histo)
</p>
</div>
<p>Figure @ref(fig:fst-circos) shows that with very few exceptions, <span class="math inline">\(F_{ST}\)</span> remains below 0.6, although specific regions do appear to be differentially selected, as indicated be the peaks on chromosomes 1, 2, and 17. Figure @ref(fig:fst-histo) makes it clear that the vast majority of SNPs have very low <span class="math inline">\(F_{ST}\)</span>, with a genome-wide mean of 0.019. The question then is whether the SNPs that have a detectable effect on traits tend to have higher <span class="math inline">\(F_{ST}\)</span> values – and therefore greater variation in allele frequencies between populations – than random SNPs with similar allele frequencies in European populations. If so, this would suggest that variation in allele frequencies might contribute significantly to the non-transferability of PRS scores derived from European-focused GWAS.</p>
</div>
</div>
<div id="Fst-analysis-chap" class="section level2" number="6.2">
<h2><span class="header-section-number">6.2</span> Analysis</h2>
<p>In this analysis I explore the distribution of <span class="math inline">\(F_{ST}\)</span> scores for loci associated with 587 traits, a subset of the GWAS Catalog that passed our criteria for suitable polygenic traits (see section @ref(Fst-analysis-chap)). Using high-coverage sequence data for 2,504 individuals from the 1000 Genomes Project phase 3 release, for each trait in the GWAS Catalog I calculated the distribution of <span class="math inline">\(F_{ST}\)</span> across all approximately-unlinked SNPs associated with it (<strong>trait SNPs</strong>), and compared these <span class="math inline">\(F_{ST}\)</span> distributions with the <span class="math inline">\(F_{ST}\)</span> distributions of random-selected SNPs that were matched to the trait SNPs by their allele frequencies in European populations (<strong>control SNPs</strong>).</p>
<div id="gwas-catalog" class="section level4" number="6.2.0.1">
<h4><span class="header-section-number">6.2.0.1</span> GWAS Catalog</h4>
<p>I used the R package <em>gwasrapidd</em><span class="citation"><a href="#fn363" class="footnote-ref" id="fnref363"><sup>363</sup></a></span> to query all traits in the GWAS Catalog<span class="citation"><a href="#fn364" class="footnote-ref" id="fnref364"><sup>364</sup></a></span> as of 9 August 2021 (<span class="math inline">\(N_{TRAITS}\)</span> = 3,459). For 541 of these traits, no matching variant IDs could be pulled out from the 1000 Genomes VCFs, leaving <span class="math inline">\(N_{TRAITS}\)</span> = 3,008.</p>
</div>
<div id="linkage-disequilibrium" class="section level3" number="6.2.1">
<h3><span class="header-section-number">6.2.1</span> Linkage disequilibrium</h3>
<p>To obtain the “trait SNP” dataset, for each trait, I sought to isolate the SNP closest to each of its true causal variants, and exclude the SNPs in LD with them. To this end, I used PLINK 1.9<span class="citation"><a href="#fn365" class="footnote-ref" id="fnref365"><sup>365</sup></a></span> to “clump” the SNPs associated with each of the remaining 3,008 traits, using an “index” SNP p-value threshold of <span class="math inline">\(10^{-8}\)</span>,<span class="citation"><a href="#fn366" class="footnote-ref" id="fnref366"><sup>366</sup></a></span> <span class="math inline">\(r^2\)</span> threshold of 0.1,<span class="citation"><a href="#fn367" class="footnote-ref" id="fnref367"><sup>367</sup></a></span> and base window size of 1 Mb. This process revealed 2,045 traits with at least one index SNP that met the p-value threshold. The index SNPs for each trait formed the set of trait SNPs, and <strong>Figure @ref(fig:FstSnpCount)</strong> shows the counts of unique SNP IDs associated with each trait before and after clumping. In order to target relatively polygenic traits, I further filtered out traits with fewer than 10 trait SNPs, leaving <span class="math inline">\(N_{TRAITS}\)</span> = 587.</p>
<p>(ref:FstSnpCount) log<sub>10</sub> counts of associated SNPs for each trait before and after the clumping process, which involved: a) excluding all SNPs with a <em>p</em>-value greater than <span class="math inline">\(10^{-8}\)</span>; and b) starting with the SNPs with the lowest <em>p</em>-values (“index” SNPs), excluding all other SNPs within a 1 Mb region of the index SNP with an LD <span class="math inline">\(r^2\)</span> of more than 0.1. [CHANGE COLOUR]</p>
<div class="figure">
<img src="figs/fst/0.1_1000_20220314_snp_counts.png" alt="(ref:FstSnpCount)" width="100%" />
<p class="caption">
(#fig:FstSnpCount)(ref:FstSnpCount)
</p>
</div>
</div>
<div id="control-snps" class="section level3" number="6.2.2">
<h3><span class="header-section-number">6.2.2</span> Control SNPs</h3>
<p>To obtain the “control SNP” dataset, I assigned each trait SNP to one of 20 bins based on its minor allele frequency in European populations (as provided in the original 1000 Genomes .vcf files under the column header ‘INFO/AC_EUR’). For example, if a trait SNP had a minor allele frequency of 0.08 in European populations, it was assigned to the <span class="math inline">\((0.05, 0.1]\)</span> bin. I did the same for all (un-associated) SNPs in the .vcf files, then paired each trait SNP with a random SNP from the .vcf file in the equivalent bin. These allele-frequency-paired random SNPs formed the set of “control SNPs”, which I used to infer the <span class="math inline">\(F_{ST}\)</span> distribution of a random set of SNPs with the same allele frequencies as the trait SNPs, and against which I could compare the <span class="math inline">\(F_{ST}\)</span> distribution of the trait SNPs.</p>
</div>
<div id="f_st-and-ranking-traits-by-signed-kolmogorov-smirnov-d-statistic" class="section level3" number="6.2.3">
<h3><span class="header-section-number">6.2.3</span> <span class="math inline">\(F_{ST}\)</span> and ranking traits by signed Kolmogorov-Smirnov <span class="math inline">\(D\)</span> statistic</h3>
<p>I then calculated <span class="math inline">\(F_{ST}\)</span> for each of the trait SNPs and their matched control SNPs using the Weir and Cockerham method,<span class="citation"><a href="#fn368" class="footnote-ref" id="fnref368"><sup>368</sup></a></span> as implemented in the R package <em>pegas</em>.<span class="citation"><a href="#fn369" class="footnote-ref" id="fnref369"><sup>369</sup></a></span> I then sought to rank all traits based on the directional difference in <span class="math inline">\(F_{ST}\)</span> distributions between trait and control SNPs. The Kolmogorov-Smirnov (KS) test is an appropriate test to measure the differences between distributions,<span class="citation"><a href="#fn370" class="footnote-ref" id="fnref370"><sup>370</sup></a></span> but the <em>p</em>-values are strongly affected by the number of samples in each distribution (here, the number of loci in each trait), and it does not measure whether one distribution is overall higher or lower than the other distribution. These limitations made it impossible to use the KS test <em>p</em>-values to compare the directional distances between trait and control SNPs across traits that have very different numbers of SNPs. I accordingly formulated the following method for ranking the traits based on the directional distance of the <span class="math inline">\(F_{ST}\)</span> distributions of the trait SNPs compared to the control SNPs:</p>
<p>First, I ran three KS tests for each trait <span class="math inline">\(t\)</span> with <span class="math inline">\(x_t\)</span> = <span class="math inline">\(F_{ST, trait SNPs}\)</span> and <span class="math inline">\(y_t\)</span> = <span class="math inline">\(F_{ST, control SNPs}\)</span>:</p>
<ol style="list-style-type: decimal">
<li><p>two-sided (<span class="math inline">\(D_t\)</span>) ;</p></li>
<li><p>one-sided “greater” (<span class="math inline">\(D_t^+\)</span>) ; and</p></li>
<li><p>one-sided “less” (<span class="math inline">\(D_t^-\)</span>).</p></li>
</ol>
<p>I note that <span class="math inline">\(D_{t} = max(D_t^+, D_t^-)\)</span>, where <span class="math inline">\(D_t^+\)</span> is the greatest vertical distance attained by the eCDF of <span class="math inline">\(x_t\)</span> over the eCDF of <span class="math inline">\(y_t\)</span>, and <span class="math inline">\(D_t^-\)</span> is the greatest vertical distance attained by the eCDF of <span class="math inline">\(y_t\)</span> over the eCDF of <span class="math inline">\(x_t\)</span>.<span class="citation"><a href="#fn371" class="footnote-ref" id="fnref371"><sup>371</sup></a></span> I then used a comparison of <span class="math inline">\(D_t^+\)</span> and <span class="math inline">\(D_t^-\)</span> to formulate a “signed D statistic” (<span class="math inline">\(D_t^S\)</span>), based on the logic that trait SNPs with a lower overall <span class="math inline">\(F_{ST}\)</span> than control SNPs tend to have a higher <span class="math inline">\(D\)</span> under the “greater” test than the “less” test, and vice versa.</p>
<p>Therefore, <span class="math inline">\({D_t^S}\)</span>:</p>
<p><span class="math display">\[
\begin{aligned}
D_t^- &gt; D_t^+ &amp;: &amp;-D_t \\
D_t^- = D_t^+ &amp;: &amp;0 \\
D_t^- &lt; D_t^+ &amp;: &amp;D_t \\
\end{aligned}
\]</span></p>
<p>In <strong>Figure @ref(fig:FstMain)</strong> I present the <span class="math inline">\(F_{ST}\)</span> distributions of trait SNPs for an illustrative subset of 28 human traits, ranked by <span class="math inline">\({D_t^S}\)</span> when compared with their matched control SNPs. <strong>Figure @ref(fig:FstMain)A</strong> shows the densities of SNPs as a function of <span class="math inline">\(F_{ST}\)</span>, and <strong>Figure @ref(fig:FstMain)B and C</strong> show their empirical Cumulative Distribution Functions (eCDFs). <strong>Figure @ref(fig:FstMain)B</strong> includes the eCDFs of control SNPs in grey. eCDF figures for all 587 traits that passed our filters (Methods) are provided in the Appendix @ref(fig:eCDFall).</p>
<p>(ref:FstMain) Distributions of <span class="math inline">\(F_{ST}\)</span> across 28 illustrative human traits, ranked by signed-D (Kolmogorov-Smirnov test) comparing trait and control SNPs. <strong>A</strong>. <span class="math inline">\(F_{ST}\)</span> density ridge plots with SNP markers. <strong>B</strong>. Empirical Cumulative Distribution Functions (eCDFs) of <span class="math inline">\(F_{ST}\)</span> for trait-associated (colour) and random control (grey) SNPs, faceted by trait. <strong>C</strong>. Consolidated eCDFs of trait-associated SNPs from (B). eCDFs for all traits are included in the Appendix.</p>
<div class="figure">
<img src="figs/fst/0.1_1000_20220314_final.png" alt="(ref:FstMain)" width="100%" />
<p class="caption">
(#fig:FstMain)(ref:FstMain)
</p>
</div>
<p>These results show that for most traits, especially those that are highly polygenic, the <span class="math inline">\(F_{ST}\)</span> of their associated alleles tend to be low, and to differ little from their matched control SNPs. This suggests that the causal variants of polygenic traits are generally shared across global populations at similar frequencies. <span class="citation">Wang et al.<a href="#fn372" class="footnote-ref" id="fnref372"><sup>372</sup></a></span> determined through simulations that differences in LD and allele frequencies between populations can explain 70-80% of the loss of PGS relative accuracy for traits like body mass index and type 2 diabetes. This builds on the work of others [CITE PAPERS CITED BY BERG AND COOP AND ELABORATE]. As most of these variants have been discovered in European populations, the poor transferability of PGS between populations does not appear to be primarily driven by differences in the allele frequencies of these discovered variants, but perhaps rather by as-yet undiscovered variants of larger effect sizes in non-European populations, as well as differences between these populations in LD structure and environment.</p>
<p>There is a question as to whether the higher distributions of <span class="math inline">\(F_{ST}\)</span> observed for hair shape, eye colour, skin pigmentation, and HIV infection, are due to those variants being discovered in diverse populations, such as those in Africa. For instance, if the alleles were predominantly discovered through GWAS on African populations, which contain an outsized proportion of genetic variation, those alleles would be more likely to show lower frequencies in other populations, and consequently higher <span class="math inline">\(F_{ST}\)</span>. To examine the number of individuals from different ancestries that were used in these studies, I generated Figure @ref(fig:fst-n-indivs). This figure shows that GWAS performed for the more comprehensively-studied phenotypes such as body height, BMI, and educational attainment, included individuals from many diverse populations, whereas the GWAS on pigmentation-related traits tended to focus on European or Latin American populations, as well as those of African ancestry in some cases. As expected, the GWAS on HIV-related traits included a higher proportion of individuals of African ancestry, which may explain the higher levels of <span class="math inline">\(F_{ST}\)</span> observed there. I note that the GWAS Catalog data did not allow me to determine whether different studies on the same trait used the same cohorts (e.g. from UK Biobank), which would have the effect of inflating the counts shown in Figure @ref(fig:fst-n-indivs).</p>
<div class="figure">
<img src="figs/fst/20220324_n_individuals.png" alt="(ref:fst-n-indivs)" width="100%" />
<p class="caption">
(#fig:fst-n-indivs)(ref:fst-n-indivs)
</p>
</div>
</div>
</div>
<div id="implications" class="section level2" number="6.3">
<h2><span class="header-section-number">6.3</span> Implications</h2>
<div id="limitation-of-f_st" class="section level3" number="6.3.1">
<h3><span class="header-section-number">6.3.1</span> Limitation of <span class="math inline">\(F_{ST}\)</span></h3>
<p>The first relevant limitation of <span class="math inline">\(F_{ST}\)</span> is that it does not correspond to similarities or differences between populations in trait values. This is because <span class="math inline">\(F_{ST}\)</span> does not take into account the effect size or the direction of effect of the trait-associated allele, so for highly-polygenic traits like the ones shown here, <span class="math inline">\(F_{ST}\)</span> is almost entirely decoupled from the mean additive genetic value between populations.<span class="citation"><a href="#fn373" class="footnote-ref" id="fnref373"><sup>373</sup></a></span> This means that even when a trait’s associated alleles have low <span class="math inline">\(F_{ST}\)</span>, as we see for most traits here, it does not follow that the population mean values for that trait would be the same due to genetic similarities between those populations. As <span class="citation">A. Kremer and V. Le Corre<a href="#fn374" class="footnote-ref" id="fnref374"><sup>374</sup></a></span> determined by through simulations, for highly polygenic traits, mean trait differences are driven by small, coordinated shifts in allele frequencies, which would not be detected by <span class="math inline">\(F_{ST}\)</span>.<span class="citation"><a href="#fn375" class="footnote-ref" id="fnref375"><sup>375</sup></a></span></p>
<p>The second relevant limitation is that <span class="math inline">\(F_{ST}\)</span> is not sufficiently powered to detect selection for polygenic traits,<span class="citation"><a href="#fn376" class="footnote-ref" id="fnref376"><sup>376</sup></a></span> which means that these results cannot be used to determine the extent to which alleles are under selection in a given population. This is apparent in our analysis, where in contrast with <span class="citation">Fernando Racimo, Jeremy J Berg, and Joseph K Pickrell<a href="#fn377" class="footnote-ref" id="fnref377"><sup>377</sup></a></span>, height and BMI, which have been found to be under strong differential selection, appear similar to other polygenic traits.</p>
</div>
<div id="the-importance-of-environment-and-a-push-for-diversity-in-genetic-studies" class="section level3" number="6.3.2">
<h3><span class="header-section-number">6.3.2</span> The importance of environment, and a push for diversity in genetic studies</h3>
<p>The obvious solution to this issue of non-transferability of PGS scores to diverse populations is to increase the representation of those populations in GWAS and PGS studies, as has been often proposed elsewhere.<span class="citation"><a href="#fn378" class="footnote-ref" id="fnref378"><sup>378</sup></a></span> [ELABORATE]</p>
<!--chapter:end:06-Fst.Rmd-->
</div>
</div>
</div>
<div id="references" class="section level1 unnumbered">
<h1 class="unnumbered">References</h1>
<div id="refs" class="references csl-bib-body hanging-indent">
<div id="ref-GlobalReferenceHuman2015" class="csl-entry">
<span>“A Global Reference for Human Genetic Variation.”</span> <em>Nature</em> 526, no. 7571 (2015): 68–74. <a href="https://doi.org/10.1038/nature15393">https://doi.org/10.1038/nature15393</a>.
</div>
<div id="ref-adhikariGWASLatinAmericans2019" class="csl-entry">
Adhikari, Kaustubh, Javier Mendoza-Revilla, Anood Sohail, Macarena Fuentes-Guajardo, Jodie Lampert, Juan Camilo Chac’on-Duque, Malena Hurtado, et al. <span>“A <span>GWAS</span> in <span>Latin Americans</span> Highlights the Convergent Evolution of Lighter Skin Pigmentation in <span>Eurasia</span>.”</span> <em>Nature Communications</em> 10, no. 1, 1 (January 21, 2019): 358. <a href="https://doi.org/10.1038/s41467-018-08147-0">https://doi.org/10.1038/s41467-018-08147-0</a>.
</div>
<div id="ref-aidaInheritanceColorFreshWater1921" class="csl-entry">
Aida, Tatuo. <span>“On the <span>Inheritance</span> of <span>Color</span> in a <span>Fresh-Water Fish</span>, <span>Aplocheilus Latipes Temmick</span> and <span>Schlegel</span>, with <span>Special Reference</span> to <span>Sex-Linked Inheritance</span>.”</span> <em>Genetics</em> 6, no. 6 (November 1921): 554–73. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1200522/">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1200522/</a>.
</div>
<div id="ref-akeyInterrogatingHighDensitySNP2002" class="csl-entry">
Akey, Joshua M., Ge Zhang, Kun Zhang, Li Jin, and Mark D. Shriver. <span>“Interrogating a <span>High-Density SNP Map</span> for <span>Signatures</span> of <span>Natural Selection</span>.”</span> <em>Genome Research</em> 12, no. 12 (January 12, 2002): 1805–14. <a href="https://doi.org/10.1101/gr.631202">https://doi.org/10.1101/gr.631202</a>.
</div>
<div id="ref-anderssonDomesticanimalGenomicsDeciphering2004" class="csl-entry">
Andersson, Leif, and Michel Georges. <span>“Domestic-Animal Genomics: Deciphering the Genetics of Complex Traits.”</span> <em>Nature Reviews Genetics</em> 5, no. 3, 3 (March 2004): 202–12. <a href="https://doi.org/10.1038/nrg1294">https://doi.org/10.1038/nrg1294</a>.
</div>
<div id="ref-aristotleGenerationAnimals2021" class="csl-entry">
Aristotle. <em>On the <span>Generation</span> of <span>Animals</span></em>. <span>Good Press</span>, 2021. <a href="https://books.google.com?id=aLUqEAAAQBAJ">https://books.google.com?id=aLUqEAAAQBAJ</a>.
</div>
<div id="ref-bancroftPartIntroductionGouldian" class="csl-entry">
Bancroft, Paul. <span>“Part 1: <span>An</span> Introduction to the <span>Gouldian</span> Finch - <span>Planet Aviary</span>.”</span> Accessed August 22, 2022. <a href="https://planetaviary.com/part-1-an-introduction-to-the-gouldian-finch/">https://planetaviary.com/part-1-an-introduction-to-the-gouldian-finch/</a>.
</div>
<div id="ref-batesonMaterialsStudyVariation1901" class="csl-entry">
Bateson, William. <em>Materials for the <span>Study</span> of <span>Variation</span>: <span>Treated With Especial Regard</span> to <span>Discontinuity</span> in the <span>Origin</span> of <span>Species</span></em>. <span>Forgotten Books.</span>, 1901.
</div>
<div id="ref-batesonMendelPrinciplesHeredity1909" class="csl-entry">
———. <span>“Mendel’s <span>Principles</span> of <span>Heredity</span>: <span>Cambridge University Press</span>.”</span> <em>März 1909; 2nd Impr</em> 3 (1909): 1913.
</div>
<div id="ref-baudGeneticVariationSocial2017" class="csl-entry">
Baud, Amelie, Megan K. Mulligan, Francesco Paolo Casale, Jesse F. Ingels, Casey J. Bohl, Jacques Callebert, Jean-Marie Launay, et al. <span>“Genetic <span>Variation</span> in the <span>Social Environment Contributes</span> to <span>Health</span> and <span>Disease</span>.”</span> <em>PLOS Genetics</em> 13, no. 1 (January 25, 2017): e1006498. <a href="https://doi.org/10.1371/journal.pgen.1006498">https://doi.org/10.1371/journal.pgen.1006498</a>.
</div>
<div id="ref-bergPopulationGeneticSignal2014a" class="csl-entry">
Berg, Jeremy J., and Graham Coop. <span>“A <span>Population Genetic Signal</span> of <span>Polygenic Adaptation</span>.”</span> <em>PLOS Genetics</em> 10, no. 8 (August 7, 2014): e1004412. <a href="https://doi.org/10.1371/journal.pgen.1004412">https://doi.org/10.1371/journal.pgen.1004412</a>.
</div>
<div id="ref-bergPopulationGeneticSignal2014" class="csl-entry">
———. <span>“A <span>Population Genetic Signal</span> of <span>Polygenic Adaptation</span>.”</span> <em>PLOS Genetics</em> 10, no. 8 (August 7, 2014): e1004412. <a href="https://doi.org/10.1371/journal.pgen.1004412">https://doi.org/10.1371/journal.pgen.1004412</a>.
</div>
<div id="ref-bergReducedSignalPolygenic2019" class="csl-entry">
Berg, Jeremy J, Arbel Harpak, Nasa Sinnott-Armstrong, Anja Moltke Joergensen, Hakhamanesh Mostafavi, Yair Field, Evan August Boyle, et al. <span>“Reduced Signal for Polygenic Adaptation of Height in <span>UK Biobank</span>.”</span> Edited by Magnus Nordborg, Mark I McCarthy, Magnus Nordborg, Nicholas H Barton, and Joachim Hermisson. <em>eLife</em> 8 (March 21, 2019): e39725. <a href="https://doi.org/10.7554/eLife.39725">https://doi.org/10.7554/eLife.39725</a>.
</div>
<div id="ref-bergelsonIdentifyingGenesUnderlying2010" class="csl-entry">
Bergelson, Joy, and Fabrice Roux. <span>“Towards Identifying Genes Underlying Ecologically Relevant Traits in <span>Arabidopsis</span> Thaliana.”</span> <em>Nature Reviews Genetics</em> 11, no. 12, 12 (December 2010): 867–79. <a href="https://doi.org/10.1038/nrg2896">https://doi.org/10.1038/nrg2896</a>.
</div>
<div id="ref-bienFutureGenomicStudies2019" class="csl-entry">
Bien, Stephanie A., Genevieve L. Wojcik, Chani J. Hodonsky, Christopher R. Gignoux, Iona Cheng, Tara C. Matise, Ulrike Peters, Eimear E. Kenny, and Kari E. North. <span>“The <span>Future</span> of <span>Genomic Studies Must Be Globally Representative</span>: <span>Perspectives</span> from <span>PAGE</span>.”</span> <em>Annual Review of Genomics and Human Genetics</em> 20, no. 1 (August 31, 2019): 181–200. <a href="https://doi.org/10.1146/annurev-genom-091416-035517">https://doi.org/10.1146/annurev-genom-091416-035517</a>.
</div>
<div id="ref-blakeMouseGenomeDatabase2021" class="csl-entry">
Blake, Judith A, Richard Baldarelli, James A Kadin, Joel E Richardson, Cynthia L Smith, Carol J Bult, and the Mouse Genome Database Group. <span>“Mouse <span>Genome Database</span> (<span>MGD</span>): <span>Knowledgebase</span> for Mouse–Human Comparative Biology.”</span> <em>Nucleic Acids Research</em> 49, no. D1 (January 8, 2021): D981–87. <a href="https://doi.org/10.1093/nar/gkaa1083">https://doi.org/10.1093/nar/gkaa1083</a>.
</div>
<div id="ref-brownCorrelationBoldnessBody2007" class="csl-entry">
Brown, C., F. Jones, and V. A. Braithwaite. <span>“Correlation Between Boldness and Body Mass in Natural Populations of the Poeciliid <span>Brachyrhaphis</span> Episcopi.”</span> <em>Journal of Fish Biology</em> 71, no. 6 (2007): 1590–1601. <a href="https://doi.org/10.1111/j.1095-8649.2007.01627.x">https://doi.org/10.1111/j.1095-8649.2007.01627.x</a>.
</div>
<div id="ref-brownHeritableExperientialEffects2007" class="csl-entry">
Brown, Culum, Fiona Burgess, and Victoria A. Braithwaite. <span>“Heritable and Experiential Effects on Boldness in a Tropical Poeciliid.”</span> <em>Behavioral Ecology and Sociobiology</em> 62, no. 2 (December 1, 2007): 237–43. <a href="https://doi.org/10.1007/s00265-007-0458-3">https://doi.org/10.1007/s00265-007-0458-3</a>.
</div>
<div id="ref-bussEvolutionaryPersonalityPsychology1991" class="csl-entry">
Buss, David M. <span>“Evolutionary Personality Psychology.”</span> <em>Annual Review of Psychology</em> 42 (1991): 459–91. <a href="https://doi.org/10.1146/annurev.ps.42.020191.002331">https://doi.org/10.1146/annurev.ps.42.020191.002331</a>.
</div>
<div id="ref-campbellKingJamesBible2010" class="csl-entry">
Campbell, Gordon. <em>King <span>James Bible</span>: 400th <span>Anniversary Edition</span></em>. <span>OUP Oxford</span>, 2010.
</div>
<div id="ref-carolaHiddenMarkovModel2011" class="csl-entry">
Carola, Valeria, Olivier Mirabeau, and Cornelius T. Gross. <span>“Hidden <span>Markov Model Analysis</span> of <span>Maternal Behavior Patterns</span> in <span>Inbred</span> and <span>Reciprocal Hybrid Mice</span>.”</span> <em>PLOS ONE</em> 6, no. 3 (March 8, 2011): e14753. <a href="https://doi.org/10.1371/journal.pone.0014753">https://doi.org/10.1371/journal.pone.0014753</a>.
</div>
<div id="ref-changSecondgenerationPLINKRising2015" class="csl-entry">
Chang, Christopher C, Carson C Chow, Laurent CAM Tellier, Shashaank Vattikuti, Shaun M Purcell, and James J Lee. <span>“Second-Generation <span>PLINK</span>: Rising to the Challenge of Larger and Richer Datasets.”</span> <em>GigaScience</em> 4, no. 1 (December 2015): 7. <a href="https://doi.org/10.1186/s13742-015-0047-8">https://doi.org/10.1186/s13742-015-0047-8</a>.
</div>
<div id="ref-clarkImportanceInformationRelatives2012" class="csl-entry">
Clark, Samuel A., John M. Hickey, Hans D. Daetwyler, and Julius HJ van der Werf. <span>“The Importance of Information on Relatives for the Prediction of Genomic Breeding Values and the Implications for the Makeup of Reference Data Sets in Livestock Breeding Schemes.”</span> <em>Genetics Selection Evolution</em> 44, no. 1 (February 9, 2012): 4. <a href="https://doi.org/10.1186/1297-9686-44-4">https://doi.org/10.1186/1297-9686-44-4</a>.
</div>
<div id="ref-alfredWallace1895" class="csl-entry">
Commons, Wikimedia. <span>“File:alfred-Russel-Wallace-C1895.jpg — Wikimedia Commons<span>,</span> the Free Media Repository,”</span> 2022. <a href="https://commons.wikimedia.org/w/index.php?title=File:Alfred-Russel-Wallace-c1895.jpg&amp;oldid=642853479">https://commons.wikimedia.org/w/index.php?title=File:Alfred-Russel-Wallace-c1895.jpg&amp;oldid=642853479</a>.
</div>
<div id="ref-batesonPhoto" class="csl-entry">
———. <span>“File:Bateson2.jpg — Wikimedia Commons<span>,</span> the Free Media Repository,”</span> 2017. <a href="\url{https://commons.wikimedia.org/w/index.php?title=File:Bateson2.jpg&amp;oldid=255958176}">\url{https://commons.wikimedia.org/w/index.php?title=File:Bateson2.jpg&amp;oldid=255958176}</a>.
</div>
<div id="ref-charlesDarwinYoung" class="csl-entry">
———. <span>“File:charles Darwin by g. Richmond.png — Wikimedia Commons<span>,</span> the Free Media Repository,”</span> 2017. <a href="\url{https://commons.wikimedia.org/w/index.php?title=File:Charles_Darwin_by_G._Richmond.png&amp;oldid=260695102}">\url{https://commons.wikimedia.org/w/index.php?title=File:Charles_Darwin_by_G._Richmond.png&amp;oldid=260695102}</a>.
</div>
<div id="ref-darwinFinches" class="csl-entry">
———. <span>“File:darwin’s Finches by Gould.jpg — Wikimedia Commons<span>,</span> the Free Media Repository,”</span> 2022. <a href="\url{https://commons.wikimedia.org/w/index.php?title=File:Darwin%27s_finches_by_Gould.jpg&amp;oldid=643906085}">\url{https://commons.wikimedia.org/w/index.php?title=File:Darwin%27s_finches_by_Gould.jpg&amp;oldid=643906085}</a>.
</div>
<div id="ref-galtonPhoto" class="csl-entry">
———. <span>“File:francis Galton 1850s.jpg — Wikimedia Commons<span>,</span> the Free Media Repository,”</span> 2021. <a href="\url{https://commons.wikimedia.org/w/index.php?title=File:Francis_Galton_1850s.jpg&amp;oldid=569302173}">\url{https://commons.wikimedia.org/w/index.php?title=File:Francis_Galton_1850s.jpg&amp;oldid=569302173}</a>.
</div>
<div id="ref-sanger" class="csl-entry">
———. <span>“File:frederick Sanger2.jpg — Wikimedia Commons<span>,</span> the Free Media Repository,”</span> 2020. <a href="\url{https://commons.wikimedia.org/w/index.php?title=File:Frederick_Sanger2.jpg&amp;oldid=395411415}">\url{https://commons.wikimedia.org/w/index.php?title=File:Frederick_Sanger2.jpg&amp;oldid=395411415}</a>.
</div>
<div id="ref-mendelPhoto" class="csl-entry">
———. <span>“File:gregor Mendel 2.jpg — Wikimedia Commons<span>,</span> the Free Media Repository,”</span> 2021. <a href="\url{https://commons.wikimedia.org/w/index.php?title=File:Gregor_Mendel_2.jpg&amp;oldid=534728192}">\url{https://commons.wikimedia.org/w/index.php?title=File:Gregor_Mendel_2.jpg&amp;oldid=534728192}</a>.
</div>
<div id="ref-lamarck1802" class="csl-entry">
———. <span>“File:jean-Baptiste de Lamarck.jpg — Wikimedia Commons<span>,</span> the Free Media Repository,”</span> 2022. <a href="\url{https://commons.wikimedia.org/w/index.php?title=File:Jean-Baptiste_de_Lamarck.jpg&amp;oldid=641428959}">\url{https://commons.wikimedia.org/w/index.php?title=File:Jean-Baptiste_de_Lamarck.jpg&amp;oldid=641428959}</a>.
</div>
<div id="ref-pearsonPhoto" class="csl-entry">
———. <span>“File:karl Pearson, 1910.jpg — Wikimedia Commons<span>,</span> the Free Media Repository,”</span> 2020. <a href="\url{https://commons.wikimedia.org/w/index.php?title=File:Karl_Pearson,_1910.jpg&amp;oldid=477532248}">\url{https://commons.wikimedia.org/w/index.php?title=File:Karl_Pearson,_1910.jpg&amp;oldid=477532248}</a>.
</div>
<div id="ref-homunculusImage" class="csl-entry">
———. <span>“File:preformation.GIF — Wikimedia Commons<span>,</span> the Free Media Repository,”</span> 2019. <a href="\url{https://commons.wikimedia.org/w/index.php?title=File:Preformation.GIF&amp;oldid=372547016}">\url{https://commons.wikimedia.org/w/index.php?title=File:Preformation.GIF&amp;oldid=372547016}</a>.
</div>
<div id="ref-conoverPracticalNonparametricStatistics1999" class="csl-entry">
Conover, W. J. <em>Practical <span>Nonparametric Statistics</span></em>. <span>John Wiley &amp; Sons</span>, 1999. <a href="https://books.google.com?id=n_39DwAAQBAJ">https://books.google.com?id=n_39DwAAQBAJ</a>.
</div>
<div id="ref-10002015global" class="csl-entry">
Consortium, 1000 Genomes Project et al. <span>“A Global Reference for Human Genetic Variation.”</span> <em>Nature</em> 526, no. 7571 (2015): 68.
</div>
<div id="ref-internationalhumangenomesequencingconsortiumInitialSequencingAnalysis2001a" class="csl-entry">
Consortium, International Human Genome Sequencing. <span>“Initial Sequencing and Analysis of the Human Genome.”</span> <em>Nature</em> 409, no. 6822, 6822 (February 2001): 860–921. <a href="https://doi.org/10.1038/35057062">https://doi.org/10.1038/35057062</a>.
</div>
<div id="ref-theuk10kconsortiumUK10KProjectIdentifies2015" class="csl-entry">
Consortium, The UK10K. <span>“The <span>Uk10k</span> Project Identifies Rare Variants in Health and Disease.”</span> <em>Nature</em> 526, no. 7571, 7571 (October 2015): 82–90. <a href="https://doi.org/10.1038/nature14962">https://doi.org/10.1038/nature14962</a>.
</div>
<div id="ref-coyneTheodosiusDobzhanskyHybrid2016" class="csl-entry">
Coyne, Jerry A. <span>“Theodosius <span>Dobzhansky</span> on <span>Hybrid Sterility</span> and <span>Speciation</span>.”</span> <em>Genetics</em> 202, no. 1 (January 1, 2016): 5–7. <a href="https://doi.org/10.1534/genetics.115.184770">https://doi.org/10.1534/genetics.115.184770</a>.
</div>
<div id="ref-danecekTwelveYearsSAMtools2021" class="csl-entry">
Danecek, Petr, James K Bonfield, Jennifer Liddle, John Marshall, Valeriu Ohan, Martin O Pollard, Andrew Whitwham, et al. <span>“Twelve Years of <span>SAMtools</span> and <span>BCFtools</span>.”</span> <em>GigaScience</em> 10, no. 2 (February 1, 2021): giab008. <a href="https://doi.org/10.1093/gigascience/giab008">https://doi.org/10.1093/gigascience/giab008</a>.
</div>
<div id="ref-darwinNaturalistVoyageJournal1882" class="csl-entry">
Darwin, Charles. <em>A <span>Naturalist</span>’s <span>Voyage</span>: <span>Journal</span> of <span>Researches Into</span> the <span>Natural History</span> and <span>Geology</span> of the <span>Countries Visited During</span> the <span>Voyage</span> of <span>H</span>.<span>M</span>.<span>S</span>. <span>Beagle</span> : With <span>Maps</span> and <span>Illustrations</span></em>. <span>Murray</span>, 1882. <a href="https://books.google.com?id=JyR7HY3FMtkC">https://books.google.com?id=JyR7HY3FMtkC</a>.
</div>
<div id="ref-darwinOriginSpeciesMeans1859" class="csl-entry">
———. <em>On the <span>Origin</span> of <span>Species</span> by <span>Means</span> of <span>Natural Selection</span>, <span>Or</span>, <span>The Preservation</span> of <span>Favoured Races</span> in the <span>Struggle</span> for <span>Life</span></em>. <span>J. Murray</span>, 1859. <a href="https://books.google.com?id=gGUYt2PsDJcC">https://books.google.com?id=gGUYt2PsDJcC</a>.
</div>
<div id="ref-depristoFrameworkVariationDiscovery2011" class="csl-entry">
DePristo, Mark A., Eric Banks, Ryan Poplin, Kiran V. Garimella, Jared R. Maguire, Christopher Hartl, Anthony A. Philippakis, et al. <span>“A Framework for Variation Discovery and Genotyping Using Next-Generation <span>DNA</span> Sequencing Data.”</span> <em>Nature Genetics</em> 43, no. 5, 5 (May 2011): 491–98. <a href="https://doi.org/10.1038/ng.806">https://doi.org/10.1038/ng.806</a>.
</div>
<div id="ref-DNAThenNow" class="csl-entry">
<span>“<span>DNA</span> Then and Now.”</span> Accessed August 30, 2022. <a href="https://undsci.berkeley.edu/article/0_0_0/dna_14">https://undsci.berkeley.edu/article/0_0_0/dna_14</a>.
</div>
<div id="ref-domingueSocialGenomeFriends2018" class="csl-entry">
Domingue, Benjamin W., Daniel W. Belsky, Jason M. Fletcher, Dalton Conley, Jason D. Boardman, and Kathleen Mullan Harris. <span>“The Social Genome of Friends and Schoolmates in the <span>National Longitudinal Study</span> of <span>Adolescent</span> to <span>Adult Health</span>.”</span> <em>Proceedings of the National Academy of Sciences</em> 115, no. 4 (January 23, 2018): 702–7. <a href="https://doi.org/10.1073/pnas.1711803115">https://doi.org/10.1073/pnas.1711803115</a>.
</div>
<div id="ref-drueryExperimentsPlantHybridization1901" class="csl-entry">
Druery, C. T., and William Bateson. <span>“Experiments in Plant Hybridization.”</span> <em>Journal of the Royal Horticultural Society</em> 26 (1901): 1–32.
</div>
<div id="ref-duncanAnalysisPolygenicRisk2019" class="csl-entry">
Duncan, L., H. Shen, B. Gelaye, J. Meijsen, K. Ressler, M. Feldman, R. Peterson, and B. Domingue. <span>“Analysis of Polygenic Risk Score Usage and Performance in Diverse Human Populations.”</span> <em>Nature Communications</em> 10, no. 1, 1 (July 25, 2019): 3328. <a href="https://doi.org/10.1038/s41467-019-11112-0">https://doi.org/10.1038/s41467-019-11112-0</a>.
</div>
<div id="ref-durbinDistributionTheoryTests1973" class="csl-entry">
Durbin, J. <em>Distribution <span>Theory</span> for <span>Tests Based</span> on <span>Sample Distribution Function</span></em>. <span>SIAM</span>, 1973. <a href="https://books.google.com?id=zAryCrT1IUYC">https://books.google.com?id=zAryCrT1IUYC</a>.
</div>
<div id="ref-eatonQuartercenturyInbreedingGuineapigs1932" class="csl-entry">
Eaton, Orson N. <span>“A Quarter-Century of Inbreeding in Guinea-Pigs.”</span> <em>Journal of Experimental Zoology</em> 63, no. 2 (1932): 261–90. <a href="https://doi.org/10.1002/jez.1400630202">https://doi.org/10.1002/jez.1400630202</a>.
</div>
<div id="ref-edgeReconstructingHistoryPolygenic2019" class="csl-entry">
Edge, Michael D, and Graham Coop. <span>“Reconstructing the <span>History</span> of <span>Polygenic Scores Using Coalescent Trees</span>.”</span> <em>Genetics</em> 211, no. 1 (January 1, 2019): 235–62. <a href="https://doi.org/10.1534/genetics.118.301687">https://doi.org/10.1534/genetics.118.301687</a>.
</div>
<div id="ref-ellenProspectsSelectionSocial2014" class="csl-entry">
Ellen, Esther D., T. Bas Rodenburg, Gerard A. A. Albers, J. Elizabeth Bolhuis, Irene Camerlink, Naomi Duijvesteijn, Egbert F. Knol, et al. <span>“The Prospects of Selection for Social Genetic Effects to Improve Welfare and Productivity in Livestock.”</span> <em>Frontiers in Genetics</em> 5 (2014). <a href="https://www.frontiersin.org/articles/10.3389/fgene.2014.00377">https://www.frontiersin.org/articles/10.3389/fgene.2014.00377</a>.
</div>
<div id="ref-evansQTLGeneElegans2021" class="csl-entry">
Evans, Kathryn S., Marijke H. van Wijk, Patrick T. McGrath, Erik C. Andersen, and Mark G. Sterken. <span>“From <span>QTL</span> to Gene: <span>C</span>. Elegans Facilitates Discoveries of the Genetic Mechanisms Underlying Natural Variation.”</span> <em>Trends in Genetics</em> 37, no. 10 (October 1, 2021): 933–47. <a href="https://doi.org/10.1016/j.tig.2021.06.005">https://doi.org/10.1016/j.tig.2021.06.005</a>.
</div>
<div id="ref-ewensTransmissionDisequilibriumTest1995" class="csl-entry">
Ewens, W J, and R S Spielman. <span>“The Transmission/Disequilibrium Test: History, Subdivision, and Admixture.”</span> <em>American Journal of Human Genetics</em> 57, no. 2 (August 1995): 455–64. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1801556/">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1801556/</a>.
</div>
<div id="ref-falconerIntroductionQuantitativeGenetics1996" class="csl-entry">
Falconer, D. S., and T. F. C. Mackay. <em>Introduction to Quantitative Genetics</em>. <span>UK: Longman Group</span>, 1996.
</div>
<div id="ref-falkImagingOnsetOscillatory2022" class="csl-entry">
Falk, Henning J, Takehito Tomita, Gregor Mönke, Katie McDole, and Alexander Aulehla. <span>“Imaging the Onset of Oscillatory Signaling Dynamics During Mouse Embryo Gastrulation.”</span> <em>Development (Cambridge, England)</em> 149, no. 13 (July 1, 2022): dev200083. <a href="https://doi.org/10.1242/dev.200083">https://doi.org/10.1242/dev.200083</a>.
</div>
<div id="ref-fisherXVCorrelationRelatives1919" class="csl-entry">
Fisher, R. A. <span>“<span>XV</span>.—<span>The Correlation</span> Between <span>Relatives</span> on the <span>Supposition</span> of <span>Mendelian Inheritance</span>.”</span> January 1, 1919. <a href="https://doi.org/10.1017/s0080456800012163">https://doi.org/10.1017/s0080456800012163</a>.
</div>
<div id="ref-fisherStudiesCropVariation1923" class="csl-entry">
Fisher, R. A., and W. A. Mackenzie. <span>“Studies in Crop Variation. <span>II</span>. <span>The</span> Manurial Response of Different Potato Varieties.”</span> <em>The Journal of Agricultural Science</em> 13, no. 3 (July 1923): 311–20. <a href="https://doi.org/10.1017/S0021859600003592">https://doi.org/10.1017/S0021859600003592</a>.
</div>
<div id="ref-fitzgeraldMedakaInbredKiyosuKarlsruhe2022" class="csl-entry">
Fitzgerald, Tomas, Ian Brettell, Adrien Leger, Nadeshda Wolf, Natalja Kusminski, Jack Monahan, Carl Barton, et al. <span>“The <span>Medaka Inbred Kiyosu-Karlsruhe</span> (<span>MIKK</span>) Panel.”</span> <em>Genome Biology</em> 23, no. 1 (February 21, 2022): 59. <a href="https://doi.org/10.1186/s13059-022-02623-z">https://doi.org/10.1186/s13059-022-02623-z</a>.
</div>
<div id="ref-fredmanComplexSNPrelatedSequence2004" class="csl-entry">
Fredman, David, Stefan J. White, Susanna Potter, Evan E. Eichler, Johan T. Den Dunnen, and Anthony J. Brookes. <span>“Complex <span class="nocase">SNP-related</span> Sequence Variation in Segmental Genome Duplications.”</span> <em>Nature Genetics</em> 36, no. 8, 8 (August 2004): 861–66. <a href="https://doi.org/10.1038/ng1401">https://doi.org/10.1038/ng1401</a>.
</div>
<div id="ref-fryComparisonSociodemographicHealthRelated2017" class="csl-entry">
Fry, Anna, Thomas J Littlejohns, Cathie Sudlow, Nicola Doherty, Ligia Adamska, Tim Sprosen, Rory Collins, and Naomi E Allen. <span>“Comparison of <span>Sociodemographic</span> and <span>Health-Related Characteristics</span> of <span>UK Biobank Participants With Those</span> of the <span>General Population</span>.”</span> <em>American Journal of Epidemiology</em> 186, no. 9 (November 1, 2017): 1026–34. <a href="https://doi.org/10.1093/aje/kwx246">https://doi.org/10.1093/aje/kwx246</a>.
</div>
<div id="ref-fukamachi100YearsMedaka2021" class="csl-entry">
Fukamachi, Shoji, and Kiyoshi Naruse. <span>“100 Years Since the Medaka’s International Debut: <span>Aida</span>’s Legacy.”</span> <span>Genes to Genomes</span>, August 20, 2021. <a href="https://genestogenomes.org/100-years-since-the-medakas-international-debut-aidas-legacy/">https://genestogenomes.org/100-years-since-the-medakas-international-debut-aidas-legacy/</a>.
</div>
<div id="ref-galtonEugenicsItsDefinition1904" class="csl-entry">
Galton, Francis. <span>“Eugenics: <span>Its Definition</span>, <span>Scope</span>, and <span>Aims</span>.”</span> <em>American Journal of Sociology</em> 10, no. 1 (July 1904): 1–25. <a href="https://doi.org/10.1086/211280">https://doi.org/10.1086/211280</a>.
</div>
<div id="ref-galtonMenScienceTheir1874a" class="csl-entry">
———. <span>“On <span>Men</span> of <span>Science</span>, Their <span>Nature</span> and Their <span>Nurture</span>.”</span> In <em>Proceedings of the <span>Royal Institution</span> of <span>Great Britain</span></em>, 7:227–36, 1874.
</div>
<div id="ref-gomezControlSegmentNumber2008" class="csl-entry">
Gomez, C’eline, Ertuğrul M. Özbudak, Joshua Wunderlich, Diana Baumann, Julian Lewis, and Olivier Pourqui’e. <span>“Control of Segment Number in Vertebrate Embryos.”</span> <em>Nature</em> 454, no. 7202, 7202 (July 2008): 335–39. <a href="https://doi.org/10.1038/nature07020">https://doi.org/10.1038/nature07020</a>.
</div>
<div id="ref-gravel2011demographic" class="csl-entry">
Gravel, Simon, Brenna M Henn, Ryan N Gutenkunst, Amit R Indap, Gabor T Marth, Andrew G Clark, Fuli Yu, et al. <span>“Demographic History and Rare Allele Sharing Among Human Populations.”</span> <em>Proceedings of the National Academy of Sciences</em> 108, no. 29 (2011): 11983–88. <a href="https://doi.org/10.1073/pnas.1019276108">https://doi.org/10.1073/pnas.1019276108</a>.
</div>
<div id="ref-gridleyLongShortIt2006" class="csl-entry">
Gridley, Thomas. <span>“The Long and Short of It: <span>Somite</span> Formation in Mice.”</span> <em>Developmental Dynamics</em> 235, no. 9 (2006): 2330–36. <a href="https://doi.org/10.1002/dvdy.20850">https://doi.org/10.1002/dvdy.20850</a>.
</div>
<div id="ref-grosEvolutionEpistasisIts2009" class="csl-entry">
Gros, Pierre-Alexis, Herv’e Le Nagard, and Olivier Tenaillon. <span>“The <span>Evolution</span> of <span>Epistasis</span> and <span>Its Links With Genetic Robustness</span>, <span>Complexity</span> and <span>Drift</span> in a <span>Phenotypic Model</span> of <span>Adaptation</span>.”</span> <em>Genetics</em> 182, no. 1 (May 1, 2009): 277–93. <a href="https://doi.org/10.1534/genetics.108.099127">https://doi.org/10.1534/genetics.108.099127</a>.
</div>
<div id="ref-guCirclizeImplementsEnhances2014" class="csl-entry">
Gu, Zuguang, Lei Gu, Roland Eils, Matthias Schlesner, and Benedikt Brors. <span>“Circlize Implements and Enhances Circular Visualization in <span>R</span>.”</span> <em>Bioinformatics</em> 30, no. 19 (October 1, 2014): 2811–12. <a href="https://doi.org/10.1093/bioinformatics/btu393">https://doi.org/10.1093/bioinformatics/btu393</a>.
</div>
<div id="ref-guoBayesianHierarchicalModel2009" class="csl-entry">
Guo, Feng, Dipak K. Dey, and Kent E. Holsinger. <span>“A <span>Bayesian Hierarchical Model</span> for <span>Analysis</span> of <span>Single-Nucleotide Polymorphisms Diversity</span> in <span>Multilocus</span>, <span>Multipopulation Samples</span>.”</span> <em>Journal of the American Statistical Association</em> 104, no. 485 (March 1, 2009): 142–54. <a href="https://doi.org/10.1198/jasa.2009.0010">https://doi.org/10.1198/jasa.2009.0010</a>.
</div>
<div id="ref-habierImpactGeneticRelationship2010" class="csl-entry">
Habier, David, Jens Tetens, Franz-Reinhold Seefried, Peter Lichtner, and Georg Thaller. <span>“The Impact of Genetic Relationship Information on Genomic Breeding Values in <span>German Holstein</span> Cattle.”</span> <em>Genetics Selection Evolution</em> 42, no. 1 (February 19, 2010): 5. <a href="https://doi.org/10.1186/1297-9686-42-5">https://doi.org/10.1186/1297-9686-42-5</a>.
</div>
<div id="ref-haenelMetaanalysisChromosomescaleCrossover2018" class="csl-entry">
Haenel, Quiterie, Telma G. Laurentino, Marius Roesti, and Daniel Berner. <span>“Meta-Analysis of Chromosome-Scale Crossover Rate Variation in Eukaryotes and Its Significance to Evolutionary Genomics.”</span> <em>Molecular Ecology</em> 27, no. 11 (2018): 2477–97. <a href="https://doi.org/10.1111/mec.14699">https://doi.org/10.1111/mec.14699</a>.
</div>
<div id="ref-hardenGeneticLotteryWhy2021" class="csl-entry">
Harden, Kathryn Paige. <em>The <span>Genetic Lottery</span>: <span>Why DNA Matters</span> for <span>Social Equality</span></em>. <span>Princeton University Press</span>, 2021. <a href="https://books.google.com?id=R9oiEAAAQBAJ">https://books.google.com?id=R9oiEAAAQBAJ</a>.
</div>
<div id="ref-harpakEvolutionGroupDifferences2021" class="csl-entry">
Harpak, Arbel, and Molly Przeworski. <span>“The Evolution of Group Differences in Changing Environments.”</span> <em>PLOS Biology</em> 19, no. 1 (January 25, 2021): e3001072. <a href="https://doi.org/10.1371/journal.pbio.3001072">https://doi.org/10.1371/journal.pbio.3001072</a>.
</div>
<div id="ref-hillLinkageDisequilibriumFinite1968" class="csl-entry">
Hill, W. G., and Alan Robertson. <span>“Linkage Disequilibrium in Finite Populations.”</span> <em>Theoretical and Applied Genetics</em> 38, no. 6 (June 1, 1968): 226–31. <a href="https://doi.org/10.1007/BF01245622">https://doi.org/10.1007/BF01245622</a>.
</div>
<div id="ref-hillsDeletionsGRID2Lead2013" class="csl-entry">
Hills, L. Benjamin, Amira Masri, Kotaro Konno, Wataru Kakegawa, Anh-Thu N. Lam, Elizabeth Lim-Melia, Nandini Chandy, et al. <span>“Deletions in <span>Grid2</span> Lead to a Recessive Syndrome of Cerebellar Ataxia and Tonic Upgaze in Humans.”</span> <em>Neurology</em> 81, no. 16 (October 15, 2013): 1378–86. <a href="https://doi.org/10.1212/WNL.0b013e3182a841a3">https://doi.org/10.1212/WNL.0b013e3182a841a3</a>.
</div>
<div id="ref-HmmlearnHmmlearn2022" class="csl-entry">
<em>Hmmlearn/Hmmlearn</em>. 2014. Reprint, <span>hmmlearn</span>, 2022. <a href="https://github.com/hmmlearn/hmmlearn">https://github.com/hmmlearn/hmmlearn</a>.
</div>
<div id="ref-hoffmannChromosomalInversionPolymorphisms2004" class="csl-entry">
Hoffmann, Ary A., Carla M. Sgr‘o, and Andrew R. Weeks. <span>“Chromosomal Inversion Polymorphisms and Adaptation.”</span> <em>Trends in Ecology &amp; Evolution</em> 19, no. 9 (September 1, 2004): 482–88. <a href="https://doi.org/10.1016/j.tree.2004.06.013">https://doi.org/10.1016/j.tree.2004.06.013</a>.
</div>
<div id="ref-holsingerGeneticsGeographicallyStructured2009" class="csl-entry">
Holsinger, Kent E., and Bruce S. Weir. <span>“Genetics in Geographically Structured Populations: Defining, Estimating and Interpreting <span>FST</span>.”</span> <em>Nature Reviews Genetics</em> 10, no. 9, 9 (September 2009): 639–50. <a href="https://doi.org/10.1038/nrg2611">https://doi.org/10.1038/nrg2611</a>.
</div>
<div id="ref-howeGeneticEvidenceAssortative2019" class="csl-entry">
Howe, Laurence J., Daniel J. Lawson, Neil M. Davies, Beate St. Pourcain, Sarah J. Lewis, George Davey Smith, and Gibran Hemani. <span>“Genetic Evidence for Assortative Mating on Alcohol Consumption in the <span>UK Biobank</span>.”</span> <em>Nature Communications</em> 10, no. 1, 1 (November 19, 2019): 5039. <a href="https://doi.org/10.1038/s41467-019-12424-x">https://doi.org/10.1038/s41467-019-12424-x</a>.
</div>
<div id="ref-hubaudSignallingDynamicsVertebrate2014" class="csl-entry">
Hubaud, Alexis, and Olivier Pourqui’e. <span>“Signalling Dynamics in Vertebrate Segmentation.”</span> <em>Nature Reviews Molecular Cell Biology</em> 15, no. 11, 11 (November 2014): 709–21. <a href="https://doi.org/10.1038/nrm3891">https://doi.org/10.1038/nrm3891</a>.
</div>
<div id="ref-huxleyEvolutionModernSynthesis1942" class="csl-entry">
Huxley, Julian. <span>“Evolution. <span>The</span> Modern Synthesis.”</span> <em>Evolution. The Modern Synthesis.</em>, 1942.
</div>
<div id="ref-hwangEstimatingIndirectParental2020" class="csl-entry">
Hwang, Liang-Dar, Justin D. Tubbs, Justin Luong, Mischa Lundberg, Gunn-Helen Moen, Geng Wang, Nicole M. Warrington, Pak C. Sham, Gabriel Cuellar-Partida, and David M. Evans. <span>“Estimating Indirect Parental Genetic Effects on Offspring Phenotypes Using Virtual Parental Genotypes Derived from Sibling and Half Sibling Pairs.”</span> <em>PLOS Genetics</em> 16, no. 10 (October 26, 2020): e1009154. <a href="https://doi.org/10.1371/journal.pgen.1009154">https://doi.org/10.1371/journal.pgen.1009154</a>.
</div>
<div id="ref-IndexPubRelease102" class="csl-entry">
<span>“Index of /Pub/Release-102/Emf/Ensembl-Compara/Multiple_alignments/50_fish.epo/.”</span> Accessed January 25, 2022. <a href="https://ftp.ensembl.org/pub/release-102/emf/ensembl-compara/multiple_alignments/50_fish.epo/">https://ftp.ensembl.org/pub/release-102/emf/ensembl-compara/multiple_alignments/50_fish.epo/</a>.
</div>
<div id="ref-IndexVol1Ftp" class="csl-entry">
<span>“Index of /Vol1/Ftp/Data_collections/<span>1000g</span>_2504_high_coverage/Working/20201028_3202_phased/.”</span> Accessed March 24, 2022. <a href="http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/data_collections/1000G_2504_high_coverage/working/20201028_3202_phased/">http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/data_collections/1000G_2504_high_coverage/working/20201028_3202_phased/</a>.
</div>
<div id="ref-jamainMutationsXlinkedGenes2003" class="csl-entry">
Jamain, St’ephane, H’el‘ene Quach, Catalina Betancur, Maria Råstam, Catherine Colineaux, I. Carina Gillberg, Henrik Soderstrom, et al. <span>“Mutations of the <span class="nocase">X-linked</span> Genes Encoding Neuroligins <span>Nlgn3</span> and <span>Nlgn4</span> Are Associated with Autism.”</span> <em>Nature Genetics</em> 34, no. 1, 1 (May 2003): 27–29. <a href="https://doi.org/10.1038/ng1136">https://doi.org/10.1038/ng1136</a>.
</div>
<div id="ref-johnsonSegregationPerformanceRecombinant1999" class="csl-entry">
Johnson, William C., and Paul Gepts. <span>“Segregation for Performance in Recombinant Inbred Populations Resulting from Inter-Gene Pool Crosses of Common Bean ( <span>Phaseolus</span> Vulgaris <span>L</span>.).”</span> <em>Euphytica</em> 106, no. 1 (March 1, 1999): 45–56. <a href="https://doi.org/10.1023/A:1003541201923">https://doi.org/10.1023/A:1003541201923</a>.
</div>
<div id="ref-junghansPostsynapticDifferentialLocalization2008" class="csl-entry">
Junghans, Dirk, Matthias Heidenreich, Iris Hack, Verdon Taylor, Michael Frotscher, and Rolf Kemler. <span>“Postsynaptic and Differential Localization to Neuronal Subtypes of Protocadherin Β16 in the Mammalian Central Nervous System.”</span> <em>European Journal of Neuroscience</em> 27, no. 3 (2008): 559–71. <a href="https://doi.org/10.1111/j.1460-9568.2008.06052.x">https://doi.org/10.1111/j.1460-9568.2008.06052.x</a>.
</div>
<div id="ref-katsumuraMedakaPopulationGenome2019" class="csl-entry">
Katsumura, Takafumi, Shoji Oda, Hiroshi Mitani, and Hiroki Oota. <span>“Medaka <span>Population Genome Structure</span> and <span>Demographic History Described</span> via <span class="nocase">Genotyping-by-Sequencing</span>.”</span> <em>G3 Genes|Genomes|Genetics</em> 9, no. 1 (January 1, 2019): 217–28. <a href="https://doi.org/10.1534/g3.118.200779">https://doi.org/10.1534/g3.118.200779</a>.
</div>
<div id="ref-kerminenGeographicVariationBias2019" class="csl-entry">
Kerminen, Sini, Alicia R. Martin, Jukka Koskela, Sanni E. Ruotsalainen, Aki S. Havulinna, Ida Surakka, Aarno Palotie, et al. <span>“Geographic <span>Variation</span> and <span>Bias</span> in the <span>Polygenic Scores</span> of <span>Complex Diseases</span> and <span>Traits</span> in <span>Finland</span>.”</span> <em>The American Journal of Human Genetics</em> 104, no. 6 (June 6, 2019): 1169–81. <a href="https://doi.org/10.1016/j.ajhg.2019.05.001">https://doi.org/10.1016/j.ajhg.2019.05.001</a>.
</div>
<div id="ref-kevlesNameEugenicsGenetics1995" class="csl-entry">
Kevles, Daniel J. <em>In the <span>Name</span> of <span>Eugenics</span>: <span>Genetics</span> and the <span>Uses</span> of <span>Human Heredity</span></em>. <span>Harvard University Press</span>, 1995. <a href="https://books.google.com?id=NWBi9kyb8xUC">https://books.google.com?id=NWBi9kyb8xUC</a>.
</div>
<div id="ref-khannaBamreadcountRapidGeneration2022" class="csl-entry">
Khanna, Ajay, David E. Larson, Sridhar Nonavinkere Srivatsan, Matthew Mosior, Travis E. Abbott, Susanna Kiwala, Timothy J. Ley, et al. <span>“Bam-Readcount - Rapid Generation of Basepair-Resolution Sequence Metrics.”</span> <em>Journal of Open Source Software</em> 7, no. 69 (January 29, 2022): 3722. <a href="https://doi.org/10.21105/joss.03722">https://doi.org/10.21105/joss.03722</a>.
</div>
<div id="ref-kheraGenomewidePolygenicScores2018" class="csl-entry">
Khera, Amit V., Mark Chaffin, Krishna G. Aragam, Mary E. Haas, Carolina Roselli, Seung Hoan Choi, Pradeep Natarajan, et al. <span>“Genome-Wide Polygenic Scores for Common Diseases Identify Individuals with Risk Equivalent to Monogenic Mutations.”</span> <em>Nature Genetics</em> 50, no. 9, 9 (September 2018): 1219–24. <a href="https://doi.org/10.1038/s41588-018-0183-z">https://doi.org/10.1038/s41588-018-0183-z</a>.
</div>
<div id="ref-kimPeriodSomiteSegmentation2011" class="csl-entry">
Kim, Woong, Takaaki Matsui, Masataka Yamao, Makoto Ishibashi, Kota Tamada, Toru Takumi, Kenji Kohno, et al. <span>“The Period of the Somite Segmentation Clock Is Sensitive to <span>Notch</span> Activity.”</span> <em>Molecular Biology of the Cell</em> 22, no. 18 (September 15, 2011): 3541–49. <a href="https://doi.org/10.1091/mbc.e11-02-0139">https://doi.org/10.1091/mbc.e11-02-0139</a>.
</div>
<div id="ref-knopikBehavioralGenetics2019" class="csl-entry">
Knopik, Valerie S., Jenae M. Neiderhiser, John C. DeFries, and Robert Plomin. <em>Behavioral <span>Genetics</span></em>. <span>Macmillan Learning</span>, 2019. <a href="https://books.google.com?id=mtDJvQEACAAJ">https://books.google.com?id=mtDJvQEACAAJ</a>.
</div>
<div id="ref-kongNatureNurtureEffects2018" class="csl-entry">
Kong, Augustine, Gudmar Thorleifsson, Michael L. Frigge, Bjarni J. Vilhjalmsson, Alexander I. Young, Thorgeir E. Thorgeirsson, Stefania Benonisdottir, et al. <span>“The Nature of Nurture: <span>Effects</span> of Parental Genotypes.”</span> <em>Science</em> 359, no. 6374 (January 26, 2018): 424–28. <a href="https://doi.org/10.1126/science.aan6877">https://doi.org/10.1126/science.aan6877</a>.
</div>
<div id="ref-kremerDecouplingDifferentiationTraits2012" class="csl-entry">
Kremer, A., and V. Le Corre. <span>“Decoupling of Differentiation Between Traits and Their Underlying Genes in Response to Divergent Selection.”</span> <em>Heredity</em> 108, no. 4, 4 (April 2012): 375–85. <a href="https://doi.org/10.1038/hdy.2011.81">https://doi.org/10.1038/hdy.2011.81</a>.
</div>
<div id="ref-kulloIncorporatingGeneticRisk2016" class="csl-entry">
Kullo, Iftikhar J., Hayan Jouni, Erin E. Austin, Sherry-Ann Brown, Teresa M. Kruisselbrink, Iyad N. Isseh, Raad A. Haddad, et al. <span>“Incorporating a <span>Genetic Risk Score Into Coronary Heart Disease Risk Estimates</span>.”</span> <em>Circulation</em> 133, no. 12 (March 22, 2016): 1181–88. <a href="https://doi.org/10.1161/CIRCULATIONAHA.115.020109">https://doi.org/10.1161/CIRCULATIONAHA.115.020109</a>.
</div>
<div id="ref-lopez-olmedaLongPhotoperiodImpairs2021" class="csl-entry">
L’opez-Olmeda, Jose Fernando, Haiyu Zhao, Markus Reischl, Christian Pylatiuk, Tyrone Lucon-Xiccato, Felix Loosli, and Nicholas S. Foulkes. <span>“Long Photoperiod Impairs Learning in Male but Not Female Medaka.”</span> <em>iScience</em> 24, no. 7 (July 23, 2021): 102784. <a href="https://doi.org/10.1016/j.isci.2021.102784">https://doi.org/10.1016/j.isci.2021.102784</a>.
</div>
<div id="ref-lalandFishCognitionBehavior2011" class="csl-entry">
Laland, Kevin, Jens Krause, and Culum Brown. <em>Fish <span>Cognition</span> and <span>Behavior</span></em>. <span>John Wiley &amp; Sons</span>, 2011. <a href="https://books.google.com?id=cI9gbVyH6lsC">https://books.google.com?id=cI9gbVyH6lsC</a>.
</div>
<div id="ref-lecorreGeneticDifferentiationQuantitative2012" class="csl-entry">
Le Corre, Val’erie, and Antoine Kremer. <span>“The Genetic Differentiation at Quantitative Trait Loci Under Local Adaptation.”</span> <em>Molecular Ecology</em> 21, no. 7 (2012): 1548–66. <a href="https://doi.org/10.1111/j.1365-294X.2012.05479.x">https://doi.org/10.1111/j.1365-294X.2012.05479.x</a>.
</div>
<div id="ref-legerGenomicVariationsEpigenomic2022" class="csl-entry">
Leger, Adrien, Ian Brettell, Jack Monahan, Carl Barton, Nadeshda Wolf, Natalja Kusminski, Cathrin Herder, et al. <span>“Genomic Variations and Epigenomic Landscape of the <span>Medaka Inbred Kiyosu-Karlsruhe</span> (<span>MIKK</span>) Panel.”</span> <em>Genome Biology</em> 23, no. 1 (February 21, 2022): 58. <a href="https://doi.org/10.1186/s13059-022-02602-4">https://doi.org/10.1186/s13059-022-02602-4</a>.
</div>
<div id="ref-lekAnalysisProteincodingGenetic2016" class="csl-entry">
Lek, Monkol, Konrad J. Karczewski, Eric V. Minikel, Kaitlin E. Samocha, Eric Banks, Timothy Fennell, Anne H. O’NADonnell-Luria, et al. <span>“Analysis of Protein-Coding Genetic Variation in 60,706 Humans.”</span> <em>Nature</em> 536, no. 7616, 7616 (August 2016): 285–91. <a href="https://doi.org/10.1038/nature19057">https://doi.org/10.1038/nature19057</a>.
</div>
<div id="ref-liSequenceAlignmentMap2009" class="csl-entry">
Li, Heng, Bob Handsaker, Alec Wysoker, Tim Fennell, Jue Ruan, Nils Homer, Gabor Marth, Goncalo Abecasis, Richard Durbin, and 1000 Genome Project Data Processing Subgroup. <span>“The <span>Sequence</span> Alignment/Map (<span>SAM</span>) Format and <span>SAMtools</span>.”</span> <em>Bioinformatics</em> 25, no. 16 (2009): 2078–79.
</div>
<div id="ref-limaBehavioralDecisionsMade1990" class="csl-entry">
Lima, Steven L., and Lawrence M. Dill. <span>“Behavioral Decisions Made Under the Risk of Predation: A Review and Prospectus.”</span> <em>Canadian Journal of Zoology</em> 68, no. 4 (April 1990): 619–40. <a href="https://doi.org/10.1139/z90-092">https://doi.org/10.1139/z90-092</a>.
</div>
<div id="ref-limamiGeneticPhysiologicalAnalysis2002" class="csl-entry">
Limami, Anis M., Clothilde Rouillon, Gaëlle Glevarec, Andr’e Gallais, and Bertrand Hirel. <span>“Genetic and <span>Physiological Analysis</span> of <span>Germination Efficiency</span> in <span>Maize</span> in <span>Relation</span> to <span>Nitrogen Metabolism Reveals</span> the <span>Importance</span> of <span>Cytosolic Glutamine Synthetase</span>.”</span> <em>Plant Physiology</em> 130, no. 4 (December 1, 2002): 1860–70. <a href="https://doi.org/10.1104/pp.009647">https://doi.org/10.1104/pp.009647</a>.
</div>
<div id="ref-lucon-xiccatoIndividualDifferencesCognition2017" class="csl-entry">
Lucon-Xiccato, Tyrone, and Angelo Bisazza. <span>“Individual Differences in Cognition Among Teleost Fishes.”</span> <em>Behavioural Processes</em>, The <span>Cognition</span> of <span>Fish</span>, 141 (August 1, 2017): 184–95. <a href="https://doi.org/10.1016/j.beproc.2017.01.015">https://doi.org/10.1016/j.beproc.2017.01.015</a>.
</div>
<div id="ref-lucon-xiccatoDevelopmentOpenFieldBehaviour2020" class="csl-entry">
Lucon-Xiccato, Tyrone, Francesca Conti, Felix Loosli, Nicholas S. Foulkes, and Cristiano Bertolucci. <span>“Development of <span>Open-Field Behaviour</span> in the <span>Medaka</span>, <span>Oryzias</span> Latipes.”</span> <em>Biology</em> 9, no. 11, 11 (November 2020): 389. <a href="https://doi.org/10.3390/biology9110389">https://doi.org/10.3390/biology9110389</a>.
</div>
<div id="ref-lucon-xiccatoComparisonAnxietylikeSocial2022" class="csl-entry">
Lucon-Xiccato, Tyrone, Felix Loosli, Francesca Conti, Nicholas S. Foulkes, and Cristiano Bertolucci. <span>“Comparison of Anxiety-Like and Social Behaviour in Medaka and Zebrafish.”</span> <em>Scientific Reports</em> 12, no. 1, 1 (June 28, 2022): 10926. <a href="https://doi.org/10.1038/s41598-022-14978-1">https://doi.org/10.1038/s41598-022-14978-1</a>.
</div>
<div id="ref-maasBreastCancerRisk2016" class="csl-entry">
Maas, Paige, Myrto Barrdahl, Amit D. Joshi, Paul L. Auer, Mia M. Gaudet, Roger L. Milne, Fredrick R. Schumacher, et al. <span>“Breast <span>Cancer Risk From Modifiable</span> and <span>Nonmodifiable Risk Factors Among White Women</span> in the <span>United States</span>.”</span> <em>JAMA Oncology</em> 2, no. 10 (October 1, 2016): 1295–1302. <a href="https://doi.org/10.1001/jamaoncol.2016.1025">https://doi.org/10.1001/jamaoncol.2016.1025</a>.
</div>
<div id="ref-macarthurNewNHGRIEBICatalog2017" class="csl-entry">
MacArthur, Jacqueline, Emily Bowler, Maria Cerezo, Laurent Gil, Peggy Hall, Emma Hastings, Heather Junkins, et al. <span>“The New <span>NHGRI-EBI Catalog</span> of Published Genome-Wide Association Studies (<span>GWAS Catalog</span>).”</span> <em>Nucleic Acids Research</em> 45, no. D1 (January 4, 2017): D896–901. <a href="https://doi.org/10.1093/nar/gkw1133">https://doi.org/10.1093/nar/gkw1133</a>.
</div>
<div id="ref-mackayChartingGenotypePhenotype2018" class="csl-entry">
Mackay, Trudy F. C., and Wen Huang. <span>“Charting the Genotype–Phenotype Map: Lessons from the <span>Drosophila</span> Melanogaster <span>Genetic Reference Panel</span>.”</span> <em>WIREs Developmental Biology</em> 7, no. 1 (2018): e289. <a href="https://doi.org/10.1002/wdev.289">https://doi.org/10.1002/wdev.289</a>.
</div>
<div id="ref-magnoGwasrapiddPackageQuery2020" class="csl-entry">
Magno, Ramiro, and Ana-Teresa Maia. <span>“Gwasrapidd: An <span>R</span> Package to Query, Download and Wrangle <span>GWAS</span> Catalog Data.”</span> <em>Bioinformatics</em> 36, no. 2 (January 15, 2020): 649–50. <a href="https://doi.org/10.1093/bioinformatics/btz605">https://doi.org/10.1093/bioinformatics/btz605</a>.
</div>
<div id="ref-malecotMathematiquesHeredite1948" class="csl-entry">
Mal’ecot, Gustave. <span>“Mathématiques de l’hérédité,”</span> 1948.
</div>
<div id="ref-manolioFindingMissingHeritability2009" class="csl-entry">
Manolio, Teri A., Francis S. Collins, Nancy J. Cox, David B. Goldstein, Lucia A. Hindorff, David J. Hunter, Mark I. McCarthy, et al. <span>“Finding the Missing Heritability of Complex Diseases.”</span> <em>Nature</em> 461, no. 7265, 7265 (October 2009): 747–53. <a href="https://doi.org/10.1038/nature08494">https://doi.org/10.1038/nature08494</a>.
</div>
<div id="ref-martinHumanDemographicHistory2017" class="csl-entry">
Martin, Alicia R, Christopher R Gignoux, Raymond K Walters, Genevieve L Wojcik, Benjamin M Neale, Simon Gravel, Mark J Daly, Carlos D Bustamante, and Eimear E Kenny. <span>“Human Demographic History Impacts Genetic Risk Prediction Across Diverse Populations.”</span> <em>The American Journal of Human Genetics</em> 100, no. 4 (2017): 635–49. <a href="https://doi.org/10.1016/j.ajhg.2017.03.004">https://doi.org/10.1016/j.ajhg.2017.03.004</a>.
</div>
<div id="ref-martinClinicalUseCurrent2019" class="csl-entry">
Martin, Alicia R., Masahiro Kanai, Yoichiro Kamatani, Yukinori Okada, Benjamin M. Neale, and Mark J. Daly. <span>“Clinical Use of Current Polygenic Risk Scores May Exacerbate Health Disparities.”</span> <em>Nature Genetics</em> 51, no. 4, 4 (April 2019): 584–91. <a href="https://doi.org/10.1038/s41588-019-0379-x">https://doi.org/10.1038/s41588-019-0379-x</a>.
</div>
<div id="ref-martinCriticalNeedsChallenges2018" class="csl-entry">
Martin, Alicia R, Solomon Teferra, Marlo Möller, Eileen G Hoal, and Mark J Daly. <span>“The Critical Needs and Challenges for Genetic Architecture Studies in <span>Africa</span>.”</span> <em>Current Opinion in Genetics &amp; Development</em>, Genetics of <span>Human Origins</span>, 53 (December 1, 2018): 113–20. <a href="https://doi.org/10.1016/j.gde.2018.08.005">https://doi.org/10.1016/j.gde.2018.08.005</a>.
</div>
<div id="ref-martinSimonhmartinGenomicsGeneral2022" class="csl-entry">
martin, Simon. <em>Simonhmartin/Genomics_general</em>, 2022. <a href="https://github.com/simonhmartin/genomics_general">https://github.com/simonhmartin/genomics_general</a>.
</div>
<div id="ref-martinEvaluatingUseABBA2015" class="csl-entry">
Martin, Simon H., John W. Davey, and Chris D. Jiggins. <span>“Evaluating the <span>Use</span> of <span>ABBA</span>–<span>BABA Statistics</span> to <span>Locate Introgressed Loci</span>.”</span> <em>Molecular Biology and Evolution</em> 32, no. 1 (January 1, 2015): 244–57. <a href="https://doi.org/10.1093/molbev/msu269">https://doi.org/10.1093/molbev/msu269</a>.
</div>
<div id="ref-matsudaSpeciesspecificSegmentationClock2020" class="csl-entry">
Matsuda, Mitsuhiro, Hanako Hayashi, Jordi Garcia-Ojalvo, Kumiko Yoshioka-Kobayashi, Ryoichiro Kageyama, Yoshihiro Yamanaka, Makoto Ikeya, Junya Toguchida, Cantas Alev, and Miki Ebisuya. <span>“Species-Specific Segmentation Clock Periods Are Due to Differential Biochemical Reaction Speeds.”</span> <em>Science</em> 369, no. 6510 (2020): 1450–55.
</div>
<div id="ref-matsunagaHabituationMedakaOryzias2010" class="csl-entry">
Matsunaga, Wataru, and Eiji Watanabe. <span>“Habituation of Medaka (<span>Oryzias</span> Latipes) Demonstrated by Open-Field Testing.”</span> <em>Behavioural Processes</em> 85, no. 2 (October 1, 2010): 142–50. <a href="https://doi.org/10.1016/j.beproc.2010.06.019">https://doi.org/10.1016/j.beproc.2010.06.019</a>.
</div>
<div id="ref-mazumdarEugenicsHumanGenetics2005" class="csl-entry">
Mazumdar, Pauline. <em>Eugenics, Human Genetics and Human Failings: The <span>Eugenics Society</span>, Its Sources and Its Critics in <span>Britain</span></em>. <span>Routledge</span>, 2005.
</div>
<div id="ref-mccarthyReferencePanel642016" class="csl-entry">
McCarthy, Shane, Sayantan Das, Warren Kretzschmar, Olivier Delaneau, Andrew R Wood, Alexander Teumer, Hyun Min Kang, et al. <span>“A Reference Panel of 64,976 Haplotypes for Genotype Imputation.”</span> <em>Nature Genetics</em> 48, no. 10, 10 (October 2016): 1279–83. <a href="https://doi.org/10.1038/ng.3643">https://doi.org/10.1038/ng.3643</a>.
</div>
<div id="ref-mckennaGenomeAnalysisToolkit2010" class="csl-entry">
McKenna, Aaron, Matthew Hanna, Eric Banks, Andrey Sivachenko, Kristian Cibulskis, Andrew Kernytsky, Kiran Garimella, et al. <span>“The <span>Genome Analysis Toolkit</span>: <span>A MapReduce</span> Framework for Analyzing Next-Generation <span>DNA</span> Sequencing Data.”</span> <em>Genome Research</em> 20, no. 9 (January 9, 2010): 1297–1303. <a href="https://doi.org/10.1101/gr.107524.110">https://doi.org/10.1101/gr.107524.110</a>.
</div>
<div id="ref-mclarenEnsemblVariantEffect2016" class="csl-entry">
McLaren, William, Laurent Gil, Sarah E. Hunt, Harpreet Singh Riat, Graham R. S. Ritchie, Anja Thormann, Paul Flicek, and Fiona Cunningham. <span>“The <span>Ensembl Variant Effect Predictor</span>.”</span> <em>Genome Biology</em> 17, no. 1 (June 6, 2016): 122. <a href="https://doi.org/10.1186/s13059-016-0974-4">https://doi.org/10.1186/s13059-016-0974-4</a>.
</div>
<div id="ref-membersofthecomplextraitconsortiumNatureIdentificationQuantitative2003" class="csl-entry">
Members of the Complex Trait Consortium. <span>“The Nature and Identification of Quantitative Trait Loci: A Community’s View.”</span> <em>Nature Reviews Genetics</em> 4, no. 11, 11 (November 2003): 911–16. <a href="https://doi.org/10.1038/nrg1206">https://doi.org/10.1038/nrg1206</a>.
</div>
<div id="ref-mendelVersucheUberPflanzenhybriden1866" class="csl-entry">
Mendel, Gregor. <span>“Versuche Uber Pflanzen-Hybriden.”</span> <em>Verhandlungen Des Naturforschenden Vereins in Brunn Fur</em> 4 (1866): 3–47.
</div>
<div id="ref-morrisPopulationPhenomenaInflate2020" class="csl-entry">
Morris, Tim T., Neil M. Davies, Gibran Hemani, and George Davey Smith. <span>“Population Phenomena Inflate Genetic Associations of Complex Social Traits.”</span> <em>Science Advances</em> 6, no. 16 (April 15, 2020): eaay0328. <a href="https://doi.org/10.1126/sciadv.aay0328">https://doi.org/10.1126/sciadv.aay0328</a>.
</div>
<div id="ref-mostafaviVariablePredictionAccuracy2020" class="csl-entry">
Mostafavi, Hakhamanesh, Arbel Harpak, Ipsita Agarwal, Dalton Conley, Jonathan K Pritchard, and Molly Przeworski. <span>“Variable Prediction Accuracy of Polygenic Scores Within an Ancestry Group.”</span> Edited by Ruth Loos, Michael B Eisen, and Paul O’Reilly. <em>eLife</em> 9 (January 30, 2020): e48376. <a href="https://doi.org/10.7554/eLife.48376">https://doi.org/10.7554/eLife.48376</a>.
</div>
<div id="ref-mukherjeeGeneIntimateHistory2016" class="csl-entry">
Mukherjee, Siddhartha. <em>The <span>Gene</span>: <span>An Intimate History</span></em>. <span>Simon and Schuster</span>, 2016. <a href="https://books.google.com?id=XvAsDAAAQBAJ">https://books.google.com?id=XvAsDAAAQBAJ</a>.
</div>
<div id="ref-naruseDetailedLinkageMap2000" class="csl-entry">
Naruse, Kiyoshi, Shoji Fukamachi, Hiroshi Mitani, Mariko Kondo, Tomoko Matsuoka, Shu Kondo, Nana Hanamura, et al. <span>“A <span>Detailed Linkage Map</span> of <span>Medaka</span>, <span>Oryzias</span> Latipes: <span>Comparative Genomics</span> and <span>Genome Evolution</span>.”</span> <em>Genetics</em> 154, no. 4 (April 1, 2000): 1773–84. <a href="https://www.genetics.org/content/154/4/1773">https://www.genetics.org/content/154/4/1773</a>.
</div>
<div id="ref-naruseMedakaModelOrganogenesis2011a" class="csl-entry">
Naruse, Kiyoshi, Minoru Tanaka, and Hiroyuki Takeda. <em>Medaka: A Model for Organogenesis, Human Disease, and Evolution</em>. <span>Springer Science &amp; Business Media</span>, 2011.
</div>
<div id="ref-natarajanPolygenicRiskScore2017" class="csl-entry">
Natarajan, Pradeep, Robin Young, Nathan O. Stitziel, Sandosh Padmanabhan, Usman Baber, Roxana Mehran, Samantha Sartori, et al. <span>“Polygenic <span>Risk Score Identifies Subgroup With Higher Burden</span> of <span>Atherosclerosis</span> and <span>Greater Relative Benefit From Statin Therapy</span> in the <span>Primary Prevention Setting</span>.”</span> <em>Circulation</em> 135, no. 22 (May 30, 2017): 2091–101. <a href="https://doi.org/10.1161/CIRCULATIONAHA.116.024436">https://doi.org/10.1161/CIRCULATIONAHA.116.024436</a>.
</div>
<div id="ref-nurkCompleteSequenceHuman2022" class="csl-entry">
Nurk, Sergey, Sergey Koren, Arang Rhie, Mikko Rautiainen, Andrey V. Bzikadze, Alla Mikheenko, Mitchell R. Vollger, et al. <span>“The Complete Sequence of a Human Genome.”</span> <em>Science</em> 376, no. 6588 (April 2022): 44–53. <a href="https://doi.org/10.1126/science.abj6987">https://doi.org/10.1126/science.abj6987</a>.
</div>
<div id="ref-okbayPolygenicPredictionEducational2022" class="csl-entry">
Okbay, Aysu, Yeda Wu, Nancy Wang, Hariharan Jayashankar, Michael Bennett, Seyed Moeen Nehzati, Julia Sidorenko, et al. <span>“Polygenic Prediction of Educational Attainment Within and Between Families from Genome-Wide Association Analyses in 3 Million Individuals.”</span> <em>Nature Genetics</em>, March 31, 2022, 1–13. <a href="https://doi.org/10.1038/s41588-022-01016-z">https://doi.org/10.1038/s41588-022-01016-z</a>.
</div>
<div id="ref-panagiotouWhatShouldGenomewide2012" class="csl-entry">
Panagiotou, Orestis A., John P. A. Ioannidis, and Genome-Wide Significance Project. <span>“What Should the Genome-Wide Significance Threshold Be? <span>Empirical</span> Replication of Borderline Genetic Associations.”</span> <em>International Journal of Epidemiology</em> 41, no. 1 (February 2012): 273–86. <a href="https://doi.org/10.1093/ije/dyr178">https://doi.org/10.1093/ije/dyr178</a>.
</div>
<div id="ref-paquettePolygenicRiskScore2017" class="csl-entry">
Paquette, Martine, Michael Chong, S’ebastien Th’eriault, Robert Dufour, Guillaume Par’e, and Alexis Baass. <span>“Polygenic Risk Score Predicts Prevalence of Cardiovascular Disease in Patients with Familial Hypercholesterolemia.”</span> <em>Journal of Clinical Lipidology</em> 11, no. 3 (May 1, 2017): 725–732.e5. <a href="https://doi.org/10.1016/j.jacl.2017.03.019">https://doi.org/10.1016/j.jacl.2017.03.019</a>.
</div>
<div id="ref-paradisPegasPackagePopulation2010" class="csl-entry">
Paradis, Emmanuel. <span>“Pegas: An <span>R</span> Package for Population Genetics with an Integrated–Modular Approach.”</span> <em>Bioinformatics</em> 26, no. 3 (February 1, 2010): 419–20. <a href="https://doi.org/10.1093/bioinformatics/btp696">https://doi.org/10.1093/bioinformatics/btp696</a>.
</div>
<div id="ref-paradisApeEnvironmentModern2019" class="csl-entry">
Paradis, Emmanuel, and Klaus Schliep. <span>“Ape 5.0: An Environment for Modern Phylogenetics and Evolutionary Analyses in <span>R</span>.”</span> <em>Bioinformatics</em> 35, no. 3 (February 1, 2019): 526–28. <a href="https://doi.org/10.1093/bioinformatics/bty633">https://doi.org/10.1093/bioinformatics/bty633</a>.
</div>
<div id="ref-parkDistributionAlleleFrequencies2011" class="csl-entry">
Park, Ju-Hyun, Mitchell H. Gail, Clarice R. Weinberg, Raymond J. Carroll, Charles C. Chung, Zhaoming Wang, Stephen J. Chanock, Joseph F. Fraumeni, and Nilanjan Chatterjee. <span>“Distribution of Allele Frequencies and Effect Sizes and Their Interrelationships for Common Genetic Susceptibility Variants.”</span> <em>Proceedings of the National Academy of Sciences</em> 108, no. 44 (November 2011): 18026–31. <a href="https://doi.org/10.1073/pnas.1114759108">https://doi.org/10.1073/pnas.1114759108</a>.
</div>
<div id="ref-payneBulkVisGraphicalViewer2019" class="csl-entry">
Payne, Alexander, Nadine Holmes, Vardhman Rakyan, and Matthew Loose. <span>“<span>BulkVis</span>: A Graphical Viewer for <span>Oxford</span> Nanopore Bulk <span>Fast5</span> Files.”</span> <em>Bioinformatics</em> 35, no. 13 (July 1, 2019): 2193–98. <a href="https://doi.org/10.1093/bioinformatics/bty841">https://doi.org/10.1093/bioinformatics/bty841</a>.
</div>
<div id="ref-pearsonLIIILinesPlanes1901" class="csl-entry">
Pearson, Karl. <span>“<span>LIII</span>. <span>On</span> Lines and Planes of Closest Fit to Systems of Points in Space.”</span> <em>The London, Edinburgh, and Dublin Philosophical Magazine and Journal of Science</em> 2, no. 11 (November 1, 1901): 559–72. <a href="https://doi.org/10.1080/14786440109462720">https://doi.org/10.1080/14786440109462720</a>.
</div>
<div id="ref-pearsonMathematicalContributionsTheory1898" class="csl-entry">
———. <span>“Mathematical Contributions to the Theory of Evolution, <span>On</span> the Law of Ancestral Heredity.”</span> <em>Proceedings of the Royal Society of London</em> 62, no. 379-387 (January 1898): 386–412. <a href="https://doi.org/10.1098/rspl.1897.0128">https://doi.org/10.1098/rspl.1897.0128</a>.
</div>
<div id="ref-pearsonLifeLettersLabours2011" class="csl-entry">
———. <em>The <span>Life</span>, <span>Letters</span> and <span>Labours</span> of <span>Francis Galton</span></em>. <span>Cambridge University Press</span>, 2011. <a href="https://books.google.com?id=sz7vSOwOGzgC">https://books.google.com?id=sz7vSOwOGzgC</a>.
</div>
<div id="ref-pearsonCriterionThatGiven1900" class="csl-entry">
———. <span>“X. <span>On</span> the Criterion That a Given System of Deviations from the Probable in the Case of a Correlated System of Variables Is Such That It Can Be Reasonably Supposed to Have Arisen from Random Sampling.”</span> <em>The London, Edinburgh, and Dublin Philosophical Magazine and Journal of Science</em> 50, no. 302 (July 1, 1900): 157–75. <a href="https://doi.org/10.1080/14786440009463897">https://doi.org/10.1080/14786440009463897</a>.
</div>
<div id="ref-pearsonIIIContributionsMathematical1894" class="csl-entry">
Pearson, Karl, and Olaus Magnus Friedrich Erdmann Henrici. <span>“<span>III</span>. <span>Contributions</span> to the Mathematical Theory of Evolution.”</span> <em>Philosophical Transactions of the Royal Society of London. (A.)</em> 185 (January 1894): 71–110. <a href="https://doi.org/10.1098/rsta.1894.0003">https://doi.org/10.1098/rsta.1894.0003</a>.
</div>
<div id="ref-peirceNewSetBXD2004" class="csl-entry">
Peirce, Jeremy L., Lu Lu, Jing Gu, Lee M. Silver, and Robert W. Williams. <span>“A New Set of <span>BXD</span> Recombinant Inbred Lines from Advanced Intercross Populations in Mice.”</span> <em>BMC Genetics</em> 5, no. 1 (April 29, 2004): 7. <a href="https://doi.org/10.1186/1471-2156-5-7">https://doi.org/10.1186/1471-2156-5-7</a>.
</div>
<div id="ref-penceEarlyHistoryChance2015" class="csl-entry">
Pence, Charles H. <span>“The Early History of Chance in Evolution.”</span> <em>Studies in History and Philosophy of Science Part A</em>, Integrated <span>History</span> and <span>Philosophy</span> of <span>Science</span> in <span>Practice</span>, 50 (April 1, 2015): 48–58. <a href="https://doi.org/10.1016/j.shpsa.2014.09.006">https://doi.org/10.1016/j.shpsa.2014.09.006</a>.
</div>
<div id="ref-pennisi100GenomeNew2022" class="csl-entry">
Pennisi, Elizabeth. <span>“A $100 Genome? <span>New DNA</span> Sequencers Could Be a <span>‘Game Changer’</span> for Biology, Medicine | <span>Science</span> | <span>AAAS</span>.”</span> <em>Science</em>, June 15, 2022. <a href="https://www.science.org/content/article/100-genome-new-dna-sequencers-could-be-game-changer-biology-medicine">https://www.science.org/content/article/100-genome-new-dna-sequencers-could-be-game-changer-biology-medicine</a>.
</div>
<div id="ref-phillipsGProtocadherinsAreTargeted2003" class="csl-entry">
Phillips, Greg R., Hidekazu Tanaka, Marcus Frank, Alice Elste, Lazar Fidler, Deanna L. Benson, and David R. Colman. <span>“Γ-<span>Protocadherins Are Targeted</span> to <span>Subsets</span> of <span>Synapses</span> and <span>Intracellular Organelles</span> in <span>Neurons</span>.”</span> <em>Journal of Neuroscience</em> 23, no. 12 (June 15, 2003): 5096–5104. <a href="https://doi.org/10.1523/JNEUROSCI.23-12-05096.2003">https://doi.org/10.1523/JNEUROSCI.23-12-05096.2003</a>.
</div>
<div id="ref-Picard2019toolkit" class="csl-entry">
<span>“Picard Toolkit.”</span> <em>Broad Institute, GitHub Repository</em>. <a href="https://broadinstitute.github.io/picard/" class="uri">https://broadinstitute.github.io/picard/</a>; Broad Institute, 2019.
</div>
<div id="ref-plominNatureNurtureGenetic2005" class="csl-entry">
Plomin, Robert, and Kathryn Asbury. <span>“Nature and <span>Nurture</span>: <span>Genetic</span> and <span>Environmental Influences</span> on <span>Behavior</span>.”</span> <em>The ANNALS of the American Academy of Political and Social Science</em> 600, no. 1 (July 1, 2005): 86–98. <a href="https://doi.org/10.1177/0002716205277184">https://doi.org/10.1177/0002716205277184</a>.
</div>
<div id="ref-poldermanMetaanalysisHeritabilityHuman2015" class="csl-entry">
Polderman, Tinca J. C., Beben Benyamin, Christiaan A. de Leeuw, Patrick F. Sullivan, Arjen van Bochoven, Peter M. Visscher, and Danielle Posthuma. <span>“Meta-Analysis of the Heritability of Human Traits Based on Fifty Years of Twin Studies.”</span> <em>Nature Genetics</em> 47, no. 7, 7 (July 2015): 702–9. <a href="https://doi.org/10.1038/ng.3285">https://doi.org/10.1038/ng.3285</a>.
</div>
<div id="ref-popejoyGenomicsFailingDiversity2016" class="csl-entry">
Popejoy, Alice B., and Stephanie M. Fullerton. <span>“Genomics Is Failing on Diversity.”</span> <em>Nature</em> 538, no. 7624, 7624 (October 2016): 161–64. <a href="https://doi.org/10.1038/538161a">https://doi.org/10.1038/538161a</a>.
</div>
<div id="ref-poplinScalingAccurateGenetic2018" class="csl-entry">
Poplin, Ryan, Valentin Ruano-Rubio, Mark A. DePristo, Tim J. Fennell, Mauricio O. Carneiro, Geraldine A. Van der Auwera, David E. Kling, et al. <span>“Scaling Accurate Genetic Variant Discovery to Tens of Thousands of Samples.”</span> <span>bioRxiv</span>, July 24, 2018. <a href="https://doi.org/10.1101/201178">https://doi.org/10.1101/201178</a>.
</div>
<div id="ref-porcuGenotypeImputationGenomeWide2013" class="csl-entry">
Porcu, Eleonora, Serena Sanna, Christian Fuchsberger, and Lars G. Fritsche. <span>“Genotype <span>Imputation</span> in <span>Genome-Wide Association Studies</span>.”</span> <em>Current Protocols in Human Genetics</em> 78, no. 1 (2013): 1.25.1–14. <a href="https://doi.org/10.1002/0471142905.hg0125s78">https://doi.org/10.1002/0471142905.hg0125s78</a>.
</div>
<div id="ref-posthumaTheoryPracticeQuantitative2003" class="csl-entry">
Posthuma, Daniëlle, A. Leo Beem, Eco J. C. de Geus, G. Caroline M. van Baal, Jacob B. von Hjelmborg, Ivan Iachine, and Dorret I. Boomsma. <span>“Theory and <span>Practice</span> in <span>Quantitative Genetics</span>.”</span> <em>Twin Research and Human Genetics</em> 6, no. 5 (October 2003): 361–76. <a href="https://doi.org/10.1375/twin.6.5.361">https://doi.org/10.1375/twin.6.5.361</a>.
</div>
<div id="ref-pritchardLinkageDisequilibriumHumans2001" class="csl-entry">
Pritchard, Jonathan K., and Molly Przeworski. <span>“Linkage <span>Disequilibrium</span> in <span>Humans</span>: <span>Models</span> and <span>Data</span>.”</span> <em>The American Journal of Human Genetics</em> 69, no. 1 (July 1, 2001): 1–14. <a href="https://doi.org/10.1086/321275">https://doi.org/10.1086/321275</a>.
</div>
<div id="ref-provineOriginsTheoreticalPopulation2001" class="csl-entry">
Provine, William B. <em>The Origins of Theoretical Population Genetics: With a New Afterword</em>. <span>University of Chicago Press</span>, 2001.
</div>
<div id="ref-prykeRedDominatesBlack2006" class="csl-entry">
Pryke, Sarah R, and Simon C Griffith. <span>“Red Dominates Black: Agonistic Signalling Among Head Morphs in the Colour Polymorphic <span>Gouldian</span> Finch.”</span> <em>Proceedings of the Royal Society B: Biological Sciences</em> 273, no. 1589 (April 22, 2006): 949–57. <a href="https://doi.org/10.1098/rspb.2005.3362">https://doi.org/10.1098/rspb.2005.3362</a>.
</div>
<div id="ref-pszczolaReliabilityDirectGenomic2012" class="csl-entry">
Pszczola, M., T. Strabel, H. A. Mulder, and M. P. L. Calus. <span>“Reliability of Direct Genomic Values for Animals with Different Relationships Within and to the Reference Population.”</span> <em>Journal of Dairy Science</em> 95, no. 1 (January 1, 2012): 389–400. <a href="https://doi.org/10.3168/jds.2011-4338">https://doi.org/10.3168/jds.2011-4338</a>.
</div>
<div id="ref-purcellPLINK" class="csl-entry">
Purcell, Shaun M, and Christopher C Chang. <em><span>PLINK</span> 1.9</em>, n.d. <a href="https://www.cog-genomics.org/plink/1.9/">www.cog-genomics.org/plink/1.9/</a>.
</div>
<div id="ref-racimoDetectingPolygenicAdaptation2018" class="csl-entry">
Racimo, Fernando, Jeremy J Berg, and Joseph K Pickrell. <span>“Detecting <span>Polygenic Adaptation</span> in <span>Admixture Graphs</span>.”</span> <em>Genetics</em> 208, no. 4 (April 1, 2018): 1565–84. <a href="https://doi.org/10.1534/genetics.117.300489">https://doi.org/10.1534/genetics.117.300489</a>.
</div>
<div id="ref-romero-ferreroIdtrackerAiTracking2019" class="csl-entry">
Romero-Ferrero, Francisco, Mattia G. Bergomi, Robert C. Hinz, Francisco J. H. Heras, and Gonzalo G. de Polavieja. <span>“Idtracker.ai: Tracking All Individuals in Small or Large Collectives of Unmarked Animals.”</span> <em>Nature Methods</em> 16, no. 2, 2 (February 2019): 179–82. <a href="https://doi.org/10.1038/s41592-018-0295-5">https://doi.org/10.1038/s41592-018-0295-5</a>.
</div>
<div id="ref-rutherfordHowArgueRacist2020" class="csl-entry">
Rutherford, Adam. <em>How to Argue with a Racist: <span>History</span>, Science, Race and Reality</em>. <span>Hachette UK</span>, 2020.
</div>
<div id="ref-ruxtonUnequalVarianceTtest2006" class="csl-entry">
Ruxton, Graeme D. <span>“The Unequal Variance t-Test Is an Underused Alternative to <span>Student</span>’s t-Test and the <span>Mann</span>–<span>Whitney U</span> Test.”</span> <em>Behavioral Ecology</em> 17, no. 4 (July 1, 2006): 688–90. <a href="https://doi.org/10.1093/beheco/ark016">https://doi.org/10.1093/beheco/ark016</a>.
</div>
<div id="ref-ruzzanteBehaviouralGrowthResponses1990" class="csl-entry">
Ruzzante, D. E., and R. W. Doyle. <span>“Behavioural and Growth Responses to the Intensity of Intraspecific Social Interaction Among Medaka, <span>Oryzias</span> Latipes (<span>Temminck</span> and <span>Schlegel</span>) (<span>Pisces</span>, <span>Cyprinodontidae</span>).”</span> <em>Journal of Fish Biology</em> 37, no. 5 (1990): 663–73. <a href="https://doi.org/10.1111/j.1095-8649.1990.tb02531.x">https://doi.org/10.1111/j.1095-8649.1990.tb02531.x</a>.
</div>
<div id="ref-segurelEvolutionLactasePersistence2017a" class="csl-entry">
S’egurel, Laure, and C’eline Bon. <span>“On the <span>Evolution</span> of <span>Lactase Persistence</span> in <span>Humans</span>.”</span> <em>Annual Review of Genomics and Human Genetics</em> 18 (August 1, 2017): 297–319. <a href="https://doi.org/10.1146/annurev-genom-091416-035340">https://doi.org/10.1146/annurev-genom-091416-035340</a>.
</div>
<div id="ref-saliba-colombaniEfficiencyRFLPRAPD2000" class="csl-entry">
Saliba-Colombani, Vera, Mathilde Causse, Laurent Gervais, and Jacqueline Philouze. <span>“Efficiency of <span>RFLP</span>, <span>RAPD</span>, and <span>AFLP</span> Markers for the Construction of an Intraspecific Map of the Tomato Genome.”</span> <em>Genome</em> 43, no. 1 (February 2000): 29–40. <a href="https://doi.org/10.1139/g99-096">https://doi.org/10.1139/g99-096</a>.
</div>
<div id="ref-sangerDNASequencingChainterminating1977" class="csl-entry">
Sanger, Frederick, Steven Nicklen, and Alan R. Coulson. <span>“<span>DNA</span> Sequencing with Chain-Terminating Inhibitors.”</span> <em>Proceedings of the National Academy of Sciences</em> 74, no. 12 (1977): 5463–67.
</div>
<div id="ref-saulHighDiversityMousePopulations2019" class="csl-entry">
Saul, Michael C., Vivek M. Philip, Laura G. Reinholdt, and Elissa J. Chesler. <span>“High-<span>Diversity Mouse Populations</span> for <span>Complex Traits</span>.”</span> <em>Trends in Genetics</em> 35, no. 7 (July 1, 2019): 501–14. <a href="https://doi.org/10.1016/j.tig.2019.04.003">https://doi.org/10.1016/j.tig.2019.04.003</a>.
</div>
<div id="ref-schjoldenDoesIndividualVariation2005" class="csl-entry">
Schjolden, Joachim, Argaudas Stoskhus, and Svante Winberg. <span>“Does <span>Individual Variation</span> in <span>Stress Responses</span> and <span>Agonistic Behavior Reflect Divergent Stress Coping Strategies</span> in <span>Juvenile Rainbow Trout</span>?”</span> <em>Physiological and Biochemical Zoology</em> 78, no. 5 (September 2005): 715–23. <a href="https://doi.org/10.1086/432153">https://doi.org/10.1086/432153</a>.
</div>
<div id="ref-schmalAnalysisComplexCircadian2022" class="csl-entry">
Schmal, Christoph, Gregor Mönke, and Adri’an E. Granada. <span>“Analysis of <span>Complex Circadian Time Series Data Using Wavelets</span>.”</span> In <em>Circadian <span>Regulation</span>: <span>Methods</span> and <span>Protocols</span></em>, edited by Guiomar Solanas and Patrick -Simon Welz, 35–54. Methods in <span>Molecular Biology</span>. <span>New York, NY</span>: <span>Springer US</span>, 2022. <a href="https://doi.org/10.1007/978-1-0716-2249-0_3">https://doi.org/10.1007/978-1-0716-2249-0_3</a>.
</div>
<div id="ref-schumacherAssociationAnalysesMore2018" class="csl-entry">
Schumacher, Fredrick R., Ali Amin Al Olama, Sonja I. Berndt, Sara Benlloch, Mahbubl Ahmed, Edward J. Saunders, Tokhir Dadaev, et al. <span>“Association Analyses of More Than 140,000 Men Identify 63 New Prostate Cancer Susceptibility Loci.”</span> <em>Nature Genetics</em> 50, no. 7, 7 (July 2018): 928–36. <a href="https://doi.org/10.1038/s41588-018-0142-8">https://doi.org/10.1038/s41588-018-0142-8</a>.
</div>
<div id="ref-schwarzeAreWholeexomeWholegenome2018" class="csl-entry">
Schwarze, Katharina, James Buchanan, Jenny C. Taylor, and Sarah Wordsworth. <span>“Are Whole-Exome and Whole-Genome Sequencing Approaches Cost-Effective? <span>A</span> Systematic Review of the Literature.”</span> <em>Genetics in Medicine</em> 20, no. 10 (October 1, 2018): 1122–30. <a href="https://doi.org/10.1038/gim.2017.247">https://doi.org/10.1038/gim.2017.247</a>.
</div>
<div id="ref-scutariUsingGeneticDistance2016" class="csl-entry">
Scutari, Marco, Ian Mackay, and David Balding. <span>“Using <span>Genetic Distance</span> to <span>Infer</span> the <span>Accuracy</span> of <span>Genomic Prediction</span>.”</span> <em>PLOS Genetics</em> 12, no. 9 (September 2, 2016): e1006288. <a href="https://doi.org/10.1371/journal.pgen.1006288">https://doi.org/10.1371/journal.pgen.1006288</a>.
</div>
<div id="ref-seleitEndogenousProteinTagging2021" class="csl-entry">
Seleit, Ali, Alexander Aulehla, and Alexandre Paix. <span>“Endogenous Protein Tagging in Medaka Using a Simplified <span>CRISPR</span>/<span>Cas9</span> Knock-in Approach.”</span> <em>eLife</em> 10 (December 1, 2021): e75050. <a href="https://doi.org/10.7554/elife.75050">https://doi.org/10.7554/elife.75050</a>.
</div>
<div id="ref-sellaThinkingEvolutionComplex2019" class="csl-entry">
Sella, G., and N. Barton. <span>“Thinking <span>About</span> the <span>Evolution</span> of <span>Complex Traits</span> in the <span>Era</span> of <span>Genome-Wide Association Studies</span>.”</span> <em>Annual Review of Genomics and Human Genetics</em>, 2019. <a href="https://doi.org/10.1146/annurev-genom-083115-022316">https://doi.org/10.1146/annurev-genom-083115-022316</a>.
</div>
<div id="ref-shamPowerLinkageAssociation2000" class="csl-entry">
Sham, P. C., S. S. Cherny, S. Purcell, and J. K. Hewitt. <span>“Power of <span>Linkage</span> Versus <span>Association Analysis</span> of <span>Quantitative Traits</span>, by <span>Use</span> of <span>Variance-Components Models</span>, for <span>Sibship Data</span>.”</span> <em>The American Journal of Human Genetics</em> 66, no. 5 (May 1, 2000): 1616–30. <a href="https://doi.org/10.1086/302891">https://doi.org/10.1086/302891</a>.
</div>
<div id="ref-sharpDevelopmentStandardizationImproved2019" class="csl-entry">
Sharp, Seth A., Stephen S. Rich, Andrew R. Wood, Samuel E. Jones, Robin N. Beaumont, James W. Harrison, Darius A. Schneider, et al. <span>“Development and <span>Standardization</span> of an <span>Improved Type</span> 1 <span>Diabetes Genetic Risk Score</span> for <span>Use</span> in <span>Newborn Screening</span> and <span>Incident Diagnosis</span>.”</span> <em>Diabetes Care</em> 42, no. 2 (January 11, 2019): 200–207. <a href="https://doi.org/10.2337/dc18-1785">https://doi.org/10.2337/dc18-1785</a>.
</div>
<div id="ref-sirugoMissingDiversityHuman2019" class="csl-entry">
Sirugo, Giorgio, Scott M. Williams, and Sarah A. Tishkoff. <span>“The <span>Missing Diversity</span> in <span>Human Genetic Studies</span>.”</span> <em>Cell</em> 177, no. 1 (March 21, 2019): 26–31. <a href="https://doi.org/10.1016/j.cell.2019.02.048">https://doi.org/10.1016/j.cell.2019.02.048</a>.
</div>
<div id="ref-sloanwilsonShynessBoldnessHumans1994" class="csl-entry">
Sloan Wilson, David, Anne B. Clark, Kristine Coleman, and Ted Dearstyne. <span>“Shyness and Boldness in Humans and Other Animals.”</span> <em>Trends in Ecology &amp; Evolution</em> 9, no. 11 (November 1, 1994): 442–46. <a href="https://doi.org/10.1016/0169-5347(94)90134-1">https://doi.org/10.1016/0169-5347(94)90134-1</a>.
</div>
<div id="ref-smigielskiDbSNPDatabaseSingle2000" class="csl-entry">
Smigielski, Elizabeth M., Karl Sirotkin, Minghong Ward, and Stephen T. Sherry. <span>“<span class="nocase">dbSNP</span>: A Database of Single Nucleotide Polymorphisms.”</span> <em>Nucleic Acids Research</em> 28, no. 1 (January 1, 2000): 352–55. <a href="https://doi.org/10.1093/nar/28.1.352">https://doi.org/10.1093/nar/28.1.352</a>.
</div>
<div id="ref-smithYearRatRat2020" class="csl-entry">
Smith, Jennifer R, G Thomas Hayman, Shur-Jen Wang, Stanley J F Laulederkind, Matthew J Hoffman, Mary L Kaldunski, Monika Tutaj, et al. <span>“The <span>Year</span> of the <span>Rat</span>: <span>The Rat Genome Database</span> at 20: A Multi-Species Knowledgebase and Analysis Platform.”</span> <em>Nucleic Acids Research</em> 48, no. D1 (January 8, 2020): D731–42. <a href="https://doi.org/10.1093/nar/gkz1041">https://doi.org/10.1093/nar/gkz1041</a>.
</div>
<div id="ref-spivakovGenomicPhenotypicCharacterization2014" class="csl-entry">
Spivakov, Mikhail, Thomas O Auer, Ravindra Peravali, Ian Dunham, Dirk Dolle, Asao Fujiyama, Atsushi Toyoda, et al. <span>“Genomic and <span>Phenotypic Characterization</span> of a <span>Wild Medaka Population</span>: <span>Towards</span> the <span>Establishment</span> of an <span>Isogenic Population Genetic Resource</span> in <span>Fish</span>.”</span> <em>G3 Genes|Genomes|Genetics</em> 4, no. 3 (March 1, 2014): 433–45. <a href="https://doi.org/10.1534/g3.113.008722">https://doi.org/10.1534/g3.113.008722</a>.
</div>
<div id="ref-steensmaAbbieLathropMouse2010" class="csl-entry">
Steensma, David P., Robert A. Kyle, and Marc A. Shampo. <span>“Abbie <span>Lathrop</span>, the <span>‘<span>Mouse Woman</span> of <span>Granby</span>’</span>: <span>Rodent Fancier</span> and <span>Accidental Genetics Pioneer</span>.”</span> <em>Mayo Clinic Proceedings</em> 85, no. 11 (November 1, 2010): e83. <a href="https://doi.org/10.4065/mcp.2010.0647">https://doi.org/10.4065/mcp.2010.0647</a>.
</div>
<div id="ref-sunPolygenicRiskScores2021" class="csl-entry">
Sun, Luanluan, Lisa Pennells, Stephen Kaptoge, Christopher P. Nelson, Scott C. Ritchie, Gad Abraham, Matthew Arnold, et al. <span>“Polygenic Risk Scores in Cardiovascular Risk Prediction: <span>A</span> Cohort Study and Modelling Analyses.”</span> <em>PLOS Medicine</em> 18, no. 1 (January 14, 2021): e1003498. <a href="https://doi.org/10.1371/journal.pmed.1003498">https://doi.org/10.1371/journal.pmed.1003498</a>.
</div>
<div id="ref-svartbergShynessBoldnessPredicts2002" class="csl-entry">
Svartberg, Kenth. <span>“Shyness–Boldness Predicts Performance in Working Dogs.”</span> <em>Applied Animal Behaviour Science</em> 79, no. 2 (October 20, 2002): 157–74. <a href="https://doi.org/10.1016/S0168-1591(02)00120-X">https://doi.org/10.1016/S0168-1591(02)00120-X</a>.
</div>
<div id="ref-svensonHighResolutionGeneticMapping2012" class="csl-entry">
Svenson, Karen L, Daniel M Gatti, William Valdar, Catherine E Welsh, Riyan Cheng, Elissa J Chesler, Abraham A Palmer, Leonard McMillan, and Gary A Churchill. <span>“High-<span>Resolution Genetic Mapping Using</span> the <span>Mouse Diversity Outbred Population</span>.”</span> <em>Genetics</em> 190, no. 2 (February 1, 2012): 437–47. <a href="https://doi.org/10.1534/genetics.111.132597">https://doi.org/10.1534/genetics.111.132597</a>.
</div>
<div id="ref-taberyStruggleUnderstandInteraction2014" class="csl-entry">
Tabery, James. <em>Beyond Versus: <span>The</span> Struggle to Understand the Interaction of Nature and Nurture</em>. <span>mit Press</span>, 2014.
</div>
<div id="ref-11TrillionGlobal2019" class="csl-entry">
<span>“The $11.9 <span>Trillion Global Healthcare Market</span>: <span>Key Opportunities</span> &amp; <span>Strategies</span> (2014-2022) - <span>ResearchAndMarkets</span>.com,”</span> June 25, 2019. <a href="https://www.businesswire.com/news/home/20190625005862/en/The-11.9-Trillion-Global-Healthcare-Market-Key-Opportunities-Strategies-2014-2022---ResearchAndMarkets.com">https://www.businesswire.com/news/home/20190625005862/en/The-11.9-Trillion-Global-Healthcare-Market-Key-Opportunities-Strategies-2014-2022---ResearchAndMarkets.com</a>.
</div>
<div id="ref-NobelPrizeChemistry" class="csl-entry">
<span>“The <span>Nobel Prize</span> in <span>Chemistry</span> 1980.”</span> <span>NobelPrize.org</span>. Accessed August 30, 2022. <a href="https://www.nobelprize.org/prizes/chemistry/1980/summary/">https://www.nobelprize.org/prizes/chemistry/1980/summary/</a>.
</div>
<div id="ref-threadgillCollaborativeCrossRecombinant2011" class="csl-entry">
Threadgill, David W., Darla R. Miller, Gary A. Churchill, and Fernando Pardo-Manuel de Villena. <span>“The <span>Collaborative Cross</span>: <span>A Recombinant Inbred Mouse Population</span> for the <span>Systems Genetic Era</span>.”</span> <em>ILAR Journal</em> 52, no. 1 (January 1, 2011): 24–31. <a href="https://doi.org/10.1093/ilar.52.1.24">https://doi.org/10.1093/ilar.52.1.24</a>.
</div>
<div id="ref-tikkanenGeneticRiskPrediction2013" class="csl-entry">
Tikkanen, Emmi, Aki S. Havulinna, Aarno Palotie, Veikko Salomaa, and Samuli Ripatti. <span>“Genetic <span>Risk Prediction</span> and a 2-<span>Stage Risk Screening Strategy</span> for <span>Coronary Heart Disease</span>.”</span> <em>Arteriosclerosis, Thrombosis, and Vascular Biology</em> 33, no. 9 (September 2013): 2261–66. <a href="https://doi.org/10.1161/ATVBAHA.112.301120">https://doi.org/10.1161/ATVBAHA.112.301120</a>.
</div>
<div id="ref-turleyMultitraitAnalysisGenomewide2018" class="csl-entry">
Turley, Patrick, Raymond K. Walters, Omeed Maghzian, Aysu Okbay, James J. Lee, Mark Alan Fontana, Tuan Anh Nguyen-Viet, et al. <span>“Multi-Trait Analysis of Genome-Wide Association Summary Statistics Using <span>MTAG</span>.”</span> <em>Nature Genetics</em> 50, no. 2, 2 (February 2018): 229–37. <a href="https://doi.org/10.1038/s41588-017-0009-4">https://doi.org/10.1038/s41588-017-0009-4</a>.
</div>
<div id="ref-uffelmannGenomewideAssociationStudies2021" class="csl-entry">
Uffelmann, Emil, Qin Qin Huang, Nchangwi Syntia Munung, Jantina de Vries, Yukinori Okada, Alicia R. Martin, Hilary C. Martin, Tuuli Lappalainen, and Danielle Posthuma. <span>“Genome-Wide Association Studies.”</span> <em>Nature Reviews Methods Primers</em> 1, no. 1, 1 (August 26, 2021): 1–21. <a href="https://doi.org/10.1038/s43586-021-00056-9">https://doi.org/10.1038/s43586-021-00056-9</a>.
</div>
<div id="ref-utineHomozygousDeletionGRID22013" class="csl-entry">
Utine, G. Eda, Göknur Haliloğlu, Bilge Salancı, Arda Çetinkaya, P. Özlem Kiper, Yasemin Alanay, Dilek Aktaş, Koray Boduroğlu, and Mehmet Alikaşifoğlu. <span>“A <span>Homozygous Deletion</span> in <span>Grid2 Causes</span> a <span>Human Phenotype With Cerebellar Ataxia</span> and <span>Atrophy</span>.”</span> <em>Journal of Child Neurology</em> 28, no. 7 (July 1, 2013): 926–32. <a href="https://doi.org/10.1177/0883073813484967">https://doi.org/10.1177/0883073813484967</a>.
</div>
<div id="ref-vanderauweraGenomicsCloudUsing2020" class="csl-entry">
Van der Auwera, Geraldine A., and Brian D. O’Connor. <em>Genomics in the Cloud: Using <span>Docker</span>, <span>GATK</span>, and <span>WDL</span> in <span>Terra</span></em>. <span>O’Reilly Media</span>, 2020.
</div>
<div id="ref-vasimuddinEfficientArchitectureawareAcceleration2019" class="csl-entry">
Vasimuddin, Md, Sanchit Misra, Heng Li, and Srinivas Aluru. <span>“Efficient Architecture-Aware Acceleration of <span>BWA-MEM</span> for Multicore Systems.”</span> In <em>2019 <span>IEEE International Parallel</span> and <span>Distributed Processing Symposium</span> (<span>IPDPS</span>)</em>, 314–24. <span>IEEE</span>, 2019.
</div>
<div id="ref-vilhjalmsson2015modeling" class="csl-entry">
Vilhjálmsson, Bjarni J, Jian Yang, Hilary K Finucane, Alexander Gusev, Sara Lindström, Stephan Ripke, Giulio Genovese, et al. <span>“Modeling Linkage Disequilibrium Increases Accuracy of Polygenic Risk Scores.”</span> <em>The American Journal of Human Genetics</em> 97, no. 4 (2015): 576–92.
</div>
<div id="ref-visscherHeritabilityGenomicsEra2008" class="csl-entry">
Visscher, Peter M., William G. Hill, and Naomi R. Wray. <span>“Heritability in the Genomics Era — Concepts and Misconceptions.”</span> <em>Nature Reviews Genetics</em> 9, no. 4, 4 (April 2008): 255–66. <a href="https://doi.org/10.1038/nrg2322">https://doi.org/10.1038/nrg2322</a>.
</div>
<div id="ref-wangTheoreticalEmpiricalQuantification2020" class="csl-entry">
Wang, Ying, Jing Guo, Guiyan Ni, Jian Yang, Peter M. Visscher, and Loic Yengo. <span>“Theoretical and Empirical Quantification of the Accuracy of Polygenic Scores in Ancestry Divergent Populations.”</span> <em>Nature Communications</em> 11, no. 1, 1 (July 31, 2020): 3865. <a href="https://doi.org/10.1038/s41467-020-17719-y">https://doi.org/10.1038/s41467-020-17719-y</a>.
</div>
<div id="ref-wangNanoporeSequencingTechnology2021" class="csl-entry">
Wang, Yunhao, Yue Zhao, Audrey Bollas, Yuru Wang, and Kin Fai Au. <span>“Nanopore Sequencing Technology, Bioinformatics and Applications.”</span> <em>Nature Biotechnology</em> 39, no. 11, 11 (November 2021): 1348–65. <a href="https://doi.org/10.1038/s41587-021-01108-x">https://doi.org/10.1038/s41587-021-01108-x</a>.
</div>
<div id="ref-watanabeGlobalOverviewPleiotropy2019" class="csl-entry">
Watanabe, Kyoko, Sven Stringer, Oleksandr Frei, Maša Umi’cevi’c Mirkov, Christiaan de Leeuw, Tinca J. C. Polderman, Sophie van der Sluis, Ole A. Andreassen, Benjamin M. Neale, and Danielle Posthuma. <span>“A Global Overview of Pleiotropy and Genetic Architecture in Complex Traits.”</span> <em>Nature Genetics</em> 51, no. 9, 9 (September 2019): 1339–48. <a href="https://doi.org/10.1038/s41588-019-0481-0">https://doi.org/10.1038/s41588-019-0481-0</a>.
</div>
<div id="ref-watsonDNASecretLife2009" class="csl-entry">
Watson, James D., and Andrew Berry. <em><span>DNA</span>: <span>The</span> Secret of Life</em>. <span>Knopf</span>, 2009.
</div>
<div id="ref-weirEstimatingFStatisticsAnalysis1984" class="csl-entry">
Weir, B. S., and C. Clark Cockerham. <span>“Estimating <span>F-Statistics</span> for the <span>Analysis</span> of <span>Population Structure</span>.”</span> <em>Evolution</em> 38, no. 6 (1984): 1358–70. <a href="https://doi.org/10.2307/2408641">https://doi.org/10.2307/2408641</a>.
</div>
<div id="ref-weirMeasuresHumanPopulation2005" class="csl-entry">
Weir, Bruce S., Lon R. Cardon, Amy D. Anderson, Dahlia M. Nielsen, and William G. Hill. <span>“Measures of Human Population Structure Show Heterogeneity Among Genomic Regions.”</span> <em>Genome Research</em> 15, no. 11 (January 11, 2005): 1468–76. <a href="https://doi.org/10.1101/gr.4398405">https://doi.org/10.1101/gr.4398405</a>.
</div>
<div id="ref-WhatWasMegatherium" class="csl-entry">
<span>“What Was <span>Megatherium</span>?”</span> Accessed August 17, 2022. <a href="https://www.nhm.ac.uk/discover/what-was-megatherium.html">https://www.nhm.ac.uk/discover/what-was-megatherium.html</a>.
</div>
<div id="ref-wichuraAlgorithm241Percentage1988" class="csl-entry">
Wichura, Michael J. <span>“Algorithm <span>AS</span> 241: <span>The Percentage Points</span> of the <span>Normal Distribution</span>.”</span> <em>Journal of the Royal Statistical Society. Series C (Applied Statistics)</em> 37, no. 3 (1988): 477–84. <a href="https://doi.org/10.2307/2347330">https://doi.org/10.2307/2347330</a>.
</div>
<div id="ref-wilsonShyboldContinuumPumpkinseed1993" class="csl-entry">
Wilson, David S., Kristine Coleman, Anne B. Clark, and Laurence Biederman. <span>“Shy-Bold Continuum in Pumpkinseed Sunfish (<span>Lepomis</span> Gibbosus): <span>An</span> Ecological Study of a Psychological Trait.”</span> <em>Journal of Comparative Psychology</em> 107, no. 3 (1993): 250–60. <a href="https://doi.org/10.1037/0735-7036.107.3.250">https://doi.org/10.1037/0735-7036.107.3.250</a>.
</div>
<div id="ref-wittbrodtMedakaModelOrganism2002" class="csl-entry">
Wittbrodt, Joachim, Akihiro Shima, and Manfred Schartl. <span>“Medaka — a Model Organism from the Far East.”</span> <em>Nature Reviews Genetics</em> 3, no. 1, 1 (January 2002): 53–64. <a href="https://doi.org/10.1038/nrg704">https://doi.org/10.1038/nrg704</a>.
</div>
<div id="ref-wojcikGeneticAnalysesDiverse2019" class="csl-entry">
Wojcik, Genevieve L., Mariaelisa Graff, Katherine K. Nishimura, Ran Tao, Jeffrey Haessler, Christopher R. Gignoux, Heather M. Highland, et al. <span>“Genetic Analyses of Diverse Populations Improves Discovery for Complex Traits.”</span> <em>Nature</em> 570, no. 7762, 7762 (June 2019): 514–18. <a href="https://doi.org/10.1038/s41586-019-1310-4">https://doi.org/10.1038/s41586-019-1310-4</a>.
</div>
<div id="ref-wrayPredictionIndividualGenetic2007" class="csl-entry">
Wray, Naomi R., Michael E. Goddard, and Peter M. Visscher. <span>“Prediction of Individual Genetic Risk to Disease from Genome-Wide Association Studies.”</span> <em>Genome Research</em> 17, no. 10 (January 10, 2007): 1520–28. <a href="https://doi.org/10.1101/gr.6665407">https://doi.org/10.1101/gr.6665407</a>.
</div>
<div id="ref-wrightEpistaticRegulationBehavioural2006" class="csl-entry">
Wright, Dominic, Roger K. Butlin, and Örjan Carlborg. <span>“Epistatic Regulation of Behavioural and Morphological Traits in the Zebrafish (<span>Danio</span> Rerio).”</span> <em>Behavior Genetics</em> 36, no. 6 (November 1, 2006): 914–22. <a href="https://doi.org/10.1007/s10519-006-9080-9">https://doi.org/10.1007/s10519-006-9080-9</a>.
</div>
<div id="ref-wrightInterIntrapopulationVariation2003" class="csl-entry">
Wright, D., L. B. Rimmer, V. L. Pritchard, R. K. Butlin, and J. Krause. <span>“Inter and Intra-Population Variation in Shoaling and Boldness in the Zebrafish (<span>Danio</span> Rerio).”</span> <em>Journal of Fish Biology</em> 63, no. s1 (2003): 258–59. <a href="https://doi.org/10.1111/j.1095-8649.2003.216bw.x">https://doi.org/10.1111/j.1095-8649.2003.216bw.x</a>.
</div>
<div id="ref-wrightCoefficientsInbreedingRelationship1922" class="csl-entry">
Wright, Sewall. <span>“Coefficients of <span>Inbreeding</span> and <span>Relationship</span>.”</span> <em>The American Naturalist</em> 56, no. 645 (July 1922): 330–38. <a href="https://doi.org/10.1086/279872">https://doi.org/10.1086/279872</a>.
</div>
<div id="ref-wrightGeneticalStructurePopulations1949" class="csl-entry">
———. <span>“The <span>Genetical Structure</span> of <span>Populations</span>.”</span> <em>Annals of Eugenics</em> 15, no. 1 (1949): 323–54. <a href="https://doi.org/10.1111/j.1469-1809.1949.tb02451.x">https://doi.org/10.1111/j.1469-1809.1949.tb02451.x</a>.
</div>
<div id="ref-yangCommonSNPsExplain2010" class="csl-entry">
Yang, Jian, Beben Benyamin, Brian P. McEvoy, Scott Gordon, Anjali K. Henders, Dale R. Nyholt, Pamela A. Madden, et al. <span>“Common <span>SNPs</span> Explain a Large Proportion of the Heritability for Human Height.”</span> <em>Nature Genetics</em> 42, no. 7, 7 (July 2010): 565–69. <a href="https://doi.org/10.1038/ng.608">https://doi.org/10.1038/ng.608</a>.
</div>
<div id="ref-yangGCTAToolGenomewide2011" class="csl-entry">
Yang, Jian, S. Hong Lee, Michael E. Goddard, and Peter M. Visscher. <span>“<span>GCTA</span>: <span>A Tool</span> for <span class="nocase">Genome-wide Complex Trait Analysis</span>.”</span> <em>The American Journal of Human Genetics</em> 88, no. 1 (January 7, 2011): 76–82. <a href="https://doi.org/10.1016/j.ajhg.2010.11.011">https://doi.org/10.1016/j.ajhg.2010.11.011</a>.
</div>
<div id="ref-yengoMetaanalysisGenomewideAssociation2018" class="csl-entry">
Yengo, Loic, Julia Sidorenko, Kathryn E Kemper, Zhili Zheng, Andrew R Wood, Michael N Weedon, Timothy M Frayling, et al. <span>“Meta-Analysis of Genome-Wide Association Studies for Height and Body Mass Index in  700000 Individuals of <span>European</span> Ancestry.”</span> <em>Human Molecular Genetics</em> 27, no. 20 (October 15, 2018): 3641–49. <a href="https://doi.org/10.1093/hmg/ddy271">https://doi.org/10.1093/hmg/ddy271</a>.
</div>
<div id="ref-youngNeurobiologyHumanSocial2008" class="csl-entry">
Young, Simon N. <span>“The Neurobiology of Human Social Behaviour: An Important but Neglected Topic.”</span> <em>Journal of Psychiatry and Neuroscience</em> 33, no. 5 (September 1, 2008): 391–92. <a href="https://www.jpn.ca/content/33/5/391">https://www.jpn.ca/content/33/5/391</a>.
</div>
</div>
<!--chapter:end:07-references.Rmd-->
</div>
<div id="appendix-appendix" class="section level1 unnumbered">
<h1 class="unnumbered">(APPENDIX) Appendix</h1>
</div>
<div id="hmm-states-with-0.05-second-interval" class="section level1" number="7">
<h1><span class="header-section-number">7</span> 15 HMM states with 0.05 second interval</h1>
<p>(ref:hmm-states-05) HMM states predicted for the F0 dataset, using the combination of a 0.05-second interval between which the distance and angle variables were calculated.</p>
<div class="figure">
<img src="figs/mikk_behaviour/0.05_15_polar_all_dge.png" alt="(ref:hmm-states-05)" width="100%" />
<p class="caption">
(#fig:hmm-states-05)(ref:hmm-states-05)
</p>
</div>
</div>
<div id="four-panel-app" class="section level1" number="8">
<h1><span class="header-section-number">8</span> Four-panel plots for MIKK lines selected for the F2 cross</h1>
<p>All frame grabs were taken at the 300th second (5-minute mark) of each 10-minute assay component.</p>
<div id="open-field" class="section level2" number="8.1">
<h2><span class="header-section-number">8.1</span> Open field</h2>
<div id="section" class="section level3" number="8.1.1">
<h3><span class="header-section-number">8.1.1</span> 8-2</h3>
<p>(ref:4p-8-2-of-A) Line 8-2, open field, run A.</p>
<div class="figure">
<img src="figs/mikk_behaviour/four_panel_plots/open_field_20191114_1040_8-2_L_A_300.png" alt="(ref:4p-8-2-of-A)" width="100%" />
<p class="caption">
(#fig:4p-8-2-of-A)(ref:4p-8-2-of-A)
</p>
</div>
<p>(ref:4p-8-2-of-B) Line 8-2, open field, run B.</p>
<div class="figure">
<img src="figs/mikk_behaviour/four_panel_plots/open_field_20191118_1224_8-2_R_B_300.png" alt="(ref:4p-8-2-of-B)" width="100%" />
<p class="caption">
(#fig:4p-8-2-of-B)(ref:4p-8-2-of-B)
</p>
</div>
</div>
<div id="section-1" class="section level3" number="8.1.2">
<h3><span class="header-section-number">8.1.2</span> 18-2</h3>
<p>(ref:4p-18-2-of-A) Line 18-2, open field, run A.</p>
<div class="figure">
<img src="figs/mikk_behaviour/four_panel_plots/open_field_20191112_1236_18-2_L_A_300.png" alt="(ref:4p-18-2-of-A)" width="100%" />
<p class="caption">
(#fig:4p-18-2-of-A)(ref:4p-18-2-of-A)
</p>
</div>
<p>(ref:4p-18-2-of-B) Line 18-2, open field, run B.</p>
<div class="figure">
<img src="figs/mikk_behaviour/four_panel_plots/open_field_20191116_1039_18-2_L_B_300.png" alt="(ref:4p-18-2-of-B)" width="100%" />
<p class="caption">
(#fig:4p-18-2-of-B)(ref:4p-18-2-of-B)
</p>
</div>
<p>(ref:4p-18-2-of-C) Line 18-2, open field, run C.</p>
<div class="figure">
<img src="figs/mikk_behaviour/four_panel_plots/open_field_20191119_1459_18-2_R_C_300.png" alt="(ref:4p-18-2-of-C)" width="100%" />
<p class="caption">
(#fig:4p-18-2-of-C)(ref:4p-18-2-of-C)
</p>
</div>
</div>
<div id="section-2" class="section level3" number="8.1.3">
<h3><span class="header-section-number">8.1.3</span> 50-2</h3>
<p>(ref:4p-50-2-of-A) Line 50-2, open field, run A.</p>
<div class="figure">
<img src="figs/mikk_behaviour/four_panel_plots/open_field_20191114_1305_50-2_R_A_300.png" alt="(ref:4p-50-2-of-A)" width="100%" />
<p class="caption">
(#fig:4p-50-2-of-A)(ref:4p-50-2-of-A)
</p>
</div>
<p>(ref:4p-50-2-of-B) Line 50-2, open field, run B.</p>
<div class="figure">
<img src="figs/mikk_behaviour/four_panel_plots/open_field_20191116_1438_50-2_R_B_300.png" alt="(ref:4p-50-2-of-B)" width="100%" />
<p class="caption">
(#fig:4p-50-2-of-B)(ref:4p-50-2-of-B)
</p>
</div>
<p>(ref:4p-50-2-of-C) Line 50-2, open field, run C.</p>
<div class="figure">
<img src="figs/mikk_behaviour/four_panel_plots/open_field_20191120_1150_50-2_R_C_300.png" alt="(ref:4p-50-2-of-C)" width="100%" />
<p class="caption">
(#fig:4p-50-2-of-C)(ref:4p-50-2-of-C)
</p>
</div>
</div>
<div id="section-3" class="section level3" number="8.1.4">
<h3><span class="header-section-number">8.1.4</span> 38-2</h3>
<p>(ref:4p-38-2-of-A) Line 38-2, open field, run A.</p>
<div class="figure">
<img src="figs/mikk_behaviour/four_panel_plots/open_field_20191112_1456_38-2_L_A_300.png" alt="(ref:4p-38-2-of-A)" width="100%" />
<p class="caption">
(#fig:4p-38-2-of-A)(ref:4p-38-2-of-A)
</p>
</div>
<p>(ref:4p-38-2-of-B) Line 38-2, open field, run B.</p>
<div class="figure">
<img src="figs/mikk_behaviour/four_panel_plots/open_field_20191116_1203_38-2_L_B_300.png" alt="(ref:4p-38-2-of-B)" width="100%" />
<p class="caption">
(#fig:4p-38-2-of-B)(ref:4p-38-2-of-B)
</p>
</div>
</div>
<div id="section-4" class="section level3" number="8.1.5">
<h3><span class="header-section-number">8.1.5</span> 21-2</h3>
<p>(ref:4p-21-2-of-A) Line 38-2, open field, run A.</p>
<div class="figure">
<img src="figs/mikk_behaviour/four_panel_plots/open_field_20191113_1527_21-2_L_A_300.png" alt="(ref:4p-21-2-of-A)" width="100%" />
<p class="caption">
(#fig:4p-21-2-of-A)(ref:4p-21-2-of-A)
</p>
</div>
<p>(ref:4p-21-2-of-B) Line 38-2, open field, run B.</p>
<div class="figure">
<img src="figs/mikk_behaviour/four_panel_plots/open_field_20191116_1309_21-2_R_B_300.png" alt="(ref:4p-21-2-of-B)" width="100%" />
<p class="caption">
(#fig:4p-21-2-of-B)(ref:4p-21-2-of-B)
</p>
</div>
</div>
<div id="section-5" class="section level3" number="8.1.6">
<h3><span class="header-section-number">8.1.6</span> 40-1</h3>
<p>(ref:4p-40-1-of-A) Line 40-1, open field, run A.</p>
<div class="figure">
<img src="figs/mikk_behaviour/four_panel_plots/open_field_20191114_1217_40-1_L_A_300.png" alt="(ref:4p-40-1-of-A)" width="100%" />
<p class="caption">
(#fig:4p-40-1-of-A)(ref:4p-40-1-of-A)
</p>
</div>
<p>(ref:4p-40-1-of-B) Line 40-1, open field, run B.</p>
<div class="figure">
<img src="figs/mikk_behaviour/four_panel_plots/open_field_20191118_1129_40-1_R_B_300.png" alt="(ref:4p-40-1-of-B)" width="100%" />
<p class="caption">
(#fig:4p-40-1-of-B)(ref:4p-40-1-of-B)
</p>
</div>
</div>
</div>
<div id="novel-object" class="section level2" number="8.2">
<h2><span class="header-section-number">8.2</span> Novel object</h2>
<div id="section-6" class="section level3" number="8.2.1">
<h3><span class="header-section-number">8.2.1</span> 8-2</h3>
<p>(ref:4p-8-2-no-A) Line 8-2, novel object, run A.</p>
<div class="figure">
<img src="figs/mikk_behaviour/four_panel_plots/novel_object_20191114_1040_8-2_L_A_300.png" alt="(ref:4p-8-2-no-A)" width="100%" />
<p class="caption">
(#fig:4p-8-2-no-A)(ref:4p-8-2-no-A)
</p>
</div>
<p>(ref:4p-8-2-no-B) Line 8-2, novel object, run B.</p>
<div class="figure">
<img src="figs/mikk_behaviour/four_panel_plots/novel_object_20191118_1224_8-2_R_B_300.png" alt="(ref:4p-8-2-no-B)" width="100%" />
<p class="caption">
(#fig:4p-8-2-no-B)(ref:4p-8-2-no-B)
</p>
</div>
</div>
<div id="section-7" class="section level3" number="8.2.2">
<h3><span class="header-section-number">8.2.2</span> 18-2</h3>
<p>(ref:4p-18-2-no-A) Line 18-2, novel object, run A.</p>
<div class="figure">
<img src="figs/mikk_behaviour/four_panel_plots/novel_object_20191112_1236_18-2_L_A_300.png" alt="(ref:4p-18-2-no-A)" width="100%" />
<p class="caption">
(#fig:4p-18-2-no-A)(ref:4p-18-2-no-A)
</p>
</div>
<p>(ref:4p-18-2-no-B) Line 18-2, novel object, run B.</p>
<div class="figure">
<img src="figs/mikk_behaviour/four_panel_plots/novel_object_20191116_1039_18-2_L_B_300.png" alt="(ref:4p-18-2-no-B)" width="100%" />
<p class="caption">
(#fig:4p-18-2-no-B)(ref:4p-18-2-no-B)
</p>
</div>
<p>(ref:4p-18-2-no-C) Line 18-2, novel object, run C.</p>
<div class="figure">
<img src="figs/mikk_behaviour/four_panel_plots/novel_object_20191119_1459_18-2_R_C_300.png" alt="(ref:4p-18-2-no-C)" width="100%" />
<p class="caption">
(#fig:4p-18-2-no-C)(ref:4p-18-2-no-C)
</p>
</div>
</div>
<div id="section-8" class="section level3" number="8.2.3">
<h3><span class="header-section-number">8.2.3</span> 50-2</h3>
<p>(ref:4p-50-2-no-A) Line 50-2, novel object, run A.</p>
<div class="figure">
<img src="figs/mikk_behaviour/four_panel_plots/novel_object_20191114_1305_50-2_R_A_300.png" alt="(ref:4p-50-2-no-A)" width="100%" />
<p class="caption">
(#fig:4p-50-2-no-A)(ref:4p-50-2-no-A)
</p>
</div>
<p>(ref:4p-50-2-no-B) Line 50-2, novel object, run B.</p>
<div class="figure">
<img src="figs/mikk_behaviour/four_panel_plots/novel_object_20191116_1438_50-2_R_B_300.png" alt="(ref:4p-50-2-no-B)" width="100%" />
<p class="caption">
(#fig:4p-50-2-no-B)(ref:4p-50-2-no-B)
</p>
</div>
<p>(ref:4p-50-2-no-C) Line 50-2, novel object, run C.</p>
<div class="figure">
<img src="figs/mikk_behaviour/four_panel_plots/novel_object_20191120_1150_50-2_R_C_300.png" alt="(ref:4p-50-2-no-C)" width="100%" />
<p class="caption">
(#fig:4p-50-2-no-C)(ref:4p-50-2-no-C)
</p>
</div>
</div>
<div id="section-9" class="section level3" number="8.2.4">
<h3><span class="header-section-number">8.2.4</span> 38-2</h3>
<p>(ref:4p-38-2-no-A) Line 38-2, novel object, run A.</p>
<div class="figure">
<img src="figs/mikk_behaviour/four_panel_plots/novel_object_20191112_1456_38-2_L_A_300.png" alt="(ref:4p-38-2-no-A)" width="100%" />
<p class="caption">
(#fig:4p-38-2-no-A)(ref:4p-38-2-no-A)
</p>
</div>
<p>(ref:4p-38-2-no-B) Line 38-2, novel object, run B.</p>
<div class="figure">
<img src="figs/mikk_behaviour/four_panel_plots/novel_object_20191116_1203_38-2_L_B_300.png" alt="(ref:4p-38-2-no-B)" width="100%" />
<p class="caption">
(#fig:4p-38-2-no-B)(ref:4p-38-2-no-B)
</p>
</div>
</div>
<div id="section-10" class="section level3" number="8.2.5">
<h3><span class="header-section-number">8.2.5</span> 21-2</h3>
<p>(ref:4p-21-2-no-A) Line 38-2, novel object, run A.</p>
<div class="figure">
<img src="figs/mikk_behaviour/four_panel_plots/novel_object_20191113_1527_21-2_L_A_300.png" alt="(ref:4p-21-2-no-A)" width="100%" />
<p class="caption">
(#fig:4p-21-2-no-A)(ref:4p-21-2-no-A)
</p>
</div>
<p>(ref:4p-21-2-no-B) Line 38-2, novel object, run B.</p>
<div class="figure">
<img src="figs/mikk_behaviour/four_panel_plots/novel_object_20191116_1309_21-2_R_B_300.png" alt="(ref:4p-21-2-no-B)" width="100%" />
<p class="caption">
(#fig:4p-21-2-no-B)(ref:4p-21-2-no-B)
</p>
</div>
</div>
<div id="section-11" class="section level3" number="8.2.6">
<h3><span class="header-section-number">8.2.6</span> 40-1</h3>
<p>(ref:4p-40-1-no-A) Line 40-1, novel object, run A.</p>
<div class="figure">
<img src="figs/mikk_behaviour/four_panel_plots/novel_object_20191114_1217_40-1_L_A_300.png" alt="(ref:4p-40-1-no-A)" width="100%" />
<p class="caption">
(#fig:4p-40-1-no-A)(ref:4p-40-1-no-A)
</p>
</div>
<p>(ref:4p-40-1-no-B) Line 40-1, novel object, run B.</p>
<div class="figure">
<img src="figs/mikk_behaviour/four_panel_plots/novel_object_20191118_1129_40-1_R_B_300.png" alt="(ref:4p-40-1-no-B)" width="100%" />
<p class="caption">
(#fig:4p-40-1-no-B)(ref:4p-40-1-no-B)
</p>
</div>
</div>
</div>
</div>
<div id="hmm-state-time-dependence-for-all-mikk-panel-lines" class="section level1" number="9">
<h1><span class="header-section-number">9</span> HMM state time dependence for all MIKK panel lines</h1>
<div id="direct-genetic-effects-3" class="section level2" number="9.1">
<h2><span class="header-section-number">9.1</span> Direct genetic effects</h2>
<div id="tile-plot" class="section level3" number="9.1.1">
<h3><span class="header-section-number">9.1.1</span> Tile plot</h3>
<p>(ref:mikk-all-tile-dge) Tile plot for all 1610 test fishes included in the MIKK panel behaviour analysis, ordered by each line’s group median for individual mean speed over the course of the 20-minute video (open field and novel object combined). The order of lines is identical to that shown in <strong>Figure @ref(fig:mikk-mean-speed)</strong>.</p>
<div class="figure">
<img src="figs/mikk_behaviour/0.08_15_tile_dge.png" alt="(ref:mikk-all-tile-dge)" width="100%" />
<p class="caption">
(#fig:mikk-all-tile-dge)(ref:mikk-all-tile-dge)
</p>
</div>
</div>
<div id="density-plot" class="section level3" number="9.1.2">
<h3><span class="header-section-number">9.1.2</span> Density plot</h3>
<p>(ref:mikk-all-dens-dge) Density plot for all 1610 test fishes included in the MIKK panel behaviour analysis, ordered by each line’s group median for individual mean speed over the course of the 20-minute video (open field and novel object combined). The order of lines is identical to that shown in <strong>Figure @ref(fig:mikk-mean-speed)</strong>.</p>
<div class="figure">
<img src="figs/mikk_behaviour/0.08_15_dens_dge.png" alt="(ref:mikk-all-tile-dge)" width="100%" />
<p class="caption">
(#fig:mikk-all-dens-dge)(ref:mikk-all-tile-dge)
</p>
</div>
</div>
</div>
<div id="social-genetic-effects-4" class="section level2" number="9.2">
<h2><span class="header-section-number">9.2</span> Social genetic effects</h2>
<div id="tile-plot-1" class="section level3" number="9.2.1">
<h3><span class="header-section-number">9.2.1</span> Tile plot</h3>
<p>(ref:mikk-all-tile-sge) Tile plot for all 1610 reference fishes included in the MIKK panel behaviour analysis, ordered by each line’s group median for individual mean speed over the course of the 20-minute video (open field and novel object combined). The order of lines is identical to that shown in <strong>Figure @ref(fig:mikk-mean-speed)</strong>.</p>
<div class="figure">
<img src="figs/mikk_behaviour/0.08_15_tile_sge.png" alt="(ref:mikk-all-tile-sge)" width="100%" />
<p class="caption">
(#fig:mikk-all-tile-sge)(ref:mikk-all-tile-sge)
</p>
</div>
</div>
<div id="density-plot-1" class="section level3" number="9.2.2">
<h3><span class="header-section-number">9.2.2</span> Density plot</h3>
<p>(ref:mikk-all-dens-sge) Density plot for all 1610 test fishes included in the MIKK panel behaviour analysis, ordered by each line’s group median for individual mean speed over the course of the 20-minute video (open field and novel object combined). The order of lines is identical to that shown in <strong>Figure @ref(fig:mikk-mean-speed)</strong>.</p>
<div class="figure">
<img src="figs/mikk_behaviour/0.08_15_dens_sge.png" alt="(ref:mikk-all-tile-sge)" width="100%" />
<p class="caption">
(#fig:mikk-all-dens-sge)(ref:mikk-all-tile-sge)
</p>
</div>
</div>
</div>
</div>
<div id="f2-recombination-karyoplot-with-missing-calls" class="section level1" number="10">
<h1><span class="header-section-number">10</span> F2 recombination karyoplot with missing calls</h1>
<p>(ref:F2-recomb-blocks-missing) Karyoplot for F2 samples, coloured by genotype. Samples are sorted in the order in which they were phenotyped. Blocks are filled with the colour of the paternal F0 line for the homozygous paternal haplotype block, black for heterozygous, and the colour of the maternal F0 line for the homozygous maternal haplotype block.</p>
<div class="figure">
<img src="figs/mikk_behaviour/karyoplot_wi_missing.png" alt="(ref:F2-recomb-blocks-missing)" width="100%" />
<p class="caption">
(#fig:F2-recomb-blocks-missing)(ref:F2-recomb-blocks-missing)
</p>
</div>
</div>
<div id="loco-grm-for-chromosome-1" class="section level1" number="11">
<h1><span class="header-section-number">11</span> LOCO GRM for chromosome 1</h1>
<p>(ref:loco-grm-chr1) “Leave-one-chromosome-out” genetic relationship matrix for 271 F2 samples based on 44,284 non-missing SNPs, having excluded 76 SNPs on chromosome 1.</p>
<div class="figure">
<img src="figs/mikk_behaviour/grm_loco_man_chr1.png" alt="(ref:loco-grm-chr1)" width="100%" />
<p class="caption">
(#fig:loco-grm-chr1)(ref:loco-grm-chr1)
</p>
</div>
</div>
<div id="ecdf-of-all-polygenic-traits-in-the-gwas-catalog-ranked-by-d_ts" class="section level1" number="12">
<h1><span class="header-section-number">12</span> eCDF of all polygenic traits in the GWAS Catalog ranked by <span class="math inline">\({D_t^S}\)</span></h1>
<p>(ref:eCDFall) 587 traits from the GWAS Catalog that passed our filters for polygenic traits, ranked by <span class="math inline">\({D_t^S}\)</span>.</p>
<div class="figure">
<img src="figs/fst/0.1_1000_20220314_ecdf_all_faceted_with_controls_d_rank_long.png" alt="(ref:eCDFall)" width="100%" />
<p class="caption">
(#fig:eCDFall)(ref:eCDFall)
</p>
</div>
<!--chapter:end:Appendix.Rmd-->
</div>
<div class="footnotes footnotes-end-of-document">
<hr />
<ol>
<li id="fn1"><p>Siddhartha Mukherjee, <em>The <span>Gene</span>: <span>An Intimate History</span></em> (<span>Simon and Schuster</span>, 2016), <a href="https://books.google.com?id=XvAsDAAAQBAJ" role="doc-biblioref">https://books.google.com?id=XvAsDAAAQBAJ</a>.<a href="#fnref1" class="footnote-back">↩︎</a></p></li>
<li id="fn2"><p>Mukherjee.<a href="#fnref2" class="footnote-back">↩︎</a></p></li>
<li id="fn3"><p>Aristotle, <em>On the <span>Generation</span> of <span>Animals</span></em> (<span>Good Press</span>, 2021), <a href="https://books.google.com?id=aLUqEAAAQBAJ" role="doc-biblioref">https://books.google.com?id=aLUqEAAAQBAJ</a>.<a href="#fnref3" class="footnote-back">↩︎</a></p></li>
<li id="fn4"><p>Mukherjee, <em>The <span>Gene</span></em>.<a href="#fnref4" class="footnote-back">↩︎</a></p></li>
<li id="fn5"><p>Mukherjee.<a href="#fnref5" class="footnote-back">↩︎</a></p></li>
<li id="fn6"><p><span>“File:preformation.GIF — Wikimedia Commons<span>,</span> the Free Media Repository,”</span> 2019, <a href="\url{https://commons.wikimedia.org/w/index.php?title=File:Preformation.GIF&amp;oldid=372547016}" role="doc-biblioref">\url{https://commons.wikimedia.org/w/index.php?title=File:Preformation.GIF&amp;oldid=372547016}</a>.<a href="#fnref6" class="footnote-back">↩︎</a></p></li>
<li id="fn7"><p>Gordon Campbell, <em>King <span>James Bible</span>: 400th <span>Anniversary Edition</span></em> (<span>OUP Oxford</span>, 2010).<a href="#fnref7" class="footnote-back">↩︎</a></p></li>
<li id="fn8"><p>Mukherjee, <em>The <span>Gene</span></em>.<a href="#fnref8" class="footnote-back">↩︎</a></p></li>
<li id="fn9"><p>Mukherjee.<a href="#fnref9" class="footnote-back">↩︎</a></p></li>
<li id="fn10"><p>Wikimedia Commons, <span>“File:charles Darwin by g. Richmond.png — Wikimedia Commons<span>,</span> the Free Media Repository,”</span> 2017, <a href="\url{https://commons.wikimedia.org/w/index.php?title=File:Charles_Darwin_by_G._Richmond.png&amp;oldid=260695102}" role="doc-biblioref">\url{https://commons.wikimedia.org/w/index.php?title=File:Charles_Darwin_by_G._Richmond.png&amp;oldid=260695102}</a>.<a href="#fnref10" class="footnote-back">↩︎</a></p></li>
<li id="fn11"><p>Mukherjee, <em>The <span>Gene</span></em>.<a href="#fnref11" class="footnote-back">↩︎</a></p></li>
<li id="fn12"><p><span>“What Was <span>Megatherium</span>?”</span> accessed August 17, 2022, <a href="https://www.nhm.ac.uk/discover/what-was-megatherium.html" role="doc-biblioref">https://www.nhm.ac.uk/discover/what-was-megatherium.html</a>.<a href="#fnref12" class="footnote-back">↩︎</a></p></li>
<li id="fn13"><p>Mukherjee, <em>The <span>Gene</span></em>.<a href="#fnref13" class="footnote-back">↩︎</a></p></li>
<li id="fn14"><p>Mukherjee.<a href="#fnref14" class="footnote-back">↩︎</a></p></li>
<li id="fn15"><p>Mukherjee.<a href="#fnref15" class="footnote-back">↩︎</a></p></li>
<li id="fn16"><p>Mukherjee.<a href="#fnref16" class="footnote-back">↩︎</a></p></li>
<li id="fn17"><p>Paul Bancroft, <span>“Part 1: <span>An</span> Introduction to the <span>Gouldian</span> Finch - <span>Planet Aviary</span>,”</span> accessed August 22, 2022, <a href="https://planetaviary.com/part-1-an-introduction-to-the-gouldian-finch/" role="doc-biblioref">https://planetaviary.com/part-1-an-introduction-to-the-gouldian-finch/</a>.<a href="#fnref17" class="footnote-back">↩︎</a></p></li>
<li id="fn18"><p><span>“Red Dominates Black: Agonistic Signalling Among Head Morphs in the Colour Polymorphic <span>Gouldian</span> Finch,”</span> <em>Proceedings of the Royal Society B: Biological Sciences</em> 273, no. 1589 (April 22, 2006): 949–57, <a href="https://doi.org/10.1098/rspb.2005.3362" role="doc-biblioref">https://doi.org/10.1098/rspb.2005.3362</a>.<a href="#fnref18" class="footnote-back">↩︎</a></p></li>
<li id="fn19"><p>Mukherjee, <em>The <span>Gene</span></em>.<a href="#fnref19" class="footnote-back">↩︎</a></p></li>
<li id="fn20"><p>Mukherjee.<a href="#fnref20" class="footnote-back">↩︎</a></p></li>
<li id="fn21"><p><em>A <span>Naturalist</span>’s <span>Voyage</span>: <span>Journal</span> of <span>Researches Into</span> the <span>Natural History</span> and <span>Geology</span> of the <span>Countries Visited During</span> the <span>Voyage</span> of <span>H</span>.<span>M</span>.<span>S</span>. <span>Beagle</span> : With <span>Maps</span> and <span>Illustrations</span></em> (<span>Murray</span>, 1882), <a href="https://books.google.com?id=JyR7HY3FMtkC" role="doc-biblioref">https://books.google.com?id=JyR7HY3FMtkC</a>.<a href="#fnref21" class="footnote-back">↩︎</a></p></li>
<li id="fn22"><p><span>“File:darwin’s Finches by Gould.jpg — Wikimedia Commons<span>,</span> the Free Media Repository,”</span> 2022, <a href="\url{https://commons.wikimedia.org/w/index.php?title=File:Darwin%27s_finches_by_Gould.jpg&amp;oldid=643906085}" role="doc-biblioref">\url{https://commons.wikimedia.org/w/index.php?title=File:Darwin%27s_finches_by_Gould.jpg&amp;oldid=643906085}</a>.<a href="#fnref22" class="footnote-back">↩︎</a></p></li>
<li id="fn23"><p>Mukherjee, <em>The <span>Gene</span></em>.<a href="#fnref23" class="footnote-back">↩︎</a></p></li>
<li id="fn24"><p>Mukherjee.<a href="#fnref24" class="footnote-back">↩︎</a></p></li>
<li id="fn25"><p>Mukherjee.<a href="#fnref25" class="footnote-back">↩︎</a></p></li>
<li id="fn26"><p>Mukherjee.<a href="#fnref26" class="footnote-back">↩︎</a></p></li>
<li id="fn27"><p>Mukherjee.<a href="#fnref27" class="footnote-back">↩︎</a></p></li>
<li id="fn28"><p>Mukherjee.<a href="#fnref28" class="footnote-back">↩︎</a></p></li>
<li id="fn29"><p><span>“File:alfred-Russel-Wallace-C1895.jpg — Wikimedia Commons<span>,</span> the Free Media Repository,”</span> 2022, <a href="https://commons.wikimedia.org/w/index.php?title=File:Alfred-Russel-Wallace-c1895.jpg&amp;oldid=642853479" role="doc-biblioref">https://commons.wikimedia.org/w/index.php?title=File:Alfred-Russel-Wallace-c1895.jpg&amp;oldid=642853479</a>.<a href="#fnref29" class="footnote-back">↩︎</a></p></li>
<li id="fn30"><p>Charles Darwin, <em>On the <span>Origin</span> of <span>Species</span> by <span>Means</span> of <span>Natural Selection</span>, <span>Or</span>, <span>The Preservation</span> of <span>Favoured Races</span> in the <span>Struggle</span> for <span>Life</span></em> (<span>J. Murray</span>, 1859), <a href="https://books.google.com?id=gGUYt2PsDJcC" role="doc-biblioref">https://books.google.com?id=gGUYt2PsDJcC</a>.<a href="#fnref30" class="footnote-back">↩︎</a></p></li>
<li id="fn31"><p>Mukherjee, <em>The <span>Gene</span></em>.<a href="#fnref31" class="footnote-back">↩︎</a></p></li>
<li id="fn32"><p>Mukherjee.<a href="#fnref32" class="footnote-back">↩︎</a></p></li>
<li id="fn33"><p>Mukherjee.<a href="#fnref33" class="footnote-back">↩︎</a></p></li>
<li id="fn34"><p>Mukherjee.<a href="#fnref34" class="footnote-back">↩︎</a></p></li>
<li id="fn35"><p>Wikimedia Commons, <span>“File:jean-Baptiste de Lamarck.jpg — Wikimedia Commons<span>,</span> the Free Media Repository,”</span> 2022, <a href="\url{https://commons.wikimedia.org/w/index.php?title=File:Jean-Baptiste_de_Lamarck.jpg&amp;oldid=641428959}" role="doc-biblioref">\url{https://commons.wikimedia.org/w/index.php?title=File:Jean-Baptiste_de_Lamarck.jpg&amp;oldid=641428959}</a>.<a href="#fnref35" class="footnote-back">↩︎</a></p></li>
<li id="fn36"><p>This theory was experimentally tested by the German embryologist August Weismann, who amputated the tails of mice to determine whether the offspring would be born tailless. The offspring were all born with tails intact <span class="citation"> (Mukherjee, <em>The <span>Gene</span></em>)</span>.<a href="#fnref36" class="footnote-back">↩︎</a></p></li>
<li id="fn37"><p>Mukherjee.<a href="#fnref37" class="footnote-back">↩︎</a></p></li>
<li id="fn38"><p>Mukherjee.<a href="#fnref38" class="footnote-back">↩︎</a></p></li>
<li id="fn39"><p><span>“File:gregor Mendel 2.jpg — Wikimedia Commons<span>,</span> the Free Media Repository,”</span> 2021, <a href="\url{https://commons.wikimedia.org/w/index.php?title=File:Gregor_Mendel_2.jpg&amp;oldid=534728192}" role="doc-biblioref">\url{https://commons.wikimedia.org/w/index.php?title=File:Gregor_Mendel_2.jpg&amp;oldid=534728192}</a>.<a href="#fnref39" class="footnote-back">↩︎</a></p></li>
<li id="fn40"><p>Mukherjee, <em>The <span>Gene</span></em>.<a href="#fnref40" class="footnote-back">↩︎</a></p></li>
<li id="fn41"><p>Mukherjee.<a href="#fnref41" class="footnote-back">↩︎</a></p></li>
<li id="fn42"><p>Mukherjee.<a href="#fnref42" class="footnote-back">↩︎</a></p></li>
<li id="fn43"><p>Mukherjee.<a href="#fnref43" class="footnote-back">↩︎</a></p></li>
<li id="fn44"><p>Based around 28,000 plants, 40,000 flowers, and almost 400,000 seeds.<a href="#fnref44" class="footnote-back">↩︎</a></p></li>
<li id="fn45"><p>Mukherjee, <em>The <span>Gene</span></em>.<a href="#fnref45" class="footnote-back">↩︎</a></p></li>
<li id="fn46"><p>Mukherjee.<a href="#fnref46" class="footnote-back">↩︎</a></p></li>
<li id="fn47"><p>Mukherjee.<a href="#fnref47" class="footnote-back">↩︎</a></p></li>
<li id="fn48"><p>Mukherjee.<a href="#fnref48" class="footnote-back">↩︎</a></p></li>
<li id="fn49"><p>Gregor Mendel, <span>“Versuche Uber Pflanzen-Hybriden,”</span> <em>Verhandlungen Des Naturforschenden Vereins in Brunn Fur</em> 4 (1866): 3–47.<a href="#fnref49" class="footnote-back">↩︎</a></p></li>
<li id="fn50"><p>Mukherjee, <em>The <span>Gene</span></em>.<a href="#fnref50" class="footnote-back">↩︎</a></p></li>
<li id="fn51"><p>Mukherjee.<a href="#fnref51" class="footnote-back">↩︎</a></p></li>
<li id="fn52"><p>Mukherjee.<a href="#fnref52" class="footnote-back">↩︎</a></p></li>
<li id="fn53"><p>An ironic twist given the plant was named after Lamarck.<a href="#fnref53" class="footnote-back">↩︎</a></p></li>
<li id="fn54"><p>Mukherjee, <em>The <span>Gene</span></em>.<a href="#fnref54" class="footnote-back">↩︎</a></p></li>
<li id="fn55"><p>Mukherjee.<a href="#fnref55" class="footnote-back">↩︎</a></p></li>
<li id="fn56"><p>Mukherjee.<a href="#fnref56" class="footnote-back">↩︎</a></p></li>
<li id="fn57"><p>William Bateson, <em>Materials for the <span>Study</span> of <span>Variation</span>: <span>Treated With Especial Regard</span> to <span>Discontinuity</span> in the <span>Origin</span> of <span>Species</span></em> (<span>Forgotten Books.</span>, 1901).<a href="#fnref57" class="footnote-back">↩︎</a></p></li>
<li id="fn58"><p>Mukherjee, <em>The <span>Gene</span></em>.<a href="#fnref58" class="footnote-back">↩︎</a></p></li>
<li id="fn59"><p>Mukherjee.<a href="#fnref59" class="footnote-back">↩︎</a></p></li>
<li id="fn60"><p>C. T. Druery and William Bateson, <span>“Experiments in Plant Hybridization,”</span> <em>Journal of the Royal Horticultural Society</em> 26 (1901): 1–32.<a href="#fnref60" class="footnote-back">↩︎</a></p></li>
<li id="fn61"><p>Mukherjee, <em>The <span>Gene</span></em>.<a href="#fnref61" class="footnote-back">↩︎</a></p></li>
<li id="fn62"><p>Mukherjee.<a href="#fnref62" class="footnote-back">↩︎</a></p></li>
<li id="fn63"><p>Wikimedia Commons, <span>“File:Bateson2.jpg — Wikimedia Commons<span>,</span> the Free Media Repository,”</span> 2017, <a href="\url{https://commons.wikimedia.org/w/index.php?title=File:Bateson2.jpg&amp;oldid=255958176}" role="doc-biblioref">\url{https://commons.wikimedia.org/w/index.php?title=File:Bateson2.jpg&amp;oldid=255958176}</a>.<a href="#fnref63" class="footnote-back">↩︎</a></p></li>
<li id="fn64"><p>Mukherjee, <em>The <span>Gene</span></em>.<a href="#fnref64" class="footnote-back">↩︎</a></p></li>
<li id="fn65"><p>Mukherjee.<a href="#fnref65" class="footnote-back">↩︎</a></p></li>
<li id="fn66"><p>Among Galton’s somewhat eccentric activities, he would stroll through England and Scotland secretly tabulating beauty by ranking the women he met as “attractive”, “indifferent,” or “repellent” using pinpricks on a card in his pocket <span class="citation"> (Mukherjee)</span>.<a href="#fnref66" class="footnote-back">↩︎</a></p></li>
<li id="fn67"><p>Francis Galton, <span>“On <span>Men</span> of <span>Science</span>, Their <span>Nature</span> and Their <span>Nurture</span>,”</span> in <em>Proceedings of the <span>Royal Institution</span> of <span>Great Britain</span></em>, vol. 7, 1874, 227–36.<a href="#fnref67" class="footnote-back">↩︎</a></p></li>
<li id="fn68"><p><em>The <span>Life</span>, <span>Letters</span> and <span>Labours</span> of <span>Francis Galton</span></em> (<span>Cambridge University Press</span>, 2011), <a href="https://books.google.com?id=sz7vSOwOGzgC" role="doc-biblioref">https://books.google.com?id=sz7vSOwOGzgC</a>; Wikimedia Commons, <span>“File:francis Galton 1850s.jpg — Wikimedia Commons<span>,</span> the Free Media Repository,”</span> 2021, <a href="\url{https://commons.wikimedia.org/w/index.php?title=File:Francis_Galton_1850s.jpg&amp;oldid=569302173}" role="doc-biblioref">\url{https://commons.wikimedia.org/w/index.php?title=File:Francis_Galton_1850s.jpg&amp;oldid=569302173}</a>.<a href="#fnref68" class="footnote-back">↩︎</a></p></li>
<li id="fn69"><p>Mukherjee, <em>The <span>Gene</span></em>.<a href="#fnref69" class="footnote-back">↩︎</a></p></li>
<li id="fn70"><p>Mukherjee.<a href="#fnref70" class="footnote-back">↩︎</a></p></li>
<li id="fn71"><p>Mukherjee.<a href="#fnref71" class="footnote-back">↩︎</a></p></li>
<li id="fn72"><p>Mukherjee.<a href="#fnref72" class="footnote-back">↩︎</a></p></li>
<li id="fn73"><p>James Tabery, <em>Beyond Versus: <span>The</span> Struggle to Understand the Interaction of Nature and Nurture</em> (<span>mit Press</span>, 2014).<a href="#fnref73" class="footnote-back">↩︎</a></p></li>
<li id="fn74"><p>James D. Watson and Andrew Berry, <em><span>DNA</span>: <span>The</span> Secret of Life</em> (<span>Knopf</span>, 2009).<a href="#fnref74" class="footnote-back">↩︎</a></p></li>
<li id="fn75"><p>Francis Galton, <span>“Eugenics: <span>Its Definition</span>, <span>Scope</span>, and <span>Aims</span>,”</span> <em>American Journal of Sociology</em> 10, no. 1 (July 1904): 1–25, <a href="https://doi.org/10.1086/211280" role="doc-biblioref">https://doi.org/10.1086/211280</a>; Tabery, <em>Beyond Versus</em>.<a href="#fnref75" class="footnote-back">↩︎</a></p></li>
<li id="fn76"><p>Mukherjee, <em>The <span>Gene</span></em>.<a href="#fnref76" class="footnote-back">↩︎</a></p></li>
<li id="fn77"><p>Karl Pearson, <span>“X. <span>On</span> the Criterion That a Given System of Deviations from the Probable in the Case of a Correlated System of Variables Is Such That It Can Be Reasonably Supposed to Have Arisen from Random Sampling,”</span> <em>The London, Edinburgh, and Dublin Philosophical Magazine and Journal of Science</em> 50, no. 302 (July 1, 1900): 157–75, <a href="https://doi.org/10.1080/14786440009463897" role="doc-biblioref">https://doi.org/10.1080/14786440009463897</a>.<a href="#fnref77" class="footnote-back">↩︎</a></p></li>
<li id="fn78"><p>Karl Pearson and Olaus Magnus Friedrich Erdmann Henrici, <span>“<span>III</span>. <span>Contributions</span> to the Mathematical Theory of Evolution,”</span> <em>Philosophical Transactions of the Royal Society of London. (A.)</em> 185 (January 1894): 71–110, <a href="https://doi.org/10.1098/rsta.1894.0003" role="doc-biblioref">https://doi.org/10.1098/rsta.1894.0003</a>.<a href="#fnref78" class="footnote-back">↩︎</a></p></li>
<li id="fn79"><p>Karl Pearson, <span>“Mathematical Contributions to the Theory of Evolution, <span>On</span> the Law of Ancestral Heredity,”</span> <em>Proceedings of the Royal Society of London</em> 62, no. 379-387 (January 1898): 386–412, <a href="https://doi.org/10.1098/rspl.1897.0128" role="doc-biblioref">https://doi.org/10.1098/rspl.1897.0128</a>.<a href="#fnref79" class="footnote-back">↩︎</a></p></li>
<li id="fn80"><p>Karl Pearson, <span>“<span>LIII</span>. <span>On</span> Lines and Planes of Closest Fit to Systems of Points in Space,”</span> <em>The London, Edinburgh, and Dublin Philosophical Magazine and Journal of Science</em> 2, no. 11 (November 1, 1901): 559–72, <a href="https://doi.org/10.1080/14786440109462720" role="doc-biblioref">https://doi.org/10.1080/14786440109462720</a>.<a href="#fnref80" class="footnote-back">↩︎</a></p></li>
<li id="fn81"><p>Charles H. Pence, <span>“The Early History of Chance in Evolution,”</span> <em>Studies in History and Philosophy of Science Part A</em>, Integrated <span>History</span> and <span>Philosophy</span> of <span>Science</span> in <span>Practice</span>, 50 (April 1, 2015): 48–58, <a href="https://doi.org/10.1016/j.shpsa.2014.09.006" role="doc-biblioref">https://doi.org/10.1016/j.shpsa.2014.09.006</a>.<a href="#fnref81" class="footnote-back">↩︎</a></p></li>
<li id="fn82"><p>Wikimedia Commons, <span>“File:karl Pearson, 1910.jpg — Wikimedia Commons<span>,</span> the Free Media Repository,”</span> 2020, <a href="\url{https://commons.wikimedia.org/w/index.php?title=File:Karl_Pearson,_1910.jpg&amp;oldid=477532248}" role="doc-biblioref">\url{https://commons.wikimedia.org/w/index.php?title=File:Karl_Pearson,_1910.jpg&amp;oldid=477532248}</a>.<a href="#fnref82" class="footnote-back">↩︎</a></p></li>
<li id="fn83"><p>Daniel J. Kevles, <em>In the <span>Name</span> of <span>Eugenics</span>: <span>Genetics</span> and the <span>Uses</span> of <span>Human Heredity</span></em> (<span>Harvard University Press</span>, 1995), <a href="https://books.google.com?id=NWBi9kyb8xUC" role="doc-biblioref">https://books.google.com?id=NWBi9kyb8xUC</a>; Kathryn Paige Harden, <em>The <span>Genetic Lottery</span>: <span>Why DNA Matters</span> for <span>Social Equality</span></em> (<span>Princeton University Press</span>, 2021), <a href="https://books.google.com?id=R9oiEAAAQBAJ" role="doc-biblioref">https://books.google.com?id=R9oiEAAAQBAJ</a>.<a href="#fnref83" class="footnote-back">↩︎</a></p></li>
<li id="fn84"><p>Mukherjee, <em>The <span>Gene</span></em>.<a href="#fnref84" class="footnote-back">↩︎</a></p></li>
<li id="fn85"><p>R. A. Fisher, <span>“<span>XV</span>.—<span>The Correlation</span> Between <span>Relatives</span> on the <span>Supposition</span> of <span>Mendelian Inheritance</span>.”</span> January 1, 1919, <a href="https://doi.org/10.1017/s0080456800012163" role="doc-biblioref">https://doi.org/10.1017/s0080456800012163</a>.<a href="#fnref85" class="footnote-back">↩︎</a></p></li>
<li id="fn86"><p>Julian Huxley, <span>“Evolution. <span>The</span> Modern Synthesis.”</span> <em>Evolution. The Modern Synthesis.</em>, 1942; Tabery, <em>Beyond Versus</em>.<a href="#fnref86" class="footnote-back">↩︎</a></p></li>
<li id="fn87"><p>William B. Provine, <em>The Origins of Theoretical Population Genetics: With a New Afterword</em> (<span>University of Chicago Press</span>, 2001); Tabery, <em>Beyond Versus</em>.<a href="#fnref87" class="footnote-back">↩︎</a></p></li>
<li id="fn88"><p><span>“<span>XV</span>.—<span>The Correlation</span> Between <span>Relatives</span> on the <span>Supposition</span> of <span>Mendelian Inheritance</span>.”</span><a href="#fnref88" class="footnote-back">↩︎</a></p></li>
<li id="fn89"><p><a href="#fnref89" class="footnote-back">↩︎</a></p></li>
<li id="fn90"><p>Tabery, <em>Beyond Versus</em>.<a href="#fnref90" class="footnote-back">↩︎</a></p></li>
<li id="fn91"><p>Tabery.<a href="#fnref91" class="footnote-back">↩︎</a></p></li>
<li id="fn92"><p>R. A. Fisher and W. A. Mackenzie, <span>“Studies in Crop Variation. <span>II</span>. <span>The</span> Manurial Response of Different Potato Varieties,”</span> <em>The Journal of Agricultural Science</em> 13, no. 3 (July 1923): 311–20, <a href="https://doi.org/10.1017/S0021859600003592" role="doc-biblioref">https://doi.org/10.1017/S0021859600003592</a>.<a href="#fnref92" class="footnote-back">↩︎</a></p></li>
<li id="fn93"><p>Tabery, <em>Beyond Versus</em>.<a href="#fnref93" class="footnote-back">↩︎</a></p></li>
<li id="fn94"><p>Tabery.<a href="#fnref94" class="footnote-back">↩︎</a></p></li>
<li id="fn95"><p>Pauline Mazumdar, <em>Eugenics, Human Genetics and Human Failings: The <span>Eugenics Society</span>, Its Sources and Its Critics in <span>Britain</span></em> (<span>Routledge</span>, 2005); Tabery, <em>Beyond Versus</em>.<a href="#fnref95" class="footnote-back">↩︎</a></p></li>
<li id="fn96"><p>Tabery, <em>Beyond Versus</em>.<a href="#fnref96" class="footnote-back">↩︎</a></p></li>
<li id="fn97"><p>Adam Rutherford, <em>How to Argue with a Racist: <span>History</span>, Science, Race and Reality</em> (<span>Hachette UK</span>, 2020).<a href="#fnref97" class="footnote-back">↩︎</a></p></li>
<li id="fn98"><p>Mukherjee, <em>The <span>Gene</span></em>.<a href="#fnref98" class="footnote-back">↩︎</a></p></li>
<li id="fn99"><p>Mukherjee.<a href="#fnref99" class="footnote-back">↩︎</a></p></li>
<li id="fn100"><p>Mukherjee.<a href="#fnref100" class="footnote-back">↩︎</a></p></li>
<li id="fn101"><p>Mukherjee.<a href="#fnref101" class="footnote-back">↩︎</a></p></li>
<li id="fn102"><p>Mukherjee.<a href="#fnref102" class="footnote-back">↩︎</a></p></li>
<li id="fn103"><p>Mukherjee.<a href="#fnref103" class="footnote-back">↩︎</a></p></li>
<li id="fn104"><p>Mukherjee.<a href="#fnref104" class="footnote-back">↩︎</a></p></li>
<li id="fn105"><p>Mukherjee.<a href="#fnref105" class="footnote-back">↩︎</a></p></li>
<li id="fn106"><p>Shoji Fukamachi and Kiyoshi Naruse, <span>“100 Years Since the Medaka’s International Debut: <span>Aida</span>’s Legacy”</span> (<span>Genes to Genomes</span>, August 20, 2021), <a href="https://genestogenomes.org/100-years-since-the-medakas-international-debut-aidas-legacy/" role="doc-biblioref">https://genestogenomes.org/100-years-since-the-medakas-international-debut-aidas-legacy/</a>.<a href="#fnref106" class="footnote-back">↩︎</a></p></li>
<li id="fn107"><p>Tatuo Aida, <span>“On the <span>Inheritance</span> of <span>Color</span> in a <span>Fresh-Water Fish</span>, <span>Aplocheilus Latipes Temmick</span> and <span>Schlegel</span>, with <span>Special Reference</span> to <span>Sex-Linked Inheritance</span>,”</span> <em>Genetics</em> 6, no. 6 (November 1921): 554–73, <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1200522/" role="doc-biblioref">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1200522/</a>.<a href="#fnref107" class="footnote-back">↩︎</a></p></li>
<li id="fn108"><p>Kiyoshi Naruse, Minoru Tanaka, and Hiroyuki Takeda, <em>Medaka: A Model for Organogenesis, Human Disease, and Evolution</em> (<span>Springer Science &amp; Business Media</span>, 2011).<a href="#fnref108" class="footnote-back">↩︎</a></p></li>
<li id="fn109"><p><span>“On the <span>Inheritance</span> of <span>Color</span> in a <span>Fresh-Water Fish</span>, <span>Aplocheilus Latipes Temmick</span> and <span>Schlegel</span>, with <span>Special Reference</span> to <span>Sex-Linked Inheritance</span>.”</span><a href="#fnref109" class="footnote-back">↩︎</a></p></li>
<li id="fn110"><p>Mukherjee, <em>The <span>Gene</span></em>.<a href="#fnref110" class="footnote-back">↩︎</a></p></li>
<li id="fn111"><p>Mukherjee.<a href="#fnref111" class="footnote-back">↩︎</a></p></li>
<li id="fn112"><p>Mukherjee.<a href="#fnref112" class="footnote-back">↩︎</a></p></li>
<li id="fn113"><p><span>“Theodosius <span>Dobzhansky</span> on <span>Hybrid Sterility</span> and <span>Speciation</span>,”</span> <em>Genetics</em> 202, no. 1 (January 1, 2016): 5–7, <a href="https://doi.org/10.1534/genetics.115.184770" role="doc-biblioref">https://doi.org/10.1534/genetics.115.184770</a>.<a href="#fnref113" class="footnote-back">↩︎</a></p></li>
<li id="fn114"><p>Mukherjee, <em>The <span>Gene</span></em>.<a href="#fnref114" class="footnote-back">↩︎</a></p></li>
<li id="fn115"><p>Mukherjee.<a href="#fnref115" class="footnote-back">↩︎</a></p></li>
<li id="fn116"><p>Mukherjee.<a href="#fnref116" class="footnote-back">↩︎</a></p></li>
<li id="fn117"><p><span>“<span>DNA</span> Then and Now,”</span> accessed August 30, 2022, <a href="https://undsci.berkeley.edu/article/0_0_0/dna_14" role="doc-biblioref">https://undsci.berkeley.edu/article/0_0_0/dna_14</a>.<a href="#fnref117" class="footnote-back">↩︎</a></p></li>
<li id="fn118"><p>Mukherjee, <em>The <span>Gene</span></em>.<a href="#fnref118" class="footnote-back">↩︎</a></p></li>
<li id="fn119"><p>Mukherjee.<a href="#fnref119" class="footnote-back">↩︎</a></p></li>
<li id="fn120"><p>Frederick Sanger, Steven Nicklen, and Alan R. Coulson, <span>“<span>DNA</span> Sequencing with Chain-Terminating Inhibitors,”</span> <em>Proceedings of the National Academy of Sciences</em> 74, no. 12 (1977): 5463–67.<a href="#fnref120" class="footnote-back">↩︎</a></p></li>
<li id="fn121"><p><span>“The <span>Nobel Prize</span> in <span>Chemistry</span> 1980”</span> (<span>NobelPrize.org</span>), accessed August 30, 2022, <a href="https://www.nobelprize.org/prizes/chemistry/1980/summary/" role="doc-biblioref">https://www.nobelprize.org/prizes/chemistry/1980/summary/</a>.<a href="#fnref121" class="footnote-back">↩︎</a></p></li>
<li id="fn122"><p><span>“File:frederick Sanger2.jpg — Wikimedia Commons<span>,</span> the Free Media Repository,”</span> 2020, <a href="\url{https://commons.wikimedia.org/w/index.php?title=File:Frederick_Sanger2.jpg&amp;oldid=395411415}" role="doc-biblioref">\url{https://commons.wikimedia.org/w/index.php?title=File:Frederick_Sanger2.jpg&amp;oldid=395411415}</a>.<a href="#fnref122" class="footnote-back">↩︎</a></p></li>
<li id="fn123"><p>International Human Genome Sequencing Consortium, <span>“Initial Sequencing and Analysis of the Human Genome,”</span> <em>Nature</em> 409, no. 6822, 6822 (February 2001): 860–921, <a href="https://doi.org/10.1038/35057062" role="doc-biblioref">https://doi.org/10.1038/35057062</a>.<a href="#fnref123" class="footnote-back">↩︎</a></p></li>
<li id="fn124"><p>Sergey Nurk et al., <span>“The Complete Sequence of a Human Genome,”</span> <em>Science</em> 376, no. 6588 (April 2022): 44–53, <a href="https://doi.org/10.1126/science.abj6987" role="doc-biblioref">https://doi.org/10.1126/science.abj6987</a>.<a href="#fnref124" class="footnote-back">↩︎</a></p></li>
<li id="fn125"><p>Elizabeth Pennisi, <span>“A $100 Genome? <span>New DNA</span> Sequencers Could Be a <span>‘Game Changer’</span> for Biology, Medicine | <span>Science</span> | <span>AAAS</span>,”</span> <em>Science</em>, June 15, 2022, <a href="https://www.science.org/content/article/100-genome-new-dna-sequencers-could-be-game-changer-biology-medicine" role="doc-biblioref">https://www.science.org/content/article/100-genome-new-dna-sequencers-could-be-game-changer-biology-medicine</a>.<a href="#fnref125" class="footnote-back">↩︎</a></p></li>
<li id="fn126"><p>Alexander Payne et al., <span>“<span>BulkVis</span>: A Graphical Viewer for <span>Oxford</span> Nanopore Bulk <span>Fast5</span> Files,”</span> <em>Bioinformatics</em> 35, no. 13 (July 1, 2019): 2193–98, <a href="https://doi.org/10.1093/bioinformatics/bty841" role="doc-biblioref">https://doi.org/10.1093/bioinformatics/bty841</a>.<a href="#fnref126" class="footnote-back">↩︎</a></p></li>
<li id="fn127"><p>Yunhao Wang et al., <span>“Nanopore Sequencing Technology, Bioinformatics and Applications,”</span> <em>Nature Biotechnology</em> 39, no. 11, 11 (November 2021): 1348–65, <a href="https://doi.org/10.1038/s41587-021-01108-x" role="doc-biblioref">https://doi.org/10.1038/s41587-021-01108-x</a>.<a href="#fnref127" class="footnote-back">↩︎</a></p></li>
<li id="fn128"><p>Tinca J. C. Polderman et al., <span>“Meta-Analysis of the Heritability of Human Traits Based on Fifty Years of Twin Studies,”</span> <em>Nature Genetics</em> 47, no. 7, 7 (July 2015): 702–9, <a href="https://doi.org/10.1038/ng.3285" role="doc-biblioref">https://doi.org/10.1038/ng.3285</a>.<a href="#fnref128" class="footnote-back">↩︎</a></p></li>
<li id="fn129"><p>Polderman et al.<a href="#fnref129" class="footnote-back">↩︎</a></p></li>
<li id="fn130"><p><span>“Population Phenomena Inflate Genetic Associations of Complex Social Traits,”</span> <em>Science Advances</em> 6, no. 16 (April 15, 2020): eaay0328, <a href="https://doi.org/10.1126/sciadv.aay0328" role="doc-biblioref">https://doi.org/10.1126/sciadv.aay0328</a>.<a href="#fnref130" class="footnote-back">↩︎</a></p></li>
<li id="fn131"><p>Augustine Kong et al., <span>“The Nature of Nurture: <span>Effects</span> of Parental Genotypes,”</span> <em>Science</em> 359, no. 6374 (January 26, 2018): 424–28, <a href="https://doi.org/10.1126/science.aan6877" role="doc-biblioref">https://doi.org/10.1126/science.aan6877</a>.<a href="#fnref131" class="footnote-back">↩︎</a></p></li>
<li id="fn132"><p>Laurence J. Howe et al., <span>“Genetic Evidence for Assortative Mating on Alcohol Consumption in the <span>UK Biobank</span>,”</span> <em>Nature Communications</em> 10, no. 1, 1 (November 19, 2019): 5039, <a href="https://doi.org/10.1038/s41467-019-12424-x" role="doc-biblioref">https://doi.org/10.1038/s41467-019-12424-x</a>.<a href="#fnref132" class="footnote-back">↩︎</a></p></li>
<li id="fn133"><p><span>“Population Phenomena Inflate Genetic Associations of Complex Social Traits.”</span><a href="#fnref133" class="footnote-back">↩︎</a></p></li>
<li id="fn134"><p>Robert Plomin and Kathryn Asbury, <span>“Nature and <span>Nurture</span>: <span>Genetic</span> and <span>Environmental Influences</span> on <span>Behavior</span>,”</span> <em>The ANNALS of the American Academy of Political and Social Science</em> 600, no. 1 (July 1, 2005): 86–98, <a href="https://doi.org/10.1177/0002716205277184" role="doc-biblioref">https://doi.org/10.1177/0002716205277184</a>.<a href="#fnref134" class="footnote-back">↩︎</a></p></li>
<li id="fn135"><p>D. S. Falconer and T. F. C. Mackay, <em>Introduction to Quantitative Genetics</em> (<span>UK: Longman Group</span>, 1996).<a href="#fnref135" class="footnote-back">↩︎</a></p></li>
<li id="fn136"><p>William Bateson, <span>“Mendel’s <span>Principles</span> of <span>Heredity</span>: <span>Cambridge University Press</span>,”</span> <em>März 1909; 2nd Impr</em> 3 (1909): 1913; Falconer and Mackay, <em>Introduction to Quantitative Genetics</em>; Daniëlle Posthuma et al., <span>“Theory and <span>Practice</span> in <span>Quantitative Genetics</span>,”</span> <em>Twin Research and Human Genetics</em> 6, no. 5 (October 2003): 361–76, <a href="https://doi.org/10.1375/twin.6.5.361" role="doc-biblioref">https://doi.org/10.1375/twin.6.5.361</a>.<a href="#fnref136" class="footnote-back">↩︎</a></p></li>
<li id="fn137"><p>Peter M. Visscher, William G. Hill, and Naomi R. Wray, <span>“Heritability in the Genomics Era — Concepts and Misconceptions,”</span> <em>Nature Reviews Genetics</em> 9, no. 4, 4 (April 2008): 255–66, <a href="https://doi.org/10.1038/nrg2322" role="doc-biblioref">https://doi.org/10.1038/nrg2322</a>.<a href="#fnref137" class="footnote-back">↩︎</a></p></li>
<li id="fn138"><p>Falconer and Mackay, <em>Introduction to Quantitative Genetics</em>.<a href="#fnref138" class="footnote-back">↩︎</a></p></li>
<li id="fn139"><p>Falconer and Mackay.<a href="#fnref139" class="footnote-back">↩︎</a></p></li>
<li id="fn140"><p>Polderman et al., <span>“Meta-Analysis of the Heritability of Human Traits Based on Fifty Years of Twin Studies.”</span><a href="#fnref140" class="footnote-back">↩︎</a></p></li>
<li id="fn141"><p>Valerie S. Knopik et al., <em>Behavioral <span>Genetics</span></em> (<span>Macmillan Learning</span>, 2019), <a href="https://books.google.com?id=mtDJvQEACAAJ" role="doc-biblioref">https://books.google.com?id=mtDJvQEACAAJ</a>.<a href="#fnref141" class="footnote-back">↩︎</a></p></li>
<li id="fn142"><p>Polderman et al., <span>“Meta-Analysis of the Heritability of Human Traits Based on Fifty Years of Twin Studies.”</span><a href="#fnref142" class="footnote-back">↩︎</a></p></li>
<li id="fn143"><p>Polderman et al.<a href="#fnref143" class="footnote-back">↩︎</a></p></li>
<li id="fn144"><p><span>“The Nature of Nurture.”</span><a href="#fnref144" class="footnote-back">↩︎</a></p></li>
<li id="fn145"><p>Emil Uffelmann et al., <span>“Genome-Wide Association Studies,”</span> <em>Nature Reviews Methods Primers</em> 1, no. 1, 1 (August 26, 2021): 1–21, <a href="https://doi.org/10.1038/s43586-021-00056-9" role="doc-biblioref">https://doi.org/10.1038/s43586-021-00056-9</a>.<a href="#fnref145" class="footnote-back">↩︎</a></p></li>
<li id="fn146"><p>Uffelmann et al.<a href="#fnref146" class="footnote-back">↩︎</a></p></li>
<li id="fn147"><p>Uffelmann et al.<a href="#fnref147" class="footnote-back">↩︎</a></p></li>
<li id="fn148"><p>1000 Genomes Project Consortium et al., <span>“A Global Reference for Human Genetic Variation,”</span> <em>Nature</em> 526, no. 7571 (2015): 68.<a href="#fnref148" class="footnote-back">↩︎</a></p></li>
<li id="fn149"><p>Uffelmann et al., <span>“Genome-Wide Association Studies.”</span><a href="#fnref149" class="footnote-back">↩︎</a></p></li>
<li id="fn150"><p>W J Ewens and R S Spielman, <span>“The Transmission/Disequilibrium Test: History, Subdivision, and Admixture.”</span> <em>American Journal of Human Genetics</em> 57, no. 2 (August 1995): 455–64, <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1801556/" role="doc-biblioref">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1801556/</a>; Members of the Complex Trait Consortium, <span>“The Nature and Identification of Quantitative Trait Loci: A Community’s View,”</span> <em>Nature Reviews Genetics</em> 4, no. 11, 11 (November 2003): 911–16, <a href="https://doi.org/10.1038/nrg1206" role="doc-biblioref">https://doi.org/10.1038/nrg1206</a>.<a href="#fnref150" class="footnote-back">↩︎</a></p></li>
<li id="fn151"><p>Teri A. Manolio et al., <span>“Finding the Missing Heritability of Complex Diseases,”</span> <em>Nature</em> 461, no. 7265, 7265 (October 2009): 747–53, <a href="https://doi.org/10.1038/nature08494" role="doc-biblioref">https://doi.org/10.1038/nature08494</a>.<a href="#fnref151" class="footnote-back">↩︎</a></p></li>
<li id="fn152"><p>Jian Yang et al., <span>“Common <span>SNPs</span> Explain a Large Proportion of the Heritability for Human Height,”</span> <em>Nature Genetics</em> 42, no. 7, 7 (July 2010): 565–69, <a href="https://doi.org/10.1038/ng.608" role="doc-biblioref">https://doi.org/10.1038/ng.608</a>.<a href="#fnref152" class="footnote-back">↩︎</a></p></li>
<li id="fn153"><p>Jian Yang et al., <span>“<span>GCTA</span>: <span>A Tool</span> for <span class="nocase">Genome-wide Complex Trait Analysis</span>,”</span> <em>The American Journal of Human Genetics</em> 88, no. 1 (January 7, 2011): 76–82, <a href="https://doi.org/10.1016/j.ajhg.2010.11.011" role="doc-biblioref">https://doi.org/10.1016/j.ajhg.2010.11.011</a>.<a href="#fnref153" class="footnote-back">↩︎</a></p></li>
<li id="fn154"><p>Orson N. Eaton, <span>“A Quarter-Century of Inbreeding in Guinea-Pigs,”</span> <em>Journal of Experimental Zoology</em> 63, no. 2 (1932): 261–90, <a href="https://doi.org/10.1002/jez.1400630202" role="doc-biblioref">https://doi.org/10.1002/jez.1400630202</a>.<a href="#fnref154" class="footnote-back">↩︎</a></p></li>
<li id="fn155"><p>Sewall Wright, <span>“Coefficients of <span>Inbreeding</span> and <span>Relationship</span>,”</span> <em>The American Naturalist</em> 56, no. 645 (July 1922): 330–38, <a href="https://doi.org/10.1086/279872" role="doc-biblioref">https://doi.org/10.1086/279872</a>.<a href="#fnref155" class="footnote-back">↩︎</a></p></li>
<li id="fn156"><p>David P. Steensma, Robert A. Kyle, and Marc A. Shampo, <span>“Abbie <span>Lathrop</span>, the <span>‘<span>Mouse Woman</span> of <span>Granby</span>’</span>: <span>Rodent Fancier</span> and <span>Accidental Genetics Pioneer</span>,”</span> <em>Mayo Clinic Proceedings</em> 85, no. 11 (November 1, 2010): e83, <a href="https://doi.org/10.4065/mcp.2010.0647" role="doc-biblioref">https://doi.org/10.4065/mcp.2010.0647</a>.<a href="#fnref156" class="footnote-back">↩︎</a></p></li>
<li id="fn157"><p>Steensma, Kyle, and Shampo.<a href="#fnref157" class="footnote-back">↩︎</a></p></li>
<li id="fn158"><p>Steensma, Kyle, and Shampo.<a href="#fnref158" class="footnote-back">↩︎</a></p></li>
<li id="fn159"><p>Joy Bergelson and Fabrice Roux, <span>“Towards Identifying Genes Underlying Ecologically Relevant Traits in <span>Arabidopsis</span> Thaliana,”</span> <em>Nature Reviews Genetics</em> 11, no. 12, 12 (December 2010): 867–79, <a href="https://doi.org/10.1038/nrg2896" role="doc-biblioref">https://doi.org/10.1038/nrg2896</a>.<a href="#fnref159" class="footnote-back">↩︎</a></p></li>
<li id="fn160"><p>William C. Johnson and Paul Gepts, <span>“Segregation for Performance in Recombinant Inbred Populations Resulting from Inter-Gene Pool Crosses of Common Bean ( <span>Phaseolus</span> Vulgaris <span>L</span>.),”</span> <em>Euphytica</em> 106, no. 1 (March 1, 1999): 45–56, <a href="https://doi.org/10.1023/A:1003541201923" role="doc-biblioref">https://doi.org/10.1023/A:1003541201923</a>.<a href="#fnref160" class="footnote-back">↩︎</a></p></li>
<li id="fn161"><p>Vera Saliba-Colombani et al., <span>“Efficiency of <span>RFLP</span>, <span>RAPD</span>, and <span>AFLP</span> Markers for the Construction of an Intraspecific Map of the Tomato Genome,”</span> <em>Genome</em> 43, no. 1 (February 2000): 29–40, <a href="https://doi.org/10.1139/g99-096" role="doc-biblioref">https://doi.org/10.1139/g99-096</a>.<a href="#fnref161" class="footnote-back">↩︎</a></p></li>
<li id="fn162"><p>Anis M. Limami et al., <span>“Genetic and <span>Physiological Analysis</span> of <span>Germination Efficiency</span> in <span>Maize</span> in <span>Relation</span> to <span>Nitrogen Metabolism Reveals</span> the <span>Importance</span> of <span>Cytosolic Glutamine Synthetase</span>,”</span> <em>Plant Physiology</em> 130, no. 4 (December 1, 2002): 1860–70, <a href="https://doi.org/10.1104/pp.009647" role="doc-biblioref">https://doi.org/10.1104/pp.009647</a>.<a href="#fnref162" class="footnote-back">↩︎</a></p></li>
<li id="fn163"><p>Kathryn S. Evans et al., <span>“From <span>QTL</span> to Gene: <span>C</span>. Elegans Facilitates Discoveries of the Genetic Mechanisms Underlying Natural Variation,”</span> <em>Trends in Genetics</em> 37, no. 10 (October 1, 2021): 933–47, <a href="https://doi.org/10.1016/j.tig.2021.06.005" role="doc-biblioref">https://doi.org/10.1016/j.tig.2021.06.005</a>.<a href="#fnref163" class="footnote-back">↩︎</a></p></li>
<li id="fn164"><p>Trudy F. C. Mackay and Wen Huang, <span>“Charting the Genotype–Phenotype Map: Lessons from the <span>Drosophila</span> Melanogaster <span>Genetic Reference Panel</span>,”</span> <em>WIREs Developmental Biology</em> 7, no. 1 (2018): e289, <a href="https://doi.org/10.1002/wdev.289" role="doc-biblioref">https://doi.org/10.1002/wdev.289</a>.<a href="#fnref164" class="footnote-back">↩︎</a></p></li>
<li id="fn165"><p>Michael C. Saul et al., <span>“High-<span>Diversity Mouse Populations</span> for <span>Complex Traits</span>,”</span> <em>Trends in Genetics</em> 35, no. 7 (July 1, 2019): 501–14, <a href="https://doi.org/10.1016/j.tig.2019.04.003" role="doc-biblioref">https://doi.org/10.1016/j.tig.2019.04.003</a>.<a href="#fnref165" class="footnote-back">↩︎</a></p></li>
<li id="fn166"><p>David W. Threadgill et al., <span>“The <span>Collaborative Cross</span>: <span>A Recombinant Inbred Mouse Population</span> for the <span>Systems Genetic Era</span>,”</span> <em>ILAR Journal</em> 52, no. 1 (January 1, 2011): 24–31, <a href="https://doi.org/10.1093/ilar.52.1.24" role="doc-biblioref">https://doi.org/10.1093/ilar.52.1.24</a>.<a href="#fnref166" class="footnote-back">↩︎</a></p></li>
<li id="fn167"><p>Karen L Svenson et al., <span>“High-<span>Resolution Genetic Mapping Using</span> the <span>Mouse Diversity Outbred Population</span>,”</span> <em>Genetics</em> 190, no. 2 (February 1, 2012): 437–47, <a href="https://doi.org/10.1534/genetics.111.132597" role="doc-biblioref">https://doi.org/10.1534/genetics.111.132597</a>.<a href="#fnref167" class="footnote-back">↩︎</a></p></li>
<li id="fn168"><p>Jeremy L. Peirce et al., <span>“A New Set of <span>BXD</span> Recombinant Inbred Lines from Advanced Intercross Populations in Mice,”</span> <em>BMC Genetics</em> 5, no. 1 (April 29, 2004): 7, <a href="https://doi.org/10.1186/1471-2156-5-7" role="doc-biblioref">https://doi.org/10.1186/1471-2156-5-7</a>.<a href="#fnref168" class="footnote-back">↩︎</a></p></li>
<li id="fn169"><p>Joachim Wittbrodt, Akihiro Shima, and Manfred Schartl, <span>“Medaka — a Model Organism from the Far East,”</span> <em>Nature Reviews Genetics</em> 3, no. 1, 1 (January 2002): 53–64, <a href="https://doi.org/10.1038/nrg704" role="doc-biblioref">https://doi.org/10.1038/nrg704</a>.<a href="#fnref169" class="footnote-back">↩︎</a></p></li>
<li id="fn170"><p>Mikhail Spivakov et al., <span>“Genomic and <span>Phenotypic Characterization</span> of a <span>Wild Medaka Population</span>: <span>Towards</span> the <span>Establishment</span> of an <span>Isogenic Population Genetic Resource</span> in <span>Fish</span>,”</span> <em>G3 Genes|Genomes|Genetics</em> 4, no. 3 (March 1, 2014): 433–45, <a href="https://doi.org/10.1534/g3.113.008722" role="doc-biblioref">https://doi.org/10.1534/g3.113.008722</a>.<a href="#fnref170" class="footnote-back">↩︎</a></p></li>
<li id="fn171"><p>Fukamachi and Naruse, <span>“100 Years Since the Medaka’s International Debut.”</span><a href="#fnref171" class="footnote-back">↩︎</a></p></li>
<li id="fn172"><p>Spivakov et al., <span>“Genomic and <span>Phenotypic Characterization</span> of a <span>Wild Medaka Population</span>.”</span><a href="#fnref172" class="footnote-back">↩︎</a></p></li>
<li id="fn173"><p>Tomas Fitzgerald et al., <span>“The <span>Medaka Inbred Kiyosu-Karlsruhe</span> (<span>MIKK</span>) Panel,”</span> <em>Genome Biology</em> 23, no. 1 (February 21, 2022): 59, <a href="https://doi.org/10.1186/s13059-022-02623-z" role="doc-biblioref">https://doi.org/10.1186/s13059-022-02623-z</a>.<a href="#fnref173" class="footnote-back">↩︎</a></p></li>
<li id="fn174"><p><a href="#fnref174" class="footnote-back">↩︎</a></p></li>
<li id="fn175"><p><span>“Genomic Variations and Epigenomic Landscape of the <span>Medaka Inbred Kiyosu-Karlsruhe</span> (<span>MIKK</span>) Panel,”</span> <em>Genome Biology</em> 23, no. 1 (February 21, 2022): 58, <a href="https://doi.org/10.1186/s13059-022-02602-4" role="doc-biblioref">https://doi.org/10.1186/s13059-022-02602-4</a>.<a href="#fnref175" class="footnote-back">↩︎</a></p></li>
<li id="fn176"><p>Ju-Hyun Park et al., <span>“Distribution of Allele Frequencies and Effect Sizes and Their Interrelationships for Common Genetic Susceptibility Variants,”</span> <em>Proceedings of the National Academy of Sciences</em> 108, no. 44 (November 2011): 18026–31, <a href="https://doi.org/10.1073/pnas.1114759108" role="doc-biblioref">https://doi.org/10.1073/pnas.1114759108</a>; The UK10K Consortium, <span>“The <span>Uk10k</span> Project Identifies Rare Variants in Health and Disease,”</span> <em>Nature</em> 526, no. 7571, 7571 (October 2015): 82–90, <a href="https://doi.org/10.1038/nature14962" role="doc-biblioref">https://doi.org/10.1038/nature14962</a>.<a href="#fnref176" class="footnote-back">↩︎</a></p></li>
<li id="fn177"><p>Woong Kim et al., <span>“The Period of the Somite Segmentation Clock Is Sensitive to <span>Notch</span> Activity,”</span> <em>Molecular Biology of the Cell</em> 22, no. 18 (September 15, 2011): 3541–49, <a href="https://doi.org/10.1091/mbc.e11-02-0139" role="doc-biblioref">https://doi.org/10.1091/mbc.e11-02-0139</a>.<a href="#fnref177" class="footnote-back">↩︎</a></p></li>
<li id="fn178"><p><span>“The Long and Short of It: <span>Somite</span> Formation in Mice,”</span> <em>Developmental Dynamics</em> 235, no. 9 (2006): 2330–36, <a href="https://doi.org/10.1002/dvdy.20850" role="doc-biblioref">https://doi.org/10.1002/dvdy.20850</a>.<a href="#fnref178" class="footnote-back">↩︎</a></p></li>
<li id="fn179"><p>Alexis Hubaud and Olivier Pourqui’e, <span>“Signalling Dynamics in Vertebrate Segmentation,”</span> <em>Nature Reviews Molecular Cell Biology</em> 15, no. 11, 11 (November 2014): 709–21, <a href="https://doi.org/10.1038/nrm3891" role="doc-biblioref">https://doi.org/10.1038/nrm3891</a>; Mitsuhiro Matsuda et al., <span>“Species-Specific Segmentation Clock Periods Are Due to Differential Biochemical Reaction Speeds,”</span> <em>Science</em> 369, no. 6510 (2020): 1450–55.<a href="#fnref179" class="footnote-back">↩︎</a></p></li>
<li id="fn180"><p>C’eline Gomez et al., <span>“Control of Segment Number in Vertebrate Embryos,”</span> <em>Nature</em> 454, no. 7202, 7202 (July 2008): 335–39, <a href="https://doi.org/10.1038/nature07020" role="doc-biblioref">https://doi.org/10.1038/nature07020</a>.<a href="#fnref180" class="footnote-back">↩︎</a></p></li>
<li id="fn181"><p>Gomez et al.<a href="#fnref181" class="footnote-back">↩︎</a></p></li>
<li id="fn182"><p>Henning J Falk et al., <span>“Imaging the Onset of Oscillatory Signaling Dynamics During Mouse Embryo Gastrulation,”</span> <em>Development (Cambridge, England)</em> 149, no. 13 (July 1, 2022): dev200083, <a href="https://doi.org/10.1242/dev.200083" role="doc-biblioref">https://doi.org/10.1242/dev.200083</a>.<a href="#fnref182" class="footnote-back">↩︎</a></p></li>
<li id="fn183"><p><span>“Species-Specific Segmentation Clock Periods Are Due to Differential Biochemical Reaction Speeds.”</span><a href="#fnref183" class="footnote-back">↩︎</a></p></li>
<li id="fn184"><p>Ali Seleit, Alexander Aulehla, and Alexandre Paix, <span>“Endogenous Protein Tagging in Medaka Using a Simplified <span>CRISPR</span>/<span>Cas9</span> Knock-in Approach.”</span> <em>eLife</em> 10 (December 1, 2021): e75050, <a href="https://doi.org/10.7554/elife.75050" role="doc-biblioref">https://doi.org/10.7554/elife.75050</a>.<a href="#fnref184" class="footnote-back">↩︎</a></p></li>
<li id="fn185"><p>D. E. Ruzzante and R. W. Doyle, <span>“Behavioural and Growth Responses to the Intensity of Intraspecific Social Interaction Among Medaka, <span>Oryzias</span> Latipes (<span>Temminck</span> and <span>Schlegel</span>) (<span>Pisces</span>, <span>Cyprinodontidae</span>),”</span> <em>Journal of Fish Biology</em> 37, no. 5 (1990): 663–73, <a href="https://doi.org/10.1111/j.1095-8649.1990.tb02531.x" role="doc-biblioref">https://doi.org/10.1111/j.1095-8649.1990.tb02531.x</a>; Simon N. Young, <span>“The Neurobiology of Human Social Behaviour: An Important but Neglected Topic,”</span> <em>Journal of Psychiatry and Neuroscience</em> 33, no. 5 (September 1, 2008): 391–92, <a href="https://www.jpn.ca/content/33/5/391" role="doc-biblioref">https://www.jpn.ca/content/33/5/391</a>.<a href="#fnref185" class="footnote-back">↩︎</a></p></li>
<li id="fn186"><p>David M. Buss, <span>“Evolutionary Personality Psychology,”</span> <em>Annual Review of Psychology</em> 42 (1991): 459–91, <a href="https://doi.org/10.1146/annurev.ps.42.020191.002331" role="doc-biblioref">https://doi.org/10.1146/annurev.ps.42.020191.002331</a>; Darwin, <em>On the <span>Origin</span> of <span>Species</span> by <span>Means</span> of <span>Natural Selection</span>, <span>Or</span>, <span>The Preservation</span> of <span>Favoured Races</span> in the <span>Struggle</span> for <span>Life</span></em>.<a href="#fnref186" class="footnote-back">↩︎</a></p></li>
<li id="fn187"><p>Steven L. Lima and Lawrence M. Dill, <span>“Behavioral Decisions Made Under the Risk of Predation: A Review and Prospectus,”</span> <em>Canadian Journal of Zoology</em> 68, no. 4 (April 1990): 619–40, <a href="https://doi.org/10.1139/z90-092" role="doc-biblioref">https://doi.org/10.1139/z90-092</a>.<a href="#fnref187" class="footnote-back">↩︎</a></p></li>
<li id="fn188"><p>David Sloan Wilson et al., <span>“Shyness and Boldness in Humans and Other Animals,”</span> <em>Trends in Ecology &amp; Evolution</em> 9, no. 11 (November 1, 1994): 442–46, <a href="https://doi.org/10.1016/0169-5347(94)90134-1" role="doc-biblioref">https://doi.org/10.1016/0169-5347(94)90134-1</a>.<a href="#fnref188" class="footnote-back">↩︎</a></p></li>
<li id="fn189"><p>Lima and Dill, <span>“Behavioral Decisions Made Under the Risk of Predation.”</span><a href="#fnref189" class="footnote-back">↩︎</a></p></li>
<li id="fn190"><p>Kenth Svartberg, <span>“Shyness–Boldness Predicts Performance in Working Dogs,”</span> <em>Applied Animal Behaviour Science</em> 79, no. 2 (October 20, 2002): 157–74, <a href="https://doi.org/10.1016/S0168-1591(02)00120-X" role="doc-biblioref">https://doi.org/10.1016/S0168-1591(02)00120-X</a>; Culum Brown, Fiona Burgess, and Victoria A. Braithwaite, <span>“Heritable and Experiential Effects on Boldness in a Tropical Poeciliid,”</span> <em>Behavioral Ecology and Sociobiology</em> 62, no. 2 (December 1, 2007): 237–43, <a href="https://doi.org/10.1007/s00265-007-0458-3" role="doc-biblioref">https://doi.org/10.1007/s00265-007-0458-3</a>.<a href="#fnref190" class="footnote-back">↩︎</a></p></li>
<li id="fn191"><p>Brown, Burgess, and Braithwaite, <span>“Heritable and Experiential Effects on Boldness in a Tropical Poeciliid.”</span><a href="#fnref191" class="footnote-back">↩︎</a></p></li>
<li id="fn192"><p>C. Brown, F. Jones, and V. A. Braithwaite, <span>“Correlation Between Boldness and Body Mass in Natural Populations of the Poeciliid <span>Brachyrhaphis</span> Episcopi,”</span> <em>Journal of Fish Biology</em> 71, no. 6 (2007): 1590–1601, <a href="https://doi.org/10.1111/j.1095-8649.2007.01627.x" role="doc-biblioref">https://doi.org/10.1111/j.1095-8649.2007.01627.x</a>.<a href="#fnref192" class="footnote-back">↩︎</a></p></li>
<li id="fn193"><p>Brown, Jones, and Braithwaite; Kevin Laland, Jens Krause, and Culum Brown, <em>Fish <span>Cognition</span> and <span>Behavior</span></em> (<span>John Wiley &amp; Sons</span>, 2011), <a href="https://books.google.com?id=cI9gbVyH6lsC" role="doc-biblioref">https://books.google.com?id=cI9gbVyH6lsC</a>; Tyrone Lucon-Xiccato et al., <span>“Comparison of Anxiety-Like and Social Behaviour in Medaka and Zebrafish,”</span> <em>Scientific Reports</em> 12, no. 1, 1 (June 28, 2022): 10926, <a href="https://doi.org/10.1038/s41598-022-14978-1" role="doc-biblioref">https://doi.org/10.1038/s41598-022-14978-1</a>; Tyrone Lucon-Xiccato et al., <span>“Development of <span>Open-Field Behaviour</span> in the <span>Medaka</span>, <span>Oryzias</span> Latipes,”</span> <em>Biology</em> 9, no. 11, 11 (November 2020): 389, <a href="https://doi.org/10.3390/biology9110389" role="doc-biblioref">https://doi.org/10.3390/biology9110389</a>; Tyrone Lucon-Xiccato and Angelo Bisazza, <span>“Individual Differences in Cognition Among Teleost Fishes,”</span> <em>Behavioural Processes</em>, The <span>Cognition</span> of <span>Fish</span>, 141 (August 1, 2017): 184–95, <a href="https://doi.org/10.1016/j.beproc.2017.01.015" role="doc-biblioref">https://doi.org/10.1016/j.beproc.2017.01.015</a>; Wataru Matsunaga and Eiji Watanabe, <span>“Habituation of Medaka (<span>Oryzias</span> Latipes) Demonstrated by Open-Field Testing,”</span> <em>Behavioural Processes</em> 85, no. 2 (October 1, 2010): 142–50, <a href="https://doi.org/10.1016/j.beproc.2010.06.019" role="doc-biblioref">https://doi.org/10.1016/j.beproc.2010.06.019</a>.<a href="#fnref193" class="footnote-back">↩︎</a></p></li>
<li id="fn194"><p>Brown, Jones, and Braithwaite, <span>“Correlation Between Boldness and Body Mass in Natural Populations of the Poeciliid <span>Brachyrhaphis</span> Episcopi”</span>; Joachim Schjolden, Argaudas Stoskhus, and Svante Winberg, <span>“Does <span>Individual Variation</span> in <span>Stress Responses</span> and <span>Agonistic Behavior Reflect Divergent Stress Coping Strategies</span> in <span>Juvenile Rainbow Trout</span>?”</span> <em>Physiological and Biochemical Zoology</em> 78, no. 5 (September 2005): 715–23, <a href="https://doi.org/10.1086/432153" role="doc-biblioref">https://doi.org/10.1086/432153</a>; David S. Wilson et al., <span>“Shy-Bold Continuum in Pumpkinseed Sunfish (<span>Lepomis</span> Gibbosus): <span>An</span> Ecological Study of a Psychological Trait,”</span> <em>Journal of Comparative Psychology</em> 107, no. 3 (1993): 250–60, <a href="https://doi.org/10.1037/0735-7036.107.3.250" role="doc-biblioref">https://doi.org/10.1037/0735-7036.107.3.250</a>; Dominic Wright, Roger K. Butlin, and Örjan Carlborg, <span>“Epistatic Regulation of Behavioural and Morphological Traits in the Zebrafish (<span>Danio</span> Rerio),”</span> <em>Behavior Genetics</em> 36, no. 6 (November 1, 2006): 914–22, <a href="https://doi.org/10.1007/s10519-006-9080-9" role="doc-biblioref">https://doi.org/10.1007/s10519-006-9080-9</a>; D. Wright et al., <span>“Inter and Intra-Population Variation in Shoaling and Boldness in the Zebrafish (<span>Danio</span> Rerio),”</span> <em>Journal of Fish Biology</em> 63, no. s1 (2003): 258–59, <a href="https://doi.org/10.1111/j.1095-8649.2003.216bw.x" role="doc-biblioref">https://doi.org/10.1111/j.1095-8649.2003.216bw.x</a>.<a href="#fnref194" class="footnote-back">↩︎</a></p></li>
<li id="fn195"><p>Brown, Jones, and Braithwaite, <span>“Correlation Between Boldness and Body Mass in Natural Populations of the Poeciliid <span>Brachyrhaphis</span> Episcopi.”</span><a href="#fnref195" class="footnote-back">↩︎</a></p></li>
<li id="fn196"><p>Amelie Baud et al., <span>“Genetic <span>Variation</span> in the <span>Social Environment Contributes</span> to <span>Health</span> and <span>Disease</span>,”</span> <em>PLOS Genetics</em> 13, no. 1 (January 25, 2017): e1006498, <a href="https://doi.org/10.1371/journal.pgen.1006498" role="doc-biblioref">https://doi.org/10.1371/journal.pgen.1006498</a>.<a href="#fnref196" class="footnote-back">↩︎</a></p></li>
<li id="fn197"><p>Baud et al.<a href="#fnref197" class="footnote-back">↩︎</a></p></li>
<li id="fn198"><p>Esther D. Ellen et al., <span>“The Prospects of Selection for Social Genetic Effects to Improve Welfare and Productivity in Livestock,”</span> <em>Frontiers in Genetics</em> 5 (2014), <a href="https://www.frontiersin.org/articles/10.3389/fgene.2014.00377" role="doc-biblioref">https://www.frontiersin.org/articles/10.3389/fgene.2014.00377</a>.<a href="#fnref198" class="footnote-back">↩︎</a></p></li>
<li id="fn199"><p>Kong et al., <span>“The Nature of Nurture.”</span><a href="#fnref199" class="footnote-back">↩︎</a></p></li>
<li id="fn200"><p><span>“Estimating Indirect Parental Genetic Effects on Offspring Phenotypes Using Virtual Parental Genotypes Derived from Sibling and Half Sibling Pairs,”</span> <em>PLOS Genetics</em> 16, no. 10 (October 26, 2020): e1009154, <a href="https://doi.org/10.1371/journal.pgen.1009154" role="doc-biblioref">https://doi.org/10.1371/journal.pgen.1009154</a>.<a href="#fnref200" class="footnote-back">↩︎</a></p></li>
<li id="fn201"><p><span>“The <span>Medaka Inbred Kiyosu-Karlsruhe</span> (<span>MIKK</span>) Panel.”</span><a href="#fnref201" class="footnote-back">↩︎</a></p></li>
<li id="fn202"><p><span>“Genomic Variations and Epigenomic Landscape of the <span>Medaka Inbred Kiyosu-Karlsruhe</span> (<span>MIKK</span>) Panel.”</span><a href="#fnref202" class="footnote-back">↩︎</a></p></li>
<li id="fn203"><p>Plomin and Asbury, <span>“Nature and <span>Nurture</span>.”</span><a href="#fnref203" class="footnote-back">↩︎</a></p></li>
<li id="fn204"><p>Bergelson and Roux, <span>“Towards Identifying Genes Underlying Ecologically Relevant Traits in <span>Arabidopsis</span> Thaliana.”</span><a href="#fnref204" class="footnote-back">↩︎</a></p></li>
<li id="fn205"><p>Johnson and Gepts, <span>“Segregation for Performance in Recombinant Inbred Populations Resulting from Inter-Gene Pool Crosses of Common Bean ( <span>Phaseolus</span> Vulgaris <span>L</span>.).”</span><a href="#fnref205" class="footnote-back">↩︎</a></p></li>
<li id="fn206"><p>Saliba-Colombani et al., <span>“Efficiency of <span>RFLP</span>, <span>RAPD</span>, and <span>AFLP</span> Markers for the Construction of an Intraspecific Map of the Tomato Genome.”</span><a href="#fnref206" class="footnote-back">↩︎</a></p></li>
<li id="fn207"><p>Limami et al., <span>“Genetic and <span>Physiological Analysis</span> of <span>Germination Efficiency</span> in <span>Maize</span> in <span>Relation</span> to <span>Nitrogen Metabolism Reveals</span> the <span>Importance</span> of <span>Cytosolic Glutamine Synthetase</span>.”</span><a href="#fnref207" class="footnote-back">↩︎</a></p></li>
<li id="fn208"><p>Evans et al., <span>“From <span>QTL</span> to Gene.”</span><a href="#fnref208" class="footnote-back">↩︎</a></p></li>
<li id="fn209"><p>Mackay and Huang, <span>“Charting the Genotype–Phenotype Map.”</span><a href="#fnref209" class="footnote-back">↩︎</a></p></li>
<li id="fn210"><p>Saul et al., <span>“High-<span>Diversity Mouse Populations</span> for <span>Complex Traits</span>.”</span><a href="#fnref210" class="footnote-back">↩︎</a></p></li>
<li id="fn211"><p>Threadgill et al., <span>“The <span>Collaborative Cross</span>.”</span><a href="#fnref211" class="footnote-back">↩︎</a></p></li>
<li id="fn212"><p>Svenson et al., <span>“High-<span>Resolution Genetic Mapping Using</span> the <span>Mouse Diversity Outbred Population</span>.”</span><a href="#fnref212" class="footnote-back">↩︎</a></p></li>
<li id="fn213"><p>Peirce et al., <span>“A New Set of <span>BXD</span> Recombinant Inbred Lines from Advanced Intercross Populations in Mice.”</span><a href="#fnref213" class="footnote-back">↩︎</a></p></li>
<li id="fn214"><p>Wittbrodt, Shima, and Schartl, <span>“Medaka — a Model Organism from the Far East.”</span><a href="#fnref214" class="footnote-back">↩︎</a></p></li>
<li id="fn215"><p>Spivakov et al., <span>“Genomic and <span>Phenotypic Characterization</span> of a <span>Wild Medaka Population</span>.”</span><a href="#fnref215" class="footnote-back">↩︎</a></p></li>
<li id="fn216"><p>Fitzgerald et al., <span>“The <span>Medaka Inbred Kiyosu-Karlsruhe</span> (<span>MIKK</span>) Panel.”</span><a href="#fnref216" class="footnote-back">↩︎</a></p></li>
<li id="fn217"><p><a href="#fnref217" class="footnote-back">↩︎</a></p></li>
<li id="fn218"><p><span>“Genomic Variations and Epigenomic Landscape of the <span>Medaka Inbred Kiyosu-Karlsruhe</span> (<span>MIKK</span>) Panel.”</span><a href="#fnref218" class="footnote-back">↩︎</a></p></li>
<li id="fn219"><p><span>“The <span>Medaka Inbred Kiyosu-Karlsruhe</span> (<span>MIKK</span>) Panel.”</span><a href="#fnref219" class="footnote-back">↩︎</a></p></li>
<li id="fn220"><p><span>“Genomic Variations and Epigenomic Landscape of the <span>Medaka Inbred Kiyosu-Karlsruhe</span> (<span>MIKK</span>) Panel.”</span><a href="#fnref220" class="footnote-back">↩︎</a></p></li>
<li id="fn221"><p>Spivakov et al., <span>“Genomic and <span>Phenotypic Characterization</span> of a <span>Wild Medaka Population</span>.”</span><a href="#fnref221" class="footnote-back">↩︎</a></p></li>
<li id="fn222"><p><span>“Index of /Pub/Release-102/Emf/Ensembl-Compara/Multiple_alignments/50_fish.epo/,”</span> accessed January 25, 2022, <a href="https://ftp.ensembl.org/pub/release-102/emf/ensembl-compara/multiple_alignments/50_fish.epo/" role="doc-biblioref">https://ftp.ensembl.org/pub/release-102/emf/ensembl-compara/multiple_alignments/50_fish.epo/</a>.<a href="#fnref222" class="footnote-back">↩︎</a></p></li>
<li id="fn223"><p>Emmanuel Paradis and Klaus Schliep, <span>“Ape 5.0: An Environment for Modern Phylogenetics and Evolutionary Analyses in <span>R</span>,”</span> <em>Bioinformatics</em> 35, no. 3 (February 1, 2019): 526–28, <a href="https://doi.org/10.1093/bioinformatics/bty633" role="doc-biblioref">https://doi.org/10.1093/bioinformatics/bty633</a>.<a href="#fnref223" class="footnote-back">↩︎</a></p></li>
<li id="fn224"><p>Simon H. Martin, John W. Davey, and Chris D. Jiggins, <span>“Evaluating the <span>Use</span> of <span>ABBA</span>–<span>BABA Statistics</span> to <span>Locate Introgressed Loci</span>,”</span> <em>Molecular Biology and Evolution</em> 32, no. 1 (January 1, 2015): 244–57, <a href="https://doi.org/10.1093/molbev/msu269" role="doc-biblioref">https://doi.org/10.1093/molbev/msu269</a>.<a href="#fnref224" class="footnote-back">↩︎</a></p></li>
<li id="fn225"><p><a href="#fnref225" class="footnote-back">↩︎</a></p></li>
<li id="fn226"><p>Simon martin, <em>Simonhmartin/Genomics_general</em>, 2022, <a href="https://github.com/simonhmartin/genomics_general" role="doc-biblioref">https://github.com/simonhmartin/genomics_general</a>.<a href="#fnref226" class="footnote-back">↩︎</a></p></li>
<li id="fn227"><p><span>“Genomic and <span>Phenotypic Characterization</span> of a <span>Wild Medaka Population</span>.”</span><a href="#fnref227" class="footnote-back">↩︎</a></p></li>
<li id="fn228"><p>Takafumi Katsumura et al., <span>“Medaka <span>Population Genome Structure</span> and <span>Demographic History Described</span> via <span class="nocase">Genotyping-by-Sequencing</span>,”</span> <em>G3 Genes|Genomes|Genetics</em> 9, no. 1 (January 1, 2019): 217–28, <a href="https://doi.org/10.1534/g3.118.200779" role="doc-biblioref">https://doi.org/10.1534/g3.118.200779</a>.<a href="#fnref228" class="footnote-back">↩︎</a></p></li>
<li id="fn229"><p>Leger et al., <span>“Genomic Variations and Epigenomic Landscape of the <span>Medaka Inbred Kiyosu-Karlsruhe</span> (<span>MIKK</span>) Panel.”</span><a href="#fnref229" class="footnote-back">↩︎</a></p></li>
<li id="fn230"><p>Ary A. Hoffmann, Carla M. Sgr‘o, and Andrew R. Weeks, <span>“Chromosomal Inversion Polymorphisms and Adaptation,”</span> <em>Trends in Ecology &amp; Evolution</em> 19, no. 9 (September 1, 2004): 482–88, <a href="https://doi.org/10.1016/j.tree.2004.06.013" role="doc-biblioref">https://doi.org/10.1016/j.tree.2004.06.013</a>.<a href="#fnref230" class="footnote-back">↩︎</a></p></li>
<li id="fn231"><p>David Fredman et al., <span>“Complex <span class="nocase">SNP-related</span> Sequence Variation in Segmental Genome Duplications,”</span> <em>Nature Genetics</em> 36, no. 8, 8 (August 2004): 861–66, <a href="https://doi.org/10.1038/ng1401" role="doc-biblioref">https://doi.org/10.1038/ng1401</a>.<a href="#fnref231" class="footnote-back">↩︎</a></p></li>
<li id="fn232"><p><span>“A Global Reference for Human Genetic Variation,”</span> <em>Nature</em> 526, no. 7571 (2015): 68–74, <a href="https://doi.org/10.1038/nature15393" role="doc-biblioref">https://doi.org/10.1038/nature15393</a>.<a href="#fnref232" class="footnote-back">↩︎</a></p></li>
<li id="fn233"><p>Aaron McKenna et al., <span>“The <span>Genome Analysis Toolkit</span>: <span>A MapReduce</span> Framework for Analyzing Next-Generation <span>DNA</span> Sequencing Data,”</span> <em>Genome Research</em> 20, no. 9 (January 9, 2010): 1297–1303, <a href="https://doi.org/10.1101/gr.107524.110" role="doc-biblioref">https://doi.org/10.1101/gr.107524.110</a>.<a href="#fnref233" class="footnote-back">↩︎</a></p></li>
<li id="fn234"><p>Christopher C Chang et al., <span>“Second-Generation <span>PLINK</span>: Rising to the Challenge of Larger and Richer Datasets,”</span> <em>GigaScience</em> 4, no. 1 (December 2015): 7, <a href="https://doi.org/10.1186/s13742-015-0047-8" role="doc-biblioref">https://doi.org/10.1186/s13742-015-0047-8</a>; Shaun M Purcell and Christopher C Chang, <em><span>PLINK</span> 1.9</em>, n.d., <a href="https://www.cog-genomics.org/plink/1.9/" role="doc-biblioref">www.cog-genomics.org/plink/1.9/</a>.<a href="#fnref234" class="footnote-back">↩︎</a></p></li>
<li id="fn235"><p><span>“A <span>Detailed Linkage Map</span> of <span>Medaka</span>, <span>Oryzias</span> Latipes: <span>Comparative Genomics</span> and <span>Genome Evolution</span>,”</span> <em>Genetics</em> 154, no. 4 (April 1, 2000): 1773–84, <a href="https://www.genetics.org/content/154/4/1773" role="doc-biblioref">https://www.genetics.org/content/154/4/1773</a>.<a href="#fnref235" class="footnote-back">↩︎</a></p></li>
<li id="fn236"><p><span>“Genomic Variations and Epigenomic Landscape of the <span>Medaka Inbred Kiyosu-Karlsruhe</span> (<span>MIKK</span>) Panel.”</span><a href="#fnref236" class="footnote-back">↩︎</a></p></li>
<li id="fn237"><p>Monkol Lek et al., <span>“Analysis of Protein-Coding Genetic Variation in 60,706 Humans,”</span> <em>Nature</em> 536, no. 7616, 7616 (August 2016): 285–91, <a href="https://doi.org/10.1038/nature19057" role="doc-biblioref">https://doi.org/10.1038/nature19057</a>.<a href="#fnref237" class="footnote-back">↩︎</a></p></li>
<li id="fn238"><p><span>“Genomic Variations and Epigenomic Landscape of the <span>Medaka Inbred Kiyosu-Karlsruhe</span> (<span>MIKK</span>) Panel.”</span><a href="#fnref238" class="footnote-back">↩︎</a></p></li>
<li id="fn239"><p>Jose Fernando L’opez-Olmeda et al., <span>“Long Photoperiod Impairs Learning in Male but Not Female Medaka,”</span> <em>iScience</em> 24, no. 7 (July 23, 2021): 102784, <a href="https://doi.org/10.1016/j.isci.2021.102784" role="doc-biblioref">https://doi.org/10.1016/j.isci.2021.102784</a>.<a href="#fnref239" class="footnote-back">↩︎</a></p></li>
<li id="fn240"><p>Spivakov et al., <span>“Genomic and <span>Phenotypic Characterization</span> of a <span>Wild Medaka Population</span>.”</span><a href="#fnref240" class="footnote-back">↩︎</a></p></li>
<li id="fn241"><p>Francisco Romero-Ferrero et al., <span>“Idtracker.ai: Tracking All Individuals in Small or Large Collectives of Unmarked Animals,”</span> <em>Nature Methods</em> 16, no. 2, 2 (February 2019): 179–82, <a href="https://doi.org/10.1038/s41592-018-0295-5" role="doc-biblioref">https://doi.org/10.1038/s41592-018-0295-5</a>.<a href="#fnref241" class="footnote-back">↩︎</a></p></li>
<li id="fn242"><p><em>Hmmlearn/Hmmlearn</em> (2014; repr., <span>hmmlearn</span>, 2022), <a href="https://github.com/hmmlearn/hmmlearn" role="doc-biblioref">https://github.com/hmmlearn/hmmlearn</a>.<a href="#fnref242" class="footnote-back">↩︎</a></p></li>
<li id="fn243"><p>Valeria Carola, Olivier Mirabeau, and Cornelius T. Gross, <span>“Hidden <span>Markov Model Analysis</span> of <span>Maternal Behavior Patterns</span> in <span>Inbred</span> and <span>Reciprocal Hybrid Mice</span>,”</span> <em>PLOS ONE</em> 6, no. 3 (March 8, 2011): e14753, <a href="https://doi.org/10.1371/journal.pone.0014753" role="doc-biblioref">https://doi.org/10.1371/journal.pone.0014753</a>.<a href="#fnref243" class="footnote-back">↩︎</a></p></li>
<li id="fn244"><p>Lucon-Xiccato et al., <span>“Comparison of Anxiety-Like and Social Behaviour in Medaka and Zebrafish.”</span><a href="#fnref244" class="footnote-back">↩︎</a></p></li>
<li id="fn245"><p>Fitzgerald et al., <span>“The <span>Medaka Inbred Kiyosu-Karlsruhe</span> (<span>MIKK</span>) Panel.”</span><a href="#fnref245" class="footnote-back">↩︎</a></p></li>
<li id="fn246"><p>Kim et al., <span>“The Period of the Somite Segmentation Clock Is Sensitive to <span>Notch</span> Activity.”</span><a href="#fnref246" class="footnote-back">↩︎</a></p></li>
<li id="fn247"><p><span>“The Long and Short of It.”</span><a href="#fnref247" class="footnote-back">↩︎</a></p></li>
<li id="fn248"><p>Hubaud and Pourqui’e, <span>“Signalling Dynamics in Vertebrate Segmentation”</span>; Matsuda et al., <span>“Species-Specific Segmentation Clock Periods Are Due to Differential Biochemical Reaction Speeds.”</span><a href="#fnref248" class="footnote-back">↩︎</a></p></li>
<li id="fn249"><p>Gomez et al., <span>“Control of Segment Number in Vertebrate Embryos.”</span><a href="#fnref249" class="footnote-back">↩︎</a></p></li>
<li id="fn250"><p>Gomez et al.<a href="#fnref250" class="footnote-back">↩︎</a></p></li>
<li id="fn251"><p>Falk et al., <span>“Imaging the Onset of Oscillatory Signaling Dynamics During Mouse Embryo Gastrulation.”</span><a href="#fnref251" class="footnote-back">↩︎</a></p></li>
<li id="fn252"><p><span>“Species-Specific Segmentation Clock Periods Are Due to Differential Biochemical Reaction Speeds.”</span><a href="#fnref252" class="footnote-back">↩︎</a></p></li>
<li id="fn253"><p>Seleit, Aulehla, and Paix, <span>“Endogenous Protein Tagging in Medaka Using a Simplified <span>CRISPR</span>/<span>Cas9</span> Knock-in Approach.”</span><a href="#fnref253" class="footnote-back">↩︎</a></p></li>
<li id="fn254"><p>Christoph Schmal, Gregor Mönke, and Adri’an E. Granada, <span>“Analysis of <span>Complex Circadian Time Series Data Using Wavelets</span>,”</span> in <em>Circadian <span>Regulation</span>: <span>Methods</span> and <span>Protocols</span></em>, ed. Guiomar Solanas and Patrick -Simon Welz, Methods in <span>Molecular Biology</span> (<span>New York, NY</span>: <span>Springer US</span>, 2022), 35–54, <a href="https://doi.org/10.1007/978-1-0716-2249-0_3" role="doc-biblioref">https://doi.org/10.1007/978-1-0716-2249-0_3</a>.<a href="#fnref254" class="footnote-back">↩︎</a></p></li>
<li id="fn255"><p>Petr Danecek et al., <span>“Twelve Years of <span>SAMtools</span> and <span>BCFtools</span>,”</span> <em>GigaScience</em> 10, no. 2 (February 1, 2021): giab008, <a href="https://doi.org/10.1093/gigascience/giab008" role="doc-biblioref">https://doi.org/10.1093/gigascience/giab008</a>.<a href="#fnref255" class="footnote-back">↩︎</a></p></li>
<li id="fn256"><p><span>“Picard Toolkit,”</span> <em>Broad Institute, GitHub Repository</em> (<a href="https://broadinstitute.github.io/picard/" class="uri" role="doc-biblioref">https://broadinstitute.github.io/picard/</a>; Broad Institute, 2019).<a href="#fnref256" class="footnote-back">↩︎</a></p></li>
<li id="fn257"><p>Heng Li et al., <span>“The <span>Sequence</span> Alignment/Map (<span>SAM</span>) Format and <span>SAMtools</span>,”</span> <em>Bioinformatics</em> 25, no. 16 (2009): 2078–79.<a href="#fnref257" class="footnote-back">↩︎</a></p></li>
<li id="fn258"><p>McKenna et al., <span>“The <span>Genome Analysis Toolkit</span>”</span>; Mark A. DePristo et al., <span>“A Framework for Variation Discovery and Genotyping Using Next-Generation <span>DNA</span> Sequencing Data,”</span> <em>Nature Genetics</em> 43, no. 5, 5 (May 2011): 491–98, <a href="https://doi.org/10.1038/ng.806" role="doc-biblioref">https://doi.org/10.1038/ng.806</a>; Geraldine A. Van der Auwera and Brian D. O’Connor, <em>Genomics in the Cloud: Using <span>Docker</span>, <span>GATK</span>, and <span>WDL</span> in <span>Terra</span></em> (<span>O’Reilly Media</span>, 2020).<a href="#fnref258" class="footnote-back">↩︎</a></p></li>
<li id="fn259"><p>Ryan Poplin et al., <span>“Scaling Accurate Genetic Variant Discovery to Tens of Thousands of Samples”</span> (<span>bioRxiv</span>, July 24, 2018), <a href="https://doi.org/10.1101/201178" role="doc-biblioref">https://doi.org/10.1101/201178</a>.<a href="#fnref259" class="footnote-back">↩︎</a></p></li>
<li id="fn260"><p><span>“Picard Toolkit.”</span><a href="#fnref260" class="footnote-back">↩︎</a></p></li>
<li id="fn261"><p>Danecek et al., <span>“Twelve Years of <span>SAMtools</span> and <span>BCFtools</span>.”</span><a href="#fnref261" class="footnote-back">↩︎</a></p></li>
<li id="fn262"><p>Zuguang Gu et al., <span>“Circlize Implements and Enhances Circular Visualization in <span>R</span>,”</span> <em>Bioinformatics</em> 30, no. 19 (October 1, 2014): 2811–12, <a href="https://doi.org/10.1093/bioinformatics/btu393" role="doc-biblioref">https://doi.org/10.1093/bioinformatics/btu393</a>.<a href="#fnref262" class="footnote-back">↩︎</a></p></li>
<li id="fn263"><p>Md Vasimuddin et al., <span>“Efficient Architecture-Aware Acceleration of <span>BWA-MEM</span> for Multicore Systems,”</span> in <em>2019 <span>IEEE International Parallel</span> and <span>Distributed Processing Symposium</span> (<span>IPDPS</span>)</em> (<span>IEEE</span>, 2019), 314–24.<a href="#fnref263" class="footnote-back">↩︎</a></p></li>
<li id="fn264"><p><span>“Picard Toolkit.”</span><a href="#fnref264" class="footnote-back">↩︎</a></p></li>
<li id="fn265"><p>Danecek et al., <span>“Twelve Years of <span>SAMtools</span> and <span>BCFtools</span>.”</span><a href="#fnref265" class="footnote-back">↩︎</a></p></li>
<li id="fn266"><p>Ajay Khanna et al., <span>“Bam-Readcount - Rapid Generation of Basepair-Resolution Sequence Metrics,”</span> <em>Journal of Open Source Software</em> 7, no. 69 (January 29, 2022): 3722, <a href="https://doi.org/10.21105/joss.03722" role="doc-biblioref">https://doi.org/10.21105/joss.03722</a>.<a href="#fnref266" class="footnote-back">↩︎</a></p></li>
<li id="fn267"><p><em>Hmmlearn/Hmmlearn</em>.<a href="#fnref267" class="footnote-back">↩︎</a></p></li>
<li id="fn268"><p>Quiterie Haenel et al., <span>“Meta-Analysis of Chromosome-Scale Crossover Rate Variation in Eukaryotes and Its Significance to Evolutionary Genomics,”</span> <em>Molecular Ecology</em> 27, no. 11 (2018): 2477–97, <a href="https://doi.org/10.1111/mec.14699" role="doc-biblioref">https://doi.org/10.1111/mec.14699</a>.<a href="#fnref268" class="footnote-back">↩︎</a></p></li>
<li id="fn269"><p><a href="#fnref269" class="footnote-back">↩︎</a></p></li>
<li id="fn270"><p>Yang et al., <span>“<span>GCTA</span>.”</span><a href="#fnref270" class="footnote-back">↩︎</a></p></li>
<li id="fn271"><p>William McLaren et al., <span>“The <span>Ensembl Variant Effect Predictor</span>,”</span> <em>Genome Biology</em> 17, no. 1 (June 6, 2016): 122, <a href="https://doi.org/10.1186/s13059-016-0974-4" role="doc-biblioref">https://doi.org/10.1186/s13059-016-0974-4</a>.<a href="#fnref271" class="footnote-back">↩︎</a></p></li>
<li id="fn272"><p>McLaren et al.<a href="#fnref272" class="footnote-back">↩︎</a></p></li>
<li id="fn273"><p>Romero-Ferrero et al., <span>“Idtracker.ai.”</span><a href="#fnref273" class="footnote-back">↩︎</a></p></li>
<li id="fn274"><p>Graeme D. Ruxton, <span>“The Unequal Variance t-Test Is an Underused Alternative to <span>Student</span>’s t-Test and the <span>Mann</span>–<span>Whitney U</span> Test,”</span> <em>Behavioral Ecology</em> 17, no. 4 (July 1, 2006): 688–90, <a href="https://doi.org/10.1093/beheco/ark016" role="doc-biblioref">https://doi.org/10.1093/beheco/ark016</a>.<a href="#fnref274" class="footnote-back">↩︎</a></p></li>
<li id="fn275"><p>Although the breeding program began in 2019, I understand from our collaborators that from around mid-2020 the F1 generation was producing poorly. After lengthy investigations, they discovered that the Covid pandemic had caused disruptions to the supply chains of their fish food manufacturer, which had compelled the manufacturer to modify the recipe. This change in the fish food recipe was the cause of the poor breeding. While the issue has since been resolved, it has resulted in a much smaller number of F2 individuals to be produced than was originally anticipated.<a href="#fnref275" class="footnote-back">↩︎</a></p></li>
<li id="fn276"><p>Romero-Ferrero et al., <span>“Idtracker.ai.”</span><a href="#fnref276" class="footnote-back">↩︎</a></p></li>
<li id="fn277"><p>Michael J. Wichura, <span>“Algorithm <span>AS</span> 241: <span>The Percentage Points</span> of the <span>Normal Distribution</span>,”</span> <em>Journal of the Royal Statistical Society. Series C (Applied Statistics)</em> 37, no. 3 (1988): 477–84, <a href="https://doi.org/10.2307/2347330" role="doc-biblioref">https://doi.org/10.2307/2347330</a>.<a href="#fnref277" class="footnote-back">↩︎</a></p></li>
<li id="fn278"><p>Vasimuddin et al., <span>“Efficient Architecture-Aware Acceleration of <span>BWA-MEM</span> for Multicore Systems.”</span><a href="#fnref278" class="footnote-back">↩︎</a></p></li>
<li id="fn279"><p><span>“Picard Toolkit.”</span><a href="#fnref279" class="footnote-back">↩︎</a></p></li>
<li id="fn280"><p>Danecek et al., <span>“Twelve Years of <span>SAMtools</span> and <span>BCFtools</span>.”</span><a href="#fnref280" class="footnote-back">↩︎</a></p></li>
<li id="fn281"><p>Khanna et al., <span>“Bam-Readcount - Rapid Generation of Basepair-Resolution Sequence Metrics.”</span><a href="#fnref281" class="footnote-back">↩︎</a></p></li>
<li id="fn282"><p><em>Hmmlearn/Hmmlearn</em>.<a href="#fnref282" class="footnote-back">↩︎</a></p></li>
<li id="fn283"><p>Yang et al., <span>“<span>GCTA</span>.”</span><a href="#fnref283" class="footnote-back">↩︎</a></p></li>
<li id="fn284"><p>Yang et al.<a href="#fnref284" class="footnote-back">↩︎</a></p></li>
<li id="fn285"><p>Yang et al.<a href="#fnref285" class="footnote-back">↩︎</a></p></li>
<li id="fn286"><p>Yang et al.<a href="#fnref286" class="footnote-back">↩︎</a></p></li>
<li id="fn287"><p>McLaren et al., <span>“The <span>Ensembl Variant Effect Predictor</span>.”</span><a href="#fnref287" class="footnote-back">↩︎</a></p></li>
<li id="fn288"><p>Dirk Junghans et al., <span>“Postsynaptic and Differential Localization to Neuronal Subtypes of Protocadherin Β16 in the Mammalian Central Nervous System,”</span> <em>European Journal of Neuroscience</em> 27, no. 3 (2008): 559–71, <a href="https://doi.org/10.1111/j.1460-9568.2008.06052.x" role="doc-biblioref">https://doi.org/10.1111/j.1460-9568.2008.06052.x</a>; Greg R. Phillips et al., <span>“Γ-<span>Protocadherins Are Targeted</span> to <span>Subsets</span> of <span>Synapses</span> and <span>Intracellular Organelles</span> in <span>Neurons</span>,”</span> <em>Journal of Neuroscience</em> 23, no. 12 (June 15, 2003): 5096–5104, <a href="https://doi.org/10.1523/JNEUROSCI.23-12-05096.2003" role="doc-biblioref">https://doi.org/10.1523/JNEUROSCI.23-12-05096.2003</a>.<a href="#fnref288" class="footnote-back">↩︎</a></p></li>
<li id="fn289"><p>Mouse Genome Database, Judith A Blake et al., <span>“Mouse <span>Genome Database</span> (<span>MGD</span>): <span>Knowledgebase</span> for Mouse–Human Comparative Biology,”</span> <em>Nucleic Acids Research</em> 49, no. D1 (January 8, 2021): D981–87, <a href="https://doi.org/10.1093/nar/gkaa1083" role="doc-biblioref">https://doi.org/10.1093/nar/gkaa1083</a>.<a href="#fnref289" class="footnote-back">↩︎</a></p></li>
<li id="fn290"><p>Rat Genome Database, Jennifer R Smith et al., <span>“The <span>Year</span> of the <span>Rat</span>: <span>The Rat Genome Database</span> at 20: A Multi-Species Knowledgebase and Analysis Platform,”</span> <em>Nucleic Acids Research</em> 48, no. D1 (January 8, 2020): D731–42, <a href="https://doi.org/10.1093/nar/gkz1041" role="doc-biblioref">https://doi.org/10.1093/nar/gkz1041</a>.<a href="#fnref290" class="footnote-back">↩︎</a></p></li>
<li id="fn291"><p>L. Benjamin Hills et al., <span>“Deletions in <span>Grid2</span> Lead to a Recessive Syndrome of Cerebellar Ataxia and Tonic Upgaze in Humans,”</span> <em>Neurology</em> 81, no. 16 (October 15, 2013): 1378–86, <a href="https://doi.org/10.1212/WNL.0b013e3182a841a3" role="doc-biblioref">https://doi.org/10.1212/WNL.0b013e3182a841a3</a>; G. Eda Utine et al., <span>“A <span>Homozygous Deletion</span> in <span>Grid2 Causes</span> a <span>Human Phenotype With Cerebellar Ataxia</span> and <span>Atrophy</span>,”</span> <em>Journal of Child Neurology</em> 28, no. 7 (July 1, 2013): 926–32, <a href="https://doi.org/10.1177/0883073813484967" role="doc-biblioref">https://doi.org/10.1177/0883073813484967</a>.<a href="#fnref291" class="footnote-back">↩︎</a></p></li>
<li id="fn292"><p>Mouse Genome Database, Blake et al., <span>“Mouse <span>Genome Database</span> (<span>MGD</span>).”</span><a href="#fnref292" class="footnote-back">↩︎</a></p></li>
<li id="fn293"><p>St’ephane Jamain et al., <span>“Mutations of the <span class="nocase">X-linked</span> Genes Encoding Neuroligins <span>Nlgn3</span> and <span>Nlgn4</span> Are Associated with Autism,”</span> <em>Nature Genetics</em> 34, no. 1, 1 (May 2003): 27–29, <a href="https://doi.org/10.1038/ng1136" role="doc-biblioref">https://doi.org/10.1038/ng1136</a>.<a href="#fnref293" class="footnote-back">↩︎</a></p></li>
<li id="fn294"><p>Blake et al., <span>“Mouse <span>Genome Database</span> (<span>MGD</span>)”</span>; Smith et al., <span>“The <span>Year</span> of the <span>Rat</span>.”</span><a href="#fnref294" class="footnote-back">↩︎</a></p></li>
<li id="fn295"><p>Multi-Trait Analysis of Genome-wide summary statistics , Patrick Turley et al., <span>“Multi-Trait Analysis of Genome-Wide Association Summary Statistics Using <span>MTAG</span>,”</span> <em>Nature Genetics</em> 50, no. 2, 2 (February 2018): 229–37, <a href="https://doi.org/10.1038/s41588-017-0009-4" role="doc-biblioref">https://doi.org/10.1038/s41588-017-0009-4</a>.<a href="#fnref295" class="footnote-back">↩︎</a></p></li>
<li id="fn296"><p>G. Sella and N. Barton, <span>“Thinking <span>About</span> the <span>Evolution</span> of <span>Complex Traits</span> in the <span>Era</span> of <span>Genome-Wide Association Studies</span>.”</span> <em>Annual Review of Genomics and Human Genetics</em>, 2019, <a href="https://doi.org/10.1146/annurev-genom-083115-022316" role="doc-biblioref">https://doi.org/10.1146/annurev-genom-083115-022316</a>.<a href="#fnref296" class="footnote-back">↩︎</a></p></li>
<li id="fn297"><p><span>“The $11.9 <span>Trillion Global Healthcare Market</span>: <span>Key Opportunities</span> &amp; <span>Strategies</span> (2014-2022) - <span>ResearchAndMarkets</span>.com,”</span> June 25, 2019, <a href="https://www.businesswire.com/news/home/20190625005862/en/The-11.9-Trillion-Global-Healthcare-Market-Key-Opportunities-Strategies-2014-2022---ResearchAndMarkets.com" role="doc-biblioref">https://www.businesswire.com/news/home/20190625005862/en/The-11.9-Trillion-Global-Healthcare-Market-Key-Opportunities-Strategies-2014-2022---ResearchAndMarkets.com</a>.<a href="#fnref297" class="footnote-back">↩︎</a></p></li>
<li id="fn298"><p>Naomi R. Wray, Michael E. Goddard, and Peter M. Visscher, <span>“Prediction of Individual Genetic Risk to Disease from Genome-Wide Association Studies,”</span> <em>Genome Research</em> 17, no. 10 (January 10, 2007): 1520–28, <a href="https://doi.org/10.1101/gr.6665407" role="doc-biblioref">https://doi.org/10.1101/gr.6665407</a>.<a href="#fnref298" class="footnote-back">↩︎</a></p></li>
<li id="fn299"><p>Alicia R. Martin et al., <span>“Clinical Use of Current Polygenic Risk Scores May Exacerbate Health Disparities,”</span> <em>Nature Genetics</em> 51, no. 4, 4 (April 2019): 584–91, <a href="https://doi.org/10.1038/s41588-019-0379-x" role="doc-biblioref">https://doi.org/10.1038/s41588-019-0379-x</a>.<a href="#fnref299" class="footnote-back">↩︎</a></p></li>
<li id="fn300"><p>Loic Yengo et al., <span>“Meta-Analysis of Genome-Wide Association Studies for Height and Body Mass Index in  700000 Individuals of <span>European</span> Ancestry,”</span> <em>Human Molecular Genetics</em> 27, no. 20 (October 15, 2018): 3641–49, <a href="https://doi.org/10.1093/hmg/ddy271" role="doc-biblioref">https://doi.org/10.1093/hmg/ddy271</a>.<a href="#fnref300" class="footnote-back">↩︎</a></p></li>
<li id="fn301"><p>Aysu Okbay et al., <span>“Polygenic Prediction of Educational Attainment Within and Between Families from Genome-Wide Association Analyses in 3 Million Individuals,”</span> <em>Nature Genetics</em>, March 31, 2022, 1–13, <a href="https://doi.org/10.1038/s41588-022-01016-z" role="doc-biblioref">https://doi.org/10.1038/s41588-022-01016-z</a>.<a href="#fnref301" class="footnote-back">↩︎</a></p></li>
<li id="fn302"><p>Paige Maas et al., <span>“Breast <span>Cancer Risk From Modifiable</span> and <span>Nonmodifiable Risk Factors Among White Women</span> in the <span>United States</span>,”</span> <em>JAMA Oncology</em> 2, no. 10 (October 1, 2016): 1295–1302, <a href="https://doi.org/10.1001/jamaoncol.2016.1025" role="doc-biblioref">https://doi.org/10.1001/jamaoncol.2016.1025</a>.<a href="#fnref302" class="footnote-back">↩︎</a></p></li>
<li id="fn303"><p>Fredrick R. Schumacher et al., <span>“Association Analyses of More Than 140,000 Men Identify 63 New Prostate Cancer Susceptibility Loci,”</span> <em>Nature Genetics</em> 50, no. 7, 7 (July 2018): 928–36, <a href="https://doi.org/10.1038/s41588-018-0142-8" role="doc-biblioref">https://doi.org/10.1038/s41588-018-0142-8</a>.<a href="#fnref303" class="footnote-back">↩︎</a></p></li>
<li id="fn304"><p>Seth A. Sharp et al., <span>“Development and <span>Standardization</span> of an <span>Improved Type</span> 1 <span>Diabetes Genetic Risk Score</span> for <span>Use</span> in <span>Newborn Screening</span> and <span>Incident Diagnosis</span>,”</span> <em>Diabetes Care</em> 42, no. 2 (January 11, 2019): 200–207, <a href="https://doi.org/10.2337/dc18-1785" role="doc-biblioref">https://doi.org/10.2337/dc18-1785</a>.<a href="#fnref304" class="footnote-back">↩︎</a></p></li>
<li id="fn305"><p>Amit V. Khera et al., <span>“Genome-Wide Polygenic Scores for Common Diseases Identify Individuals with Risk Equivalent to Monogenic Mutations,”</span> <em>Nature Genetics</em> 50, no. 9, 9 (September 2018): 1219–24, <a href="https://doi.org/10.1038/s41588-018-0183-z" role="doc-biblioref">https://doi.org/10.1038/s41588-018-0183-z</a>; Iftikhar J. Kullo et al., <span>“Incorporating a <span>Genetic Risk Score Into Coronary Heart Disease Risk Estimates</span>,”</span> <em>Circulation</em> 133, no. 12 (March 22, 2016): 1181–88, <a href="https://doi.org/10.1161/CIRCULATIONAHA.115.020109" role="doc-biblioref">https://doi.org/10.1161/CIRCULATIONAHA.115.020109</a>; Pradeep Natarajan et al., <span>“Polygenic <span>Risk Score Identifies Subgroup With Higher Burden</span> of <span>Atherosclerosis</span> and <span>Greater Relative Benefit From Statin Therapy</span> in the <span>Primary Prevention Setting</span>,”</span> <em>Circulation</em> 135, no. 22 (May 30, 2017): 2091–101, <a href="https://doi.org/10.1161/CIRCULATIONAHA.116.024436" role="doc-biblioref">https://doi.org/10.1161/CIRCULATIONAHA.116.024436</a>; Martine Paquette et al., <span>“Polygenic Risk Score Predicts Prevalence of Cardiovascular Disease in Patients with Familial Hypercholesterolemia,”</span> <em>Journal of Clinical Lipidology</em> 11, no. 3 (May 1, 2017): 725–732.e5, <a href="https://doi.org/10.1016/j.jacl.2017.03.019" role="doc-biblioref">https://doi.org/10.1016/j.jacl.2017.03.019</a>; Emmi Tikkanen et al., <span>“Genetic <span>Risk Prediction</span> and a 2-<span>Stage Risk Screening Strategy</span> for <span>Coronary Heart Disease</span>,”</span> <em>Arteriosclerosis, Thrombosis, and Vascular Biology</em> 33, no. 9 (September 2013): 2261–66, <a href="https://doi.org/10.1161/ATVBAHA.112.301120" role="doc-biblioref">https://doi.org/10.1161/ATVBAHA.112.301120</a>; Luanluan Sun et al., <span>“Polygenic Risk Scores in Cardiovascular Risk Prediction: <span>A</span> Cohort Study and Modelling Analyses,”</span> <em>PLOS Medicine</em> 18, no. 1 (January 14, 2021): e1003498, <a href="https://doi.org/10.1371/journal.pmed.1003498" role="doc-biblioref">https://doi.org/10.1371/journal.pmed.1003498</a>.<a href="#fnref305" class="footnote-back">↩︎</a></p></li>
<li id="fn306"><p>L. Duncan et al., <span>“Analysis of Polygenic Risk Score Usage and Performance in Diverse Human Populations,”</span> <em>Nature Communications</em> 10, no. 1, 1 (July 25, 2019): 3328, <a href="https://doi.org/10.1038/s41467-019-11112-0" role="doc-biblioref">https://doi.org/10.1038/s41467-019-11112-0</a>.<a href="#fnref306" class="footnote-back">↩︎</a></p></li>
<li id="fn307"><p>Uffelmann et al., <span>“Genome-Wide Association Studies.”</span><a href="#fnref307" class="footnote-back">↩︎</a></p></li>
<li id="fn308"><p>Uffelmann et al.<a href="#fnref308" class="footnote-back">↩︎</a></p></li>
<li id="fn309"><p>Alice B. Popejoy and Stephanie M. Fullerton, <span>“Genomics Is Failing on Diversity,”</span> <em>Nature</em> 538, no. 7624, 7624 (October 2016): 161–64, <a href="https://doi.org/10.1038/538161a" role="doc-biblioref">https://doi.org/10.1038/538161a</a>.<a href="#fnref309" class="footnote-back">↩︎</a></p></li>
<li id="fn310"><p>Martin et al., <span>“Clinical Use of Current Polygenic Risk Scores May Exacerbate Health Disparities.”</span><a href="#fnref310" class="footnote-back">↩︎</a></p></li>
<li id="fn311"><p>Duncan et al., <span>“Analysis of Polygenic Risk Score Usage and Performance in Diverse Human Populations.”</span><a href="#fnref311" class="footnote-back">↩︎</a></p></li>
<li id="fn312"><p>Alicia R Martin et al., <span>“Human Demographic History Impacts Genetic Risk Prediction Across Diverse Populations,”</span> <em>The American Journal of Human Genetics</em> 100, no. 4 (2017): 635–49, <a href="https://doi.org/10.1016/j.ajhg.2017.03.004" role="doc-biblioref">https://doi.org/10.1016/j.ajhg.2017.03.004</a>; Martin et al., <span>“Clinical Use of Current Polygenic Risk Scores May Exacerbate Health Disparities.”</span><a href="#fnref312" class="footnote-back">↩︎</a></p></li>
<li id="fn313"><p>Martin et al., <span>“Human Demographic History Impacts Genetic Risk Prediction Across Diverse Populations”</span>; Martin et al., <span>“Clinical Use of Current Polygenic Risk Scores May Exacerbate Health Disparities.”</span><a href="#fnref313" class="footnote-back">↩︎</a></p></li>
<li id="fn314"><p>Samuel A. Clark et al., <span>“The Importance of Information on Relatives for the Prediction of Genomic Breeding Values and the Implications for the Makeup of Reference Data Sets in Livestock Breeding Schemes,”</span> <em>Genetics Selection Evolution</em> 44, no. 1 (February 9, 2012): 4, <a href="https://doi.org/10.1186/1297-9686-44-4" role="doc-biblioref">https://doi.org/10.1186/1297-9686-44-4</a>; David Habier et al., <span>“The Impact of Genetic Relationship Information on Genomic Breeding Values in <span>German Holstein</span> Cattle,”</span> <em>Genetics Selection Evolution</em> 42, no. 1 (February 19, 2010): 5, <a href="https://doi.org/10.1186/1297-9686-42-5" role="doc-biblioref">https://doi.org/10.1186/1297-9686-42-5</a>; M. Pszczola et al., <span>“Reliability of Direct Genomic Values for Animals with Different Relationships Within and to the Reference Population,”</span> <em>Journal of Dairy Science</em> 95, no. 1 (January 1, 2012): 389–400, <a href="https://doi.org/10.3168/jds.2011-4338" role="doc-biblioref">https://doi.org/10.3168/jds.2011-4338</a>.<a href="#fnref314" class="footnote-back">↩︎</a></p></li>
<li id="fn315"><p>Duncan et al., <span>“Analysis of Polygenic Risk Score Usage and Performance in Diverse Human Populations”</span>; Martin et al., <span>“Clinical Use of Current Polygenic Risk Scores May Exacerbate Health Disparities.”</span><a href="#fnref315" class="footnote-back">↩︎</a></p></li>
<li id="fn316"><p>Martin et al., <span>“Clinical Use of Current Polygenic Risk Scores May Exacerbate Health Disparities.”</span><a href="#fnref316" class="footnote-back">↩︎</a></p></li>
<li id="fn317"><p>Martin et al.<a href="#fnref317" class="footnote-back">↩︎</a></p></li>
<li id="fn318"><p>Martin et al.<a href="#fnref318" class="footnote-back">↩︎</a></p></li>
<li id="fn319"><p>Martin et al., <span>“Human Demographic History Impacts Genetic Risk Prediction Across Diverse Populations”</span>; Marco Scutari, Ian Mackay, and David Balding, <span>“Using <span>Genetic Distance</span> to <span>Infer</span> the <span>Accuracy</span> of <span>Genomic Prediction</span>,”</span> <em>PLOS Genetics</em> 12, no. 9 (September 2, 2016): e1006288, <a href="https://doi.org/10.1371/journal.pgen.1006288" role="doc-biblioref">https://doi.org/10.1371/journal.pgen.1006288</a>.<a href="#fnref319" class="footnote-back">↩︎</a></p></li>
<li id="fn320"><p>Martin et al., <span>“Clinical Use of Current Polygenic Risk Scores May Exacerbate Health Disparities”</span>; Sini Kerminen et al., <span>“Geographic <span>Variation</span> and <span>Bias</span> in the <span>Polygenic Scores</span> of <span>Complex Diseases</span> and <span>Traits</span> in <span>Finland</span>,”</span> <em>The American Journal of Human Genetics</em> 104, no. 6 (June 6, 2019): 1169–81, <a href="https://doi.org/10.1016/j.ajhg.2019.05.001" role="doc-biblioref">https://doi.org/10.1016/j.ajhg.2019.05.001</a>.<a href="#fnref320" class="footnote-back">↩︎</a></p></li>
<li id="fn321"><p>Martin et al., <span>“Clinical Use of Current Polygenic Risk Scores May Exacerbate Health Disparities.”</span><a href="#fnref321" class="footnote-back">↩︎</a></p></li>
<li id="fn322"><p>Martin et al.; P. C. Sham et al., <span>“Power of <span>Linkage</span> Versus <span>Association Analysis</span> of <span>Quantitative Traits</span>, by <span>Use</span> of <span>Variance-Components Models</span>, for <span>Sibship Data</span>,”</span> <em>The American Journal of Human Genetics</em> 66, no. 5 (May 1, 2000): 1616–30, <a href="https://doi.org/10.1086/302891" role="doc-biblioref">https://doi.org/10.1086/302891</a>.<a href="#fnref322" class="footnote-back">↩︎</a></p></li>
<li id="fn323"><p>Kyoko Watanabe et al., <span>“A Global Overview of Pleiotropy and Genetic Architecture in Complex Traits,”</span> <em>Nature Genetics</em> 51, no. 9, 9 (September 2019): 1339–48, <a href="https://doi.org/10.1038/s41588-019-0481-0" role="doc-biblioref">https://doi.org/10.1038/s41588-019-0481-0</a>.<a href="#fnref323" class="footnote-back">↩︎</a></p></li>
<li id="fn324"><p>Park et al., <span>“Distribution of Allele Frequencies and Effect Sizes and Their Interrelationships for Common Genetic Susceptibility Variants.”</span><a href="#fnref324" class="footnote-back">↩︎</a></p></li>
<li id="fn325"><p>Simon Gravel et al., <span>“Demographic History and Rare Allele Sharing Among Human Populations,”</span> <em>Proceedings of the National Academy of Sciences</em> 108, no. 29 (2011): 11983–88, <a href="https://doi.org/10.1073/pnas.1019276108" role="doc-biblioref">https://doi.org/10.1073/pnas.1019276108</a>; Consortium et al., <span>“A Global Reference for Human Genetic Variation.”</span><a href="#fnref325" class="footnote-back">↩︎</a></p></li>
<li id="fn326"><p>Consortium et al., <span>“A Global Reference for Human Genetic Variation.”</span><a href="#fnref326" class="footnote-back">↩︎</a></p></li>
<li id="fn327"><p>Eleonora Porcu et al., <span>“Genotype <span>Imputation</span> in <span>Genome-Wide Association Studies</span>,”</span> <em>Current Protocols in Human Genetics</em> 78, no. 1 (2013): 1.25.1–14, <a href="https://doi.org/10.1002/0471142905.hg0125s78" role="doc-biblioref">https://doi.org/10.1002/0471142905.hg0125s78</a>.<a href="#fnref327" class="footnote-back">↩︎</a></p></li>
<li id="fn328"><p>Uffelmann et al., <span>“Genome-Wide Association Studies.”</span><a href="#fnref328" class="footnote-back">↩︎</a></p></li>
<li id="fn329"><p>Porcu et al., <span>“Genotype <span>Imputation</span> in <span>Genome-Wide Association Studies</span>.”</span><a href="#fnref329" class="footnote-back">↩︎</a></p></li>
<li id="fn330"><p>Shane McCarthy et al., <span>“A Reference Panel of 64,976 Haplotypes for Genotype Imputation,”</span> <em>Nature Genetics</em> 48, no. 10, 10 (October 2016): 1279–83, <a href="https://doi.org/10.1038/ng.3643" role="doc-biblioref">https://doi.org/10.1038/ng.3643</a>.<a href="#fnref330" class="footnote-back">↩︎</a></p></li>
<li id="fn331"><p>Martin et al., <span>“Clinical Use of Current Polygenic Risk Scores May Exacerbate Health Disparities.”</span><a href="#fnref331" class="footnote-back">↩︎</a></p></li>
<li id="fn332"><p>Katharina Schwarze et al., <span>“Are Whole-Exome and Whole-Genome Sequencing Approaches Cost-Effective? <span>A</span> Systematic Review of the Literature,”</span> <em>Genetics in Medicine</em> 20, no. 10 (October 1, 2018): 1122–30, <a href="https://doi.org/10.1038/gim.2017.247" role="doc-biblioref">https://doi.org/10.1038/gim.2017.247</a>.<a href="#fnref332" class="footnote-back">↩︎</a></p></li>
<li id="fn333"><p>Uffelmann et al., <span>“Genome-Wide Association Studies.”</span><a href="#fnref333" class="footnote-back">↩︎</a></p></li>
<li id="fn334"><p>Anna Fry et al., <span>“Comparison of <span>Sociodemographic</span> and <span>Health-Related Characteristics</span> of <span>UK Biobank Participants With Those</span> of the <span>General Population</span>,”</span> <em>American Journal of Epidemiology</em> 186, no. 9 (November 1, 2017): 1026–34, <a href="https://doi.org/10.1093/aje/kwx246" role="doc-biblioref">https://doi.org/10.1093/aje/kwx246</a>.<a href="#fnref334" class="footnote-back">↩︎</a></p></li>
<li id="fn335"><p>Uffelmann et al., <span>“Genome-Wide Association Studies.”</span><a href="#fnref335" class="footnote-back">↩︎</a></p></li>
<li id="fn336"><p>Jeremy J Berg et al., <span>“Reduced Signal for Polygenic Adaptation of Height in <span>UK Biobank</span>,”</span> ed. Magnus Nordborg et al., <em>eLife</em> 8 (March 21, 2019): e39725, <a href="https://doi.org/10.7554/eLife.39725" role="doc-biblioref">https://doi.org/10.7554/eLife.39725</a>.<a href="#fnref336" class="footnote-back">↩︎</a></p></li>
<li id="fn337"><p>Hakhamanesh Mostafavi et al., <span>“Variable Prediction Accuracy of Polygenic Scores Within an Ancestry Group,”</span> ed. Ruth Loos, Michael B Eisen, and Paul O’Reilly, <em>eLife</em> 9 (January 30, 2020): e48376, <a href="https://doi.org/10.7554/eLife.48376" role="doc-biblioref">https://doi.org/10.7554/eLife.48376</a>; Jonathan K. Pritchard and Molly Przeworski, <span>“Linkage <span>Disequilibrium</span> in <span>Humans</span>: <span>Models</span> and <span>Data</span>,”</span> <em>The American Journal of Human Genetics</em> 69, no. 1 (July 1, 2001): 1–14, <a href="https://doi.org/10.1086/321275" role="doc-biblioref">https://doi.org/10.1086/321275</a>.<a href="#fnref337" class="footnote-back">↩︎</a></p></li>
<li id="fn338"><p>Ying Wang et al., <span>“Theoretical and Empirical Quantification of the Accuracy of Polygenic Scores in Ancestry Divergent Populations,”</span> <em>Nature Communications</em> 11, no. 1, 1 (July 31, 2020): 3865, <a href="https://doi.org/10.1038/s41467-020-17719-y" role="doc-biblioref">https://doi.org/10.1038/s41467-020-17719-y</a>.<a href="#fnref338" class="footnote-back">↩︎</a></p></li>
<li id="fn339"><p>Martin et al., <span>“Clinical Use of Current Polygenic Risk Scores May Exacerbate Health Disparities”</span>; Berg et al., <span>“Reduced Signal for Polygenic Adaptation of Height in <span>UK Biobank</span>.”</span><a href="#fnref339" class="footnote-back">↩︎</a></p></li>
<li id="fn340"><p>Habier et al., <span>“The Impact of Genetic Relationship Information on Genomic Breeding Values in <span>German Holstein</span> Cattle”</span>; Pszczola et al., <span>“Reliability of Direct Genomic Values for Animals with Different Relationships Within and to the Reference Population”</span>; Scutari, Mackay, and Balding, <span>“Using <span>Genetic Distance</span> to <span>Infer</span> the <span>Accuracy</span> of <span>Genomic Prediction</span>”</span>; Wang et al., <span>“Theoretical and Empirical Quantification of the Accuracy of Polygenic Scores in Ancestry Divergent Populations.”</span><a href="#fnref340" class="footnote-back">↩︎</a></p></li>
<li id="fn341"><p><span>“Modeling Linkage Disequilibrium Increases Accuracy of Polygenic Risk Scores,”</span> <em>The American Journal of Human Genetics</em> 97, no. 4 (2015): 576–92.<a href="#fnref341" class="footnote-back">↩︎</a></p></li>
<li id="fn342"><p>Duncan et al., <span>“Analysis of Polygenic Risk Score Usage and Performance in Diverse Human Populations.”</span><a href="#fnref342" class="footnote-back">↩︎</a></p></li>
<li id="fn343"><p>Laure S’egurel and C’eline Bon, <span>“On the <span>Evolution</span> of <span>Lactase Persistence</span> in <span>Humans</span>,”</span> <em>Annual Review of Genomics and Human Genetics</em> 18 (August 1, 2017): 297–319, <a href="https://doi.org/10.1146/annurev-genom-091416-035340" role="doc-biblioref">https://doi.org/10.1146/annurev-genom-091416-035340</a>.<a href="#fnref343" class="footnote-back">↩︎</a></p></li>
<li id="fn344"><p>Kaustubh Adhikari et al., <span>“A <span>GWAS</span> in <span>Latin Americans</span> Highlights the Convergent Evolution of Lighter Skin Pigmentation in <span>Eurasia</span>,”</span> <em>Nature Communications</em> 10, no. 1, 1 (January 21, 2019): 358, <a href="https://doi.org/10.1038/s41467-018-08147-0" role="doc-biblioref">https://doi.org/10.1038/s41467-018-08147-0</a>.<a href="#fnref344" class="footnote-back">↩︎</a></p></li>
<li id="fn345"><p>Arbel Harpak and Molly Przeworski, <span>“The Evolution of Group Differences in Changing Environments,”</span> <em>PLOS Biology</em> 19, no. 1 (January 25, 2021): e3001072, <a href="https://doi.org/10.1371/journal.pbio.3001072" role="doc-biblioref">https://doi.org/10.1371/journal.pbio.3001072</a>.<a href="#fnref345" class="footnote-back">↩︎</a></p></li>
<li id="fn346"><p>Berg et al., <span>“Reduced Signal for Polygenic Adaptation of Height in <span>UK Biobank</span>”</span>; Michael D Edge and Graham Coop, <span>“Reconstructing the <span>History</span> of <span>Polygenic Scores Using Coalescent Trees</span>,”</span> <em>Genetics</em> 211, no. 1 (January 1, 2019): 235–62, <a href="https://doi.org/10.1534/genetics.118.301687" role="doc-biblioref">https://doi.org/10.1534/genetics.118.301687</a>.<a href="#fnref346" class="footnote-back">↩︎</a></p></li>
<li id="fn347"><p>Pierre-Alexis Gros, Herv’e Le Nagard, and Olivier Tenaillon, <span>“The <span>Evolution</span> of <span>Epistasis</span> and <span>Its Links With Genetic Robustness</span>, <span>Complexity</span> and <span>Drift</span> in a <span>Phenotypic Model</span> of <span>Adaptation</span>,”</span> <em>Genetics</em> 182, no. 1 (May 1, 2009): 277–93, <a href="https://doi.org/10.1534/genetics.108.099127" role="doc-biblioref">https://doi.org/10.1534/genetics.108.099127</a>.<a href="#fnref347" class="footnote-back">↩︎</a></p></li>
<li id="fn348"><p><span>“The Nature of Nurture.”</span><a href="#fnref348" class="footnote-back">↩︎</a></p></li>
<li id="fn349"><p>Benjamin W. Domingue et al., <span>“The Social Genome of Friends and Schoolmates in the <span>National Longitudinal Study</span> of <span>Adolescent</span> to <span>Adult Health</span>,”</span> <em>Proceedings of the National Academy of Sciences</em> 115, no. 4 (January 23, 2018): 702–7, <a href="https://doi.org/10.1073/pnas.1711803115" role="doc-biblioref">https://doi.org/10.1073/pnas.1711803115</a>; Baud et al., <span>“Genetic <span>Variation</span> in the <span>Social Environment Contributes</span> to <span>Health</span> and <span>Disease</span>.”</span><a href="#fnref349" class="footnote-back">↩︎</a></p></li>
<li id="fn350"><p>As also explored in Chapters @ref(Pilot-chap) and @ref(MIKK-F2-chap)<a href="#fnref350" class="footnote-back">↩︎</a></p></li>
<li id="fn351"><p>Berg et al., <span>“Reduced Signal for Polygenic Adaptation of Height in <span>UK Biobank</span>.”</span><a href="#fnref351" class="footnote-back">↩︎</a></p></li>
<li id="fn352"><p>Leif Andersson and Michel Georges, <span>“Domestic-Animal Genomics: Deciphering the Genetics of Complex Traits,”</span> <em>Nature Reviews Genetics</em> 5, no. 3, 3 (March 2004): 202–12, <a href="https://doi.org/10.1038/nrg1294" role="doc-biblioref">https://doi.org/10.1038/nrg1294</a>.<a href="#fnref352" class="footnote-back">↩︎</a></p></li>
<li id="fn353"><p>Sewall Wright, <span>“The <span>Genetical Structure</span> of <span>Populations</span>,”</span> <em>Annals of Eugenics</em> 15, no. 1 (1949): 323–54, <a href="https://doi.org/10.1111/j.1469-1809.1949.tb02451.x" role="doc-biblioref">https://doi.org/10.1111/j.1469-1809.1949.tb02451.x</a>.<a href="#fnref353" class="footnote-back">↩︎</a></p></li>
<li id="fn354"><p>Gustave Mal’ecot, <span>“Mathématiques de l’hérédité,”</span> 1948.<a href="#fnref354" class="footnote-back">↩︎</a></p></li>
<li id="fn355"><p>In Wright’s notation, <span class="math inline">\(F\)</span> refers to “fixation” of an allele, and <span class="math inline">\(_{ST}\)</span> refers to “subpopulations within the total population”.<a href="#fnref355" class="footnote-back">↩︎</a></p></li>
<li id="fn356"><p>Kent E. Holsinger and Bruce S. Weir, <span>“Genetics in Geographically Structured Populations: Defining, Estimating and Interpreting <span>FST</span>,”</span> <em>Nature Reviews Genetics</em> 10, no. 9, 9 (September 2009): 639–50, <a href="https://doi.org/10.1038/nrg2611" role="doc-biblioref">https://doi.org/10.1038/nrg2611</a>.<a href="#fnref356" class="footnote-back">↩︎</a></p></li>
<li id="fn357"><p>Holsinger and Weir.<a href="#fnref357" class="footnote-back">↩︎</a></p></li>
<li id="fn358"><p>Joshua M. Akey et al., <span>“Interrogating a <span>High-Density SNP Map</span> for <span>Signatures</span> of <span>Natural Selection</span>,”</span> <em>Genome Research</em> 12, no. 12 (January 12, 2002): 1805–14, <a href="https://doi.org/10.1101/gr.631202" role="doc-biblioref">https://doi.org/10.1101/gr.631202</a>; Feng Guo, Dipak K. Dey, and Kent E. Holsinger, <span>“A <span>Bayesian Hierarchical Model</span> for <span>Analysis</span> of <span>Single-Nucleotide Polymorphisms Diversity</span> in <span>Multilocus</span>, <span>Multipopulation Samples</span>,”</span> <em>Journal of the American Statistical Association</em> 104, no. 485 (March 1, 2009): 142–54, <a href="https://doi.org/10.1198/jasa.2009.0010" role="doc-biblioref">https://doi.org/10.1198/jasa.2009.0010</a>; Bruce S. Weir et al., <span>“Measures of Human Population Structure Show Heterogeneity Among Genomic Regions,”</span> <em>Genome Research</em> 15, no. 11 (January 11, 2005): 1468–76, <a href="https://doi.org/10.1101/gr.4398405" role="doc-biblioref">https://doi.org/10.1101/gr.4398405</a>.<a href="#fnref358" class="footnote-back">↩︎</a></p></li>
<li id="fn359"><p><span>“Index of /Vol1/Ftp/Data_collections/<span>1000g</span>_2504_high_coverage/Working/20201028_3202_phased/,”</span> accessed March 24, 2022, <a href="http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/data_collections/1000G_2504_high_coverage/working/20201028_3202_phased/" role="doc-biblioref">http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/data_collections/1000G_2504_high_coverage/working/20201028_3202_phased/</a>.<a href="#fnref359" class="footnote-back">↩︎</a></p></li>
<li id="fn360"><p>Consortium et al., <span>“A Global Reference for Human Genetic Variation.”</span><a href="#fnref360" class="footnote-back">↩︎</a></p></li>
<li id="fn361"><p>Elizabeth M. Smigielski et al., <span>“<span class="nocase">dbSNP</span>: A Database of Single Nucleotide Polymorphisms,”</span> <em>Nucleic Acids Research</em> 28, no. 1 (January 1, 2000): 352–55, <a href="https://doi.org/10.1093/nar/28.1.352" role="doc-biblioref">https://doi.org/10.1093/nar/28.1.352</a>.<a href="#fnref361" class="footnote-back">↩︎</a></p></li>
<li id="fn362"><p>Chang et al., <span>“Second-Generation <span>PLINK</span>”</span>; Purcell and Chang, <em><span>PLINK</span> 1.9</em>.<a href="#fnref362" class="footnote-back">↩︎</a></p></li>
<li id="fn363"><p>Ramiro Magno and Ana-Teresa Maia, <span>“Gwasrapidd: An <span>R</span> Package to Query, Download and Wrangle <span>GWAS</span> Catalog Data,”</span> <em>Bioinformatics</em> 36, no. 2 (January 15, 2020): 649–50, <a href="https://doi.org/10.1093/bioinformatics/btz605" role="doc-biblioref">https://doi.org/10.1093/bioinformatics/btz605</a>.<a href="#fnref363" class="footnote-back">↩︎</a></p></li>
<li id="fn364"><p>Jacqueline MacArthur et al., <span>“The New <span>NHGRI-EBI Catalog</span> of Published Genome-Wide Association Studies (<span>GWAS Catalog</span>),”</span> <em>Nucleic Acids Research</em> 45, no. D1 (January 4, 2017): D896–901, <a href="https://doi.org/10.1093/nar/gkw1133" role="doc-biblioref">https://doi.org/10.1093/nar/gkw1133</a>.<a href="#fnref364" class="footnote-back">↩︎</a></p></li>
<li id="fn365"><p>Chang et al., <span>“Second-Generation <span>PLINK</span>”</span>; Purcell and Chang, <em><span>PLINK</span> 1.9</em>.<a href="#fnref365" class="footnote-back">↩︎</a></p></li>
<li id="fn366"><p>Orestis A. Panagiotou, John P. A. Ioannidis, and Genome-Wide Significance Project, <span>“What Should the Genome-Wide Significance Threshold Be? <span>Empirical</span> Replication of Borderline Genetic Associations,”</span> <em>International Journal of Epidemiology</em> 41, no. 1 (February 2012): 273–86, <a href="https://doi.org/10.1093/ije/dyr178" role="doc-biblioref">https://doi.org/10.1093/ije/dyr178</a>.<a href="#fnref366" class="footnote-back">↩︎</a></p></li>
<li id="fn367"><p>W. G. Hill and Alan Robertson, <span>“Linkage Disequilibrium in Finite Populations,”</span> <em>Theoretical and Applied Genetics</em> 38, no. 6 (June 1, 1968): 226–31, <a href="https://doi.org/10.1007/BF01245622" role="doc-biblioref">https://doi.org/10.1007/BF01245622</a>.<a href="#fnref367" class="footnote-back">↩︎</a></p></li>
<li id="fn368"><p>B. S. Weir and C. Clark Cockerham, <span>“Estimating <span>F-Statistics</span> for the <span>Analysis</span> of <span>Population Structure</span>,”</span> <em>Evolution</em> 38, no. 6 (1984): 1358–70, <a href="https://doi.org/10.2307/2408641" role="doc-biblioref">https://doi.org/10.2307/2408641</a>.<a href="#fnref368" class="footnote-back">↩︎</a></p></li>
<li id="fn369"><p>Emmanuel Paradis, <span>“Pegas: An <span>R</span> Package for Population Genetics with an Integrated–Modular Approach,”</span> <em>Bioinformatics</em> 26, no. 3 (February 1, 2010): 419–20, <a href="https://doi.org/10.1093/bioinformatics/btp696" role="doc-biblioref">https://doi.org/10.1093/bioinformatics/btp696</a>.<a href="#fnref369" class="footnote-back">↩︎</a></p></li>
<li id="fn370"><p>W. J. Conover, <em>Practical <span>Nonparametric Statistics</span></em> (<span>John Wiley &amp; Sons</span>, 1999), <a href="https://books.google.com?id=n_39DwAAQBAJ" role="doc-biblioref">https://books.google.com?id=n_39DwAAQBAJ</a>.<a href="#fnref370" class="footnote-back">↩︎</a></p></li>
<li id="fn371"><p>Conover; J. Durbin, <em>Distribution <span>Theory</span> for <span>Tests Based</span> on <span>Sample Distribution Function</span></em> (<span>SIAM</span>, 1973), <a href="https://books.google.com?id=zAryCrT1IUYC" role="doc-biblioref">https://books.google.com?id=zAryCrT1IUYC</a>.<a href="#fnref371" class="footnote-back">↩︎</a></p></li>
<li id="fn372"><p><span>“Theoretical and Empirical Quantification of the Accuracy of Polygenic Scores in Ancestry Divergent Populations.”</span><a href="#fnref372" class="footnote-back">↩︎</a></p></li>
<li id="fn373"><p>Jeremy J. Berg and Graham Coop, <span>“A <span>Population Genetic Signal</span> of <span>Polygenic Adaptation</span>,”</span> <em>PLOS Genetics</em> 10, no. 8 (August 7, 2014): e1004412, <a href="https://doi.org/10.1371/journal.pgen.1004412" role="doc-biblioref">https://doi.org/10.1371/journal.pgen.1004412</a>.<a href="#fnref373" class="footnote-back">↩︎</a></p></li>
<li id="fn374"><p><span>“Decoupling of Differentiation Between Traits and Their Underlying Genes in Response to Divergent Selection,”</span> <em>Heredity</em> 108, no. 4, 4 (April 2012): 375–85, <a href="https://doi.org/10.1038/hdy.2011.81" role="doc-biblioref">https://doi.org/10.1038/hdy.2011.81</a>.<a href="#fnref374" class="footnote-back">↩︎</a></p></li>
<li id="fn375"><p>Val’erie Le Corre and Antoine Kremer, <span>“The Genetic Differentiation at Quantitative Trait Loci Under Local Adaptation,”</span> <em>Molecular Ecology</em> 21, no. 7 (2012): 1548–66, <a href="https://doi.org/10.1111/j.1365-294X.2012.05479.x" role="doc-biblioref">https://doi.org/10.1111/j.1365-294X.2012.05479.x</a>.<a href="#fnref375" class="footnote-back">↩︎</a></p></li>
<li id="fn376"><p>Jeremy J. Berg and Graham Coop, <span>“A <span>Population Genetic Signal</span> of <span>Polygenic Adaptation</span>,”</span> <em>PLOS Genetics</em> 10, no. 8 (August 7, 2014): e1004412, <a href="https://doi.org/10.1371/journal.pgen.1004412" role="doc-biblioref">https://doi.org/10.1371/journal.pgen.1004412</a>.<a href="#fnref376" class="footnote-back">↩︎</a></p></li>
<li id="fn377"><p><span>“Detecting <span>Polygenic Adaptation</span> in <span>Admixture Graphs</span>,”</span> <em>Genetics</em> 208, no. 4 (April 1, 2018): 1565–84, <a href="https://doi.org/10.1534/genetics.117.300489" role="doc-biblioref">https://doi.org/10.1534/genetics.117.300489</a>.<a href="#fnref377" class="footnote-back">↩︎</a></p></li>
<li id="fn378"><p>Martin et al., <span>“Human Demographic History Impacts Genetic Risk Prediction Across Diverse Populations”</span>; Alicia R Martin et al., <span>“The Critical Needs and Challenges for Genetic Architecture Studies in <span>Africa</span>,”</span> <em>Current Opinion in Genetics &amp; Development</em>, Genetics of <span>Human Origins</span>, 53 (December 1, 2018): 113–20, <a href="https://doi.org/10.1016/j.gde.2018.08.005" role="doc-biblioref">https://doi.org/10.1016/j.gde.2018.08.005</a>; Stephanie A. Bien et al., <span>“The <span>Future</span> of <span>Genomic Studies Must Be Globally Representative</span>: <span>Perspectives</span> from <span>PAGE</span>,”</span> <em>Annual Review of Genomics and Human Genetics</em> 20, no. 1 (August 31, 2019): 181–200, <a href="https://doi.org/10.1146/annurev-genom-091416-035517" role="doc-biblioref">https://doi.org/10.1146/annurev-genom-091416-035517</a>; Martin et al., <span>“Clinical Use of Current Polygenic Risk Scores May Exacerbate Health Disparities”</span>; Giorgio Sirugo, Scott M. Williams, and Sarah A. Tishkoff, <span>“The <span>Missing Diversity</span> in <span>Human Genetic Studies</span>,”</span> <em>Cell</em> 177, no. 1 (March 21, 2019): 26–31, <a href="https://doi.org/10.1016/j.cell.2019.02.048" role="doc-biblioref">https://doi.org/10.1016/j.cell.2019.02.048</a>; Genevieve L. Wojcik et al., <span>“Genetic Analyses of Diverse Populations Improves Discovery for Complex Traits,”</span> <em>Nature</em> 570, no. 7762, 7762 (June 2019): 514–18, <a href="https://doi.org/10.1038/s41586-019-1310-4" role="doc-biblioref">https://doi.org/10.1038/s41586-019-1310-4</a>.<a href="#fnref378" class="footnote-back">↩︎</a></p></li>
</ol>
</div>
<!--bookdown:body:end-->
  </main>

  <div class="col-md-3 col-lg-2 d-none d-md-block sidebar sidebar-chapter">
    <nav id="toc" data-toggle="toc" aria-label="On this page">
      <h2>On this page</h2>
      <div id="book-on-this-page"></div>

      <div class="book-extra">
        <ul class="list-unstyled">
          <li><a id="book-source" href="#">View source <i class="fab fa-github"></i></a></li>
          <li><a id="book-edit" href="#">Edit this page <i class="fab fa-github"></i></a></li>
        </ul>
      </div>
    </nav>
  </div>

</div>
</div> <!-- .container -->

<footer class="bg-primary text-light mt-5">
  <div class="container"><div class="row">

  <div class="col-12 col-md-6 mt-3">
    <p>"<strong>Japanese courage: a genetic analysis of complex traits in medaka fish and humans</strong>" was written by Ian Brettell. It was last built on 2022-08-31.</p>
  </div>

  <div class="col-12 col-md-6 mt-3">
    <p>This book was built by the <a class="text-light" href="https://bookdown.org">bookdown</a> R package.</p>
  </div>

</div></div>
</footer>


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.9/latest.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
<script type="text/x-mathjax-config">const popovers = document.querySelectorAll('a.footnote-ref[data-toggle="popover"]');
for (let popover of popovers) {
  const div = document.createElement('div');
  div.setAttribute('style', 'position: absolute; top: 0, left:0; width:0, height:0, overflow: hidden; visibility: hidden;');
  div.innerHTML = popover.getAttribute('data-content');

  var has_math = div.querySelector("span.math");
  if (has_math) {
    document.body.appendChild(div);
    MathJax.Hub.Queue(["Typeset", MathJax.Hub, div]);
    MathJax.Hub.Queue(function() {
      popover.setAttribute('data-content', div.innerHTML);
      document.body.removeChild(div);
    })
  }
}
</script>
</body>

</html>
